{{Template:20.109}}

==Introduction==
Based on the numerous applications of PCR, it may seem that the technique has been around forever. In fact it is only a little more than 20 years old. In 1984, Kary Mullis described this technique for amplifying DNA of known or unknown sequence, realizing immediately the significance of his insight. 

[[Image:Be109karymullis.jpg|thumb|left|150px|'''Kary Mullis''']]

''"Dear Thor!" I exclaimed. I had solved the most annoying problems in DNA chemistry in a single lightening bolt. Abundance and distinction. With two oligonucleotides, DNA polymerase, and the four nucleosidetriphosphates I could make as much of a DNA sequence as I wanted and I could make it on a fragment of a specific size that I could distinguish easily. Somehow, I thought, it had to be an illusion. Otherwise it would change DNA chemistry forever. Otherwise it would make me famous. It was too easy. Someone else would have done it and I would surely have heard of it. We would be doing it all the time. What was I failing to see? "Jennifer, wake up. I've thought of something incredible." '' --Kary Mullis from his Nobel lecture; December 8, 1983

To begin today’s experiment, you will “pop” some ura+ yeast from your transformation plates and then amplify the relevant portion of the released genomic DNA. The primers you will use are expected to give an ~800 base pair product only if <i> URA3 </i> is inserted into the gene you have tried to delete. [[Image:Macintosh HD-Users-nkuldell-Desktop-colonyPCRprimers.png|thumb|right| PCR primers to check candidates]] This is accomplished by using a forward primer that anneals upstream of the gene you're deleting and a reverse primer that anneals to a region within the URA3 gene. If the <i>URA3</i> gene has inserted elsewhere in the genome (giving rise to the ura+ phenotype) but the SAGA-subunit remains in place, then you will see no product from this reaction. Note, however, that a negative result from these reactions (i.e. no PCR product) can just as easily be explained as a failed PCR (bad primers, dead enzyme, wrong reaction conditions, etc), and might incorrectly lead you to toss out perfectly correct samples. Only the positive result is meaningful in this experiment and we will not know the result until we run the gel next lab.  
We will remain optimistic and set up overnight cultures of all the candidates you are examining today. In this way you will have cells to examine next time, when you will screen them for phenotypes arising from loss of the SAGA subunit and when you will isolate total RNA from them for microarray analysis. 
 
==Protocols==
===Part 1: Colony PCR===
#Begin by counting the colonies that arose from the transformation experiment you performed last time. Choose three candidates (when possible) to pursue, circling the colonies on the back of the petri dish and clearly labeling them "A," "B," and "C." If you do not have enough colonies, don't despair. You can get some additional ones from another group or from the teaching faculty. Note, however, that by doing so, you may be switching SAGA-subunits and you will have to rethink the work you have planned so far. And if applicable, it will be worth noting the "negative" result in your lab report, since it may indicate that despite the subunit being deemed non-essential, cells may fair less well than wild-type cells and so deletions may be difficult to isolate. 
#You will use the microwave to release the DNA from the yeast. On the tip of a sterile toothpick, pick-up 1/2 of the colony that is candidate "A" and swirl the cells in 20 &mu;l of sterile water in an eppendorf tube. Be sure to label the tube so you know it belongs to your group and which candidate it contains. Repeat with the second and third candidates you've selected. 
#The teaching faculty will have the parent strain, FY2068, streaked out on a petri dish. You should mix 1/2 of a colony from this plate with 20 &mu;l of sterile water in a fourth eppendorf tube. 
#Close the caps to the tubes and microwave them in an eppendorf rack for 15 seconds. 
#Move 2.5 &mu;l of the microwaved mixes, yeast debris and all, into a 200 &mu;l PCR tube. Again label your tubes well (write small!).
#In a new eppendorf tube (normal size this time), prepare a PCR cocktail enough for 5 reactions. Each reaction will include:
#*10 &mu;l  2.5X PCR mastermix
#*0.5 &mu;l  forward primer specific to your gene of interest
#*0.5 &mu;l  reverse primer specific for URA3
#*water to a final volume of 22.5 &mu;l.
#Add 22.5 &mu;l of PCR cocktail to each PCR tube with yeast and leave the tubes on ice until everyone is ready.  
#Cycle the reactions as: 	
#*95&deg; 4 minutes
#*95&deg; 1 minute
#*52&deg; 1 minute
#*72&deg; 2 minute
#*repeat steps 2-4 35 times 
#*72&deg; 10 minutes
#*4&deg; forever

===Part 2: Overnight cultures===
#Using sterile technique, aliquot 2.5 ml of YPD into four sterile test tubes. 
#Label the caps with your team color, and either "parent" "A" "B" or "C."
#Use sterile dowels to move the second 1/2 of each yeast colony from your transformation plates to the media. Swirl the dowel to remove the yeast from the stick and vortex the solution to fully resuspend them. 
#Use a sterile dowel to innoculate the "parent" culture, using the FY2068 plate that the teaching faculty have prepared. 
#Place the tubes on the roller drum in the 30° incubator. They will grow for 24 hours and then be placed in the 4° fridge until next lab. 
#When everyone is ready we will discuss the journal article that was assigned for today.
===Part 3: Journal article discussion===
As part of your assignment for today, you have read the relevant article by [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=15260971&query_hl=3&itool=pubmed_docsum Wu et al, published in Mol Cell in 2004]. There is also an [http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6TCV-4DYW4W3-4&_user=501045&_coverDate=01%2F01%2F2005&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000022659&_version=1&_urlVersion=0&_userid=501045&md5=b1c5e4e2f9e0094a16d7b6ac47413f7f associated review article] that was written to celebrate and highlight this important work on the structure of the SAGA complex. You and your partner will be randomly assigned a portion of the text to describe to the class but be prepared to engage in conversation about other parts of this paper even if you're not "assigned" to the section being discussed.

DONE!

==For next time==
#Prepare a table with your transformation results. Include a legend for the table. This legend should have a number, title and description of the table similar to the tables you might find in a scientific publication. Ideally this homework assignment will be included in the lab report you'll hand in a few weeks from now. 
#You are not required to hand in a revised Materials and Methods section, but consider adding the experiment you performed today to the assignment from last time.

==Reagents list==


{| border="1"  
! primer name  
! primer number
! sequence
! anneals
|-   
| URA3_230to191_rev 
| NO179
| 5' CTG TGC CCT CCA TGG AAA AAT CAG TCA AGA 
| 191-220 (btm strand) after ATG 
|-   
| LEU2_230to191_rev 
| NO194
| 5' CAG CAC CTA ACA AAA CGG CAT CAG CCT TCT TGG AGG CTT
| 191-230 (btm strand) after ATG 
|-   
| HIS3_230to191_rev 
| NO195
| 5' GCG ACC AGC CGG AAT GCT TGG CCA GAG CAT GTA TCA TAT 
| 191-230 (btm strand) after ATG 
|-   
| KAN B primer to check KAN insertions
| FO1310
| 5'CTGCAGCGAGGAGCCGTAAT 
| -174 to -193 (+1 ATG of KanMX ORF)
|-
| KANC primer to check KanMX deletions
| FO1311
| 5'TGATTTTGATGACGAGCGTAAT
| +486 to +507 (+1 ATG of KanMX ORF) 
|-  
| ada1_minus500_fwd 
| NO180
| 5' AGCAGAAAAG CTGACACGTT TCTCCCACTG
| 500bp upstream of ATG
|-   
| ada2_minus530_fwd 
| NO181
| 5' ATCTCGTGGT ACGTACCATT TTTATCTGCT
| 530bp upstream of ATG
|-   
| ada3_minus530_fwd 
| NO182
| 5'ACCCAATCTA GCCATTCTCT CCCAATATAA  
| 530bp upsteam of ATG
|-   
| gcn5_minus530_fwd 
| NO183
| 5'ATTCTAGCCA AGGCAATTAT TGCATACTGC 
| 530bp upstream of ATG 
|-   
| ada5_minus500_rev 
| NO184
| 5' AAGAAGACTATTCTGGGATGCTGCAGATTC 
| 500bp upstream of ATG  
|-   
| spt3_minus490_fwd 
| NO185
| 5' AAC GTC TGC TGT GAT TTG GAG TGA AAT ATC 
| 490 upstream of ATG  
|-   
| spt7_minus550_fwd 
| NO186
| 5' TGTATGACAA TTCATTTGTT TTTGGTGTCG GC
| 550bp upstream of ATG  
|-   
| spt8_minus520_fwd 
| NO187
| 5' CACCTTCTAT TGTGCCACTT ATACGAGCCT
| 520bp upstream of ATG 
|-   
| spt20_minus520_fwd 
| NO188
| 5' CAAGACAGGT ACACCGCGTT AAGAAGACTA
| 520bp upstream of ATG 
|-  
| sgf73_minus510_fwd 
| NO189
| 5' AATCATTCAA GCTTTGGATA AGTACGCCGG
| 510bp upstream of ATG
|-
| sgf29_minus530_fwd 
| NO190
| 5' CGATATCACC AACTTGACCA ACAACTTTGG
| 530bp upstream of ATG
|-
| sgf11_minus530_fwd 
| NO191
| 5' CAACAACAAC GGTATTGTTG GTGCAGTTAT 
| 530bp upstream of ATG
|-
| ubp8_minus470_fwd 
| NO192
| 5' GTCTTAGAGG CAAAGTAGTC ACCTGACTG
| 470bp upstream of ATG
|-
| sus1_minus520_fwd 
| NO193
| 5' AATGGTTAAG ATACCAATGC CGTCTACACC
| 520bp upstream of ATG
|}