{{Template:20.109}}

==Introduction==
Today you will ligate your linearized M13K07 backbone with your epitope insert by mixing the two in the presence of ATP and an enzyme, T4 DNA ligase. During the ligation reactions, hydrogen bonds will form between the overhangs on the fragments, and then the ligase will repair the phosphate backbone, creating a stable circular plasmid.

[[Image:Macintosh HD-Users-nkuldell-Desktop-20109Ligation.png|thumb|left|300px| '''DNA ligation''']]

Hopefully, your ligation reactions will generate your desired construct, namely the M13K07 backbone carrying a myc-tag in either p8 or p3. Alternative ligation products may arise, including a simple reclosing of the singly cut M13K07 backbone. You can distinguish these reclosures from the correct constructs in a later step by looking at the PstI or BamHI sites, since these will be absent from the ligation products you are hoping to make. 

You will transform today's ligation reactions into bacteria. During “transformation,” a single plasmid from the ligation mixture enters a single bacterium and, once inside, replicates and expresses the genes it encodes. One of the genes on the M13K07  genome leads to kanamycin-resistance. Thus, a transformed bacterium will grow on agar medium containing kanamycin. Untransformed cells will die before they can form a colony on the agar surface. 

Most bacteria do not usually exist in a “transformation ready” state, but the bacteria can be made permeable to the plasmid DNA, and cells that are capable of transformation are referred to as “competent.” Competent cells are extremely fragile and should be handled gently, specifically kept cold and not vortexed. The transformation procedure is efficient enough for most lab purposes, with efficiencies as high as 10^9 transformed cells per microgram of DNA, but it is important to realize that even with high efficiency cells only 1 DNA molecule in about 10,000 is successfully transformed. 

[[Image:Be109transformation.jpg|thumb|left|200px|'''Bacterial transformation''']]
<br style="clear:both" />

==Protocols==
When you have a break from the work described below, be sure to examine the plaques you plated last time. Record the number of plaques on each plate, their appearance and any observations or conclusions you can draw.  
===Part 1: Ligation reactions===
For your ligation, you will mix the M13K07 backbone you prepared with the annealed oligonucleotides. As control reactions you will also prepare a “bkb, no ligase” reaction to control for any errant uncut plasmid that might have wandered into your solutions. Additionally you will prepare “bkb only, plus ligase” reaction to assess the frequency of backbone religation. 

The contents of each ligation will be

{| border="1"
|-
|
| '''bkb,<br> no ligase'''
| '''bkb only,<br> plus ligase'''
| '''bkb + insert,<br> plus ligase'''
|-
| '''M13K07 bkb'''
| 4 &mu;l
| 4 &mu;l
| 4 &mu;l
|-
| '''insert'''
| none 
| none
| 5 &mu;l
|-
| '''10X Ligation Buffer^'''
| 1.0 &mu;l
| 1.0 &mu;l
| 1.0 &mu;l
|-
| '''T4 DNA Ligase'''
| none
| 0.5 &mu;l
| 0.5 &mu;l
|-
| '''Water'''
| colspan="3" | To 10 &mu;l not including volume of enzyme
|}

^New England Biolabs sells 10X Ligation buffer to use with their ligase. It contains ATP so must be kept on ice. 
#Assemble the reactions in eppendorf tubes but not in the order listed. Please ask if you are unsure what order to assemble the components. 
#When the ligation mixes are complete, flick the tubes to mix the contents, quick spin them in the microfuge to bring down any droplets, then incubate the reactions at room temperature for at least 10 minutes. 
#Recall that the ligation of your insert into the M13K07 backbone will destroy the PstI or BamHI restriction sites originally used to linearize the genome. Consequently, you can enrich for the proper ligation products by digesting the ligation reactions with the enzyme used to open up the backbone. These are called "kill cuts" since in theory they destroy unwanted ligation products. So while your samples are ligating, you should prepare a "killcut" cocktail according to the table below. By making one mixture that contains water, buffer and enzyme, you can add the same mixture to all reactions, minimizing effects of pipet-error and possible accidental errors, like leaving one component out of one reaction that you remember in all the others.You will prepare enough cocktail to perform 4 killcut reactions, even though you have only three. This will assure you have enough volume for the three reactions. 

{| border="1"
|-
|
| '''volume in each reaction'''
|
| '''x4 <br>= volume in cocktail'''
|-
| '''water'''
| 8 &mu;l
| 
| ? &mu;l
|-
| '''10X NEB buffer'''
| 2 &mu;l
|
| ? &mu;l
|-
| '''Enzyme'''
| 0.25 &mu;l PstI or BamHI
| 
| ? &mu;l
|}
4. Add 10 ul of the killcut cocktail to each of the ligation reactions you've prepared, pipetting up and down to mix. <br>
5. Incubate at 37° for 15 minutes.

===Part 2: Precipitation of DNA===
In this step, salts and buffers are removed from the reactions. DNA is precipitated with salt and ethanol. Yeast tRNA is added to the precipitation as “carrier,” allowing you to better visualize the DNA pellets. The salts are washed from the pellets with 70% ethanol. The tRNA is not removed. Rather it enters the bacteria with the ligation DNA, but is then rapidly degraded. 

#Add 20 &mu;l 3M sodium acetate to each tube.
#Add 5 &mu;l tRNA to each tube. 
#Add 200 &mu;l cold 100% ethanol to each tube and vortex. 
#Spin in a room temperature microfuge 15 minutes. '''Be sure to orient your tubes''' in the microfuge so you know where your pellets should be and balance your tubes with those of another group or with a water-filled eppendorf tube. 
#When the spin is done, locate the pellets in each of the eppendorf tubes. They may appear as solid white dots at the bottom corner of the tube or they may appear to be a diffuse white smear along the wall of the tube. Both are OK. Carefully remove the ethanol from the pellets with your P1000, taking care not to disturb the pellet. You do not have to remove every last drop. 
#Wash the pellets with 500 &mu;l cold 70% ethanol. This is done by dribbling the 70% ethanol along the wall of the eppendorf tube that is opposite your pellet and then removing the 70% ethanol with the same pipet tip. Again you should not disturb the pellet and you do not have to remove every drop of liquid in the tube. If the pellet seems to float away from the wall of the tube, you can re-spin the tubes for 2 minutes with the liquid to adhere the pellets to the wall again. 
#Once you have washed all your pellets, give the tubes a quick spin in the microfuge to bring down any droplets of ethanol that cling to the sides of the tube then remove any remaining liquid from the tubes using your P200. Allow your tubes to dry in the hood for 10 minutes. All the ethanol must be removed or evaporated.
#Resuspend the pellets in 15 &mu;l sterile water. This is done by adding water to the tubes and mixing.  If the DNA does not readily go into solution, it helps to heat the DNA in the 42&deg;C heat block, then vortex and pipet up and down several times. Bring any droplets down to the bottom of the tubes with a quick spin in the microfuge.

===Part 3: Bacterial transformation===

You will perform 4 bacterial transformations, one for each of the ligation mixtures as well as one transformation with 5 ng of plasmid DNA to assess transformation frequency. 

#Prewarm and dry five LB+Kan plates by placing them in the 37&deg;C incubator, media side up with the lids ajar.  You are only doing four transformations, but your truly experimental one will be plated twice.
#Get an aliquot of competent cells from one of the teaching faculty. Keep these cells on ice at all times. There should be at least 200 &mu;l of cells in each tube. Aliquot 50 &mu;l of cells into 4 clean eppendorf tubes. 
#Add DNA to each tube of cells as shown in the table below.
#Flick to mix the contents and leave the tubes on ice for at least 5 minutes. 
#Heat shock the cells at 42&deg;C for 90 seconds exactly. Use your timer. 
#Move the samples to a rack on your bench then use your P1000 to add 0.5 ml of LB media to each eppendorf tube. Invert each tube to mix. 
#Incubate the tubes in the 37&deg;C incubator for at least 30 minutes. This gives the kanamycin-resistance gene some time be expressed in the transformed bacterial cells.
#While you are waiting, label 4 large test tubes with your team color and numbers 1, 2, 3, 4. Mix 10 ml LB with 10 &mu;l of the Kanamycin stock. Aliquot 2.5 ml/tube. This will help the teaching faculty to set up overnight cultures for you for next time. 
#Plate 200 &mu;l of each transformation mix on LB+Kan plates, plating the bkb+insert+ligase transformation '''twice'''.  Note: After dipping the glass spreader in the ethanol jar, then pass it through the flame of the alcohol burner '''just long enough to ignite the ethanol'''.  '''After letting the ethanol burn off, the spreader may still be very hot''', and it is advisable to tap it gently on a portion of the agar plate without cells in order to equilibrate it with the agar (if it sizzles, it's way too hot).  Once the plates are done, wrap them with colored tape and incubate them in the 37&deg;C incubator overnight. One of the teaching faculty will remove them from the incubator and set up liquid cultures for you to use next time.

{| border="1"
|-
!Tube
!Transformation
!Add
|-
|1
|positive control plasmid
|1 &mu;l (5 ng) of M13K07 DNA
|-
|2
|bkb, no ligase 
|5 &mu;l
|-
|3
|bkb, plus ligase
|5 &mu;l 
|-
|4
|bkb+insert, plus ligase
|5 &mu;l 
|}


DONE!

==For next time==
Questions 1 and 2 are theoretical but they should help prepare you to interpret the results you will collect next time. 

#You have purchased some supercompetent bacteria that are provided at a transformation efficiency of 10<sup>9</sup> colony forming units/ug of DNA. You transform the cells with 1 ng of plasmid DNA and plate 1/1000th of the cells. How many colonies do you expect? Next you transform another aliquot of cells, also at 10<sup>9</sup> colony forming units/ug of DNA, with 2 &mu;l of plasmid DNA. You spread 1/100th of the cells and find 50 colonies growing on the plate after 24 hours at 37&deg;C. What is the concentration of plasmid? 
#To illustrate your understanding and the importance of the controls you performed today, please write a one-sentence interpretation for each of the following transformation outcomes. 
#*Outcome 1: no colonies on any plate. 
#*Outcome 2: thousands of colonies on all the plates. 
#*There may be more than one valid interpretation for some of the data (only one answer for each is required for the assignment).
# Next time you will prepare DNA from four transformants and begin to characterize the plasmids in these bacteria. Using the plasmid map you drew last time, plan at least two restriction digests that will confirm the presence of the PCR insert. It will help to read the introduction for the next lab before you complete this part of the assignment. Be sure to predict the size of the fragments you expect when the plasmid does and doesn’t have the PCR insert. Also include reaction conditions such as buffer and temperature. Use the NEB website for details on various enzymes and reaction conditions (http://www.neb.com).
<center>

{| border="1"
! Diagnostic digest 1
| plasmid with insert
| plasmid no insert
|-
| Enzyme(s) used 
| 
|
|-
| Buffer used
| 
|
|-
| Temperature
| 
|
|-
| Predicted fragments		
| 
|
|-
| 
|
|
|-
! Diagnostic digest 2	 	
| 
|
|-
| Enzyme(s) used		
| 
|
|-
| Buffer used		
| 
|
|-
| Temperature		
| 
|
|-
| Predicted fragments		
| 
|
|}
</center>
<br>
4. Based on the results of your plaque assay, what is the titer of each stock solution of phage? Please show your work. If the plaques appeared different, please consider how the phage genomes differ (M13K07 is a "helper phage" while E4 is identical the the M13 genome except four glutamic acids are presented on the N-terminus of the p8 protein) and suggest how these differences might account for the differences in plaque morphology. <br><br>
5. Read the article by Chan, Kosuri and Endy. "Refactoring bacteriophage T7" ''Nature/EMBO Molecular Systems Biology'' 13 September 2005 [[doi:10.1038/msb4100025]] and [http://www.nature.com/msb/journal/v1/n1/full/msb4100028.html News & Views]. Come prepared to discuss this paper during lab next time. To guide your reading and test your understanding, try to answer the following questions (<b>note: these questions are just to guide your reading and the answers do not have to be turned in</b>):
*'''from the Introduction:'''
**What is "refactoring" and what makes is T7 an attractive candidate for this approach?
**What experimental techniques give us "compenent level" understanding, i.e. allow us to attribute particular functions to particular sequences in the genome? How completely can "component level" understanding provide "system level" understanding?  
**How predictive have computational and quantitative models for T7 behavior proven to be? What's important about predicting behavior?  
*'''from the Results:'''
**What design principles were the authors pursuing? How well do these map to our class effort at M13 re-design? 
**Was the entire T7 genome refactored? 
**What techniques were used to verify the refactoring? What techniques were used to evaluate it? 
*'''from the Discussion:'''
**How do the authors' findings extend knowledge of T7 biology? 
**Does T7.1 resolve disagreements between model-based behavior predictions and those that are observed though experimental approaches? 
**Could nature have produced the T7 phage that now exists in the Endy lab in Building 68?
**What's next for this phage?
6. Decide if you want to be a "Discoverer" or a "Designer" for the M13 refactoring project.  Discoverers will work to identify, define, refine, and perhaps erase regions of the wild-type M13 genome.  Designers will work to engineer new features into the genome in support of  applications (e.g., what Professor Belcher discussed in class 15 Feb).  Note: it would be great to have both types of folks (i.e., discoverers & designers).  Please state what type of work you want to do [[20.109:Module 1:RefactorM13 | here]].<br><br>
7. If you have not done so already, register for an account on the [http://parts.mit.edu/ Registry of Standard Biological Parts]; join the 20.109 spring '07 group.<br><br>
8. On the Parts Registry, annotate BBa_M1307 by adding two "features" namely a genetic element of your choosing (an ORF, promoter, or RBS) and one single-cutter restriction endonuclease site.  You'll have to choose a genetic element and restriction site that hasn't been added by one of your classmates. Adding a feature requires that you 
** login to the registry
** pull up BBa_M1307 using the search window on the lefthand navigation bar
** click on the "Hard Information" link that is in the lefthand navigation bar
** click on the edit <b>tab</b> that is at the top of the page (near the "article" and "discussion" tabs)
** click on the edit <b>link</b> that is on the right side of the page above the "sequence and features" box 
** click the "add a feature" link that is near the features box
** fill in the "type" "label" "start" and "stop" details for the feature you're adding
Note what you added to the annotation next to your name [[20.109:Module 1:RefactorM13 | here]]. Hint!  You can find a table listing the Registry Part numbers for M13 genes, promoters, and RBSs [http://openwetware.org/wiki/20.109%28S07%29:_Western_analysis#Part_3:_M13.1 here].  Use the part numbers to find the DNA sequence information of the genetic elements and then use this information to annotate the full M13K07 sequence.  Hint #2!  Remember that you can find a listing of single cutters [http://openwetware.org/images/9/91/M13KO7_singlecutters.txt here].

==Reagents list==

*5 ng M13K07 DNA
*T4 DNA Ligase Buffer (1X)
**50 mM Tris-HCl
**10 mM MgCl2
**10 mM DTT
**1 mM ATP
**25 &mu;g/ml BSA
*LB
**1% Tryptone
**0.5% Yeast Extract
**1% NaCl
*LB+kan plates
**LB with 2% agar and 25 &mu;g/ml Kanamycin