{{Template:20.109(F07)}}
==Introduction==
[[Image:Macintosh HD-Users-nkuldell-Desktop-divideandconquer.jpeg|thumb|left|400 px| image credit: Justin Lo, MIT Course 20]] 

With your molecular manipulations to add unique restriction sites to the gene encoding the phage p3, you are both testing our existing knowledge of the system and extending it. Today you will perform experiments to see if the manipulation you have carried out is tolerated by the phage, specifically, if the modified protein is detectable in bacteria that are infected by your manipulated phage and if the phage life cycle is altered. These will be determined through Western analysis, looking for proteins that react to an anti-p3 antibody, and by plaque assay, with which you are already familiar.

Western analysis begins with SDS-PAGE, an acronym for "Sodium Dodecyl Sulfate-Polyacrylamide Electrophoresis." The last word, "electrophoresis," is something you're already familiar with from your DNA work. SDS-PAGE uses charge to separate proteins rather than DNA, but is not identical to agarose gel electrophoresis since proteins are chemically and physically so different from DNA. 
*Unlike DNA, proteins are not uniformly charged. Consequently, proteins are coated with a detergent ("SDS"---same thing that's in many shampoos) to allow them to be drawn through the electrophoresis matrix according to their size. 
*Proteins are too big to move through most agarose gels, so a different matrix ("polyacrylamide" is used). Polyacrylamide is more hazardous than agarose (check out the [http://physchem.ox.ac.uk/MSDS/AC/acrylamide.html MSDS]) so extra care must be used when handling these gels.
*Proteins come in a wide wide variety of shapes and exist in complexes with other proteins. In order for each protein to move through the gel according to its size, the higher order folds are removed by boiling the proteins in the presence of a reducing agent. 

The electrophoresis itself you're already familiar with. It may feel different because the gels are much thinner and are run vertically rather than horizontally. After the run is done, you'll transfer the proteins out of the gel (which is why the gel is so thin) onto a charged membrane which will be probed with antibody next time. 


==Protocol==
===Part 1: SDS-PAGE===

Each group will run a lane of molecular weight markers, a lane with a positive control for the Western (e.g. a bacterial strain infected with M13K07 or an aliquot of phage), and two lanes with bacterial cells expressing the M13KO7 genome you've modified. 

The bacterial strains will be lysed to release all their proteins. 

#Retrieve the bacterial cultures carrying the modified M13KO7 genomes that have been stored at 4Â°C since last time. You should also get a bacterial sample that will serve as a positive control for the anti-p3 antibody. This sample carries the M13K07 genome. 
#To compare intensities between lanes on the protein gel it's necessary that equal numbers of cells be loaded into each well. Make a 1:10 dilution of the two candidate strains that you'll follow-up and of the M13K07 infected control (50 ul cells plus 450 ul water). Transfer each to a cuvette and use the spectrophotometer to measure the density of the samples at a wavelength of 600 nm. If you do not remember how to use the spectrophotometer, please ask the teaching faculty to help. 
#The cells will scatter light in proportion to the density, at least within a certain range of densities, and the measurement is called an "OD reading", for optical density. The value tells you something about the number of cells in a millileter of liquid. For example a reading of 0.7 says the sample has 0.7 OD units of cells / ml. If you wanted to collect the number of cells equivalent to 1 OD unit, then you would have to collect 1/0.7 = 1.4 ml of that sample to get 1 OD's worth of cells. Calculate the volume of your cells needed to give 1 OD. Don't forget that your spectrophotometric reading is for a 1:10 dilution of the original (undiluted) samples. 
#Move the calculated volume of cells to well-labeled eppendorf tubes, and spin the tubes in a microfuge for 1 minute to pellet the bacteria. 
#Move the supernatant of each to a new, labelled eppendorf tube. Later today, you will test the supernatant of these samples for phage using the plaque assay and by counting the plaques that develop, know the titer of the phage in the supernatant.  
#Resuspend the bacterial pellets in 100 ul of 1X sample buffer. Sample Buffer contains glycerol to help your samples sink into the wells of the gel, SDS to coat amino acids with negative charge, BME to reduce disulfide bonds, and bromophenol blue to track the migration of the smallest proteins through the gel. <b>Wear gloves when using sample buffer or your hands will get blue and smelly. </b>
#Put lid locks on the eppendorf tubes and boil for 5 minutes. 
#Put on gloves. Load the indicated volumes of each sample onto your acrylamide gel in the order below. Once you have loaded a sample from one tube, move it to a different row in your eppendorf tube rack. This will help you keep track of which samples you have loaded. 
<center>
{| border="1"
! Lane 
! Sample 
! Volume to load
|-
| 1 
|[http://www.bio-rad.com/B2B/BioRad/product/br_category.jsp?BV_SessionID=@@@@2015247057.1166812647@@@@&BV_EngineID=cccfaddjjkijiikcfngcfkmdhkkdfll.0&categoryPath=%2fCatalogs%2fLife+Science+Research%2fElectrophoresis%2fProtein+Electrophoresis%2fElectrophoresis+Reagents+and+Standards%2fProtein+Standards+for+Electrophoresis%2fPrecision+Plus+Protein+Standards&divName=Corporate&loggedIn=false&lang=English&country=HQ&catLevel=7&catOID=-17283&isPA=false&serviceLevel=Lit+Request&searchStr=kaleidoscope+markers&cateName=Ordering+Information "Kaleidoscope" protein molecular weight standards] 
| 5 ul
|-
| 2 
| M13K07 positive control strain
| 25 ul
|-
| 3
| candidate 1
| 25 ul
|-
| 4 
| candidate 2 
| 25 ul
|-
| 5
| empty
| 
|-
| 6 
|[http://www.bio-rad.com/B2B/BioRad/product/br_category.jsp?BV_SessionID=@@@@2015247057.1166812647@@@@&BV_EngineID=cccfaddjjkijiikcfngcfkmdhkkdfll.0&categoryPath=%2fCatalogs%2fLife+Science+Research%2fElectrophoresis%2fProtein+Electrophoresis%2fElectrophoresis+Reagents+and+Standards%2fProtein+Standards+for+Electrophoresis%2fPrecision+Plus+Protein+Standards&divName=Corporate&loggedIn=false&lang=English&country=HQ&catLevel=7&catOID=-17283&isPA=false&serviceLevel=Lit+Request&searchStr=kaleidoscope+markers&cateName=Ordering+Information "Kaleidoscope" protein molecular weight standards]
| 5 ul
|-
| 7
| M13KO7 positive control strain
| 25 ul
|-
| 8
| candidate 1  
| 25 ul
|-
| 9 
| candidate 2
| 25 ul
|-
| 10
| empty
| 
|}
</center>

9. Once all the samples are loaded, turn on the power and run the gel at 200 V. The molecular weight standards are pre-stained and will separate as the gel runs. The gel should take approximately one hour to run. During that hour, you should work on part two of today's protocol.<br>
10. Wearing gloves, disassemble the electrophoresis chamber.<br>
11. Blot the gel to nitrocellulose as follows:
*Place the gray side of the transfer cassette in a tupperware container which is half full of transfer buffer. The transfer cassette is color-coded so the gray side should end up facing the cathode (black electrode) and the clear side facing the anode (red). 
*Place a ScotchBrite pad on the gray side of the cassette.
*Place 1 piece of filter paper on top of the ScotchBrite pad.
*Place your gel on top of the filter paper.
*Place a piece of nitrocellulose filter on top of the gel.  The nitrocellulose filter is white and can be found between the blue protective paper sheets. Wear gloves when handling the nitrocellulose to avoid transferring proteins from your fingers to the filter. 
*Gently but thoroughly press out any air bubbles caught between the gel and the nitrocellulose.  
*Place another piece of filter paper on top of the nitrocellulose.
*Place a second ScotchBrite pad on top of the filter paper.
*Close the cassette then push the clasp down and slide it along the top to hold it shut.
*Place the transfer cassette into the blotting tank so that the clear side faces the red pole and the gray side faces the black pole.<br>
12. Two blots can be run in each tank. When both are in place, insert the ice compartment into the tank. Fill the tank with buffer. Be sure the stir bar is able to circulate the buffer. Connect the power supply and transfer at 100 V for one hour. You can use this time to complete part 2 of today's protocol.<br>
13. After an hour, turn off the current, disconnect the tank from the power supply and remove the holders. Retrieve the nitrocellulose filter and confirm that the pre-stained markers have transferred from the gel to the blot. Move the blot to blocking buffer (TBS-T +5% milk) and store it in the refrigerator until next time.

===Part 2: Plaque assay===
You will use the plaque assay to titer the supernatant from the candidate strains you are following. This will allow you to determine (if/how many) phage have been secreted by your cells carrying the modified p3 phage protein. You will include a positive and negative control for the assay as well. 

#Titer the supernatant from the candidates you're running on the SDS-PAGE, examining dilutions that are 10^-8, 10^-10, and 10^-12 of the starting concentration. 
#As a negative control, you should set up a plate with bacteria and top agar but without any phage.
#As a positive control, you should set up a plate with bacteria and top agar and 10 ul of 10^-8 dilution of the supernatant from the M13K07-infected cells you're running on the SDS-PAGE. 
#Consult your lab notebook or the protocol from [[20.109(F07): Agarose gel electrophoresis#Protocols| day 2 of this module]] if you need reminding about how to perform the plaque assay.


DONE!

==For next time==
#Remind yourself what size you expect to see for p3. Determine which of the [http://www.bio-rad.com/B2B/BioRad/product/br_category.jsp?BV_SessionID=@@@@2015247057.1166812647@@@@&BV_EngineID=cccfaddjjkijiikcfngcfkmdhkkdfll.0&categoryPath=%2fCatalogs%2fLife+Science+Research%2fElectrophoresis%2fProtein+Electrophoresis%2fElectrophoresis+Reagents+and+Standards%2fProtein+Standards+for+Electrophoresis%2fPrecision+Plus+Protein+Standards&divName=Corporate&loggedIn=false&lang=English&country=HQ&catLevel=7&catOID=-17283&isPA=false&serviceLevel=Lit+Request&searchStr=kaleidoscope+markers&cateName=Ordering+Information kaleidoscope markers] you expect to see nearby. 
#Your first draft of parts 1, 2, and 4 of your portfolio will be due when you arrive in lab next time. Detailed information about this assignment can be found on [[20.109(F07): Genome engineering assessment| this page]]. Send your draft prior to arriving in lab, emailing it to nkuldell AT mit DOT edu, astachow AT mit DOT edu, endy AT mit DOT edu and nlerner AT mit DOT edu. Please follow the document naming conventions [[20.109(F07): Genome engineering assessment| as noted here]].

==Reagents list==
*Transfer Buffer
**25 mM Tris
**192 mM glycine
**20% v/v methanol