{{Template:20.109(F07)}}
==Introduction==
[[Image:Abalone.jpg|thumb|left|200 px| Abalone shell]]
[[Image:Diatoms.jpg|thumb|right|225 px| diatoms]]<br>
The accomplishments of the natural world can inspire us to great engineering feats. Biomineralization is one particularly impressive trick nature pulls off. Vertebrates, invertebrates and plants all have ways to precisely position inorganic substrates into crystalline order. For example, calcium carbonate will form unstructured dust in the absence of genetically-programmed organizers, but the same material can be made into the hard and luminous shells of sea creatures. Similarly, diatoms organize silicon dioxide into intricate patterns that manufacturers of electronic components can’t begin to approach. In one more instance, bacteria align iron inside their cytoplasm to form magnetic rods on the submicron scale. Even more amazing, these feats are accomplished without harsh chemicals, without extreme temperatures, and without noxious wastes that poison the nests of the organisms themselves. Humans have a lot to learn from nature’s successes. In the upcoming weeks we’ll use a biological character from earlier in the term, namely M13, and rely on the self-assembling coat to template a crystal of iridium oxide. The interaction of iridium with the p8 protein of the phage coat yields nanoscale-particles with useful optic properties, as we’ll see (literally). 

[[Image:Phagefusions.png|thumb| 150 px| phage display]]Thinking back to the work we've done so far this term, there seems to be nothing in our initial studies of M13 that would lead us to believe that the phage could interact with inorganic materials. It's unlikely this phage would have run into lots of iridium in its natural environment. It seems even less likely that the phage might, by happy coincidence, bind this metal. It's in fact through some clever human intervention, namely selection through phage display, that a phage with such an unusual property was found. [http://www.dyax.com/discovery/phagedisplay.html Phage display] has been used for decades as a tool for discovery. This technique exploits natural selection and identifies functional peptide sequences that can be fused to the phage coat. Most often it’s the p3 protein that’s used for phage display because, despite the limited number of displayed peptides per phage (on the order of 5), there is enough flexibility to accommodate peptides of [http://www.neb.com/nebecomm/products/productE8101.asp  20 to 30 amino acids]. The other protein used for phage display, p8, is presented at much higher copy number per phage (on the order of 2700) but it has limited flexibilty. The semi-crystalline packing of p8 on the phage coat restricts fusions to only 4 to 6 neutral or negatively charged amino acids. For scientists who can tolerate a mix of p8 proteins on the phage coat for their work, there are phage-display variations that mix and match fusion and wild-type proteins on a phage coat, but for those who want phage of a particular form, the options are limited. 

Nonetheless, peptides with remarkably diverse functions have been isolated with phage display. Once the fusion site is chosen, a library of sequences encoding random peptides can be synthesized and cloned. In this way a pool of phage, each with different fusions, can be made. Finally, the phage pool can be screened for interesting behaviors or properties. Peptide-fusion proteins to p8 or p3 that include stop codons or intolerable sequences are largely lost from the population after the first round of “panning.” Other phage that can bind to a substance of interest or show enzyme activity or glow green…, these remain and can be directly isolated from the pool or further enriched by a second, third, fourth round of panning. Ultimately anywhere from 10 to 1000 candidate sequences may remain from a starting pool of 1 billion [http://www.dyax.com/discovery/phagedisplay.html]. 

Despite phage display techniques being available for more than a generation, this tool been applied only recently to the search for novel materials. Largely it’s been Angela Belcher and her lab who highlighted and then demonstrated the usefulness of this search tool for finding peptides that interact with materials to meet human needs. Phage that can bind to cobalt oxide, gold, iridium and indium tin oxide are all in-hand thanks to their work (e.g see reference [http://www.sciencemag.org/cgi/content/abstract/312/5775/885]). Today you will harvest and titer a phage that can bind to iridium since these can be used to build nanowires next time.

==Protocols==
In advance of this lab, a bacterial host ([[E. coli genotypes| XL1-blue]]) was infected with the modified M13 phage called "3-12." Though the precise details of the 3-12 modification are still unpublished, it was isolated from a screen for changes in the M13 p8 that enable the phage to bind a metal (iridium). Today you will harvest the phage from the supernatant of the infected bacterial culture and then titer it. 
===Part 1: Phage purification===
#Spin 2 eppendorf tubes with infected cells in a room temperature microfuge, 1 minute at full speed.
#Remove the supernatant to clean fresh eppendorf tubes.
#Use your P1000 to measure the volume, then add a 1/6th volume of 20% PEG-8000/2.5M NaCl solution.
#Invert to mix then incubate on ice 60 minutes. 
#Spin in a room temperature microfuge, 15 minutes at full speed. A very tiny white pellet may be visible...these are your precipitated phage. If you can't see a pellet keep going, but be aware of where the pellet you can't see is in the tube and don't scrape a tip against it or you will accidentally remove it. 
#Remove the supernatant by aspiration (carefully so as not to disturb the pellet) or using your P1000. 
#Spin the tubes 1 minute more to pellet any droplets stuck to the walls of the tube and use your P200 to remove the last drops of liquid from the pellet. 
#Resuspend the pellet in 100 ul sterile H2O, pool the volumes of both eppendorf tubes into one and add a 1/6th volume of 20% PEG-8000/2.5M NaCl solution.
#Invert to mix.  Then incubate on ice for 15 minutes. 
#Balance your sample against that of another group, or against an eppendorf with water. Spin in a room temperature microfuge, 10 minutes at full speed. 
#Aspirate the supernatant and resuspend the pellet (if you can see it) in 100 ul sterile H2O. 
#If the solution looks at all cloudy, spin in a room temperature microfuge for 1 minute more and move supernatant with the phage to a new eppendorf tube.

===Part 2: Preparation of Ir(III)Cl3=== 
[[Image:Macintosh HD-Users-nkuldell-Desktop-20.109(F07)-20.109(F07) Mod3 ECD-Mod3F07 wiki images-Ir(III)Cl.png|thumb|Ir(III)Cl3]] 
A solution of 25mM IrCl3 was prepared by dissolving 1.7 g in 200 ml H20. You will adjust the pH of an aliquot to somewhere between 7.0 and 7.5, and then you will store the solution in the dark until next time. This "rest" period between pH-ing and usefulness seems necessary, though the reasons for it are unclear. One idea is that the water structures around the iridium ions in a way that's important for their deposition on the phage, a process you'll work on next time. 
# Move ~4 ml of the 25 mM IrCl3 solution from the stock bottle to a 15 ml falcon tube. The volume you pH can be approximate, estimating the amount using the markings on the side of the tube. 
# Measure the starting pH by dribbling a small amount on pH paper. A pasteur pipet and bulb work well to transfer the liquid to the paper, or your P200 set to 50 ul. Do not dip the paper into the solution. 
# Add one drop of 1N NaOH to your aliquot of IrCl3. Invert the tube to mix and check pH again using pH paper. 
# If you're still well below pH 7 then add another drop of 1N NaOH. If you're close, dilute the 1N to 0.1N with water and add a drop of the dilute base. If you've overshot the range and are at 10 or above, then add 500 ul of un-pH'd IrCl3 to your solution. 
# Recheck with pH paper. 
# When the pH paper tells you that the pH is close, use the pH meter to know precisely what the pH of your solution is. Add this pH value to the table of [[20.109(F07): ECD experimental variations| experimental variables.]] If you've forgotten how to use the pH meter, ask one of the teaching faculty.
# Store your pH'd solution of IrCl3 wrapped in foil in the dark until next lab.

===Part 3: Titering phage===
This protocol is identical to one you performed [[20.109(F07): Agarose gel electrophoresis| earlier in the term]]. You should dilute the phage you've prepared today so the final concentrations of phage are 10^6, 10^8, and 10^10th less concentrated than your stock. In addition to 10 ul of these three dilutions, you should include a "no phage" sample.

Before you leave you should place your petri dishes in the 37° incubator and give the phage stock to the teaching faculty, who will store it at 4° for you until next time.    

DONE!

==For next time==
# [[20.109(F07): Module 3 oral presentations| The primary assignment]] for this experimental module will be for you to develop a research proposal and present your idea to the class. For next time, please describe five recent findings that might define an interesting research question. You should hand in a 3-5 sentence description of each topic and list the reference that led you to each item. The topics you pick can be related to any aspect of the class, i.e. genome, expression, or bio-material engineering. During lab next time, you and your partner will review the topics and narrow your choices, identifying one or perhaps two topics for further research.
# Ultimately you'll be using phage nanowires to make an electrochromic device. The application for these devices is in displays of all sorts. For next time you should describe one of the following types of displays: LCD, plasma screen, rear projection, or DLP. Include a description of how they work (in two or three sentences) and draw a figure to illustrate the workings of the display in question. A hand-drawn figure is fine, or a computer-drawn version, but not something that's just cut and paste from a webpage. Include the references you used to learn about the display.

==Reagents list==
25 mM IrCl3
* 1.7 g dissolved in 200 ml H2O
1N NaOH