<div style="padding: 10px; width: 7.5in; border: 3px solid #000000;">

{|
|- valign="top"
| __TOC__
| [[Image:DNA Melting Apparatus Picture|thumb|center|top|500px|DNA Melting Apparatus]]
|}

==Introduction== 

[[Image:Example Melting Curve|thumb|right|250px|Example DNA melting curves showing the effect of varying ionic strength. The data has been filtered to reduce noise.]]
[[Image:Example Melting Curve Derivative|thumb|right|250px|Differentiating the melting curve simplifies finding T<sub>m</sub>.]]
In this lab, you will measure the melting temperature of several DNA samples to determine the effect of sequence length, ionic strength and complementarity. A common application of this technique exploits the length dependence of DNA melting temperatures to examine PCR products in order to determine whether a desired sequence was successfully amplified.  

The measurement technique utilizes a fluorescent dye that binds preferentially to double stranded DNA (dsDNA). This characteristic of the dye allows the relative concentration of dsDNA to be determined by measuring the intensity of fluorescent light given off by an excited sample. 

The DNA melting apparatus you will construct consists of four major subsystems: excitation, fluorescence measurement, temperature sensing, and data acquisition. You will build these subsystems out of an LED, a photodiode, a resistance temperature detector (RTD), and a PC data acquisition system.

The goal of your time in the lab will be to measure fluorescence intensity versus temperature for each of the samples over a range of about 90°C to room temperature. This will provide a basis for estimating the melting temperature, T<sub>m</sub> of each sample. (T<sub>m</sub> is defined as the temperature where half of the DNA in the sample remains hybridized.)

Three of the samples will be unknown. All the unknowns will have the same length, but different degrees of complementarity: complete match, single mismatch, and complete mismatch. Using the data you gather, you will attempt to identify these three samples.

===Overview of the apparatus===

In most DNA melting apparatuses, the temperature of the sample is ramped up at a controlled rate and the concentration of dsDNA recorded. In our homebrew setup, however, we will first heat up the sample in a bath. That way, natural cooling will provide the range of temperature conditions needed. As the sample cools, a PC data acquisition card will record the photodiode and RTD outputs over time. During data analysis, you will convert these voltages to temperature and relative dsDNA concentration. The melting temperature, T<sub>m</sub> can be estimated from a graph of this data or its derivative.

[http://www.sigmaaldrich.com/sigma-aldrich/datasheet/s9430dat.pdf SYBR Green I] is most sensitive to blue light with a wavelength of 498 nm. The dye emits green light with a wavelength of 522 nm. You can easily observe this – a room-temperature sample of dsDNA and SYBR green looks yellow from the combination of blue excitation and green fluorescence. At higher temperatures, when there is no dsDNA to bind to, the sample will appear blue or clear. 

<center>[[Image:DNA Melting Block Diagram.jpg|frame|The DNA melting apparatus includes excitation, fluorescence measurement, temperature sensing, and data acquisition functions.]]</center>

As shown in the diagram, an aluminum heating block holds a cuvette containing the sample under test. The sample is a combination of DNA and a fluorescent dye called SYBR Green. In addition to being a convenient holder, the block gives the setup enough thermal inertia to facilitate a measurement from natural cooling. (Without the block, the sample would cool too quickly.) 

Blue light from an LED illuminates one side of the cuvette. An optical filter shapes the output spectrum of the LED so that only the desired wavelengths of light fall on the sample. 

A photodiode placed at 90 degrees to the LED source detects the green light emitted by bound SYBR Green. The photodiode is placed behind an optical filter to ensure that only the fluorescent light given off by the sample is detected. 

Since photodiodes produce only a very small amount of current, it will be necessary to build a very high gain transimpedence amplifier to produce a signal that is measurable by the PC data acquisition cards. Photodiode amplifiers are particularly challenging because many of the non-ideal characteristics of op amps become apparent at high gain.

An RTD attached to the heating block and wired to a voltage divider provides an indication of temperature. The temperature of the heating block will be a proxy for the sample temperature. Unfortunately, the block cools faster when it is hot than when it is near room temperature. You will have to get the heating block set up in your apparatus quickly after you remove it from the heating block.

A PC data acquisition card digitizes the amplified photodiode and RTD signals. A LabVIEW virtual instrument (VI) records the signals over time. Data from the DNA melting VI can be saved to a file. The file can be loaded into Matlab for analysis.

===Objectives and learning goals=== 

*Measure temperature with an RTD. 
*Implement a high gain transimpedence amplifier for photodiode current multiplication. 
*Measure light intensity with a photodiode.
*Build an optical system for exciting the sample with blue light and gathering the fluorescence output on the photodiode. 
*Record dsDNA concentration versus temperature curves for several samples. 
*Estimate T<sub>m</sub> from your data.
*Compare the measured curves with theoretical models.
*Identify unknown DNA samples.

==Lab procedure== 

===Roadmap ===

#Build an optical system containing the LED, heating block, sample, photodiode, filters, and lenses.
#Hook up a three terminal voltage regulator to create an electrical power supply for the LED.
#Build, test, and calibrate the temperature-sensing circuit. 
#Build an amplification/offset circuit for the DNA fluorescence signal. 
#Troubleshoot and optimize your system.
#Heat a samples of DNA with SYBR Green dye and record DNA melting curves as the samples cool.
#Analyze the data. Identify the three unknown samples. Compare your observations to theoretical models.

===Optical system===

<center>[[Image:DNA Melting Optical System Diagram.jpg|700 px|thumb|center|DNA Melting Optical System Diagram]]</center>

The optical system consists of an LED, excitation filter, sample cuvette, heating block, emission filter, photodiode, optional lenses, and associated mounting hardware. Construct your system on an [http://www.thorlabs.com/thorProduct.cfm?partNumber=MB1224 optical breadboard]. The breadboard has a grid of tapped holes for mounting all kinds of optical and mechanical hardware. [http://www.thorlabs.com ThorLabs] manufactures most of the hardware stocked in the lab. A few of the components you will certainly use include:  [http://www.thorlabs.com/newgrouppage9.cfm?objectgroup_id=1266 1/2" diameter posts], [http://www.thorlabs.com/thorProduct.cfm?partNumber=CP02 CP02 cage plates], and [http://www.thorlabs.com/NewGroupPage9.cfm?ObjectGroup_ID=1521 1” diameter lens tubes]. 

Use [http://www.thorlabs.com/thorProduct.cfm?partNumber=RLA0300 optical rails] and [http://www.thorlabs.com/thorProduct.cfm?partNumber=RC1 rail carriers] or [http://www.thorlabs.com/NewGroupPage9.cfm?ObjectGroup_ID=47 optical bases] to mount 1/2” posts on the breadboard. [http://www.thorlabs.com/thorProduct.cfm?partNumber=RA90 RA90] right angle post clamps and [http://www.thorlabs.com/thorProduct.cfm?partNumber=PH2-ST post holders]can also be useful.

There are a variety of ways to construct the apparatus. A good design will be compact, stable, and simple. It will be necessary to shield the optical system from ambient light, so a small footprint will be advantageous. 

====Illumination====

Begin by mounting the LED on your breadboard. Note that there are two styles of LEDs. The [http://www.laminaceramics.com/docs/BL_2_Blue.pdf Lamina LED Array] is mounted on an aluminum heat sink and bolted to a CP02 cage plate. The CP02 attaches to the top of a post. It has an SM1 threaded hole through the middle that connects to 1” diameter lens tubes. The [http://www.allelectronics.com/spec/LED-112.pdf Cree LEDs] are already mounted in a 1” lens tube. 

Both styles of LED emit a range of wavelengths with a peak at 475 nm. A Chroma Technology [http://web.mit.edu/~20.309/Students/Datasheets/Chroma%20D470-40.pdf D470] filter eliminates unwanted parts of the spectrum that might interfere with detection of the fluorescence signal. The filters have exposed, delicate coatings and must be [http://www.chroma.com/index.php?option=com_content&task=view&id=56&Itemid=65 handled carefully]. In addition, the filter [http://www.chroma.com/index.php?option=com_content&task=view&id=57&Itemid=66 works better in one direction than the other].

Light from both kinds of LEDs diverges in a cone with an angle of about 100 degrees, so place the device close to the sample. You can also use a lens to concentrate the LED's output. Several lenses are available in the lab:

*[http://www.thorlabs.com/thorProduct.cfm?partNumber=LA1951 f=25.4mm]
*[http://www.thorlabs.com/thorProduct.cfm?partNumber=LA1131 f=50mm]
*[http://www.thorlabs.com/thorProduct.cfm?partNumber=LA1509 f=100mm]
*[http://www.thorlabs.com/thorProduct.cfm?partNumber=LA1708 f=200mm]

====Fluorescence detection====

The [http://www.thorlabs.com/Thorcat/8700/8770-D02.pdf SM05PD1A] photodiode is mounted in a short tube with SM05 threads. Use a [http://www.thorlabs.com/thorProduct.cfm?partNumber=SM1A6 SM1A6] adapter to mount the photodiode in a CP02 cage plate. Mount the photodiode assembly to the breadboard at 90 degrees to the LED. Build a system to hold the Chroma [http://web.mit.edu/~20.309/Students/Datasheets/Chroma%20E515lpv2.pdf E515LPV2] emission filter in front of the photodiode. You can use a lens to focus light from the sample on to the detector to improve performance, if you like.

Put an optical quick connect at the end of the photodiode assembly to facilitate easy attachment of the heating block during experimental runs. The other half of the quick connect goes into the CP02 cage plate mounted on the heating block.

===Electrical System===

===LED driver===
[[Image:Lamina LED Driver Electronics.jpg|right|thumb|250px|Current feedback to the ''adjust'' pin of the LM317T variable voltage regulator provides a steady source of illumination. (Note that 4.2&Omega; should read 4.3&Omega;)]]
'''Or: how to make a current source'''

Drive the LED array with an [http://www.national.com/ds/LM/LM117.pdf LM317T] variable voltage regulator as shown. The LM317T has a feedback circuit that strives to maintain 1.25 volts between its output and adjustment pins. Thus, in the circuit shown, the LM317T sources a constant current of approximately .29A through the load (and the feedback resistor). 

It is possible to drive an LED with a voltage source; however, the steepness of a diode's I-V curve results in large current swings for small changes in supply voltage. LED brightness is proportional to current. A current source will provide a more stable light output.

The LM317T and 4.3&Omega; resistor both dissipate quite a bit of power in this connection. They will become toasty. Use a heat sink on the LM317T. Double check your wiring before connecting the LED array. The array can be damaged by excessive current. Remember the rule of finger: if you can't keep your finger on a component indefinitely, it is too hot. Use a larger feedback resistor to keep the electronics cooler (at the expense of light output), but never a smaller one.

===Temperature=== 

The electrical resistance of most materials varies with temperature. An RTD is a special resistor (usually made out of platinum) that exhibits a nearly linear change in its value with temperature. An RTD may be used to accurately measure temperature by including it as an element in a voltage divider. As the resistance of the RTD changes, so will the voltage across it. 

A [http://www.ussensor.com/prod_rtds_thin_film.htm  PPG102A1 RTD] has been pre-mounted to the DNA heating block. This RTD has a nominal resistance of 1 K&Omega; and its value increses with temperature. Note that the maximum current carrying capacity of this device is 1 ma. Hook up the RTD in a voltage divider. Make sure the divider has no more than 1 mA flowing through it. Use freeze spray or heat the block on the warmer to test the circuit.

===Fluorescence intensity===

====Amplification circuit====

[[Image:Hw1highgopamp.JPG|thumb|300px|right|Schematic diagram of a high gain transimpedence amplifier.]]
The photodiode produces only a tiny current – on the order of nanoamps. Its output must be amplified and converted to a voltage measurable by the PC data acquisition system. A transimpedance amplifier (sometimes called a current-to-voltage converter) with a gain of approximately 10<sup>8</sup> V/A will be required. The circuit considered in [[20.309:Homeworks/Homework1|Homework 1]] is capable of providing this gain. (Optional question: why not simply use a resistor, and omit the op-amp?)

Photodiode amplifiers can be fiddly under the best of circumstances. At such high gain, many of the non-ideal behaviors of op amps become apparent. It will be important to keep your wiring short and neat. The amplifier and witing will also be susceptible to physical movement, so prevent things from getting bumped during experimental runs. In addition select an op amp that has a very low input bias current as possible. (Why?) Op amps with JFET inputs like the [http://www.ortodoxism.ro/datasheets/nationalsemiconductor/DS005655.PDF LF411] and [http://www.ortodoxism.ro/datasheets/nationalsemiconductor/DS005648.PDF LF351] generally have the lowest input current.

====Offset circuit====

The positive and negative input channels of an op amp cannot be perfectly matched during manufacturing. Because the open loop gain of an op amp is huge &mdash; usually  10<sup>5</sup> or more &mdash; even a slight mismatch will cause a non-ieal behavior called input offset voltage. (In other words, if you apply a zero voltage to the across the plus and minus pins by shorting them together, the output will probably saturate at the full positive or negative output limit.) ''V<sub>os</sub>'' is the voltage that must be applied across the inputs to achieve a zero output. Most op amp datasheets specify a maximum value for ''V<sub>os</sub>''. In terms of the ideal circuit elements, input offset acts like a small voltage source connected in series with one of the input pins. As a real world example, the maximum specified offset voltage of the LF411 is 2.0 mV. 

In the lab, you will find it useful to be able to adjust the quiescent output level of the photodiode amplifier. Many op amps provide a means for externally balancing the mismatch between plus and minus inputs. Pins 1 and 5 of both the LF351 and the LF411 are connected to the current sources that drive the differential input stage. As suggested by the name, these balance pins allow slight changes in the balance of current flowing through each side of the input stage. A potentiometer with both ends hooked across these pins and the wiper hooked to the negative supply voltage allows ''V<sub>os''</sub> to be virtually eliminated with a single adjustment. See the ''Typical Connection'' schematic diagram on page 1 of the [http://www.ortodoxism.ro/datasheets/nationalsemiconductor/DS005655.PDF LF411] datasheet.

Although the primary intent of the balance pins is to null out  ''V<sub>os</sub>'', they will also work quite nicely as an output level adjustment. Use a 10 turn pot so that you can get the output to settle where you want it. Adjust the dark output of the amplifier to be approximately zero.

Unfortunately, input offset voltage varies with temperature. (The LF411, for example, specifies a maximum temperature coefficient of 20&mu;V/°C.) This sensitivity is one of the chief causes of output drift in the high gain amplifier, which you will undoubtedly notice in the lab. Try spraying a little freeze spray on the op amp to observe the effect. (Don’t freeze your op amp right before you do an experimental run &mdash; it takes quite a while to stabilize.)

===Practical matters===

<blockquote>
<div>
''As with most amplifiers, care should be taken with lead dress, component placement and supply decoupling in order to ensure stability.''

''&mdash;[http://www.ortodoxism.ro/datasheets/nationalsemiconductor/DS005655.PDF LF411 Datasheet]''

''In theory, there is no difference between theory and practice. But, in practice, there is.'' 

''&mdash;[http://en.wikiquote.org/wiki/Jan_L._A._van_de_Snepscheut Jan L. A. van de Snepscheut]/[http://en.wikiquote.org/wiki/Yogi_Berra Yogi Berra] 

</div>
</blockquote>
The universe is rife with electrical noise. Keeping the noise out of sensitive electronic instruments requires a great deal of care. Unfortunately, electronic breadboards are a poor environment in which to construct high gain amplifiers. A few simple tricks can improve things.

*Strap the ground of your breadboard to the optical table by connecting it with a short wire to a screw in the table.
*Use power supply bypass capacitors. Connect a large capacitor between all supply voltges and ground. Large, electrolytic capacitors of at least 0.1 &mu;Fd work well for this purpose. Electrolytic capacitors are polarized. Make sure to put them in the right direction.
*What happens when the shield of a BNC cable touches the optical table? If you notice an effect, take precautions to prevent this from happening during an experimental run.
*Move your hands around dfferent parts of the circuit. What effects do you see?

===PC Data Acquisition System===

{{:20.309:DAQ System}}

====LabVIEW VI====

The DNA Melting LabVIEW VI is located in the <code>Students/Labs/DNA Melting</code> folder of the course locker. Double click to launch the VI. (The current version is R1.0)

Click the run arrow or select Operate->Run from the menu to start the VI. The top two charts show the digitized voltage at the RTD and diode inputs over time. Use the range settings to get a good view of the signal. 

Press <code>Start Recording</code> to begin taking data. The sample rate for recorded data can be set in increments of 0.1 seconds. Press <code>Stop Recording</code> at the end of an experimental run and use the <code>Write Data</code> button to save the most recent result in a comma delimited file that can be read into Matlab or Excel.

===Debugging the apparatus===

#Use freeze spray and the heat gun to make sure the temperature circuit is working properly.
#Cover and uncover the photodiode to verify operation of the fluorescence measurement system.
#Use a box and a piece of black cloth to shield your apparatus from ambient light. Can you measure the difference between a cuvette filed with water and one with DNA and SYBR Green?
#Observe every electrical signal node with the oscilloscope. Are any signals noisy? Is there a way to improve the quality of poor signals?
#Watch the fluorescence readout over time. Is it stable or does it drift?

==Experimental procedure== 

Once your instrument is running to your satisfaction, measure melting curves each of the 5 conditions: 

*40bp perfect match 
*3 unknown 20 bp sequences (perfect match, single mismatch, and complete mismatch) 
*20 bp perfect match at different ionic strength

If you have time, you can run additional experiments. For example, you could gather additional ionic strength data points.

The DNA melting apparatus will generate the best data when both the amplifier circuit and LED have been on for a while and all the components have reached their steady state temperature. Make sure the outupts of the system are stable before you begin taking data. Turn your apparatus on and measure the difference between a cool DNA sample and water. Run the DNA melting LabVIEW VI in the DNAMelting directory of the course locker. Adjust the range controls for each channel to provide the greatest measurement resolution.

The steps for each experimental run are:

#Heat up the sample on the hot plate
#Quickly transfer the sample to your setup
#Cover the apparatus to block out ambient light 
#Start recording RTD and photodiode output with the LabVIEW VI. 
#Wait for the block to cool to below 40°C 

===Prepare your apparatus===

Use the potentiometer to adjust the amplifier voltage offset until it reads close to 0 Volts in the dark. 
Make sure your apparatus has reached the steady state and the fluorescence readout is stable. 

===Make a sample===


{{Template:Safety Warning|message=SYBR Green I in DMSO is readily absorbed through skin. Synthetic oligonucleotides may be harmful by inhalation, ingestion, or skin absorption. Wear gloves when handling samples. Wear safety goggles if there is a danger of liquid splashing into your eyes. Do not create aerosols. The health effects of SYBR Green I have not been thoroughly investigated. See the SYBR Green I and synthetic oligonucleotide MSDS in the couse locker for more information.}}

Pipet 500&mu;l of DNA plus dye solution into a disposable plastic cuvette. Pipet 20&mu;l of mineral oil on top of the sample to help prevent evaporation. Put a top on the cuvette and mark it with a permanent marker. Keep the sample vertical to make sure the oil stays on top. You should be able to use the same sample for many heating/cooling cycles. Only discard it if you lose significant volume due to evaporation. If you need to leave the sample overnight, store it in the lab refrigerator. If you finish with a sample and it is still in good shape, pass it on to another group.

===Heat up the sample===

Place your heating block and sample in the hot water bath. You can use a DVM to monitor the temperature of the holder. It takes longer than you think to reach equilibrium. The block will cool down a bit while you transfer it to your setup, so heat it to a temperature well above where the DNA melts (at least 85°C, preferably 90°C). The double boiler arrangement will not allow the sample to boil.

===Transfer the sample to your apparatus and take data===

Use tongs to remove the heating block from the bath. Remember to keep everything upright. Set the block down on a paper towel. Use leather gloves to pick up the sample and connect it optically and electrically to your apparatus. Once everything is hooked up, press the ''Start Recording'' button on the LabView DNA Melting VI.

{{Template:Environmental Warning|message=Discard pipette tips with DNA sample residue in the ''Biohazard Sharps'' container. Do not pour synthetic oligonucleotides or SYBR Green down the drain. Empty the liquid into the waste container provided. Dispose of plastic cuvettes in the ''Biohazard'' container.}}

==Report Requirements== 

===Data Analysis=== 

Use Matlab to convert your raw data to fraction hybridized and temperature.  (Can you learn anything from the absolute value of the photodiode output?) Filter the data to remove noise. This can involve smoothing the data from individual experimental runs as well as combining data from multiple runs. Plot relative fluorescence versus temperature comparing:

*20 bp oligos in solutions of varying ionic strength
*Perfect match, single mismatch, and complete mismatch (unknown) 20 bp oligos
*40 bp versus 20 bp perfect match oligos

In addition, plot the derivative of each curve. Note that the differentiation operation is particularly sensitive to noise. If your derivative plots look noisy, apply better filtering to the raw data. It will be helpful to create a matlab script to process the data. ''T<sub>m</sub>'' is taken to be the peak value of the derivative.

You may process the data how you wish, however a useful command in Matlab is <tt> resample </tt>.  This function can not only resample data, as the name implies, but will also apply a ''low''-pass filter (decreasing the ''high''-frequency noise).  A larger vector of filter coefficients or number of samples on each side of the current sample will smooth the data more.  Using this command, pay attention to the resulting length of your new data, as well as any inaccuracies at the ends (what does resample assume for the data points before and after your data?).  Derivatives may require filtering as well.

===Model vs. reality=== 

In class, we derived an expression that relates the melting temperature to the enthalpy change &Delta;H° and entropy change &Delta;S° of the hybridization reaction: 

<br/>
<center>
<math>
T(f) = \frac{\Delta H^{\circ}}{\Delta S^{\circ}-R \ln
(2f/C_T(1-f)^2)}
</math>.
</center>
<br/>

Here, <math>f</math> is the fraction of DNA strands hybridized (dimerized) at a particular temperature (at <math>T_m</math>, this is 1/2), and <math>C_T</math> is the total concentration of single-strand oligonucleotides (or 2X the dsDNA concentration when all strands are hybridized). Choose one of the perfect-match sequences that you measured, and use matlab to fit the model to your measured data, which will allow you to extract the &Delta;H° and &Delta;S° parameters. To perform the fit, you will need a matlab function that will evaluate <math>T(f)</math> given an input <code>const</code> for the &Delta;H° and &Delta;S° parameters. The function will be something like this: 

<pre style="background:#DDDDFF">
function Tf = melt(const, f) 
R=8.3; 
C_T=33e-6; 
dH = const(1); 
dS = const(2); 
Tf = dH./(dS - R*log(2*f./(C_T*(1-f).^2))); 
</pre>

You can then invoke matlab's <code>lsqcurvefit</code> routine to do the fit, which will return the best values for &Delta;H° and &Delta;S°. 

<pre style="background:#DDDDFF">
FitVals = lsqcurvefit(@melt, [dH_guess, dS_guess], frac_vector, temp_vector) 
</pre>

===Discussion===

Document and discuss the electrical and optical systems you built. What practical problems did you run into? What kinds of noise did you see in different parts of the circuit? How effective were the measures you took to reduce the noise? 

Briefly describe how you processed the raw data.

For each of the three cases, discuss the melting temperatures and shapes of the melting curves you recorded. Why do the curves look as the do? Can you identify the unknown samples? Compare your melting curves with those of other students in the class. You may find large differences. What might cause these variations? What factors affect the DNA melting temperature and the sharpness of the melting transition?

How does the theoretical model compare with the curves you measured?

Bonus (optional): 

Calculate &Delta;H° and &Delta;S° for one of the perfet match sequences using the nearest-neighbor model from class. Compare the calculated values to the best fit parameters. What might explain the differences? What factors affect &Delta;H° and &Delta;S°?

==External references==

*[http://www.ni.com/pdf/manuals/370503k.pdf National Instruments data acquisition card user manual]
*[http://www.sigmaaldrich.com/sigma-aldrich/datasheet/s9430dat.pdf SYBR Green I datasheet]
*[http://www.national.com/mpf/LM/LM317.html LM317 datasheet]

</div>