<!-- Replace the OWW.101 below with your course's number (after you've copied it to your course's people page) -->
{{Template:7.342}}

<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

'''Post discussion, questions, or comments about the Week 3 course material here.'''

==Amber==

Lachner et al.

What does it mean to "mock-infect" the fibroblasts during the experiment to look for nuclear localization of SUV39H1 proteins? 

Wysocka el at.

When the authors look for defects from knockdown of xWDR5, they use use embryos derived from eggs of multiple frogs so that there will not be phenotypic differences from different genetic backgrounds. How does using eggs from many frogs do this? It seems like they would want to use embryos from the same frog instead. 

==Elizabeth==

Wysocha et al

I found their suggestion that WDR5 specifically recognizes the H3 tail methylated at K4 a bit confusing, and do not understand how they made that leap from the analysis of Ash2 and RbBP5.

Lachner et al

I really enjoyed how they used the GFP analysis to better show that HP1 localization is dependent on Suv39h activity and not Suv39h proteins, and the result of 80-90% was so high. I would however, have liked to hear more about how Lys9 in H3 acts like a molecular switch, because I think that is one of the strongest suggestions in their paper. 

==Georgi==

Wysocka et al.

I think the increase in DiMeK4 in HOXA9 P2 can be explained by the decrease in trimethylation, e.g. less DiMeK4 is converted to TriMeK4, that's why the levels go up; which fits the model; moreover they see the same in HOXC8 P4; but in HOXC8 P3 the levels go down which can not be explained at all, I think. Maybe if they had more WDR5 targets the results would me more consistent. They say there might be differences in the dynamic of regulation between the two Hox genes anyway

Lachner et al.

In the WDR5 paper they do ChIP and gene expression analysis. Maybe they might have done it for the HP1 paper too, if, of course it was possible to do in 2001 (which I don't know). Also, is there any reason for using (Myc)3-tagging other than just not having specific anibodies against the proteins of interest?

==Holly==

Wysocka et al

They suggest that their data provides evidence that WDR5 acts as a "sensor" protein - in what way? It does not seem to show how the timing of WDR5 binding and subsequent activation is related to signals that indicate expression would be appropriate (although it does show that WDR5 would be a good target for manipulating/ regulating expression)

Lachner et al

This paper discusses what features of HP1 are required for it to bind the H3 lys 9 N terminus. However the experiments only show the characteristics needed in vitro - I think they should have done the pull down assay with nucleosomes. Also, what is the significance of understanding the binding? Why would dimerization be important and do these results actually show that it is necessary?

==Kathy==

Lachner: What are the mechanisms for passing on the histone code to the next generation? And, what are some characteristics of the chromo domain?

Wysocka: What is the functional difference between methylations, di-methylation, and tri-methylation? Does more methylation equal more silencing, or is the relationship not linear, but more dependent on recognition of the elements involved?

==Manpreet==

Lachner et al:
Presumably the authors of this paper are short of time. It would have been nice if they'd replicated some of the studies shown in the second paper; including perhaps a loss of function analysis in a model organism.

Wysocka et al:

Figure 5D shows an increase in the levels of dimethylated K4 in HOXA9, when WDR5 is knocked down, using WDR5 siRNA. The authors say this data is reproducible; but it doesn't seem to be explained by their model.

==Zak==
Lachner, et al.:

How come we never learned about histone phosphorylation? Has it been
studied much?

In figure 1.c, they simply assume that adding the Myc tags does not 
interfere with the function of the protein. How valid is this 
assumption? When does adding Myc or FLAG tags interfere, and what 
experimental methods could be used to verify that tag addition is not 
affecting protein function?

Wysocka, et al.:

I don't entirely understand the hypothetical model (page 870). Do they know how a histone methyltransferase is recruited to a completely unmodified histone tail? WDR5 seems to control a positive feedback loop - if there is no H3 K4 methylation then it cannot methylate, but if there is some H3 K4 methylation then it causes more methylation. What stops WDR5 from running wild and causing a huge amount of methylation in response to one mistakenly methylated histone tail? They call WDR5 a "sensor" protein, because it reads and writes the same mark, but I think it is more of a "positive feedback" protein.