<!-- Replace the OWW.101 below with your course's number (after you've copied it to your course's people page) -->
{{Template:7.342}}

<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

'''Post discussion, questions, or comments about the Week 10 course material here.'''

==Amber==

Morris et al

What is a lentiviral vector? 

Also, what do the authors mean when they say that siRNA silencing reflects a "genome defense mechanism"? 

Kim et al

I don't really understand what polycomb silencing involves.

==Elizabeth==
Kim et al

In figure 5, I do not understand the different levels of GFP expression.

Morris et al

In Figure 1b they use a nuclear run-on assay. How does that work?

==Georgi==

Morris et al.

Kim & Villeneuve et al.

How is the siRNA targeted to a non-coding promoter region? They propose a nascent transcript model for a RNA transcipted through the promoter region by RNAPolII but how does the polymerase transcribes the promoter? Morris et al. use a siRNA targeting a sequence between -106 and -86.

==Holly==
Morris et al:

What is a nuclear run-on analysis? How is it carried out? Also, is it known how siRNAs mediate posttranscriptional mRNA destruction?


Kim et al:

Fig 2c indicates that there is little DNA methylation. In contrast Morris et al show that DNA methylation is involved in the silencing. What are the implications these different silencing mechanisms? Does it effect the nature of the repression (e.g. duration, extent etc) or just allow more opportunities for gene regulation?

==Kathy==

Morris: Is the mechanism of this silencing known? It's interesting that these small RNAs can direct both DNA and histone methylation. Also, how does a nuclear run-on assay work?

Kim: Have people done studies on TGS siRNA targetting like those that have been done for miRNAs? E.g., whether base pairing at certain areas of the siRNA is more important than at other parts, or whether perfect base pairing elicits a different response than imperfect?

==Manpreet==
Morris et al:
An siRNA of only 21 to 25 nucleotides can target DNA for methylation (and silencing), when the siRNA has mismatches with the DNA. 21-25 nucleotide motifs must be common in the genome (particularly if you are allowed a certain degree of mismatch). How is specificity of silencing acheived?

Kim et al:
The authors talk about the RNA-induced silencing complex (RISC) in human cells. When is this active?

==Zak==
.