<!-- Replace the OWW.101 below with your course's number (after you've copied it to your course's people page) -->
{{Template:7.342}}

<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

'''Post discussion, questions, or comments about the Week 2 course material here.'''

==Amber==

1) For the Weintraub paper, I was wondering if today we have images of the
histone conformations they were speculating about at that time.

2) For the Meshorer one, I was curious about the leukemia inhibitory factor
(LIF) and why the cell differentiates when you deplete it.

==Cathlene==

For "Chromosomal subunits in active genes have an altered conformation", I'm
confused by the passage on page 849, in the middle column, where it describes
the DNA being 'nibbled'.  If the DNA is being digested (and presumably
differently in different cells, since different cells have different active
genes), doesn't that mean that then each type of cell has a different set of
DNA?  I thought all the cells in an organism had the same set of DNA...  Do
they just start off with the same set, and then they're modified?

For "Hyperdynamic plasticity of chromatin proteins in pluripotent embryonic stem
cells", I was confused by the discussion in the summary about chromatin binding
proteins.  Are these proteins that simply bind to the chromatin, or are they
proteins that bind to multiple, pieces of chromatin and thus bind the chromatin
together, affecting the shape?

==Elizabeth==

==Georgi==

Weintraub et al.

OK, I know I'm required to ask questions about each paper but I really have no questions about this one; it's 30 years old and whatever questions might come out of it, have already been answered 

11/07/2006

Meshorer et al.

I think several questions follow up from this study: which regions in the genome change their chromatin stucture during differentiation and how is this regulated? is there any defined structure of the nucleus in vivo and if there is how does it correlate with transcriptional regulation and cell differentiation?

==Holly==

Questions relating to Paper 1 (Chromosomal subunits in active genes have an
altered conformation)
-It says that mature adult RBCs that don't synthesize RNA are also sensitive to
the nuclease - does this suggest that the original structure is not reinstated?
Could this be due to the lack of hyperdynamic chromatin proteins shown in paper
2? Does it also suggest that it isn't the structure of DNA that is manipulated
to silence the gene? In which case what is used?

- Why does staph nuclease not normally show preferential digestion - is it
because even with the more open conformation of active genes the enzyme is
still to bulky(?) to access them?

- A possible control to check that the preferential digestion is due to
the structural conformation would be to digest the histones with a protease,
then subsequently add the nuclease - would expect 100% digestion?

Questions relating to Paper 2 (Hyperdynamic etc etc)
- Possible follow up questions - What are the interactions of the hyperdynamic
chromatin proteins; what is the signal that instigates their actions in the
remodeling process? Is there anything that could reinstate the hyperdynamic
nature and would this cause the cell to revert back to pluripotency?

==Kathy==

First paper:

What exactly are they doing with the acid precipitations/what do the acid-soluble fractions contain? (e.g. in pg. 849 middle column, pg. 850 middle column, figure 2 legend, pg. 853 middle column).

Having a hard time fully understanding the x-axis of their graphs... Cot is [DNA]*(time digested)?? Concentration of which DNA? I don't know how to predict what the curves should look like with those units.

Second paper:

What is FISH?

The HirA-/- data are the opposite of what I would expect. Since lack of HirA makes it harder to assemble complete nucleosomes ("reduced incorporation of core histones H3 and H3.3"), then it would seem that HirA-/- cells would be less able to form heterochromatin, and therefore prefer to stay more ESC-like rather than differentiating quickly. The authors' argument is that since H3 and H3.3 cannot be incorporated as well, they are more of them floating around; but in the end wouldn't you s
till need HirA function to use those extra H3/H3.3s?
==Manpreet==

Questions about the first paper:

I don't really understand why ATV genes are sensitive to digestion. The paper says the globin genes are active, and therefore more have an altered subunit structure, which is more susceptible to pancreatic DNAse digestion. I don't understand how this fits with less active RNA tumour virus genes also being digested.


Question about the second paper:

In figure 4C, the NPCs show a very clear line on the gel electrophoresis, which is not seen on the ESCs. This is present both before and after digestion with micrococcal nuclease. What does this line represent? It suggests some undigestible element of the nucleus, present in NPCs, and not in ESCs. I'd investigate this further.

==Zak==
Weintraub, et al.:

They use many different methods and models to show that more transcribed
genes are easier to digest, but they don't have any quantitative
correlation between gene activity and ease of digestion. Is chromatin
conformation equally important to the activity of all genes, or is it
more or less important for some genes? I suspect that there are certain
genes which can be completely silenced with very little increase in
chromatin packing.

Meshorer, et al.:

What mechanism accounts for the decrease in chromatin protein mobility
during differentiation? Are the hyperdynamic chromatin proteins replaced
by other chromatin proteins that bind to DNA more tightly and
irreversibly, or do the hyperdynamic proteins simply change their
behavior during differentiation?

I don't entirely understand why they used salt extraction, how the procedure works, and what it accomplishes. It seems like an interesting technique for separating the components of cells.