<!-- Replace the OWW.101 below with your course's number (after you've copied it to your course's people page) -->
{{Template:7.342}}

<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

'''Post discussion, questions, or comments about the Week 5 course material here.'''

==Amber==
Nielson et al

Figure 2d is confusing me. Is it trying to show that Rb and SUV39H1 repressed E2F1? And why would that matter? 

Ayyanathan et al

This may have been the paper's main point, but I still am not sure how the gene silencing can continue to occur in the next several generations of the original cells. What causes the gene silencing to then stop after about 40 divisions?

==Elizabeth==

Nielsen et al.

I agree with Manpreet in not understand the presence of HA in the HA immunoprecipitation. I in theory understand what they are trying to prove in “This methylase binds the pocket domain of Rb because tumour-derived mutations in the pocket (F706C), or truncations of the pocket (delta928 and delta737), abolish binding to the methylase,” (p563 top left) but I do not understand how figure 1d explains this.

Ayyanathan et al.

I understand that 4-OHT is important for repression activity, but what is it?

==Georgi==

Nielsen et al.

The authors propose a model involving K9 deacetylation by HDAC complexed with RB and E2F and subsequent methylation by Suv39H1. They don't do any experiments to test this hypotheses and I think they could have done it by testing for Rb-associated histonedeacetylase actvity the same way as they identified the assiciated histonemethylase activity

Ayyanathan et al.

The paper ends proposing some molecular basics for DNA methylation. What is known now about the proteins involved in CpG methylation?

I agree with Manpreet that we see people choosing different HP1 isoforms to work with and it would be good to know if they possess any distinct functional specificity.

==Holly==
Nielson et al:
This paper shows that SUV39H1 and HP1 are both involved in the gene silencing mechanism of the tumour suppressor Rb. What is the functional significance/ therapeutic potential of this knowledge?  Have mutations in either HP1 or SUV39H1 themselves been implicated in cancer? Also, the 2nd paper makes reference to the dose dependent nature of HP1 repression. Column 4 in fig 2d (of Nielson paper) shows that SUV39H1 mediates some repression even without Rb being present – would increasing levels of HP1 compensate for Rb loss of function? 

Ayyanathan et al:
Why is the KRAB-PAX3-HBD protein not required to maintain the silent state? I.e. why are the other components of the repression pathway able to remain bound when they presumably couldn’t initially bind without KRAB-PAX3-HBD? Also, how (and is) the repression reversed? (Presumably the HP1 etc must dissociate, what causes this?)

==Kathy==

Nielsen: How do they know HP1 is directly responsible for silencing, and that silencing isn't just due to the methylation? It would've been interesting to see +/- HP1 on their [cyclin E promoter]-[luciferase] assay, or an anti-cyclin E blot in their Rb+/+ and Rb-/- cells (to go with figure 4e). I'm also a little bothered by the lack of error bars on their reporter assays in figure 2. Otherwise, their results are really interesting, because if SUV39H1 and HP1 are involved in both heterochromatic silencing ''and'' transcription repression, then if they repressed other genes involved in histone modification, that could lead to propagation of the histone code. This could also make sense if (I'm assuming) cyclin E is also involved in the cell cycle (maybe somehow related to S phase)?

Ayyanathan: The biggest question I have after reading this is still how the histone code is propagated (through >50 mitotic divisions!). The authors address this in the last section of the discussion; it seems like the DNA methylations could definitely mark where silencing is supposed to occur for the next generation. What makes CpG prone to methylation? In what other contexts does DNA become methylated? (I think bacteria methylate their DNA, but they don't have histones, is that right?). What are some complexes that specifically associate with methylated DNA? Are their any other modifications made to DNA (e.g. acetylation)? Also, I seem to remember that recently someone reported RNAi knockdowns could be propagated for many generations, and someone else discovered a white-tipped phenotype in mice that might be passed down through some small RNAs. Do any small RNAs associate with histone- or DNA-methyltransferases? (And a super naive question: does RNA ever get methylated?)

==Manpreet==

Nielsen et al: In figure 1c, the control immunoprecipation (i.e. using the HA) shows some H3 methylase activity. I'm sure the activity shown isn't statisically significant, but I can't think of any reason for the control to show any activity.

Ayyanathan et al: The paper mentions two different homologs of HP1 - Alpha and Gamma. They also mention that HP1 Alpha is inducibly bound; and that HP1 gamma is constituitively bound. This is briefly mentioned again in the conclusion - have studies been done showing specific functioanl differences between the isoforms?

==Zak==
Nielsen et al.: How exactly do they assay for histone methylase activity? Do all methyltransferase enzymes get the spare methyl groups from methyl-S-adenosyl methionine? When DNA gets methylated, where exactly is the methyl group added?

Ayyanathan et al.: I think it is cool that they engineered cells that respond in a specific way to a hormone of their choice. They clearly explain the mechanism by which 4-OHT exposure represses the luciferase gene, but how is that repression/silencing passed on during DNA replication? They don't even hypothesize as to the mechanism of inheritance. They also use a zeocin gene, but what protein does the zeocin gene code for, and how did they measure the frequency with which it was transcribed?