[[Image:Abhishek tiwari.PNG|750px]]
<div style="padding: 10px; color: #ffffff; background-color: #000; width: 730px">
<center>
[[Abhishek Tiwari | <font face="trebuchet ms" style="color:#ffffff"> '''Home''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:About | <font face="trebuchet ms" style="color:#ffffff"> '''About''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Resources| <font face="trebuchet ms" style="color:#ffffff"> '''Resources''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Projects | <font face="trebuchet ms" style="color:#ffffff"> '''Research & Projects''' </font>]] 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Softwares | <font face="trebuchet ms" style="color:#ffffff"> '''Softwares''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Reprints | <font face="trebuchet ms" style="color:#ffffff"> '''Publications''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Links | <font face="trebuchet ms" style="color:#ffffff"> '''ImpLinks''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Contact | <font face="trebuchet ms" style="color:#ffffff"> '''Contact''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</center>
</div>

= INTERACTOMICS & BIOLOGICAL NETWORKS =

'''''PLoS Computational Biology''''' Volume 2 | Issue 12 | DECEMBER 2006

*'''Modularity and Dynamics of Cellular Networks'''

'''Synopsis'''

Modularity and dynamics both underlie the functionality of cellular networks, ranging from transcriptional regulation to cell signaling. Technological innovations in both data generation and computational methods may advance our understanding significantly. Furthermore, integrating currently available data from various sources helps us to gain a more accurate and comprehensive understanding of cellular processes [45,46] (Box 1). Currently, the data quality and coverage of high-throughput datasets impose limitations on inferring accurate networks. Many computational methods used for analyzing biological systems do not make full use of available data and/or make strong assumptions that might not be realistic. With progress toward solving these problems, the phenotypes and behaviors of cells could potentially be predicted with higher confidence, and we might realize the promise to re-engineer cellular networks to produce desired properties.

The integration of a variety of datasets, including binary interactions, protein complexes, and expression profiles enables the identification of subnetworks that are active under certain conditions.

'''''PLoS Computational Biology''''' Volume 3 | Issue 3 | MARCH 2007

*'''Deciphering Protein–Protein Interactions. Part I. Experimental Techniques and Databases'''

'''Synopsis'''

Proteins interact with each other in a highly specific manner, and protein interactions play a key role in many cellular processes; in particular, the distortion of protein interfaces may lead to the development of many diseases. To understand the mechanisms of protein recognition at the molecular level and to unravel the global picture of protein interactions in the cell, different experimental techniques have been developed. Some methods characterize individual protein interactions while others are advanced for screening interactions on a genome-wide scale. In this review we describe different experimental techniques of protein interaction identification together with various databases which attempt to classify the large array of experimental data. We discuss the main promises and pitfalls of different methods and present several approaches to verify and validate the diverse experimental data produced by high-throughput techniques.

'''''PLoS Computational Biology''''' Volume 3 | Issue 4 | APRIL 2007

*'''Deciphering Protein–Protein Interactions. Part II. Computational Methods to Predict Protein and Domain Interaction Partners'''

'''Synopsis'''

Recent advances in high-throughput experimental methods for the identification of protein interactions have resulted in a large amount of diverse data that are somewhat incomplete and contradictory. As valuable as they are, such experimental approaches studying protein interactomes have certain limitations that can be complemented by the computational methods for predicting protein interactions. In this review we describe different approaches to predict protein interaction partners as well as highlight recent achievements in the prediction of specific domains mediating protein–protein interactions. We discuss the applicability of computational methods to different types of prediction problems and point out limitations common to all of them.

'''''PLoS Computational Biology''''' Volume 2 | Issue 12 | DECEMBER 2006

*'''Modularity and Dynamics of Cellular Networks'''

'''Synopsis'''

At very first sight I can say that Yuan Qi and  Hui Ge has written an excellent review article on the recent advances  on analyzing the architecture and dynamics of cellular networks. Using mammalian cell signaling as case studies article also summarize how computational modeling yields insight about cell signaling pathways or how computational analyses of networks shed light on specific biological processes. Yuan and Hui also addressed a series of important questions like

1.What are the characteristics of cellular network structures that distinguish them from randomly generated networks? 

2.Are the network structures relevant for biological functions? If so, are they evolutionarily conserved and how do they evolve? 

3.Are some topological patterns preferred at certain times or conditions?

Modularity and dynamics both underlie the functionality of cellular networks, ranging from transcriptional regulation to cell signaling.Unlike random networks, cellular networks contain characteristic topological patterns that enable their functionality. Modularity exists in a variety of biological contexts, including protein complexes, metabolic pathways, signaling pathways, and transcriptional programs.Network topologies reveal dynamic properties that contribute to cellular functions. Computationally, general graphical models such as dynamic Bayesian networks may be applied to analyze the dynamics of cellular network structures. Article also gives a nice summary of Computational Methods in Network Modeling Using “Omic” Data. 

'''''PLoS Computational Biology''''' Volume 2 | Issue 12 | DECEMBER 2006

*'''Protein Homology Network Families Reveal Step-Wise Diversification of Type III and Type IV Secretion Systems'''

'''Synopsis'''

Proteins evolved from a common ancestor are said to be homologues and to constitute a “family” with potentially similar structures, functions, and interactions. The problem of identifying “real” protein families based on amino acid sequence conservation is still doubtful  because algorithms that search for pairwise homologies can miss important relations and produce false hits. Problem of reconstructing the evolutionary relationships amongst proteins and of classifying them into families from a topological point of view was addressed  by defining the Protein Homology Network (PHN). In the PHN, proteins are seen as nodes connected by links that represent the homology relations inferred by sequence similarity. In such a representation, protein families should appear as dense clusters disconnected from the rest of the network. The availability of a large number of sequenced genomes now allows us to map the full set of protein similarity relationships into a Protein Homology Network (PHN), and protein families appear naturally as dense, highly connected regions of the network. In this study, Medini, Covacci, and Donati describe a new method that identifies these regions of the PHN, and generate a set of protein families (PHN-Families) that correlate with protein function and phylogeny. A comparison with an external protein domain database suggests that this approach produces results with a quality comparable to the ones generated by human experts.The method is completely unsupervised and can be applied to any number of genomes. In summary, the PHN-Families provide a comprehensive catalogue of the protein repertoire, also useful for the detection of inheritance patterns.Given the increasing number of bacterial genome sequences, and the number of genes with unknown function, PHN-Families could provide a powerful annotation tool, allowing straightforward comparisons of whole genomes and the discovery of novel and previously uncharacterized functions.

'''''Oxford Bioinformatics''''' Volume 22 | Number 17 | 1 September 2006

*'''Constructing biological networks through combined literature mining and microarray analysis: a LMMA approach
'''

'''Synopsis'''

Network representation of biological entities is very important for understanding biological processes, system organization and interaction between entities. Shao Li et al. integrated both the literatures and microarray gene-expression data, and developed a combined literature mining and microarray analysis (LMMA) approach to construct gene networks of a specific biological system. In the LMMA approach, a global network is first constructed using the literature-based co-occurrence method. It is then refined using microarray data.

This work may be a start towards an Integrated Biological Network representation where different data sets from literature, genomics and proteomics will be used to give precise and useful multidimensional interaction networks.