[[Image:Abhishek tiwari.PNG|750px]]
<div style="padding: 10px; color: #ffffff; background-color: #000; width: 730px">
<center>
[[Abhishek Tiwari | <font face="trebuchet ms" style="color:#ffffff"> '''Home''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:About | <font face="trebuchet ms" style="color:#ffffff"> '''About''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Resources| <font face="trebuchet ms" style="color:#ffffff"> '''Resources''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Projects | <font face="trebuchet ms" style="color:#ffffff"> '''Research & Projects''' </font>]] 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Softwares | <font face="trebuchet ms" style="color:#ffffff"> '''Softwares''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Reprints | <font face="trebuchet ms" style="color:#ffffff"> '''Publications''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Links | <font face="trebuchet ms" style="color:#ffffff"> '''ImpLinks''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Contact | <font face="trebuchet ms" style="color:#ffffff"> '''Contact''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</center>
</div>
= STRUCTURAL BIOINFORMATICS =

'''''Bioinformation''''' Volume 2 | Issue 1 | 2007
*'''HBNG: Graph theory-based visualization of hydrogen bond networks in protein structures''' 

'''Synopsis'''

HBNG is a graph theory-based tool for visualization of Hydrogen Bond Network in 2D. Digraphs generated by HBNG facilitate visualization of cooperativity and anticooperativity chains & rings in Protein Structures. HBNG takes Hydrogen Bonds list files (Output from HBAT, HBexplore, HBPLUS and STRIDE) as input and generates a DOT language script and constructs digraphs using freeware AT&T Graphviz tool. HBNG is useful in the enumeration of favorable topologies of hydrogen bond networks in protein structures and determining the effect of cooperativity and anticooperativity on protein stability and folding. HBNG can be applied to protein structure comparison and in the identification of secondary structural regions in protein structures.


'''''PLoS Computational Biology''''' Volume 2 | Issue 12 | DECEMBER 2006
*'''On Side-Chain Conformational Entropy of Proteins''' 

'''Synopsis'''

Side-chains of amino acids determine a protein's three-dimensional structure. The flexible nature of side-chains introduces a significant amount of conformational entropy associated with both protein folding and interactions. Despite many studies, the role that this side-chain entropy (SCE) plays in the process of folding and interactions has not been fully understood. Some basic questions about SCE have not been systematically studied. In this study, Zhang and Liu developed an efficient sequential Monte Carlo strategy to accurately estimate the SCE of proteins of arbitrary lengths with a given potential energy function. Using this novel tool, they studied how the SCE scales with the length of the protein, and how the SCE differs among a protein's X-ray, NMR, and decoy structures. They observed that X-ray structures pack more “smartly” than the corresponding decoy and NMR structures: with the same compactness, X-ray structures tend to have larger SCE. A combination of an SCE term with a contact potential energy significantly improved the discrimination between native and decoy structures. The implication of this study is that the SCE contributes so significantly to protein stability that it should be included explicitly in tasks such as structure prediction, protein design, and NMR structure refinement.


'''''In Silico Biology''''' Volume 7 | Article 7 | 2006
*'''Evolved cellular automata for protein secondary structure prediction imitate the determinants for folding observed in nature'''

'''Synopsis'''

Authors have developed a simple algorithm based on cellular automata for secondary structure predictions of proteins.Cellular Automata have been used in Bioinformatics for different purposes like predicting protein subcellular location and modeling biochemical pathways.Cellular automata make use of local interactions to simulate global and decentralized phenomena which can help to study protein folding phenomenon.Protein Folding depends on both local interactions between adjacent residues and global interactions between distant residues. A cellular automaton can capture both of these interactions making it an ideal candidate for studying protein folding. Developed program takes protein sequence as input and output is the corresponding sequence of secondary conformations adopted by each residue.


'''''PLoS Computational Biology''''' Volume 2 | Issue 10 | OCTOBER 2006
*'''3D Complex: a Structural Classification of Protein Complexes''' 

'''Synopsis'''

Most of the proteins in a cell assemble into complexes to carry out their function. It is therefore crucial to understand the physico-chemical properties as well as the evolution of interactions between proteins. The Protein Data Bank represents an important source of information for such studies, because more than half of the structures are homo- or heteromeric protein complexes. Here Emmanuel D Levy,etl. propose the first hierarchical classification of whole protein complexes of known three-dimensional structure, based on representing their fundamental structural features as a graph. This classification provides the first overview of all the complexes in the Protein Data Bank and allows non-redundant sets to be derived at different levels of detail. This reveals that between one half and two thirds of known structures are multimeric, depending on the level of redundancy accepted. We also analyse the structures in terms of the topological arrangement of their subunits, and find that they form a small number of arrangements compared to all theoretically possible ones. This is because most complexes contain four subunits or less, and the large majority are homomeric. In addition, there is a strong tendency for symmetry in complexes, even for heteromeric complexes. Finally, through comparison of Biological Units in the Protein Data Bank with the Protein Quaternary Structure database, they identified many possible errors in quaternary structure assignments. Our classification, available as a database and web server at http://www.3Dcomplex.org, will be a starting point for future work aimed at understanding the structure and evolution of protein complexes.

'''''Oxford Bioinformatics''''' Volume 22 | Number 22 | 15 November 2006
*'''Protein–RNA interactions: exploring binding patterns with a three-dimensional superposition analysis of high resolution structures'''

'''Synopsis'''

The recognition of specific RNA sequences and structures by proteins is critical to our understanding of RNA processing, gene expression and viral replication. The diversity of RNA structures suggests that RNA recognition is substantially different than that of DNA.The atomic coordinates of 41 protein–RNA complexes have been used to probe composite nucleoside binding pockets that form the structural and chemical underpinnings of base recognition. Composite nucleoside binding pockets were constructed using three-dimensional superpositions of each RNA nucleoside. Unlike protein–DNA interactions which are dominated by accessibility, RNA recognition frequently occurs in non-canonical and single-strand-like structures that allow interactions to occur from a much wider set of geometries and make fuller use of unique base shapes and hydrogen-bonding ability. By constructing composites that include all van der Waals, hydrogen-bonding, stacking and general non-polar interactions made to a particular nucleoside, the strategies employed are made readily visible. Protein–RNA interactions can result in the formation of a glove-like tight binding pocket around RNA bases, but the size, shape and non-polar binding patterns differ between specific RNA bases. We show that adenine can be distinguished from guanine based on the size and shape of the binding pocket and steric exclusion of the guanine N2 exocyclic amino group. The unique shape and hydrogen-bonding pattern for each RNA base allow proteins to make specific interactions through a very small number of contacts, as few as two in some cases.