[[Image:Abhishek tiwari.PNG|750px]]
<div style="padding: 10px; color: #ffffff; background-color: #000; width: 730px">
<center>
[[Abhishek Tiwari | <font face="trebuchet ms" style="color:#ffffff"> '''Home''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:About | <font face="trebuchet ms" style="color:#ffffff"> '''About''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Resources| <font face="trebuchet ms" style="color:#ffffff"> '''Resources''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Projects | <font face="trebuchet ms" style="color:#ffffff"> '''Research & Projects''' </font>]] 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Softwares | <font face="trebuchet ms" style="color:#ffffff"> '''Softwares''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Reprints | <font face="trebuchet ms" style="color:#ffffff"> '''Publications''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Links | <font face="trebuchet ms" style="color:#ffffff"> '''ImpLinks''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Contact | <font face="trebuchet ms" style="color:#ffffff"> '''Contact''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</center>
</div>
=Bioinformatics Toolkits=

'''Geneious''' http://www.geneious.com/

Geneious(Free for academic use) provides sophisticated genome & proteome research tools, protein or molecular visualization, publication literature search and storage tools and more, with revolutionary usability.Plugins extend Geneious's functionality by providing additional document importers or exporters, document viewers, database access or operations such as alignment or tree building. Anyone can write plugins using our freely available Public API â€“ in fact, most of Geneious's standard functionality is internally provided by plugins.

'''BioPerl''' http://www.bioperl.org

Bioperl is a collection of Perl modules that facilitate the development of Perl scripts for bioinformatics applications. As such, it does not include ready to use programs in the sense that many commercial packages and free web-based interfaces do (e.g. Entrez, SRS). On the other hand, Bioperl does provide reusable Perl modules that facilitate writing Perl scripts for sequence manipulation, accessing of databases using a range of data formats and execution and parsing of the results of various molecular biology programs including Blast, clustalw, TCoffee, genscan, ESTscan and HMMER. Consequently, bioperl enables developing scripts that can analyze large quantities of sequence data in ways that are typically difficult or impossible with web based systems.In order to take advantage of bioperl, the user needs a basic understanding of the Perl programming language including an understanding of how to use Perl references, modules, objects and methods.Bioperl is open source software that is still under active development.

'''BioJava''' http://biojava.org

BioJava is an open-source project dedicated to providing a Java framework for processing biological data. It include objects for manipulating biological sequences, file parsers, DAS client and server support, access to BioSQL and Ensembl databases, tools for making sequence analysis GUIs and powerful analysis and statistical routines including a dynamic programming toolkit.

BioJava is used in several real-world bioinformatics applications and has been used for bioinformatics analysis in a number of published studies. For example Biosense a commercial informatics offering from Inforsense that uses BioJava under the hood.

'''BioPython''' http://biopython.org/

The Biopython Project is an international association of developers of freely available Python tools for computational molecular biology.Project develop reusable Python code for anything related to biology and computing.Using BioPython one can do nearly all works which are possible using BioPerl and BioJava.

'''BioRuby''' http://bioruby.org/

BioRuby is an open-source project which aims to provide a reusable library for biological tasks for
the Ruby language . Ruby is an interpreted object-oriented scripting language with a simple and
powerful syntax and native object-oriented programming support.Recently  BioRuby was integrated with KEGG web services (API and DAS) to automate  genome and pathway analysis procedure. Note KEGG API is a SOAP/WSDL based web service providing genes and pathway information. KEGG DAS is also a web service providing genomic sequences and gene annotations via DAS protocol.

'''MathWorks''' http://www.mathworks.com/products/bioinfo/

MathWorks provides Bioinformatics Toolbox for extending MATLAB.The Bioinformatics Toolbox offers computational molecular biologists and other research scientists an open and extensible environment in which to explore ideas, prototype new algorithms, and build applications in drug research, genetic engineering, and other genomics and proteomics projects. The toolbox provides access to genomic and proteomic data formats, analysis techniques, and specialized visualizations for genomic and proteomic sequence and microarray analysis. Most functions are implemented in the open MATLAB language, enabling you to customize the algorithms or develop your own.

'''NCBI ToolBox''' http://www.ncbi.nlm.nih.gov/IEB/ToolBox/MainPage/index.html

The NCBI ToolBox model and code is used extensively within NCBI for the internal pipelines and tools such as GenBank, Entrez, BLAST, Sequin, OMIM, RefSeq, and others. These tools are supported in the sense that they are designed to work in many environments outside NCBI. The NCBI ToolBox consists of three major parts:

* Data Model - An explicit data model of biological sequences, structures, bibliographic data, and associated annotations.
* Data Encoding - A formal specification and encoding rules. The telecommunications standard, ASN.1, has been used for this. Recently it has been mapped to a similar language, XML.
* Programming Libraries - Originally written in a portable dialect of C. Recently a new generation is being written in C++.

'''Entrez Programming Utilities (eUtils)''' http://www.ncbi.nlm.nih.gov/

The Entrez Programming Utilities (eUtils[http://eutils.ncbi.nlm.nih.gov/entrez/query/static/eutils_help.html]
) are a set of seven server-side programs that provide a
stable interface into the Entrez query and database system at the National Center for Biotechnology
Information (NCBI). The eUtils use a fixed URL syntax that translates a standard set of input
parameters into the values necessary for various NCBI software components to search for and
retrieve the requested data. The eUtils are therefore the structured interface to the Entrez system,
which currently includes 23 databases covering a variety of biomedical data, including nucleotide
and protein sequences, gene records, three-dimensional molecular structures, and the biomedical
literature.
To access these data, a piece of software first posts an eUtils URL to NCBI, then retrieves the
results of this posting, after which it processes the data as required. The software can thus use any
computer language that can send a URL to the eUtils server and interpret the XML response;
examples of such languages are Perl, Python, Java, and C++. Combining eUtils components to
form customized data pipelines within these applications is a powerful approach to data
manipulation.

*Read More about eUtils[http://www.ncbi.nlm.nih.gov/books/bv.fcgi?rid=coursework.TOC&depth=2]

'''SLRI Toolkit''' http://www.blueprint.org/products/toolkit/index.html

The Samuel Lunenfeld Research Institute (SLRI) Toolkit is a cross-platform toolkit for manipulating biological information. The SLRI toolkit is based mainly in C and derives many functions from the NCBI toolkit. The SLRI toolkit was developed mainly for data pertaining to protein structure and function but can be used to manipulate other data such as gene sequences.

'''Molecular Biology Toolkit (MBT)''' http://mbt.sdsc.edu/

The Molecular Biology Toolkit (MBT) is a Java-based protein visualization and analysis toolkit. The toolkit provides classes for efficiently loading, managing and manipulating protein structure and sequence data. The MBT also provides a rich set of graphical 3D and 2D visualization components which can be easily "plugged together" to produce applications having sophisticated graphical user interfaces. Yet, with all of the GUI components provided in the toolkit, the core data i/o and manipulation classes may be used to write completely non-graphical applications (say, for implementing pure analysis codes, or, for producing a non-graphical "back end" for web-based applications).

----

<html>
<a href="http://www.ploscompbiol.org"><img src="http://www.plos.org/images/pcbi_234x60.png" alt="PLoS Computational Biology - www.ploscompbiol.org"  border="0"></a>
<a href="http://www.plos.org/journals/"><img src="http://www.plos.org/images/banners/library_button.gif" alt="I Support the Public Library of Science"  border="0"></a>
<a href="http://www.biomedcentral.com/home/"><img src="http://www.biomedcentral.com/graphics/advocacy/bmclogo2.gif" alt="Support Open Access. Spread the Word"  border="0"></a>
<a href="http://www.iscb.org/"><img src="http://iscb.org/images/index_r3_c1.gif" alt=" The International Society for Computational Biology (ISCB)"  border="0"></a>

</html>