[[Image:Abhishek_tiwari.PNG|750px]]
<div style="padding: 10px; color: #ffffff; background-color: #000; width: 730px">
<center>
[[Abhishek Tiwari | <font face="trebuchet ms" style="color:#ffffff"> '''Home''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:About | <font face="trebuchet ms" style="color:#ffffff"> '''About''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Resources| <font face="trebuchet ms" style="color:#ffffff"> '''Resources''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Projects | <font face="trebuchet ms" style="color:#ffffff"> '''Research & Projects''' </font>]] 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Softwares | <font face="trebuchet ms" style="color:#ffffff"> '''Softwares''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Reprints | <font face="trebuchet ms" style="color:#ffffff"> '''Publications''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Links | <font face="trebuchet ms" style="color:#ffffff"> '''ImpLinks''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Contact | <font face="trebuchet ms" style="color:#ffffff"> '''Contact''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</center>
</div>


'''Powered by'''  [[Image:Sflogo.png]]   [[Image:Gnu-head-banner.png]]   

= Developed Softwares/Tools/Programs =

== HBAT(Hydrogen Bond Analysis Tool) ==

The program HBAT, is a tool to automate the analysis of potential hydrogen bonds and similar type of weak interactions like halogen bonds and non-canonical Ï€ interactions in macromolecular structures, available in Brookhaven Protein Database (PDB) file format. HBAT is written using PERL and TK languages. The program generates an MSOFFICE Excel compatible output file for statistical analysis. HBAT identify potential interactions based on geometrical criteria. A series of analysis reports like frequency tables, geometry distribution tables, furcations list are generated. A user friendly GUI offers freedom to select several parameters and options. Graphviz based visualization of hydrogen bond networks in 2D helps to study the cooperativity and anticooperativity geometry in hydrogen bond. HBAT supports post docking interaction analysis between PDB files for any target/receptor (in PDB files) and docked ligands/poses (in SDF). This tool can be implemented in active site interaction analysis, structure based drug design and molecular dynamics simulations.

Availability https://sourceforge.net/projects/hbat

==HBNG(Hydrogen Bond Network Graph) ==

HBNG is a graph theory based tool for visualization of Hydrogen Bond Network in 2D. Digraphs generated by HBNG give easy visualization of cooperativity and anticooperativity chains & rings in Protein Structures. HBNG takes Hydrogen Bonds list files (Output from HBAT, HBexplore and HBPLUS) as input and generated a DOT language script and draw digraphs using free ware AT&T Graphviz tool. HBNG is useful in study of enumeration of favorable topologies of hydrogen bond networks in protein structures, effect of cooperativity and anticooperativity on protein stability and folding.

Availability https://sourceforge.net/projects/hbat 

== MeltDNA ==

Prediction of DNA duplex stability and thermodynamics is invaluable for many molecular biology applications involving sequence dependent hybridization reactions. Sequence dependent stability of duplex DNA plays a major role in fundamental processes of the living cell, such as replication, transcription, and recombination. Many techniques in molecular biology depend on the oligonucleotide melting temperature (Tm), and several formulas have been developed to estimate Tm. We have developed a Perl based tool to predict DNA duplex hybridization & melting thermo- dynamics. This tool consider all factors those determine thermodynamic values like-Base composition,NN parameters, salt dependency (Na+, Mg++), internal mismatch, dangling ends ,loops etc. For improve NN model based calculations we introduced fusion matrices. MeltDNA provides better prediction accuracy for DNA duplex hybridization especially its ability to handle the structural features in DNA duplex like loops, loop size etc. It has got wide applicability in primer selection in PCR, probe selection in Microarrays, DNA secondary structure stability prediction, DNA computing etc.

Availability https://sourceforge.net/projects/meltdna

== PMSAT(Peptide MS Analysis Tool)==

PMSAT is a project to develop an integrated software tool for Mass Spectroscopy analysis of peptides. PMSAT-Peptide MS Analysis Tool determines the peptide composition for the query mass and also generates fragmentation pattern for Peptide Sequence. It finds utility in checking whether the denovo sequence established accurately matches the values obtained from the mass spectra. This software determines the likely amino acid composition and not the sequence. It will also find Peptide masses from enzymatic digestion of protein,AA compositions fitting parent or fragment mass and immonium ions and Isotope patterns of peptides.

Availability https://sourceforge.net/projects/peptide

=Open Source Links=
*SourceForge.net http://sourceforge.net/
*GNU http://www.gnu.org/
*The Free Software Foundation http://www.fsf.org/
*Open Source Initiative (OSI) http://www.opensource.org/


----

<html>
<a href="http://www.ploscompbiol.org"><img src="http://www.plos.org/images/pcbi_234x60.png" alt="PLoS Computational Biology - www.ploscompbiol.org"  border="0"></a>
<a href="http://www.plos.org/journals/"><img src="http://www.plos.org/images/banners/library_button.gif" alt="I Support the Public Library of Science"  border="0"></a>
<a href="http://www.biomedcentral.com/home/"><img src="http://www.biomedcentral.com/graphics/advocacy/bmclogo2.gif" alt="Support Open Access. Spread the Word"  border="0"></a>
<a href="http://www.iscb.org/"><img src="http://iscb.org/images/index_r3_c1.gif" alt=" The International Society for Computational Biology (ISCB)"  border="0"></a>
</html>