[[Image:Abhishek_tiwari.PNG|750px]]
<div style="padding: 10px; color: #ffffff; background-color: #000; width: 730px">
<center>
[[Abhishek Tiwari | <font face="trebuchet ms" style="color:#ffffff"> '''Home''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:About | <font face="trebuchet ms" style="color:#ffffff"> '''About''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Resources| <font face="trebuchet ms" style="color:#ffffff"> '''Resources''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Projects | <font face="trebuchet ms" style="color:#ffffff"> '''Research & Projects''' </font>]] 
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Softwares | <font face="trebuchet ms" style="color:#ffffff"> '''Softwares''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Reprints | <font face="trebuchet ms" style="color:#ffffff"> '''Publications''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Links | <font face="trebuchet ms" style="color:#ffffff"> '''ImpLinks''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Abhishek Tiwari:Contact | <font face="trebuchet ms" style="color:#ffffff"> '''Contact''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</center>
</div>
= PROTEOMICS =
'''''Proteomics''''' Volume 7 | Issue S1 | SEPTEMBER 2007

*'''High-performance Proteomics as a Tool in Biomarker Discovery'''

'''Synopsis'''

Biomarkers allowing early detection of disease or therapy control have a huge influence in curing a disease. A wide variety of methods were applied to find new biomarkers. In contrast to methods focused on DNA or mRNA techniques, approaches considering proteins as potential biomarker candidates have the advantage that proteins are more diverse than DNA or RNA and are more reflective of a biological system. Here, we present an approach for the identification of new biomarkers relying on our experience from the past 10 years of proteomics, outlining a concept of "high-performance proteomics" This approach is based on quantitative proteome analysis using a sufficient number of clinical samples and statistical validation of proteomics data by independent methods, such as Western blot analysis or immunohistochemistry.

'''''PLoS Computational Biology''''' Volume 3 | Issue 7 | JULY 2007

*'''Introduction to Computational Proteomics'''

'''Synopsis'''

Proteomics plays an ever-increasing and pivotal role in biological research, and there are a range of technologies available that can generate large quantities of data. The analysis of such data opens new and challenging areas of interest for bioinformatics. In addition to the utilisation of classical methods and resources, new types of data require modelling and processing. Perhaps the best example is the mass spectrum itself, which contains continuous and discrete information simultaneously. Such issues are reflected in the difficulty of designing high-performance scoring functions and de novo sequencing algorithms.

To provide an introduction to this fascinating field of research, we have presented general concepts of proteomics. The central problem of MS data identification by database searching has been explained at an introductory level, and should allow any interested reader to grasp the fundamental concepts of this area of research. 

'''''BMC Bioinformatics''''' Volume 7 |  29 October 2006
*'''HybGFS: a hybrid method for genome-fingerprint scanning'''

'''Synopsis'''

Protein identification based on mass spectrometry (MS) has previously been performed using peptide mass fingerprinting (PMF) or tandem MS (MS/MS) database searching. However, these methods cannot identify proteins that are not already listed in existing databases. Moreover, the alternative approach of de novo sequencing requires costly equipment and the interpretation of complex MS/MS spectra. Thus, there is a need for novel high-throughput protein-identification methods that are independent of existing predefined protein databases.A hybrid method for genome-fingerprint scanning, known as HybGFS is reported by Kosaku Shinoda, etl. This technique combines genome sequence-based peptide MS/MS ion searching with liquid-chromatography elution-time (LC-ET) prediction, to improve the reliability of identification. The hybrid method allows the simultaneous identification and mapping of proteins without a priori information about their coding sequences. The current study used standard LC-MS/MS data to query an in silico-generated six-reading-frame translation and the enzymatic digest of an entire genome. Used in conjunction with precursor/product ion-mass searching, the LC-ETs increased confidence in the peptide-identification process and reduced the number of false-positive matches. The power of this method was demonstrated using recombinant proteins from the Escherichia coli K12 strain.The novel hybrid method described  will be useful for the large-scale experimental confirmation of genome coding sequences, without the need for transcriptome-level expression analysis or costly MS database searching.