== Affymetrix One Cycle Eukaryotic Gene Expression Sample Processing ==

This protocol will allow you to perform the labelling of Eukaryotic total RNA ready for Affymetrix genechip analysis. This protocol is a supplement to instructions provided in the Affymetrix Expression Manual and uses reagents in the Affymetrix One Cycle Synthesis kit. You can use from 1 to 15ug of starting total RNA. '''This protocol assumes you are using 5ug starting total RNA'''

==RNA sample quality==
RNA needs to be high quality for this protocol refer to the [[RNA Quality Control|RNA QC]] page.

==Workflow==

==Materials==
[http://www.affymetrix.com/products/reagents/specific/one_cycle_target_control.affx One-Cycle Target Labeling and Control Reagents], Affymetrix, P/N 900493
This is the preferred way to get all the reagents you need. This pack is suitable for 30 reactions from 1-15ug of starting total RNA. Each of these components may be ordered individually (described below) as well as in this complete kit.
Contains:
*1 Poly-A RNA Control Kit (Affymetrix, P/N 900433)
*1 One-Cycle cDNA Synthesis Kit (Affymetrix, P/N 900431)
*1 Sample Cleanup Module (Affymetrix, P/N 900371)
*1 IVT Labeling Kit (Affymetrix, P/N 900449)
*1 Hybridization Control Kit (Affymetrix, P/N 900454)

Do not store enzymes in a frost-free freezer.


Miscellaneous Reagents
*Absolute ethanol (stored at -20°C for RNA precipitation; store ethanol at room temperature for use with the GeneChip Sample Cleanup Module)
*80% ethanol (stored at -20°C for RNA precipitation; store ethanol at room temperature for use with the GeneChip Sample Cleanup Module)
*Glycogen, Ambion, P/N 9510 (optional)
*3M Sodium Acetate (NaOAc), Sigma-Aldrich, P/N S7899
*1N NaOH
*1N HCl


Optional extra enzymes from the One Cycle cDNA synthesis kit are obtainable from Invitrogen
*SuperScript™ II, Invitrogen Life Technologies, P/N 18064-014
*E. coli DNA Ligase, Invitrogen Life Technologies, P/N 18052-019
*E. coli DNA Polymerase I, Invitrogen Life Technologies, P/N 18010-025
*E. coli RNaseH, Invitrogen Life Technologies, P/N 18021-071
*T4 DNA Polymerase, Invitrogen Life Technologies, P/N 18005-025
*5x Second-strand buffer, Invitrogen Life Technologies, P/N 10812-014
*10 mM dNTP, Invitrogen Life Technologies, P/N 18427-013

==cDNA Synthesis==

=== Poly-A Controls ===
Poly-A RNA Control Stock and Poly-A Control Dil Buffer come with the complete Affymetrix One Cycle Kit.
Prepare enough of the third dilution for all the samples processing that day. Make separate mixes if jobs are using different starting anmounts of RNA. Usually you will be using 5ug of RNA.

{| class="wikitable" style="margin: 1em auto 1em auto"
! Starting RNA || 1st Dilution || 2nd Dilution || 3rd Dilution
|-
| 1 ug  || 1:20 || 1:50 || 1:50
|-
| 5 ug  || 1:20 || 1:50 || 1:10
|-
| 10 ug  || 1:20 || 1:50 || 1:5
|}

For example, to prepare the poly-A RNA dilutions for 20x 5 µg total RNA samples:
#In tube A: Add 2 µL of the Poly-A Control Stock to 38 µL of Poly-A Control Dil Buffer (Mix thoroughly and spin down).
#In tube B: Add 2 µL of the First Dilution from tube A to 98 µL of Poly-A Control Dil Buffer (Mix thoroughly and spin down).
#In tube C: Add 2 µL of the Second Dilution from tube B to 18 µL of Poly-A Control Dil Buffer (Mix thoroughly and spin down).

You will use 2 µL of this final dilution in tube C for each sample starting with 5µg of total RNA.

''The First Dilution of the poly-A RNA controls, tube A, can be stored up to six weeks in a nonfrost-free freezer at -20°C and frozen-thawed up to eight times.''

=== First-Strand cDNA Synthesis ===
One-Cycle cDNA Synthesis Kit is used for this step.
Briefly spin down all tubes in the Kit before using the reagents.
Perform all of the incubations in thermal cyclers. The following program is used to perform the first-strand cDNA synthesis reaction in a thermal cycler; the 4°C holds are for reagent addition steps:

Protocol Name: AFFY1CYC

#70°C 10 minutes
#4°C hold
#42°C 2 minutes
#42°C 1 hour
#4°C hold
#16°C 2 hour
#4°C hold
#16°C 5 minutes
#4°C hold
#End

Use AffyRxnSetUpcDNA_Template spreadsheet for calculations using Nanodrop data. (I need to put a file download link here to the reaction setup spreadsheet and I am not sure how to direct that or where to direct it to!)

{| class="wikitable" style="margin: 1em auto 1em auto"
! || Amount
|-
| Sample RNA (5µg)  || variable
|-
| Diluted poly-A RNA controls (Tube C from above link to poly a controls bit)  || 2 µL
|-
| T7-Oligo(dT) Primer, 50 µM  || 2 µL 
|-
| RNase-free Water (to 12µl) || variable
|-
| Total Volume || 12 µL
|}

	
#Place total RNA in a 0.2 mL PCRtube.
#Add poly-A RNA controls, 50 µM T7-Oligo(dT) Primer and RNase-free Water to a final volume of 12 µL, as directed in the table above. Gently flick the tube a few times to mix, and then centrifuge briefly (~5 seconds) to collect the reaction at the bottom of the tube.
#Incubate the reaction for 10 minutes at 70°C and then cool the sample for 2 minutes at 4°Cl (using PCR machine and protocol as above).


'''Make First-Strand Master Mix.'''
Prepare sufficient First-Strand Master Mix for all of the RNA samples. When there are more than 2 samples, it is prudent to include additional material to compensate for potential pipetting inaccuracy or solution lost during the process. The following recipe is for a single reaction. 

{| class="wikitable" style="margin: 1em auto 1em auto"
! Reagent || Amount
|-
|5X 1st Strand Reaction Mix||4 µL
|-
| DTT, 0.1M||2 µL	
|-	
| dNTP, 10 mM||1 µL 
|-
| Superscript II||2 µL	
|-
| Total Volume || 9 µL
|}


#Add reagents in order to a clean eppendorf tube. Mix well by flicking the tube a few times. Centrifuge briefly (~5 seconds) to collect the master mix at the bottom of the tube.
#Add master mix to the reaction tubes in the PCR machine and mix by pipetting
#Incubate for 1 hour at 42°C; then cool the sample for at least 2 minutes at 4°C.
#Proceed to Second-Strand cDNA Synthesis


=== Second-Strand cDNA Synthesis ===
Second-Strand cDNA Synthesis 
It is recommended to prepare Second-Strand Master Mix immediately before use.

One-Cycle cDNA Synthesis Kit is used for this step.
Make Second-Strand Master Mix.
Prepare sufficient Second-Strand Master Mix for all of the samples. When there are more than 2 samples, it is prudent to include additional material to compensate for potential pipetting inaccuracy or solution lost during the process. The following recipe is for a single reaction.

{| class="wikitable" style="margin: 1em auto 1em auto"
! Reagent || Amount
|-
|RNase-free Water||91 µL
|-
| 5X 2nd Strand Reaction Mix||30 µL	
|-	
| dNTP, 10 mM||3 µL 
|-
| E. coli DNA ligase||1 µL	
|-
| E. coli DNA Polymerase||4 µL 
|-
| RNase H||1 µL	
|-| Total Volume || 130 µL
|}

#Add reagents in order to a clean eppendorf tube. Mix well by flicking the tube a few times. Centrifuge briefly (~5 seconds) to collect the master mix at the bottom of the tube.
#Add 130ul master mix to the reaction tubes in the PCR machine and mix by pipetting.
#Incubate for 2 hours at 16°C.
#Add 2 µL of T4 DNA Polymerase to each sample and incubate for 5 minutes at 16°C.
#After incubation with T4 DNA Polymerase add 10 µL of EDTA, 0.5M and proceed directly to Cleanup of Double-Stranded cDNA (link) or store cDNA syntheis reactions at -20°C.
#Proceed to Cleanup of Double-Stranded cDNA.

''Do not leave the reactions at 4°C for long periods of time.Incubate for 1 hour at 42°C; then cool the sample for at least 2 minutes at 4°C.''

=== Cleanup of Double-Stranded cDNA ===

The Sample Cleanup Module (link) is used for cleaning up double-stranded cDNA.
All components, except 100% Ethanol, needed for cleanup of double-stranded cDNA are supplied with the GeneChip Sample Cleanup Module.

'''BEFORE STARTING, please note:'''
*You will need to transfer cDNA reactions from 200µl tubes into labeled 1.5ml tubes. ''Label 1.5ml tubes on the side '''AND''' the lid''
*cDNA Wash Buffer is supplied as a concentrate. Before using for the first time, add 24 mL of ethanol (96-100%), as indicated on the bottle, to obtain a working solution, and checkmark the box on the left-hand side of the bottle label to avoid confusion.
*All steps of the protocol should be performed at room temperature. During the procedure, work without interruption. If the color of the mixture is orange or violet, add 10 µL of 3M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow.


#	Add 600 µL of cDNA Binding Buffer to the double-stranded cDNA synthesis preparation. Mix by shaking well. (Check that the color of the mixture is yellow similar to cDNA Binding Buffer without the cDNA synthesis reaction).
#	Apply the whole sample (750µL) to the cDNA Cleanup Spin Column sitting in a 2 mL Collection Tube (supplied), and centrifuge for 1 minute at ≥ 8,000 x g (≥ 10,000 rpm). Discard flow-through.
#	Transfer spin column into a new 2 mL Collection Tube (supplied).
#	Pipet 750 µL of the cDNA Wash Buffer onto the spin column. Centrifuge for 1 minute at ≥ 8,000 x g (≥ 10,000 rpm). Discard flow-through.
#	Open the cap of the spin column and centrifuge for 5 minutes at maximum speed (≤ 25,000 x g). Discard flow-through and Collection Tube. Place columns into the centrifuge using every second bucket. Position caps over the adjoining bucket so that they are oriented in the opposite direction to the rotation. (Label the collection tubes with the sample name. During centrifugation some column caps may break, resulting in loss of sample information.)
#	Transfer spin column into a 1.5mL Collection Tube, and pipet 23µL of cDNA Elution Buffer directly onto the spin column membrane. Incubate for 1 minute at room temperature and centrifuge 1 minute at maximum speed (≤ 25,000 x g) to elute. Ensure that the cDNA Elution Buffer is dispensed directly onto the membrane. The average volume of eluate is 20µL from 23µL Elution Buffer. (The volume of elution buffer is increased over the Affymetrix protocol to allow the same tube to be used for IVT in the next step with no water needed.)
#Proceed to Synthesis of Biotin-Labeled cRNA

cDNA can be kept at -20°C for long term storage.

== cRNA in-vitro transcription==
=== Synthesis of Biotin-Labeled cRNA ===
IVT Labeling Kit (link) is used for this step.
Use only nuclease-free water, buffers, and pipette tips.
Store all reagents in a -20°C freezer that is not self-defrosting. Prior to use, centrifuge all reagents briefly to ensure that the solution is collected at the bottom of the tube.

'''BEFORE STARTING, please note:'''
*Use all 20ul of cDNA for each IVT reaction (Half reactions do work equally well). Perform reactions in the original cDNA elution tubes.
*The Target Hybridizations (link) and Array Washing (link) protocols have been optimized specifically for this IVT Labeling Protocol.
*


'''Make IVT Master Mix.''' (Do not assemble the reaction on ice, since spermidine in the 10X IVT Labeling Buffer can lead to precipitation of the template cDNA.)
*Prepare sufficient IVT Master Mix for all of the samples. When there are more than 2 samples, it is prudent to include additional material to compensate for potential pipetting inaccuracy or solution lost during the process. The following recipe is for a single reaction.

{| class="wikitable" style="margin: 1em auto 1em auto"
! Reagent || Amount
|-
|Template cDNA variable*||Usually 20 µL
|-
| RNase-free Water variable**||Usually 0 µL	
|-	
| 10X IVT Labeling Buffer||4 µL 
|-
| IVT Labeling NTP Mix||12 µL	
|-
| IVT Labeling Enzyme Mix||4 µL 
|-
| Total Volume || 40 µL
|}
"*"		Use the entire cDNA template from a 23ul elution.
"**" 	You should not need any water for a standard reaction

#Make the master mix of the required size, mix by shaking and then briefly spin (~5 seconds).
#Add 20µL of IVT master mix to each cDNA reaction tube. label the tube to indicate that this is an IVT reaction now!!!
#Incubate at 37°C for 16 hours. To prevent condensation that may result from water bath-style incubators, incubations are best performed in oven incubators for even temperature distribution, or in a thermal cycler, or in thermomixer.
#Proceed to Cleanup and Quantification of Biotin-Labeled cRNA (link)

cRNA can be kept at -20°C, or -70°C for long term storage.

=== Cleanup and Quantification of Biotin-Labeled cRNA ===
Cleanup and Quantification of Biotin-Labeled cRNA

Sample Cleanup Module (link) is used for cleaning up the biotin-labeled cRNA.
Reagents to be Supplied by User
Ethanol, 96-100% (v/v)
Ethanol, 80% (v/v)

'''BEFORE STARTING, please note:'''
*It is essential to remove unincorporated NTPs, so that the concentration and purity of cRNA can be accurately determined by 260 nm absorbance.
*DO NOT extract biotin-labeled RNA with phenol-chloroform. The biotin will cause some of the RNA to partition into the organic phase. This will result in low yields.
*Save an aliquot of the unpurified IVT product for analysis by gel electrophoresis.
*IVT cRNA Wash Buffer is supplied as a concentrate. Before using for the first time, add 20 mL of ethanol (96-100%), as indicated on the bottle, to obtain a working solution, and checkmark the box on the left-hand side of the bottle label to avoid confusion.
*IVT cRNA Binding Buffer may form a precipitate upon storage. If necessary, redissolve by warming in a water bath at 30°C, and then place the buffer at room temperature.
*All steps of the protocol should be performed at room temperature.
*During the procedure, work without interruption.


#	Add 60 µL of RNase-free Water to the IVT reaction and mix by vortexing for 3 seconds.
#	Add 350 µL IVT cRNA Binding Buffer to the sample and mix by vortexing for 3 seconds.
#	Add 250 µL ethanol (96-100%) to the lysate, and mix well by pipetting. Do not centrifuge.
#	Apply sample (700 µL) to the IVT cRNA Cleanup Spin Column sitting in a 2 mL Collection Tube. Centrifuge for 15 seconds at ≥ 8,000 x g (≥ 10,000 rpm). Discard flow-through and Collection Tube.
#	Transfer the spin column into a new 2 mL Collection Tube (supplied).
#	Pipet 500 µL IVT cRNA Wash Buffer onto the spin column. Centrifuge for 15 seconds at ≥ 8,000 x g (≥ 10,000 rpm) to wash. Discard flow-through.
#	Pipet 500 µL 80% (v/v) ethanol onto the spin column and centrifuge for 15 seconds at ≥ 8,000 x g (≥ 10,000 rpm). Discard flow-through.
#	Open the cap of the spin column and centrifuge for 5 minutes at maximum speed (≤ 25,000 x g). Discard flow-through and Collection Tube. (Label the collection tubes with the sample name. During centrifugation some column caps may break, resulting in loss of sample information.)
#	Place columns into the centrifuge using every second bucket. Position caps over the adjoining bucket so that they are oriented in the opposite direction to the rotation (i.e., if the microcentrifuge rotates in a clockwise direction, orient the caps in a counterclockwise direction). This avoids damage of the caps. Centrifugation with open caps allows complete drying of the membrane.
#	Transfer spin column into a new 1.5 mL Collection Tube (supplied), and pipet 50µL of RNase-free Water directly onto the spin column membrane. Ensure that the water is dispensed directly onto the membrane. Centrifuge 1 minute at maximum speed (≤ 25,000 x g) to elute. (This volume of water is significantly different to the Affymetrix manual but works very well, concentrations should be around 1ug/ul)
#Proceed to Quality Control of cRNA (link).

cRNA can be kept at -20°C, or -70°C for long term storage.

===Quality Control of cRNA===
[[RNA Quality Control|RNA QC]]

*Concentrations of cRNA are generally around 1000-1500ng/µl for a 40µl IVT reaction using 20µl of cDNA template.

 For quantification of cRNA when using total RNA as starting material, an adjusted cRNA <br>yield must be calculated to reflect carryover of unlabeled total RNA. Using an estimate of <br>100% carryover, use the formula below to determine adjusted cRNA yield:
adjusted cRNA yield = RNAm - (total RNAi) (y) where:
RNAm = amount of cRNA measured after IVT (µg)
total RNAi = starting amount of total RNA (µg)
y = fraction of cDNA reaction used in IVT

Example: Starting with 5 µg total RNA, 100% of the cDNA reaction is added to the IVT, giving a yield of 50 µg cRNA. Therefore, adjusted cRNA yield = 50 µg cRNA - (5 µg total RNA) (1 cDNA reaction) = 45.0 µg.
*Use adjusted yield and proceed to Fragmenting cRNA for hybridisation

===Fragmenting cRNA for hybridisation===
*Sample Cleanup Module (link) is used for cleaning up the biotin-labeled cRNA.
*The Fragmentation Buffer has been optimized to break down full-length cRNA to 35 to 200 base fragments by metal-induced hydrolysis. Fragmentation of cRNA target before hybridization onto GeneChip probe arrays has been shown to be critical in obtaining optimal assay sensitivity.
Affymetrix recommends that the cRNA used in the fragmentation procedure be sufficiently concentrated to maintain a small volume during the procedure. This will minimize the amount of magnesium in the final hybridization cocktail. Fragment an appropriate amount of cRNA for hybridization cocktail preparation and gel analysis.
*Use adjusted cRNA concentration, as described above in Quality Control of cRNA.
*The following program is used to perform the first-strand cDNA synthesis reaction in a thermal cycler; the 4°C holds are for reagent addition steps:

Protocol Name: FRAGAFFY

#94°C 35 minutes
#4°C hold
#End

''Use AffyRxnSetUpcRNAIVT_Template spreadsheet for calculations using Nanodrop data and adjusted yields. (I need to put a file download link here to the reaction setup spreadsheet and I am not sure how to direct that or where to direct it to!)''

 
The following table shows suggested fragmentation reaction mix for cRNA samples at a final concentration of 0.5µg/µL.

{| class="wikitable" style="margin: 1em auto 1em auto"
! Reagent || Amount
|-
|cRNA variable*||Usually 20 µg
|-
| RNase-free Water (to 40µL)||variable	
|-	
| 5x Fragmentation Buffer||8 µL 
|}

*Mix reagents in individual 200µL PCR tubes or 96well PCR plate.
*Incubate at 94°C for 35 minutes. Put on ice following the incubation.
*Proceed to [[Affymetrix Target Hybridization|target hybridization.]]
You can save an aliquot for analysis on the BioAnalyser, typically this is not necessary. A typical fragmented target is shown below. The standard fragmentation procedure should produce a distribution of RNA fragment sizes from approximately 35 to 200 bases. '''High quality cRNA does not nomally need further QC analysis after Agilent BioAnalyser QC.'''
Undiluted, fragmented sample cRNA can be kept -20°C (or -70°C for long-term storage).

==Notes==
#List troubleshooting tips here.  
#You can also link to FAQs/tips provided by other sources such as the manufacturer or other websites.
#Anecdotal observations that might be of use to others can also be posted here.  

Please sign your name to your note by adding <font face="courier"><nowiki>('''~~~~''')</nowiki></font> to the end of your tip.

==References==
<!-- Have you referenced material in this protocol? Do not plagarise and list those references here.  See the [[OpenWetWare:Biblio]] page for more information. -->
<biblio>
# none
</biblio>

==Contact==
*mail to jamesdothadfieldatcancerdotorgdotuk
*This page was created by James Hadfield on 12 October 2006. I hope you found it useful.

[[Category:Protocol]]
[[Category:In vitro]]
[[Category:RNA]]