= 9/19/05 - 9/23/05 =
Goal: make list of all features on chromosome III, start to figure out 5' and 3' ends required to specify a "functional element". 

Next week: [[AlexLabNotebook/ChrIIIRebuild/9/24/05-9/30/05|9/24/05-9/30/05]]
== To-do list/Questions ==

=== ChrIII specific ===
* Get list of types of features -- ''done''.
* Determine position of ARS, centromere  -- ''done''.
* Make list of genes with introns -- ''done''.
* Determine whether there are any polycistronic transcripts -- ''done''
* Make list of overlapping functional elements -- ''done''.
* For each feature, determine 3' and 5' flanking sequences that are needed [eg promoters and 3' UTR for genes] --> this is a ''functional element''
** Determine transcription start and end sites for Pol I, II and III genes
** Figure out what cis-acting sequences are required for functions of all other types of elements
* Determine whether orientation matters for HML, HMR and MAT loci.
* Make list of essential genes.  
* Find genes controlled by same TF that are divergently transcribed.


=== General ===
* If 2 genes controlled by same TF are divergently transcribed, doesn't separating them while preserving the promoter sequence mean that you now need 2x the concentration of TF as before, because you've duplicated the binding sites of the TF ? Alternatively, given the large number of alternative binding sites that already exist, maybe having one more is really just a very minor change in the concentration of TF seen by the "real" binding site ...
* Is there any known correlation between whether a gene is on the leading/lagging strand and its mutation rate, expression level etc ? In other words, is there a model for what could/should happen if a gene is moved from the plus strand to the minus strand, or vice versa ?
* Consider taking out pseudogenes, introns [have to be careful not to eliminate snoRNAs that are made from introns].

== Data ==
=== Features ===
* Chr III features:
** repeat_family (6): [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=TEL03L-TR Terminal bits of telomeres], [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=TEL03L-XR X-element combinatorial repeats] and [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=TEL03L-XC X-element core sequence]. Occur on both right and left ends of chromosome. 
** telomere (2): Full [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=TEL03L left] and [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=TEL03R right] telomeres.
** nucleotide_match (17): [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=ARS300 ARS] and [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=TEL03L-XC X-element] core sequences. 
** binding_site (2): [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=TEL03L-XC X-elements] which contain Tbp1 binding site. 
** ARS (19): Full [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=ARS300 ARS] sequence, not just consensus sequence.  
** repeat_region (19): LTR of Ty 1, 2, 4 and 5 elements, like [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCLWomega1 YCLWomega1].
** transposable_element (2): Full-length Ty5 and Ty2 elements, like [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCLWTy5-1 YCLWTy5-1]. 
** gene (182): Full sequence for gene, spanning all introns and exons. 
** CDS (195): Coding sequences for "real" genes, pseudogenes and transposable element genes;   each exon is considered a separate coding sequence.  
** pseudogene (2): [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCL075W YCL075W] and [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCL074W YCL074W] encode fragments of Ty Pol protein.
** region (7): [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=HMRA1 HMRa], [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=HMLalpha1 HMLalpha], [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=MATalpha1 MATalpha] and consensus elements CDEI, CDEII and CDEIII of [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=CEN3 centromere]  
** tRNA (10): Full tRNAs; some have introns, eg [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=tL(CAA)C tRNA-Leu]
** ncRNA (16): tRNAs and snoRNAs, like [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=snR65 snR65].
** transposable_element_gene (2): Genes encoded by Ty elements, like [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCL020W YCL020W].
** intron (10): Introns of 8 genes and 2 tRNAs. 
** snoRNA (4): snoRNAs. 
** centromere (1): Full centromeric sequence for [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=CEN3 CEN3].

* Elements that contain introns:
** [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=tL(CAA)C tL(CAA)C]
** [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=tS(CGA)C tS(CGA)C]
** [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCL012C YCL012C]
** [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCL005W-A YCL005W-A]
** [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCL002C YCL002C]
** [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCR028C-A YCR028C-A]
** [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCR031C YCR031C]
** [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCR097W YCR097W aka HMRa]

* Some snoRNAs are polycistronic; see, for example, [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=9649442&query_hl=1 Chanfreau et al].

=== 3' and 5' ends ===
* Pol I is only required for rRNA gene transcription; [http://db.yeastgenome.org/cgi-bin/locus.pl?locus=YCR003W YCR003W] is only rRNA gene on chromosome III. 
* [Textbook] Pol II promoters are ~40bp long, up/downstream of transcription start site, consist of 
**TFIIB responsive element [BRE], (G/C)(G/C)(G/A)CGCCC, -37 to -32
**TATA box, TATA(A/T)(A/T), -31 to -26
**initiator, (C/T)(C/T)AN(T/A)(C/T)(C/T), -2 to +4
**downstream promoter elements [DPE], (A/G)G(A/T)CGTG, +28 to +32 
**(all numbers relative to transcription start site)
* [Textbook] Transcription of Pol II genes is terminated by polyA signal sequences that signal cleavage and addition of the polyA tail.
* [Textbook] Pol III promoters tend to be downstream of transcription start sites. Have Box A and Box B motifs.
* [http://bighost.area.ba.cnr.it/BIG/UTRHome/ UTRdb] is database of published 3' and 5' UTRs.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15905473&itool=iconpmc&query_hl=3 Zhang and Dietrich paper] describes experimental mapping of transcription start sites. Raw data available [http://data.cgt.duke.edu/5sage.php here].
* 3' end formation for Pol II genes:
** [http://harlequin.jax.org/polyA/ 3' UTR prediction website], has downloadable data on predicted 3' UTRs for yeast genes.
*** Downloadable data doesn't have UTRs for all genes because some genes didn't have a predicted 3' UTR that scored above the cutoff. In particular, 98 genes on chromosome III don't have predicted 3' UTRs, so maybe the thing to do for these is to manually run the prediction and pick the most probable site, or just choose a particular length of UTR for those genes, like the max UTR length found so far.
*** Also, downloadable data has some strange UTR lengths eg genes YBR264C, YFR026C etc supposedly have a 3' UTR of length 1. 
** [http://mmbr.asm.org/cgi/reprint/63/2/405 Review of mRNA 3' end formation]
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=9009831&itool=iconabstr&query_hl=9 Guo & Sherman] review of cis elements directing 3' end formation; contains consensus sequences.
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=9889288&itool=iconpmc&query_hl=12 Graber et al] bioinformatic analysis of 3'-end-processing signals; contains consensus sequences, ranked by strength.

=== Overlapping elements ===
* Top-level feature types:
** Telomere [includes repeat_family, binding_site, some of the nucleotide_match elements]
** ARS [includes rest of nucleotide_match elements]
** Transposable_element [includes some repeat_regions, pseudogene elements]
** repeat_region 
** Gene [includes CDS elements, some region elements]
** tRNA [includes some ncRNA elements]
** snoRNA [includes rest of ncRNA elements]
** Centromere [includes rest of region elements]
* Assertion: Interesting overlaps to remove are between the top-level feature types above.

Next week: [[AlexLabNotebook/YeastRebuild/9/24/05-9/30/05|9/24/05-9/30/05]]