==Overview of Cloning==
'''Cloning means you are going to insert a piece of DNA into a plasmid, introduce it into E. coli, and then isolate a pure population of bacteria that contain this piece of DNA.  It is the most basic operation we have in genetic engineering, synthetic biology, molecular biology, or recombinant DNA technology.  There are many variations on the theme, but the figure at right describes them in a general way.'''
[[Image:JCA_Arkin-Tutorial7Pic1.jpg|250px|right]]
In the figure, the piece of DNA you are trying to clone is designated in green.  The plasmid you are trying to put it into is in blue.  The plasmid fragment is almost always obtained by enzymatic digestion of plasmid DNA extracted from a bacterial culture.  Essentially, the plasmid is first purified from all other biological components.  Restriction enzymes are then added to cut the DNA at specific positions, and the desired DNA fragment is isolated by gel electrophoresis and purified.  The insert for cloning can be obtained several ways.  Sometimes, the DNA comes from an existing plasmid.  In that case, the source plasmid is cut with restriction enzymes and purified.  More often, however, the sequence is either obtained from genomic DNA or is synthesized chemically.  Genomic DNA is the long stretch of sequence of DNA present in every cell as its genetic blueprint.  This DNA can be extracted from any organism and used as the template for PCR amplification.  Using PCR, any sequence present in the sample can be amplified into a pure linear DNA fragment.  The ends of the DNA are then cut with restriction enzymes to produce the insert for cloning.

Regardless of where the insert and vector digests come from, they must contain compatible cohesive ends (also called "sticky ends") so that the fragments can be combined using T4 DNA ligase.  This enzyme can join the ends of two double stranded DNAs provided their ends are complementary.  So, blunt cut ends can be ligated to other blunt ends, ends with the sequence TTAA can be joined to ones with AATT ends, and so on.  After ligation, circular DNAs are produced.  These DNAs are then introduced into E. coli cells by a process called transformation.  Only circular DNAs are accepted by E. coli, but there are some additional requirements.

All plasmids must have one essential feature--an origin of replication.  This is a stretch of sequence that the cell uses to initiate synthesis of new DNAs.  This is actually the only feature ''necessary'' for propogation of a plasmid in E. coli.  However, there is almost always a selectable marker present in the plasmid.  Most markers for use in E. coli are genes encoding resistance to an antibiotic.  Therefore, cells harboring these plasmids will survive on media containing particular antibiotics.  This feature is necessary since we are doing transformation.  Only a small fraction of transformed cells will successfully receive and replicate a plasmid.  Since this subpopulation is antibiotic resistant, they can be selected for growth on antibiotic media.

So, to recap what we have thus far--we generate linear double stranded DNAs with compatible cohesive ends, we ligate them together, we transform E. coli, and we plate on selective media.  We now wait about 16 hours (aka overnight).  When we come in the next day, we'll hopefully have colonies.

Each colony on the plate is a clonal population of bacteria.  By this, we mean they are all genetically identical.  What has happened is that a single bacterium received a single plasmid DNA, sat down on the plate at a specific position, and started growing.  E. coli grows very fast, so after sitting there overnight about a million bacteria are piled up in a lump.  Each bacterium in that colony is identical, but each colony on the plate may be different from every other.  In this case, we suppose that the green insert we inserted into the plasmid makes the bacteria turn green.  Some of the bacteria that received a plasmid received the insert, and some didn't.  Therefore, some turned green while others didn't.
[[Image:JCA_Arkin-Tutorial7Pic2.jpg|200px|right]]
==== ====
In the second stage of cloning, we must identify a colony of bacteria that has the gene of interest.  In this case, it's easy to pick the colonies because they are green!  However, this is rare.  Usually they all look the same.  You might be thinking, what are those other colonies that ''don't'' receive an insert?  Well, the most common side product in cloning is parent vector.  Often uncut plasmids bleed through purifiction and give some colonies.  However, you will encounter many other unwanted products.  Particularly when PCR is involved, mutations will occur within you insert fragment generating colonies in which the plasmid is really close to the expected sequence but has maybe one point mutation in it.  And then there are the weird products--sequences that got amplified by accident, recombination products, deletion products, random genomic DNA fragments--lots of weird things can happen, but fortunely they are pretty rare.  To find a colony that is the clone you want, you have to pick colonies.

This is quite simple, you dip a sterile tip or toothpick into the colony and drop it into ("innoculate") a test tube with a few milliliters of culture media.  You let these cultures grow overnight, and when you come in it will be full of bacteria.  Typically, you will screen around 4 colonies.  The next step is to isolate the plasmid DNA from this culture, and analyze the sequence either using mapping with restriciton enzymes, colony PCR, and/or sequencing.  Generally, though, what you are trying to establish is whether the clone matches the sequences you are looking for.

And on that note, every plasmid you construct has an associated sequence model file.  You have the physical sequence of bases in the DNA in your plasmid which hangs out in your bacteria and also in the tube you'll keep in the freezer.  You also have a file in your computer that describes the sequence of your plasmid.

Most of this tutorial tells you how to manipulate these sequence files on the computer.  First, we'll go through how you design a cloning experiment to make a particular sequence, and how to predict the sequence for the model file.  Then, we'll describe how you would analyze sequencing data after making the plasmid to determine whether it matches your model file.  Finally, we'll tell you how to execute the cloning experiment in the lab.
{{Template:JCA_Arkin_tutorialfooter}}