__NOTOC__
==Basic cloning:  How to implement the Construction File==
This tutorial describes what we'll be doing in the lab as a demonstration of basic cloning.  This procedure describes what you'll be doing and seeing during cloning.  There's no quiz at the end, but you'll have plenty of real-life experience with this procedure.  First of all, let's examine the construction file:

  Construction of GFP Biobrick 2.0 basic part
  PCR ca1123F/ca1123R on pSB1A2-I13522      (748 bp, BglII/XhoI/DpnI)
  Sub into pBca1102                         (BglII/XhoI, 2159+697, L)
  Product is pBca1102-Bca1123
  ----------------------------------------
  ca1123F  Forward BglII for eXtreme GFP basic part  ctctgAGATCTatgcgtaaaggagaagaac
  ca1123R  Reverse XhoI for eXtreme GFP basic part   gcaaaCTCGAGttaGGATCCttatttgtatagttcatccatgc

Go ahead and download all the files and simulate it in your editor to confirm that it "works":
  [[Media:JCAseq_pSB1A2-I13522.str]]
  [[Media:JCAseq_pBca1102.str]]
  [[Media:JCAseq_pBca1102-Bca1123.str]]

In this construction, we are using a plasmid pBca1102 I call "ORF expressor."  This plasmid holds open reading frame basic parts, but it places a constitutive (meaning always on) promoter and ribosome binding site upstream of the Biobrick BglII site.  The result of this is that the open reading frame will make the encoded protein when the plasmid is inserted into E. coli cells.  However, it is still a "basic part" in the sense that the ORF is still flanked by BamHI and BglII restriction sites.  In this construction file, we start with the parent plasmid pBca1102 which has an RFP gene inside these restriction sites.  Cells with pBca1102 are therefore red.  When we digest with BglII/XhoI, we remove the RFP gene.  We'll paste in the PCR-amplified GFP gene, and upon insertion of the new plasmid into E. coli, the cells should be green.  So, that's the game here, now let's see how specifically to do it.

==Step 1:  The PCR==
When you receive oligos from IDT, it comes as a liophylized solid (ie, a dry powder).  You must resuspend it in water to a final concentration of 100uM (uM is 10^-6 molar).  Fortunately, IDT measures how many moles of DNA are present in the tube and writes it on the tube.  Let's say the tube says it's 23.4nmoles of material.  You would therefore add 234uL of water to the tube.  I leave it to you to do the math and confirm that.  To set up the PCR, you'll want to use 10uM stocks, so do a 10x dilution of the oligos.  To achieve this, add 1uL of 100uM oligo to 9uL of water, then mix it by tapping it on the bench.  Now you can set up the PCR.

A general procedure for setting up a PCR reaction is to make the following cocktail:
  39 uL water
  5 uL 2 mM in each dNTPs (meaning a mixture of 2mM dATP, 2mM dCTP, 2mM dTTP, and 2mM dGTP)
  5 uL 10x Expand Buffer "2"
  0.75 uL of 0.75 uL Expand Polymerase "1"
  1 uL of 10uM Oligo 1
  1 uL of 10uM Oligo 2
  0.5 uL of template DNA

Add the ingredients in order.  In general, never add an enzyme to a reaction until after you've added the buffer.  The reason for this is that proteins can denature if they are diluted into a solution that is not buffered.  It doesn't matter when you add the oligos and template, though.

Now mix up the sample by tapping and place it in the thermocycler.  Choose the temperature program based on the size of the fragment:
  '''PCR product Size     Program'''
  Under 1kb            55
  1kb to 2kb           2K55
  2kb to 4kb           4K55
  over 4kb             8K55
These programs principly differ in the length of time for each extension step during the cycles.  This is necessary because longer PCR products take more time to polymerize.  In general, it is OK to run a longer program than is necessary for the PCR product, but going shorter will cause failure.  Sometimes doing longer extension times than are necessary will lead to aberrant products, so you should make an effort to use the right program.  Alright, run the program.

Now you want to run an analytical gel to make sure you got the right product.  There is another kind of gel you'll run later called a "preparative" gel.  The distinction is that the analytical gel is run simply to examine the products.  The prepartive gel is done to separate DNA fragments, and bands are individually excised from the gel and purified.  You will add 5uL of 10x loading buffer to the PCR reaction, and then load 5 uL of material to a well of the gel.  In another lane, add 5uL of molecular weight marker.  You'll use the marker to estimate the size of your PCR band.  Below is an example of a PCR product.<br>
[[Image:090806-pMP908bamecoscreen.jpg|95px]]
[[Image:bisc300-lab_Hi-Lo_DNA_Marker.jpg|80px]]
==== ==== 
At the right of the PCR product is the known sizes present in the Hi-Lo molecular weight markers.  The size of this PCR product is therefore somewhere around 3kb.  After running this gel, estimate the size of your PCR product.  Does it match the size you predicted for the construction file?  Also, look at the quality of the band.  Is it a single band or multiple bands?  Is it tight or kindov fuzzy?  If any of that fails, you'll need to [[ArkinJCA_troubleshoot1 | troubleshoot]].  If it all looks good, let's move on to the cleanup reaction.

==Step 2:  Purify the PCR product==
The general flow of the experiment goes 1) do the PCR, 2) clean up the PCR product, 3) digest the PCR product, 4) purify the digest.  In parallel, you'll want to prepare the vector if it doesn't already exist.  Then 5) set up the ligation reaction, and finally 6) transform the ligation.

You now need to purify the PCR product.  We'll use the Zymo kit for this.  ADB buffer is high concentration Guanidinium Chloride.  This salt forms a complex with nucleic acids which makes them bind to silica membranes.  Also, it denatures proteins.  So, adding ADB will destroy the polymerase present in the PCR.  When you pass the mixture through the column, the DNA binds, but the denatured polymerse, dNTPs, buffer, and salts present in the mixture flow through.  
# Start by adding 200uL of ADB buffer to to the PCR reaction and transfer the mixture to a Zymo column.
# Spin the column 15sec at full speed to pass the liquid through the column.
# Wash with 200uL of wash buffer, which is just 70% ethanol in water and again spin for 15sec.  This will dissolve any extra guanidinium chloride and salts sitting on the membrane.
# Repeat the wash buffer step 3.  Now nothing is present on the membrane but the DNA and a little ethanol and water.
# Now spin the column for 90sec at full speed to remove all traces of water/ethanol.
# Finally, add 90uL of water to the membrane and spin 30sec collecting the eluent in an eppendorf tube.
The water re-dissolves the DNA which is now entirely pure.  All that's left in your collection tube is DNA and water.  This procedure achieves several things:
*It removes the polymerase which if left behind could either fill in or degrade the sticky ends you'll generate during the restriction digest
*It exchanges out the buffer because you'll want a different buffer for the restriction digest
*It removes most of the oligonucleotides and dNTPs left over from the PCR reaction, though they aren't much of an issue

==Step 3:  Set up the Restriction Digest==
You now have about 87uL of DNA and water as some of the liquid tends to stay behind on the membrane.  Buffers generally are made at 10x.  So, set up the following reaction:
  87uL DNA and water
  10uL of NEB Buffer 2
  1uL BglII
  1uL XhoI
  1uL DpnI
Tap the reaction on the bench to mix it thoroughly, give it a quick spin, then put it in the 37deg incubator for 1hr.  

==Step 4:  Clean up the Digest==
After 1hr, add 400uL of ADB buffer and repeat the Zymo column procedure, elute with 25uL of water.  This procedure will:
*Remove the leftover buffer
*Remove the restriction enzymes
*Partially remove the short DNA "tails" liberated by digestion
*Concentrate the DNA

==Step 5:  Set up the ligation==
Let's assume you already did the vector digest.  Once purified, digested DNA will last for years in the freezer.  So, you can save vector digests and use them over and over.  In a later section, I'll describe how you do that digest, but for now, let's assume you already have done it.  Set up the following reaction (in order):
  6.5uL water
  1uL 10x T4 DNA Ligase Buffer (in aliquots in freezer)
  1uL vector Digest
  1uL PCR product digest
  0.5 uL T4 DNA Ligase
'''Make sure you are using T4 DNA ligase and not T4 DNA polymerase!'''  Tap the reaction to mix, give it a quick spin, then put it on the bench (at room temp) to ligate for 30min.

==Step 6:  Do the transformation==
Now we're ready to put the DNA into cells.  There are 2 general ways to do this:  heat shocking and electroporation.  Both involve things called "competent cells".  We make these in big batches and store them in the -80 freezer.  We'll cover how to make them elsewhere.  For now, just take one out of the -80 freezer and put it on ice to thaw out.  There is 220uL of cells in the aliquot.  Add to it (on ice):
  50uL of water
  30uL of KCM
Pipette up and down or invert the tube to mix it.  Don't let it warm up--keep it on ice the whole time.  Don't vortex it as cells are a little sensitive in this state.

Put your ligation on the ice to cool it down, then transfer 100uL of the competent cell mixture to the ligation, pipette up and down a few times to mix it.

Let that sit on ice for 10min, then heat shock it in the 42deg water bath for 90sec.  Transfer it back on ice for 2min (the time isn't critical).

Now you have to do different things based on the antibiotic marker you're going to use.  If it is an ampicillin/carbenicillin resistance marker, you can plate immediately.  This is because only growing bacteria are sensitive to the penicillin family of antibiotics.  In contrast, all other antibiotics must be incubated at least 30min to give the cells time to express the marker.  This is because most antibiotics directly inhibit translation.  So, if you add the antibiotics immediately, you'll inhibit translation and the restance protien never gets made.  The cells will just die.  So, instead add 100uL of LB media to the transformation and put it in the shaker for 30min to 1hr.  After that incubation, you can plate 100uL of the reaction on the plates.

Now you're done for the day!  Tomorrow you should have colonies, and you will go into the screening/characterization phase of the experiment.

==Step 4b:  Digesting the vector==
In step 5 we set up ligations and assumed you already had a restriction digest of the vector (ie, plasmid pBca9145-Bca1089, pBca1102--the third line in your construction file).  If you don't already have this in the freezer, you need to make it.  You'll need a good quality miniprep (purified DNA) of your plasmid, then you'll set up the following:
  3uL water
  5uL miniprep
  1uL NEB Buffer 2
  0.5uL restriction enzyme 1
  0.5uL restriction enzyme 2
Often you'll want to scale the above numbers by a factor of 2 to 4.  It all depends on how often you think you'll need this digest.  Tap the mixture on the bench to mix, give it a quick spin, then put it at 37deg for 1hr.

After incubating, add 1uL of 10x Loading Buffer (not marker!) load the entire mixutre into one well of a 1% agarose gel.  Put 5uL of marker in another lane and run the gel.  Make sure you wear gloves whenever working with agarose gels.  The dye used to visualize DNA, ethidium bromide, is a carcinogen.  Put the gel in a dish and put it on the UV lamp.  Be sure you are using the shield while doing this.  Otherwise, you're basically staring into the sun--it's  bad for your eyes.  Now, look at the bands.  Are they the size predicted by the construction file?

Let's look at some gels:<br>
[[Image:042507gel2.jpg|300px]]<br>
'''A''' is an example of a "large" band from a plasmid digest.  Usually, this is the sort of band you want.  '''B''' is singly cut DNA.  Often you'll predict 2 fragments from your digestion and will observe 3.  If this is the case, look at the largest band--is it consistant with the full length of your plasmid?  If so, it most likely is plasmid that only got cut once.  The band you want is the middle band.  In '''C''' you see an example of what I call a "smile" of the gel.  Often some lanes of the gel run a little slower than others.  If this happens, you can't just line up the  bands with the marker to estimate the size.  You have to mentally draw an arc through the gel.  On this gel, all 4 lanes are the same plasmid, but as you can see they all look a little different.  In '''D''', you see a high molecular weight band in the lane.  This can be one of two things.  Sometimes, genomic DNA bleeds into your miniprep, and will look like a high molecular weight smear in the lane.  Also, sometimes you get a "glycerol shadow", which is just a chemical phenomena--it's not DNA at all.

Cut out the appropriate band described in the construction file.  Keep as little of the bordering agarose as possible, and transfer it to an eppendorf tube.  Add ~750uL of ADB buffer.  Really, what you need is a minimum of 3 gel volumes worth of ADB, but you'll need to estimate that.  It won't hurt to add too much ADB, but you can add too little.  Melt it at 55deg.  Shake it occasionally, and keep heating until the whole thing is homogeneous.  Spin it through a Zymo column, wash twice with wash buffer, spin to dry, elute it in about same volume you initially started with.  So, if you digested 10uL of miniprep, elute with 10uL of water.  If you did 5uL of miniprep, elute with 6uL of water--6 is the lowest volume you can elute with.
{{Template:JCA_Arkin_tutorialfooter}}