Bacterial artificial chromosomes or BACS are circular DNA molecules which contain a replicon that is based on the F factor.  BACs have ''oriS'' and ''repE'' which encode an ATP-driven helicase and ''parA'', ''parB'' and ''parC'' for partitioning.  The original BAC vectors are 7.4 kb while others are 8-9kb in length.  You can clone 80-300kb fragments in BACs.  Generally, the host needs to be deficient in homologous recombination machinery in order for the BAC to be stable (i.e. ''recA<sup>-</sup>'').

==Existing vectors==

====pBAC108L====
*no selection system for inserts
*one of the original vectors
*<cite>Shizuya-PNAS-1992</cite>

====pBeloBAC11====
*recommended by Tom, have this in the lab
*screen for inserts via &alpha;-complementation (blue-white screening on IPTG/Xgal plates)
*does not have a selection marker for transfection into mammalian cells
*[http://www.neb.com/nebecomm/products/productE4154.asp pBeloBAC11 from NEB]
*cos site
*loxP site
*inserts up to 1Mb
*T7 and SP6 promoters flank the insertion site
*[http://www.rzpd.de/info/vectors/pBeloBAC11_pic.shtml map]
*Genbank accession number [http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&val=1817728 U51113]

====pBACe3.6====
*transferred the pUC-link and SacBII sequences from a PAC vector to pBAC108L. See PAC vectors below.
*does not have a selection marker for transfection into mammalian cells
*[http://bacpac.chori.org/pbace36.htm pBACe3.6 information/map]
*<cite>Frengen-Genomics-1999</cite>
*Genbank accession number [http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&val=4878025 U80929]
*has a loxP site for Cre recombinase protein
*11.5 kb
*has lots of BioBricks sites internal to the vector backbone
*[http://www.rzpd.de/info/vectors/pBACe3.6_pic.shtml map] 

====pFW11 and F' factor====
*<cite>Whipple-NAR-1998</cite> (from Natalie)
*available through Natalie from an HMS lab
*clone insert into pFW11 between 3' end of ''lacI'' and 5' end of ''lacZ''.  Transferred via homologous recombination to the complete lac operon (lacZYA) on the F' episome.  Then F' moves to another strain via conjugation which is distinguished from the original strain by streptomycin resistance.
*distinguish between single recombinants (both episomes) and double recombinant F' via the marker sensitivity.
*method designed to test DNA binding proteins and cognate promoters via &beta;-galatosidase activity.
*pFW11 has a strong terminator upstream of the polylinker
*may be a more sophisticated system than we need?

====pCC1BAC====
*[http://www.epibio.com/item.asp?ID=380&CatID=125&SubCatID=33 CopyControl pCC1BAC vector from Epicentre]
*"contains the E. coli F-factor single-copy origin of replication and an inducible high-copy oriV origin of replication." 
*"grown at single-copy number to ensure insert stability. Then, clones can be induced to 10-20 copies per cell within 2 hours of adding CopyControl Induction Solution to the culture, for higher yields and higher purity DNA. Although large-insert clones are less stable when cloned and maintained in high-copy vectors, the short 2-hour induction of CopyControl BAC clones to high-copy-number does not decrease their stability, based on analysis of Hind III restriction patterns of a large number of induced versus uninduced clones that ranged in size up to ~200 kb."
*see information on pSMART VC vectors below, I think it uses the same system for inducible copy control.

====pSMART VC Vectors====
*[http://www.lucigen.com/products/Cloning_Systems/CopyRight.html pSMART VC Vectors from Lucigen]
*Flanking transcriptional terminators to stabilize recombinant clones.  Strong promoters have successfully been cloned between these two terminators.
*Transcription/translation-free cloning for unstable DNAs
*Inducible replication origin: ''oriV'' origin can be induced by addition of L-arabinose.  L-arabinose induces the expression of the RK2-encoded TrfA replication protein upon which replication at ''oriV'' depends.  Vector copy number increases 20-50 fold upon induction of this protein.  
*Replicator Cells contain the ''trfA'' gene necessary for conditional amplification of vector copy number. These cells have copy-up TrfA mutant protein controlled by the P<sub>''ara''BAD</sub> promoter.  If copy amplification is not needed, CopyRight Kits can be used with Lucigen’s E. cloni Chemically Competent or Electrocompetent Cells, or with any other competent E. coli cells.
*Minimal vector size
*Bacteriophage lambda cos site for fosmid packaging or lambda terminase cleavage
*Phage T7 promoter for in vitro transcription (pEZ BAC only)
*loxP sites for Cre recombinase cleavage
*Rare-cutting restriction sites on either side of the insert
*Chloramphenicol-resistance gene
* Not all electroporation cuvettes are created equal! Lucigen has found that use of certain cuvettes with our systems results in less than one tenth the cloning efficiency. In all cases, we recommend a 0.1 cm gap width. Recommended: BioRad 165-2089 or BTX model 610. Not Recommended: Invitrogen 65-0030
*<cite>Wild-Genome-Res-2002</cite>
*GenBank accession number [http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&val=68262428 AY643800]
*the sequence in GenBank does not seem to match the sequence on the website (at least the linker appears to be different).  According to Ron Godiska at Lucigen, "There were supposed to be two NotI sites in the vector (one on each side of the cloning site), but the right-side site was mutated. The latest version of the genBank sequence shows the mutation. Some of the earlier drawings of the vector still show two NotI sites."  This doesn't seem to account for all the differences I see so I am still checking on this.

====pIndigoBAC-5====
*[http://www.epibio.com/item.asp?ID=328&CatID=125&SubCatID=33 pIndigoBAC-5 from Epicentre]
*"derived from pBeloBAC11 and pIndigoBAC1 and will accommodate and stably maintain DNA inserts of >100 kb."
*"linearized at its unique BamH I or Hind III site, dephosphorylated, and highly purified and is ready for cloning BamH I- or Hind III-cut genomic DNA. The linearized and dephosphorylated vectors are tested to ensure the completeness of linearization, dephosphorylation, and the integrity of the BamH I and Hind III ends."
*"The complete sequence and restriction map of pIndigoBAC-5 is available."
*"Enhanced blue/white screening of recombinants."  The ''lacZ&alpha;'' fragment is reengineered so that detection of DNA insertions between amino acids 11-36 of &beta;-galactosidase occurs.  This offers improved accuracy in blue-white screening.

===Cosmids===

*conventional vectors that contain a cohesive end site (cos) from bacteriophage &lambda;

====pWEB-TNC<sup>TM</sup>====
*[http://www.epibio.com/item.asp?id=332 pWEB-TNC from Epicentre]
*"The kits utilize a novel strategy of cloning end-repaired, randomly sheared DNA instead of the conventional approach of cloning fragments generated by partial restriction endonuclease digestion. First, genomic DNA is sheared by passing it through a syringe needle. The sheared DNA is end-repaired to generate 5’-phosphorylated blunt ends and size-selected using a low melting point agarose gel. The size-selected DNA is then ligated into the supplied linearized and dephosphorylated pWEB-TNC<sup>TM</sup> or pWEB<sup>TM</sup> Cosmid Vector, packaged using ultra-high efficiency MaxPlax<sup>TM</sup> Lambda Packaging Extracts (>109 pfu/&mu;g for phage lambda) and plated on phage T1-resistant EPI100<sup>TM</sup>-T1R E. coli plating cells, all included in the kit. The result is a complete and unbiased primary cosmid library."
*Has EcoRI, NotI ... NotI, EcoRI in the multiple cloning site making it fairly easy to create a BioBricks insertion site in the plasmid.
*Has a ColEI origin annotated on the vector.

====SuperCos 1====
*[http://www.stratagene.com/manuals/251301.pdf SuperCos 1 from Stratagene]
*we have it?
*Tom says we got it from Epicentre
*"Cosmid vectors are valuable tools in this analysis, because they can accommodate genomic DNA fragments ranging in size from 30 to 42 kb."
*"SuperCos 1 is a novel, 7.9-kb cosmid vector that contains bacteriophage promoter sequences flanking a unique cloning site."
*"The SuperCos 1 vector is also engineered to contain genes for the amplification and expression of cosmid clones in eukaryotic cells."
*"Most genomic inserts can be excised as a single large restriction fragment using the Not I restriction site that flanks the SuperCos 1 polylinker."
*cos sites are for in vitro packaging with &lambda;
*has a pUC origin on the map.

====pFos1====
*[http://www.neb.com/nebecomm/products/productE4153.asp pFos1 from NEB]
*"Grow cells at 30&deg;C. The plasmid is unstable at 37&deg;C."
*"The plasmid pFOS1 carries a gene encoding resistance to ampicillin (amp) and a gene encoding resistance to chloramphenicol (cam). To maintain the plasmid, cells should be grown with 100 &mu;g/ml amp and 15-20 &mu;g/ml cam."
*<cite>Kim-NAR-1992</cite>.
*"constructed by fusing pBAC (Shizuya et al., in preparation), an F replicon based plasmid with lambda cosN site, to pUCcos, a pUC derivative containing cosN site, by homologous recombination in E. coli through the shared cosN site. pBAC and pUCcos were transformed to the strain C600r-m+ sequentially, and the transformants grown on LB plates containing ampicillin and chloramphenicol were picked. Miniprep plasmid DNA prepared from the transformant containing fused replicons was diluted and transformed to the strain D 1OpolA-, in which pUC replication origin cannot function and therefore only the plasmids containing the F factor replication origin can survive.
*thus for this vector to be at single copy, it must be in a strain in which pUC origin is inactive
*[http://www.rzpd.de/info/vectors/pFOS1_pic.shtml map]

===PAC vectors===
*derived from bacteriophage P1 vector
*15-16kb
*positive selection for inserts via ''sacB'' the levansucrase gene which converts sucrose to levan which is toxic to ''E. coli''
*a pUC-vector-derived fragment is inserted into the cloning site which inactivates ''sacB'' and allows large amounts of vector to be purified.  Your insert replaces this pUC vector thereby returning the vector to single copy.  Must be careful to ensure that no uncut vector is present in cloning because it will transform as well as the insert containing vectors.  Thus, the ''sacB'' marker only prevents against vector religation productions not intact vector.

==Other information==
*Usually DH10B is used as the host.
*can retrofit BAC vectors with a particular insert by using a retrofitting vector like pRetroES (Ref 3)

==Protocol notes==

===BAC purification===
*exist as supercoiled circular plasmids and are resistant to shearing making purification easier
*most purification protocols appear to be geared for preparation of genomic libraries for sequencing.  It is not clear if we need such elaborate protocols for simple clonings.
*Typical DNA yields
*#1&mu;g from 5mL LB culture sing alkaline BAC DNA purification
*#0.8&mu;g from 1.3mL TB culture using Qiagen R.E.A.L.
*#4&mu;g from 20mL LB culture using Qiagen-tip 20
*#100&mu;g from 500mL LB culture using Qiagen-tip 500

===Putting the BAC into ''E. coli''===
#Transformation
#*has low efficiencies
#*works on any size
#In vitro packaging
#*package the vector inside &lambda; bacteriophage particles and infect cells
#*limits the size of the DNA because of phage head stability.  The phage head is not stable if the vector is too small or too large.  Usually ~40kb (between 25-45 kb okay) insert with an 8-9kb vector works.
#*upon introduction into the ''E. coli'' cells the vector is circularized to for a large plasmid with the insert.

==References==
<biblio>
#Hohn-Gene-1980 pmid=6260575
#Ogura-Cell-1983 pmid=6297791
#Abeles-JMB-1985 pmid=3903163
#OConnor-Science-1989 pmid=2660262
#Evans-Gene-1989 pmid=2777090
#Shizuya-PNAS-1992 pmid=1528894
#Kim-NAR-1992 pmid=1549470
#Whipple-NAR-1998 pmid=9685485 (from Natalie)
#Frengen-Genomics-1999 pmid=10373322
#Wild-Genome-Res-2002 pmid=12213781
#Casali-Ecoli-plasmids-2003 isbn=1-58829-151-0
#Zhao-BAC-2004-v1 isbn=0-89603-988-9
#Zhao-BAC-2004-v2 isbn=0-89603-989-7
</biblio>
#[http://www.rzpd.de/info/vectors/ RZPD vector information]
#[http://www.emunix.emich.edu/~rwinning/genetics/glossary.htm Biol301 at Eastern Michigan University]

==Related pages==
*[[F plasmids]] serve as the basis of the replication origins in BACs.
*See the [[Synthetic Biology:Vectors/Single copy plasmid | BioBricks compatible single copy plasmid]] project.

[[Category:Escherichia coli]]