{{Template:BE.109}}

==Schedule for Module 1 laboratory report==

===March 16/17, 2006===
*First draft is due by 10:00 a.m. on either the 16th or the 17th, depending on your section.  Please turn in your lab reports electronically by emailing them to Neal Lerner, Bevin Engelward and Natalie Kuldell (nlerner@mit.edu, nkuldell@mit.edu, bevin@mit.edu).

===April 4/5, 2006===
*First draft is returned to students.

===April 11/12, 2006===
*Final reports are due.
*Your draft will be graded, and you will have the opportunity to improve your grade as follows:
**Anything above a B can become an A+.
**Anything above a B- can become an A
**Anything above a C+ can become an A-
**Anything above a C can become a B+
**etc.

==Guidelines for Module 1 laboratory report==

===Originality===
* This report must be written by you, and thus should not be written with your lab partner.

===Guidelines on length===
Not counting figures, using 12 pt font, double spaced text and 1” margins (be sure to turn in your final report double spaced), the length should not exceed the following (concise writing is appreciated and rewarded!):

*Introduction - 2-3 pages
*M&M –  3 pages
*Results – 5 pages
*Discussion – 3 pages
*Abstract – Under 300 words total
*Total: Your final report should be ~13-15 pages (certainly not longer than 17 pages).
*Figures:  Your figures should be in a separate power point file.

===Grading===
Your first draft will be graded, and you will have the opportunity to improve your report after you get comments from Neal Lerner and Bevin Engelward.  

===Order of assembly===

It is recommended that you assemble your lab report in the following order:

#'''Figures and tables:''' Put together the figures and tables, so you know exactly what you will be talking about.  This includes statistical analysis, making the graphs look nice, putting them into a word document and writing the figure captions.  It is important to do the statistical analysis before writing the results, since your interpretation of the results will be affected by whether or not the observations appear to be significant.
#'''Results:''' Write the results to describe how and why you did what you did, in a logical flow (tell the ‘story’ of what you did).  Before you start writing anything, make an outline and think about the flow of the text.  Decide what your subheadings will be. 
#*The results should interpret the data.  For example, you might write:  “As expected for the correct d5 vector (see map Fig. 1A), Lane 2 of Figure 1B shows that we observed two fragments that were ~x and ~y kb.  Thus, the ligation resulted in creation of the desired construct.”  Or you might write: “Unexpectedly, we observed an extra band that was ~3 kb (Fig. 1B, lane 2), which may be the result of a partial digestion.”    
#*Tell the reader what to look at – walk them through your results.  For example, for the flow cytometry results, you might write “Figure X shows the results of flow cytometry.  Results for wild type cells are shown in part A where each dot represents X.  These results show that normal wild type cells have a certain level of normal fluorescence in the X and Y wavelength brackets.” Be sure to explain what R2 is.

#'''Discussion:''' After you write your results, you will have thought about the discussion and you’ll be ready to write it.  The discussion is where you accomplish two major objectives: 1) talk about results that weren’t as expected, and come up with ideas about what could have happened.  Try to avoid simplistic explanations (e.g., the enzyme was not added), unless you have very good reason to think that you did.  2) Put the results into the context of the bigger picture.  What are the limitations of this approach?  You used ES cells for these experiments – what is this cell type, and what might be different in other cell types (e.g., how general are your results)? What other ways could you imagine improving the assay? For what types of applications might this assay be used?
#'''Introduction:''' Write the introduction.  Knowing the whole story, you now can ask yourself ‘what does the reader need to know to understand what I’ve written?’. 
#*Your introduction should address the following questions:  What is homologous recombination?  Why should the reader care about this repair pathway? What is the objective of the experiments? What is the value of the assay you are helping to develop? 
#*Generally, people usually end the introduction with a paragraph summarizing in very brief terms the major results.  For example, “Here, we have created a system for measuring xxxx, and we have compared xxxx.  Our results show xxxx, indicating that xxxx.”  You will certainly want to tell the reader what you set out to do - the extent to which you disclose results here is a matter of style, so we will leave this choice up to you. 
#'''Abstract:''' Write the abstract last.  It’s easiest to write the abstract just after writing the main text, when everything is fresh in your mind, and you have a good sense of how confidently you can make your statements.
#'''Materials and Methods:'''  This can be written at any time.  It should be brief!    
#*Assume the reader can refer to the NEB catalog for temperatures and amounts of enzyme to add to cut DNA.  They just need to know what enzyme was used.  
#*Regarding PCR, they need to know what the key features of the primers were. (The sequence is not necessary and is generally not meaningful to the reader.. ideally you would state something like –‘The forward primer anneals to the first 22 nucleotides, beginning at the ATG.  To the 5’ end of the primer was added the X and Y restriction sites.  An additional X nucleotides were added to the X end in order to assure that ... Sequence is available upon request.”
#*Regarding lipofection – assume they can go to the commercial site to get the details.  Tell them what they need to know about your experiment.  How much DNA did you use?  Were the cells attached to dishes or floating around when you put the DNA into the cells?  Were the cells crowded in the dish or was there room to divide when you added the DNA?
#*Regarding flow cytometry – they need to know what the excitation wavelength was, and what the axis are – what is FL1?  what is FL2?

===Figures===

Figures:  All figures should be prepared in PowerPoint.  You need not integrate the figures into the text, but rather you can submit the text and figures as separate files.

#'''Figure 1:''' HR figure in the introduction
#*You are welcomed to add figures to the introduction.  Hand drawn figures are encouraged (but they would need to be scanned in to put them into PPT).  You can use published figures, but if you do so, you must include the reference to the published work (e.g., the figure should not be downloaded from a web site). 
#*The introduction should not include pictures of the plasmids, since that is too much detail for the introduction. 
#'''Figure 2:''' Diagram showing what you did.  There are two levels to this and you should start with the big picture and then zoom in.  Part A for this figure would ideally show a summary figure that describes how plasmids are used to detect homologous recombination in mammalian cells.  The reader needs to know what the goal is: which plasmid did you set out to create? Part B would ideally be a simplified diagram that shows the design strategy for the ligation (e.g., such as the 'Roadmap for Plasmid Construction' that appears on the wiki for Day 1). 
#'''Figure 3:''' Your gel showing your PCR results. Keep in mind that all images of gels must include markers indicating the estimated sizes of the observed fragments (these can be estimated by eye based by comparing to the markers).  You will certainly need markers indicating the sizes of two or three fragments in the ladder.
#'''Figure 4:''' The gel showing the purified products.  See above comments on for Fig. 3  regarding correct labeling.
#'''Figure 5:''' Diagrams showing rough plasmid maps to explain the logic of how you know whether or not you obtained the desired vector. You should indicate the positions of the restriction sites that you used for your analysis, and indicate the sizes that you expect for digestion. Ideally you would also indicate the position of the coding sequences.  You will need diagrams of the parental backbone and the correct product.  Please explain to the reader how you know that you do not have multiple inserts (if you indeed know this to be the case).  
#*The restriction site positions need not be perfect to the base pair, but should be correct within ~50 bp (e.g., the reader need not know the fragment is 2.543 kb – it is sufficient to state that the fragment is 2.5 kb). 
#'''Figure 6:''' The gel showing the results of your restriction analysis.  See above comments on for Fig. 3 regarding correct labeling.
#'''Figure 7:''' Plots from your flow cytometry experiment.  For flow cytometry, you should not show all of your flow data, but you should show representative results – positive control, negative controls, and representative examples of the conditions tested.  Yoon Sung has generously offered to scan and compile this figure for you.
#'''Figure 8:''' Graph showing the results of your experiment.  Your final results should be an excel graph with error bars.  Be sure to include all the conditions (e.g., negative and positive controls), even if the result is a number too low to show up in a bar graph.  If your positive control 'swamps' your results, you can separate this into two separate graphs.  You must state in the figure caption what the error bars indicate (‘error bars indicate the 95% confidence interval’) and you must state whether or not the results are statistically significantly different (specify exactly which two samples are being compared) [‘asterix indicates that X is statistically significantly greater than the uncut control (p < 0.05, Student’s T-test)’].

===References===
You should include at least one reference to a primary research report.