{{Template:BE.109}}


==Introduction==
DNA can be put into mammalian cells in a process called transfection. If you wanted to make a mouse cell fluoresce green, you could transfect it with DNA carrying the EGFP open reading frame, a promoter directing transcription of EGFP and a signal sequence for polyadenylation of the mRNA. The promoter tells the cell that the EGFP sequence should be transcribed by RNA polymerase.  The polyadenylation sequence assists in the export and stability of the mRNA so that it gets translated by the ribosome.  The coding sequence tells the ribosome which amino acids should be joined together.

Mammalian cells can be transiently or stably transfected. For transient transfection, DNA is put into a cell and the transgene is expressed, but eventually the DNA is degraded and transgene expression is lost ("transgene" is used to describe any gene that is introduced into a cell).  For stable transfection, the DNA is introduced in such a way that it is maintained indefinitely.  Today you will be transiently transfecting your cultures of mouse embryonic stem cells. 

There are several approaches that researchers have used to introduce DNA into a cell's nucleus.  At one extreme there is ballistics.  In essence, a small gun is used to shoot the DNA into the cell.  This is both technically difficult and inefficient, and so we won't be using this approach!  More common approaches are electroporation and lipofection.  During electroporation, mammalian cells are mixed with DNA and subjected to a brief pulse of electrical current within a capacitor.  The current causes the membranes (which are charged in a polar fashion) to momentarily flip around, making small holes in the cell membrane that the DNA can pass through.

The most popular chemical approach for getting DNA into cells is "lipofection." With this technique, a DNA sample is coated with a special kind of lipid that is able to fuse with mammalian cell membranes.  When the coated DNA is mixed with the cells, they engulf it through endocytosis. The DNA stays in the cytoplasm of the cell until the next cell division at which time the cell’s nuclear membrane dissolves and the DNA has a chance to enter the nucleus.

Today you will lipofect DNA into your mouse embryonic stem cells. As a positive control for the transfection, you will include a plasmid encoding EGFP. This will cause the cells to fluoresce green.  Next time we will measure fluorescence to assess the success of the transfection. 

You will also be transfecting two experimental plasmids, one of which you have been constructing for weeks. The EGFP coding sequence on these plasmids is truncated at either the 5’ or 3’ end of the gene. Cells expressing these truncated EGFPs should not fluoresce green. The plasmids provide a wonderful tool for studying recombination since a cell will fluoresce green if it has been transfected with both plasmids and has recombined the genes to regenerate a full-length EGFP. Finally, you will be using the truncated EGFP plasmids to investigate the effect of double strand breaks on the rate of recombination. 
Elaborate mechanisms for coping with DNA breaks have evolved since these forms of DNA damage are so dangerous for the cell. DNA with double strand breaks can be repaired according to the Szostak model, named after the person who first described it. According to this model, illustrated below, a gap in one chromosome is repaired through an interaction with its homologous chromosome. The repair first requires “homology searching” and invasion of the gapped DNA (blue) into the undamaged copy (red). The undamaged DNA serves as a template for missing sequences, which are copied and then resolved as indicated by the open triangles on the figure below. Resolution can lead to exchange of sequences flanking the original double strand break, resulting in recombination of genetic information. Alternatively the integrity of the original genetic material is preserved when the repaired strands resolve without crossover. 

[[Image:Be109szostak.jpg|thumb|left|500px|'''Szostak model for double strand break repair''']]
<br style="clear:both" />
 
Different, and far simpler means of repair are possible if the broken ends of the DNA can be held together, either through base pairing of the overhangs or through the chromatin structure surrounding the damage. Your investigation today will assess recombination rates for different DNA lesions. All the lesions are double stranded breaks but they differ in length of the single stranded overhangs. 

==Protocol==

In anticipation of your lipofection experiment, one of the teaching faculty plated 1x10<sup>5</sup> cells in each well of a pregelatinized 24-well dish 24 hours ago. Special media without antibiotics has been used. 

A schematic for your experiment is described below. We will discuss in class different things we might test, and we will use wells labeled 'A' and 'B' to test two parameters that might affect the frequency of inter-plasmid homologous recombination.
[[Image:Be109lipofectionschematic.jpg|thumb|left|500px|'''Experiment schematic''']]
<br style="clear:both" />

'''All manipulations are to be done with sterile technique in the TC facility.'''

'''Timing is important for this experiment, so calculate all dilutions and be sure of all manipulations before you begin.'''

For each lipofection you will need
 '''DNA:'''	 	0.1 ug (of &Delta;3) and/or 0.1 ug (of &Delta;5) in 50 ul OptiMEM 
 '''Carrier:''' 	2.5 ul Lipofectamine 2000 in 50ul OptiMEM 

*1. Dilute enough carrier for 16.5 lipofections. Let the dilution sit in the hood undisturbed for at least 5 minutes but not more than 30. 
*2. For the lipofections to be done just once, you’ll need 50 ul of DNA in OptiMEM. We will distribute to you a table indicating the appropriate volumes to use in your experiment.

{| border="1"
! Tube
! [DNA] stock
! DNA/lipofection
! Volume DNA
! Volume OptiMEM
|-
| mock
| NA
| NA
| NA
| 50ul
|-
| EGFP
| 0.25 ug/ul
| 0.1 ug		
|
|
|-
| &Delta;5
| 0.04 ug/ul
| 0.1 ug		
|
|
|-
| &Delta;3
| 0.04 ug/ul
| 0.1 ug		
|
|
|}

*3. For the lipofections to be done in triplicate, you’ll need a total of 150 ul of DNA diluted in OptiMEM. By making one lipofection cocktail that is later divide between replicates, you can be confident that each replicate was treated identically. The basic protocol is to do a lipofection with 0.1 ug of ∆5 and 0.1 ug of ∆3. 

{| border="1"
! Tube
! [DNA] stock
! DNA/lipofection
! Volume DNA
! Volume OptiMEM
|-
| &Delta;5/&Delta;3
| 0.25 ug/ul for &Delta;5<br> 0.05 ug/ul for &Delta;3
| 0.1 ug<br> 0.1 ug
|
|
|}

*4. You should now have eight eppendorf tubes, four with 50 ul of OptiMEM +/- DNA and three with mixtures of &Delta;5 and &Delta;3 DNA in a volume of 150 ul. Add an equal volume of diluted lipofectamine to each eppendorf (i.e., 50 ul if the tube has 50 ul). Pipet up and down to mix. 
*5. Incubate the DNA and lipofectamine cocktails at room temperature for 20 minutes. To allow the DNA/carrier complexes to form, it is important that you do not disturb the tubes during this incubation. During this time, aspirate the media from the cells in your 24-well dish, wash the wells with 0.5 ml PBS, then put 0.5 ml of fresh media on the cells. The PBS and media can be aliquoted with a 5 ml pipet.
*6. After the 20 minute incubation is over, use your P200 to add 95 ul of the appropriate DNA:lipofectamine complexes to each well. Since the carrier is quite toxic to the cells it’s a good idea to gently rock the plate back and forth after each addition. 
*7. Return the plate to the 37&deg;C incubator. 
*8. Tomorrow, one of the teaching faculty will remove the lipofection media from your cells and will replace it with 1 ml of fresh media. Approximately 48 hours after performing the lipofection, you and your partner will collect the cells and analyze their fluorescence by FACS. Before you leave today, see one of the teaching faculty to schedule your FACS time. 

DONE!

==For next time==
Write the Materials and Methods section of your lab report. This assignment will not be collected but, by writing it now, you will have time to get some feedback if you want it and you will stay on track for completing the lab report which is due next week.