{{Template:BE.109}}

==Introduction==
“Looking for a needle in a haystack” is one of several familiar phrases to describe the search for something rare and particular. Today you will try to find your “needle:” a protein that binds gold. Using the same panning protocol as last time, you will look through a pool of randomized 36-base pair sequences each of which will form a 12- amino acid peptide fused to Aga2.

Thanks to recent advances in DNA synthesis chemistry, high quality pools of oligonucleotides (“oligos” for short) are available at reasonable concentrations and prices. If you have time to look around the Internet for companies performing DNA synthesis (companies such as Integrated DNA Technologies or Operon), you’ll see that it’s possible to get next day delivery on a 20 or 30 base sequence of your choosing for around 20 dollars. 

[[Image:Be109synthesis.jpg|thumb|left|400px|'''Synthesis chemistry''']]
<br style="clear:both" />

The synthesis chemistry requires only a few things: a solid support on which to build the oligos, nucleotides that can be protected from further nucleotide additions, a way of deprotecting the nucleotides once they’ve been added to the oligo chain, and a way of releasing the finished oligos from the support at the end of the synthesis. An automated synthesizer is shown below. 

[[Image:Be109synthesizer.jpg|thumb|left|200px|'''DNA synthesizer''']]

Consider making a 4-mer: GATC. To begin the synthesis, the 3’ hydroxyl group of the first nucleotide (guanosine) is coupled to a 500 angstrom bead (”cpg”). The guanosine is then treated with a solution (dichloroacteic acid) to remove the cap (a dimethoxytrityl group) protecting its 5’ phosphate. The phosphate can then react with the 3’ hydroxyl of the incoming nucleotide (adenosine), which is also capped on its 5’ end to prevent the addition of multiple adenines to the growing chain. Unincorporated adenine is washed away before the adenine on the bound GA-dinucleotide gets deprotected and reacted with a protected thymidine. Next, the GAT-trinucleotide gets washed to remove unincorporated T, deprotected and reacted with cytosine. Ultimately, the final 4-mer gets released from the solid support to be further purified by chromatography or electrophoresis if needed. 

The same chemistry was used to make the library you’ll be working with but at each nucleotide addition step, an equimolar mixture of all four bases was reacted generating a pool of random DNAs. After uncoupling the oligos from the solid support, the pool was made double stranded with a DNA polymerase and then inserted into the pCT-CON plasmid, with each plasmid getting one version of the oligo. Considerable time in experimental module 3 will be spent on the precise details of cutting and pasting DNA pieces and so the process will not be described in detail here. For now, it will be sufficient to know that the junction of the Aga2 sequence with the 36-base pair oligo is the same in every plasmid but that the 12 amino acid sequence fused to each Aga2 protein can be different.

[[Image:Be109yeastdisplayplasmid36mer.jpg|thumb|left|400px]]
<br style="clear:both" />

With 20 possible bases amino acids at each of the 12 positions, the library you’ll be sorting through today could have 2012 possibilities. In practice, it’s more likely that the number of variants (called the library’s “complexity”) is around 1x10<sup>7</sup>. How many of these will bind gold is unknown. As an extreme example, consider the possibility that there is only one sequence in the library that confers gold binding. If you looked at one yeast from the library there’s only a 1:10<sup>7</sup> chance that it will stick to the gold slide. Clearly your chances of finding the gold-binder improve by looking at all 10<sup>7</sup> yeast (as a point of reference, recall from your earlier experiments that 1 OD has 1x10<sup>7</sup> yeast/ml). In practice, people tend to screen at least 2 or 3 times the library’s complexity (in this case 2-3x10<sup>7</sup>th cells) and only feel confident that they’ve exhausted the library (and themselves) if they begin isolating the same variants multiple times. 

Your experiment today has three parts. At some point you should count the number of colonies on your Petri dishes from last time. This part of the protocol is listed first but can be done at any time during the lab when you get a break. You will also repeat the gold panning from last time, changing one parameter that may improve the protocol. Consider the contents of the solutions you used, the times and temperatures for the binding and washing steps and the mechanics of the yeast-gold incubation. Any one of these can be modified today. Alternatively, you may repeat the panning experiment exactly as before but compare the ability of pAu1 to bind other metals. We have aluminum slides or silver slides for you to try. Finally, you will screen the library of 36 base pair randomized sequences following the protocol from last time. 

==Protocol==

===Part 1: Gold Binding Results===
Examine the Petri dishes that you spread last time. Count the number of colonies on each plate. It is OK to count only 1/2 or 1/4 or 1/8 of the colonies if they are densely growing. In these cases, try to find a representative sector to count and don’t forget to multiply for the total number of colonies on the plate.  

===Part 2: Optimizing Experimental Conditions===
You’ll want to compare pCT-CON and pAu1 in the DY strain grown in galactose. What you vary in the panning protocol is up to you. The drawing below can be used to sketch your experimental set up. You may want to perform this part of the experiment in parallel with the library screen you are performing (Part 3 of today’s protocol). Alternatively, you may want to divide the work so you and your partner each have one protocol to follow. Either is OK but do coordinate and keep notes on both parts of the experiment.

[[Image:Be109emptywells.jpg|thumb|left|400px]]
<br style="clear:both" />

===Part 3: Screening Library===
The protocol you will follow is identical to the one from last time but today you will work with only the DY strains grown in galactose to display the Aga2 fusion proteins. One sample (pCT-CON) serves as a negative control for binding since it displays an Aga2 fusion you’ve seen does not bind gold. The positive control for binding will be the same pAu1 variant you used last time.  Your third sample will be a pool of yeast displaying variants from the 12 amino-acid peptide library. 

[[Image:Be109displaywells.jpg|thumb|left|500px]]
<br style="clear:both" />

===A few important notes:===
*Since all the samples were grown in CAA(Gal), you do not need Glu Binding/Blocking buffer.
*Since you are using only three wells for this part of the experiment, you may be able to do the washing in the remaining three wells of the same 6-well dish. 
*When it comes time to plate the yeast that you’ve eluted from the slides, you should plate 100 ul of your eluted pCT-CON and pAu1 controls but use two –trp plates for your eluted library, plating 100 ul on each. This will double the number of gold binding candidates you’ll have next time.
*Before you leave today be sure to set up four tubes with 2.5 ml of glucose-containing media and four tubes with 5 ml of galactose-containing media. Use colored dots to label the tops of each “A” “B” “C” or “D.” Use sterile technique since these will be used to grow any gold binding candidates for next time.

DONE!

==For next time==
#Prepare a table presenting the plating results of your gold binding experiment from last time as well as a short description of the experiment to go with the table. Include enough information so a person who didn’t do the experiment would understand the table, but not a step-by-step protocol.  
#Prepare two figures with your digital photographs from today’s experiment. Same rules apply for the figure legends. 
#Read the article:  
#*Wackett LP, Dodge AG, and LBM Ellis. “Microbial Genomics and the Periodic Table” ''Appl. Environ. Microbiol'' (2004) '''70''':647-655.

You will be asked to lead the class through some section of this article next time.