{{Template:BE.109}}


==Introduction==

Throughout the term you have used PCR. In the first experimental module you used it to engineer restriction sites that flanked the GFP truncation you were amplifying. In the third experimental module you used it to quantitate the number of transcripts of the lacZ gene when cells were grown under different experimental conditions. It is, perhaps, fitting then that the last bit of "wet lab work" you'll do this term will include PCR. Today you will be performing PCR to amplify the sequence fused to Aga2 in your best gold-binding library clones. The PCR will provide enough material to send to the biopolymer sequencing facility. 

Based on the numerous applications of PCR, it may seem that the technique has been around forever. In fact it is only 20 years old. In 1984, Kary Mullis described this technique for amplifying DNA of known or unknown sequence, realizing immediately the significance of his insight. 

[[Image:Be109karymullis.jpg|thumb|left|150px|'''Kary Mullis''']]

''"Dear Thor!," I exclaimed. I had solved the most annoying problems in DNA chemistry in a single lightening bolt. Abundance and distinction. With two oligonucleotides, DNA polymerase, and the four nucleosidetriphosphates I could make as much of a DNA sequence as I wanted and I could make it on a fragment of a specific size that I could distinguish easily. Somehow, I thought, it had to be an illusion. Otherwise it would change DNA chemistry forever. Otherwise it would make me famous. It was too easy. Someone else would have done it and I would surely have heard of it. We would be doing it all the time. What was I failing to see? "Jennifer, wake up. I've thought of something incredible." '' --Kary Mullis from his Nobel lecture; December 8, 1983

The PCR primers you will use today recognize sequences flanking the Aga2 fusion on the yeast display plasmid. After performing PCR, the DNA produced will be mixed with yet another primer that will serve as a starting point for the sequencing reactions. Sequencing chemistry will be described on the last day of this experimental module [[http://openwetware.org/wiki/BE.109:Bio-material_engineering/Sequence_analysis]], before you examine the sequence data for your own gold-binding clones. 

To being today’s experiment, you will “pop” the gold-binding yeast from your library screen and then amplify the relevant portion of the released plasmid to send for sequencing. As the PCR runs, we will discuss the details of the upcoming presentation and you will have time to work on it with your partner.

==Protocol==

#Begin by counting the colonies that arose from the gold binding experiment you performed last time. The two library candidates that seem to have the highest affinity for gold will be sequenced. 
#You will use the microwave to release the DNA from the yeast. On the tip of a sterile toothpick, pick-up a dab of the correct colony from your Petri dish and swirl it in 20 &mu;l of sterile water in an eppendorf tube. Be sure to label the tube so you know it belongs to your group and which candidate it contains. Repeat with the second candidate you would like to sequence. 
#Close the caps and microwave the tubes in an eppendorf rack for 15 seconds. If you haven’t been wearing gloves, start here. PCR is a sensitive technique and trace amounts of DNA from your fingertips can be detected. Before you proceed, clean up a bit, e.g. wash the barrels of your pipetmen with a paper towel and some 70% EtOH. You could also wash your bench area. Next, move 5 &mu;l of the microwaved mix, yeast debris and all, into a 200 &mu;l PCR tube that you will be given. Again label your tube well (write small!).
#Add 45 &mu;l of PCR cocktail to each PCR tube and leave the tubes on ice until everyone is ready. The details of PCR were discussed in [[BE.109:DNA engineering/DNA engineering by PCR | experimental module 1]] but the recipe and template sequences are listed at the end of today’s protocol if you have immediate questions. 
#Cycle the reactions as: 	
##94&deg; 4 minutes
##94&deg; 1 minute
##52&deg; 1 minute
##72&deg; 2 minute
##repeat steps 2-4 35 times 
##72&deg; 10 minutes
##4&deg; forever 

While these reactions are cycling, you and your partner should work on the research proposal that you will present to the class next time (see FNT, below). 

If we had more time today we would confirm that there is product in your reactions and we might even measure its concentration and remove the PCR salts and buffers before sending it off to the sequencing facility. Since there is insufficient time, we will blindly send it off and hope for the best. Sequencing reactions require 100-200 ng of DNA, and 6.4 pmoles of sequencing primer in a final volume of 24 &mu;l. For each PCR product, make a 1:10 dilution in water and aliquot 2 &mu;l to a full-sized eppendorf tube, labeled properly. A mixture of water and sequencing primer will be added (22&mu;l) and the samples will be taken to the Biopolymer facility in E17 for sequencing. The data will be available for you to examine one week from today. Keep your fingers crossed. 

DONE!

==For next time==
Prepare a 10 minute powerpoint talk that describes the research question you have identified, how you propose to study the question and what you hope to learn. More detailed descriptions of the elements of the oral presentation can be found in the FNT assignments and the protocols associate with this Module. When it is ready, please email your presention to nkuldell AT mit DOT edu. Speaking order will be determined by the order that presentations are recieved.

On the day you present (see announcements on front page for when and where) your team should print out and bring <b>three copies</b> of your powerpoint slides. Black and white is fine and you can print more than one slide per page if you like. You should also write and print out your "talking points" into the comments box of each of the slides you'll present. These are speaking notes for your presentation. They should include the words you'll use to describe each slide and the transitions you've planned between them. For example from last year's presentations, one slide's talking points were: <br>
<i>"Slide shows normalized data (we took logs)
**Red color used for down regulated genes
**Lime green for upregulated
**Olive green used when nothing changed
We dictated what would be considered “Nothing” by putting them into bins
Arbitrarily assigned ‘nothing’ as anything between -1 and 1, because it could just have to do with background and the such

Note many open reading frames and hypothetical proteins

Now let’s look at each component individually!"
</i>


You will be graded on the integrated success of your presentation: concepts, slides, talking points, and presentation.

==Reagents list==
*PCR forward primer (100 pmole/ul)<br> CAAATGTAATAAAAGATCGAATTCTACTTCATACATTTTCAATT
*PCR reverse primer (100 pmole/ul)<br> GAACAAAGTCGTTTTTGTTACCTATACACTGTTG
*Sequencing primer (1 pmole/ul)<br> GACAATAGCTCGACGATTGAAGGTAG
*PCR Master Mix (2.5X)
**62.5 U/ml Taq DNA Polymerase
**125 mM KCl
**75 mM Tris-HCl, pH 8.3
**3.75 mM Mg(OAc)<sub>2</sub>
**500 uM each dNTP