{{Template:BE.109}}

==Introduction==
There’s a saying that goes something like: “it’s not what you don’t know; it’s what you know that isn’t so.” In other words, it’s easy to head down the wrong path based on an incorrect presumption or misunderstanding. The results from quantitative PCR can easily mislead an investigator. Errors are introduced because the number of sample manipulations is large, and the readout is indirect. A pessimist might argue that the number of assumptions makes the data uninformative. A more optimistic observer might believe that good experimental controls and a healthy dose of skepticism can make the results from this technique informative. To decide for yourself, it will be important to understand the theory underlying quantitative PCR and the experimental design that you are using.

[[Image:Be109PCRrounds.jpg|thumb|left|600px|'''Exponential increase in PCR product through rounds of amplification.''']]	
<br style="clear:both" />

Quantitative PCR is possible since each round of amplification should theoretically double the number of DNA products in the reaction tube. This theoretical efficiency can only be achieved with an abundant pool of dNTPs and primers and it will futher depend on experimental conditions that might vary, like the length of the sequence to be amplified and its sequence. Invariably, however, as the pool of dNTPs and primers deplete, fewer transcripts are made in each round. A graph of transcript number vs. PCR cycle number would reveal the exponential increase in transcript number and its plateau as the reagents are depleted. 

[[Image:Be109qPCR.jpg|thumb|left|400px|'''Graph of transcript number versus PCR cycle number.''' <br>Data from: [http://bio.takara.co.jp/BIO_EN/application.asp?C_ID=C1280&ID=559 Takara products catalog]]]	        
<br style="clear:both" />

In the sample data above, the product of seven independent reactions is being monitored and shown with curves of different colors. It’s clear from the data that each reaction is initiating the exponential phase of PCR and then reaching a plateau, except for the last two reactions that didn’t start their exponential phase until later in the PCR cycling program. The data illustrates how reactions may begin their exponential phase at a different cycle numbers.  What is fundamentally important to understanding this method is the fact that '''the cycle number at which the exponential phase begins depends on the number of starting templates in the reaction'''. Fortunately, the number of transcripts (shown on the y-axis) can be sensitively and accurately determined by measuring fluorescence units. Recall the reactions you set up last time included SYBR Green, a dye that fluoresces nearly 300X more strongly when it is bound to double-stranded DNA. Thus fluorescence is used as a proxy measurement for amplified product. 

“Ct” is the term used to compare quantitative PCR profiles like the seven shown above. Ct is a user-defined fluorescence threshold that the reactions all cross. It is possible to set the threshold anywhere in the exponential phase. A second user-defined parameter is the reaction’s baseline, which must be subtracted to remove machine noise. As you work with your data today, you should consider how your choices for these two parameters affect your data interpretation. 

With the baseline subtracted and the Ct defined, it is possible to compare the amount of starting template because in the exponential phase of amplification, the amount of double stranded DNA (y) is equal to the starting number of templates (N<sub>0</sub>) times 2<sup>n</sup>, where n is the number of PCR cycles. For example, by starting with 1000 copies of template DNA, you find that reactions crosses Ct at 28th cycle, then a reaction that crosses Ct at the 29th cycle had 500 copies of the target initially, a reaction that crosses Ct at the 30th cycle started with 250 copies of the amplified target, etc. In other words: reactions that differ by 1 Ct began with a 2X difference in starting target.

What Ct can reveal is dependent on the design of your experiment. Commonly, Ct is expressed relative to some “housekeeping” gene whose expression is expected to be unchanged by the experimental manipulations that affect the genes of interest. In your experiment, you are using cell numbers as standards, assuming that each cell will have one copy of the LacZ gene to amplify. Today you will use the data from these standards to generate a standard curve that relates amount of product to cell number. Using this standard curve, it may be possible to know the absolute number of ''LacZ'' transcripts/cell in each of the experimental conditions you tried. 

Before any grand conclusions can be drawn from your data, there is one important question to ask. How do you know that changes in fluorescence arise from lacZ amplification? It is easy to imagine that the PCR primers can recognize other templates in your reactions and so are producing many species of double stranded DNA molecules. Since you’re not running the products out on a gel, how can you detect other products? The most common method is to run a “melting curve” at the very end of the amplification cycles. By raising the temperature of the final reactions, you can denature the double stranded DNA, changing its fluorescence. Sequence and length of the DNA will determine the temperature at which the melting occurs. For example, in the data shown below, there is one species of DNA that is denaturing at the same temperature in all the samples. Before you begin to look at your own data you should imagine what the melting curve might look like for a reaction with multiple products.

[[Image:Be109meltingcurve.jpg|thumb|left|400px|'''Melting curves.''' <br>Data from: [http://bio.takara.co.jp/BIO_EN/application.asp?C_ID=C1280&ID=559 Takara product catalog]]]
<br style="clear:both" />

==Protocol==
===Data Analysis===
The data from your q-PCR run should now be loaded onto the lab laptops as a file with a light-bulb icon and ending in .tad2. Double click on the file to open it, ignoring the warnings that your laptop can't directly communicate with the q-PCR machine itself. Everyone's samples will be available for you to examine but begin by analyzing your own. 

One place to start is to examine the data for the DNA standards you've run. Find your samples in the 96-well dish shown in the upper left of the window and select the 6 wells that you included to generate your standard curve. The data related to these wells will appear in the lower left window, color coded as shown in the 96-well dish shown in the upper left of the window.By scrolling your mouse over the curves in the lower left window you can see the data for each sample, data that is generated using the program's default options. Before this data is meaningful, though, you will need to enter the values for the DNA standards. 

This can be done by selecting the "Master" file from the upper left list on the left-most portion of the window. Next select "edit" from the "plate setup" portion of the window. This will allow you to specify your DNA standards. Select the 6 relevant wells then click the "specify quant standards" found on the lower right of the window. Choose "copies" from the pull-down units menu and enter the number of lacZ copies in each of the 6 standard wells. You must use the "enter" button on your computer to move from cell to cell on the spreadsheet (don't know why). When you've entered all the relevant values, select "OK" then "OK" again to return to the master data file.   

Select "quantitation" and you'll be able to play with the threshold options, found in the right-most section of that window. Settings that have worked in the past include selection of "subtract blanks" as well as selection of "subtract baseline" options. See how the different minima affect the output of your DNA standards, selectable by scrolling the mouse over the wells shown in the upper left of the window. You can increase the cycle range to 35 (since we ran 35 cycles) and you can manually set the threshold to 0.07. You can try this or any other setting you prefer and examine how these affect the titration curves. You should use the analysis software to determine the equation and r-squared for your standard curve. You'll have to "select wells" and deselect the other standards from consideration. Deselected wells will appear as red-points on the data curve. Alternatively you can modify the master data file so the other blanks in the class are now "blanks." If you decide to leave off certain points, note which and why. 

Next you'll want to assess the number of lacZ copies that were measured in each of your two experimental samples. This can be "eyeballed" by selecting the relevant wells from the 96-well dish shown in the upper left of the window and scrolling over the curves shown in the lower left of the window. Do you see a difference in the plus and minus RT samples? Do the triplicates correlate well? Do you see a difference under the two conditions you measured? Do these differences agree with the b-gal data you measured on the cells? Record all this in your lab notebook and also begin to consider how you will enter it in the Registry of Standard Biological Parts. 

The data from your run has been collected by the analysis software and can be seen by selecting the "quantify calculations" tab at the bottom of the window. You will have to select all 16 of your samples to see them listed and then choose "show selected wells" from the well selection menu. The data here includes the Ct and the copy number according to the standard curve you used. Export it to an excel spreadsheet by copying it to the clipboard then opening the data in Excel. You will have to copy and paste the standard curve equation and r-squared separately. You should also note which primer pair you used for your experiment since different primer pairs may behave quite differently depending on the efficiency with which they amplify and the stability of the region they anneal to. Hand in this spreadsheet before you leave lab. You may also want a copy for your lab notebook.
 
The copy number shown on the spreadsheet refers to the number in the q-PCR itself....not a particularly meaningful number on its own. Knowing the number of cells you used for your RNA preparation and knowing what fraction of the RNA you used for the RT reactions, then the q-PCR, you can try to convert the number to copies per cell, recording the data in your notebook and preparing it for the Registry. 

Since the q-PCR results are indirect readouts of dsDNA you should examine the melting curve associated with your samples. What can this tell you about each reaction? Note your findings in your notebook as well as describing them on the Registry.

===Jamboree-lite===
For the past few summers, teams made mostly of undergraduate students have participated in a genetically engineered machines competition. The teams then gathered in the fall for a "Jamboree" to show off their successes from the summer's work. The bacterial photography system was initially presented at such a jamboree in November 2004. To finish today's lab, you'll have a chance to share with the group what you have proposed to build in your essay. What do you think about its usefulness? Are you motivated to solve a particular problem? Are there existing designs that inspired you? Are there people who gave you some good ideas? Did you need to specify special parts for your idea? Do you know how to get them? What do you think are your chances for success? Does your design fit cleanly into your model for synthetic biology? What will be needed to make it a reality? Do you want to build it? Do you want someone else to build it? 

Say what you like but be prepared to talk about your essay and to ask others about theirs. 

DONE!

....on to Mod4.