Protocol for Real-Time RT-PCR<br><br>



This protocol describes the detailed experimental procedure for real-time RT-PCR using SYBR Green I as mentioned in Xiaowei Wang and Brian Seed (2003) A PCR primer bank for quantitative gene expression analysis. Nucleic Acids Research 31(24): e154; pp.1-8. Please refer to this paper and the PrimerBank Help page for more background information. The procedure begins with reverse transcription of total RNA. The cDNA is then used as template for real-time PCR with gene specific primers. You may need to modify this protocol if you use different reagents or instruments for real-time PCR.<br><br>


Time required<br><br>

cDNA synthesis: 2 hours.<br>
real-time PCR: 2 hours.<br>
Dissociation curve analysis: 0.5 hour.<br><br>

Reagents and Equipments<br><br>


•	Oligonucleotide Primers. Gene specific primers are retrieved from PrimerBank (http://pga.mgh.harvard.edu/primerbank/). These primers are ordered from the MGH DNA Core facility (https://dnacore.mgh.harvard.edu/synthesis/index.shtml). All the primers are desalted and both UV absorbance and capillary electrophoresis are used to assess the quality of primer synthesis. <br>
•	Mouse total liver RNA (Stratagene).<br>
•	Mouse total RNA master panel (BD Biosciences / Clontech). <br>
•	SYBR Green PCR master mix, 200 reactions  (Applied Biosystems).<br>
•	Optical tube and cap strips (Applied Biosystems).<br>
•	SuperScript First-Strand Synthesis System for RT-PCR (Invitrogen).<br>
•	25 bp DNA ladder (Invitrogen).<br>
•	ABI Prism 7000 Sequence Detection System (Applied Biosystems).<br>
•	ABI Prism 7000 SDS software (Applied Biosystems).<br>
•	3% ReadyAgarose Precast Gel (Bio-Rad).<br>
•	Agarose gel electrophoresis apparatus (Bio-Rad).<br><br>

Detailed procedure<br><br>

Reverse Transcription<br><br>

Reverse Transcription is carried out with the SuperScript First-Strand Synthesis System for RT-PCR. The following procedure is based on Invitrogen’s protocol. <br><br>

1. Prepare the following RNA/primer mixture in each tube:<br><br>

Total RNA	5 μg<br>
random hexamers (50 ng/μl)	3 μl<br>
10 mM dNTP mix	1 μl<br>
DEPC H2O	to 10 μl<br><br>

2. Incubate the samples at 65°C for 5 min and then on ice for at least 1 min.<br>
3. Prepare reaction master mixture. For each reaction:<br><br>

10x RT buffer	2 μl<br>
25 mM MgCl2	4 μl<br>
0.1 M DTT 	2 μl<br>
RNAaseOUT	1 μl<br><br>

4. Add the reaction mixture to the RNA/primer mixture, mix briefly, and then place at room temperature for 2 min.<br>
5. Add 1 μl (50 units) of SuperScript II RT to each tube, mix and incubate at 25°C for 10 min.<br>
6. Incubate the tubes at 42°C for 50 min, heat inactivate at 70°C for 15 min, and then chill on ice.<br>
7. Add 1 μl RNase H and incubate at 37°C for 20 min. <br>
8. Store the 1st strand cDNA at -20°C until use for real-time PCR.<br><br>

Real-time PCR<br><br>

1. Normalize the primer concentrations and mix gene-specific forward and reverse primer pair. Each primer (forward or reverse) concentration in the mixture is 5 pmol/μl.<br>
2. Set up the experiment and the following PCR program on ABI Prism SDS 7000. Do not click on the dissociation protocol if you want to check the PCR result by agarose gel. Save a copy of the setup file and delete all PCR cycles (used for later dissociation curve analysis). Please note the extension steps are slightly different from described in our paper.<br>
1.	50°C 2 min, 1 cycle<br>
2.	95°C 10 min, 1 cycle<br>
3.	95 °C 15 s -> 60 °C 30 s -> 72 °C 30 s, 40 cycles<br>
4.	72°C 10 min, 1 cycle<br>
3. A real-time PCR reaction mixture can be either 50 μl or 25 μl. Prepare the following mixture in each optical tube.<br><br>

25 μl SYBR Green Mix (2x)<br>
0.5 μl liver cDNA<br>
2 μl primer pair mix (5 pmol/μl each primer)<br>
22.5 μl H2O	<br>
OR	<br>
12.5 μl SYBR Green Mix (2x)<br>
0.2 μl liver cDNA<br>
1 μl primer pair mix (5 pmol/μl each primer)<br>
11.3 μl H2O<br>

4. After PCR is finished, remove the tubes from the machine. The PCR specificity is examined by 3% agarose gel using 5 μl from each reaction. <br>

5. Put the tubes back in SDS 7000 and perform dissociation curve analysis with the saved copy of the setup file.<br>

6. Analyze the real-time PCR result with the SDS 7000 software. Check to see if there is any bimodal dissociation curve or abnormal amplification plot.<br>



Troubleshooting<br>
<br>


Here I listed a few major causes for real-time PCR failures. Please read the PrimerBank Help page for more details.<br><br>

Little or no PCR product. Poor quality of PCR templates, primers, or reagents may lead to PCR failures. First, please include appropriate PCR controls to eliminate these possibilities. Some genes are expressed transiently or only in certain tissues. In our experience, this is the most likely cause for negative PCR results. Please read literature for the gene expression patterns. One caveat is that microarrays are not always reliable at measuring gene expressions. After switching to the appropriate templates, we obtained positive PCR results in contrast to the otherwise negative PCRs (see our paper for more details). <br><br>

Poor PCR amplification efficiency. The accuracy of real-time PCR is highly dependent on PCR efficiency. A reasonable efficiency should be at least 80%. Poor primer quality is the leading cause for poor PCR efficiency. In this case, the PCR amplification curve usually reaches plateau early and the final fluorescence intensity is significantly lower than that of most other PCRs. This problem may be solved with re-synthesized primers.<br><br>

Primer dimer. Primer dimer may be occasionally observed if the gene expression level is very low. If this is the case, increasing the template amount may help eliminate the primer dimer formation.<br><br>

Multiple bands on gel or multiple peaks in the melting curve. Agarose gel electrophoresis or melting curve analysis may not always reliably measure PCR specificity. From our experience, bimodal melting curves are sometimes observed for long amplicons (> 200 bp) even when the PCRs are specific. The observed heterogeneity in melting temperature is due to internal sequence inhomogeneity (e.g. independently melting blocks of high and low GC content) rather than non-specific amplicon. On the other hand, for short amplicons (< 150 bp) very weak (and fussy) bands migrating ahead of the major specific bands are sometimes observed on agarose gel. These weak bands are super-structured or single-stranded version of the specific amplicons in equilibrium state and therefore should be considered specific. Although gel electrophoresis or melting curve analysis alone may not be 100% reliable, the combination of both can always reveal PCR specificity in our experience.<br><br>

Non-specific amplicons. Non-specific amplicons, identified by both gel electrophoresis and melting curve analysis, give misleading real-time PCR result. To avoid this problem, please make sure to perform hot-start PCR and use at least 60°C annealing temperature. We noticed not all hot-start Taq polymerases are equally efficient at suppressing polymerase activity during sample setup. The SYBR Green PCR master mix described here always gives us satisfactory results. If the non-specific amplicon is persistent, you have to choose a different primer pair for the gene of interest. You are also encouraged to report bad primers to Xiaowei Wang (xwang@molbio.mgh.harvard.edu). <br><br>