So, the first method allows you to actually insert in a different intron, different exons, etc without issue.  This alternative method would allow you to make mutations, just not such large wholesale mutations.<br>
<br>
Here, you would be working with the same DEK exon, intron, exon combo.<br>
<br>
Let's say that your interest lies in the 3'splice site area (right before the second exon).  This is what you would do.<br>
<br>
You would design a normal short 5' primer to act as your forward, instead of a large one.<br>
5'-AAAAGAGTCTCATCGTGGAAGGCAAG-3' (Tm=60°C)<br><br>

Then you would use Primer 4 as your first reverse.<br><br>

Your template would be genomic DNA (in this case from human).  <br><br>

Do about 10 rounds of PCR off of 100ng of genomic DNA in a 30µl reaction (more rounds because this is now real life, not an artificial construction).<br><br>

Clean up the PCR by gel purifying the product (need to get rid of the genomic)<br><br>

Now, take your short forward primer and do 10 more cycles with primer 5 (30µl reaction)<br><br>

Now, increase the volume of your PCR reaction to 100µl, adding more short forward primer, plus adding primer 6 (this dilutes primer 5 and allows primer 6 to do it's job).<br><br>

Why I designed it this way:<br><br>

Primer 4 is the primer of interest.  It spans the 3'end of the intron.  By using this primer, you can also use other primers(4, 5, or 6) that are mutants, just hook 4 with 5 and 6 and you are good to go.  By doing it this way, instead of short forward primer  with primer 6 on the genomic, you are also testing to see if you can get this to amplify up right using these primers.  There are lots of options.  Imagination is all you need.