Return to [http://www.openwetware.org/wiki/IGEM:UC_Berkeley/2006 main page]
----
==[[User:JCAnderson|JCAnderson]] 13:48, 15 November 2006 (EST)==
The new riboregulator lock, pSB3C6-J23090 was a perfect match, saw the whole thing EcoRI to PstI in read jca313.  The EcoRI site had an extra 'A' in the read, but that must be a call error.  Jen is replating the relavent controls for assaying this guy's activity.  Sam will transform it with the key today.

==[[User:JCAnderson|JCAnderson]] 17:24, 19 September 2006 (EDT)==
Tecan of pSB3C6-J23077 #1 with the current best key, pSB1A2-J23066 came about as expected.
*TG1  21
*J01122 27
*J01122/1129  71
*J01022  4629
*J23077 alone  43
*J23077/J23066 598
*J23077/J23066 507
*J23077/J23066 538
*J23077/J23066 502

We will proceed with this lock for the conjugation and NAND gate.

'''KAD_08/11/06'''
*renamed pSB1A2-J23063 and pSB1A2-J23064 constructs. 
*grew up one colony of pSB3C6-pBca1025-1 in cam. 
*grew up 1 colony of pJ23006-J23022-1 to tecan (along with all controls)
*grew a 5ml cultures of pSB1A2-J01129
*ran a tecan on pJ23006-J23041, 42, 43, and 44. 

TGI: 23, 1122: 37, 1122x1129: 53, 1022: 3634<br>
41A: 96, 41B: 96, 41C: 86<br>
42A: 125, 42B: 141, 42C: 141<br>
43A: 286, 43B: 272, 43C: 284<br>
44A: 217, 44B: 185, 44C: 66<br>


'''KAD_08/10/06'''
*miniprepped pJ23006-J23058-1, pJ23006-J23022-1, pJ23006-J23033-D, pJ23006-J23033-E, pJ23006-J23033-E, pJ23006-J23033-F, pJ23006-J23034-D, pJ23006-J23034-E, pJ23006-J23034-F. 
*transformed pJ23006-J23022-1 into 1122s. 
*digested pJ2306-J23058-1 with BamHI/HindIII
*started extension pcr with KB030/KB031 and KB028/KB029
*grew up 3 colonies each pJ23006-J23043, pJ23006-J23044, pJ23006-J23042, pJ23006-J23041, along with TGIs, pSB3C6-J01122, pSB3C6-J01022, and pSB3C6-J01122xpSB1A2-J01129.




'''KAD_08/09/06'''
*J23054 is wrong. 
*pSB1A2-J23046 was miniprepped and screened yesterday.  all clones appear correct.[[Image:080806Gel.jpg|right|100px]]
*miniprepped pSB1A2-J23061. screened with EcoRI/AlwnI to yield 1510bp and 800bp bands. 2 and 4 look the best. [[Image:080906Gel.jpg|right|100px]]
*changed pSB1A2-J23057 to pSB1A2-J23061 to elminate confusion between 2 constructs with the same name. 
*sent pSB1A2-J23046-1 and 2, pSB1A2-J23061-2 and 4 for sequencing. (ca998)




'''KAD_08/07/06'''
*plate of pSB1A2-J23046 shows few colonies, even after growing multiple days in incubator.  construct may be toxic.  will try using a variety of promoters to figure out which works best with 4 base suppressor. grew up 4 in AMP/CARB. 
*grew 4 pJ23006-J23009-1 x pSB1A2-J23023 (pSB1A2-J23054) in AMP/CARB
*redigested new RA2 promoter (pSB1A2-J23104) with AlwnI/SpeI/PstI to ligate with pSB1A2-J23037 digest (XbaI/AlwnI) to make pSB1A2-J23057 construct. Made construction and ApE file. 
*transformed pSB1A2-J23057 into TGIs and plated on AMP. 
*confirmed sequencing with stickers on correct miniprep tubes


'''KAD_08/04/06'''
*ligated pJ23006-J23009-1 with pSB1A2-J23023 (to construct pSB1A2-J23054 construct)
*transformed pSB1A2-J23023 x pJ23006-J23009-1 into TGI's, plated on AMP (pSB1A2-J23054)
*digested pSB1A3-J23037-4 (XbaI/AlwnI) and pBca1020-Bca1022-E(J23102) (SpeI/AlwnI/PstI), purified, ligated, transformed, plated on AMP. (for pSB1A2-J23046 construct)
*tecan of LuxI key construct showed that there was no increase in fluroescence with this construct.  stick with terminator constructs. 




'''KAD_08/03/06'''
*grew up pSB1A2-J23029-4D x 1122 (along with 1122, 1022, TGIs, and 1122 x 1129 from fridge plates) for tecan on fri
*sequenced pSB3C6-Bca1025-1 and 2 with KB009, made -80 of both
*will wait for sequencing to proceed with construction of pBca1020-J23037
*tecan of pSB1A2-J01129-B0015 (pSB1A2-J23023), 1122, 1022, TGIs, and 1122 x 1129.  Big improvement with the double terminator! (@ 120)
 - pSB1A2-J01129-B0015-1xpSB3C6-J01122 = 565
 - pSB1A2-J01129-B0015-2xpSB3C6-J01122 = 445
 - pSB3C6-J01022 = 8728
 - pSB3C6-J01122 = 57
 - TGI = 40
 - pSB3C6-J01122xpSB1A2-J01129=139
*will go ahead with Key3D-TT construct, using pJ23006-J23009-1(SpeI/AlwnI) and pSB1A2-J23023(XbaI/AlwnI) 
*digested pJ23006-J23009-1(SpeI/AlwnI) and pSB1A2-J23023(XbaI/AlwnI). purified pSB1A2-J23023, had to redigest pJ23006-J23009-1. 



'''KAD_08/02/06'''
*grew up pSB1A2-J01129-B0015 (pSB1A2-J23023) from plate for tecan (along with 1122, 1022, TGIs, and 1122 x 1129 from fridge plates for tecan on thurs
*miniprep pSB1A2-J23029-2D and 4D
*grew up pBca1020-Bca1022-E (J23102) from -80
*sent pSB1A2-J23029-2D and 4D for sequencing
*transformed pSB1A2-J23029-2D and 4D into 1122 for a tecan on Friday
*looked at sequencing results for iG043 and iG044, and confirmed the presence of AAAAAAGGG in the pSB1A2-J23023 contructs, and an A to T bp variation in pSB3C6-J01129. changed the ApE files accordingly.  
*made Ape construct and construction file of pBca1020-J23037, added Ser2(AGGA) to ApE library



'''KAD_07/31/06'''
*none of the pSB1A2-J23029 cultures grew, must have been in wrong antibiotic? regrew new cultures. 
*updated sequencing (iG043 and iG044, pSB1A2-J01129-B0015-1 and 2)
*colony PCR of pSB1A2-J23029 C and D.  Screened product, and only 2D and 4D looked correct.  
*screened colony PCR of pSB1A3-J23037. If correct clones, should yield 364 bp fragments. Clones 3 and 4 looked correct. 
*transformed pSB1A2-J01129-B0015-1 and 2 into 1122, plated
*sent pSB1A3-J23037-3 and 4 for sequencing




'''KAD_07/31/06'''
*grew cultures of pSB1A2-J23029 plate A and plate B. 
*colony PCR of pSB1A3-J23037 with ca998 and BBa_G00101, should yield 364 bp fragment
*miniprep of pSB1A3-J23037



'''KAD_07/27/06'''
*plate of pSB1A3-J23037 in TGI shows ~10 colonies.  grew 4 in AMP
*digested pSB1A2-J01129 with SpeI/PstI, pSB1AK3-F1610 with XbaI/PstI for new approach to construction of pSB1A3-J23029 in case sequencing shows inaccuracy of previous construct
*religated products of digestion made 07/21 in case construction was complicated by ligation or transformation. transformed into TGI
*purified pSB1A2-J01129, pSB1AK3-F1610; then ligated and transformed into TGI


'''KAD_07/26/06'''
*grew up pK2 and pK3 in AMP, pAC-SerZAGGA in CAM from plates
*cleaned up PCR of pACSerZAGGA
*miniprepped pSB1A2-I13450, pJ23025-J01129 1-4.
*mapped pSB1A2-I13450 with MfeI/AlwnI, expected one large band and one small 200bp band, and saw both, but the 200 bp fragment was faint.  sequencing will confirm result. 
*sent pACSerZAGGA PCR, pSB1A2-J23029, and pSB1A3-I13450 for sequencing.
*will wait for sequencing data confirmation to proceed with the luxI (J23029), because mapping was not definitive and unclear. 
*digested pSB1A2-I13450 with EcoRI/PstI, digested pACSerZAGGA with NsiI/MfeI. 
*gel purified pSB1A2-I13450, short fragment cleanup of pACSerZAGGA 
*ligated pSB1A2-I13450 with pACSerZAGGA





'''KAD_07/25/06'''
*transformation of pJ23006-J23009-1 into J01122s was successful, with a 35+ colonies.
*transformation of pJ23024-J01129 into TGI yields no colonies. 
*transformation of pJ23025-J01129 into TGI yields 4 very small colonies. 
*transformation of pSB1A3-I13450 into TGIs shows 2 colonies only. 
*grew 4 cultures of pJ23006-J23009-1, 2 of pSB1A3-I13450, 4 of pJ23025-J01129, all in AMP. 
*[J23024 = pSB1A2-J01129 + J01007(pK2)]
*[J23025 = pSB1A2-J01129 + J01006(pK3)]
*reconstructed pJ23024 and J23025 (in J01122 due to a labeling error of the previous construct), plate on AMP
*made APE files of pSB1A2-J23024 and pSB1A2-J23025
*miniprep pSB1A2-J23018 and pSB1A2-J01129-F1610
*replated pK2 and pK3 from -80.  the cause of smearing on gel when trying to subclone into pSB1A2-J01129 is probably an unclean miniprep of pK2 and pK3. 
*transformed pACSerZAGGA in TGI, plated on CAM
*Phusion PCR of pACSerZAGGA with ca1000 (cagggcagggtcgttaaatagc) and ca1001 ( ggcggttttttcgttttcagagc), hope to yield a 1647bp product. came out clean.
*gel mapping of pSB1A2-J01129-F1610 (pSB1A2-J23029) with AlwnI/SpeI.  Correct clone will have 2 bands, 2495bp and 624bp. Parent will have 2 bands, 1689bp and 546bp.  Both have the same restriction sites so it was difficult to pick enzymes that would leave one band in one, and two bands in the other, but hopefully the thousand bp difference will be enough.  Problems with AlwnI resulted in smearing on the gel, but despite that complication, clone 1 is wrong.  It was difficult to tell band sizes, because of the smearing, but clones 2,3 and 4 showed identical bands and may be correct. 





'''KAD_07/24/06'''
*plate of pSB1A2-J23018 shows very few clones, grew up 4 in AMP  (maybe we should re-do)
*plate of pSB1A2-J01129-F1610 shows a good number of clones, grew up 4 in AMP
*miniprepped pJ23006-J23009 (constructs 1-4) and digested with AlwnI/BamHI, then mapped.  Correct clones yield 2 bands, 1749 bp and 627 bp. 
[[Image:pJ23006-J23009 1-4.jpg|right|100px]]
*mapped Friday's PCR of Key3 constructs with TT (pJ23006-J23007-B0015 1 and 2, pJ23006-J23008-B0015 1 and 2, pSB1A2-J01129-B0015 1 and 2)[[Image:Key3 plus TT.jpg|right|100px]]
*sent pJ23006-J23009-1, pSB1A2-J01129-B0015-1, and pSB1A2-J01129-B0015-2 for sequencing. 
*transformed pSB1A3-I13450 into TGIs. 
*made construction and APE files for pSB1A2-J23024 and pSB1A2-J23025 (key3 parts with pK3 and pK2 promoters.
*digested pSB1A2-J01129, (pSB1A2)-J01006 (pK3), and (pSB1A2)-J01007 (pK2) with XbaI/AlwnI, SpeI/AlwnI, and SpeI/AlwnI respectively, for subclone to make pSB1A2-J23024 and pSB1A2-J23025.  We are unsure as to which plasmid the pK promoters are actually on, but we assume they are both on pSB1A2.



'''KAD_07/21/06'''
*grew up 4 clones of pJ23006-J23009 (1, 2, 3 and 4) 
*mini prepped pSB1AK3-F1610, pJ23006-J23007-B0015-1, pJ23006-J23007-B0015-2, pJ23006-J23008-B0015-1, pJ23006-J23008-B-15-2, pSB1A2-J01129-B0015-1, pSB1A2-J01129-B0015-2, and pSB2K3-E1010. 
*digested, gel purified, ligated, transformed, plated pSB1A2-J01129 with SpeI/AlwnI and pSB1AK3-F1610 to make pSB1A2-J01129-F1610.
*digested, gel purified, ligated, transformed, plated pBca1021-E0040 with XbaI/PstI and pSB2K3-E1010 with XbaI/PstI to make pSB1A2-J23018.
*colony PCR of Key3 constructs with TT with ca998 (gtatcacgaggcagaatttcag) and ca1020R (ccggactgcagcggccgcttctagtatataaacg) (pJ23006-J23007-B0015 1 and 2, pJ23006-J23008-B0015 1 and 2, pSB1A2-J01129-B0015 1 and 2)'''still needs to be mapped'''
*made/finished construction files for pSB1A2-J23018 and pSB1A2-J01129-F1610.
*made APE files for pSB1A2-J23018, pSB1A2-J01129-F1610, and pJ23006-J23008-B0015.



for friday:<br
*Grow up pJ23006-J23009 ('''if you do this early enough then you can miniprep and transorm into j01122 the same day.''')
*Begin contruction of Lock variants.  You are going to want to use Chris's construct that is {EcoRI][TT][Ptet][RBS][XbaI][GFP][TT][SpeI][PstI].  Digest this with XbaI/PstI and instert a basic RFP part.  Im not sure what part this is but im sure we have it somewhere.  Tranform this product and ask someone to grow you up a colony.
*order oligos for lock3; chris will have these.
*Continue the construction of the pK3-Key3, pK2-Key3 constructs.
*miniprep: LuxI (pSB1AK3-F1610), pJ23006-J23008-B0015, pJ23006-J23007-B0015, and pSB1A2-J01129-B0015, pSB2K3-E1010
*read up on rtPCR

*look at iG031 and make sure that the point mutation is not an issue.



'''BCH_07/20/06'''
#Minipreped: pK3, pK2 (J01006, J01007)
#Grew up: LuxI (pSB1AK3-F1610), pJ23006-J23008-B0015, pJ23006-J23007-B0015, and pSB1A2-J01129-B0015, pSB2K3-E1010 
#remade and transformed pJ23006-J23009 (key3d) and plated.
#Sequenced pJ23006-994Lib-1, and pJ23006-994Lib-8
#Discussed rtPCR plausibility.
#made oligos for lock3 modifications

'''BCH_07/19/06'''
*construct pSB1A2-J01129-B0015
*threw out pJ23006 -80 stock because it was obviously contaminated.

'''KAD_07/19/06'''
*ran Tecan of pJ23006-J23007, pJ23006-J23008, 1122, 1122 x 1129, and TG1: 332, 1103, 451, 1181, and 336 (@ 120). We had the greatest reaction with keys that had slight mismatches and hairpins, so we will go ahead with 1122 x 1129 and transform with the double terminator. Hopefully this will produce even greater fluroescent results.  
*mini prepped pSB1AK3-B0015 and pJ23006
*grew up pK3 and pK2 (J01006 and J01007) from -80s. the key 3 basic part and an RBS-lacZ alpha-double terminator parts do exist (BBa_J01086 and BBa_I12012) but we do not currently have them. 
*transformed lacZ alpha (BBa_E0033) into TGIs. 
*transformed pJ23006-J23008-B0015, pJ23006-J23007-B0015, and pSB1A2-J01129-B0015 constructs after ligation. 


'''KAD_07/18/06'''
*fixed sequencing page error, put white stickers on mini preps to confirm sequencing accurace. 
*grew up key 3b and 3c in 1122
*mini prepped 994library in TG1s (1-10)and pSB1A2-R0040.
*plated pJ23006 from -80 to see if that stock is also contaminated.
*the next step in key modification: inserting the key and pTet into pSB1A2-B0015 before the double terminator.  
*regrew TG1s, 1122, 1122/1129, and 1022 for Tecan tomorrow. 



'''BCH_07/17/06'''
*took tecan results from TGI, 1122, 1122 x 1129, and I13521. The results were, respectively, (31, 44, 78, 6415).  The expriment will be redone because the flouresence numbers that are shown here were obtained from cultures that were allowed to sit out over the weekend.  New cultures have been grown and the reads will be repeated tomorrow.
*grew up a culture of pSB1A2-R0040.
*grew up cultures of 994Lib in TGI to miniprep and sequence to make sure that the Library was even getting into our cells and no funny business was going down.
*results from assay show that B0015 grows on Kan and Amp and is almost certainly on pSB1AK3.
*Digested pJ23006-J23009 miniprep with BamHI and AlwnI to confirm that the key was subcloned into the backbone.  All 3 are parent vector or possibly contamination.  Because of excessive fail, we will retransform the J23006 plasmid into fresh TG1 and attempt to insert keys all over again. 
*Ran a PCR of pJ23006-J23007 and pJ23006-J23008. Expected bands are 407 and 474 repepectively.  All bands look to be correct except one.  see image.
[[Image:23007,8 key screens.tif|thumb|100px]]















'''KAD_07/14/06'''
*plate of pSB1A2-R0040 shows no colonies. and so we retransformed them and also plated from the -80 stock.
*plates of 994 library in both TG1 and pSB3C6-J01122 show few colonies, none of which are red. as such, we are retransforming yet again.
*ran colony PCR to screen keys (pJ23006-J23007-1.1, 2.1, 3.1 and 4.1, pJ23006-J23008-1.1, 2.1, 3.1, and 4.1, and pJ23006-J23009-1.1, 2.1, 3.1, and 4.1) these failed miserably so we miniprepped them and will map them moday.
*ran colony PCR of the 994library in TG1 (10 colonies screened).  see image.
*grew up more B0015 in Amp and LB. (we arent sure if its in pSB1A2 or pSB1AK3 so i assayed and plated on an amp plate and a kan plate.)
[[Image:994library gel digest.jpg|100px]]




'''KAD_071206'''
*grew up colonies of keys (7, 8, 9) in amp/carb
*sequencing read failed for iG028 and iG029 (pJ23006-J23007-3 and pJ23006-J23008-2).  This sequencing primer (ca998) has worked for other inserts in pJ23006, so we will send the same samples again but with more DNA in the sample. 
*scraped 994library in TG1 plate to grow up for -80. this was a mistake so we just retransformed.
*scraped 994library in pSB3C6-J01122 plate to regrow for more colonies.  None were red. this was a mistake so we just retransformed.
*Made construction files for pSB3C6-J01011 and all intermediates.
*transformed pSB1A2-R0040 into TG1. they are on a plate.
*ordered oligos for Ptet, Lock2, mRFP. (KB009, KB010, KB011)
 


'''KAD_07/12/06'''
*sequencing confirmed accuracy of pJ23006, pSB1A2-J01129, pJ23006-I13522-1, and pJ23006-I13522-2, but showed that pSB1A2-J01122-1 was the wrong clone. 
*miniprepped and sent pSB1A2-J01055 and pSb1A2-J01020 (from Box2) for sequencing
*transformed 994library into TG1 cells
*grew liquid culture of pSB3C6-J01122-A3 (iG021) in order to make competent cells
*screened more Key3 colonies
*made pSB3C6-j01122 competent cells.
*cotransformed j01122 x j01129
*resubcloned J23007, 8, 9 and plated
*threw away old pJ23006, pJ23006-I13522
*transformed 994library into 1122

'''KAD_07/11/06'''
*digested pJ23006-J23009-1 and pJ23006-J23009-2 with BamHI and AlwnI for analytic gel mapping.  results from this gel show both versions of pJ23006-J23009 were in fact parent vector.  these results imply that the long fragments from yesterday's gel are the correct clones, contrary to what we had first thought.  we will need to colony pcr at least 4 more colonies  of pJ23006-J23009 in order to find the correct clone.  
*-80 stocks of pJ23006-J23007-3 and pJ23006-J23008-2 have been made, and both clones were also sent for sequencing.
*still waiting for seqencing results from 7/7 and 7/10, so we have yet to confirm accuracy of Lock 1 (J01063), Lock 2 (J01068), pSB3C6-J01122-A3, pSB3C6-J01122-A4, and pJ23006, as well as pJ23006, pSB3C6-J01122-1, pSB1A2-J01129, pJ23006-I13522-1, and pJ23006-I13522-2.
*scraped plate and mini-prepped pJ23006-994library
*sequencing confirmed the accuracy of pSB3C6-J01122-A3 and pSB3C6-J01122-A4, so the A3 clone was transformed into TG1 cells.  
*grew cultures of key 1 and 2 that were plated yesterday: J01055 (on amp) and J01020 (from box2 on amp)

'''BCH_07/11/06'''<br>

'''colony pCR to screen Keys, again'''<br>
Oligos used: ca998 and G00101
pJ23006-J23007: 487 if clone, 1381 if parent vector<br>
pJ23006-J23008: 338 if clone, 1381 if parent vector<br>
pJ23006-J23009: 556 if clone, 1382 if parent vector<br>
''gel map1:'' latter, 7.5, 7.6, 7.7, 7.8, 7.9, 7.10, 7.11, 7.12<br>
''gel map2:'' latter, 8.5, 8.6, 8.7, 8.8, 8.9, 8.10, 8.11, 8.12<br>
''gel map3:'' latter, 9.5, 9.6, 9.7, 9.8, 9.9, 9.10, 9.11, 9.12<br>


to be done on tuesday:
*scrape 994 library plate and miniprep it.
*if J01122 sequence data has been received then make competent cells of these. 
*all sequence data received needs to be filled in the wiki and noted on the miniprep tubes.  
*run an analytical gel on pJ23006-J23007-2, pJ23006-J23007-3, pJ23006-J23008-1, pJ23006-J23008-2, pJ23006-J23009-1, pJ23006-J23009-2  minipreps to determine the correct clones.  Digest with BamHI and AlwnI but double check this in Ape. sequence the ones that come out correctly.
*pick colony from the pSB3C6-J01122 plate and grow a liquid culture to miniprep and -80 stock it assuming sequence data comes back groovy.
*transform 994Library, J23007, J23008, and J23009 into the J01122 competent cells that were made.<br>


'''BCH_07/10/06'''
*Subcloned 994 library into pJ2306-I13522 to make pJ23006-994Library.  They are plated on a big plate in incubator.
*Screened pJ23007, 8, 9 by colony PCR and results need to be put up by kaitlin.
*pJ23006-J23007-2, pJ23006-J23007-3, pJ23006-J23008-1, pJ23006-J23008-2, pJ23006-J23009-1, pJ23006-J23009-2 were minipreped and need to be sequenced. 

'''KAD_07/10/06'''
*miniprepped pJ23006-I13522 verions 1 and 2
*plated pSB3C6-J01122-1 from liquid culture
*sent pJ23006, pSB3C6-J01122-1, pSB1A2-J01129, pJ23006-I13522-1, and pJ23006-I13522-2 for sequencing
*still waiting for sequencing results of Lock 1 (J01093), Lock 1 (J01068), pSB3C6-J01122-A3, pSB3C6-J01122-A4, and pJ23006 
*plated J01055 on AMP from -80 stock
*plated J01129 on AMP from -80 stock
*plated versions of J01020 from -8o Box 1 and Box 2 on AMP, KAN, and CAM plates, because no information could be found relating to the part's antibiotic resistance


'''colony pCR to screen Keys'''<br>
pJ23006-J23007: 487 if clone, 1381 if parent vector<br>
pJ23006-J23008: 338 if clone, 1381 if parent vector<br>
pJ23006-J23009: 556 if clone, 1382 if parent vector<br>
gel map: latter, 7-1, 7-2, 7-3, 7-4, 8-1, 8-2, 8-3, 8-4, 9-1, 9-2, 9-3, 9-4<br>
Of all versions of pJ23006-J23007, only clone 3 appears to have cloned correctly. The bands only differ slightly, however, so we will miniprep a band of each length just to be sure. (7-2, 7-3) <br>
Of pJ23006-J23008, we had similar results, so we will miniprep a versions of each (8-1, 8-2).<br>
Of pJ23006-J23009, all bands were the same short length.  We will miniprep 2 to be sure that these results were accurate, otherwise more colonies will need to be screened in order to find the long variety.  (9-1, 9-2)<br>
7-3 and 8-2 were large variants, all others were short bands.  
[[Image:071006gel.jpg]]

*there are two different clone versions for pJ23006-I13522.  For the construction of pJ23006-994Library, i am using pJ23006-I13522-1. note if the sequence is incorrect for this clone, the construction must be redone.  Also, make sure to throw out -80 stock of pJ23006-I13522 if sequence data comes back incorrect.  Throw away pJ23006-I13522-2 if pJ23006-I13522-1 comes back correct.  -BCH





'''KAD_07/07/06'''
*miniprepped cultures of pSB3C6-J01122 (versions A1, A2, A3, and A4) 
*miniprepped Lock 1 (J01063) and Lock 2 (J01068)
*analytically digested and mapped pSB3C60-J01122-A1, pSB3C6-J01122-A2, pSB3C6-J01122-A3, and pSB3C6-J01122-A4.  results on the gel suggest that all four versions were correct, and most likely have the right genetic sequence. 
*then sent Lock 1 (J01063), Lock 2 (J01068), pSB3C6-J01122-A3, pSB3C6-J01122-A4, and pJ23006 out for sequencing



'''BCH_07/07/06'''
*colony PCR to confirm subclone of pSB3C6-J01122:
**using oligos ca1000 (forward, cagggcagggtcgttaaatagc) ca1001 (reverse, ggcggttttttcgttttcagagc). These are sequencing oligos for pSB3C6 (pAC997).  
**Parent vector will product fragments of size 173 bp, whereas the cloned vector will produce fragments of 1101 bp.

*Colony PCRs ALL came out perfect.  I grew up clones 1 and 2 and minipreped them both. we will sequence 1 and keep 2 until it's confirmed.
*pJ23006 is growing up to make -80 stocks.  this needs to be minipreped and potentially sequenced again because we ran out.
*Subcloned key3b, key3c, and key3d (J23007, 8, 9) into pJ23006.
*Subcloned I13522 into pJ23006.  994 library needs to be subcloned and transformed after this is screened and minipreped.



'''KAD_07/06/06'''<br>
*sequencing data confirms accuracy of pJ23006, but also shows that both pSB3C6-J01122 and pJ23004 do not match expected results.  
*disposed of -80 stocks of pSB3C6-J01122-1, pSB3C6-J01122-4, pJ23004-1, pJ23004-4, pJ23006-1, and pJ23006-2.  
*grew up 4 new cultures from original plate of pSB3C6-J01122 (grown 6/30/06)
*grew cultures of Lock1 and Lock2 from plates grown yesterday
*minipreped pJ23006, however this still needs to be sequenced.
*subcloned and transformed pSB3C6-J01122 yet again and im not sure if we need to sequence this.  needs to be verified some how (colony PCR)
 


'''KAD_07/05/06'''<br>
*still waiting for sequencing confirmation of pSB3C6-J01122, pJ23004, and pJ23006.
*digested 994 key3 library, J23007 (key3B), J23008 (key3C), J23009 (key3D), and pSB1A2-I13522 (to make J23006-I13522 with GFP instead of RFP)
*made overnight cultures of pSB3C6-J01122-1, pSB3C6-J01122-4, pJ23004-1, pJ23004-4, pJ23006-1 and pJ23006-2 in case original minipreps cannot be located. 
*gel purified pSB1A2-I13522 digest, small fragment clean-up of J23007, J23008, J23009, and 994 key3 library to prep for ligation and transformation to test key3 variants.  
*plated Lock 1, Lock 2, pSB3C6-J01122-1, and pSB3C6-J01122-4.


'''DLK_07/03/06'''<br>
-80 Stocks of pSB3C6-J0112 #1, pSB3C6-J0112 #4, pJ23004 #1, pJ23004 4, pJ23006 #1, pJ23006 #2 created and stored in box3.<br>

'''KAD_07/03/06'''<br>
Today we miniprepped overnight cultures of pSB3C6-J01122, pJ23004, and pJ23006.  <br>
We then mapped each plasmid:
* cut pSB3C6-J01122 with EcoRI and PstI to produce fragments 2042 and 974 bp. (Parent vector would yield one 3203 bp fragment)
* cut pJ23004 with SpeI and PstI to produce fragments 2135 and 955 bp.  (Parent vector would yield two fragments: 3066 and 14 bp)
* cut pJ23006 with EcoRI and XbaI to produce fragments 3065 and 214 bp.  (Parent vector would yield two fragments: 3065 and 15 bp)
Results from the analytical digest suggest that clones have correct respective sequences, but nothing can be confimed until sequencing results are received and analyzed. <br>
Lanes 1-4: pSB3C6-J01122, Lanes 5-8: pJ23004, Lane 9: ladder 
[[Image:70306gel.jpg]]<br>
Lanes 1-4: pJ23006, Lane 5: ladder[[Image:Example0703gel2.jpg]]


'''BCH_06/29/06'''
*Sequence data for the -80C stock shows that pSB3C6-J01122 is in fact the parent vector pSB3C6.  We suspect this is sue to a tube switch early on in the project. 
*We extended the keys that we fabricated (parts b, c and d). they are in the riboregulator -20 box and have been purified from their extension reactions.
*We ran an assay on the 1122 and 1129 constructs on the tecan and found bad results.  the key+lock transformation showed absolutely zero increase in flouresence suggesting that the key doesnt unlock shit.  We later found out that the lock, 1122, that we thought o be working perfectly was in fact the parent vector, pSB3C6.  Some tube switched went down and we had to throw out pSB3C6-J01122 mini and -80 stock, all -20 tubes labeled pSB3C6-J01122 or pSB3C6-J01022, or J23000 or j01129X. it was a massacre.
*Kaitlin cut out J01122 from pSB1A2-J01122 and ligated it into pAC997 (pSB3C6) to make pSB3C6-J01122.
*Trashed pSB3C6-J01122 competent cells (-80 stock).
*Kaitlin resubcloned and transformed pSB1A2dXS into TG1s, and they are on a plate and incubating.  
*Subcloned and chris transformed J23006 (Ptet-X-GFP-S).

'''KAD_06/29/06'''<br>
Tecan scan results: 
*Lane 1A: CAM, J01022-B
*Lane 2A: LB + CAM, J01122-A
*Lane 3A: Carb + Kam, J01129 + J01122
*Lane 4A: LB, TG1 Cells

{| border="2" cellpadding="10"
!width="5"|A   
!width="10"|1
!width="10"|2
!width="10"|3
!width="10"|4    
|-
|'''A'''||13357||104||71||71
|}






'''WRB_06/28/06'''<br>
* colony pcr 994/TGI's; colony pcr -> analytical gel to check key3 library to verify presence of key. The gel results showed the expected size in 9 out of 10 cases. The 10th case was not even close. 
* transformed psb1A2_dXS into TG1 cells (chemically competent); the last attempt was unsuccesful. 
* started overnights of psB3C6-J01122, -J01124, -J01022, and psB1A2-J01129

'''DLK_06/26/06'''<br>
*Evening ligation and transformation
**The purified DNA product from Kaitlin was ligated using the standard ligation procedure. Additional 5.5ul backbone (6.5ul total) was subsituted for 5.5ul water

*pSB1A2dXS, pGLN1003, DNA-2/23H,DNA-2/23F were transformed into TGI, with KCM and plated onto Amp plates. A slightly modified transformation procedure was used: 
**	A 280ul cocktail was created	
***	200 ul TGI
***	30 ul KCM
***	50 ul h20
**	1ul of the plasmids (DNA-2/23f DNA-2/23h) was added to tubes (single plasmid per tube, 40ul cocktail) and 10ul of ligation (pSB1A2dXS, pGLN1003) was added to tubes. (single ligation product per tube 100ul cocktail) 4 tubes total, each with single vector.
**	10min on ice, 42C heat shock for 1.5min, 5min on ice, plated


'''BCH_06/26/06'''<br>
*Minipreped and sequenced pSB3C6 J01122-A
**Sequence data from what we thought was J01122 came in and it turned out that it was in fact unlocked RFP (J01022) thereby resulting in constitutive RFP production.  As such, we minipreped the actual J01122-A
*Cotransformed J01122/J01129 into TGI cells in order to determine relative RFP flouresence levels.  Our assay includes untransformed TGI cells, J01122 transformed, J01129 transformed, and J01122/J01129 cotransformed cells.
*KAITLIN GOT HER BADGE! AND COMPLETED HER SAFTEY TRAINING!
*Finished ligating and subcloning 994 Library into Gene Hogs and plated.
*Kaitlin started to make pSBdXS construct described in the construction file.  Kaitlin will set up ligation and Dan will transform it into TGI cells.

'''KAD_06/22/06'''<br>
1.  We grew cultures of pSB3C6-J01022, pSB3C6-J01122-A, pSB3C6-J01093, and pSB3C6-J01093 (all in TG1 cells) to saturdation.<br>
2.  Put samples in tecan<br>
3.  Measured the fluorescence at wavelengths appropriate for RFP:<br>
*pSB3C6-J01122; at saturation: 20, at midlog: 31
*pSB3C6-J01022; at saturation: 2002, at midlog: 267
*pSB3C6-J01093: at saturation: 24, at midlog: 19
*pSB3C6-J01122/pSBIA2-J01129; at saturation: 2484, at midlog: 311
We did this experiment to test our lock3 and key3.  In order to do this, we had to compare results with this test of cells with RFP only (pSB3C6-J01122), the lock only (pSB3C6-J01122), both the lock and the key (pSB3C6-J01122/pSBIA2-J01129), and cells containing no RFP gene (pSB3C6-J01093).<br>

The results from these experiments seem too good to be true. The only way this could be wrong is if we in reality cotransformed J01022 (OnRFP) with the key plasmid instead of the lock3 plasmid, J01122.  To resolve this, we have also decided to sequence our miniprep of pSB3C6-J01122-A to confirm the sequence of the lock reporter.   <br>

A second attempt to make the Lib994 key3 variant library failed to generate colonies.  Therefore, we tried to figure out why it failed.  We ran an analytical gel of the ligation product for the library.  We cut the plasmid with AlwnI because this enzyme would theoretically produce two bands (726 and 1509 bp) from the unligated strand and only one band (2235 bp) from the ligated/circular strand. Unfortunetly, the gel produced unexpected results--multiple bands appeared the gel.  Apparently what happened was that when we had previously cut with XbaI, only one site was cut, and the sticky ends (in some cases) ligated back together, and then fragments were cut with the AlwnI, producing an array of fragments varying in length.<br>

We've designed a new strategy to make the 994 library.  We will do a PCR product off J01129 and do a suffix insertion into pSB1A2-R0040 as described in the modified construction file.

'''KAD_06/19/06'''
* Ran analytical gel on PCR products (A (the 994 key3 library) and B (J01093)) made 06/17/06.  Visible bands present for each sample confirmed success of PCR. 
 [[Image:060618gel.jpg|thumb|EIPCR of 994 key3 library]]
* Digested and recircularized 994 key3 library created by PCR. 
* Transformed 994 key3 library into GeneHogs and TGI with pSB3C6-J01122 by electroboration.  Plated products.
* Cotransformed J01122 and J01129 by heat shock.  Plated product.
B