{{Template:BIO254}}
<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

__TOC__

{{Template:Bio254_Toolbox}}

==Paper 1 Model Systems==
<i>What are the advantages of each?</i><br>
===Chick optic tectum===
===Mouse superior colliculus===
===Mouse retina===
<br>
==Paper 1 Techniques==
<i>What can these be used for?</i><br>

===HEK293 cells===
	HEK 293 cells are an epithelial cell line originally derived from embryonic human kidney. As an experimentally transformed cell line, HEK cells are not a particularly good model for normal cells, cancer cells, or any other kind of cell that is a fundamental object of research. However, they are extremely easy to work with, being straightforward to culture and to transfect, and so can be used in experiments in which the behavior of the cell itself is not of interest. Typically, these experiments involve transfecting in a gene (or combination of genes) of interest, and then analyzing the expressed protein; essentially, the cell is used simply as a test tube with a membrane.

Examples of such experiments include:
*A study of the effects of a drug on sodium channels
*Testing of an inducible RNA interference system
*Testing of an isoform-selective protein kinase C agonist
*Investigation of the interaction between two proteins
*Analysis of a nuclear export signal in a protein

In the Schmitt et al (2006), HEK 293 cells were used in a preliminary test to determine whether Wnt3 can regulate the growth of RGC axons. Schmitt el al created HEK293 cells transfected with the wnt3 gene in order to have the Wnt3 protein expressed in membrane fractions of HEK293 cells (Wnt3 is highly hydrophobic and associates tightly with cell membranes). They found that Wnt3-transfected HEK293 cell membranes inhibited the growth of both dorsal and ventral mouse RGC axons at higher concentrations, and stimulated the growth of dorsal but not ventral RGC axons at lower concentrations (data was not shown).

===SF9 cells===
An insect cell line used for the production of recombinant protein. The Sf9 cell line is derived from pupal ovarian tissue of the Fall armyworm Spodoptera frugiperda. The Sf9 cell line is highly susceptible to infection with Autographa california nuclear polyhedrosis virus (AcNPV baculovirus), and can be used with all baculovirus expression vectors. Sf9 cells are commonly used to isolate and propagate recombinant baculoviral stocks and to produce recombinant proteins. In the Schmitt et al. paper, Sf9 cells were used to overexpress Wnt3 (using the Baculovirus system) to obtain sufficient and consistent amounts of Wnt3.<br>

===Baculovirus system===
Baculovirus is a natural pathogen of the caterpillars producing the SF9 cell line.  In the lab, genes are encoded into a baculovirus vector which is then used to infect SF9 cells.<br>

===Affinity-purified protein===
A protein purified by passing a solution of protein through a column where the protein becomes associated with a matrix of immobilized [[ligand]] somehow attatched to the column.  In most cases the protein must be tagged, or appended to a functional motif called a [[fusion tag]].  Common fusion tag-ligand pairs include:  [[Histidine tag]] (6 or more extra Histidines) and the "ligands" Chelated Nickel or Cobalt, [[Maltose Binding Protein]] and its ligand [[dextrin]], [[Glutathione S-transferase]] and its ligand [[reduced glutathione]], and [[Green Fluorescent Protein]] and Anti-GFP [[antibody]].

===Mock infection===
A control used in infection experiments. Two specimens are used: one that is infected with the virus or vector of interest and the other is treated the same way except with the virus. Sometimes, a non-virulent strain is used in the mock-in the mock-infection experiments.

===Blocking with antibodies or proteins===
===Western Blot, α-tubulin===
Western blots are a technique using immunolabeling to detect proteins from a tissue sample or extract that is run on a gel. Western blots use gel electrophoresis (such as SDS-PAGE) to separate proteins by mass. Several key steps define the methods germaine to a western blot:

First, tissue preparation is required: tissues of interest are frozen rapidly and homogenized using sonication to lyse cells. This homogenate may then be centrifuged in several separate steps to isolate cystolic proteins or membrane proteins. Second, the collected protein samples are run using gel electrophoresis which separates them according to weight. Third, proteins are prepped for detection by being transferred to a nitrocellulose membrane. This procedure is done by placing the membrane face-on-face with the gal and using a current to move charged proteins across the gel-membrane surfaces. Fourth, the membrane proteins are blocked to prevent non-specific binding. Finally, detection of the proteins of interest are done via blotting, such as using antibodies to detect the protein under study. After blotting the membrane is usually washed with developing solution and the amount of antibody signal is transfered onto film for visual quantification. The α-tubulin is a protein that is observed in nearly the same amount in most cells.  Its antibody signal is used to normalize the data to account for the error in loading different amount of protein into each well on the gel.  

===Retina explant assay===

Tissue from the retina at different locations was removed and cultured on cover slips, to allow the chemical interactions and environment to be tightly controlled and the resulting axon growth to be monitored.  Axon growth in the presence of a protein, hormone, or other factor, compared to no or little growth in control, can be used to indicate the sufficiency of that factor for axon growth.  Or, relative lack of growth relative to control can be used to demonstrate inhibition.  Note that different concentrations of the same factor may have opposite effects on an explant, so testing multiple concentrations in addition to control is advised.  The use of explants from different locations in the retina is necessary because a factor may have the opposite effect on nerve tissue depending on its location,  depending on the destination of those axons in vivo, relative to the expression patterns of the factor.

===Electroporation into ventricular zone===
===Dominant-negative===

A dominant-negative is a mutation whose gene product adversely affects the normal, wild-type gene product within the same cell. This usually occurs if the product can still interact with the same elements as the wild-type product, but block some aspect of its function.


Examples:
1. A mutation in a transcription factor that removes the activation domain, but still contains the DNA binding domain. This product can then block the wild-type transcription factor from binding the DNA site leading to reduced levels of gene activation.
2. A protein that is functional as a dimer. A mutation that removes the functional domain, but retains the dimerization domain would cause a dominate negative phenotype, because some fraction of protein dimers would be missing one of the functional domains.


The Shmitt et al Ryk dominant negative:
Wnt3 knockout mice fail in early embryonic patterning because Wnt3 is important for early nervous system development. This makes it impossible to examine the function of the Wnt3 gradient on the medial-lateral axis of the mouse superior colliculus because knockout Wnt3 mice die at birth and axon termination zones form at postnatal day 8. To circumvent this difficulty, Schmitt et al generated a dominant-negative form of Ryk. This truncated Ryk protein only contained Ryk ectodomain (extracellular) and the transmembrane domain, missing the intracellular domain. This dominant negative Ryk allowed Schmitt et al to test in vivo whether blocking Wnt3-Ryk function will shift the termination zone of RGX neurons in the superior colliculus of mice medially.

===In ovo electroporation===

===DAPI staining===
DAPI is a fluorescent stain that binds strongly to DNA. It is used extensively in fluorescence microscopy. Since DAPI will pass through an intact cell membrane, it may be used to stain live and fixed cells.
For fluorescence microscopy, DAPI is excited with ultraviolet light. When bound to double-stranded DNA its absorption maximum is at 358 nm and its emission maximum is at 461 nm, and appears blue/cyan. DAPI will also bind to RNA, though it is not as strongly fluorescent. Its emission shifts to around 400 nm when bound to RNA.
DAPI's blue emission is convenient for microscopists who wish to use multiple fluorescent stains in a single sample. Apart from labelling cell nuclei, the most popular application of DAPI is in detection of mycoplasma or virus DNA in cell cultures. Because DAPI readily enters live cells and binds tightly to DNA, it is toxic and mutagenic.

===AP (alkaline phosphatase)===
Alkaline phosphatase is an enzyme that removes phosphate groups from many molecules such as nucleotides and proteins.  This enzyme is present in practically all organisms, ranging from E. coli to shrimp to humans.  In the lab, AP is a tool frequently used in molecular biology.  One example of its use is in cloning of DNA plasmids.  When a vector and an insert need to be ligated together, measures must be taken to prevent self ligation of the vector (vector religating with itself without the insert).  AP selectively removes the 5' phosphate group on the vector DNA.  Thus, the only phosphate groups left are on the inserts, which then can be ligated onto the vectors using DNA ligase that forms a phosphodiester bond between the 5' end of the insert to the 3' hydroxyl on the vector.  This procedure increases the efficiency of the ligation step.

Another use for AP is in ELISA.  AP is an enzyme that catalyzes colorless pnitrophenylphosphate (pNPP) into p-nitrophenol, a yellow compound.  In a sandwich ELISA, the first antibody is fixed onto the bottom of the well, an sample that contains antigen is added, then a second antibody is added to recognize the antigen.  This second antibody is complexed with AP.  Then the substrate for AP, pNPP, is added and the amount of color change from clear to yellow is measured.  With the aid of AP, scientists can tell the amount of antigen present in the sample by the amount of color change.        
===Protein overexpression===
Selective gene amplification by a cell results in more templates for transcription, which is the basis of natural protein overexpression.  The cell can make more of a certain gene product by increasing the number of copies of the appropriate gene and transcribing them all.  This strategy takes advantage of the transcription mechanisms already in place within the cell and merely feeds them more material to transcribe.
In the lab, the polymerase chain reaction (PCR) technique makes multiple copies of a DNA sequence by copying a short region of DNA many times in a test tube.  It is a cyclic process in which a sequence of steps is repeated over and over:  A DNA molecule with a target sequence to be copied is heated to denature it.  When the mixture cools, short, artificially synthesized primers bond to the single-stranded DNA.  Then dNTPs (four deoxyribonucleotide triphosphates dATP, dGTP, dCTP, and dTTP) and DNA polymerase are added to synthesize two new strands of DNA.
One goal of recombinant DNA technology is to produce many copies (clones) of a particular gene either for the purposes of analysis or to produce its protein product in quantity.  Scientists normally use bacteria as hosts because they are easily grown and manipulated.  Bacteria also contain plasmids, small circular chromosomes, which can be manipulated to carry recombinant DNA into the cell.  But bacterium is not ideal for studying and expressing eukaryotic genes because they lack the splicing machinery to excise introns from the initial RNA transcript of eukaryotic genes.  Many eukaryotic proteins are extensively modified after translation, so scientists use vectors rather than bacteria to carry the new DNA into host cells.
Vectors already have a built-in origin of replication.  The new DNA has to become part of a segment of DNA that contains an origin of replication (i.e. join a replication unit) in order to be replicated in the host cell as it divides.  Plasmids are often used as vectors because they are small, naturally occurring in bacteria, often have only a single recognition site for a given restriction enzyme, and allows for the insertion of DNA at only one location.  When the plasmid is cut with a restriction enzyme, it is transformed into a linear molecule with sticky ends that can pair with the sticky ends of another DNA fragment cut with the same restriction enzyme.  Viruses can also be used as vectors to insert large numbers of base pairs into a genome.  Even if the genes that cause the host cell to die and lyse are gone, the virus can still attach to a host cell and inject its DNA, which in our case is the new DNA to be expressed.
Finally, expression vectors allow foreign genes to be expressed in host cells and can turns cells into protein factories, contributing to protein overexpression.  Expression vectors contain the sequences for promotion, termination, and ribosome binding, which are necessary for protein synthesis in a foreign cell.  The protein to be overexpressed is inserted at the restriction site, the bacteria of choice is transfected with the expression vector, and the protein is synthesized because of its locale in the DNA.  Inducible promoters that respond to a specific signal, thus initiating protein synthesis, can be inserted into the expression vector so that the production of the target protein can be controlled.

===sFRP2===
secreted frizzled related protein 2 is an [[antagonist]] of the [[Wnt]] ligand in [[Wnt-Frizzled]] mediated [[cell signalling]].

===DiI===
A [[lipophillic]] compound used to label cells.  DiI has [[affinity]] for any [[cell membrane]] and is therefore not cell specific, but will only label the cell individually injected with DiI.<br>

</div>