{{Template:BIO254}}
<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

__TOC__

{{Template:Bio254_Toolbox}}

==Lecture 2 Model Systems==
<i>What are the advantages of each?</i><br>
===Frog visual system===
Some of the most groundbreaking work in understanding vision in humans were completed using the visual system of frogs. The retina of frogs has a uniform distribution, while mammalian eyes--such as those of humans--contain areas of higher resolution, called the fovea. The frog, however, has no fovea and it only moves its eyes to compensate for its own motion (either intended or accidental). This makes it easier to compare the patterns of light reaching the frog's retina with the signals leaving along its optic nerve. In early experiments, scientists placed electrodes along the nerve fibers emanating from the eye. Then, they shone a light on to the retina and looked at the pattern of responses along the outgoing nerves. The first studies of the responses in the frog's visual system were conducted in the late 1930's by the physiologist Hartline. Hartline introduced the term "receptive field" to describe the region of the retina to which an individual output responded. The discover of ON, OFF, and ON-OFF cells were made in the frog visual system and allowed us to gain a stronger understanding of how visual systems sharpen the constrast of perceived images.

===Vertebrate spinal cord===
===C. elegans sensory and motor neurons===

''Caenorhabditis elegans'' is a model system due to its short life cycle (about four days) and ease of maintenance.  These nematodes are also transparent, which allow for visual assays (such as with fluorescent proteins).  In the study of neurobiology, ''C. elegans'' is useful because it has a simple nervous system of 302 neurons.  For each of these neurons, the morphology and the connectivity to other neurons is known through electron micrographs.

Most sensory neurons of ''C. elegans'' can be categorized into two groups.  The first type consists of chemosensory neurons with channels that are exposed to the external environment.  The second group consists of mechanosensory neurons that lack these channels to the environment.  

The interneurons of ''C. elegans'' vary in their function and types of connections.  Some receive synaptic input from only a few neurons, while others receive signals from many neurons.

''C. elegans'' neurons have the interesting property in which the presynapses and postsynapses are not localized to only after or before the cell body, respectively.  For example, the interneuron AVE has a postsynaptic region and presynaptic region which both follow after the cell body.  Another unique property is that ''C. elegans'' neurons do not conduct action potentials.

===Drosophila embryo===
Embryonic development has been studied extensively in the Drosophila embryo, particularly the establishment of the dorsal-ventral and anterior-posterior axis, as well as segmentation in Drosophila.
The body axes of the embryo are prefigured in the oocyte by maternal effect genes.  These are prelocalized cytoplasmic determinants as well as localized extracellular signals (signals in the egg shell covering).  Scientists have identified about 50 maternal mRNA types that build up localized determinants during oogenesis that are pre-determined before zygotic genome is turned on.  More specifically, the dorsal-ventral axis is specified by an extracellular signal called spatzle.  Molecules laid down in the ventral extracellular egg covering during oogenesis locally activate a ligand (spatzle).  Spatzle quickly and locally binds its receptor, activating a signal transduction cascade that releases the transcription factor dorsal from a cytoplasmic inhibitor called cactus by degrading cactus.  As a result, dorsal enters nuclei on the ventral side.  dorsal protein is localized in the nucleus in a gradient on the ventral side of the blastoderm embryo.  The gradient of activity of the dorsal transcription factor sets up several different domains of target gene expression.  The gradient of activation of transcription factor (high ventral, low dorsal) and affinity of binding sites, determines how embryo pattern will develop based on signaling pathways.
The action of the bicoid gene during oogenesis is required to set up conditions for development of anterior structures in the embryo.  Bicoid mRNA, like dorsal, is a localized cytoplasmic determinant that is localized to the anterior pole of the oocyte during oogenesis (by motor proteins moving on microtubule tracks).  After fertilization the mRNA is translated and bicoid protein diffuses out to form a gradient.  It functions as a DNA binding protein that turns on transcription of the hunchback gene in the embryo.  Hunchback, in turn, acts to pattern the embryo as a gap gene.  The levels of bicoid protein sets the position of hunchback expression along the anterior-posterior axis since a threshold level of bicoid is required to turn on hunchback transcription.  Hunchback transcription is blocked in the posterior region by the nanos protein.  Nanos mRNA is localized posteriorly and similarly sends out a protein gradient that opposes the bicoid protein gradient as well as the maternal Hb mRNA that is distributed uniformly throughout the cell.
The gap genes, which include hunchback, giant, kruppel, and knirps, are transcription factors for segmentation.  There is a tapering of the proteins in each direction from where it is expressed to create gap gene domain expression overlap, creating combinations of more than one of their protein products.  The peaks overall to foster complexity since across head to tail axis there are different amounts and different locations of transcription factors, and genes sensitive to these differ in triggering influences that lead to segmented expression of genes.
Pair rule gene transcription is under gap protein control.  There are different DNA control elements for different stripes.  Combinations of transcription factors act on particular silencers/enhancers to control segmentation.  The regulation of lateral segments require different combinations of transcription factors, such as Wnt and Hedgehog signals that organize the pattern of bristles within each Drosophila segment.  Hedgehog and Wnt are both short range signals, but Hedgehod is a secreted protein and Wnt is a signaling pathway.  Adjacent cells talking to each other for feedback to reinforce each other’s signals in positive feedback.  If the Hedgehog signal fades then there’s no communication within the poles of the segments and Wnt causes bristles to form on all cells.
Finally, homeotic genes (Hox genes) are single transcription factors that can affect where development occurs by conferring different fates upon repeating body segments, inducing limb growth, and organizing organ placement.  This is based upon their select expression along the dorsal-ventral and anterior-posterior axis in accordance to combinatorial coding of the genes described above.

===Cell culture===
Small amounts of undifferentiated or single cells (normally from excised animal tissue) are placed in an artificial environment.  The nutrient medium depends on the experiment being conducted, but usually the medium favors cellular growth and differentiation.  By using cell cultures it is possible to pin down a cause and effect relationship between the carefully controlled culture and the development of the maturing cells.  Cell cultures can be manipulated by adding chemicals, nutrients, etc. to the cellular environment to test a hypothesis or achieve desired characteristic results.  Favorable qualities of cells can be precisely controlled, so that each cell is identical for the particular quality being sought, allowing for repetition within experimental methods.  In the case of neuroscience, axon growth, protein secretion, receptor up/down-regulation, and neurotransmitter release can all be studied and manipulated within culture to test the effects of a wide variety of cellular environments.

===Grasshopper===
Much of the early work on the study of axon growth was done on grasshoppers. Many of their individual motor neurons were identified and their pathways characterized.

===Xenopus axons in culture===
<br>
Xenopus is one of the most commonly used model organism in studying developmental biology. Their embryos are large, easy to obtain and are developed externally, allowing easy access to experimental manipulation. Such large cells are relatively easy to culture and manipulate experimentally. Thus, growing their neurons in culture is a optimal method for studying axon guidance and growth cone cues.

==Lecture 2 Techniques==
<i>What can these be used for?</i><br>
===Biochemistry===
Biochemistry is the study of chemical processes in a bioloical environment. The field deals mostly with structure and function of large biomolecules such as large proteins (enzymes, other cellular machinery), carbohydrates, nucleic acids, etc.  In terms of experimental assays, biochemical assays can be used to evaluate enzymatic functions, discover inhibitors for a certain enzyme, or to extract novel proteins within a certain tissue. For many biochemical assays, commons reagents are used and are easier to prepare on a daily basis, which gives this type of assay flexiblity.  A typical biochemical assay for detecting kinase activity of a certain protein would consist of a reaction with the enzyme, the substrate, ATP, and magnesium. Also biochemical assays usually do not require the cells to be alive.  

===Genetics: mutation and over expression===
A genetic mutation is a permanent change in the DNA sequence that makes up a gene. Mutations can affect a single DNA building block or even a large segment of an entire chromosome. Mutations may be induced in an egg or sperm cell or after fertilization; these changes are termed new (de novo) mutations, and may be experimentally beneficial for studying genetic diseases or for creating transgenic animal models that mimic aspects of human disease. 

The protein encoded by a particular gene may be expressed in an increased quantity ("over-expression") such that the phenotype of the organism can be significantly altered. Two commonly used techniques to create gene over-expression are to either increase the number of the copies of the gene, or, to increase the binding strength of the promotor.

===Co-culture on a 3D collagen gel matrix===
Cultured cells usually reflect growth in a 2D environment, whereas cells in the body exist in 3D matrices. Thus, it is important to have models where cell growth may traverse in a three-dimensional fashion. A 3D collage gel matrix allows this type of investigation. Cells in organisms exist in a 3D extracellular matrix environment rich in type I collagen, and a co-culture on a collage gel matrix provides a model for scientists to recreate this environment in a laboratory setting.  This 3D system provides a simple and rapid method to analyze cell angiogenesis, migration, apoptosis, proliferation and tissue formation in a 3D-collagen matrix. Cells suspended in this 3D system are easily visualized by phase contrast or fluorescence light microscopy. Cells can be directly fixed and stained within the matrix and treated with antibodies for visualization of specific intra- and extracellular proteins.

===Antibody Staining===
Antibody staining, also known as immunostaining, is a general term in biochemistry that applies to any use of an antibody-based method to detect a specific protein in a sample. The term immunostaining was originally used to refer to the immunohistochemical staining of tissue sections, as first described by Albert Coons in 1941.   Now however, immunostaining encompasses a broad range of techniques used in histology, cell biology, and molecular biology that utilise antibody-based staining methods.

===Cloning genes and expressing them in cell culture===
Cloning involves producing a large number of daughter molecules identical to the original gene product. Target genes can be inserted into plasmids/vectors which are subsequently used to transform bacterial cells for amplification; the bacterium Escherichia coli is commonly used. Plasmids are circular, double-stranded, extrachromosomal DNA that replicate upon insertion into a host cell and that confer their properties (e.g. antibiotic resistance) to the host. To insert a gene into a plasmid, first, specific restriction endonucleases are used to isolate a gene of interest from its original genomic DNA; the same restriction endonuclease is used to specifically cut the plasmid (e.g. at the BamHI, EcoRI, or SalI cloning sites) that will receive the gene insert, allowing for complementary base pairing between the “sticky” ends (single strand overhangs resulting from restriction endonuclease action). For each restriction enzyme, a corresponding modification enzyme prevents the host’s own DNA from being cut. DNA ligase is used to seal the gene insert within the plasmid, and this genetically engineered plasmid-gene construct is then used to transform competent E.coli host cells, through electroporation. The plasmid (with gene insert) then utilizes the host cell machinery to replicate itself until it reaches the cell’s maximum copy number. Cell culture of the transformed E. coli allows for exponential host cell growth and further amplifies plasmid replicate number. Plasmids are passed on to daughter cells in roughly equal proportions upon cell division. Gene product expression is driven by a promoter on the plasmid, located upstream of the cloning site. This technique is extremely valuable for research. Not only does it enable the study and characterization of individual genes of interest, but also it can be used in conjunction with various reporter assays to assess the effects of different mutations on the functionality of a gene of interest. Additionally, applications outside of the lab exist. For instance, this method enables industrial production of large quantities of insulin to meet the demands of diabetic patients.
===Forward genetic screen===
Genetic screens test and identify organisms with a specific phenotype. A forward genetic screen searches for new genes or mutant alleles, which rarely occur in nature. Hence, scientists perform a forward genetic screen by exposing the individual to a mutagen in order to induce mutations in their chromosome(s). Mutagens such as random DNA insertions by transformation or active transposons can also be used to generate new mutants.

Forward genetic approaches make no assumptions about the genes involved to illicit a given behavior. Random point mutations are introduced into the genome with chemical agents and the mutant organism is identified based on its altered phenotype. This is the opposite of reverse genetic approaches, which move from gene to phenotype rather than phenotype to gene. The use of  ENU (see Lecture 15 techniques) to create circadian mutant mice strains is a good example of how forward genetics uses altered phenotype to explore gene function.
===The Poo Assay===
The Poo Assay is used to assess growth cone turning responses to gradients of extracellular guidance factors.  It is named after its originator, Mu-Ming Poo, who used it to demonstrate the attractive turning of a growth cone towards a gradient of netrin-1 and the repulsive turning of a growth cone away from a gradient of semaphorin 3A. Isolated growth cones are cultured in a cell-free environment in vitro and then are exposed to gradients of a potential signaling molecule. Within an hour turning of the growth cone is evident and the angle of turning can be used to gauge the strength of the molecule’s signal. Turning should not be observed when the culture medium is supplemented with an antibody against the signaling molecule of interest.

===Explant overlay assay===
The explant overlay assay, known more commonly as the slice overlay assay, is an in vitro assay in which neuronal explants are cultured over cortical slices. The principal use of the explant overlay assay is to characterize extracellular signaling molecules that regulate neuronal differentiation and patterning. The two methods used for this purpose before the innovation of the explant overlay assay had significant shortcomings. An in vitro assay using neuronal explants cultured on an artificial substrate was problematic because the substrate was no substitute for the actual in vivo environment in which neuronal outgrowth takes place. The limitation of the second method, an in vivo assay that involved transplanting and monitoring labeled neurons, was that the chemical environment could not be manipulated like in an in vitro assay.  The explant overlay assay is able to resolve both problems, making it the most effective method for studying neuronal guidance molecules and mechanisms. Franck Polleux developed the explant overlay assay in 1998 to show that the initial growth of cortical axons toward the white matter is regulated by a semaphorin signal that is expressed in the marginal zone.

===Incubating slices in media with chemical cues===
===Mammalian pyramidal neurons===
Pyramidal cells are the primary projection neurons in the cerebral cortex and the hippocampus of the central nervous system (CNS, brain). Pyramidal cells have a pyramid-shaped cell a long and branching dendritic tree. An axon that carries nerve impulses emerges from one end of the cell. The axon may have local collateral branches but also project outside their region. These cells are multipolar neurons with a single apical dendrite and compose up to 80% of the neurons in the mammalian cortex. Pyramidal cells are excitatory neurons and release glutamate as their neurotransmitter.

</div>