{{Template:BIO254}}
<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

__TOC__

{{Template:Bio254_Toolbox}}

==Lecture 5==  

===Enhancer Promoter Screen===
A screen for over expression mutant phenotypes.  The genotype is created through random insertion of a strong promoter into the genome.

===Lethal Enhancer Screen===
A screen for a second mutation that enhances a phenotype of another mutation which by itself is not lethal.  In other words, to screen for a second mutation that by itself is viable but when combined with the first mutation produces a lethal phenotype.  This type of genetic screen is usually used to delineate gene interactions because the second mutation must be involved in some pathway with the first in order to it be a lethal enhancer.  Such a genetic screen usually use a chemical mutagen (such as EMS) to induce the second mutation and then crossing the mutated flies with those with the first mutation.  Those with desired lethal phenotypes were then selected.  

In lecture 4, ubiquitination processes were implicated in the development of synapses at the Drosophila neuromuscular junction.  DiAntonio et al (Nature, 2001) showed that overexpression of fat facets, a deubiquinating protease, led to disrupted synaptic function and an overgrowth of synapses.  A lethal enhancer screen was employed to find other genes that enhance the fat facets phenotype.  The screen looked for viable mutations that produced lethal phenotypes when combined with fat facets overexpression.  The Highwire gene was one such mutation isolated in the screen.  Highwire loss of function mutations resulted in the same synaptic overgrowth as fat facets overexpression and biochemical analysis of highwire revealed it to be an ubiquitin ligase.  Take together, these data show that fat facets and highwire antagonize each other in the ubiquitination process and strongly support the idea that synaptic development is controlled by positive (highwire) and negative (fat facets) regulators  of ubiquitination.
==Lecture 5 Model Systems==
===Electric eel===
The electric eel (Electrophorus electricus) is a species of fish native to South America that is able to generate electric shocks using the electric organ of its body.  The organ is composed of electroplaques, a stack of plates that can generate charges.  The organ is also extremely rich in voltage-gated sodium channels.  In 1986, a team of Japanese researchers led by Shosaku Numa cloned the cDNA for the sodium channel using the electric organ of the eel.  From that scientists were able to deduce the biochemical structure of the sodium channel.  

==Lecture 5 Techniques==
===Aldicarb===
Aldicarb (chemical name: 2-methyl-2-(methylthio)propionaldehyde O-methylcarbamoyloxime) is an carbamate-class insecticide applied directly to the soil and is used to control mites, nematodes, and aphids.  In the laboratory, aldicarb acts as an inhibitor of acetylcholinesterase (AChE), the enzyme present in the basal lamina of the post-synaptic cell of the neuromuscular junction (NMJ) which breaks down the neurotransmitter acetylcholine (ACh).  ACh is the excitatory neurotransmitter for muscular contraction; therefore, inhibition of AChE using aldicarb results in prolonged activation of ACh receptors in the post-synaptic cell, causing paralysis and eventually death.

Aldicarb is a useful tool in genetic screens that search for mutants resistant or hypersensitive to the aldicarb-induced paralysis.  These mutants will likely have mutated genes involved in the ACh signaling pathway at the NMJ.  Mutants resistant to aldicarb will likely have an impairment of normal ACh signaling, while mutants hypersensitive to aldicarb will likely have an exaggeration of normal ACh signaling.

===PCR===
PCR, or the polymerase chain reaction, is an experimental technique devised by Kary Mullis in 1984 and is used to amplify a targeted DNA segment for further experimental analysis.  PCR does not require knowledge of the target DNA sequence; however, knowledge of the DNA sequences flanking the target is necessary.

A PCR cycle proceeds as follows: 1) The parent DNA duplex is separated by heating the solution to 95°C for 15 seconds, exposing the bases on each strand.  2) Primers for the flanking DNA sequences anneal to the 3'-end of each parent DNA strand when the solution is cooled to 54°C.  3) A heat-stable DNA polymerase called Taq DNA polymerase (derived from the thermophilic bacterium Thermus aquaticus) synthesizes complementary DNA strands starting at the primers and using available nucleotides in solution when the solution is heated to 72°C.  Repeated cycles of PCR allow for an exponential amplification of the target DNA sequence.

===E. coli===
===Ion replacement===
===Low transmitter release conditions===
===Caged calcium===

</div>