{{Template:BIO254}}
<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

==Introduction==
In activity-dependent [http://en.wikipedia.org/wiki/Transcription_%28genetics%29 transcription], a postsynaptic neuron changes the level of transcription of specific genes in response to synaptic activity. This mechanism underlies the ability of the nervous system to convert the effects of various stimuli into long-term changes in brain function.  While both long-term and short-term changes are produced by synaptic activity, long-term changes, such as synaptic plasticity and neuronal development, are often caused by changes in gene expression. 

Activity-dependent transcription is critical for the proper function and development of the neural system including in long term potentiation (LTP) and memory formation, synaptic plasticity, neuronal survival, neurogenesis, dendritic arborization, and wiring.

==General Mechanism for Activity-Dependent Transcription==

Following are the steps leading to changes in gene expression:

===A) Neuronal Activity===
The presynaptic neuron excites or inhibits the postsynaptic neuron using neurotransmitters, such as glutamate or GABA.

===B) Postsynaptic Response===
The neurotransmitter activates metabotropic and/or ionotropic receptors activating G-protein/second messenger systems or producing ionic currents, respectively.

===C) Signaling Pathways===
Signaling proteins are activated (phosporylated/dephosphorylated) by either the G-protein pathways or the pathways affected by ionic concentrations (usually Ca2+ dependent). These pathways are often referred to as cascades because of the multiple proteins phosphorylated/dephosphorylated sequentially in a pathway.

===D) Nuclear Localization & Transcription Factor Activation/Inactivation===
An activated protein enters the nucleus and activates/inactivates an activating or repressing transcription factor. In some cases, the protein that enters the nucleus is itself an activated transcription factor which goes directly and binds to its specific DNA sequence.

===E) Gene expression Increased/Decreased===
Depending on whether the transcription factor is activated or inactivated and whether the transcription factor itself activates or represses gene expression, the transcription of the target gene(s) will be changed.

====Transcription Factors====
Along with protein-coding regions, the gene's DNA includes sequences that are involved in the control of the gene's expression. These sequences are called regulatory elements; transcription factors are usually sequence specific and bind to regulatory elements containing its binding sequence (~7-12 base pairs long). For example, CREB (CRE binding protein) binds to the DNA sequence CRE (cAMP response element). In eukaryotes, the transcription of DNA to mRNA is carried out by RNA Polymerase II, which binds to the gene's promoter region. Transcription factors affect the ability of the polymerase to bind to the promoter region, thereby influencing the level of expression of the gene. Often, adaptor proteins (which interact with both the transcription factor and the polymerase) are required for the transcription factor (bound to its binding sequence within several hundred base pairs from the promoter region) to influence gene expression.

==Activity-dependent Chromatin Remodeling==
Recently, chromatin remodeling has been studied as a mechanism of activity-dependent transcription. Generally, studies have shown that neuronal activity can lead to the modification of nuclear chromatin. The new chromatin state presents a new set of genes to the cellular transcription machinery leading to changes in mRNA expression (Crosio et al. 2003). This is a powerful new mechanism of activity-dependent gene regulation as the chromatin state can perhaps regulate many more genes than individual transcription factors can. 

===Chromatin===
Chromatin is the DNA-protein complex found in all eukaryotic nuclei. Histone is the protein. Nucleosomes are the repeating subunits of DNA-protein complexes found in chromatin. Covalent, post-translational, modifications to histone proteins affect the chemical and steric environment around DNA and are classified as "epigenetic mechanisms" of gene regulation. Each histone protein consists of a globular domain that interacts with other histones and DNA, and flexible tails, which are highly basic and are the substrates for various enzymes that introduce modifications including acetylation, methylation, phosphorylation, and ubiquitylation (Cosgrove and Wolberger 2005).

===What Studies Have Shown===
Neuronal activity-dependent chromatin remodeling is still largely uncharacterized. However, initial studies have been promising. For instance, in Histone H3, Ser10 phosphorylation has been shown to be induced by neuronal activity (Bode et al. 2005). Another study looked at the effect of dopaminergic, glutamergic, and muscarinic receptor agonists on hippocampal neurons. It was found that these pharmacological manipulations induced rapid phosphorylation at H3 Ser 10. In addition, H3 Ser10 phosphorylation is coupled to acetylation at Lys14, a modification that is linked to the opening of chromatin structure (Crosio et al. 2003). 



==Examples of Pathways that Affect Transcription==

===Activation of CREB by Ca2+ Influx===

CREB, the cyclic AMP response element binding protein, is a transcription factor involved in various signaling pathways influencing synaptic plasticity, neuronal survival, and neuronal development. CREB remains constitutively bound to its DNA binding sequence, CRE; CREB must be phosphorylated on Ser-133 and bound to its activator CBP to become active and increase or decrease the expression of specific genes. The pathway that leads to CREB activation initially requires an increase in Ca2+ concentration, the activation of G-protein coupled receptors, or the binding of neurotrophins to cell receptors. For Ca2+ to enter the postsynaptic neuron, an excited presynaptic neuron has to release glutamate, which binds glutamate receptors (AMPA, NMDA, Kainate) and depolarize the postsynaptic neuron. In hippocampal neurons (and possibly in other neurons), Ca2+ ion flow through the NMDA receptor and the L-type voltage-gated calcium channel are required for the signaling cascade to begin. In other words, Ca2+ increases by themselves are not sufficient to activate CREB (Deisseroth 2003). This could be because L-type Ca2+ channel seems to carry a CaM binding site and the NMDA receptor has a calcium microdomain near it. Four Ca2+ ions then bind to calmodulin forming a Ca2+/calmodulin complex.

<BR>
<center>[[Image:ca2toCREB4.jpg|frame|center|Figure 1. Activation of CREB by Ca2+ Influx (Deisseroth 2003)]]</center>
<BR>
The Ca2+/CaM complex activates CaMKII outside the nucleus and also translocates into the nucleus to activate CaM dependent protein kinase kinase (CaMKK) (as shown in Figure 1). This protein then activates CaMKIV which phosphorylates CREB at Ser-133. An adaptor protein, CBP then binds to CREB and the RNA polymerase. The pathway through which CaMKII affects CREB activation is not yet clear (Curtis 1999).

The formation of the Ca2+/CaM complex also activates the MAPK (mitogen-activated protein kinase) pathway by first activating (through an as yet undetermined process) the small GTP binding protein Ras, which then activates Raf, which activates MEK, which activates ERK (extra-cellular regulated kinase), which in turn activates p90 Rsk. Rsk then phosphorylates CREB.

Certain isoforms of adenylate cyclate (AC) can be stimulated by Ca2+/CaM. The AC then converts ATP into cAMP. cAMP in turn activates PKA by binding to its regulatory units and freeing the catalytic units to translocate to the nucleus and phosphorylate CREB.

CREB affects a variety of genes, including many immediate early genes (first set of genes to be turned on after a stimulus), such as c-Fos. CREB plays an interesting role in neuronal survival – while activation of CREB leads to neuronal survival, overactivation of CREB can lead to apoptosis.

Activation of CREB is the first step in long-term facilitation (LTP), which is the cellular homolog of long-term memory.  CREB is present both as a repressor (CREB-2) and as an activator (CREB-1) of transcription – this suggests that the mechanism for putting information into long-term memory is highly  regulated. When the action of CREB-2 is blocked artificially, LTP can occur after a single pulse of serotonin, which normally produces only short-term facilitation.

The combined activation of CREB-1 and deactivation of CREB-2 leads to induction of expression of two genes: 1) the enzyme ubiquitin  carboxyterminal hydrolase, which activates proteasomes to make PKA persistently active and thus removing inhibitory constraints on memory.  2) Transcription factor C/EBP, which is part of a gene cascade necessary of the growth of new synaptic connections  - a process that provides  for the maintance of long-term memory.

Eric Kandel pioneered the used of the sea mollusk Aplysia to study the process of long-term potentiation (LTP).  This process can be stimulated in Aplysia by activation of the CREB pathway through sensitizing  the gill-withdrawal reflex.  Animals that have been “sensitized” have twice as many presynaptic terminals in the sensory neurons of the gill-withdrawal pathway  as untrained animals.  Furthermore, in sensitized Aplysia there is also an increase in the dendrites of the motor neurons .



====More Images of Pathways Leading to CREB Activation====
[http://nba.uth.tmc.edu/darpa/images/CREB.jpg UT Houston] Simple picture of neurotransmitter activation of G-protein, increase of cAMP concentration, activation of PKA, and activation of CREB<br>
[http://edoc.hu-berlin.de/dissertationen/miksa-michael-2004-03-26/HTML/Miksa_html_m312ec272.png Humboldt University, Berlin] Simple picture of pathway from Ca2+ increase to CREB activation<br>
[http://www.panomics.com/images/CREBStablecellline/crebsignalingpathway.jpg Panomics] Simple picture of various pathways influencing CREB<br>
[http://www.eurekah.com/dbimages/Riedel5.1Frankland.jpg Eurekah] Small picture that shows effect of neurotransmitter, Ca2+ increase, and growth factors on CREB<br>
[http://www.sigmaaldrich.com/img/assets/6460/Trans_fact_reg.gif Sigma Aldrich] CREB activation by growth factors<br>
[http://www.biocarta.com/pathfiles/m_crebPathway.gif Biocarta] Complicated picture of various pathways influencing CREB

====Abbreviations====
CaM: calmodulin<br>
Ca2+/CaM: calcium calmodulin (calcium bound to calmodulin)<br>
CaMK: calmodulin-dependent protein kinase<br>
CREB: Ca2+/cAMP responsive element binding protein<br>
CBP: CREB binding protein<br>
CRE: cAMP response element<br>
CaMKK: CaM dependent protein kinase kinase <br>
CaMKII:CaM dependent protein kinase II<br>
CaMKIV:CaM dependent protein kinase IV<br>
NMDAR: N-methyl-D-aspartate receptor
PKA: protein kinase A<br>
ERK: extracellular regulated kinase<br>
MEK: MAPK kinase<br>
ROS: reactive oxygen species<br>
PI3-K: phopshatidyl inositol3 kinase<br>
GSK3: glycogen synthase kinase 3B<br>

===Other Examples===

CREB is the main transcription factor involved in actvity-dependent transcription. Along with being activated by an increase in Ca2+ concentration, CREB can be activated by G-proteins and neurotrophins. Transcription factors like NF-kB are also activated or inactivated by neuronal activity. Also, there are many activity-dependent regulators of transcription whose mechanism are unclear, such as the intracellular domain of neuregulin-1 which translocates to the nucleus and represses the expression of genes that regulate apoptosis.


==Research History of Activity-Dependent Transcription==

In 1984, Stanton and Sarvey showed that long-term potentiation in the hippocampus can be blocked by inhibitors of protein synthesis.

In 1986, Morgan and Curran studied the relationship between the Ca2+ influx and gene expression, specifically of c-fos. They suggested that Ca2+ would enter through the L-type Ca2+ channel and bind to calmodulin. Then, a calmodulin sensitive kinase would phosphorylate a transcription factor, which would then move into the nucleus.

In 1988, Sheng et. al. found a regulatory sequence upstream of c-Fos, which would turn out to be the CRE where CREB binds.

In 1989, Gonzalez and Montminy discovered that CREB needed to be phosphorylated at serine 133 to become active.

In 2000, Eric Kandel received a Nobel Prize in Physiology or Medicine partly for his work showing the importance of activity-dependent transcription in memory formation.


==Notes==

This article has focused on activity-dependent transcription in neurons. Activity-dependent transcription occurs in various excitable cells, such as muscle cells (ie, Wamhoff 2006).

Also, it is important to note that phosphorylation of a protein activates some proteins while it inactivates others.


== References ==
Cosgrove M, Wolberger C. 2005. How does the histone code work?. Biochem. Cell Biol 83:468-476. 

Crosio C, Heitz E, Allis C, Borrelli E, Sassone-Corse P. 2003. Chromatin remodeling and neuronal response: multiple signaling pathways induce specific histone H3 modifications and early gene expression in hippocampal neurons. Journal of Cell Science 116:4905- 4914. 

Curtis J, Finkbeiner S. Sending signals from the synapse to the nucleus: possible roles for CaMK, Ras/ERK, and SAPK pathways in the regulation of synaptic plasticity and neuronal growth. J Neurosci Res. 1999 Oct 1;58(1):88-95.

Deisseroth K, Heist EK, Tsien RW. Translocation of calmodulin to the nucleus supports CREB phosphorylation in hippocampal neurons. Nature. 1998 Mar 12;392(6672):198-202.

Deisseroth K, Mermelstein PG, Xia H, Tsien RW. Signaling from synapse to nucleus: the logic behind the mechanisms.Curr Opin Neurobiol. 2003 Jun;13(3):354-65.

Gonzalez GA, Montminy MR. AMP stimulates somatostatin gene transcription by phosphorylation of CREB at serine 133. Cell. 1989 Nov 17;59(4):675-80.

Hardingham GE, Arnold FJ, Bading H. A calcium microdomain near NMDA receptors: on switch for ERK-dependent synapse-to-nucleus communication. Nat Neurosci. 2001 Jun;4(6):565-6.

Hardingham GE, Cruzalegui FH, Chawla S, Bading H. Mechanisms controlling gene expression by nuclear calcium signals. Cell Calcium. 1998 Feb-Mar;23(2-3):131-4.

Johannessen M, Delghandi MP, Moens U. What turns CREB on? Cell Signal. 2004 Nov;16(11):1211-27.

Kandel ER. The molecular biology of memory storage: a dialog between genes and synapses.   Science. 2001 Nov 2;294(5544):1030-8.

Morgan JI, Curran T. Role of ion flux in the control of c-fos expression. Nature 1986, 322:552-555.

Sheng M, Dougan ST, McFadden G, Greenberg ME. Calcium and growth factor pathways of c-fos transcriptional activation require distinct upstream regulatory sequences. Mol Cell Biol. 1988 Jul;8(7):2787-96.

Silva AJ, Kogan JH, Frankland PW, Kida S. CREB and memory. Annu Rev Neurosci. 1998;21:127-48.

Stanton PK, Sarvey JM. Blockade of long-term potentiation in rat hippocampal CA1 region by inhibitors of protein synthesis. J Neurosci. 1984 Dec;4(12):3080-8.

Squire LR, et al. Fundamental Neuroscience. Academic Press, Boston, 2003.

Wamhoff BR, Bowles DK, Owens GK. Excitation-transcription coupling in arterial smooth muscle. Circ Res. 2006 Apr 14;98(7):868-78.

==<h3>Recent updates to the site:</h3>==
{{Special:Recentchanges/BIO254&limit=50}}
</div>