{{Template:BIO254}}
<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">
__FORCETOC__<br>
==Lecture 1 Model Systems==
<i>What are the advantages of each?</i><br>
===Mammalian visual system===
The Mammalian visual system, of which the human visual system is an important member, is among the most complex in the animal world. The mammalian visual system is composed of several components, listed in the order which they act: the eye itself (consisting most importantly of the lens and the retina), the optic nerve, the optic chiasm, the left and right optic tracts, the lateral geninculate nucleus, the optic radiation and lastly the visual cortex itself. The light focused by the lens falls on photoreceptors in the retina, which, in humans, are bound to two different proteins called <i>opsins</i>: rodopsins (in rods) and cone opsins (in cones). The human, and some primates', visual system is different from most other mammals in that we have one extra type of cone opsin that allows us to distinguish colour. Most mammals are colourblind; colour evolved fairly recently in a sub-branch of the primate evolutionary tree. Despite this, the mammalian visual system (i.e. that of mammals other than the higher primates) is a powerful model for study as most of its components are analogous to those in humans. For further information on the specific components of the visual system, see lecture 8 below.
===Mammalian olfactory system===
Mammalian sense of smell depends on chemoreceptors.  The olfactory sensors are sensory neurons embedded in a layer of epithelial tissue at the top of the nasal cavity.  These neurons directly project their axons to the olfactory bulb of the brain (i.e. they are not relayed through the thalamus like other senses' axons).  Their dendrites end in olfactory hairs on the surface of the nasal epithelium.  Each functional olfactory receptor protein that is expressed is found in a limited number of sensory cells in the olfactory epithelium.  All of the cells that express the same receptor protein project to the same regions in the olfactory bulb.  A given odorant molecule may bind to one or to more than one receptor protein.  Therefore each odorant molecule can excite a unique combination of cells in the olfactory bulb, so an olfactory system with even hundred of different receptor proteins can discriminate a large number of cells.  Interestingly, the more odorant molecule that bind to receptors, the more action potentials are generated, and the greater the intensity of the perceived smell.

===Spinal cord motor neurons===
The spinal cord may be divided up into sections based on the nerve roots that extend from it. At each segment, rootlets come out of both the dorsal and ventral halves of the spinal cord. In fact, nerves go in through the dorsal side and then exit through the ventral face. The dorsal horns respond to sensory perception by receiving axons from the periphery via the dorsal root. The ventral horns contain motor neurons with axons that leave the cord via the ventral roots and travel to stimulate the muscles. The dorsal root ganglion contains a collection of cell bodies that have all the receptor neurons sending processes to the peripheral muscles. These are the motor neurons located in the spinal cord. Lower motor neurons are either alpha or gamma cells. Alpha cells comprise the principle motor neurons of the spinal cord and are a part of the main portion of the common reflex pathway. These neurons conduct rapid motor impulses; each alpha cell innervates about 200 muscle fibers. Gamma neurons, also part of the common pathway, are only half as numerous as alpha cells. They conduct slower motor impulses and their major function is to stretch muscle spindles.

===Human brain===
As the most anterior part of the central nervous system, the human brain acts as the primary control center for the peripheral nervous system. Autonomic functions of the brain include controlling heartbeat, digestion, respiration, sensation, and movement. Higher order functions of the human brain include conscious activities like thinking and reasoning. The human brain is unique from other mammalian and vertebrate brains because it contains millions of billions of synaptic connectins, resulting in a complex and dense neural circuit. Neuroscience is the study of the brain and its functions; psychology is the study of the mind and behavior; and neurophysiology is the study of normal healthy brain activity. Advantages of using the human brain system for study is that results obtained will be the closest model system for determining the causes and pathways of mental illness in humans. Neuroimaging allow scientists to study the brain not only in detail, but in real time (functional neuroimaging). The human brain is most often compared with a computer: individual neurons are analogous to microchips, and specific areas of the brain are said to resemble graphic cards. However, no clear sequential (one-to-one command) set of instructions are consistently observed in the brain, and the study of the human brain directly cannot be replaced with computer models with great accuracy.

==Lecture 1 Techniques== 
<i>What can these be used for?</i><br>
===Golgi staining===
Also called the Black Reaction, Golgi staining stains a subset of cells within the brain, because staining all neurons and cellular processes would make anatomical analyses difficult and cumbersome. While the exact mechanisms behind the Golgi stain is not well understood, this technique labels axons, dendrites, and cell somas in black and brown along their entire length. Hence, neural ciruits can be visualized, tracked, and mapped. Golgi stains are made by injection of potassiumdichromate and silver nitrate; the brown-black color of neurons stems from the microcrystallization of silver chromate.

===Tissue culture===
Tissue cultures allow researchers to grow tissues and/or cells outside of the organism under investigation. Primary cell cultures usually have a finite life span in culture compared to cell lines which are abnormal or transformed cell lines. The availability of tissue cultures enable the study of cells in a controlled environment without the external influences found in the organisms' physiological environment. Advantages of such a technique include the ability to study specific cellular mechanisms alone, and the opportunity to manipulate cell lines to better understand developmental abnormalities.

===Electron microscopy===
Through the use of electrons to create an image of the object, electron microscopy provides higher magnification and superior resolving power than a light microscope by almost a magnitude of two million. Various electron microscopy techniques exist for exploring morphology and mechanisms: scanning electron microscopes give a 3D image of the sample; transmission electrion microscopes produce 2D images at impressive magnifications (up to 500 million times); and scanning tunneling microscopes determine the height of the sample surface.

===Biolistic transfection (gene gun)===
This technique injects cells with a heavy metal coated with plasmid DNA, and is capable of transforming almost all types of cells including their genetic information and cellular organelles. Gene guns are also effective in delivering DNA vaccines to mammals for therapy. 

===Genetic labeling===
A recent method developed for detecting transposition and may genetically label conjugative plasmids that do not result in an apparently identifiable phenotype. Genetic labeling results in the transformation of the host organism with a plasmid containing a heterologous DNA fragment. This DNA labels genetic areas of interest which may then be visualized. The method is also beginning to be studied in vivo, where mice can be orally inoculated with genetically-labelled probiotic bacteria plasmids to study products of the digestive tract.

===Patch clamp===
The patch clamp method allowed detailed understanding of the action potential after it was invented by Kenneth Cole in the 1940s. This method enables us to measure the membrane potential, or voltage, at any level desired by the experimenter through use of a microelectrode placed inside the cell. There are severable variations of the technique. An “inside out” patch is created when a portion of the cell membrane is ripped off, leaving the intracellular surface of the membrane oriented towards the media. This enables the study of intracellular components’ (e.g. ligands) that influence ion channels. An “outside-out” patch, on the other hand, allows the researcher to study the properties of an ion channel when it is exposed to a novel extracellular environment while having the same intracellular components.   The voltage clamp technique reveals how membrane potential influences ionic current flow across the membrane, and was instrumental in providing Hodgkin and Huxley with information leading to membrane ion gradients and the action potential.

===Electrical stimulation===
Coupled with the patch clamp technique, electrical stimulation is a form of manual current that may be delivered to the neuron under investigation in order to study its electrophysiological properties in a controlled manner. Usually only a few milliAmps are applied to the neuron to evoke passive responses. Greater current is introduced to evoke action potentials. Through this manner, the firing threshold for various types of neurons may be quantitatively determined with great accuracy. Furthermore, recent experiments have used this method to purposefully stimulate neocortical neurons to study the effects of prolonged activity (or "stimulation") on axonal growth.

===fMRI===

Functional magnetic resonance imaging is a technique used to visualize not only the neural anatomical images created by traditional MRI scans, but also overlaid images of event-related hemodynamic responses in the brain. The hemodynamic activation levels refer to the amount of blood oxygenation ocurring at a particular "voxel" of the image, which is a kind of three-dimensional pixel. This hemodynamic response is often referred to as BOLD (blood-oxygen level dependent) contrast. High BOLD contrast reflects a decreased amount of deoxygenated hemoglobin present in the brain. General changes in BOLD signal are highly correlated with changes in blood flow to different regions of the brain. 
Images of both anatomical and functional (BOLD) data are recorded every few seconds. Data can be analyzed in such a way as to contrast the activations associated with two separate paradigms, effectively subtracting the activation of one dataset from another and presenting the difference visually. This technique is generally applied to psychophysical ventures, quantifying the results of a multitude of psychological questions.

==Lecture 2 Model Systems==
<i>What are the advantages of each?</i><br>
===Frog visual system===
Some of the most groundbreaking work in understanding vision in humans were completed using the visual system of frogs. The retina of frogs has a uniform distribution, while mammalian eyes--such as those of humans--contain areas of higher resolution, called the fovea. The frog, however, has no fovea and it only moves its eyes to compensate for its own motion (either intended or accidental). This makes it easier to compare the patterns of light reaching the frog's retina with the signals leaving along its optic nerve. In early experiments, scientists placed electrodes along the nerve fibers emanating from the eye. Then, they shone a light on to the retina and looked at the pattern of responses along the outgoing nerves. The first studies of the responses in the frog's visual system were conducted in the late 1930's by the physiologist Hartline. Hartline introduced the term "receptive field" to describe the region of the retina to which an individual output responded. The discover of ON, OFF, and ON-OFF cells were made in the frog visual system and allowed us to gain a stronger understanding of how visual systems sharpen the constrast of perceived images.

===Vertebrate spinal cord===
===C. elegans sensory and motor neurons===

''Caenorhabditis elegans'' is a model system due to its short life cycle (about four days) and ease of maintenance.  These nematodes are also transparent, which allow for visual assays (such as with fluorescent proteins).  In the study of neurobiology, ''C. elegans'' is useful because it has a simple nervous system of 302 neurons.  For each of these neurons, the morphology and the connectivity to other neurons is known through electron micrographs.

Most sensory neurons of ''C. elegans'' can be categorized into two groups.  The first type consists of chemosensory neurons with channels that are exposed to the external environment.  The second group consists of mechanosensory neurons that lack these channels to the environment.  

The interneurons of ''C. elegans'' vary in their function and types of connections.  Some receive synaptic input from only a few neurons, while others receive signals from many neurons.

''C. elegans'' neurons have the interesting property in which the presynapses and postsynapses are not localized to only after or before the cell body, respectively.  For example, the interneuron AVE has a postsynaptic region and presynaptic region which both follow after the cell body.  Another unique property is that ''C. elegans'' neurons do not conduct action potentials.

===Drosophila embryo===
Embryonic development has been studied extensively in the Drosophila embryo, particularly the establishment of the dorsal-ventral and anterior-posterior axis, as well as segmentation in Drosophila.
The body axes of the embryo are prefigured in the oocyte by maternal effect genes.  These are prelocalized cytoplasmic determinants as well as localized extracellular signals (signals in the egg shell covering).  Scientists have identified about 50 maternal mRNA types that build up localized determinants during oogenesis that are pre-determined before zygotic genome is turned on.  More specifically, the dorsal-ventral axis is specified by an extracellular signal called spatzle.  Molecules laid down in the ventral extracellular egg covering during oogenesis locally activate a ligand (spatzle).  Spatzle quickly and locally binds its receptor, activating a signal transduction cascade that releases the transcription factor dorsal from a cytoplasmic inhibitor called cactus by degrading cactus.  As a result, dorsal enters nuclei on the ventral side.  dorsal protein is localized in the nucleus in a gradient on the ventral side of the blastoderm embryo.  The gradient of activity of the dorsal transcription factor sets up several different domains of target gene expression.  The gradient of activation of transcription factor (high ventral, low dorsal) and affinity of binding sites, determines how embryo pattern will develop based on signaling pathways.
The action of the bicoid gene during oogenesis is required to set up conditions for development of anterior structures in the embryo.  Bicoid mRNA, like dorsal, is a localized cytoplasmic determinant that is localized to the anterior pole of the oocyte during oogenesis (by motor proteins moving on microtubule tracks).  After fertilization the mRNA is translated and bicoid protein diffuses out to form a gradient.  It functions as a DNA binding protein that turns on transcription of the hunchback gene in the embryo.  Hunchback, in turn, acts to pattern the embryo as a gap gene.  The levels of bicoid protein sets the position of hunchback expression along the anterior-posterior axis since a threshold level of bicoid is required to turn on hunchback transcription.  Hunchback transcription is blocked in the posterior region by the nanos protein.  Nanos mRNA is localized posteriorly and similarly sends out a protein gradient that opposes the bicoid protein gradient as well as the maternal Hb mRNA that is distributed uniformly throughout the cell.
The gap genes, which include hunchback, giant, kruppel, and knirps, are transcription factors for segmentation.  There is a tapering of the proteins in each direction from where it is expressed to create gap gene domain expression overlap, creating combinations of more than one of their protein products.  The peaks overall to foster complexity since across head to tail axis there are different amounts and different locations of transcription factors, and genes sensitive to these differ in triggering influences that lead to segmented expression of genes.
Pair rule gene transcription is under gap protein control.  There are different DNA control elements for different stripes.  Combinations of transcription factors act on particular silencers/enhancers to control segmentation.  The regulation of lateral segments require different combinations of transcription factors, such as Wnt and Hedgehog signals that organize the pattern of bristles within each Drosophila segment.  Hedgehog and Wnt are both short range signals, but Hedgehod is a secreted protein and Wnt is a signaling pathway.  Adjacent cells talking to each other for feedback to reinforce each other’s signals in positive feedback.  If the Hedgehog signal fades then there’s no communication within the poles of the segments and Wnt causes bristles to form on all cells.
Finally, homeotic genes (Hox genes) are single transcription factors that can affect where development occurs by conferring different fates upon repeating body segments, inducing limb growth, and organizing organ placement.  This is based upon their select expression along the dorsal-ventral and anterior-posterior axis in accordance to combinatorial coding of the genes described above.

===Cell culture===
Small amounts of undifferentiated or single cells (normally from excised animal tissue) are placed in an artificial environment.  The nutrient medium depends on the experiment being conducted, but usually the medium favors cellular growth and differentiation.  By using cell cultures it is possible to pin down a cause and effect relationship between the carefully controlled culture and the development of the maturing cells.  Cell cultures can be manipulated by adding chemicals, nutrients, etc. to the cellular environment to test a hypothesis or achieve desired characteristic results.  Favorable qualities of cells can be precisely controlled, so that each cell is identical for the particular quality being sought, allowing for repetition within experimental methods.  In the case of neuroscience, axon growth, protein secretion, receptor up/down-regulation, and neurotransmitter release can all be studied and manipulated within culture to test the effects of a wide variety of cellular environments.

===Grasshopper===
Much of the early work on the study of axon growth was done on grasshoppers. Many of their individual motor neurons were identified and their pathways characterized.

===Xenopus axons in culture===
<br>
Xenopus is one of the most commonly used model organism in studying developmental biology. Their embryos are large, easy to obtain and are developed externally, allowing easy access to experimental manipulation. Such large cells are relatively easy to culture and manipulate experimentally. Thus, growing their neurons in culture is a optimal method for studying axon guidance and growth cone cues.

==Lecture 2 Techniques==
<i>What can these be used for?</i><br>
===Biochemistry===
Biochemistry is the study of chemical processes in a bioloical environment. The field deals mostly with structure and function of large biomolecules such as large proteins (enzymes, other cellular machinery), carbohydrates, nucleic acids, etc.  In terms of experimental assays, biochemical assays can be used to evaluate enzymatic functions, discover inhibitors for a certain enzyme, or to extract novel proteins within a certain tissue. For many biochemical assays, commons reagents are used and are easier to prepare on a daily basis, which gives this type of assay flexiblity.  A typical biochemical assay for detecting kinase activity of a certain protein would consist of a reaction with the enzyme, the substrate, ATP, and magnesium. Also biochemical assays usually do not require the cells to be alive.  

===Genetics: mutation and over expression===
A genetic mutation is a permanent change in the DNA sequence that makes up a gene. Mutations can affect a single DNA building block or even a large segment of an entire chromosome. Mutations may be induced in an egg or sperm cell or after fertilization; these changes are termed new (de novo) mutations, and may be experimentally beneficial for studying genetic diseases or for creating transgenic animal models that mimic aspects of human disease. 

The protein encoded by a particular gene may be expressed in an increased quantity ("over-expression") such that the phenotype of the organism can be significantly altered. Two commonly used techniques to create gene over-expression are to either increase the number of the copies of the gene, or, to increase the binding strength of the promotor.

===Co-culture on a 3D collagen gel matrix===
Cultured cells usually reflect growth in a 2D environment, whereas cells in the body exist in 3D matrices. Thus, it is important to have models where cell growth may traverse in a three-dimensional fashion. A 3D collage gel matrix allows this type of investigation. Cells in organisms exist in a 3D extracellular matrix environment rich in type I collagen, and a co-culture on a collage gel matrix provides a model for scientists to recreate this environment in a laboratory setting.  This 3D system provides a simple and rapid method to analyze cell angiogenesis, migration, apoptosis, proliferation and tissue formation in a 3D-collagen matrix. Cells suspended in this 3D system are easily visualized by phase contrast or fluorescence light microscopy. Cells can be directly fixed and stained within the matrix and treated with antibodies for visualization of specific intra- and extracellular proteins.

===Antibody Staining===
Antibody staining, also known as immunostaining, is a general term in biochemistry that applies to any use of an antibody-based method to detect a specific protein in a sample. The term immunostaining was originally used to refer to the immunohistochemical staining of tissue sections, as first described by Albert Coons in 1941.   Now however, immunostaining encompasses a broad range of techniques used in histology, cell biology, and molecular biology that utilise antibody-based staining methods.

===Cloning genes and expressing them in cell culture===
Cloning involves producing a large number of daughter molecules identical to the original gene product. Target genes can be inserted into plasmids/vectors which are subsequently used to transform bacterial cells for amplification; the bacterium Escherichia coli is commonly used. Plasmids are circular, double-stranded, extrachromosomal DNA that replicate upon insertion into a host cell and that confer their properties (e.g. antibiotic resistance) to the host. To insert a gene into a plasmid, first, specific restriction endonucleases are used to isolate a gene of interest from its original genomic DNA; the same restriction endonuclease is used to specifically cut the plasmid (e.g. at the BamHI, EcoRI, or SalI cloning sites) that will receive the gene insert, allowing for complementary base pairing between the “sticky” ends (single strand overhangs resulting from restriction endonuclease action). For each restriction enzyme, a corresponding modification enzyme prevents the host’s own DNA from being cut. DNA ligase is used to seal the gene insert within the plasmid, and this genetically engineered plasmid-gene construct is then used to transform competent E.coli host cells, through electroporation. The plasmid (with gene insert) then utilizes the host cell machinery to replicate itself until it reaches the cell’s maximum copy number. Cell culture of the transformed E. coli allows for exponential host cell growth and further amplifies plasmid replicate number. Plasmids are passed on to daughter cells in roughly equal proportions upon cell division. Gene product expression is driven by a promoter on the plasmid, located upstream of the cloning site. This technique is extremely valuable for research. Not only does it enable the study and characterization of individual genes of interest, but also it can be used in conjunction with various reporter assays to assess the effects of different mutations on the functionality of a gene of interest. Additionally, applications outside of the lab exist. For instance, this method enables industrial production of large quantities of insulin to meet the demands of diabetic patients.
===Forward genetic screen===
Genetic screens test and identify organisms with a specific phenotype. A forward genetic screen searches for new genes or mutant alleles, which rarely occur in nature. Hence, scientists perform a forward genetic screen by exposing the individual to a mutagen in order to induce mutations in their chromosome(s). Mutagens such as random DNA insertions by transformation or active transposons can also be used to generate new mutants.

Forward genetic approaches make no assumptions about the genes involved to illicit a given behavior. Random point mutations are introduced into the genome with chemical agents and the mutant organism is identified based on its altered phenotype. This is the opposite of reverse genetic approaches, which move from gene to phenotype rather than phenotype to gene. The use of  ENU (see Lecture 15 techniques) to create circadian mutant mice strains is a good example of how forward genetics uses altered phenotype to explore gene function.
===The Poo Assay===
The Poo Assay is used to assess growth cone turning responses to gradients of extracellular guidance factors.  It is named after its originator, Mu-Ming Poo, who used it to demonstrate the attractive turning of a growth cone towards a gradient of netrin-1 and the repulsive turning of a growth cone away from a gradient of semaphorin 3A. Isolated growth cones are cultured in a cell-free environment in vitro and then are exposed to gradients of a potential signaling molecule. Within an hour turning of the growth cone is evident and the angle of turning can be used to gauge the strength of the molecule’s signal. Turning should not be observed when the culture medium is supplemented with an antibody against the signaling molecule of interest.

===Explant overlay assay===
The explant overlay assay, known more commonly as the slice overlay assay, is an in vitro assay in which neuronal explants are cultured over cortical slices. The principal use of the explant overlay assay is to characterize extracellular signaling molecules that regulate neuronal differentiation and patterning. The two methods used for this purpose before the innovation of the explant overlay assay had significant shortcomings. An in vitro assay using neuronal explants cultured on an artificial substrate was problematic because the substrate was no substitute for the actual in vivo environment in which neuronal outgrowth takes place. The limitation of the second method, an in vivo assay that involved transplanting and monitoring labeled neurons, was that the chemical environment could not be manipulated like in an in vitro assay.  The explant overlay assay is able to resolve both problems, making it the most effective method for studying neuronal guidance molecules and mechanisms. Franck Polleux developed the explant overlay assay in 1998 to show that the initial growth of cortical axons toward the white matter is regulated by a semaphorin signal that is expressed in the marginal zone.

===Incubating slices in media with chemical cues===
===Mammalian pyramidal neurons===
Pyramidal cells are the primary projection neurons in the cerebral cortex and the hippocampus of the central nervous system (CNS, brain). Pyramidal cells have a pyramid-shaped cell a long and branching dendritic tree. An axon that carries nerve impulses emerges from one end of the cell. The axon may have local collateral branches but also project outside their region. These cells are multipolar neurons with a single apical dendrite and compose up to 80% of the neurons in the mammalian cortex. Pyramidal cells are excitatory neurons and release glutamate as their neurotransmitter.

==Lecture 3 Model Systems==
<i>What are the advantages of each?</i><br>
===Drosophila olfactory system===
The ''Drosophila'' olfactory system is a great model system for understanding how precise connections are made, what are the genes important for the formation of precise connections, and how formation of these precise connections are relevant for encoding olfactory information. Olfactory sensory neurons project their axons to discrete circular centers called glomeruli. At these glomeruli they connect with the dendrites of second order neurons, projection neurons. The projection neurons then send axons to the mushroom body calyx and the lateral horn for higher processing of olfactory information. The power of genetics has allowed scientists to label projection neurons. Since the advent of MARCM (Mosaic Analysis with a Repressible Cell Marker) one can label a subset of these projection neurons. One can even label a single projection neuron. Using MARCM, studies have shown that lineage and birth timing of projection neurons is correlated with their glomerular projections. MARCM has also been used to study the branching patterns of individual classes of projection neurons and the genes involved in the precise projections to single glomeruli (e.g. Sema1a, N-cadherin, Dscam).

===Three-eye frogs===
"An extra eye primordium was implanted into the forebrain region of embryonic Rana pipiens. During development both normal and supernumerary optic tracts terminated within a single, previously uninnervated tectal lobe. Autoradiographic tracing of either the normal or supernumerary eye's projection revealed distinct, eye-specific bands of radioactivity running rostrocaudally through the dually innervated tectum. Interactions among axons of retinal ganglion cells, possibly mediated through tectal neurons, must be invoked to explain this stereotyped disruption of the normally continuous retinal termination pattern." ("Eye-specific termination bands in tecta of three-eyed frogs" [http://www.sciencemag.org/cgi/content/abstract/202/4368/639])
<br><br>
Frogs do not have binocular vision because the outputs of the left and right eye do not converge. All retinal ganglion cells (RGCs; the cells that relay information from eye to the next level of information processing) from the left eye project their axons to the optic tectum on the right side. All RGCs from the right eye project their axons to the optic tectum on the left size. Because the left and right eyes are completely segregated there is no competition during development and no stripe formation is seen. However, when you transplant a third eye, you induce competition among axons projecting to the optic tectum. The competion between RGC axons from the transplanted and non-transplanted eyes to the same optic tectum gives rise stripes.
<br>

==Lecture 3 Techniques==
<i>What can these be used for?</i><br>
===In vitro stripe assay===

Creating a stripe assay involves affixing various substrates of interest into thin (~50 micrometers width) stripes onto a tissue-culture dish (thus, "in vitro"). One can then apply another substance to the culture dish and observe the effects of combination of both substances on the dish. For instance, one might wish to understand the molecular differences between anterior and posterior tectum to explain retinal axon patterning (this was done by Walter et al. in 1987, pg 13 of lecture 3 notes). To do this using the stripe assay, one would extract the membranes from anterior or posterior tectum and place them in alternating stripes, using flourescent labels to distinguish the two types of tissue. Then, temporal or nasal axons are allowed to grow on the stripes. Observing the results of such a test reveals that temporal retinal axons do indeed recognize the position-specific properties of the tectal cell membranes, because the temporal axons are attracted by the anterior membranes and repelled by the posterior tectal membranes. Thus, the in vitro stripe assay is a useful tool for understanding in vivo processes.

===2D gel electrophoresis===

A 2D gel electrophoresis is a process whereby proteins may be compared visually. The "gel" refers to a matrix of a specifically chosen polymer used to separate the  molecules of analysis. "Electrophoresis" is the term that describes the electro-motive force that is used to push the molecules along the gel matrix. Molecules are applied to wells at one end of the matrix, and an electric current is applied, causing the molecules to move in a certain direction (depending on their electric charge, towards the anode if negative and towards the cathode if positive. Visualization of the progress of the molecules is made possible by dyes. The example in lecture three comes from Drescher et al. (1995): the gel electrophoresis is used to comopare proteins from anterior and posterior tectal membrane (thus, "2D"). The ligand Ephrin for the Eph receptor tyrosine  kinase was found to be present in posterior, but not anterior tectal membrane. The Ephrin mRNA was revealed to be expressed in a gradient from posterior to anterior tectum.

A two-dimensional electrophoresis combines isoelectric focusing with SDS-PAGE, thus separating proteins with very high resolution. First, a sample of protein mixture is “isoelectrically focused”. When a mixture of small multicharged polymers (having many pI values) called polyampholytes undergo electrophoresis in a single-gel lane, they create a pH gradient in the gel. Thus, when the protein sample is subjected to electrophoresis in a gel with such a pH gradient, the proteins will each move to a position in the gel at which pH=pI, where pI is the isoelectric point of the protein, where its net charge and electrophoretic mobility are zero. This gel is then placed horizontally on an SDS-polyacrylamide slab. When the proteins are again subjected to electrophoresis (now in a vertical, rather than horizontal, direction, they will be now be separated by mass, creating a 2-D separation. (Berg, Tymoczko, Stryer, Biochem 2001, 5th Ed.) If the protein molecules are negatively charged, the smaller molecules will travel closer to the positive electrode end of the gel matrix, and the larger molecules will remain relatively closer to negative end.
===Transplantation===
In humans, tranplanted organs are used to replace a failing or damaged organ with a working organ from a donor. In research, transplatation is useful for exploring interactions between individual organisms--for example, the unique responses of an organism's immune system or the three-eyed frog to study axonal competition during neuron growth. Several types of transplatations are done: 

1) Allografts = transplanting organs or tissues from a genetically non-identical member within the same species;
2) Autografts = transplanting tissue from one area of one's body to another, usually with surplus tissue to replace damaged areas;
3) Xenografts = transplanting organs or tissues across species (example, pig's heart to human body);
4) Isografts = transplanting organs or tissues to a genetically identical member of the same species (such as a twin). This type of transplantation may overcome difficulties associated with organ rejection or triggering a recipient's immune system.

===Radiolabel injection===
Using radiolabeled injections, neurobiologists are able to observe cellular mechanisms and metabolisms in real-time, such as the influx and efflux of calcium within a cell. This technique is completed by making and attaching a radiolabeled tag to the compound of interest, then injecting this compound into the organism or cell system under study. Through neuroimaging techniques such as MRI, fMRI and PET, we are able to see the brain regions where certain chemicals are taken up and metabolized.

===TTX===
<b>Tetrodotoxin. (Also: anhydrotetrodotoxin 4-epitetrodotoxin, tetrodonic acid)</b>  A toxin from the [[puffer fish]] that blocks [[wikipedia:sodium channels|voltage gated sodium channels]]. Although originally found in the puffer fish and a few other organisms, TTX is now known to be synthetized by certain bacteria such as Pseudoalteromonas tetraodonis, some species of Pseudomonas and Vibrio, as well as others. The toxin works by blocking action potentials being created by binding to the pores of the voltage gated sodium channels in the neuron cell membranes. Tetrodotoxin binds to what is known as site 1 of the voltage-gated sodium channel. Site 1 is located at the extracellular pore opening of the ion channel. The binding of any molecules to this site will temporarily disable the function of the ion channel. Saxitoxin and several of the conotoxins also bind the same site. In humans, two categories of sodium channels with respect to TTX have been found: the tetrodotoxin-sensitive voltage-gated sodium channel (TTX-s Na+ channel) and the tetrodotoxin-resistant voltage-gated sodium channel (TTX-r Na+ channel). Nerve cells contain many TTX-s Na+ channels and thus TTX is a valuable tool in inducing paralysis of neurons in culture.<br>

===TEA===
<b>Tetraethylammonium.</b> A compound which selectively blocks [[wikipedia:potassium channels|voltage gated potassium channels]]. Unlike TTX, TEA is synthesized for the purpose of being used as a potassium channel blocker in neuropharmacological experiments. The K+ eflux is responsible for the trailing part of the action potential so stopping it has a definite effect on the shape of the action potential.<br>

===Differential Display===
A technique used to determine the differences in [[expression]] of mRNA between two cells under different conditions or between two different cell, using [[mRNA probes]].  This technique is rapidly being replaced by [[expression profiles]] using [[microarray]]s.<br>

===In-situ hybridization===
In-situ uses mRNA probes (also called oligos) that anneal to the mRNA strand of interest in fixed animal tissue. Because the probes are usually fluorescently-tagged, this technique allows visualization of mRNA in cells/tissue, providing quantitative data on the amount of genetic information being expressed.

===Knockout mice===
Knock-out mice are genetically engineered animals with one or more genes that are made inoperable through a gene knock-out. Knock-out animals are significant to research because they allow us to test and identify the function of an identified gene whose effect is partially or fully unknown. Knock-out techniques are usually performed in mice, which are genetically similar to humans; this procedure is also easier to perform in mice compared to rats, in which knock-outs have only been possible since 2003. A typical procedure for creating knock-out mice are as follows:

1) Isolate the gene to be knocked-out from a mice genome library. A similar DNA sequence to the gene of interest is synthesized, but is made with significant changes so that the gene is inoperable.
2) Isolate stem cells from a mouse morulla, which can be grown in vitro.
3) Combine the stems cells with the re-created DNA sequence. Some of the cells will be able to incorporate the new DNA into their genomic sequence.
4) Insert stem cells into mouse blastocyst cells, then implant into a mouse uterus to complete the pregnancy.
5) Newborn mice are chimeras, sometimes not fully knocked-out mice. These animals are then crossed with other chimeras to potentially produce an offspring that is a full knock-out transgenic mouse.

===Monocular enucleation===
Monocular enucleation is a useful experimental tool for analyzing the mechanisms of neural plasticity. The process involves depriving one eye of environmental input during an early postnatal period and then observing the structural changes that result in the brain. What results is an adaptive reaction in which the visual system compensates for the lost sensory capacity. Afferent neurons from the deprived eye are connecting to the lateral geniculate nucleus (LGN) and the superior colliculus (CS) are destroyed as a result of the degeneration of retino-geniculate and collicular synapses, which receive no stimulation from the deprived eye. Inserting a tracer into the LGN of ME mice reveals a thinning of the ocular dominance columns. This is because the retinogeniculate fibers coming from the remaining eye replace begin to replace these synapses. Ipsalateral representation of the remaining eye thus becomes extended in the left LGN and in the left visual area. This raises the interesting question of how the deprivation of input results in adaptive plastic change. This is the primary question that is experimentally explored with monoculear enucleated organisms. (Reviewed by [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=8735877&query_hl=18&itool=pubmed_DocSum Toldi et al 1996])
==Paper 1 Model Systems==
<i>What are the advantages of each?</i><br>
===Chick optic tectum===
===Mouse superior colliculus===
===Mouse retina===
<br>
==Paper 1 Techniques==
<i>What can these be used for?</i><br>

===HEK293 cells===
	HEK 293 cells are an epithelial cell line originally derived from embryonic human kidney. As an experimentally transformed cell line, HEK cells are not a particularly good model for normal cells, cancer cells, or any other kind of cell that is a fundamental object of research. However, they are extremely easy to work with, being straightforward to culture and to transfect, and so can be used in experiments in which the behavior of the cell itself is not of interest. Typically, these experiments involve transfecting in a gene (or combination of genes) of interest, and then analyzing the expressed protein; essentially, the cell is used simply as a test tube with a membrane.

Examples of such experiments include:
*A study of the effects of a drug on sodium channels
*Testing of an inducible RNA interference system
*Testing of an isoform-selective protein kinase C agonist
*Investigation of the interaction between two proteins
*Analysis of a nuclear export signal in a protein

In the Schmitt et al (2006), HEK 293 cells were used in a preliminary test to determine whether Wnt3 can regulate the growth of RGC axons. Schmitt el al created HEK293 cells transfected with the wnt3 gene in order to have the Wnt3 protein expressed in membrane fractions of HEK293 cells (Wnt3 is highly hydrophobic and associates tightly with cell membranes). They found that Wnt3-transfected HEK293 cell membranes inhibited the growth of both dorsal and ventral mouse RGC axons at higher concentrations, and stimulated the growth of dorsal but not ventral RGC axons at lower concentrations (data was not shown).

===SF9 cells===
An insect cell line used for the production of recombinant protein. The Sf9 cell line is derived from pupal ovarian tissue of the Fall armyworm Spodoptera frugiperda. The Sf9 cell line is highly susceptible to infection with Autographa california nuclear polyhedrosis virus (AcNPV baculovirus), and can be used with all baculovirus expression vectors. Sf9 cells are commonly used to isolate and propagate recombinant baculoviral stocks and to produce recombinant proteins. In the Schmitt et al. paper, Sf9 cells were used to overexpress Wnt3 (using the Baculovirus system) to obtain sufficient and consistent amounts of Wnt3.<br>

===Baculovirus system===
Baculovirus is a natural pathogen of the caterpillars producing the SF9 cell line.  In the lab, genes are encoded into a baculovirus vector which is then used to infect SF9 cells.<br>

===Affinity-purified protein===
A protein purified by passing a solution of protein through a column where the protein becomes associated with a matrix of immobilized [[ligand]] somehow attatched to the column.  In most cases the protein must be tagged, or appended to a functional motif called a [[fusion tag]].  Common fusion tag-ligand pairs include:  [[Histidine tag]] (6 or more extra Histidines) and the "ligands" Chelated Nickel or Cobalt, [[Maltose Binding Protein]] and its ligand [[dextrin]], [[Glutathione S-transferase]] and its ligand [[reduced glutathione]], and [[Green Fluorescent Protein]] and Anti-GFP [[antibody]].

===Mock infection===
A control used in infection experiments. Two specimens are used: one that is infected with the virus or vector of interest and the other is treated the same way except with the virus. Sometimes, a non-virulent strain is used in the mock-in the mock-infection experiments.

===Blocking with antibodies or proteins===
===Western Blot, α-tubulin===
Western blots are a technique using immunolabeling to detect proteins from a tissue sample or extract that is run on a gel. Western blots use gel electrophoresis (such as SDS-PAGE) to separate proteins by mass. Several key steps define the methods germaine to a western blot:

First, tissue preparation is required: tissues of interest are frozen rapidly and homogenized using sonication to lyse cells. This homogenate may then be centrifuged in several separate steps to isolate cystolic proteins or membrane proteins. Second, the collected protein samples are run using gel electrophoresis which separates them according to weight. Third, proteins are prepped for detection by being transferred to a nitrocellulose membrane. This procedure is done by placing the membrane face-on-face with the gal and using a current to move charged proteins across the gel-membrane surfaces. Fourth, the membrane proteins are blocked to prevent non-specific binding. Finally, detection of the proteins of interest are done via blotting, such as using antibodies to detect the protein under study. After blotting the membrane is usually washed with developing solution and the amount of antibody signal is transfered onto film for visual quantification. The α-tubulin is a protein that is observed in nearly the same amount in most cells.  Its antibody signal is used to normalize the data to account for the error in loading different amount of protein into each well on the gel.  

===Retina explant assay===

Tissue from the retina at different locations was removed and cultured on cover slips, to allow the chemical interactions and environment to be tightly controlled and the resulting axon growth to be monitored.  Axon growth in the presence of a protein, hormone, or other factor, compared to no or little growth in control, can be used to indicate the sufficiency of that factor for axon growth.  Or, relative lack of growth relative to control can be used to demonstrate inhibition.  Note that different concentrations of the same factor may have opposite effects on an explant, so testing multiple concentrations in addition to control is advised.  The use of explants from different locations in the retina is necessary because a factor may have the opposite effect on nerve tissue depending on its location,  depending on the destination of those axons in vivo, relative to the expression patterns of the factor.

===Electroporation into ventricular zone===
===Dominant-negative===

A dominant-negative is a mutation whose gene product adversely affects the normal, wild-type gene product within the same cell. This usually occurs if the product can still interact with the same elements as the wild-type product, but block some aspect of its function.


Examples:
1. A mutation in a transcription factor that removes the activation domain, but still contains the DNA binding domain. This product can then block the wild-type transcription factor from binding the DNA site leading to reduced levels of gene activation.
2. A protein that is functional as a dimer. A mutation that removes the functional domain, but retains the dimerization domain would cause a dominate negative phenotype, because some fraction of protein dimers would be missing one of the functional domains.


The Shmitt et al Ryk dominant negative:
Wnt3 knockout mice fail in early embryonic patterning because Wnt3 is important for early nervous system development. This makes it impossible to examine the function of the Wnt3 gradient on the medial-lateral axis of the mouse superior colliculus because knockout Wnt3 mice die at birth and axon termination zones form at postnatal day 8. To circumvent this difficulty, Schmitt et al generated a dominant-negative form of Ryk. This truncated Ryk protein only contained Ryk ectodomain (extracellular) and the transmembrane domain, missing the intracellular domain. This dominant negative Ryk allowed Schmitt et al to test in vivo whether blocking Wnt3-Ryk function will shift the termination zone of RGX neurons in the superior colliculus of mice medially.

===In ovo electroporation===

===DAPI staining===
DAPI is a fluorescent stain that binds strongly to DNA. It is used extensively in fluorescence microscopy. Since DAPI will pass through an intact cell membrane, it may be used to stain live and fixed cells.
For fluorescence microscopy, DAPI is excited with ultraviolet light. When bound to double-stranded DNA its absorption maximum is at 358 nm and its emission maximum is at 461 nm, and appears blue/cyan. DAPI will also bind to RNA, though it is not as strongly fluorescent. Its emission shifts to around 400 nm when bound to RNA.
DAPI's blue emission is convenient for microscopists who wish to use multiple fluorescent stains in a single sample. Apart from labelling cell nuclei, the most popular application of DAPI is in detection of mycoplasma or virus DNA in cell cultures. Because DAPI readily enters live cells and binds tightly to DNA, it is toxic and mutagenic.

===AP (alkaline phosphatase)===
Alkaline phosphatase is an enzyme that removes phosphate groups from many molecules such as nucleotides and proteins.  This enzyme is present in practically all organisms, ranging from E. coli to shrimp to humans.  In the lab, AP is a tool frequently used in molecular biology.  One example of its use is in cloning of DNA plasmids.  When a vector and an insert need to be ligated together, measures must be taken to prevent self ligation of the vector (vector religating with itself without the insert).  AP selectively removes the 5' phosphate group on the vector DNA.  Thus, the only phosphate groups left are on the inserts, which then can be ligated onto the vectors using DNA ligase that forms a phosphodiester bond between the 5' end of the insert to the 3' hydroxyl on the vector.  This procedure increases the efficiency of the ligation step.

Another use for AP is in ELISA.  AP is an enzyme that catalyzes colorless pnitrophenylphosphate (pNPP) into p-nitrophenol, a yellow compound.  In a sandwich ELISA, the first antibody is fixed onto the bottom of the well, an sample that contains antigen is added, then a second antibody is added to recognize the antigen.  This second antibody is complexed with AP.  Then the substrate for AP, pNPP, is added and the amount of color change from clear to yellow is measured.  With the aid of AP, scientists can tell the amount of antigen present in the sample by the amount of color change.        
===Protein overexpression===
Selective gene amplification by a cell results in more templates for transcription, which is the basis of natural protein overexpression.  The cell can make more of a certain gene product by increasing the number of copies of the appropriate gene and transcribing them all.  This strategy takes advantage of the transcription mechanisms already in place within the cell and merely feeds them more material to transcribe.
In the lab, the polymerase chain reaction (PCR) technique makes multiple copies of a DNA sequence by copying a short region of DNA many times in a test tube.  It is a cyclic process in which a sequence of steps is repeated over and over:  A DNA molecule with a target sequence to be copied is heated to denature it.  When the mixture cools, short, artificially synthesized primers bond to the single-stranded DNA.  Then dNTPs (four deoxyribonucleotide triphosphates dATP, dGTP, dCTP, and dTTP) and DNA polymerase are added to synthesize two new strands of DNA.
One goal of recombinant DNA technology is to produce many copies (clones) of a particular gene either for the purposes of analysis or to produce its protein product in quantity.  Scientists normally use bacteria as hosts because they are easily grown and manipulated.  Bacteria also contain plasmids, small circular chromosomes, which can be manipulated to carry recombinant DNA into the cell.  But bacterium is not ideal for studying and expressing eukaryotic genes because they lack the splicing machinery to excise introns from the initial RNA transcript of eukaryotic genes.  Many eukaryotic proteins are extensively modified after translation, so scientists use vectors rather than bacteria to carry the new DNA into host cells.
Vectors already have a built-in origin of replication.  The new DNA has to become part of a segment of DNA that contains an origin of replication (i.e. join a replication unit) in order to be replicated in the host cell as it divides.  Plasmids are often used as vectors because they are small, naturally occurring in bacteria, often have only a single recognition site for a given restriction enzyme, and allows for the insertion of DNA at only one location.  When the plasmid is cut with a restriction enzyme, it is transformed into a linear molecule with sticky ends that can pair with the sticky ends of another DNA fragment cut with the same restriction enzyme.  Viruses can also be used as vectors to insert large numbers of base pairs into a genome.  Even if the genes that cause the host cell to die and lyse are gone, the virus can still attach to a host cell and inject its DNA, which in our case is the new DNA to be expressed.
Finally, expression vectors allow foreign genes to be expressed in host cells and can turns cells into protein factories, contributing to protein overexpression.  Expression vectors contain the sequences for promotion, termination, and ribosome binding, which are necessary for protein synthesis in a foreign cell.  The protein to be overexpressed is inserted at the restriction site, the bacteria of choice is transfected with the expression vector, and the protein is synthesized because of its locale in the DNA.  Inducible promoters that respond to a specific signal, thus initiating protein synthesis, can be inserted into the expression vector so that the production of the target protein can be controlled.

===sFRP2===
secreted frizzled related protein 2 is an [[antagonist]] of the [[Wnt]] ligand in [[Wnt-Frizzled]] mediated [[cell signalling]].

===DiI===
A [[lipophillic]] compound used to label cells.  DiI has [[affinity]] for any [[cell membrane]] and is therefore not cell specific, but will only label the cell individually injected with DiI.<br>

==Lecture 4==

===Bungarotoxin===
Toxin harvested from the snake species ''Bungarus multicinctus'' that binds [[Acetylcholine receptors]] and therefore paralyzes its prey.   Alpha bungarotoxin is used as a label for [[Acetylcholine receptors]].

===Agrin===
A [[proteoglycan]] made by nerve and glia.  Agrin is transported to the [[nerve terminal]] and [[synaptic cleft]].  Due to the phenotype of agrin knockout mice (dispersed [[acetylcholine receptors]]), agrin was believed to be the factor which organizes the aggregation of [[acetylcholine receptors]] into clusters.  Later experiments in model systems in which agrin could not have been present due to the absence of the pre synaptic nerve (Homeobox 9 or [[HB9]] knockouts) showed that Agrin was not necessary for clustering.  It has since been elucidated that agrin stops the dispersion of [[acetylcholine receptors]].  Dispersion of [[acetylcholine receptors]] is caused by the receptor's own [[ligand]], the [[neurotransmitter]] [[acetylcholine]].

===Proteoglycan=== 
A class of [[glycoproteins]] which contain glycosaminoglycan chains

===Glycan===
The polysaccharides which form the carbohydrate moiety of [[glycoproteins]].   

===MuSk===
A receptor [[tyrosine kinase]] found in muscle necessary for aggregation of [[Acetylcholine receptors]] into clusters.  MuSK co-localizes with [[Acetylcholine receptors]].  Its expression peaks during the formation of neuro-muscular junctions.

===Rapsyn===
A cytosolic protein necessary for proper Acetylcholine aggregation.  During early stages of muscle development Rapsyn co-localizes with acetylcholine receptors.

===ChAT===
Choline Acetyl transferase.  The enzyme is responsible for the synthesis of Acetylcholine and Coenzyme A from Choline and Acetyl-CoA. In ChAT mutant mice, there is an overabundance of acetylcholine receptor clusters. Additionally, in a ChAT-/-, agrin-/- double knock-out, the agrin single knock-out defect (lack of acetylcholine receptor clustering) is rescued. This suggested that acetylcholine is the negative regulator of acetylcholine receptor clustering, thereby leading to "Paradigm lost!" and a modified agrin hypothesis model to be formed.
===Neuregulin===
A protein which is a known [[ligand]] for the erbB type [[receptor]] [[tyrosine kinase]].

==Lecture 4 Model Systems==

===Neuromuscular junction===
A neuromuscular junction is the synapse or junction of the axon terminal of a motoneuron with the motor end plate, the highly-excitable region of muscle fiber plasma membrane responsible for initiation of action potentials across the muscle's surface, ultimately causing the muscle to contract. The signal passes through the neuromusclar junction via the neurotransmitter acetylcholine.

===Electric ray===
Generates electric current. A large amount of ACHR and other proteins involved in the neuromuscular junction are present at high density in the electric ray, making it a suitable model organism for study of the neuromuscular junction.

===Zebrafish===
Zebrafish are a useful model organism since their fertilization process is external, which allows scientists to study all stages of development with ease. In addition, they have a large number of offspring which allows for easy analysis of general trends in offspring. Most importantly, the embryos and early stages are transparent, which makes it easy to view internal structures. It also allows for labeling and tracking of proteins or cells labeled with GFP that can be seen through the skin of the animal.

===Squid giant axon===
The giant squid axon has been used to elucidate many of the most exciting problems in the history of neurobiology, such as how action potentials initiate and what ion channels contribute to the resting membrane potential. The squid giant axon is about 1 mm in diameter, making its large size easy for axons to be visualized. In particular, Hodgkin and Huxley were able to perform their work on action potentials by inserting electrodes directly into the luman of the axon. The squid giant axon has provided the highest rate in measurement accuracy of action potentials, and they are still used in contemporary studies.

===Genetic mosaic animals===

==Lecture 4 Techniques==

===α-bungarotoxin===
α-bungarotoxin is one of the toxic components of the venom of the elapid snake Taiwanese banded krait (Bungarus multicinctus). It binds irreversibly to the acetylcholine receptor found at the neuromuscular junction, causing paralysis, respiratory failure and death. Bungarotoxin was discovered by researchers of the National Taiwan University in 1963. α-bungarotoxin is also a selective antagonist of the α7 nicotinic acetylcholine receptor in the brain.

===Protein extracts===
===anti-NF===
===YFP===
Yellow fluorescent protein allows scientists to visualize and monitor cellular processes in organisms using optical microscopy and confocal microscopy. YFP is made usuing molecular cloning methods by fusing the fluorophore to a diversity of proteins or enzyme targets. After the crystal structure of green fluorescent proteins was elucidated, it was found that threonine residue 203 was near the chromophore. By mutating this residue to tyrosine, the excited state of the chromophore may be manipulated and shifted to longer wavelengths, resulting in YFP or proteins of various colors.

===CFP===
CFP, or cyan fluorescent protein, is a mutant form of GFP (Green Fluorescent Protein) that floresceses with the color cyan instead of green. The obvious benefit is to allow for staining of different markers in the same organism. Yellow Fluorescent Protein (YFP) is also an analogue of CFP and GFP. Fluorescent proteins are used in Fluorescence resonance energy transfer (FRET) experiments, in which these proteins can be coupled to important cellular proteins to visualize cell activity in vivo easily without causing undue disturbance to the living cell. For more information, please see the entry for [[GFP]].

===Radiolabeled amino acids===
Radiolabeled amino acids are made by replacing a carbon atom with 11C in a physiologic amino acid.  This does not chemically change the molecule, but allows for detection through positive electron tomography.  Or radiolabeled amino acids can be imaged after being infused into cells and incorporated into synthesized proteins while in culture or in vivo.  Tracking where the amino acids travel and their activities serves to garner information about cellular function and can aid in imaging structures.  Methionine is the most popular amino acid for PET when made into L-[methyl-11C]-methionine (MET), and it is extremely effective at diagnosing brain tumors.

===Time lapse===
===GFP fusion protein===
GFP has been widely used to visualize receptors and various intracellular proteins and their movements in live cells. To investigate the dynamics of protein trafficking or membrane receptor diffusion, GFP-fusion proteins may be used. GFP may be fused to the protein of interest, and using Fluorescence Recovery After Photobleaching (FRAP), the rates of diffusion or trafficking of proteins may be determined. This process is completed by repeatedly bleaching areas of the cell containing the protein under investigation, then measuring the recovery intensity of the region after bleaching.

===Double knock-out===
===EP screen===
===Lethal enhancer screen===

==Lecture 5==  

===Enhancer Promoter Screen===
A screen for over expression mutant phenotypes.  The genotype is created through random insertion of a strong promoter into the genome.

===Lethal Enhancer Screen===
A screen for a second mutation that enhances a phenotype of another mutation which by itself is not lethal.  In other words, to screen for a second mutation that by itself is viable but when combined with the first mutation produces a lethal phenotype.  This type of genetic screen is usually used to delineate gene interactions because the second mutation must be involved in some pathway with the first in order to it be a lethal enhancer.  Such a genetic screen usually use a chemical mutagen (such as EMS) to induce the second mutation and then crossing the mutated flies with those with the first mutation.  Those with desired lethal phenotypes were then selected.  

In lecture 4, ubiquitination processes were implicated in the development of synapses at the Drosophila neuromuscular junction.  DiAntonio et al (Nature, 2001) showed that overexpression of fat facets, a deubiquinating protease, led to disrupted synaptic function and an overgrowth of synapses.  A lethal enhancer screen was employed to find other genes that enhance the fat facets phenotype.  The screen looked for viable mutations that produced lethal phenotypes when combined with fat facets overexpression.  The Highwire gene was one such mutation isolated in the screen.  Highwire loss of function mutations resulted in the same synaptic overgrowth as fat facets overexpression and biochemical analysis of highwire revealed it to be an ubiquitin ligase.  Take together, these data show that fat facets and highwire antagonize each other in the ubiquitination process and strongly support the idea that synaptic development is controlled by positive (highwire) and negative (fat facets) regulators  of ubiquitination.
==Lecture 5 Model Systems==
===Electric eel===
The electric eel (Electrophorus electricus) is a species of fish native to South America that is able to generate electric shocks using the electric organ of its body.  The organ is composed of electroplaques, a stack of plates that can generate charges.  The organ is also extremely rich in voltage-gated sodium channels.  In 1986, a team of Japanese researchers led by Shosaku Numa cloned the cDNA for the sodium channel using the electric organ of the eel.  From that scientists were able to deduce the biochemical structure of the sodium channel.  

==Lecture 5 Techniques==
===Aldicarb===
Aldicarb (chemical name: 2-methyl-2-(methylthio)propionaldehyde O-methylcarbamoyloxime) is an carbamate-class insecticide applied directly to the soil and is used to control mites, nematodes, and aphids.  In the laboratory, aldicarb acts as an inhibitor of acetylcholinesterase (AChE), the enzyme present in the basal lamina of the post-synaptic cell of the neuromuscular junction (NMJ) which breaks down the neurotransmitter acetylcholine (ACh).  ACh is the excitatory neurotransmitter for muscular contraction; therefore, inhibition of AChE using aldicarb results in prolonged activation of ACh receptors in the post-synaptic cell, causing paralysis and eventually death.

Aldicarb is a useful tool in genetic screens that search for mutants resistant or hypersensitive to the aldicarb-induced paralysis.  These mutants will likely have mutated genes involved in the ACh signaling pathway at the NMJ.  Mutants resistant to aldicarb will likely have an impairment of normal ACh signaling, while mutants hypersensitive to aldicarb will likely have an exaggeration of normal ACh signaling.

===PCR===
PCR, or the polymerase chain reaction, is an experimental technique devised by Kary Mullis in 1984 and is used to amplify a targeted DNA segment for further experimental analysis.  PCR does not require knowledge of the target DNA sequence; however, knowledge of the DNA sequences flanking the target is necessary.

A PCR cycle proceeds as follows: 1) The parent DNA duplex is separated by heating the solution to 95°C for 15 seconds, exposing the bases on each strand.  2) Primers for the flanking DNA sequences anneal to the 3'-end of each parent DNA strand when the solution is cooled to 54°C.  3) A heat-stable DNA polymerase called Taq DNA polymerase (derived from the thermophilic bacterium Thermus aquaticus) synthesizes complementary DNA strands starting at the primers and using available nucleotides in solution when the solution is heated to 72°C.  Repeated cycles of PCR allow for an exponential amplification of the target DNA sequence.

===E. coli===
===Ion replacement===
===Low transmitter release conditions===
===Caged calcium===

==Lecture 6==

===''Shibire''===
The shibire mutation causes temperature-sensitive paralysis in adult Drosophila. The shibire gene encodes a homologue of rat dynamin protein. Dynamin is a GTPase that localizes to the neck of budding vesicles and is involved in vesicle scission from the parent membrane. When shibire mutants are exposed to temperatures exceeding a restrictive temperature, the function of the shibire protein is disrupted and endocytic vesicles can no longer be separated from parent membranes. As a result, endocytosis is blocked, membrane cycling is prevented and synaptic vesicles are depleted, especially in synaptic terminals. Because synaptic vesicles are depleted, when shibire mutants are exposed to temperatures exceeding a restrictive temperature they become paralyzed. The shibire protein can regain its function and the paralyzed phenotype can be reversed upon lowering to a permissive temperature.

===Electron Microscopy===
Electron microscopy, invented in 1931, is a form of microscopy that uses electrons (as opposed to photons in a classical optical microscope) to create a visualization of very small objects. Before the invention of electron microscopy, microscopes were limited by the wavelength of light, which meant that the minimum distance objects could be from each other before the microscope blurred them together and was incapable of distinguishing them was too high for many applications of microscopy. Electrons have a much smaller wavelength than light and thus microscopy based on them is capable of distinguishing much more detail. Advantages to electron microscopy include this increased resolution (0.5nm vs around 200nm for light microscopes) and much higher maginifications (250,000x vs 1500x). All this comes at a price, however: electron microscopes are expensive both to build and maintain. Furthermore, qualities of electrons mean that samples can only be observed in a vacuum, meaning that they have to be specially prepared. A critical consequence of this is that all samples will be killed in the process, and artifacts might be created. Contrast this with light microscopy, which allows viewing of even moving samples. As electrons have no color, electron microscopy creates a "monochrome" image, one which is often stained with arbitrary colors to increase viewability. Cells are sliced for viewing under an electron microscope using a tool called a microtome.

===MK801===
MK801 is an experimental drug that, like PCP, blocks the flow of calcium ions through the NMDA receptor channel.  It binds to a site within the open channel pore and thus is considered a non-competitive antagonist of the NMDA receptor. It binds with a high affinity and its radiolabeled form has been used to label NMDA receptor populations in brain slices. Blockage of the NMDA receptor results in hallucinations similar to those seen in schizophrenia, leading to the hypothesis that schizophrenia may be the result of a defect in NMDA receptor function. MK801 has been studied as a potential treatment for diseases that are the result of excitotoxic neurodegeneration such as stroke and Alzheimer’s.

===APV (AP-5)===
APV (also called AP-5) is a selective NMDA receptor (NMDAR) antagonist that competitively inhibits the active site of NMDAR. Its chemical name is R-2-amino-5-phosphonopentanoate. APV is generally very fast acting within in vitro preparations, and can block NMDA receptor action at a reasonably small concentration.

===Hippocampus to study plasticity mechanisms===

==Lecture 6 Model Systems==

===Xenopus oocytes===

Xenopus oocytes are an ideal system in which to perform gene and protein expression experiments, as well as knockdown studies, given their large size (~1 mm in diameter). Their size makes them easy to culture, as well as to inject RNA from other species into.

===Mammalian hippocampus===

==Lecture 6 Techniques==
===Sequence comparison===
===Hydrophobicity plot===

Hydrophobicity plots are used to determine the relative polarities of amino acids found in a protein sequence.  The main use of these plots is to predict transmembrane regions of proteins, which are usually characterized by long sequences of hydrophobic residues.  In order to generate a hydrophobic plot, each amino acid in a sequence is scored based on one of two scales: the Kyte-Doolittle scale and the Hopp-Woods scale.  In the Kyte-Doolittle scale, highly hydrophobic regions achieve large positive values, and this scale is used predominantly to pick out potential transmembrane regions of the protein.  The Hopp-Woods scale was developed to predict potential globular protein binding sites, which are usually characterized by many polar residues.  This scale can be views as a hydrophilic index, where the Kyte-Doolittle represents a hydrophobic index.

===Tetanic stimulation===
Tetanic stimulation is a technique used in neurobiology to induce long term potentiation in post-synaptic neurons.  It is performed by applying a sequence of high-frequency stimulations to artificially induce rapid EPSP temporal summative effects, mimicking increased neurotransmitter release and binding by postsynaptic receptors (i.e. large amounts of glutamate binding to AMPA receptors on the postsynaptic membrane).  This results in greater postsynaptic depolarization if each successive pulse of tetanic stimulus reaches the postsynaptic cell before the previous EPSP can appreciably decay.  The progressive and prolonged depolarization removes the magnesium blockage of the NMDA receptor, and subsequent stimuli promote an extremely rapid calcium influx through the NMDA receptor-coupled ion channel.  Rapidly dramatically raising calcium’s intracellular concentration triggers a sequence of events ending in enhanced excitability to future stimuli.  The calcium made available by the NMDA channel binds to chelator calmodulin  and among other things activates the calcium/calmodulin dependent CaMKII.  CaMKII increases the excitability of existing AMPA receptors and voltage-gated potassium channels by phosphorylating them and also initiates the MAP kinase cascade which adds new AMPA receptors to the postsynaptic membrane.  The incoming calcium also binds adenyl-cyclase, indirectly raising the level of cAMP in the neuron.  cAMP turns on PKA, an important protein kinase that phosphorylates voltage-dependent potassium channels and also calcium channels thus lowering their threshold for opening in response to future stimuli.

===Protein synthesis blockers===

==Lecture 7==

===adenylate cyclase===
An enzyme that takes [[AMP]] as substrate and synthesizes [[cyclic AMP]] ([[cAMP]]).  cAMP is [[secondary messenger]] in many [[signal transduction cascades]].  There are nine different isoforms of adenylate cyclase that respond differently to the various kinases, elevated calcium levels, and to G protein subunits other than the stimulatory alpha subunit (by which all adenylate cylcases are activated.)

===cAMP Phosphodiestrase=== 
An enzyme that takes the [[second messenger]] [[cyclic AMP]] ([[cAMP]]) as a substrate and hydrolyses the phosphodiester bond, creating AMP.  Important enzyme in shutting off signal transduction cascades.

===Isoproternol===
An agonist of the [[beta andrenergic receptor]], a receptor for the catelchomines [[epinepherine]] ([[adrenaline]]), and [[norepinepherine]] ([[noradrenaline]]).

===Beta andrenergic receptor===
A receptor for the catelchomines epinepherine (adrenaline), and norepinepherine (noradrenaline).  It is one type of G-protein coupled receptor or GPCR, in which ligand receptor binding activates a G protein which in turn activates an adenylate cyclase.



===G-proteins===
G protein is a heterotrimeric protein that exists in two forms, a [[GTP]] bound form, and a [[GDP]] bond form, and acts as an on/off switch in signal transduction cascades.  The trimer is activated when it binds GTP.  When activated the trimer separates into two subunits, one being the alpha subunit, the other being a heterodimer of the beta and gamma subunits.  The alpha subunit activates its target adenylate cyclase, causing synthesis of the [[cAMP]] [[second messenger]].  Recent research suggests that the beta/gamma subunit also has downstream targets.  G protein to GTP binding is mediated by an enzyme called the [[Guanine nucleotide Exchange Factor]] or [[GEF]] which exchanges GDP for GTP in the G-protein binding site.  The alpha subunit has intrinsic GTPase action, and therefore can turn the entire trimer into the inactivated GDP-bound form, however hydrolysis of GTP to GDP by the alpha subunit is accelerated by the enzyme [[GAP]] or [[GTPase activating protein]].  When in the inactivated GDP bound form, G protein reforms the heterotrimer.

===Sodium Fluoride and G protein activation===
Sodium fluoride and aluminum fluoride are G protein activators.

===Narcoleptic dogs===
Narcolepsy is a primary sleep disorder, whose prominent symptom is excessive sleepiness. It was first identified by Jean-Babtiste in 1880. In the 1950s the narcoleptic syndrome was defines as consisting of four symptoms: (1) daytime sleepiness, (2) cataplexy,thbe reversible loss of muscle tone  (3) sleep paralysis, and (4) hypnagogic hallucinations. After the discovery of REM sleep, it was discovered that patients with narcolepsy begin sleep with REM sleep, whereas normal sleep begins with non-REM sleep. 
Narcolepsy has been associated with a class II antigen of the major histocompatibility complex on chromosome 6 at the HLA-DR2 or HLA-DQW1 locus. HLA-DR2 is also associated with autoimmune diseases such as multiple sclerosis and rheumatoid arthiritis, raising the possibility that narcolepsyt has an immunological basis.
Some dogs are narcoleptic, and their narcolepsies are similar in most respects to human narcolepsy, except for the mode of genetic transmission. In narcoleptic dogs, abnormalities have been found in cholinergic and monoaminergic synaptic transmission, important components of REM sleep regulation. Dogs with narcolepsy have more muscaniric M2 receptors in the pons, suggesting a defect in cholinergic sensitivity. Consistent with this, cholinergic antagonists inhibit and agonists exacerbate canine cataplexy. Norepinephrine function also seems abnormal in that the number of α-2 receptors in the locus ceruleus is larger than normal. Moreover, the density of dopamine D2 receptors is greater both in dogs and in humans with narcolepsy. Some of the selective serotonin reuptake inhibitors reduce cataplexy in dogs and humans, implicating serotonergic systems at least in cataplexy.   
A group of researchers at Stanford University led by Emmanuel Mignot, MD, PhD associate professor of psychiatry at Stanford University School of Medicine, used a technique called positional cloning to pinpoint the “narcolepsy gene” in dogs. In the August 6 issue of Cell (*Mignot, E., et al. "The Sleep Disorder Canine Narcolepsy Is Caused by a Mutation in the Hypocretin (Orexin) Receptor 2 Gene." Cell, August 6, 1999.), Mignot and his colleagues report locating two defective versions of the narcolepsy gene, one in Doberman pinschers, the other in Labrador retrievers. The gene, known as hypocretin receptor 2, codes for a protein that juts out from the surface of brain cells and that functions as an antenna, allowing the cell to receive messages - transmitted via small molecules called hypocretins - from other cells. The defective versions of the gene encode proteins that cannot recognize these messages, in effect cutting the cell off from essential directives, including perhaps messages that promote wakefulness.
Mignot predicts the finding will not only help the roughly 135,000 Americans who suffer from narcolepsy, but in time it will shed light on two of the biggest mysteries in sleep research: how and why we sleep.

===Fluorescent Proteins (e.g. GFP)===
Green fluorescent protein, isolated from the jellyfish ''Aequorea victoria'', fluoresces upon exposure to blue light.  The structure was solved by James Remington and colleagues - the fluorescent group is contained within the B-barrel.  Fluorescent proteins are available in many colors and are most often used as reporters of protein expression to better understand cellular signaling.

===FRET Imaging===

Fluorescence Resonance Energy Transfer (FRET) is the radiationless transfer of energy between two fluorescent proteins.  The fluorescent donor is excited at a specific wavelength.  This energy can then be transferred to the fluorescent acceptor through a dipole-dipole coupling mechanism.  FRET imaging can be used to determine protein-protein interactions, protein-DNA interactions, and protein conformational changes.  

For example, one part of a protein is tagged with CFP and another part is tagged with YFP.  When the protein is in a certain conformation in which the two fluorophores are far apart, the CFP will be excited but will not transfer its energy to the YFP.  The assay would result in the visualization of the CFP wavelength.  If a conformational change takes place and allows the CFP and YFP to come close together, the energy transfer will take place.  In this case, the assay would result in the visualization of YFP wavelength.

One limitation of FRET is the inherent background noise that results from the direct excitation of the acceptor fluorescent protein.  To avoid this problem Bioluminescence Resonance Energy Transfer (BRET) is used in which a bioluminescent luciferase is used instead of CFP as the fluorescent donor.  Another solution to determine protein-protein interactions is BiFC.  This technique attaches one half of the YFP molecule to one protein and the other half to another protein.  When the two halves come together, the complete YFP is now functional.

===CaM Kinase II===
CaM Kinase II is a calmodulin-dependent protein kinase that shows history-dependent activity, remembering previous calcium pulses through autophosphorylation. CaM kinase II forms a complex with 12 subunits, arranged in two hexamer rings. It is activated by calcium-bound calmodulin, which relieves the autoinhibitory interaction between CaM kinase subunits. Once activated, CaM kinase II proceeds to phosphorylate itself and remains partially active even after the lowering of calcium levels, thereby prolonging the duration of its kinase activity. Consequently, activated CaM kinase II responds non-linearly to calcium oscillations, as (in the absence of calcium) its activity falls more slowly the more it is phosphorylated.

See [[BIO254:CaMKII]]

==Lecture 7 - No New Model Systems!==

==Lecture 7 Techniques==



===Calcium channel inhibitors===
===Calcium imaging===
Calcium imaging is a scientific technique designed to reflect the calcium status of a particular tissue or medium.  In calcium imaging a substance called Fura is used to bind to calcium.  When Fura binds to calcium after being exposed to fluorescent light, it fluoresces.  The Fura-Ca complex affects the wavelength typically associated with unbound Fura and the resulting fluorescence can be detected by a camera adapted (usually through a microscope) for fluorescent light imaging. A computer-generated image is thus created which can be analyzed according to intensity, which reflects calcium status in the given medium or tissue.

==Lecture 8==

===Eye Anatomy===
The eye is a light-sensitive organ through which visual information about the external world is transmitted to the brain. The pupil, a black, circular opening centered in the front of the eyeball, controls the amount of light entering the eye, widening when surroundings are dark and constricting when they are bright. The lens of the eye focuses light from the external world onto the retina, a thin, multilayered region composed of photoreceptors (rods and cones) and interneurons that lines the back of the eyeball. The fovea, located in the retina, is the focus point of the lens and contains the highest density of photoreceptors (only cones), making it responsible for high-acuity vision. Retinal axons leave the eye through the optic disc, an area also known as the “blind spot” for its lack of photoreceptors, and merge into the optic nerve, which transmits visual information to the brain. The photoreceptor cells in the eye are categorized into rods and cones.  The rods expresses rhodopsin as their photoreceptor molecule while cones express red, green, or blue opsins.  Traditionally, each cone cell is considered to express only a single type of opsin.  This theory is largely supported by morphological evidence in many primates and adult humans. Having more than one opsin molecule per cone suggests that each cone will respond to multiple wavelength of light, which seems illogical from the wiring point of view.  However, recent stud-ies have found many mammalian examples that reject this one-opsin-one-cone theory. Such co-expression of multiple opsin photoreceptors was first discovered in mice. The mouse retina expresses the M-opsin and S-opsin in two gradients with the M-opsin forming a dorsal-ventral increasing gradient and S-opsin in a ventral-dorsal increasing gradient.  There is a substancial region of overlap in the gradient where M/S-opsins are both expressed in the mouse retina.  Such co-expression of more than one opsin in a single cone is also displayed in human retinal development.

===Isolated Retinal Rod Cell===

===Phototransduction===
See [[BIO254:Phototransduction]]

===Adaptation===
See [[BIO254:Adaptation]]

==Lecture 8 Model Systems==

===Vertebrate eye===
===Isolated retinal rod cell===

==Lecture 8 Techniques==

===Exciting specific areas of the retina===
===Shearing rod cells===
===Sucrose density gradient===

==Lecture 9 Model Systems==

===Cat retina===

==Lecture 9 Techniques==

===Multielectrode array===
Multi-electrode arrays are used to stimulate and record extracellular electrical activity in many recording sites. This multi-unit approach offers the advantage of simultaneously recording tens or even hundreds of neurons at the same time; one can choose to analyze the activity of a single cell or any combination of cells. The technology can be used to record electrophysiological data both in vivo and in vitro.
==Lecture 10 Model Systems==

===Toads===
The toad’s stereotyped prey capture response was studied in order to better understand how releasers, features of a stimulus that activate a fixed action pattern, are detected. For the toad, the fixed action pattern was orienting its head towards potential prey and the releaser was a cardboard cutout that resembled a worm. Three different types of stimuli were placed in front of the frog and moved in a horizontal plane. The toad’s orienting response for each stimulus was measured. When a rectangular cardboard stimulus was moved across the toad’s visual field in the direction of its long axis, the so-called “worm” configuration, a strong orienting response was elicited. When a rectangular cardboard stimulus was moved across the toad’s visual field in the direction of its short axis, the so-called “anti-worm” configuration, no response was elicited. When the stimulus was a square piece of cardboard, the toad moved toward it if the square was small and moved away from it if the square was large, the shift in orientation taking place at the point where the square stopped being viewed as prey by the toad and started being viewed as a predator. A type of tectal neuron called TH5(2) which demonstrated frequent impulses in response to the worm configuration, infrequent impulses in response to the anti-worm configuration, and impulses of decreasing frequency in response to square stimuli of increasing size is a strong candidate for the feature detector.

==Lecture 10 Techniques==

===Infrared camera===
===High speed monitor for stimulation===
===Gal4 inhibition of neurotransmission===
The Gal4/UAS system is used for targeted gene expression in Drosophila as well as other model organisms.  Gal4 is a transcription factor that does not activate native Drosophila genes but activates the expression of genes that are under the control of UAS.  Gal4 can be used neurobiologically to selectively inhibit the neurotransmission of a subset of neurons. To do this, a Gal4-encoding gene is placed between some gene, present in a certain subset of neurons, and the promoter for this gene. In this way, whenever the gene is expressed, Gal4 is expressed. Next, UAS is placed next to a transgene that is incorporated into the neuronal DNA and that codes for the shibire protein. Gal4 will bind to UAS, activating the expression of the transgene’s shibire protein product. When exposed to temperatures higher than a permissive temperature, the shibire protein is disrupted and results in the blockage of synaptic signaling. Thus by manipulating the temperature a subset of neurons can be turned on and off.
<br>

==Paper 3 Model Systems==

===Drosophila eye===

==Paper 3 Techniques==

===FLP/FRT system===
FLP/FRT Recombination is a directed recombination technology involving the recombination of sequences between FRT sites by the flippase recombination enzyme derived from Saccharomyces cerevisiae. This allows the precise manipulation of an organism's DNA under controlled conditions in vivo.

FLP/FRT is a technique for inducing mosaics of +/+ and -/- (mutant, or carrying an allele of interest) cells in heterozygote organisms.  This technique is particularly useful when dealing with traits that are necessary for development in the entire organism thus preventing the use of a homozygous organism.  The FLP/FRT system enhances the rate of mitotic recombination, which refers to crossing over in mitosis rather than meiosis, and is ordinarily very rare.  By expressing an FRT region nearer to the centromere than the gene of interest, the chromosomes may cross over at this point, so that each chromosome will have a chromatid with the + allele and one with the - allele, instead of having identical chromatids as usual in mitosis.  The two chromosomes will separate independently in mitosis, so there is a 50% probability that one daughter cell will have both - alleles and the other will have both + alleles.  The crossing over at the FRT sites is mediated by FLP, flippase recombination enzyme derived from Saccharomyces cerevisiae.  Depending on the experimental design, it may be expressed in a heat-inducible form or regulated by a tissue-specific enhancer.

===MARCM system===
In Mosaic Analysis, it is sometimes inconvenient to use negatively marked clones, especially when generating very small patches of cells, where it is more difficult to see a dark spot on a bright background than vice versa. It is possible to create positively marked clones using the so called MARCM system, which stands for "Mosaic Analysis with a Repressible Cell Marker". This technique was developed by Liqun Luo, a professor at Stanford University. In this system the GAL4/UAS system is used to globally express GFP. However, the gene GAL80 is used to repress the action of GAL4, preventing the expression of GFP. Instead of using GFP to mark the wild type chromosome, GAL80 serves this purpose, so that when it is removed, GAL4 is allowed to function, and GFP turns on. This results in the cells of interest being marked brightly in a dark background.

===GAL4-UAS system===
The GAL4-UAS system allows for targeted gene expression in vertebrates using the transcription factor GAL4 and the gene of interest attached to a promoter specific to GAL4, the Upstream Activating Sequence (UAS).  The system was originally discovered in the yeast <i>Saccharomyces cerevisiae</i>.

The system works by creating two transgenic strains of the organism and crossing them to yield progeny with both transgenes.  In one strain, an activator line is introduced by inserting the gene for GAL4 near a specific promoter with a known expression pattern (e.g., a promoter used in the development of <i>Drosophila melanogaster</i> legs), while, in the other strain, an effector line is created by fusing the UAS upstream of the gene of interest.  When the two lines are crossed, the progeny with both transgenes will express the gene of interest according to the expression profile of the promoter near which the GAL4 gene was placed.  It is by this control over the insertion of the GAL4 gene that targeted expression (e.g., ectopic expression or expression of mutated genes) can be achieved.

===Heat shock===
Heat shock proteins (HSP) are a part of the cell's internal repair mechanism. They are also called stress-proteins. They respond to heat, cold and oxygen deprivation by activating several cascade pathways. applying a heat shock means subjecting cells to a higher temperature than the ideal body temperature of the organism from which the cell line was derived.
Heat shock is a method in which genes can be introduced into a vector.

===Confocal imaging===

Confocal imaging is a microscopy technique used to obtain high resolution images and 3-D reconstructions.  

In confocal microscopy, one or more beams of light are focused by an objective lens on to a small volume of a fluorescent specimen.  The fluorescent light emitted from this small area is separated from the source laser light by a beam splitter.  The fluorescence is detected by computer detection device.  This process to obtain the information for a single pixel is repeated many times as the laser is scanned over a single plane.  After scanning several layers of a specimen, the computer reconstructs a 3-D image by stacking the 2-D planes together.

Confocal imaging has several advantages.  Confocal microscopy has better resolution than other microscopy techniques.  It also allows for images of thick, living specimens to be obtained with minimal preparation.  Note: Fluorescence is required for this technique.  This can be achieved by using fluorescent dyes or by transgenically expressing fluorescent proteins such as GFP.

For some sample images, click here [http://www.microscopyu.com/galleries/confocal/index.html].

===Frozen sectioning===

In Discussion Paper 3 (Wernet et al, 2006), mid-pupal and adult cryosectioned (frozen sectioned) retinas were immuno-stained for the various rhodopsin types. Frozen sections are often used in immunohistochemistry in order to preserve certain cell antigens that do not survive routine tissue fixation and/or paraffin embedding. However, the disadvantages of frozen sections include: poor morphology, poor resolution at higher magnifications, special storage requirements, limited retrospective studies, and increased cutting difficulty over paraffin sections.

There are two primary methods to sectioning frozen brains. The microtome is used for sectioning frozen, fixed brains. For this procedure, the fixed brains must be cryoprotected by infiltration of sucrose to prevent freezing artifacts during sectioning. Sucrose-infiltrated brains are frozen and maintained frozen during sectioning. 	Sections are collected off the microtome knife and placed in a buffered saline solution during sectioning. 

Wernet et al. used 10 micrometer horizontal eye sections, sectioned using a cryostat. A cryostat is a microtome housed within a freezing chamber that allows the sectioning process to be performed at a temperature of -20 to -30 degrees Celsius. The cryostat is required for sectioning fresh-frozen brains, as the unfixed brain sections must be maintained in a frozen state until they are affixed to a microscope slide; however, cryostat sectioning may also be used for perfusion-fixed brains. Cryostat sections retain morphological integrity after being sections and can be affixed directly onto microscopic slides for immunohistochemistry use.

===Water immersion microscopy===
In this modified form of microscopy, water is placed between the front lens element and the specimen so as to increase the numerical aperture of the microscope system. Water immersion objectives allow for high-resolution imaging through aqueous layers on the order of 200-micrometers thickness. As with oil immersion microscopy, the main advantage of water immersion objectives is improved imaging capabilities in thick sections of biological specimens. However, water has distinct advantages over oil. It has no inherent fluorescence to complicate image interpretation, it’s very cheap, there is almost no risk of contaminating the specimen, and there is no special cleanup method required.
===Plastic sectioning===

==Lecture 11 Model Systems==

===Mouse gustatory system===
===Mouse olfactory system===

==Lecture 11 Techniques==

===Two-bottle preference test===
===Calcium imaging===
Calcium imaging is a scientific technique designed to reflect the calcium status of a particular tissue or medium. In calcium imaging a substance called Fura is used to bind to calcium. When Fura binds to calcium after being exposed to fluorescent light, it fluoresces. The Fura-Ca complex affects the wavelength typically associated with unbound Fura and the resulting fluorescence can be detected by a camera adapted (usually through a microscope) for fluorescent light imaging. A computer-generated image is thus created which can be analyzed according to intensity, which reflects calcium status in the given medium or tissue.

===G15===
G15 is able to bind to 7 transmembrane proteins so that they can be easily labeled.

===Fura-2===
Fura-2 is a ratiometric fluorescent dye which binds to free intracellular calcium. Fura-2 is excited at 340 nm and 380 nm of light, and the ratio of the emissions at those wavelengths is directly correlated to the amount of intracellular calcium. The use of the ratio allows freedom from a myriad of confounding factors, such as ambient light, making Fura-2 one of the most preferred tools to quantify calcium levels.

===Ace K===
Acesulfame potassium is a calorie-free artificial sweetener, also known as Acesulfame K or Ace K, and marketed under the trade names Sunett and Sweet One.

Chemically, acesulfame potassium is the potassium salt of 6-methyl-1,2,3- oxathiazine-4(3H)-one 2,2-dioxide. It is white crystalline powder with molecular formula of C4H4KNO4S and molecular weight of 201.24.

Acesulfame K is 180-200 times sweeter than sucrose (table sugar), as sweet as aspartame, about half as sweet as saccharin, and one-quarter the sweetness of sucralose. Like saccharin, it has a slightly bitter aftertaste, especially at high concentrations.

The studies that purport to show safety have been challenged by a number of individuals and organizations, most notably the Center for Science in the Public Interest in the USA. They claim that the existing studies are inadequate (despite being peer-reviewed), that there are flaws in the research protocols, dosing, and time length of the studies, and that as a result the carcinogenicity of acesulfame K may not be properly understood. In particular they note that there have not been long-term human studies, so they doubt the studies which show that acesulfame is rapidly absorbed and then excreted unchanged (i.e., not metabolized by the human body) are representative of the long-term. Currently, the scientific community's official position is that acesulfame K is safe to consume, which is the view put forth on the sweetener industry's public relations website, IFIC.

===Inosinate===
Inosinate increases the effect of L-glutamate in the gustatory system.
===Cycloheximide===

Cycloheximide is chemical that inhibits protein synthesis in eukaryotic cells.  In is made by the bacteria Streptomyces griseus, and functions by interfering with peptidyl transferase in the 60S ribosome.  When this chemical is added, transcriptional elongation is arrested, and protein synthesis stops. Cycloheximide is a commonly used reagent in biological research to inhibit protein synthesis in in vitro tissue culture experiments. The effects of the agent are rapid and can be reversed simply by replacing the cell culture media.
===Diphtheria toxin (DTA)===
Diphtheria toxin (DTA) is an exotoxin secreted by Corynebacterium diphtheriae, the pathogen bacterium that causes diphtheria.
DTA is a single polypeptide chain of 535 amino acids consisting of two subunits linked by disulfide bridges. Binding to the cell surface of the less stable of these two subunits allows the more stable part of the protein to penetrate the host cell. It catalyzes the ADP-ribosylation and inactivates the eucaryotic elongation factor-2 (eEF2). It does so by ADP-ribosylating the unusual aminoacid diphtamide. In this way, it acts as a DNA translational inhibitor. The lethal dose for humans is about 0.1 μg/kg of pure protein.

===Enzyme digestion of PCR===

==Lecture 12 Model Systems==

===C. elegans olfactory system===
According to lecture, C. Elegans olfaction consists of three sets of neurons: AWA, AWB, and AWC. AWA and AWC control attraction while AWB regulates repulsion. An interesting finding is that the C. Elegans olfactory system is spatially encoded; it doesn't matter what the signal is, it matters which neuron fires. For example, if an attractant receptor is expressed in the repulsive or AWB neurons, the animal will be repelled by the attractant. 

==Lecture 12 Techniques==

===Intrinsic imaging===
===Gas chromatography-electrophysiology===
===Trans-neuronal labeling===
===G-CaMP===
G-CaMP is a Ca2+ probe based on a single green fluorescent protein (GFP). G-CaMP shows a large fluorescence increase upon Ca2+ binding, but its fluorescence is dim and pH sensitive, similar to other single GFP-based probes.

===Drosophila brain warping===

Every Drosophila brain is different. Thus, deriving information from many fly brains may be difficult. Averaging can produce a fairly fuzzy image, so researchers developed a technique in which they use Drosophila brain warping--also used in human fMRI studies--to form a clearer picture. It is a method whereby many images of many fly brains are "warped" to adhere to a more coherent image. The method used is called translation rotation scaling, or non-linear warping. Researchers can thus "register" many different brains onto a preselected standard brain. This method has been used in studying projection neuron axon pathways. The method allows investigators to compare structures across different individuals of the same species.

===IRES===
An internal ribosome entry site (IRES) is a nucleotide sequence that allows for translation initiation in the middle of a messenger RNA (mRNA) sequence. Normally, in eukaryotes, translation can only be initiated at the 5' end of the mRNA molecule, since 5' cap recognition is required for the assembly of the initiation complex. IRES mimics the 5' cap structure, and is recognized by the 43S pre-initiation complex (the 40S ribosomal subunit plus eIF1A, eIF3, and eIF2-GTP-bound to the initiator tRNA, required for translation). IRES are located in the 5’ un-translated region (UTR) of RNA viruses and allow translation of the RNAs in a cap-independent manner. Some mammalian mRNAs have also been reported to have IRES, although their existence is still controversial. One hypothesis is that the IRES function in eukaryotic mRNAs as housekeeping genes involved in cellular survival. 

When an IRES segment is located in between two reporter genes in eukaryotic mRNA molecules as bi-cistronic mRNA, it can drive translation of the downstream protein coding region independently of the 5'-cap structure bound to the 5' end of the mRNA molecule. As such, both proteins are produced in the cell. The first reporter protein located in the first cistron is synthesized by the cap-dependent initiation approach while translation initiation of the second protein is directed by the IRES segment located in the inter-cistronic spacer region between the two reporter protein coding regions. However, this so-called “method of bi-cistronic constructs” has several bottlenecks when used in vivo and can often be deceitful. 

In Lecture 12, Professor Luo presented data from Mombaerts et al. (1996), which used IRES to enable the translation of the P2 receptor (an olfactory receptor gene which is expressed in a restricted subpopulation of olfactory sensory neurons) and the tau-lacZ fusion protein (the protein tau functions in microtubule binding, while lacZ is a common reporter gene) from one mRNA. Their strategy allowed them to visualize axons from olfactory sensory neurons expressing a given odorant receptor as they project to the olfactory bulb. The experiment showed that neurons expressing a given receptor, and therefore responsive to a given odorant, project with precision to 2 of the 1800 glomeruli within the olfactory bulb.

==Lecture 13 Model Systems==

===Electric fish===
Its additional sensory capability is to generate an electric field. This represents an adaptation to living in murky water. For instance, when a potential prey animal enters the fish's electric field, it will create a disturbance and alert the fish as to its proximity.
===Desert ant===
Its additional sensory capability is its ability to figure out its relative position to the sun, as it possesses polarized light detectors in its ommatidia.
===Bat===
Its additional sensory capability is echolocation-ultrasonic imaging of its environment.
===Pigeon===
Its additional sensory capability is detection of magnetism; this capability informs the pigeons' flight direction.
===Pit viper===
Its additional sensory capability is the ability to detect heat waves, using its pit organs.
===Star nosed mole===
The star nosed mole is a blind mole with no eyes and an elaborated star derived from whiskers found in the Eastern United States only in swamps.  Kenneth Catania is a pioneer that examines thes alternate systems.  The star nosed mole picks up food in complete darkness with a directed set of movements (video from Lecture 13 shown by Professor Clandinin).  In real time, the movements are remarkably fast and demonstrate evolution of sensory structures to quickly puruse prey in a specialized way.  The behavioral strategy of the star nosed mole is to first make contact to the star with a directed movement over the prey item.  The saccade, or sensory fovea, then decides whether the prey is worth eating or not.  Catania has studied how the underlying brain structure of the star nosed mole has changed, in particular showing that the somatosensory cortex of the star nosed mole is built around a star.  The sensory periphery has the characteristic structure of the star where star number 11 is the sensory fovea.  There is a densely packed array of nasal mechanoreceptors that serve as a sensitive touch organ.  In the cortex, barrels correspond one-to-one to the stars on the nose.  Number 11 is overrepresented in the cortex as well.  The star nosed mole serves as an exmaple of balancing cost and reward.  Profitability is equal to energy/handling time.  The standard mole has large prey and a large handling time (low cost, low payoff), whereas the star nosed mole has small food items and a low handling time (high cost, high payoff).

===Euglena===
Euglena is a photosynthetic protist with eyespots, demonstrating photosensitivity within unicellular eukaryotes.  The eyespot is contained within a chloroplast where there is a stack of photoreceptor pigments.  Rhodopsin encodes a 7-transmembrane protein that is fused directly to an ion channel.

===Cubozoan jellyfish===
Otherwise known as the sea wasp, the cubozoan jellyfish has very advanced optics, and has eye 'master regulatory genes.' Its eyes are set up to be eyes on stalks. It has a small lens eye, a large lens eye, and four small eye spots, all joined to a simple nervous system.
===Yeast===
The studies of yeast give us some clues about the prevalence of G protein cascades in the majority of organisms. Yeast have two sexes, alpha and beta, which are each related to their own unique pheromones. The mating of alpha and beta is regulated by a G protein cascade that triggers the joining of two yeast partners of opposite pheromones.

==Lecture 13 Techniques==

===Fossil record===
Analyzing the fossil record is one way to study evolution that is useful for learning about the evolution of skeletal structure, in particular.  The fossil record, however, proves to be less useful when studying sensory systems whose components are poorly preserved.

===Comparing anatomy===
===Comparing DNA and protein sequences===
===Molecular resurrection===
===Selective breeding===
===Long-term field studies===
Long-term field studies of emerging species can provide greater understanding of variation in species.  For example, more than 30 years of studying Darwin's finches provides systematic analysis of how changes in the Galapagos islands relates to changes in beak structure.

==Lecture 14 Model Systems==
===Drosophila eclosion===
Eclosion is the emergence of fruit flies from their pupas. Even when pupas are moved into complete darkness, their emergence happens at a fairly consistent point in time, at 'dawn'. This indicates that there is an internal rhythm within the fruit fly that is not affected by the constant darkness.

===Mouse locomotor activity===
Mouse locomotor activity, specifically running wheel activity, is useful for identifying circadian rhythm defects.  In Lecture 14, mouse running wheel activity was measured in constant dark (DD) for 60 days as an example of self-sustained circadian rhythm in the absence of environmental cues (Pittendrigh 1993).  The heavy bars of the actogram represent wheel utilization, while the period between activity onsets is slightly less than 24 hours.  Using a forward genetic screen, Takahashi and colleagues were able to identify a semi-dominant mutant Clock in a mouse with a period of 24 hours.  The homozygous mutant had a longer initial rhythm of 27 hours and lost rhythmicity completely after shifting to DD.

===Golden hamster===
The golden hamster is one of the most frequently used laboratory animals in chronobiological studies mainly because of its very strong and predictable rhythms. A spontaneous mutation in a Golden hamster was found to result in a significant decrease in the period of its circadian rhythm. The mutation occurs at a single gene locus called tau. Whereas a normal wild type hamster has a 24 hour period, a hamster heterozygous for the tau mutation has a 22 hour period, and a hamster homozygous for the tau mutation has a 20 hour period. When the tau mutant was discovered in 1988 by Ralph and Menaker, it was the first single gene mutation that causes an effect on circadian rhythm periodicity  to be described in a mammalian species. Recently it was found that tau is an allele of the gene casein kinase I epsilon in the golden hamster. The tau mutation provided the opportunity to further test the hypothesis that SCN is the circadian pacemaker. When SCNs were transplanted between wild type and tau mutant hamsters the period of the host's circadian rhythm was always the same as the period of the donor's transplanted SCN. Thus a tau mutant with a wild-type SCN graft showed a 24 hour rhythm while a wild type hamster with a mutant SCN graft showed a 20 hour rhythm. From these findings it was concluded that the SCN contains all the necessary information for the proper functioning of circadian rhythms.

==Lecture 14 Techniques==
===Yeast two-hybrid===
The yeast two-hybrid system is commonly used to study protein-protein interactions between two proteins. This system takes advantage of the properties of the GAL4 protein of the yeast <i>Saccharomyces cerevisiae</i>. The GAL4 protein is a transcriptional activator that is required for the expression of genes encoding enzymes of galactose utilization. It consists of two separable and functionally essential domains: an N-terminal domain which binds to specific DNA sequences (UAS<sub>G</sub> for upstream activating sequence of GAL4); and a C-terminal domain containing acidic regions necessary to activate gene transcription. In other words, GAL4 protein consists of two domains: GAL4 DNA binding domain (GAL4-DBD) and GAL4 activating domain (GAL4-AD).

In the two-hybrid system, you create two hybrid proteins: protein X bound to GAL4-DBD (otherwise known as the “bait” fusion, containing the protein of interest) and protein Y bound to GAL4-AD (the “prey” fusion, containing the potential interacting partner of the protein of interest). These fusion proteins are created in plasmid vectors and transfected into a host yeast cell. If protein X interacts with protein Y, the binding of these two will reconstitute the proximity of the GAL4 domains, forming an intact and functional transcriptional activator. This newly formed transcriptional activator will bind to the UAS<sub>G</sub>, which regulates a reporter gene, a gene whose protein product can be easily detected and measured (e.g., LacZ, luciferase, GFP). In this way, the amount of the reporter produced can be used as a measure of interaction between the protein of interest and its potential partner (Fields and Song, 1989).

In Lecture 14 on circadian rhythms, Professor Luo presented results that showed that mClock does not form dimers with itself, instead binding to a second bHLH-PAS domain protein, bMAL, which was identified using a yeast two-hybrid system (slide 20). Furthermore, a yeast two-hybrid assay was also used to determine that dClock binds to the <i>Drosophila</i> homolog of bMAL, Cycle.
===ZT===
ZT stands for Zeitberger time. The number represented after the ZT indicates the hours since dawn.

===Serum shock===
Serum shock is the introduction of large amounts of foreign serum into cells. Injecting fibroblasts with high levels of serum can cause a fluctuation in protein levels that follows a circadian-like rhythm that can persist for many days.
===DNA microarray===
DNA microarray is a method for profiling gene expression levels for thousands of genes simultaneously. A DNA chip is made by designing florescent probes that match parts of the sequence of known mRNA. After inserting each desired mRNA reporter into the organism, microscopic DNA spots are attached to the solid chip surface. Each spot fluoresces according to the level of reporter binding to mRNA. To compare expression levels between two conditions (e.g. comparing gene expression between clock mutant and wildtype) it is necessary to create two chips, one for each strain. Observing which reporters decrease in expression with the mutant DNA chip will reveal which genes are affected by the loss of Clock function. (see [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=11733057&query_hl=7&itool=pubmed_docsum McDonald and Rosbash 2001]).

From Lecture 15, Slides 37 and 38:
Because biotech companies are capable of making DNA chips that contain the entire 13,500 genes in Drosophila, it is possible to do genome wide analyses to identify which genes undergo circadian cycle. Experimenters can cluster genes according to their phases (high day expression, high night expression, etc.), and identify which genes are direct or indirect targets of central clock regulators (such as the Clock or bMAL). Microarray analysis in flies has already revealed that circadian genes fall into several large groups of proteins, including detoxification enzymes, ligand binding/carrier, transporter, immune/host defense genes and neuropeptide modulaters.

===Tissue graft===

Tissue grafting involves surgically transplanting tissue from one organism to another, or from one part of an organism to another part. The tissue is transplanted without a blood supply and must thus obtain blood from the new vascular bed to survive. Grafting is often used in medical procedures to replace damaged tissue, ranging from skin to bone. Grafting is also used in biological research to see the effects of foreign tissue on the host organism. There are three specific categories of grafts, xenografs, allografts and isografts. A xenograft involves the transplanation of tissue from one species to another, an allograft involves two heterogenic members of the same species and an isograft involves two isogenic members of the same species (ie. twins or clones). 

In slide 15 of lecture 14 Professor Luo mentions a study done by Ralph et al. in 1990 in which grafting of the suprachiasmatic nucleus (SCN) was used to see whether the genotype of the SCN alone was sufficient to produce rhythmicity in a golden hamster. This procedure was an allograft since two different genotypes of the animal were used. One was the wildtype (~24 hour rhythmicity) and one was a homozygous tau mutant which exhibited a shorter ~20 hour rhythmicity. When the SCN of the wildtype was ablated and replaced with a mutant SCN, the circadian rhythm changed to the mutant phenotype of 20 hours. When the SCN of the mutant was ablated and replaced with a wildtype SCN, the circadian rhythm changed to the wildtype phenotype of 24 hours. Thus the study showed that the genotype of the SCN determines the rhythmicity of the animal. 

Note: Slide 15 mentions heterotopic grafting, which may be an error since heterotopic grafting involves transplanting tissue into a region of the organism in which it did not originally belong (eg. putting the ear of one organism on the foot of another). However, Ralph et al. 1990 placed the foreign SCN in the same position of the brain which originally held the host SCN before it was ablated. Thus this seems to be an example of an isotopic graft.

===ENU===
ENU is also known as N-ethyl-N-nitrosourea (chemical formula C3H6N3O2) and is a very potent and powerful mutagen (Russell et al., 1979). It is used in mutagenesis screens to identify mouse behavioral mutants using forward genetic approaches (mutagenizing genome first, then observing resulting phenotype). On a molecular level, the chemical works by transferring the ethyl group of ENU to nucleobases (usally thymine) in nuclei acids.

ENU is often used to study circadian behavior in mice. Male mice are treated with the highest dose of ENU that can be tolerated without causing infertility. The primary target of ENU in the germline of mice is the spermatogonia, which can be highly mutagenized and go on to produce mutant gametes (ENU induces about 1 mutation in every 700 gametes). Mutant mice with germ lines that display mutant circadian phenotypes are then bred and gene function is explored.

[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=15817290&query_hl=10&itool=pubmed_docsum Siepka & Takahashi 2005]
===Phase response curve===
Plotting phase response curve is a way to reveal the relationship between a circadian phase shifting drug or treatment, and its effect on circadian timing. A standard PRC describes changes of the phase of the circadian rhythm in response to a brief pulse of light. A variety of agents aside from light are also capable of phase shifting the mammalian clock in the SCN (e.g. 5-HT agonists, histamine, melatonin, glutamate, neuropeptide Y, vasoactive intestinal polypeptide (VIP), gastrin-releasing peptide (GRP)). Testing how various treatments effect the circadian clock at different moments of the subjective day or night give insight into the nature of the clock mechanism. An example is the gated nature of the photic effect on circadian timing in all organisms studied (Lecture 14, Slide 25). During the subjective day, a pulse of light has no significant change to the phase. In the first half of the subjective night, a pulse of light results in a phase delay, or shift back to the beginning of the night. In the second half of the subjective night, a pulse of light results in a phase advance, or shift to the beginning of the morning. 
===Sucrose gradient===

==Lecture 15 Model Systems==

===Drosophila courtship behavior===

The courtship behavior of Drosophila is an interesting system for genetic study because, like all sexual behavior, it is  innate, making it a convenient system from which to obtain links between genotype and behavior. Visual, olfactory, gustatory, and auditory sensory cues produce a robust and stereotyped courtship behavior in Drosophila males. Wildtype males consistently orient themselves near the female, "tap" them and sing to them before attempting copulation. Studies of this behavior have investigated the strong role of the fruitless gene, which encodes for a set of male-specific transcription factors Fru^M in males whose action controls their sexual behavior (Manoli et al. 2005).

===Voles===

Different species of voles exhibit robust stereotyped sexual behavior, much like the courtship rituals of Drosophila males. Specifically, prairie voles are found to spend more time huddling after mating and cohabitating with a sexual mate than meadow voles, which tend to exhibit more independence. This pair bonding behavior is attributed to the roles of neuropeptides oxytocin and vasopressin. Oxytocin is exhibited in the female (it also is responsible for mother/infant bonding) and vasopressin is exhibited in the male (vasopressin is also associated with male social responses such as aggression, scent marking, and courtship). The neuropeptides are released after mating, and can induce pair-bonding even when sexual activity is not present. The bonding is thought to be due to the intersection of social discrimination circuits from the olfactory pathway to the oxytocin/vasopressin and dopamine reward systems. Prairie voles, consequently, possess more vasopressin V1a receptor than meadow voles in the ventral pallidum of the ventral forebrain.

==Lecture 15 Techniques==

===Retrograde pseudorabies virus===
Pseudorabies can be used as a transneuronal marker that infects in the retrograde direction: it infects and enters the presynaptic neuron. As a result, it can be used to study the connections between neurons.

===Recording in behaving mice===

==Lecture 16 Model Systems==

===Human epilepsy patients===

Case studies of human patients with extreme forms of epilepsy that have been treated with lesions or removal of parts of the brain have been very important in defining the neuroanatomy of human memory. Perhaps the most well known of these patients is H.M. His epilepsy was so disruptive that surgery to remove both his medial temporal lobes, the locus of his seizures, was required. The results of this surgery produced specific disruptions in his memory. He retained normal short term memory and perfect long term memory for events that ocurred prior to the surgery, but experienced anterograde amnesia, unable to retain memory for events that ocurred after the surgery. Interestingly, though, he retained his implicit memory, and could improve his skill in motor tasks, even if he did not remember practicing the tasks in the past.
Another famous epilepsy patient was R.B., who underwent a bilateral hippocampal lesion and consequently also experienced anterograde amnesia. Patients like R.B. and H.M. have allowed investigators to establish the medial temporal lobes, which include the hipposcampus, are necessary in the processes of encoding memory.

===Xenopus tadpole retinal ganglion cell – tectum synapses===
[http://www.nature.com/nature/journal/v395/n6697/abs/395037a0.html Zhang et al 1998] used the developing tadpole RGC-tectum synapses as a model system to study spike timing dependent synaptic plasticity (see Lecture 15, Slides 12-15). The Xenopus tadpole is a good model system for manipulating synaptic plasticity because it has fewer total neurons, neurons that are easier to patch clamp, and because synapses have not yet been pruned in the young brain.

Zhang et al 1998 were thus able to manipulate synapse strength by inserting two electrodes into two individual RGC neurons that were both connected to a single postsynaptic cell in the tectum, which was patch clamped in order to measure membrane potential following temporal manipulation of electrical firing from the two presynaptic RGCs. The results of the paper resulted in a modification of Hebb’s rule of synaptic plasticity: both the initial synaptic strength and the temporal order of activation of presynaptic neurons are critical for synaptic interactions among convergent synaptic inputs during activity-dependent refinement of developing neural networks ([http://www.nature.com/nature/journal/v395/n6697/abs/395037a0.html Zhang et al 1998]). Specifically, they found that firing right before a postsynaptic spike was rewarded with increased robustness of synaptic strenth, while arriving immediately after spikes was punished and arriving too far or too late had little effect in modifying synaptic strength. 

===Mouse barrel cortex===
The barrel cortex is the part of the somatosensory cortex of rodents where sensory inputs from the whiskers in the contralateral side of the body are represented. Inputs carrying information from a given whisker terminate in discrete areas of layer IV forming anatomically distinguishable areas called barrels.

===Drosophila mushroom bodies===
Mushroom bodies are known to be involved in learning and memory, particularly for smell. They are largest in the Hymenoptera, which are known to have particularly elaborate olfactory control over behaviour. In fruit flies, studies suggest that mushroom bodies have other learning and memory functions, like associative memory, sensory filtering, motor control, and place memory. The mushroom bodies are currently the subject of intense research. Because they are small compared to the brain structures of vertebrates, and yet many arthropods are capable of quite complex learning, it is hoped that investigations of the mushroom bodies will allow a clear view of the neurophysiology of animal cognition. The most recent research is also beginning to reveal the genetic control of processes within the mushroom bodies.

===Aplysia===

Aplysia, a type of sea slug, is a model organism for the study of learning and memory because of its simple nervous system and large neurons. It is relatively easy to perform electrophysiological recordings and molecular analyses, given the large amount of mRNAs found in a single Aplysia cell. Kandel et al. examined this animal and found that its learning behaviors are analogous to those of higher animals, and it can also form long-term memories. Kandel et al.'s most useful study involved studying habituation and non-associative learning in the Aplysia. The animal performs a simple reflex (retraction) when its siphon is touched that exhibits three main features of non-associative learning in vertebrates:
   1. Habituation: the retraction response of its siphon and gills progressively decreases in intensity after repeated stimulus.
   2. Dishabituation: the partial or complete restoration of an already habituated response following the presentation of another stimulus.
   3. Sensitization: an enhanced reflex response when a stimulus is paired with a novel, strong and noxious stimulus (a tail shock).
The Aplysia displayed such non-associative learning for time intervals between minutes and hours. With similar stimulus setups, the Aplysia also exhibited classical conditioning behaviors. The molecular mechanisms of this memory behavior was studied in the Aplysia's motor neurons and dopaminergic 5HT neurons.

===Honeybees===
Experiments coupling food/reward (sugar water) to certain colors show that honeybees can be trained to recognize color through classical conditioning. Initially, the sugar water is always placed with a certain color, so the honeybees go to the sugar water. But later, when the bees detect just the color, they will move toward it (expecting reward/sugar water). At a more subtle level, honeybees can be trained to even distinguish among variations of patterns. This suggests that honeybees may be capable of processing more abstract data/concepts, for instance sameness versus difference. When honeybees show recognition of the "correct" pattern, they in essence are recognizing the same pattern that previously was associated with food/reward. In these experiments, the visual patterning that is associated with reward can be manipulated to be more and more specific/subtle, for instance beginning with a solid black color, then going to black stripes, then testing for horizontal versus vertical stripe orientation, and so forth.
==Lecture 16 Techniques==

===Mirror trace===
The mirror trace task is used as a test of implicit memory in the form of skill learning.  The task involves tracing the outline of a figure while viewing only the mirror image of one's progress (the subject's hands and the actual figure are covered from view).  Normal subjects and amnesics with damage to the medial temporal lobe (such as patient H.M.) show improved speed and accuracy with practice on the mirror trace task, demonstrating that the medial temporal lobe is not required for this type of skill learning.  Diseases of the basal ganglia, such as Huntington's disease and Parkinson's disease, cause impairment on the mirror trace task and other tests of skill learning.

===Priming===
This technique helps to “jog” the memory by activating certain associations in the brain just prior to carrying out a task or action. These associations can work both unconsciously as well as consciously. In either case, they are thought to activate clusters of neurons, which makes it more likely that the memory will come into consciousness. Though HM had poor long term memory after surgery, he still could perform 
well in priming tasks. For example, if presented with a list of words, and asked to memorize them, he would not be able to remember them later. However, if you presented him later with the first three letters of each word he was supposed to remember, he was able to complete the words. He could recall ABSENT if presented with (ABS), INCOME when presented with (INC), etcetera.

===EM reconstruction===
===Two-photon imaging===
Two-photon excitation microscopy is a technique that allows imaging living tissue up to a depth of one millimeter. The concept of two-photon excitation is based on the idea that two photons of low energy can excite a fluorophore in a quantum event, resulting in the emission of fluorescence. The probability of the simultaneous absorption of two photons is extremely low. Therefore a high flux of excitation photons is typically required.

===Morris Water Maze===
Morris Water Maze is a behavioral procedure developed by Richard Morris in 1984 to test memory function.  Specifically, the maze tests hippocampal learning and spatial memory formation.  In one paradigm of the procedure, a mouse is placed into a small pool of water that contains a black and visible platform.  While mice can swim, they prefer not to and thus are always motivated to get onto the platform and rest.  Since the platform in this case is visible, the mouse can use it as a beacon and swim to it easily.  After several training sessions the platform is removed but the mouse still swims towards it.  This kind of learning does not involve the hippocampus.

In a second paradigm, the mouse is placed in a pool of water that contains a clear and thus invisible platform.  Several visual cues, such as different colored shapes, are placed around the pool in plain sight to the mouse.  When the platform is removed, the mouse is able to remember the position of the platform based on the external visual cues and swims to the usual position where the old platform was placed.  This kind of learning does require the hippocampus and spatial memory formation.

Using this model, scientists can explore the mechanisms of memory formation.  Lesions of the hippocampus resulted in decreased maze performance.  In addition, both NMDA receptor KO mice and those treated with APV, a NMDA receptor antagonist, showed reduced maze proficiency.  Based on observed results, scientists were able to eludicate the importance of the hippocampus and NMDA receptors in memory formation.       
===Multielectrode array===
===Classical conditioning===
Classical conditioning is an associative learning process where an animal learns to associate a conditioned stimulus with an unconditioned stimulus. The unconditioned stimulus (US) is something the animal naturally has a reaction to, such as food, or a shock. The conditioned stimulus (CS) is an innocuous or neutral stimulus that doesn't trigger any reaction on its own. An example of 
classical conditioning is the Pavlovian dog, that associates the bell (CS) with food (US), or the tone-shock training used on mice. Just as the dog learns to salivate when it hears the bell, the mice display a startle response when the tone is played, in anticipation of a noxious stimulus. The CS-US coupling has to be fairly tight, since if they are too temporally distant, the animal will not learn to associate the two. Also, the CS has to precede the US in order to trigger the association.


==Lecture 17 Model Systems==

===C. elegans vulva formation===
===Drosophila spermatogenesis===

==Lecture 17 Techniques==

===Cantab Paired Associates Learning===
Cambridge Neuropsychological Test Automated Battery Paired Assocation Learning is a type of visual memory test that characterizes episodic memory. (www.cantab.com). There are six boxes displayed around the edge of the screen. In random order, the boxes will display different, unique patterns. The patient is 
then shown the same patterns in a box in the center of the screen, and must recall where they saw each pattern (in which box).
===Human genetics===
The difficulty, obviously, is that we cannot study human genetics directly. We cannot manipulate who mates with whom. However, there are certain strategies for studying human genetics, particularly in relation to disease. Twin studies can suggest the relative role of genetic and environmental factors in inducing the onset age or severity of disease. For instance, a twins study suggested that genetic factors do not play a major role in causing sporadic PD that begins after age 50. Also, study of pedigrees is useful for tracing familial history of genetic diseases, for instance X-linked inheritance disorders. Additionally, transgenic animal models (mouse, fly) can be used to elucidate the molecular pathways that lead to diseases. 
===T-maze===
The T-Maze is used to measure rat learning. The rat is placed at the bottom of the "T," and it is allowed to choose either the right or left branch of the T. A reward is waiting at one of the branches. When this process is repeated over days, the rat (hopefully) learns and remembers to which side to run. In the Chapman article (1998) referenced in class, the rat's task was to learn to alternate branches to find the reward. That is, the sugar-water reward would consistently be located at the other arm than the rat chose previously.
===Reversal learning===
===Radial water-maze===
The radial water maze, or the radial arm water maze, is an adaptation of such classic learning apparati for rodents as the T maze and the dry radial arm maze.  The maze consists of a large, round basin of water with metal wall inserts that form lanes, or arms, from a central open area.  One arm is chosen as the target arm, in which an escape platform is placed that allows the mouse to climb atop it, thus exiting the water, and rest.  The result is a maze with the navigating area appearing like an asterisk, although the apparatus can be changed to include more or less arms.  There are several advantages to the radial arm water maze over other mazes, namely, that it has "the spatial complexity and performance measurement simplicity of the dry radial arm maze combined with the rapid learning and strong motivation observed in the Morris water maze without requiring foot shock or food deprivation as motivating factors." (Alamed et al., Nature Protocols 2006)

The radial arm water maze is used to test spatial learning and working memory.  Typically, experimenters train mice to navigate to the target arm by inserting mice into a start arm and maintaining the same target arm during training.  During the experimental trials, the target arm is moved from its original position.  In Morgan et al., Nature 2000, the radial arm water maze was used to test the restoration of spatial learning ability in TG2576 x PS1 double-transgenic mice, a mouse model of Alzheimer's Disease, via amyloid-β vaccination.  The findings suggested that such vaccination might be an effective therapeutic approach to treating Alzheimer's dementia.
===Longitudinal study===
A study done over a period of time, involving observing/examining certain variable(s) in a set group of study subjects. Also called a diachronic study. The opposite of a cross-sectional/synchronic study which is done at one point in time (e.g. to learn about a certain disease's prevalence and distribution within a population at one time), rather than with the passage of time. In the last lecture, we learned that recent longitudinal studies have shown that PD may occur more acutely. Using F-18-Dopa to image dopamine terminals of clinically unaffected twins of PD patients suggests that disease progression is rather acute, with a 5-10 year span from normal imaging results to abnormal to the onset of clinical symptoms.
===ELIZA===
'''ELISA''' stands for '''E'''nzyme-'''L'''inked '''I'''mmuno'''s'''orbent '''A'''ssay. 
This technique uses antibodies (type of protein) as extremely specific analytic reagents to detect and quantify the amount of antigen (a substance that when introduced into the body, stimulates the production of an antibody. Antigens can be proteins, toxins, bacteria, foreign blood cells, cells of transplanted organs, etc.). Here, most likely, we are dealing with ''protein'' antigens.

Overall Method:
#'''Enzyme''': You have an enzyme which reacts with a colorless substrate to produce a colored product.
#'''Antibody-Enzyme Complex''': The enzyme (from part 1) is covalently linked to a specific antibody that recognizes a target antigen.
#'''Antibody-Enzyme Complex + Antigen = Reaction''': If the antigen is present, the antibody-enzyme complex (from part 2) will bind to it, and the ''enzyme part'' of the antibody-enzyme complex (from part 2) will catalyze the reaction generating the colored product.
#'''What does the reaction (i.e. getting a colored product) mean?''' The presence of a colored product indicates the presence of the antigen (see part 3).

'''Why do people use the ELISA method?'''
#It is fast and convenient.
#It is sensitive--can detect less than 10^(-9) grams of a protein.
#It can be performed with either '''polyclonal''' (mixture of different antibodies that bind to the same antigen) or '''monoclonal''' (identical antibodies cloned from a single antibody-producing cell, used to bind to one type of antigen) antibodies, but using monoclonal antibodies yields ''more reliable'' results.
*Note: We can use polyclonal and monoclonal antibodies to a specific antigen because antigens have different surface markers/binding sites that specifically recognize different types of antibodies.

There are several types of ELISA. Here are two popularly used types.
*'''Indirect ELISA''': used to detect the ''presence'' of ''antibody'' 
**Indirect ELISA is basis of test for HIV infection.
*'''Sandwich ELISA''': allows ''both detection and quantification'' of ''antigen''
#Antibody for a particular antigen is absorbed to bottom of a well.
#Antigen (or blood/urine containing the antigen) is added to the well and binds to the antibody.
#A second, different antibody (this second antibody is linked to an enzyme, whereas the first antibody was not linked to an enzyme) to the antigen is added. Remember from the note above that different antibodies can bind to the same antigen because the antigen has different surface markers that recognize and bind to these different antibodies. Basically in this step, you are adding the '''antibody-enzyme complex''' (described in the Overall Methods above) to the well.
#A colorless substrate is added to the well. The enzyme part of the antibody-enzyme complex (from part (3) above) will react with the substrate and produce color. The rate of this color formation is proportional to the amount of antigen present (because the enzyme is linked to the second antibody which is linked to the antigen).
===Suppressor screen===
===Immunotherapy===
Immunotherapy is a broad class of therapies that uses the natural molecules of a body's immune system to either treat or prevent diseases.  There are many types of immunotherapies, the most common form being anti-microbial vaccinations.  Other kinds of immunotherapies do exist, such as cancer immunotherapy, where the patient's immune system is stimulated to kill tumor cells, and allergy immunotherapy, where increased dosage of injected allergens leads to a desensitization response.

In the context of the class, immunotherapy was explored as a treatment for Alzheimer's disease.  Schenk et al (Nature, 1999) found that young mice predispositioned to get AD did not develop beta-amyloid (Ab) plaques when injected with Ab over a period of time and older mice that already had plaques showed retarded plaque growth following Ab injection.  The reason this therapy works is that injection with Ab causes the body's immune system to synthesize high titer anti-Ab antibodies.  These antibodies then bind to Ab, causing its clearance from the brain and the prevention of plaques.  The result is improved learning and memory.  However, human trials were curtailed in 2002 after some patients showed inflammatory responses. 

==Paper 4 Model Systems==

===Drosophila nervous system===

==Paper 4 Techniques==

===Homologous recombination in Drosophila===
Homologous recombination is a technique that allows you to replace an allele with an engineered construct while not affecting any other loci in the genome. In the Manoli et al paper, homologous recombination was used to generate the fruP1-Gal4 construct.

Procedure: design the DNA construct that you want to insert --> flank this construct with DNA sequences identical to the sequences in the target locus (so that your construct can later find the homologous chromosome during mitosis/meiosis) --> add your engineered construct to cells containing the targeted gene of interest --> during mitosis/meiosis, homologous chromosomes will align and your engineered construct can find the targeted gene --> recombination takes place (switching of the target gene with your engineered construct) --> end result is the altered targeted locus, while the rest of the genome remains unchanged

===Inverted repeats===
Double-stranded RNA (RNA-mediated interference) has become a standard tool used to silence post-transcriptional expression of a single gene, and the results can provide some insight on the gene’s function. dsRNA can be induced using inverted repeats, which consist of two arms of DNA that surround a spacer region. Inverted repeats have been associated with genomic instability, and are also known to induced homologous recombination. 
===Male-male habituation===
Male fruit flies will court other male flies when exposed to them, initially. Over time, they learn not to court each other. This is called male-male habituation, or "Experience-Dependent Courtship Modification". (Vaias, et al. 1993) The courtship displayed towards other males is the essentially the same courtship behavior displayed towards other females: the behavior is identical for courtship of either sex, intially. The Vaias paper demonstrates that exposure to pheromones released by young males causes male-male habituation.
In the discussion paper 'Baker, et al. 2005' on the fruitless gene, they inhibited the expression of Fru-M in olfactory neurons with UAS-Fru-M- IR, and observed that the inhibited males courted males far longer than males with a control transgene.
===Courtship conditioning===
Recently mated females display a rejection response to males trying to court them. In courtship conditioning, males learn to no longer attempt to court these females, and have a longer courtship delay time in approaching virgin females after exposure to a mated female. In the Fruitless paper (Baker, et al. 2005), they found that the inhibition by UAS-fruM-IR of Fru-M expression in mushroom body neurons decreased any conditioning caused by exposure to mated females.
==<h3>Recent updates to the site:</h3>==
{{Special:Recentchanges/BIO254&limit=50}}
</div>