[[Bioinformatics Discussion Group]]

==Software==
*[http://pfaat.sourceforge.net/ PFAAT] - Protein Family Alignment Annotation Tool

===Frameworks===
*[http://www.bioinformatics.org/biococoa/ BioCocoa] is an open source Cocoa framework for bioinformatics written in Objective-C

==Online services==
*[http://taverna.sourceforge.net/index.php?doc=services.html Biological web services] - Taverna Project
*[http://weblogo.berkeley.edu/logo.cgi WebLogo] - create sequence logos
*[http://www.ch.embnet.org/software/TCoffee.html T-COFFEE server]
*[http://bioinformatics.uams.edu/mafft/ MAFFT]: A Program for Multiple Sequence Alignment
*[http://blocks.fhcrc.org/ Block WWW Server]
**[http://blocks.fhcrc.org/sift/SIFT.html SIFT] Sorting Intolerant From Tolerant - will return predictions for what amino acid substitutions will affect protein function
**[http://www.proweb.org/coddle/ CODDLE] Codons Optimized to Discover Deleterious Lesions - will identify regions of a gene where mutations are likely to be deleterious
*[http://rsat.ulb.ac.be/rsat/ Regulatory Sequence Analysis Tools (RSAT)] provide a series of modular computer programs specifically designed for the detection of regulatory signals in non-coding sequences
*[http://supfam.org/SUPERFAMILY/ Superfamily] - HMM library and genome assignments server

===Synthetic biology===
*[[Synthetic_Biology:Tools|Synthetic biology tools]] @ OpenWetWare
*[http://slam.bs.jhmi.edu/gd/ GeneDesign &beta;2.0]
*[http://www.evolvingcode.net/codon/ Gene Re-design Software Collection] by EvolvingCode Group @ UMBC

===Databases===
*[http://www.ncbi.nlm.nih.gov/Entrez/ Entrez] cross-database search
*[http://expasy.org/ ExPASy Proteomics Server] is dedicated to the analysis of protein sequences and structures as well as 2-D PAGE
*[http://www.uniprot.org/ UniProt (Universal Protein Resource)] is the world's most comprehensive catalog of information on proteins. It is a central repository of protein sequence and function created by joining the information contained in Swiss-Prot, TrEMBL, and PIR
*[http://dip.doe-mbi.ucla.edu/ DIP] catalogs experimentally determined interactions between proteins, it combines information from a variety of sources to create a single, consistent set of protein-protein interactions
*[http://www.genome.ad.jp/kegg/ KEGG]: Kyoto Encyclopedia of Genes and Genomes
*[http://www.bind.ca/Action BIND] - Biomolecular Interaction Network Database
*[http://scop.berkeley.edu/ SCOP]: Structural Classification of Proteins
*[http://www.ebi.ac.uk/InterProScan/ InterProScan]: sequence search
*[http://www.sanger.ac.uk/Software/Pfam/search.shtml Pfam search]
*[http://expasy.org/prosite/ PROSITE] is a database of protein families and domains. It consists of biologically significant sites, patterns and profiles that help to reliably identify to which known protein family (if any) a new sequence belongs
*[http://bioinf.man.ac.uk/dbbrowser/PRINTS/ PRINTS] is a compendium of protein fingerprints

====Kinases====
*[http://mapk.8m.com/ MAP Kinase Resource]
*[http://www.kinasenet.org/pkr/Welcome.do Protein Kinase Resource]
*[http://hodgkin.mbu.iisc.ernet.in/~king/ KinG]: Kinases in Genomes is a comprehensive collection of Ser/Thr/Tyr specific protein kinases and similar sequences encoded in the completed genomes of eukaryotes and prokaryotes

==Miscellaneous==
[http://www.gene-regulation.com/ Gene Regulation] - list of resources