<div align=center>{{Click || image=Bionumbers-logo-v1.jpg | link=BioNumbers | width=437px | height=55px }}</div>

'''Use the textbox on the right to search or filter the list using the boxes.'''
'''Press on any of the column headings to sort by organism, property etc.'''

<exhibit>
  <source id="bionumbers" 
          columns="label:number,propfamily,propsubfamily,property,organism,value:number,units,reference,primaryref,keywords,method,range,comments,enteredby" 
          type="Number" label="Number" pluralLabel="Numbers" 
          />
  <facet facetClass="TextSearch" facetLabel="Search" />
  <facet expression=".organism" />
  <facet expression=".property" />
  <facet expression=".propfamily" />
  <facet expression=".propsubfamily" />
  <view viewClass="Tabular"
        label="Table"
        columns=".label,.property,.organism,.value,.units"
        sortColumn="1"
        />
  <view viewClass="Tile"
        label="Details"
        showAll="false"
        orders=".property,.organism,.label"
        />
</exhibit>

{| id="bionumbers" border="1"
|-
!ID
!Property family
!Property subfamily
!Property
!Organism
!Value
!Units
!Reference (where data was taken from)
!Primary source of reference (if different from previous)
!Keywords
!measurement method
!Standard deviation or range if available
!comments (cell type, measurement conditions etc.)
!Entered by:
|-
|1
|Sizes
|Lengths
|Cell length
|E. coli
|2
|microns
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi Shan Sundararaj, Anchi Guo, Bahram Habibi-Nazhad, Melania Rouani,Paul Stothard, Michael Ellison,and David S. Wishart "The CyberCell Database (CCDB): a comprehensive, self-updating, relational database to coordinate and facilitate in silico modeling of Escherichia coli" Nucleic Acids Res. 2004 January 1; 32 (Database issue): D293.D295
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|2
|Sizes
|Lengths
|Cell diameter
|E. coli
|0.8
|microns
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|3
|Sizes
|Volumes
|Cell total volume
|E. coli
|1x10-18
|m^3
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|4
|Sizes
|Volumes
|Cell total volume
|E. coli
|0.44-1.79*10^-18
|m^3
|Determination of bacterial cell volume with the Coulter Counter. HE Kubitschek, JA Friske - Journal of Bacteriology, 1986, p. 1466-1467
|
|
|Coulter counter
|
|value depends on medium and growth rate, see ref.
|Ron Milo
|-
|5
|Sizes
|Volumes
|Cell aqueous volume
|E. coli
|7 x 10-16
|L
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|6
|Sizes
|Surface
|Cell surface area
|E. coli
|6x10-12
|m^2
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|7
|Sizes
|Weights
|Cell wet weight
|E. coli
|1x10-15
|kg
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|8
|Sizes
|Weights
|Cell dry weight
|E. coli
|3.0x10-16
|kg
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|9
|Sizes
|Weights
|Cell dry weight
|E. coli
|6.4x10-16
|kg
|Bremer, H., Dennis, P. P. (1996) Modulation of chemical composition and other parameters of the cell by growth rate. Neidhardt, et al. eds. Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. chapter 97, Table 2
|
|
|Indirect calculation based on the ratio: (dry mass)/OD460
|
|calculated based on data at 30 minutes cell division time. For other division times see reference.
|Ron Milo
|-
|10
|Sizes
|Volumes
|Periplasm volume
|E. coli
|6.5x10-17
|L
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|11
|Sizes
|Volumes
|Cytoplasm volume
|E. coli
|6.7x10-16
|L
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|12
|Sizes
|Volumes
|Envelope volume
|E. coli
|1.6x10-16
|L
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|13
|Sizes
|Volumes
|Nuclear (DNA+protein) volume
|E. coli
|1.6x10-16
|L
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|14
|Sizes
|Lengths
|Inner Membrane thickness
|E. coli
|8x10-9
|m
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|15
|Sizes
|Lengths
|Outer Membrane thickness
|E. coli
|8x10-9 - 15x10-9
|m
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|16
|Sizes
|Lengths
|Periplasm thickness
|E. coli
|1x10-8
|m
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|17
|Sizes
|Sizes
|Average size of protein
|E. coli
|360
|aa
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|18
|Sizes
|Lengths
|Average diameter of ave. protein
|E. coli
|5
|nm
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|19
|Sizes
|Weights
|Average MW of protein
|E. coli
|40
|kDalton
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|20
|Sizes
|Complexes
|Average prot. oligomerization state
|E. coli
|4
|proteins/complex
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|21
|Sizes
|Weights
|Average MW of protein entity
|E. coli
|160
|kDalton
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|22
|Sizes
|mRNA
|Average size of mRNA
|E. coli
|1100
|bp
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|23
|Sizes
|mRNA
|Average length of mRNA
|E. coli
|370
|nm
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|24
|Sizes
|Weights
|Average MW of all RNAs
|E. coli
|400
|kDalton
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|25
|Sizes
|Weights
|Average MW of single DNA
|E. coli
|3.0x10^6
|kDalton
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|26
|Sizes
|Weights
|Average MW of all DNA
|E. coli
|7x10^6
|kDalton
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|27
|Sizes
|Lengths
|Average length of DNA (chrom.)
|E. coli
|1.55
|mm
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|28
|Sizes
|Lengths
|Diameter of chromosome
|E. coli
|490
|micron
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|29
|Sizes
|Lengths
|Diameter of condensed chromosome
|E. coli
|17
|micron
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|30
|Sizes
|Lengths
|Spacing between small organics
|E. coli
|3.6
|nm/molecule
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|31
|Sizes
|Lengths
|Spacing between ions
|E. coli
|2.1
|nm/molecule
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|32
|Sizes
|Lengths
|Ave. spacing between proteins
|E. coli
|7
|nm/molecule
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|33
|Sizes
|Lengths
|Spacing between protein entities
|E. coli
|9
|nm/molecule
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|34
|General Statistics
|Velocities
|Mean Velocity of 70 kD protein (cytoplasm)
|E. coli
|3
|nm/ms
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|35
|General Statistics
|Velocities
|Mean Velocity of 40 kD protein (cytoplasm)
|E. coli
|5
|nm/ms
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|36
|General Statistics
|Velocities
|Mean Velocity of 30 kD protein (cytoplasm)
|E. coli
|7
|nm/ms
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|37
|General Statistics
|Velocities
|Mean Velocity of 14 kD protein (cytoplasm)
|E. coli
|10
|nm/ms
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|38
|General Statistics
|Velocities
|Mean Velocity of small molecules (cytoplasm)
|E. coli
|50
|nm/ms
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|39
|General Statistics
|Velocities
|Mean Velocity of protein in H2O
|E. coli
|27
|nm/ms
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|40
|General Statistics
|Velocities
|Mean Velocity of small molecules in H2O
|E. coli
|87
|nm/ms
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|41
|General Statistics
|Concentrations
|Concentration of protein in cell
|E. coli
|200-320
|mg/ml
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|42
|General Statistics
|Concentrations
|Concentration of RNA in cell
|E. coli
|75-120
|mg/ml
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|43
|General Statistics
|Concentrations
|Concentration of DNA in cell
|E. coli
|11 - 18
|mg/ml
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|44
|General Statistics
|Volumes
|Volume occupied by water
|E. coli
|70
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|45
|General Statistics
|Volumes
|Volume occupied by protein
|E. coli
|17
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|46
|General Statistics
|Volumes
|Volume occupied by all RNA
|E. coli
|6
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|47
|General Statistics
|Volumes
|Volume occupied by rRNA
|E. coli
|5
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|48
|General Statistics
|Volumes
|Volume occupied by tRNA
|E. coli
|0.8
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|49
|General Statistics
|Volumes
|Volume occupied by mRNA
|E. coli
|0.2
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|50
|General Statistics
|Volumes
|Volume occupied by DNA
|E. coli
|1
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|51
|General Statistics
|Volumes
|Volume occupied by ribosomes
|E. coli
|8
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|52
|General Statistics
|Volumes
|Volume occupied by lipid
|E. coli
|3
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|53
|General Statistics
|Volumes
|Volume occupied by LPS
|E. coli
|1
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|54
|General Statistics
|Volumes
|Volume occupied by murein
|E. coli
|1
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|55
|General Statistics
|Volumes
|Volume occupied by glycogen
|E. coli
|1
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|56
|General Statistics
|Volumes
|Volume occupied by ions
|E. coli
|0.3
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|57
|General Statistics
|Volumes
|Volume occupied by small organics
|E. coli
|1
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|58
|General Statistics
|Rates
|Translation rate
|E. coli
|40
|aa/sec
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|59
|General Statistics
|Rates
|Translation rate
|E. coli
|20
|aa/sec
|Bremer, H., Dennis, P. P. (1996) Modulation of chemical composition and other parameters of the cell by growth rate. Neidhardt, et al. eds. Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. chapter 97, Table 3
|Dennis, P. P., and H. Bremer. 1974. Differential rate of ribosomal protein synthesis in Escherichia coli B/r. J. Mol. Biol. 84:407�$-1òó422.;Dennis, P. P., and M. Nomura. 1974. Stringent control of ribosomal protein gene expression in Escherichia coli. Proc. Natl. Acad. Sci. USA 71:3819òó3823.
|
|Indirect calculation based on the number of ribosomes, ribosome activity fraction and protein synthesis rate
|
|calculated based on data at 30 minutes cell division time. For other division times see reference. This value is probably an underestimate based on some newer results
|Ron Milo
|-
|60
|General Statistics
|Rates
|RNA polymerase transcription rate
|E. coli
|70
|nt/sec
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|coli B/r. J. Mol. Biol. 84:407�$-1òó422.
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|61
|Copy numbers
|Large Molecule Copy Numbers
|Number of cell walls/cell
|E. coli
|1
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|62
|Copy numbers
|Large Molecule Copy Numbers
|Number of membranes/cell
|E. coli
|2
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|63
|Copy numbers
|Large Molecule Copy Numbers
|Number of chromosomes/cell
|E. coli
|2.3
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|at mid log phase
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|64
|Copy numbers
|Large Molecule Copy Numbers
|Number of mRNA/cell
|E. coli
|4000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|65
|Copy numbers
|Large Molecule Copy Numbers
|Number of rRNA/cell
|E. coli
|18000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|66
|Copy numbers
|Large Molecule Copy Numbers
|Number of tRNA/cell
|E. coli
|200000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|67
|Copy numbers
|Large Molecule Copy Numbers
|Number of all RNA/cell
|E. coli
|222000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|68
|Copy numbers
|Large Molecule Copy Numbers
|Number of polysaccharides/cell
|E. coli
|39000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|69
|Copy numbers
|Large Molecule Copy Numbers
|Number of murein molecules/cell
|E. coli
|240,000-700,000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|70
|Copy numbers
|Large Molecule Copy Numbers
|Number of lipopolysaccharide/cell
|E. coli
|600000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|71
|Copy numbers
|Large Molecule Copy Numbers
|Number of lipids/cell
|E. coli
|25000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|72
|Copy numbers
|Large Molecule Copy Numbers
|Number of all lipids/cell
|E. coli
|25000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|73
|Copy numbers
|Large Molecule Copy Numbers
|Number of phosphatidylethanolamine
|E. coli
|18500000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|74
|Copy numbers
|Large Molecule Copy Numbers
|Number of phosphatidylglycerol
|E. coli
|5000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|75
|Copy numbers
|Large Molecule Copy Numbers
|Number of cardiolipin
|E. coli
|1200000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|76
|Copy numbers
|Large Molecule Copy Numbers
|Number of phosphatidylserine
|E. coli
|500000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|77
|Copy numbers
|Large Molecule Copy Numbers
|Number of LPS (MW = 10kD)
|E. coli
|600000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|78
|Copy numbers
|Large Molecule Copy Numbers
|Average SA of lipid molecule
|E. coli
|25
| Ang2
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|79
|Copy numbers
|Large Molecule Copy Numbers
|Fraction of lipid bilayer=lipid
|E. coli
|40
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|80
|Copy numbers
|Large Molecule Copy Numbers
|Fraction of lipid bilayer=protein
|E. coli
|60
|Percent
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|81
|Copy numbers
|Large Molecule Copy Numbers
|Number of outer membrane proteins
|E. coli
|300000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|82
|Copy numbers
|Large Molecule Copy Numbers
|Number of porins (subset of OM)
|E. coli
|60000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|83
|Copy numbers
|Large Molecule Copy Numbers
|Number of lipoproteins (OM)
|E. coli
|240000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|84
|Copy numbers
|Large Molecule Copy Numbers
|Number of inner membrane proteins
|E. coli
|200000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|85
|Copy numbers
|Large Molecule Copy Numbers
|Number of nuclear proteins
|E. coli
|100000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|86
|Copy numbers
|Large Molecule Copy Numbers
|Number of cytoplasmic proteins
|E. coli
|1000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|excluding ribosomal proteins
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|87
|Copy numbers
|Large Molecule Copy Numbers
|Number of ribosomal proteins
|E. coli
|900000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|88
|Copy numbers
|Large Molecule Copy Numbers
|Number of periplasmic proteins
|E. coli
|80000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|89
|Copy numbers
|Large Molecule Copy Numbers
|Number of all proteins in cell
|E. coli
|2600000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|90
|Copy numbers
|Large Molecule Copy Numbers
|Number of all proteins in cell in amino acids
|E. coli
|1.89*10^9
|amino acids
|Neidhardt, F.C. et al. (1987) Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. chapter 97, Table 2
|
|
|
|
|For a cell cycle of 30 minutes at 37deg, see reference for value at other division rates
|Ron Milo
|-
|91
|Genome
|Replication
|Time to replicate the chromosome
|E. coli
|40-67
|Minutes
|Neidhardt, F.C. et al. (1987) Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. chapter 97, Table 1, 3
|
|
|
|
|Value in range depends on division rate
|Ron Milo
|-
|92
|Translation
|Ribosomes
|Average distance between ribosomes on mRNA
|E. coli
|41-79
|nucleotides
|Neidhardt, F.C. et al. (1987) Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. chapter 97, Table 3
|
|
|Indirectly calculated from other parameters, see reference
|
|Value in range depends on division rate
|Ron Milo
|-
|93
|Copy numbers
|Large Molecule Copy Numbers
|Number of external proteins (flag/pili)
|E. coli
|1000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|94
|Copy numbers
|Large Molecule Copy Numbers
|Number of all proteins
|E. coli
|3600000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|95
|Complexes
|Statistics on Larger Molecule Complexes
|Number of protein types to make flagella
|E. coli
|42
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|96
|Sizes
|Statistics on Larger Molecule Complexes
|Length of flagella
|E. coli
|10 - 20
|microns
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|97
|Sizes
|Statistics on Larger Molecule Complexes
|Diameter of flagella
|E. coli
|25
|nm
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|98
|Complexes
|Statistics on Larger Molecule Complexes
|Number of protofilaments in flagellum
|E. coli
|11
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|99
|Sizes
|Statistics on Larger Molecule Complexes
|Diameter of each fliC monomer
|E. coli
|5
|nm
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|100
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of fliC monomers in filament
|E. coli
|33000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|101
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of flagella/cell
|E. coli
|10
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|102
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of fliC proteins
|E. coli
|330000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|103
|Velocities
|Statistics on Larger Molecule Complexes
|Speed at which E. coli move
|E. coli
|50
|micron/sec
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|104
|
|Statistics on Larger Molecule Complexes
|Number of protein types to make pilus
|E. coli
|1
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|105
|Lengths
|Statistics on Larger Molecule Complexes
|Length of pili/fimbrae
|E. coli
|200-2000
|nm
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|106
|Lengths
|Statistics on Larger Molecule Complexes
|Diamter of pili
|E. coli
|6.5
|nm
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|107
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of papA/nm pilus
|E. coli
|1.5
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|108
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of papA monomers/pilus
|E. coli
|3000-30,000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|109
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of pili/cell
|E. coli
|100-300
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|110
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of papA/cell
|E. coli
|300,000-900,000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|111
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of ribosomes/cell
|E. coli
|18000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|112
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of protein types to make ribosome
|E. coli
|55
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|113
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number rRNA types to make ribosome
|E. coli
|3
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|114
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of proteins in 30S subunit
|E. coli
|21
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|115
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of proteins in 50S subunit
|E. coli
|34
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|116
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of rRNA in 30S subunit
|E. coli
|1
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|117
|Copy numbers
|Statistics on Larger Molecule Complexes
|Number of rRNA in 50S subunit
|E. coli
|2
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|118
|Lengths
|Statistics on Larger Molecule Complexes
|Length of all rRNA
|E. coli
|5520
|nt
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|119
|Weights
|Statistics on Larger Molecule Complexes
|MW of ribosome
|E. coli
|2700
|kDalton
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|120
|Weights
|Statistics on Larger Molecule Complexes
|MW of RNA component
|E. coli
|1700
|kDalton
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|121
|Weights
|Statistics on Larger Molecule Complexes
|MW of protein component
|E. coli
|1000
|kDalton
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|122
|Lengths
|Statistics on Larger Molecule Complexes
|Diameter of ribosome
|E. coli
|20
|nm
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|123
|Volumes
|Statistics on Larger Molecule Complexes
|Volume of ribosome
|E. coli
|4.2 x 10^-24
|m^3
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|124
|Copy numbers
|Small Molecule Copy Numbers
|Number of water molecules/cell
|E. coli
|2.34x10^10
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|125
|Copy numbers
|Small Molecule Copy Numbers
|Number of ions/cell
|E. coli
|120000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|120,000,000 (300 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|126
|Copy numbers
|Small Molecule Copy Numbers
|Number of small organics/cell
|E. coli
|18000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|18,000,000 (40-50 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|127
|Copy numbers
|Small Molecule Copy Numbers
|Number of K ions
|E. coli
|90000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|90,000,000 (200-250 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|128
|Copy numbers
|Small Molecule Copy Numbers
|Number of Na ions
|E. coli
|2000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|2,000,000 (5 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|129
|Copy numbers
|Small Molecule Copy Numbers
|Na (in): Na (out)
|E. coli
|1 : 20
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|1:20 (in concentration)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|130
|Copy numbers
|Small Molecule Copy Numbers
|Number of Ca ions
|E. coli
|2300000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|2,300,000 (6 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|131
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Ca ions
|E. coli
|40
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|40 (100 nM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|132
|Copy numbers
|Small Molecule Copy Numbers
|Number of Cl ions
|E. coli
|2400000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|2,400,000 (6 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|133
|Copy numbers
|Small Molecule Copy Numbers
|Number of Mg ions
|E. coli
|4000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|4,000,000 (10 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|134
|Copy numbers
|Small Molecule Copy Numbers
|Number of Fe ions
|E. coli
|7000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|7,000,000 (18 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|135
|Copy numbers
|Small Molecule Copy Numbers
|Number of Mn ions
|E. coli
|1700000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|1,700,000 (4 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|136
|Copy numbers
|Small Molecule Copy Numbers
|Number of Zn ions
|E. coli
|1700000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|1,700,000 (4 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|137
|Copy numbers
|Small Molecule Copy Numbers
|Number of Mo ions
|E. coli
|1700000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|1,700,000 (4 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|138
|Copy numbers
|Small Molecule Copy Numbers
|Number of Cu ions
|E. coli
|1700000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|1,700,000 (4 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|139
|Copy numbers
|Small Molecule Copy Numbers
|Number of PO4 ions
|E. coli
|2000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|2,000,000 (5 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|140
|Copy numbers
|Small Molecule Copy Numbers
|Number of glucose/cell
|E. coli
|200,000-400,000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|200,000-400,000 (0.5-1 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|141
|Copy numbers
|Small Molecule Copy Numbers
|Number of PEP/cell
|E. coli
|1100000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|1,100,000 (2.8 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|142
|Copy numbers
|Small Molecule Copy Numbers
|Number of pyruvate/cell
|E. coli
|370000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|370,000 (0.9 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|143
|Copy numbers
|Small Molecule Copy Numbers
|Number of gluc-6-PO4/cell
|E. coli
|20000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|20,000 (0.05 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|144
|Copy numbers
|Small Molecule Copy Numbers
|Number of ATP/cell
|E. coli
|500,000 - 3,000,000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|500,000 - 3,000,000 (1.3-7.0 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|145
|Copy numbers
|Small Molecule Copy Numbers
|Number of ADP/cell
|E. coli
|70000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|70,000 (0.17 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|146
|Copy numbers
|Small Molecule Copy Numbers
|Number of NADP/cell
|E. coli
|240000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|240,000 (0.63 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|147
|Copy numbers
|Small Molecule Copy Numbers
|Number of NADPH/cell
|E. coli
|220000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|220,000 (0.56 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|148
|Copy numbers
|Small Molecule Copy Numbers
|Number of all amino acids/cell
|E. coli
|6000000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|6,000,000 (1.5 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|149
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Alanine/cell
|E. coli
|350000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|350,000 (0.8 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|150
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Cysteine/cell
|E. coli
|80000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|80,000 (0.2 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|151
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Aspartate/cell
|E. coli
|530000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|530,000 (1.34 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|152
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Glutamate/cell
|E. coli
|200000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|200,000 (0.5 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|153
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Phenylalanine/cell
|E. coli
|170000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|170,000 (0.4 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|154
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Glycine/cell
|E. coli
|350000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|350,000 (0.8 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|155
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Histidine/cell
|E. coli
|80000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|80,000 (0.2 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|156
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Isoleucine/cell
|E. coli
|200000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|200,000 (0.5 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|157
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Lysine/cell
|E. coli
|190000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|190,000 (0.46 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|158
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Leucine/cell
|E. coli
|300000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|300,000 (0.7 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|159
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Methionine/cell
|E. coli
|40000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|40,000 (0.1 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|160
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Asparagine/cell
|E. coli
|200000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|200,000 (0.5 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|161
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Proline/cell
|E. coli
|200000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|200,000 (0.5 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|162
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Glutamine/cell
|E. coli
|200000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|200,000 (0.5 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|163
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Arginine/cell
|E. coli
|170000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|170,000 (0.4 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|164
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Serine/cell
|E. coli
|300000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|300,000 (0.7 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|165
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Threonine/cell
|E. coli
|1400000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|1,400,000 (3.49 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|166
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Valine/cell
|E. coli
|240000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|240,000 (0.6 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|167
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Tryptophan/cell
|E. coli
|80000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|80,000 (0.2 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|168
|Copy numbers
|Small Molecule Copy Numbers
|Number of free Tyrosine/cell
|E. coli
|300000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|300,000 (0.7 mM)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|169
|Pressure
|Small Molecule Copy Numbers
|Osmotic pressure (pushing out)
|E. coli
|75
|lb/in2
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|inner, pressure, osmotic, internal
|
|
|Equivalent to about 5 Atm (50ton/m^2)
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|170
|Energetic cost
|E. coli Metabolism
|One glucose generates (total)
|E. coli
|36-38
|ATP
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|171
|Energetic cost
|E. coli Metabolism
|Glycolysis yields
|E. coli
|39241
|ATP
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|172
|Energetic cost
|E. coli Metabolism
|Oxidation of pyruvate yields
|E. coli
|6
|ATP
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|173
|Energetic cost
|E. coli Metabolism
|Krebs cycle/e- transport yields
|E. coli
|24
|ATP
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|174
|Energetic cost
|E. coli Metabolism
|Number ATP to make 1 DNA
|E. coli
|72289000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|175
|Energetic cost
|E. coli Metabolism
|Number ATP to make 1 protein (360 aa)
|E. coli
|1500
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|176
|Energetic cost
|E. coli Metabolism
|Number ATP to make 1 lipid
|E. coli
|7
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|177
|Energetic cost
|E. coli Metabolism
|Number ATP to make 1 polysaccharide
|E. coli
|2000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|178
|Energetic cost
|E. coli Metabolism
|Number ATP to make 1 RNA (1000 nt)
|E. coli
|2000
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|179
|Energetic cost
|E. coli Metabolism
|Number ATP to make 1 cell
|E. coli
|55*10^9
|ATP
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|180
|Energetic cost
|E. coli Metabolism
|Number Glucose molecules consumed
|E. coli
|1.4*10^6
|Unitless
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|181
|Growth
|E. coli Metabolism
|Cell generation time
|E. coli
|30
|minutes
|http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
|A list of primary sources is given in the reference
|
|
|
|
|Ron Milo, Paul Jorgensen, Mike Springer
|-
|182
|General statistics
|DNA
|Chromosomal DNA weight
|X. laevis
|12
|pg
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|A list of primary sources is given in the reference
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|183
|General statistics
|DNA
|Nucleolar rDNA weight
|X. laevis
|25
|pg
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|184
|General statistics
|DNA
|Nucleolar rDNA
|X. laevis
|2 x 10^6
|rDNA repeats
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|185
|General statistics
|DNA
|Mitochondrial DNA weight
|X. laevis
|4000
|pg
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|Igor and Dawid, Dev. Biol 27:504-518 (1972)
|
|Egg
|
|
|Paul Jorgensen
|-
|186
|General statistics
|DNA
|Mitochondrial DNA
|X. laevis
|10^8
|genomes
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|187
|General statistics
|RNA
|rRNA
|X. laevis
|5
|ug
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|188
|General statistics
|RNA
|5S RNA
|X. laevis
|60
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|189
|General statistics
|RNA
|5S RNA
|X. laevis
|10^12
|RNAs
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|190
|General statistics
|RNA
|tRNA
|X. laevis
|60
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|191
|General statistics
|RNA
|tRNA
|X. laevis
|1.5 x 10^12
|tRNAs
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|192
|General statistics
|RNA
|snRNA U1
|X. laevis
|0.07
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|193
|General statistics
|RNA
|snRNA U1
|X. laevis
|8 x 10^8
|RNAs
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|194
|General statistics
|RNA
|polyA+ RNA
|X. laevis
|80
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|195
|General statistics
|RNA
|polyA+ RNA
|X. laevis
|5 x 10^10
|RNAs
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|if 2500bp long on average
|Paul Jorgensen
|-
|196
|General statistics
|RNA
|ribosomal protein mRNA
|X. laevis
|10
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|197
|General statistics
|RNA
|ribosomal protein mRNA
|X. laevis
|2 x 10^10
|transcripts
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|198
|General statistics
|RNA
|actin mRNA
|X. laevis
|1
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|199
|General statistics
|RNA
|actin mRNA
|X. laevis
|5 x 10^8
|transcripts
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|200
|General statistics
|RNA
|Hsp70 mRNA
|X. laevis
|0.004
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|201
|General statistics
|RNA
|Hsp70 mRNA
|X. laevis
|10^6
|transcripts
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|202
|General statistics
|RNA
|Ribosomes
|X. laevis
|10^12
|ribosomes
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|203
|General statistics
|Protein
|Yolk protein
|X. laevis
|250
|ug
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|204
|General statistics
|Protein
|non-yolk protein
|X. laevis
|25
|ug
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|205
|General statistics
|Protein
|non-yolk protein
|X. laevis
|5 x 10^14
|proteins
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|if 30K average
|Paul Jorgensen
|-
|206
|General statistics
|Protein
|Histones
|X. laevis
|140
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|207
|General statistics
|Protein
|Histones
|X. laevis
|5 x 10^12
|histone proteins
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|208
|General statistics
|Protein
|Nucleoplasmin
|X. laevis
|250
|ng
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|209
|General statistics
|Protein
|Nucleoplasmin
|X. laevis
|5 x 10^12
|nucleoplasmin proteins
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|210
|General statistics
|Protein
|RNA polymerase I and II
|X. laevis
|10^5
| greater than a somatic cell
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|211
|General statistics
|Protein
|RNA polymerase III
|X. laevis
|5 x 10^5
| greater than a somatic cell
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|212
|General statistics
|Metabolism
|Each dNTP
|X. laevis
|10 - 15
|pmol
|Woodland and Pestell, Biochem J. 127:597-605 (1972)
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|213
|General statistics
|Metabolism
|rGTP
|X. laevis
|250
|pmol
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|214
|General statistics
|Metabolism
|Methionine
|X. laevis
|40
|pmol
|Hausen and Riebesell, The Early Development of Xenopus laevis, 1991
|
|
|Stage VI oocyte
|
|
|Paul Jorgensen
|-
|215
|General statistics
|Genome
|Genome Size
|Escherichia coli (Bacterium)
|4.6*10^6
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|216
|General statistics
|Genome
|Genome Size
|Sacchromyces cerevisiae (Yeast)
|12156678
|Base pairs
|http://www.yeastgenome.org/cache/genomeSnapshot.html
|
|
|
|
|
|Ron Milo
|-
|217
|General statistics
|Genome
|Genome Size
|Neurospora crassa (Fungus)
|39.9*10^6
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|218
|General statistics
|Genome
|Genome Size
|C. elegans (Nematode)
|100*10^6
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|219
|General statistics
|Genome
|Genome Size
|C. briggsae (Nematode)
|104*10^6
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|220
|General statistics
|Genome
|Genome Size
|Drosophila melanogaster (Fruit fly)
|120*10^6
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|221
|General statistics
|Genome
|Genome Size
|Anapheles gambiae (Mosquito)
|280*10^6
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|222
|General statistics
|Genome
|Genome Size
|Gallus gallus (Chicken)
|120*10^6
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|223
|General statistics
|Genome
|Genome Size
|Fugu rubripes (Pufferfish)
|365*10^6
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|224
|General statistics
|Genome
|Genome Size
|Danio rario (Zebrafish)
|1.7*10^9
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
| (?)
|Ron Milo
|-
|225
|General statistics
|Genome
|Genome Size
|Mouse
|3*10^9
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|226
|General statistics
|Genome
|Genome Size
|Xenopus laevis (Frog)
|3*10^9
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|~
|Ron Milo
|-
|227
|General statistics
|Genome
|Genome Size
|Human
|3*10^9
|Base pairs
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|228
|General statistics
|Genome
|Number of genes
|Escherichia coli (Bacterium)
|3000
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|229
|General statistics
|Genome
|Number of genes
|Yeast
|6609
|ORF
|http://www.yeastgenome.org/cache/genomeSnapshot.html
|
|
|
|
|
|Ron Milo
|-
|230
|General statistics
|Genome
|Number of genes
|Neurospora crassa (Fungus)
|10000
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
| (Release Apr 2003)
|Ron Milo
|-
|231
|General statistics
|Genome
|Number of genes
|C. elegans (Nematode)
|20621
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
| (Release WS108)
|Ron Milo
|-
|232
|General statistics
|Genome
|Number of genes
|C. briggsae (Nematode)
|19507
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
| (Release Sept 2003)
|Ron Milo
|-
|233
|General statistics
|Genome
|Number of genes
|Drosophila melanogaster (Fruit fly)
|13647
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
| (Release 3.1)
|Ron Milo
|-
|234
|General statistics
|Genome
|Number of genes
|Anapheles gambiae (Mosquito)
|13600
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|235
|General statistics
|Genome
|Number of genes
|Fugu rubripes (Pufferfish)
|38000
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|236
|General statistics
|Genome
|Number of genes
|Mouse
|30000
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|237
|General statistics
|Genome
|Number of genes
|Human
|31000
|Genes
|http://wormlab.caltech.edu/briggsae/genomeSize.html
|
|
|
|
|
|Ron Milo
|-
|238
|General statistics
|Genome
|Number of tRNA
|Yeast
|299
|unitless
|http://www.yeastgenome.org/cache/genomeSnapshot.html
|
|
|
|
|
|Ron Milo
|-
|239
|General statistics
|Genome
|Number of rRNA
|Yeast
|27
|unitless
|http://www.yeastgenome.org/cache/genomeSnapshot.html
|
|
|
|
|
|Ron Milo
|-
|240
|Growth
|Generation time
|Minimal generation time
|Vibrio natriegens
|9.8
|Minutes
|R. G. Eagon, J. Bacteriol. 83:736-737, 1962
|
|
|Optimal growth occurred at 37 C in brain heart infusion broth supplemented with 1.5% sea salt.
|
|
|Ron Milo
|-
|241
|Growth
|Generation time
|Minimal generation time
|Escherichia coli
|17
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Glucose-salts
|
|
|Ron Milo
|-
|242
|Growth
|Generation time
|Minimal generation time
|Bacillus megaterium
|25
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Sucrose-salts
|
|
|Ron Milo
|-
|243
|Growth
|Generation time
|Minimal generation time
|Streptococcus lactis
|26
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Milk
|
|
|Ron Milo
|-
|244
|Growth
|Generation time
|Minimal generation time
|Streptococcus lactis
|48
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Lactose broth
|
|
|Ron Milo
|-
|245
|Growth
|Generation time
|Minimal generation time
|Staphylococcus aureus
|27-30
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Heart infusion broth
|
|
|Ron Milo
|-
|246
|Growth
|Generation time
|Minimal generation time
|Lactobacillus acidophilus
|66-87
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Milk
|
|
|Ron Milo
|-
|247
|Growth
|Generation time
|Minimal generation time
|Rhizobium japonicum
|344-461
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Mannitol-salts-yeast extract
|
|
|Ron Milo
|-
|248
|Growth
|Generation time
|Minimal generation time
|Mycobacterium tuberculosis
|792-932
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Synthetic
|
|
|Ron Milo
|-
|249
|Growth
|Generation time
|Minimal generation time
|Treponema pallidum
|1980
|Minutes
|http://textbookofbacteriology.net/growth.html
|
|
|medium: Rabbit testes
|
|
|Ron Milo
|-
|250
|copy numbers
|Chemotaxis
|Number of chemoreceptors (total)
|E. coli
|15000
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|251
|copy numbers
|Chemotaxis
|Number of major receptors (Tsr+Tar)
|E. coli
|14000
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|Li & Hazelbauer (PNAS, 2004)
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|252
|copy numbers
|Chemotaxis
|Number of minor receptors (Trg+Tap+Aer)
|E. coli
|1000
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|Li & Hazelbauer (PNAS, 2004)
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|253
|copy numbers
|Chemotaxis
|Number of kinases (CheA)
|E. coli
|4500
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|Li & Hazelbauer (PNAS, 2004)
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|254
|copy numbers
|Chemotaxis
|Number of receptor-kinase couplers (CheW)
|E. coli
|6700
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|Li & Hazelbauer (PNAS, 2004)
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|255
|copy numbers
|Chemotaxis
|Number of output response regulators (CheY)
|E.coli
|8200
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|Li & Hazelbauer (PNAS, 2004)
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|256
|copy numbers
|Chemotaxis
|Number of phosphatases (CheZ)
|E. coli
|3200
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|Li & Hazelbauer (PNAS, 2004)
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|257
|copy numbers
|Chemotaxis
|Number of feedback response regulators (CheB)
|E. coli
|240
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|Li & Hazelbauer (PNAS, 2004)
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|258
|copy numbers
|Chemotaxis
|Number of methyltransferases
|E. coli
|140
|proteins
|http://www.pdn.cam.ac.uk/groups/comp-cell/Rates.html
|Li & Hazelbauer (PNAS, 2004)
|
|strain RP437 grown in TB medium
|
|
|Tom Shimizu
|-
|259
|Rates
|Chemotaxis
|Time constant for sub-saturating response
|E. coli
|4
|seconds
|Segall, Block & Berg (PNAS, 1986)
|Li & Hazelbauer (PNAS, 2004)
|
|
|
|
|Tom Shimizu
|-
|260
|Dynamic range
|Chemotaxis
|Dynamic range for gradient sensing of alpha-methylaspartate
|E.coli
|5
|orders of magnitude
|Mesibov, Ordal & Adler (J. Gen. Physiol., 1973)
|
|
|
|
|this is minimal estimation, i.e. larger than 5 orders of magnitude
|Tom Shimizu
|-
|261
|Biophysical parameters
|Diffusion rates
|Diffusion rate of small molecule
|Generic
|50
|10^-7*cm^2/sec
|"Cell movements - from molecules to motility", pp. 6, Dennis Bray, 2nd ed. Garland, 2001
|Hille (1992), Berg (1993), Atkins (1994)
|
|order of magnitude. ~1msec for a distance of 1 micron
|
|
|Ron Milo
|-
|262
|Biophysical parameters
|Diffusion rates
|Diffusion rate of protein molecule
|Generic
|5
|10^-7*cm^2/sec
|"Cell movements - from molecules to motility", pp. 6, Dennis Bray, 2nd ed. Garland, 2001
|Hille (1992), Berg (1993), Atkins (1994)
|
|order of magnitude. ~10msec for a distance of 1 micron
|
|
|Ron Milo
|-
|263
|Biophysical parameters
|Diffusion rates
|Diffusion rate of virus particle
|Generic
|0.5
|10^-7*cm^2/sec
|"Cell movements - from molecules to motility", pp. 6, Dennis Bray, 2nd ed. Garland, 2001
|Hille (1992), Berg (1993), Atkins (1994)
|
|order of magnitude. ~0.1sec for a distance of 1 micron
|
|
|Ron Milo
|-
|264
|Biophysical parameters
|Diffusion rates
|Diffusion rate of bacterial cell
|Generic
|0.05
|10^-7*cm^2/sec
|"Cell movements - from molecules to motility", pp. 6, Dennis Bray, 2nd ed. Garland, 2001
|Hille (1992), Berg (1993), Atkins (1994)
|
|order of magnitude. ~1sec for a distance of 1 micron
|
|
|Ron Milo
|-
|265
|Biophysical parameters
|Diffusion rates
|Diffusion rate of animal cell
|Generic
|0.005
|10^-7*cm^2/sec
|"Cell movements - from molecules to motility", pp. 6, Dennis Bray, 2nd ed. Garland, 2001
|Hille (1992), Berg (1993), Atkins (1994)
|
|order of magnitude. ~10sec for a distance of 1 micron
|
|
|Ron Milo
|-
|266
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Pancreatic trypsin inhibitor
|Bovine
|12.9
|10^-7*cm^2/sec
|"Proteins", Creighton 1984
|
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|267
|Biophysical parameters
|Diffusion rates
|Diffusion rate of cytochrome c
|Equine
|13
|10^-7*cm^2/sec
|"Proteins", Creighton 1985
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1979
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|268
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Ribonuclease A
|Bovine
|10.7
|10^-7*cm^2/sec
|"Proteins", Creighton 1986
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1980
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|269
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Lysozyme
|Hen
|11.3
|10^-7*cm^2/sec
|"Proteins", Creighton 1987
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1981
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|270
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Myoglobin
|Sperm whale
|11.3
|10^-7*cm^2/sec
|"Proteins", Creighton 1988
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1982
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|271
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Adenylate kinase
|Porcine
|10.2
|10^-7*cm^2/sec
|"Proteins", Creighton 1989
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1983
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|272
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Trypsin
|Bovine
|9.3
|10^-7*cm^2/sec
|"Proteins", Creighton 1990
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1984
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|273
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Bence Jones REI
|Porcine
|10
|10^-7*cm^2/sec
|"Proteins", Creighton 1991
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1985
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|274
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Chymotrypsinogen
|Human
|9.48
|10^-7*cm^2/sec
|"Proteins", Creighton 1992
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1986
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|275
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Elastase
|Bovine
|9.5
|10^-7*cm^2/sec
|"Proteins", Creighton 1993
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1987
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|276
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Subtilisin novo
|Porcine
|9.04
|10^-7*cm^2/sec
|"Proteins", Creighton 1994
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1988
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|277
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Carbonic anhydrase
|B. amyloliq.
|10.7
|10^-7*cm^2/sec
|"Proteins", Creighton 1995
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1989
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|278
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Superoxide dismutase
|Human
|8.92
|10^-7*cm^2/sec
|"Proteins", Creighton 1996
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1990
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|279
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Carboxypeptidase A
|Bovine
|9.2
|10^-7*cm^2/sec
|"Proteins", Creighton 1997
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1991
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|280
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Phosphoglycerate kinase
|Bovine
|6.38
|10^-7*cm^2/sec
|"Proteins", Creighton 1998
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1992
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|281
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Concanavalin A
|Yeast
|6.34
|10^-7*cm^2/sec
|"Proteins", Creighton 1999
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1993
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|282
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Hemoglobin, oxy
|Equine
|6.02
|10^-7*cm^2/sec
|"Proteins", Creighton 2000
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1994
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|283
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Malate dehydrogenase
|Porcine
|5.76
|10^-7*cm^2/sec
|"Proteins", Creighton 2001
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1995
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|284
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Alcohol dehydrogenaseh
|Equine
|6.23
|10^-7*cm^2/sec
|"Proteins", Creighton 2002
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1996
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|285
|Biophysical parameters
|Diffusion rates
|Diffusion rate of Lactate dehydrogenase
|Dogfish
|4.99
|10^-7*cm^2/sec
|"Proteins", Creighton 2003
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1997
|
|Translational diffusion coefficient at 20degC in water, extraploated to zero protein concentration
|
|
|Ron Milo
|-
|286
|Biophysical parameters
|Molecular weights
|Molecular weight of Pancreatic trypsin inhibitor
|Bovine
|6520
|Dalton
|"Proteins", Creighton 2004
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1998
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|287
|Biophysical parameters
|Molecular weights
|Molecular weight of cytochrome c
|Equine
|12310
|Dalton
|"Proteins", Creighton 2005
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 1999
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|288
|Biophysical parameters
|Molecular weights
|Molecular weight of Ribonuclease A
|Bovine
|13690
|Dalton
|"Proteins", Creighton 2006
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2000
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|289
|Biophysical parameters
|Molecular weights
|Molecular weight of Lysozyme
|Hen
|14320
|Dalton
|"Proteins", Creighton 2007
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2001
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|290
|Biophysical parameters
|Molecular weights
|Molecular weight of Myoglobin
|Sperm whale
|17800
|Dalton
|"Proteins", Creighton 2008
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2002
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|291
|Biophysical parameters
|Molecular weights
|Molecular weight of Adenylate kinase
|Porcine
|21640
|Dalton
|"Proteins", Creighton 2009
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2003
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|292
|Biophysical parameters
|Molecular weights
|Molecular weight of Trypsin
|Bovine
|23200
|Dalton
|"Proteins", Creighton 2010
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2004
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|293
|Biophysical parameters
|Molecular weights
|Molecular weight of Bence Jones REI
|Porcine
|23500
|Dalton
|"Proteins", Creighton 2011
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2005
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|294
|Biophysical parameters
|Molecular weights
|Molecular weight of Chymotrypsinogen
|Human
|25670
|Dalton
|"Proteins", Creighton 2012
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2006
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|295
|Biophysical parameters
|Molecular weights
|Molecular weight of Elastase
|Bovine
|25900
|Dalton
|"Proteins", Creighton 2013
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2007
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|296
|Biophysical parameters
|Molecular weights
|Molecular weight of Subtilisin novo
|Porcine
|27530
|Dalton
|"Proteins", Creighton 2014
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2008
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|297
|Biophysical parameters
|Molecular weights
|Molecular weight of Carbonic anhydrase
|B. amyloliq.
|28800
|Dalton
|"Proteins", Creighton 2015
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2009
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|298
|Biophysical parameters
|Molecular weights
|Molecular weight of Superoxide dismutase
|Human
|33900
|Dalton
|"Proteins", Creighton 2016
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2010
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|299
|Biophysical parameters
|Molecular weights
|Molecular weight of Carboxypeptidase A
|Bovine
|34500
|Dalton
|"Proteins", Creighton 2017
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2011
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|300
|Biophysical parameters
|Molecular weights
|Molecular weight of Phosphoglycerate kinase
|Bovine
|45800
|Dalton
|"Proteins", Creighton 2018
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2012
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|301
|Biophysical parameters
|Molecular weights
|Molecular weight of Concanavalin A
|Yeast
|51260
|Dalton
|"Proteins", Creighton 2019
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2013
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|302
|Biophysical parameters
|Molecular weights
|Molecular weight of Hemoglobin, oxy
|Equine
|64610
|Dalton
|"Proteins", Creighton 2020
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2014
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|303
|Biophysical parameters
|Molecular weights
|Molecular weight of Malate dehydrogenase
|Porcine
|74900
|Dalton
|"Proteins", Creighton 2021
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2015
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|304
|Biophysical parameters
|Molecular weights
|Molecular weight of Alcohol dehydrogenaseh
|Equine
|79870
|Dalton
|"Proteins", Creighton 2022
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2016
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|305
|Biophysical parameters
|Molecular weights
|Molecular weight of Lactate dehydrogenase
|Dogfish
|146200
|Dalton
|"Proteins", Creighton 2023
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2017
|
|Calculated from covalent structure
|
|
|Ron Milo
|-
|306
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Bacteriophage M13
|7.2 x 10-7
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 1998
|Squire & Himmel, Arch. Biochem. Biophys. 196:165-177, 2018
|
|
|
|
|Ron Milo
|-
|307
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Bacteriophage {lambda}
|7.7 x 10-8
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 1999
|
|
|
|
|
|Ron Milo
|-
|308
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Bacteriophages T2 and T4
|2.4 x 10-8
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2000
|
|
|
|
|
|Ron Milo
|-
|309
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Escherichia coli
|5.4 x 10-10
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2001
|
|
|
|
|
|Ron Milo
|-
|310
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Saccharomyces cerevisiae
|2.2 x 10-10
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 1998
|
|
|
|
|
|Ron Milo
|-
|311
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Neurospora crassa
|7.2 x 10-11
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 1999
|
|
|
|
|
|Ron Milo
|-
|312
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|C. elegans
|2.3 x 10-10
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 1998
|
|
|
|
|
|Ron Milo
|-
|313
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Drosophila
|3.4 x 10-10
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 1999
|
|
|
|
|
|Ron Milo
|-
|314
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Mouse
|1.8 x 10-10
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2000
|
|
|
|
|
|Ron Milo
|-
|315
|DNA
|Mutation Rates
|Mutation rate per base pair per replication
|Human
|5.0 x 10-11
|mutation/bp/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2001
|
|
|
|
|
|Ron Milo
|-
|316
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Bacteriophage M13
|0.0046
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2001
|
|
|Notice that this is per gemone replication not per sexual generation
|
|
|Ron Milo
|-
|317
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Bacteriophage {lambda}
|0.0038
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2002
|
|
|Notice that this is per gemone replication not per sexual generation
|
|
|Ron Milo
|-
|318
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Bacteriophages T2 and T4
|0.004
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2003
|
|
|Notice that this is per gemone replication not per sexual generation
|
|
|Ron Milo
|-
|319
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Escherichia coli
|0.0025
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2004
|
|
|
|
|
|Ron Milo
|-
|320
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Saccharomyces cerevisiae
|0.0027
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2005
|
|
|
|
|
|Ron Milo
|-
|321
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Neurospora crassa
|0.003
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2006
|
|
|Notice that this is per gemone replication not per sexual generation
|
|
|Ron Milo
|-
|322
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|C. elegans
|0.018
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2007
|
|
|Notice that this is per gemone replication not per sexual generation
|
|
|Ron Milo
|-
|323
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Drosophila
|0.058
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2008
|
|
|Notice that this is per gemone replication not per sexual generation
|
|
|Ron Milo
|-
|324
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Mouse
|0.49
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2009
|
|
|Notice that this is per gemone replication not per sexual generation
|
|
|Ron Milo
|-
|325
|DNA
|Mutation Rates
|Mutation rate per genome per replication
|Human
|0.16
|mutation/genome/replication
|Drake et. al., Genetics, Vol. 148, 1667-1686, April 2010
|
|
|Notice that this is per gemone replication not per sexual generation
|
|
|Ron Milo
|-
|326
|Growth
|Growth
|Generation time
|Chlamydomonas reinhardtii
|5
|Hours
|http://www.yale.edu/rosenbaum/green_yeast.html
|
|
|
|
|
|Ron Milo
|-
|327
|DNA
|Genome
|Number of chromosomes/cell
|Chlamydomonas reinhardtii
|17
|Unitless
|http://www.yale.edu/rosenbaum/green_yeast.html
|
|
|
|
|
|Ron Milo
|-
|328
|DNA
|Genome
|Genome Size
|Chlamydomonas reinhardtii
|120*10^6
|Base pairs
|http://en.wikipedia.org/wiki/Chlamydomonas_reinhardtii
|
|
|
|
|
|Ron Milo
|-
|329
|Composition
|Water
|percentage of the total wet weight of cells
|bacteria
|70
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|330
|Composition
|Water
|percentage of the total wet weight of cells
|chicken heart fibroblasts
|77
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|331
|Composition
|Water
|percentage of the total wet weight of cells
|rat kidney cells
|74
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|332
|Composition
|Water
|percentage of the total wet weight of cells
|HeLa cells
|70
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|333
|Composition
|Water
|percentage of the total wet weight of cells
|axon
|87
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|334
|Composition
|Protein
|Percentage by Weight
|bacteria
|20-32
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|335
|Composition
|Protein
|Percentage by Weight
|muscle cells
|23
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|336
|Composition
|Protein
|Percentage by Weight
|red blood cells
|35
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|337
|Composition
|Protein
|Percentage by Weight
|mammalian tissue cells
|17-26
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|338
|Composition
|Protein
|Percentage by Weight
|squid axoplasm
|5
|Percent
|"Cell movements - from molecules to motility", pp. 303, Dennis Bray, 2nd ed. Garland, 2001
|not given
|
|
|
|
|Ron Milo
|-
|339
|Sizes
|Volume
|median cell volume
|Yeast
|42
|micron^3
|http://yeastpheromonemodel.org/wiki/Cell_volume
|Jorgensen et al. 2002 PMID 120894449
|
|The median volume of a haploid yeast cell in exponential phase growing in YPD at 30°C
|
|
|Ron Milo
|-
|340
|Sizes
|Volume
|median cell volume
|Yeast
|37
|micron^3
|http://yeastpheromonemodel.org/wiki/Cell_volume
|Tyson et al. 1979 PMID 374379
|
|The mean volume of a haploid yeast cell in exponential phase growing in YPD at 30°C
|
|
|Ron Milo
|-
|341
|Transcription
|Ribosome
|Percent of total transcription devoted to ribosomal RNA
|Yeast
|60
|Percent
|The economics of ribosome biosynthesis in yeast, JR Warner - Trends Biochem. Sci, 1999
|
|
|
|
|
|Ron Milo
|-
|342
|Transcription
|
|RNA to DNA ratio
|Yeast
|50
|Unitless
|The economics of ribosome biosynthesis in yeast, JR Warner - Trends Biochem. Sci, 1999
|
|
|
|
|
|Ron Milo
|-
|343
|Transcription
|
|Percent of total RNA that is ribosomal RNA
|Yeast
|80
|Percent
|The economics of ribosome biosynthesis in yeast, JR Warner - Trends Biochem. Sci, 1999
|
|
|
|
|
|Ron Milo
|-
|344
|Transcription
|
|Percent of total RNA that is tRNA
|Yeast
|15
|Percent
|The economics of ribosome biosynthesis in yeast, JR Warner - Trends Biochem. Sci, 1999
|
|
|
|
|
|Ron Milo
|-
|345
|Transcription
|
|Percent of total RNA that is ribosomal mRNA
|Yeast
|5
|Percent
|The economics of ribosome biosynthesis in yeast, JR Warner - Trends Biochem. Sci, 1999
|
|
|
|
|
|Ron Milo
|-
|346
|Complexes
|copy number
|Number of ribosomes
|Yeast
|200000
|Unitless
|The economics of ribosome biosynthesis in yeast, JR Warner - Trends Biochem. Sci, 1999
|
|
|Based on comparison of the size of the genome (1.4 3 107 bp) with the RNA in a ribosome (5469 nucleotides), and using ratio of DNA to rRNA
|
|
|Ron Milo
|-
|347
|Growth
|
|Generation time
|Yeast
|100
|minutes
|The economics of ribosome biosynthesis in yeast, JR Warner - Trends Biochem. Sci, 1999
|
|
|
|
|
|Ron Milo
|-
|348
|mRNA
|copy number
|Number of mRNAs in cell
|Yeast
|15000
|Unitless
|The economics of ribosome biosynthesis in yeast, JR Warner - Trends Biochem. Sci, 1999
|Holstege, F. C. P. et al. (1998) Cell 95, 717�$-1òó728
|
|
|
|
|Ron Milo
|-
|349
|Weight
|genome
|Weight of DNA base pair
|generic
|665
|Dalton
|http://www.roche-applied-science.com/PROD_INF/MANUALS/napi_man/pdf/chapter9/page_212-215.pdf
|Ausubel et al., 1988
|
|Notice this is a base PAIR
|
|
|Itai Yanai
|-
|350
|Weight
|genome
|Weight of single stranded DNA base
|generic
|325
|Dalton
|http://www.roche-applied-science.com/PROD_INF/MANUALS/napi_man/pdf/chapter9/page_212-215.pdf
|Ausubel et al., 1988
|
|
|
|
|Itai Yanai
|-
|351
|Weight
|genome
|Weight of single stranded RNA base
|generic
|340
|Dalton
|http://www.roche-applied-science.com/PROD_INF/MANUALS/napi_man/pdf/chapter9/page_212-215.pdf
|Ausubel et al., 1988
|
|
|
|
|Itai Yanai
|-
|352
|Weight
|genome
|Conversion from weight to bp
|generic
|978
|Mbp/pgram
|Gregory, T.R. (2007). Animal Genome Size Database. http://www.genomesize.com
|Dolezel et al., Cytometry 51A: 127-128, 2003
|
|
|
|
|Ron Milo
|-
|353
|Thermosensitivity
|TRP channels
|Q10 (24-36°C) of TRPV4
|mouse
|19.1
|Unitless
|Watanabe et al., J. Biol. Chem. 2002
|
|
|In HEK293 Cell
|
|
|Daniel Ramot
|-
|354
|Thermosensitivity
|TRP channels
|TRPV1 Q10 (41�$-1òó50°C)
|rat
|27
|Unitless
|Liu, Hui, & Qin, Biophys J. 2003
|
|
|Single-channel activity in Xenopus laevis oocytes
|
|
|Daniel Ramot
|-
|355
|Thermosensitivity
|Sensory neurons
|Q10 (47�$-1òó51°C) of heat activated current in DRG neurons
|rat
|17.8
|Unitless
|Vyklický et al., J. Physiol 1999
|
|
|Cultured neurons
|
|
|Daniel Ramot
|-
|356
|Thermosensitivity
|TRP channels
|TRPV3 Q10
|mouse
|6.62
|Unitless
|Peier et al., Science 2002
|
|
|In CHO cells
|
|
|Daniel Ramot
|-
|357
|Thermosensitivity
|K+ Channels
|Q10 (5�$-1òó20°C) of Shaker  K+ channel peak amplitude
|Drosophila
|1.51
|Unitless
|Nobile et al., Exp Brain Res, 1997
|
|
|In Xenopus laevis oocytes - cut open oocyte method
|
|
|Daniel Ramot
|-
|358
|Thermosensitivity
|K+ Channels
|Q10 (5�$-1òó20°C) of Shaker K+ channel activation time constant
|Drosophila
|3.14
|Unitless
|Nobile et al., Exp Brain Res, 1997
|
|
|In Xenopus laevis oocytes - cut open oocyte method
|
|
|Daniel Ramot
|-
|359
|Thermosensitivity
|K+ Channels
|Q10 (5�$-1òó20°C) of Shaker K+ channel inactivation time constant
|Drosophila
|7.2
|Unitless
|Nobile et al., Exp Brain Res, 1997
|
|
|In Xenopus laevis oocytes - cut open oocyte method
|
|
|Daniel Ramot
|-
|360
|Thermosensitivity
|K+ Channels
|Q10 (5�$-1òó20°C) of Shaker  K+ channel ratio b/w steady-state & peak current
|Drosophila
|2.95
|Unitless
|Nobile et al., Exp Brain Res, 1997
|
|
|In Xenopus laevis oocytes - cut open oocyte method
|
|
|Daniel Ramot
|-
|361
|Thermosensitivity
|TRP channels
|TRPV1 Q10
|
|20.6
|Unitless
|Welch et al., PNAS 2000
|
|
|
|
|
|Daniel Ramot
|-
|362
|Thermosensitivity
|TRP channels
|TRPV1 Q10
|
|25.6
|Unitless
|Vlachová et al., J. Neurosci. 2003
|
|
|
|
|
|Daniel Ramot
|-
|363
|Copy numbers
|Small Molecule Copy Numbers
|Number of Zn ions in YPD
|yeast
|8e7
|
|Eide et al. Genome Biology 2005
|
|
|
|
|
|Mike Springer
|-
|364
|Copy numbers
|Small Molecule Copy Numbers
|Number of Se ions in YPD
|yeast
|6e7
|
|Eide et al. Genome Biology 2005
|
|
|
|
|
|Mike Springer
|-
|365
|Copy numbers
|Small Molecule Copy Numbers
|Number of S ions in YPD
|yeast
|5e8
|
|Eide et al. Genome Biology 2005
|
|
|
|
|
|Mike Springer
|-
|366
|Copy numbers
|Small Molecule Copy Numbers
|Number of P ions in YPD
|yeast
|5e9
|
|Eide et al. Genome Biology 2005
|
|
|
|
|
|Mike Springer
|-
|367
|Copy numbers
|Small Molecule Copy Numbers
|Number of Ni ions in YPD
|yeast
|2e7
|
|Eide et al. Genome Biology 2005
|
|
|
|
|
|Mike Springer
|-
|368
|Copy numbers
|Small Molecule Copy Numbers
|Number of Na ions in YPD
|yeast
|2e8
|
|Eide et al. Genome Biology 2005
|
|
|
|
|
|Mike Springer
|-
|369
|Copy numbers
|Small Molecule Copy Numbers
|Number of Mn ions in YPD
|yeast
|5e5
|
|Eide et al. Genome Biology 2005
|
|
|
|
|
|Mike Springer
|-
|370
|Sizes
|Volumes
|Nuclear volume (average)
|S. cerevisiae
|2.9
|um3
|Jorgensen et al.  MBC 2007
|
|
|Morphometry
|
|SD 2% glucose medium, wild-type haploid cells
|Paul Jorgensen
|-
|371
|Sizes
|Volumes
|Nuclear:cell volume ratio
|S. cerevisiae
|0.07
|Unitless
|Jorgensen et al.  MBC 2007
|
|
|
|
|SD 2% glucose medium, wild-type haploid cells
|Paul Jorgensen
|-
|372
|Sizes
|Volumes
|Cell volume
|S. cerevisiae
|42
|um3
|Jorgensen et al.  MBC 2007
|
|
|Coulter counter
|17
|SD 2% glucose medium, wild-type haploid cells
|Paul Jorgensen
|-
|373
|General statistics
|Proliferation
|Number of cell divisions in an average human lifespan
|H. sapiens
|10e17
|Cell divisions
|Weinberg, R. The Biology of Cancer 2007
|
|
|
|
|
|Paul Jorgensen
|-
|374
|Sizes
|Dimensions
|Cell diameter
|S. cerevisiae
|5
|um
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|375
|Sizes
|Volumes
|Cell volume
|S. cerevisiae
|66
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|376
|Sizes
|Dimensions
|Cell dimensions
|S. pombe
|2 x 7
|um
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|377
|Sizes
|Volumes
|Cell volume
|S. pombe
|22
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|378
|Sizes
|Dimensions
|Cell diameter
|Mammalian cell
|10 - 20
|um
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|379
|Sizes
|Volumes
|Cell volume
|Mammalian cell
|500 - 4000
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|380
|Sizes
|Dimensions
|Cell dimensions
|E. coli
|1 x 3
|um
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|381
|Sizes
|Volumes
|Cell volume
|E. coli
|2
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|382
|Sizes
|Organelle size
|Mitochonrdia dimensions
|Mammalian cell
|1
|um
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|383
|Sizes
|Organelle size
|Mitochondria volume
|Mammalian cell
|0.5
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|384
|Sizes
|Organelle size
|Nuclear diameter
|Mammalian cell
|5 - 10
|um
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|385
|Sizes
|Organelle size
|Nuclear volume
|Mammalian cell
|66 - 500
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|386
|Sizes
|Organelle size
|Chloroplast dimensions
|Plant cell
|1 x 4
|um
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|387
|Sizes
|Organelle size
|Chloroplast volume
|Plant cell
|3
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|388
|Sizes
|
|Virus diameter, head only
|Bacteriophase Lambda
|6.6e-5
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|389
|Sizes
|
|Virus diameter, head only
|Bacteriophase Lambda
|50
|nm
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|390
|Sizes
|Protein size
|Ribosome diameter
|?
|30
|nm
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|391
|Sizes
|Protein size
|Ribosome volume
|?
|1.4e-5
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|392
|Sizes
|Protein size
|Globular protein diameter
|?
|5
|nm
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|393
|Sizes
|Protein size
|Globular protein volume
|?
|6.6e-8
|um3
|Roskams and Rodgers, LabRef
|
|
|
|
|
|Paul Jorgensen
|-
|394
|Transporters
|Rates
|Water flow through aquaporin 1 channel
|?
|5*10^8
|molecules/second
|Nelson and Cox, "Lehninger - Principles of biochemistry", 3rd ed. p.410
|
|
|
|
|
|Ron Milo
|-
|395
|Photosynthesis
|Sun
|Flux of photons
|generic
|4*10^21
|Photons/m^2/sec
|Design and engineering of photosynthetic light-harvesting and electron transfer using length, time, and energy scales. Noy D, Moser CC, Dutton PL. PMID: 16457774
|
|
|Intergration
|
|at sea level, based on reference solar irradiance 1.5 air masses, integrated over the whole spectrum
|Ron Milo
|-
|396
|Photosynthesis
|Sun
|Max Photosynthetic photon flux
|generic
|2000
|micromol/m^2/sec
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.243
|
|
|
|
|in the range 400-700nm on a horizontal plane
|Ron Milo
|-
|397
|Photosynthesis
|Sun
|Mean Photosynthetic photon flux
|generic
|800
|micromol/m^2/sec
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.245
|
|
|
|
|in the range 400-700nm averaged on daytime over the earth's surface on clear days
|Ron Milo
|-
|398
|Photosynthesis
|concentrations
|Chlorophyll concentration
|chloroplast
|30
|mol/m^3
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.243
|
|
|
|
|average
|Ron Milo
|-
|399
|Photosynthesis
|leaves
|Leaf area index
|Biosphere
|4.3
|micromol/m^2/sec
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.245
|
|
|
|
|Total area of all leaves divided by the total land area where vegetation occurs
|Ron Milo
|-
|400
|Photosynthesis
|Cross section
|Effective cross section of chlorophyll for useful photons
|generic
|0.09
|Angstrom^2
|Design and engineering of photosynthetic light-harvesting and electron transfer using length, time, and energy scales. Noy D, Moser CC, Dutton PL. PMID: 16457774
|
|
|
|
|
|Ron Milo
|-
|401
|Photosynthesis
|Absorption
|Maximal absorption rate under sun illumination of chlorophyll pigment
|generic
|4
|1/sec
|Design and engineering of photosynthetic light-harvesting and electron transfer using length, time, and energy scales. Noy D, Moser CC, Dutton PL. PMID: 16457774
|
|
|
|
|
|Ron Milo
|-
|402
|Photosynthesis
|Rates
|Catalysis rates of photosystem units
|generic
|10^2-10^4
|1/sec
|Design and engineering of photosynthetic light-harvesting and electron transfer using length, time, and energy scales. Noy D, Moser CC, Dutton PL. PMID: 16457774
|
|
|
|
|
|Ron Milo
|-
|403
|Photosynthesis
|Rates
|Catalysis rates of photosystem units
|generic
|200
|1/sec
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.243
|
|
|
|
|
|Ron Milo
|-
|404
|Photosynthesis
|copy numbers
|number of chlorophyll pigments in PSI
|Synechococcus elongatus
|96
|unitless
|Three-dimensional structure of cyanobacterial photosystem I at 2.5 A resolution.Nature. 2001 Jun 21;411(6840):909-17.PMID: 11418848
|
|
|
|
|
|Ron Milo
|-
|405
|Photosynthesis
|copy numbers
|number of chlorophyll per reaction center
|plants
|100-700
|unitless
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.245
|
|
|
|
|depending on illumination level
|Ron Milo
|-
|406
|Photosynthesis
|copy numbers
|number of bacteriochlorophylls per reaction center
|bacteria
|40-100
|unitless
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.245
|
|
|
|
|depending on illumination level
|Ron Milo
|-
|407
|DNA damage
|Rates
|number of depurination events
|Mammalian cell
|10000
|bases depurinated per cell per day
|R. Weinberg, The Biology of Cancer
|
|
|
|
|
|Paul Jorgensen
|-
|408
|DNA damage
|Rates
|number of depyrimidination events
|Mammalian cell
|500
|bases depyrimidinated per cell per day
|R. Weinberg, The Biology of Cancer
|
|
|
|
|
|Paul Jorgensen
|-
|409
|DNA damage
|Rates
|number of DNA damage events caused by oxidative damage
|H. sapiens cell
|10e3
|bases damaged per cell per day
|R. Weinberg, The Biology of Cancer
|
|
|
|
|
|Paul Jorgensen
|-
|410
|DNA damage
|Rates
|number of DNA damage events caused by oxidative damage
|Rat cell
|10e4
|bases damaged per cell per day
|R. Weinberg, The Biology of Cancer
|
|
|
|
|
|Paul Jorgensen
|-
|411
|Proteins
|Diffusion rate
|GFP diffusion rate in cytoplasm
|E. Coli
|7.7 ± 2.5
|µm^2/s
|Elowitz, M.B. et al. (1999) Protein Mobility in the Cytoplasm of E. coli, J. Bacteriol 181, 197-203.
|Elowitz, M.B. et al. (1999) Protein Mobility in the Cytoplasm of E. coli, J. Bacteriol 181, 197-203.
|
|FRAP and photoactivation
|
|
|Ron Milo
|-
|412
|Proteins
|Diffusion rate
|GFP diffusion rate in water
|Water
|87
|µm^2/s
|Elowitz, M.B. et al. (1999) Protein Mobility in the Cytoplasm of E. coli, J. Bacteriol 181, 197-203.
|Swaminathan, R., C. P. Hoang, and A. S. Verkman. 1997. Photobleaching recovery and anisotropy decay of green fluorescent protein GFP-S65T in solution and cells: cytoplasmic viscosity probed by green fluorescent protein translational and rotational diffusion. Biophys. J. 72:1900-1907; Terry, B. R., E. K. Matthews, and J. Haseloff. 1995. Molecular characterisation of recombinant green fluorescent protein by fluorescence correlation microscopy. Biochem. Biophys. Res. Commun. 217:21-27
|
|
|
|
|Ron Milo
|-
|413
|Proteins
|Diffusion rate
|GFP diffusion rate in cytoplasm
|Eukaryotic
|27
|µm^2/s
|Elowitz, M.B. et al. (1999) Protein Mobility in the Cytoplasm of E. coli, J. Bacteriol 181, 197-203.
|Swaminathan, R., C. P. Hoang, and A. S. Verkman. 1997. Photobleaching recovery and anisotropy decay of green fluorescent protein GFP-S65T in solution and cells: cytoplasmic viscosity probed by green fluorescent protein translational and rotational diffusion. Biophys. J. 72:1900-1907
|
|
|
|
|Ron Milo
|-
|414
|Proteins
|Diffusion rate
|GFP diffusion rate in mitochondria
|Mitochondria
|20-30
|µm^2/s
|Elowitz, M.B. et al. (1999) Protein Mobility in the Cytoplasm of E. coli, J. Bacteriol 181, 197-203.
|Partikian, A., B. Olveczky, R. Swaminathan, Y. Li, and A. S. Verkman. 1998. Rapid diffusion of green fluorescent protein in the mitochondrial matrix. J. Cell Biol. 140:821-829
|
|
|
|
|Ron Milo
|-
|415
|Sizes
|Dimensions
|Cell length at division
|S. pombe
|13.5
|um
|Das et al.  (2007) MBC 18:2090-2101.
|
|
|Morphometry
|1.3
|
|Paul Jorgensen
|-
|416
|Biosphere
|Photosynthesis
|Global primary productivity - [carbon]
|Biosphere
|10^14
|kg
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.219
|
|
|
|
|
|Ron Milo
|-
|417
|Biosphere
|Atmosphere content
|Total CO2
|Atmosphere
|8*10^14
|kg
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.220
|
|
|
|
|
|Ron Milo
|-
|418
|Biosphere
|lakes and oceans
|Total CO2
|Biosphere
|400*10^14
|kg
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.220
|
|
|
|
|
|Ron Milo
|-
|419
|Biosphere
|Photosynthesis
|Time for O2 replenishment by photosynthesis
|Atmosphere
|2000
|years
|Nobel, P. S. "Plant Physiology", 3rd ed. pp.220
|
|
|
|
|
|Ron Milo
|-
|420
|Copy number
|
|p53
|human
|15 ± 6 x 10^5
|unitless
|Ma et al., PNAS (2005) 102:14266
|
|
|Measured by ELISA, concentration ranges from 0.06 to 0.5microM depending on cell line
|
|MCF7 breast cancer cell line
|Alex Loewer
|-
|421
|DNA
|DNA damage
|DNA double strand breaks induced by ionizing radiation [DSB per Gray per cell]
|human
|36
|unitless
|Rothkamm et al.,