=Computational identification of tumour suppressor protein p53 target genes=
'''Author(s): '''[[Special:Mypage|Siew Chinn Fong, Fiona E.M. Paulin, Alastair M. Thompson]]<br> 
'''Affiliations:''' University of Dundee  <br>
'''Contact:'''email: s.fong@dundee.ac.uk <br>
'''Keywords:''' 'p53' 'tumour suppressor protein' 'response element' 'position weight matrices' 'bioinformatics' 

[[Category:BioSysBio Keywords add_p53]]
[[Category:BioSysBio Keywords add_tumout suppressor]]
[[Category:BioSysBio Keywords add_response element]]
[[Category:BioSysBio Keywords add_position weight matrices]]
[[Category:BioSysBio Keywords add_bioinformatics]]


==Background/Introduction==

The tumour suppressor protein p53 plays a major role in modulating cellular functions such as DNA repair, cell cycle arrest and apoptosis. Mutation in p53 gene is known to be one of the most frequently observed genetic alterations during tumorigenesis. The p53 protein is kept at a very low level, ‘stand-by’ mode under normal cellular conditions. In response to cellular stresses like hypoxia or nucleotide depletion, p53 is activated and binds to the promoter elements of its downstream target genes (p53 response element) and regulates their transcriptions, which initiates the cellular activities that account for most of its tumour suppressor functions. The p53 response element (RE) consists of two repeated copies of 5’-PuPuPuC(A/T)(A/T)GPyPyPy-3’with a spacer of up to 13 bp in between. By identifying genes regulated by p53, we hope to achieve a better understanding of their interactions in maintaining cellular integrity.  

Many studies have been carried out on p53 binding sites and in developing different algorithms to identify p53 RE in human genomes. However, the prediction of p53 target genes based on p53 RE alone might be insufficient. Thus, we investigated these target genes to determine whether additional elements adjacent to the p53 binding site may also have a role in modulating p53 binding. Genes experimentally found to be regulated by p53 were aligned with respect to their p53 REs and six conserved consensus motifs were found present at the upstream of the p53 REs. Each motif is represented by a Positional Weight Matrix (PWM). As a result, a program was written using Perl script based on the criteria of searching for a p53 binding site as well as the six upstream elements as an additional confirmation for the search result. The efficacy of this program in detecting a potential p53 target gene was tested using appropriate statistical tests and validated using lists of experimentally proven p53 target genes.  

This program can be potentially use as a preliminary scanning tool to predict p53 target genes for data generated from high throughput experiments like DNA microarrays. Recently, it has also been used in predicting metabolites potentially regulated by p53 in a cancer cell line when treated with a non genotoxic drug.

(Pu, Purine; Py, Pyramidine)

==Results==
Add your text here

==Images/Tables==
Add your images or tables here

==Materials/Methods==
Add your text here

==Conclusion==
Add your text here

==References==

For further help on editing, please see [http://en.wikipedia.org/wiki/Help:Editing here] 
__NOTOC__