* Add or delete the sections that you require.



==Similarity in expression accounts for correlated evolution of interacting proteins==

'''Author(s): '''[[User:your_oww_username|Luke Hakes]], Simon C Lovell, Stephen G Oliver, David L Robertson<br> 
'''Affiliations:''' The University of Manchester  <br>
'''Contact:'''email: Luke.hakes@postgrad.manchester.ac.uk <br>
'''Keywords:''' 'coevolution' 'protein interaction' 'mirrortree' 'gene expression' 

[[Category:coevolution]] 
[[Category:protein interaction]]
[[Category:mirrortree]]
[[Category:gene expression]]

==Abstract==
Studies of interacting proteins have found correlated evolution of the sequences of binding partners, apparently as a result of compensating mutations to maintain specificity.  Here we analyse co-evolution of interacting proteins in yeast, demonstrating correlated evolution of binding partners in eukaryotes. However, detailed investigation of this apparent co-evolution, focussing on the proteinsâ€™ surface and binding interface, surprisingly leads to no improvement in the correlation.  Thus, we conclude that true co-evolution, as characterised by compensatory mutations between binding partners, is unlikely to be chiefly responsible for the apparent correlated evolution.  Rather, we find that absolute expression level outperforms correlated evolution for predicting interacting protein partners.  A correlation between sequence alignments can also be identified not only between pairs of proteins that physically interact, but also between those that are merely functionally related, i.e., within the same protein complex.  Thus, the correlation between sequence alignments is most likely to be due to levels of gene expression being similar for interacting proteins, thus, influencing the rate of sequence evolution in similar ways. 





For further help on editing, please see [ http://en.wikipedia.org/wiki/Help:Editing here] 

__NOTOC__