=Navigational Control of Bacteria=
<i>The Design, Test & Modelling of a synthetic chemotactic biological system</i>

'''Author(s): '''[[Special:Mypage|James Brown]]<br>
'''Affiliations:''' University of Cambridge - Department of Engineering / Department of Plant Science<br>
'''Contact:'''email: jrb62@cam.ac.uk <br>
'''Keywords:''' 'e.coli' 'navigation' 'chemotaxis' 'control' 

[[Category:BioSysBio Keywords e.coli]] 
[[Category:BioSysBio Keywords navigation]]
[[Category:BioSysBio Keywords chemotaxis]]
[[Category:BioSysBio Keywords control]]

==Technical Abstract==
Synthetic Biology is a rapidly developing field, which sees engineering principles applied to biological systems. Here I focus on chemotaxis, the natural directed motion of a micro-organism toward environmental conditions it deems attractive, with the aim to demonstrate navigational control over bacteria.  

Initially my study focused on the internal mechanism behind the sensory system of E. coli and it’s ability to preferentially swim toward attractants that constitutes chemotaxis. Traditional assays such as swarm plates and the new field of microfluidics were examined with a view to developing a test assay for engineered strains. A review of previous stochastic and deterministic methods for the computational modelling of chemotaxis highlighted the stochastic simulator StochSim and the associated spatial modeller AgentCell as likely starting points for the modelling work that would later be adapted to the chosen maltose system.

Discovery of the periplasmic binding-proteins revealed a possible method for exhibiting control which was far more attractive than attempting to interfere with the closely regulated and highly evolved internal workings of the natural system.  Maltose was identified as a starting point for such work and the investigation progressed to consider placing the essential maltose-binding protein (MBP) under external control. 

The re-engineering of the natural chemotaxis system was ultimately achieved by cloning of three engineered E. coli strains with the critical MBP-encoding malE gene knocked-out. It was later successfully demonstrated on both the macro and micro scale. 

The simulator StochSim was modified from the typical aspartate system to reflect the maltose regulon. The importance of the maltose-binding protein was considered through a series of simulations that examined the dependence on the amount of MBP and the system’s response to various step changes in concentration. 

This proved to be a valid model, with simulated data for the adaptation time to a saturating maltose concentration closely matching experimental data. The dependence of chemotacric response on MBP also appeared to follow the documented experimental case.

The full Masters thesis concerning this work can be found [[http://www.synbio.co.uk/ftp/MEng_JamesBrown.pdf here]]:

==References==
1. Endy, D. (2005) “Foundations for Engineering Biology” Nature 438, 449-453

2. Berg, H. C. and Anderson, R. A. (1973) “Bacteria swim by rotating their flagellar filaments” Nature. 245, 380-382

3. Manson, M. D. et al. (2003) “A sensitive, versatile microfluidic assay for bacterial chemotaxis” PNAS Vol 100 No.9, 5449-5454

4. Whitesides, G. M.  (2003) “Microfluidic devices fabricated in Polydimethylsiloxane for biological studies” Electrophoresis, 
Volume 24, Issue 21, Pages 3563 – 3576

5. Bray, D. et al. (1993) “Computer simulation of the phosphorylation cascade controlling bacterial chemotaxis” Mol. Biol. Cell 4, 469-482

6. Bray, D., & Bourret, R. B. (1995) “Computer analysis of the binding reactions leading to a transmembrane receptor-linked multiprotein complex involved in bacterial chemotaxis.” Mol. Biol. Cell 6, 1367-1380

7. Morton-Firth, C. J., Shimizu, T. S., & Bray, D. (1999) “A Free-energy-based Stochastic Simulation of the Tar Receptor Complex” J. Mol. Biol. 286, 1059-1074

8. Shimizu, T. S., Aksenov, S. V., & Bray, D. (2003) “A Spatially Extended Stochastic Model of the Bacterial Chemotaxis Signalling Pathway” J. Mol. Biol. 329, 291-309

9. Emonet, T. et al. (2005)  “AgentCell :a digital single-cell assay for bacterial chemotaxis” Bioinformatics 21, 2714-2721

10. Boos, W. Shuman, H. (1998) “Maltose System of E. coli: Transport, Metabolism and Regulation” Microbiol. Mol. Biol. Rev 62 No.1 204-22924. 

11. Larsson, G. et al. “Solubility and proteolysis of the Zb-MalE and Zb-MalE31 proteins during overproduction in Escherichia coli” Biotechnology and Bioengineering Volume 90, Issue 2 , Pages 239 - 247

12. Yu, Hyung Suk & Alam, Maqsudul
An agarose-in-plug bridge method to study chemotaxis in the Archaeon Halobacterium salinarum. FEMS Microbiology Letters 156 (2), 265-269.

13. Manson, M. and Boos, W. (1985) “Dependence of maltose transport and chemotaxis on the amount of maltose- binding protein” Biol. Chem., Vol. 260, Issue 17, 9727-9733



__NOTOC__