* Add or delete the sections that you require.

=Set theoretical and algebraic model for redundancies in the genetic code=
'''Author(s): '''[[User:Karl|Karl Heuer]]<br> 
'''Affiliations:''' University of Hamburg <br>
'''Contact:'''email: kmh@exmpl.de <br>
'''Keywords:''' 'syllogisms' 'Shegalkin-Polynomials' 'cellular automaton' 'tetrahedral numbers' 'redundancy' 'bijectivity' 'amino acids'

[[Category:BioSysBio Keywords add_keyword_1]] 
[[Category:BioSysBio Keywords add_keyword_2]]
[[Category:BioSysBio Keywords add_keyword_3]]
[[Category:BioSysBio Keywords add_keyword_4]]
[[Category:BioSysBio Keywords add_keyword_5]]
[[Category:BioSysBio Keywords add_keyword_6]]
[[Category:BioSysBio Keywords add_keyword_7]]

==Background/Introduction==
Codons are similar to syllogisms. Each codon has a complement – an anticodon – and syllogisms have such too. An other point is, that codons and syllogisms consist of 3 of 4 possible building blocks. So 64 syllogisms should exist, but only 19 make sense.
Each syllogism is a set operation, which I can represent with Shegalkin-Polynomials [[#References | [1]]]. This needs a continuous view, which is ordinary in biological systems. It accrues matrices with special properties. They are self-similar, that means they can be generated with a tensor product of their own. Other methods work with power sets or multidimensional tetrahedral numbers. Because of these different generation methods it can be shown, that the matrices can be generated with a cellular automaton. Each cellular automaton is producible with a linear automaton [[#References | [2, 3, 4, 5, 6, 7]]], so there is an other aspect of dependence of the this model and decoding of amino acids. The decoding of amino acids follows the rules of a linear automaton. In order to combine the fields of syllogisms and amino acids a different representation for them is used – a tetrahedral grid. This is possible, because the number of syllogisms and amino acids is not only a square number, it is also a triangle number. ''Prof. Dr. Franke´s'' matrices [[#References | [8]]] let me construct such a grid, because of their structure of a Pascal triangle.

==Results== 
My work shows the existence of bijective mappings between syllogisms and amino acids. This is discernible with the tetrahedral grid. The smaller triangles in the grid can not be ordered arbitrarily without loosing informations. It follows because of triangle rotations. Nevertheless the triangles of redundant coded amino acids with degeneracy 6 (''Arg'', ''Leu'' and ''Ser'') seem to be ordered arbitrarily in the grid. Their distribution in the grid fits to model of family boxes [[#References | [9, 10]]] and to a dynamical model by ''Magini'' and ''Hornos'' [[#References | [11]]]. My model has some other similarities with the last one; for example a basic approach with a Platonic solid (tetrahedron and octahedron). ''Mathematical'' invalid syllogisms are very important in my model too, because in strong logics they do not work every time, but they are useful in Biology.

==Images/Tables==
Fig.1: Red marked triangles in the left tetrahedral grid are valid syllogisms.
Black framed triangles mean, that the amino acid with the same colour is assigned to one syllogism. The triangles with red font are stop-sequences and are not assigned to a syllogism.
The triangle with green font is the start-sequence and is not assigned to a syllogism (Gödel, DeMorgan).
The other triangles with yellow font are not assigned to a syllogism in the left tetrahedral grid. They are assigned to so called invalid syllogisms i.e. d'''a'''r'''a'''pt'''i'''.
----
Fig.2: This matrix is an example for ''Prof. Dr. Franke´s'' matrices [[#References | [8]]], which are used in my work. This matrix belongs to a boolean function with three variables.
[[Image:pic1.jpg|thumb|200px|right|Fig.1 Bijective mapping with tetrahedral grid of syllogisms and of coded amino acids]]
[[Image:pic2.jpg|thumb|200px|right|Fig.2 Coefficient matrix of a boolean function with three variables]]

==Conclusion==
The bijective mapping could explain ''Prof. Dr. Freeland´s'' presumption [[#References | [12]]], that fewer amino acids existed in the past. ''Mathematical'' invalid syllogisms are very interesting here. But the further work is also going to aim on a more intensive test of the position of the amino acids and syllogisms. The positions seem to have something in common with chemical properties. This would help to explain that amino acids, which drift from the genetic code, have similar chemical properties i.e. hydrophobia. ''Prof. Dr. Freeland'' wrote about this [[#References | [12]]] too. Shegalkin-Polynomials shall be used here.

==References==


[1] I.I. Shegalkin, Die Arithmetisierung der symbolischen Logik, (Mat. co, p. 311-377, 1928).
----
[2] S. Wolfram, J.D. Farmer, and T. Toffoli, Cellular Automata: Proceedings of an Interdisciplinary Workshop, (Physica D, vol. 10D, no 1 and 2, 1984).
----
[3] A.K. Dewdney, The Planiverse, (Poseidon Press, 1984).
----
[4] B. Hayes, Tabellenkalkulation, (Spektrum der Wissenschaft, Sonderhaft, Computer-Kurzweil II, p. 46-52, 1988).
----
[5] B. Hayes, Endliche Automaten, (Spektrum der Wissenschaft, Sonderheft, Computer-Kurzweil II, p. 53-59, 1988).
----
[6] B. Hayes, Zelluläre Automaten, (Spektrum der Wissenschaft, Sonderheft, Computer-Kurzweil II, p. 60-67, 1988).
----
[7] A.K. Dewdney, Lineare Automaten, (Spektrum der Wissenschaft, Sonderheft, Computer-Kurzweil II, p. 68-73, 1988).
----
[8] D. Franke, Sequentielle Systeme, (Vieweg Verlag, ISBN 3-528-06527, 1994).
----
[9] S. Osawa, T.H. Jukes, K. Watanabe, and A. Muto, Microb. Rev. 56 229 (1992).
----
[10] S. Osawa, Evolution of the Genetic Code, (Oxford University Press, Oxford, 1995).
----
[11] M. Magini, and J.E.M. Hornos, A Dynamical System for the Algebraic Approach to the Genetic Code, (Brazilian Journal of Physics, vol. 33, no. 4, 2003).
----
[12] S.J. Freeland, and L.D. Hurst, Der raffinierte Code des Lebens, (Spektrum der Wissenschaft, Dossier, Das neue Genom, p. 18-25, 2006).