=SHARKview: a tool for the visualization of systems biology data=
'''Authors: '''John M Starkovich, Peter Li, David L Robertson and [[User:bmbjwp|John W Pinney]]<br> 
'''Affiliations:''' University of Manchester  <br>
'''Contact:''' john.pinney@manchester.ac.uk <br>
'''Keywords:''' 'Systems Biology' 'BioPAX' 'SBML' 'visualization' 

[[Category:BioSysBio Keywords Systems_Biology]] 
[[Category:BioSysBio Keywords BioPAX]]
[[Category:BioSysBio Keywords SBML]]
[[Category:BioSysBio Keywords visualization]]

==Background/Introduction==
SHARKview is a freely available program to enable the interactive exploration of biological models encoded in the BioPAX or SBML exchange formats. This includes the ability to browse a model by pathway, physical entity, or interaction as well as the ability to overlay quantitative data such as gene expression levels onto the components of a model. 
==Results== 
Using ''Saccharomyces cerevisiae'' models obtained from online resources in conjunction with expression data from several different studies, we show how SHARKview can be used to visualise mRNA expression level changes for the proteins composing a pathway.  Pathways analysed include both metabolic and genetic regulatory pathways. 
==Conclusion==
Beyond simply displaying the model structure, the facility to incorporate and visualise transcriptomic, proteomic or metabolomic data in the context of a number of different cellular pathways will make SHARKview a very useful tool for systems biologists.
==Images/Tables==
[[Image:sv1b.png | 650px]]

__NOTOC__