=Modelling the cAMP pathway using BioNessie, and the use of BVP techniques for solving ODEs= 
'''Author(s):''' Ms Xu Gu and Prof. David Gilbert <br> 
'''Affiliations:''' Bioinformatics Research Centre, University of Glasgow, Scotland, UK<br>
'''Collaborators:''' Prof. Des Higham and Prof. Miles Houslay<br>
'''Contact:''' gux@dcs.gla.ac.uk<br>
'''Keywords:''' 'BioNessies', 'cAMP signalling pathway', 'Ordinary differential equations', 'Boundary value problems'  

[[Category:BioSysBio Keywords add_keyword_1]]
[[Category:BioSysBio Keywords add_keyword_2]]
[[Category:BioSysBio Keywords add_keyword_3]]
[[Category:BioSysBio Keywords add_keyword_4]]

===Abstract===
Biochemists often conduct experiments in-vivo in order to explore observable behaviours and understand the dynamics of many inter-cellular and intracelluar processes. However an intuitive understanding of their dynamics is hard to obtain because most pathways of interest involve components connected via interlocking loops. Formal methods for modelling and analysis of biochemical pathways are therefore indispensable. To this end, ODEs (Ordinary Differential Equations) have been widely adopted as a method to model biochemical pathways bacause they have an unambiguous mathematical format and amenable to rigorous quantitaive analysis. BioNessie (www.bionessie.com) is a workbench for the composition, simulation and analysis of biochemical networks which is being developed by the Systems Biology team at the Bioinformatics Research Centre as a part of a large DTI funded project 'BPS: A Software Tool for the Simulation ana Analysis of Biochemical Networks' (www.brc.dcs.gla.ac.uk/projects/dti_beacon).

cAMP is an ackowledged second messenger which is involved in mediating the activion of a wide range of host of processes in specific cells, for example, the control of various metabolism events, gene expression, cell growth and division and cell apotosis, muscel contraction, secretion and memory. Part of the aims of my PhD project are to use ODE modelling technique to investigate the dynamic behaviour of cAMP signalling pathway and its connections between ERK pathway for which no computational model currently exists. BioNessie has been used for the modelling and simulation of cAMP pathway model. 

The computational modelling of signalling pathways using ODEs relies on exact values for rate constants and initial concentrations; however, the lack of sufficient data seriously limits the use of this approach. Thus a further aim of my project is to address this chanllenge by investigating methods to permit users to explore the range of possible solutions for a model by taking into account inequality constraints in order to permit an ODE solver to operate over partial data. Towards the end, I have implemented a two-point BVP (Boundary Value Problem) solver in Matlab, for the solution of problems where only partial data is available. In contrast to other BVP solver that require exact boundary value conditions to be provided, this solver can solver inequality constraints and plot solution with certain error distributions. While not in a polished form, the use of the solver illustrates the usability and flexibility of the preliminary version. To summarise, the resulting project will produce a powerful analytical tool for rapid and extensive analysis and testing of models of biochemical networks in order to provide novel hypotheses that can then be evaluated experimentally with data generated by Professor Houslay's wet laboratory.
==Results==

==Images/Tables==
Add your images or tables here

==Materials/Methods==
Add your text here

==Conclusion==
Add your text here

==References==

For further help on editing, please see [http://en.wikipedia.org/wiki/Help:Editing here] 
__NOTOC__