* Add or delete the sections that you require.

= Chaos Game Representation of Mitochondrial DNA: Is it useful in phylogenetic studies? =

'''Author(s): '''Iman Tavassoly, Omid Tavassoly, Mohammad Soltany Rezaee Rad<br>
'''Affiliations:''' Mazandaran University of Medical Sciences and Tarbiat Modares University, IRAN <br>
'''Contact:'''iman_tavassoly@hotmail.com <br>
'''Keywords:''' chaos game representation, mtDNA, evolution 

[[Category:BioSysBio Keywords add_keyword_1]] 
[[Category:BioSysBio Keywords add_keyword_2]]
[[Category:BioSysBio Keywords add_keyword_3]]
[[Category:BioSysBio Keywords add_keyword_4]]

==Background/Introduction==
Chaos Game Representation (CGR) is an iterative mapping method which can convert a nucleotide sequence to a unique and scale-independent image.
There are some studies indicating different applications of CGR images of nucleotide sequences but there is still limited information on their possible application in phylogenetics. In this study we have tried to have a evolutionary evaluation of CGR images of mtDNA in different species using different image processing tools.

==Results== 
Our results showed that there are some evolutionary information in CGR images of mtDNA.

==Images/Tables==
Add your images or tables here

==Materials/Methods==
The algorithms of chaos game for four points as four bases in nucleotide sequences were coded in Matlab 7.0 language licensed by the Mathworks Inc. (http://www.mathworks.com). We produced CGR images of complete mtDNA genomes of 15 species.
We used different methods of image processing for comparing these images in different species.

==Conclusion==
In CGR images of a DNA sequence both global and local patterns are displayed. These images allow us to investigate patterns in the sequence and help our eye to recognize hidden structures. Our results indicated that CGR images of mtDNAs may be useful tools in phylogenetic studies although we need more studies to build a hypothesis about possible applications  of these images in molecular evolutionary studies.

==References==


__NOTOC__
1. C.L. Berthelsen, J.A. Glazier, M.H. Skolnick, Global fractal dimension of human DNA sequences treated as pseudorandom walks. Physical Review A. 45 (1992) 8902-8913.

2. B. Hao, H.C. Lee, S. Zhang, Fractals related to long DNA sequences and complete genomes, Chaos, Solitons, Fractals 11 (2000) 825-836.
3. N.N. Oiwa, J.A. Glazier, Self-similar mitochondrial DNA, Cell Biochemistry and biophysics 41 (2004) 41-61.

4. H.J. Jeffrey, Chaos game representation of gene structure, Nucleic Acids Res. 18 (1990) 2163-2170.

5. D.L. Hartl, E.W. Jones, Genetics analysis of genes and genomes, 5th ed. (Jones and Bartlett Publishers INC, 2001) p 696.

6. J.A. Glazier, S. Raghavachari, C.L. Berthelsen, M.H. Skolnick Reconstructing phylogeny from the multifractal spectrum of mitochondrial DNA, Physical Review E 5 (1995) 2665-2668.

7. P.J. Deschavanne, A. Giron, J. Vilain, G. Fagot, B. Fertil, Genomic signature: characterization and classification of species assessed by chaos game representation of sequences. Mol.Biol.Evol. 16 (1999) 1391-1399.

8. J.C. Sprott, Chaos and time-series analysis, 1st ed. (Oxford University Press, New York, 2004) pp 336-339 and 358-364.