* Add or delete the sections that you require.

=Abstract: Predicting gene co-expression from CIS-regulatory regions=
'''Author(s): '''[[User:jamieallen|Jamie Allen,Ajay Kohli,Ehsan Mesbahi]]<br> 
'''Affiliations:''' University of Newcastle upon Tyne  <br>
'''Contact:'''email: jamie.allen@ncl.ac.uk <br>
'''Keywords:''' 'add-keyword_1' 'add_keyword_2' 'add_keyword_3' 'add_keyword_4' 

[[Category:BioSysBio Keywords add_keyword_1]] 
[[Category:BioSysBio Keywords add_keyword_2]]
[[Category:BioSysBio Keywords add_keyword_3]]
[[Category:BioSysBio Keywords add_keyword_4]]

==Background/Introduction==
Single Nucleotide Polymorphisms (SNPs) affecting phenotype are
frequently found in the CIS regulatory factors of a gene. However
current prediction tools ignore positional data and thus over estimate
the co-expression of genes. In order to reduce this error, this work 
intends to produce a method of correlating quantitative trait loci
(QTLs) simultaneously to their CIS element and the expressed phenotype.

To decipher the cryptic code in this non-coding DNA autoassociative
neural network tools and multidimensional self-organising maps (Kohenen
maps) are being used.

Preliminary work produced a tool using a 2 dimensional Kohenen map has
produced some promising results. This work aims to further improve the 
prediction rate by increasing the dimensions used in the self-organising
map so more positional data is incorporated.

Once an effective prediction tool has been developed it will be trained
using existing genomic data from Arabidopsis pollen and its 
effectiveness assessed.

Once trained the prediction tool can then be used on novel data and the
predictions, if not available in published literature can be tested in
vitro using Transcript and Locus profiling techniques within the 
university.

The ultimate aim being to use the prediction tool on regions of the rice
genome mapped for high density heterosis QTLs to predicted co-regulated
genes. Then using in-house genomics techniques meaningful biological 
information may be assigned to the co-expression.

==Results== 
Results are currently being analysed.

==Images/Tables==
n/a at present.

==Materials/Methods==
All work computer based. Scripts have been written in Perl, Kohenon map has been written in LabView and will be translated to Java.

==Conclusion==

==References==

For further help on editing, please see [ http://en.wikipedia.org/wiki/Help:Editing here] 

__NOTOC__