* Add or delete the sections that you require.

=A systems biology approach to modelling Tea (<i>Camellia sinensis</i>)=
'''Author(s): '''Marshall, A.A.<sup>1</sup>, Gollapudi, S.<sup>1</sup>, De Silva, J.<sup>2</sup> & Hodgman, C.<sup>1</sup><br> 
'''Affiliations:''' <sup>1</sup>University of Nottingham and <sup>2</sup>Unilever<br>
'''Contact:'''  email: alex[at]mycib.ac.uk <br>
'''Keywords:'''  '<i>Camellia sinensis</i>', 'Cross-species array', 'Network analysis'

==Abstract==
Tea manufacture induces a variety of stresses that affect Tea quality. We are using microarray data to track transcriptional changes ocurring during wounding and withering of the leaves to identify metabolic pathways that could influence Tea aroma and flavour. Current transcriptomic approaches include the use of a partial, Tea-specific array. In order to monitor a larger number of genes we have performed cross-species analyses using Affymetrix Arabidopsis genome arrays [1]. Arabidopsis metabolic SBML [2] network data from AraCyc [3], KEGG and Reactome were collated and merged, then subsequently overlaid with the Tea expression data. Subnetworks were constructed by connecting the shortest paths between the differentially expressed genes and the downstream secondary metabolites, therefore identifying potential pathways involved in these stress responses.

http://www.mycib.ac.uk/~alex/Conference/pic.png
<p>This figure shows Cytoscape [4] layouts of (a) the merged AraCyc, KEGG and Reactome network, (b) the AraCyc metabolic network with gene identifiers, (c) the subgraph based on the Tea wounding and withering expression data.  

==Conclusion==
We present the initial output of this project and address how cross-species expression data can be used to colour a network and analysed using a variety of subgraph analyses. 

==Acknowledgments==
Many thanks to the NASC Arrays X-species service, Peter Clarke, Shao Chih Kuo and Thomas Spriggs for their help throughout the project. 

==References==

1. Hammond J.P, Broadley M.R, Craigon D.J, Higgins J, Emmerson, Z.F., Townsend, H.J., White, P.J. & May, S.T. (2005) Using genomic DNA-based probe-selection to improve the sensitivity of high-density oligonucleotide arrays when applied to heterologous species. Plant Methods 2005, 1:10

2. Hucka M, Finney A, Sauro H.M, Bolouri H, Doyle J.C, Kitano H, Arkin A.P, Bornstein B.J, Bray D, Cornish-Bowden A. et al. The Systems Biology Markup Language (SBML): A medium for the representation and exchange of biochemical network models. Bioinformatics 2003, 19: 524-531

3. Mueller L.A, Zhang P. & Rhee, S.Y. AraCyc: A Biochemical Pathway Database for Arabidopsis. Plant Physiology 2003, 132: 453 - 460

4. Shannon P, Markiel A, Ozier O, Baliga NS, Wang J.T, Ramage D, Amin N, Schwikowski B. & Ideker T. Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks. Genome Research 2003, 13: 2498 - 2504

==Links==

Alex Marshall
http://www.mycib.ac.uk/~alex

Multidisciplinary Centre for Integrative Biology
http://www.mycib.ac.uk/

Unilever
http://www.unilever.co.uk/

__NOTOC__