=Protein Annotation=
'''Author(s): '''Jennifer McDowell <br> 
'''Affiliations:''' EBI  <br>
'''Contact:'''  <br>
'''Keywords:''' protein; network; annotation; tutorial

==Abstract==

Modelling networks in Systems Biology benefits from a sound understanding of the proteins making up individual pathways. With the sequencing of various
genomes, many more hypothetical proteins have been catalogued than have been well characterised. Bioinformatics methods can be employed to help predict
the function, structure and interaction potential of these proteins, thereby providing greater insight into the systems being modelled.

===Content===

This tutorial focuses on methods of protein annotation, discussing how to use UniProt, InterPro, IntAct and GO to annotate proteins in a network. Each of these methods complement one another. UniProt provides a comprehensive catalogue of proteins. InterPro provides a  library of signatures, including patterns, motifs, profiles and Hidden Markov Models, gained from integrating 10 major signature databases. These signatures cover over two million proteins, providing predicted functional and structural annotation. IntAct provides protein-protein interaction data, which is being linked to InterPro to provide information on
domain-domain interactions. GO annotation is provided for proteins in UniProt, and is used by InterPro to predict GO terms for uncharacterised proteins, as well as in IntAct to display common GO terms between members of an interaction pathway. Together these methods can provide valuable information for
the proteins modelled within a system.