=Analysis of metabolome data by a maximum likelihood approach=

'''Author(s): '''[[User:Claudi|Claudia Choi]]<sup>1</sup>, Claudia Hundertmark<sup>2</sup>, Bernhard Thielen<sup>3</sup>, Beatrice Benkert<sup>1</sup>, Richard Münch<sup>1</sup>, Max Schobert<sup>1</sup>, Dietmar Schomburg<sup>3</sup>, Dieter Jahn<sup>1</sup>, Frank Klawonn<sup>4</sup> <br>

'''Affiliations:''' <sup>1</sup>Institut für Mikrobiologie, Technische Universitält Braunschweig, Germany; <sup>2</sup>Helmholtz-Zentrum für Infektionsforschung GmbH, Germany; <sup>3</sup>Institut für Biochemie, Universität zu Köln, Germany; <sup>4</sup>Fachbereich Informatik, Fachhochschule Wolfenbüttel, Germany <br>

'''Contact:''' email: c.choi AT tu-bs.de <br>

'''Keywords:''' 'metabolomics' 

[[Category:BioSysBio metabolomics]] 


==Abstract==
Metabolomics emerges as one key aspect of systems biology, since quantifying the dynamic set of metabolites reveals the effect of altered gene expression and protein pattern and thus complements transcriptomics and proteomics. By high-throughput techniques, such as measuring metabolites by gas chromatography-mass spectrometry (GC-MS), enormous data amounts are produced, that need to be analysed. At present, a variety of methods are available for cluster analysis of metabolome data.<br>
Our maximum likelihood approach identifies significantly altered metabolites between <i>Pseudomonas aeruginosa</i> samples grown under different conditions and measured by GC-MS. <i>P. aeruginosa</i> is a versatile soil bacterium and an important opportunistic pathogen causing persistent infection in immunocompromised patients. This statistical approach estimates the inherent noise of the samples and thereby evaluates the significance of altered metabolite composition. Identified key metabolites with significantly altered pattern under different conditions, will be interesting for further investigation of the metabolic network and flux analysis.


__NOTOC__