<br>
==''The NetBuilder' project: development of a tool for constructing, simulating, evolving, and analysing complex regulatory networks''==
<br>

'''Author(s): ''' Katja Wegner, Johannes Knabe, Mark Robinson, Attila Egri-Nagy and Maria Schilstra<br> 
'''Affiliations:''' STRI Biocomputation, University of Hertfordshire  <br>
'''Contact:''' email: k.wegner@herts.ac.uk <br>
'''Keywords:''' 'modelling' 'regulation' 'GRN' 'Petri Nets' 

[[Category:BioSysBio modelling]] 
[[Category:BioSysBio regulation]]
[[Category:BioSysBio GRN]]
[[Category:BioSysBio Petri Nets]]
<br>



<html>
<font face="Arial">
<head>

<title>NetBuilder' abstract</title>
</head>

<h2>Regulatory networks</h2>
<p>Regulatory networks specify how processes such as gene expression 
or cell-cell signalling are controlled by regulators. Such regulatory networks 
may contain feedback loops, non-linear 
interactions and processes working on varying timescales, and can be very 
complex. Moreover, regulators often act in concert. Particularly in genetic 
regulatory networks, regulatory agents such as transcription factors may take 
part in various combinations, each of which may have a specific effect, and may 
be present at specific developmental stages in specific cells or tissues. To handle this complexity and to understand different relationships in 
the network, modelling and simulation techniques are used.  &quot;In silico&quot; 
techniques enable researchers to study networks in a very flexible, cheap and 
fast way compared to &quot;in vitro&quot; and &quot;in vivo&quot; experiments. Therefore, we are developing
<a href="http://strc.herts.ac.uk/bio/maria/Apostrophe/">NetBuilder'</a>, a tool 
for integrating the construction, simulation, evolution, and analysis of complex 
regulatory networks. NetBuilder' uses the Petri Net formalism to aid the 
modelling and analysis processes, and to support understanding of the network 
structure and the role of the various elements.  </p>

<br>

<h2>NetBuilder'</h2>
<p>NetBuilder'  consists of a model designer and a simulation engine, 
and modules for visualisation and analysis of the network structure and 
simulation results are under development.</p>
<ul>
  <li><b>Network construction:</b> the creation of Petri Net network representations is 
  facilitated by a graphical model designer, but can also be done relatively 
  easily (and in principle in a more flexible way) by programming the 
  appropriate commands in a Python driver module. </li>
  <li><b>Simulation:</b> NetBuilder' automatically converts the Petri Net model into 
  the set of vectors required to perform the simulation of its dynamics (a state 
  vector, a stoichiometric matrix, and a vector containing information about the 
  regulatory interactions). This mathematical model can then be simulated by using either 
  stochastic (Gillespie algorithm) or deterministic (standard numerical ODE 
  integration) methods. Techniques for carrying out hybrid 
  stochastic-deterministic simulations are being developed and implemented.</li>
  <li><b>Evolution:</b> we have developed a (genetic) algorithm to evolve genetic 
  regulatory networks that show a specified output behaviour on the basis of a 
  given input function. We are currently attempting to generalise the procedures 
  somewhat, and make them suitable for integration into NetBuilder'.</li>
  <li><b>Analysis:</b> methods for reducing the complexity of networks, for finding 
  network motifs, and for comparing dynamic behaviour are being developed, 
  implemented, and integrated. </li>
  <li><b>Interoperability:</b> NetBuilder' supports SBML. Connection to Systems Biology 
  Workbench (offering further simulation and analysis tools) is planned. </li>
</ul>
<br>

<h4>Emphasis on regulatory interactions </h4>
<p>The model designer and simulation engine can be used to create and simulate 
the dynamics of a wide range of biological reaction networks. Although its 
graphical user interface does not allow definition of arbitrary rate equations 
(a large number of existing modelling and simulation tools can be used to do 
this), NetBuilder is intended to support the specification of complex 
combinatorial regulatory interactions on the basis of a limited set of 
relatively simple primitives (to describe the interaction) and operators (to 
describe their combination).</p>
<br>

<h2>Conclusion</h2>
<p>NetBuilder' aims to offer a range of methods and 
tools that are useful for modelling, simulating, evolving, and analysing complex 
regulatory networks, and it is hoped that it will thereby assist in the 
understanding of the structure and dynamics of these networks.</p>
<br><br>


</font>