<font size=4>
== A global statistical test for improved detection of gene activity ==
</font><br/>
'''David J G Bakewell ''' <sup>1</sup> and '''Ernst Wit''' <sup>2</sup> <br/><font size=2>

Department of Electrical Engineering and Electronics, University of Liverpool, Liverpool, L69 3GJ, UK <sup>1</sup><br/>
Department of Mathematics and Statistics, Lancaster University, Lancaster LA1 4YF, UK <sup>2</sup><br/>
--[[User:Dbakewel|Dbakewel]] 10:54, 29 September 2006 (EDT)
== Abstract ==
This presentation describes a global likelihood ratio test for detecting low levels of genomic (mRNA transcript) activity in a group of related genes or ‘pathway’.  The method yields a far more sensitive detection of differential expression than testing each gene separately. Results are given for published experimental data and simulations. 

== Background and introduction  ==

Interpreting gene transcription for understanding cellular processes using DNA microarray measurements presents a number of challenges to the investigator.  One of these is the relatively large number of genes (mRNA transcript identities) that are measured with relatively few independent biological samples. The small number of independent samples limits the statistical inference that can be made about the behaviour of individual genes.  Consequently, the microarray end-user is faced with bewildering lists of differentially expressed genes that typically contain many false positives - a direct result of the well-known ‘few samples – many genes’ dimensionality problem.  

At the same time investigators are using microarrays to understand processes involving the collective action of a number of genes, often organized as a complex or pathway [1].  To address these needs and remedy the problem of dimensionality, we consider a global, likelihood ratio (LR) test for examining the behaviour of a pre-assigned group of genes - rather than individual genes.
 
This presentation describes the development of the LR test for detecting low levels of activity in a group of related genes or `pathway'.  The underlying principle of the global LR test is to share variance information across related genes so that small concordant changes in expression in a group of genes can be detected more sensitively than current multiple univariate tests.  

The LR test takes into account only the magnitude and uncertainty of the changes, that is, it is not affected by whether the genes within the group are up- or down-regulated.  Compared with other exploratory methods [2] – [4] that find patterns of differentially expressed genes, our method is more a confirmatory method.  It is particularly well suited to investigating the involvement of a particular functional group of genes in explaining the difference between two or more phenotypes.

== Methods  ==

The LR method uses a two-layer hierarchy that models variation between genes within a pathway and experimental variation due to effects of sample preparation, microarray measurements and other sources.  Hierarchical maximum likelihood estimation is used to determine the likelihood ratios, which are in-turn used to determine P-values (i.e. how likely the observed data would be if in fact none of the pathway genes are differentially expressed).  The global maximum of the likelihood is evaluated using standard differential calculus techniques and numerical methods along similar lines described in previous work [5].   Further details are given in the presentation. 

Comparisons are made with P-values evaluated using standard procedures, such as, the univariate regularised t-test [6] with decision rules that take into account the effect of multiple or `sequential' testing.  In the examples described in the presentation, the Family Wise Error Rate is controlled by adjusted P-values calculated using the Bonferroni correction or Hochberg multiple step-down procedure [7] – [9].

== Results == 

The P-values determined by LR and sequential t-test methods are compared using data from Monte Carlo (MC) simulations and a published two-channel spotted microarray experiment [10].  The MC simulations generated LR and T-statistics that enabled pathways to be identified as True Positive (Tp) and False Positive (Fp) for a particular cut-off value. Counts of Tp and Fp were combined into Receiver Operating Characteristics (ROCs) similar to earlier work [5]. The MC simulations show the hierarchical LR test leads to a more sensitive detection of differential gene-group expression compared to sequential application of the t-test. 

The LR and t-test methods were applied to data about three transforming growth factor genes implicated in the regulation of breast cancer tumorigenesis [10], [11].  The effect of sample size on the P-values for the two methods was investigated by randomly selecting samples (maximum 10) and taking combinatorial averages.  Computed relationships of the P-values versus number of samples shows the LR test requires up to three times fewer independent biological samples to give the same gene-group P-value compared with a multiple comparison t-test.

== Concluding remarks ==
A global, likelihood ratio test is developed that shares variance information across a pre-assigned group of genes.  The test is shown to be a more sensitive detector of differential gene expression than conventional techniques, such as, the multiple comparison t-test. This improvement is particularly advantageous for typical of microarray measurements where only a few independent biological samples are available.

== Acknowledgement ==
D. B. would like to thank Cancer Research U.K. and The Wellcome Trust (Project 062511) for financial support and facilities.

== References ==
<font size=1>
[1]  B. Lewin. Genes VII. Oxford University Press, 2002.

[2] F. Al-Shahrour, R. Díaz-Uriarte, and J. Dopazo. Discovering molecular functions significantly related to phenotypes by combining gene expression data and biological information. Bioinformatics, 21(13):2988-93, 2005.

[3]  R. Breitling, A. Amtmann, and P. Herzyk. Iterative group analysis (iGA): a simple tool to enhance sensivity and faciltate interpretation of microarray experiments. BMC Bioinformatics, 5(34), 2004.

[4]  D. Martin, C. Brun, E. Remy, P. Mouren, D. Thieffry, and B. Jacq. Gotoolbox: functional analysis of gene datasets based on gene ontology. Genome Biology, 5(12:R101), 2004.

[5] D. J. Bakewell and E. Wit. Weighted analysis of microarray gene expression using maximum-likelihood. Bioinformatics, 21(6): 723-729, 2005.

[6]  X. Cui and G. A. Churchill. Statistical tests for differential expression in cDNA microarray experiments. Genome Biology, 4(4):210.1-210.10, 2003.

[7]  S. Dudoit, J. P. Shaffer, and J. C. Boldrick. Multiple hypothesis testing in microarray experiments. Working Paper Series 110, U. C. Berkeley Division of Biostatistics, 2002.

[8] S. Dudoit, Y. H. Yang, M. J. Callow, and T. P. Speed. Statistical methods for identifying differentially expressed genes in replicated cDNA microarray experiments. Statistica Sinica, 7:111-139, 2002.

[9]  Y. Hochberg. A sharper Bonferroni procedure for multiple tests of significance. Biometrika, 75(4):800-802, 1988

[10]  R. B. West, D. S. A. Nuyten, S. Subramanian, T. O. Nielsen, C. L. Corless, P. R. Rubin, K.Montgomery, S. Zhu, R. Patel, T. Hernandez-Boussard, J. R. Goldblum, P. O. Brown, M. van de Vijver, and M. van de Rijn. Determination of stromal signatures in breast carcinoma. PLoS Biology, 3(6):e187, 2005.

[11]  R.J. Akhurst and R. Derynck. Tgf-beta signalling in cancer - a double-edged sword. Trends in Cell Biology, 11(11):S44-51, 2001.