* Add or delete the sections that you require.

=Huntington’s Disease: From Experimental Results to Interaction Networks, Patho-Pathway Construction and Disease Hypothesis=
'''Author(s): '''[[User:gonedu|Eduardo GONZALEZ-COUTO]]<br> 
'''Affiliations:''' Siena Biotech S.p.A.  <br>
'''Contact:'''email: egcouto AT sienabiotech DOT it <br>
'''Keywords:''' 'Neurodegeneration' 'Proteomics' 'Pathways' 'Bioinformatics' 

[[Category:BioSysBio Neurodegeneration]] 
[[Category:BioSysBio Proteomics]]
[[Category:BioSysBio Pathways]]
[[Category:BioSysBio Bioinformatics]]

== Huntington’s Disease: From Experimental Results to Interaction Networks, Patho-Pathway Construction and Disease Hypothesis ==

'''Eduardo Gonzalez-Couto, Silvia Matteoni, Stefano Gotta, Letizia Magnoni, Freddy Heitz, Roberto Raggiaschi, Georg C. Terstappen, and Andreas Kremer'''

'''Siena Biotech S.p.A., Discovery Research, Siena, Italy'''

==Background==
Protein-protein interaction networks and mechanistic pathway models are excellent tools in the drug discovery process. They can be used to identify and select targets for a given disease hypothesis. Combining information from diverse sources, like in house experiments as well as literature, allows further development of interaction networks into detailed descriptions of cellular pathways. Computerized pathway diagrams allow integrating all relevant data regarding a project into one framework by linking the different data sources.
Interaction networks analysis and pathway design tools are used to support target identification and validation activities. Experimental results (e.g. from differential proteomics experiments) are incorporated in protein interaction networks, analyzed and further developed into bio molecular pathopathways including literature findings to understand the underlying modulation mechanisms. The pathway diagrams are also used as communication tools, particularly for interdisciplinary project teams, thus ensuring a common understanding and facilitating critical interrogations about disease hypotheses.
The analyses of experimental results, the initial construction of an HD pathopathway are presented and two mechanistic disease hypotheses are discussed.

==Results== 
Three independent differential proteomics experiments were performed and the modulated proteins, confirmed in at least two experiments, were analyzed in the context of protein networks. In a typical experiment, 121 differential spots were picked and MS identification produced 3671 entries grouped into 359 proteins, an expected average of about 3 proteins per spot[6]. In the end, 48 proteins were confirmed to be modulated by the expression of PolyQ Htt across at least two experiments, and were therefore considered for bioinformatics analysis. The functional process distribution of the confirmed proteins (see Figure 1) indicates that the “Stress response & Chaperones”, “Energy metabolism” and “Proteasome degradation” are the best represented processes in the final results set.<br>
Since the ubiquitin-proteasome system is a particularly important biological process shown to be involved in neurodegeneration[7], the focus was put on the “Proteasome degradation” class to enrich the mechanistic HD pathopathway, originally exported from Panther pathway database[4], using CellDesigner[5] pathway editor. In this class of cellular function, the UCH-L1 protein was of particular interest, since it is known to be associated to Parkinson’s disease[8], Alzheimer’s disease[9], and was described as a genetic modifier of the age of onset of HD[10,11]. As the modulation by PolyQ Htt of UCH-L1 at the RNA level was confirmed by RT-PCR (data not shown); we linked UCH-L1 into our HD pathopathway (see Figure 2) and build disease hypotheses around this protein.<br>
Divergent hypotheses can be elaborated for the role of the protein UCH-L1 in HD. UCH-L1 could play a positive role by contributing to Ubiquitin recycling, and thus maintaining normal Proteasome pathway function. A reduced Proteasome activity would favor the accumulation of insoluble PolyQ Htt. On the other hand, based on the recent discovery that peptide sequences can modulate the toxicity of PolyQ tracts in cis or trans[13], an alternative disease hypothesis can be formulated; the transient interaction between UCH-L1 and PolyQ Htt to recycle Ubiquitin could actually increase the toxicity of the extended PolyQ tract of mutant Htt by initiating the first step of the formation of stable PolyQ Htt aggregates.<br>
We thus tested if the fibrillogenic reference structure, the Josephin domain of Ataxin-3[14,15] recently shown to initiate the aggregation of the entire protein independently of its PolyQ tract[16], shares structural features with UCH-L1. This is indeed the case; the 3D structures of UCH-L1 and of the Josephin domain can be superimposed, as shown in Figure 3.

==Images/Tables==
[[Image:Figure1.PNG|300 px|Figure 1]]<br>
'''Figure 1: Functional classification by cellular process of the proteins confirmed to be modulated in the differential proteomics experiments'''<br>
Numeric values indicate the number of proteins in each class. In agreement with the design of the experiment, the apoptosis process is not present amongst the functional classes.<br>
<br>
[[Image:Figure2v3.png|300 px|Figure 2]]<br>
'''Figure 2: Mechanistic pathopathway linking UCH-L1 in HD'''<br>
Literature and experimental findings are integrated to link proteins modulated by the expression of PolyQ Htt with intracellular pathways. Inside the nucleus of HD patients, the mutant PolyQ Htt gene is transcribed into a messenger RNA with a potential stem secondary structure[12]. Intranuclear aggregation of PolyQ Htt sequesters proteins binding to PolyQ Htt, including CBP, hence reducing the cAMP response element-mediated transcription of the CREB-target genes[1]. UCH-L1 was reported to potentiate CREB-target genes transcription by restoring normal proteasomal degradation of the PKA-regulatory subunit II alpha (PKA-r), PKA activity (PKA-c), and CREB phosphorylation, hence resulting in contextual memory retrieval[9].<br>
<br>
[[Image:Figure3.PNG|300 px|Figure 3]]<br>
'''Figure 3: Common structural features of UCH-L1 and of the fibrillogenic Josephin domain of Ataxin-3'''<br>
The superposition of UCH-L1 (green, 2ETL) with the Josephin domain (orange, 1YZB) reveals a common core containing a large beta sheet surrounded by 3 superposed alpha helices.<br>

==Materials/Methods==
Differential proteomics experiments were performed on PC12 rat cells containing either wild-type or mutant full-length (PolyQ) Huntingtin (Htt) under control of a doxycycline-inducible promoter[1]. Cell extracts were prepared at 0, 12, and 48 hours post-induction to identify proteins involved in pre-apoptotic intracellular events. Protein expression modulation was measured using dye-swapping DIGE technology[2] followed by statistical analysis for spot selection and automated spot picking. The protein content of each picked spot was analyzed by mass spectrometry (MS) for entries matching the UniProt [''http://www.uniprot.org''], and rat ENSEMBL+GENSCAN [''http://www.ensembl.org''] databanks. The MS identification results were stored in a relational database designed in-house following the Proteomics Standards Initiative guidelines [''http://psidev.sourceforge.net/'']. Redundancy introduced by the usage of partially overlapping databanks at the MS identification step was removed and the lists of genes encoding the identified proteins were mapped on networks using MetaCore[3] for analysis. As a starting point, an HD pathway from the Panther pathway database[4] was used and subsequently enriched with proteins and events functionally involved in cellular processes linked to neurodegeneration using CellDesigner[5].
Protein structure superposition was obtained using the DaliLite server [''http://www.ebi.ac.uk/DaliLite/''] and the structures were visualized using the Swiss PDB viewer [''http://www.expasy.org/spdbv/''].

==Conclusion==
Based on the exploration of differential proteomics-based experimental results, an HD pathopathway including proteins modulated by PolyQ Htt expression was started to be developed. The perturbation of the Ubiquitin Proteasome pathway was chosen as the focus from amongst the different cellular functional classes represented in this experimental data set. UCH-L1 was identified as an element of this pathway, and the potential roles this protein could play in HD was revealed by integrating different types of biological results, previously analyzed by a wide spectrum of bioinformatics tools. Validation studies are ongoing to uncover which of the potential mechanisms might be active in HD. Finally, the proteins from the other cellular functional classes represented in the proteomics data set are under investigation, and it is expected that the integration of DNA chip-based studies[17] and RNAi-based screens of focused genes or of the druggable genome[18] will lead to a better mechanistic understanding of HD, thus enhancing the prospects for treatments in the field of rare diseases.

'''Acknowledgments.''' The authors would like to thank Prof. David C. Rubinsztein for the recombinant PC12 cell lines and Dr. Mike Palmer for critical reading of the manuscript.

==References==
[1] KL Sugars, R Brown, LJ Cook, J Swartz, DC Rubinsztein: Decreased cAMP response element-mediated transcription: an early event in exon 1 and full-length cell models of Huntington's disease that contributes to polyglutamine pathogenesis. J Biol Chem 2004, 279: 4988-4999.<br>
[2] R Raggiaschi, S Gotta, GC Terstappen: Phosphoproteome analysis. Biosci Rep 2005, 25: 33-44.<br>
[3] S Ekins, Y Nikolsky, A Bugrim, E Kirillov, T Nikolskaya: Pathway mapping tools for analysis of high content data. Methods Mol Biol 2007, 356: 319-350.<br>
[4] PD Thomas, A Kejariwal, N Guo, H Mi, MJ Campbell, A Muruganujan, B Lazareva-Ulitsky: Applications for protein sequence-function evolution data: mRNA/protein expression analysis and coding SNP scoring tools. Nucleic Acids Res 2006, 34: W645-W650.<br>
[5] H Kitano, A Funahashi, Y Matsuoka, K Oda: Using process diagrams for the graphical representation of biological networks. Nat Biotechnol 2005, 23: 961-966.<br>
[6] SP Gygi, GL Corthals, Y Zhang, Y Rochon, R Aebersold: Evaluation of two-dimensional gel electrophoresis-based proteome analysis technology. Proc Natl Acad Sci U S A 2000, 97: 9390-9395.<br>
[7] EM Hol, DF Fischer, H Ovaa, W Scheper: Ubiquitin proteasome system as a pharmacological target in neurodegeneration. Expert Rev Neurother 2006, 6: 1337-1347.<br>
[8] Y Liu, L Fallon, HA Lashuel, Z Liu, PT Lansbury, Jr.: The UCH-L1 gene encodes two opposing enzymatic activities that affect alpha-synuclein degradation and Parkinson's disease susceptibility. Cell 2002, 111: 209-218.<br>
[9] B Gong, Z Cao, P Zheng, OV Vitolo, S Liu, A Staniszewski, D Moolman, H Zhang, M Shelanski, O Arancio: Ubiquitin hydrolase Uch-L1 rescues beta-amyloid-induced decreases in synaptic function and contextual memory. Cell 2006, 126: 775-788.<br>
[10] P Naze, I Vuillaume, A Destee, F Pasquier, B Sablonniere: Mutation analysis and association studies of the ubiquitin carboxy-terminal hydrolase L1 gene in Huntington's disease. Neurosci Lett 2002, 328: 1-4.<br>
[11] S Metzger, P Bauer, J Tomiuk, F Laccone, S Didonato, C Gellera, P Soliveri, HW Lange, H Weirich-Schwaiger, GK Wenning et al.: The S18Y polymorphism in the UCHL1 gene is a genetic modifier in Huntington's disease. Neurogenetics 2006, 7: 27-30.<br>
[12] R Galvao, L Mendes-Soares, J Camara, I Jaco, M Carmo-Fonseca: Triplet repeats, RNA secondary structure and toxic gain-of-function models for pathogenesis. Brain Res Bull 2001, 56: 191-201.<br>
[13] ML Duennwald, S Jagadish, F Giorgini, PJ Muchowski, S Lindquist: A network of protein interactions determines polyglutamine toxicity. Proc Natl Acad Sci U S A 2006, 103: 11051-11056.<br>
[14] G Nicastro, RP Menon, L Masino, PP Knowles, NQ McDonald, A Pastore: The solution structure of the Josephin domain of ataxin-3: structural determinants for molecular recognition. Proc Natl Acad Sci U S A 2005, 102: 10493-10498.<br>
[15] RP Menon, A Pastore: Expansion of amino acid homo-sequences in proteins: insights into the role of amino acid homo-polymers and of the protein context in aggregation. Cell Mol Life Sci 2006, 63: 1677-1685.<br>
[16] AM Ellisdon, B Thomas, SP Bottomley: The two-stage pathway of ataxin-3 fibrillogenesis involves a polyglutamine-independent step. J Biol Chem 2006, 281: 16888-16896.<br>
[17] A Hodges, AD Strand, AK Aragaki, A Kuhn, T Sengstag, G Hughes, LA Elliston, C Hartog, DR Goldstein, D Thu et al.: Regional and cellular gene expression changes in human Huntington's disease brain. Hum Mol Genet 2006, 15: 965-977.<br>
[18] AL Hopkins, CR Groom: The druggable genome. Nat Rev Drug Discov 2002, 1: 727-730.<br>

__NOTOC__