{{Template:BMCB625}}

<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

==(Homework) Questions==
===Jon===
No question here (it is implied), however, I want you all to think about nuclesome mobility.

Also if you have time play around with the following pdb file: [http://www.pdb.org/pdb/files/1kx3.pdb 1kx3.pdb]

This one is slightly higher resolution, and referred to in their paper more.  It has about 3 times the number of water molecules added in:
[http://www.pdb.org/pdb/files/1kx5.pdb 1kx5.pdb]
--[[User:Chayne|Chayne]] 15:41, 14 May 2007 (EDT)

----

===Chris===
Discussion point 1

An interesting finding in the study be Segal, et al, is that their model supports an idea that nucleosomes appear quite stable near genes that contain a TATA-box, the classical transcription start site for a large number of genes, and appear particularly “likely” to be bound near the translational start site (see figure 5 for translation).  

In support of this model, the histone variant H2A.Z is found to be near the translation start site.

In contrast, the in vivo data suggests that the TATA-box is a relatively nucleosome-depleted region (the “average predicted occupancy of their model is conflicting between the red and purple curves figure 5a).


Discussion Point 2

While Segal, et al, are attempting to look at the global binding, might it be beneficial to take a subset of examples (one or two genes that are “well characterized”) in detail over a temporal study to see if their model holds up during gene activation and/or repression.  This appears to be a critical component to “identifying” positioning and may support or refute their model as noted in Point #1

----
===Mahta===

Q1.  To determine binding affinities of their DNA sequences (Fig 1c-1e)(sequences with additional dinucleotide motifs, removed dinucleotide motifs, and disrupted spacing between the motifs) the authors use the core histone tetramer H3(2)H4(2).  Do you think this is a valid approach for determining binding affinity?  Why or why not?

A1.  In my opinion it would be better to use the H3(2)H4(2) tetramer in association with the two H2A-H2B dimers but the methods used in the paper are valid due to the greater affinity, plus increased number of contacts, with the H3-H4 tetramer with DNA in each nucleosome.  In dissociation studies with increasing salt concentration H2A and H2B are freed from the nucleosome before H3 and H4.  (One could argue that the "full" bending of DNA required for formation of nucleosomes does not take place without the entire octamer present.)

Q2.  What are the benefits of having DNA sequences that "code" for nucleosome organization and transcriptional regulation (transcription factor binding)?  Do you see the competition for binding as an effective mode of regulation?

A2.  It is of great benefit to organisms in terms of control of transcription to have nucleosome binding sites (preferred locations for them) at regulatory sites.  Nucleosomes can be thought of as maybe a more dynamic form of a repressor-corepressor system?  Going from a switch from nucleosome coverage to one in which transcription factors can bind dictates that an area must be subject to constant modulation.   It allows for constant remodeling of the area.  
----

===Larry===
Q1.  Do you think it is a stretch to claim that this discovery might lead to better transgenes?
Q2.  Is the thermodynamic model data sufficient to cause doubt on the nucleosome-nucleosome interations?
----

===Chayne===
Q:  Why might the following quote from the article be, strictly-speaking, an overstatement (from page 777): "Overall, our results establish that genomes encode the positioning and stability of nucleosomes in regions that are critical for gene regulation...", and what is the significance of the caveat in that statement "...critical for gene regulation..."?

A:  The positioning of nucleosomes are dependent upon many factors, such as DNA methylation, competition with other DNA binding proteins, modification state of histones, local concentration of other nucleosomes, etc.  Of this, the genome sequence seems only to be a single piece of the information that determines nucleosome position.  As such, specific sequence motifs that favor nucleosome architecture do so by contributing a limited amount of local thermodynamic stability, making the free-energy change associated with placing a nucleosome at that position either more negative or less positive.  Conceptually, the sequence can provide energy "depressions" for nucleosome structure to "fall into", and if isolated from all other factors, will statistically favor nucleosome formation. In the physiological context of all of the factors that determine nucleosome position, this may or may not be significant, dependent of the relative magnitude of this thermodynamic quantity compared to the other forces that are present at any point in time to determine chromatin structure.

I call their sentence fragment "...critical for gene regulation..." a caveat because it seems that this model is most robust for genes that are critically regulated, that is genes that would most likely be found in a heterochromatic state.  This suggests that the cell must provide more robust levels of signalling (forces) to remodel that chromatin into a euchromatic state.  Thus, the thermodynamic stability offered by the sequence specifc motif at that position results in a stronger silent "default" state, making the gene more tightly regulated.  This could be contrasted to, perhaps, genes that undergo position-effect variegation.  Perhaps the sequence specific motifs are not as strong, thus the factors that determine the nucleosome positions when those genes are down-regulated might be more variable, without a significant "default" energy depression to stabilize nucleosome structures.  PEV might also arise even if the sequences provide energy terms identical to more critically regulated genes, but are regularly being outweighed by the other myriad forces that regulate transcription, eg a high level of expression of transcription factors that compete with the nucleosomes for binding at a gene locus, or even a change in temperature. (Le Chatelier's principle).
--[[User:Chayne|Chayne]] 22:39, 13 May 2007 (EDT)
----