{{Template:BMCB625}}

<div style="padding: 10px; width: 720px; border: 5px solid #B3CD4E;">

==(Homework) Questions==
===Larry===
====Lopes et al====
# What observations lead to the authors' suggestion that discontinuites left randomly behind the fork are not immediately sealed after the passage of the replication fork? 

Answer:  The finding of internal gaps at long distances from the fork with distributions along replicated duplexes that mirrored the length distribution of replicated strands.

[http://www.openwetware.org/wiki/User:LarryGray:Questions Larry's Question Page]

====Heller and Marians====
# How did the author cleverly show that the same DnaB from the lagging strand was involved in leading strand reinitation?

Answer:  The authors exploited the fact that SSB could exclude further binding of DnaB to Wild-type DnaC.  So, the authors pre-loaded DnaB onto the DNA in the absence of SSB then followed up with an SSb incubation that prevented anymore DnaB from loading.  This allowed the authors to conclude that leading strand reinitation involved the same DnaB fro mthe lagging strand.
 
 [http://www.openwetware.org/wiki/User:LarryGray:Questions Larry's Question Page]


----

===Chayne===
#In Lopes, et al, why does ablation of TLS activity in rad14 mutants increase the number of internal ssDNA gaps and shorten the distance of internal gaps from the elongation point?
#Considering what we know from Heller, et al, how could the in vivo conditions in Lopes, et al be altered to predictably shorten the average observed internal ssDNA gap?
--[[User:Chayne|Chayne]] 03:03, 11 April 2007 (EDT)
----
===Jon===


====Lopes et al====
# If ssDNA breaks are not an artifact of the EM preparation and do indeed occur in vivo, how does DNA damage checkpoint facilitate efficient fork progression?

====Heller et al====
# DnaG (primase) is thought to facilitate replication restart on the leading strand after a stalled fork.  What mechanisms does the cell employ to repair these gaps?

[http://www.openwetware.org/wiki/User:Jonfay:Questions Jon's Question Page]


----

===Chris===
====Lopes et al (UV-irrad of ''S. cerevisiae''; EM grids and 2-D gel electrophoresis)====
# Lopes and colleagues never actually demonstrate that the ssDNA is located on the strand opposite the UV photoproducts.  Suggest a possible way in which this may be elucidated.

====Heller and Marians (Biochemical purification and modifications from ''E.coli'' and chemical Mods.)====
# Topic for Discussion Thursday: It appears in both papers that specialized translesion polymerases are needed.  How broadly applicable are these proposed mechanisms (i.e., can we really assume that what occurs in a severely damaged DNA strand is the same process as “healthy” DNA synthesis? Are they specific to single-celled organisms which do not participate in the complex process of apoptosis that is found in multi-cellular organisms)?
--[[User:Chris|Chris]] 00:55, 9 April 2007 (EDT)

----



===Jeremy===




'''''Lopes et al'''''

Despite the remarkable discovery that DNA polymerase can transcribe through UV lesions when yeast cells lack NER, list potential genotoxic consequences of transcribing through these lesions? 

'''Answer:'''

Leaving behind DNA lesions and ssDNA, poses many problems to the cell.  The lack of nucleotide excision repair is sufficient to introduce mutations into the genome by mis-pairing to DNA UV-adducts.  Additionally, ssDNA gaps that are filled in post-replication, would be expected to have abberant chromatin structure, according to current models of DNA replication and chromatin structure.  Current models suggests nucleosomes are added during replication, and subsequently modified.  Filled in ssDNA post-replication, would likely produce hypersensitive sites, if in the right place, this could dramatically upregulate a gene.  It would be interesting to know whether nucleosomes at these sites can be added post-replication, or if neighboring nucleosomes can slide and fill in the gaps to allow for proper chromatin stucture.  


'''''Heller and Marians'''''

Why might the mechanism described by Heller and Marians be considered heretical?

'''Answer:'''

The mechanism proposed by Heller and Marians could certainly be considered heretical.  The mechanism proposed, where DNAG primase fills in ssDNA gaps post replication suggests DNAG can act independently of the transcription machinery.  DNAG is most often associated with the lagging strand during DNA replication, acting in concert with the replication fork and replication machinery.  This mechanism suggests DNAG can be recruited to ssDNA independent of the replication machinery, and independent of an origin of replication, of which is tightly regulated.  This mechanism in some ways shatters the current models of DNA replication and repair, and revives old ones.  The work by Heller and Marians redefines the functionality of DNAG, and mechanisms of repair.