{{Template:20.109}}

===Day 2===
*--[[User:Bryanh|Bryanh]] 17:02, 14 February 2007 (EST)
*'''Purpose:''' to separate dna fragments based on size from vector digest.

==Introduction==
Electrophoresis is a technique that separates large molecules by size using an applied electrical field and a sieving matrix. DNA, RNA and proteins are the molecules most often studied with this technique; agarose and acrylamide gels are the two most common sieves. The molecules to be separated enter the matrix through a well at one end and are pulled through the matrix when a current is applied across it. The larger molecules get entwined in the matrix and retarded; the smaller molecules wind through the matrix more easily and travel further from the well. Molecules of the same size and charge migrate the same distance from the well and collect into a band. 

[[Image:Be109gelmigration.jpg|thumb|center|500px]]
<br style="clear:both" />

DNA and RNA are negatively charged molecules due to their phosphate backbone, and they naturally travel toward the positive charge at the far end of the gel. They are typically examined with agarose gels. Proteins are composed of amino acids that can be positively, negatively or uncharged. To give proteins a uniformly negative charge, they are coated with a detergent, SDS, prior to running them on a gel. Protein samples are also boiled to remove any secondary structure that might make two molecules of the same size migrate differently. Polyacrylamide is the matrix commonly used to separate proteins.  
These gels are typically run vertically while agarose gels are run horizontally but gravity has nothing to do with the separation. 

[[Image:Be109agarosegelsetup.jpg|thumb|left|500px]]

Today you will separate DNA fragments using an agarose matrix. Agarose is a polymer that comes from seaweed and if you’ve ever made Jell-O™, then you already have all the skills for pouring an agarose gel. To prepare these gels, agarose and buffer are microwaved until the agarose is melted. The molten agar is then poured into a horizontal casting tray, and a comb is added. Once the agar has solidified, the comb is removed, leaving wells into which the DNA sample can be loaded. 

[[Image:Be109gelpercentage.jpg|right|thumb|150px]]

The distance a DNA fragment travels is inversely proportional to its length. Over time fragments of similar length accumulate into “bands” in the gel. Higher concentrations of agarose can be used to resolve smaller DNA fragments. This figure shows the same DNA fragments resolved with three agarose concentrations. The 1000 base pair fragment is indicated in each. <br><br><br><br><br>
				 	 
[[Image:Be109agarosegelEM.jpg|left|thumb|250px|'''Scanning EM image of agarose polymer''']]
[[Image:Be109sizevsmigrationdistance.jpg|right|thumb|200px|'''Size vs. migration distance''']]

<br style="clear:both" />

[[Ethidium Bromide]] is a fluorescent dye that is commonly added to agarose gels. This dye intercalates between the bases of DNA, allowing DNA fragments to be located in the gel under UV light and photographed. The intensity of the band reflects the concentration of molecules that size, although there are upper and lower limits to the sensitivity of dyes. Because of its interaction with DNA, '''ethidium bromide is a powerful mutagen''' and will interact with the DNA in your body just as it does with any DNA on a gel. You should always handle all gels and gel equipment with gloves. Agarose gels with Ethidium Bromide must be disposed as hazardous waste in the labelled container in the fume hood. 
 
Today you will run your M13K07 digested samples on an agarose gel, cut the linearized backbone out of your gel and then purify the DNA from the agarose. You will also anneal the insert you designed last time, making a small double stranded fragment of DNA that has sticky ends compatible to the ones in the M13K07 backbone. Next time you will mix the backbone and insert in a ligation reaction.

==Protocols==
===Part 1: Running your gel===
#Add 2.5 ul loading dye to M13K07 samples from last time. Loading dye contains xylene cyanol as a tracking dye to follow the progress of the electrophoresis (so you don’t run the smallest fragments off the end of your gel!) as well as glycerol to help the samples sink into the well.  
#Flick the eppendorf tubes to mix the contents then quick spin them in the microfuge to bring the contents of the tubes to the bottom. 
#Load the gel in the order shown in the table below. <br>To load your samples, draw 25 ul into the tip of your P200. Lower the tip below the surface of the buffer and directly over the well. You risk puncturing the bottom of the well if you lower the tip too far into the well itself (puncturing well = bad!). Expel your sample into the well. Do not release the pipet plunger until after you have removed the tip from the gel box (or you'll draw your sample back into the tip!). 
#Once all the samples have been loaded, attach the gel box to the power supply and run the gel at 125V for no more than 45 minutes. 
#You will be shown how to photograph your gel and excise the relevant bands of DNA. <br>

<center>
{| border="1"
! Lane 
! Sample 
! Volume to load
|-
| 1 
| [http://www.neb.com/nebecomm/products/productN0468.asp 1 kb Marker] 
| 5 ul
|-
| 2 
| M13K07 uncut sample
| all of reaction volume (~30ul)
|-
| 3
| M13K07 digested sample 
| all of reaction volume (~30ul)
|-
| 4 
| empty
| 
|-
| 5
| [http://www.neb.com/nebecomm/products/productN0468.asp 1 kb Marker] 
| 5 ul
|-
| 6 
| M13K07 uncut sample
| all of reaction volume (~30ul)
|-
| 7
| M13K07 digested sample 
| all of reaction volume (~30ul)
|-
| 8
| empty
| 
|-
| 9
| sample of annealed oligos
| from teaching faculty
|-
| 10
| empty
|
|}

[[Image:Be109gelloading.jpg|thumb|center|200px|'''Loading a gel''']]
</center>

===Part 2: Anneal Oligonucleotide insert===
#Begin by looking at the primer record you wrote last time and review your planning of the myc-tag that you designed. Do the single-stranded overhangs still make sense to you? Verify the reading frame. Will the PstI or BamHI site be regenerated if the oligo anneals to the M13 backbone as you hope? What restriction site will you use to check that the oligo is inserted into the backbone? Add that information and any other you think helpful to the [http://openwetware.org/wiki/Talk%3A20.109%28S07%29%3AStart-up_genome_engineering talk page associated with the last lab].
#If new oligonucleotides were ordered for you, then look at the information sheet sent to you by the company. Are the oligos the correct sequence? If they are right, then you should resuspend the samples in water. This is done by first giving the tubes a quick spin in the microfuge to bring down any material that is stuck on the lid. Add sterile water to resuspend the oligos at a concentration of 100 pmoles/ul, and write this concentration on the spec. sheets from the company. 
#You will anneal the oligos in a PCR tube that you can get from the faculty. These are very small (relative to eppendorf tubes) and fit nicely into the holes of the yellow pipet tips boxes. Label the top or side of one tube as best you can with your team ID and the contents of the tube. 
#Mix 10 ul of each of the appropriate oligonucleotides in the PCR tube. 
#Heat the sample to 94° for 2 minutes in the lab's thermal cycler ("PCR machine") and then let it cool slowly. 
#The teaching faculty will store your insert until you need it for cloning next time.

===Part 3: Isolating/Purifying DNA===
To purify your DNA from the agarose, you will use a kit sold by Qiagen. The reagents have uninformative names and their contents are proprietary. The Agarose Purification kit requires binding the DNA to a spin-column with a silica-gel membrane, washing away salts and eluting the DNA from the membrane.

*'''ok, so i think this protocol is overly complicated so im going to do it a slightly simpler way that will still work just as good.'''
*#add 550uL QG to eppendorf with gel in it.
*#vortex until the gel dissolves completely. if you cant get it to dissolve, then heat it at 50C.
*#add 100uL isopropanol to the eppendorf. mix.
*#pour resulting liquid into the QIAquick
*#spin through column. dump out liquid in well.
*#wash column with 750uL PE by spinning it through the column.
*#dry column by spinning it for 2 minutes.
*# elute with 30uL of EB into a clean eppendorf.

#Estimate the volume of your gel slice by weighing it.
#Add 3 volumes of '''QG''' for every 1 volume of agarose, where 100 mg ~ 100 uL. (The maximum advised volume is 550 ul.)
#Incubate at 50&deg;C for 10 minutes, until the agarose is completely dissolved. Every few minutes, you can remove your tubes from the 50&deg;C heat to flick the contents. This will help dissolve the agarose. 
#Add 125 ul of isopropanol to each eppendorf tube. 
#Get a QIAquick spin column (purple) with collection tube (clear) from the teaching faculty. Label the spin column (not the collection tube!) then pipet the dissolved agarose mixture in the top of the column. Microfuge the column in the collection tube for 60 seconds. '''The maximum capacity of the QIAquick columns is 800 uL!''' If you have more than 800 uL in your mixture, you will need to repeat this step.
#Discard the flow-through in the sink and replace the spin-column in their collection tube. Add 750 ul of '''PE''' to the top of the column and spin as before. 
#Discard the flow-through in the sink and replace the spin-column in the collection tube. Add nothing to the top but spin for 60 seconds more to dry the membrane. 
#Trim the cap off a new eppendorf tube and label the side with your team color and the date. Place the spin-column in the trimmed eppendorf tube and add 30 ul of '''EB''' to the center of the membrane.
#Allow the column to sit at room temperature for one minute and then spin as before. The material that collects in the bottom of the eppendorf tube is your purified plasmid backbone, ready to be ligated. Give it to the teaching faculty who will store it with your insert until next time.

===Part 4: Titering phage===
One technique you will see several times this term is plating for plaques. The idea of this technique is simple. Since phage infection slows down the growth of bacteria, any phage-infected cell will grow less quickly than an uninfected one, giving rise to a zone that is more clear on a lawn of fully grown cells. This zone is called a plaque and by counting the number of plaques formed, it is possible to measure the number of infective phage in the sample you are testing. The number of infected phages is measured as PFUs, which is "plaque forming units per ml."
[[Image:Macintosh HD-Users-nkuldell-Desktop-plaques.jpg| thumb|right| 200 px| plaques formed by bacteriophage upon infection of susceptible bacteria]]  
# Start by placing 6 LB plates in the 37° incubator to prewarm them. If there is any condensation on the surface of the plates, then you can leave the lids slightly ajar to dry the plate surface. 
# Aliquot 200 ul of bacteria into 6 small, sterile test tubes. The bacteria you are using are the strain [http://www.neb.com/nebecomm/products/productE4103.asp ER2267] since this strain has a selectible F'. Label the tops of each tube with a colored sticker and one of the following: none, none, 10^-4 E4, 10^-6 E4, 10^-4 K07, 10^-6 K07. 
# The teaching faculty have two phage stocks for you to compare, an M13K07 phage stock and a stock of another M13 phage called E4. You will need to serially dilute each stock, making stepwise 1/100 dilutions in eppendorf tubes. For example, add 10 ul of a phage stock to 990 ul sterile water for a 10^-2 dilution, then repeat, using 10 ul of the 10^-2 dilution into 990 ul sterile water to make a 10^-4 dilution. Vortex the dilutions before removing any liquid and change pipet tips to prepare each new dilution. Continue serially diluting the phage to final concentrations that are 10^-4th and 10^-6th as concentrated as the starting stock.
# Mix 10 ul of one of the 10^-4 dilutions into a tube with bacteria. 
# Mix 10 ul of one of the 10^-6 dilution into another tube with bacteria. 
# Repeat with the dilutions of the other phage stock. 
# One of the teaching faculty will show you how to mix 3 ml of top agar into one of the uninfected samples you have prepared and how to pour the molten mix onto the surface of a prewarmed LB plate. 
# You and your partner should add top agar to the other uninfected sample and the four phage infected ones. 
# Allow the top agar to solidify by leaving the plates on the bench at least 5 minutes then stack them and wrap them with your colored tape and finally move them to the 37° incubator to grow overnight. One of the teaching faculty will remove them from the incubator tomorrow and store them for you until next time. 

DONE!

==Calculations==

'''dilute oligos'''
*an easy way to figure out how much water to add to your dna is to multiply the number of nmol by 10 and then add that number of uL to the dna. this will always dilute to 100uM, or 100pmol/uL as you requested.  our two oligos were 30.2nmol and 30.5nmol and so we added 302uL and 305ul to each sample, respectively.

==Summary/Interpretation==

'''Gel'''
Our digest of the m13ko7 genome with PstI was unsuccessful as can be seen from our gel.  There appears to be a lot of supercoiled plasmid and open-loop plasmid which were not cut and only a very little bit of singly cut plasmid. we interpreted this as a failed digest because it is very unlikely that we'd be able to purify enough dna to be successful in our ligation.  

[[Image:Macintosh HD-Users-nkuldell-Desktop-WFbkbs purplered.jpg|200px|gel before cutting]] [[Image:Macintosh HD-Users-nkuldell-Desktop-WFbkbgel red cut.jpg|200px|gel after cutting]]<br>

'''Phage infection'''
We infected e. coli, strain ER2267, with E4 phage and ko7 phage.  everything went well. next class we will count PFUs.

'''oligo annealing'''
We double-stranded out oligos by heating them up to 92C in the thermocycler and then letting them cool which will allow the two strands to anneal to each other forming the appropriate sticky ends that will be used in ligation into the backbone.

==For next time==
#Take the log<sub>10</sub> of the length of each molecular weight marker you can identify on your agarose gel photograph. Graph the log<sub>10</sub> of their length on the y-axis versus the distance they migrated from the well on the x-axis, measured in mm using a ruler and the picture of your agarose gel. An example of such a graph is found in the introduction to today’s experiment. Use the equation of the line from your graph to determine the size of your M13K07 backbone (use the band in the lane in which you loaded the cut DNA). How does this measurement compare with the predicted size?
# How many plaques do you expect if you plated 10 ul of a 10^-8 dilution of phage, if the titer of phage was 10^12th plaque forming units/ml? How many plaques would you expect if you tested the phage stock on strain [http://www.atcc.org/common/catalog/numSearch/numResults.cfm?atccNum=53868 DH5]?
# Print out the M13K07 plasmid map from NEB by using their [http://tools.neb.com/NEBcutter2/index.php NEB Cutter tool], selecting M13K07 from the "Viral and phage" drop down menu on the right, changing the default minimum ORF to 25 amino acids (do you remember which of the M13 proteins are very small?), and finally telling the program that you are entering circular DNA. Print out two copies of the plasmid map. Modify them by hand to indicate which ORF you are epitope tagging, how many basepairs of DNA you are adding and what restriction enzymes you are adding to or deleting from the plasmid. Keep one copy for your notebook and hand the other copy in next time. 
# The myc-epitope you are adding to p8 or p3 uses traditional genetic engineering ("recombinant") techniques. These are powerful and precise ways to move single genes from one organism to another and to make useful chimeric protein products like the one you are now working on. Synthetic biology is a newer approach to programming cells. Please read one (or more!) of the following articles and then write a paragraph exploring the legitimacy of the following statement: "synthetic biology is about engineering while genetic engineering is about biology."
#*[http://www.thenewatlantis.com/archive/12/tuckerzilinskas.htm New Atlantis, Spring 2006]
#*[http://www.asm.org/microbe/index.asp?bid=47155 Microbe, Fall 2006]
#*[http://www.journals.royalsoc.ac.uk/openurl.asp?genre=article&id=doi:10.1098/rsif.2006.0206 Journal of the Royal Society Interface, Winter 2007]

==Reagents list==
*Loading Dye
**0.25% xylene cyanol
**30% glycerol
**RNase

*1% agarose gel in 1X TAE

*1X TAE
**40 mM Tris-acetate
**1 mM EDTA
**pH 8.3

*[http://www.neb.com/nebecomm/products/productN0468.asp 1 kb Marker] 

*Oligos for adding myc tag to p8
**NO150 = g8p_myc'_Top:       CTC GAA CAG AAg CTt ATC TCT GAA GAA GAC CTt gca
**NO151 = g8p_myc'_Btm: a cgt GAG CTT GTC TTc GAa TAG AGA CTT CTT CTG GA 
**This results in 11aa insertion (Gly +10aamyc) at Ala24, followed by natural seq Glu25 

*Oligos for adding myc tag to p3
**NO154 = g3p_myc'_Top: gat cTA GAA CAG AAg CTt ATC TCT GAA GAA GAC CTA
**NO155 = g3p_myc'_Btm:      AT CTT GTC TTc GAa TAG AGA CTT CTT CTG GAT cta g
**This results in 11 aa insertion (Leu+10aamyc) at D215, followed by natural seq picking up at D215P216F217