--[[User:Bryanh|Bryanh]] 14:09, 20 April 2007 (EDT)

'''purpose:'''
To convert the RNA isolated from our yeast cells into cDNA and hybridize the cDNA to a DNA microarray. 

==Protocols==
===Part 1: cDNA synthesis===
Creating cDNA from RNA is done using an enzyme called reverse transcriptase. Like all DNA polymerases, this enzyme can only add sequence to an existing chain and so needs a short “primer” to begin synthesis. To perform the cDNA synthesis, you will use a kit from a company named Genisphere. The primers in this kit have a special “capture sequence” at their 5’ end. The capture sequences allow the cDNA to be reacted with Cy3 or Cy5 later. 
Before you begin today’s protocol, prepare your bench for working with RNA. This involves cleaning your pipetmen, retrieving your “RNA only” pipet tips and solutions and wiping down your bench. You should work on a fresh piece of benchpaper and remember to wear gloves when working with RNA.

Calculate the volume needed for 2 ug of each RNA sample, then assemble the annealing reactions in RNase-free eppendorf tubes according to the following table. If you need more than 10 ul for 2 ug of RNA then you should calculate the mass in 10 ul of the more dilute sample and match that mass for the more concentrated sample. You cannot put a volume of RNA greater than 10 ul in the tubes.
{| border="1"
!  
! TUBE A  
! TUBE B
|-     
| RNA
| .944 ug of RNA from parental strain
| .944 ug of RNA from deletion strain
|-    
| RNase-free H20
| bring volume to 10 ul
| bring volume to 10 ul 
|-    
| RT primer  
| 1 ul Capture Sequence I <br> vial 11, <font color = red> red </font color>
| 1 ul Capture Sequence II <br> vial 11, <font color = blue> blue </font color>  
|}
#Heat the annealing reactions to 80°C for 10 minutes then place the tubes on ice for 2 minutes. 
#Microfuge the tubes briefly to spin any condensation or droplets down to the bottom of the tube then add 1 ul of "Superase," an RNase inhibitor (Vial 4), and 8 ul of cDNA synthesis cocktail. Because reverse transcriptase is an unstable enzyme, this cocktail must be prepared just before use. The teaching faculty will prepare some for you when you are ready for it. 
#Pipet the contents of your tubes up and down to gently mix, then incubate the cDNA synthesis reactions at 42° for 1.5 hours. During this time, work with the sample array data file that is available (see Part 2 of today’s protocol).
#Microfuge the tubes briefly then add 3.5 ul of 0.5M NaOH/0.5M EDTA to each tube and pipet up and down to mix. Heat to 65°C for 10 minutes. This step will denature your RNA/cDNA hybrids and degrade the RNA. 
#Add 5 ul of 1M Tris, pH7 to neutralize the contents of each tube.

===Part 2: practice array data analysis===
practiced array analysis to make sure we understood how to reduce our data to meaningful imformation.

===Part 3: hybridize microarrays===
The arrays we will use are the [http://www.chem.agilent.com/scripts/pds.asp?lPage=37069 yeast v2 DNA microarrays] from Agilent. Each slide has two arrays , each with 11,000 60-mer oligonucleotides. These “oligos” were first built on glass wafers and then printed onto the slide surface. The oligos represent more than 6000 yeast genes, many spotted on the slide more than once. 

====The success of your experiment is absolutely dependent on the following:====
*You must <b>hold the slides by the edges only.</b> If you touch the array that is printed on the slide’s surface, you will obscure the DNA that is printed there.  
*Each array has a barcode printed on some stickers on one end of each slide. <b>The array is printed on the side of the slide that says “Agilent.” </b> If you try to hybridize your cDNA to the numbered side of the slide, there will be no array there to bind. <br> 

====To hybridize the arrays====
#Begin by mixing the cDNA pools you have synthesized into one eppendorf tube. The total volume should be 57 ul. Add 43 ul of H2O and 100 ul of 2X Hybridization Buffer. Pipet up and down several times to mix the contents. 
#Heat your hybridization solutions to 80° for 10 minutes then cool them to room temperature. During this time you will be shown how to assemble the hybridization chambers. 
#Load your samples into the hybridization chambers as follows:
#*open the gasket and place the "Agilent" sticker facing up in the rectangular side of the holder. 
#*Pipet 200 ul of the hyb solution into place 1 or place 2 (see figure)[[Image:Macintosh HD-Users-nkuldell-Desktop-agilentarray.png]]
#*Place the microarray <b><font color = red> "agilent" sticker down</font color> </b> onto the hybridization solution that's in the gasket. 
#*Slide on the top portion of the hybridization chamber and the brace. 
#*Tighten and tape them with your colored tape. 

When every group is ready, we will walk the arrays over to the BioMicroCenter in Building 68 to put them in the 60°C hybridization oven.

==summary/interpretation==

Took RNA samples and reverse transcribed them to make cDNA.  We hybridized the cDNA to the microarray which will allow us to read the gene expression profile of the genome comparing the mutant to the parent FY2068 strain.  It was a good day's worth of work.