{| border="1" cellpadding="2"
!width="10"|protein/known region
!width="1000000"|function
!width="10"|overlaps?
!width="10"|manipulatable? 
!width="10"|copy number
|-
|IX, VII|| pack together to make blunt end of phage.  33 and 32 AA long, respectively || [[Bryan Hernandez/20.109/M13o7k degisn ideas/overlaps|overlaps]] || not wildtype || 5, 5 ||
|-
|III, VI|| pack together to make the round end of the phage. It's the 1st and last point of contact with host. the secreation of the phage is terminated by adding this cap.|| slightly between III and VI RBS || ?|| 5, 5 ||
|-
|II, V, X||II nicks the dsDNA initiating replication. V competes with II and dsDNA formation by packaging ssDNA for phage production. these need to be at correct ratios. X regulates the + strand accumulation (allows accumulation).  all three of these need to be in harmony with each other.|| [[Bryan Hernandez/20.109/M13o7k degisn ideas/overlaps|overlaps]] || ? || ? ||
|-
|IV, I, XI||these form the channel that lets the phage out of the cell.||yes ||? || ? ||
|-
|Packaging Sequence||IX and VII interact with the V-ssDNA complex here.|| i need to find this sequence ||? || n/a ||
|-
|V, VIII||V is replaced with VIII as it goes through the I, IV, XI pore.|| [[Bryan Hernandez/20.109/M13o7k degisn ideas/overlaps|overlaps]]|| ? || ?, 2700 ||
|}




'''For Next Time'''<br><br>



{| border="1"
! protein
! function
! design ideas
|-
| I 
| assembly
| completely decouple from rest of genes.
|-
| II 
| replication of DNA + strand
| completely decouple from rest of genes.
|-
| III 
| phage tail protein (5 copies)
| 
*completely decouple from rest of genes.
*delete this protein to prevent cap from forming. this might allow for an arbitrarily long phage which could be good for nanowire production.
|-
| IV 
| assembly
| completely decouple from rest of genes.
|-
| V 
| binds ssDNA
| completely decouple from rest of genes.
|-
| VI 
| phage tail protein (5 copies)
| 
*completely decouple from rest of genes.
*delete this protein to prevent cap from forming. this might allow for an arbitrarily long phage which could be good for nanowire production.
|-
| VII 
| phage head protein (5 copies)
| completely decouple from rest of genes.
|-
| VIII 
| phage coat protein (2700 copies) 
| this is the protein that we decided to modify.  Our current design plan is to add an myc epitope tag into the middle of this protein that will hopefully stick off the side of the phage.  if successful, the tag will allow anitbodies to bind.
apparently there are a lot of other cool things that one can do with this surface protein in the field of construction of nanomaterials.  This is a particularly good protein to mess with for making a solar cell, for instance, because the packing of p8 is  very uniform and there are 2700 copies to utilize.  I would like to figure out a way to use two synthetic amino acids so that one can be a binding site for GaN and one can be for LiN (i dont really remember which elements they were.)  then we'd charge half the tRNAs for this position with the amino acid for GaN and half charged with the amino acid for LiN.  this should evenly disperse the GaN and LiN over the surface protein.   
|-
| IX 
| phage head protein (5 copies)
| completely decouple from rest of genes.
|-
| X 
| DNA replication
| 
*completely decouple from rest of genes.
*potentiall mess with this to make a really long genome. this might allow for an arbitrarily long phage which could be good for nanowire production.
|-
| XI 
| assembly
| completely decouple from rest of genes.
|}


#* Would you expect the phage to tolerate p8 modifications that:
#**make the protein neutral rather than charged at the C-terminus?
#***yes?
#**encode all Leucines with the CTA codon instead of the CTG codon?
#***it would tolerate it but not very well probably because im sure there isnt as many tRNAs with the respective anticodon and since p8 is needed at 2700 copies this will be a major bottle neck in the phage production if there isnt enough tRNAs to complete the protein.
#**double the size of the protein? Justify your answers. Please assume that the p8 modifications do not destabilize the protein itself.
#***if the size change didnt interfere with the packing of the proteins then this shouldnt be a problem; however if it did then it wouldnt work.
#* Would you expect the phage to tolerate these same modifications to p3? 
#** yeah, this protein isnt packed very tightly and has a lot of room on the end of the phage.
#* Would you expect the phage to tolerate transcriptional terminators that are 
#**2X stronger
#*** maybe, there is too much orf sharing and polysystronic genes for this.
#**100X stronger
#***no, there is too much orf sharing and polysystronic genes for this.
#**2X weaker
#*** yes, there are a lot of genes that are polysystronic already so this probably wouldnt hurt it that bad.
#**100X weaker? Again please justify your answer.
#*** no, i think it would be an over kill. there would be no regulation of anything and phage probably wouldnt be viable.
# Nature often preserves functionally critical genomic elements, and evolutionary cousins can help us identify which genetic elements are disposable, which are interchangeable, and which are essential. Who are M13's closest evolutionary relatives and how do they differ from the phage you're working with?<br>
fd and f1.  "M13 differs from fd for replacement of a negatively charged Aspartate with a neutral Asparagine in the coat protein." (D. A. Marvin et al., J. Mol. Biol. 355, 294 (2006).)<br>
"Phage f1 DNA differs from that of phage M13 by 52 nucleotide changes, which lead to 5 amino acid substitutions in the corresponding proteins of the two phages, and from phage fd DNA by 186 nucleotide changes (including the single-nucleotide deletion), which lead to 12 amino acid differences between the proteins of phages f1 and fd. More than one-half of the nucleotide changes in each case are found in the sequence of 1,786 nucleotides comprising gene IV and the major intergenic region between gene IV and gene II. The sequence of this intergenic region (nucleotides 5501 to 6005) of phage f1 differs from the sequence reported by others through the inclusion of additional single nucleotides in eight positions and of a run of 13 nucleotides between positions 5885 and 5897, a point of uncertainty in the earlier published sequence. The differences between the sequence of bacteriophage f1 DNA now presented and a complete sequence for the DNA previously published by others are discussed, and the f1 DNA sequence is compared with those of bacteriophages M13 and fd." (J Virol. 1982 October; 44(1): 32â€“46.)
# As you heard on the first day of class, the writing you are doing for 20.109 is the subject of an academic study and will eventually become a chapter in a forthcoming book, "The Idea of a Writing Laboratory." The author, Neal Lerner, has requested that you download the following file [[Media:Macintosh HD-Users-nkuldell-Desktop-Student Writing Survey.doc | Student Writing Survey]], fill the information out electronically, then email the completed survey to "nlerner AT mit DOT edu". Please cc "nkuldell AT mit DOT edu" on your message. He will directly follow up with some 109ers. Thanks in advance.