{{Template:20.109}}

==Western analysis==
--[[User:Bryanh|Bryanh]] 14:00, 28 February 2007 (EST)

===Purpose===
We want to see if protein VIII with the added myc tag will be tolerated by the phage production.  We are going to use a western analysis to confirm the presence (or absence) of the myc tag.

==Protocol==
===Part 1: SDS-PAGE===

Each group will run a lane with a positive control (a bacterial strain expressing a myc-tagged protein), a lane of molecular weight markers, a lane with bacterial cells expressing the protein you've myc-tagged, and supernatant from that strain. These will be run in duplicate on the gel, so we can cut the blot in half and you will have two blots to probe next time. 

#Retrieve the bacterial cultures with your myc-tagged candidates that have been stored at 4Â°C since last time. You should also get a bacterial sample that will serve as a positive control for the myc-probe. This sample is known to express a ClpP-myc fusion protein (ClpP is a 207 amino acid bacterial protease). 
#Make a 1:10 dilution of your myc-tagged candidate that you'll follow-up and of the myc-positive control (50 ul cells plus 450 ul water) and measure their density at a wavelength of 600 nm using the spectrophotometer. If you do not remember how to use the spectrophotometer, please ask the teaching faculty to help. The cells will scatter light in proportion to  their density, at least within a certain range of densities, and the measurement is called an "OD reading", for optical density. 
#*+ control OD600: .3070
#*clone 3": .1008      
#Calculate the volume of cells needed to give the equivalent number of cells as 1 ml at 1 OD. For example, if the original undiluted sample was at 2 OD, then 500 ul would give you 1 OD of cells. 
#* 1/.3070 = 3.26ml of 1:10 diluted cells. so 326 uL of undiluted cells.
#* 1/.1008 = 9.92ml of 1:10 diluted cells. so 992 uL of undiluted cells.
#Move the calculated volume of cells to an eppendorf tube, and spin the tubes in a microfuge for 1 minute to pellet the bacteria. 
#Move the supernatant of each to a new, labelled eppendorf tube. You will <b>not</b> need the myc-positive control supernatant, but the supernatant from your myc-candidate will be tested for phage using the Western blot and plaque assays.  
#Resuspend the bacterial pellets in 100 ul of 1X sample buffer. Sample Buffer contains glycerol to help your samples sink into the wells of the gel, SDS to coat amino acids with negative charge, BME to reduce disulfide bonds, and bromophenol blue to track the migration of the smallest proteins through the gel. <b>Wear gloves when using sample buffer or your hands will get blue and smelly. </b>
#Mix 50 ul of the supernatant from your myc-candidate with 50 ul of '''2X''' sample buffer. Save the rest of the supernatant so you can measure its titer. 
#Put lid locks on the eppendorf tubes and boil for 5 minutes. 
#Put on gloves. Load the indicated volumes of each sample onto your acrylamide gel in the order below. Once you have loaded a sample from one tube, move it to a different row in your eppendorf tube rack. This will help you keep track of which samples you have loaded. 
{| border="1"
! Lane 
! Sample 
! Volume to load
|-
| 1 
|[http://www.bio-rad.com/B2B/BioRad/product/br_category.jsp?BV_SessionID=@@@@2015247057.1166812647@@@@&BV_EngineID=cccfaddjjkijiikcfngcfkmdhkkdfll.0&categoryPath=%2fCatalogs%2fLife+Science+Research%2fElectrophoresis%2fProtein+Electrophoresis%2fElectrophoresis+Reagents+and+Standards%2fProtein+Standards+for+Electrophoresis%2fPrecision+Plus+Protein+Standards&divName=Corporate&loggedIn=false&lang=English&country=HQ&catLevel=7&catOID=-17283&isPA=false&serviceLevel=Lit+Request&searchStr=kaleidoscope+markers&cateName=Ordering+Information "Kaleidoscope" protein molecular weight standards] 
| 5 ul
|-
| 2 
| myc positive control, from teaching faculty
| 25 ul
|-
| 3
| p8+myc or p3+myc candidate, cells
| 25 ul
|-
| 4 
| p8+myc or p3+myc candidate, supernatant
| 25 ul
|-
| 5
| empty
| 
|-
| 6 
|[http://www.bio-rad.com/B2B/BioRad/product/br_category.jsp?BV_SessionID=@@@@2015247057.1166812647@@@@&BV_EngineID=cccfaddjjkijiikcfngcfkmdhkkdfll.0&categoryPath=%2fCatalogs%2fLife+Science+Research%2fElectrophoresis%2fProtein+Electrophoresis%2fElectrophoresis+Reagents+and+Standards%2fProtein+Standards+for+Electrophoresis%2fPrecision+Plus+Protein+Standards&divName=Corporate&loggedIn=false&lang=English&country=HQ&catLevel=7&catOID=-17283&isPA=false&serviceLevel=Lit+Request&searchStr=kaleidoscope+markers&cateName=Ordering+Information "Kaleidoscope" protein molecular weight standards]
| 5 ul
|-
| 7
| myc positive control, from teaching faculty
| 25 ul
|-
| 8
| p8+myc or p3+myc candidate, cells
| 25 ul
|-
| 9 
| p8+myc or p3+myc candidate, supernatant
| 25 ul
|-
| 10
| M13K07 phage, from teaching faculty
| 25 ul
|}

10. Once all the samples are loaded, turn on the power and run the gel at 200 V. The molecular weight standards are pre-stained and will separate as the gel runs. The gel should take approximately one hour to run. During that hour, you should work on parts two and three of today's protocol.<br>
11. Wearing gloves, disassemble the electrophoresis chamber.<br>
12. Blot the gel to nitrocellulose as follows:
*Place the gray side of the transfer cassette in a tupperware container which is half full of transfer buffer. The transfer cassette is color-coded so the gray side should end up facing the cathode (black electrode) and the clear side facing the anode (red). 
*Place a ScotchBrite pad on the gray side of the cassette.
*Place 1 pieces of filter paper on top of the ScotchBrite pad.
*Place your gel on top of the filter paper.
*Place a piece of nitrocellulose filter on top of the gel.  The nitrocellulose filter is white and can be found between the blue protective paper sheets. Wear gloves when handling the nitrocellulose to avoid transferring proteins from your fingers to the filter. 
*Gently but thoroughly press out any air bubbles caught between the gel and the nitrocellulose.  
*Place another piece of filter paper on top of the nitrocellulose.
*Place a second ScotchBrite pad on top of the filter paper.
*Close the cassette then push the clasp down and slide it along the top to hold it shut.
*Place the transfer cassette into the blotting tank so that the clear side faces the red pole and the gray side faces the black pole.<br>
13. Two blots can be run in each tank. When both are in place, insert the ice compartment into the tank. Fill the tank with buffer. Be sure the stir bar is able to circulate the buffer. Connect the power supply and transfer at 100 V for one hour. You can use this time to complete parts 2 and 3 of today's protocol.<br>
14. After an hour, turn off the current, disconnect the tank from the power supply and remove the holders. Retrieve the nitrocellulose filter and confirm that the pre-stained markers have transferred from the gel to the blot. Move the blot to blocking buffer (TBS-T +5% milk) and store it in the refrigerator until next time.

===Part 2: Plaque assay===
You will use the plaque assay to titer the supernatant from the myc-tagged candidate you are following. This will allow you to determine (if/how many) phage have been secreted by your cells carrying the myc-tagged phage protein. You will include a positive and negative control for the assay as well. 

#Titer the supernatant from your candidate, examining the starting concentration (=10^0 dilution) as well as dilutions that are 10^-2, 10^-4, and 10^-6 of the starting concentration. 
#As a negative control, you should test bacteria without any phage.
#As a positive control, you should test bacteria infected with 10 ul of 10^-6 dilution of M13K07. You can get an aliquot of this phage dilution from the teaching faculty. 
#Consult your lab notebook or the protocol from [[20.109(S07): Agarose gel electrophoresis| day 2 of this module]] if you need reminding about how to perform the plaque assay.

===Part 3: M13.1===
This is a rough sketch to refine before we request a DNA synthesis company to compile the program for us. All elements are specified at the [http://parts.mit.edu/registry/index.php/Main_Page Registry of Standard Biological Parts]. Proposed refinements can be noted there as well as on our [[20.109:Module 1:RefactorM13| M13 refactoring workpage]]

{| border="1"
! genetic element
! promoter
! RBS
! coding
|-
|
! colspan="3" | start synthesis with
|-
|<font color = red> gII </font color>
|BBa_M13102 <br>(need 5' UTR?) 
|BBa_M13502 
|BBa_M13002'<br>(modified to remove gene 10 promoter)
|-
|<font color = red> gX </font color>
|BBa_M13110 <br>(need 5' UTR?) 
|BBa_M13510 
|BBa_M13010'<br>(modified to remove gene 5 promoter)
|-
|<font color = red> gV </font color>
|BBa_M13105 <br>(need 5' UTR?) 
|BBa_M13505 
|BBa_M13005
|- 
|<font color = red> gVII </font color>
|
|BBa_M13507 
|BBa_M13007' <br>(modified to remove overlap with gene 9 dwnstm)
|-
|<font color = red> gIX </font color>
|
|BBa_M13509 
|BBa_M13009' <br>(modified to remove overlap with gene 8 dwnstm)
|-
|<font color = red> gVIII </font color>
|BBa_M13108 <br>(need 5' UTR?) 
|BBa_M13508 
|BBa_M13008
|-
|
! colspan="3" |Transcriptional terminator (if M13K07 part, then need to modify to remove gene 3 promoter)
|-
||<font color = red> gIII </font color>
|BBa_M13103 
|BBa_M13503 
|BBa_M13003' <br>(modified to remove gene 6 promoter, change GTG start?) 
|-
|<font color = red> gVI </font color>
|BBa_M13106 
|BBa_M13506 
|BBa_M13006' <br>(modified to remove gene 1 promoter)
|-
|<font color = red> gI </font color>
|BBa_M13101 
|BBa_M13501 
|BBa_M13001' <br>(modified to remove gene 11 RBS, gene 4 promoter, RBS, start)
|-
|<font color = red> gXI </font color>
|
|BBa_M13511 
|BBa_M13011' <br>(modified to remove gene 4 promoter, RBS, start)
|-
|<font color = red> gIV </font color>
|BBa_M13104 <br>(need 5' UTR?) 
|BBa_M13504 
|BBa_M13004'
|-
|
! colspan="3" |M13K07 ori/KanR/p15a ori
|-
|
! colspan="3" | end synthesis
|}


Note: modified parts codon varied to remove direct repeats.
<br>
'''Summary:'''
we are waiting for all results and so no summaries can really be made

==For next time==
#Remind yourself what size you expect to see for p8 or p3, depending on which you myc-tagged. Determine which of the [http://www.bio-rad.com/B2B/BioRad/product/br_category.jsp?BV_SessionID=@@@@2015247057.1166812647@@@@&BV_EngineID=cccfaddjjkijiikcfngcfkmdhkkdfll.0&categoryPath=%2fCatalogs%2fLife+Science+Research%2fElectrophoresis%2fProtein+Electrophoresis%2fElectrophoresis+Reagents+and+Standards%2fProtein+Standards+for+Electrophoresis%2fPrecision+Plus+Protein+Standards&divName=Corporate&loggedIn=false&lang=English&country=HQ&catLevel=7&catOID=-17283&isPA=false&serviceLevel=Lit+Request&searchStr=kaleidoscope+markers&cateName=Ordering+Information kaleidoscope markers] you expect to see nearby. 
#Your first draft of your essay will be due when you arrive in lab next time. Detailed information about this assignment can be found at [[20.109(S07): Genome engineering essay]]. Send your draft prior to arriving in lab, emailing it to nkuldell AT mit DOT edu, endy AT mit DOT edu and breindel AT mit DOT edu.

==Reagents list==
*Transfer Buffer
**25 mM Tris
**192 mM glycine
**20% v/v methanol