==Making a Broad Host Range BioBrick Vector==

As part of the iGEM2007 competition, the University of Edinburgh team sought to make a broad host range BioBrick vector that could be used to transfer BioBrick constructs into hosts other than ''Escherichia coli''. Since we were working on production of flavour and pigment compounds in yoghurt, we initially wanted a shuttle vector that could operate in both ''E. coli'' and in ''Lactobacillus'' spp. We contacted Prof. Mike Gasson at the Institute of Food Research, Norwich, and he and his colleague, Dr. Claire Shearman, kindly provided us with vector '''pTG262'''. This 5.6 kb rolling-circle plasmid was known to replicate in ''E. coli'', ''Bacillus subtilis'', ''Lactococcus'' and ''Lactobacillus'' spp., all from the same replication origin. This raised the question: if this replication origin works in ''E. coli'', might it also work in other useful Gram negative bacteria such as ''Pseudomonas'', ''Shewanella'', ''Agrobacterium'' and ''Rhizobium''?

===Making pTG262 into a BioBrick vector===

pTG262 possessed a multi-cloning site with PstI and EcoRI sites, and also a unique XbaI site between them. There was no SpeI site present. By inserting a BioBrick between the EcoRI and PstI sites, we removed the XbaI site and replaced it with a full set of BioBrick cloning sites including EcoRI, NotI, XbaI...(insert)...SpeI, NotI, PstI, thus converting pTG262 to a fully compliant BioBrick vector which we have temporarily named pTG262-BB, until it can be given a proper BioBrick vector name by the Registry. We inserted three different BioBricks in this way: two encoding the monomeric DsRed version of Red Fluorescent Protein (BBa_J04450, with a ''lac'' promoter, and BBa_I13521, with a ''tet'' promoter) and one with the lacZ' minigene (BBa_J33207). Red and blue colonies were obtained in ''E. coli'' JM109, though transformation efficiencies were not high (see below). (In fact, it turns out that pTG262 already had a ''lacZ''' minigene, so it is not clear whether or not this part worked - see below). We also transformed ''B. subtilis'' 168 with pTG262BB-BBa_I13521 and obtained chloramphenicol-resistant colonies, but RFP was apparently not produced. This may be due to a promoter issue, or DsRed may not work in ''B. subtilis'' for some reason. We are currently investigating this further. Results will be posted soon.

===Information about pTG262===

Most of the information I have been able to find about the history of pTG262 comes from W.M. de Vos and G.F.M. Simons (1994) 'Gene cloning and expression systems in Lactococci', chapter 2 (pages 52 to 105) in 'Genetics and Biotechnology of Lactic Acid Bacteria', edited by M.J. Gasson and W.M. de Vos, Blackie Academic and Professional, London, ISBN 0 7514 0098 X, Darwin Library classmark QR 121 Gen. The good stuff is in the section 'Replicative Gene Cloning' on pages 53 to 66. It appears that pTG262 is ultimately derived from a ''Lactococcus'' plasmid of 2062 bp named '''pSH71''', which is extremely similar to another plasmid of 2178 bp named '''pWV01''', so that the two plasmids can be considered to possess the same replicon. pSH71 and pWV01 have been used as the bases for the widely used lactococcal cloning vector series pNZ, pCK and pGK. pTG262 would seem to be derived from pCK1, which was made from pSH71 by addition of staphylococcal resistance determinants for chloramphenicol (''cat-194'') and neomycin/kanamycin (''knt-110''). According to de Vos and Simons (op cit), pTG262 was made by adding the ''E. coli lacZ' ''minigene encoding the alpha-peptide N-terminus of beta-galactosidase, plus a multi-cloning site. After hunting through the literature, the earliest reference I have found so far is C.A. Shearman, H. Underwood, K. Jury and M.J. Gasson (1989) Cloning and sequence of a ''Lactococcus'' bacteriophage lysin gene, ''Mol. Gen. Genet'' '''218''', 214-221, which lists pTG262 in Table 1, page 215, with the annotation: '0.6 kb HaeII lac alpha fragment ex pUC18 in pCK17 (EcoRI-BamHI): A. Mercenier, Transgene, Strasbourg, France'. Clearly there is scope for tracking this back a bit further.

But back to pSH71. According to de Vos and Simons (op cit), pSH71 is a rolling circle plasmid bearing two ''rep'' genes encoding the 6 kD repressor protein RepC and the 27 kD replication protein RepA. pSH71, pWV01 and their derivatives replicate in all lactic acid bacteria as well as ''B. subtilis'' and a wide range of other listed Gram positive bacteria, and also in ''E. coli''; however, it is stated that segregational stability is relatively low in ''E. coli'', despite a high copy number (probably not a problem if antibiotic selection is maintained) and that transformation efficiency is low in ''recA- E. coli'' host strains (including most common lab strains such as JM109 and DH5-alpha), possibly due to the unusual method for lagging strand synthesis. Certainly we were not getting stormingly high efficiencies in JM109, so it might be an idea to try a ''recA+'' host strain. Gram negative bacteriaa other than ''E. coli'' do not seem to be mentioned, but if this replicon works in ''E. coli'' there seems no reason why it should not work in other Gram negative hosts such as ''Pseudomonas'', ''Shewanella'', and ''Agrobacterium''. We are currently investigating this. If the vector does work in all of these organisms, it could be a very useful broad host range BioBrick vector. 

Drs. Shearman and Gasson were kind enough to send me the entire sequence of pTG262 as well as instructions on selection conditions. Genes are present encoding resistance to chloramphenicol, kanamycin/neomycin and (I believe) gentamicin. Recommended selection is with chloramphenicol at 15 mg/l in ''E. coli'', 10 mg/l in ''Bacillus'', 7.5 mg/l in ''Lactobacillus'' and 5 mg/l in ''Lactococcus lactis''. After our replacement of most of the MCS with a BioBrick, the only remaining site is HindIII, which is at the PstI end of the insert.

We have submitted pTG262 to the Registry as BBa_I742103 and pTG262BB-BBa_I13521 as BBa_I742123. We will also be happy to supply this DNA directly on request (C.French@ed.ac.uk). We hope that a new name will be assigned to pTG262BB according to Registry vector nomenclature. We are currently investigating expression of BioBricks in ''B. subtilis'', and transformation of other Gram negative chassis organisms with this vector, and will post results here.

C. French, 19 Nov 2007.

===Further Information: 27 Nov 07===

From examination of the pTG262 sequence, it is clear that it contains a full ''lac'' promoter region including the CAP binding site and LacI binding site, and the C-terminal 28 residues of ''lacI'', as well as the pUC18-derived (ie M13mp18-derived) multi-cloning site and the N-terminal region of ''lacZ''. The region of similarity with ''lacZ'' includes the first 58 codons; after that there are a further 9 unrelated codons followed by a stop codon. (By comparison, pUC18 has the first60 codons of ''lacZ'' followed by a further 29 unrelated codons and a stop codon.) However, surprisingly, in my hands so far, pTG262 in ''E. coli'' JM109 does not give blue colonies in the presence of IPTG and Xgal. 

As noted above, we inserted BioBrick BBa_J33207, including a ''lac'' promoter and the N-terminal 76 residues of ''lacZ'', into pTG262 and obtained blue colonies; however, we could not confirm this by sequencing, since it turns out, obviously in retrospect, that our sequencing primers lie within the ''lac''-related region, hence giving double sequence (which in itself might be taken as evidence that the insertion was successful). However, the repetitive nature of the result may make it unstable, and in any case we would prefer a marker that will work in organisms other than ''E. coli''. We are therefore currently inserting BioBrick BBa_J33207 into pTG262; this includes ''xylE'' encoding catechol-2,3-dioxygenase, which produces a yellow pigment (2-hydroxy-''cis,cis''-muconic semialdehyde) when catechol is added (see BBa_J33207 documentation for further details). BBa_J33207 does not have a promoter, but insertion between EcoRI and PstI of pTG262 should mean that the ''lac'' promoter can drive expression.

For those planning to use pTG262 in this way, note that the presence of this ''lac'' promoter may lead to unwanted expression; you may need to include a terminator upstream of your construct.

[[French_Lab|Back to main page]]