==Catalog of primers==

===Primers for iGEM2007 flavours/fragrances project===

'''echf1''' aat gaattc gcggccgc t tctag atg agc aaa tac gaa ggt c
*Purpose: forward primer for ''ech (ferA)'' coding sequence from ''Ps. fluorescens'' NCIMB 9046.
*Restriction sites: EcoRI, XbaI
*Notes: no ribosome binding site included.

'''echr1''' ct actagt a tta tta gcg ttt ata ggc ttg cag c
*Purpose: reverse primer for ''ech (ferA)'' coding sequence from ''Ps. fluorescens'' NCIMB 9046.
*Restriction sites: SpeI only.
*Notes: replaces TGA stop codon with TAA TAA. Also replaces PstI site in coding sequence with silent mutation.

'''fcsf1''' aat gaattc gcggccgc t tctag atg cgc tcc ctg gaa ccc
*Purpose: forward primer for ''fcs (ferB)'' coding sequence from ''Ps. fluorescens'' NCIMB 9046.
*Restriction sites: EcoRI, XbaI
*Notes: no ribosome binding site included.

'''fcsr1''' ct actagt a tta tta cgg ttt ggg ccc ggc ac
*Purpose: reverse primer for ''fcs (ferB)'' coding sequence from ''Ps. fluorescens'' NCIMB 9046.
*Restriction sites: SpeI only.
*Notes: replaces TGA stop codon with TAA TAA. 

'''Ms_COMTf1''' aat gaattc gcggccgc t tctag atg ggt tca aca ggt gaa ac
*Purpose: forward primer for caffeate O-methltransferase coding sequence from ''Medicago sativa'' (alfalfa).
*Restriction sites: EcoRI, XbaI.
*Notes: eukaryotic gene, no ribosome binding site.

'''Ms_COMTr1''' ct actagt a tta tta  aac ctt ctt aag aaa ctc c
*Purpose: reverse primer for caffeate O-methltransferase coding sequence from ''Medicago sativa'' (alfalfa).
*Restriction sites: SpeI only (NOT PstI).
*Notes: adds TAA TAA stop codons.

'''echf2''' atc gagctc acacc cagaa caaga gc
*Date: 9 August 2007.
*Purpose: amplify ''ech'' from ''Pseudomonas'' with rbs and some surrounding DNA.
*Restriction sites: SacI
*Notes: this was made because no product was obtained with echf1 and echr1.

'''echr2''' tt actagt atcgg gaaca cgttc aagc
*Date: 9 August 2007.
*Purpose: amplify ''ech'' from ''Pseudomonas'' with rbs and some surrounding DNA.
*Restriction sites: SpeI
*Notes: this was made because no product was obtained with echf1 and echr1.

'''fcsf2''' gtg gagctc actga agaac agggc gtg
*Date: 9 August 2007.
*Purpose: amplify ''fcs'' from ''Pseudomonas'' with rbs and some surrounding DNA.
*Restriction sites: SacI
*Notes: this was made because no product was obtained with fcsf1 and fcsr1.

'''fcsr2''' aa actagt atgcc gtgac agcaa atagg
*Date: 9 August 2007.
*Purpose: amplify ''fcs'' from ''Pseudomonas'' with rbs and some surrounding DNA.
*Restriction sites: SpeI
*Notes: this was made because no product was obtained with fcsf1 and fcsr1.

'''sam5f1''' at gaattc gcggccgc t tctag atg acc atc acg tca cct g
*Date: 9 August 2007.
*Purpose: amplify ''sam5'' from ''Saccharothrix espanaensis''.
*Restriction sites: EcoRI, NotI, XbaI (not SacI!)
*Notes: coding sequence only, no ribosome binding site.

'''sam5f2''' tc gaattc gcggccgc t tctaga g acgg agaa gcag cgaa atg
*Date: 21 Sep 07
*Purpose: amplifies ''sam5'' with native ribosome binding site region.
*Restriction sites: EcoRI, NotI, XbaI (not SacI!)
*Notes: the ''sam5'' gene has one internal site each for SacI and PstI. Amplification OK from plasmid DNA.

'''sam5r1''' ct actagt a tta tta ggt gcc ggg gtt gat cag
*Date: 9 August 2007.
*Purpose: amplify ''sam5'' from ''Saccharothrix espanaensis''.
*Restriction sites: SpeI (not PstI!)
*Notes: coding sequence modified to end with TAA TAA.

'''sam8f1''' at gaattc gcggccgc t tctag atg acg cag gtc gtg gaa cg
*Date: 9 August 2007.
*Purpose: amplify ''sam8'' from ''Saccharothrix espanaensis''.
*Restriction sites: EcoRI, NotI, XbaI (not SacI!)
*Notes: coding sequence only, no ribosome binding site.

'''sam8f2''' tc gaattc gcggccgc t tctaga g gagg ttgg gaca agat gac
*Date: 21 Sep 07
*Purpose: amplify ''sam8'' with native ribosome binding site region.
*Restriction sites: EcoRI, NotI, XbaI (not SacI!)
*Notes: coding sequence contains an internal SacI site. Amplification OK from plasmid DNA.

'''sam8r1''' ct actagt a tta tta tcc gaa atc ctt ccc gtc
*Date: 9 August 2007.
*Purpose: amplify ''sam8'' from ''Saccharothrix espanaensis''.
*Restriction sites: SpeI (not PstI!)
*Notes: coding sequence modified to end with TAA TAA.

'''crtEf1''' gca gagctc gcgt tgcc gtaa atgt atc
*Date: 13 August 2007
*Purpose: amplification of ''crtE'' from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SacI
*Notes: amplification from cell suspension was successful.

'''crtEr1''' aa actagt gcga tcgc cgcg aaat g
*Date: 13 August 2007
*Purpose: amplification of ''crtE'' from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SpeI
*Notes: amplification from cell suspension was successful. Product size 997 bp.

'''crtEr2''' cgt actagt a tta tta act gac ggc agc gag
*Date: 4 Oct 07
*Purpose: amplifying ''crtE'' from ''Pantoea ananatis'' leaving a BioBrick-compliant end
*Restriction sites: SpeI (not PstI!)
*Notes: leaves TAA TAA stop codon. Worked fine.

'''crtYf1''' cag gagctc ttaa gtgg gagc ggct atg
*Date: 13 August 2007
*Purpose: amplification of ''crtY'' from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SacI
*Notes: amplification from cell suspension was successful.

'''crtYr1''' ac actagt tggt ttca tgta gtcg
*Date: 13 August 2007
*Purpose: amplification of ''crtY'' from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SpeI
*Notes: amplification from cell suspension was successful. Product size 1202 bp.

'''crtYr2''' cgt actagt a tta tta acg atg agt cgt cat aat g
*Date: 4 Oct 07
*Purpose: amplifies ''crtY'' from ''Pantoea ananatis'' leaving a BioBrick compliant end.
*Restriction sites: SpeI (not PstI!)
*Notes: leaves TAA TAA at end. Worked.

'''crtIf1''' tga gagctc atcg ttaa agag cgac
*Date: 13 August 2007
*Purpose: amplification of ''crtIB'' from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SacI
*Notes: amplification from cell suspension was successful. Note that ''crtI'' contains two PstI sites.

'''crtBr1''' gc actagt caaa actt cagg cgac
*Date: 13 August 2007
*Purpose: amplification of ''crtIB'' from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SpeI
*Notes: amplification from cell suspension was successful. Note that ''crtI'' contains two PstI sites. Product size 2477 bp.

'''crtBr2''' cgt actagt a tta tta gag cgg gcg ctg cca g
*Date: 4 Oct 07
*Purpose: amplifies ''crtIB'' from ''Pantoea ananatis'' with a BioBrick compliant end.
*Restriction sites: SpeI (not PstI!)
*Notes: leaves TAA TAA at end of ''crtB''. Note that ''crtI'' contains 2 PstI sites. Worked.

'''crtZf1''' aga gagctc tacc ggag aaat tatg
*Date: 13 August 2007
*Purpose: amplification of ''crtZ'' from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SacI
*Notes: amplification from cell suspension was successful. Note that ''crtZ'' is encoded on the opposite strand to ''crtEXYBI''. 

'''crtZr1''' cc actagt cagg ccct tact tccc
*Date: 13 August 2007
*Purpose: amplification of ''crtZ'' from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SpeI
*Notes: amplification from cell suspension was successful. Note that ''crtZ'' is encoded on the opposite strand to ''crtEXYBI''. Product size 565 bp.

'''crtZr2''' cgt actagt a tta tta ctt ccc gga tgc ggg
*Date: 4 Oct 07
*Purpose: amplifies ''crtZ'' from ''Pantoea ananatis'' with a BioBrick compliant end.
*Restriction sites: SpeI (not PstI!)
*Notes: leaves TAA TAA at end. Worked.

'''crtopr1''' cc actagt ggag tact tccc gatg
*Date: 13 August 2007
*Purpose: amplification of ''crt'' region (with primer crtEf1) from ''Pantoea ananatis'' ATCC 19321 (DSM 30080)
*Restriction sites: SpeI
*Notes: amplification from cell suspension was successful. Note that ''crtZ'' is encoded on the opposite strand to ''crtEXYBI'', so reverse primer crtZr1 could not be used to amplify the whole region. (crtZf1 would have given a product but with SacI sites at both ends.) Product size 6490 bp.


===Primers for iGEM2007 bioswitches project===

'''revPlacf1''' tcta gagctc tgtgtgaaattgttatccg
*Purpose: forward biobrick primer for making a reverse ''lac'' promoter.
*Restriction sites: XbaI (probably won't cut as right at end), SacI
'''revPlacr1''' ct actagt a caatacgcaaaccgcctc
*Purpose: reverse biobrick primer for making a reverse ''lac'' promoter.
*Restriction sites: SpeI only.

===Primers for working with biobricks===
'''pSB1A3f1''' tccttagctttcgctaagg
*Purpose: sequencing or amplification of biobrick inserts in pSB1A3 and derivatives thereof.
*Restriction sites: none (but amplification products will have full biobrick prefix)

'''pSB1A3r1''' agggtggtgacaccttgc
*Purpose: sequencing or amplification of biobrick inserts in pSB1A3 and derivatives thereof.
*Restriction sites: none (but amplification products will have full biobrick suffix)

'''pSB1A2insf1''' cgctaaggatgatttctgg
*Purpose: sequencing or amplification of biobrick inserts in pSB1A2 and derivatives thereof.
*Restriction sites: none (but amplification products will have full biobrick prefix)

*Notes: excellent resuts for sequencing.
'''pSB1A2insr1''' gtcagtgagcgaggaagc
*Purpose: sequencing or amplification of biobrick inserts in pSB1A2 and derivatives thereof.
*Restriction sites: none (but amplification products will have full biobrick suffix)
*Notes: excellent resuts for sequencing.

'''bbinsertf1''' attc gcggccgc t tctag
*Purpose: sequencing or amplification of any biobrick insert
*Restriction sites: only NotI, but amplification product will have XbaI, and SacI if present.
*Note: gives good results for sequencing, but pSB1A2insf1 is better.

'''bbinsertr1''' tgcag cggccgc t actag
*Purpose: sequencing or amplification of any biobrick insert
*Restriction sites: only NotI, but amplification product will have SpeI.
*Note: usually not very good results for sequencing.

'''bbvectorf1''' ag t actag ta gcggccg ctgc
*Purpose: amplification of vector + biobrick for PCR-based cloning procedures; binds to biobrick suffix and amplifies vector region.
*Restriction sites: SpeI, NotI; amplification product will also have PstI.

'''bbvectr1''' cctt gagctc taga a gcggccgc gaattc
*Purpose: amplification of vector + biobrick for PCR-based cloning procedures; binds to biobrick prefix and amplifies vector region.
*Restriction sites: SacI, XbaI, EcoRI.

'''pTG262f1''' gttgtgtggaattgtgagc
*Date: 13 sep 07
*Purpose: sequencing of inserts in pTG262 and derivatives thereof.
*Restriction sites: none
*Notes: 

'''pTG262r1''' ggttttcccagtcacgacg
*Date: 13 sep 07
*Purpose: sequencing of inserts in pTG262 and derivatives thereof.
*Restriction sites: none
*Notes: 

===Primers for 'Bacillobrick' project===

'''specf3''' aac gagctc aacg aggt gaaa tcat g
*Date: 22 June 2007
*Purpose: amplification of spectinomycin resistance gene from plasmid pVK168 (Patrick Piggot, Temple U).
*Restriction sites: SacI
*Notes: A product was obtained (in contrast to primers specf1 and specf2, which never produced a product; these two primers were much further upstream, in the hope of including the promoter region, whereas specf3 just includes the rbs; this suggests that there is some problem with the sequence data I was given). However, numerous attempts to clone the PCR product in pGemT-easy and Edinbricks 1, 2 and 3 were unsuccessful. The reason for this is unclear. At this date (21 Aug 2007) this product has not been successfully cloned.

===Primers for ''Bacillus subtilis ureABC'' genes===

'''Bsuref1''' ga gagctc cgca aatt cgta gtag c
*Date: 
*Purpose: Amplification of urease gene cluster ''ureABC'' from ''B. subtilis'' 168.
*Restriction sites: SacI
*Notes: Worked.

'''Bsurer1''' ct ggatcc atgg gttt tgtg cacc g
*Date: 
*Purpose: Amplification of urease gene cluster ''ureABC'' from ''B. subtilis'' 168.
*Restriction sites: BamHI
*Notes: The urease gene cluster contained one EcoRI and one SpeI site, so could not be initially cloned as a biobrick, so it was cloned SacI/BamHI into pBluescript SK+ to use as a template for mutagenesis to remove the offending sites.

'''Bsure1r1''' tcc tctaga gaga ccgt tttc tgct ctc
*Date: 
*Purpose: elimination of an SpeI site in ''ureABC'' by amplifying the gene cluster in two overlapping parts, converting one of the SpeI sites to an XbaI site, ligating together to generate a non-cleavable fusion site.
*Restriction sites: XbaI
*Notes: used with Bsure2f1 (see below)

'''Bsure2f1''' aaa acgg tctc actagt gg
*Date: 
*Purpose: elimination of an SpeI site in ''ureABC'' by amplifying the gene cluster in two overlapping parts, converting one of the SpeI sites to an XbaI site, ligating together to generate a non-cleavable fusion site.
*Restriction sites: SpeI
*Notes: used with Bsure1r1 (see above)

'''Bsure2r1''' cat ctcgag cttg ctca gctt tctg
*Date: 
*Purpose: mutating an EcoRI site in ''ureABC'' by amplifying the gene cluster as two overlapping fragments, mutating out the EcoRI and introducing an XhoI which could be used to ligate the fragments together.
*Restriction sites: XhoI
*Notes: used with Bsure3f2 (see below)

'''Bsure3f2''' aag ctcgag atga agct gaac tcgg cttt g
*Date: 
*Purpose: mutating an EcoRI site in ''ureABC'' by amplifying the gene cluster as two overlapping fragments, mutating out the EcoRI and introducing an XhoI which could be used to ligate the fragments together.
*Restriction sites: XhoI
*Notes: used with Bsure2r1 (see above)

'''Bsure3r2''' tac actagt tttg tgca ccgt tttt ag
*Date: 
*Purpose: converting the final ''ureABC'' product to a biobrick by adding the 3' SpeI site.
*Restriction sites: SpeI
*Notes: the alert reader will notice that this would not produce a compliant biobrick since it is missing the A basee after the SpeI site. Oops.

===Primers for ''Escherichia coli lac'' genes===

'''laczf1''' cctt tctaga gg aggaaacagct atg acc
*Date: 11 Jul 06
*Purpose: amplification of ''lacZ''' from ''E. coli'' BS with its native ribosome binding site (actually the rbs is slightly improved by adding 2 extra G residues at the start).
*Restriction sites: XbaI only
*Notes: Worked fine.

'''laczf2''' gg tctaga gctc atgt tata tccc g
*Date: 18 July 2006
*Purpose: Construction of Edinbrick1 (BBa_J33207).
*Restriction sites: XbaI, SacI
*Notes: Puts ''lacZ'' into pSB1A2 with a SacI site overlapping the XbaI site. This allows other PCR products to be cloned in replacing the ''lacZ'' marker gene after digestion with SacI and SpeI, meaning that primers can be much shorter than if a full prefix or suffix was required. Products are fully compliant biobricks. The complementary part of this primer starts about 370 bases upstream of the start codon, so the full promoter region is present.

'''laczf3''' gg gtcgac aggt ttccc gact g
*Date: 18 July 2006
*Purpose: Construction of a biobrick vector based on pT7-7.
*Restriction sites: SalI
*Notes: This vector was never completed. The complementary part of this primer starts 30 bases upstream of the CAP binding site, about 150 bases upstream of the start codon, so the promoter region is also included.

'''laczr1''' aagg ctgcag cggccgc t actagt a tca ctc cag cca gct ttc
*Date: 11 July 2006
*Purpose: Designed to produce a truncated version of ''lacZ'' with stop codon TGA at position 77.
*Restriction sites: PstI, NotI, SpeI
*Notes: Also produces minor product truncated with stop codon at position 137, reason unclear. Still works fine though.

'''Eclacr2''' gtt actagt gtgaaattgttatccgc
*Date: 21 August 2006
*Purpose: 
*Restriction sites: SpeI
*Notes: Don't remember why this was made. Maybe it was to amplify the whole ''lacZ'' coding sequence rather than just the N-terminus?

'''XWlaczZf1''' cctt tctag atg acc atg att acg gat tc
*Date: 27 Feb 2007
*Purpose: amplification of ''lacZ'' coding sequence (without rbs) from ''E. coli''.
*Restriction sites: XbaI
*Notes: 

'''XWlacZr1''' aagg ctgcag cggccgc t actagt a tta tta ctc cag cca gct ttc c
*Date: 27 Feb 2007
*Purpose: amplification of ''lacZ'' coding sequence with TAA TAA double stop codon introduced after codon 76, to comply with Registry recommendations for coding sequence biobricks.
*Restriction sites: PstI, NotI, SpeI
*Notes: 

'''eclacyf1''' ggcgagctctgcccgtatttcgcgtaag
*Date: 1 Sep 2006
*Purpose: testing for presence of intact ''lacY'' in ''E. coli'' strains. 
*Restriction sites: SacI.
*Notes: Also could be used to clone ''lacY'' as a biobrick using SacI/SpeI sites in an Edinbrick vector.

'''eclacyr1''' gatactagtcgcgccgcatccgacattg
*Date: 1 Sep 2006
*Purpose: as above.
*Restriction sites: SpeI
*Notes: as above.

'''bbtagf1''' gga tctaga ctagt gagc gcaa cgca atta atg
*Date: 22 June 2007
*Purpose: amplify lacZ' with rbs using a reverse primer such as lacZr1 with a full biobrick suffix, in order to make a cleavable ''lacZ'' tag to facilitate combining biobricks.
*Restriction sites: XbaI, SpeI
*Notes: Complementary part lies at the start of the CAP-binding site, about 128 bases upstream of the start codon, so the promoter region is present. Generated a good PCR product and was cloned in a suitable vector, but later I realised that cleavage at this SpeI site would generate a non-compliant biobrick lacking one of the required residues in the suffix (the T before the SpeI site). Therefore I designed primer bbtagf2, see below.

'''t7tagf1''' ggc aat catatg ttgc ccgt ctca ctgg
*Date: 22 June 2007
*Purpose: Amplify ''lacZ'' for insertion into pT7-7 as a removable tag so that Sahreena could insert PCR products into pT7-7 and have a way of telling which clones had the insert by blue white selection.
*Restriction sites: NdeI
*Notes: The complementary region starts about 255 bases upstream of the ''lacZ'' start codon, so the promoter is included, and it will work in a strain like JM109 rather than relying on the T7 promoter in the vector. Worked fine. Insertion is between NdeI and EcoRI so that EcoRI and all other sites in the MCS are available for cloning.

'''t7tagr1''' gaa gaat tca ctc cag cca gct ttc c
*Date: 22 June 2007
*Purpose: Amplify ''lacZ'' for insertion into pT7-7 as a removable tag so that Sahreena could insert PCR products into pT7-7 and have a way of telling which clones had the insert by blue white selection. A TGA stop codon (overlaps the EcoRI site) is introduced at codon 77 of ''lacZ''. 
*Restriction sites: EcoRI
*Notes: Worked fine.

===Primers for chromogenic reporter genes===

'''PpxylEf2''' ct gagctc atga acta tgaa gagg tg
*Date: 
*Purpose: amplification of ''xylE'' encoding catechol-2,3-dioxygenase from TOL plasmid pWW0 (subclone kindly provided by Prof. Peter Williams)
*Restriction sites: SacI
*Notes: Used to make J33204, aka Edinbrick 2 vector. Detect XylE by adding a drop of 10 mM catechol to patches; yellow colour develops rapidly. Works much better on patches than on primary colonies from transformation.

'''PpxylEr2''' ta actagt a ccgg acca tcag gtc
*Date: 
*Purpose: amplification of ''xylE'' encoding catechol-2,3-dioxygenase from TOL plasmid pWW0 (subclone kindly provided by Prof. Peter Williams)
*Restriction sites: SpeI
*Notes: Used to make J33204, aka Edinbrick 2 vector. Detect XylE by adding a drop of 10 mM catechol to patches; yellow colour develops rapidly. Works much better on patches than on primary colonies from transformation.


===Primers for heavy metal sensing components===

'''Ecarsf1''' cctt tctaga g ccaa ctca aaat tcac
*Date: 
*Purpose: amplification of ''ars'' promoter and ''arsR'' gene from ''E. coli'' K12. 
*Restriction sites: XbaI
*Notes: Amplification successful. Used to make J33201.

'''Ecarsr1''' aagg ctgcag cggccgc t actagt a cccg gata aaac acat c
*Date: 
*Purpose: amplification of ''ars'' promoter and ''arsR'' gene from ''E. coli'' K12. 
*Restriction sites: PstI, NotI, SpeI
*Notes: Amplification successful. Used to make J33201. 

'''Bsarsf2''' at gagctc cgtt gctg tagt agc
*Date: 
*Purpose: amplification of ''ars'' promoter and ''arsR'' gene from ''B. subtilis'' 168. 
*Restriction sites: XbaI
*Notes: Amplification successful. Used to make J33206.

'''Bsarsr2''' aa actagt ttag cagc aatc tcct tc
*Date: 
*Purpose: amplification of ''ars'' promoter and ''arsR'' gene from ''B. subtilis'' 168. 
*Restriction sites: SpeI
*Notes: Amplification successful. Used to make J33206. However, due to an error in design, this primer does not produce a compliant biobrick since it is missing the A after the SpeI site. To get around this, we added another biobrick to get a compliant 3' end. Also note that constructs using this as a control system did not show good regulation in ''E. coli'' in our experiments, in contrast to previous experience with this promoter using other reporter genes such as ''xylE''.

'''PzntAf1''' tcta gagctc tctg cgtttgtt gg
*Date: 9 August 2007
*Purpose: amplification of the ''zntA'' promoter region from ''E. coli'' K12.
*Restriction sites: XbaI (probably won't cut as right at end), SacI
*Notes: amplification successful.

'''PzntAr1''' ct actagt atta accg aagg atac actc
*Date: 9 August 2007
*Purpose: amplification of the ''zntA'' promoter region from ''E. coli'' K12.
*Restriction sites: SpeI
*Notes: amplification successful. 

'''yhdMf1''' aga gaattc gcggccgc t tctag atg tat cgc att ggt gag c
*Date: 9 August 2007
*Purpose: amplification of ''yhdM'' (aka ''copR'', ''cueR'') encoding copper-dependent MerR-like activator protein from ''E. coli'' K12.
*Restriction sites: EcoRI, NotI, XbaI 
*Notes: amplification successful. 

'''yhdMr1''' ct actagt a tta tta aca acc act ctt aac gcc
*Date: 9 August 2007
*Purpose: amplification of ''yhdM'' (aka ''copR'', ''cueR'') encoding copper-dependent MerR-like activator protein from ''E. coli'' K12.
*Restriction sites: SpeI
*Notes: introduces TAA TAA double stop codon to comply with Registry recommendations for coding sequence biobricks. Amplification successful.



===Template===
'''primer'''
*Date: 
*Purpose: 
*Restriction sites: 
*Notes: 


[[French_Lab|Back to Main Page]]