{{ChandraLab}}

__NOTOC__
The current focus of our lab is the application of systems biology to understand diseases and immunological processes. An obvious outcome will be the application of the obtained knowledge in drug and vaccine discovery.

Development of virtual mycobacterial cell models and their analysis to
*Address mechanism of drug action.
*Identify most appropriate drug targets.
*Study carbohydrate recognition in pathogenesis.
*Carbohydrate recognition and immunological triggers

The present focus in the laboratory is towards different areas for deriving appropriate system landscapes at different levels of hierarchy and different levels of model abstraction, and build mathematical models to simulate them. Such models will then be used to address 
:(a) role of recognition of carbohydrates in tuberculosis pathogenesis 
:(b) consequences of carbohydrate recognition in triggering specific immune mechanisms relevant to tuberculosis, 
:(c) identify appropriate drug targets for tuberculosis and 
:(d) study the pharmacodynamic profiles of the existing anti-mycobacterial drugs.

Work has already been initiated in this direction by building a first level virtual cell of the mycobacterium. A model of this system is built using a systems biology modelling toolkit. A new algorithm that encodes advanced bioinformatics tools and methods, is under development, to define the minimum system model for a given purpose. As proof-of-concept, a simulation of the system defined for H2-antihistamines has been carried out, which reveals the paradoxical accumulation of histamine available for binding by H1-receptor, upon using H2-antihistamines, thus explaining their adverse effects.

A long-term goal in the laboratory is to derive general principles and sub-system ontologies, their representation and incorporation into tractable models, so as to enable disease modelling.

We also have [[Chandra:Research:Collaborations|collaborative projects]] with other groups in IISc.


==Systems Biology==
*Flux Balance Analysis, Identification of newer drug targets based on flux analyses
*Identification of proteinâ€“protein influences based on metabolic pathway participation
*[[Chandra:Research:Cytoview|Cell Modelling]]
*[[Chandra:Research:Systems biology|More ...]]

==Structural bioinformatics==
*Algorithms for detecting signature interaction patterns in protein structures
*Study of determinants of molecular recognition
*Analysis of fold and function determinants
*[[Chandra:Research:Structural bioinformatics|More ...]]

==Genome and proteome sequence analysis: ''Mycobacterium tuberculosis''==
*Identification of pathways
*Identification of function for several proteins
*Identification and feasibility analysis of drug targets
*Comparative genomics
*[[Chandra:Research:Sequence analysis|More ...]]

==Lectin Databases==
*Development of an integrated lectin knowledge base, its usage in analysis of lectins, identification of potential applications of lectins in clinical practice, biotechnology and agriculture
*Cancer-lectin database

==Computational Immunology==
*Study of peptide-HLA complexes
*[[Chandra:Research:Computational immunology|More ...]]