{{ChandraLab}}
==Comparative Genomics of Mycolic Acid Pathway==
[[Image:Chandra-MAP_clustering.png|thumbnail|200px|left|Clustering of proteins, based on phylogenetic profiles]] [[Image:Chandra-MAP_taxonomy.png|thumbnail|200px|left|Molecular taxonomy]]
We have carried out a comprehensive phylogenetic pathway profile analysis, for 318 fully sequenced bacterial genomes, for each of the proteins present in the MAP. The organisms were clustered on the basis of co-occurrence of the MAP proteins in their proteome, while the proteins were clustered on the basis of their phylogenetic profiles. The MAP proteins were also searched against the nonredundant sequence database, to identify similar proteins from other phyla. The pathway profiles indicate that four proteins and certain protein domains stand out as more characteristic to mycolate producing organisms. Further analysis leads to the identification of the desaturases DesA1 and DesA2 and certain domains of Fas and Pks13 as hallmarks of the pathway. The clustering of organisms, carried out to group organisms with similar profiles, provides a means of obtaining finer classification as compared to the standard taxonomic method. The results indicate that the MAP and hence the capacity of mycolic acid production in mycobacteria is an example of an emergent property that has come about by recruiting enzymes from unrelated pathways in plants, presumably through lateral gene transfer. ''Original Paper:'' Karthik Raman, Preethi Rajagopalan and Nagasuma Chandra (2007, ''In Press'') "Hallmarks of Mycolic Acid Biosynthesis: A Comparative Genomics Study" ''Proteins: Structure, Function and Bioinformatics''. '''PMID 17600834'''