{{Chen Lab Top}}

{|  
|- valign="top"|
|width=800px; class="MainPageBG" style="border: 5px solid #669900; color: #000000; background-color: #FFFFFF" |

<div style="padding: .4em .9em .9em; font-family:courier new">

<h3>RNA interference of influenza virus infection:</h3>

Influenza A virus causes the most prevalent infection of the respiratory tract in humans. In a typical year, 10-20% of the population in the United States is afflicted by the virus, resulting in up to 40,000 deaths. In what was one of the most devastating human catastrophes in history, an estimated of 40-50 million people died worldwide during the 1918 influenza pandemic. The virulence of influenza A virus results from i) its easy spread by aerosol, ii) its ability to escape from protective immunity by frequent changes in viral antigens (antigenic drift), and iii) the periodic emergence of new virulent strains by reassortment of gene segments between viruses from two different species (antigenic shift). The threat of a new influenza pandemic persists as indicated by frequent outbreak of avian influenza viruses. RNA interference (RNAi) is a process by which double-stranded RNA (dsRNA) directs sequence-specific degradation of messenger RNA (mRNA). Studies have shown that in mammalian cells RNAi can be triggered by synthetic 19-29-nucleotide short interfering RNA duplexes (siRNA). Because siRNAs are too short to induce an interferon response in mammalian cells, yet still able to confer transient interference of gene expression in a sequence specific manner, they represent a new class of molecules that may have significant medical applications. We have found that siRNAs specific for conserved regions of influenza viruses potently inhibit virus production in cultured cells, embryonated chicken eggs, and mice. We are interested in developing siRNA as prophylaxis and/or therapies for influenza virus infection in humans.
</div>
|}