The relative frequency of codon use varies widely depending on the organism and organelle.  Many design programs for synthetic protein coding sequences allow the choice of organism.  The [http://www.kazusa.or.jp/codon/ codon usage database] has codon usage statistics for many common and sequenced organisms.  However, many times expression in more than one organism is desirable, often E. coli and a target organism, or S. cerevisiae and a target organism.

For these applications, a compromise codon usage table is required.  The codon usage table database lists the relative frequency of each possible codon for a particular amino acid.  By multiplying these relative frequencies and taking the square root, we calculate the geometric mean of each probability, which reflects the desirable compromise value.  The resulting numbers are then normalized such that the relative frequencies for each amino acid sum to 1.0, by dividing each result by the sum of all the codon frequencies for each amino acid.

The resulting compromise table can be saved and used as input to many of the protein coding region design programs, such as [http://www.dna20.com Gene Designer].

Codon tables:
* [[Escherichia coli/Codon usage | E. coli]]
* [[media:mf-codon.txt | Mesoplasma florum]]
* [[media:ec-mf-codon.txt | compromise E. coli and Mesoplasma florum codon usage table]]