<center>[[Davidson Missouri W| <span style="color:red">Home</span>]] | [[Davidson Missouri W/Background Information| <span style="color:red">Background Information</span>]] | [[Davidson Missouri W/Solving the HPP in vivo| <span style="color:red">Current Project: Solving the Hamiltonian Path Problem ''in vivo''</span>]] | [[Davidson Missouri W/Mathematical Modeling| <span style="color:red">Mathematical Modeling</span>]] | [[Davidson Missouri W/Gene splitting| <span style="color:black"> Gene Splitting </span>]] | [[Davidson Missouri W/Results| <span style="color:red">Results</span>]] | [[Davidson Missouri W/Traveling Salesperson Problem| <span style="color:red">Traveling Salesperson Problem</span> ]] | [[Davidson Missouri W/Software|<span style="color:red">Software</span>]] | [[Davidson Missouri W/Resources and Citations|<span style="color:red">Resources and Citations</span>]]</center>
<hr>

=What is Gene Splitting?=

"Gene splitting" refers to the insertion of a ''hixC'' site within the coding region of a gene.  Although this allows us to create edges for the simulation of a graph, it will change the protein sequence, potentially interfering with proper functionality.  We successfully inserted ''hixC'' in two different reporter genes, GFP and RFP.  Cells transformed with plasmids containing these "split" genes still fluoresce the appropriate color.

[[Image:Unsplit-gene.png|thumb|440px|none|A promoter upstream of a reporter gene will produce the expected phenotype.]]

[[Image:Split-gene.png|thumb|830px|none|The insertion of a ''hixC'' site, or 13 extra amino acids, might prevent the expected phenotype.]]

To facilitate the splitting process we developed software to help us.  Our [http://gcat.davidson.edu/iGEM07/genesplitter.html online] gene splitting web tool (click [[Davidson Missouri W/Web tool| here]] for a tutorial) helps us choose PCR primers that will amplify the appropriate segments of a gene of interest.

=The Genes=

{| border="1" cellpadding="5" cellspacing="0" align="center" width="100%"
|-
!style="color: red; background-color: black;" width="5%"| Gene
!style="color: red; background-color: black;" width="65%"| Description
!style="color: red; background-color: black;" width="30%"| Protein 3D Structure and ''hixC'' Insertion Point
|-

|style="color: black; background-color: green;" align="center"|GFP - Green Fluorescent Protein

|In the spring of 2007, Karen Acker pioneered the gene-splitting process by splitting GFP.  According to previous research, it had been shown that it was possible to insert extra amino acids within a particular region of the protein without disrupting green fluorescence.  Acker successfully inserted a ''hixC'' site at the same location, between amino acids 157 and 158.

|align="center"|[[Image:Gfp_insertion.png]]

|-

|style="color: black; background-color: red;" align="center"|DsRed - Red Fluorescent Protein

|After Acker demonstrated that GFP fluoresces despite a ''hixC'' insertion, we tested the same process on RFP, [http://parts.mit.edu/registry/index.php/Part:BBa_E1010 RFP] (from [http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=86600 ''Discosoma sp.'']).  Although its DNA sequence is markedly different from GFP's, it has some amino acid similarity and a remarkable structural similarity.  Both proteins have a Beta-barrel structure which surrounds an internal chromophore.

Fortunately, the similarity between GFP and RFP allowed us to make an educated guess for where to insert.  RFP's amino acid position 154 is homologous to GFP's amino acid position 157, which is where GFP was split.  This was therefore our best guess for where to insert the ''hixC'' site.

RFP was also successfully split.  However, the color is much weaker and takes a day at room temperature to become visible by eye.

|align="center"|[[Image:Rfp_insertion.png]]

|}

=A Closer Look=

Here is a more detailed view of splitting GFP.  The image on the left, with the protein structure highlighted in green, shows the normal, wild-type structure of the protein.  The image on the right highlights the two parts of the protein on either side of the ''hixC'' insertion, shown in pink.  The additional 13 amino acids are not shown, but they would extend from one pink amino acid into the next.

From these 3-dimensional PDB images it becomes apparent that the insertion likely sticks into open space away from the rest of the protein.  This increases the likelihood that the addition does not interfere with the beta barrel structure or with the chromophore.

As a consequence of RFP's similar 3-dimensional structure it is assumed the ''hixC'' insertion acts in a similar fashion.

[[Image:Gfp-nosplit.png|thumb|340px|left|The unmodified structure of GFP.]]
[[Image:Gfp-split.png|thumb|340px|none|The sections before and after the ''hixC'' insertion are highlighted in blue and yellow.  The amino acids at the insertion point are shown in pink.]]

<br>
<hr>
<center>
<[[Davidson Missouri W/Mathematical Modeling | Previous Section]] | [[Davidson Missouri W/Results | Next Section>]]
</center>