<font face="Arial" size="5"><center>'''A Genomics-enabled Microbial Observatory <br>in the Monterey Bay National Marine Sanctuary'''</center></font ><br>

Developing "Community Genome" Microarrays for Marine Microbial Ecology

{| cellspacing="0"
|- valign="center"
|width="50%" |
<div style="clear: right; text-align: center; float: center; padding: .1em 1.5em .9em">

[[Image:nsfe.gif|195px|center]]

</div>
|width="50%" |
<div style="clear: right; text-align: center; float: center; padding: .1em 1.5em .9em">

[[Image:ProSlideExample.jpg|195px|center]]


</div>
|width="50%" |
<div style="clear: right; text-align: center; float: center; padding: .1em 1.5em .9em">

[[Image:MontereyBay.gif|350px|center]]
</div>
|}


Previous efforts by the DeLong Lab in Monterey Bay have generated large-insert genomic “libraries” from genomic fragments of Monterey Bay microbes (each “book” in the library containing a large fragment of the DNA from a single marine microbe from the Bay). These libraries have been characterized extensively, both phylogenetically (who’s there?) through surveys of rDNA genes and functionally (what might they be doing?) through surveys of genes known to be involved in, for example, pulling carbon dioxide out of the atmosphere and turning it into cellular building blocks, or capturing sunlight and transforming it into energy. In this current Monterey Bay [http://www.lternet.edu/microbial_ecology/ Microbial Observatory] project funded by NSF, the genomic information in the libraries is now being used to develop microarrays targeting specific microbes within the Bay, to allow a high-throughput means for studying their ecology in the Bay across space and time. (Current technologies allow the tracking of one or a few microbial groups but generally not representatives across the whole community.) 


Monterey Bay is a long-term ecological research site, with the efforts of several major research institutions focused on understanding the Bay's oceanography. DNA samples have been collected by the DeLong Lab at monthly intervals since 1998, in tandem with the [http://www.mbari.org Monterey Bay Aquarium Research Institute's] (MBARI) [http://www.mbari.org/bog/ Biological Oceanography Group's] (BOG) research cruises. DNA samples from several sites and depths in Monterey Bay are collected and archived monthly by the DeLong Lab group, in conjunction with the MBARI Biological Oceanography Group’s regular CTD cruises. The microarrays will be used to query these archived time-series DNA samples, in order to track the targeted organisms across depth, space and over time. 


Among the specific questions we hope to address in this project are: What are the seasonal microbial population changes in Monterey Bay? How do these changes differ between the photic and subphotic communities? How do discrete oceanographic events like transient upwelling plumes influence the microbial community? What about larger-scale oceanographic events, such as El Ninos? In what ways, if any, does the community change along the coastal to offshore transect? The prototype array has been made and is currently being tested, and the next generation array will allow us to begin answering these questions. The central motivator underlying all these specific questions is our shocking ignorance about marine microbes, these fundamentally important microscopic drivers of our planet’s biogeochemistry. As our countries and citizens grapple with the complex issue of global change, we must provide the best information we can about how marine microbes respond to and control atmospheric composition, and to do that we must begin by figuring out who is there and how the communities respond to “normal” environmental perturbations.


'''Results:'''<br> 
Using a microarray design approach developed by the DeRisi Lab at UCSF for identifying and tracking viruses involved in human illness, we designed microarrays specific to the fragments of microbial DNA captured in the DeLong Lab genomic libraries. When tested against their perfect match target DNA, these microarrays show hybridization under standard conditions, and also can discriminate between closely related microbes. This specifity occurs not only with laboratory DNA samples but also with cells spiked into a coastal seawater - i.e., in a background of comples community DNA. We are testing the array further in the lab with increasingly complex mixtures of DNA, and diminishingly small amounts, to test where the limit of detection is for this array, and also to develop the optimal DNA amplification protocol to allow us to work with environmental DNA samples without skewing the community composition captured in the DNA.