==Types of Primers==
In general, primers are designed to identify specific locations within a long region of DNA, either plasmid or genomic.  Primer binding sites are ideally unique within the range of DNA  found in the reaction tube.  Single primers are used to amplify and label DNA fragments for sequencing reactions, or as probes for Southern blots.  Pairs of primers are used to delimit the range of DNA amplified during a PCR reaction.  Because biological enzymes selectively add bases on the 3' end of primed double stranded DNA, the binding of the 3' end of the primer is especially important, while the 5' end of the primer can either bind or not, with relatively little effect on most uses of the primer.  The binding of the 4-e 3' to the 5' end, the opposite from the DNA synthesis direction found in biological systems.  Synthesis starts with a glass bead filled column containing a specific 3' base bound to the glass.   Chemically activated nucleotides, termed phosphoramidites, are linked in 3' to 5' order to fabricate the DNA, which is then cleaved from the column using aqueous ammonia.  The use of chemical synthesis provides many opportunities to build DNA fragments with unnatural bases, which can be very useful in experimental settings.

The  fact that primers are synthesized 3' to 5' means that all or nearly all primers are correct and identical at the 3' end, which is important for locating the site of biological synthesis.  Synthesized primers typically have a significant error rate, with most 100 bp primers being incorrect in one or more locations.  The most common error is a truncation of the 5' end, followed by point deletions of single bases.  Long strings of G's are problematic for synthesis.

==Adding Restriction Sites onto your primers==
When adding restriction sites to your primers, check that there is sufficient space if that site is near the end of the primer for the enzyme to be able to sufficiently bind and cut.  As a general rule, 4-6 bp is a good length to protect your site with although this can vary to be as few as 1bp and as many as 9bp.  For more information, visit the [http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/cleavage_linearized_vector.asp NEB information site]


==Primer binding and melting temperature==

==Degenerate Codon Table==

::{| border="1" cellpadding="2" cellspacing="0"
|-
!Letter||Base(s)||mnemonic||complement
|-
|A||||Adenosine||T
|-
|C||||Cytosine||G
|-
|G||||Guanine||C
|-
|T||||Thymine||A
|-
|U||||Uracil||A
|-
|N||A C G T||aNy||N
|-
|Y||C T||pYrimidine||R
|-
|R||A G||puRine||Y
|-
|W||A T||Weak||W
|-
|S||C G||Strong||S
|-
|M||A C||aMino||K
|-
|K||G T||Keto||M
|-
|B||C G T||not A||
|-
|D||A G T||not C||
|-
|H||A C T||not G||
|-
|V||A C G||not T||

|}

==Unusual Bases and 5' modifications==

::{| border="1" cellpadding="2" cellspacing="0"
|-
!Base or modfication||Use
|-
|5-methyl Cytosine||methylated base, protects restriction site
|-
|Inosine||Pairs with C, A, less well with T, little with G
|-
|Abasic site||deoxyribose backbone without a base
|-
|2' methoxy-ribose bases||Terminates DNA replication on the opposite strand
|-
|5' phosphate||Allows ligation of PCR products
|-
|5'-Biotin||
|-
|5'-Fluorescein||
|-
|5'-amino||
|}

See the catalog of [http://www.glenresearch.com/ Glen Research] for overwhelming choice in modified nucleotides.

==General guidelines==
*Avoid runs over 3 nucleotide (GGGG)
*Long stretches of G, in particular, give problems.
*18-30bp in length.  [[Molecular Cloning]] says that 5' tails do not significantly affect annealing.
*Primer pairs should differ in length by less than 3bp.
*3â€™ end should be G or C (stronger bond)
*Primer melting temp (Tm) should be 50-60&deg;C with low FIR difference (<5&deg;C, <2&deg;C better) 
*[[Molecular Cloning]] advises GC content between 40% and 60%
*Avoid palindromes and inverted repeat sequences.
*Avoid complementarity between members of a primer pair.
*Check for dimer binding and hairpins in Vector NTI.
**Want to avoid structures with &Delta;G < -5kcal/mol
*Long primers (those approximately >50 bp or those needed for sensitive applications) should be purified.  Note that the purification step costs extra.  See the [https://www.invitrogen.com/contentFrame.cfm?pageid=100&contentPage=https://invitrogen.custhelp.com/cgi-bin/invitrogen.cfg/php/enduser/std_adp.php?p_faqid=22 Invitrogen FAQ on purification options] for more information on which purification method to choose.
**Fraction of synthesized oligo that will have the correct length = 0.99<sup>n-1</sup> where n = primer length.  [From [http://www.invitrogen.com/content.cfm?pageid=10039 Invitrogen's Custom DNA Oligo FAQ]]
*Verify that your primers are designed and ordered in the correct orientation (oligos are always specified 5' to 3', left to right).
*If you plan to cut your PCR product near the ends of the linear DNA fragment, note that some enzymes do not cut efficiently at the ends of linear DNA.  So include extra bases to increase the efficiency of cutting.  Many enzymes work with 4 bases supposedly but XhoI was found to require more than 4 bases (8 bases was used successfully).  Thus, to be on the safe side, use 8 bases whenever possible. [http://www.neb.com/ NEB] has more information [http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/cleavage_linearized_vector.asp here].  Read the information at NEB carefully ... they recommend adding 4 bases to the numbers listed in their table.
*[[User:tk | Tom's]] rule of thumb is that if a PCR fails, try it again.  The second time around, work a bit harder by varying the annealing temperature or something else.  If it fails again, redesign your primers.
*For primers used to generate PCR products for TA cloning, see [[Knight:TOPO TA cloning#Notes|notes]] for more information on efficient addition of 3'A to PCR products.

==BioBrick primers==
To BioBrick a part, the following tails should be added to your primers:
*PREFIX Primer        cctttctagag        11 bp
*SUFFIX Primer        tactagtagcggccgctgcagcctt        25 bp

The prefix primer adds an Xba I site, and the suffix adds the entire BB suffix (Spe I-Not I-Pst I)

Check the annealing temperature both without the tail (the first cycle or so) and with the tail (the later cycles).

==Turnaround time==

===Ordering primers within the US from Invitrogen===
''update based on your experience''
*2 days on primers without any purification
*3 days on primers with cartridge purification
*4+ days on primers with PAGE purification
*Orders placed after 4pm do not get processed until the following day.
*Standard Delivery Schedule: Primer orders (<45bp) placed Monday through Wednesday will be delivered within 24 to 48 hours. Orders placed Thursday through Friday will be delivered Tuesday. [From Invitrogen]

==Resources==

===Useful primer design tools===
*[http://synbio.mit.edu/tools/clipboard.cgi Austin's clipboard tool] - online tool for generating the complement, reverse complement and restriction enzyme site analysis of a DNA sequence.  It also translates the sequence and gives the amino acids properties.
*[http://scitools.idtdna.com/analyzer/Applications/OligoAnalyzer/] - A relatively complete suite of online tools for primer analysis.
*[https://catalog.invitrogen.com/index.cfm Invitrogen]
*[http://www.bioinformatics.vg/biolinks/bioinformatics/verbose/PCR%2520and%2520Primer%2520Design.shtml List of Primer Design Software]
*[http://perlprimer.sourceforge.net/ Perlprimer] - Open-source, cross-platform PCR primer design software
*[http://frodo.wi.mit.edu/primer3/primer3_code.html Primer3]
*[http://bioinformatics.org/primerx/index.htm PrimerX] - This is a useful online tool for designing primers for site directed mutagenesis.
*[[VectorNTI]]

===Oligo synthesis information===
*[http://www.invitrogen.com/content.cfm?pageid=10039 Invitrogen custom DNA oligo FAQ]
*[https://www.invitrogen.com/contentFrame.cfm?pageid=100&contentPage=https://invitrogen.custhelp.com/cgi-bin/invitrogen.cfg/php/enduser/std_adp.php?p_faqid=22 Invitrogen FAQ on oligo purification options]
*[https://www.invitrogen.com/contentFrame.cfm?pageid=100&contentPage=https://invitrogen.custhelp.com/cgi-bin/invitrogen.cfg/php/enduser/std_adp.php?p_faqid=1141 Invitrogen FAQ on size limit of HPLC or PAGE purified oligo]

[[Category:Protocol]]
[[Category:DNA]]
[[Category:In vitro]]