back to [[Evolutionary Stability]]

===Develop an algorithm for codon optimization of evolutionary stability===
I will develop an algorithm to codon optimize a system for increased or decreased genetic and performance stability.  Codon optimization enables the underlying DNA sequence to be changed while keeping the amino acid sequence constant, by taking advantage of redundancy in the genetic code.  I will use codon optimization to modulate the evolutionary stability of the system while leaving the system performance unaffected.  

By varying the DNA sequence through codon optimization I will modulate both genetic and performance stability.  For instance, genetic stability will be increased or decreased by changing the number of repeat homology regions, thereby encouraging or discouraging recombination.  Similarly, the frequency of G:C pairs may encourage or discourage point mutations and influence genetic stability.  Performance stability can be varied by choosing more or less ‘volatile’ codons (Plotkin et al., 2004).  Each codon can be given a volatility score based on its likelihood to mutate into a stop codon or a different amino acid, in particular ones with very different chemical properties.  By choosing a set of codons that are less volatile I expect system performance to be more reliable in the face of mutations.

The algorithm for codon optimization will need to consider both genetic and performance stability.  For example, choosing less volatile codons to increase performance stability could create repeated homologous regions that result in genetic instability, leading to a decrease in overall evolutionary stability.  Although the amino acid sequence remains unchanged following codon optimization, there may be second order effects from altering the codon sequence that affect system performance.  For example, genes with codon frequencies that are not equivalent to their respective tRNA frequencies in the host cell may have reduced expression rates due to inefficient use of cellular tRNA resources (Kurland, 1991).  In refining the algorithm I will develop a version that takes into account this ‘codon bias’ of the host organism in determining the optimal codon sequence.  Another concern is altered mRNA secondary structure that may influence regulation of protein synthesis and mRNA stability.

===Experimentally validate model predictions===
I will evaluate the effectiveness of the algorithm for codon optimization by measuring the stability of different codon sequences of a [[Endy:Counter-selectable markers for studying evolutionary stability|counter-selectable marker]] that are generated by de novo synthesis.  The codon sequence variats will be designed to have a range of predicted stabilities, both higher and lower than the wild-type codon sequence.  These variants will be grown in a chemostat and the culture will be sampled periodically and plated on media containing the counter selection in order to assay the performance stability of each codon sequence variant.  Mutants will be sequenced to better evaluate the genetic stability of the device.  Constitutive expression of GFP may provide an alternative system where loss of GFP function by mutation can be assayed via FACS.  The experimental results will be used to evaluate as well as improve the codon optimization algorithm.

==OWW Resources==
*[[Endy:Counter-selectable markers for studying evolutionary stability|Possible counter-selectable markers]]
*[[Endy:Evolutionary stability project]]

===Contact===
[[Jason Kelly]]