==General Information==
Brief notes of relevance to me at least, taken from [[Escherichia coli & Salmonella]] and [http://ecocyc.org ecocyc]--[[User:Bcanton|BC]] 16:37, 31 Jul 2005 (EDT)

*Restriction systems are only found in unicellular organisms.
**Either the cell modifies certain sequences so they are not restricted (classic R-M systems)
**or certain foreign modifications are restricted.

*'''''E. Coli'' K12 (MG1655) restriction systems'''
**According to information from [http://ecocyc.org Ecocyc] and [[Escherichia coli & Salmonella]], the following restriction enzymes are found in ''E. coli'' K12.
**Contains the type I restriction enzyme EcoK encoded by hsdR - cleaves -AAC(N<sub>6</sub>)GTCG- if the second A is unmethylated.
**Contains the Mcr systems described below.  These restriction systems do not appear to have a recognition sequence, they just cut at the methylated residues.
**Contains the Mrr system which cleaves m6A or m5c residues.  The sequence specificity is unknown.
''The presence of the hsdR gene in MG1655 may explain why plasmids (e.g pSB4A3 and pCH1497-ASD1) that contain the recognition sequence for EcoK function unreliably in MG1655.  By contrast, constructs (e.g I13273 on pSB3k3) that do not contain the recognition sequence for EcoK function well in MG1655.''

*'''Mcr Systems (found in ''E. coli'' strains)'''
**''McrBC'' - cleaves hm5C, m5C and m4C (hydroxymethylated or methylated cytosine at positions 4 or 5).  The modified  bases must be paired about 40-80bp apart.  (genes ''mcrB'' and ''mcrC'')
**''McrA'' - cleaves DNA methylated by HpaII and SssI methylases.  (encoded by the prophagelike e14 element)

==Classic R-M systems==
''A classic R-M system includes an endonuclease that cleaves a specific DNA sequence and a DNA methyltransferase that methylates either adenosyl or cytosyl residues within the same DNA sequence.''
*'''Type I R-M systems'''
**Relatively rare.
**One, three-subunit protein acts as both endonuclease and methylase.
**Requires AdoMet as a cofactor.
**Endonuclease activity also requires Mg<sup>2+</sup> and ATP.
**E. coli has a type 1 restriction system encoded by ''hsdR'', ''hsdM'', and ''hsdS''.

*'''Type II R-M systems'''
**Relatively abundant - these are the restriction enzymes found in the NEB catalog etc.
**Separate nuclease and methylation enzymes.
**Requires AdoMet as a cofactor.
**Endonuclease activity also requires Mg<sup>2+</sup>
**Little homology between restriction and methylation enzymes despite the shared DNA recognition sequence.
**Most recognition sequences are close to symmetric.
**Endonucleases normally act as homodimers, acting on sense and anti-sense strands.
**A subgroup of Type II restriction enzymes (Type IIS) cleave a fixed distance away from the recognition sequence.

*'''Type III R-M systems'''
**Relatively rare.
**Hetero-oligomeric protein catalyzes both the restriction and modification reactions.
**Requires AdoMet as a cofactor for modification.
**Endonuclease activity also requires Mg<sup>2+</sup> and ATP.
**Encoded by ''mod'' and ''res'' genes in ''E. coli''.
**Only one strand is methylated unlike the other systems.
**The restriction enzyme requires two recognition sites in inverse orientation.

*'''Regulation'''
**There is no evidence of transcriptional regulation of Type I R-M systems in ''E. coli''
**However, there must be some regulation as the modification subunit is always expressed before the restriction subunit.
**There is little clear information about the regulation of Type II R-M systems, in part because many of them come from poorly understood bacteria.
**Type II R-M regulation is also not very well understood.  However, the modification enzyme is again expressed prior to the restriction enzyme.

[[Category:Escherichia coli]]