==Nomenclature & Abbreviations==
{{:Escherichia coli/Nomenclature & Abbreviations}}

<!-- To edit, visit http://openwetware.org/index.php?title=Escherichia_coli/Nomenclature_%26_Abbreviations&action=edit -->

==Methylation Issues in ''E. coli''==
{{:Escherichia coli/Methylation}}

<!-- To edit, visit http://openwetware.org/index.php?title=Escherichia_coli/Methylation&action=edit -->

==Commonly used strains==

===AG1===
endA1 recA1 gyrA96 thi-1 relA1 glnV44 hsdR17(r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>)

===AB1157===
thr-1 leu-6 thi-1 lacY1 galK2 ara-14 xyl-5 mtl-1 proA2 his-4 argE3 str-31 tsx-33 supE44 rec+<br>
*Bachmann BJ: Derivation and genotypes of some mutant derivatives of Escherichia coli K-12.
Escherichia coli and Salmonella typhimurium. Cellular and Molecular Biology (Edited by: F C Neidhardt J L Ingraham KB Low B Magasanik M Schaechter H E Umbarger). Washington, D.C., American Society for Microbiology 1987, 2:1190-1219.

===BL21(AI)===
F<sup>–</sup> ompT gal dcm lon hsdS<sub>B</sub>(r<sub>B</sub><sup>-</sup> m<sub>B</sub><sup>-</sup>) araB::T7RNAP-tetA
*an ''E. coli'' B strain with carrying the T7 RNA polymerase gene in the ''araB'' locus of the ''araBAD'' operon<sup>q</sup>.
*Transformed plasmids containing T7 promoter driven expression are repressed until L-arabinose induction of T7 RNA polymerase.
*Derived from BL21.
*See the [https://catalog.invitrogen.com/index.cfm?fuseaction=viewCatalog.viewProductDetails&productDescription=595 product page] for more information.

===BL21(DE3)===
F<sup>–</sup> ompT gal dcm lon hsdS<sub>B</sub>(r<sub>B</sub><sup>-</sup> m<sub>B</sub><sup>-</sup>) &lambda;(DE3 [lacI lacUV5-T7 gene 1 ind1 sam7 nin5])
*an ''E. coli'' B strain with DE3, a &lambda; prophage carrying the T7 RNA polymerase gene and lacI<sup>q</sup>
*Transformed plasmids containing T7 promoter driven expression are repressed until IPTG induction of T7 RNA polymerase from a lac promoter.
*Derived from B834 ([http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?CMD=File&DB=pubmed Wood, 1966]) by transducing to Met<sup>+</sup>.  
*See the original Studier [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=3537305&query_hl=14 paper] or the summary in [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=2199796&query_hl=13&itool=pubmed_docsum Methods in Enzymology] for more details.

===BL21 (DE3) pLysS=== 
F<sup>-</sup> ompT gal dcm lon hsdS<sub>B</sub>(r<sub>B</sub><sup>-</sup> m<sub>B</sub><sup>-</sup>) &lambda;(DE3) pLysS(cm<sup>R</sup>)
* pLysS plasmid chloramphenicol resistant; grow with chloramphenicol  to retain plasmid
* Chloramphenicol resistant
* The pLysS plasmid encodes T7 phage lysozyme, an inhibitor for T7 polymerase which reduces and almost eliminates expression from transformed T7 promoter containing plasmids when not induced.
* see Moffatt87 for details of pLysS and pLysE plasmids

===BNN93===
F<sup>-</sup> tonA21 thi-1 thr-1 leuB6 lacY1 glnV44 rfbC1 fhuA1 mcrB e14-(mcrA<sup>-</sup>) hsdR(r<sub>K</sub>m<sub>K</sub>) &lambda;<sup>-</sup>
* Some C600 strains are really BNN93

===BW26434, CGSC Strain # 7658===
&Delta;(araD-araB)567, &Delta;(lacA-lacZ)514(::kan), lacIp-4000(lacI<sup>q</sup>), &lambda;<sup>-</sup>, rpoS396(Am)?, rph-1, &Delta;(rhaD-rhaB)568, hsdR514<br>
*This information is from a printout sent by the [http://cgsc.biology.yale.edu ''E. coli'' Genetic Stock Center] with the strain.
*B.L. Wanner strain
*rph-1 is a 1bp deletion that results in a frameshift over last 15 codons and has a polar effect on pyrE leading to suboptimal pyrimidine levels on minimal medium. (Jensen 1993 J Bact. 175:3401)
*&Delta;(araD-araB)567 was formerly called &Delta;araBAD<sub>AH33</sub> by Datsenko and Wanner
*Am = amber(UAG) mutation
*Reference: Datsenko and Wanner, 2000, PNAS, 97:6640

'''NOTE:''' 
*This promoter driving the expression of lacI was sequenced in this strain using a primer in mhpR (upstream of lacI) and a primer in the opposite orientation in lacI.  The lac promoter was found to be identical to wildtype.  Thus, the -35 sequence was GCGCAA not GTGCAA as expected with lacI<sup>q</sup>.  Therefore this strain (or at least the version obtained from the [http://cgsc.biology.yale.edu ''E. coli'' Genetic Stock Center]) does NOT appear to be lacI<sup>q</sup>.  According to Barry Wanner, this is an unexpected result. -[[User:Rshetty|Reshma]] 13:19, 5 May 2005 (EDT)
*"We have now confirmed that BW25113, BW25141, and BW26434 are all lacI<sup>+</sup>, and not lacI<sup>q</sup>.  We thank you for alerting us to the error with respect to BW26434.  Apparently, the lacI region was restored to wild-type in a predecessor of BW25113." (from Barry Wanner November 18, 2005)

===C600===
F<sup>-</sup> tonA21 thi-1 thr-1 leuB6 lacY1 glnV44 rfbC1 fhuA1 &lambda;<sup>-</sup>
*There are strains circulating with both e14+(mcrA<sup>+</sup>) and e14-(mcrA<sup>-</sup>)
*General purpose host
*See [http://cgsc.biology.yale.edu:80/cgi-bin/sybgw/cgsc/Strain/11195 CGSC#3004]
*References: Appleyard, R.K. (1954) Genetics 39, 440; Hanahan, D. (1983) J. Mol. Biol. 166, 577.

===C600 hflA150 (Y1073, BNN102) ===
F<sup>-</sup> thi-1 thr-1 leuB6 lacY1 tonA21 glnV44 &lambda;<sup>-</sup> hflA150(chr::Tn10)
*host for repressing plaques of &lambda;gt10 when establishing cDNA libraries
*Reference Young R.A. and Davis, R. (1983) Proc. Natl. Acad. Sci. USA 80, 1194.
*Tetracycline resistance from the Tn10 insertion

===CSH50===
ara &Delta;(lac-pro) rpsL thi supD+ fimE::IS1
*Reference: Blomfeld et al., J.Bact. 173: 5298-5307, 1991.

===D1210===
HB101 lacI<sup>q</sup>  lacY<sup>+</sup>

===DB3.1===
F- gyrA462 endA1 glnV44 &Delta;(sr1-recA) mcrB mrr hsdS20(r<sub>B</sub><sup>-</sup>, m<sub>B</sub><sup>-</sup>) ara14 galK2 lacY1 proA2 rpsL20(Sm<sup>r</sup>) xyl5 &Delta;leu mtl1
*useful for propagating plasmids containing the [[ccdB]] operon.
*gyrA462 enables ccdB containing plasmid propagation
*streptomycin resistant
*appears to NOT contain lacI (based on a colony PCR) --[[User:Austin J. Che|Austin Che]] 16:16, 18 June 2007 (EDT)
<biblio>
#Bernard-JMolBiol-1992 pmid=1324324
#Miki-JMolBiol-1992 pmid=1316444
</biblio>

===DH1===
endA1 recA1 gyrA96 thi-1 glnV44 relA1 hsdR17(r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>) &lambda;<sup>-</sup>
* parent of DH5&alpha;
*An Hoffman-Berling 1100 strain derivative (Meselson68)
* more efficient at transforming large (40-60Kb) plasmids
*nalidixic acid resistant
*Reference: Meselson M. and Yuan R. (1968) Nature 217:1110 PMID 4868368.

===DH5&alpha;===
F<sup>-</sup> endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG &Phi;80d''lacZ''&Delta;M15 &Delta;(''lacZYA-argF'')U169, hsdR17(r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>), &lambda;–

*An Hoffman-Berling 1100 strain derivative (Meselson68)
*Promega also lists phoA
*nalidixic acid resistant
*References:
**FOCUS (1986) 8:2, 9.
**Hanahan, D. (1985) in DNA Cloning: A Practical Approach (Glover, D.M., ed.), Vol. 1, p. 109, IRL Press, McLean, Virginia.
**Grant, S.G.N. et al. (1990) Proc. Natl. Acad. Sci. USA 87: 4645-4649 PMID 2162051.
** Meselson M. and Yuan R. (1968) Nature 217:1110 PMID 4868368.

===DH10B (Invitrogen)===
F<sup>-</sup> endA1 recA1 galE15 galK16 deoR nupG rpsL &Delta;lacX74 &Phi;80lacZ&Delta;M15 araD139 &Delta;(ara,leu)7697 mcrA &Delta;(mrr-hsdRMS-mcrBC) &lambda;<sup>-</sup>
*suitable for cloning methylated cytosine or adenine containing DNA
*an MC1061 derivative (Casadaban80).  Prepare cells for chemical transformation with CCMB80 buffer
*blue/white selection
*While DH10B has been classically reported to be galU galK, the preliminary genome sequence for DH10B indicates that DH10B (and by their lineage also TOP10 and any other MC1061 derivatives) is actually galE galK galU<sup>+</sup>.  '''[[User:Dcekiert|Dcekiert]] 16:37, 23 January 2008 (CST)'''
*Promega also lists deoR (but see Top10 cell genotype)
*Streptomycin resistant
*References: 
**Casdaban, M. and Cohen, S. (1980) J Mol Biol 138:179   PMID 6997493.
**Grant, S.G.N. et al. (1990) Proc. Natl. Acad. Sci. USA 87: 4645-4649 PMID 2162051.
**[http://cgsc2.biology.yale.edu/Strain.php?ID=11225 E. coli Genetic Stock Center, MC1061 Record]
**[http://www.hgsc.bcm.tmc.edu/projects/microbial/microbial-detail.xsp?project_id=105 DH10B Genome Sequencing Project, Baylor College of Medicine]

===DH12S (Invitrogen)===
mcrA &Delta;(mrr-hsdRMS-mcrBC) &phi;80d lacZ&Delta;M15 &Delta;lacX74 recA1 deoR &Delta;(ara, leu)7697 araD139 galU galK rpsL F' [proAB<sup>+</sup> lacI<sup>q</sup>Z&Delta;M15]
*host for phagemid and M13 vectors
*useful for generating genomic libraries containing methylated cytosine or adenine residues
*streptomycin resistant
*References: Lin, J.J., Smith, M., Jessee, J., and Bloom, F. (1991) FOCUS 13, 96.; Lin, J.J., Smith, M., Jessee, J., and Bloom, F. (1992) BioTechniques 12, 718.

===DM1 (Invitrogen)===
F- dam-13::Tn9(Cm<sup>R</sup>) dcm- mcrB hsdR-M+ gal1 gal2 ara- lac- thr- leu- tonR tsxR Su0

*Host for pBR322 and other non-pUC19 plasmids; useful for generating plasmids that can be cleaved with dam and dcm sensitive enzymes
*Chloramphenicol resistant
*Promega lists as F' not F-
*Reference: Lorow-Murray D and Bloom F (1991) Focus 13:20

===ER2566 (NEB)===
F- &lambda;- fhuA2 [lon] ompT lacZ::T7 gene 1 gal sulA11 &Delta;(mcrC-mrr)114::IS10 R(mcr-73::miniTn10-TetS)2 R(zgb-210::Tn10)(TetS) endA1 [dcm]

*Host strain for the expression of a target gene cloned in the pTYB vectors.
*Carry a chromosomal copy of the T7 RNA polymerase gene inserted into ''lacZ'' gene and thus under the control of the lac promoter.  In the absence of IPTG induction expression of T7 RNA polymerase is suppressed by the binding of ''lac'' I repressor to the ''lac'' promoter.
*Deficient in both ''lon'' and ''ompT'' proteases.

===ER2267 (NEB)===
F´ proA+B+ lacIq Δ(lacZ)M15 zzf::mini-Tn10 (KanR)/ Δ(argF-lacZ)U169 glnV44 e14-(McrA-) rfbD1? recA1 relA1? endA1 spoT1? thi-1 Δ(mcrC-mrr)114::IS10

*Commonly used for titering M13 phage because of the strain's F' plasmid, which carries KanR, and its slow growth, which promotes easy visualization of plaques.

===HB101===
F<sup>-</sup> mcrB mrr hsdS20(r<sub>B</sub><sup>-</sup> m<sub>B</sub><sup>-</sup>) recA13 leuB6 ara-14 proA2 lacY1 galK2 xyl-5 mtl-1 rpsL20(Sm<sup>R</sup>) glnV44 &lambda;<sup>-</sup>

Please note that different sources have different genotypes so treat this information with caution.<br>
*From a GIBCO BRL list of competent cells.

*Hybrid of E. coli K12 and E. coli B
*Host for pBR322 and many plasmids
*Sigma lists the deletion &Delta;(gpt,proA).  Check this.
*Promega does not list F-, mcrB, or mrr
*Streptomycin resistant
*References:
**Boyer, H.W. and Roulland-Dussoix, D. (1969) J. Mol. Biol. 41, 459.
**Smith, M., Lorow, D., and Jessee, J. (1989) FOCUS 11, 56.
**Lacks S and Greenberg JR (1977) J Mol Biol 114:153.
===IJ1126===
E. coli K-12 recB21 recC22 sbcA5 endA gal thi Su+ Δ(mcrC-mrr)102::Tn10<br>
See [[Endy:IJ1126]]
===IJ1127===
[[#IJ1126|IJ1126]] lacUV5 lacZ::T7 gene1-Knr<br>
See [[Endy:IJ1127]]

===JM83===
rpsL ara &Delta;(lac-proAB) &Phi;80dlacZ&Delta;M15
*Sigma lists thi.  Check this.
*streptomycin resistant

===JM101===
glnV44 thi-1 &Delta;(lac-proAB) F'[lacI<sup>q</sup>Z&Delta;M15 traD36 proAB<sup>+</sup>]
*host for M13mp vectors
*recA<sup>+</sup>, r<sub>K</sub><sup>+</sup>
*original blue/white cloning strain
*has all wt restriction systems
*References: Messing, J. et al. (1981) Nucleic Acids Res. 9, 309; Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Gene 33, 103.

===JM103===
endA1 glnV44 sbcBC rpsL thi-1 &Delta;(lac-proAB) F'[traD36 proAB<sup>+</sup> lacI<sup>q</sup> lacZ&Delta;M15]
*streptomycin resistant
*References: Hanahan, D. (1983) J. Mol. Biol. 166:557-80.
*NEB says this strain encodes a prophage encoded EcoP1 endonuclease.
*Sigma lists (P1) (r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup> rP1<sup>+</sup> mP1<sup>+</sup>)

===JM105===
endA1 glnV44 sbcB15 rpsL thi-1 &Delta;(lac-proAB) [F'  traD36 proAB<sup>+</sup> lacI<sup>q</sup> lacZ&Delta;M15] hsdR4(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup>)
*Sigma lists sbcC
*streptomycin resistant
*References: Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Gene 33, 103.

===JM106===
endA1 glnV44 thi-1 relA1 gyrA96 &Delta;(lac-proAB) F<sup>-</sup> hsdR17(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup>)
*References: Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Gene 33, 103.

===JM107===
endA1 glnV44 thi-1 relA1 gyrA96 &Delta;(lac-proAB) [F'  traD36 proAB<sup>+</sup> lacI<sup>q</sup> lacZ&Delta;M15]
hsdR17(R<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>) &lambda;<sup>-</sup>
*host for M13mp vectors
*recA+, r<sub>K</sub>+
*Sigma lists e14- (McrA-)
*nalidixic acid resistant
*References: Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Gene 33, 103.

===JM108===
endA1 recA1 gyrA96 thi-1 relA1 glnV44 &Delta;(lac-proAB) hsdR17 (r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>)
*nalidixic acid resistant

===JM109===
endA1 glnV44 thi-1 relA1 gyrA96 recA1 mcrB<sup>+</sup> &Delta;(lac-proAB) e14- [F'  traD36 proAB<sup>+</sup> lacI<sup>q</sup> lacZ&Delta;M15] hsdR17(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup>)
*From [http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/ecoli_genotypes.asp NEB]
*Partly restriction-deficient; good strain for cloning repetitive DNA (RecA<sup>–</sup>).
*Suppresses many amber mutations when glutamine is acceptable but not the S<sub>100</sub> or S<sub>7</sub> mutations of &lambda;, e.g., &lambda;gt11. 
*Can also be used for M13 cloning/sequencing and blue/white screening.<br>
*Sigma lists e14-
*nalidixic acid resistant
*From C. Yanisch-Perron, J. Vieira, and J. Messing. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. ''Gene'', 33(1):103–19, 1985.<br>
* Some information from Mary Berlyn at the [http://cgsc.biology.yale.edu/ ''E. coli'' Genetic Stock Center]: One of the reasons the original curator of this collection did not accession the JM109, JM103, etc. strains was because she found it impossible to be sure of the derivation and therefore the details of the genotype. But I think it's safe to assume that the F' in this strain is derived from or similar to F128 which extends from the proBA region through the lac operon.  It thus carries the wildtype genes for all loci in that region except those indicated as mutant for the genotype of the F'.  So it carries the lacZ (alpha-complementation) deletion lacZ58(M150 and the lacI mutation lacIq, but it has the lacY+ gene also on the F-prime.  On the chromosome it lacks all the lac operon genes.

'''NOTE:''' The promoter driving the expression of lacI was sequenced in this strain using a primer in mhpR (upstream of lacI) and a primer in the opposite orientation in lacI.  The lac promoter was found to be identical to wildtype.  Thus, the -35 sequence was GCGCAA not GTGCAA as expected with lacI<sup>Q</sup>.  Therefore this strain (or at least the version we have) does NOT appear to be lacI<sup>Q</sup> unless there is another copy of lacI elsewhere. This result is somewhat confirmed by the fact that a lacI regulated promoter driving expression of YFP on a medium copy vector does not repress completely. -[[User:Rshetty|Reshma]] 13:48, 5 May 2005 (EDT)

===JM109(DE3)===
JM109 + &lambda;(DE3)
* DE3 prophage carrying T7 polymerase expression cassette
* Same cassette as BL21(DE3) carrying a lac inducible T7 RNA polymerase and lacI<sup>q</sup>
*nalidixic acid resistant

===JM110===
rpsL thr leu thi lacY galK galT ara tonA tsx dam dcm glnV44 &Delta;(lac-proAB) e14- [F'  traD36 proAB<sup>+</sup> lacI<sup>q</sup> lacZ&Delta;M15] hsdR17(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup>)
*Sigma fails to list tonA tsx e14 fhuA hsdR17
* (e14-) status uncertain
*streptomycin resistant

===JM2.300===
*Some folks have been using this strain (i.e., Elowitz, Gardner) and it took me too long to find the CGSC#.  
*The CGSC# is 5002.  [http://cgsc.biology.yale.edu/cgi-bin/sybgw/cgsc/Strain/query Check it out] if you need.
*CGSC does not like this strain number -- tk 12/7/05

===LE392===
glnV44 supF58 (lacY1 or &Delta;lacZY) galK2 galT22 metB1 trpR55 hsdR514(r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup>)
*Sigma lists F- e14-

===Mach1===
&Delta;recA1398 endA1 tonA &Phi;80&Delta;lacM15 &Delta;lacX74 hsdR(r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>)
* From Invitrogen
* Doubling time approx. 50 min and supposedly fastest growing chemically competent cloning strain available
* Mach1 cells are derivatives of E. coli W strains (ATCC 9637, S. A. Waksman), rather than E. coli K-12.  This may have implications for BL-1 status for some facilities (apparently not for MIT).
* See [[Media:pat6709852.pdf | Bloom04]] patent for details on the construction and properties of this strain.

===MC1061===
hsdR2 hsdM+ hsdS+ araD139 &Delta;(ara-leu)7697 &Delta;(lac)X74  galE15 galK16 rpsL (StrR) mcrA mcrB1
*Streptomycin resistant
*Parent of DH10B/TOP10 and derived strains
*References:
**[http://cgsc2.biology.yale.edu/Strain.php?ID=11225 E. coli Genetic Stock Center, MC1061 Record]
**Casdaban, M. and Cohen, S. (1980) J Mol Biol 138:179   PMID 6997493.

===MC4100===
F<sup>-</sup> araD139 &Delta;(argF-lac)U169* rspL150 relA1 flbB5301 fruA25&Dagger; deoC1 ptsF25  e14-

This [http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=12618467 paper] compares MC4100 to MG1655 and describes the significant deletions.

<nowiki>*</nowiki>The paper referenced above showed that this deletion was larger than previously known.  The deletion now covers ykfD-b0350.  

&Dagger;The fruA25 allele is attributed to the deletion of fruB-yeiR.  This means fruA is present but its promoter has been deleted.  

The paper also shows that the e14 element is deleted in MC4100.  One of the genes removed by this deletion is mcrA, which encodes an enzyme that restricts DNA containing methylcytosine.  However, other ''E. coli'' K-12 restriction/modification systems are still present in MC4100.  MC4100 still encodes the McrBC 5-methylcytosine=specific restriction enzyme and the HsdR/HsdS/HsdM type I restriction-modification complex.

Table three of the paper lists all genes believed to be deleted in MC4100.  The methods used in the paper can detect deletions but not loss of function mutations.

===MG1655===
F<sup>-</sup> &lambda;<sup>-</sup> ilvG- rfb-50 rph-1

This is the "wild type" K-12 strain which was sequenced, and should be used when PCRing genes from the sequenced genome.  It also looks very healthy under the microscope -- a dramatic difference from most of the cloning strains, which appear sick.

*See [http://cgsc.biology.yale.edu:80/cgi-bin/sybgw/cgsc/Strain/4837 CGSC#6300]
*See ATCC 700926
<biblio>
#Blattner-Science-1997 pmid=9278503
</biblio>
*More accurate sequence correcting 243 errors in the original sequencing<cite>Horiuchi2006</cite>.  New Genbank accession number U00096.2

===Rosetta(DE3)pLysS===
F<sup>-</sup> ompT hsdS<sub>B</sub>(R<sub>B</sub><sup>-</sup> m<sub>B</sub><sup>-</sup>) gal dcm &lambda;(DE3 [lacI lacUV5-T7 gene 1 ind1 sam7 nin5]) pLysSRARE (Cam<sup>R</sup>)
*an ''E. coli'' B strain with DE3, a &lambda; prophage carrying the T7 RNA polymerase gene and lacI<sup>q</sup>
*Transformed plasmids containing T7 promoter driven expression are repressed until IPTG induction of T7 RNA polymerase from a lac promoter.
* Chloramphenicol resistant
* pLysSRARE contains tRNA genes argU, argW, ileX, glyT, leuW, proL, metT, thrT, tyrU, and thrU.  The rare codons AGG, AGA, AUA, CUA, CCC, and GGA are supplemented.
* The pLysS plasmid encodes T7 phage lysozyme, an inhibitor for T7 polymerase which reduces and almost eliminates expression from transformed T7 promoter containing plasmids when not induced.
* see Moffatt87 for details of pLysS and pLysE plasmids
*[http://www.clinalfa.com/docs/docs/PROT/TB009.pdf Novagen strain manual]

===Rosetta-gami(DE3)pLysS===
&Delta;(ara-leu)7697 &Delta;lacX74 &Delta;phoA PvuII phoR araD139 ahpC galE galK rpsL (DE3) F'[lac<sup>+</sup> lacI<sup>q</sup> pro] gor522::Tn10 trxB pLysSRARE (Cam<sup>R</sup>, Str<sup>R</sup>, Tet<sup>R</sup>)
*an ''E. coli'' K-12 strain with DE3, a &lambda; prophage carrying the T7 RNA polymerase gene and lacI<sup>q</sup>
*Transformed plasmids containing T7 promoter driven expression are repressed until IPTG induction of T7 RNA polymerase from a lac promoter.
* ahpC mutation allows trxB/gor double mutants to grow in the absence of reducing medium
* pLysSRARE contains tRNA genes argU, argW, ileX, glyT, leuW, proL, metT, thrT, tyrU, and thrU.  The rare codons AGG, AGA, AUA, CUA, CCC, and GGA are supplemented.
* The pLysS plasmid encodes T7 phage lysozyme, an inhibitor for T7 polymerase which reduces and almost eliminates expression from transformed T7 promoter containing plasmids when not induced.
* see Moffatt87 for details of pLysS and pLysE plasmids
* Chloramphenicol resistant
* Kanamycin resistant
* Tetracycline resistant
* Streptomycin resistant
*[http://www.clinalfa.com/docs/docs/PROT/TB009.pdf Novagen strain manual]

===RR1===
HB101 recA<sup>+</sup>

===STBL2 (Invitrogen)===
F- endA1 glnV44 thi-1 recA1 gyrA96 relA1 &Delta;(lac-proAB) mcrA &Delta;(mcrBC-hsdRMS-mrr)  &lambda;<sup>-</sup>
*host for unstable sequences such as retroviral sequences and direct repeats
*nalidixic acid resistant
*References: Trinh, T., Jessee, J., Bloom, F.R., and Hirsch, V. (1994) FOCUS 16, 78.

===STBL4===
endA1 glnV44 thi-1 recA1 gyrA96 relA1 &Delta;(lac-proAB) mcrA &Delta;(mcrBC-hsdRMS-mrr)  &lambda;<sup>-</sup> gal F'[ proAB<sup>+</sup> lacI<sup>q</sup> lacZ&Delta;M15 Tn10]
*Tetracycline resistant (from Tn10 insertion)
*STBL2 + blue/white selection

===SURE (Stratagene)===
endA1 glnV44 thi-1 gyrA96 relA1 lac recB recJ sbcC umuC::Tn5 uvrC e14- &Delta;(mcrCB-hsdSMR-mrr)171 F'[ proAB<sup>+</sup> lacI<sup>q</sup> lacZ&Delta;M15 Tn10]
* uncertain status of TraD36 in F plasmid
* increased stability for inverted repeats and Z-DNA
* nalidixic acid resistant
* kanamycin resistant
* tetracycline resistant

===SURE2 (Stratagene)===
endA1 glnV44 thi-1 gyrA96 relA1 lac recB recJ sbcC umuC::Tn5 uvrC e14- &Delta;(mcrCB-hsdSMR-mrr)171 F'[ proAB<sup>+</sup> lacI<sup>q</sup> lacZ&Delta;M15 Tn10 Amy Cm<sup>R</sup>]
* increased stability for inverted repeats and Z-DNA
* nalidixic acid resistant
* kanamycin resistant
* tetracycline resistant
* chloramphenicol resistant for < 40 &mu;g/ml, sensitive for > 100 &mu;g/ml

===TOP10 (Invitrogen)===
F- mcrA &Delta;(mrr-hsdRMS-mcrBC) &phi;80lacZ&Delta;M15 &Delta;lacX74 deoR nupG recA1 araD139 &Delta;(ara-leu)7697 galE15 galK16 rpsL(Str<sup>R</sup>) endA1 &lambda;<sup>-</sup>
*Very similar to DH10B
**I actually emailed Invitrogen and asked if DH10B and TOP10 are the same strain or what.  Their response: "Thank you for contacting Invitrogen Technical Support.TOP10 and DH10B competent cells are closely related. They have the same genotypes and can used for the same applications. You can also choose from those that are Chemically competent or electrocomp cells. I hope this information answers your questions."  So either there is a difference that they don't want to put out there, or they have rebranded DH10B as TOP10 for marketing purposes... --[[User:Dcekiert|Dcekiert]] 18:55, 23 January 2008 (CST)
*While DH10B has been classically reported to be galU galK, the preliminary genome sequence for DH10B indicates that DH10B (and by their lineage also TOP10 and any other MC1061 derivatives) is actually galE galK galU<sup>+</sup>.  --[[User:Dcekiert|Dcekiert]] 16:45, 23 January 2008 (CST)
*Promega also lists deoR (but see Top10 cell genotype)
*Streptomycin resistant
*an MC1061 derivative <cite>Casadaban-JMolBiol-1980</cite>
*Streptomycin resistant
* Prepare cells for chemical transformation with CCMB80 buffer [[TOP10 chemically competent cells|Here]]
* Contain lacI based on a colony PCR (even though lacX74 supposedly deletes the lac operon!) --[[User:Austin J. Che|Austin Che]] 16:16, 18 June 2007 (EDT)
** φ80lacZ&Delta;M15 actually contains the entire lac operon, including lacI<sup>q</sup> --[[User:Dcekiert|Dcekiert]] 16:45, 23 January 2008 (CST)
*References:
**[http://cgsc2.biology.yale.edu/Strain.php?ID=11225 E. coli Genetic Stock Center, MC1061 Record]
**[http://www.hgsc.bcm.tmc.edu/projects/microbial/microbial-detail.xsp?project_id=105 DH10B Genome Sequencing Project, Baylor College of Medicine]
<biblio>
#Casadaban-JMolBiol-1980 pmid=6997493
#Grant-PNAS-1990 pmid=2162051
</biblio>

===Top10F' (Invitrogen)===
F'[lacI<sup>q</sup> Tn10(tet<sup>R</sup>)] mcrA &Delta;(mrr-hsdRMS-mcrBC) &phi;80lacZ&Delta;M15 &Delta;lacX74 deoR nupG recA1 araD139 &Delta;(ara-leu)7697 galU galK rpsL(Str<sup>R</sup>) endA1 &lambda;<sup>-</sup>
*Very similar to DH10B with F plasmid containing lacI<sup>q</sup> and Tn10
*While DH10B has been classically reported to be galU galK, the preliminary genome sequence for DH10B indicates that DH10B (and by their lineage also TOP10 and any other MC1061 derivatives) is actually galE galK galU<sup>+</sup>.  --[[User:Dcekiert|Dcekiert]] 16:45, 23 January 2008 (CST)
*Tetracycline resistant
*Streptomycin resistant
*an MC1061 derivative <cite>Casadaban-JMolBiol-1980</cite>
*References:
**[http://cgsc2.biology.yale.edu/Strain.php?ID=11225 E. coli Genetic Stock Center, MC1061 Record]
**[http://www.hgsc.bcm.tmc.edu/projects/microbial/microbial-detail.xsp?project_id=105 DH10B Genome Sequencing Project, Baylor College of Medicine]

===W3110===
F<sup>-</sup> &lambda;<sup>-</sup> rph-1 INV(rrnD, rrnE)
* See [http://cgsc.biology.yale.edu:80/cgi-bin/sybgw/cgsc/Strain/5446 CGSC#4474]
* See ATCC 39936
* See <cite>Horiuchi-MSB-2006</cite>. Briefly, there are 8 site (9nt) differences between W3110 and MG1655.  They reside in 7 orgs and one rRNA gene.  Two are nonfunctional (''rpoS'' and ''dcuA'') and 5 are unknown missense mutations.  
* New annotation has accession number DDBJ AP009048.

===XL1-Blue (Stratagene)===
endA1 gyrA96(nal<sup>R</sup>)  thi-1  recA1 relA1 lac glnV44 F'[ ::Tn10 proAB<sup>+</sup> lacI<sup>q</sup> &Delta;(lacZ)M15] hsdR17(r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>)
*nalidixic acid resistant
*tetracycline resistant (carried on the F plasmid)

===XL2-Blue (Stratagene)===
endA1 gyrA96(nal<sup>R</sup>)  thi-1  recA1 relA1 lac glnV44 F'[ ::Tn10 proAB<sup>+</sup> lacI<sup>q</sup> &Delta;(lacZ)M15 Amy Cm<sup>R</sup>] hsdR17(r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>)
*nalidixic acid resistant
*tetracycline resistant (carried on the F plasmid)
*chloramphenicol resistant for <40 &mu;g/ml; sensitive for >100 &mu;g/ml

===XL2-Blue MRF' (Stratagene)===
endA1 gyrA96(nal<sup>R</sup>)  thi-1  recA1 relA1 lac glnV44 e14- &Delta;(mcrCB-hsdSMR-mrr)171 recB recJ sbcC umuC::Tn5 uvrC
F'[ ::Tn10 proAB<sup>+</sup> lacI<sup>q</sup> &Delta;(lacZ)M15 Amy Cm<sup>R</sup>] 

*Minus Restriction strain (minus mcrA mcrCB mcrF mrr hsdR)
*nalidixic acid resistant
*kanamycin resistant
*tetracycline resistant (carried on the F plasmid)
*chloramphenicol resistant <40 &mu;g/ml, sensitive >100&mu;g/ml

===XL1-Red (Stratagene)===
F- endA1 gyrA96(nal<sup>R</sup>)  thi-1  relA1 lac glnV44 hsdR17(r<sub>K</sub><sup>-</sup> m<sub>K</sub><sup>+</sup>) mutS mutT mutD5 Tn10
*nalidixic acid resistant
*tetracycline resistant
*mutator strain, produces highly unstable DNA changes
*colonies grow and mutate so quickly that the strain is sick and mutated constructs must be moved rapidly to stable strains for plasmid isolation

===XL10-Gold (Stratagene)===
endA1 glnV44 recA1 thi-1 gyrA96 relA1 lac Hte &Delta;(mcrA)183 &Delta;(mcrCB-hsdSMR-mrr)173 tet<sup>R</sup>  F'[proAB lacI<sup>q</sup>Z&Delta;M15 Tn10(Tet<sup>R</sup>  Amy Cm<sup>R</sup>)]
* Tetracycline and Chloramphenicol resistant
*Nalidixic acid resistant
*Hte phenotype allows high transformation with large plasmid inserts

===XL10-Gold KanR (Stratagene)===
endA1 glnV44 recA1 thi-1 gyrA96 relA1 lac Hte &Delta;(mcrA)183 &Delta;(mcrCB-hsdSMR-mrr)173 tet<sup>R</sup>  F'[proAB lacI<sup>q</sup>Z&Delta;M15 Tn10(Tet<sup>R</sup>  Amy Tn5(Kan<sup>R</sup>)]
* Tetracycline and Kanamycin resistant
*Nalidixic acid resistant
*Hte phenotype allows high transformation with large plasmid inserts

==Other genotype information sources==

* [[E. coli genotypes/Exhibit]]: Test of moving information on this page into a wiki database

*[http://www.neb.com/nebecomm/products/category64.asp? NEB strains] [http://www.neb.com/nebecomm/products/category95.asp Other NEB strains]
*[http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/genetic_markers.asp NEB genotype information]
*[http://216.247.204.235/technical-main.html Teknova]
*[http://www.promega.com/guides/cloning_guide/appendix.pdf Promega]
*[http://www.embl-hamburg.de/~geerlof/webPP/straindb/bact_strains/bact_strains.html EMBL]
*[http://www.emdbiosciences.com/docs/docs/PROT/TB009.pdf Novagen/EMD]
*[http://www.sigmaaldrich.com/Area_of_Interest/Life_Science/Molecular_Biology/Cloning_and_Expression/Key_Resources/Genotypes_of_E_coli.html Sigma]

*[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=genome&cmd=Retrieve&dopt=Overview&list_uids=115 Escherichia coli K12 genome browser]

==References==
<biblio>
#Horiuchi-MSB-2006 pmid=16738553
#Novick-BacteriolRev-1976 pmid=1267736
#Lim46 Lim A, Dimalanta ET, Potamousis KD, Apodaca J, Ananthara-man TS, and Witkin, EM.  ''Inherited differences in sensitivity to radiation in Escherichia coli.'' Proc Natl Acad Sci USA 1946 32:59-68  (the original B strain reference).
#Moffatt87 pmid=3568126
</biblio>

[[Category:Escherichia coli]]