__NOTOC__
==Overview==
==Brink orthogonal ribosomes==
{{size|2|'''RBS design'''}} {{hide|1=[[Image:DedicatedTranslation.gif|thumb|right|400px]]
The first RBS I designed was [http://parts2.mit.edu/r/parts/partsdb/view.cgi?part_id=4699 BBa_B0036].  This part uses the RBS described by [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=7758959&query_hl=1 Brink et al.].  The spacing between the RBS and the start codon was set to be the same as that of the consensus ''E. coli'' RBS sequence.  

The RiPS from [http://parts2.mit.edu/r/parts/partsdb/view.cgi?part_id=4699 BBa_B0036] are not as high as I would like.  Hence I am designing a new RBS.  For this new [http://parts2.mit.edu/r/parts/partsdb/view.cgi?part_id=6348 RBS], I will use the spacing between RBS and start codon used by Brink and coworkers.  I have obtained high levels of translation using this spacing as shown in the protein gel below using the reporter construct pSD-rbsK2 (obtained from Chris Hayes)

To increase the spacing, I am using the same 5bp sequence as used by Brink et al.  This sequence doesn't appear to form a restriction site (it is the first 5bp of an XbaI site) or an E. coli RBS.  

<br style="clear:both" />}}

==Chin orthogonal ribosomes==
{{size|2|'''RBS Design'''}}<br>
[http://parts.mit.edu/registry/index.php/User:Bcanton/Dedicated_RBSs#Design_notes_for_mRNA-A_derivatives Registry page for chin RBSs]

{{size|2|'''rRNA Construction'''}}{{hide|1===Testing a new RBS-ribosome pair==
I'm going to construct a new mutant 16srRNA.  I'm going to try two of the sequences published by Chin and Rackham<cite>Rackham</cite>.  I'm picking two of the mutants that don't require me to mutate the distant bases that the authors also mutate.

===Orthogonal ribosome construction===
I'm going to make this via [['Round-the-horn site-directed mutagenesis]].  This seems like the quickest way to make the full 5 base mutation in one go.

====Primer design====
pCH1497-ASD1      5'-CacaCTTAccttaaagaagcgtactttgtagtgctcacacag attgtctgatagaaagtga-3'<br>
4rRNA-f primer    5'-TTGTGGTAccttaaagaagcgtactttgtagtgctcacacag-3'  Tm = 61.8C<br>
10rRNA-f primer   5'-TGGGATTAccttaaagaagcgtactttgtagtgctcacacag-3'  Tm = 61.9C<br>
4/10rRNA-r primer 5'-TGATCCAACCGCAGGTTCCCCTAC-3'  Tm = 62.8C<br>

====Experimental details====
[[Image:PCH1497RTH.jpg|thumb|left|100px|'''Lane1''' - 10kb ladder<br>
'''Lane 2''' - 4rRNA, phusion<br>
'''Lane 3''' - 10rRNA, phusion<br>
'''Lane 4''' - 4rRNA, pfu ultra<br>
'''Lane 5''' - 10rRNA, pfu ultra]]
[[Image:PCH1497RTH-dpnI.jpg|thumb|left|100px|Following dpnI digestion<br>
'''Lane1''' - 10kb ladder<br>
'''Lane 2''' - 4rRNA, phusion<br>
'''Lane 3''' - 10rRNA, phusion]]
[[Image:BC4&10rRNAPCR.jpg|thumb|left|100px|Colony PCR<br>'''Lane 1''' - 10kb ladder<br>
'''Lanes 2-7''' - 6 colonies of pCH1497->4rRNA<br>
'''Lane 8''' - positive control PCR on pCH1497<br>
'''Lane 9''' - positive control PCR on I7101<br>
'''Lane 10''' - 10kb ladder<br>
'''Lane 11-16''' - 6 colonies of pCH1497->10rRNA<br>
'''Lane 17''' - negative control, no template]]
[[Image:BC.PCH1497-4or10rRNA.jpg|thumb|left|100px|2nd Colony PCR, lanes same as left]]
*My plasmid is 10kB so [[Sean Moore]] suggested adding fresh polymerase in half-way through.  I didn't actually manage to do that on the first attempt.
*[[Sean Moore|Sean]] also suggested dropping the melting temperature by a few degrees and also the extension temperature.  I did 91C for melting and 69C for extension.  
*I used [[Pfu Ultra]] and [[Phusion]] as my polymerases.
*According to the gel I ran of the PCR product (I ran 10&mu;l), phusion worked(?) but pfu ultra did not.  Not sure yet why the phusion lanes are smeary.  I think I'll redo this and try to add in the fresh polymerase this time.
*Not sure why the PCR product of 10rRNA looks slightly longer than 4rRNA.  Maybe just because there is more DNA?
*I did a 10&mu;l ligation rather than the 5 that [[Sean Moore|Sean]] recommends.
*Got ~50 colonies on each plate of BL21(DE3) with a wide range of sizes(?).  I PCR'd 3 big and 3 small on each plate.  Only got the right band from the small colonies, not sure why.
*Will start cultures of 4-6 and 10-4, 10-5 and 10-6.
*Sequencing says that 4-6 worked, although it mutated an A to a G just upstream of the ASD.  I'll test this anyway to see if it works.  I should also check to see if this mutation is due to an error in my reverse primer as it would have been in this region.
*I got no sequence results for 10-6 and 10-5 seems to have had the ASD cut out.  I'll try re-prepping 10-6 and see how that sequences.
*From the second gel, start cultures of lanes 3, 6, 11 and 12.
*Colonies 10-1 and 4-2 sequenced correctly. nice.
*In addition, I'm keeping 10-5 from the first sequencing since it has the ASD deleted and 4-5 from the second sequencing as it has a slightly different ASD than expected.  
<br style="clear:both" />

[[:Image:BC-ChinASDs.apr|alignment results]]
}}

{{size|2|'''O-ribosome performance'''}}
[[Image:E7020x(Trial4)BL21EndogenousTranslation.png|thumb|400px|left|BL21(DE3) Endogenous Translation]]
[[Image:E7020x(Trial4)BL21Orthogonal Translation.png|thumb|400px|right|BL21(DE3) Orthogonal Translation]]
[[Image:E7020x(Trial4)TOP10EndogenousTranslation.png|thumb|400px|left|TOP10 Endogenous Translation]]
[[Image:E7020x(Trial4)TOP10Orthogonal Translation.png|thumb|400px|right|TOP10 Orthogonal Translation]]
[[Endy:Dedicated systems/Translation/100406]]