''Back to [[Engineering the Host/System Interface|Engineering the chassis/system interface]]''
==Introduction==
I will design and implement ways to place a certain demand on the chassis and ways to measure the cellular chassis' response to that demand.  These methods will not be specific to a given chassis but can be used to characterize a range of potential cellular chassis.

==Design test systems with specific demands==
Interactions between the chassis and an engineered system can be tested by placing clearly specified demands on the chassis.  I will design and build a suite of test systems that will allow me to independently increment some of the demands placed on the chassis.  Some of these test systems use the host cell's transcription and translation systems and some require the dedicated systems described in the previous subsection.  The individual demands of an engineered system are coupled.  For example, the number of actively transcribing RNAP determines the number of nucleotides polymerized per unit time, in other words, the transcription machinery demand determines the materials demand.  Despite this coupling of demands, if single demands can be independently placed on the chassis, the effect of demands on the chassis can be probed in a more detailed way.  

I will build the following systems to examine transcription and translation demand.  It is not yet clear that this is an adequate list of systems to successfully probe the range of demands, I will continue to examine others.  In Specific Aim 3, I will detail my proposal to test chassis response to these system demands.

===Total Transcription Demand===
I will build a family of promoters of different strengths controlling untranslated transcripts of two different lengths.  These test systems will place varying total transcription demands on the chassis.  I will use a subset of a library of promoters constructed by Moyle and coworkers.  The library is generated from 36 single-base pair mutations in the -35 and -10 hexamers of the ''ant'' promoter of P22.  LacZ fusions were used to measure the relative activity of the promoter mutants in ''S. typhimurium''.  Should this method of varying transcription demand prove unsuccessful, transcription demand could be varied by using the P<sub>BAD</sub> linear induction system of (Khlebnikov, 2002).  I believe using a family of promoters is preferable to using linear induction as it removes the need to produce a repressor or consider non-specific effects of the inducer on the chassis.  I will measure the rate of accumulation of the reporter RNA via Northern blotting.  This data specifies the number of RNAP actively transcribing the reporter RNA and hence, the total transcription demand.  To ensure that the test systems only place a transcription demand on the chassis, there must be negligible degradation of the reporter RNA, i.e. the test system must not place a significant demand on the chassis degradation systems.  For this reason, a tRNA is an ideal choice for the reporter RNA (Lopez, 1994).  Total transcription demand can be varied either by increasing promoter strength, and hence loading more RNAP onto the reporter coding region, or by making the coding region longer.  I will use both promoter strength and coding region length as independent ways to vary the total transcription demand of an engineered system.  I expect that the total transcription demand should depend only on the number of RNAP engaged in transcription, not the way the demand is generated.

===Total Translation Demand===
This can be varied with a set of test systems that produce a single protein but that have RBSs of varying strength.  Such a set of RBSs has been studied by Ringquist and coworkers.  I will determine the rate of accumulation of reporter protein via quantitative Western blotting and hence, specify the total translation demand.  An alternative to using Western blotting might be measure the numbers of ribosomes on a single mRNA via an RNA gel shift assay (Plambeck, 2003).  Similar to the transcription case above, a highly stable protein (e.g GFP) should be used to ensure that the test system does not place a significant demand on the chassis degradation system.  It will be necessary to take into account the fact that varying RBS strength will affect mRNA stability (Yarchuk, 1992).

The dedicated transcription and translation systems described above will allow me to vary the material and energy demands on the chassis independent of the machinery demand.

===[[Endy:Material Demand|Material Demand of Transcription]]===
The material demand of transcription can be varied independent of the machinery demand if a test system does not use the same transcription machinery as the chassis' own machinery.  This can be achieved by using the T7 expression system of Studier and coworkers.   I will build a family of T7 promoters of different strengths controlling an untranslated transcript.  The relative activities of all single-base mutation variants of the T7 consensus promoter sequence are known and I will use a subset of these promoters in the test systems (Imburgio, 2000).  The demand can be measured in a similar way to that outlined above to measure the total transcription demand.

===Material and Energy Demand of Translation===
Similar to above, the materials and energy demand of translation can be varied independent of the demand for translation machinery by using the dedicated translation system described in Specific Aim 1.  I will use a set of test systems that produce a single stable protein but that have RBSs of varying strength.  Such a set of RBSs compatible with the dedicated translation system does not exist.  Hence, I will need to design and build a small set of RBSs of varying strength.  Measurement of the demand these test systems place on the chassis will be similar to that described above for total translation demand.

==References==
#	Moyle, H., Waldburger, C., and Susskind, M. M. Hierarchies of base pair preferences in the p22 ant promoter. J Bacteriol 173, 6 (1991), 1944–50.
#	Khlebnikov, A., Skaug, T., and Keasling, J. D. Modulation of gene expression from the arabinose-inducible arabad promoter. J Ind Microbiol Biotechnol 29, 1 (2002), 34–7.
#	Lopez, P. J., Iost, I., and Dreyfus, M. The use of a trna as a transcriptional reporter: the t7 late promoter is extremely efficient in escherichia coli but its transcripts are poorly expressed. Nucleic Acids Res 22, 7 (1994), 1186–93.
#       Ringquist, S., Shinedling, S., Barrick, D., Green, L., Binkley, J., Stormo, G. D., and Gold, L. Translation initiation in escherichia coli: sequences within the ribosome-binding site. Mol Microbiol 6, 9 (1992), 1219–29.
#	Plambeck, C. A., Kwan, A. H. Y., Adams, D. J., Westman, B. J., van der Weyden, L., Medcalf, R. L., Morris, B. J., and Mackay, J. P. The structure of the zinc finger domain from human splicing factor znf265 fold. J Biol Chem 278, 25 (2003), 22805–11.
#	Yarchuk, O., Jacques, N., Guillerez, J., and Dreyfus, M. Interdependence of translation, transcription and mrna degradation in the lacz gene. J Mol Biol 226, 3 (1992), 581–96.
#	Imburgio, D., Rong, M., Ma, K., and McAllister, W. T. Studies of promoter recognition and start site selection by t7 rna polymerase using a comprehensive collection of promoter variants. Biochemistry 39, 34 (2000), 10419–30.