A listed gene name means that gene carries a loss of function mutation, a &Delta; preceding a gene name means the gene is deleted.  If a gene is not listed, it is not known to be mutated. Prophages present in wt K-12 strains (F, &lambda;, e14, rac) are listed only if absent.  E. coli B strains are naturally lon- and dcm-.

*'''F<sup>-</sup>''' = Does not carry the F plasmid
*'''F<sup>+</sup>''' = Carries the F plasmid.  The cell is able to mate with F<sup>-</sup> through conjugation.
*'''F'[  ]''' = Carries an F plasmid that has host chromosomal genes on it from a previous recombination event.  This cell can also mate with F<sup>-</sup> through conjugation.  Chromosomal genes carried in the F plasmid are listed in brackets.
*'''r<sub>B/K</sub><sup>+/-</sup>''' = The (B/K) defines the strain lineage.  The +/- indicates whether the strain has or hasn't got the restriction system.
*'''m<sub>B/K</sub><sup>+/-</sup>''' = The (B/K) defines the strain lineage.  The +/- indicates whether the strain has or hasn't got the modification (methylation) system. 
*'''hsdS''' = Both restriction and methylation of certain sequences is deleted from the strain.  If you transform DNA from such a strain into a wild type strain, it will be degraded.
*'''hsdR''' = For efficient transformation of cloned unmethylated DNA from PCR amplifications
*'''INV( )''' = chromosomal inversion between locations indicated
*'''ahpC''' = mutation to alkyl hydroperoxide reductase conferring disulfide reductase activity
*'''ara-14''' = cannot metabolize arabinose
*'''araD''' = mutation in L-ribulose-phosphate 4-epimerase blocks arabinose metabolism
*'''cycA''' = mutation in alanine transporter; cannot use alanine as a carbon source
*'''dapD''' = mutation in succinyl diaminopimelate aminotransferase leads to succinate or (lysine + methionine) requirement
*'''&Delta;( )''' = chromosomal deletion of genes between the listed genes (may include unlisted genes!)
*'''dam''' = adenine methylation at GATC sequences abolished; high recombination efficiency; DNA repair turned on
*'''dcm''' = cytosine methylation at second C of CCWGG sites abolished
*'''deoR''' = regulatory gene that allows constitutive expression of deoxyribose synthesis genes; permits uptake of large plasmids.  See Hanahan D, [http://patft.uspto.gov/netacgi/nph-Parser?u=/netahtml/srchnum.htm&Sect1=PTO1&Sect2=HITOFF&p=1&r=1&l=50&f=G&d=PALL&s1=4851348.WKU.&OS=PN/4851348&RS=PN/4851348 US Patent 4,851,348].
*'''dnaJ''' = one of the chaparonins inactivated; stabilizes some mutant proteins
*'''dut1''' = dUTPase activity abolished, leading to increased dUTP concentrations, allowing uracil instead of thymine incorporation in DNA.  Stable U incorporation requires ung gene mutation as well.
*'''endA1''' = For cleaner preparations of DNA and better results in downstream applications due to the elimination of non-specific digestion by Endonuclease I
*'''(e14)''' = excisable prophage like element containing mcrA gene; present in K-12 but missing in many other strains
*'''galE''' = mutations are associated with high competence, increased resistance to phage P1 infection, and 2-deoxygalactose resistance.  galE mutations block the production of UDP-galactose, resulting in truncation of LPS glycans to the minimal, "inner core".  The exceptional competence of DH10B/TOP10 is thought to be a result of a reduced interference from LPS in the binding and/or uptake of transforming DNA.  galE15 is a point mutation resulting in a Ser123 -> Phe conversion near the enzyme's active site.  See van Die, et al. PMID 6373734, Hanahan, et al. PMID 1943786, and EcoSal ISBN 1555811647.  --[[User:Dcekiert|Dcekiert]] 16:56, 23 January 2008 (CST)
*'''galK''' = mutants cannot metabolize galactose and are resistant to 2-deoxygalactose.  galK16 is an IS2 insertion ~170bp downstream of the galK start codon.  See EcoSal ISBN 1555811647.  --[[User:Dcekiert|Dcekiert]] 16:56, 23 January 2008 (CST)
*'''galU''' = mutants cannot metabolize galactose
*'''gor''' = mutation in glutathione reductase; enhances disulphide bond formation
*'''glnV'''  = suppression of amber (UAG) stop codons by insertion of glutamine; required for some phage growth
*'''gyrA96''' = mutation in DNA gyrase; conveys nalidixic acid resistance
*'''gyrA462''' = mutation in DNA gyrase; conveys resistance to ccdB colicin gene product
*'''hflA150''' = protease mutation stabilizing phage cII protein;  high frequency of lysogenization by &lambda;
*'''&Delta;(lac)X74''' = Deletion of the entire ''lac'' operon as well as some flanking DNA.
*'''lacI<sup>q</sup>''' or '''lacI<sup>Q</sup>''' = overproduction of the lac repressor protein; -35 site in promoter upstream of ''lacI'' is mutated from GCGCAA to GTGCAA
*'''lacI<sup>Q1</sup>'''  = overproduction of the lac repressor protein; contains a 15 bp deletion to create optimal -35 site in promoter upstream of ''lacI''
*'''lacY''' = deficient in lactose transport; deletion of lactose permease (M protein)
*'''lacZ&Delta;M15''' = partial deletion of the lacZ gene that allows &alpha; complementation of the &beta;-galactosidase gene; required for blue/white selection on XGal plates.  Deletes the amino portion of lacZ (aa 11-41).
*'''leuB''' = requires leucine
*'''&Delta;lon''' =  deletion of the lon protease
*'''malA''' = cannot metabolize maltose
*'''mcrA''' = Mutation eliminating restriction of DNA methylated at the sequence C<sup>m</sup>CGG (possibly <sup>m</sup>CG).  Carried on the e14 prophage (q.v.)
*'''mcrB''' = Mutation eliminating restriction of DNA methylated at the sequence R<sup>m</sup>C
*'''metB''' = requires methionine
*'''metC''' = requires methionine
*'''mrr''' = Mutation eliminating restriction of DNA methylated at the sequence C<sup>m</sup>AG or G<sup>m</sup>AC
*'''mtlA''' = cannot metabilize mannitol
*'''(Mu)''' = Mu prophage present.  Mu&delta; means the phage is defective.
*'''mutS''' - mutation inhibits DNA repair of mismatches in unmethylated newly synthesized strands
*'''nupG''' = same as deoR
*'''ompT''' = mutation in outer membrane protein protease VII, reducing proteolysis of expressed proteins
*'''(P1)''' = Cell carries a P1 prophage.  Cells express the P1 restriction system.
*'''(P2)''' = Cell carries a P2 prophage.  Allows selection against Red+ Gam+ &lambda;
*'''(&phi;80)''' = Cell carries the lambdoid prophage &phi;80.  A defective version of this phage carrying lacZM15 deletion (as well as lacIq, lacYA, and flanking sequences) is present in some strains.  The &phi;80 attachment site is just adjacent to tonB.
*'''pLysS''' = contains pLysS plasmid carrying chloramphenicol resistance and phage T7 lysozyme, effective at attenuating activity of T7 RNA polymerase, for better inhibition of expression under non-induced conditions.  The sequence can be found [http://www.emdbiosciences.com/docs/NDIS/69659-000.HTML here].
*'''proA/B''' = requires proline
*'''recA1''' = For reduced occurrence of unwanted recombination in cloned DNA; cells UV sensitive, deficient in DNA repair
*'''recA13''' = as for recA1, but inserts less stable.
*'''recBCD''' = Exonuclease V; mutation in RecB or RecC reduces general recombination by a factor of 100; impaired DNA repair; UV sensitive, easier propagation of inverted repeats
*'''recJ''' Exonuclease involved in alternate recombination
*'''relA''' = relaxed phenotype; permits RNA synthesis in absence of protein synthesis
*'''rha''' = blocked rhamose metabolism
*'''rnc''' = encodes RnaseIII (rnc-14 is a common null mutant)
*'''rne''' = encodes RnaseE (rne-3071 is a common temperature sensitive mutant)
*'''rpsL''' =  mutation in ribosomal protein S12 conveying streptomycin resistance; also called strA
*'''sbcBC''' = ExoI activity abolished; usually present in recBC strains; recombination proficient, stable inverted repeats
*'''srl''' = cannot metabolize sorbitol
*'''supE''' = glnV
*'''supF''' = tyrT
*'''thi''' = requires thiamine
*'''thyA''' = requires thymidine
*'''Tn10''' = transposon normally carrying Tetracycline resistance
*'''Tn5''' = transposon normally carrying Kanamycin resistance
*'''tonA''' = Mutation in outer membrane protein conveying resistance to phage T1 and phage T5
*'''traD''' = Mutation eliminating transfer factor; prevents transfer of F plasmid
*'''trxB''' = mutation in thioredoxin reductase; enhances disulphide bond formation in the cytoplasm
*'''tsx''' = outer membrane protein mutation conveying resistance to phage T6 and colicin K
*'''tyrT''' = suppression of amber (UAG) stop codons by insertion of tyrosine; needed for some phage infection such as &lambda;gt11.
*'''ung1''' = allows uracil to exist in plasmid DNA
*'''xyl-5''' = blocked xylose metabolism

*'''Sm<sup>R</sup>''' = Streptomycin resistance

[[Category:Escherichia coli]]