[[User:Federico_Castro_M|Return to home page]]

<small>Last updated January/08/2008</small>

__TOC__

THIS PAGE IS UNDER CONSTRUCTION 
Project status: <font color="green">'''Active'''</font>

Objective: To design and assemble a devise that would allow us to map the adaptive landscape resulting from artificial genes that confer antibiotic resistance.

== The theory==

[[Image:FEDII.JPG|right|frame|An adaptive landscape]]
In the adaptive landscape, of S.Wright, the fitness associated with different genotypes as well as the relation between two genotypes is visualized in a map. Different evolutionary forces such as endogamy, genetic drift, horizontal transfer and natural selection move the population across the adaptive landscape.[http://www.blackwellpublishing.com/ridley/classictexts/wright.pdf] An analogy to the movement of populations across adaptive landscape could be a ball falling into a cone by the action of gravity (although natural selection usually moves populations uphill).

The antibiotic resistance acquired by bacteria and drug resistance developed by virus can be seen as a movement of the population to peak of fitness in the adaptive landscape. 
The speed and direction of a population that moves across the adaptive landscape, are given by the topology of the landscape and the evolutionary forces that act upon the population. Interestingly, this is supported by the observation of twins infected with the same strain of VIH; having the same genotype they had the same adaptive landscape and developed similar immunological responses to the virus[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2118231].
The forthcoming of personal DNA sequencing stresses the need to understand how the genotype of one individual shapes the adaptive landscape for a pathogen.

Given the medical importance of the process that lead to drug resistance, some strategies have been devise to prevent it:
*High doses of drugs are administered (Ex. antibiotics) to reduce a population and hence reduce the number of new individuals with a mutation that are produced.
*Many different drugs are administered at the same time as cocktails (Such as the HAART that is used for AIDS treatment)so that the population has to move a very large distance in the adaptive landscape in order to reach a fitness peak.
*Drugs are designed in such a way that their action is not inhibit by a single mutation, they have a less specific active area thereby widening the valley the populations have to cross.
 
== The design==
I aim to construct a system in which the fitness associated with different genotypes as well as the relation between the different genotypes and some of the different evolutionary forces are known and then to predict the movement of the population across the adaptive landscape. 

[[Image:FedIII.JPG|right|frame|The first devise]]

In order to achieve such an ambitious goal I intend to construct a devise with two different regions susceptible to the action of invertases. An inversion in the system would allow the expression of a reporter and will confer the bacteria with resistance to certain antibiotic. 

I intend to culture all the bacteria with nonlethal concentration of Kanamycin and Tetracycline and to estimate the fitness by means of colony counting.
Having only two region susceptible to inversions the system will represent two different genes each one with two alleles. The topography resulting from the mutations will be fairly simple being that there are no intermediate mutations between the two alleles.

The Promoters are unspecified and I just require them to be constitutive. The reporters are unspecified since most reporters will do the job, I will choose the ones that will allow to assemble the construction easy with few ligation steps.

Some proteins are inverted to highlight that they will only be expresed with the inversion.

==Further Work==
*Note:We should also build a RCT protein generator of Kan. Aparently the guys at the registry forgot to make one, the protein generator might also be used by the horizontal transfer team and it could earn us some points. Also a Hin protein generator is missing... so lets add it to the list.