PHYLOGENETIC ANALAYSIS OF CYTOCHROME P450 STRUCTURES
Gowri Shankar B.A1, Michael Charleston1, Michael Murray2 and David Hibbs2
1 School of Information Technology, University of Sydney, NSW
2Faculty of Pharmacy, University of Sydney, NSW

Abstract

Cytochrome P450 (CYPs) play an important role in metabolizing a variety of exogenous and endogenous compounds. There are 18 different families of CYPs, and each family has different properties and characteristics. The individual CYPs have their unique substrate specificity with little structural variations, even across families. CYPs are ubiquitous for example, they present in distant species like archaea and also in mammals. This research is aimed at resolving the evolutionary dynamics of CYP Protein Structures using phylogenetic and standard protein structural comparison methods.
Fourteen CYP proteins from different species were analysed. These proteins are from human, rabbit, bacteria, pseudomonas, fungi and archaea. All structures were compared structurally using FATCAT (Flexible structure Alignment by Chaining Aligned fragment pairs allowing Twists). Phylogenetic analysis of these proteins were performed using PAUP* and Phylip. The evolutionary distances are calculated from with Phylip using Maximum Likelihood with a molecular clock model of amino acid change. 
From CLUSTAL X alignment we observed that sites from 308 – 330 and from 450- 470 are conserved. These conserved residues are mostly hydrophobic residues like leucine, isoleucine, valine and alanine. The 441 glycine and 450 cystine are perfectly conserved among all sequences. The structures are also compared using RMSD. The RMSD of these species are much less, from 0-3.00 Å, whereas between mammalian and archaea it is as much as 6.00 Å. The evolutionary distances, measured as the expected number of changes among amino acids at a given locus, per unit time, go up to 4.5 for archaea and mammals but are very low between mammals and bacteria. The conserved similarities in the amino acids between CYPs were found only in regions like substrate binding regions and functionally active sites. Some of the hydrophobic residues are conserved and are found in the helices.