Major new collaborations are being established and are summarised below (website still in constuction).  In the mean time you can learn about our current work on the [[Grierson Lab:Back Door|Lab Members]] and [[Grierson Lab:Lab Members|Research]] pages.


== '''Summary''' == 
We are pioneering an integrative, predictive biology approach to describing the mechanisms of differentiation, morphogenesis and directed elongation of an individual cell.  Root hair (RH) cells of [http://www.bioone.org/perlserv/?request=get-document&issn=1543-8120&volume=41&issue=1&page=1  Arabidopsis] are an ideal model for this because their development is exceptionally well understood and they are relatively easy to study experimentally. This project brings together a range of life and theoretical scientists to piece together our current knowledge and use novel experiments, mathematical models, and biocomputation to begin to understand the system as a whole.  This work will contribute to the global research effort on a “virtual” plant by linking to the virtual root model being developed at the Centre for Plant Integrative Biology (CPIB) at Nottingham [http://www.cpib.info/ (Nottingham CPIB homepage)]. In addition the project will develop insights and approaches relevant to a wide range of biological systems, and boost our understanding of complex systems in general.

Root hair development presents a unique set of challenges that are beyond the scope of the CPIB, and are outlined here. The research builds on skills already at Bristol: a world-leading root hair biology group [[Grierson_Lab|(Grierson Lab frontpage)]], expert mathematical modelling in dynamical systems [http://www.enm.bris.ac.uk/anm/staff/arc.html (Prof. Champneys, University of Bristol][http://www.bio.bris.ac.uk/people/staff.cfm?key=1046 Dr Payne, University of Bristol)], mechanics [http://www.maths.bris.ac.uk/~maivc/ (Dr Chenchiah, University of Bristol)], bioinformatics [http://supfam.org/gough/ (Dr Gough, Institut Pasteur Paris)], and computational [http://www.cs.bristol.ac.uk/People/personal.jsp?person=16910 (Prof. Flach, University of Bristol)], and statistical [http://www.maths.bris.ac.uk/~mapjg/ (Prof. Green FRS, University of Bristol)] modelling plus leading edge techniques in light [http://www.bris.ac.uk/biochemistry/mrccif/index.html (cell imaging facilities, University of Bristol)] and atomic force [http://www.phy.bris.ac.uk/people/miles_m/index.html (Prof. Miles, University of Bristol)] microscopy and image analysis [http://www.cs.bristol.ac.uk/People/personal.jsp?person=16601 (Dr Mirmehdi, University of Bristol)]. Other UK expertise in modelling and experimentation at Sheffield [http://www.dcs.shef.ac.uk/~nmonk/ (Dr Monk, University of Sheffield)], Norwich [http://www.jic.bbsrc.ac.uk/science/cdb/dolanWebpage.htm (Prof Dolan, John Innes Centre)] and Nottingham [http://www.cpib.info/ (Nottingham CPIB homepage)].


== '''Rationale''' ==
Root hairs are agronomically important. They make up the majority of the root surface area of many crops, where they play an essential role in taking up nutrients and water from the soil, in interacting with pathogens and symbionts, and in anchorage. Research into optimisation of root hair properties is vital, since current agricultural usage levels of fertiliser and fresh water are not sustainable. Historically root hair research has been multidisciplinary and has involved developmental, genetic and cellular approaches to investigate the network controlling formation of the cell. More is known about the mechanism underpinning root hair cell development than any other plant cell type. Hair cells are an exemplary experimental system: they develop in a predictable spatial pattern (Fig.1), allowing cells to be imaged throughout their development, and they develop cylindrical “hairs” that grow away from the surface of the plant into surrounding medium, and which are transparent thus facilitating quality imaging. Genetic knowledge of root hairs is excellent and there are many viable mutants and transgenic lines available, along with other outstanding international resources. These mutants are often characterised by specific aberrant morphologies, and the ability to explain these mutant forms will be a key bench-mark of our work. Root hairs are also an outstanding system for generic modelling of plant cell development differentiation and growth, posing a series of profound biological questions that are relevant to many other biological systems.
[[Image:Objectives figure.jpg|frame|Figure 1. Schematic diagram showing sequence of RH development, with numbers to indicate the relevant research objectives described in the text. Commitment to form a RH is made very late, as the cell stops elongating. A patch of cell wall about 20 microns high and 10 microns across loosens and bulges out. Specialised machinery for tip growth assembles in the bulge and the RH starts to grow.  Tip growth stops once the RH reaches its final length (0.7 – 1.0 mm).]] 



== '''Objectives''' ==
We will focus on six objectives.


=== '''Objective 1''' ===
Understanding which epidermal cells become root hair cells. Epidermal cells differentiate into RH and non-RH cells in a predictable pattern (Fig. 2), which has yet to be understood in a context that permits an understanding of root auxin flow and RH outgrowth. Monk & Dolan have developed an ordinary differential equation model for the gene expression and cell-cell movement of transcription factors that control epidermal cell differentiation. We will add a longitudinal dimension to these models to address how the assembly of the transcription factor gene network might interact with the transport of long-range regulatory molecules, like auxin. This novel multi-level modelling approach is needed because key regulators such as ethylene and auxin are known to modify, and sometimes even to prevent manifestation of the RH and non-RH phenotypes.[[Image:Root_hair_cross_section.png|frame|Figure 2. Cross-section of an Arabidopsis root tip showing the two types of cell in the epidermis. RH cells (yellow) overly the junction between two cortical cells in the layer below, whereas non-RH cells (blue) overlie a single cortical cell. Specific transcription factors establish this pattern by moving between the cell types and regulating each other’s activity.]]


=== '''Objective 2''' ===
The position on the root where RHs form is influenced, inter alia, by auxin-responsive transcription factors. New results from the Grierson lab show RH cells express very different levels of certain auxin transporters from non-hair cells. Considered in the light of Kramer’s [http://www.simons-rock.edu/~ekramer/] model of auxin flow, these new data suggest that RH cells should contain different levels of auxin from non-RH cells. We are testing this prediction by measuring the auxin content of RH and non-RH cells in collaboration with  Ljung [http://static.upsc.se/kljung.htm]. We are also ascertaining the contributions that auxin and ethylene make to RH development using a combination of hormone treatments, fluorescent imaging, genetic manipulation, and measurements of auxin content and response. In parallel we will build a mathematical model of hair morphogenesis using modified reaction-diffusion equations, and test whether the robustness of the morphogenesis is explained by a Turing-like process. Preliminary results from Dolan’s lab indicate that the ability of auxin and ethylene to trigger RH formation depends on transcription factors whose role will be examined using transgenic reporters, high throughput microarray data, suppressor mutants, and hormone treatments. Bioinformatics tools could identify similarities between the promoter sequences of putative targets and suppressors, look for likely transcription factor binding sites, and identify possible mechanistic links with auxin and ethylene.


=== '''Objective 3''' ===
Root hair growth begins with an ellipse of bulging and thinning of the cell wall end nearest the root tip. Previous work has identified an number of gene products which play key roles in this process including RHD6 [http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=159671&blobtype=pdf], RHD1 UDP-glucose 4-epimerase [http://www.plantphysiol.org/cgi/reprint/140/4/1406][http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6VRT-475B5XM-T&_coverDate=10%2F29%2F2002&_alid=494902988&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=6243&_sort=d&view=c&_acct=C000010018&_version=1&_urlVersion=0&_userid=121739&md5=f0e91e686bbcd39efae3315cdbbcd0dd][http://www.jbc.org/cgi/content/full/281/25/17276], TIP1 S-acyl transferase [http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=1197434&blobtype=pdf], and ROP small GTPases [http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=150681&blobtype=pdf][http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=150680&blobtype=pdf]  [http://www.pubmedcentral.nih.gov/picrender.fcgi?artid=111158&blobtype=pdf] and their negative regulator ROPGDI1 [http://www.nature.com/nature/journal/v438/n7070/abs/nature04198.html]. In order to investigate the mechanisms by which these factors might operate, we will build mathematical models to describe the main factors which affect cell wall properties: inter- and intra-cellular auxin flow, the shape of the bulge, and the intra-cellular structures and networks. This will require close interaction with the results of models in Objectives 2 and 4. In tandem we will investigate changes in cell wall properties using AFM microscopy of biological tissue and novel approaches that are being developed at the CPIB. This work will contribute useful data to our integrative approach of experimentation, imaging, and development of biomechanical models of the cell wall.


=== '''Objective 4''' ===
Understanding how root hair elongation and shape are controlled..  Any predictive model of RH elongation must take into account various experimentally measured features, e.g., relative time scales associated with initiation, growth and chemical (including auxin) transport within the root hair; clarify the behaviour of at least some genetic mutants, e.g., those with branched, wavy or corkscrew hairs; and explain different morphological forms associated with changes in the internal or external environments.
Our modelling will form two complementary strands: Strand (a): Molecular machines involved in tip growth. We will investigate transcriptional control of RH growth using data generated in Objectives 1 & 2, and recent data (Dolan, unpublished) on transcription factors that maintain RH elongation. This will be combined with novel graph-theory representation and analysis of the network of genes and gene products that contribute to the molecular machines for growth. The networks will be built using microarray data, bioinformatics, and high throughput phenotyping methods to be developed collaboratively by Mirmehdi, Pridmore and our industrial collaborator. We will get insights into the way that the growth process develops, and the order in which the RH growth machinery forms in the cell. Up to a third of the tip growth network consists of genes of hitherto unknown function (Breen and Grierson, unpublished). Hidden Markov modelling will enable us to predict functions for many of these genes by identifying their association in the network with genes of known function (Fig. 3). Our results will help to launch work on the evolution, structure, function, and organisation of tip growth networks (including those in fungal hyphae, pollen tubes, and nerve cells). The models will also incorporate methods for representation of spatial information about the subcellular localisation of components, essential for understanding development and growth, but not commonly included in cell network models.([[Identifying genes and molecular machines that drive root hair tip growth|Breen]] and Grierson, unpublished)
[http://www.nature.com/nature/journal/v436/n7052/full/nature03876.html See relevant work by Gunsalus et al.] 

=== '''Objective 5''' ===
Understanding how cessation of root hair growth is controlled.  Mechanisms of growth cessation are tightly coordinated so that the end of the mature RH is a tidy dome-shape. Several mutants and treatments disrupt this process and produce tips with distortions such as bulbous, pointed, or branched ends. Two sets of questions arise. Firstly, do local changes in auxin transport or levels affect this process? We will address this by collecting image data on the locations of auxin transporters in the region of the root where growth ceases and including the results in our models of auxin flow. Secondly, how is the activity of the tip growth machinery stopped, and how is this coordinated? Our phenotyping of mutants in the genes of the tip growth network will include assays for phenotypes affecting the end of growth. Ultimately these conditions will be fed into the models from Objective 4 to seek a coherent explanation for RH growth cessation.  


=== '''Objective 6''' ===
The virtual root hair.
Our modelling approaches will be designed to be integrated into the CPIB Virtual Root and hence the international Virtual Plant. Models arising from Objectives 1-5 will be feed into a unified software environment compatible with the CPIB project. This will enable consideration of the RH cell within both the root and the plant as a whole, and virtual experiments will help prioritise future theory work and experiments.


'''The following have provisionally agreed to sit on an International Steering Committee:''' [http://www.biology.duke.edu/benfeylab/funding_nsf0606873.htm Philip Benfey (Root development, Duke University)], [http://www.deas.harvard.edu/softmat/ Lalith Mahadevan (Biomechanics, Harvard)], [http://www.pcb.wur.nl/UK/people/Faculty/Anne+Mie+Emons/?WBCMODE=PresentationUnpublished Anne-Mie Emons (Root hair cytoskeleton, Wageningen)], [http://www.missouri.edu/~physwww/people/GaborForgacs.html Gabor Forgacs (Biological viscoelasticity, Missouri)], [http://www.ma.ic.ac.uk/~mjhowa/ Martin Howard (Intracellular dynamics, Imperial)], [http://www.maths.ox.ac.uk/~maini/ Philip Maini (Mathematical Biology, Oxford)], [http://www.lsa.umich.edu/mcdb1/faculty/schiefel/lab/index.html John Schiefelbein (Root hair development, Michigan)], [http://www.personal.psu.edu/faculty/j/p/jpl4/ Jonathan Lynch (Root hairs and plant nutrition, Penn State).

[[Grierson_Lab|Grierson Lab frontpage]]