[[Image:LylemutS.jpg]]

DNA replication, chromosome partitioning and condensation: Prokaryotes are not known to have a eukaryotic-like mitotic apparatus, and little is known about the mechanisms controlling chromosome partitioning. Our studies on the bacterial DNA polymerase and DNA replication have led to a new model for chromosome partitioning in prokaryotes. We visualized DNA polymerase in living cells of B. subtilis using a fusion of the catalytic subunit (PolC) to Green Fluorescent Protein (GFP). We found that PolC-GFP is at discrete intracellular positions, predominantly at or near mid-cell, and is not distributed randomly on the DNA which largely fills the cell. Our results indicate that the replisome is stationary, like a factory, and that during replication, the DNA template moves through the polymerase. Recent experiments visualizing a site on the DNA confirm that the DNA moves through the centrally positioned replisome. 

Newly duplicated regions of the bacterial chromosome separate from each other while the remainder of the chromosome is replicated, that is, before completion of DNA synthesis. We suspect that the cell harnesses the energy from the replication machine to drive separation of the newly duplicated regions. The centrally positioned replisome would redirect the sister regions, one towards each end of the cell. Current work focuses on testing this model and on understanding how the replisome position is established and maintained. 

Clearly there are components other than the DNA polymerase that are involved in chromosome partitioning. We have identified a DNA site involved in chromosome partitioning in B. subtilis. This site was identified as the binding site for the chromosome partitioning protein Spo0J (ParB). Spo0J is a site-specific DNA binding protein that recognizes a 16 bp sequence found in spo0J. Allowing two mismatches, this sequence occurs 10 times in the entire B. subtilis chromosome, all in the origin-proximal ~20%. Eight of the 10 sequences are bound to Spo0J in vivo. The presence of a site on an otherwise unstable plasmid stabilizes the plasmid in a Spo0J-dependent manner, demonstrating that this site, called parS, can function as a partitioning site. This site and Spo0J are conserved in a wide range of bacterial species. 

The B. subtilis Smc protein, like its eukaryotic counterparts, plays an important role in chromosome structure and partitioning. A null mutation in B. subtilis smc causes a Ts-lethal phenotype in rich medium. Under permissive conditions, the mutant has abnormal nucleoids and ~10% of the cells are anucleate. In combination with a null mutation in spo0J, the smc mutation caused a synthetic phenotype; cell growth was slower and ~25% of the cells were anucleate. 


[[Image:Alex-HeatMap.jpg]]

DNA arrays and control of gene expression: We are using DNA arrays to monitor expression of almost all B. subtilis genes at once. PCR products of each identified open reading frame are spotted on glass slides and used to monitor amounts of RNA corresponding to each gene. We are using these arrays to monitor gene expression under a variety of different growth conditions and in a variety of different regulatory mutants. The combination of this powerful technology with more classical genetic and molecular analyses should provide new insights into the complex regulatory networks controlling growth and development. 

[[Grossman Lab| Back to the Grossman Lab Webpage]]