==Goals==
*Look around for what can be done with an rs# besides OMIM
*Talk to Chris about the dbSNP and OMIM DOA 

==Progress==
*Bounced some ideas with Chris, you can see some of our thoughts [[Harvard:Biophysics_101/2007/Notebook:Christopher_Nabel/2007-4-24|here]]

===Random Findings===
*The ALelle FREquency Database [http://alfred.med.yale.edu/ ALFRED], as the name implies, gives allele frequencies!
**It can take dbSNP rs#'s, so this can be pretty useful in looking at frequency, penetrance, ...
**Unfortunately, their database is quite limited.  Our little macular degeneration friend (rs11200638) has not yet made it into the db.
===Non-OMIM PubMed===
*Why didn't we bother with going straight to PubMed from dbSNP?
**I see the value in OMIM, but might OMIM occasionally lack something that PubMed has?
**Maybe this is only worthwhile when the rs# doesn't show up in OMIM
**The search is pretty easy, just modified the code I had worked on with Resmi to search the OMIM results to now just search the rs #
*Script
<pre>
from Bio import PubMed
from Bio import Medline

article_ids = PubMed.search_for("rs11200638")

rec_parser = Medline.RecordParser()
medline_dict = PubMed.Dictionary(parser = rec_parser)

count = 1
for did in article_ids[0:5]:
    cur_record = medline_dict[did]
    print '\n', count, ')  ', string.rstrip(cur_record.title), cur_record.authors, string.strip(cur_record.source)
    count=count+1
</pre>
*Output
**The output doesn't quite give the top articles used by OMIM, and I'm supposing review articles don't reference rs#'s, but this doesn't look too bad as a non-OMIM option
**Output:
<pre>
1 )   HTRA1 promoter polymorphism predisposes Japanese to age-related macular
degeneration. ['Yoshida T', 'DeWan A', 'Zhang H', 'Sakamoto R', 'Okamoto H', 'Minami M', 'Obazawa M', 'Mizota A', 'Tanaka M', 'Saito Y', 'Takagi I', 'Hoh J', 'Iwata T'] Mol Vis. 2007 Apr 4;13:545-8.

2 )   HTRA1 Variant Confers Similar Risks to Geographic Atrophy and Neovascular
Age-related Macular Degeneration. ['Cameron DJ', 'Yang Z', 'Gibbs D', 'Chen H', 'Kaminoh Y', 'Jorgensen A', 'Zeng J', 'Luo L', 'Brinton E', 'Brinton G', 'Brand JM', 'Bernstein PS', 'Zabriskie NA', 'Tang S', 'Constantine R', 'Tong Z', 'Zhang K'] Cell Cycle. 2007 May 16;6(9).

3 )   A variant of the HTRA1 gene increases susceptibility to age-related
macular degeneration. ['Yang Z', 'Camp NJ', 'Sun H', 'Tong Z', 'Gibbs D', 'Cameron DJ', 'Chen H', 'Zhao Y', 'Pearson E', 'Li X', 'Chien J', 'Dewan A', 'Harmon J', 'Bernstein PS', 'Shridhar V', 'Zabriskie NA', 'Hoh J', 'Howes K', 'Zhang K'] Science. 2006 Nov 10;314(5801):992-3. Epub 2006 Oct 19.
</pre>

===Other Databases===
'''Human Variome Project'''
*Deniz had talked about this in class as an option for non dbSNP, non OMIM
*Seems like they look at alot of privately funded databases, such as 
**Human Genome Variation Database [http://hgvbase.cgb.ki.se/cgi-bin/main.pl?page=index_new1.htm HGVbase] 
***... umm, are they doing what we're doing?
***"The primary purpose of HGVbase is to facilitate genotype-phenotype association analyses that explore how single nucleotide polymorphisms (SNPs) and other common sequence variations may influence phenotypes such as common disease risk and drug response differences."
***They're not accepting submissions currently, but if you check out the databases they use [http://hgvbase.cgb.ki.se/cgi-bin/main.pl?page=databases_.htm here], it looks a little worrisome