'''GENERAL BACKGROUND'''

Fanconi anemia (FA) belongs to the group of caretaker gene diseases including Ataxia telangiectasia (AT), Nijmegen breakage syndrome (NBS), Bloom syndrome (BLM) and hereditary breast cancer. Mutations in these caretaker genes result in genomic instability and cancer predisposition (1). Fanconi anemia patients have a high incidence of neoplasia and chromosomal aberrations as well as skeletal, organ and developmental abnormalities. FA has a worldwide incidence of 1-5 per million, with a heterozygote carrier frequency of 0.3-1.0%. The average life expectancy of FA patients is less than 20 years, with bone marrow failure and fatal malignancies (acute myeloid leukemia, AML; or solid tumors) as the major course of death (1, Niedernhofer, 2005 #4779). 

The pleiotrophy of congenital defects observed in FA patients, ranging from skeletal abnormalities, growth retardation, ear malformation and hearing loss to hypogenitalia, kidney abnormalities and cardiac and cardiopulmonary involvement, suggests global developmental derangements (2). Despite being genetically heterogeneous (at least twelve different FA genes), the cellular FA phenotype appears to be quite homogenous. Cells of FA patients show hypersensitivity towards alkylating agents like Mitomycin C (MMC) or diepoxybutane (DEB) that cause DNA interstrand crosslinks. Another hallmark of FA cells is the highly elevated spontaneous chromosomal breakage rate resulting in typical aberrations, e.g. quadriradial figures. A third typical feature of FA cells is a delay of the G2 phase and accumulation in the G2/M compartment of the cell cycle (reviewed in (1).

A role for FA proteins in the DNA damage response during replication has recently emerged. We and others showed that FA proteins are involved in (I) repair of replication-associated DNA double strand breaks (3-8), and (II) control of the S-phase checkpoint of the cell cycle (9-12) (6, 7, 13, 14). The identification of the breast cancer susceptibility protein 2 (BRCA2) as a Fanconi protein (FANCD1)(15) and the interaction of BRCA1 and BRCA2, with FANCD2 (16-19), indicate a close interconnection of the FA proteins with the BRCA1/BRCA2 caretaker pathways that have a central role in maintaining genomic stability. Moreover, the identification of two new FA proteins, FANCM and FANCJ, with defined DNA helicase domains (20-23), strongly favors a model where the FA pathway acts at structure-specific sites of damaged DNA that arise during replication. 


'''SPECIFIC BACKGROUND'''

'''''(I) Involvement of FA proteins in DNA repair'''''

The FA proteins appear to be positioned at the crossroads between cell cycle checkpoints and DNA repair. DNA interstrand crosslinks, the major genotoxic event for FA cells, are repaired by the concerted action of at least two repair pathways, nucleotide excision repair and DNA double strand break repair via homologous recombination (HR) (24, 25). Several findings support a role for FA proteins in the latter repair pathway. FA cells are defective in the removal of DNA interstrand crosslink-induced DNA double strand breaks that are specifically generated during S-phase (3). In response to DNA damage, the FA core complex consisting of at least eight proteins (FANCA, -B, -C, -E, -F, -G, -L, and –M) is required for activation (monoubiquitination) of the downstream FANCD2 protein (1, 16, 26-29). Activated FANCD2 interacts with BRCA1 (major breast cancer associated protein 1) and Rad51, major components of HR repair (16, 17). FANCD2 was also shown to directly interact with the FANCD1/BCRA2 protein (15, 19), which is involved in HR, moreover, cells defective in BRCA2 show a decrease in HR (30, 31). In addition, other FA proteins have been shown to overlap with the HR pathway (32-34).
On the other hand, several reports suggest involvement of the FA proteins in different DNA break repair pathways such as single-strand annealing (SSA) (35), or non-homologous end-joining (NHEJ) (36-38).
One of the most exciting findings was the recent identification of two additional FA proteins, FANCM and FANCJ. FANCM, an FA core complex member (20, 39, 40), has strong homology to the archeal protein Hef (helicase-associated endonuclease for fork-structured DNA), which resolves stalled replication forks (41). So far, the human FANCM protein was shown to possess in vitro DNA translocase activity (20). FANCJ, a downstream protein in the FA pathway, was shown to be identical to BACH1/BRIP1 (21, 22, 42), a 5'-to-3' DNA helicase that directly interacts with BRCA1 (43). The perspective that the FA pathway contains two DNA helicases (with possibly different DNA structure specificities) is in strong favor of a direct, sequential involvement of the FA proteins in repair/processing of DNA lesions. 

'''''(II) Function of FA proteins in replication and the replication checkpoint'''''

Following induction of interstrand crosslinks, FA cells show a defect in arrest at the early S-phase checkpoint, accompanied by damage-resistant DNA synthesis (DRDS) (13, 44). The later accumulation of FA cells in the G2 phase of the cell cycle is due to an arrest in late S-phase, possibly caused by a deficiency in the resolution of interstrand crosslinks (14). In agreement with this hypothesis, Rothfuss et al. showed that interstrand crosslink-induced formation of DNA double strand breaks is a replication-associated event and that removal of such breaks is defect/delayed in FA cells (3). Our laboratory demonstrated recently that the FA proteins are recruited to chromatin in a strictly replication-dependent manner, even in the presence of exogenous DNA damage. In addition, we showed that absence of a functional FA pathway results in accumulation of chromosomal breaks during unperturbed replicative DNA synthesis (6). These results support a model where the FA proteins are recruited to sites of DNA damage that are generated when the moving replication fork stalls at certain DNA lesions. 

''''''(III) Using the Xenopus model to explore caretaker functions of FA proteins'''''

For studying the FA proteins, the Xenopus cell free-system offers significant advantages over other systems. For example, Xenopus egg extracts are naturally synchronized in S-phase and can independently remodel a DNA template into a functional nucleus, followed by semi-conservative replication of the nuclear DNA (for an overview, see Menut et al., 1999). Using this unique system, we recently showed that the FA proteins have DNA repair functions during the replication process (see also above) (6). Besides other advantages, the Xenopus cell-free system is uniquely suitable for studying the tightly interconnected FA pathway proteins because it is the only in vitro system that allows us to analyze the behavior of any one FA protein in a functionally relevant environment, i.e. in the presence of the entire, fully-functional FA pathway ensemble.  


'''LITERATURE'''

1.	Joenje, H. & Patel, K. J. (2001) Nat Rev Genet 2, 446-57.

2.	Krasnoshtein, F. & Buchwald, M. (1996) Hum Mol Genet 5, 85-93.

3.	Rothfuss, A. & Grompe, M. (2004) Mol Cell Biol 24, 123-34.

4.	Houghtaling, S., Newell, A., Akkari, Y., Taniguchi, T., Olson, S. & Grompe, M. (2005) Hum Mol Genet 14, 3027-33.

5.	Yamamoto, K., Hirano, S., Ishiai, M., Morishima, K., Kitao, H., Namikoshi, K., Kimura, M., Matsushita, N., Arakawa, H., Buerstedde, J. M., Komatsu, K., Thompson, L. H. & Takata, M. (2005) Mol Cell Biol 25, 34-43.

6.	Sobeck, A., Stone, S., Costanzo, V., de Graaf, B., Reuter, T., de Winter, J., Wallisch, M., Akkari, Y., Olson, S., Wang, W., Joenje, H., Christian, J. L., Lupardus, P. J., Cimprich, K. A., Gautier, J. & Hoatlin, M. E. (2006) Mol Cell Biol 26, 425-37.

7.	Hirano, S., Yamamoto, K., Ishiai, M., Yamazoe, M., Seki, M., Matsushita, N., Ohzeki, M., Yamashita, Y. M., Arakawa, H., Buerstedde, J. M., Enomoto, T., Takeda, S., Thompson, L. H. & Takata, M. (2005) Embo J 24, 418-27.

8.	Niedzwiedz, W., Mosedale, G., Johnson, M., Ong, C. Y., Pace, P. & Patel, K. J. (2004) Mol Cell 15, 607-20.

9.	Taniguchi, T., Garcia-Higuera, I., Xu, B., Andreassen, P., Gregory, R., Kim, S., Lane, W., Kastan, M. & D'Andrea, A. (2002) Cell 109, 459-472.

10.	Nakanishi, K., Taniguchi, T., Ranganathan, V., New, H. V., Moreau, L. A., Stotsky, M., Mathew, C. G., Kastan, M. B., Weaver, D. T. & D'Andrea, A. D. (2002) Nat Cell Biol 4, 913-20.

11.	Pichierri, P. & Rosselli, F. (2004) Embo J.

12.	Pichierri, P. & Rosselli, F. (2004) Cell Cycle 3.

13.	Sala-Trepat, M., Rouillard, D., Escarceller, M., Laquerbe, A., Moustacchi, E. & Papadopoulo, D. (2000) Exp Cell Res 260, 208-15.

14.	Akkari, Y. M., Bateman, R. L., Reifsteck, C. A., D'Andrea, A. D., Olson, S. B. & Grompe, M. (2001) Mol Genet Metab 74, 403-12.

15.	Howlett, N. G., Taniguchi, T., Olson, S., Cox, B., Waisfisz, Q., De Die-Smulders, C., Persky, N., Grompe, M., Joenje, H., Pals, G., Ikeda, H., Fox, E. & D’Andrea, A. (2002) Science.

16.	Garcia-Higuera, I., Taniguchi, T., Ganesan, S., Meyn, M. S., Timmers, C., Hejna, J., Grompe, M. & D'Andrea, A. D. (2001) Mol Cell 7, 249-62.

17.	Taniguchi, T., Garcia-Higuera, I., Andreassen, P. R., Gregory, R. C., Grompe, M. & D'Andrea, A. D. (2002) Blood 100, 2414-20.

18.	Hussain, S., Wilson, J. B., Medhurst, A. L., Hejna, J., Witt, E., Ananth, S., Davies, A., Masson, J. Y., Moses, R., West, S. C., De Winter, J. P., Ashworth, A., Jones, N. J. & Mathew, C. G. (2004) Hum Mol Genet.

19.	Wang, X., Andreassen, P. R. & D'Andrea, A. D. (2004) Mol Cell Biol 24, 5850-62.

20.	Meetei, A. R., Medhurst, A. L., Ling, C., Xue, Y., Singh, T. R., Bier, P., Steltenpool, J., Stone, S., Dokal, I., Mathew, C. G., Hoatlin, M., Joenje, H., de Winter, J. P. & Wang, W. (2005) Nat Genet 37, 958-63.

21.	Levitus, M., Waisfisz, Q., Godthelp, B. C., de Vries, Y., Hussain, S., Wiegant, W. W., Elghalbzouri-Maghrani, E., Steltenpool, J., Rooimans, M. A., Pals, G., Arwert, F., Mathew, C. G., Zdzienicka, M. Z., Hiom, K., De Winter, J. P. & Joenje, H. (2005) Nat Genet 37, 934-5.

22.	Levran, O., Attwooll, C., Henry, R. T., Milton, K. L., Neveling, K., Rio, P., Batish, S. D., Kalb, R., Velleuer, E., Barral, S., Ott, J., Petrini, J., Schindler, D., Hanenberg, H. & Auerbach, A. D. (2005) Nat Genet 37, 931-3.

23.	Bridge, W. L., Vandenberg, C. J., Franklin, R. J. & Hiom, K. (2005) Nat Genet 37, 953-7.

24.	Dronkert, M. L. & Kanaar, R. (2001) Mutat Res 486, 217-47.

25.	McHugh, P. J., Spanswick, V. J. & Hartley, J. A. (2001) Lancet Oncol 2, 483-90.

26.	de Winter, J. P., van Der Weel, L., de Groot, J., Stone, S., Waisfisz, Q., Arwert, F., Scheper, R. J., Kruyt, F. A., Hoatlin, M. E. & Joenje, H. (2000) Hum Mol Genet 9, 2665-74.

27.	Grompe, M. & D'Andrea, A. (2001) Hum Mol Genet 10, 2253-9.

28.	Medhurst, A. L., Huber, P. A., Waisfisz, Q., de Winter, J. P. & Mathew, C. G. (2001) Hum Mol Genet 10, 423-9.

29.	Niedernhofer, L. J., Lalai, A. S. & Hoeijmakers, J. H. (2005) Cell 123, 1191-8.

30.	Davies, A. A., Masson, J. Y., McIlwraith, M. J., Stasiak, A. Z., Stasiak, A., Venkitaraman, A. R. & West, S. C. (2001) Mol Cell 7, 273-82.

31.	Larminat, F., Germanier, M., Papouli, E. & Defais, M. (2002) Oncogene 21, 5188-92.

32.	Hussain, S., Witt, E., Huber, P. A., Medhurst, A. L., Ashworth, A. & Mathew, C. G. (2003) Hum Mol Genet 12, 2503-10.

33.	Kitao, H., Yamamoto, K., Matsushita, N., Ohzeki, M., Ishiai, M. & Takata, M. (2006) J Biol Chem.

34.	Hussain, S., Wilson, J. B., Blom, E., Thompson, L. H., Sung, P., Gordon, S. M., Kupfer, G. M., Joenje, H., Mathew, C. G. & Jones, N. J. (2006) DNA Repair (Amst) 5, 629-40.

35.	Nakanishi, K., Yang, Y. G., Pierce, A. J., Taniguchi, T., Digweed, M., D'Andrea, A. D., Wang, Z. Q. & Jasin, M. (2005) Proc Natl Acad Sci U S A 102, 1110-5.

36.	Donahue, S. L. & Campbell, C. (2002) J Biol Chem 277, 46243-7.

37.	Escarceller, M., Buchwald, M., Singleton, B. K., Jeggo, P. A., Jackson, S. P., Moustacchi, E. & Papadopoulo, D. (1998) J Mol Biol 279, 375-85.

38.	Lundberg, R., Mavinakere, M. & Campbell, C. (2001) J Biol Chem 276, 9543-9.

39.	Meetei, A. R., Sechi, S., Wallisch, M., Yang, D., Young, M. K., Joenje, H., Hoatlin, M. & Wang, W. (2003) Mol Biol Cell in press.

40.	Mosedale, G., Niedzwiedz, W., Alpi, A., Perrina, F., Pereira-Leal, J. B., Johnson, M., Langevin, F., Pace, P. & Patel, K. J. (2005) Nat Struct Mol Biol 12, 763-71.

41.	Komori, K., Fujikane, R., Shinagawa, H. & Ishino, Y. (2002) Genes Genet Syst 77, 227-41.

42.	Peng, M., Litman, R., Jin, Z., Fong, G. & Cantor, S. B. (2006) Oncogene 25, 2245-53.

43.	Cantor, S. B., Bell, D. W., Ganesan, S., Kass, E. M., Drapkin, R., Grossman, S., Wahrer, D. C., Sgroi, D. C., Lane, W. S., Haber, D. A. & Livingston, D. M. (2001) Cell 105, 149-60.

44.	Centurion, S. A., Kuo, H. R. & Lambert, W. C. (2000) Exp Cell Res 260, 216-21.




[http://www.nature.com/ng/journal/v37/n9/full/ng0905-913.html Nature Genetics Article about FA research]

[http://en.wikipedia.org/wiki/Fanconi_anemia  Wikipedia article]

Back to Main Page [[Hoatlin Lab]]