<div class="tabs-blue">
<ul>
<li >[[IGEM:IMPERIAL/2006/project/popsblocker | Specifications]]</li>
<li id="current">[[IGEM:IMPERIAL/2006/project/popsblocker/Design | Design]]</li>
<li >[[IGEM:IMPERIAL/2006/project/popsblocker/Modelling| Modelling]]</li>
<li >[[IGEM:IMPERIAL/2006/project/popsblocker/Implementation| Implementation]]</li>
<li >[[IGEM:IMPERIAL/2006/project/popsblocker/Testing| Validation]]</li>
</ul>
</div>
<br style="clear:both">


===Cre-LoxP System===

==Registry==

Two devices are required:
*1) A complementary sequence for a reporter with loxP sites either side 
*2) Cre recombinase under the control of a promoter

1) The LoxP device J37027 can be found [http://parts.mit.edu/registry/index.php/Part:BBa_J37027 here]


[[Image:Part_J37027.jpg|300px]]



2) Cre recombinase device in registry [http://parts.mit.edu/registry/index.php/Part:BBa_J37030 J37030]


[[Image:J37030.PNG]]

==Design choices==



'''Overview'''

This part is designed to be placed downstream of a promoter and prevent any Pops from the Promoter passing through this part. It will do this until an accompanying Cre Recombinase plasmid becomes activated. Once the Cre recombinase is activated the enzyme produced will permanently cut a section of DNA from the plasmid containing this part. This excised section of DNA is degraded. This short section of DNA contains stop codons and terminator sequences so once these are removed the polymerase can pass through this part and transcribe downstream genes. 

'''Reporter  Gene'''

The Part also contains a RFP reporter which is transcribed in the 3’-5’ direction. This means that un-activated parts will fluoresce red and activated parts will not fluoresce. This allows you to see that the part is working in your system. It also allows you to observe the efficiency of activation of the part in your system. The part runs the other way to the rest of the system to prevent any fluorescence due to external Pops passing into the device

'''Lox Sites'''

The Cre lox system is very complex and it will not be described fully. There are multiple Lox sites which exist - we have chosen Lox sites 66 and 71. They are both placed in the forward direction on the sense strand of the DNA. This causes the DNA between the lox sites to be excised rather than reversed. Lox 66 and 71 have been mutated so that the reversed strand of DNA cannot be re-inserted by cre. Lox 66 should be upstream of Lox 71. The DNA between the Lox sites is seen as a plasmid in the cells once it has been excised however it has no origin of replication so will be quickly degraded inside the cell.

Sequences - [http://parts.mit.edu/registry/index.php/Part:BBa_J37026 Lox66] and [http://parts.mit.edu/registry/index.php/Part:BBa_J37028 Lox71]

'''Terminator Sequences'''

There are two main terminator sequences in the registry: 1) B0015 which consists of two separate sequences B0010 and B0012. 2) B0012 which is a single terminator sequence. B0015 is much more efficient than B0021, 98.4% and 60% respectively. The two terminator sequences in this part cannot be the same as when you try to PCR them they will bind together and form a 131 base pair hairpin loop which the polymerase cannot pass through. Therefore the two terminator sequences must be different. We just have to accept that the stopping will not be as efficient. In theory we could have multiple copies of B0021 but this would require several ligations and we do not have time for this.

[[Image:Cre2.PNG]]


===Cre recombinase===

*A Cre recombinase device has been designed in the registry - [http://parts.mit.edu/registry/index.php/Part:BBa_J37030 J37030]

*This device produces the enzyme Cre recombinase under the control of the lacI promoter. Therefore by introducing IPTG, one would be able to induce the production of Cre. 

*It is also possible to obtain commercially available Cre plasmids as well

Cre sequence and primers that will be used to remove the sequence [http://openwetware.org/images/f/ff/Cre_Primers.doc here]


[[Image:Cre.PNG]]


===System Incorporation===

There is one further step that we need to take if we want to successfully incorporate this system into the whole system. The plasmid containing the Cre enzyme coding region will be going into the prey cell. However the prey cell will already contain the main coding plasmid, which has ampicillin resistance. Therefore to ensure that the second plasmid is taken up into the cell as well, we need to swap plasmids by ligation. Ligating into a kanamycin resistant plasmid is probably our best bet. We could do this after the device has been constructed in the ampicillin resistant plasmid.  

*[http://openwetware.org/wiki/IGEM:IMPERIAL/2006/project/Oscillator/cellcre Further Discussion]on the incorporation of the Cre recombinase into our system

{| border="1" width="100%"
|- style="background:lightgreen"
! Inputs !! Biological Component !! Comments
|-
| Cre|| J37030 or any other Cre plasmid || Plasmid requires different resistance 
|-
| Pops || Any active promoter || -
|- style="background:orange"
! Ouptuts !! Biological Component !! Comments
|-
| Reporter|| There will be a loss of RFP. A positive reporter such as GFP could also be added|| - 
|}


==Open issues==
*The parts are not available in the registry however, which means that we would have to make them. 
*The loxP sites are around 20 bases in length and so sequences can be be worked out and bought fairly easily. 
*The Cre recombinase site is around a kilobase in length. However we have been able to obtain the enzyme from one of the labs, and as it is isolated in a plasmid, this will make it much simpler for us to incorporate it into our design.
*The Cre recombinase will also need a promoter - perhaps we could use one such as lacI? (we need to find out whether IPTG has any effect on other promoters though)
*With all the parts available, we estimate that the length of time for implementation should be under a week.

==Further Reading==

*[http://www3.interscience.wiley.com/cgi-bin/fulltext/70001850/PDFSTART Cre recombinase: The universal reagent for genome tailoring]
*[http://nar.oxfordjournals.org/cgi/content/full/30/17/e90 Cre recombinase-mediated inversion using lox66 and lox71: method to introduce conditional point mutations into the CREB-binding protein]

<html>
<!-- Start of StatCounter Code -->
<script type="text/javascript" language="javascript">
var sc_project=1999441; 
var sc_invisible=1; 
var sc_partition=18; 
var sc_security="18996820"; 
</script>

<script type="text/javascript" language="javascript" src="http://www.statcounter.com/counter/frames.js"></script><noscript><a href="http://www.statcounter.com/" target="_blank"><img  src="http://c19.statcounter.com/counter.php?sc_project=1999441&amp;java=0&amp;security=18996820&amp;invisible=1" alt="website statistics" border="0"></a> </noscript>
<!-- End of StatCounter Code -->
</html>