Looking back on the ideas that were presented, I think some key elements we should all look for in candidate projects for the summer are: modularity, feasibility and 'iGEMness.'That last one refers to the creation of standard parts and such.
In that vein, engineering an entire metabolic pathway to generate some given vitamin may be infeasible granted that we haven't much time. However, I think cell & tissue targeting could be a first step in increasing the '''bioavailability''' of whatever we intend to deliver -- vitamins, minerals, drugs, hormones etc. Other things we could look at if we have time would be controlled delivery and efficacy.

So, I'd like to work in this direction.

Sammy Sambu
------------------------------------------------------------------------------------------------------------------------------------------
Hi all! Great meeting yesterday, with a lot of terrific ideas proposed.  Now comes the hard part ... deciding which project to pursue this summer.  I put an overview slide of some of the projects proposed over the last three brainstorming sessions here ([[Media: Brainstorming_Overview.ppt| Brainstorming Overview]])  I hope it's of use.  You can use the section below (between the ---) to hash out final ideas, proposals, and to lobby for your favorite projects.  If you have any questions on updating the wiki you can contact me or Nick via email.  Thanks, Mike (4/24/07)

------------------------------------------------------------------------------------------Final Project Ideas, Discussions, and Lobbying (post 4-23-07)<br>
<br>
I really like the idea of pathway engineering. I think we should stick to <i> E. coli </i> for several reasons: 1) the knowledge of E. coli, both within our group members and from outside papers, seems to have a comparative advantage over other organisms; 2) many of the pathways we're considering have been explored using E. coli already; 3) E. coli seems relatively easy to work with (though I can't argue this from experience). I found a really nice paper that summarizes some of the pathways we've considered, and what we might be able to do. [http://www.ibmb.uni.wroc.pl/prace2/praca12.pdf Commercial Production of Chemicals Using Pathway Engineering]. I particularly like the following excerpt: <br>
We have constructed a strain of E. coli containing the genes from S. cerevisiae for glycerol production and the genes from K. pneumoniae for 1,3-propanediol production. E. coli provides several advantages of other systems. E. coli is the most completely studied organism. E. coli provides a rich set of genetic tools: sequenced genome, vectors, promoters, etc. E. coli's metabolism and physiology are well characterized and a large number of metabolic mutants have been constructed and analyzed. E. coli has been used in large scale fermentations and production on an industrial level. In addition, E. coli is closely related to the natural 1,3-propanediol producers: K. pneumoniae and C. freundii. Since E. coli does not naturally produce glycerol or 1,3-propanediol, there is no natural regulation to overcome. Through the construction of arti¢cial operons for the optimized expression of the genes for the 1,3-propanediol pathway, we have built an E. coli strain, which can produce 1,3-propanediol from glucose. Currently, the 1,3-propanediol production performance of our glucose to 1,3-propanediol organism equals or surpasses that of any glycerol to 1,3-propanediol natural organism.<br>[As a side note, 1,3-propanediol is a monomer with potential utility for production of polyester fibers and manufacture of polyurethanes. In itself, it's not a project I'm suggesting, but I think the excerpt captures the relative ease of working with E.coli for engineering pathways, and it shows the efficacy of E.coli in other engineering studies.]<br>
... So I think I've done enough encouragement for using E. coli in general, and from personal discussions I've had with some of you, it seems like we're in agreement. I think the engineering for EtOH synthesis was a popular topic, but I'm not sure exactly where we want to go with it- it seems that a lot of groups have already engineered relatively efficient strains, and with our time restraint, I doubt that we'll get results that even come close to meeting the standard that has already been set. I like the idea of vitamin synthesis - it seems that plant engineering has been extremely popular in that regard, but not as much attention has been given to organisms such as E. coli - so I think it might be an interesting path to try. Thoughts? 

--Stephanie (4/25)


Hi Stephanie, thanks for posting your ideas on the web.  In response to your post, yes, I agree that E. coli would be the most tractable organism to use this summer, and I do like the idea of metabolic engineering as well as surface engineering.  I’ll post some examples from the literature on engineering pathways in microorganisms in the next few days.  Two papers you may be interested in:

1. Engineering a mevalonate pathway in Escherichia coli for production of terpenoids.  Martin VJ, Pitera DJ, Withers ST, Newman JD, Keasling JD.    ([[Media: Martin.pdf| Nat Biotechnol. 2003 Jul;21(7):796-802.]])

2. Combinatorial engineering of intergenic regions in operons tunes expression of multiple genes.  Pfleger BF, Pitera DJ, Smolke CD, Keasling JD.   ([[Media: Pfleger.pdf| Nat Biotechnol. 2006 Aug;24(8):1027-32. ]])

You may also want to examine how many steps (reactions or proteins) are involved in the pathways you are interested in, since this will give you an idea of the complexity of the potential project.  Thanks again for posting your idea.  I'll add some more feedback in the next day or so.

--Mike (4/30)


Wow, thanks for the papers. I think it would be a good idea for everyone to read them - but if anyone on the team is willing to read/bullet/summarize the papers so it is easy for us the extract what we think is important, that would be wonderful. I will probably be reading them within the next few days, and I'll post my thoughts afterward. 

--Stephanie (5/1)


'''Bacterial Surface Expression and Cellular Targeting''' (compiled 5/2/07 by Mike)
 

Hi all, I put together a project outline here related to Perry’s and Stephanie’s presentation on surface targeting of bacteria, focusing on an area that many of you expressed interest in: '''Medical Applications of Synthetic Biology'''.  Importantly, it is a project that is feasible to finish during the summer time limit (a big consideration), a project that would be a great resource for future researchers, has the potential for quality research with wide ranging applications, and a project that each of you could really contribute to.  Also it embraces the iGEM idea of creating modular genetic systems.


The project outline involves the creation of a modular system used to '''express a peptide library on the surface of E. coli'''.  This would utilize the power of combinatorial constructions, selections, and surface expression (Three underutilized strengths George Church talked about during our first meeting).  


Essentially, we could use the BioBrick strategy to create two "parts": 1 ) a membrane protein that will anchor a protein or peptide library of interest to the outer membrane of E. coli, and 2) a random peptide library that we can fuse to the outer membrane portion.  We could use this to select peptides that bind various tissue types or cells, enabling us to target bacteria to specific cell types.


An outline of the strategy is as follows:


1) '''PCR or have synthesized the genes for two or three membrane proteins''' that will serve as the membrane anchors.  PCR primers will contain the BioBrick sequence ends to create fusion proteins.  Two candidate membrane proteins are OmpA and AIDA1 (see references).  We can also use some constructs already made by myself, Perry, and the MCB100 students last quarter to facilitate the project.

''References:
Display of heterologous proteins on the surface of microorganisms: from the screening of combinatorial libraries to live recombinant vaccines. Georgiou G, Stathopoulos C, Daugherty PS, Nayak AR, Iverson BL, Curtiss R 3rd.
Nat Biotechnol. 1997 Jan;15(1):29-34. Review.'' [[Media:Georgiou.pdf]]   

''Autodisplay: efficient bacterial surface display of recombinant proteins. Jose J.
Appl Microbiol Biotechnol. 2006 Feb;69(6):607-14. Epub 2005 Dec 20. Review.'' [[Media:Jose.pdf]]


2. Next we can '''synthesize a double stranded nucleic acid sequence that encodes a random peptide library'''.  This is straightforward to have synthesized since we can synthesize a random oligonucleotide (equal mix of A,T,C,G at each position) flanked by the BioBrick sequences.


3. Next we can '''ligate the Membrane Portion and Random Peptide Library into an expression plasmid''' and overexpress the membrane protein-peptide library in E. coli.


4. We can then '''characterize''' the overexpression by running protein denaturing gels, assay transformation effeciency and coverage of the library, and assay for surface expression by using a variety of control surface proteins like histidine tags (bind nickel) and streptavidin binding tags (bind streptavidin).  I have some control vectors made that we can use during our testing.


5. After we construct the fusion library we can use the power of selection to identify '''surface peptide sequences that target our bacteria to various tissue types or cell types''' we are interested in.  This has '''important medical implications''' since we may be able to target "microbial factories" to various areas of the body using this method (related to many of your brainstorming ideas realting to '''synthetic symbiosis''', ie for vitamin production etc).  We can also select for peptides that bind to other cell types.  The list goes on.


This system is analogous to what has been done using phage (see reference), but in our case we can use bacteria, since bacteria have the added advantage that once bound they can produce desired products or perform desired functions (see references), such as targeting cancer cells (see references).
*Organ targeting in vivo using phage display peptide libraries. Pasqualini R, Ruoslahti E.
Nature. 1996 Mar 28;380(6572):364-6.  [[Media:Pasqualini.pdf]]

''Environmentally controlled invasion of cancer cells by engineered bacteria.
J Mol Biol. 2006 Jan 27;355(4):619-27.  Anderson JC, Clarke EJ, Arkin AP, Voigt CA.'' [[Media:Anderson.pdf]]

''A bacterial protein enhances the release and efficacy of liposomal cancer drugs.
Science. 2006 Nov 24;314(5803):1308-11. Cheong I, Huang X, Bettegowda C, Diaz LA Jr, Kinzler KW, Zhou S, Vogelstein B.'' [[Media:Cheong.pdf]]


This would be a really exciting project that would be feasible for the summer timeframe.  We could contribute many new BioBricks to iGEM at the end (membrane proteins and all the peptides we identified as binders), and potentially create a system that many other scientists could use in their own future projects.  The project also employs the power of combinatorial constructions (random peptide library), selections (select for binding to specific tissue types and cell types), and much needed advancements and modularity of cellular surface expression.  We can also build on work done by Perry, myself, and some of the MCB100 students this past quarter on related surface expression projects.


Please add comments or let me know if I can provide any more information on this.  I think it would be a great project to pursue, that would utilize all your talents, and help you learn many important experimental methods. We also have some members, postdocs, and TFs with experience and interest in this area, which will help the project move quicker this summer.  Thanks, Mike (post 5/2)


Wow. Mike, I really like the idea, and the way you presented it was very clear and helpful. Aside from reading over the references that you've provided, what are the most important things for us to do between now and the next meeting?

--Stephanie (5/2)

I really think this sounds good too. It seems like there's a solid experimental plan that we could actually follow and manage to accomplish a lot. I have not read the papers yet, but I plan on doing so tomorrow (Thursday) or by Friday at the latest. I will post back my thoughts within that time frame. But for now, I'm pretty excited. --Shaunak (5/2)

Ok, so I've read/skimmed the papers above (a day or so later than originally planned, but oh well)--here are my impressions. Firstly, I think this is definitely a good idea to pursue, and I think given that there are TFs and students who have interest and experience in related fields, we would be very well positioned to work through such a project. That said, on to some more substantive issues/questions:

1. Is there any advantage/difference to using an Lpp-OmpA system over an AIDA1 system? Or are they effectively equal? From what I understood, Lpp-OmpA has only failed in expressing alkaline phosphatase on the surface, but it apparently can't deal well with complex secondary/tertiary structures? Jose presented AIDA1 as being superior to Lpp-OmpA, so I'm assuming that it doesn't have these problems? From what I got from that article, AIDA1's main limitation is the formation of highly stable folded conformations (spec. disulfide bonds) that can't be threaded through the beta barrel. He said though (in only one sentence) that a Dsb-A negative (I'm assuming this means no beta barrel) could overcome this problem--would we want to consider using this type of a transporter? I'm not clear on what the drawbacks are, although I presume there must be some. You were also saying there are possible other constructs from MCB100--what are these, what are their advantages/disadvantages?

2. I'm a little unclear on how the random peptide library construction and testing works. Could you explain it a little more? Are you suggesting that we essentially just have random DNA bases put in corresponding to the regions that will be extracellular? Is this the standard methodology for creating random peptide libraries? I'm guessing we make many of these random sequences and then make it so that each bacteria only takes up one of these plasmids, right? For this type of strategy to work for targeting, I'm guessing we'd have to have the number of peptide molecules we're expressing number in at least the thousands. Will we be able to make random peptide sequences on these orders of magnitude and then test them all at once? I really like the idea of using random peptides though as long as the library can be large enough and we can do large scale screens (although we may want to consider also using known binding for particular tissue types as well). 

3. How are we going to test for successful interactions? In these articles, they mainly use FACS, with the exception of the one where they just see where the cells accumulate. I'm guessing we can all talk about what types of tissues we'd want to test--would we just test them all at once? Would we first try to find one that targets a particular tissue, and then move onto the next? We might maybe want to consider just picking a particular type of tissue (maybe some type of cancerous cell), and see if we can then use the targeting to then make something else happen (is there anyway we can create a switch that only turns on if there is successful targeting? this may be beyond our capabilities). 

These are just some preliminary thoughts/questions. I like this project a lot though, and right now I think it'd be a good area for us to pursue. Do others of you have thoughts? --Shaunak (5/6)


Shaunak, some great questions! I'll try to answer some of them, though others should definitely jump in if I have anything wrong. 

1. I think the other constructs from MCB100 that he mentions are the ones I presented on - adaptamers. It seems that they would be a round-about way to approach this specific project, since if we were to use adaptamers, we would have to have the bacterial surface protein, AND the adaptamer, AND a site on the target cell/tissue that is specific for one end of the adaptamer. Hence, at least three parts are necessary for a direct adaptamer-based approach, whereas I think the idea is to have a simple bacterial surface<->target tissue interaction (though the implications of having a "control" molecule in between could be interesting). Also, if we wanted the adaptamer-approach, we'd have to have a few more controls to test whether the adaptamer is binding, whether the surface protein<->tissue is occurring without the adaptamer, etc. ... But I think Mike's point was that we can use some of the adaptamer sequences that we already found in MCB100. ... As the membrane-proteing group has probably found, with time-crunch to get research done, sometimes it's best to adopt known sequences that are known to interact with other proteins. These can be found in other papers, often in the "methods" section, or, in the case of our project, I believe that Perry and the MCB 100 team have some sequences. For the adaptamer project, we have some sequences that are specific for streptavidin and thrombin ... Mike mentioned using streptavidin to characterize for surface expression. In MCB100 fall term, we did this by using streptavidin magnetic beads, so that the thrombin with the adaptamers (which also had streptavidin specificity) would "link" to the streptavidin and get pulled down in the presence of magnet. We then washed, incubated with antibodies a couple times, and the like, and eventually we could use this protocol to detect for fluorescence, allowing us to compare relative binding or presence/absence of binding. My guess is that we can use similar methods for the project in question. [... This method answers some of your questions in the next part, I believe].

2. Your thoughts on random peptide library construction seem generally correct. Have you heard of DNA microarrays? It allows for thousands of little DNA spots, with known locations on the slide, to be tested. In a bacterial whole genome array analysis, there is a number of issues that emerge, but for procedure seems relatively standardized now, with some prep kits available. I definitely suggest that you look at the following site: [http://www.ambion.com/techlib/tn/114/8.html Bacterial Whole Genome Array Analysis]

I'll try to cover the other questions soon, as time allows. Thanks for all your thoughts, and let me know if I should clarify/elaborate on anything I said!

--Stephanie (5/6)

'''Re: Surface Expression Questions''' (Post by Mike 5/7)

Hi all, thanks for all your great comments and suggestions.  You have had some very insightful discussions.  In regard to some of your questions, both AIDA-1 and Llp-OmpA have been shown to be effective for surface expression.  As Shaunak pointed out, there are some challenges in this realm, such as proteins that fold into complex tertiary structures as well as proteins that form disulfide bonds.  This can sometimes inhibit proteins from getting out to the outer surface.  One way that Jose overcame the disulfide bond issue in the AIDA1 case was to use a strain of E. coli that is deficient in a protein called Dsb-A, which facilitates disulfide bond formation.  If we choose to work on random peptide libraries, we may not have to worry so much about some of these considerations since our peptides will be much shorter than most full length proteins, and therefore are not likely to have such complex tertiary structures as some larger protein counterparts.

Yes, MCB 100 has some constructs that we may want to use, either directly or to make our own expression vectors.  Some of the vectors they and I have are fusion constructs made from both the Llp-OmpA and the AIDA-1 membrane proteins.  They (and I) have some control protein fusions that we can use to optimize some of our screens before screening a peptide library if you would like.  We can also use other membrane proteins of your choosing.

To make a random peptide library, we can order oligonucleotides from a DNA synthesis company.  We will design the sequence and have them make it for us to use.  A random library is quite straightforward to synthesize, because we can request that for each position in the nucleotide sequence that they incorporate an equal mixture of A,G,C, and T nucleotides, for 30 bases or so.  These random bases can be flanked by BioBrick sequences, for insertion into a surface expression fusion vector.  Such a strategy would yield a random mixture of peptides (each 10 amino acid long).  There are other strategies for reducing possible stop codons in the random sequence that we can use too. 

The random oligonucleotides (encoding a random peptide library) can then be fused to a membrane protein like OmpA or AIDA-1 (in an expression plasmid).  After transformation of the plasmids into E. coli, we could then screen and select for binding to various targets using various strategies.  At the end of comment #1, Stephanie, described some of the control experiments we could do using magnetic beads that have affinity for various peptide or protein sequences.

Hope this helps, keep up the great discussions!  Mike (5/7)


Small addendum: ... I'm thinking about what Shaunak said in (3). Not sure if FACS is our simplest option. With Protein-Tissue interactions, we could do a Co-IP (immunoprecipitation), but not sure if we could do that (or a variation of it) given bacteria. I'm sure one of the more experienced TFs, who has worked with E. coli, has a suggestion ...? I'll continue to think about it.

Still have to read through Mike's comments (directly above), will continue to brainstorm and post of course ...

--Stephanie (5/7)

I've been in term-paper hell for the past few days, and so haven't been able to keep up with the brainstorming.  I really like Mike's idea, though.  Some people have asked me to flesh out the death gene idea, so I've attached a sketch of the project: 

Components: 
•	Suicide gene
o	Lambda Phage
o	LlaIR+ restriction cassette
•	Promoter
o	LacZ
o	Other promoters available on biobricks website, but they generally require purified protein as an inducer—that’s not optimal. 
Methods:
•	Use endonucleases to restrict out normal promoters from suicide gene. 
•	Separate out promoters from coding sequence using gel electrophoresis
•	Combine LacZ or other promoter with suicide gene using ligases. 
•	Integrate into bacterial genome

This paper outlines the death L1aIR+ idea:
Djordjevic GM, O'Sullivan DJ, Walker SA, Conkling MA, Klaenhammer TR. Related Articles, Links 
 
A triggered-suicide system designed as a defense against bacteriophages.
J Bacteriol. 1997 Nov;179(21):6741-8. 
PMID: 9352925 [PubMed - indexed for MEDLINE]
The only new thing that this project does is combining the confirmed death gene with a new promoter.  It ought not to be that difficult and might work as a minor project alongside Mike's -- Alexander


Thanks, Alexander. Logistically, I don't think splitting into two projects is a good idea - really reduces group interaction, and if one project "finishes" (... which is unlikely, but still), then merging with the group that has already developed a group dynamic will be difficult (... I know this from an attempted merge of groups in MCB 100 ...). Something we can talk about when we see the group soon.

--Stephanie (5/9)