[[Image:CloningBBs.jpg|thumb|right|Cloning BB Schematic Example]]

===Phenol-chloroform extraction===
 Add 1:1 ratio of Phenol-chloroform --> vortex --> spin
 Remove layer of Phenol-chloroform (usually at the bottom)
 Transfer top layer to a new tube
 Add 1:1 ratio of chloroform --> vortex --> spin
 Remove chloroform layer at the bottom
 Transfer the remains to a new tube
 Add 1/10 amount of Sodium acetate --> vortex --> spin
 Add 2.5x amount of 100% EtOH (thats ethanol for people who don't know) --> vortex --> spin
 Put in -80 degrees celsius freezer for approx 20 min. till it becomes gel-like
 Centrifuge at high speed (18000 rpm) for approx 20 min.
 Look for a pellet
 Remove supernatant
 Add a decent amount of 70% EtOH to wash salt away
 invert upside down, vortex lightly if needed
 Centrifuge for short amount of time to see the pellet again
 Remove EtOH --> air dry --> DO NOT OVERDRY, SMELL IF ETOH IS THERE OR GONE
 Resuspend in dH20 (20 microlitres)

===Resuspend Primers===
 Centrifuge vial 1 min. to allow "dry" DNA to fall to bottom.
 Resuspend primers to working concentration. 
 [[Image:Conversions.xls|PrimerResuspensionConversion.xls]]
 Find <b>nmoles</b> on order form for primer.
 Convert nmoles to 20 picomoles/uL to find X. (This means multiply X nmoles by 50)
 Pipette X uL 1X TE into vial. 
 Vortex 1 min.
 Insert <font color=blue>BLUE</font> label-cap. Label appropriately.
 Store bottom shelf -20 Freezer in PRIMER box.

===<font color=purple>PCR</font>===
 1 uL each primer <font color=#cccccc>(-20 freezer, BLUE caps)</font>
 3 uL template DNA 
 45 uL PCR Super Mix <font color=#cccccc>(-20 freezer)</font>
 ---
 50 uL rxn
 [[../Denature/]], [[../Anneal/]], [[../Extend/]] in thermocycler (see links for details)
 <font color=red>Negative Control: No template DNA</font>

===Making a Gel===
 1%
 0.5 g agarose into blue weight boat <font color=#cccccc>(bench C)</font>
 50 mL 1X TAE into graduated cylinder <font color=#cccccc>(counter with solution stocks)</font>
 Add agarose into 250 mL sterile flask <font color=#cccccc>(cabinet, back of room, left)</font>
 Add TAE into flask. Swirl.
 Insert Kimwipes into top of flask to prevent boiling out
 Microwave till boil. Continue to swirl during time until all dissolved. <font color=#cccccc>(Alcove 1)</font>
 Add Ethidium Bromide to flask. (<font color=red>0.6uL/100mL</font>) <font color=#cccccc>(Cold Room, front shelf, metal bin)</font>
 Pour into gel tray of gelbox. Let sit until hardened. 
 Use TAE as running buffer.<br>
 2%
 1 g agarose
 50 mL 1X TAE
 Same procedure as above.
 
 1% in endy lab
 12 well boxes need 30-35mL gel.
 EtBr added @ ~1uL/35mL.
 should have 1% agarose stock most of the time.

===Test PCR on gel===
 5 uL PCR rxn
 5 uL H20 or TE
 2 uL 6x sample buffer <font color=#cccccc>(-20 freezer)</font>
 ---
 12 uL total load into 12 well gel.

===<font color=Purple>Clean PCR</font>===
 Use Qiagen PCR cleanup kit.<Br>
 1. Add 5 volumes Buffer PB to 1 volume PCR mix.
 2. Apply sample to QIAquick spin column.
 3. Let sit 1' to bind DNA. Spin 1'.
 4. Discard flowthrough. Place back in same tube.
 5. Add 750 uL Buffer PE to wash. Spin 1'.
 6. Discard flow-through. Place back in same tube.
 7. Spin 1' to completely remove residual ethanol.
 8. Place column in clean 1.5 mL microcentrifuge tube.
 9. Elute with 50 uL Buffer EB. Let sit 1', spin 1'.
 Label with <font color=purple><b>PURPLE</B></font>, store in Cold Room.

===Ethanol Precipitation===
 1/10 vol of 3M sodium acetate (pH5.2)
 X solution of DNA<br>
 Flick tube several times to mix.<br>
 Add 2 vol or more of ice-cold 100% ethanol. 
 Vortex. 
 Place in -20 freezer 30 min, or -80 freezer 15 min.
 Spin 5 min. in microcentrifuge.
 Add 1mL 70% ethanol (room temperature).
 Invert tube several times. Spin. 
 Pipette off supernatant.
 Dry pellet under vacuum or Speecvac (or on the bench).
 Dissolve pellet in appropriate volume with EB Buffer.
 Dissolve by flicking or vortex briefly.

===<font color=green>Digest</font>===
 <font color=red><b>Enzymes should be added last. Keep enzymes in freezer until needed!!</b></font>
 5 uL 10X NEB Buffer #_ (1, 2, 3 according to REs used) <font color=#cccccc>(-20 freezer, bottom shelf)</font>
 0.5 uL 100X BSA <font color=#cccccc>(-20 freezer, bottom shelf)</font>
 2.5 uL each restriction enzyme <font color=#cccccc>(-20 freezer, bottom shelf)</font>
 X uL plasmid (at least 1-10ug in 35uL is needed)
 X H20
 ---
 50 uL digestion - in PCR tube
 37°C 6 hr. (2 hrs works, too).
 65°C 10'
 4°C forever<BR><BR>Label with <font color=green><b>GREEN</b></font>, store in Cold Room.

===Test Digest===
 X% 12-well gel(s)             
 5 uL cut product
 5 uL H20
 2 uL 6X sample buffer
 ---
 12 uL total
 <font color=red><b>Negative control with uncut product!!!</b></font>

===Gel Purify===
 20 uL cut product
 4 uL 6X sample buffer <font color=#cccccc>(-20 freezer, bottom shelf)</font>
 ---
 24 uL total

===<font color=blue>Gel Extract Digest</font>===
 Use Qiagen Gel Extraction Kit.
 1. Excise DNA fragment from agarose gel using a clean razorblade.
 2. Weigh gel slice.
 3. Add 3 volumes of <font color=gold>Buffer QG</font> to 1 volume gel.
 4. Incubate at 50°C until gel slice completely dissolves.
 5. Apply sample to QIAquick spin column. Spin 1'. 
 6. Discard flowthrough. Place column back into same tube.
 6. Add 700 uL Buffer PE to wash. Spin 1'.
 7. Discard flowthrough. Place back into same tube.
 8. Centrifuge for addition 1' to remove all ethanol residue.
 9. Place column in clean 1.5 mL microcentrifuge tube.
 10. Add 50 uL Buffer EB, let sit 1', spin 1' to elute.
 11. Label with <font color=blue><b>BLUE</b></font> label. Store in Cold Room (4°C).

===Nanodrop Product===
 1. 1 uL blank (usu. EB)
 2. Hit REBLANK
 3. Record figure
 4. Clean with H20 using Kimwipe
 5. 1 uL product
 6. Hit MEASURE
 7. Record figure
 8. Clean with H20 using Kimwipe
 9. Repeat #5 and on if necessary.<BR>
 <u>Location</u>: Endy Lab or Knight Lab
 Obtain concentration (ng/uL) from Nanodrop.
 Use [http://openwetware.org/images/c/c6/Conversions.xls Conversions.xls] to find <b>fm</b> from concentration and size (bp) of DNA.
 Need 20 fm recipient, 60 fm insert for ligation.

===<font color=gold>Ligation</font>===
 <font color=green>For good results, use a 1:3 mole ratio of recipient:donor to force ligation rxn.</font>
 20 fm recipient
 60 fm insert
 2 uL 10X ligase buffer <font color=#cccccc>(-20 freezer)</font>
 0.5 uL T4 DNA ligase <font color=#cccccc>(-20 freezer)</font>
 X uL H20
 ---
 20 uL total ligation
 <font color="red"><b>Negative controls: 1 without insert, 1 without DNA, 1 with only insert</B></font>
 16°C 15'
 4°C forever<br><br>Label <font color=gold><b>YELLOW</b></font>, store in Cold Room.

 note: go to [http://openwetware.mit.edu/index.php?title=Endy:DNA_ligation_using_T4_DNA_ligase Endy Digest Protocol] for help figuring out gram amounts of DNA to add

===Pouring Plates===
 <b>Amp Plates</b>
 10mL Amp
 1000mL LB
 <b>Kan Plates</b>
 1mL Kan
 1000mL LB

===Transformation (CaCl2 competent cells)===
 <font size=2><B>Protocol per tube</b></font>
 50 uL competent cells
 3 uL ligation reaction<br>
 GENTLY mix ligation/cells with pipette tip.
 Incubate on ice 10'.
 Heat shock in 42°C 2'.
 Immediately place on ice.
 Add 1 mL LB to each tube, transfer contents to culture tube. <br>
 Place tubes in Warm Room rollerdrum 37°C 15'.<br>
 <b>Use sterile technique.</b>
 Plate 100 uL transformation onto LB-DrugMarker plate. 
 Incubate upsidedown overnight at 37°C.
 <font color=red>Negative Controls from ligation! 1 vector only, 1 insert only, 1 without DNA.</font><br>
 '''Another Method'''
         1. Thaw cells on ice.
         2. Add 2 uL rxn
         3. Let incubate on ice 30'
         4. Heat shock 42 for 30 sec.
         5. Add 250 uL SOC or LB
         6. Grow at 37 rollerdrum 1 hr.
         7. Spin 1'
         8. Pull off SOC/LB
         9. Resuspend in 100 uL SOC/LB
         10. Plate.<br>
 '''Even Better Method'''
         1. 20 uL ligation
         2. 1000 uL 1-butanol
         3. Mix
         4. -80 for 15'
         5. Spin at 13K rpm, 30'
         6. Pull off s/n
         7. Dry pellet in SpeedVac
         8. Resuspend in 10 uL H20
         9. Transform all 10 uL in 50 uL Omnimax E. coli cells.

===Overnights===
 <b>for amp resistant strains</b>
 5 mL LB per culture
 25 uL 200X Amp per culture<br>
 <b>for kan resistant strains</b>
 5 mL LB per culture 
 5 uL 1000X Kan per culture<br>
 <b>...continued</b>
 X culture tubes (big ones with blue cap)
 <b><font color=red>Use sterile technique!!!</font></b>
 Use a sterile toothpick to dab the single colony desired.
 Stir into culture tube of LB+DrugMarker. 
 Incubate in Warm Room on rollerdrums for no more than 16 hours.

===Glycerol Stocks===
 900 uL overnight
 300 uL 40% glycerol
 freezer tube, labeled legibly
 -80 Freezer.

===<font color=red>MiniPrep Plasmid</font>===
 <b>Qiagen Kit</b>
 1. Pellet cells in 1.5mL microcentrifuge tube 13K rpm, 1'.
 2. Resuspend pelleted cells in 250 μl Buffer P1. <font color=red>No visible clumps!</font>
 3. Add 250 μl Buffer P2, invert 4-6X to mix. <font color=red>Do not vortex!</font>
 4. Add 350 μl Buffer N3, shake 4-6X to mix. <font color=red>Should be cloudy</font>
 5. Centrifuge for 10' @ 13K rpm. <font color=red>compact white pellet should form</font>
 6. Pipette supernatant to the QIAprep spin column. Dispose tube w/ white gunk. Spin 1'.
 7. Add 750 μl Buffer PE, spin 1'.
 8. Discard flowthrough, spin dry 1'.
 9. Place in clean 1.5mL microcentrifuge tube.
 10. Elute with 50 μl EB, spin 1'.
 11. Nanodrop for concentration (ng/μl)
 12. Label with <font color=RED><b>RED</b></font> label. Store in Cold Room (4°C).

===Lambda Red===
 1.Grow 5mL of overnight culture at 30 degree
 2.Turn on water bath to 32 degree. Turn on another water bath for 42 degree.
 3.In a baffled flask, dilute 2 mL of cluture into 100 mL LB. Grow at 32 degree to an OD of 0.4-0.6
 4. Make an ice water slurry.
 5.In a 50 mL baffled flask, put 10 mL of this culture. Make a 10mL control.
   Put in a 42 degree shaker at 200 rev/min for 15 mins.
 6.Swirl the flasks in the ice-water slurry for 10 mins. 
 7.Centrifuge for 8 mins at 5,500g at 4 degree.
 8.Resuspend pellet in 1 mL ice-cold sterile water. Transfer to 1.5mL tubes. Spin 20s at 4 degree in microfuge
 9.Wash pellet twice with cold, sterile water.
 10.Suspend in 100microL ice-cold sterile water. Store on ice.
 11.Aliquot into 50microL tubes. Add desired amount of plasmid/chromosomal DNA and insert/linear DNA, for example:
    a. 0.1 ng plasmid/chromosomal DNA
    b. 300 ng insert/linear DNA
 12.Pipette cells into a precooled cuvette, and electroporate.
 13.Immediately after electroporation, add 1mL of SOC media.
 14.Incubate at at 30 degree after electroporation.

===QuikChange===
 *Only use QuikChange I or QuikChange II
 *[http://stratagene.com/manuals/200523.pdf QuikChange II Protocol (use Ultra poly)]
 *[http://www.stratagene.com/manuals/200518.pdf QuikChange I Protocol (use Turbo poly)]
 *Things you'll need (can get it from TK)<br>
 '''Kit includes:'''
 1. PfuUltra High-Fidelity DNA poly (25 U for 10 rxns)
 2. 10X rxn buffer
 3. Dpn I RE (100 U)
 4. control primer 1 (750 ng)
 5. control primer 2 (750 ng)
 6. pWhitescript (50 ng)
 7. dNTP mix (10 microlitters)  use straight from TK’s LAB
 8. XL1-Blue cells (3 x 200 microlitters) 
 9. pUC18 (10 microlitters)<br>
 '''Note:''' XL1-Blue needs to be stored in -80; the rest are stored in -20.
 XL1-Blue and dNTP need to be kept in freezer prior to usage because
 Transfering tubes from one freezer to another decreases efficiency.
 '''Other things:''''<br>
 1. 14ml Falcon polypropylene tybes
 2. X-gal in DMF
 3. IPTG in dwater
 4. LB Amp X-gal IPTG plates
 5. FecA in BB plasmid
 6. FecR in BB plasmid
 7. mutagenesis primers for FecA
 8. utagenesis primers for FecR
 9. NZY+ broth (can be replaced by LB or superbroth)

===Sequencing=== 

 -In order to be sure we have what we think we have, we need to sequence our constructs.
 -This is done by the [http://web.mit.edu/biopolymers/www/dna_sequencing.html Biopolymers Lab]
 
 1.[[../Fill out/]] the sequencing request form
 2.For each construct that is to be sequenced, you will need two tubes: 
     One with forward primer, one with reverse
   For longer constructs (>1.5 kb) you need internal primers. 
     You can get ~800 bp of sequence info from a single primer
   Make up one reaction mixture <b>per primer:</b>
   100-200 ng of DNA (prepped ds plasmid DNA)
   3.2 pmoles of sequencing primer (1 ul of 20uM standard stock primer)
   ------
   final volume of 12 ul, in sterile DI water. Use tiny PCR tubes. 
   ------
   Label tubes with the number on the order form that corresponds to the sequence ID.
 3.Place all tubes in a large 50 ml falcon tube; label the side of the large falcon tube with your name, 
   the date, and Endy Lab.
 -Deliver tube and 1 of the 3 copies of the form to the Biopolymers Lab, E17-415. 
 -Place tube in brown fridge near front of room, and request form in the receptacle on the fridge. 
 -Keep one copy of the form for your records, and be sure to place the third copy in the 
  "Completed" file in the Endy lab. 

 notes: 
  1. you can get the concentration of your miniprep from nanodropping and use this to get the correct amt of DNA
  2. be sure to follow the instructions in the miniprep protocol that are required for direct sequencing.
  
===M9 Media===
Materials
*2X M9 (from the kitchen: this contains sodium phosphate, potassium phosphate, sodium chloride, and ammonium chloride)
*0.1 M MgSO4 (from kitchen)
*18% glucose (TK's lab)
*L-arginine powder
*1M CaCl2 solution (TK's lab)
*Sterile H2O

To make 400 ml of m9, add:
*200 ml of Fx m9
*4 ml of 0.1 M MgSo4
*4.44 ml of 18% glucose
*16 mg of L-arginine
*40 ul 1M CaCl2 solution
*135.56 ml sterlie H2O

-------------------

==Notes from Lab Bootcamp==
#[[../Monday, June 6th/]] (Pipetman use, Working with bacteria, PCR)
#[[../Tuesday, June 7th/]] (Miniprepping DNA, Agarose gels, RE digests)
#[[../Wednesday, June 8th/]] (Excising DNA from gels, purifying DNA from gels)
#[[../Thursday, June 9th/]] (Ligations, Transformations)