==Sequencing==
Submitted samples for sequencing; order #2852.  Samples were the BSMT+B0015 and SAMT+B0015.

==Digests of BSMT.B0015s, SAMT.B0015s, B0030, and B0032==

'''SAMT A + C + BSMT C + D'''

35.5 uL H20

5 uL NEB Buffer 2

.5 uL BSA

8 uL DNA (100-150 ng/uL) 

.5 uL EcoRI

.5 uL XbaI


'''SAMT D + BSMT A + B'''

27.5 uL H20

5 uL NEB Buffer 2

.5 uL BSA

16 uL DNA (50-100 ng/uL)

.5 uL EcoRI

.5 uL XbaI

'''SAMT B'''

37.5 uL H20

5 uL NEB Buffer 2

.5 uL BSA

6 uL DNA (158.4 ng/uL)

.5 uL EcoRI

.5 uL XbaI

'''B0030'''

33.5 uL H20

5 uL NEB Buffer 2

.5 uL BSA

10 uL DNA (87.5 ng/uL)

.5 uL EcoRI

.5 uL SpeI

'''B0032'''

22.5 uL H20

5 uL NEB Buffer 2

.5 uL BSA

21 uL DNA (38.0 ng/uL) 

.5 uL EcoRI

.5 uL SpeI

==PCR Cleanups==

Were done on the above digests.

==Ligation==

Each Enzyme+B.0015 part was ligated with both B0030 and B0032.  Here was the reaction mix:

1.5 uL H20 

3.5 uL RBS (15-25 ng/uL) 

3.5 uL Enzyme+B.0015 (15-25 ng/uL)

1 uL T4 DNA Ligase Buffer

.5 uL T4 DNA Ligase


These mixes were then transformed into top 10 cells.  However, we did not plate the BSMT mixes since when run on the gel, no bands were present.  The gel results showed that SAMT A, B, and D were of the wrong length, most likely since they were grown on AMP plates instead of AMP + KAN plates.  Note that the terminator's backbone (the desired ligated product's backbone) was 1AK3, while the coding region's original backbone was 1A3.  Thus, we believe that SAMT A, B, and D contained the coding region and the terminator in the wrong orientation.

==Transformations==
#Digested ATF1 and transformed into Top10 competent cells
#Transformed BAMT and BSMT into indole-knockout competent cells
#*ATF1->Top10
#*Control (Puc19)->Top10
#*BAMT->IK
#*SAMT->IK
#*Control->IK

==pchBA: designing primers==
*Genbank gene ID for pchA: 881821
*Genbank gene ID for pchB: 881846