<h1 style="color: #AAAAAA">'''[[User:Luw134|Lucien Weiss]] Lab Notebook'''</h1>
*Penn State iGEM 2005-2007
**[[http://parts.mit.edu/wiki/index.php/Penn_State_2005 PSU iGEM 2005]]
**[[http://openwetware.org/wiki/IGEM:PennState/2006 PSU iGEM 2006]]
**[[http://openwetware.org/wiki/IGEM:PennState/2007 PSU iGEM 2007]]
*Penn State Undergraduate
*Email [mailto:luw134 luw134 AT psu DOT edu]
*Phone (814) 880 5909

<h2>'''March 11, 2007: Motility Assay'''</h2>
'''Experiment Aims'''

Test motility on a variety of surface hardnesses in order to determine optimal level to use for assays.

'''Procedure'''

Eiken Agar Plates were made using a modified version of general procedure [[IGEM:PennState/2006/Eikenplates|1]].  Increased agar concentrations were made to increase hardness of gel and reduce the speed of motility.

''.20% Eiken Agar Plates:''
*.25g Bactotryptone
*.20g NaCl
*.15g sodium citrate
*.05g Eiken Agar (for 20EikenAgar)
''.30% Eiken Agar Plates:''
*.25g Bactotryptone
*.20g NaCl
*.15g sodium citrate
*.075g Eiken Agar (for 30EikenAgar)
''.40% Eiken Agar Plates:''
*.25g Bactotryptone
*.20g NaCl
*.15g sodium citrate
*.10g Eiken Agar (for 40EikenAgar


#Constructs were transformed into RP3087
#*YFP Sender Cells-Sender (LVA+)-RP3087 w/ [I14032+B0030+C0161 A3]+E0430
#*Sender Cells-Sender-RP3087 w/ I14032+B0030+C0161 (B2)
#*Receiver-RFP+Receiver-RP3087 w/ I13321+[J09855+B0033+J09274]
#Grown overnight at 37 degrees C on 50 micromolar Kan LB plates
#3mL cultures inoculated off of the plate and grown at 30 degrees C until OD590~.45
#3uL placed of receiver cells placed in various distances from 3uL inoculated sender colony.

'''Results:'''

<table>
 <tr>
  <td>'''.20% Eiken Agar Sender and Receivers'''</td>
 </tr>
 <tr>
  <td>[[Image:LucMotilityAssay3-12 20EikenAgar Sender+Receiver 8hours.JPG|thumb|.30% Eiken Agar with sender and receiver cells after 8 hours]]</td>
 </tr>
 <tr>
  <td>'''.30% Eiken Agar Sender and Receivers'''</td>
 </tr>
 <tr>
  <td>[[Image:30EikenAgar Sender+Receiver 8hours.JPG|thumb|.30% Eiken Agar with sender and receiver cells after 8 hours]]</td>
  <td>[[Image:30EikenAgar Sender+Receiver 20hours.JPG|thumb|.30% Eiken Agar with regular sender and receiver cells after 20 hours]]</td>
 <tr>
  <td>'''.30% Eiken Agar with Degradation Tagged AHL'''</td>
 </tr>
 <tr>
  <td>[[Image:30EikenAgar YFPSender+Receiver 8hours.JPG|thumb|.30% Eiken Agar with sender produced AHL tagged for degradation and receiver cells after 8 hours]]</td>
  <td>[[Image:30EikenAgar YFPSender+Receiver 20Hours.JPG|thumb|.30% Eiken Agar with sender cells producing AHL tagged for degradation and receiver cells after 20 hours]]</td>
 <tr>
  <td>'''.40% Eiken Agar Sender and Receivers'''</td>
 </tr>
 <tr>
  <td>[[Image:40EikenAgar YFPSender+Receiver 8hours.JPG|thumb|.40% Eiken Agar with sender produced AHL tagged for degradation after 8 hours]] </td>
  <td>[[Image:40EikenAgar Sender+Receiver 8hours.JPG|thumb|.40% Eiken Agar with regular sender and receiver cells after 8 hours]]</td>
 </tr>
</table>


'''Observations:'''

.20% agar plate difficult to handle as agar was extremely fragile.  Results were diminished by heavy diffusion.
.30% agar results in fair motility and is much easier to handle
.40% agar results in no motility

'''Conclusions:'''


'''Future Experiments:'''

Retry .20% Agar experiments with set distances away from sender
Experiment with temperature changes

<h2>'''March 13, 2007: Motility Assay'''</h2>
'''Experiment Aims'''

Test induced motility at a variety of distances from sender cells.

'''Procedure'''

Eiken Agar Plates were made using a modified version of general procedure [[IGEM:PennState/2006/Eikenplates|1]].  Agar concentrations were kept at .20%

''.20% Eiken Agar Plates:''
*.25g Bactotryptone
*.20g NaCl
*.15g sodium citrate
*.05g Eiken Agar (for 20EikenAgar)


#Constructs were transformed into RP3087
#*YFP Sender Cells-Sender (LVA+)-RP3087 w/ [I14032+B0030+C0161 A3]+E0430
#*Sender Cells-Sender-RP3087 w/ I14032+B0030+C0161 (B2)
#*Receiver-RFP+Receiver-RP3087 w/ I13321+[J09855+B0033+J09274]
#Grown overnight at 37 degrees C on 50 micromolar Kan LB plates
#3mL cultures inoculated off of the plate and grown at 30 degrees C until OD590~.45
#3uL placed of receiver cells placed in various distances from 3uL inoculated sender colony.

'''Results:'''

<table>
 <tr>
  <td>'''.20% Eiken Agar Sender and Receivers'''</td>
 </tr>
 <tr>
  <td>[[Image:LucMotilityAssay3-13 20EikenAgar Sender+Receiver 1 8hours.JPG|thumb|.20% Eiken Agar with sender and receiver cells after 8 hours]]</td>
  <td>[[Image:LucMotilityAssay3-13 20EikenAgar YFPSender+Receiver 1 8hours.JPG|thumb|.20% Eiken Agar with sender and receiver cells after 8 hours]]</td>
 </tr>
</table>


'''Observations:'''

Difficult to discern motility on .20% Eiken Agar plates

'''Conclusions:'''


'''Future Experiments:'''
Increase Agar Concentration
Experiment with temperature changes

<h2>'''March 16, 2007: Motility Assay'''</h2>

<h2>'''March 29-April 1, 2007: Institute of Biological Engineering'''</h2>
<table>
 <tr>
  <td>'''Poster: Bacterial Relay Race'''</td>
 </tr>
 <tr>
  <td>[[Image:IBE2007MeetingPennStateBacterialRelayRace.jpg|thumb|IBE Poster]]</td>
 </tr>
 <tr>
  <td></td>
 </tr>
 <tr>
  <td></td>
 </tr>
</table>

<h2>'''April 22, 2007: Project Proposal Presentations'''</h2>
<table|left>
 <tr>
  <td>'''Project Proposal: Pressure Sensing in Bacteria'''</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007ProjectProposalPressureSensingSlide1.JPG|thumb|Slide1]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalPressureSensingSlide2.JPG|thumb|Slide2]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalPressureSensingSlide3.JPG|thumb|Slide3]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007ProjectProposalPressureSensingSlide4.JPG|thumb|Slide4]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalPressureSensingSlide5.JPG|thumb|Slide5]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalPressureSensingSlide6.JPG|thumb|Slide6]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007ProjectProposalPressureSensingSlide7.JPG|thumb|Slide7]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalPressureSensingSlide8.JPG|thumb|Slide8]]</td>
 </tr>
</table>

<table|left>
 <tr>
  <td>'''Project Proposal: Bi and Sept Stable Oscillator'''</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007ProjectProposalBiSeptStableOscillatorSlide1.JPG|thumb|Slide1]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalBiSeptStableOscillatorSlide2.JPG|thumb|Slide2]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalBiSeptStableOscillatorSlide3.JPG|thumb|Slide3]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007ProjectProposalBiSeptStableOscillatorSlide4.JPG|thumb|Slide4]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalBiSeptStableOscillatorSlide5.JPG|thumb|Slide5]]</td>
  <td>[[Image:PSUiGEM2007ProjectProposalBiSeptStableOscillatorSlide6.JPG|thumb|Slide6]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007ProjectProposalBiSeptStableOscillatorSlide7.JPG|thumb|Slide7]]</td>
 </tr>
</table>

<h2>'''April 28, 2007: Synthetic Biology 3.0'''</h2>
[http://www.syntheticbiology3.ethz.ch/index.htm|Synthetic Biology 3.0 Website]

<h2>'''May 15, 2007: General Procedures'''</h2>
These are all the general cloning kinds of things that will be used to build our constructs.  We will probably get into getting parts sequenced if things do not seem to be working, but that gets expensive and takes a couple of days, so my thoughts are we should save that until it looks like we are getting into trouble.  Labeling is key.

'''Labeling'''
#First note this is the most important thing for how easy it is.  In the past we have had a lot of problems however so I have come up with a system.  Remember everyone is going to need to be able to read this, so use a fine tipped sharpie always.  Also EtOH is an organic solvent and will remove your title.  Be careful with this especially during minipreps.  Write on the outside of the tube, Thats really obvious it would seem, but just a warning.

Microcentrifuge tubes
On Cap:
<table>
 <tr>
  <td> Row 1: Initials Date</td>
 </tr>
 <tr>
  <td> Row 2: Part/parts (this is a quick reference, always know what it is that you are using) </td>
 </tr>
 <tr>
  <td> Row 3: ID number 0001-9999 (each member will have a list of every tube they have online)
*[[IGEM:PennState/2007/Microcentrifuge_tube_lists]] </td>
 </tr>
</table>
On Side:
you may want to duplicate all the information especiall if it is very messy or you needed more room.

EX-on Cap:
<table>
 <tr>
  <td> LW 5-19-07</td>
 </tr>
 <tr>
  <td> B33+J2971 </td>
 </tr>
 <tr>
  <td> 0012</td>
 </tr>
</table>
On Side:
<table>
 <tr>
  <td> LW 5-19-07</td>
 </tr>
 <tr>
  <td> B0033+J02971 miniprep </td>
 </tr>
 <tr>
  <td> Row 3: 0012</td>
 </tr>
</table>

'''Innoculation from plate'''
#flame loop
#cool loop on plate
#pick up individual colony
#place in culture and shake loop
#flame loop

'''Culture Dilutions'''
EX 1-1:100 Dilution
#Add 3mL fresh LB to a new culture tube
#Add 3uL of appropriate (1:1000)antibiotic solution
#Take 30uL from starter culture and place in fresh media.
#*Becareful to not touch the sides of the tube with the pipetmen.

'''Checking cell OD. Using Spectrophotometer'''
#Turn on spectrophotometer (switch on the back to the right)
#Allow the startup to run, and then turn off deuterium Light (needed at hv <300nm)
#Turn wavelength to 590nm
#Zero with 1mL LB in Cuvette
#Do an appropriate dilution (suggested 1:5, 1:10) of culture with fresh LB into a fresh Cuvette
#*note 1: if you are trying to find out how long it will be until cells are at OD .6 (mid log) you can take the OD at two seperate times and use http://molbiol.edu.ru/eng/scripts/01_02.html.
#*note 2: diluting a culture down to OD .6 will not put it in mid log phase; dilute down to .15 and allow it to grow back up to .6 if your culture has grown too long.
note: wavelength for DNA = 260nm, wavelength for protein=280nm, to find purity take ratio between peaks.

'''Miniprep'''
#Spin down 3mL of saturated cell culture, remove supernatant
#Resuspend pelleted bacteria in 250uL Buffer P1 and transfer to a 1.5mL micro centrifuge tube.
#Add 250uL Buffer P2 and invert the tube gently 4-6 times to mix.
#Add 350uL Buffer N3 and invert the tube immediately but gently 4-6 times.
#*the solution should be come cloudy
#Centrifuge for 10 minutes at 13,000 rpm in a table-top microcentrifuge
#*allign tubes such that the cap clip is facing outward
#decant supernatant into spin column
#Centrifuge for 60s. Discard flow-through.
#Add .75mL Buffer PE and centifuge for 60s.
#Discard flowthrough and centrifuge for 60s
#Place the spin column into a new 1.5mL microcentrifuge tube.  
#Add 30-50uL of water (8.5>pH>7) to the center of each spin column.
#Let tubes stand vertically for 1min, then centrifuge for 1min.

'''Digest'''
#Add components generally by largest volume to smallest.  I tend to add water first just so I do not waste pipettips if I am using different vectors, but cetainly do not start with .5 or 1 uL components.
#Choosing your Enzymes: The biobricks are built with the restriction sites as following: -EcoR1-Xba1-Part-Spe1-Pst1- it is important to remember that your plasmid may have some of these restriction sites too, so if too many bands are appearing on the gel becareful of this.  Keep these enzymes on ice when they are out of the feezer, and put them back immediately after you use them.  If enzymes happen to be left out please let me know and we will decide what to do with them.  Do not put enzymes that have been left out back in the freezer, or future restrictions will not work.
#Choosing your buffer: Check the compatibility with each enzyme and the buffer. There should be a list for double digestions.
#Forgetting to add BSA will make the whole thing not work.

*Examples demonstrate the two main digets that will be used in this lab.
EX1-10uL Digest
#3.0uL H2O
#4.0uL Vector
#1.0uL Buffer X
#1.0uL BSA
#0.5uL Enzyme 1
#0.5uL Enzyme 2

EX2-20uL Digest
#4.0uL H2O
#11.uL Vector
#2.0uL Buffer X
#2.0uL BSA
#0.5uL Enzyme 1
#0.5uL Enzyme 2

'''Running a Gel'''
TAE Buffer-if you are extracting from your gel
TBE Buffer-for better imaging 
Do not load DNA in high concentrations.  It may be tempting to put in as much concentrated DNA as you can, but this is not a good thing.

'''Gel Extraction'''


'''Ligation'''

'''Transformation'''

<h2>'''May 21, 2007: Summer Schedule and Restriction Enzymes'''</h2>
<table|left>
 <tr>
  <td>'''Summer Schedule and Week's Plan'''</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007May21Slide1.JPG|thumb|Slide1]]</td>
  <td>[[Image:PSUiGEM2007May21Slide2.JPG|thumb|Slide2]]</td>
  <td>[[Image:PSUiGEM2007May21Slide3.JPG|thumb|Slide3]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007May21Slide4.JPG|thumb|Slide4]]</td>
  <td>[[Image:PSUiGEM2007May21Slide5.JPG|thumb|Slide5]]</td>
  <td>[[Image:PSUiGEM2007May21Slide6.JPG|thumb|Slide6]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007May21Slide7.JPG|thumb|Slide7]]</td>
  <td>[[Image:PSUiGEM2007May21Slide8.JPG|thumb|Slide8]]</td>
  <td>[[Image:PSUiGEM2007May21Slide9.JPG|thumb|Slide9]]</td>
 </tr>
</table>

'''Restriction Enzymes'''
<table>
 <tr>
  <td>|---------EcoRI--------|</td>
  <td>|---------XbaI---------|</td>
  <td>|---------PART---------|</td>
  <td>|---------SpeI---------|</td>
  <td>|---------PstI---------|</td>
 </tr>
 <tr>
  <td>5' G|AATT C 3'</td>
  <td>5' T|CTAG A 3'</td>
  <td></td>
  <td>5' A|CTAG T 3'</td>
  <td>5' C TGCA|G 3'</td>
 </tr>
 <tr>
  <td>3' C TTAA|G 5'</td>
  <td>3' A GATC|T 5'</td>
  <td></td>
  <td>3' T GATC|A 5'</td>
  <td>3' G|ACGT C 5'</td>
 </tr>
</table>

<h2>'''May 22, 2007: Growing Parts From Registry Plates'''</h2>
Registry plates are kept in the -4 degree freezer in a box labeled registry plates.
#Take a 10uL pipetment and pipet 10uL of dd-water up and down inside target well on the registry plate
#Once the water appears red place that in a labeled microcentrifuge tube (Initials Date, Part name well #, and ID number). 
#*Now the registry part is ready to be transformed.  Proceed with regular transformation.

<h2>'''May 24, 2007: Gel Results: J32038 & J31004 Digests'''</h2>

'''Gel 1 Results: J32038 & J31004 Digests'''
[http://openwetware.org/wiki/IGEM:PennState/Labbook/NoahJohnson/2007-5-24|From Digests in Noah Johnsons Labnotebook 5-24-07 Restriction Digest]
[http://openwetware.org/wiki/IGEM:PennState/Labbook/GarrettTobin#5-24-07|Extracted DNA]

<table>
 <tr>
  <td>EcoRI+SpeI J31004 A</td>
  <td>EcoRI+SpeI J31004 B</td>
  <td>EcoRI+SpeI J31004 C</td>
  <td>EcoRI+XbaI J32038 A</td>
  <td>EcoRI+XbaI J32038 B</td>
  <td>EcoRI+XbaI J32038 C</td>
  <td>EcoRI J31004 A</td>
  <td>XbaI J31004 A</td>
  <td>SpeI J31004 A</td>
  <td>J31004 A</td>
 </tr>
 <tr>
  <td>LW0003</td>
  <td>LW0004</td>
  <td>LW0005</td>
  <td>LW0006</td>
  <td>LW0007</td>
  <td>LW0008</td>
  <td>LW0003</td>
  <td>LW0003</td>
  <td>LW0003</td>
  <td>LW0003</td>
 </tr>
 <tr>
  <td>No bands</td>
  <td>No bands</td>
  <td>No bands</td>
  <td>No bands</td>
  <td>4 bands</td>
  <td>2 bands</td>
  <td>1 band</td>
  <td>1 band</td>
  <td>1 band</td>
  <td>no bands</td>
 </tr>
 <tr>
  <td>Gel Leaked</td>
  <td>Gel Leaked</td>
  <td>Gel Leaked</td>
  <td>Gel Leaked</td>
  <td>Incomp Digestion</td>
  <td>Success</td>
  <td>Success</td>
  <td>Success</td>
  <td>Success</td>
  <td>Gel Leaked</td>
 </tr>
 <tr>
  <td>Wrong Buff</td>
  <td>Wrong Buff</td>
  <td>Wrong Buff</td>
  <td></td>
  <td></td>
  <td></td>
  <td></td>
  <td></td>
  <td></td>
  <td></td>
 </tr>
 <tr>
  <td></td>
  <td></td>
  <td></td>
  <td></td>
  <td></td>
  <td>600bp Extracted</td>
  <td></td>
  <td>3kbp Extracted</td>
  <td>3kbp Extracted</td>
  <td></td>
 </tr>
</table>

'''Gel 2 Results: J32038 & J31004 Digests'''
[http://openwetware.org/wiki/IGEM:PennState/Labbook/NoahJohsnon#5-24-07|From Digests in Noah Johnsons Labnotebook 5-24-07 Restriction Digest]

<table>
 <tr>
  <td>EcoRI+XbaI J32038 A</td>
  <td>EcoRI+XbaI J32038 B</td>
  <td>EcoRI+XbaI J32038 C</td>
  <td>EcoRI+SpeI J31004 A</td>
  <td>EcoRI+SpeI J31004 B</td>
  <td>EcoRI+SpeI J31004 C</td>
  <td>J32038 C</td>
  <td>J31004 C</td>
 </tr>
 <tr>
  <td>LW0006</td>
  <td>LW0007</td>
  <td>LW0008</td>
  <td>LW0003</td>
  <td>LW0004</td>
  <td>LW0005</td>
  <td>LW0008</td>
  <td>LW0005</td>
 </tr>
 <tr>
  <td>1 band</td>
  <td>1 band</td>
  <td>1 band</td>
  <td>2 bands</td>
  <td>2 bands</td>
  <td>2 bands</td>
  <td>4 bands</td>
  <td>4 bands</td>
 </tr>
 <tr>
  <td>Success</td>
  <td>Success</td>
  <td>Success</td>
  <td>Success</td>
  <td>Success</td>
  <td>Success</td>
  <td>Success</td>
  <td>Success</td>
 </tr>
 <tr>
  <td>3kbp Extracted</td>
  <td>3kbp Extracted</td>
  <td>3kbp Extracted</td>
  <td>600bp Extracted</td>
  <td>600bp Extracted</td>
  <td>600bp Extracted</td>
  <td></td>
  <td></td>
 </tr>
</table>

<h2>'''June 4, 2007: Bacterial Relay Race Plan'''</h2>
<table|left>
 <tr>
  <td>'''Bacterial Relay Race'''</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007May30BacterialRelaySlide1.JPG|thumb|Slide1]]</td>
  <td>[[Image:PSUiGEM2007May30BacterialRelaySlide2.JPG|thumb|Slide2]]</td>
  <td>[[Image:PSUiGEM2007May30BacterialRelaySlide3.JPG|thumb|Slide3]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM2007May30BacterialRelaySlide4.JPG|thumb|Slide4]]</td>
  <td>[[Image:PSUiGEM2007May30BacterialRelaySlide5.JPG|thumb|Slide5]]</td>
  <td>[[Image:PSUiGEM2007May30BacterialRelaySlide6.JPG|thumb|Slide6]]</td>
 </tr>
</table>

<h2>'''June 08, 2007: Bacterial Relay Race Plan'''</h2>

#Receiver Cells:
#*BBa_I14032 LacIQ constit Promoter
#*BBa_B0034 RBS
#*BBa_C0062 LuxR (to bind to HSL)
#*BBa_B0010 Terminator
#*BBa_B0012 Terminator
#*BBa_R0062 PluxR HSL+LuxR activates
#*BBa_B0033 Weak RBS
#*BBa_J09271 motB without RBS
#*BBa_J52008 Rluc Luciferase
#*BBa_B0010 Terminator
#*BBa_B0012 Terminator
#Built so far and in registry:
#*BBa_I14032 LacIQ constit Promoter
#*BBa_B0034 RBS
#*BBa_C0062 LuxR (to bind to HSL)
#*BBa_B0010 Terminator
#*BBa_B0012 Terminator
#*BBa_R0062 PluxR HSL+LuxR activates
#*=BBa_J09855
#**24G iGEM 2007 Parts Kit Plate 3 pSB1A2 V1001

#Cloning Plan Step 1:
#*Prefix Insertion BBa_J52008 Rluc Luciferase before BBa_B0015 (BBa_B0010 Terminator + BBa_B0012 Terminator)
#**BBa_J52008 12E iGEM 2007 Parts Kit Plate 3 pSB1AK3 V1001
#**BBa_B0015 1I iGEM 2007 Parts Kit Plate 1 pSB1AK3 V1001 
#**BBa_B0015 3O iGEM 2007 Parts Kit Plate 3 pSB1AK3 V1010 
#*Suffix Insertion BBa_J09271 motB without RBS after BBa_B0033 Weak RBS
#**24G iGEM 2007 Parts Kit Plate 3 pSB1A2 V1001 
#**3M iGEM 2007 Parts Kit Plate 1 pSB1A2 V1004 
#Cloning Plan Step 2:
#*Prefix Insertion BBa_J09271+BBa_B0033 before BBa_J52008+BBa+B0010+BBa_B0012
#*Suffix Insertion BBa_J52008+BBa+B0010+BBa_B0012 after BBa_J09271+BBa_B0033
#*(use whichever works)
#Cloning Plan Step 3:
#*Suffix Insertion: BBa_J09271+BBa_B0033+BBa_J52008+BBa+B0010+BBa_B0012 after BBa_J09855
#*Prefix Insertion: BBa_J09855 before BBa_J09271+BBa_B0033+BBa_J52008+BBa+B0010+BBa_B0012


<h2>'''June 10, 2007: Promoters in progress'''</h2>
Full Sequence ecocyc 3728788->3729154
[http://parts.mit.edu/r/parts/partsdb/part_info.cgi?part_name=BBa_G00000]
[http://parts.mit.edu/r/parts/partsdb/part_info.cgi?part_name=BBa_G00001]
[http://www.mcb.uct.ac.za/pcroptim.htm]

Forward Facing Promoter Region between XylA and XylF
<html><pre>TATTGAACTC CATAATCAGG TAATGCCGCG GGTGATGGAT GATGTCGTAA TATTGGGCAC
TCCCTTTCAG TTGCTCAATT ATGTTATTTC ACACTGCTAT TGAGATAATT CACAAGTGTG
CGCTCGCTCG CAAAATAAAA TGGAATGATG AAACTGGGTA ATTCCTCGAA GAGAAAAATG
CAATAAGTAC AATTGCGCAA CAAAAGTAAG ATCTCGGTCA TAAATCAAGA AATAAACCAA
AAATCGTAAT CGAAAGATAA AAATCTGTAA TTGTTTTCCC CTGTTTAGTT GCTAAAAATT
GGTTACGTTT ATCGCGGTGA TTGTTACTTA TTAAAACTGT CCTCTAACTA CAGAAGGCCC
TACACCA</pre></html>

Coding Forward Primers-
F - gaattcgcggccgcttctagag TATTGAACTC CATAATCAGG TAATG
* %GC34, TmC 61 Dimers:AAT TTA Stability .5 Runs: 2 Repeats: none, Hairpins none.

R - ctgcagcggccgctactagta TGGTGTA GGGCCTTCTG TAG 
* %GC 55, TMC 62, Dimers: 2bp Stability: 3.0 Runs GGG repeats:2, airpins: none

Reverse Complement Facing Promoter Region between XylA and XylF
<html><pre>TGGTGTAGGG CCTTCTGTAG TTAGAGGACA GTTTTAATAA GTAACAATCA CCGCGATAAA
CGTAACCAAT TTTTAGCAAC TAAACAGGGG AAAACAATTA CAGATTTTTA TCTTTCGATT
ACGATTTTTG GTTTATTTCT TGATTTATGA CCGAGATCTT ACTTTTGTTG CGCAATTGTA
CTTATTGCAT TTTTCTCTTC GAGGAATTAC CCAGTTTCAT CATTCCATTT TATTTTGCGA
GCGAGCGCAC ACTTGTGAAT TATCTCAATA GCAGTGTGAA ATAACATAAT TGAGCAACTG
AAAGGGAGTG CCCAATATTA CGACATCATC CATCACCCGC GGCATTACCT GATTATGGAG
TTCAATA</pre></html>

Coding Left Facing Primers-
   F - gaattcgcggccgcttctagag TGGTGTAGGG CCTTCTGTA 
*TmC 61
   R - ctgcagcggccgctactagta TATTGAACTCCATAATCAGGTAATG
*TmC 58

Forward Facing ORF Promoter Region between XylA and XylF
<html><pre><br>                                     ATGGAT GATGTCGTAA TATTGGGCAC
TCCCTTTCAG TTGCTCAATT ATGTTATTTC ACACTGCTAT TGAGATAATT CACAAGTGTG
CGCTCGCTCG CAAAATAAAA TGGAATGATG AAACTGGGTA ATTCCTCGAA GAGAAAAATG
CAATAAGTAC AATTGCGCAA CAAAAGTAAG ATCTCGGTCA TAAATCAAGA AATAAACCAA
AAATCGTAAT CGAAAGATAA AAATCTGTAA TTGTTTTCCC CTGTTTAGTT GCTAAAAATT
GGTT</pre></html>

Coding Forward ORF Primers-
   F - gaattcgcggccgcttctagag ATGGAT GATGTCGTAA TA
*&GC 33, TmC: 51, Dimers:OK Stability 3.2 Runs OK Hairpin: OK
   R - ctgcagcggccgctactagta AACC AATTTTTAGC AACTAA
*%GC 25, TmC: 53, Dimers: CTAA, Stability 1.8, Runs TTTT, Hairpin: ok

Reverse Complement Facing ORF Promoter Region between XylA and XylF
<html><pre>AACCAAT TTTTAGCAAC TAAACAGGGG AAAACAATTA CAGATTTTTA TCTTTCGATT
ACGATTTTTG GTTTATTTCT TGATTTATGA CCGAGATCTT ACTTTTGTTG CGCAATTGTA
CTTATTGCAT TTTTCTCTTC GAGGAATTAC CCAGTTTCAT CATTCCATTT TATTTTGCGA
GCGAGCGCAC ACTTGTGAAT TATCTCAATA GCAGTGTGAA ATAACATAAT TGAGCAACTG
AAAGGGAGTG CCCAATATTA CGACATCATC CAT</pre></html>

Left Facing ORF Promoters
   F - gaattcgcggccgcttctagag AACCAATTTTTAGCAACTAAACAG
*TmC 58
   R - ctgcagcggccgctactagta ATGGATGATGTCGTAATATTGG
*TmC 59

crp ORF
http://biocyc.org/ECOLI/sequence?type=GENE&object=EG10164

Ggaattcgcggccgcttctagag ATGGTGCTTGGCAAACC left side
*TmC 60

actagtagcggccgctgcag
ctgcagcggccgctactagta

ctgcagcggccgctactagta TTAACGAGTGCCGTAAAC right side
*TmC 57

<h2>'''June 11, 2007: Planning Stuff and more primers'''</h2>
<table|left>
 <tr>
  <td>'''6-11 Weekly Plan'''</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM07weeklyplan611Slide1.JPG|thumb|Slide1]]</td>
  <td>[[Image:PSUiGEM07weeklyplan611Slide2.JPG|thumb|Slide2]]</td>
  <td>[[Image:PSUiGEM07weeklyplan611Slide3.JPG|thumb|Slide3]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM07weeklyplan611Slide4.JPG|thumb|Slide4]]</td>
  <td>[[Image:PSUiGEM07weeklyplan611Slide5.JPG|thumb|Slide5]]</td>
 </tr>
</table>

'''primers'''
*XylR XylR Fis Sequence Ecocyc 3729090-3729153 :
*TTACGTTTAT CGCGGTGATT GTTACTTATT AAAACTGTCC TCTAACTACA GAAGGCCCTACACC
*Reverse Complementary sequence:
*TGGTGTAGGG CCTTCTGTAG TTAGAGGACA GTTTTAATAA GTAACAATCA CCGCGATAAACGTAA

*Forward Primer gaattcgcggccgcttctagag TTACGTTTATCGCGGTGATT
*approved by SECentral TmC 59
*Reverse Primer ctgcagcggccgctactagta GGTGTAGGGCCTTCTGTAG
*approved by SECentral TmC 60

*CRP XylR XylR Fis Sequence Ecocyc 3728848-3729153
*TCCCTTTCAG TTGCTCAATT ATGTTATTTC ACACTGCTAT TGAGATAATT CACAAGTGTG
*CGCTCGCTCG CAAAATAAAA TGGAATGATG AAACTGGGTA ATTCCTCGAA GAGAAAAATG
*CAATAAGTAC AATTGCGCAA CAAAAGTAAG ATCTCGGTCA TAAATCAAGA AATAAACCAA
*AAATCGTAAT CGAAAGATAA AAATCTGTAA TTGTTTTCCC CTGTTTAGTT GCTAAAAATT
*GGTTACGTTT ATCGCGGTGA TTGTTACTTA TTAAAACTGT CCTCTAACTA CAGAAGGCCC
*TACACC

*Forward Primer gaattcgcggccgcttctagag TCCCTTTCAGTTGCTCAAT
*approved by SECentral TmC 58
*Reverse Primer ctgcagcggccgctactagta GGTGTAGGGC CTTCTGTAG
*approved by SECentral TmC 60

*Project on hold:

*CRP XylR2 XylR1 xylAp (Promoter for xylAB)
*3728788-3729092
*(Reverse Complement to make promoter face forward)
*TAACCAATTT TTAGCAACTA AACAGGGGAA AACAATTACA GATTTTTATC TTTCGATTAC
*GATTTTTGGT TTATTTCTTG ATTTATGACC GAGATCTTAC TTTTGTTGCG CAATTGTACT
*TATTGCATTT TTCTCTTCGA GGAATTACCC AGTTTCATCA TTCCATTTTA TTTTGCGAGC
*GAGCGCACAC TTGTGAATTA TCTCAATAGC AGTGTGAAAT AACATAATTG AGCAACTGAA
*AGGGAGTGCC CAATATTACG ACATCATCCA TCACCCGCGG CATTACCTGA TTATGGAGTT
*CAATA

*Forward Primer: gaattcgcggccgcttctagag TAACCAATTT TTAGCAACTA 

*Reverse Primer: ctgcagcggccgctactagta AACTCCATAA TCAGGTAATG

<h2>'''June 13, 2007: Final Primers for Promoter Region'''</h2>
BB Forward gaattcgcggccgcttctagag


Creates A to F biobrick of Promoter Region
*Forward:59
*GGTGATGGATGATGTCGTAATA
*GAATTCGCGGCCGCTTCTAGAGGGTGATGGATGATGTCGTAATA(61,77)
*Reverse:60
*ATAAGTAACAATCACCGCGATA
*CTGCAGCGGCCGCTACTAGTAATAAGTAACAATCACCGCGATAA(60,76)

Creates F to A biobrick of Promoter Region
*Forward:59
*AGTAACAATCACCGCGATAA
*GAATTCGCGGCCGCTTCTAGAGTAAGTAACAATCACCGCGATAA(58,77)
*Reverse:59
*CGGGTGATGGATGATGTC
*CTGCAGCGGCCGCTACTAGTACGGGTGATGGATGATGTC(59,79)

Creates F biobrick of Half Promoter Missing CRP Binding Site
*Forward: 59
*TGGAATGATGAAACTGGGTAATTC
*GAATTCGCGGCCGCTTCTAGAGTGGAATGATGAAACTGGGTAATTC(60,77)
*Reverse:60
*ATAAGTAACAATCACCGCGATA
*CTGCAGCGGCCGCTACTAGTAATAAGTAACAATCACCGCGATAA(60,76)

Creates A biobrick of Half promoter  Missing CRP Binding Site
*Forward:
*GGAATTCGCGGCCGCTTCTAGAGTTGTGAATTATCTCAATAGCAGT(57,76)
*Reverse:59
*CGGGTGATGGATGATGTC
*CTGCAGCGGCCGCTACTAGTACGGGTGATGGATGATGTC(59,79)

Creates F biobrick half promoter including CRP Binding Site
*Forward:
*GGAATTCGCGGCCGCTTCTAGAGCACAAGTGTGCGCTC(57,80)
*Reverse:60
*ATAAGTAACAATCACCGCGATA
*CTGCAGCGGCCGCTACTAGTAATAAGTAACAATCACCGCGATAA(60,76)

Creates A biobrick half promoter including CRP Binding Site
*Forward:
*GAATTCGCGGCCGCTTCTAGAGACCCAGTTTCATCATTCCATTT(60,77)
*Reverse:59
*CGGGTGATGGATGATGTC
*CTGCAGCGGCCGCTACTAGTACGGGTGATGGATGATGTC(59,79)

<h2>'''June 18, 2007: Plan'''</h2>
This week
Put together a small group of parts:
RBS+GFP+TT

Prepare Competent Cells:
DH5a & Death Gene Resistant

Grow and isolate Death Gene Vector

Isolate Chromosomal DNA

Receive Primers
PCR Selected parts of the chromosome

Ligate with Death Gene Vector
Transform into DH5a

Put all constructs on computer (in parts page or software from ChE).  Full plan by next Monday.

PCR
**use ~100fmol DNA template-check prot.
Clean
Digest
Clean/("freeze & squeeze" filtration method)
Ligate

<h2>'''June 18, 2007: Plan'''</h2>
This week
Put together a small group of parts:
RBS+GFP+TT

Prepare Competent Cells:
DH5a & Death Gene Resistant

Grow and isolate Death Gene Vector

Isolate Chromosomal DNA

Receive Primers
PCR Selected parts of the chromosome

Ligate with Death Gene Vector
Transform into DH5a

Put all constructs on computer (in parts page or software from ChE).  Full plan by next Monday.

PCR
**use ~100fmol DNA template-check prot.
Clean
Digest
Clean/("freeze & squeeze" filtration method)
Ligate

<h2>'''June 19, 2007: mid week schedule'''</h2>
Plan for Monday
#Group A
#*Chromsomal DNA prep
#*PCR
#*store at 4C
#Group B
#*Innoculate E0040+B0015
#*Innoculate Death Gene in pSB1A3 (amp+strep)
#*Innoculate pSB1A3 (amp+strep)
#Group C
#*Form overall plan on openwetware
#Group D
#*Develop Lambda project
Plan for Thursday
#Group A
#*Digest PCR Products
#*Purify Gel
#Group B
#*Miniprep Death Gene in pSB1A3
#*Digest Death Gene in pSB1A3
#*Gel/Purify
#*Ligate with digested PCR products
#Group C
#*Miniprep pSB1A3 and Transform into Dh5a
#Group D
#*Miniprep E0040+B0015
#*Digest E0040+B0015
#*Digest B0030
#*Run Gel/Gel Purify
#*Ligate
#*Transform into DH5a


<h2>'''June 25, 2007: iGEM Group Meeting'''</h2>
<table|left>
 <tr>
  <td>'''6-11 Weekly Plan'''</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM07weeklyplan625Slide1.JPG|thumb|Slide1]]</td>
  <td>[[Image:PSUiGEM07weeklyplan625Slide2.JPG|thumb|Slide2]]</td>
  <td>[[Image:PSUiGEM07weeklyplan625Slide3.JPG|thumb|Slide3]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM07weeklyplan625Slide4.JPG|thumb|Slide1]]</td>
  <td>[[Image:PSUiGEM07weeklyplan625Slide5.JPG|thumb|Slide2]]</td>
  <td>[[Image:PSUiGEM07weeklyplan625Slide6.JPG|thumb|Slide3]]</td>
 </tr>
 <tr>
  <td>[[Image:PSUiGEM07weeklyplan625Slide7.JPG|thumb|Slide1]]</td>
  <td>[[Image:PSUiGEM07weeklyplan625Slide8.JPG|thumb|Slide2]]</td>
  <td>[[Image:PSUiGEM07weeklyplan625Slide9.JPG|thumb|Slide3]]</td>
 </tr>
</table>


<h2>'''June 27, 2007: Competent Cell Prep & P1010 Transformation'''</h2>
#Competent cells using Inoue method.
#Immediately after preperation cells were tested wtihout being frozen in liq N2 with Puc19, P1010 in pSB1A2, and a Neg Control.  270uL were plated
*The Neg control grew nothing
*Puc19 grew 30 colonies (low yield due to <.5 uL used to transform
*P1010 in pSB1A2 created many.
#PCR was done on BBa_I741015 and BBa_171017 using both Taq and PFx Polymerase
*Results were:
Taq=success
Pfx=no DNA found (except primers)

<h2>'''June 28, 2007: PCR on Promoter Regions'''</h2>
'''PCR Setup:'''
#10X Pfx Amplification Buffer 5 μl/10x Thermo Polymerase Buffer 5uL
#10 mM dNTP mixture* 1.5 μl
#50 mM MgSO4 1 μl
#Primer mix (10 μM each)* 1.5 μl
#Chromosomal DNA 2ul
#Platinum® Pfx DNA Polymerase 0.5 μl/TAq Polymerase .5uL
#Autoclaved, distilled water 38.5 μl

Reaction:
#+Pfx Primers 3398+3399 '''Part:BBa_I741015'''
#+Pfx Primers 3400+3401 '''Part:BBa_I741017'''
#+Pfx Primers 3406+3407 '''Part:BBa_I741018'''
#+Pfx Primers 3408+3409 '''Part:BBa_I741019'''
#+Pfx Primers 3402+3403 '''Part:BBa_I741020'''
#+Pfx Primers 3404+3405 '''Part:BBa_I741021'''
#+TAq Primers 3398+3399 ''Part:BBa_I741015''
#+TAq Primers 3400+3401 ''Part:BBa_I741017''
#+TAq Primers 3406+3407 ''Part:BBa_I741018''
#+TAq Primers 3408+3409 ''Part:BBa_I741019''
#+TAq Primers 3402+3403 ''Part:BBa_I741020''
#+TAq Primers 3404+3405 ''Part:BBa_I741021''
#-TAq Primers 3398+3399 ''Negative Control''
#-PFx Primers 3398+3399 ''Negative Control''

<h2>'''June 29, 2007: PCR using Pfx Thermal Gradient Test'''</h2>
'''PCR Setup:'''
#10X Pfx Amplification Buffer 5 μl/10x Thermo Polymerase Buffer 50uL
#10 mM dNTP mixture* 15 μl
#50 mM MgSO4 10 μl
#Primer mix (10 μM each)* 15 μl
#Chromosomal DNA 20ul
#Platinum® Pfx DNA Polymerase 6 μl/TAq Polymerase .6uL
#Autoclaved, distilled water 38.5 μl

Reaction:
#+Pfx Primers 3398+3399 '''Part:BBa_I741015'''
#+Pfx Primers 3400+3401 '''Part:BBa_I741017'''
#+Pfx Primers 3406+3407 '''Part:BBa_I741018'''
#+Pfx Primers 3408+3409 '''Part:BBa_I741019'''
#+Pfx Primers 3402+3403 '''Part:BBa_I741020'''
#+Pfx Primers 3404+3405 '''Part:BBa_I741021'''
#+TAq Primers 3398+3399 ''Part:BBa_I741015''
#+TAq Primers 3400+3401 ''Part:BBa_I741017''
#+TAq Primers 3406+3407 ''Part:BBa_I741018''
#+TAq Primers 3408+3409 ''Part:BBa_I741019''
#+TAq Primers 3402+3403 ''Part:BBa_I741020''
#+TAq Primers 3404+3405 ''Part:BBa_I741021''
#-TAq Primers 3398+3399 ''Negative Control''
#-PFx Primers 3398+3399 ''Negative Control''

<h2>'''July 5, 2007: Plan and Motility Preperations'''</h2>
'''plan'''
#Motility:
##Miniprep Sender and Receiver cells (elute with 30uL)
##*Digest with ECO and SPE
##*Run Gel/Extract
##*Ligate Sender with psB1A2/transform
##*Digest with Xba and PST
##*Run Gel/Extract
##*Ligate sender with psB1A2
##Make 16x Eiken Agar Plates
##Transform low copy plasmid (amp)
##*Prepare to add Receiver into the amp plasmid
#Diauxie:
##Plate rest of transformation
##Retry Ligation and transformation
##Redigest pSB1A2 
##*Gel/extract
##*Ligate with extracted I741015
##Attempt PCR with Pfx on 4 different temperatures
##Redigest I741015/pSB1A2 with
##*1uL SPE
##*.5uL XBA
##*For 20 hours
#Misc:
##pH new water and autoclave
##Prepare more LB amp plates

'''Motility Information'''

<h2>'''July 6, 2007: Motility Assay'''</h2>
'''Experiment Aims'''

Test motility at a variety of different temperatures.

'''Procedure'''

Eiken Agar Plates were made using a modified version of general procedure [[IGEM:PennState/2006/Eikenplates|1]].  An ncreased agar concentration was made to increase durability of plate and reduce the motility.

''.28% Eiken Agar Plates:''
*.25g Bactotryptone
*.20g NaCl
*.15g sodium citrate
*.07g Eiken Agar (for 20EikenAgar)

#Receiver cells grown at 37 degrees to OD500 ~1.2
#*Rediluted to OD500 ~ .2 and grown for 1.5 hours at 30 degrees
#.3uL plated of receiver cells with 1uL of sender cells (.3uL used with wildtype RP4037)
#grown at temperatures 21, 26, 30, and 37C

Controls
*Motile Strain RP4037
*+HSL Plate
*-Sender Cell Plate

'''Results:'''


'''Observations:'''

21 degree plates have no exhibited swarming as of ~14 hours.
26 degree plates seemed quite successful with controls, however room temperature may have dropped from 26 to 22 overnight
30 degree plates wild type cells swarmed very quickly when compared to controls
37 degree plates exhibited leaky expression in negative control

'''Conclusions:'''


'''Future Experiments:'''

Try a variety of agar concentrations at the 26 degree temperature

<h2>'''July 8, 2007: Motility Assay'''</h2>

'''Experiment Aims'''

Test motility at a variety of different temperatures.

'''Procedure'''

Eiken Agar Plates were made using a modified version of general procedure [[IGEM:PennState/2006/Eikenplates|1]].  An ncreased agar concentration was made to increase durability of plate and reduce the motility.

''.24-.32% Eiken Agar Plates:''
*.25g Bactotryptone
*.20g NaCl
*.15g sodium citrate
*.varied amount of Eiken Agar


Controls
*Motile Strain RP4037
*+HSL Plate
*-Sender Cell Plate

'''Results:'''

insert pictures here

'''Observations:'''

.24% plates seem too fragile to use.
.26-.32% seem fine

'''Conclusions:'''

use a .30% Eiken Agar Plate

<h2>'''July 10, 2007: Changing the sender to a high copy plasmid'''</h2>
Part BBa_F1610 was grown up off registry plate and transformed into DH5a.
Motility assay will be retried Wednesday.

Digested P1010 PSB1A2
Gel extracted
Ligated with
Extracted products 7/10 in LW-1
Transformed


<h2>'''July 16, 2007: Testing what we have and preparing for sequencing'''</h2>
Grew up 6-7 colonies per plate of ligations. (large numbers on negative plates, thus a quck miniprep, and subsequent digestion should reveal if the constructs were made.

Also, a gigantic gel was made to test the dna concentration of everything we have.  The full list can be found in Noah's lab notebook.

<h2>'''July 17, 2007: Giant Digest'''</h2>


32 +4X
0.25u EcoRI
0.25u SpeI
1.00u EcoRI Buffer (10x)
3.00u DNA Miniprep
5.50u H2O

Mix:
9uL EcoRI
9uL SpeI
36uL EcoRI Buffer
DNA to be added
198uL H2O

<h2>'''July 23, 2007: Motility Assay Protocol'''</h2>
Motility Protocol:
# For each plate to be made, add the following to a 125mL flask: 
*0.25g Bacto tryptone 
*0.20g NaCl 
*0.15g sodium citrate 
*0.075g Eiken agar (for a 0.30% agar swarm plate)
note: <.25% plates are difficult to deal with as they break very easily (Weiss motility assay 3/11/07) .26%-.32% is what I tend to use.  Cells did not swarmed at .40% agar concentration (Weiss motility assay 3/11/07) -LWeiss
# Add 25mL purified water to the flask and swirl to dissolve 
# Autoclave 25 minutes with slow exhaust 
# Upon cooling, add appropriate antibiotics and inducers 
*Kan 50µg/mL
*Amp 100µg/mL
*HSL between 10-4 - 10-8
*IPTG 1mM
# Allow plates to solidify for ~3 hours before use 
*Plates can be prepared the previous day before inoculated, but tend to try out if prepared too far in advance-LW

Day 1: Inoculate O'N Culture
# Inoculate 3mL overnight cultures of all samples to be tested 
# Shake at 200 RPM in 30°C incubator 
*I have done this at 37°C and it seems to be ok, I would recommend shaking at whatever temperature the motility assay will be run at-LW 

Day 2: Assay
# Dilute overnight culture 1:100 in LB (30uL of culture in 3mL LB) 
# Grow in a 30°C incubator at 200 RPM to an A590nm O.D. between 0.3-0.4 
# Add about .5µL-3µL of each culture to plates
# Grow in a 26-30°C incubator 8 hours 
*At 37°C our constructs tend to swarm regardless of HSL’s presence-LW (motility assay 7/8/2007)
# Take picture of plate with digital camera
*I prefer to monitor the cells over a 20 hour period.-LW

<table>
<tr>
<td>[[Image:PSUIGEM7232007Slide1.JPG|thumb|left|Description]]</td>
<td>[[Image:PSUIGEM7232007Slide2.JPG|thumb|left|Description]]</td>
<td>[[Image:PSUIGEM7232007Slide3.JPG|thumb|left|Description]]</td>
</tr><tr>
<td>[[Image:PSUIGEM7232007Slide4.JPG|thumb|left|Description]]</td>
<td>[[Image:PSUIGEM7232007Slide5.JPG|thumb|left|Description]]</td>
<td>[[Image:PSUIGEM7232007Slide6.JPG|thumb|left|Description]]</td>
</tr>
</table>
<p>

<h2>'''August 5, 2007: Motility Assay Protocol'''</h2>

primers from monday aug 6th

Do reverse complement of this one:
ATCAATGCTTATCTGATATGACTA

lacI
Do the reverse complement:
atgccggtgtctcttatcagaccgt

<h2>'''August 16, 2007: Sender Construct Primers'''</h2>

Strategy:  We would like to read both ways from inside the plasmid to ensure correct sequencing:

Primer 1 (Near end of psb2k3

taggcgtatcacgaggcag

Primer 2 (rc of beginning psb2k3)

gcctgcataacgcgaag


<h2>'''November 8, 2007: Designing Promoters for Synthesis'''</h2>
'''Template:
Promoter -35 and -10=N, caps are changed CRP binding site:'''
gaattcgcggccgcttctagagggtgatggatgatgtcgtNNNNNNgggcactccctttcagttNNNNNN
ttatgttatttcacactgctattgagataattcacaaGTCGTACGCCGTGCTGCaaataaaatggaatga
tgaaactgggtaattcctcgaagagaaaaatgcaataagtacaattgcgcaacaaaagtaagatctcggt
cataaatcaagaaataaaccaaaaatcgtaatcgaaagataaaaatctgtaaNNNNNNtcccctgtttag
ttgcNNNNNNttggttacgtttatcgcggtgattgttacttattactagtagcggccgctgcag

Original promoters(Level 1):
gaattcgcggccgcttctagagggtgatggatgatgtcgtAATATTgggcactccctttcagttGCTCAAttatgttatttcacactgctattgagataat
tcacaaGTCGTACGCCGTGCTGCaaataaaatggaatgatgaaactgggtaattcctcgaagagaaaaatgcaataagtacaattgcgcaacaaaagta
agatctcggtcataaatcaagaaataaaccaaaaatcgtaatcgaaagataaaaatctgtaaTTGTTTtcccctgtttagttgcTAAAAAttggttacgtt
tatcgcggtgattgttacttattactagtagcggccgctgcag

Level 2:
gaattcgcggccgcttctagagggtgatggatgatgtcgtATTATTgggcactccctttcagttGGTCAAttatgttatttcacactgctattgagataat
tcacaaGTCGTACGCCGTGCTGCaaataaaatggaatgatgaaactgggtaattcctcgaagagaaaaatgcaataagtacaattgcgcaacaaaagta
agatctcggtcataaatcaagaaataaaccaaaaatcgtaatcgaaagataaaaatctgtaaTTGTTTtcccctgtttagttgcTAAAATttggttacgtt
tatcgcggtgattgttacttattactagtagcggccgctgcag

Level 3:
gaattcgcggccgcttctagagggtgatggatgatgtcgtAATATAgggcactccctttcagttTCTCAAttatgttatttcacactgctattgagataat
tcacaaGTCGTACGCCGTGCTGCaaataaaatggaatgatgaaactgggtaattcctcgaagagaaaaatgcaataagtacaattgcgcaacaaaagta
agatctcggtcataaatcaagaaataaaccaaaaatcgtaatcgaaagataaaaatctgtaaTTGATTtcccctgtttagttgcTAAAATttggttacgtt
tatcgcggtgattgttacttattactagtagcggccgctgcag

Level 4:
gaattcgcggccgcttctagagggtgatggatgatgtcgtATTATTgggcactccctttcagttTGTCAAttatgttatttcacactgctattgagataat
tcacaaGTCGTACGCCGTGCTGCaaataaaatggaatgatgaaactgggtaattcctcgaagagaaaaatgcaataagtacaattgcgcaacaaaagta
agatctcggtcataaatcaagaaataaaccaaaaatcgtaatcgaaagataaaaatctgtaaTTGATAtcccctgtttagttgcTAAAATttggttacgtt
tatcgcggtgattgttacttattactagtagcggccgctgcag

Consensus promoters Level 5:
gaattcgcggccgcttctagagggtgatggatgatgtcgtATTATAgggcactccctttcagttTGTCAAttatgttatttcacactgctattgagataat
tcacaaGTCGTACGCCGTGCTGCaaataaaatggaatgatgaaactgggtaattcctcgaagagaaaaatgcaataagtacaattgcgcaacaaaagta
agatctcggtcataaatcaagaaataaaccaaaaatcgtaatcgaaagataaaaatctgtaaTTGACAtcccctgtttagttgcTATAATttggttacgtt
tatcgcggtgattgttacttattactagtagcggccgctgcag