==Cre-LoxP System==


The use of Cre recombinase to excise regions of DNA between two loxP sites could be incorporated into part [http://parts.mit.edu/registry/index.php?title=Part:BBa_J37015 J37015]. Our aim is to insert two loxP sites into this part, which, when in place, would block the synthesis of any genes downstream of it, and when removed would allow their synthesis. Thus the Cre-loxP system would effectively act as a switch controlled by the activity of the promoter in front of the Cre enzyme.   

*Using Cre recombinase as a switch appears to be, in theory, a very efficient and solid method. 
*The parts are not available in the registry however, which means that we would have to make them. 
*The loxP sites are around 20 bases in length and so sequences can be be worked out and bought fairly easily. 
*The Cre recombinase site is around a kilobase in length. However we have been able to obtain the enzyme from one of the labs, and as it is isolated in a plasmid, this will make it much simpler for us to incorporate it into our design.
*The Cre recombinase will also need a promoter - perhaps we could use one such as lacI? (we need to find out whether IPTG has any effect on other promoters though)
*With all the parts available, we estimate that the length of time for implementation should be under a week.

==Design==

Two devices are required:
*1) A complementary sequence for a reporter with loxP sites either side 
*2) Cre recombinase under the control of a promoter

The device J37027 can be found [http://parts.mit.edu/registry/index.php/Part:BBa_J37027 here] 

[http://openwetware.org/images/5/5b/Cre_PCR.doc Design of primers and method of construction]

[http://openwetware.org/wiki/Image:Cre_Pops_Blocker.ppt Proposed Implementation of Cre-LoxP system]

==Cre recombinase==

Cre recombinase device in registry [http://parts.mit.edu/registry/index.php/Part:BBa_J37030 J37030]

Sequence and primers [http://openwetware.org/images/f/ff/Cre_Primers.doc here]

There is one further step that we need to take if we want to successfully incorporate this system into the whole system. The plasmid containing the Cre enzyme coding region will be going into the prey cell. However the prey cell will already contain the main coding plasmid, which has ampicillin resistance. Therefore to ensure that the second plasmid is taken up into the cell as well, we need to swap plasmids by ligation. Ligating into a kanamycin resistant plasmid is probably our best bet. We could do this after the device has been constructed in the ampicillin resistant plasmid.  

==LoxP sites==

The two LoxP sites that we are going to use are Lox66 and Lox71 - mutant LoxP sites

Sequences - [http://parts.mit.edu/registry/index.php/Part:BBa_J37026 Lox66] and [http://parts.mit.edu/registry/index.php/Part:BBa_J37028 Lox71]

[http://openwetware.org/images/b/b3/Redoing_Cre2.doc Alterations to Cre-Lox System] 

==Further Reading==

*[http://www3.interscience.wiley.com/cgi-bin/fulltext/70001850/PDFSTART Cre recombinase: The universal reagent for genome tailoring]
*[http://nar.oxfordjournals.org/cgi/content/full/30/17/e90 Cre recombinase-mediated inversion using lox66 and lox71: method to introduce conditional point mutations into the CREB-binding protein]