A very brief description of the mechanism for making cinnamon smells:

http://docs.lib.purdue.edu/dissertations/AAI9939353/



*[[IGEM:MIT/2006/System brainstorming#Bacterial scents|Brainstorming page for scents]]
We'll dump all this information on today's page for now but you guys should think about what page structure you want to use to store all your information!
==notes for Veena and Giovanni==

Hey Veena and Giovanni-- so a super quick recap of the last two days is that we have picked the bacterial scents project. Hopefully you will be as excited about this project as we are! Anyways, we have lots of cool application ideas and specifically want to hone in on control. With help from our great grads and faculty, we have started to do some hard core research and planning, so if you guys are on internet please definitely jump in and help. I think that doing the following things would be good if you have time...

1: Keep up to date with the Wiki posts and get familiar with some of the linked literature

2: Try to to think about a scent subproject that might interest you so that we can get orders in asap for necessary materials

3: Use the Wiki. Help research a specific enzyme and help to fill in a table following the 'enzyme information template' so that we can pursue acquisition of the enzyme (if it looks promising)

4. We have looked at producing wintergreen, fresh/floral, cinnamon, and jasmin so far, but there are many other scented esters, so could check into say banana ester/scent or something else too

---ayt, well thanks and hope you both are well !!

==BAMT: SAM-Benzoic Acid Carboxyl Methyltransferase==
#Species
#* ''Antirrhinum majus'' (snapdragon); Just to note, this is the species that Dudareva is apparently sending us.
#Sequence & Structure
#*Codon optimized?
#*Restriction sites:  '''There is an SpeI site, but that is it.'''
#*Post-translational modification
#*Material availability
#**Request from authors: '''Already requested!!!'''
#***Form of DNA: '''pET-28a vector'''
#Reaction catalyzed
#*Substrate: '''Benzoic Acid (BA)'''
#**Synthesized in our chassis or supplied exogenously: '''supplied exogenously''' (used at '''5&mu;g/mL''') <cite>dudareva2</cite>
#*Product: '''Methyl Benzoate''' (MB)
#**''Transport'': They measured MB concentrations without needing to lyse the cells, so MB '''can''' (presumably) exit the cell
#*Reaction rates - The BAMT (Met1) purified from E. coli had apparent K<sub>m</sub> values for benzoic acid and SAM of 1.5 mM and 87 μM, respectively.  
#Does it smell??
#*The scent comes from methyl benzoate, a volatile ester.
#*Regulation issues/Transport issues for substrate/product
#**They induced BAMT expression with IPTG (0.4mM final conc.) in ''E. coli''.  We'd need to make modifications to the plasmid/cassette as necessary.
#Ported to '' '''E. coli''' ''
#*1.1kb fragment cloned into NdeI/BamHI sites of pET-28a (Kan<sup>R</sup>)
#*Contains an N-terminal His-tag
#*They cloned the BAMT gene from two different Met codons.  The second Met is two codons downstream (position 4 of the protein) <cite>dudareva1</cite>.  
#What is the regulation ''in planta''
#*Enzyme is active as a dimer.
#*There are two Met codons at the begnning of the BAMT cDNA.  The BAMT resulting from amplification from the second Met shows a 2.4 times higher specific activity than from the first Met in protein purified from cell lysate; on the other hand, BAMT purified from ''E. coli'' shows no difference in specific activity (from both Met codons).  The K<sub>m</sub> and k<sub>cat</sub> are the same from both codons in ''E. coli''.
#*Inhibition by SAH was competitive with respect to SAM and noncompetitive with respect to BA. The K<sub>i</sub> value of SAH was determined to be 7 μM for SAM and 14 μM for BA. SAM appears to be the first substrate to bind to the enzyme. Methyl benzoate, would be the first to be released and SAH the last.
#*BAMT activity may be regulated by the intracellular SAM/SAH concentration ratio rather than BA availability.
#References & Links
#*Genbank accession number(s): AF198492

<biblio>

#dudareva1 pmid=11051108
#dudareva2 pmid=10852939

</biblio>

==Kate's enzyme/SAMT-type enzyme==
#Species: 
#*from Antirrhinum majus or A. majus. This is the common snapdragon flower
#*this carboxyl methyltransferase gene encodes an enzyme that catalyzes the transfer of the methyl group of SAM to the carboxyl group of salicylic acid to make the volatile ester methyl salicylate
#Sequence & Structure: 
#*Codon optimized? 
#**yes, already been ported into e. coli
#**It is a homodimer with subunit molecular mass of about 45 kDA
#*Restriction sites: NONE in enzyme's ORF
#**On the pET-28a expression vector there is a convenient NdeI site at the initiating ATG codon as well as a BamHI site downstream of the stop codon
#*Post-translational modification: no. 
#**all of these enzymes lack an organelle-specific transit signal peptide, so they catalyze the formation of their corresponding esters in the cytoplasm (note: this is an in vivo study in petals where immunogold labeling studies localized BAMT to cytosol)
#*Material availability: 
#**YES - it is in a pET 28 vector and is kindly being shipped by Natalia Dutareva
#*cDNA from arabadopsis: no
#*Request from authors: yes, Reshma has secured the vector already
#**Form of DNA? pET 28 plasmid from cDNA
#*** the isolated cDNA clone was first found to contain a total of 1333 nucleotides. It encodes an ORF of 1149 nt corresponding to a protein of 383 amino acids with a calculated molecular mass of 43,613 and pI of 8.04 (GenBank Accession No. AF515284). Also, the protein encoded by this cDNA has 54–56% amino acid identity to SAMT from C. breweri (note: C. breweri is the highly researched protein with 3D crystal structure models, etc. that we currently have 'pains' in obtaining)
#***the coding region of the isolated cDNA clone was then subcloned into the expression vector pET-28a, which contains an N-terminal polyhistidine (6x His) tag
#**need to synthesize? no
#***AF515284, 1333 bp, mRNA
#***link: http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&val=23957317
#Reaction catalyzed
#*Substrate: 
#**salicylic acid (100%) or benzoic acid (45% relative activity)
#*Synthesized in our chassis or supplied exogenously? 
#**supplied exogeneously, genes activated by induction (At least for now)
#*Product
#**''Coolness'' (Austin): very cool, painless
#*Reaction rates - K<sub>m</sub>,k<sub>cat</sub>: 
#**E. coli-expressed SAMT protein catalyzes the formation of the volatile ester methyl salicylate from salicylic acid with a K(m) value of 83 microM. (This is pretty small = good). so Km (SA) is 83 and Km (SAM) is 3-4. It can also methylate benzoic acid to form methyl benzoate, but its K(m) value for benzoic acid is 1720 M (larger). Relative activity with substrate salicylic acid is 100%, with substrate benzoic acid is 45%, and with all other acids is 0%. Numerically, Kcat/Km (SA) s-1 M-1 is 132
#Does it smell?? 
#*It is not involved in floral scent production in snapdragons because snapdragon flowers do not emit methyl salicylate. Snapdragons seem to use their SAMT gene to create methyl salicylate for defense purposes. Snapdragon flowers do emit their methyl benzoate, which is their most abundant scent compound. When the SAMT gene was put into E.coli, some wintergreen smell was detected by Dutareva, and the gene products ARE DEFINATELY PRESENT, so i don't think we should worry. Specifically, the culture medium of the E. coli cells expressing SAMT contained methyl salicylate (2.1 μg/ml) when the growing medium was supplemented with 5 μg/ml salicylic acid, and methyl benzoate (0.86 μg/ml) when the growing medium was supplemented with 5 μg/ml benzoic acid. Note that E. coli cells that contained a pET-28a plasmid without the SAMT coding region did not have any detectable enzyme activity and did not produce methyl salicylate or methyl benzoate.
#*Regulation issues:
#**the expressed recombinant SAMT enzyme possesses a pH optimum from 7.0 to 7.5, and has 80%-90& maximum activity in pH from 5.0 to 8.0. The enzyme was active in both Tris- and phosphate-citrate-based buffers, although phosphate was 20% lower. Snapdragon SAMT has the broadest temperature stability of this family of enzymes. Incubation for 30 mins at 42 C had no effect on activity. Also, SAMT activity not affected by the presence of 5mM Mg2+, K+, NH4+, Ca2+ in assay reaction. It does not require any metal ions as cofactors. Fe2+ and Cu2+ have a strong inhibitory effect.
#*Transport issues for substrate/product: no
#Ported to ''E. coli'' or yeast? 
#*Yes, extensively studies with E.coli. -- PMID: 12361714
#*Also, the SAMT gene is already on a pET plasmid with ease of access to our team. This also means that the gene is transcribed by T7 polymerase rather than E. coli polymerase, and  T7 polymerase is much faster than E. coli so we can expect to get very high expression levels.  
#What is the regulation ''in planta'':  it is thought to serve a physiological role in plant defense
#References & Links
#*Corresponding authors: Natalia Dudareva
#*Genbank accession numbers: AF515284
#*Pubmed: PMID: 12361714
#*Websites: http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=12361714&dopt=Abstract
#*Any companies or patents? no

==JMT: Jasmonic acid carboxyl methyltransferase==
#Species: Arabidopsis thaliana (thale cress)
#Sequence & Structure
#*Use cDNA from A. thaliana (multistep- screened with Brassica NTR1)
#*JMT cDNA inserted into the pGEX-2T vector (Amersham Pharmacia-GE) at the EcoRI site and fused with glutathione-S-transferase gene (GST). 
#*The recombinant was transformed into E. coli BL21. Fusion protein GST-JMT was expressed
#**Purified to get JMT- issue?
#Reaction catalyzed
#*Substrate: Jasmonic acid and SAM added exogenously
#*Product: Methyl jasmonate
#**Given off by plants as defense, also in jasmin oil
#*Reaction rates - K<sub>m</sub>: 38.5 µM; K<sub>cat</sub>:25 S<sup>−1</sup>
#Methyl jasmonate has jasmine smell
#*Reactions done in vitro with purified product- GST is tag? 
#*Probable that jasmonic acid and methyl jasmonate (volatile) can be transported (see function)
#Ported to E. coli (see above for method)
#Regulated ''in planta'': External stimuli such as wounding or methyl jasmonate can cause gene expression
#References & Links
#*<biblio>Seo01 pmid=11287667</biblio>
#*Genbank accession numbers: BD441046, BD441047
#*Patents for synthesizing inferior jasmine substitutes are commercial
#Link= <biobrick>BBa_J45010

==PAL & SAM:CCMT==
#Species: PAL (Phenylalanine Ammonia-lyase) is found in several organisms, but for our interests, it seems the best option is the gene from ''Rhodosporidium toruloides''. SAM:CCMT (S-adenosyl-L-methionine:cinnamate carboxyl methyltransferase) can only be found in ''Ocimum basilicum'' L., Lamieaceae (a basil plant) as far as I can tell.
#Sequence & Structure
#*Codon optimized?: The good news is that PAL from ''Rhodosporidium toruloides'' has been successfully transported and expressed in ''E. coli'' and ''S. cerevisiae'' via plasmids by Faulkner, et al in 1994.  The bad news is that SAM:CCMT seems to be only present in the cDNA library created by Dr. Hao in 1998 and in the raw ''Ocimum basilicum'' L., Lamieaceae form.  Qualley also discussed a cDNA library that he had possession of at Purdue.  
#*Restriction sites
#*Post-translational modification
#*Material availability
#**cDNA from arabadopsis: No
#**Request from authors: Yes Faulkner for PAL, Hao or his former advisor Simon for SAM:CCMT, Also Qualley at Purdue          Progress: contacted both Simon and Hao about access to Hao's dissertation, contacted Simon for access to cDNA library containing SAM:CCMT
#***Form of DNA? PAL- E. coli & S. cerevisiae vector form; SAM:CCMT- Ocimum bailicum cDNA library
#**need to synthesize? An option for SAM:CCMT
#Reaction catalyzed: (1) Phenylalanine-->Trans-cinnamic Acid + NH4+ with PAL as a catalyst (2) Trans-cinnamic Acid-->Methylcinnamate with SAM:CCMT as a catalyst
#*Substrate: Phenylalanine *Advantage-Phenylalanine already needed for other cell processes in E. coli.  Thus, no extraneous compound would need to be added to the media.
#**Synthesized in our chassis or supplied exogenously?: Phenylalanine must be supplied in the media.
#*Product: Methylcinnamate
#**''Coolness'' (Austin): Very cool.
#*Reaction rates - K<sub>m</sub>,k<sub>cat</sub> KM for S-adenosyl-L-homocysteine: 30 μM, KM for trans-cinnamate: 60 μM, NOTE: catalytic activity for SAM:CCMT (kcat/Km) found to be 10-80 times higher affinity for cinnamic acid than for benzoic acid and p-coumaric acid
#Does it smell?: Smells like cinnamon
#*Regulation issues
#*Transport issues for substrate/product: None.
#Ported to ''E. coli'' or yeast?: Ported to both for PAL; not ported to either for SAM:CCMT
#What is the regulation ''in planta'': For PAL, chavicol O-methyltransferase (CVOMT) and eugenol O-methyltransferase (EOMT) involved in methylchavicol biosynthesis increase the enzyme's activity.  For SAM:CCMT, there is Cu2+.  
#References & Links
#*Corresponding authors: Faulkner, Simon, Hao, Rasmussen, Orum
#*Genbank accession numbers: PAL=X12702
#*Pubmed: 
<biblio>
#**orum92 pmid=1368015
#**rasmussen91 pmid=1773059
#**faulkner94 pmid=8200528
</biblio>
#*Websites:
http://www.ncbi.nih.gov/entrez/query.fcgi?db=nucleotide&cmd=search&term=X51513&doptcmdl=GenBank

http://docs.lib.purdue.edu/dissertations/AAI9939353/

http://en.wikipedia.org/wiki/Cinnamic_acid

http://docs.lib.purdue.edu/dissertations/AAI3104932/

http://www.plantphysiol.org/cgi/content/full/125/2/539

Last link shows known pathways in basil.

http://biocyc.org/META/NEW-IMAGE?type=ENZYME&object=CPLX-6223#QUALLEY05

http://www.psna-online.org/PSNAabst05.pdf

#*Any companies or patents?: None

==BSMT: Benzoic acid/salicylic acid carboxyl methyltransferases==
#Species: <b>Petunia x hybrida</b>, A. thaliana and N. suaveolens
#Sequence & Structure
#*Screen cDNA-> subcloned into the NdeI-EcoRI site of the expression vector pET-28a; E. coli
#*Purified/Washed afterwards
#*Material availability: cDNA from Petunia x hybrida- phBSMT1, phBSMT2; (sequence not available? from A. thaliana)
#Reaction catalyzed
#*Substrate: benzoic acid or salicylic acid supplied exogenously
#*Produces methyl benzoate/methyl salicylate- SA 100%, BA 16.3/27.3%
#**Potential dual function- handles multiple inputs
#*Reaction rates (A.t.): K<sub>m</sub>(SA) 51.6uM, K<sub>m</sub>(BA) 1273uM, K<sub>cat</sub>/K<sub>m</sub>(SA) 230 s<sup>-1</sup>M<sup>-1</sup>, K<sub>cat</sub>/K<sub>m</sub>(BA) 3 s<sup>-1</sup>M<sup>-1</sup>:
#*Standard potential issues (transfer out of cell etc.)
#Characteristics of dual products-same smell etc.
#BSMT ported to E. coli
#Down-regulated "in planta" by ethylene (regulated spatially and temporally)
#References & Links
#*<biblio>Underwood05 pmid=15849311</biblio>
#*<biblio>Negre03 pmid=14630969</biblio>
#*GenBank: 28629494, 28629496 (in Petunia)

==Alcohol Acetyltransferase(s): ATF1 & ATF2==
NOTE- The first pubmed link paper demonstrated that ATF1 is more effective in producing the banana scent compound than ATF2.
#Species: S. cerevisiae
#Sequence & Structure (see <bbpart>BBa_J45006</bbpart>)
#*Codon optimized?: Yes. Already done in E. coli and S. cerevisiae
#*Restriction sites: EcoRI site found in the sequence at base pair 412.
#*Post-translational modification
#*Material availability
#**cDNA from arabadopsis: No
#**Request from authors: Horton, et. al, vectors that work in E. coli and S. cerevisiae
#***Form of DNA?: plasmid
#**need to synthesize?: not if we can get it from the authors
#Reaction catalyzed: isoamyl alcohol--->isoamyl acetate
#*Substrates: isoamyl alcohol & acetyl CoA
#**Synthesized in our chassis or supplied exogenously?: isoamyl alcohol in media
#*Product: isoamyl acetate
#**'''Banana smell'''; you get the pear smell when isoamyl acetate is dissolved in ethanol
#*Reaction rates - K<sub>m</sub>,k<sub>cat</sub>
#Does it smell??
#*Regulation issues
#*Transport issues for substrate/product: None. The substrate, isoamyl alcohol, is taken up by E. coli when added to the medium. (The concentration of isoamyl alcohol used by Horton et al. was 10mM.) Additionally, Horton et al. measured isoamyl acetate concentrations by running gas chromotography on the supernatant, so we know that the product will leave the cell.
#Ported to ''E. coli'' or yeast?: Successfully done by Horton et al.
#What is the regulation ''in planta'': ATF1 repressed by unsaturated fatty acids or oxygen
#References & Links
<biblio>
#*Horton03 pmid=12937998
#*Fujii94 pmid=9836419
#*Nagasawa98 pmid=8085822
</biblio>
#*Corresponding authors
#*Genbank accession numbers: ATF1= Z75285
#*Pubmed http://www.pubmedcentral.gov/articlerender.fcgi?tool=pubmed&pubmedid=12957907
#*Websites
Regulation of ATF1 here:

http://www3.interscience.wiley.com/cgi-bin/fulltext/3987/PDFSTART
#*Any companies or patents?

==SAMT with 3D crystal structure ==
#Species: 
#*from Clarkia breweri
#*is an S-adenosyl-L-methionine:salicylic acid carboxyl methyltransferase (SAMT) enzyme
#*enzyme is a dimer with subunit mol. weight of 40.3 kDa
#Sequence & Structure: 3D crystal structure of protein available!
#*Codon optimized?
#*Restriction sites: NONE in enzyme's ORF
#**NdeI-BamHI before and after coding region of gene on the expression vector constructs pET-11a (non-tagged polypeptide) and pET-28 (has 6x histidine codons upstream of NdeI site-- i.e. a His tag)
#*Post-translational modification
#*Material availability
#**cDNA from arabadopsis: no
#**Request from authors: as of now lab wants a material transfer sheet (pains)
#***Form of DNA? The accession no. of C. breweri SAMT cDNA is AF133053, 1321 bp
#**need to synthesize? possibly an option 
#Reaction catalyzed
#*Substrate: salicylic acid
#**Synthesized in our chassis or supplied exogenously? supplied
#*Product
#**''Coolness'' (Austin)
#*Reaction rates - K<sub>m</sub>,k<sub>cat</sub>:
#**K(m) for salicylic acid is 24 microM and Km for SAM is 9 microM
#**Kcat/Km (SA) s-1, M-1 = 116.7
#Does it smell?? YESSS - Pichersky says strong smell in e.coli
#*Regulation issues
#*Transport issues for substrate/product
#Ported to ''E. coli'' or yeast?
#*Yes, the C. breweri SAMT cDNA was expressed in e. coli and the bacterial cells synthesized a functional SAMT protein with properties nearly identical to those of the plant purified enzyme 
#What is the regulation ''in planta'' : floral scent production and plant defense
#References & Links
#*Corresponding authors: Eran Pichersky
#*Genbank accession numbers: AF133053
#*Pubmed: http://www.ncbi.nlm.nih.gov/entrez/query.fcgicmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=10375393&query_hl=35&itool=pubmed_docsum
#*Websites: this shows the structural basis for substrate recognition of SA: http://www.biology.lsa.umich.edu/research/labs/pichersky/references/Plant%20cell-2003.pdf
#*Any companies or patents?
#Helpful Bonus: Crystal structure!!!
#*The recently obtained crystal structure of C. breweri SAMT has identified the specific residues responsible for substrate binding in carboxyl methyltransferases. Asp-57, Asp-98, and Phe-130 were found to be involved in SAM binding, whereas Gln-25, Met-150, Trp-151, and Met-308 are responsible for salicylic acid binding.
#*The N-terminal sequence is involved in dimer formation, while C-terminal domain is primarily involved in substrate binding