[[IGEM_project_tsinghua|Back to Project Page]]
==Introduction==

===Standards in engineering cell-cell communications.===

When engineering prokaryotic cell networks, one of the most 
difficulties is isolating signals. For one, it's necessary to get various kinds 
of signals to allow specific communication. In the widely-used quorum-sensing 
system, the quorum-sensing signal is excreted out of the cells, which in space broadcasts an identical signal to 
every recipient cell. In spite of the concentration gradient, that means cells with same receptor 
will be triggered simultaneously. And when trying to engineering a more complex 
system, the freedom of design is severely limited by available kinds of 
quorum-sensing signal producers and 
receptors. But if a framework allows one signal being isolated from another, it 
will break the those limits and allow different cells being triggered 
specifically. For another, space isolation is equally important in 
engineering. Take the isolated wire cable as an example, although all the wires 
utilized identical electric current as the medium for signal transport, the isolation 
allows identical electric current in different wires to generate different combinations of 
signals. While the excreted broadcasting prokaryotic signals are unable to work in this 
way because they share the same medium or space. 
Therefore, those requirements bring the problem posed in the field of 
bioengineering into a general topic in engineering cell-cell communications. 
It's necessary to establish a system with standards in cell-cell communication with expected isolation and variance in the kinds of signals.

===Cell-cell communication platform can serve as the framework of cellular circuits.===

Just take the simple case. If a system allows one cell to deliver specific 
signal to a specific group of cells, any complicated cellular signal systems can 
be built up with combinations of a proper number of such systems. It will be 
also feasible to construct a logical network with such framework.


===Conjugation can perform as cell-cell specific signal pathway.===


It is reported that relaxases can transport fusion proteins 
to recipient cells during conjugation, such as Cre. Since relaxases are DNA binding proteins, we 
further postulated that other DNA binding proteins might also be transported 
into the recipient cells, which would be a potential useful phenomenon in 
engineering cell-cell communication. Therefore, both the relaxases and 
the DNA binding proteins are chosen as the transport for cell-cell signals during 
conjugation. Though extensively studied, the detailed mechanism of conjugation is still controversial. A popular theory postulates that a single strand DNA of host cell is carried to receptive cell by relaxase and then induce rolling replication. Some other works, however, do not support this model. Thus, we chose two kind of DNA binding proteins, including double strand DNA binding protein and single strand DNA binding protein. In both cases, several protein binding sites can be integrated in one 
conjugation plasmid to allow the transport of multiple copies of the signals to 
the recipient cells in order to amplify the effecting. The conjugation system has many advantages. First, 
the signals are transferred from one cell to another in a point-to-point manner, 
rather than the excreted broadcast way. Second, protein signals can be easily 
fused to the DNA-binding transport with pre-constructed system, allowing 
parallel engineering of many different pairs of signal producer and receptor. Third, the protein signal is produced in 
donor cells and degraded by the receptor cells. This allows reutilization of the 
same signals and indicates that the cellular network can be reset to the initial 
condition for reuse. At last, it is facile to find various kinds of protein signal 
producer/receptor pairs from two hybrid systems.

===Prokaryotic two hybrid system is a large pool of specific signals.===


During the last two decades, at least three prokaryotic two hybrid system has 
been established. If the two hybrid system is integrated with the cell-cell 
signal transport system, they can provide a large pool for specific signals in 
cell-cell communications.

===Strains &amp; Plasmids===

===Strains===
{| border="1" cellspacing="0"
|-
|  Strain ||  Source
|-
|  <i>E. coli</i> TOP10 ||  Transgen biotech company, Beijing.
|-
|  <i>E. coli</i> S17-1 ||  Prof. Chen's lab collection
|-
|  <i>E. coli</i> JM109 ||  Prof. Chen's lab collection
|}

===Plasmids===
{| border="1" cellspacing="0"
|-
|  Plasmid ||  Source
|-
|  pEASY-Blunt ||  Transgen biotech company, Beijing.
|-
|  pBBR1MCS ||  Prof. Chen's lab collection
|-
|  pBluescript II SK(-) ||  Prof. Chen's lab collection
|-
|  pMD18-T ||  Takara, Japan.
|-
|  pLZZGPp ||  Prof. Chen's lab collection
|}


==Flowchart==
The initial part of the project pursuits the verification of relaxase and C1 
transport.<p>[[Image:CelCuitv1.5.gif]]


</p>


==Experiment Records==

===Thread 1: Celcuit carrier plasmids.===

{| border="1" cellspacing="0"
|-
|  Nubmer ||  Date ||  Inherit ||  Chart Index ||  Notes ||  Results ||  Plan ||  Expected Time ||  Operator
|-
|  1 ||  6-20 ||  - ||  1-1 ||  Anneal the phosphated C1 primers and ligate the annealed primers with T4 DNA ligase for 16 hours. ||  - ||  Check the ligation result. ||  6-21 ||  Peng Dong
|-
|  2 ||  6-20 ||  - ||  1-3 ||  Extract the pLZZGPp plasmid. Use HindIII to cut the plasmid and add CIAP to dephosphate the digested plasmid. ||  - ||  Recover the digested &amp; dephosphated plasmid. ||  6-21 ||  Peng Dong
|-
|  3 ||  6-21 ||  2 ||  1-3 ||  Use DNA extraction kit to purify the digested pLZZGPp plasmid. ||  Purified digested plasmid. ||  Ligation. ||  6-21 ||  　
|-
|  4 ||  6-21 ||  1 ||  1-2 ||  Check the ligation products. ||  6 bands were seen in the gel. The annealed C1 primers are ligated into multiple copies. ||  Ligation. ||  6-21 ||  Peng Dong
|-
|  5 ||  6-21 ||  3,4 ||  1-3 ||  Add the the purified digested pLZZGPp plasmid to the C1 ligation system to insert the C1 fragments into the pLZZGPp plasmid. The negative control ligation used only the digested pLZZGPp plasmids. ||  - ||  Transformation. ||  6-22 ||  Peng Dong
|-
|  6 ||  6-22 ||  5 ||  1-3 ||  Transformation. The ligation system is transformed into E. coli TOP10 competent cells. ||  - ||  Pick clones for verification. ||  6-23 ||  Peng Dong
|-
|  7 ||  6-23 ||  6 ||  1-3 ||  8 clones on the ligation plate were picked. ||  There are 50% more clones on the plate of ligation than on the negative control. ||  Store the cultures and send them for sequencing. ||  6-24 ||  Peng Dong
|-
|  8 ||  6-24 ||  7 ||  1-3 ||  The 8 clones are stored sent to sequencing. ||  - ||  Wait for the sequencing results. ||  6-30 ||  Peng Dong
|-
|  9 ||  7-9 ||  　 ||  1-1 ||  Solve the C1 primer and anneal the C1 primer for further ligation. ||  - ||  Ligate the C1  ||  8-5 ||  Yexing Liu
|-
|  10 ||  7-11 ||  　 ||  1-2 ||  Extract the pLZZGPp plasmid and cut it with HindIII and then apply CIAP for dephosphoration.<p>Ligate the dephosphorated plasmid with C1 fragment by annealing for positive and with dd H2O for negative control and transformation. ||  - ||  　 ||  　 ||  Keyu Li
|-
|  11 ||  7-11 ||  9 ||  1-5 ||  PCR amplification to delete the Mob sequence from pBBR1MCS. ||  No PCR product detected. ||  Try other conditions. ||  7-12 ||  Keyu Li
|-
|  12 ||  7-12 ||  10 ||  1-2 ||  Check the clones on the positive and negative plates. ||  The numbers of clones on the positive and negative plates do not have significant difference.<p>Especially, there are quite a lot of clones on the negative plates.</p> <p>One problem may attribute to the extremely high density of plasmid. ||  Try other conditions to reduce the number of negative clones. ||  7-14 ||  Keyu Li 
|-
|  13 ||  7-12 ||  11 ||  1-5 ||  Use Takara LA Taq with GC buffer to perform gradiant PCR amplification to delete the Mob sequence of pBBR1MCS. ||  There is PCR product at specific annealing temperature. ||  Use the optimal PCR contidion to get enough PCR products. ||  7-14 ||  Keyu Li
|-
|  14 ||  7-14 ||  12 ||  1-2 ||  Apply both HindIII and CIAP from Fermentas at the same time with Buffer R for overnight. ||  　 ||  Rescue the dephosphated pLZZGPp. ||  7-15 ||  Keyu Li
|-
|  15 ||  7-14 ||  13 ||  1-5 ||  Use Takara LA Taq with GC buffer to perform PCR to amplify the pBBR1MCS at optimized annealing temperature. ||  - ||  Extract the PCR product. ||  7-15 ||  Keyu Li
|-
|  16 ||  7-15 ||  14 ||  1-2 ||  Extract the dephosphated&nbsp; product. Ligate the product with additional dd H2O as negative control assay and transform.  ||  - ||  Check the plates. ||  7-16 ||  Keyu Li
|-
|  17 ||  7-15 ||  15 ||  1-5 ||  Extract the PCR product, and perform phophoration to the PCR product.  Ligate the products and trasform. ||  - ||  Check the plates. ||  7-16 ||  Keyu Li
|-
|  18 ||  7-16 ||  16 ||  1-2 ||  Check the plates. ||  There are still about 80 clones on the plates. ||  Need to try further.<p>Wait for Peng Dong's results. ||  8-2 ||  Keyu Li
|-
|  19 ||  7-16 ||  15 ||  1-5 ||  Pick the clones from the plate and culture them. ||  - ||  Extract plasmids and enzyme digest to check the products. ||  7-17 ||  Keyu Li
|-
|  20 ||  7-17 ||  19 ||  1-5 ||  Extract the plasmids and enzyme digest. ||  Those are correct clones.<p>The mob carrier was successfully 
  constructed. ||  　 ||  　 ||  Keyu Li
|-
|  21 ||  7-28 ||  18 ||  1-2 ||  Get 3 possible clones with Lambda C1 binding sites from Peng Dong.  Culture them and the strain with pLZZGPp empty plasmid. ||  　 ||  Extract plasmids for enzyme digestion and sequencing. ||  7-29 ||  Keyu Li
|-
|  22 ||  7-29 ||  21 ||  1-2 ||  Extract the plasmid and perform enzyme digestion.<p>Send the plasmids for sequencing by M13F general primer. ||  The enzyme digestion products from the 3 clones do not have significant differences from the pLZZGPp empty plasmid.  ||  Wait for the sequencing results.<p>Try other methods.</p> <p>From Peng Dong's results, we still see that there're a lot of negative clones. The pLZZGPp plasmid may need further modification to serve as a dephosphorated plasmid. Perhaps an additional HindIII site would give better result. ||  8-2<p>7-30 ||  Keyu Li
|-
|  23 ||  7-30 ||  22 ||  1-2 ||  Culture the strains with pGL-3 Basic plasmid and pLZZGPp. ||  - ||  Extract the plasmid and digest with KpnI and SalI. ||  7-31 ||  Keyu Li
|-
|  24 ||  7-31 ||  23 ||  1-2 ||  Extract the plasmid and digest with KpnI and SalI.<p>Extract the 3.8K fragment from pLZZGPp digestion. ||  The pGL-3 Basic was not completely digested. ||  Check the pGL-3 Basic and extract the 2K fragment. ||  8-1 ||  Keyu Li
|-
|  25 ||  8-1 ||  24 ||  1-2 ||  Check the KpnI-SalI digestion of pGL-3 Basic plasmid.<p>Extract the 2K fragment from the pGL-3 Basic.</p> <p>Ligate the 2K fragment from pGL-3 Basic and pLZZGPp.</p> <p>Transform. ||  　 ||  Check the plates. ||  8-2 ||  Keyu Li
|-
|  26 ||  8-2 ||  21 ||  1-3 ||  The 8 sequencing results are analyzed and unfortunately none of them was found to be desired clones. ||  Failed in construction the clones by Peng Dong. ||  　 ||  - ||  Zhenyu Shi
|-
|  27 ||  8-2 ||  25 ||  1-2 ||  Culture 4 clones from the positive ligation.<p>　 ||  The plate of positive ligation has many more clones than the negative 
  control.  ||  Extract the plasmid and perform enzyme digestion to verify. ||  8-3 ||  Keyu Li
|-
|  28 ||  8-3 ||  27 ||  1-2 ||  Extract the plasmid and digest with HindIII. ||  All of the clones are positive. ||  Extract more plasmid. Use HindIII and CIAP to digest and dephosphorate at the same time. ||  8-4 ||  Keyu Li
|-
|  29 ||  8-4 ||  28 ||  1-2 ||  Extract pLZZGPp-pGL plasmid. Use HindIII and CIAP to digest and dephosphorate.<p>Ligate and transform. ||  The pLZZGPp-pGL was not digested completely. ||  Check the clones. ||  8-5 ||  Keyu Li
|-
|  30 ||  8-5 ||  29 ||  1-2 ||  Digest and dephosphorate again. ||  There are about more than 200 clones on the plate. ||  Extract the digested and dephosphorated products. ||  8-6 ||  Keyu Li
|-
|  31 ||  8-6 ||  30 ||  1-2 ||  Extract the products.<p>Ligate and transform. ||  The digestion result was completely and clear. ||  Check the plates. ||  8-7 ||  Keyu Li
|-
|  32 ||  8-7 ||  31 ||  1-2 ||  　 ||  Only less than 20 clones. ||  Wait for the phosphorated primer. ||  8-13 ||  Keyu Li
|-
|  33 ||  8-14 ||  32 ||  1-1 ||  Dilute the C1 primer to about 6ng/ul, 1.2ng/ul and 2.4ng/ul respectively.<p>Anneal the primer. ||  - ||  Insert the primer to pLZZGPp. ||  8-14 ||  Keyu Li
|-
|  34 ||  8-15 ||  33 ||  1-2 ||  One group of the annealed primers are pre-ligated for 5 hours and then ligated with pLZZGPp. The other group is ligated with pLZZGPp directly.  In addition, pLZZGPp self-ligation was performed as negative control.  <p>Transform.  ||  - ||  Check the plates. ||  8-16 ||  Keyu Li
|-
|  35 ||  8-16 ||  34 ||  1-2 ||  Check the plates.<p>Pick 40 clones for colony PCR verfication. ||  The 6ng/ul with pre-ligation group turn out has 1 times more clones than the negative plate.<p>Two PCR products were found to be significantly longer than the negative contorl. ||  Send 4 clones for sequencing. ||  8-17 ||  Keyu Li
|-
|  36 ||  8-17 ||  35 ||  1-2 ||  Send the clones for sequencing. ||  - ||  Wait for sequencing result. ||  8-21 ||  Keyu Li
|-
|  37 ||  8-21 ||  36 ||  1-2 ||  2 clones were found to have 3 C1 binding sites. ||  - ||  Insert the C1 binding site to pBBR1MCS.<p>Culture pBBR1MCS.  ||  - ||  Keyu Li
|-
|  38 ||  8-23 ||  37 ||  1-3 ||  Culturing the E. coli strain harboring the pBBR1MCS plasmid. ||  - ||  Extract plasmid and enzyme digestion. ||  8-24 ||  Keyu Li
|-
|  39 ||  8-23 ||  37 ||  1-3 ||  PCR clone fragment containing multiple C1 binding sites. ||  - ||  Extract the PCR products and enzyme digestion. ||  8-24 ||  Keyu Li
|-
|  40 ||  8-24 ||  38,39 ||  1-3 ||  Extract the plasmid and PCR production and Enzyme digestion with SacI and XhoI.<p>Extract the digestion products , ligate and transform. And transform a control system without inserts. ||  - ||  Pick clones from the plates. ||  8-25 ||  Keyu Li
|-
|  41 ||  8-25 ||  40 ||  1-3 ||  6 clones were picked from the ligation plate for PCR verfication.<p>And correct clones are cultured. ||  There only hundreds of colonies on the ligation plate but few on control plate.  <p>All the 4 clones are verified to be right. ||  　 ||  　 ||  Keyu Li
|-
|  42 ||  8-25 ||  - ||  1-1 ||  Use PCR to clone pBBR1MCS to delete the Mob gene with Takara LA Taq. The PCR product was extracted and phosphorated. The phosphorated products was ligated and transformed. ||  　 ||  Check the colonies. ||  8-26 ||  Keyu Li
|-
|  43 ||  8-26 ||  42 ||  1-2 ||  The colonies on the plate were picked and cultured. Plasmids were extracted and verified by enzyme digestion. ||  2 clones were found to be right. ||  　 ||  　 ||  Keyu Li
 |}
 
===Thread 2: Celcuit Initiator.===

{| border="1" cellspacing="0"
|-
|  Nubmer ||  Date ||  Inherit ||  Chart Index ||  Notes ||  Results ||  Plan ||  Expected Time ||  Operator
|-
|  1 ||  4-28 ||  - ||  2-3 ||  PCR clone the FLP gene and insert the PCR product into pEASY-B cloning vector. Transform the ligation system. ||  - ||  Pick clones. ||  4-29 ||  Xinyu Zhao
|-
|  2 ||  4-29 ||  1 ||  2-3 ||  4 clones were picked and cultured. ||  - ||  Extract plasmid and perform double digestion verification. ||  4-30 ||  Xinyu Zhao
|-
|  3 ||  4-30 ||  2 ||  2-3 ||  Extract the plasmid and use double digest to screen. ||  Two clones are found to be right. ||  Wait the sequencing result. ||  5-4 ||  Xinyu Zhao
|-
|  4 ||  5-5 ||  3 ||  2-3 ||  The sequencing results were analyzed. ||  Both the two were found to be correct clones. ||  Wait for other Fragments and the expression plasmid. ||  - ||  Zhenyu Shi
|-
|  5 ||  5-5 ||  - ||  2-1 ||  Use PCR to clone the lambda C1 gene and insert the PCR product into pEASY-B cloning vector. Transform the ligation system. ||  - ||  Pick clones. ||  　 ||  Zhenyu Shi
|-
|  6 ||  5-5 ||  - ||  2-2 ||  Use PCR to clone the Mob gene and insert the PCR product into pEASY-B cloning vector. Transform the ligation system. ||  - ||  Pick clones. ||  　 ||  Zhenyu Shi
|-
|  7 ||  5-6 ||  5,6 ||  2-1,2-2 ||  4 clones of both lambda C1 and Mob are picked and use PCR to screened. ||  All the clones are found to be posive in PCR screen. ||  Send the clones for sequencing. ||  5-7 ||  Zhenyu Shi
|-
|  8 ||  5-8 ||  7 ||  2-1,2-2 ||  The clones are sent for sequencing. ||  - ||  Wait the sequencing results. ||  5-12 ||  Zhenyu Shi
|-
|  9 ||  5-15 ||  8 ||  2-1,2-2 ||  A Mob clone and a C1 clone were found to be correct. ||  - ||  Wait for the expression vector. ||  - ||  Zhenyu Shi
|-
|  10 ||  8-4 ||  9 ||  2-4 ||  Use PCR to clone part of the pBluscript II SK(-) vector to construct a vector without T7 promoter.<p>The PCR products were phosphorated, ligated and transformed. ||  - ||  Check the transformation result. ||  8-5 ||  Zhou Yu
|-
|  11 ||  8-5 ||  10 ||  2-4 ||  Four white colonies were picked and cultured. ||  There are a lot of white colonies. With dozens of blue colonies. ||  Verify by enzyme digestion. ||  8-6 ||  Zhou Yu
|-
|  12 ||  8-6 ||  11 ||  2-4 ||  Extract the plasmid and verify the results by enzyme digestion.<p> Culture the C1 and Mob cloning plasmid to construct the initiator plasmid.</p> <p>Culture more strains harboring pCCInt. ||  One of the four clones was found to be correct. The correct clone was named as pCCInt. ||  Extract the C1 and Mob plasmids. ||  8-7 ||  Zhou Yu
|-
|  13 ||  8-7 ||  12 ||  2-4 ||  Extract the C1 and Mob plasmids.<p>The pCCInt was extracted and digested.</p> <p>The FLP gene was PCR cloned from pCP20. And the PCR products was extracted and digested. ||  The C1 plasmids show very light results. ||  Use PCR to amplify C1 gene from the cloning plasmid. ||  8-8 ||  Zhou Yu
|-
|  14 ||  8-8 ||  13 ||  2-4 ||  Use PCR to Amplify the C1 from the cloning plasmid.<p>Both the C1 PCR products and the Mob plasmids were digested. ||  　 ||  Extract the enzyme digestion products and perform ligation. ||  8-9 ||  Zhou Yu
|-
|  15 ||  8-9 ||  14 ||  2-4 ||  The FLP, pCCInt, C1 and Mob digestion products were extracted. FLP, pCCInt with either C1 or Mob were ligated and transformed with a control which does not contain C1 or Mob. ||  　 ||  Check the two plates. ||  8-10 ||  Zhou Yu
|-
|  16 ||  8-10 ||  15 ||  2-4 ||  From either ligation plates, 10 clones were picked to culture for further verification. ||  There are much more clones on the plates of the positive ligation than the negative controls. ||  Enzyme digestion and sequencing. ||  8-11 ||  Zhou Yu
|-
|  17 ||  8-11 ||  16 ||  2-4 ||  Plasmids were extracted from all the 20 cultures were and digested. ||  6 of the FLP-C1 clones and 8 of the FLP-Mob clones were seemed to be correct. ||  Send for sequencing. ||  8-13 ||  Zhou Yu
|-
|  18 ||  8-13 ||  17 ||  2-4 ||  The correct clones were sent for sequencing. ||  　 ||  Wait for the sequencing result. ||  8-17 ||  Zhou Yu
|-
|  19 ||  8-17 ||  18 ||  2-4 ||  The sequencing results were analyzed. ||  2 FLP-C1 clones and 4 FLP-Mob clones were found to be correct. ||  　 ||  　 ||  Zhou Yu, Zhenyu Shi
 |}

===Thread 3: Celcuit Reporter.===

{| border="1" cellspacing="0"
|-
|  Nubmer ||  Date ||  Inherit ||  Chart Index ||  Notes ||  Results ||  Plan ||  Expected Time ||  Operator
|-
|  1 ||  7-9 ||  - ||  3-1 ||  Culture the E. coli BW25141 harboring pKD13 plasmids. ||  - ||  Extract the plasmids. Store the strain.  ||  7-10 ||  Yuan Zhao, Jing Ma, Zhenyu Shi 
|-
|  2 ||  7-9 ||  - ||  3-1 ||  PCR clone the lambda and rnnB terminators directly from the culture of BW25141. ||  The two terminators are successfully cloned. ||  Enlarge the PCR system and extract the PCR product for further overlap extention PCR. ||  7-9 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  3 ||  7-9 ||  2 ||  3-1 ||  PCR clone the lambda and rnnB terminators in large system. ||  Success. ||  Extract the products. ||  7-9 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  4 ||  7-9 ||  3 ||  3-1 ||  Extract the PCR product. ||  Success. ||  Perform Overlap extention PCR. ||  7-10 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  5 ||  7-10 ||  4 ||  3-2,3-3 ||  Anneal the two PCR products and use gradient overlap extension PCR to optimize the conditions. ||  Success. ||  Use the optimal condition to clone the Products. ||  7-10 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  6 ||  7-10 ||  5 ||  3-3 ||  Overlap extension PCR to clone in large system. ||  Success. ||  Extract the products. ||  7-10 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  7 ||  7-10 ||  6 ||  3-3 ||  Extract the PCR product. ||  Success. ||  Insert the product into pEASY-B cloning vector. ||  wait for FLP. ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  8 ||  7-12 ||  7 ||  3-4 ||  Insert PCR product to pEASY-B vector. Transformation ||  - ||  Pick clones from the plate amd perform PCR screening. ||  7-13 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  9 ||  7-13 ||  8 ||  3-4 ||  PCR screen the clones. ||  Success. ||  Culture the right clones for sequencing. ||  7-14 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  10 ||  7-14 ||  9 ||  3-4 ||  culture the right clones for enzyme digestion verification. ||  - ||  Send the cultures for sequencing. ||  7-15 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  11 ||  7-15 ||  10 ||  3-4 ||  Store the cultures of different clones. ||  　 ||  Extract the plasmids for verification and send for sequencing. ||  7-16 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  12 ||  7-16 ||  11 ||  3-4 ||  Enzyme digestion of the plamids with (XhoI &amp; SalI) for the positive screening and with (SalI &amp; SpeI) for the negative screening.  ||  Failed. None of the digest turn out to show right DNA strips. ||  Send for sequencing. Use different enzyme sites. ||  7-16 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  13 ||  7-16 ||  12 ||  3-4 ||  Enzyme digestino with Fermentas XhoI &amp; SalI.  ||  Failed. Maybe the cloning vector is wrong. ||  Send the cultures for sequencing. ||  7-16 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  14 ||  7-16 ||  13 ||  3-4 ||  Send the cultures for sequencing. ||  - ||  Wait for sequencing results. ||  - ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  15 ||  7-19 ||  14 ||  3-4 ||  The sequencing results are analyzed and 2 clones were found to be right clones. But the restriction enzyme sites on the vector were not shown in sequencing results. It's even more possible that the cloning vector is wrong. ||  - ||  Culture the rights clones for verification. ||  7-17 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  16 ||  7-20 ||  15 ||  3-4 ||  Extract the plasmid and perform enzyme digestion for 8 hours. ||  None of the clones were correctly digested. ||  Digest longer lest the digestion is not completed. ||  7-21 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  17 ||  7-21 ||  16 ||  3-4 ||  Check the digestion products. ||  None of the plasmids from those clones were digested. ||  Reorder another box of pEASY-B cloning vector. ||  7-22 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  18 ||  7-22 ||  17 ||  3-4 ||  Order another box of the pEASY-B vector. ||  - ||  　 ||  　 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  19 ||  7-25 ||  18 ||  3-4 ||  The transgene company comfirmed that the previous pEASY-B vector was wrong. ||  - ||  Repeat the cloning of the lambda-rrnB terminators. ||  　 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  20 ||  7-26 ||  19 ||  3-1,3-2,3-3 ||  PCR clone the lambda-rrnB terminators. ||  Success. ||  Insert the PCR product into pEASY-B vector. ||  7-26 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  21 ||  7-26 ||  20 ||  3-4 ||  Insert the PCR product into pEASY-B cloning vector. Transformation. ||  - ||  Pick clones for PCR screening. ||  7-27 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  22 ||  7-27 ||  21 ||  3-4 ||  PCR screen the clones. ||  2 clones were found to be correct. ||  Culture the clones for sequencing. ||  7-28 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  23 ||  7-28 ||  22 ||  3-4 ||  Culture the correct clones. ||  - ||  Send for sequencing. ||  7-29 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  24 ||  7-29 ||  23 ||  3-4 ||  Send the culture for sequencing. ||  - ||  wait for the sequencing results. ||  - ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  25 ||  8-4 ||  24 ||  3-5 ||  The sequencing results were analyzed.<p>The pEASY-LRT plasmid was culture. ||  Both 2 clones were found to be correct, the clones were named as pEASY-LRT. ||  Culture the correct clones for further verifications. ||  8-5 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  26 ||  8-5 ||  25 ||  3-6 ||  The pEASY-LRT plasmid was extracted and digest by ScaI and Eam1105I(MBI), then blunted, ligated and transformed. ||  　 ||  Check the plates the next day. ||  8-6 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  27 ||  8-6 ||  26 ||  3-6 ||  20 of the clones were picked and negatively screened for Ampicillin resistance. ||  　 ||  Check the plates the next day. ||  8-7 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  28 ||  8-7 ||  27 ||  3-6 ||  Check the resistance.<p>Culture the strain harboring the pEASY-LRT Amp(-), and extract the plasmids. ||  8 of the 20 clones do not have ampicillin resisntance, which were named as pEASY-LRT Amp(-). ||  Check the FLP's function. See thread 4. ||  8-8 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  29 ||  8-8 ||  28 ||  3-7,3-8 ||  PCR cloned part of the pBluescript II SK(-).  <p>PCR cloned part of the fragment containing Km resistance gene from lambda pKD13. </p> <p>The two PCR productes were phosphorated, ligated and transformed and sprayed on a&nbsp; plate with both Km and Amp resistance gene. ||  　 ||  Check the plate. ||  8-9 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  30 ||  8-9 ||  29 ||  3-9 ||  4 clones were picked from the plasmid to culture for verification by restriction analysis. ||  　 ||  Extract the plasmids and digest the plasmids. ||  8-10 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  31 ||  8-10 ||  30 ||  3-9 ||  Extract the plasmids of the pBL-pKD plasmids and digest the plasmids with SphI and PstI. ||  2 of the 4 clones were found to be correct and ligated in the right direction, which are named as pBL-pKD ||  Blunting the digestion products. ||  8-11 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  32 ||  8-11 ||  31 ||  3-9 ||  The digestion products were blunted with T4 DNA polymerase, then ligated and transformed. ||  　 ||  Check the plates. ||  8-12 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  33 ||  8-12 ||  32 ||  3-9 ||  8 clones were picked and cultured.<p>The plasmids were extracted and enzyme digested to examine products. </p> <p>The strains harboring pBL-Linker was cultured. ||  3 of the 8 clones were found correct, which were named as pBL-Linker. ||  　 ||  8-13 ||  Han Li, Yuting Wang, Zhenyu Shi
 |}

===Thread 4: Celcuit FLP verifier.===

{| border="1" cellspacing="0"
|-
|  Nubmer ||  Date ||  Inherit ||  Chart Index ||  Notes ||  Results ||  Plan ||  Expected Time ||  Operator
|-
|  1 ||  7-12 ||  - ||  4-1 ||  Culture the plasmid pCP20 harboring FLP gene. ||  - ||  Extract the plasmid. ||  7-13 ||  Yuan Zhao, Jing Ma, Zhenyu Shi 
|-
|  2 ||  7-12 ||  - ||  4-1 ||  PCR clone the FLP gene directly from the culture. ||  Success. ||  Extract the PCR product. ||  7-12 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  3 ||  7-12 ||  2 ||  4-1 ||  Extract the PCR product. ||  Success. ||  Insert the product into pEASY-B cloning vector. ||  7-12 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  4 ||  7-12 ||  3 ||  4-1 ||  Insert the FLP PCR product into pEASY-B plasmid. Transformation. ||  - ||  Pick clones from the plate amd perform PCR screening. ||  7-13 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  5 ||  7-13 ||  4 ||  4-1 ||  PCR screen the clones. ||  Success. ||  Culture the right clones for sequencing. ||  7-14 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  6 ||  7-14 ||  5 ||  4-1 ||  culture the right clones for sequencing. ||  - ||  Send the cultures for sequencing. ||  7-15 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  7 ||  7-15 ||  6 ||  4-1 ||  Store the cultures of different clones. ||  　 ||  Extract the plasmids for verification and send for sequencing. ||  7-16 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  8 ||  7-16 ||  7 ||  4-1 ||  Enzyme digestion of the plamids with (SacI &amp; XbaI) for the positive screening.  ||  Failed. None of the digest turn out to show right DNA strips. ||  Send for sequencing. Use different enzyme sites. ||  7-16 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  9 ||  7-16 ||  8 ||  4-1 ||  Enzyme digestino with Fermentas SacI &amp; XbaI.  ||  Failed. Maybe the cloning vector is wrong. ||  Send the cultures for sequencing. ||  7-16 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  10 ||  7-16 ||  9 ||  4-1 ||  Send the cultures for sequencing. ||  - ||  Wait for sequencing results. ||  - ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  11 ||  7-19 ||  10 ||  4-1 ||  The sequencing results are analyzed and 3 clones were found to be right clones. But the restriction enzyme sites on the vector were not shown in sequencing results. It's even more possible that the cloning vector is wrong. ||  - ||  Culture the rights clones for verification. ||  7-17 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  12 ||  7-20 ||  11 ||  4-1 ||  Extract the plasmid and perform enzyme digestion for 8 hours. ||  None of the clones were correctly digested. ||  Digest longer lest the digestion is not completed. ||  7-21 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  13 ||  7-21 ||  12 ||  4-1 ||  Check the digestion products. ||  None of the plasmids from those clones were digested. ||  Reorder another box of pEASY-B cloning vector. ||  7-22 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  14 ||  7-22 ||  13 ||  4-1 ||  Order another box of the pEASY-B vector. ||  - ||  　 ||  　 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  15 ||  7-25 ||  14 ||  4-1 ||  The transgene company comfirmed that the previous pEASY-B vector was wrong. ||  - ||  Repeat the cloning of the FLP gene. ||  7-26 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  16 ||  7-26 ||  15 ||  4-1 ||  PCR clone the LFP gene. ||  Success. ||  Insert the PCR product into pEASY-B vector. ||  7-26 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  17 ||  7-26 ||  16 ||  4-1 ||  Insert the PCR product into pEASY-B cloning vector. Transformation. ||  - ||  Pick clones for PCR screening. ||  7-27 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  18 ||  7-27 ||  17 ||  4-1 ||  PCR screen the clones. ||  2 clones were found to be correct. ||  Culture the clones for sequencing. ||  7-28 ||  Yuan Zhao, Jing Ma, Zhenyu Shi
|-
|  19 ||  7-28 ||  18 ||  4-1 ||  Culture the correct clones. ||  - ||  Send for sequencing. ||  7-29 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  20 ||  7-29 ||  19 ||  4-1 ||  Send the culture for sequencing. ||  - ||  wait for the sequencing results. ||  - ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  21 ||  8-3 ||  20 ||  4-1 ||  The sequencing results were analyzed.<p>The clone was cultured for further verification.</p> <p>The strain harboring pBluescript II SK(-) was cultured. ||  1 clone was found to be correct ||  　 ||  8-3 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  22 ||  8-4 ||  21 ||  4-2 ||  Extract the FLP cloning plasmid and pBluescript II SK(-) vector and digest them.<p>The digested products were ligated and transformed. ||  - ||  Check the plates tomorrow. ||  8-4 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  23 ||  8-5 ||  22 ||  4-2 ||  4 white clones picked and cultured. ||  - ||  　 ||  8-5 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  24 ||  8-6 ||  23 ||  4-2 ||  Culture the pBlueFLP and extract the plasmids. ||  All the 4 clones were found to be correct. The plasmid was named as pBlueFLP. ||  Wait for pEASY-LRT Amp(-) to check the FLP's function. ||  - ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  25 ||  8-7 ||  24 ||  4-2,3-6 ||  The pBlueFLP and pEASY-LRT Amp(-) were co-transformed. And IPTG was added when plating. ||  　 ||  Check the plate. ||  8-8 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  26 ||  8-8 ||  25 ||  4-2,3-6 ||  1 clone was picked and cultured with IPTG induction for 8 hours, then diluted by 10^5 time and sprayed on a plates.<p>The strain harboring pCP20 was cultured. ||  There is no blue colonies on the plates. ||  Check the plate. ||  8-9 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  27 ||  8-9 ||  26 ||  4-2,3-6 ||  pCP20 was extracted and co-transformed with pESAY-LRT Amp(-). The plate was cultured at 42 degrees for half an hour and then 30 degrees for overnight since the pCP20 is a temperature sensitive plasmid but can be induced by heat. ||  There is still no blue clones on the plate yesterday. ||  Check the plate. ||  8-10 ||  Han Li, Yuting Wang, Zhenyu Shi
|-
|  28 ||  8-10 ||  27 ||  4-2 ||  Check the plate. ||  One blue was found on the plates.<p>It seems that the pEASY-LRT Amp(-) does work. ||  　 ||  　 ||  Han Li, Yuting Wang, Zhenyu Shi
 |}