=Bacterial Targeting (LppOmpA)=

==Goals==
*To make bacteria adhere to targets with a high degree of specificity.
*To make bacteria transduce a signal upon adhesion to their targets. 

== Planned Work ==

===6/27/07===

Transform strains of our controls - strep and his <br>
Prepare them in liquid culture

===6/28/07===

Innoculate and Induce controls <br>
Run them on the MACS system to see if the assay works.

===6/29 to 7/11===

Continue to try and cell sort using MACS<br>
Order Anti-his and strep with fluorescent tags for FACS.

===7/11/07===

PCR OmpA1 + Random library

===7/12/07===

Ligation of OmpA+library<br>
Grow an overnight culture for the FACS

===7/13/07===

Induce bacteria, add antibodies according to the [[IGEM:Harvard/2007/Protocols/Fluorescent Labeling Protocol|Fluorescent Labeling Protocol]].<br>
Take the bacteria to the FACS cell sorter and sort cells, plating them afterwards.

== Completed Work ==

See [[IGEM:Harvard/2007/Laboratory Notebooks/Bacterial Targeting Protocol|Bacterial Targeting Protocol]]

== Brainstorming ==

=== Initial Plan (6/25/07) ===

'''Bacterial Surface Expression and Cellular Targeting''' (compiled 5/2/07 by Mike)
 

Hi all, I put together a project outline here related to Perry’s and Stephanie’s presentation on surface targeting of bacteria, focusing on an area that many of you expressed interest in: '''Medical Applications of Synthetic Biology'''.  Importantly, it is a project that is feasible to finish during the summer time limit (a big consideration), a project that would be a great resource for future researchers, has the potential for quality research with wide ranging applications, and a project that each of you could really contribute to.  Also it embraces the iGEM idea of creating modular genetic systems.


The project outline involves the creation of a modular system used to '''express a peptide library on the surface of E. coli'''.  This would utilize the power of combinatorial constructions, selections, and surface expression (Three underutilized strengths George Church talked about during our first meeting).  


Essentially, we could use the BioBrick strategy to create two "parts": 1 ) a membrane protein that will anchor a protein or peptide library of interest to the outer membrane of E. coli, and 2) a random peptide library that we can fuse to the outer membrane portion.  We could use this to select peptides that bind various tissue types or cells, enabling us to target bacteria to specific cell types.


An outline of the strategy is as follows:


1) '''PCR or have synthesized the genes for two or three membrane proteins''' that will serve as the membrane anchors.  PCR primers will contain the BioBrick sequence ends to create fusion proteins.  Two candidate membrane proteins are OmpA and AIDA1 (see references).  We can also use some constructs already made by myself, Perry, and the MCB100 students last quarter to facilitate the project.

''References:
Display of heterologous proteins on the surface of microorganisms: from the screening of combinatorial libraries to live recombinant vaccines. Georgiou G, Stathopoulos C, Daugherty PS, Nayak AR, Iverson BL, Curtiss R 3rd.
Nat Biotechnol. 1997 Jan;15(1):29-34. Review.'' [[Media:Georgiou.pdf]]   

''Autodisplay: efficient bacterial surface display of recombinant proteins. Jose J.
Appl Microbiol Biotechnol. 2006 Feb;69(6):607-14. Epub 2005 Dec 20. Review.'' [[Media:Jose.pdf]]


2. Next we can '''synthesize a double stranded nucleic acid sequence that encodes a random peptide library'''.  This is straightforward to have synthesized since we can synthesize a random oligonucleotide (equal mix of A,T,C,G at each position) flanked by the BioBrick sequences.


3. Next we can '''ligate the Membrane Portion and Random Peptide Library into an expression plasmid''' and overexpress the membrane protein-peptide library in E. coli.


4. We can then '''characterize''' the overexpression by running protein denaturing gels, assay transformation effeciency and coverage of the library, and assay for surface expression by using a variety of control surface proteins like histidine tags (bind nickel) and streptavidin binding tags (bind streptavidin).  I have some control vectors made that we can use during our testing.


5. After we construct the fusion library we can use the power of selection to identify '''surface peptide sequences that target our bacteria to various tissue types or cell types''' we are interested in.  This has '''important medical implications''' since we may be able to target "microbial factories" to various areas of the body using this method (related to many of your brainstorming ideas realting to '''synthetic symbiosis''', ie for vitamin production etc).  We can also select for peptides that bind to other cell types.  The list goes on.


This system is analogous to what has been done using phage (see reference), but in our case we can use bacteria, since bacteria have the added advantage that once bound they can produce desired products or perform desired functions (see references), such as targeting cancer cells (see references).
*Organ targeting in vivo using phage display peptide libraries. Pasqualini R, Ruoslahti E.
Nature. 1996 Mar 28;380(6572):364-6.  [[Media:Pasqualini.pdf]]

''Environmentally controlled invasion of cancer cells by engineered bacteria.
J Mol Biol. 2006 Jan 27;355(4):619-27.  Anderson JC, Clarke EJ, Arkin AP, Voigt CA.'' [[Media:Anderson.pdf]]

''A bacterial protein enhances the release and efficacy of liposomal cancer drugs.
Science. 2006 Nov 24;314(5803):1308-11. Cheong I, Huang X, Bettegowda C, Diaz LA Jr, Kinzler KW, Zhou S, Vogelstein B.'' [[Media:Cheong.pdf]]


This would be a really exciting project that would be feasible for the summer timeframe.  We could contribute many new BioBricks to iGEM at the end (membrane proteins and all the peptides we identified as binders), and potentially create a system that many other scientists could use in their own future projects.  The project also employs the power of combinatorial constructions (random peptide library), selections (select for binding to specific tissue types and cell types), and much needed advancements and modularity of cellular surface expression.  We can also build on work done by Perry, myself, and some of the MCB100 students this past quarter on related surface expression projects.


Please add comments or let me know if I can provide any more information on this.  I think it would be a great project to pursue, that would utilize all your talents, and help you learn many important experimental methods. We also have some members, postdocs, and TFs with experience and interest in this area, which will help the project move quicker this summer.  Thanks, Mike (post 5/2)

==Readings==


<biblio>
#lppompA1 pmid=16391137
#lppompA2 pmid=9005446
#lppompA3 pmid=8920186
#lppompA4 pmid=6313361
#lppompA5 pmid=1557377 
#lppompA6 pmid=12209821
#lppompA7 pmid=11036660
#lppompA8 pmid=16448666
#lppompA9 pmid=11137298
</biblio>