'''MIT iGEM 2005 Laboratory "bootcamp"'''
(run by Natalie Kuldell and Kate Bacon Schneider)

I.  Tour of 68-074


II. Pipetman use 
*Background Information/Demonstration (asked students what they knew):
**settings on different Pipetman
**different sizes of tips used (blue for P1000, yellow or white for P20, P200)
**first stop to pull up liquid/push through to second stop to release liquid
**releasing tips with ejector button-->dispose of in SHARPS
**change tips frequently!
*Exercise (each student):
**pipette minimum and maximum from each Pipetman (1 µl and 20µl for P20;  20 µL and 200 µL for P200; 200 µL and 1000 µL for P1000) using Xylene Cyanol dye
**allowed students to see how consistent pipetmen were, get a feel for what a certain volume looks like in an Eppendorf tubes


III. Sterile technique, serial dilutions, plating bacteria 
*Background Information/Demonstration (asked students to help explain):
**keep sterile things sterile by not touching them with nonsterile things (fingers, benchtop, etc.)-->work CLEANLY!
**flaming tubes after uncapping, after adding/removing liquid from tubes 
**dilutions:  a 1:10 dilution is 1 part concentrated solution + 9 parts dilution (10 parts total) 
**serial dilutions:  
***dilutions "in series"-->make 1:10 dilution, take 1 volume of 1:10 into 9 parts diluent to create 1:100 dilution, etc.  
***More accuracy/less error than pipetting small volumes (e.g. 1 µL in 100 µL total)
**plating cells:  
***label plates on media side with initials, date, strain name, dilution, etc.
***add liquid to be plated to center of plate (keep lid face down on bench top while adding, then cover)
***sterilize glass spreader by dipping in EtOH and passing through flame (don't bake spreader!)
***touch spreader to plate away from cells (to avoid killing them!)
***move spreader back and forth while spinning plate (about 45 seconds to "dryness"-->notice change in resistance as you spread)
*Exercise:
**Students made 1:10 and 1:100 serial dilutions of log phase bacteria and plated 50 µl of each (plus undiluted) on LB plates, then grew overnight at 37˚C
 

IV. Streaking bacteria for single colonies
*Background Information/Demonstration:
**the fabulous Beckwith lab 3 toothpick method for obtaining single colonies
***remember:  it's a serial dilution on a plate!
*Exercise:
**had students streak out 1 AmpR and 1 AmpS strain on an LB Amp plate, then grew plates overnight at 37˚C

 
V. Making media with antibiotic and inoculating liquid cultures
*Background Information/Demonstration:
**How to make LB Amp media from LB and Ampicillin stock solution (1000X working concentration)
**how to inoculate O/N culture 
***use "stick" (dowel) and touch colony (gets ~106 cells on stick)
***open tube (sterile technique) and touch stick to liquid media
*** flame tube and cap tube quickly
***don't use small pick (have to drop toothpick in media, part of stick that touched your fingers is not sterile and will contaminate culture!)
*Exercise:  
**had each group of 2 students make 25 mL of LB Amp media
**each student inoculated 3, 3 mL cultures:  one "media alone," one AmpR strain, one AmpS strain
**grew cells O/N @ 37˚C
 

VI. Polymerase Chain Reaction (PCR)
*Background Information:
**Basics of DNA, DNA synthesis and PCR cycle
***DNA 
****DNA is double stranded in E. coli cells
****Two strands of DNA are held together by basepairing (A and T, G and C)
****DNA has a 5' and 3' end (numbers= those of carbons of the sugar of the nucleotide...here deoxyribose) 
****DNA sequences written 5'->3' left to right
***DNA synthesis
****requires a primer (provides 3'OH for nucleotide addition)
****proceeds in the 5'-->3' direction
***PCR cycle:
**** Denaturation:  separation of template strands (95˚C)
**** Annealing:  primers bind via basepairing to template (50-65˚C, based on primer sequence and length)
**** Extension:  DNA strand is "extended" from primers using the rules of basepairing (e.g. if template says A, add T onto primer) (72˚C, optimal temperature for thermostable polymerase)
**Components of a PCR reaction 
*** template (genomic DNA, plasmid)
*** primers (short DNA molecules, give specificity to amplification, usually ~1000X excess of template)
*** dNTPs
*** DNA polymerase (thermostable, like Taq, so don't have to add enzyme after each denaturation cycle)
*** pH buffer and salts 
***Mg+2 (required for DNA polymerase function, concentration can be altered in PCR reaction)
*Exercise:
**students set up PCR with template and primers from Natalie's lab
**standard PCR master mix—ran cycle O/N.