== 6/25/07 ==
===Nucleotide Removal===
#Add 10 volumes of Buffer PN to 1 volume of random library extensions (single primer) and mix.
##B1, B2, B3, B4 and S1, S2, S3, S4 (about 50 ul each)
#Place a QIAquick spin column in a provided 2 ml collection tube.
#To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min at 6000 rpm.
##For the Bs, do the same process using the same 2 ml collection tube/spin column
##Do the same for the Ss
###Basically, we are combining the DNA for 1-4.
#Discard the flow-through and place QIAquick column back into the same tube.
#To wash QIAquick column, add 750 µl of Buffer PE and centrifuge for 1 min at 6000 rpm.
#Discard the flow-through and place the QIAquick column back in the same tube, which should be empty. Centrifuge for an additional 1 min at 13,000 rpm (~17,900 x g).
#Place the QIAquick column in a clean 1.5 ml microcentrifuge tube.
#To elute DNA, add 100 ul of Buffer EB (10 mM Tris·Cl, pH 8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min at 13,000 rpm (~17,900 x g). Alternatively, for increased DNA concentration, add 30–50 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge.

----------------------------------------------------------------------------------------------

===Transformation protocol of ligated DNA===
#Thaw the 4 tubes of cells on ice and mix gently to ensure that the cells are evenly suspended.  We will be using the following DNA:
##OmpA1/G1/S2
##OmpA1/G1/S
##OmpA2/G2/B
##OmpA2/G2/S
#Add 1 µl of the DNA solution directly to the NovaBlue cells.  Stir gently to mix.
#Place the tubes on ice for a little more than 5 minutes
#Heat the tubes for about 45 seconds in a 42°C water bath; do not shake
#Place on ice for 2 minutes
#Add 250 µl of room temperature SOC Medium to each tube
#Shake at 37°C (300 rpm) for more than 30 minutes.
#Light the bunsen burner
#Using petri dishes containing LB agar w/ Kanamycin (only with 275uL of the cells)
#For each plate, use sterile beads to spread the bacteria over the mix.
#Afterwards, turn the plates upside down and incubate at 37°C

------------------

===Transformation protocol of ligated DNA using Top10 cells===


The instructions provided below are for general use. Specific instructions for 
particular applications such as Zero Blunt® PCR Cloning are provided in the 
manual for that kit.  <br>
1. Centrifuge the vial(s) containing the ligation reaction(s) briefly and place on 
ice.<br> 
2. Thaw, on ice, one 50 μl vial of One Shot® cells for each 
ligation/transformation. <br>
3. Pipet 6 μl of each ligation reaction directly into the vial of competent cells 
and mix by tapping gently. Do not mix by pipetting up and down. The 
remaining ligation mixture(s) can be stored at -20°C. <br>
4. Incubate the vial(s) on ice for 30 minutes. <br>
5. Incubate for exactly 30 seconds in the 42°C water bath. Do not mix or shake. <br>
6. Remove vial(s) from the 42°C bath and place them on ice. <br>
7. Add 250 μl of pre-warmed S.O.C medium to each vial. S.O.C is a rich 
medium; sterile technique must be practiced to avoid contamination.<br> 
8. Place the vial(s) in a microcentrifuge rack on its side and secure with tape to 
avoid loss of the vial(s). Shake the vial(s) at 37°C for exactly 1 hour at 225 rpm 
in a shaking incubator. (Note: I actually put them in for 300 rpm because other samples were in the shaker. --Stephanie) <br>
9. Spread 20 μl to 200 μl from each transformation vial on separate, labeled LB 
agar plates. The remaining transformation mix may be stored at +4°C and 
plated out the next day, if desired.<br> 
10. Invert the plate(s) and incubate at 37°C overnight. <br>
11. Select colonies and analyze by plasmid isolation, PCR, or sequencing. <br>

From Invitrogen: [http://www.invitrogen.com/content/sfs/manuals/oneshottop10_man.pdf External Link]

----

===Digestion of OmpA1 protocol===
#Add 24.7 ul of transformed DNA directly above (OMPA1 ext or PCR)
#Add 3ul 10x NEB buffer2
#Add 1ul Nhe1
#Add 1ul Pst1
#Add 0.3ul BSA 100x
##This should all amount to 30ul total
#In a PCR machine:
##Incubate for 4 hrs at 37°C
##Heat inactivate for 20 min at 65°C
##Store overnight at 4°C

----------------------------------------------------------------------------------------------

===Sequencing DNA===
#Determine how many sequencing reactions you will be sending out, and ask Alain for a purchase order number for Genewiz, a DNA sequencing service.
#Take as many 8-tube strips as you need, and label them with your initials and a three-digit number, e.g. "PT001, PT002...". Plan and record which sequencing reactions you'll put into which tubes.
#Into each tube, pipette 8ul of the DNA to be sequenced. If you're premixing your own primer, add 4ul primer (2uM).
##Each sequencing reaction should contain only one primer. For each DNA sequence you want to be analyzed, you should have two sequencing reactions, one with forward primer and one with reverse primer.
##You may leave out primer if you are using a vector with annealing sites for universal primers. For example, the Topo vector has M13F and M13R primer annealing sites flanking the cloning site, and Genewiz can add those primers for you.
#Snap the tubes closed, wrap the strip in parafilm, and seal in a Ziplock bag.
#Log into www.genewiz.com with Alain's email aviel@fas, and password ******.
#Click "Place DNA Sequencing Order".
#Fill in "Number of Reactions", select "Premix", and select yes or no for "Same Day Service". #Click "Submit".
#Fill in the "PO Number". Fill in the sizes of your DNA samples to be sequenced. Select "DNA Type", usually plasmid or PCR. If you've already added primer, under Primer, select "Already added", and if you want Genewiz to add universal primers, select the proper primer "to be added". #Review the order, and click "Submit."
##If you are sequencing an insert within a plasmid, fill in the size of the entire plasmid plus insert.
#Print out two copies, one for Genewiz and one for yourself. Fold up the copy for Genewiz, and place into the Ziplock bag with the tubes. Drop off the bag in the dropbox at Conant 113 before 4pm Monday-Friday.
##The entrance to Conant is in the walkway between Lowell Lecture Hall and Fairchild.
##*<b>A Big Thanks to Perry for this Sequencing Protocol</b>

----------

==6/26/07==

===Bacterial Innoculation and Induction===

#Add 5 ul of Kanamycin
#Add 5 ml of LB
#Add 200 ul of bacteria:
##OmpA1 + his
##OmpA1 + strep
#Add 5 ul of IPTG
##Add when the culture is at log phase.

----------------------------------------------------------------------------------------------

===Measuring Optical Density to measure cell density===

#Using the Genesis UV scanning machine, we will measure the OD of the samples to find log phase:
##Pipette culture into the clear tubes, making sure the clear sides are facing the circle.
##Insert blank (LB only), OmpA1 + his, and OmpA1 + strep into the machine.
##Press measure blank (should go to zero)
##Log phase is between 0.2 and 0.8 OD, so watch the absorption value until it hits these values.
###See [[IGEM:Harvard/2007/week 2/absorbance|absorbance]] page for exact values calculated by the team.
#When the bacteria enters log phase, add IPTG to the sample that needs IPTG added.
#Then we're ready to induce.

-----------------------------

===Nucleotide Removal===
#Add 10 volumes of Buffer PN to 1 volume of random library extensions (single primer) and mix.
##B1, B2, B3, B4 and S1, S2, S3, S4 (about 50 ul each)
#Place a QIAquick spin column in a provided 2 ml collection tube.
#To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min at 6000 rpm.
##For the Bs, do the same process using the same 2 ml collection tube/spin column
##Do the same for the Ss
###Basically, we are combining the DNA for 1-4.
#Discard the flow-through and place QIAquick column back into the same tube.
#To wash QIAquick column, add 750 µl of Buffer PE and centrifuge for 1 min at 6000 rpm.
#Discard the flow-through and place the QIAquick column back in the same tube, which should be empty. Centrifuge for an additional 1 min at 13,000 rpm (~17,900 x g).
#Place the QIAquick column in a clean 1.5 ml microcentrifuge tube.
#To elute DNA, add 100 ul of Buffer EB (10 mM Tris·Cl, pH 8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min at 13,000 rpm (~17,900 x g). Alternatively, for increased DNA concentration, add 30–50 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge.
<br>
After nucleotide removal, we did Nanodrops and found that the 10mer had 389.8 ng/uL, the 15mer had 278.6 ng/uL, and the 20mer had 168.7 ng/uL. 
----------------------------------------------------------------------------------------------

===Transformation of MIT 2005 Constructs===
#Thaw the 4 tubes of cells on ice and mix gently to ensure that the cells are evenly suspended.  We will be using the following DNA:
##J07017 - FecA (4P AMP) - Plate 2
##J07022 - FecI (3B Amp) - Plate 2
##J07021 - FecR (4D Amp) - Plate 2
#Add 1 µl of the DNA solution directly to the Top10 cells after thawing Top10 cells on ice for 5 minutes. Tap gently to mix.
#Place the tubes on ice for approx 30 minutes
#Heat the tubes for about 45 seconds in a 42°C water bath; do not shake
#Place on ice for 2 minutes
#Add 250 µl of room temperature SOC Medium to each tube
#Shake at 37°C (200 rpm) for 60 minutes.
#Light the bunsen burner
#Using petri dishes containing LB agar w/ Ampicillin, plate using 100 uL of cells
#For each plate, use sterile beads to spread the bacteria over the mix.
#Let each plate stand until end of the day
#Afterwards, turn the plates upside down and incubate at 37°C overnight

----------------------------------------------------------------------------------------------

===Extension of the 10mer, 15mer, and 20mer Library===
Basically, this was done in preparation for the ligation reaction that we will do with the massive amounts of DNA.
#Take the dry 15mer and 20mer library in the tubes labeled MNS32 and MNS33, respectively, and spin it down for about 3 seconds
#Resuspend the 15mer and 20mer library with 50 µL of nuclease-free water. The concentrations of these libraries are now around 420 and 380 µM, respectively.
#Add 10 µL of nuclease-free water to the 10mer library (labeled MNS20). The concentration of this library is now around 600 µM.
#Prepare a PCR tube for the extension reaction of the 10mer library:
##7 µL of the PCR polymerization reaction master mix (found in the refrigerator along the back wall of the small room. The bottle is labeled Amplitaq Gold)
##3 µL of the reverse primer (found in the square green box labeled IGEM2007 right next to the door of the small room)
##2 µL of the template 10mer library
##2 µL of the nuclease-free water
#Prepare one PCR tube each for the extension reaction of the 15mer and 20mer library:
##7 µL of the PCR polymerization reaction master mix
##3 µL of the reverse primer
##4 µL of the template 15mer or 20mer library
#Put all three tubes in the black PCR machine with two lids in the small room. Run program "IGEM2."
#Return reagents to said locations

------------------

===Digestion of the 10mer, 15mer, and 20mer Library, along with vectors from week 0===

<br>
For all samples:<br>
#50uL elution
#0.7uL 100x BSA
#7uL 10x NEB2
#6.3uL H2O
<br>
For vectors: 3uL NheI, 3uL PstI<br>
For library: 3uL SpeI, 3uL PstI<br>
<br>
Samples were left in 37C incubator all night - we plan to run the nucleotide-removal on them to remove enzymes tomorrow.

------------------

===Growing Liquid Cultures of the QS parts===
We grew a large amount because we want to carry out a midiprep tomorrow
#Take 3 500 mL flasks from the cabinet in the big room with the house on the chalkboard and the person saying "Yaye"
#Put 150 mL of LB and 150 µL of ampicillin (all the parts are ampicillin resistant)
#Put into the agitating incubator overnight

------------------

===Nucleotide Removal===
#Add 10 volumes of Buffer PN to 1 volume of random library extensions (single primer) and mix.
##10-mer, 15-mer, and 20-mer extentions (about 70 ul each)
#Place a QIAquick spin column in a provided 2 ml collection tube.
#To bind DNA, apply the sample to the QIAquick column and centrifuge for 1 min at 6000 rpm.
##For the Bs, do the same process using the same 2 ml collection tube/spin column
##Do the same for the Ss
###Basically, we are combining the DNA for 1-4.
#Discard the flow-through and place QIAquick column back into the same tube.
#To wash QIAquick column, add 750 µl of Buffer PE and centrifuge for 1 min at 6000 rpm.
#Discard the flow-through and place the QIAquick column back in the same tube, which should be empty. Centrifuge for an additional 1 min at 13,000 rpm (~17,900 x g).
#Place the QIAquick column in a clean 1.5 ml microcentrifuge tube.
#To elute DNA, add 100 ul of Buffer EB (10 mM Tris·Cl, pH 8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min at 13,000 rpm (~17,900 x g). Alternatively, for increased DNA concentration, add 30–50 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge.

----------------------------------------------

<p style="text-align: right;">Protocol Recorded by Kevin Shee</p>