=Cell by Date: Notes=

__NOTOC__

<div class="tabs-blue">
<ul>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell By Date|Introduction]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/Specifications|Specifications]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/Design|Design]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/Modelling|Modelling]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/Implementation|Implementation]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/TestingValidation|Testing/Validation]]</li>
<li id="current">[[IGEM:IMPERIAL/2007/Projects/Cell by date/Notes|Notes]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/References|References]]</li>
</ul>
</div>
<br style="clear:both">

__TOC__




===Experimental design===

*Temperature increases the rate of reactions - increased temperature means faster gene expression over a given time.
*System only deals with protein synthesis and its reporter function - a visual signal.
*Coupling firefly genes with that of a well-characterized consitutive promoter, response is good.
*We will then calibrate the synthesis of fluorescent protein (which is easier as opposed to calibrating cell density) with that of the predicted shelf-life (date label).
**Assuming the cold chain process only occurs at 4oC, it will lead to the predicted outcome.
**Any temperature increases, and over a threshold would lead to over the range fluorescence.
**Degradation of protein is minimal with fine-tuning of cell free medium characterization
**Proof of principle of the application of CFE.


===Fluorescent Signal===

* The first problem is that a fluorescent signal will only be visible if a cell population is at a certain density. This is because there needs to be a great enough density of cells expressing a fluorescent protein so that there is a strong enough signal to be visible by eye. This will be a problem if the cells are exposed to varying conditions early on in the sell by date when cell population is low.
* The fluorescent proteins expressed will also need to be relatively stable from degradation. This is because if we initiate the expression of the fluorescent proteins then we want the signal to be sustained so that our 'cell-by-date' has a memory.
**A further point on this is that we can't use GFP as a reporter in hypoxic conditions GFPHypoxic
**Possible solution to this problem is to use a reporter with a long Half-Life so that there is enough time for someone to see the signal being outputted by our system. EGFP Seems to be one such reporter having a half life that may exceed 24hours1. Enocding vector found for EGFP that places it under control of Plac promoter4. DsRed seems to be another possible reporter to use3.

*In order to avoid complication in design we have opted to just use one reporter, RFP, instead of using two eg. transition from GFP to RFP when problems occour. This is due to time constraints - it make take time for the RFP to overcome the GFP.



==E.coli==

===Level 1===

Calibration: 

The process criteria for minced meat and mechanically separated meat are:

5 samples per sampling session 

Aerobic Plate Count (APC) and E. coli (EC) specified limits per gram

APC all 5 samples must be less than 5 x 106 cfu/g and 3 samples must be less than 5 x 10 5 cfu/g.

EC all 5 samples must be less than 500 cfu/g and 3 samples must be less than 50 cfu/g. 
[http://www.ukmeat.org/Assessment.htm Info obtained here]

==In-Vitro==

===Level 1===
===Level 2===
===Level 3===

==In-Veso==

===Level 1===
===Level 2===
===Level 3===

==Second stage: Calibration and Measurement of Fluorescence + Absorbance==
===Abstract===
'''1. Fluorescence'''
This involves the use of the fluorometer to measure fluorescence of a reporter gene (GFP), under the control of a range of promoters (pTet, pBad, pT7), at various temperatures (4<sup>o</sup>C, 25<sup>o</sup>C, 37<sup>o</sup>C, etc.), over 1h time intervals for 24 hours (or however it is deemed fit).

Calibration:
*fluorescence vs. intracellular [GFP]
*intracellular [GFP] vs. lysate [GFP]
*Degradation [GFP] - half-life of 24 hours
'''2. Absorbance'''
In addition, the culture is tested for absorbance use spectrophotometry to determine cell density at 600nm.

Calibration: 
*Dilution plating (or other methods of cell counting) vs. optical density (at 600nm)


===Introduction===
*Aim is to create a more generic unit to allow for modular design and easier repetition of experiments - '''GFP molecules synthesized cfu<sup>-1</sup> s<sup>-1</sup>'''.
*Required for absorbance and fluorescence measurements.


''1. Absorbance@600nm vs. Colony Forming Units (CFU)''

[[IMAGE: Absorbance_calibration_curve.jpg|thumb|right|400px|Frame| Example of Absorbance Calibration Curve ]]

Absorbance of the cultures in the wells are measured to allow to convert our absorbance raw data into a cell count, so that cell we can relate GFP synthesis ''per cell''. As cultures of ''E. coli'' are grown to various cell densities, a sample of these cultures are taken and OD<sub>600</sub> measured. Dilution plating is then carried out to work out approximate CFU ml<sup>-1</sup> of culture. This data set allows the conversion of absorbance levels at OD<sub>600</sub> to cell density measured by CFU ml<sup>-1</sup>. 

Limitations of calibration:<br>
*Sources of error in plate counting (The Great Plate Count Anomaly)
*Different strains of bacteria will require different calibration curves.

Potential Solution is the use of '''[[IGEM:IMPERIAL/2007/Projects/Cell_by_date/Implementation/Flow Cytometry|Flow Cytometry]]'''
<br clear="all">

''2. Fluorescence vs. intracellular [GFP]''

[[IMAGE: Calibration_Curve2.JPG‎ |thumb|right|400px|Frame| Example of Fluorescence Calibration Curve ]]

Calibrating intracellular [GFP] to that of fluorescence is tricky. This will however allow us to relate [GFP] to fluorescence readings, and would provide us with valuable information for the modelling studies. Three curves are required to describe this data set.

*'''''Fluorescence vs extracellular [GFP] molecules'''''. Known concentrations of GFP are diluted into a range of dilutions and are then mixed with cell lysate of a particular chassis.

*'''''Fluorescence vs Cultures of varying unknown intracellular [GFP]'''''. Measuring the fluorescence of cultures of cells at set time intervals would measure a set of unknown intracellular [GFP]. For our objectives, we would need to compare this unknown concentration in terms of its extracellular content. This problem can be circumvented with the following experiment.

*'''''Fluorescence vs Cultures of varying unknown extracellular [GFP]'''''. Lysing samples of Experiment 2 will allow the relation of an unknwon intracellular [GFP] to that of extracellular [GFP]. The results of comparing Experiments 2 & 3 can then be inferred with respect to the calibrations made in experiment 1 where we used known concentrations of GFP. The result is the correlation of intracellular [GFP] to that of fluorescence observed. This is important to the creation of our generic unit of measure.
<br clear="all">


===Materials & Methods===
*'''[[IGEM:IMPERIAL/2007/Projects/Cell_by_date/Implementation/List of Required Parts|List of Required Parts]]''' to be tested under the conditions stated above.
*'''[[IGEM:IMPERIAL/2007/Projects/Cell_by_date/Implementation/Calibrating Absorbance|Calibrating and Measuring Absorbance]]'''.
*'''[[IGEM:IMPERIAL/2007/Projects/Cell_by_date/Implementation/Calibrating Fluorescence|Calibrating and Measuring Fluorescence]]'''.

====Fluorometer====
*'''Fluorometer Reading'''
**aka. 'Twinkle' - plate reader fluorometer that measures 96 well plates
**It is hoped that absorbance@600nm and fluorescence emisssion of GFP & DsRed-Express can be measured at set time intervals
***We can thus derive '''GFP molecules synthesized cfu<sup>-1</sup> s<sup>-1</sup>'''
**To be installed on 3<sup>rd</sup> Aug @10:30am by engineer.
**General protocols for fluorometer usage to be finalised.
*'''Digital Camera''' 
**To take digital pictures of our results. Might not be useful to GFP, but colour scheme can be done for DsRed-Express.



===Results===
A summary of the stage results. More details in lab notebook.

==Third stage: Calibration and Measurement of Fluorescence + Absorbance with DsRed-Express==
===Introduction===
Similar to second stage, but with the reporter gene as DsRed-Express. Preparation of cultures to be done in conjunction with Hrp Characterization System.

Calibration to be done with fluorescence for DsRed-Express as well (Note response time).


===Materials & Methods===

*'''[[IGEM:IMPERIAL/2007/Projects/Cell_by_date/Implementation/List of Required Parts|List of Required Parts]]''' to be ligated into plasmid.
*'''[[IGEM:IMPERIAL/2007/Projects/General Protocols | General Protocols]]''' for digestion, ligation, gel electrophoresis, gene cloning, electroporation, competent cells etc. and the materials required for each.
*Fluorometer Reading
*Mass Spectrometry

===Results===
A summary of the stage results. More details in lab notebook.

==Provisional Timeline==

====Week 4====
*Stage 1 Preparation of Solutions required (2 Days)
*Stage 1 Preparation of cloning DNA constructs (in parallel) / ''together with in veso team''

====Week 5====
*Stage 1 Cloning of DNA constructs and Transformation (2 Days) / ''together with in veso team''
*Stage 2 Measurement of time-resolved fluorometry and absorbance using GFP over different temperatures (3 Days)
*Stage 2 Calibration of fluorescence and absorbance (in parallel with measurement) / ''together with Hrp team''

====Week 6====
*Stage 2 Modelling Studies on quantitative data collected. Objective is a predictive model that can relate temperature fluxes to increased GFP production with no error for false positives (2-3 Days)
*Stage 2 Testing and validation of predictive model and use of varying temperature conditions (3-4 Days)

====Week 7====
*Stage 2 Testing and validation of predictive model and use of varying temperature conditions (cont'd)
*Focus now shifts to DsRed-Express and to different chassis (in vitro/ in veso)
*Stage 3 Cloning of DNA constructs and Transformation (2 Days) / ''together with Hrp team''
*Stage 3 Measurement of time-resolved fluorometry using DsRed-Express over different temperatures. Absence of cell growth means cell population is no longer a varying parameter; absorbance omitted. (3 Days)
*Stage 3 Calibration of fluorescence of DsRed-Express (in parallel with measurement)

====Week 8====



====Week 9 & 10====
*Polish on project
*Review modelling studies and quantitative data
*

==Proposal for Lab==

====First stage: Preparation of vectors and E.coli cultures====

This involves a series of cloning and transformation experiments (e.g. digestion, ligation, transformation). Agarose gel electrophoresis will then be used to test for products formed. Miniprep will be performed to extract the plasmid (containing our construct) from the cells, to be used in the other stages.

This is done in conjunction with In-veso group (using same gene constructs and plasmids).

====Second stage: Calibration and Measurement of Fluorescence + Absorbance====

=====Flurorescence=====
This involved the use of the fluorometer to measure the fluorescence of a reporter gene (GFP), under the control of a range of promoters (pTet, pBad, pT7), at various temperatures (4<sup>o</sup>C, 25<sup>o</sup>C, 37<sup>o</sup>C, etc.), over 1h time intervals for 24 hours (or however it is deemed fit).

Calibration:
-Different [GFP] vs. fluorescence

-intracellular [GFP] vs. lysate [GFP]

-Degradation [GFP] - half-life of 24 hours


=====Absorbance=====
In addition, the culture is tested for absorbance use spectrometry to determine cell density at 600nm.

Calibration: 
Dilution plating (or other methods of cell counting) vs. optical density (at 600nm)

====Third stage: Calibration and Measurement of Fluorescence + Absorbance with DsRed-Express====

Similar to second stage, but with the reporter gene as DsRed-Express. Preparation of cultures to be done in conjunction with Hrp Characterization System.

Calibration to be done with fluorescence for DsRed-Express as well (Note response time).


----


===27/07/07===
Busy day today need to finish scheduling for weeks ahead



2. Clean up wiki: Specification are still vague , Design page contains expt / implementation data

3. Contact other universities about meeting up

4. Determine HSA standards for initial contamination

5. Determine Temperature range that our system has to operate in

6. Flowchart for modelling:
**Need to do some preliminary experiments to set up a mathematical model
***Data Analysis on preliminary experiments will reveal and equation which is our model
**With model formed we need to validate by doing further experiments 


'''Specifications'''- quantitative data -> format ok.
-get those HSA standards and assumptions... page on more about challenge testing.


'''Design'''- expand levels 1 2 3.
- List of required parts - promoters, RBS, Te, reporter genes (B-gal,  luciferase, DsRed, acGFP)
- how is it going to work? - Process that it would work - define 100% by exposure to 4deg over 24 hours. any higher temp mean we reach 100% quicker.


Modelling - Ant
Using Arrhenius Equations... bla bla


-change implementation page - Alex
List of Protocols - Collate them - Incubation... transformation bla bla


Testing/Validation
- skeleton - what experiments, what resultsdo we hope to achieve? Results

===25/07/07===
* Feedback from Profs:
**Good Application
**Would like to see CellByDate applied to all types of Meat
**would like to see more focused towards sythetic bio, make CellByDate resusable as a generic accumulator/integrator, taking 
Pops in and given Pops out ( possibility ?)

*Suggest that by end of tomorrow wiki is completed updated with approved & rejected ideas
*Suggest that we start up journal club tomorrow again
*Need to look into CFE for Cell By Date , consider realising what professors said
*By End of tomorrow should have planned out rest of IGEM
===Cell-by Date (Now Refocused - See Main Project Page)===

====Summary====
Fresh meat is a highly perishable food product and unless appropriately actions are taken, e.g., packaged, transported and stored at refrigeration temperatures, can spoil in relatively short time. Factors affecting meat spoilage include intrinsic (e.g., pH, aw, composition, type, and extent of initial contamination) and extrinsic parameters (e.g., temperature and  packaging atmosphere). 

Among these, temperature is considered the most important factor. Although most countries have established regulations with maximum temperature limits for refrigeration storage, in practice these are often violated. Survey studies have shown that temperature conditions higher than 10°C are not unusual during transportation, retail storage, and consumer handling. Such temperature abuses during any stage of the chill chain may result in an unexpected loss of quality and a significant decrease of meat shelf life.

Surveys conducted also show that 1 in 3 consumers actually use food that is past the expiry date (this has been "verified" across the imperial igem team). Consumer ignorance, together with the lack of transparency in food processing and packaging are perhaps the two main factors for the 76,000 cases of food poisoning reported in the UK in 2006, many more presumingly under-reported. 

The huge task at hand is whether or not we can find an application in synthetic biology to solve this modern-day crisis - a better indicator to distinguish meat that is fit for consumption rather than visual and olfactory examination (or the date that is written on your food labels that no one really reads). 

Challenge tests are the main current method used by the meat industry and academia to evaluate product’s shelf life. This is however valid only to the conditions that are tested in the lab, clearly impractical for a multitude of reasons.

====Specifications====
* No contact with the food.
* Want at least to inputs into our system in an 'OR' gate.
* We would like a visual signal.
* System must be stable for the lifetime of the food.
* Need system to be able to work in a suitable location, e.g. if food item is in a fridge.

====Design (now re-focsed see below)====
A preictive microbiological sensor, this has two aspects to it;

# We aim to design a biological prediction method to predict the microbiological growth within a food sample. We plan to achieve prediction by designing a system that starts with a defined low density population of E.coli and then allow this to grow under similar conditions to those found in the food, e.g. pH and temperature. Once the cell density reaches a density level that would be typical of that found in spoiled food, a fluorescent signal will be triggered. 
# The second aspect of our 'cell-by-date' system is to monitor the temperature that the food is exposed to. In various studies, temperature has been shown to be the most significant factor in the spoiling of foods. We plan to manipulate the heat shock promoter to allow us to trigger a fluorescent signal when exposed to a set temperature and so to show that the food has been exposed to a particular temperature.

====Re-Focused Design (Thanks to supervisors)====

#No More population density measurement
##Live, cant comply with standards
##Non-integrative function - no processive power
##Problems with noise; GFP degradation in a live system
##Difficulty in integration of both inuputs of cell density and temperature activation
##Requirement for a more elegant design

Re-focussed design:
#Based on in-vitro expression of cell-free extract (CFE) medium to comply with HSA standards (no bacteria next to food)
#CFE are known to be stable of up to 4 days - in working range of highly perishable foods
#Ability to perform gene expression that would lead to a cumulative response in the form of a visible, stable protein.
#Use of firefly genes cf. GFP. More stable and visible over a range of colour spectrum.
#Colour schemes provide more information to consumers - Focus on supermarket-fridge-consumption phases.  Sets out to answer the question: Is this good to eat ? Looks good semells good but...

Experimental design:
#Temperature increases the rate of reactions - increased temperature means faster gene expression over a given time.
#System only deals with protein synthesis and its reporter function - a visual signal.
#Coupling firefly genes with that of a well-characterized consitutive promoter, response is good.
#We will then calibrate the synthesis of fluorescent protein (which is easier as opposed to calibrating cell density) with that of the predicted shelf-life (date label).
##Assuming the cold chain process only occurs at 4oC, it will lead to the predicted outcome.
##Any temperature increases, and over a threshold would lead to over the range fluorescence.
##Degradation of protein is minimal with fine-tuning of cell free medium characterization
##Proof of principle of the application of CFE.

===== Problems =====

# Modelling:
## Need memory/integration Can't Give RFP in response to a pulse
## Does system act as an effective integrator ?
# Part Specific Design
## How many parts available in registry
# Does system really have WoW ! Factor ?
## What additional components could change this ?
# Promoters : Heat Schock induction
# Timeline - Proposal needs schedule for project
# Initial Cell Density enough for a GFP response ?
# Commercially available alternative ?
# NOVELTY : September 2002 - Problem seems to have been already tackled by material scientists [http://www.advance.uri.edu/pacer/september2002/story8.htm MatSciCellByDate]<br>
##Field name may be SSM (Stimuli Sensitive Materials) [http://64.233.183.104/search?q=cache:V1PlCUfkpoMJ:www.ntcresearch.org/pdf-rpts/AnRp01/M98-A16-A1.pdf+Heat+Sensitive+Materials&hl=en&ct=clnk&cd=5&gl=uk SSM]<br>
##Intelligent colling systems in addition to predictive microbiology could mean our system is unneccessary [http://aiche.confex.com/aiche/2005/techprogram/P14844.HTM IntelligenCooling] suggest refocus on Average Joe and food in his fridge<br>
## Antimicrobial Agents have been developed which prevent the formation of spoiling microbes during transport [http://www.blackwell-synergy.com/doi/pdf/10.1111/j.1750-3841.2006.00103.x AntiMicrob]<br>

===== Modelling =====
The main problem associated with modelling is that we would wish to model cell growth at varying temperatures. There are many models that are available for cell growth, however we would need to expand on these and include the additional varibale temperature.

===== Fluorescent Signal =====
* The first problem is that a fluorescent signal will only be visible if a cell population is at a certain density. This is because there needs to be a great enough density of cells expressing a fluorescent protein so that there is a strong enough signal to be visible by eye. This will be a problem if the cells are exposed to varying conditions early on in the sell by date when cell population is low.
* The fluorescent proteins expressed will also need to be relatively stable from degradation. This is because if we initiate the expression of the fluorescent proteins then we want the signal to be sustained so that our 'cell-by-date' has a memory.
**A further point on this is that we can't use GFP as a reporter in hypoxic conditions [http://cancerres.aacrjournals.org/cgi/content/abstract/61/12/4784 GFPHypoxic] <br>
**Possible solution to this problem is to use a reporter with a long Half-Life so that there is enough time for someone to see the signal being outputted by our system. EGFP Seems to be one such reporter having a half life that may exceed 24hours<sup>1</sup>. Enocding vector found for EGFP that places it under control of Plac promoter<sup>4</sup>.  DsRed seems to be another possible reporter to use<sup>3</sup>.<br>
* In order to avoid complication in design we have opted to just use one reporter, RFP, instead of using two eg. transition from GFP to RFP when problems occour.  This is due to time constraints - it make take time for the RFP to overcome the GFP.<br>


<biblio>
#1 pmid=11313827
#2 [http://www.blackwell-synergy.com/doi/pdf/10.1046/j.1365-2672.2001.01344.x?cookieSet=1 ImplementationOfEGFP]
#3 [http://www.mpibpc.gwdg.de/abteilungen/200/publications/pdf/FEBS_Lett._479_131-135.pdf DsRed]
#4 [http://66.102.1.104/scholar?hl=en&lr=&q=cache:2LQnsulowsUJ:www.bdbiosciences.com/clontech/archive/JAN97UPD/PDF/JAN97TechTip.pdf+EGFP+ecoli EGFP promoter]
</biblio>

===== Constructs =====
#The heat shock promoter is a slow protein to expresss. Studies have shown that it takes 5 hours for the promoter to go from low to high, meaning for the maximum output of PoPS from the heat shock promoter will take 5 hours.

===== Tests =====

*All sensory operations would occur within the environment of the fridge, which is maintained at a temperature of about 4 degrees celsius.
*Food spoiling processes take a long time and are hard to quantify.
*food spoilage constitutes an array of microflora, under many parameters (incl. various ways of packaging meat.

==== Solution ====
*Concept of predictive microbiology
**involves knowledge of microbial growth responses to environmental factors expressed in quantitative terms by mathematical equations (in other words, extensive modelling).
**data and models can be stored in databases and used to interpret the effect of processing, distribution, and storage conditions on microbial growth.

*the combination of data on the temperature history of the product and mathematical models may lead to “intelligent” product management systems for the optimization of food quality and safety at the time of consumption.

*we also know a great deal about the colonization of bacterial spp. under varying conditions of temperature and pH. Effects of Photobacterium phosphoreum, pseudomonads, Shewanella putrefaciens, and Brochothrix thermosphacta, have been published. 
** In particular, Psudeomonads are a strong spoilage index indicator due to its quick growth response under increased temperature

Therefore, using research modelling as a means of input to specify the growth conditions of our isolated bacteria, we could develop a temp + pH sensitive bacteria that estimates the time frame in which meat would go bad (using quorum sensing and controlled cell growth), with GFP as an output. 

Why consider the project? 
*Modelling saves lots of time (eventually the parameters will be used as an input to control and regulate cell growth of E.coli)
*Potential for extensive use in the F&B industry
*parts that are required can be easily obtained, and we could also recycle the previous proposals (so our initial efforts do not go to waste)
*Gimmicky project name tag - easily received and marketable (since everyone faces the same problems, avg. joe and microbiologists alike)
*Potential use of Hrp system? (temperature activation may require a quick response time to generate increased growth rate responses)

==== Potential problems ====
*Is this synthetic biology?! (as opposed to the classic gene cloning and insertion into different chassis)
*spoilage models remain a research tool rather than an effective industrial application due to several reasons:
**developed models were based on observations in a well-controlled laboratory environment with microbiological media. Predictions based on such models are not necessarily valid in complex food environments such as meat.
**lack of information required for the application of models for predicting the shelf life of specific food products
**environmental factors used; lag phase of bacterial growth seldom considered.
**Industrial denial and lack of consumer confidence?

*Knowledge expertise on predictive microbiology and models
**2006 paper found on recent developments of models that take into account the previous problems - can we apply it as an output to stimulate cell growth?
**Input parameters: only considering temperature at the moment, and potentially pH (pH is important for the Pseudomonads spp.)
**How are we going to go about testing the purported meat spoilage model?


====Notes====




====References====

<biblio>
#1 pmid=8913810
#2 T.A. McMeekin, Predictive microbiology: Quantitative science delivering quantifiable benefits to the meat industry and other food industries, Meat Science, Volume 77, Issue 1, September 2007, Pages 17-27 
#3 K. Koutsoumanis, P.S. Taoukis and G.J.E. Nychas, Development of a safety monitoring and assurance system for chilled food products, International Journal of Food Microbiology 100 (2005), pp. 253–260. 
#4 pmid=8913812
#5 pmid=16391034
#6 pmid=2404279


</biblio>