==Description==

Create bacterial scents/nice smelling bacteria for multiple bacterial applications, as well as scented yeast, etc. Possibly control this mechanism through light, heat, concentration sensitivity.

==Requirements==
===Design/Construction/Testing===

1. Transfer known coding region/enzyme for up to four scent producing proteins into ''E.coli'' bacteria first

2. Try to develop pathway control for intracellular creation of salicylic acid, benzoic acid, jasmonic acid, cinamic acid
* Test different acid concentrations in media for optimal concentration. Want minimum cell death and maximum desired scent
* Could use heat, light, etc. to monitor enzyme activity

3. Transfer scent coding region into yeast and other cellular organisms
* Experiment with bread, beer brewing

==Feasibility & Timeline==
The beginning phases of the project include the insertion of the enzyme into E. coli, which has already been done. This first step will take at most 2-3 weeks, giving lots of room to expand into the other possiblities for the remainder of the project timeline.

==Parallelizability==
The team can work together at first to put the enzyme in E. coli. With that first step done, it will be easy to run multiple parallel subprojects to tackle different applications in yeast, plants, sawdust, etc.

==Intermediate goals==
The first goal is to acheive results in E. coli. Following that, S. cerevisiae is a good candidate for a solid extended application (which will run concurrently with other subprojects).

==Significance==

The goal would be a working system in which we inserted genes encoding enzymes that develop delightful scents into the genomes of E.coli, yeast, etc.

Using this system in E.coli would allow for more pleasant working environments for researchers around the world. Using it in Streptococcus mutans could allow for the development of an added bonus to already FDA approved bacterial mouthwash that is currently in clinical human trials. 

Mutated yeast could be used for developing scented bread and beer. 

Also, a biological "Glade Plug-in" with multiple scent producing capacity could be tested and fine tuned to have
sensitivity to different environmental controls (i.e. light, heat)

And sawdust containing harmless bacterial cells designed to perfume smelly stables could be possible if system designed to
respond to urine's salicylic acid

Create registry parts for each enzyme, etc.

==Safety & ethics==
The extended applications in humans (mouthwash etc.) pose potential ethical issues; however, the scope of the project will not encompass testing in humans.

==Summary (pros/cons)==
There are various advantages to the project, including: 

1. Appealing demo possibilities<Br>
2. Strong case for extended commercial applications, including everyday uses<Br>
3. Solid previous work that shows proof of concept<Br>
4. Room for additional controls (light, heat, sense)<Br>

However, the difficulty is that there has been a great deal of previous work. To distinguish the project, it would be necessary to fine tune controls, which may make the project ultimately as complex.

==References==
<biblio>
#Negre-ArchBiochemBiophys-2002 pmid=12361714
#Ross-ArchBiochemBiophys-1999 pmid=10375393
#Pott-PlantPhysiol-2004 pmid=15310828
#Zubieta-PlantCell-2003 pmid=12897246
</biblio>

==Bacterial scents==

{| border="1"
|-
|'''Precursor'''
|'''Enzyme'''
|'''Compound'''
|'''Scent'''
|'''References'''
|-
|benzoic acid & ''S''-adenosyl-L-methionine (SAM)
|S-adenosyl-L-methionine:benzoic acid carboxyl methyltransferase (BAMT)
|[[Wikipedia:Methyl benzoate|methyl benzoate]]
|pleasant smell
|<cite>Pott-PlantPhysiol-2004</cite>
|-
|''trans''-cinnamic acid & ''S''-adenosyl-L-methionine (SAM)
|?
|[[Wikipedia:Methyl cinnamate|methyl cinnamate]]
|cinnamon?
|
|-
|jasmonic acid & ''S''-adenosyl-L-methionine (SAM)
|S-adenosyl-L-methionine:jasmonic acid carboxyl methyltransferase (JMT)
|[[Wikipedia:Methyl jasmonate|methyl jasmonate]]
|jasmine
|
|-
|salicylic acid (SA) & ''S''-adenosyl-L-methionine (SAM)
|S-adenosyl-L-methionine:salicylic acid carboxyl methyltransferase (SAMT)
|[[Wikipedia:Methyl salicylate|methyl salicylate]]
|wintergreen
|<cite>Ross-ArchBiochemBiophys-1999, Negre-ArchBiochemBiophys-2002, Zubieta-PlantCell-2003</cite>
|}

===Expert advice===
From Natalia Dudareva, Purdue University:
*thinks that you can smell wintergreen from E. coli cultures expressing SAMT with salicylic acid in the media

From [http://www.biology.lsa.umich.edu/research/labs/pichersky/ Eran Pichersky], University of Michigan:
*E. coli cultures expressing SAMT with salicylic acid in the media will have a detectable wintergreen smell
*eliminate indole pathway (responsible for bad E. coli smell) to strengthen the scent.
*have shown production of several scent compounds in E. coli

===SAMT===
*''C. breweri''
**DNA and protein sequence known
**Expressed in ''E. coli''
**Methyl salicylate has been extracted from spent medium of ''E. coli'' cells when medium was supplemented with salicylic acid
**Genbank AF133053
**also can use benzoic acid as a substrate but with lower efficiency
**crystal structure available
*''A. majus'' (Snapdragon)
**DNA and protein sequence known
**Expressed in ''E. coli''
**Methyl salicylate has been extracted from spent medium of ''E. coli'' cells when medium was supplemented with salicylic acid
**also can use benzoic acid as a substrate but with lower efficiency
**Methyl benzoate has been extracted from spent medium of ''E. coli'' cells when medium was supplemented with benzoic acid
*''S. floribunda''
**Genbank AJ308570
*''A belladonna''
**Genbank AB049752

===JMT===
*''A. thaliana'' AY008434

===BAMT===
*Snapdragon AF198492

===References===
<biblio>
#Negre-ArchBiochemBiophys-2002 pmid=12361714
#Ross-ArchBiochemBiophys-1999 pmid=10375393
#Pott-PlantPhysiol-2004 pmid=15310828
#Zubieta-PlantCell-2003 pmid=12897246
</biblio>

===Terpenes and terpenoids===
*Terpenes are hydrocarbons: combinations of several isoprenes.  (Sometimes encompasses terpenoids.)
*Terpernoids are modified terpenes with methyl groups added/removed or oxygens added
*From Wikipedia: "Terpenoids contribute to the scent of eucalyptus, the flavors of cinnamon, cloves and ginger and the color of yellow flowers. Well-known terpenoids include citral, menthol, camphor and the cannabinoids found in the Cannabis plant."

E. coli has the &Delta;<sup>3</sup>-isopentenyl-pyrophosphate pathway, and the enzymes to produce geranyl-PP.  This pathway is less effective than the mevalonate pathway, but this has been cloned into an E. coli strain by Keastling's group.   Many scented compounds can be made from isopentenyl-PP and geranyl-PP with one or two enzymes, including lemon, orange, pine, etc.  See [http://www.ecocyc.org Ecocyc] for the pathways (type IPP as a compound and look at the synthetic and reactant pathways that link to it).

Terpenes are also the precursor to rubber and many of the resins and gums.

===Indole elimination===
Indole is the precursor to and degradation product of tryptophan.  We could knock out the relevant two enzymes and supply tryptophan exogenously.  Also, we could supply tryptophan exogenously and see if that is sufficient to inhibit indole formation via feedback inhibition in a "normal" strain. [from TK]

[http://biocyc.org/ECOLI/NEW-IMAGE?type=COMPOUND&object=INDOLE Indole] can act as an extracellular signal so indole can probably get in and out of the cell.

====Relevant reactions====
In Pathway Reactions as a Reactant:

[http://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&object=TRPSYN-PWY tryptophan biosynthesis] :
indole + L-serine = L-tryptophan + H2O

In Pathway Reactions as a Product:

[http://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&object=TRPSYN-PWY tryptophan biosynthesis] :
indole-3-glycerol-phosphate = indole + D-glyceraldehyde-3-phosphate

[http://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&object=TRYPDEG-PWY tryptophan degradation II (via pyruvate)] :
L-tryptophan + H2O = indole + pyruvate + ammonia

====Relevant enzymes====
trpB (biosynthesis) and tnaA (degradation)

====References====
<biblio>
#Freundlich-JBacteriol-1960 pmid=13701820
#Phillips-ArchBiochemBiophys-1992 pmid=1632641
</biblio>

===Enzyme/Metabolic Engineering===
Here are two references worth studying (both from Jay Keasling's lab).  The first involves tracking down the right version of an enzyme from a plant.  The second involves the "evolution" of an enzyme for different purposes.
<biblio>
#YeastEngineering pmid=16612385
#EnzymeEvolution pmid=16495946
</biblio>