== Introduction ==

== The Molecular Biologist Toolbox Kit ==

=== Restriction Enzymes ===
Restriction enzymes are a class of enzyme that can causes specific breaks in a DNA molecule.The mechanism by which restriction enzymes cut DNA is by causing hydrolysis of the phosphodiester bonds between the nucleotides that make up DNA.
<br>
Restriction enzymes can be broadly split into two types:
*Exonucleases cleave DNA from the ends of the DNA molecule. The action of exonucleases will remove nucleotides from the ends of DNA.
*Endonucleases cleave within the interior of the DNA molecule. The action of endonucleases will cause fragmentation of DNA molecules.

=== Endonucleases ===

[[Image:Endonucleaserestrictionsite.jpg|frame|Some examples of restriction sites]]

Endonucleases are essential for recombinant DNA technology because they allow specific genes to be cut out and released from within a DNA molecule. The specificity of endonucleases is determined by a specific sequence of nucleotides that they can recognise and cleave. 
This sequence of nucleotides is called the restriction site and will vary between different types of endonucleases.
<br><br>
The key features of the restriction sites are the sequence of nucleotides, sequence length and site of cleavage. In addition most of the restriction sites are palidromic, meaning that they read the same forwards and backwards. If this is the case then there is cleavage of both strands of DNA that make up the DNA molecule. 
<br><br>
Once a DNA molecule has been cleaved two fragments will be resolved. Each of these fragments will contain an 'end' that is from the cleaved restriction site. These ends can vary, depending on the exact position of the bonds cleaved. In general there are two broad types of ends resolved from fragmentation:
*Blunt ends where the two strands of DNA are the same length, and so there is no overhanging strand. This types of ends occur when the cleavage sites on both strands are directly opposite each other.
*Sticky ends where one of the stands of DNA is longer and so overhangs the other. This type of end occurs when the cleavage sites on both strands are not opposite each other. The examples shown in the diagram produce this types of end.
<br clear = "all">

== Recombinant DNA ==

[[IMAGE:Recombinant_DNA.jpg|frame|Inserting DNA of interest into the plasmid]]

Recombinant DNA is the construction of new combinations of unrelated genes. These novel combinations can be cloned and amplified by introducing them into host cells where they are replicated by the DNA-synthesizing machinery of the host.	A DNA fragment of interest is covalently joined to a DNA vector.	A vector can replicate autonomously in an appropriate host. Plasmids (naturally occurring circles of DNA in bacteria) and bacteriophage λ (a virus) are common vectors for cloning in E.coli. 


The vector is prepared for accepting a new DNA fragment by cleaving it at a single specific site with a restriction enzyme. The DNA fragment of interest is prepared from a larger piece of DNA using the same restriction enzyme as was used to open the plasmid DNA. The single-stranded ends of the fragment are then complementary to those of the cut plasmid. The DNA fragment and the cut plasmid are annealed and then joined by DNA ligase. DNA ligase catalyzes the formation of a phosphodiester bond at a break in a DNA chain. An energy source such as ATP is required for the joining reaction.

<br clear="all">

== Gene Cloning ==

[[IMAGE:Plasmid_pBR322.jpg|frame|Plasmid pBR322]]

One of the most useful plasmids for cloning is pBR322. pBR322 contains genes for resistance to tetracycline and ampicillin (antibiotics like penicillin). Different endonucleases can cleave this plasmid at a variety of unique sites, at which DNA fragments can be inserted. Insertion of DNA at the EcoRI restriction site does not alter either of the genes for antibiotic resistance. However insertion at the SalI or PstI site inactivates the gene for tetracycline or ampicillin resistance, an effect called insertional inactivation.


Cells containing pBR322 with a DNA insert at the SalI site are resistant to ampicillin but sensitive to tetracycline, so they can be readily selected. Cells that failed to take up the vector are sensitive to both antibiotics. Cells that take up pBR322 without a DNA insert are resistant to both. The colony of cells that are shown to have taken up the correct recombinant DNA is grown to produce a large amount of the DNA of interest.

<br clear="all">

== Gel Electrophoresis ==
[[Image:Electrophoresis.JPG|frame|Gel Electrophoresis Equipment]]
Gel electrophoresis is a technique used to separate biological molecules such as RNA, DNA and proteins through a gel support. The gels are a porous network of cross-linked polymers, usually composed of agarose or polyacrylamide. The principle of separation is to apply an electric field to drive charged molecules through the gel and separated out a complex mixture into distinct bands. The separation is based upon the mobility of molecules through the gel.
<br> <br>
The mobility of molecules can be defined as:

<center>Mobility = Charge + Molecular Dimensions</center>

A key use of gel electrophoresis is to separate mixture of DNA molecules. DNA is negatively charged and so, when an electric field is applied the DNA molecules will move towards the positive node. 
Generally speaking the key factor determining the mobility of DNA is the length of the molecule(molecular dimension). The reason why charge is not a factor for DNA mobility, is that DNA of all lengths have the same mass to charge ratio. In addition other factors affecting mobility include conditions of electrophoresis and conformation. The conformation is the form that the DNA is in i.e. in a linear, circular or supercoiled form.
<br>
<br>
As the size of DNA molecules increases the distance migrated decreases, this is because larger strands of DNA will have a greater steric resistance in the gel and so migrate slower. In addition for linear DNA, the distance migrated shows a linear correlation to the length of DNA. This correlation enables a DNA marker to be run out and a graph of distance migrated vs length of DNA plotted. Using this a band of unknown DNA size can have its distance migrated measured and the approximate length worked out.
<br clear=all>

== Blotting Techniques ==
Bloting is the methd of transferring biological molecules onto a carrier such as nitrocellulose membranes. Normally this transfer is after gel electrophoresis to allow for further experiments to be carried out on a separated sample. There are several varieties of blots, which vary in the types of molecules that they separate. The three types are, Southern blotting for DNA, Northern blotting for RNA, and Western blotting for proteins.

=== Southern Blotting ===
Southern blotting is teh transfer of DNA molecules that have been separated using gel electrophoresis onto a membrane such as nitrocellulose. Once transferred, the DNA molecules can be visualised by probe hybridisation. Hybridisation is the ability of complementary single strands of DNA to form a double helix and this. Hybridisation allows us to probe for specific sequences of DNA, by adding a complementary probe, either RNA or DNA, that have been labelled to allow for visualisation. Labelling types include radioactiveand fluorescent.

===Northern Blotting ===
Northern blotting is similar to southern blotting, however teh key differece is that it does not involve the transfer of DNA molecules but rather RNA molecules. This method is normally used for testing expression variations, where mRNA samples of a cell are extracted, separated. transferred and visualised. Again for visualisation, labelled DNA or RNA probes are used and hybridised to the sample of RNA.

=== Western Blotting ===
Western blotting is a method that is used to identify protein from a sample. A mixture of protein is separated on either native or denaturing gels and then transferred to a membrane. Specific proteins can then be identified using antibodies that bind to specific proteins. These antibodies are labelled, usually by either fluorescence or radioactivity. In addition, antibodies can be coupled to enzymes such as horseradish peroxidase that allows for detection.

== PCR (Polymerase Chain Reaction) ==

[[Image:PCR.jpg|frame|Three stages of PCR]]

PCR is a method for amplifying specific DNA sequences. PCR is carried out by adding the following components to a buffer solution:
*Target sequence
*A pair of primers that are complementary to the flanking sequences of the target
*dNTPs (ATGC)
*Heat-stable DNA polymerase<br>


A PCR cycle consists of three steps.
*Strand separation. The two strands of the parent DNA molecule are separated by heating the solution to 95ºC.
*Hybridization of primers. The solution is cooled to 54ºC to allow each primer to bind to a DNA strand. One primer hybridizes to the 3’end of the target on one strand, and the other primer hybridizes to the 3’ end on the complementary target strand.
*DNA synthesis. The solution is heated to 72ºC, the optimal temperature for Taq DNA polymerase (a heat-stable DNA polymerase from a thermophilic bacterium). The polymerase elongates both primers in the direction of the target sequence because DNA synthesis is in the 5’-to-3’ direction.<br>
These three steps constitute one cycle of the PCR amplification. The duplexes are heated to begin the second cycle, which produces four duplexes, and then the third cycle is initiated. After n cycles, the sequence is amplified 2n-fold.


There are several noteworthy features of this method. First, the sequence of the target need not be known. Only knowledge of the flanking sequences is required. Second, the target can be much larger than the primers. (>10 kb) Third, primers do not have to perfectly match flanking sequences. Primers derived from a gene of known sequence can be used to amplify related genes of the same family. Fourth, stringency or the closeness of the match between primer and target can be controlled by temperature and salt (MgCl2). Fifth, PCR is very sensitive. A single DNA molecule can be amplified and detected.

<br clear="all">

== DNA Sequencing ==
[[IMAGE:Dideoxyribonucleic_Acid2.JPG|frame]]
[[IMAGE:DNA_sequencing.JPG|frame]]

DNA sequencing is a technique used to determine the nucleotide sequence of a DNA molecule. There are various DNA sequencing techniques available; the most common technique used is the dideoxy method. DNA sequencing has become a crucial technique and with the creation of automated sequencers, DNA sequencing is now a relatively time and cost effective technique.
<br><br>
The dideoxy method relies upon dideoxyribonucleic acids, these nucleic acids lack the hydroxyl groups at the 3’ position that are found on deoxyribonucleic acids (see diagram). In DNA synthesis, the addition of a dideoxyribonucleotide to a growing strand will cause termination of the synthesis because there is no 3’ hydroxyl group to allow a phosphodiester bond to form.
<br><br>
The basis of the dideoxy method is to use single strands of the DNA sample to be sequenced as a template for complementary strand synthesis. In addition to deoxyribonucleotides the four types of dideoxyribonucleotides are included in the reaction. These are randomly incorporated into synthesizing strands and so will randomly terminate DNA synthesis. If enough DNA synthesis reactions are carried out, random incorporation will mean that at some point a dideoxyribonucleotide will be found for every nucleotide in the sequence will be found at the end of a terminated strands. These randomly terminated strands are then separated with gel electrophoresis. If we know which dideoxyribonucleotide was incorporated into a strand then we can work out the sequence nucleotide at a time.
<br><br>
The basics of the practical are explained in the diagram to the right. Four synthesis reactions are carried out, however within each reaction only one type of dideoxyribonucleotide is included. This enables us to know which dideoxyribonucleotide has been incorporated to a particular strand and so, we can work out which nucleotide is found at that position on the strand.
<br clear=all>