----

----

'''TRANSFORMATIONS GONE WRONG!'''
(these cells look weird)

<table|left>
 <tr>
  <td>[[Image:AudreyIGEMPennState1.jpg|thumb|Transformation 1]]</td>
  <td>[[Image:AudreyIGEMPennState2.jpg|thumb|Transformation 2]]</td>
 </tr>
</table>
--Troubleshooting
*Purification of discarded plates from lab benches may help to increase transformation efficiency

----

----


<h1>'''June 12, 2007'''</h1>

== Primers -- crp ORF and xylR ORF ==

crp ORF (view sequence: [http://biocyc.org/ECOLI/sequence?type=GENE&object=EG10164])

 Ggaattcgcggccgcttctagag ATGGTGCTTGGCAAACC Forward primer
 TmC 60°C

 ctgcagcggccgctactagta TTAACGAGTGCCGTAAAC Reverse Primer
 TmC 57°C

xylR ORF (view sequence: [http://biocyc.org/ECOLI/sequence?type=GENE&object=EG20253])

 Ggaattcgcggccgcttctagag CCATGTTTACTAAACGTCAC Forward primer
 TmC 56°C

 ctgcagcggccgctactagta CTACAACATGACCTCGCTAT Reverse primer
 TmC 58°C

<h1>'''June 25, 2007'''</h1>

== Preparation of Ultra-competent Cells Protocol -- Inoue Method ==
Based on Inoue et al (1990), Gene, 96:23-28, with modifications
<br>--To prepare beforehand: SOB, TB (see below)

--Prepared competent cells of DB3.1 and DH5a
<br>--When centrifuging, centrifuge used only held max 50ml, so spun down the 250ml culture in two separate centrifuge tubes with 40ml culture (don't fill all the way to the top to prevent spilling) for ~5minutes each time.
 '''Procedure'''
 1. Inoculated 2ml media with cells, grew overnight
 2. Inoculated 250mL SOB with O/N culture, grew at room temperature shaking until OD600=0.5 (Optimum
 at 19°C, but no loss of efficiency if cultures are grown at 20-23°C. Doubling time is 2.5-4hours)
 3. Place flask on ice for 10 min.
 4. Pellet cell by spinning cells at 4000rpm for 10 minutes at 4°C
 5. Discard supernatent, tip centrifuge tubes upside-down over paper towels
 for 2 minutes to remove excess liquid
 6. Spin at 4000rpm for 10 minutes at 4°C
 7. Gently resuspend pellet in 20mL ice-cold TB and 1.4mL DMSO (freeze O/N -20°C)
 8. Aliquot cells to 50ul for transformation or store at -70C (we stored at -80°C)

 '''SOB Solution'''
 *0.5% yeast extract
 *2% tryptone
 *10mM NaCl
 *2.5mM KCl
 *10mM MgCl<sub>2</sub>
 *10mM MgSO<sub>4</sub>
 *Dissolve in nanopure water and autoclave to sterilize

 '''TB Solution'''
 *10mM PIPES
 *15mM CaCl<sub>2</sub>
 *250mM KCl
 *Dissolve in nanopure water and adjust pH 6.7 with KOH or HCL (solutes will dissolve as you do this) and then add
 55mM MnCl<sub>2</sub>. Adjust to final volume. Sterilize by filtration with 0.45um filter and store at 4°C

<h1>'''June 26, 2007'''</h1>
== Competent cell preparation -- Information ==
http://www.chem.uga.edu/scottgrp/GrpProtocols/Competent_cell_preparation.htm

http://www.ciwemb.edu/labs/koshland/Protocols/BACTERIA/ecolicells.html

http://www.hybtech.org/Liu/uccell.html

== Frozen Competent E.Coli Cells Protocol -- Preparation For Use ==

http://www.ciwemb.edu/labs/koshland/Protocols/BACTERIA/ecolicells.html

(Inoue et al., 1990 Gene 96:23)

   1. Inoculate a 5ml overnight of E.coli in LB+20 mM MgSO4.
   2. Next morning, inoculate 250 ml LB+20 mM Mg++ in a 2L flask with about 2ml
 overnight culture. Grow at room temp (23°C) with good aeration (250rpm) to an 
 A600 of 0.4-0.6. Temp is important--see original ref.
   3. Place cells 10 min on ice. Transfer to a sterile bottle and spin 3K, 10', 4°C.
   4. Resuspend pellet in 80 ml cold TB (swirl cells in bottle). Leave 10’/ice.
   5. Spin cells 3K, 10', 4°C.
   6. Resuspend cells in 20 ml cold TB then add 1.5 ml DMSO. Leave 10'/ice.
   7. Dispense into 220 ul and 525 ul aliquots (in cold sterile tubes) and 
 freeze in dry ice/EtOH bath. Store -70°C. Typically, competency about 5X 106
 cfu/ug DNA. Note, improves after freezing. Cells good for a year and counting.

To use:

   1. To 50 or 100 ul cells, add 5-50 ng DNA. Leave 30’ on ice.
   2. Heat shock, 45 sec at 42°C, then chill cells on ice about 2’.
   3. Spin down (15 sec, eppendorf fuge) and remove SN. (Removing the Manganese
 seems to boost efficiency about 10X) and resuspend cells in 200 ul LB.
   4. For a supercoiled plasmid, plate 1 ul of cells. For a ligation, plate 20 ul and the rest. 

TB (transformation buffer: filter sterilize and store 4°C)
Product 	[stock] 	[ ]final 	volumes to make 100ml
Pipes-NaOH pH6.7 	0.5M 	10 mM 	2 ml
CaCl2 	0.5M 	15 mM 	3 ml
KCl 	2M 	0.25M 	12.5 ml (or 1.864g)
MnCl2 	1M 	55 mM 	5.5ml (or 1.088g) add to 100ml with ddH2O
4. To use competent cells for transformation, remove from freezer and thaw for a few minutes at 37°C. Place on ice, add plasmid DNA and incubate for one hour as in the standard transformation procedure. Then heat shock at 42degC for 2 minutes, cool briefly, add 1 ml of 2xTY and incubate for 1 hour at 37°C before spreading on plates.

<h1>'''June 27, 2007'''</h1>
== DH5a and DB3.1 Ultra-Competent Cells -- Notes==
--Finished up competent cell procedure
<br>--DH5a cells grew very slowly, DB3.1 cells grew very fast
<br>--Transformed fresh DB3.1 cells with P1010 ([http://parts.mit.edu/registry/index.php/Part:BBa_P1010 cell death gene]) and pUC19 (small high copy vector)
<br>--Plated DB3.1 (P1010, pUC19, negative control, positive control [DB3.1 on strep])

<h1>'''June 28, 2007'''</h1>
== DH5a and DB3.1 Competent Cells (cont.) ==
--Transformed cells grew well
<br>--Very few pUC19 cells (probably due to very small amount of vector added during transformation)
<br>--No cells on negative control (good)
<br>--Inoculated transformed cells into cell cultures for mini-prepping (pUC19 into 2mL, death gene into 15mL)

== PCR ==
--PCR testing out Taq and Pfu DNA polymerase (during this experiment, thought Pfx was being used instead of Pfu) on the following primers (12 samples total))
 '''Primers Used'''
 *3398+3399: xyl promoter region (xylA to xylF) [http://parts.mit.edu/registry/index.php/Part:BBa_I741015 BBa_I741015]
 *3400+3401: xyl promoter region (xylF to xylA) [http://parts.mit.edu/registry/index.php/Part:BBa_I741017 BBa_I741017]
 *3406+3407: P<sub>FGH</sub> including CRP-cAMP binding site [http://parts.mit.edu/registry/index.php/Part:BBa_I741018 BBa_I741018]
 *3408+3409: P<sub>AB</sub> with CRP-cAMP binding site [http://parts.mit.edu/registry/index.php/Part:BBa_I741019 BBa_I741019]
 *3402+3403: P<sub>FGH</sub> without CRP-cAMP binding site [http://parts.mit.edu/registry/index.php/Part:BBa_I741020 BBa_I741020]
 *3404+3405: P<sub>AB</sub> without CRP-cAMP binding site [http://parts.mit.edu/registry/index.php/Part:BBa_I741021 BBa_I741021]

--PCR from the day before didn't work because MgCl<sub>2</sub> was used instead of MgSO<sub>4</sub>


<h1>'''June 29, 2007'''</h1>
== Miniprep Protocol -- pUC19 and P1010 in pSB1A2 ==
--Used O/N cultures to do miniprep. Miniprepped 1 tube of pUC19 and 10 tubes of P1010 (1ml, spin, add another 1ml, spin). Spun down at 4k for ~10 minutes

 '''QIAprep Spin Miniprep'''
 1. Resuspend pelleted bacterial cells in 250ul Buffer P1 an transfer to
 microcentrifuge tube (Ensure RNase A has been added to
 Buffer P1
 2. Add 250ul Buffer P2 and mix thoroughly by inverting the tube 4-6 times until solution is
 viscous and slightly clear. Do not vortex as doing so will shear the DNA
 3. Add 350ul Buffer N3 and mix the solution thorougly by inverting the tube 4-6 times. Solution
 should become cloudy
 4. Centrifuge for 10min. at 13000rpm (~17900 x g)
 5. Apply the supernatants from step 4 to the QIAprep spin column
 6. Centrifuge for 30-60sec. Discard flowthrough
 7. Wash QIAprep spin column by adding 0.75ml Buffer PE (w/EtOH) and centrifuge for 30-60sec.
 Discard flowthrough
 8. Centrifuge an additional 1min. to remove residual wash buffer
 9. Place QIAprep column in a clean 1.5ml microcentrifuge tube. Add 50ul dH<sub>2</sub>O to the
 center of each QIAprep, let stand for 1min. and centrifuge for 1min.

== Transformation -- pUC19 ==
--Transformed pUC19 into DH5a to test both competence of DH5a frozen stocks and pUC19 miniprep
<br>--Plated 200ul on Amp plates

<h1>'''July 2, 2007'''</h1>
== General Lab Activities ==
--Ordered Invitrogen Platinum Pfx DNA Polymerase
<br>--Made more SOB
<br>--Transformation from the day before went well

== Gradient PCR Protocol -- PfuUltra DNA polymerase ==
--Testing optimal temperature using temperature gradient during PCR using Taq and Pfu DNA polymerase. Used 3398 and 3399 (xyl promoter region xylA to xylF primers)

 '''PCR Protocol (modified)'''
 Reaction Mixture (10x volume)
 *418ul dH<sub>2</sub>O
 *50ul 10x PfuUltra HF reaction buffer or ThermolPol buffer (for Taq)
 *10ul dNTP (25mM each dNTP)
 *0.69ul Primer #1
 *0.69ul Primer #2
 Then aliquoted 39ul mixture into PCR tubes (8 samples, 1 negative) for each polymerase
 Added 1ul of chromosomal DNA (DNA template) to all tubes except negative control
 Then added 1ul of appropriate polymerase

 '''PCR program -- Pfu DNA polymerase'''
 No hotstart
 Cycle 1 (1x)  - 1min. at 95°C
 Cycle 2 (28x) - 15sec at 95°C
                 30sec at 55°C
                 25sec at 55-75°C (gradient part of PCR)
 Cycle 3 (1x)  - Stand at 4°C

Gradient temperatures: 55.0, 56.6, 59.0, 62.6, 67.7, 71.3, 73.6, 75.0 (in °C)
<br>Negative controls run at 73.6°C

<br>--Tested out SYBRGreen to PCR run
*SYBRGreen didn't show up well (later found out due to improper storage of the SYBRGreen)
*Soaked gels for an hour in TAE with EtBr. Saw bands for both Taq and Pfu, but very unclear (rerun next day)

<h1>'''July 3, 2007'''</h1>
== PCR --Temperature Gradient Experiment (cont) ==
<br>[[Image:taqpfugradient.jpg|thumb|Taq and Pfu Gradient]]
--Reran gel with EtBr with yesterday's PCR samples
*results seem very good. Looks like worked with both Pfu and Taq
*Pfu seems to work best at 67.6-71.3°C
*Did DNA gel extraction with this gel run

<br>--Ran same PCR procedure as previous day
*Negative controls run at 71.3°C
*Results don't seem as good, although PCR still seem to work
*For some reason no band at 71.3°C and 75.0°C, but band present at 73.6°C
*Decided to still go ahead and use Pfu (will not use Pfx)

== Restriction -- xyl promoter region (xylA to xylF)==
--Ran restriction overnight at 37°C
<br>--Restricted with EcoRI and SpeI
<br>--BioBrick part I741015
<br>--(Notes to self [http://openwetware.org/wiki/Silver:_BB_Strategy])
 EcoRI Recognition site: 5' AATT 3'
                         3' TTAA 5'

 SpeI Recognition site:  5' CTAG 3'
                         3' GATC 5'

<h1>'''July 4, 2007'''</h1>
Some stuff that was important enough that we had to work during the holiday,<br> but not important enough that you would need to know

Read: I don't really remember what we did this day and/or everything we did this day failed anyway so as to be of little consequence. Most likely a combination of the two.

<h1>'''July 5, 2007'''</h1>
== Digeston Protocol -- P1010 in pSB1A2 ==
--Retried from yesterday
--Used EcoRI and SpeI
 '''For 10ul'''
 0.5ul EcoRI
 0.5ul SpeI
 1ul EcoRI buffer
 5ul vector (P1010 in pSB1A2)
 3ul autoclaved, distilled water
 4 hours at 37°C
 *for negative, no enzymes, used water to make up the volume

--run on 1% agarose gel
*ladder, P1010 in pSB1A2, P1010 in pSB1A2 (restricted), - sender, + sender, - receiver, + receiver
*last four lanes are for the motility experiment
*Gel didn't end up good, strange unexpected banding pattern (the pSB1A2 restricted band didn't show up, only the P1010 and the unrestricted P1010 in pSB1A2 appeared to be present in the second lane)

== Ligation and Transformation of I741015 into pSB1A2 ==
--Ligated I741015 (xyl promoter region) and pSB1A2 (repeat of yesterday)
--Transformed into DH5a cells
*Didn't work

== Gradient PCR Protocol -- Pfx DNA polymerase ==
--Received Pfx DNA polymerase today
--Did gradient PCR from 56-68ºC, negative control at 68ºC

 '''Component'''                      '''Volume'''          '''Final Concentration'''
 10X Pfx Amplification Buffer   5ul             1X
 10mM dNTP mixture              1.5ul           0.3mM each
 50mM MgSO<sub>4</sub>                     1ul             1mM
 Primer mix (10uM each)         1.5ul           0.3mM each
 Template DNA                   >=1ul           as required
 Platinum Pfx DNA polymerase*   0.4-1ul         1.0-2.5 units
 Autoclaved, distilled water    to 50ul
 *We used 0.5ul in this experiment
 Negative control contained no template DNA. Water was used to make up the volume

--Lanes: Ladder, negative control, 56ºC, 57.1ºC, 58.8ºC, 61.3ºC, 64.9ºC, 67.5ºC, 69.1ºC, 70ºC

== PCR using Pfx DNA polymerase on promoters and xylR ==
--Did PCR for all the promoters, xylR (total), xylR ORF
*Used SYBR Green
*Bands didn't show up well on SYBR Green (perhaps due to poorly made agarose gel)
*elongation temperature at 68ºC
*Something showed up in the negative control (must have accidentally added chromosomal DNA)

 '''PCR program -- Pfx DNA polymerase'''
 No hotstart
 Cycle 1 (1x)  - 2min. at 94°C
 Cycle 2 (28x) - 15sec at 94°C
                 30sec at 55°C
                 1min. at 68°C
 Cycle 3 (1x)  - Stand at 4°C

<h1>'''July 7, 2007'''</h1>
== Gel run of promoters and xylR ==
--Reran last night's PCR on 1.5% agarose gel using EtBr instead of SYBR Green
*Showed up a lot better. Did gel extraction with these
*Lanes: Ladder, negative control, 3416+3417 (xylR ORF), 3412+3413 (xylR total), 3398+3399 (xyl promoter), 3408+3409 (forward xylA promoter), 3406+3407 (forward xylF promoter), 3404+3405 (xylA promoter w/o CRP binding sites), 3402+3403 (xylF promoter w/o CRP binding sites), 3400+3401 (reverse xyl promoter)
*Cut out xylR ORF, xylR total, and xyl promoter bands. Also cut out the other promoter bands

--Did another PCR with just xylR total, xylR ORF, and xyl promoter
*changed annealing temperature to 57ºC
*Lanes: Ladder, negative control, 3398+3399, 3398+3399, 3398+3399, 3412+3413, 3412+3413, 3412+3413, 3416+3417
*Only the second lanes from xylR total and xyl promoter seem strong. Cut those out

== Gel extraction ==
'''From yesterday's gel'''
--Used gel extraction kit with xylR genes
--Tried out freeze n' squeeze method with the promoters
*dye seems to stay and go through so that the liquid in the centrifuge tube is blue. Dye might possible interfere with restriction?? Also, a lot of end volume, so the product iws more dilute than through gel extraction

'''Today's gel'''
--Extracted both xyl promoter and xylR total using gel extraction kit

== Restriction == 
--Restricted xyl promoter, xylR total, and xylR ORF at EcoRI and SpeI sites for 20hours

<h1>'''July 8, 2007'''</h1>
== Restriction -- P1010 in pSB1A2 ==
--Restricted for 1 hour at 37°C with EcoRI and SpeI
--Run all restrictions (including the 20hr one from yesterday) on 1.5% gel
*good band for pSB1A2
*very faint bands for xyl promoter lanes
--Ligate for 1 hour at room temperature (~25C) with T4 ligase for 1 hour
--Transform into DH5a cells

<h1>'''July 9, 2007'''</h1>
== Ligation Problems/Troubleshooting ==
--No cells from transformation. Maybe problem with ligation?
*Maybe T4 ligase is not good?
--Bought new T4 ligase and more SpeI
--Retry ligation
*Tried two tests: ligation for 20 minutes and ligation for 1 hour
*Transformed into DH5a cells
--From lab meeting: Possible problems
*too much UV exposure destroying sticky ends
*too much salt (solutions: dialyze or use Princeton column)
*Try 3 hour digestion instead of 20 hour
*quantify DNA (use only 50ng of vector in 20ul ligation. Too much or too little will not work. 3x molar excess of insert. If the insert is small, use more excess insert)
*Separate T4 buffer into small aliquots so ATP does not become de-phosphorylated from repeated freezing and thawing

== Miniprep and Restriction -- E0240 and pSB1A2 ==
--Restrict at EcoRI and SpeI
--Run on gel (1.5% with EtBr)
 '''Notes'''
 E0240 = 876bp
 pSB1A2 = 2079bp
 E0240 in pSB1A2 = 2955bp
 P1010 = 675bp

<h1>'''July 10, 2007'''</h1>
== Ligation Troubleshooting ==
--Purchased PCR cleaning kit ($88)
<br>--Ligation Experiment 1
*Try restriction right in PCR products with EcoRI and SpeI (method from last year's iGEM)
*Restrict for only 3 hours
*After restriction, run gel with SYBR Green
*Gel extract
*Use PCR cleaning kit to get rid of salts from gel run
*Try restricting in Buffer 2 with BSA (method from last year's iGEM) instead of EcoRI
 '''Restriction Protocol -- approx. 20ul volume'''
 15ul insert (directly from PCR)
 0.5ul EcoRI
 0.65 SpeI
 2ul 10X Buffer 2
 2ul 10X BSA
--Ligation Experiment 2
*Using xyl promoter, xylR ORF, xylR total, GFP, pSB1A2
*Restrict on DNA already taken from gel after PCR (EcoRI and SpeI) for 3 hours at 37°C (Restricted 2 hours for GFP and pSB1A2)
*Use PCR cleaning kit afterwards (put GFP in freezer after this)
*Ligated with T4 ligase and transformed, using pUC19 as a positive control and no insert for negative control
*Grow O/N at 37°C

== Ligation and PCR Purification Protocol ==
 '''Ligation Protocol -- 20ul volume'''
 2ul T4 buffer
 4ul Restricted pSB1A2 vector (cut at EcoRI and SpeI)
 12ul insert (cut at EcoRI and SpeI)
 1ul T4 ligase
 1ul distilled, autoclaved water pH~7.8

 '''PCR Purification Protocol -- QIAquick PCR Purification Kit Protocol'''
 *All centrifugation steps carried out at 13000rpm (17900 x g)
 1. Add 5 volumes of Buffer PBI to 1 volume of the PCR sample and mix
 2. Check that the color of the mixture is yellow
 3. Place a QIAquick spin column (purple) in a provided 2ml collection tube
 4. To bind DNA, apply the sample to the QIAquick column and centrifuge for 30-60sec
 5. Discard flow-through. Place the QIAquick column back into the same tube
 6. To wash, add 0.75ml Buffer PE to the QIAquick column and centrifuge 30-60sec
 7. Discard flow-through and place the QIAquick column back in the same tube.
    Centrifuge for an additional 1min.
 8. Place QIAquick column in a clean 1.5ml microcentrifuge tube (labeled)
 9. To elute DNA, add 50ul water to the center of the QIAquick membrane, let stand for 1min, then centrifuge for 1min.

<h1>July 11, 2007</h1>
== Ligation Results ==
--Ligation Experiment 1 (digestion directly in PCR product) failed. No colonies on any of the plates
<br>--Ligation Experiment 2
*Lots of colonies on xyl promoter (I741015) plate. 20~ish colonies on the xylR ORF (I741005) plate
*Inoculate I741015 and I741005 into 2ml cultures with Amp overnight 37°C for miniprepping
*xylR total (I741011) plate failed. No colonies

== Gel extraction -- pSB1A2 and E0240 ==
--Gel extracted SpeI/EcoRI restricted pSB1A2 and E0240 from Monday's gel

== Retry Ligation of I741011 and I741017 ==
--Try to digest again from gel for 3 hours, clean with kit (same as Experiment 2 method)
*Used 10ul instead of 12ul of insert with I741011
--Plated and grew overnight at 37°C

== Preparation for creating constructs with xylR ORF ==
--Transformed various RBS (different strengths) and terminators into DH5a

<h1>July 12, 2007</h1>
== Retry Ligation of I741011 and I741017 Results ==
--Contamination. Growth on all of the plates, including negative
*Cleaned out fume hood and incubation chamber with ethanol. Also, turned on UV light on fume hood for 30 minutes.
*Retry ligation and plating (used left over ligation from yesterday)

== Beginning to Build Constructs -- I741015 with E0240 and I741005 with B0015 ==
<br>[[Image:I741015+E0240andI741005+B0015.jpg|thumb|Restrictions on Gel]]
--Did prefix insertion of I741005 into B0015 in pSB1K3 
--Did suffix insertion of E0240 into I741015 in pSB1A2
 '''Notes'''
 B0015 in pSB1K3 = 3318bp
 I741005 = 1179bp
 E0240 = 876bp
 I741015 in pSB1A2 = 2380bp

 '''Digest Protocol -- 20ul'''
 14ul DNA
 2ul 10X Buffer
 2ul 10X BSA
 1ul autoclaved, filtered water
 0.5ul Enzyme #1
 0.5ul Enzyme #2
 *add enzymes last
 *Restricted 1 hour at 37C

== Combining Two BioBricked Parts Protocol ==
 '''Combining Two Parts'''
 <u>Prefix Insertions</u>
 *Cut vector with: XbaI and EcoRI -- Use NEB Buffer 2 (w/BSA)
 *Cut insert with: SpeI and EcoRI -- Use NEB Buffer 2 (w/BSA)
 <u>Suffix Insertions</u>
 *Cut vector with: SpeI and PstI -- Use NEB Buffer 2 (w/BSA)
 *Cut insert with: XbaI and PstI -- Use NEB Buffer 3 (w/BSA)

--Notes to self
*Silver [http://openwetware.org/wiki/Silver:_BB_Strategy] suggests using EcoRI buffer instead of Buffer 2 for the XbaI and EcoRI restriction
*Manual recommends doing XbaI and EcoRI restrictions separately. (did them at the same time for this experiment)

--Ran on gel, extracted, and ran ligation for 1 hour at room temperature
<br>--Tranform and plate

== General Lab Activities ==
--Miniprepped I741015 and I741005 from DH5a
<br>--Made more 500mM glucose (Galen)
<br>--Made more LB plates and SOB (Noah)


<h1>July 13, 2007</h1>
== Ligation Results == 
--Ligation of I741015+E0240 had cells. Inoculate and grow overnight in 2ml culture tubes

<h1>July 14, 2007</h1>
--Miniprep I741015+E0240
--Tried ligating various promoters to RBS and xylR ORF to E0240. Tried with varying concentrations of insert to vector with the E0240 and xylR ORF (0.5:1, 1:1, 2:1, 3:1)
--Transform into cells and plated to grow overnight

<h1>July 15, 2007</h1>
--4-8 colonies on all plates; even on negatives :(
--Retry ligating

<h1>July 16, 2007</h1>
--Retried ligations are not good. Too much contamination on ALL plates (really small yellow colonies everywhere)
<br>--Tried to get all colonies that appeared to be good on the ligations from July 14. Grow O/N. Tomorrow will restrict and run on gel to see what DNA is present

<h1>July 17, 2007</h1>
== Ligation Troubleshooting ==
--Searched on internet for possible causes of failed ligation
<br>--Made new SOB (SOB from the 12th got contaminated, which resulted in lots of of unexpected cell growth on previous plates)
<br>--pUC19 is transforming, so competent cells are still okay
<br>--Appears that DNA (i.e. I741015+E0240) is present according to gels. I741011 looks higher than normal, though, after restriction. Should be around 1475 base pairs, but appears above 1.5kb on gel.
*Ligation at 20-30 minutes instead of 1hr
*Try less ligation product for transformation. Anything more than about 5% volume of ligation mix into the chemically competent (Invitrogen) cells started to become inhibitory [http://www.protocol-online.org/biology-forums/posts/14962.html]
*Ligase inhibits transformation. Try denaturing ligase first before transforming

== Gel ==
--Single Restrictions using EcoRI and Double Restrictions using SpeI and PstI [http://openwetware.org/wiki/IGEM:PennState/Labbook/NoahJohnson/2007-7-16]
*double digest don't look good. Only one band seen, not much dropping out. Only 3 of 13 samples appear to have restricted properly.

<h1>July 19, 2007</h1>
http://mycoplasmas.vm.iastate.edu/lab_site/methods/DNA/precip.html