{{IGEM:/Harvard/2006/Cyanobacteria}}

==Literature==
===Cyanobacteria background and practicality===
#[https://dspace.mit.edu/bitstream/1721.1/32302/1/61347289.pdf Genetic noise in the cyanobacterial oscillator]
#*Very good PhD thesis by Jeffrey Chabot which has general cyanobacteria information and culturing information. Also deals with using a GFP reporter. From the vanO lab.
# Molecular biology of circadian rhythms / edited by Amita Sehgal 
#* Peng's taking a look at it now; in Biolabs
#[http://www-cyanosite.bio.purdue.edu Cyanosite]
#* Lots of information on cyanobacteria culture mediums and good links to other resources.

===Western Blotting Protocols===
Links and descriptions taken from [http://www.protocol-online.org Protocol-Online]

#[http://www.abcam.com/assets/pdf/protocols/WB-beginner.pdf Western Blotting- A Beginner's Guide]
#*''In this 9-page protocol, you can find everything you need to know about Western blotting including lysate preparation, denaturation and reduction of protein, electrophoresis (tips on positive controls, MW markers, loading controls), tips on transferring, blocking and incubation.''
#[http://omrf.ouhsc.edu/~frank/Western.html Western Blotting Protocol (Lab of Dr. Mark Barton Frank, Oklahoma Medical Research Foundation)]
#*''The most complete Western protocol including:
Preparation of protein from human cells
Preparation of Bacterially Expressed Proteins
SDS-PAGE Separating Gels (Hoeffer baby gels)
Electrophoretic Transfer for Dry Blots 
Electrophoretic Transfer For Wet Blots
Detection of Antigen on Membranes
Photograph the membranes
Recipes ''
#[http://cancer.ucsd.edu/howelllab/Western.html Howell Lab]
#*''Procedures for preparation of cell lysate, gel, electrophoresis, transfer, antibody detection and stripping membranes.''
#[http://www.biochem.emory.edu/labs/acorbe2/protocols9.html Corbett Lab]
#[http://www.encorbio.com/protocols/blotting.htm EnCor Biotechnology]
#*''Well written Western protocol in plain English''
#*The accompanying SDS-PAGE gel electrophoresis [http://www.encorbio.com/protocols/SDS-PAGE.htm protocol].

===Papers on latest findings===
<biblio>
 #1 pmid=15831759
 #2 pmid=10649285
 #3 pmid=16729054
 #4 pmid=12727878
 #5 pmid=16882723 
 #6 pmid=16707582
 #7 pmid=12429697
 #8 pmid=15731760 
 #9 pmid=6091052 
 #10 pmid=16236728
 
</biblio>
# Implementing KaiABC in-vitro and demonstrating circadian oscillation
#* Hetmann's review [[IGEM:Harvard/2006/Brainstorming_Papers_-_Hetmann | here]]
# Good review paper on cyanobacteria oscillator, featuring clearly what is unknown
#* Peng's review [[IGEM:Harvard/2006/Brainstorming_Papers_-_Zhipeng | here]]
# General review paper of current cyanobacteria knowledge recommended by Dave
# Information on measuring Phosphorylated KaiC
#* [[IGEM:Harvard/2006/Xuetal | See here for review from paper + image]]
# [http://www.pnas.org.ezp1.harvard.edu/cgi/content/abstract/99/24/15788?ijkey=adbaa3e242bd7408f479b14da3c7164908512852&keytype2=tf_ipsecsha More on measuring phosphorylated KaiC (SDS-PAGE)]
#*Cited in the Nakajima paper
#[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=Abstract&list_uids=8985018&query_hl=2&itool=pubmed_docsum Circadian rhythms in rapidly dividing cyanobacteria], Kondo ''et al.'', 1997
#[http://jb.asm.org.ezp1.harvard.edu/cgi/content/full/183/8/2439 Independence of Circadian Timing from Cell Division in Cyanobacteria], Mori and Johnson, 2000
#[http://www.pubmedcentral.gov/articlerender.fcgi?tool=pubmed&pubmedid=12727879 Ratio of KaiABC in vivo]

==Possible Molecular Mechanisms==
After a review of the literature, one will find that not much is known about the molecular mechanism behind the cyanobacteria oscillator; we only have theories of what could be true. Below are some of them, evidence that supports it, and evidence to the contrary.

===Activator/Repressor===
[[Image: Barkai_Leibler.jpg |thumb|right| "Activator-repressor model from 2000, by Chabot 2005"]]
*Barkai and Leibler 2000
*Modeled after Eukarayotic systems
*Probably not true
**KaiABC vary as activator/repressor
**Transcription/translation not essential (invitro experiment)


===KaiC phosphorylation model===
[[Image: Xu_2003.jpg |thumb|left| "KaiC phosphorylation model from 2003, by Chabot 2005"]]
*Xu et al 2003
*Previous research showed
**Cells without KaiA had all unphosphorylated KaiC
**Cells without KaiB has all phosphorylated KaiC
**KaiA protein constant
**Iwasaki et al. 2002
*'''Note:''' If this model holds true than our experiment in E. coli should show some silencing of genes downstream of KaiBC? We could test it by putting a reporter right downstream of KaiBC easily... very interesting.
*'''Note:''' This would be a good question to ask someone: if we put a reporter right downstream of KaiBC what should happen to that reporter.
<br style="clear:both;">

===KaiB spaitotemporal localization model===
[[Image: Kitayamaetal_2003.jpg |thumb|right| "KaiB spaitotemporal localization, 2003"]]
*Kitayama et al 2003
*Idea that KaiB rotates location from the membrane to cytosol
**Doesn't the in-vitro experiment disprove this?


===Transcriptional/Translational independent model===
[[Image: Tomita 2005.jpg |thumb|left| "Transcriptional/translational independent, 2005"]]
*Tomita, Nakajima et al 2005
*Minimal oscillator and an extended timing system
*Best oscillation system currently developed
<br style="clear:both;">

===KaiC helicase model===
*Proposed by C Johnson (unpub.)
*Looked at the 2 endogenous plasmids in cyanobacteria and found that they varied supercoiling state
*Hypothesis is that KaiC acts as a helicase which controls transcription access over genes

==Primer Design==

[[IGEM:Harvard/2006/Cyanobacteria/PrimerDesign]]

*According to BLAST, KaiC (519aa) is only 81% identical between PCC7942 and 6803.
*These papers might have our promoter sequences:
<biblio>
#1 pmid=9727980
#2 pmid=16102014
#3 pmid=10564489
</biblio>
*Especially paper #3, which states that there are 4 known sigma factors.
*Paper #2 says that -55 to 14 was experimentally determined to be the promoter/operon area for kaiBC (not kaiA :(); -35 and -10 have homology to sigma 70 in E. coli

==Kai Gene sizes==

:*KaiA size: 855bp
:**903bp with BioBrick ends
:*KaiB: 309bp
:**357bp with BioBrick ends
:*KaiC: 1560bp