<div id="mainpage">
<hr width="750px">
<div id="about">
{{Caltech_iGEM}}
</div>
<hr width="750px">
__NOTOC__
{|cellspacing="5" cellpadding="10" style="background:#ffffff; width: 750px;"
|-valign="top"
|style="background:#ffffff"|

This protocol is adopted almost verbatim from Oppenheim <i>et al</i>, 2004<cite>oppenheim</cite>.

==Recombineering==
The strains used for recombineering carry a defective λ prophage containing the pL operon under control of the temperature-sensitive repressor cI857. 

# The strain of choice is grown in a shaking water bath at 32°C in LB with 0.4% maltose to mid-exponential phase, A600 0.4–0.6 (30 ml is adequate for several recombineering reactions). 
# The culture is harvested by centrifugation and resuspended in 1 ml TM (10mM Tris base, 10mM MgSO4, pH 7.4). The phage to be engineered is added at a multiplicity of infection of 1–3 phages/cell (we assume cell density of approximately 10<sup>8</sup>/ml before concentration) and allowed to adsorb at room temperature for 15 min (this step would need modification for other phages, i.e., adsorption on ice). 
# Meanwhile, two flasks with 5-ml broth are pre warmed to 32°C and 42°C in separate shaking water baths. 
# The infected culture is divided and half inoculated into each flask; the cultures are incubated an additional 15 min. The 42°C heat pulse induces prophage functions; the 32°C uninduced culture is a control. 
# After induction, the flasks are well chilled in an ice water bath and the cells transferred to chilled 35-ml centrifuge tubes and harvested by centrifugation at approximately 6500g for 7 min. 
# The cells are then washed once with 30-ml ice-cold sterile water; the pellet is quickly resuspended in 1-ml ice-cold sterile water and pelleted briefly (30 s) in a refrigerated microfuge. 
# The pellet is resuspended in 200 μL cold sterile water and 50–100 μl aliquots are used for electroporation with 100–150 ng PCR product or 10–100 ng oligonucleotide. (Oppenheim <i>et al</i> use a BioRad E. coli Gene Pulser set at 1.8 mV with 0.1-cm cuvettes.) 
# Electroporated cells are diluted into 5 ml 39°C LB medium and incubated to allow completion of the lytic cycle. 
# The resulting phage lysate is diluted and titered on appropriate bacteria to obtain single plaques. To screen for phages containing desired amber mutations in <i>N</i> and <i>Q</i>, the lysate can be titered in a [[IGEM:Caltech/2007/Protocols/Two_Layer|double layer assay]] with amber suppressing and non-suppressing bacteria.

==Oligonucleotide Design==
We purchased our oligonucleotides from IDT with PAGE purification. The oligos were homologous to the <i>N</i> and <i>Q</i> gene regions to be mutagenized, and contained a single point mutation moving a tyrosine residue to an amber stop codon. The oligo sequences are given below, with the nucleotide introducing the point mutation capitalized.

===Oligos targeting N for amber mutation===
5'-tctcctgtcagttagctttggtggtgtgtggcagttCtagtcctgaacgaaaaccccccgcgattggcac-3'

5'-tcaatacgttgcaggttgctttcaatctgtttgtgCtattcagccagcactgtaaggtctatcggattta-3'

5'-ccactgcatgttatgccgcgttcgccaggcttgctCtaccatgtgcgctgattcttgcgctcaatacgtt-3'

===Oligos targeting Q for amber mutation (courtesy of D. Court)===

5'-ttagtatttccttcaagctttgccacaccacgCtatttccccgataccttgtgtgcaaattgcatcagat-3'

==References==
<biblio>
#oppenheim pmid=14980479
</biblio>

|}
</div>