=='''Calendar'''==
Weekly Updates on the Tri-Stable Switch

<calendar>
name=iGEM:Brown/2007/Tri-Stable Switch
date=2007/08/01
view=threemonths
format=%name/%year-%month-%day
weekstart=7
</calendar>


=='''Tri-stable Toggle Switch'''==
The Tri-stable Toggle Switch will be able to produce three distinct, continuous (stable) outputs for each of the three inputs.  A chemical will induce the system to "lock into" one state while repressing the other two states.  [[Image:Tristable_Toggle_Switch_2007.jpg|thumb|left|The Tri-stable Toggle Switch Architecture]] Our three constructs are ''pBAD->LacI->TetR'',   ''pLacI->AraC->TetR'',   and   ''pTet->AraC->LacI'',   where [http://en.wikipedia.org/wiki/Bcl-2-associated_death_promoter AraC] represses pBAD, [http://en.wikipedia.org/wiki/Lac_repressor LacI] represses pLac and [http://en.wikipedia.org/wiki/Tetracycline_controlled_transcriptional_activation TetR] represses pTet.  The three chemicals ([http://en.wikipedia.org/wiki/Arabinose arabinose], [http://en.wikipedia.org/wiki/IPTG IPTG] (Isopropyl β-D-1-thiogalactopyranoside) and [http://en.wikipedia.org/wiki/Tetracycline Tetracycline], respectively), cause conformational changes in their respective repressor proteins which leads to gene expression.  For example, in the presence of arabinose, AraC cannot repress pBAD so LacI and TetR are produced which in turn repress pTet and pLac.













===Motivation for a Tri-Stable switch===

===Applications===

=='''Repressors'''==
===AraC===
The gene AraC is one of several genes (AraA, AraB, AraD, etc) originally for the metabolism of arabinose.  OF special interest for our application are araC and araE.  AraC is the repressor/transcription factor and araE is required for the import of arabinose.  [http://www.mun.ca/biochem/courses/3107/Topics/Ara_operon.html]  [[Image:Two_Dimers_of_AraC.jpg|thumb|left|Dimer structure with arabinose on the left (yellow) [http://www.bactregulators.org/structure/arac-3d.php] ]]
[[Image:AraC_Promoters.gif|thumb|The left image shows the araC dimer repressing transcription, while the right conformation enables transcription [http://www.bactregulators.org/structure/arac-3d.php]]]

The araC forms a dimer with ''and'' without arabinose but the structural change activates or represses the ara promoters. This goes for pC which promotes araC and pBAD which has been modified so that we can put what ever we want after it and pE which promotes araE.  In the presence of arabinose, araC undergoes a conformational change so that instead of repressing it allows the DNA loop to unfold and now is a transcription factor.  This makes sense because the cell would not want to be spending its valuable resources on making import and metabolic proteins if there was no arabinose present to metabolize.

Non modified forms of ara genes are already present naturally in the E. coli we use, which can be a problem.  Arabinose can be metabolized so our arabinose concentration may not stay constant, but, more importantly, pAra promoters suffer from all or none gene expression, ie a cell is all on or all off at low concentrations of inducer arabinose.  Once some arabinose slips into a cell it turns pE on bringing much more arabinose into the cell.  The cells that are turned on first take all the arabinose and there is none left to turn the other cells on.  It has been shown that this can be fixed by putting the araE gene after a constitutive promoter [Modulation of gene expression from the arabinose-inducible araBAD promoter,  Keasling et al].  We wont have time to do that this year but perhaps in the future araE can be added to the registry.  For now it seems that 0.2% arabinose is enough to turn all of the cells on.

The pC/pBAD promoter available in the registry comes with the araC gene so that araC is also under the control of its own promoter (pC>araC, reading the opposite direction as pBAD).  This means that we wouldn't have a tristable system but a "2.5 stable switch" because we would have three states but we couldnt remove arabinose from the system and have the switch remain stable.  araC would be activated, repressing pBAD and the switch would fall into whatever the stronger of the other two states are.  An attempt [http://parts.mit.edu/registry/index.php/Part:BBa_I13458:Experience] was made at decoupling the araC from pC and putting it on a different plasmid but this lead to the very confusing result where the promoter was on with out arabinose and off with arabinose.  More testing should be done here.

The AraC BBa_C0800 may have a problem as noted by PatrickKing on the reg page. "When C0080 was inserted in front of I13504 (RBS . GFP . Terminators) strong GFP production was observed. No promoters were present in the construct at all. C0080 may contain a strong forward promoter!" This could be a big problem.

Relavent araC papers:
Keasling et al., "Modulation of gene expression from the arabinose-inducible araBAD promoter", 2002.
Lee et al., "Mechanism of araC autoregulation", 1980.
Johnson and Schleif, "In vivo Induction Kinetics of Arabinose Promoters...", 1995.
Lee et al., "Arabinose-induced binding of AraC...", 1987.

===LacI===
In nature, LacI represses pLac which promotes LacYZA genes that metabolize lactose, thus LacI represses pLac except in the presence of lactose (or lactose mimics, eg IPTG).  [[Image:LacI_repressor.gif|thumb|left|Image[http://www.mun.ca/biochem/courses/3107/Topics/Lac_genetics.html]. LacI forms a tetramer and represses pLac.  However, an inducer, such as IPTG, causes a conformation change that removes LacI from the operator site.]]    Lactose causes a conformational change which inhibits LacI from binding to the operator site of pLac.  Four LacI proteins form a tetramer to inhibit pLac and four inducer molecules are required to cause the full conformational change in the inhibitor.[http://www.mun.ca/biochem/courses/3107/Topics/Lac_genetics.html]

The LacI part available in the registry has an LVA degradation tag and, as JasonK notes, "seeing significant repression with C0012 is unlikely."  He says C0013 without an LVA tag is under construction but that was a year ago and we will need strong repressors for the switch to work.  It may be time for us to make LacI+ASV ourselves.  The registry page says that the part is "Incompatible with systems containing LacI, lactose, or IPTG."  Thus, we must make sure that our cells dont have genomic DNA for LacI.















===TetR===
TetR represses the constitutive promoter pTet.  In the presence of tetracycline, an antibiotic, a conformational change in TetR inhibits the protein from binding to the operator region.  In nature, pTet promotes TetR and TetA.  The latter which acts to pump tetracycline out of the cell, thus the pump is only activated in the presence of Tetracycline.[http://en.wikipedia.org/wiki/Tetracycline_controlled_transcriptional_activation]
The TetR, as it turns out is a very tight repressor and a range of 0 to 1 ug/ml has been shown to cause a 5 order of magnitude change in luciferase production.[http://www.ncbi.nlm.nih.gov/sites/entrez?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=1319065&query_hl=1&itool=pubmed_docsum] [[Image:Tc_bound_to_TetR.jpg|thumb|left|A tetracycline molecule binds to each of the two TetR monomers to form a dimer]]
Tetracycline is highly diffusable through cell membrane (permeation coeficient or 5.6±1.9 * 10^-9 cm/s or half equilibrium time = 35 ± 15 min) and TetR shows a very high affinity for the molecule. The binding constant of TetR to [tc-Mg+] is Ka ~ 10^9 M^-1.  When bound to tc, TetR has a low binding level to DNA of 10^5 M^-1. [http://content.febsjournal.org/cgi/content/abstract/270/15/3109]

The TetR part in the registry (BBa_C0080)










=='''Modeling'''==
===Model 1.0===

Brown iGEM 2006 Matlab model code [[Media:tristable2006.txt]]

[http://parts2.mit.edu/wiki/index.php/Table_of_preliminary_model_constants Initial Table of Constants]

[http://parts2.mit.edu/wiki/index.php/Derivation_of_the_Model_Equations Derivation of Model Equations]

===Model 2.0===
[[Media:TriStable_Log_Model.txt]]


=='''Testing'''==
There are two methods we could follow in designing the Switch.  We could randomly try different RBSs, hope it works and if not try again without having much of an understanding of why our cnostructs didn't work.  Or we can test our repressors, promoters and inducers and have a systematic approach to anaylizing our system so that when something works or doesn't work we will know why.  Thus we have designed a few tests which should give us relative and absolute values of our system that we can then plug into the model.

We decided to to test for three  values.  The combined transcription/translation rate of each repressor, the cooperativity of each repressor and the concentration of ligand needed to deactivate each repressor.  We managed to design three tests all using the same constructs so as to minimize ligations.  These tests should determine our variables independantly, i.e. changing synthesis rate should not change cooperativity of repression.

===Synthesis Rate===
The combined transcription/translation rate of the repressor is the combined strength of the promoter and the RBS.  In [[Image:alphaTest.png|thumb|left|The Alpha test Architecture]]our model this is the alpha value.  Our model predicts that the alpha values for each repressor should be fairly comparable (the stable region is along the 1 to 1 to 1 line in 3D).  Since we can't change the promoter strength very easily, we will change the RBS strength to obtain similar alpha values for all repressors.













===Cooperativity of Repression===
The cooperativity describes an inherent characteristic of a repressor's repression.  In our system we want to know how [[Image:betaTest.png|thumb|left|The Beta test Architecture]]much increased repressor concentration will increase repression.  In our model, cooperativity is an exponent, beta, so that the repressor concentration is raised to the beta ([repressor]^beta = total repression).  For beta = 1, repression increases linearly with repressor concentration.  With twice as much repressor there is twice the repression.  For beta = 2, repression increases with the square of the concetration.  Twce as much repressor leads to four times the repression.  Our model predicts that our system will be more robust with greater alpha values and the tri stable region (in the graph) will be larger.  Beta must be larger than one for the system to be stable.  











===Ligand Concentration===
Naturally, we don't want to add more ligand than we need.  In what ever application the project might find, if we wanted to change the state again by adding a different [[Image:ligandTest.png|thumb|left|The Ligand test Architecture]]ligand, we wouldn't want excessive amounts of the first ligand floating around. Furthermore, tetracycline is an antibiotic and we don't want to add so much that we are compromising the activity of the cell.













=='''Standardized Characterization'''==
===Standardizing Repressor Characterization===
Ultimately we would like to make a protocol by which all other repressors could be characterized that would be easy to construct, repeatable and useful to all applications.  Hopefully, we will find the correct combination of cell concentration and growth phase, reporter readout, induction method, cell type and variables to test for so that this information can be useful to others.

===Standardizing Promoter Characterization===
The same goes for promoter characterization, the only difference is that promoters have to be characterized relative to a standard promoter defined as one, similar to RBSs.  We should also determine how off the promoter can be, whether it be a repressable promoter or inducible promoter.

=='''Questions yet to be Answered'''==
Does fluoresence increase linearly with FP concentration?
What are normal Beta value?
Rumor has it that there is a constitutive promoter region within the AraC gene (BBa_C0080).  Is this true? Should we change the order of our genes so that AraC is at the end and promotes junk DNA?  This will affect the relative values obtained for our testing contructs.
We seem to have very low FP output for our pBAD/AraC construct.