<div id="mainpage">
<hr width="750px">
<div id="about">
{{Caltech_iGEM}}
</div>
<hr width="750px">
__NOTOC__
{|cellspacing="5" cellpadding="10" style="background:#ffffff; width: 750px;"
|-valign="top"
|style="background:#ffffff"|
=<center>Recombineering</center>=

==Background==
'''What is recombineering? Generally, how do you do it?

   Recombineering (recombination-mediated genetic engineering) is a recently developed ''in vivo'' technique for making recombinant DNA.  As a fast and efficient alternative to classically used ''in vitro'' techniques, recombineering takes advantage of lambda phage's homologous recombination proteins collectively known as Red.  Previous genetically engineered systems could not successfully insert linear DNA into ''E. coli'' due to degradation by nucleases.  However, homologous recombination of ssDNA succeeded in the presence of the Red proteins, which inhibited the degrading nuclease in ''E. coli''.  Therefore, a defective lambda prophage was engineered with lysis and replication functions inhibited and Red functions retained.  After creating cell strains containing this prophage, single-stranded oligos with the desired mutationscould successively be used for recombineering the phage.

   Recombineering (recombination-mediated genetic engineering) is a recently developed ''in vivo'' technique for making recombinant DNA.  As a fast and efficient alternative to classically used ''in vitro'' techniques, recombineering takes advantage of lambda phage's homologous recombination proteins collectively known as Red.  Previous genetically engineered systems could not successfully insert linear DNA into ''E. coli'' due to degradation by nucleases.  However, homologous recombination of ssDNA succeeded in the presence of the Red proteins, which inhibited the degrading nuclease in ''E. coli''.  Therefore, a defective lambda prophage was engineered with lysis and replication functions inhibited and Red functions retained.  After creating cell strains containing this prophage, single-stranded oligos with the desired mutationscould successively be used for recombineering the phage.

   Recombineering (recombination-mediated genetic engineering) is a recently developed ''in vivo'' technique for making recombinant DNA.  As a fast and efficient alternative to classically used ''in vitro'' techniques, recombineering takes advantage of lambda phage's homologous recombination proteins collectively known as Red.  Previous genetically engineered systems could not successfully insert linear DNA into ''E. coli'' due to degradation by nucleases.  However, homologous recombination of ssDNA succeeded in the presence of the Red proteins, which inhibited the degrading nuclease in ''E. coli''.  Therefore, a defective lambda prophage was engineered with lysis and replication functions inhibited and Red functions retained.  After creating cell strains containing this prophage, single-stranded oligos with the desired mutationscould successively be used for recombineering the phage.

   Recombineering (recombination-mediated genetic engineering) is a recently developed ''in vivo'' technique for making recombinant DNA.  As a fast and efficient alternative to classically used ''in vitro'' techniques, recombineering takes advantage of lambda phage's homologous recombination proteins collectively known as Red.  Previous genetically engineered systems could not successfully insert linear DNA into ''E. coli'' due to degradation by nucleases.  However, homologous recombination of ssDNA succeeded in the presence of the Red proteins, which inhibited the degrading nuclease in ''E. coli''.  Therefore, a defective lambda prophage was engineered with lysis and replication functions inhibited and Red functions retained.  After creating cell strains containing this prophage, single-stranded oligos with the desired mutationscould successively be used for recombineering the phage.

==Integration==

We will use recombineering to create phages which will serve as suitable background strains for this project. 
Our project requires phage strains with two main characteristics. First, the phage strains must allow easy cloning 
of heterologous constructs into them -- in this case, the riboregulated <i>N, Q</i>, and <i>cro</i> expression constructs. Second, the
strains must simultaneously be defective in the relevant N, Q or cro developmental genes while still being relatively easy to propagate.

Choosing a phage strain which facilitates insertion of heterologous sequences and tolerates their presence requires some care. The process is more involved than cloning heterologous DNA into a bacterial plasmid, for two main reasons. First, wild type lambda phage does not contain many unique restriction sites, making standard cloning techniques difficult. Second, since evolution has optimized lambda's small genome to have a high density of regulation, many stretches of DNA serve multiple functions (insert Chan et al MSB 2005 reference). This means that deleting long stretches of DNA (such as the <i>N, Q,</i> or <i>cro</i> sequences) may have unpredictable consequences.

Recombineering helps address both these concerns.

==Status and Future Plans==
'''Where are we now? What's happening next?'''

The double-layer titering assay was used to screen for plaques corresponding to successfully recombineered phage (i.e. amber mutants).  Unfortunately, the "cloudy" vs "clear" difference in amber mutant and wild type plaques proved to be more subtle than expected.  Therefore, a new approach was taken consisting of carrying out the double-layer assay with the same amber suppressor layer and now a non-suppressor layer that expresses RFP.  This modified experiment would simply require identifying plaques that pierce the bacterial lawn under visible light but appear confluent with the surrounding bacteria under fluorescence.  After screening extracted plaques         via single-layer titering in amber suppressing and non-suppressing cell strains revealed no amber mutants, we decided to go back to the recombineering process and re-design the single-stranded oligos.

==Relevant Protocols==
* [[IGEM:Caltech/2007/Protocols/Titering | Titering]]
* [[IGEM:Caltech/2007/Protocols/Recombineering_Protocol | Recombineering ]]
** [[IGEM:Caltech/2007/Protocols/Two_Layer | Two Layer Plaque Assay ]]
|}
</div>