== Bacterial signal transduction network in a genomic perspective ==

author: Michael Y. Galperin 

Tables 1 and 2 show what types of signalling molecules are present in different types of prokaryotes.

It looks like S_TKc (Serine-Threonine kinase, catalytic) would work best for E. Coli because there are not many of them in E. Coli naturally. Or, we may want to use a signalling protein that doesn't exist yet in E. Coli, to prevent confusion and false signals.


[[Image:Table001.jpg]]




[[Image:Table002.jpg]]




== Signal transduction: Hair brains in bacterial chemotaxis ==


authors: Jeff Stock and Mikhail Levit 

[[Image:ChemoRec.jpg]] [[Image:ChemoRec2.jpg]]

In their ‘ON’ state, in the absence of attractants, several receptors bind to CheA in such a way that CheA is activated over 100-fold [24], and conversely CheA binding to the receptors appears to be required for long-range structural interactions that serve to organize the array. In the ‘OFF’ state, in the presence of attractants, it is as if the CheA dimer is torn apart by binding to the receptor network.

In E. coli, at least five different receptors — Tar, Tsr, Tap, Trg and Aer — appear to be intermingled within the same complex.

a) Tar: aspartate and maltose; cobalt and nickel

b) Tsr: serine

c) Tap: Taxis towards peptides

d) Trg: ribose and galactose

e) Aer: directs taxis towards ribose, galactose, maltose, malate, proline and alanine

a. CheA is a kinase that takes a phosphate off of ATP and attaches it to itself

b. CheW connects CheA to the chemoreceptor



== Binding Proteins ==

author: Dr. Leonidas G. Bachas

'''Sensing System for Zinc Based on Zinc-Binding Protein
'''

Zinc is an essential element in our diet. Too little zinc can cause problems, but too much zinc is also harmful. Severe soil zinc deficiency can cause complete crop failure. Certain microorganisms are known to survive in highly toxic environments contaminated with toxic species such as '''mercury, arsenic, cadmium, zinc, lead, copper or nickel'''. Resistance is associated with presence of resistance operons which are precisely regulated. Operon consists of gene for regulatory protein to which toxic metals binds and induce the expression of other genes of operon. Detoxification occurs either by pumping the toxic metals out of the cell or by expression of metallothionein, a cysteine rich protein that chelates heavy metals.
We plan to take advantage of this specific binding between the regulatory protein and the toxic species in order to develop a sensing system for the target toxic analyte. In this project we have replaced the genes of the operon for zinc resistance with the genes that encode for reporter proteins to develop a biosensor for zinc.

'''Detection of Sulfate Using Periplasmic Sulfate-Binding Protein
'''

Periplasmic binding proteins from E. coli undergo large conformational changes upon binding their respective ligands. By attaching a fluorescent probe at rationally selected unique sites on the protein, these conformational changes in the protein can be monitored by measuring the changes in fluorescence intensity of the probe, which allow the development of reagentless sensing systems for their corresponding ligands. On the basis of this strategy we have evaluated several sites on bacterial periplasmic sulfate-binding protein (SBP) for attachment of a fluorescent probe for rational designe of a reagentless sensing system for sulfate. Eight different mutants of SBP were prepared by employing the polymerase chain reaction (PCR) to introduce a unique cysteine residue at a specific location on the protein. The sites Gly55, Ser90, Ser129, Ala140, Leu145, Ser171, Val181, and Gly186 were chosen for mutagenesis by studying the three-dimensional X-ray crystal structure of SBP. Different environment-sensitive fluorescent probes were then attached site-specifically to the protein through the sulfhydryl group of the unique cysteine residue introduced. Each fluorescent probe-conjugated SBP mutant was characterized in terms of its fluorescence properties and Ser171 was determined to be the best site for the attachment of the fluorescent probe that would allow for the development of a reagentless sensing system for sulfate. A calibration curve for sulfate was constructed using the labeled protein and relating the change in the fluorescence intensity with the amount of sulfate present in the sample. The detection limit for sulfate was found to be in the submicromolar range using this system. The selectivity of the sensing system was demonstrated by evaluating its response to other anions. A fast and selective sensing system with detection limits for sulfate in the submicromolar range was developed.


== An Exceptionally Selective Lead(ii)-Regulatory Protein from Ralstonia Metallidurans: Development of a Fluorescent Lead(ii) Probe ==

[http://he-group.uchicago.edu/He%20publication/P%20Chen%20Angew%20Chem%202005.pdf Lead Detection Paper (THIS IS AWESOME!)]

[http://pubs.acs.org/cgi-bin/article.cgi/jacsat/2004/126/i03/html/ja0383975.html Fluorescent tests on Pbr]

[[Media:lead_supplement.pdf | Supplement to the Lead Detection Paper (mentions work done in ecoli)]]

[http://www.ansijournals.com/pjbs/2004/6-9.pdf Can ecoli survive in lead?]

'''PbrR protein [Ralstonia metallidurans CH34]'''

Other Aliases: pMOL30_092

Genomic context: Plasmid pMOL30 (Plasmid 1)

Annotation: NC_006466.1 (114932..115370)

GeneID: 3170418


MNIQIGELAKRTACPVVTIRFYEQEGLLPPPGRSRGNFRLYGEEHVERLQFIRHCRSLDMPLSDVRTLLS
YRKRPDQDCGEVNMLLDEHIRQVESRIGALLELKHHLVELREACSGARPAQSCGILQGLSDCVCDTRGTT
AHPSD

114933..115370 (including stop codon)

'''DNA sequence:'''

atgaatat ccagatcggc gagcttgcca agcgcaccgc atgcccggtg gtgaccattc gcttctacga acaagaaggg ctgttgccgc cgccgggccg cagccggggg aattttcgcc  gtatggcga ggagcacgtg gagcgcttgc agttcattcg tcactgccgg tctctggata tgccgttgag cgacgtacgg accttattga gttaccggaa gcggcccgac caggattgcg  tgaagtcaa tatgctcttg gatgagcaca tccgtcaggt cgaatctcgg atcggagctt tgctcgaact gaagcaccat ttggtggaac tgcgcgaagc ctgttctggt gccaggcccg  ccaatcgtg cgggattctg cagggactgt cggactgcgt gtgtgatacg cgggggacca ccgcccatcc aagcgactag

''PstI site found in the sequence at base pair 373.''

'''Lead Promoter Sequence:'''

GGTTGCTTCCTATAAAAAACTTGACTCTATATCTACTAGAGGTTTTCTAATGATGGCATC
CGGGGAAAACCTTGTCAATGAAGAGCGATCTATG

'''Primers Ordered (that actually work and will be kept):'''

PbrR691 5' Primer a: GTTTCTTCGAATTCGCGGCCGCTTCTAGatgaatatccagatcggcgag

PbrR691 5' Primer b: GTTTCTTCGAATTCGCGGCCGCTTCTAGatgaatatccagatcgg

PbrR691 5' Primer c: GTTTCTTCGAATTCGCGGCCGCTTCTAGatgaatatccagatcggcgagcttg

'''Primers Ordered July 2, 2007:'''

PbrR691 3' Primer: GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACTAGTCGCTTGGATGGGC 

Promoter 5' Primer: GTTTCTTCGAATTCGCGGCCGCTTCTAGAGccgcatcatggttgcttcc

Promoter 3' Primer: GTTTCTTCCTGCAGCGGCCGCTACTAGTACATAGATGATCGCTCTTCATTGACAAGG

'''PbrR691 Sequence Analysis to remove Pst1 restriction site:'''
Pst1 Restriction Site:

5'CTGCAG

3'GACGTC

5'---CTGCA  G---3'

3'---G  ACGTC---5'
 

July 3, 2007: New Developments - Email from Dr. van der Niels

PbrR691 Sequence Analysis according to GenBank:
atgaatat ccagatcggc gagcttgcca agcgcaccgc atgcccggtg gtgaccattc gcttctacga acaagaaggg ctgttgccgc cgccgggccg cagccggggg aattttcgcc gtatggcga ggagcacgtg gagcgcttgc agttcattcg tcactgccgg tctctggata tgccgttgag cgacgtacgg accttattga gttaccggaa gcggcccgac caggattgcg tgaagtcaa tatgctcttg gatgagcaca tccgtcaggt cgaatctcgg atcggagctt tgctcgaact gaagcaccat ttggtggaac tgcgcgaagc ctgttctggt gccaggcccg ccaatcgtg cgggattctg cagggactgt cggactgcgt gtgtgatacg cgggggacca ccgcccatcc aagcgactag

1   atgaatatcc agatcggcga gcttgccaag cgcaccgcat gcccggtggt gaccattcgc
61  ttctacgaac aagaagggct gttgccgccg ccgggccgca gccgggggaa ttttcgcctg
121 tatggcgagg agcacgtgga gcgcttgcag ttcattcgtc actgccggtc tctggatatg
181 ccgttgagcg acgtacggac cttattgagt taccggaagc ggcccgacca ggattgcggt
241 gaagtcaata tgctcttgga tgagcacatc cgtcaggtcg aatctcggat cggagctttg
301 ctcgaactga agcaccattt ggtggaactg cgcgaagcct gttctggtgc caggcccgcc
361 caatcgtgcg ggattctgca gggactgtcg gactgcgtgt gtgatacgcg ggggaccacc
421 gcccatccaa gcgactag

http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=NC_006466.1&from=114933&to=115370&dopt=gb

PbrR691 Sequence According to Fasta:

ATGAATATCCAGATCGGCGAGCTTGCCAAGCGCACCGCATGCCCGGTGGTGACCATTCGCTTCTACGAAC
AAGAAGGGCTGTTGCCGCCGCCGGGCCGCAGCCGGGGGAATTTTCGCCTGTATGGCGAGGAGCACGTGGA
GCGCTTGCAGTTCATTCGTCACTGCCGGTCTCTGGATATGCCGTTGAGCGACGTACGGACCTTATTGAGT
TACCGGAAGCGGCCCGACCAGGATTGCGGTGAAGTCAATATGCTCTTGGATGAGCACATCCGTCAGGTCG
AATCTCGGATCGGAGCTTTGCTCGAACTGAAGCACCATTTGGTGGAACTGCGCGAAGCCTGTTCTGGTGC
CAGGCCCGCCCAATCGTGCGGGATTCTGCAGGGACTGTCGGACTGCGTGTGTGATACGCGGGGGACCACC
GCCCATCCAAGCGACTAG

http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=NC_006466.1&from=114933&to=115370&dopt=fasta

Inverse complement of above sequence:

CTAGTCGCTTGGATGGGCGGTGGTCCCCCGCGTATCACACACGCAGTCCGACAGTCCCTG
CAGAATCCCGCACGATTGGGCGGGCCTGGCACCAGAACAGGCTTCGCGCAGTTCCACCAA
ATGGTGCTTCAGTTCGAGCAAAGCTCCGATCCGAGATTCGACCTGACGGATGTGCTCATC
CAAGAGCATATTGACTTCACCGCAATCCTGGTCGGGCCGCTTCCGGTAACTCAATAAGGT
CCGTACGTCGCTCAACGGCATATCCAGAGACCGGCAGTGACGAATGAACTGCAAGCGCTC
CACGTGCTCCTCGCCATACAGGCGAAAATTCCCCCGGCTGCGGCCCGGCGGCGGCAACAG
CCCTTCTTGTTCGTAGAAGCGAATGGTCACCACCGGGCATGCGGTGCGCTTGGCAAGCTC
GCCGATCTGGATATTCAT

438 bases  146 aa

PbrR691 Protein Sequence According to NCBI:
MNIQIGELAKRTACPVVTIRFYEQEGLLPPPGRSRGNFRLYGEEHVERLQFIRHCRSLDMPLSDVRTLLSYRKRPDQDCGEVNMLLDEHIRQVESRIGALLELKHHLVELREACSGARPAQSCGILQGLSDCVCDTRGTTAHPSD

145

According to Niels:
“Please find below the correct sequence of the PbrR691 gene. This sequence is the inverse complement and begins at the STOP codon from pbrR691. Also indicated is the ATG start codon for pbrR691 (bold, underlined). The ATG at the end is the start of the pbrA691 gene.

TCAGGCGGGCTCGGCCAAGCTGTTGAGAATGCCGCACTCGCGCGAGGTTCGGGCGCTATCGCAGGAGCGTCGCAGATCCATTAACTCGCGCTCCAAGGCGCGCAATTCCTTCATCTTGGTCCGCACTTGCGCGATATGAGCGTCGACCAGCGCGTTCACCTCGCCGCAGCCCAACTCTGGCCGATCCCGTAAGTTCAGCAGTTGACGGATCTCATCCAGCGTCATGTCCTTCGCCCGGCAGCGGCGGATGAACAGCAAGCGCTGCAAATGGACTTCGTCATAGAGCCTGAAGTTGCCCTCGCTACGTGCAGGCTCGGGCAGCAAGCCTTCTGACTCGTAAAAGCGCACGGTCTGCACCAAGCAATCTGCCTTCTTGCCCAGTTCACCGATCCGCATCATGGTTGCTTCCTATAAAAAACTTGACTCTATATCTACTAGAGGTTTTCTAATGATGGCATCCGGGGAAAACCTTGTCAATGAAGAGCGATCTATG

493  164.3 aa

399  133 aa

There are no PstI sites in this sequence.

In order to make a gene fusion that is under control of PbrR691 you will need to place your reporter gene downstream of the ATG start codon at the end of the is sequence.”

What this means: Gene orientation according to PubMed:

plasmid: pMOL30  

http://www.ncbi.nlm.nih.gov/sites/entrez?db=gene&cmd=Retrieve&dopt=full_report&list_uids=3170418#refseq

Inverse of above sequence:

GTATCTAGCGAGAAGTAACTGTTCCAAAAGGGGCCTACGGTAGTAATCTTTTGGAGATCA
TCTATATCTCAGTTCAAAAAATATCCTTCGTTGGTACTACGCCTAGCCACTTGACCCGTT
CTTCCGTCTAACGAACCACGTCTGGCACGCGAAAATGCTCAGTCTTCCGAACGACGGGCT
CGGACGTGCATCGCTCCCGTTGAAGTCCGAGATACTGCTTCAGGTAAACGTCGCGAACGA
CAAGTAGGCGGCGACGGCCCGCTTCCTGTACTGCGACCTACTCTAGGCAGTTGACGACTT
GAATGCCCTAGCCGGTCTCAACCCGACGCCGCTCCACTTGCGCGACCAGCTGCGAGTATA
GCGCGTTCACGCCTGGTTCTACTTCCTTAACGCGCGGAACCTCGCGCTCAATTACCTAGA
CGCTGCGAGGACGCTATCGCGGGCTTGGAGCGCGCTCACGCCGTAAGAGTTGTCGAACCG
GCTCGGGCGGACT

Inverse complement of above sequence:

CATAGATCGCTCTTCATTGACAAGGTTTTCCCCGGATGCCATCATTAGAAAACCTCTAGT
AGATATAGAGTCAAGTTTTTTATAGGAAGCAACCATGATGCGGATCGGTGAACTGGGCAA
GAAGGCAGATTGCTTGGTGCAGACCGTGCGCTTTTACGAGTCAGAAGGCTTGCTGCCCGA
GCCTGCACGTAGCGAGGGCAACTTCAGGCTCTATGACGAAGTCCATTTGCAGCGCTTGCT
GTTCATCCGCCGCTGCCGGGCGAAGGACATGACGCTGGATGAGATCCGTCAACTGCTGAA
CTTACGGGATCGGCCAGAGTTGGGCTGCGGCGAGGTGAACGCGCTGGTCGACGCTCATAT
CGCGCAAGTGCGGACCAAGATGAAGGAATTGCGCGCCTTGGAGCGCGAGTTAATGGATCT
GCGACGCTCCTGCGATAGCGCCCGAACCTCGCGCGAGTGCGGCATTCTCAACAGCTTGGC
CGAGCCCGCCTGA 

Inverse Complement Sequence BLASTed: 

http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?cmd=Retrieve&db=Nucleotide&list_uids=93352797&dopt=GenBank&WebEnv=0cvy0r_s7OgtB31ICc6NXtgu1iJMjqrGLyFI4oYcQxj2kT9qtcQijdQrtTUUbUUmddm6suqFMcOWS9o%40255F25AC68A58C60_0055SID&WebEnvRq=1

gb|CP000352.1| Ralstonia metallidurans CH34, complete genome

Length=3928089

CP000352.1

 Features in this part of subject sequence:

   transcriptional regulator, MerR family

   Heavy metal translocating P-type ATPase

 Score =  911 bits (493),  Expect = 0.0

 Identities = 493/493 (100%), Gaps = 0/493 (0%)

 Strand=Plus/Minus

   /product="transcriptional regulator, MerR family"

                     /protein_id="ABF06988.1"

http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=ABF06988.1

Indicates protein sequence is:

ORIGIN      

1   msaqpstvty titdlarefd itprairfye dqgllapdre gpsgrrrvyn srertrlklt

61  lrgkrlgltl neireildly esprdtapql erflhllagh rgtlerqled lqaqlaeidq

121 herqcqalla aqhaknagnk tpta

July 9, 2007: Actual PbrR691 gene sequence accepted. Earlier sequence was of PbrR.

New primers ordered for the gene sequence itself July 9, 2007 from Invitrogen as follows:

GTTTCTTCGAATTCGCGGCCGCTTCTAGATGATGCGGATCGGTGAACTG [7.9.07 5 PbrR691]   
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTAGCTCGAGCTTCAGGCGGGCTCG [7.9.07 3a PbrR691]   
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTAGCTCGAGCTTCAGGCGGGCT [7.9.07 3b PbrR691]
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTATTACACGCTCGAGCTTCAGGCG [7.9.07 3c PbrR691]




== Results of Sequencing Parts 1 through 15 (8/21/2007)==

1,2,3: PbrR691

4,5,6: Promoter

7,8,9,10,11,12: combined PbrR691 + promoter

13: promoter + PbrA

14,15: Combined PbrR691 + promoter + PbrA


1. Good prefix; no part; Good suffix

2. Bad EcoRI; no part; Good suffix

3. Bad EcoRI; Good part; Good suffix

4. Bad EcoRI; Good part; Good Suffix

5. Bad EcoRI; 2 point mutations in promoter; Good Suffix

6. Bad EcoRI; Good part; Good suffix

7. Bad EcoRI; Good PbrR691 and promoter; Good suffix

'''8. Good prefix; Good PbrR691 and promoter; Good suffix'''

9. Bad EcoRI; 1 point mutation in PbrR691 and Good promoter; Good suffix

10. Bad EcoRI; no part; Good suffix

'''11. Good prefix; Good PbrR691 and promoter; Good suffix'''

12. Bad EcoRI; Good PbrR691, 1 point mutation in promoter; Good suffix

13. Bad EcoRI; no part; Good suffix

14. No prefix, suffix, or part, just plasmid DNA

15. Good prefix; no part; Good suffix


In Summary: 

parts 8 and 11 look perfect. 

A total of 4 out of 15 EcoRI sites looked correct (GAATTC), the others all looked like GGATTC.

6 sequences (1,2,10,13,14,15) had no part in them.

The suffixes all looked perfect, except for part 14 which had nothing except plasmid DNA

The fact that the EcoRI site is bad in 11/15 parts, and all other parts of prefixes and suffixes are fine, suggests to me that maybe the plasmid vector that we put the PCR parts into may have been the source of the mutation. This would make sense since it is only the first have of the EcoRI site, which I think may have come from the plasmid itself, not from the sticky ends of our part. Of course, there would have been a problem when GGATTC tried to anneal to CTTAAG (though a slight one, because T's and G's actually aren't all that bad at binding to each other, believe it or not), but the cell's machinery would have attempted to correct this problem by changing one strand or the other. Apparently some cells changed the incorrect strand, and some cells changed the correct strand to allow full DNA hybridization. Unfortunately for us, most cells changed the strand that was correct to begin with. That is my hypothesis.