== Quorum Sensing ==

Basically all of the parts that are available are derived from the [http://www.che.caltech.edu/groups/fha/quorum.html LuxI/LuxR system] found in ''V. fishcheri''. The Voigt paper that we read a long time ago used this system to demonstrate a cell-density dependent expression of invasin in E. Coli ([[Media:Voigt.pdf]]). 

The basic idea that Perry and I discussed requires two basic parts which Perry transformed into three separate tubes of Top10 cells this afternoon: 
# An HSL signal sender ([http://parts.mit.edu/registry/index.php/Part:BBa_F1610 BBa_F1610])
# An HSL signal receiver attached to a report 
([http://parts.mit.edu/registry/index.php/Part:BBa_I13263 BBa_I13263] or 
[http://parts.mit.edu/registry/index.php/Part:BBa_I13272 BBa_I13272])
The only difference between the two signal receivers is a different YFP reporter protein.

A quick rundown of how it will work: we will stick some sort of promoter in front of the [http://parts.mit.edu/registry/index.php/Part:BBa_F1610 BBa_F1610] in order to produce  constitutive expression of HSL. We can play around with which promoter we want to use in order to tweak the sensitivity of the system. This HSL will normally diffuse quickly. Meanwhile, there is constitutive expression of the protein produced after transcription and translation of luxR (called R from now on) is continually going on in the bacteria with the receiver. The HSL will bind to the R and these bound complexes will dimerize and activate transcription of the YFP reporter. However, normally the concentration of HSL is too low and the equilibrium highly favors unbound HSL and R. In areas of high cell concentration, the concentration of HSL will be great enough to shift the equilibrium toward the bound complex. This bound complex will then activate the transcription of the reporter gene. An interesting note is that the bound complex supposedly also represses the luxR gene according to the Biobricks parts list. However, I haven't found any confirmation of this. If it is true, then it means that along with the activation of transcription of the reporter gene the amount of luxR will decrease and transcription of the YFP reporter would probably decrease. 

There are a few ways we could approach this:
# Have both the signaller and receiver+reporter parts in the same plasmid
# Have the signaller part in one plasmid and the receiver+reporter part in another plasmid, both in the same bacterium
# Have two different bacteria: one with the signaller part and one with the receiver+reporter part

Perry asked Mike about the first two possibilities and confirmed that both should be doable. The last possibility was demonstrated in the Voigt paper.

I'm not completely sure what the advantages/disadvantages of each system is, but I think we should try all three possibilities.

Also, if we can get this to work, we could potentially build more complex systems that involve logical gates. One example paper ([[Media:Pulse.pdf]]) used two bacteria and five separate parts controlled by inducible/repressible promoters two create a pulse of fluorescence.

== Signal Transduction ==

*Nick and I were talking over a paper on OmpR/PhoB and we thought of a possible method of making bacteria transduce a signal in response to binding our target.  The OmpR/PhoB family of membrane proteins dimerize and activate in response to binding a ligand.  Perhaps we can induce dimerization and activation of the proteins by holding them close together.

*We could do this by adding a library to the extramembrane portion of OmpR/PhoB and selecting for adhesion to a ligand.  Then we could co-express two different genes for OmpR+library in a single E. coli. Hopefully the two different target-binding portions will bind separate portions of the target, and will bring the two monomers of OmpR close enough together to activate them.  

*Nick refined this idea by suggesting that we use a protein with clearly defined C-terminal and N-terminal domains as a target.  We could use a protease to snip the C-terminal and N-terminal domains apart, then separately select for libraries that bind to each.  Once we have sequences that bind to the C-terminal and N-terminal domains separately, we can combine those two sequences in the same plasmid. This would ensure that the two OmpR genes target different portions of the bacteria, and hopefully that they'll be brought close enough to dimerize.  
-Alex

*Papers of Interest
** [[http://www.hhmi.org/research/investigators/stock.html OmpR/PhoB Overview See Figure 1]
** A common dimerization interface in bacterial response regulators KdpE and TorR (OmpR family members) PMID: 16322582 Protein Sci. 2005 Dec;14(12):3077-88.
== Adhesion Targets ==

Brainstorming of Possible Adhesion Targets (feel free to add/reorganize/whatever--I just thought I'd get this started):

*Keratins -- was thinking of new lines of "hair conditioners" --> these tend to add body to the hair shaft by using bonding polymers; bacteria could be the new line of "bonding polyproteinecious entities" etc etc


'''textiles''' 
*silk 
*wool 
*polyester 
*nylon fibres 
*Could clone E coli that secrete pleasant smells and similarly bind to these fibers -- a living deodorant!


'''eukaryotic cells'''
*Mosquito gut epithelial cells as targets


'''other mammalian'''
*Laminins
*fibronectin


'''Mammalian cell surface proteins'''



'''Protozoan membrane proteins'''
*By the way, I was thinking about expressing plasmodium falciparum proteins on the surface of E coli and maybe showing somehow that an immune rxn can be elicited in mice or something like that e.g.  in humans, "High levels of plasma chitotriosidase are a marker of macrophage activation..."
*Speaking on targets, heregoes:
*MSP-1 --> well characterized
*CSP
*EBA-175, a 175 kDa 'erythrocyte binding antigen' from P. falciparum 
*DBP, Duffy-binding protein from P. vivax and P. knowlesi 
*SSP2, Plasmodium sporozoite surface protein-2. Also known as TRAP (thrombospondin-related adhesive protein). 
*Proteins with homology to SSP2/TRAP from Toxoplasma (MIC2), Eimeria (Etp100), and Cryptosporidium 
*CTRP, circumsporozoite- and TRAP-related protein of Plasmodium found in the ookinete stage 
*(see http://www.tulane.edu/~wiser/malaria/cmb.html#junction)
*MSP-1 polypeptide fragments


'''Viral Membrane Proteins'''
*Hepatitis C surface proteins
**we could use the adhesion to these proteins to adapt the ELISA assay to the detection of such proteins '''in vitro'''
*[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=3882238&dopt=Abstract Influenza A M2 Protein]
*Neuraminidase
*Hemaglutinin
*Semliki Forest membrane proteins
*[http://jvi.asm.org/cgi/content/full/79/7/4415 Epstein Barr LMP1]
*[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=249393 Epstein Barr LMP2]
*HIV gp120


'''Cancer-Related Membrane Proteins'''
*[http://www.molecular-cancer.com/content/5/1/41 Eag1 Potassium Channels]
*[http://bioinformatics.org/pcgdb/Genes/muc1/muc1.htm Muc1]
*[http://cancerres.aacrjournals.org/cgi/content/abstract/67/4/1594 NGEP-L]
*[http://clincancerres.aacrjournals.org/cgi/content/abstract/7/12/4182?ck=nck HP59]
*[http://clincancerres.aacrjournals.org/cgi/content/abstract/2/11/1837 gp75] - Surface expression of gp75 on mouse melanoma cells correlates with the ability of a monoclonal antibody against gp75 to reject melanomas in syngeneic mice 



'''Assorted'''
*Presenilin

== 2 Component Systems ==

* We should start researching these as well as the tagets.  
* Perhaps The TF's (myself included, of course) can help point out some papers with which to start.
- Nick
*How about arsenic sensor using an ''ars'' regulator, ''ars'' promoter and ''GFP'' gene -- we can have it just warn one if the arsenical concentration is greater than 5 ppb.

* Maybe look over the light sensitive bacteria paper again, they use a 2 component system there.
<biblio>
# sb1 pmid=16306980
</biblio>

*Controlled spider-silk production in the presence of calcium ions or vitamin K
**Could have the bacteria attach to skin (keratin, using LppOmpA+appropriate peptide sequence) and then , since Ca2+ ions increase when one bleeds, could have the bacteria respond to this by spritzing out spider silk protein (antimicrobial). It will matter less that monocytes will attack these bacteria since we just want them to play a role early on during the wound-healing process (they may very well contribute to debris used to create a wound plug upon their death.
In case we still want to tackle the macrophage activity headon, then I suggest we immunoisolate the bacteria using alginate beads.
**A second application would be to select bacteria that have peptides with binding affinity to silks and then have them attach to individual silk yarns --> Thus, when the yarns are soaked in culture media, the bacterial secretions (silk) increase the thickness of the yarns --> conversely, when dried out, the bacteria are inactivated and subsequent washing reduces yarn thickness through shrinkage and wear. In effect, you'd have clothes that replenish themselves. Additionally, we could use the research by MIT's group last time on sweetsmelling ''E.coli'' to make yarns innately sweet-smelling.

== Cascade Network Structure ==

* Some loose brainstorming on a possible reporter network that can be activated from the membrane protein as we discussed would be beneficial as well, if we plan to do something like that.
* May be good to start with something similar to what has been done previously and hook it up to GFP.
* Some TF direction would be warranted here as well.
- Nick

== Previous Brainstorming Ideas ==

Btw, can we reorganize the brainstorming page to put the older brainstorming into folders or something and only have the stuff relevant to our project still on the front page? I have no idea how to do this...thanks. -Shaunak

Hope you like the formatting.<br>
--Stephanie (5/12)

[http://openwetware.org/wiki/IGEM:Harvard/2007/Brainstorming/423to59 Brainstorming, 4/23 to 5/9]<br>
[http://openwetware.org/wiki/IGEM:Harvard/2007/Brainstorming/miscpre59 Misc Brainstorming, pre-5/9]<br>
[http://openwetware.org/wiki/IGEM:Harvard/2007/Brainstorming/405 Brainstorming from Second Meeting, 4/05]<br>
[http://openwetware.org/wiki/IGEM:Harvard/2007/Brainstorming/319 Brainstorming from First Meeting, 3/19]<br>


------------------------------------------------------------------------------------------