AiiA 

 The aiiA gene has 750 nucleotides and encodes a protein of 250aa with a predicted molecular
 mass of 28,036Da and an isoelectric point     of 4.7. The protein has no hydrophobic signal
 pepetide at the N-terminus and therefore it is believe that it is not secreted. This is supported
 by the observation that when aiiA  is expressed in E.coli DH5alpha or Bacillus 240B1 cells no
 autoinducer inactivation is detected in the supernatants of these cultures. The AiiA protein has
 no significant overall homology with other known proteins, but based on the presence of two well
 conserved motifs it is believed that it is a metalloenzyme.
 '''Taken from [[http://parts.mit.edu/registry/index.php/Part:BBa_C0060]]'''

==='''Aiia from Dr Rupert Fray'''===

Dr Rupert fray has agreed to send us some of his functional aiiA. It has been expressed in tobacco plants. He is using it to quench any quorum sensing systems operating in the rhizosphere and thus prevent any quorum mediated virulence factors from operating, thereby preventing soil pathogens from infecting the plant's roots. I think this is ingenious however it does not guarantee that E.coli Dh15a can express the gene. 

First I checked the codon usage. This table ''(fig1)'' is the output from an analysis of the new aiia gene It shows that e-coli should be able to express this protein efficiently. I have highlighted one potential problem in that the gene from Dr Rupert Fray uses a UAG codon which is rarely used by E-coli, however there is only 1 per protein so it may not be a big problem. [[IGEM:IMPERIAL/2006/project/Oscillator/aiiA/tabels Codon usage table comparision]] 

 '''Fig1 Codon usage table of the aiiA gene from Dr Rupert Fray'''
 UUU 27.9(     7)  UCU 15.9(     4)  UAU 35.9(     9)  UGU 15.9(     4)
 UUC 15.9(     4)  UCC  4.0(     1)  UAC  8.0(     2)  UGC  0.0(     0)
 UUA 59.8(    15)  UCA  4.0(     1)  UAA  0.0(     0)  UGA  0.0(     0)
 UUG 19.9(     5)  UCG 12.0(     3)  <font color = red>UAG  4.0(     1)</font>  UGG  4.0(     1)  
 
 CUU 19.9(     5)  CCU 12.0(     3)  CAU 31.9(     8)  CGU 12.0(     3)
 CUC  0.0(     0)  CCC  0.0(     0)  CAC  4.0(     1)  CGC  4.0(     1)
 CUA  4.0(     1)  CCA 23.9(     6)  CAA  8.0(     2)  CGA  4.0(     1)
 CUG  0.0(     0)  CCG 23.9(     6)  CAG 12.0(     3)  CGG  0.0(     0)  
 
 AUU 47.8(    12)  ACU 12.0(     3)  AAU 27.9(     7)  AGU 19.9(     5)
 AUC  8.0(     2)  ACC  0.0(     0)  AAC 12.0(     3)  AGC  0.0(     0)
 AUA 12.0(     3)  ACA 35.9(     9)  AAA 39.8(    10)  AGA  4.0(     1)
 AUG 19.9(     5)  ACG  8.0(     2)  AAG 15.9(     4)  AGG  4.0(     1)
 
 GUU 35.9(     9)  GCU 15.9(     4)  GAU 35.9(     9)  GGU 23.9(     6)
 GUC  4.0(     1)  GCC  0.0(     0)  GAC  8.0(     2)  GGC  4.0(     1)
 GUA 15.9(     4)  GCA 15.9(     4)  GAA 91.6(    23)  GGA 31.9(     8)
 GUG 19.9(     5)  GCG  8.0(     2)  GAG 23.9(     6)  GGG 12.0(     3)


The gene was isolated from an unknown bacteria present in the soil, therefore will be different to the registry. There is a remote chance that the differences could be due to PCR however it is '''far more likely''' that the differences between this sequence and published/registry sequences are due to variation in the natural population and are most likely silent and therefore should not affect function. I checked the homology of the protein sequence of Dr Fray's aiia against the NCBI database and it is more than 99.99999% identical. I am sure that the mutations will not affect function. The varaitions in sequence are due to random mutations in the DNA but due to the fact that translation uses a degenerate code these mutations do not result in a changed protein sequence.

 Protein Sequence of Rupert Fray’s aiia -  MTVKKLYFVPAGRCMLDHSSVNSTLTRGELLDLPVWCYLLETEEGPILVDTGMPESAVNNEGLFNGTFVEGHILPKMTEEDRIVNILKRVGYEPEDLLYIISSHLHFDHAGGNGAFTNTPIIVQRAEYEAAQHSEEYMKECILPNLKYKIIEGDYEVVPGVQLLYTPGHTPGHQSLLIETEKSGPVLLTIDASYTKENFEDEVPFAGFDSELALSSIKRLKEVVRKENPIVFFGHDIEQEKSCKVFPEYI
 Sequence of Registry aiia -               MTVKKLYFVPAGRCMLDHSSVNSTLTPGELLDLPVWCYLLETEEGPILVDTGMPESAVNNEGLFNGTFVEGQVLPKMTEEDRIVNILKRVGYEPEDLLYIISSHLHFDHAGGNGAFINTPIIVQRAEYEAAQHSEEYLKECILPNLNYKIIEGDYEVVPGVQLLHTPGHTPGHQSLLIETEKSGPVLLTIDASYTKENFENEVPFAGFDSELALSSIKRLKEVVMKEKPIVFFGHDIEQERGCKVFPEYIAANDENYALVA

There are a few discrepancies between Dr Frays aiia and the registry such as the registry has a degradation tag and there are a few amino acid substitutions but it is unlikely they will affect the overall fold.

I also checked for restriction sites in the sequence that Dr Fray sent. There are no sites for any of the restriction enzymes involved in ligation.

'''Summary'''

'''I would expect our cells to express this protein''' However we may observe an effect where the maximum translation rate is limited by the available UAG tRNAs not the amount of mRNA in the cell, this will not happen with registry aiiA. This aiiA is not degradation tagged so it will have a long half-life in the cell but it is an available last resort if the registry aiiA really doesn’t work.

[[User:JohnChattaway|JohnChattaway]] 12:12, 23 August 2006 (EDT)

==aiiA in General==

There are two aiiA genes in the registry

*[http://parts.mit.edu/r/parts/partsdb/view.cgi?part_id=4672 Untagged]
*[http://parts.mit.edu/r/parts/partsdb/view.cgi?part_id=159 Tagged]

There are several useful devices available:

*[http://parts.mit.edu/registry/index.php/Part:BBa_I0460 BBa_C0060]
&nbsp; If the gene in the registry is correct then we could simply place this part onto the end of our predator cell construct. There is no FLAG tag however so we couldn't measure the concentrations of it. This would be a last resort. 

*[http://parts.mit.edu/registry/index.php/Part:BBa_S01007 S01007]
&nbsp; This is similar to the above part as the aiiA is un-flagged, however there is a CFP so we would get more information about it's production.

*[http://parts.mit.edu/registry/index.php/Part:BBa_I0403 I0403]
&nbsp;  I am curious as to why this was not originally chosen to test aiiA. This is the part above placed in front of an arabinose promoter so you can see if the promoter has been activated through expression of ECFP