[http://parts.mit.edu/r/parts/htdocs/Assembly/rbs_cds.cgi Here's] a page explaining why the BioBricks prefix for coding regions is different from the BioBricks prefix for all other parts as described by Reshma this morning.

==Protocols for Primer Design==

===Forward and Reverse Primer Designs===
*Forward Primer Design
#Add the following sequence to the 5' end of the primer in the 5' to 3' direction: GTT TCT TCG AAT TCG CGG CCG CTT CTA G 
#Add to the 3' end of this sequence the first 18-22 bases of the template strand's open reading frame (ATG...).

*Reverse Primer Design
#Add the following sequence to the 5' end of the primer in the 5' to 3' direction: GTT TCT TCC TGC AGC GGC CGC TAC TAG TAT TAT TA 
#Add to the 3' end of this sequence the <b>reverse complement</b> of the last 18-22 bases of the template starand's open reading frame, excluding the stop codon.

*Notes
#One can check their primers' annealing sequence's GC content, melting temperature, etc. at http://www.idtdna.com/analyzer/Applications/OligoAnalyzer/ by simply copying and pasting the sequence into the box and clicking the "analyze" button.  One should shoot for a GC content between 40-60% for each of the primers.  One should also attempt to get their two primers' melting temperatures within 2 deg. C of one another.
#By copying and pasting the whole primers into the box, one can see the energetics of possible hairpin structures that can form by clicking on the "hairpin" button.  One can also check the energetics of possible self-dimers that can form by clicking the "self-dimer" button.  In addition by clicking on the "hetero-dimer" button and entering the other primer, one can check the energetics of possible hetero-dimers that can form between the two primers.

===Site-Directed Mutagenesis Primer Design===
#Go to the site, http://bioinformatics.org/primerx/.
#Click on "I want to design a primer pair based on a mutation in my template DNA sequence" at the bottom of the page.
#Either upload a text file on the top line or copy and paste your original template DNA onto the bottom space for step 1.
#For step 2, type in the mutation code following the examples below.
#For step 3, click on the "QuikChange Site-Directed Mutagenesis Kit by Stratagene" option.
# In step 4, confirm that your mutation has been implemented correctly into the system.
#Set parameters for your desired primers in step 5.
#In the results section, choose which primers you find to be the most desirable, taking into consideration GC content, melting temperature, etc.
#If no primers result, you will most likely need to change your parameters in step 5.

==06/09/06 Preliminary Transformation Lab==

===Sources===
*Note, used 1/2 of designated source 
#SAMT in PET28 1.6 &mu;g
#BAMT in PET28 2.0 &mu;g
#BSMT in PET28 2.0 &mu;g

===Dehydrated DNA Recovery (from paper)===
*Final DNA Concentration: 1ng/&mu;L

#Add appropriate amount of TE buffer to 1.7mL Eppendorf tubes (500&mu;L for BAMT and BSMT; 400&mu;L for SAMT).
#Carefully cut out paper that contains DNA and place into Eppen.
#*Ensure submergence with toothpick 
#*Wait for 10 minutes

===Transformation===
#Followed chemical transformation protocol listed [[Transforming chemically competent cells | here]].
#*Used 2&mu;L of plasmid/DNA.
#*Did not do heat shock
#*Used 2xYT
#*Plated 200&mu;L cells