==Shortlist==
#[[IGEM:MIT/2006/System brainstorming/Smell-o-Rama]]
#[[IGEM:MIT/2006/Other]]
#[[IGEM:MIT/2006/System brainstorming/Autotrophic bacteria]] - filed some in in case its still on the table?

'''Post your project ideas here.'''

See the [[IGEM:MIT/2005/System Brainstorm|brainstorming]] of last year's team.

#[[/Scent subprojects/]]

==Rough Ideas==
#Bacterial scents (RS)
##Living Lamp (RS)
#Fast time scale switching
##Analog Clock (AC)
#Cool part domestication
#*Rhodopsin
#*Magnetotaxis
#*Competence
#*CC Signaling
#*Chemotaxis
#*Linear induction
#Organism Domestication
#*meso, square bacteria, fast ''E. coli'', cyanobacteria
#Do some work with stationary phase, auto-inducing media (a la Studier)
#Multiple genetic populations interacting
#Organisms that eat photons (JK)
##rhodopsin (AC)
##Synthetic Vesicles (Libchaber)
#meso (DE)
##laundry list of parts
##put genome in a YAC
##refactoring/redesign
##knock out restriction system 
##genome boot-up
##genome transfer (DE)
#autolyse cells
#Cell/Electrical connections

==Graveyard==
'''Some may be resurrected'''

#Mitochondrial Liberation & Synthetic organelle(BC)
#Square Bacteria (JK)
#Maze (JK)
#wood eater (SS)
#wood-o-genesis(SS)
#Diagnostic Bacteria (BC)
#Characterization methods
#*FCS
#*MS2 bound to mRNA
#Subset of composable promoters and RBSs
#Plasmid-level parts
#*Single-copy
#Synthetic genomes (''a la'' T7)
#Assembly
#*''in vitro'', recombination
#Phage

==Cell/Electrical Connection==

Connect cells to some kind of electrical system. Would allow for easy monitoring, interesting feedback systems, and allow creation of hybrid systems.

* rhodopsin (proton pump changes pH)
* [http://www.wired.com/news/technology/0,70882-0.html Magnetic] bacteria generating electricity
* Genetic controlled expression of cell surface enzyme that interacts with substrate on surface that changes electrical properties: <cite>collier06</cite>

<biblio>
#collier06 pmid=16461913
</biblio>

==Auto-prep Cells==

Under induction control, express
* Lysis gene
* protease
* either RNase or DNase (depending on whether prepping DNA/RNA)

Spin down cells, perhaps do a ethanol precipitation. Can easily select for proper performing cells.

===Things to think about===
* Is there any way to enzymatically separate chromosomal from plasmid DNA?
* If have different control pathways that also expresses BioBrick enzymes, then may be able to prep and cut at the same time. A bit further out in fantasy land: using 3-antibiotic selection or other method, we could have cells auto-assemble parts. Grow up one cell line, induce with X which preps and cuts with ES, grow up another cell with second part, induce with Y which preps and cuts with XP, mix two lysates together, add destination plasmid, ligase, and transform new cells.
* Would be pretty modular in being able to be built incrementally by externally supplying other enzymes.
* Ideally, would require no chemicals (i.e. ethanol). Probably no way to get away from one centrifuge step to get rid of cell debris (what else other than nucleic acids would remain soluble?)

===DNA prep only===

An alternative is DNA production only. Find a DNA secreting system (Agrobacterium for instance), put the secreting signal on the plasmid. The cells continually make and secrete the desired DNA (DNA bioreactor). By not lysing the cells, it makes it easier to separate the cells from the DNA.

===References===
<biblio>
#garrett81 pmid=6457237 
#jain00 pmid=10639478
#phageLL-H http://herkules.oulu.fi/isbn9514250826/html/x1288.html
</biblio>

==Organisms that eat photons==
[[Image:BeingalgaeSetup.jpg|thumb|left|Algae are very green.  Don't worry cyanobacteria are too.]]
===Cyanobacteria===
Looks like cyanobacteria are significantly easier to genetically manipulate than algae.  Cyanobacteria have Photosystem I/II and are thought to be the ancestor of chloroplasts (via endocytosis).  Not sure why GreenFuel et al don't use them in their reactors, it is possibly because they don't have as high a lipid content as the algae (Dunaliella) that they use currently.  Two popular cyanobacteria strains are PCC7942 and PCC6803.  Peter has both of these strains available, as well as some of the plasmids for genetic manipulation, he has also made & tested the growth media BG11.


====PCC6803====
*Synechocistis PCC6803 is a naturally transformable cyanobacterium which will give you colonies on hard agar in 4 days. It is also heterotrophic, so you can grow it on glucose in the dark. (thanks to [[Peter Weigele]] for the info)
*Knock in/out system available for 6803 w/ a sucrose selection/counter selection.  Keith Tyo in the Stephanopolous lab has the plasmids, etc, but Peter is in the process of getting them from him.

====PCC7942====
*S.elongus (7942) is the model system for circadian clocks in prokaryotes (its used in the vanO lab), so that might be a cool hook to play with as well.
*Genetic manipulation was worked out for 7942 largely by Sue Golden.

Another suggestion from [[Peter Weigele|Peter]]:
"Something else you might want to consider is hydrogen from a
photoheterotroph, such as Rhodopseudomonas palustris. You can get H2
from an acetate feedstock. Acetate is a waste product in many
industrial fermentations and is an energy poor carbon source. Rhodo
uses light to kick up the electrons to an energy level where they can
be used to do work. Some protons get moved around too to make a
gradient to drive ATP synthesis. The genome is sequenced and the
strain is manipulable"



===H2 sensor===
The H<sub>2</sub> sensing circuit has been studied in detail in ''Ralstonia eutropha''<cite>Kleihues</cite>.  The network consists of a hydrogenase-like protein to control gene expression and also a two-component regulatory system.  This bacterium can metabolize Hydrogen gas as an energy source and the hydrogen sensor is used to regulate the synthesis of the metabolic enzymes.

''Ralstonia eutropha'' is a gram negative bacteria.  [[Peter Weigele]] has the strain and the Sinskey lab has plasmids to transform it. It expresses two hydrogenases, one membrane bound (MBH), the other cytoplasmic (SH).  The operons for both of the multi-protein complexes are co-regulated.  HoxA is the key regulator.  HoxB,C and J form the signal transduction network.  HoxBC appears to be the hydrogen receptor and HoxJ inactivates HoxA by phosphorylation.  When HoxBC is bound to Hydrogen, it inactivates HoxJ <cite>Lenz</cite>.  

Kamachi and coworkers have produced a Light-driven hydrogen production system in Synechocystis.  Not directly related to the H<sub>2</sub> sensor but maybe cool for the photon eating stuff above.



====References====
<biblio>
#Kleihues pmid=10781538
#Kamachi pmid=16542111
#Lenz pmid=11931556
</biblio>

===Greenfuel plugin===
Samantha looking into this.

===Saccharophagus degradans ===

Very recent paper showing that Saccharophagus degradans can break down cellulose and all other major polysaccharides <cite>taylor06</cite>. There could be uses in converting plant biomass to energy.

<biblio>
#taylor06 pmid=16707677
</biblio>

===E. coli===

E. coli has receptors for blue light. Could perhaps get another light control channel.

<biblio>
#wright06 pmid=16707688
</biblio>

==Living lamp==
According to this [http://www.oozinggoo.com/howto.html site], most homemade lava lamps are built using a mixture of mineral oil and 70-90% isopropyl alcohol (with possibly some supplemented chemicals to help the lamp work better.)  Obviously such a method wouldn't work for us. 

However, according the [http://www.oozinggoo.com/ll-form1.html lava lamp patent descriptions], ''"The clear liquid is roughly 70/30% (by volume) water and a liquid which will raise the coefficient of cubic thermal expansion and encourage the movement. The patent recommends slip agents such as propylene glycol for this. However, glycerol, ethylene glycol, and polyethylene glycol (aka PEG) are also mentioned as being sufficient."''  This sentence implies that we ought to be able to use something other than alcohol.  

The other relevant patent says, ''"A display device comprising a container having two substances therein, with one of the substances being of a heavier specific gravity and immiscible with the other substance, with the first substance being of such a nature that it is either substantially solid at room temperature or is so viscous at room temperature that neither will emulsify with the other liquid, and when heat is applied to the container, the first substance will become flowable and move about in the other substance.''

''...The liquid in which the globule is suspended is usually dyed water, but not necessarily so. The other liquid is chosen with very many considerations in mind, including the relative densities of the liquids at the desired operating temperature; the fact that the liquids must be immiscible; the fact that the surface tension must be such that the globule does not adhere to the walls of the container; the relative coefficients of thermal expansion of the liquids; and the shapes that are obtained during operation. A suitable liquid for the globule has been found to comprise mineral oil, paraffin, carbon tetrachloride and a dye or dyes. However, undue shaking or sharp impacts, especially during transport of the display device, can cause total or partial emulsification of the globule."''  My guess is that most homemade lava lamps are made from an alcohol mixture because it is cheaper and possibly also easier to achieve the lava effect.

Also note that they recommend putting a dimmer switch on the bulb below the lamp to be able to regulate the heat output.

===Issues===
#We'd have to do some research to see if media or media supplemented with something would be
##nontoxic to cells
##have the necessary properties to achieve the lava effect at ~37&deg;C
#Luminescence requires oxygen so we'd have to oxygenate the contents of the lava lamp which might interfere with the lava effect. [from TK]

===References===
#[http://www.oozinggoo.com/howto.html Lava Lamp how-to]
#[http://www.freepatentsonline.com/image-3570156-1.html US Patent #3,570,156]
#[http://www.freepatentsonline.com/3387396.html US Patent #3,387,396]

==Bacterial scents==

{| border="1"
|-
|'''Precursor'''
|'''Enzyme'''
|'''Compound'''
|'''Scent'''
|'''References'''
|-
|benzoic acid & ''S''-adenosyl-L-methionine (SAM)
|S-adenosyl-L-methionine:benzoic acid carboxyl methyltransferase (BAMT)
|[[Wikipedia:Methyl benzoate|methyl benzoate]]
|pleasant smell
|<cite>Pott-PlantPhysiol-2004</cite>
|-
|''trans''-cinnamic acid & ''S''-adenosyl-L-methionine (SAM)
|?
|[[Wikipedia:Methyl cinnamate|methyl cinnamate]]
|cinnamon?
|
|-
|jasmonic acid & ''S''-adenosyl-L-methionine (SAM)
|S-adenosyl-L-methionine:jasmonic acid carboxyl methyltransferase (JMT)
|[[Wikipedia:Methyl jasmonate|methyl jasmonate]]
|jasmine
|
|-
|salicylic acid (SA) & ''S''-adenosyl-L-methionine (SAM)
|S-adenosyl-L-methionine:salicylic acid carboxyl methyltransferase (SAMT)
|[[Wikipedia:Methyl salicylate|methyl salicylate]]
|wintergreen
|<cite>Ross-ArchBiochemBiophys-1999, Negre-ArchBiochemBiophys-2002, Zubieta-PlantCell-2003</cite>
|}

===Expert advice===
From Natalia Dudareva, Purdue University:
*thinks that you can smell wintergreen from E. coli cultures expressing SAMT with salicylic acid in the media

From [http://www.biology.lsa.umich.edu/research/labs/pichersky/ Eran Pichersky], University of Michigan:
*E. coli cultures expressing SAMT with salicylic acid in the media will have a detectable wintergreen smell
*eliminate indole pathway (responsible for bad E. coli smell) to strengthen the scent.
*have shown production of several scent compounds in E. coli

===SAMT===
*''C. breweri''
**DNA and protein sequence known
**Expressed in ''E. coli''
**Methyl salicylate has been extracted from spent medium of ''E. coli'' cells when medium was supplemented with salicylic acid
**Genbank AF133053
**also can use benzoic acid as a substrate but with lower efficiency
**crystal structure available
*''A. majus'' (Snapdragon)
**DNA and protein sequence known
**Expressed in ''E. coli''
**Methyl salicylate has been extracted from spent medium of ''E. coli'' cells when medium was supplemented with salicylic acid
**also can use benzoic acid as a substrate but with lower efficiency
**Methyl benzoate has been extracted from spent medium of ''E. coli'' cells when medium was supplemented with benzoic acid
*''S. floribunda''
**Genbank AJ308570
*''A belladonna''
**Genbank AB049752

===JMT===
*''A. thaliana'' AY008434

===BAMT===
*Snapdragon AF198492

===References===
<biblio>
#Negre-ArchBiochemBiophys-2002 pmid=12361714
#Ross-ArchBiochemBiophys-1999 pmid=10375393
#Pott-PlantPhysiol-2004 pmid=15310828
#Zubieta-PlantCell-2003 pmid=12897246
</biblio>

===Terpenes and terpenoids===
*Terpenes are hydrocarbons: combinations of several isoprenes.  (Sometimes encompasses terpenoids.)
*Terpernoids are modified terpenes with methyl groups added/removed or oxygens added
*From Wikipedia: "Terpenoids contribute to the scent of eucalyptus, the flavors of cinnamon, cloves and ginger and the color of yellow flowers. Well-known terpenoids include citral, menthol, camphor and the cannabinoids found in the Cannabis plant."

E. coli has the &Delta;<sup>3</sup>-isopentenyl-pyrophosphate pathway, and the enzymes to produce geranyl-PP.  This pathway is less effective than the mevalonate pathway, but this has been cloned into an E. coli strain by Keastling's group.   Many scented compounds can be made from isopentenyl-PP and geranyl-PP with one or two enzymes, including lemon, orange, pine, etc.  See [http://www.ecocyc.org Ecocyc] for the pathways (type IPP as a compound and look at the synthetic and reactant pathways that link to it).

Terpenes are also the precursor to rubber and many of the resins and gums.

===Indole elimination===
Indole is the precursor to and degradation product of tryptophan.  We could knock out the relevant two enzymes and supply tryptophan exogenously.  Also, we could supply tryptophan exogenously and see if that is sufficient to inhibit indole formation via feedback inhibition in a "normal" strain. [from TK]

[http://biocyc.org/ECOLI/NEW-IMAGE?type=COMPOUND&object=INDOLE Indole] can act as an extracellular signal so indole can probably get in and out of the cell.

====Relevant reactions====
In Pathway Reactions as a Reactant:

[http://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&object=TRPSYN-PWY tryptophan biosynthesis] :
indole + L-serine = L-tryptophan + H2O

In Pathway Reactions as a Product:

[http://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&object=TRPSYN-PWY tryptophan biosynthesis] :
indole-3-glycerol-phosphate = indole + D-glyceraldehyde-3-phosphate

[http://biocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&object=TRYPDEG-PWY tryptophan degradation II (via pyruvate)] :
L-tryptophan + H2O = indole + pyruvate + ammonia

====Relevant enzymes====
trpB (biosynthesis) and tnaA (degradation)

====References====
<biblio>
#Freundlich-JBacteriol-1960 pmid=13701820
#Phillips-ArchBiochemBiophys-1992 pmid=1632641
</biblio>

==Diagnostic bacteria==

I'd like to be able to add a small number of diagnostic bacteria into a larger culture to detect the presence of cells containing engineered devices in the culture.  Presumably there would be BB DNA floating around from lysed bacteria (does it get cut up?).  The diagnostic bacteria would need to responsd to BB DNA by glowing green or smelling minty fresh:)  The response might be mediated by uptake of DNA into the diagnostic bacteria and then use the mixed connective site as a riboregulator or maybe have a membrane protein that binds specific DNA sequences and triggers a two component system.  Very sketchy proposal right now.  Unless we could find easy ways of doing this it would be a protein engineering project.

==Minimal Cellular Power Supply & Chassis==

It would be really great to have a cell with the following properties:
*made from known components.
*works well with any system that's placed inside the cell.

The TK lab has done some foundational work on developing ''Mesoplasma florum'' as a standard cellular power supply and chassis (e.g., sequencing its [http://www.broad.mit.edu/annotation/microbes/mesoplasma_florum/ genome]).  

Still, today, there is not a well-described simple cell that serves as a standard cellular power supply and chassis.  '''Let's get on with it!'''

The goal of this project would be to take the development of ''Mesoplasma florum'' as a chassis to the next level.  Specific parts of the project might include:
*making parts out of all the known Meso genes (e.g., ~600 new parts!)
*develop genome-scale engineering methods (e.g., cell and genome fusion techniques)
*measure the properties of a cell (e.g., transcription and translation load functions)
*design a new organism (e.g., minimal / modelable metabolism, DNA refactoring)

Strengths of this project would include:
*significant advisor interest and expertise 
*some parts of the projects are, as near as possible, guaranteed to work (i.e., turn genes into new parts)
*the project would result in foundational contributions to the field (i.e., not another stupid bacteria trick)

==Random, Environmentally-Sensitive Design Generator==

In 2000, Elowitz and Leibler created a ring oscillator system in bacteria which allowed for the oscillation of green fluorescent protein expression.  I would like to develop many plasmids of this kind with a few modifications.  First, each plasmid’s expression of the ocscillating system would be dependent on the sensing of a particular type of light (specifically, red, yellow, or blue).  In addition, following each of the three repressors constituting the oscillator would be a unique fluorescent color protein coding region (again, red, yellow, or blue).  These plasmids would be randomly distributed onto a field of cells.  Then, once a light is shined on a particular region of the field, environmentally-sensitive, multi-color light patterns will be developed on the field of cells.
Reference:
Elowitz, M. B. & Leibler, S. A synthetic oscillatory network of transcriptional regulators. 
Nature 403, 335–338 (2000).

Strength:
Cool Demo 

Weakness:
Not any real significant scientific value

==Light Amplifier==
In 2004, a team of students from the University of Texas at Austin created a system engineered inside bacteria which produced a black precipitate in the absence of red light.  This system was then used to take pictures of a projected slide with the aid of modern imaging technology.  I would like to explore the possibility of connecting the light sensor component of their system to another genetic circuit.  This genetic circuit would consist of a promoter, which would trigger subsequent red light production when enough polymerases per second (PoPS) are received from the output of the light-sensing device.  Thus, by shining red light on one cell, all of the cells in a bacterial lawn will be lit up red.  One could then place yellow, green, and blue light-sensing and light-producing systems similar to these systems in the same cells, inducing the whole lawn to amplify a light signal directed towards a single cell.
Reference:
Levskaya A, Chevalier AA, Tabor JJ, Simpson ZB, Lavery LA, Levy M, 	Davidson EA, 
Scouras A, Ellington AD, Marcotte EM, Voigt CA. 	Synthetic biology: engineering Escherichia coli to see light. Nature. 2005 Nov 24;438(7067):441-442. 

Strength: 
Could be used as an indirect measurement of PoPS which is integral to the development of synthetic biology
Physics professors may be interested
Could be used in the future as a cheaper light source

Weakness:
May seem too simple or unuseful at first glance

==''Vibrio furnissii''  hydrocarbon production==

Park has discovered a strain of ''Vibrio furnissii'' which produces significant quantities of long chain hydrocarbons from fatty acids <cite>Park01 Park05 Park05a </cite>.  The key pathway is from the aldehyde hexadecanal to the alcohol hexadecanol to the hydrocarbon hexadecane.  Remarkably, this reduction appears to take place in an oxygen environment, making the enzyme especially interesting.


The project would be to enrich for ''V. furnissii'' strains on alkaline peptone water (with 7% NaCl  see [http://www.cfsan.fda.gov/~ebam/bam-9.html here]), plate out on TCBS agar, isolate Vibrio colonies, and select hydrocarbon producers.  We would then shotgun clone genomic DNA into ''E. coli'', add exogenous hexadecanol, and attempt to isolate clones producing hexadecane.  Sequencing selected strains might reveal the important enzyme.

Several diffficulties arise: potential pathogenicity of ''V. furnissii'', likelihood of obtaining a hydrocarbon producing strain, cross reaction between hexadecanol and hexadecane.  The strain NCTC 11218 is alleged to be non-pathogenic, although the evidence for this seems thin.  It is used as a validation test for TCBS  agar in microbiology labs.  See Taylor <cite>Taylor81</cite>.

Wackett at the University of Minnesota has started looking at this with this short proposal:

<blockquote>
Petroleum Fuels in Real-Time from Renewable Resources

 Lead: Lawrence P. Wackett (BMBB)

 Funding Amount: $270,000

 
Project Abstract: Petroleum hydrocarbons took hundreds of millions of years to form naturally, but are being harvested and combusted in centuries. Either our vehicles and industry will change dramatically, or a renewable source of petroleum-like alkanes will need to be developed.
Recently, bacteria have been shown to biosynthesize middle chain-length alkanes in the time frame of enzyme-catalyzed reactions (a typical enzyme second order rate constant is 106 M-1s-1). Middle chain-length alkanes are the “sweet” part of petroleum that is used as liquid fuels. Longer chain-length alkanes are less preferred as fuels and short-chain alkanes are gaseous and thus not amenable for use as transportation fuels.
In traditional petroleum fuels, sulfur-containing ring compounds cause polluting sulfurous oxides during combustion. Bacterially-produced middle chain-length alkanes can provide a fuels source for society that derives from renewable resources and combusts efficiently and with much less pollution. Moreover, alkane-producing metabolism could be tuned to generate different chain-length alkanes to be used for different purposes; for example, C8– C14 alkanes can be used in automobile engines and C14 – C22 alkanes used in diesel engines.
Research is proposed here to better understand and engineer bacterial fermentations that transform common renewable resources into fuel hydrocarbons. A bacterium has been described that is capable of producing C14 – C22 alkanes (diesel fuel) from glucose, xylose, acetate, sugar cane molasses, xylan or chitin. The pathway and enzymes involved in lipid reduction to alkanes is proposed but still requires experimental investigation.
In this project, the researchers will begin to: (1) identify the key enzyme(s) reducing alcohols to alkanes, (2) elucidate the pathway for alkane production, (3) metabolically engineer superior alkane producing strains, (4) isolate superior alkane producing bacteria from nature, and (5) analyze the optimal economic outcomes for biobased production of alkanes from various renewable resources.
This project has a very high probability of generating new intellectual property and providing the basis for external funding from the Department of Energy, the National Science Foundation and other agencies.
</blockquote>

See also Wackett's presentation:  [http://www1.umn.edu/iree/pdfs/wackett.pdf]

<biblio>

#Park01 pmid=11549018

#Park05 pmid=15659187

#Park05a pmid=15687207

#Brenner83 pmid=6630464

#Oro67 pmid=6025301

#Taylor81 pmid=7229102

#Holden02 pmid=11976128 

#Birkeland04 Birkeland N-K, The microbial diversity of deep subsurface oil reservoirs, Chapter 14, Studies in Surface Science and Catalysis  151
Vazuez-Duhalt R and Quintero-Ramirez R (eds.), (2004) Elesevier B. V.

#Valderrama04 Valderrama B, Bacterial hydrocarbon biosynthesis revisited, Chapter 13, Studies in Surface Science and Catalysis  151
Vazuez-Duhalt R and Quintero-Ramirez R (eds.), (2004) Elesevier B. V.


#Haight66 pmid=5924270

#Hickman-Brenner84 pmid=6746884

#OHara03 pmid=14662957

#Stone52 Stone RW and Zobel CE, Bacterial aspects of the origin of petroleum, Industrial and Engineering Chemistry, Vol 44 No 11 pp 2564-7 (1952).

</biblio>

==Protein/Nucleic Acid/BioBrick(??) Intercellular Translocation==

Update: I'll put up a revised version of this idea soon.  It seems that these systems are not as I initially thought.  
I'll leave the references, though (if they are of interest at all).

===References===

<biblio>

#Journet pmid=16092523

#Spreng pmid=10727885

#Majander pmid=15806100

</biblio>

==Cell History==

I figured it wouldn't hurt to throw the idea up.  It is what I told everyone about at the Legal-lunch.  It's a pretty simple idea.  So, I like the idea of cellular memory.  Cells have intrinsic mechanisms for not only using up-to-date information (solute concentrations, cell density, you name it), but also for using past information to define a current or even future state.  The example I gave at lunch was the homing of blood stem cells in the early embryo (as they make their way to their niche).  Another example is telomeres shortening with cellular division.  As in the telomere example, a permanent DNA alteration is made to record the history of the cell.

I propose we somewhat follow this idea of recording the history of the cell.  More specifically, it would be interesting (at least to me) to create a system that can take in inputs, differentiating, at minimum, between the order of those inputs, store that information, and deliver that information at the request of the researcher.  Here's a simpler description of what I am trying to say:

There exist two inputs A and C with corresponding outputs B and D, respectively.  Both B and D would be necessary to elicit some reporter function (mint smell, perhaps?).  D would also be a "permanent" inhibitor of A (permanent meaning D would make some type of DNA alteration to eliminate A rather than simply repressing A or something like that).  Thus, only if B were already produced (i.e. if input A were received before input C) would the reporter be expressed/activated/etc.

A -> B      

C -> D -| A  

B + D -> Reporter

*This system only really works out (and you keep the whole "history-keeping" apect of this idea) if the outputs (B and D) are "permanent".

Advantages:
*This can completely be expanded to more than two inputs.  Of course, this would require greater interdependency within the system, something we don't want.  Furthermore, the complexity you add with each input is quite interesting (not in a good way, though there are some neat patterns that arise; I began working some of this stuff out)
*We can potentially start out with some parts that we already have.
*This network may be relatively easy to construct compared to the other ideas we have thrown around.

Comments:
This falls into the general application category of Information Processing.  TK mentions that a general approach for making biological Finite State Machines (FSMs) would be interesting.  Austin mentions that an old Princeton iGEM project was to implement the "Simon" game in bacteria (they just started work on a design, but never made anything).

===References===

Drew pointed out this paper to me.  They created a genetic toggle switch that can stably interconvert between two states. 

<biblio>
#Collins pmid=10659857
#Arkin pmid=16534780
</biblio>

==Autotrophic e.coli==
As a first step to constructing an autotrophic E.Coli cell, we can get a light-driven proton gradient present in the cells.  This will be accomplished by expressing Proteorhodopsin along with the genes necessary for the biosynthesis of retinal in E.Coli.  I'm not sure the steps that would follow - I guess you'd have to get the "dark" reations of photosynthesis going, which might be tough.  However, getting a light-driven proton gradient working will be easy! in fact it's already been done.

[http://biology.plosjournals.org/perlserv?request=get-document&doi=10.1371/journal.pbio.0030273 This paper] describes co-transforming E.coli, with pACCAR16<delta>crtX and a plasmid containing the ''blh'' gene and the Proteorhodopsin gene each under inducible control.
*pACCAR16<delta>crtX - contains ''crtE,crtB,crtI,''and''crtY'' genes, which are responsible for B-carotene biosynthesis in ''E.herbicola''
**Originally, specified in [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=PubMed&cmd=Retrieve&list_uids=7733978&dopt=Abstract this paper] written by researchers at [http://www.kirin.com/ Kirin], makers of a very fine product.
*Blh protein cleaves B-carotene creating retinal
in particular see [http://biology.plosjournals.org/perlserv?request=slideshow&type=figure&doi=10.1371/journal.pbio.0030273&id=32448 this figure] where they demonstrate inducing both Blh and proteorhodopsin and getting red E.coli (an indicator of proteorhodopsin bound to retinal).  The particular Blh-homolog used in this study was found in a BAC library from the same paper.

This project would involve "domesticating" this system by getting the relevant genes either chromosomally integrated or all on a single BioBrick plasmid.  We would be able to tell its working at a first pass by just seeing that the cells turn red, and we could follow up by doing pH measurements as specified by [[DeLong Lab|Ed Delong's group]].  We could also determine the other steps necessary to make E.coli truly autotrophic and start on those if we have time.  The eventual goal is e.coli that grows on minimal media with no carbon source.  bam!

===Calvin Cycle (the "dark reactions")===
So check [http://peds.oxfordjournals.org/cgi/content/abstract/19/3/113 this out]. In an effort to do directed evolution on RuBisCO, these folks partially reconstructed the Calvin cycle in E.Coli (wt E.coli naturally expresses 8 of 11 of the required calvin cycle enzymes), by adding RuBisCO & phosphoribulokinase from cyanobacteria PCC6301 and PCC7492, respectively.  Anyway, these cells needed CO2 to grow (as well as a pentose C-source), so this looks to be something close to the other piece of the puzzle.  I have to do my e.coli metabolism HWK, but it seems like it might be possible to fill in the other calvin cycle genes, hook up the light-driven proton pump and then grow on minimal media.  

===Organization===
I think this project divides up nicely into the two areas outlined above (assuming those make sense, more hwk reqd). It would also provide 2 intermediate goals - i.e. get proton pump/calvin cycle working, as well as a final goal -- integrate and grow on minimal.  shezam!




===References===
[[Image:PACCAR16.JPG|thumb|left|pACCAR16 plasmid structure]]