=Cell by Date: Design=

__NOTOC__

<div class="tabs-blue">
<ul>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell By Date|Introduction]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/Specifications|Specifications]]</li>
<li id="current">[[IGEM:IMPERIAL/2007/Projects/Cell by date/Design|Design]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/Modelling|Modelling]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/Implementation|Implementation]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/TestingValidation|Testing/Validation]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/Notes|Notes]]</li>
<li>[[IGEM:IMPERIAL/2007/Projects/Cell by date/References|References]]</li>
</ul>
</div>
<br style="clear:both">

__TOC__

===Summary===

Cell by Date has three levels of complexity and we hope to implement these levels in separate chassis: ''E. coli'', ''in vitro'' and ''in veso''.  These chassis are being characterized as one of ICGEMs sub-projects. The table below outlines which levels which hope to implement Cell by Date in.

{| style="background:#FAF5FF; border:1px solid #FFDAB9; color:#000000"
|- style="background:#FFA07A"
|| '''Chassis''' || Phase 1 || Phase 2 || Phase 3 
|- style="background:#FFDAB9"
|style="background:#FFA07A" |  E.coli || X ||  || || 
|- style="background:#FFDAB9"
| style="background:#FFA07A" | In-Vitro|| X || X || X ||
|- style="background:#FFDAB9"
| style="background:#FFA07A" | In-Veso|| X || X || X ||
|}


==Phase 1: Initial Testing==

Phase 1 involves the testing of various simple constructs to confirm that there is gene expression ''in vivo'' and ''in vitro''. DNA constructs are as summarized below:

{|{| style="background:#F5FAFF; border: 1px solid #aabadd; color:Black"
|- style="background:#aabadd; color:Black"
|| '''Type of Promoter''' ||'''Test Constructs''' || '''Notes'''
|- style="background:#d5dcef"  
| style="background:#aabadd" | Constitutive T7 Promoter 
|[[image:ICGEMS_dvc_PT7_gfp.png|left|PT7 promoting GFP]] 
|
*Entire construct is currently available in Biobricks Registry: [http://parts.mit.edu/registry/index.php/Part:BBa_E7104 BBa_E7104] (available), in [http://parts.mit.edu/registry/index.php/Part:pSB1A2 pSB1A2] (AmpR)
*pT7 can be used with T7 RNAP in vitro and in veso, and it gives constitutive expression.

|- style="background:#d5dcef"
| style="background:#aabadd" | Constitutive E.coli Promoter
|[[image:ICGEMS_dvc_Ptet_gfp.png|left|P<sub>tet</sub promoting GFP]] 
|
*Entire construct is currently in Biobricks Registry: [http://parts.mit.edu/registry/index.php/Part:BBa_I13522 BBa_I13522] (available), in [http://parts.mit.edu/registry/index.php/Part:pSB1A2 pSB1A2] (AmpR)
*pTet can be used with E.coli RNAP in vitro and maybe in veso, and it gives constitutive expression.

|- style="background:#d5dcef"
| style="background:#aabadd" | Inducible E.coli Promoter
|[[image:ICGEMS_dvc_PBad_gfp.png|left|PBad promoting GFP]] 
|
*pBad promoter and GFP available: [http://parts.mit.edu/registry/index.php/Part:BBa_J5528 BBa_J5528] (available), in [http://parts.mit.edu/registry/index.php/Part:pSB2K3 pSB2K3] (KanR)
*Need Ara C regulator protein in vitro and in veso. Ara C available with the RBS:[http://parts.mit.edu/registry/index.php/Part:BBa_S03550 BBa_S03550]
*pBad can be used with E.coli RNAP in vitro, and it is inducible by arabinose.
*To be used in veso, arabinose transport proteins have to be inserted in the phospholipid bilayer of the vesicles.

|- style="background:#d5dcef"
| style="background:#aabadd" | Constitutive E.coli Promoter 
|[[image:PcI_construct.PNG|left|PcI promoting GFP]]
|
*Entire construct is currently available in Biobricks Registry:  [http://parts.mit.edu/registry/index.php/Part:BBa_J07037 BBa_J07037] (avaiable), in [http://parts.mit.edu/registry/index.php/Part:pSB2K3 pSB2K3] (KanR)
*pcI gives constitutive expression.

|- style="background:#d5dcef"
| style="background:#aabadd" | Inducible E.coli Promoter 
|[[image:ICGEMS_dvc_PBad2_gfp.PNG‎ |left|pBad promoting GFP and pcI promoting araC]]
|
*All parts are currently available in the Biobricks Registry.
*pcI promoter: [http://parts.mit.edu/registry/index.php/Part:BBa_R0051 BBa_R0051] (available), in [http://parts.mit.edu/registry/index.php/Part:pSB1A2 pSB1A2] (AmpR)
*Arabinose C regulator protein with RBS: [http://parts.mit.edu/registry/index.php/Part:BBa_S03550 BBa_S03550] (available), in [http://parts.mit.edu/registry/index.php/Part:pSB1AC3 pSB1AC3] (A+CR)
*Stop codon: [http://parts.mit.edu/registry/index.php/Part:BBa_B0015 BBa_B0015], in [http://parts.mit.edu/registry/index.php/Part:pSB1AK3 pSB1AK3] (A+KR)
*pBAD with GFP: [http://parts.mit.edu/registry/index.php/Part:BBa_J5528 BBa_J5528], in [http://parts.mit.edu/registry/index.php/Part:pSB2K3 pSB2K3] (KanR)

|}
<br>

We are not interested in a lot of details, but rather just trying to see the most suitable promoter. The promoter construct, together with the inducer required (if the promoter is inducible) will be mixed together and the levels of fluorescence will be monitored over time in different chassis. <br>
*3 repeats to be done for each test
*Positive control: Purified GFP in E.coli, and then in the cell extract
*Negative control: Cell extract without any DNA insert or GFP

The best promoter sequence is defined as such:<br>
*Fastest expression of GFP 
*Highest detection of fluorescence

We preferably want to use a constitutive promoter for Cell by Date. If all constitutive promoters are found working then we will need to chose one of them. This decision can be made based on their levels of activity (the higher, the better) and their operating temperature range. This needs to be as close as possible to the CBD operating range of 4&deg;C - 37&deg;C <br>

==Phase 2: Characterizing specific Construct==
Once the best promoter has been chosen, we would then proceed to characterize it in the ''in vitro'' systems. Several variables will be tested for, including:<br>
*Test for operating temperature range: 4°C, 15°C, 25°C, 30°C, 37°C, 50°C.
*Life span of the system

Fluorescence level readings will be taken every 15 min intervals, with triplicates for each sample at a particular temperature. Determining the rate of fluorescence at each temperature would also provide preliminary data for modelling analysis. 

The system is then subjected to temperature changes and the effect on fluorescence levels measured:<br>
*Subject system to temperature changes and measure effect on fluorescence levels
*Type of gradients: Steep/ Gentle/ Pulse
*Temperature increment: from 4°C to 37°C
*Apply the temperature increment only after the GFP levels have reached steady state

===Problems===
*The commercial cell extracts come in 50µl, while the wells in the fluorometer plates can hold up to 200µl.
*Water evaporates when fluorescent readings are taken.

==Phase 3: Testing/Validation of Modelling Analysis==
Ideally, phase 2 would provide the results necessary for constructing our model for the system. The model should also be able to accurately predict the outcome of our temperature effects and verified through experimental testing. We would then obtain the thermal exposure device that achieves the specifications that we have defined for the project. 

Phase 2 and 3 are to be repeated using DsRed-Express as the reporter. This is subjected to the arrival of its gene sequence, and its preparation and performance overall as a better reporter (due to its stability and quicker detection time). 

===Considerations for DsRed-Express===
*When Dsred Express gene sequence arrives, clone and express it.
*Draw up a calibration curve of DsRed-Express.
*Positive control: DsRed-Express in cell extract
*Negative control: Cell extract only