{{IGEM:/Harvard/2006/Cyanobacteria}}

<div class="tabs">
<ul>
<li>[[IGEM:Harvard/2006/Cyanobacteria/Notebook/2006-7-14 | Previous Entry]]</li>
<li>[[IGEM:Harvard/2006/Cyanobacteria/Notebook/2006-7-16 | Current Entry]]</li>
<li>[[IGEM:Harvard/2006/Cyanobacteria/Notebook/2006-7-17 | Next Entry]]</li>
</ul>
</div>
<br style="clear:both">
<div class="tabcontent">


==Result of Friday Experiment==
:*Ran two gels of the PCR products from Friday
:**1% Agarose Gel: for larger segments
:**2% Agarose Gel: for smaller segments
:*The gels were hot to the touch and one of the gels crumbled (melted). 
:*The other gel displayed nothing (not even the ladder)
:*Ran two gels again, repeat of the first gel electrophoresis
:**1% Agarose Gel: for larger segments
:**1.5% Agarose Gel: for smaller segments

Result: FAILURE! Contamination has hit our shores...

[[Image: cyano_85_71606.jpg|thumb|fail1, 1%
1: 1kb+<br>
2: 5-8 HH<br>
3: 5-8 DR<br>
4: 5-8 JL<br>
5: 5-8 ZS<br>
6: 5-8 neg<br>
]]
[[Image: cyano_other_71606.jpg|thumb|fail2, 1.5%
1: 1kb+<br>
2: 4-6 HH<br>
3: 4-6 DR<br>
4: 4-6 JL<br>
5: 4-6 ZS<br>
6: 4-6 neg<br>
7: 10-7 HH<br>
8: 10-7 DR<br>
9: 10-7 JL<br>
10: 10-7 ZS<br>
11: 10-7 neg<br>
12: 5-9 HH<br>
13: 5-9 DR<br>
14: 5-9 JL<br>
15: 5-9 ZS<br>
16: 5-9 neg<br>
]]

==Housekeeping (incubator status)==
:*'''Removed'''
:**All flasks from upper shelf
:***Bleach was added to the flasks and poured down the drain
:**PCC 7942 flask w/o thiosulfate
:*'''Added'''
:**Inoculated a new flask (100 mL BG-11 w/thiosulfate) with a colony from PCC 7942 streak #1 plate
:**Inoculated a new flask (100 mL BG-11 w/thiosulfate) with 2 mL of liquid culture from the PCC 7942 flask w/thiosulfate (from 2006-7-7)
:**Control (100 mL BG-11 w/thiosulfate)

==New Experiment: diagnosis + new reagents==

To test:
*Is the water used bad?
**Going to toss old water and use nuclease free water
*Is the Vent contaminated?
**Doing a positive and negative control with the Vent + new water
*Is the Vent buffer contaminated?
**No way to test seperate of Vent due to lack of more reagents
*Does HotStar work?
**Hotstar old = #1, hotstar new = #2
*Are primers designed incorrectly?
**On monday peng will take a look at it
*Are primers contaminated?
**Made new batch
*Is the template actually what we think it is?
**Sequence! we can amplify off the template when we get the diagnosis done

===Diagnosis===
*#BB positive control (R0010+E0241) w/ new water and Vent
*#BB positive control w/ new water and old Hotstar
*#BB positive control w/ new water and new Hotstar
*#negative control w/ new water and Vent
*#negative control w/ new water and old Hotstar
*#negative control w/ new water and new Hotstar
*#negative control w/ old water and new Hotstar

====Diagnosis protocol====
{| {{table}}
| align="center" style="background:#f0f0f0;"|'''PCR mutagenesis'''
|-
| 0.5 uL template R0010+E0241
|-
| 5 μL 10x buffer
|-
| 1 μL 10 mM dNTP 
|-
| 1 μL Primer 1 
|-
| 1 μL Primer 2
|-
| 0.5 μL Taq
|-
| 41 μL dH<sub>2</sub>0 
|-
| 
| 
|}

Run time:
 *#95@15'
 *#94@30"
 *#56@30"
 *#72@3'30"
 *#Cycle to step 2, 40x
 *#72@5'
 *#4@forever



===Experimental===
*Same as previously done experimental reactions with new Hotstar, new primers, new water, and 250x diluted template instead of 500x. There are 8 reactions total: one experimental and one control for each pair of primers.

====Experimental protocol====
{| {{table}}
| align="center" style="background:#f0f0f0;"|'''PCR mutagenesis'''
|-
| 1 uL ~12 ng/uL template
|-
| 5 μL 10x buffer
|-
| 1 μL 10 mM dNTP 
|-
| 1 μL Primer 1 
|-
| 1 μL Primer 2
|-
| 0.5 μL Taq
|-
| 40.5 μL dH<sub>2</sub>0 
|-
| 
| 
|}

Sample 8-5:
 *#95@15'
 *#94@30"
 *#56@30"
 *#72@2'30"
 *#Cycle to step 2, 40x
 *#72@2'30""
 *#4@forever

Other 3:
 *#95@15'
 *#94@30"
 *#56@30"
 *#72@1'30"
 *#Cycle to step 2, 40x
 *#72@2'30""
 *#4@forever

Mixing procedure:

# To each tube, add:
#* 5 µL H20
#* 1 µL primer 1
#* 1 µL primer 2
# Make the following master mix:
#* 8 µL H20
#* 2 µL template DNA
# Add 1 µL of the above master to each experimental tube, and add 1 µL of H20 to the control (there should be leftover master mix).
# Make the following master mix:
#* 55 µL buffer
#* 11 µL dNTP
#* 390.5 µL H20
#* 5.5 µL Taq (new Hotstar)
# Add 42 µL of the above master to each tube

==Experiment design (for tomorrow's presentation)==
===Experiment 1===
; Goal
: Verify that our ''KaiA'' and ''KaiBC'' BB's can be expressed in ''E. coli''.

; Procedure : 
:# Tag KaiA/BC (e.g. with a fluorescent subunit).
:# Insert the tagged KaiA/BC into ''E. coli''.
:# Induce promoters for both KaiA and KaiBC
:# Measure the ''E. coli'' fluorescence.

; Interpretation of results
: A positive result (observation of the tagged proteins) means that our KaiA/BC genes are being expressed and our promoters are working.
: A negative result means that there's a problem with our KaiA/BC genes or our promoters. We will sequence our KaiA/BC genes to test against an error in the code, but there may be a lingering codon bias problem or some other unforeseen incompatibility with ''E. coli.''

===Experiment 2===
; Goal: Verify that KaiA, KaiB, and KaiC are interacting to phosphorylate KaiC in ''E.coli''.
; Procedure: 
:# Insert KaiA and KaiBC into ''E. coli''.
:# Induce KaiBC's promoter and measure the amount of phosphorylated KaiC via Western blot.
:# Induce KaiA's promoter and measure the amount of phosphorylated KaiC via Western blot.

Ideally, we would see an increase in the amount of phosphorylated KaiC between steps 2 and 3.  This would indicate that the Kai proteins are interacting as they do ''in vivo''.  However, some care needs to be taken in deciding when exactly to measure the phosphorylated KaiC, to determine when/if KaiC is phosphorylated in the absence of KaiA.  In that case, we may not see as much of a difference between steps 2 and 3.

===Experiment 3===
; Goal 
: To measure the oscillation of the KaiC phosphorylation cycle given pulsed KaiA and KaiBC production.
; Procedure :
:# Insert KaiA and KaiBC into ''E. coli''.
:# Induce the promoters for KaiA and KaiBC at the same time
:# After a short time, turn off the promoters.
:# Measure the amount of phosphorylated KaiC at regular intervals (e.g. every hour) via Western blot.

We hope to initially see strong oscillation, with the amplitude decreasing over time.  Since no KaiA/B/C should be produced after the initial spike, we expect that the intracellular concentrations of the Kai proteins would decrease as the E. coli divide.

===Experiment 4===
; Goal
: To measure the oscillation of the KaiC phosphorylation cycle given constant KaiA and pulsed KaiBC production.
; Procedure : 
:# Insert KaiA and KaiBC into ''E. coli''.
:# Induce the promoters for KaiA and KaiBC
:# After a short time, turn off the KaiBC promoter, leaving the KaiA promoter induced.
:# Measure the amount of phosphorylated KaiC at regular intervals

We expect that, since KaiA is produced at a constant (non-oscillating) rate in cyanobacteria, its constant production in ''E. coli'' should not interfere with KaiC's phosphorylation cycle. The constant production of KaiA may even strengthen the oscillation, since KaiA won't degrade or dilute away.

===Experiment 5===
; Goal
: To measure the oscillation of the KaiC phosphorylation cycle, given constant KaiA and KaiBC production.
; Procedure :
:# Insert KaiA and KaiBC into ''E. coli''.
:# Induce the promoters for KaiA and KaiBC 
:# Measure the amount of phosphorylated KaiC at regular intervals

We do not know how constant KaiBC transcription will interfere with the oscillation of KaiC (in cyanobacteria, KaiBC transcription oscillates on a circadian rhythm).  This experiment should help us figure it out.

<br>
[[Image:Cyano e coli experimental design.JPG|300px]]