==Purpose of this Page==
Tom is away revising for exams and John is going away to the US.  This page is a place where questions are raised and answered and a place to keep everyone aware of what is going on. Also, if anyone external to the project wishes to raise a point, contribute to a discussion or simply to ask a question please feel free!

==A message to Tom and John S.==
'''[[User:Cys|Christin]] 20:08, 6 September 2006 (EDT)''': Since labwork has become quite heavy and we have been feeling the limit in manpower since you two left - could you help us out doing some data analysis, revising of protocols & extracting parameters etc. 
After Friday, we will be down to 4 people and then soon next week there will only be 3 people be available to work ... so every help that can be done via the wiki or similar is urgently needed! 
Here are some of the most important things for now: 
*Revising & checking protocols: 
**J37015 has been revised - can it be checked and amended if necessary - it is being tested in the lab now
**J37022 protocol needs to be checked & revised - hopefully there will be a construct ready on Thursday to be tested
*Analysis of data obtained so far: 
**For now, J37016 has given quite good results - and we have a good set of data that can be used & has to be analysed more thoroughly now. Data is up on the [http://openwetware.org/wiki/IGEM:IMPERIAL/Results general results page] or on the [http://openwetware.org/wiki/IGEM:IMPERIAL/2006/Results/T9002 T9002 experimental data page]
**Lots of data to come the next days (hopefully)
If you are unsure about anything & where the project is proceeding (from just looking at the wiki) - let us know immediately. 
Also, if you find any flaw / mistake - give us a notice asap so we can still have a chance to fix any errors.

*'''[[User:Johnsy|Johnsy]] 21:03, 6 September 2006 (EDT)''': I'll try to have a look at the data for J37016 and J37020 within the next few days.  I have posted a note on the [http://openwetware.org/wiki/IGEM:IMPERIAL/Results general results page].

*'''[[User:Johnsy|Johnsy]] 21:23, 6 September 2006 (EDT)''': I have altered the first three sections of the J37022 protocol.  Have we gotten the ELISA plates and the biosensor working yet?  Have we gotten any decent results for the J37015 prey cell yet and what is happening to the CreLox part and the Riboswitch construct?  Are they working as well?  Keep me updated... Also, if anyone has any questions for Dr. Endy or anyone here at MIT, I might be able to meet with him to get an answer.

*'''[[User:JohnChattaway|JohnChattaway]] 04:41, 7 September 2006 (EDT)''' We are no longer using elisas, we are using a type of western blot which uses the imunotags.

*'''[[User:Johnsy|Johnsy]] 09:09, 7 September 2006 (EDT)''': In that case, can you please update the J37022 protocol page to reflect that.  Thanks!  (Or is there another protocol page that has been made for J37022?)

*'''[[User:Farah|Farah]] 12:48, 7 September 2006 (EDT)'''': Hi John. I'm currently updating that protocol now. The biosensor has (sadly) become a low priority in our 'to do' list. The CreLox part is a really difficult PCR and hasn't shown any promise so far (Johnny has tried to make it work many many times) so we've left it for Kirsten to have a go at if she has any time. We've more or less given up on the riboswitch. As an alternative method of controlling our positive feedback loop, we've decided to just add acylase to the medium to get rid of any AHL. We're currently testing how our acylase affects growth of the E.coli. If it works then it's probably the simplest way to control the feedback loop. As for the J37022, I am reading over your protocol for testing the half-life of aiia and I am unsure about the second half. From what I understand from your comments, both parts of the protocol seem to be testing the same thing but the second half requires a three hour wait, the reason for which I don't understand. Can you clarify please? 

*[[User:Johnsy|Johnsy]] 15:21, 7 September 2006 (EDT): Yes, indeed the two parts of the protocol measure the exact same thing.  What we were trying to do is to get the half-life from two distinct experimental methods and to see if they correlate in any way.  This could prove to be useful in the future, and it doesn't take any more time, as you still have to wait for AiiA expression to reach a high enough level for it to be measured.  However, if you don't feel comfortable with using that part of the protocol, it can just as easily be left out.  The only thing I'm worried about is the western blot technique.  Will this give us a quantitative value for the half-life of AiiA?  I'm asking since I'm not too familiar with the protocol for western blot.

*'''[[User:Farah|Farah]] 13:56, 7 September 2006 (EDT)''': This is for <font color=magenta>'''John & Tom'''</font>. We may discard the idea of testing for half-life of aiia anymore since it's stable and so we can say the half-life is equal to cell-division. Also, since we are using the chemostat, we are able to control cell growth and so we can tweak with the half-life of the enzyme. What we really need to know are the k<sub>2</sub> and K<sub>m</sub> of aiia...can you guys do some quick digging and find these values?

*'''[[User:TomH|Tom]] 14:10, 7 September 2006 (EDT)''': Check the realistic values page. It should be there.

*[[User:Johnsy|Johnsy]] 15:31, 7 September 2006 (EDT): I have to disagree with the idea of discarding measuring the half-life.  We still need to be able to compare the half-life with the average cycle period we expect.  If the half-life is longer than the cycle period we expect, we won't be producing any oscillations.  Moreover, the AiiA is using a degradation tag which will speed up the degradation of the AiiA protein.  I'm unsure of how this will affect the half-life of the AiiA.  Also, if we assume that the half-life is equal to the time needed for cell division, then we are assuming that the concentration halves every 20 minutes, which is untrue if we are considering the total population wide concentration.  In fact, the concentration will tend to double every 20 minutes as the cells begin to metabolise and make more AiiA.  BUT...this is all dependent upon how good the chemostat is at maintaining cell population.  If it is very efficient, then I guess you could assume that the AiiA would halve every 20 minutes (assuming that the rate at which AiiA is removed outweighs the actual rate that AiiA is degraded...which I'm not sure if this is a good assumption to make until we get actual values of the half-life of the AiiA we are using.

*'''[[User:Farah|Farah]] 07:43, 14 September 2006 (EDT):'''From what I know of the chemostat, it is quite efficient and so we should be able to make the assumption that the half-life of AiiA is equal to the division time. The problem is, it's the eleventh week and we still don't have the AiiA test construct to be able to determine the half-life. It turns out that the terminator we were using for the construct and the final predator actually isn't there (according to Davidson) so we're trying the ligation with a different terminators from the registry this time. Dr. Mann offered to continue ligations of the test constructs and the final predator if we're available for testing during term time.

*[[User:Johnsy|Johnsy]] 22:46, 14 September 2006 (EDT): My class has a matlab tutorial on how to use it to calculate the least square curve fit, so I will get back to you next monday on the values hopefully...


*'''[[User:JohnChattaway|JohnChattaway]] 16:39, 23 September 2006 (EDT)''' It is true that the halflife will equil the division time but the division time is dependent on the flow rate of the chemostat so it will vary between 1-2 hours aprox.

==Major Issues (Everyone please read and contribute)==

===Cre Lox===
[[IGEM:IMPERIAL/2006/project/parts/creloxprotocal|look at the protocol]]

===Non-functional AiiA <font color=red>(Unresolved)</font>===

*'''[[User:JohnChattaway|John C]]''': [[IGEM:Imperial/Results/S01656|Data showing aiiA does not work]]'''
*'''[[User:TomH|Tom]] 13:55, 26 August 2006 (EDT)''': I'd be a little hesitant to condem AiiA with that data alone, for the following reasons:
#The control of having non-induced cells I don't think is adequate. Having non-induced cells as a control we are assuming the operator sequence is functioning and intact and we're also making the assumption there is enough native LacI repressor to effectively shut off AiiA production. We need a control where we have AHL innoculated with non-AiiA/LuxI/Flourescence producing cells.
#Secondly, the method for working out whether the difference between induced and non-induced cells is statistically significant is not given. Just because a difference is between 3-4% doesn't make it statistically insignificant by default. Especially if you're consistently getting those results. I'd suggest a T test on repeated samples to determine significance.
#Thirdly, the dilution of cells and small AHL innoculation time may be having an effect. I'd suggest growing a culture overnight in the prescence of AHL and IPTG, along with appropriate controls, and taking the supernatent directly from that for the T9002 assay.
*'''[[User:Johnsy|Johnsy]] 06:49, 27 August 2006 (EDT)''':  I agree with Tom in that the AiiA may work.  There are several reasons for non-expression (or maybe non-detection?) I have not tried to increased the IPTG amount, so maybe there is not enough IPTG to induce expression.  Moreover, as our T9002 has been acting up within the past few days, it might be our T9002 cell activity which is the problem.  However, having run an SDS gel with no results might signify that there is not expression of AiiA in the first place (or that there is not enough expression to produce any significant results).  We might want to try to increase the IPTG concentration so that we are sure to induce AiiA expression and retry the SDS gel?  just a suggestion...Then again, it could be the problem with the part itself in the registry.  We can wait for another few days until our J37022 test construct finishes and try to run the tests using that construct.
*'''[[User:TomH|Tom]] 19:14, 27 August 2006 (EDT)''': I reckon first thing to do would be to induce overnight in the prescence of AHL. You can then do the T9002 assay straight from the o/n culture, and use some of the cells for the SDS page.

*'''[[User:JohnChattaway|JohnChattaway]] 07:53, 28 August 2006 (EDT)''' I think you may be right, the western blot expriment is flawed because DH5a can't express protein at high levels, to do that you need to use a different strain so the western blot may have failed for that reason. The new strain kirstin wants to use is in the next safety catagory above DH5a but that just means it has more forms to be fille in. '''Secondly''' I think that seeing as the activity of aiia is so low an hour may not be a long enough time period to get mesurable results I will try an assay which runs overnight and look into small chemostats to run it in.

*'''[[User:TomH|Tom]] 10:00, 28 August 2006 (EDT)''': I don't think a chemostat would be necessary. Just whack in some IPTG and AHL in a standard overnight culture. Just to act as a diagnostic as to whether AiiA is being produced and is working. You shouldn't need a chemostat until assembley of the final system is complete.

*'''[[User:JohnChattaway|JohnChattaway]] 04:48, 29 August 2006 (EDT)''' <font color = red> '''The reason no aiia is being produced is that we were testing a contaminant.'''</font> The part should grow in kanomycin media only but we were testing in ampacillin so it was an amp-resistant contaminant growing, not the real part. '''will retest in kan medium'''

*'''[[User:TomH|Tom]] 06:27, 29 August 2006 (EDT)''': The AHL assay test data should be okay though, as I'm pretty sure John S tested in Kanamycin (I set up a culture with Deepti, defiantley used Kan). And as we were only doing the western blot because we didn't think that was working, doing the overnight innoculation/induction is still probably the best move.

*'''[[User:Johnsy|Johnsy]] 18:00, 29 August 2006 (EDT)''': Yes indeed I have been using Kanamycin for S01656 testing.  Furthermore, I believe that we grew up the S01656 culture for SDS gel in Kanamycin, so there should still be some expression of AiiA.  However, the culture might be contaminated already, since when we were growing up the cultures, we accidentally grew up in Amp, but we did have substantial bacterial growth :S...not really a good sign...and might be a contaminant of something else...but definitely not good.  I would susupect that maybe the AiiA genome might have been mutated or as John C suspects, that the AiiA isn't being produced enough in the medium, so we are not getting enough expression to be measured.

*'''[[User:TomH|Tom]] 13:54, 31 August 2006 (EDT)''': What's going on with this? I saw in the lab-notebook that S01656 was meant to have been tested today but no mention of any results? Plus, presuming the S01656 testing hasn't been done, we still don't know if DH5a are producing AiiA yet (as an overnight inoculation ''still'' hasn't been done), so why are we transforiming BL21 cells under the pretense that DH5a isn't producing protein?

*'''[[User:TomH|Tom]] 13:59, 4 September 2006 (EDT)''': Has the overnight been done yet? Sorry to go on about it because from what's being reported in the lab book, it appears you've all but given up on the AiiA with, what I can see, no extra data then what you had a week ago.

*'''[[User:Cys|Christin]] 04:17, 5 September 2006 (EDT)''': No, S01656 has not been given up yet. It could not be tested yesterday because we did not have any T9002 culture that worked. This is another problem - T9002 does not give us any good results.... we think this might be due to a mutation in the cells (before they were frozen) because it seems that none of the frozen cells seem to give good results. T9002 has now been electroporated anew straight from the registry such that it can be tested tomorrow and hopefully, we will get better results. S01656 is going to be tested again today - I think there might have been a problem as well with the T9002 assay at that time (if you compare the data obtained for one day), so that the problem might not even be the S01656. As we know the J37016 is working, I will try to use that one to check for AHL degradation. This time, an overnight inoculation was done as well. There will be some data coming tonight ....

*[[User:Johnsy|Johnsy]] 15:18, 5 September 2006 (EDT): Are we still updating the protocols continuously?  I haven't seen any updates on them since last week :S...

*<big> '''The final constructs were sequenced and there was no aiiA It looks like it was the registary's fualt'''</big> '''[[User:JohnChattaway|JohnChattaway]] 06:53, 13 September 2006 (EDT)'''

==General Issues==

===Finalised Predator Construct===

*J37018 or J37019???
*Monocistronic finalised prey cell or Poly cistronic prey cell?? (2 promotors and single promotor respectively).
*We have to decide soon guys!!! If all goes well, we may be making this baby by the end of next week!!

*'''[[User:TomH|Tom]] 14:03, 4 September 2006 (EDT)''': My vote is for using the single promoter. Although that's just off preference rather than any data. Has anyone been analysing the results from the testing? PS. I thought one of the predators was J37020?

*'''[[User:Cys|Christin]] 04:35, 5 September 2006 (EDT)''': I am in favour of the single promoter too. My reason rather is that J37020 does NOT give any good results from the testing whereas J37016 (polycistronic predator construct) gives very good results. The actual polycistronic predator has been finished ligating and can be checked tomorrow...

===Auxostat===

anyone thought of using an auxostat rather than a chemostat? (just to make it even more complicated)
*[[User:Johnsy|Johnsy]] 17:52, 4 September 2006 (EDT): What is an auxostat?..pls. clarify...

===HPLC===

*'''[[User:TomH|Tom]] 18:55, 26 August 2006 (EDT)''': You'll all be pleased to know it is possible to assess AHL levels via HPLC. They did it [http://www.pubmedcentral.gov/picrender.fcgi?artid=38458&blobtype=pdf here.] Dunno whether you can incorporate this into testing, but I defiantley think it's a more accurate way to assess AHL levels.
*'''[[User:Johnsy|Johnsy]] 07:11, 27 August 2006 (EDT)''': Again, the problem with using HPLC is that we will not have a direct measurement of the AHL levels in the actual oscillator.  This will be okay for our testing phase, and actually might be helpful if we begin by testing the concentrations of the AHL in the stock AHL, so we are sure that we are using correct AHL concentrations.  Degradation of the AHL (or incorrect concentrations) might account for some of the bloopers we have had in our T9002/J37016/J37020 testing.
*'''[[User:TomH|Tom]] 09:16, 27 August 2006 (EDT)''': There is facility in the bioreactor suite to take samples from the chemostat and run them through a HPLC within the room. There is no reason we can't use this to test AHL levels in the oscilator, although I think the GFP flourescence would be a nicer reporter. It could, however, prove useful if things don't go according to plan and we have to troubleshoot. And if you think the AHL is messed up, it might be an idea to make up a new stock when Vincent gets back.
*'''[[User:Farah|Farah]] 18:27, 28 August 2006 (EDT)''': John, you've landed!! And you're already typing on the wiki! Just have to say - thanks for leaving your lab book...As for the HPLC - I think we should go ahead and do it so that we can check our T9002 transfer curve but to save manpower and time I'd rather give it to the ABC to sort out than do it for ourselves.
*'''[[User:Johnsy|Johnsy]] 18:02, 29 August 2006 (EDT)''': Will the other stuff in the LB medium and the bacteria cell affect the HPLC readings, or do we have to purify AHL somehow?
*'''[[User:TomH|Tom]] 14:06, 4 September 2006 (EDT)''': Unsure on this, all depends on the resolution of HPLC really. Which I don't know. Can anyone find this out? Plus if purification was needed you might be able to run some kind of hydrophobic precipitation, but again, unsure on the details.

===Protein Half-Lives===

*'''[[User:TomH|Tom]] 12:04, 24 August 2006 (EDT)''': Hi guys, something to note which I don't think has been that clear. In a growing cell culture cells grow and divide, increasing the volume of cells. If the production of a stable protein was shut off then the effective half-life of that protein would be the division time of the cells. IE, as the cell becomes two, the protein is shared between a volume twice as large as the original. Thus halving concentration. This needs to be taken into account when assessing protein half-lives.
**Although note, this does not effect AHL half-life as its volume is not constrained by the cell, but rather the fixed volume of the medium.

*'''[[User:Cys|Christin]] 12:51, 24 August 2006 (EDT)''': After cell division, the protein concentration would half - that means that that protein concentration is not in steady state any more but after a certain time, the steady state concentration would be reestablished again. How can this be taken into account? <br>Also, this time delay might have effects on the dynamics of the system.

*'''[[User:TomH|Tom]] 10:16, 25 August 2006 (EDT)''': You would still get establishment of steady states when protein is being produced. The only thing cell division is effecting is protein half-lives when we consider the volume of cells in the chemostat is always the same. IE, the washing out rate is the same as the growth rate. And thinking about it, in the chemostat AHL half-life would also be equivalent to the doubling time of Bacteria, IE ~30 minutes, as this is being washed out at the same rate. PS: Is the full model on the wiki yet?

*'''[[User:Johnsy|Johnsy]] 07:18, 27 August 2006 (EDT)''': Something else to note regarding the cell division: Once the cell divides, yes the concentration of AiiA is halved WITHIN each cell, but eventually, the AiiA concentration will again reach a steady state and return to the level before, which means that in time, there would be an actual increase in the amount of AiiA within the entire system.  However, if we consider the prey cells, they too are dividing.  Thus, also increasing the concentration of AHL (doubling it every doubling time).  Intuitively, this would suggest that our amplitude would double every time the cells divide (which is not necessarily a bad thing?), but more modelling needs to be done on this to confirm.

*'''[[User:TomH|Tom]] 11:23, 27 August 2006 (EDT)''': The changing of amplitudes won't happen within the chemostat, as the number of cells remains constant. And the actual division of cells themselves doesn't effect protein levels in the way that there is a sudden drop and increase back to steady state. The act of division itself does not effect concentrations as the resulting daughter cells will still have the combined volume of the single parental cell. What is effecting concentrations is growth, which is continuous.

*'''[[User:Cys|Christin]] 19:11, 29 August 2006 (EDT)''': Wait a second ... the ''concentration'' within the cell as it divides does not change at all. As concentration is number of moles ''per volume'', the concentration stays the same. However, are you saying that the growth of the cell ''before'' it divides will affect the concentrations? Yes, it does, but is there such a large change in cell size that it will have a significant effect on the concentrations?

*'''[[User:JohnChattaway|JohnChattaway]] 09:26, 1 September 2006 (EDT)''' I hate to point out the obvious but the cells grow by increacing their size so this can be ignored by keeping the cells OD constant in a chemostat.

*'''[[User:JohnChattaway|JohnChattaway]] 04:52, 7 September 2006 (EDT)''' The halflife of AHL in the chemostat will be affected by washout in exactly the same way as any intracellular protein. This is because media + cells is removed so as far as the chemostat is concerened there is no diference between the two areas of the media. Secondly the argument that a constant OD will cause any intracellular protein to remain constant while an extracellular protein can be washed out is flawed as there is a constant amount of empty media inside the chemostat! so they are both constant.

===[[IGEM:IMPERIAL/2006/ProjectCalendar/2006-8-21|Monitoring Cell Populations]]===

*'''[[User:TomH|Tom]] 07:35, 21 August 2006 (EDT)''': Hi everyone, just been reading through [[IGEM:IMPERIAL/2006/ProjectCalendar/2006-8-21|this]]. Big drawback of the 3 antibiotic way of monitoring cell population that you've identified is the extra ligation steps. What extra ligations steps are involved? If you just mean grafting your parts into a plasmid with appropriate antibiotic markers, can't you just co-transform a seperate plasmid with the marker on, thus saving a ligation step? I'm pretty sure those kind of plasmids would be easy to get hold of as they are the basis of a lot of gene cloning.
**'''[[User:TomH|Tom]] 10:03, 21 August 2006 (EDT)''': Actually, on second thoughts, that wouldn't work, as you'll end up losing the plasmid containing the antibiotic you're not screening for while the cells are growing
*'''[[User:Johnsy|Johnsy]] 11:30, 21 August 2006 (EDT)''': From what I understand from the team, we will use the antibiotic way of monitoring as a backup plan, in case we decide to monitor cell populations.  For now, we have not decided to monitor cell populations, but we will go ahead and electroporate & Maxi the parts we might need.  Just in case...
*'''[[User:da104|Deepti]] 15.24, 22 August 2006 (EDT)''' -Talking to Dr. Jensen yesterday morning, it seems that the 3 antibiotic method is not very accurate an requires extra ligations steps which can be avoided.
**Waiting for plated samples to grow will require atleast a day before we can assess the differences in population sizes.
**The number of bacteria in the plated samples does not necessarily accurately reflect the relative proportion in the common broth culture.
**Even if the plating method were to be an accurate measure of the colony sizes in the broth culture, we could still only record, and not control population sizes
*According to Dr. Jensen, the best way to do it would be the OD method. 
**By growing up the bacteria separately, we can measure their separate growth rates which, hopefully, should not be too disimilar. 
**When mixing the  two cultures, the ODs should be equal, and preferably, between 0.6 and 0.8.
*'''[[User:TomH|Tom]] 13:05, 22 August 2006 (EDT)''': Would it be possible to check the population ratios after they've been mixed together using the OD method?
*'''[[User:da104|Deepti]] 09.10, 23 August 2006''' - I'm afraid not. Once the cultures are mixed there is no way to tell them apart using the OD method. Best we can do, I reckon, is ensure the OD of both individual cultures is between (0.6-0.8) right before mixing and then just hope they stay more or less equal (we will have a rough estimate of the individual growth rates of each population from the OD measurement of the individual cultures done beforehand).
*'''[[User:TomH|Tom]] 05:27, 23 August 2006 (EDT)''': If we're gonna be using the chemostat for the final assembley (which I assume we are), Dr. Leak made it clear that it amplifies the smallest difference in compeitive growth ability. Meaning that once innoculated, there's a good chance the ratios are gonna change. I '''strongly''' recommend looking more into this, and looking into obtaining LacZ plasmids.
*'''[[User:JohnChattaway|JohnChattaway]] 09:24, 1 September 2006 (EDT)''' Dr david leak made it clear to me that this was the least of our problems

===Measuring LuxI===

*'''[[User:Farah|Farah]] 12:22, 21 August 2006 (EDT)''': What are we doing about luxI in J37015? Are we going ahead with measuring the GFP levels in that part? Tom mentioned talking to Vincent about it...I can model the luxI but from the assumptions we've made for our current model for J37015 we ought to get a similar curve as for the AHL levels. To Tom: I remember you saying you've talked to Vincent about the luxI - what was the conclusion?
*'''[[User:TomH|Tom]] 15:50, 21 August 2006 (EDT)''': Yeah, we decided it might be easier and more accurate to measure the GFP from the pLux transcript as opposed to AHL. We'd still like to measure AHL to see the relationship between LuxI and AHL, however the relationship may be easier to characterise and more accurate than the T9002 assay.
*'''[[User:da104|Deepti]] 23 August 2006''': What would be the point of measuring LuxI if we cannot characterise the relationship between LuxI concentration and AHL concentration? The only way we could do that is to measure both compounds (using both protocols) and come up with a curve that describes their relationship. Correct me if I am wrong, but as far as I can gather, the reason for using this alternative protocol is to achieve greater accuracy than with the AHL assay.  Using the alternative protocol we would gain a more accurate measurememnt of LuxI concentration but the AHL measurement would still depend on the original assay whether we use the curve produced by the alternative or not!! (Since the curve is extracted by comparing data from both assays).
Should we not stick with the original assay then, inaccurate as this may be? Unless ofcourse we do want to measure the concentration of LuxI in the system, for which I'm not sure I see the reason for.

===LuxR in the Predator Cell===

*'''[[User:TomH|Tom]] 15:50, 21 August 2006 (EDT)''': I remember when Farah and Christin assumed LuxR in the predator was constant we still got oscillations. If this is the case what is the point of LuxR being under the control of pLux as opposed to a constitutive promoter? Also, if this is the case and LuxR variance isn't important, than our design is exactly the same as the preivious MIT design, except spread over two cells. Maybe someone can look at the effects on the system of having LuxR vairable and LuxR constant?

*'''[[User:Farah|Farah]] 11:29, 22 August 2006 (EDT):''' When Christin and I assumed LuxR to be constant we were just 'hiding' the fact that LuxR '''is''' actually varying - as was pointed out by Vincent. I don't see the poing in modelling a simplified version of the system when our more 'accurate' model is working.
*'''[[User:TomH|Tom]] 13:39, 22 August 2006 (EDT)''': I think it would just be interesting to see the differences. We've made LuxR vary, and if we still get oscillations when it isn't varied people may ask why we bothered to make LuxR variable in the first place.
*'''[[User:Farah|Farah]] 18:20, 28 August 2006 (EDT):''' Sorry I didn't answer you sooner about this - just noticed your comments buried in the mass of text on this page. We don't really have the option of choosing whether levels of LuxR vary or are constant. Our predator (J37016 or J37020) just simply has varying levels of LuxR, so assuming constant LuxR levels in the modelling would just hide what was actually going on in our system (we tried it but this point was pointed out to us by Vincent). If we adapted the predator to produce constant LuxR then we could make the approximation but this could only be done with J37020 which has two promoters - doing so for part J37016 would mean producing constant amounts of AiiA too.

*'''[[User:TomH|Tom]] 13:12, 13 September 2006 (EDT)''': Sorry if I was unclear. I wasn't suggesting to actually make the thing, but just to model it. Just so we can see whether having LuxR fluctuate gives us an advantage. For example, MIT did this without fluctuating LuxR, and its very possible they'll ask us our rationale for making it variable. And if we don't know what varied LuxR does as opposed to constant LuxR then we won't be able to answer that question.

===Cre/Lox Part===

*'''[[User:Johnsy|Johnsy]] 07:08, 27 August 2006 (EDT)''': What is going on with the Cre/Lox part? Have we started to ligate it yet?

*'''[[User:Jonny|Jonny]] 12:59, 27 August 2006 (EDT)''' This is being done by Ms. Jensen and now I think Dr. Mann also. Basically, we have to do two seperate PCR reactions and then fuse them together. One of the PCR reactions appears to be fine, however the other has failed four times. The reason for the failures is thought to be the very long overhanging fragment for one of the primers - around 60 base pairs. They have been playing around with the different temperatures, but there has been no success so far. If it continues to fail, then an option is to order two seperate sequences that anneal to each other which should then make the PCR simpler. Unfortunately none of the other ligations steps are possible without the PCRed fragment.
*[[User:Johnsy|Johnsy]] 18:04, 29 August 2006 (EDT): Ok...cool...but if it doesn't work, is it a very necessary part to have for our prey cell?  Have we finished testing the J37015?  or rather..how is the testing going for the J37015?
*'''[[User:JohnChattaway|JohnChattaway]] 09:44, 12 September 2006 (EDT)''' The cre lox part now exists and we may test it this week

==Resolved Issues==

===[[IGEM:IMPERIAL/Protocols/J37015|Prey cell positive feedback test]]===

*'''[[User:JohnChattaway|JohnChattaway]] 05:55, 22 August 2006 (EDT)''': I am curious about what data this test will actually give us. It seems like you are measuring the time taken to reach steady state rather than characterising the positive feedback loop. I think that a much better test would be;
**Make a fresh day culture of J37015 (OD 0.4 (guess))
**Make a T9002 AHL assay 
**then every 10 mins for 2hrs
***Take 1 1ml sample
***Record OD
***Centrifuge down cells, collect supernatant 
***Put supernatant on ice
**After 2 hours this should leave you with 12 samples of different AHL conc in the ice bucket you can test these with the T9002 assay.
**We would then normalise the AHL reading against OD
*'''[[User:TomH|Tom]] 07:07, 22 August 2006 (EDT)''': That's pretty much exactly what the protocol says to do. Although it needs to be adapted to look at GFP output over time also. Also it'll have to be made more clear if you had trouble working out what was going on, any volunteers? :D

===Non-functional Prey Cells===

*'''[[User:JohnChattaway|John C]]''': The prey cells don't flouressce! '''This is a majour problem!!!''' what are we doing about it?
**[[IGEM:Imperial/Results/J37015|This data]] suggests that the prey cells are flourescing however there is no propper controll so the experiment needs to be repeated. I am pretty sure the increaced flourescence is due to the presence of cells not producing GFP. There needs to be a positive controll.
**My experiment yesterday showed that the prey cells '''DO NOT PRODUCE AHL!''' this is a big problem.
**'''How can we fix this?'''
*'''[[User:TomH|Tom]] 14:10, 26 August 2006 (EDT)''': When I did the initial testing of J37015 I initally got a AHL and flourescence flatline. However, repeating the experiment and missing out the dilution steps from the o/n culture I got [[IGEM:Imperial/Results/J37015#Results|this output]] for AHL (although I forgot to measure J37015 direct flouresence, oops!). The data does show that AHL is being produced.
*'''[[User:TomH|Tom]] 10:01, 28 August 2006 (EDT)''': Any updates on this?
*'''[[User:JohnChattaway|JohnChattaway]] 04:40, 30 August 2006 (EDT)''' I have decided that the prey cells are producing AHL based on tom's data. Here's how I think we can test the prey cell positive feedback loop, uising a chemostat.[http://openwetware.org/wiki/IGEM:IMPERIAL/Equipment/Chemostat]

==Random Ideas==

===Jamboree Demonstration===

*'''[[User:TomH|Tom]] 18:43, 26 August 2006 (EDT)''': Hi guys, just thought I'd pop in with an idea. We were saying how we wouldn't be able to have a live demonstration at the jamboree because our period is too long. However, what about a ''simulated'' live demonstration? So we have a computer generated visual representation of the culture running off our model, and we can change parameters like cell ratios, growth rates and chemostat ODs. That way people can come along and fiddle with the paramters and be like 'wow, I can chage frequency/amplitude really easily!'. Plus, as it'll be based off a mathematical model, it isn't gonna go wrong in front of them. What do you reckon? I personally think audience participation is always a big plus. Also, just generally and not thinking about the jamboree, it'll be interesting and potentially very useful to see how the model responds to these new input parameters that we can easily change.

===General===
*[[User:baijiongjun|baijiongjun]]: hi, i'm finally back at china, still have limited access to internet, will be back in action soon, how's the project going on? how's everyone, will be pop in more in the wiki soon.
*[[User:Johnsy|Johnsy]] 18:04, 4 September 2006 (EDT): As I am currently updating this webpage from the Stata Center (where the iGEM jamboree will be held - quite a funky building indeed!), I am thinking of our presentation, as it is already the 10th week.  I apologise for not visiting the site within the past few days.  My aunt uses a dial up connection that is too slow to load web pages at any reasonable speed, and accessing the internet would mean that her landline was cut off whilst surfing the web.  But finally, I have access to MIT's incredible wifi system which works just about everywhere on campus :)...I was thinking of the following sections to add into our presentation:
**Introduction
***What is an oscillator?
***What is the motivation behind a biological oscillator?
***What has been done in the past? - but of course not to step on anyone's toes
**Design
***How our design is different to previous designs (MIT & Repressilator)
***The funky stuff we though of to control our oscillator, including the riboswitch, Cre/Lox system, chemostat
***Our quest for the holy AiiA...and Vincent's chemostat
**Modelling
***Our mathematical models for the entire system
***I don't think we need to go into detail about the models for the individual testing constructs..we can just say that we obtained the values...as like MIT did in their 2004 IAP presentation
**Results/Implementation
***After all that testing, the results we obtained from our testing constructs
***Did our final assembly work in the end? if not, can we explain our errors?
***Video of working system? or at least a nice animation of what it is supposed to look like?
**Conclusion
***What have we learned to do and not to do with synthetic biology?
***Is synthetic biology worth pursuing in the future?
***Future developments with our biological oscillator
***Analysis of our project, suggestions for future iGEM teams wishing to tackle the problem of the biological oscillator.
**Any other suggestions?

*'''[[User:JohnChattaway|JohnChattaway]] 09:47, 12 September 2006 (EDT)''' I think we should focus on how our AHL system is based on predator prey dynamics because it is actually quite a complicated concept which will take 10min to explain fully, this will generate more interest in our project than all the little tricks we developed to alter how it works

*[[User:Johnsy|Johnsy]] 08:50, 13 September 2006 (EDT):  We also have a poster presentation to do.  I think we should try to keep the presentation as simple as possible and have all the details in the poster presentation.  This would prevent our presentations from getting too boged down and from preventing people from falling asleep.  Yes, we do need to go into some details, which is okay, but too much details might have the reverse effect of what we want.

==Working Bank Holiday Weekend==
*'''[[User:Farah|Farah]] 18:37, 27 August 2006 (EDT):''' This is to John. Can you come in tomorrow afternoon to read the plates and culture up stuff for tuesday? I can call/mail you to let you know when you need to come in. We've worked it out so that then you, Christin, Jimmy and I will only need to contribute a half day...Our fate lies in your hands....
*'''[[User:Farah|Farah]] 17:51, 28 August 2006 (EDT):''' We've gathered a lot of data this weekend. Could everyone process whatever data they have (using the [http://openwetware.org/images/a/a2/T9002_Data_Spreadsheet.xls spreadsheet])and update the results page?
*[[User:Johnsy|Johnsy]] 18:05, 29 August 2006 (EDT): Pity they didn't have a holiday here in the US...but we do have one next Monday 4 Sept!  :)  Good to see you guys still dedicated!