Name: Jennifer Logan <br>
Email: jlogan at mit dot edu <br>

==== M13 phage re-engineering ideas: ====
{| Border="1"
! Gene !! Ideas
|-
| I || Modify such that it is able to interact with multiple p4s and see how phage   secretion changes (this would affect the effectiveness of the channels).
|-
| II || modify such that it not only nicks the double stranded form of the genome to initiate replication of the + strand, but also nicks the - strand to impede the formation of dsDNA (this would also help g5). Care should be taken when modifying because this is also linked to g10.
|-
| III || Add myc or other tag to monitor how it affects time delay/progression of phage escape from host
|-
| IV || modify to increase its affinity for p1 and p11--this would possibly increase the effectiveness of the channels
|-
| V ||Modify such that it is able to sequester the positive stranded DNA more effectively, thus reducing the competition with dsDNA formation
|-
| VI || As an accessory protein to p3, modify such that p3 is more effective in its interaction with TolA protein
|-
| VII || Modify in such a way to allow for phage secretion speed to increase so the phage-host interaction time is decreased. 
|-
| VIII || Add myc or alternative tag to possibly aid in targeting various qualities of host
|-
| IX ||Same principle as gene VII except that care must be taken with this modification because it overlaps with gene 8.
|-
| X || Modify such that + strands are not dependent on the presense of p10. This would work in conjuction with our modification of p2 (since they are linked).
|-
| XI || same as p1
|}

===M13.1 Section Design===

{| Border="2"
! Modification/Limitation !! Description 
|-
| gene VIII and promoter-g3 || These two overlapped so I separated and added in ecoRI (g/aattc)and XmaI (c/ccggg) restriction sites to create unique sticky ends. I also modified codons at the wobble positions so that the promoter for g3 can no longer exist within gene VIII but the amino acid sequence of gene VIII was maintained. (Portion of overlap in gene VIII is now 5' aattTacctcgaaagcaagTtga 3')
|-
| myc epitope within gene 3 || Myc epitope is known as a good tag, this way when gene 3 regulation occurs the myc tag will help locate this regulation. I had to add some bases on the ends of the myc epitope so that when I cut with the restriction site (NIaIII: ccatg/t)I was able to anneal my insert. However, I designed the insert such that when it anneals with our segment, the NIaIII site is destroyed. (Myc epitope top strand: 5' GAA CAG AAA CTG ATC TCT GAA GAA GAC CTG catg 3'
bottom strand: 5' CAG GTC TTC TTC AGA GAT CAG TTT CAG TTC catg 3')

|-
|- 
|}

The modifications that were necessary for me as a Discoverer (I choose to focus on gene 3) were mainly to unstuff the overlapping regions of gene VIII and the promoter of gene III. I did this by first copying over the region of overlap between gene VIII and the promoter of gene III and then modified two bases within the gene VIII overlap sequence (the wobble positions)so that gene III can no longer exist within it. However, care was take that the amino acid sequence was maintained. Also, two restriction sites (ecoRI and XmaI) were added to create unique sticky ends. I then found another restriction site (NIaIII) and cut open inbetween gene III so that I could insert a myc epitope that I modified by adding sticky ends (but upon insertion would destroy the NIaIII site). As the table above indicates, I hoped to insert the myc epitope because since it is known as a good tag, when regulation of gene III does occur, the myc epitope will help us pinpoint more exactly when this occurs. I hope that this will help me discover more about the regulation of gene III.
<br/>
[http://parts.mit.edu/r/parts/partsdb/part_info.cgi?part_name=BBa_M31517 My Redesigned Phage Section]