== Notes on <cite>Forterre-PNAS-2006</cite> ==

* see also <cite>Forterre-Biochimie-2005</cite>
* transition from RNA to DNA genomes could have been carried out by viruses (to protect from host RNA defenses)
** cellular DNA and replication machineries originated by transfers from DNA viruses to RNA cells
*** 3 seperate such transfers could be at origin of Archea, Bacteria, and Eukarya - could explain why each domain has a specific DNA replication apparatus
** plasmids transitional forms between DNA viruses and cellular chromosomes
* unification of cellular life - all cells share a common mechanism for protein synthesis with same genetic code and thus originated from common ancestor: Last Universal Cellular Ancestor (LUCA)
** each domain characterized by a different type of ribosome
* introduction reviews pre-genomic theories to explain how these domains originated and what evolutionary relationships among them
* archea have histones <cite>Reeve-BiochemSocTrans-2004</cite>
* thought that archea evolved from bacteria by adaptation to hyperthermophily, but are cases of regular hyperthermophilic bacteria that use bacterial versions of their proteins
* Woese <cite>Woese-MicrobiolRev-1987</cite>: suggestion that the rate of protein evolution higher in time frame between LUCA and last common ancestor of each domain today
* all this analysis  based on translation and transcription apparatus
* major proteins in bacterial DNA replication (DNA polymerase, primase, helicase) not homologous to archael/eukaryotic homologs - one version of DnaG primase (bacterial), and 2 in other branches
** cellular DNA informational proteins found in only 1 or 2, not 3 versions
** can explain violation of 'one family, three versions' rule by considering multi-cellularity and viruses
* see also <cite>Forterre-ResMicrobiol-2003 Bamford-ResMirobiol-2003 </cite>
** structural similarities between capsid proteins and replicating enzymes of viruses infecting different domains - viruses older than thought <cite> Bamford-ResMirobiol-2003 </cite>
* viruses can be sources of new proteins for cells <cite> Daubin-CurrOpinGenetDev-2004 </cite>
** evolution of mitochondria from alpha-proteobacteria - original bacteria RNA polymerase, DNA polymerase and helicase replaced by T3/T7-related viral proteins <cite>Filee-TrendsMicrobiol-2005</cite>
* viruses could have invented DNA to counteract host RNA defenses - many modern viruses encode viral-specific versions of ribonucleotide reductases and thymidylate synthases (needed to make DNA precursors)
** see also <cite>Villarreal-VirusesAndTheEvolutionOfLife-2005</cite>
* host RNA could have been transformed to DNA by a persistent viral infection (via a plasmid), with gradual accumulation of host genome as more stable DNA
* propose here that this happened three, independent times giving rise to 2 DNA replication machineries (Bacteria and Archea/Eukarya) and three ribosomal machineries
** ancestral RNA cells out-competed by DNA cells which could have larger and more stable genomes - also once these DNA cells took over, would have 'fixed' the three domains
* encoding by DNA would have caused drastic drop in mutation rate, thus rate of evolution
* archael lipids have opposite chirality than bacterial and eukaryotic lipids
* plasmids originated from viruses (not vice versa because then a plasmid would have to 'invent' a capsid protein)
** archea and bacteria have plasmids, eukarya do not
* postulate that the virus that gave rise the eukarya had a linear DNA genome (possible multiple chromosomes)
** several Eukaryotic RNA and DNA polymerases could suggest eukarya was caused by integration of several viruses
* nucleocytoplasmic large DNA viruses, ex: poxviruses - replicate in cytoplasm, form small nuclei, produce envelope by recruiting membrane from ER
** such a system could have evolved into the eukaryotic nucleus
** mimivirus NCLDV with 1.2 Mb genome
*** capsid proteins homol to Adenoviruses and several bacterial and archael viruses suggesting existed before formation of eukaryotes <cite>Takemura-JMolEvol-2001</cite> 
* can test experimentally by designing RNA plasmids with reverse transcriptase and see how much gets transferred to DNA genome
* can test informatically by looking at all viral DNA informational proteins - should be viruses still around that closely resemble the founding viruses
** recent discovery bacterial prophage homolog of archaeal replicative helicase minichromosome maintenance protein (MCM)


== References ==

<biblio force=false>
#Reeve-BiochemSocTrans-2004 pmid=15046577
#Woese-MicrobiolRev-1987 pmid=2439888
#Forterre-ResMicrobiol-2003 pmid=12901366
#Bamford-ResMirobiol-2003 pmid=12798226
#Daubin-CurrOpinGenetDev-2004 pmid=15531155
#Filee-TrendsMicrobiol-2005 pmid=16157484
#Villarreal-VirusesAndTheEvolutionOfLife-2005 isbn=978-1555813093
#Takemura-JMolEvol-2001 pmid=11443345
#[[Julius B. Lucks/Bibliography]]
</biblio>