= Jean-Michel Claverie: Mimivirus and the origin of large DNA viruses infecting eukaryotes - reconciling incompatible scenarii =
* Institute de Biologie Structureale et Microbiologique

== Words to Lookup ==
* intein

== Talk ==
* 1.2 Mbp genome
* in between Eukarya and the tri-vertex of Euk, Bac, Archea
* 1992 Bradforcoccus
** gram-positive cocci-like intracellular bacterium - Bradforcoccus
** no 16S rRNA, no culture
* A giant Virus in Amoeba, Science 299, 203, 2003
* icosahedral virus-like particle
** 750 nm - same size as U. urealyticum - almost a micron size
* how prove is a virus
** typical 'eclipse phase' during infection - not divide like bacteria
*** eclipse - disappear, then reappear when start to multiple
** use confocal microscopy
* The 1.2 Mb genome of Mimivirus. Sciene 206, 1344, 2004
* linear, 72% AT, 911 predicted CDS (298 with putative function), 90.5% fraction coding, 6 tRNAs, 1 intein, Intron (Type 1b) $ + 2
* J. Virol, 75, 11720, Koonin - Common origin of four diverse families of large eukaryotic DNA viruses
** core genes of NCDLVs - Nucleo-Cytoplasmic Large DNA Viruses
** all these genes are in Mimivirus
* no transcription apparatus (RNA polymerase, etc.)
* 1st virus to have genome bigger than cellular organism (parasytic bacteria)
** bigger in complexity than some free-living bacteria
* viruses and bacteria
** v no translation app
** v no energy metab
** v no biosynthesis
** v have capsid protein genes
** v have core protein genes
* Mimi - 8 components of translation
** uses universal code
** 4 tRNA synthetases
*** one does not recognize 3rd base of codon - Tyr tRNA synth, only competition is TAT or TAG - both stop codons
**** this could be viral optimization, or a relic of a long time ago
** all translation factors to start, elongate and end translation
** tRNA modification enzyme (U to T in a t-loop)
** 6 tRNA (3 Leu, Trp, His, Cys)
** does have some metabolism genes
** all DNA repair enzymes
** using these genes, can hang on the tree of life
** only virus with MutS - mismatch repair
*** seems to be maybe on another branch of life, or at least a distant ancestor on the eukaryotic branch

=== Mimivirus predates the Eukaryotic radiation ===
* falsifiable prediction
* equidistant from all Eukaryotic kingdoms

=== Evolutionary hypotheses ===
* viruses predate everything (d'Herelle, 1922)
* viruses at origin of nucleus (Villareal)
* nucleus at origin of viruses (Bell)
* viruses invented DNA (Forterre)
* see 'iterative virual eukaryogenesis & nuclear viriogenesis' - Forterre-lke - start with RNA cell
** cell stores its own genes in viral DNA genome (virus invent DNA)
** viruses revert to viral world - every time cycle produce a new lineage

=== Where genes coming from ===
* not duplication
* not accretion
* reduction
** reduction from a past NCLDV ancestor - have lineage specific losses
* see [http://www.giantvirus.org GiantVirus.org]
** another NCDLV infect the xxx huxlei algae that occur in blooms - responsible for the loss of the bloom

=== A New way of Looking at Viruses ===

= Martine Zilversmit: Low-complexity region in the human malaria parisite ''Plasmodium falciparum'' - missing links in the evolution of an extreme genome =
* collaborated with Mark DePristo!

== Words to look Up ==
* low complexity regions
** regions selected from limited amount of AAs
** simple repeats
** aperiodic repeats
** predominantly hydrophilic
** very non-compact structure - flexible

== Talk ==
* not a monolithic type of sequence
** 3 distinct families of sequence, with non-overlapping characteristics and evol mechanisms
* pathology and public health intimately link with evolution of plasmodium
** organism can adapt to human intervention
*** evades immune system, adapts to chemicals
* eukaryotic parasite - protist
* 14 chromosome karyotype
* 81% AT
* highly recombinogenic (1% per 17 kilobases - approx 60% more than human genome)
* 87% proteins have low complex region
* LCRs
** adaptive smokescreens in surface antigens
** polymorphic in terms of size
** P. falciparum proteins longer than orthologs because of LCRs
** elevated AT-content relative to flanking coding regions
** coded for by low-complexity DNA sequence
* comp genomics
** LCRs abundant in AT-rich genomes, but not adaptive
** extreme codon bias (due to AT and hydrophilic functional pressure) - proposed results in formation of random repeat regions
** repeats initially groww by replication slyppage
** evidence of elevated recombination rates

= Christopher Lane: Impact of genome reduction on the eukaryotic proteome - nucleomorphs as a case study =
* DalHousie

== Words to Look Up ==
* ribosomal cistron

== Talk ==
* chryptomonads (engulphed red algae) and chloracniophytes (engulphed green algae)
* similar functional categaries in nucleomorphs, but not a lot of overlap
* no obvious barrier for NM-to-N gene transfer (Nucleomorph to Nuclear)
* Gillardia theta nucleomarch (2001), letters to Nature, 'The highly reduced genome of an enslaved algal nucleus', Susan Douglas et. al.
* Hemiselmis andersenii nucleomorh (chryptomonad) - sequenced
** 60% conserved genes
** all introns lost (also splicing machinery)
** 94% conserved sequences with Gillardia are syntenic
* genome compaction can significantly impact the structure and function of proteins

== Questions ==
* Koonin: spliceosomes coupled to nuclear transport - given that no introns, have looked at nuclear pore?

= Christina Toft: Identifying candidate genes for non-functionalisation in endosymbiotic genomes =
* evol genet and bioinformatics, Murfit institute of genetics, Trinity College, Dublin

== Words to Look Up ==
* Muellers ratchet

== Talk ==
* phylogenetics of the eukaryotes. Mol Biol Evol, 10, 1093,2007, Pisani, Cotton, Mcinerney
* endosymbiosis highly unstable system
** very few bacteria transferred between hosts
** bacteria are asexual - very low opportunity to recombine
** small population
** all lead to Muellers ratchet phenomenon
*** mutation load increases as goes through generations
*** can lead to non-functionalisation, disintegration, genome reduction
* Genome Reduction Analysis Software Tool (GRAST) Toft and Fares, 2006, Bioinformatics, 22, 1551

= Dinis Lavrov: Convergent evolution in animal mitochondrial DNA =
* Iowa state

== Talk ==

=== Animal mtDNA ===
* size and gene content - Buger, TIG, 2003
** 22 tRNA (trnX)
** 2 ribosomal RNA (rns, rnl)
** 13 protein genes (nad1-6,4L,cob,cox1-3)
* bizarre tRNA and rRNA structures
** 3 diff tRNA structures - some have only D-arm, some only T-arm, some both ('normal')
* multiple changes in the genetic code - Knight, TIG, 2003 (Trends in Genetics?)
** TGA: Stop to Try
** AGR: Arg to Ser
** AUA: Ile to Met
** last 2 allows animals to use 22 tRNAs instead of 24 as minimal set
* mutation rate 15-34x10^-9 /site/year - up to 25 times faster
** in plants 20 times slower
** used in criminal and phylogeography study - can detect differences between individuals

=== Animal mtDNA evolution ===
* porifera (sponges)
* Lavrov, 2005, MBE: demosponges - extra genes (atp9, tRNAs) - minimally derived genetic code, conserved structuras of RNAs, multiple intergenic regions

=== Convergent evolution of animal mtDNA ===