= Patrick Keeling: Nuclear Genome Diversity =

== Background ==
* Botany: U British Columbia

== Talk ==
* Keeling and Slamovits, 2005, Curr. Opin, Genet. Div.
* dynoflaggelites and ameoba have large genomes
* 2 small genome groups
** microsporidia - related to fungi - obligate intracell parasites
*** huge tube that can peirce cell that they can travel into
** nucleomorphs - organells found in algae: cryptomonads, chloracniophytes
*** have engulfed another algae and kept it as a symbiont
* Keeling 2005, Trends Ecol. Evol.

=== How genomes get small ===
* reduction - massive gene loss
** host-dependance - loss of whole pathways
* compaction - 2x compaction of yeost
** very short intergenic regions
** reduced number and size of introns
** genes shorter
* how does this affect how genomes work?
** most eukaryotic - promoter upstream and terminator downstream
** reduced genomes - transcripts overlap - opposite strands (transcriptional collision), or same strand (promoter occlusion)
*** like some viruses
** Williams, PNAS, 2005
* keep all tRNAs and ribosomal RNAs - ribosomal RNAs at the telomeres

=== Comparison of Antonospora and Encephalitozoon ===
* promoter regions of a gene can be inside coding region upstream of it

=== Pygmy Introns (Nucleomorphs) ===
* cryptomonid - 17 introns of about 50 bases (Douglas, Nature, 2001)
** lost many
* chloracachniophyte - 841 introns 18,19,29,21 bp (Gilson, 2006, PNAS)
** retained all, but shrunk
** GTnnnnAG - GT/AG boundaries - spliceosomal introns

=== Shrinking Proteins ===
* genes shorter - Katinka, 2003
** 9% reduction compared to yeast
* want to reduce DNA that bad?
* reduction of complexity of the proteome?
* ostreococcus tauri - picoplankton - one of smallest organisms known
* 12.5 mbp, 8,166 proteins, gene density 1.54 genes/kbp (in between yeast and the extremes)

== Questions ==
* intron loss by reverse transcriptase mediated gene replacement
* Koonin: any lifestyle differences between the cryptomonids and chloracachniophytes?

= Jeffrey Palmer: Horizontal gene transfer gone wild in plant mitochondrial genes =

== Talk ==
* Amborella trichopoda mtDNA: 4 Mb - outside:native::5:1
* bacteria - maybe 1/2 of genome can be from outside sources
* Parkinson, Adams, Palmer, Curr. Biol, 9, 1481, 1999 - Amborella one of 1st angiosperms (flowing plants)
* Amborella contains foreign mitochondrial genes of moss and angiosperm origin
* grows only on new caledania (next to Australia) (80% of plants are endemic there (only live there))
* mtDNA:
** Human 16 kb
** Arabidobsis 367 kb
** Zea 570 kb
** Amborella 4,000 kb!
* does not have 
** foreign chloroplast DNA
** foreign nuclear DNA
** bacterial DNA
** mtDNA from fungi and other 'not-green' plants
* Frequent fusion and fission fof plant mitochondria - PNAS
** phylogenetic gulf in fusion mechanisms between (animals/fungi) & plants- Hoppins et al Ann Rev Biochem 76,751,2007
** chloroplasts don't fuse like mitochondria
* mtDNA retains synteny of its foreign acquisitions

=== How does foreign mtDNA get into Amborella's mitochondria ===
* how do mitochondria get from one plant into cells of another
** direct plant-to-plant contact
*** grows in wet env - lots of epiphytes, epaphylls - mosses, lichens
*** how get into germ line?
**** large tree that gets wounded and dies back - suckers grow off at sites of wounding
**** suckering - cellular dediffertiation - recreate meristems - promotes mitochondrial fusion
***** mascerate foreign substances in a wound to get foreign sources
**** other plants that grow like this have same mitochondrial size?
** biological vectoring agents - viruses, bacteria, insects
** illegitimate pollination
* of 250 forign mt genes - probably very few functional



= Atsushi Nakabachi: Smallest cellular genome =

== Background ==
* RIKEN (Intitute of Physical and Chemical Research)

== Words to Look Up ==
* Chromosome FISH

== Talk ==
* Carsonella ruddii 160 kb genome
** endocellular symbiotic bacterium of psyllids

* E. coli - 4000 genes (Human 23,000, Yeast 6,000)
* prokaryotes - simpler body plans and lifestyles
** gene density high, no introns so gene #'s proportional to genome sizes
* endocellular prasites (prokaryotes) have smaller genomes
* insects - bacteriocytes (in body cavity) - host cells specialized to harbor symbionts
** host live in nutrient poor environments
** obligate mutualisw
* Host: Bacteriocyte : Symbiont
** Carpenter Ant: midgut epithelium : Blochmannia floidanus
** Tsetse flies: specialized epithelial cells : Wiggleswortha glossinidia
** Aphids: body cavity: Buchnera aphidicula

=== General featurs of bacterial symbiont genomes ===
* massive reduction in genome sizes (420-800kb)
* AT-richness (20-30% GC-content)
* gene inactivation and deletion - relaxed selection, small pop size, mutational pressure due to loss of DNA repair genes
* general correlation between GC-content and Genome size

=== Corsonella ===
* hakcberry petiole gall psyllid - related to aphids - body cavity bacteriacyte
** Pachypsylla venusta
** Carsonella primary symbiont
* 16.5% GC-content, 0.16 Mb
** 15.9% protein coding regieons
* single chromosome no plasmids
* 159,662 bp
** < 1/2 of 2nd smallest genome
** approx same as chloroplast genomes
* no insertion sequences, phages or transposons
* 182 predicted ORFs - 136 putative functions 
** avg length 826 bp (not shorter than in other bacteriocyte symbionts (981-1006 bp))
** in bacteria approx 1 kb
* 97.3% protein and RNA coding regions (other bacteriocyte symbionts 77-80%)
** due to numerous overlaping ORFs
** majority are tandem out of frame overlaps on same strand
* COG classification of ORFs
** mostly in translation (35%), and amino acid metabolism (18%)
* phyllids and aphids feed only on plant ploem sap - poor in essential AAs - need symbionts to synthesize some AAs
* apparently no genes for cell division
* no genes for cell envelope biogenesis (biosynthesis of fatty acid, phospholipid, lipopolysacharide)
* no genes for lipid or nucleotide metabolism
* many known genes necc for glycolysis and TCA cylle missing
* only 2 transporter genes retained
* few genes for DNA rep, recomb and repair - helicase (dnaB), primase (dnaG), alpha (dnaE) and epsilon (dnaQ) subunits of DNA pol III, and recA retained

=== How can Survive? ===
* biological processes shared by prokaryotes and metazoa
** compensation for metazoa-like processes of host psyllids (synthesis of non-essential AAs)
* proteins gained multiple functions?
* genes transferred to host nuclear genome?
** have happened to mitochondria and chloroplasts (endosymbiont to organelle)
** transfer of genes, acquisition of promoters, evolution of protein translocation

=== Questions ===
* have you seen Carsonella divide?
** tubular structure - have to be short before transfer to next generation of insect - cell div should happen before this
* biased nucleotide usage in leading and lagging strands - see this as well?
* does cell wall of Carsonella have peptidoglycan? - not known