{{Kafatos/Christophides Lab}}
<BR><BR>
== Preparation of dsRNA from PCR products (single tube synthesis) ==
=== PCR ===
• Design PCR primers for a 200 – 600 bp section of the gene of interest. The targeted region should not be located in high homology regions, and preferably located as close to the 3’ prime end of the gene as possible. Tail both your PCR primers with the T7 promoter sequence GAATTAATACGACTCACTATAGGGAGA at their 5’ end. Use this [http://www.dkfz.de/signaling/ernai/ernai_prime.html website] to assist you.<BR>
• Standard purification (desalt) of the primers is acceptable.<BR>
• Amplify the fragment by PCR using the following concentrations:<BR>
 10x PCR Buffer                  5 	μl 
 Taq Polymerase                  0.7	µl
 dNTP mix (10 mM)                1	µl
 DNA template (0.5 μg/μl)        1	μl
 forward primer (10 pmol/μl)     1	µl
 reverse primer (10 pmol/μl)     1	µl
 ddH2O                           40.3	μl 
 '''Total:                          50	μl'''
		 
cDNA should be used as PCR template, but genomic DNA (at 1 μg/μl) is acceptable if the targeted region does not contain an intron and you are confident of the gene architecture.<BR>

=== Purification of PCR product === 
The purification of the PCR product is achieved with the '''QIAquick PCR Purification kit (QIAGEN)''' using a microcentrifuge.
(Add ethanol to Buffer PE before use (see bottle label for volume); all centrifugation steps are carried out at 10,000 g (~13,000 rpm)).<BR>
• Add 5 volumes (250 μl) of Buffer PB to one volume of PCR reaction (50 μl) and mix.<BR>
• To bind the DNA, apply the sample to a QIAquick column in a 2ml collection tube and centrifuge for 30-60 sec.<BR>
• Discard the flow-through and wash by adding 0.75 ml Buffer PE to the column and centrifuging for 30-60 sec.<BR>
• Discard the flow-through and centrifuge for an additional 1 min at max. speed.<BR>
• Place the column in a clean 1.5 ml Eppendorf microfuge tube.<BR>
• To elute the DNA, add 30 μl Buffer EB (10 mM Tris-Cl, pH 8.5) or ddH2O to the center of the column membrane and centrifuge the column for 1 min. 
Repeat last step with another 30 μl into the same tube.<BR>
• Run the 3 μl of the DNA on a 1% agarose electrophoresis gel supplemented with 0,5µg/ml Ethidium bromide. Check that there is only a single band at the expected size.<BR>

=== Production of dsRNA ===
The purified PCR amplicons are now used to synthesize dsRNA with the '''MEGAscript T7 Kit (Ambion)'''. <BR>
• Add to an RNase free tube in this order:<BR>
 DEPC treated H2O      4	μl
 ATP	              2	μl
 CTP	              2	μl
 GTP	              2	μl
 UTP	              2	μl
 Buffer (warm to 37˚C) 2	μl
 cleaned PCR product   4	μl
 Enzyme mix            2	μl
 '''Total                20	μl'''

• Mix gently and spin down.<BR>
• Incubate at 37˚C for 12-16 hours (preferably with constant shaking). Incubation times may vary – check viscosity of the sample by tapping on the tube with your finger – its viscosity will increase with time.<BR>
• You may interrupt and freeze the sample at -80˚C to continue at a later time.<BR>

=== Purification of the RNA ===
There are two steps for the purification of the dsRNA: the digestion of the DNA template and the clean-up of the sample with the '''RNeasy kit (QIAGEN)'''.<BR>
(β-Mercaptoethanol (β-ME) must be added freshly to the Buffer RLT before use (under hood!). Add 10 μl β-ME per 1 ml Buffer RLT – you only need 350 μl Buffer RLT per reaction. Also make sure that 4 volumes of ethanol have been added to 1 volume of Buffer RPE.)<BR>

• ''Digestion of the DNA with DNAseI:'' add 1μl DNAse (Ambion Kit) to the 20 μl reaction and incubate at 37˚C for 15 min.<BR>
• Add 80 μl RNase-free H2O to the sample to increase the volume to 100 μl. Add 350 μl Buffer RLT, and mix thoroughly. <BR>
• Add 250 μl ethanol to the diluted RNA and mix by pipetting (do not vortex/centrifuge!). Proceed immediately with next step.<BR>
• Transfer the sample (700 μl) to an RNeasy mini column in a 2 ml collection tube. Close and centrifuge for 15 s at 8,000 g (~10,000 rpm). Discard flow-through and collection tube. Transfer column into new 2 ml collection tube. <BR>
• Pipet 500 μl Buffer RPE onto the RNeasy column. Centrifuge for 15 s at 8,000 g (~10,000 rpm) to wash the column. Discard flow through.<BR>
• Repeat last step, but centrifuge for 2 min to dry the silica-gel membrane.<BR>
• Place the column in a fresh 2 ml collection tube (you can use 2 ml Eppendorf tubes) and centrifuge at full speed for 1 min to eliminate any chance of ethanol carry over.<BR>
• To elute, place the RNeasy column in a new 1.5 ml RNase free collection tube.<BR>
• Pipet 30 μl RNase-free water directly onto the silica-gel membrane of the column. Centrifuge for 1 min at 8,000 g (10,000 rpm).<BR>
• Repeat last step into same collection tube.<BR>
• The concentration of the dsRNA is determined by photo-spectrometric concentration analysis (OD260) and adjusted to 3µg/µl by concentration with a vacuum centrifuge. <BR>
• The dsRNA is also analyzed by running 1 μg of it on a 1.7 % agarose electrophoresis gel supplemented with 0,5µg/ml Ethidium bromide (use the MEGAscript T7 Kit (Ambion) RNA loading buffer) .<BR>
• Store dsRNA at -80˚C.<BR>

 note: The dsRNA is not denatured and annealed by a boiling and cooling cycle.