=Kai's Notebook (Gulick Lab)=

==Protocols==

*[[Kai_Yuet/Protocols:Transformation | Bacterial Transformation]]
*[[#The_Bootstrapping_Process | Bootstrapping]]
*[[Kai_Yuet/Degenerate_Primers | Degenerate Primers]]
*[[Kai_Yuet/Protocols:DNA Ligation | DNA Ligation]]
*[[Kai_Yuet/Protocols:Transfection | Eukaryotic Transfection]]*
*[[Kai_Yuet/Protocols:Plasmid Maxipreparation | Plasmid Maxipreparation]]*
*[[Kai_Yuet/Protocols:Plasmid Minipreparation | Plasmid Minipreparation]]*
*[[Kai_Yuet/Protocols:RNA Isolation | RNA Isolation]]
*[[Kai_Yuet/Protocols:RT-PCR | RT-PCR]]*
*[[Kai_Yuet/Protocols:SDS-PAGE | SDS-PAGE]]*
*[[Kai_Yuet/Protocols:Western Blot | Western Blot]]*

==Laboratory Notebook==

===January 29, 2007===

I am also trying to determine the exact relationship among the introns flanking exon three of ''Ciona intestinalis'' and the two introns flanking the alpha region of ''Branchiostoma floridae'' as well as the intron separating the two alpha exons:

'''''Ciona'' (Intron Upstream) x ''Ciona'' (Intron Upstream):'''

Percent Similarity: 100.000   Percent Identity: 100.000<br>
Quality:   2830<br>
Average quality based on 100 randomizations: 459.3 +/- 44.6

'''''Ciona'' (Intron Upstream) x ''Branchiostoma'' (Intron Upstream of Alpha):'''

Percent Similarity: 70.513   Percent Identity: 70.513<br>
Quality:    285<br>
Average quality based on 100 randomizations: 293.4 +/- 39.2

'''''Ciona'' (Intron Upstream) x ''Branchiostoma'' (Intron in Alpha):'''

Percent Similarity: 74.286   Percent Identity: 74.286<br>
Quality:   283<br>
Average quality based on 100 randomizations: 260.6 +/- 38.4

'''''Ciona'' (Intron Downstream) x ''Branchiostoma'' (Intron in Alpha):'''

Percent Similarity: 70.417   Percent Identity: 70.417<br>
Quality:    283<br>
Average quality based on 100 randomizations: 320.6 +/- 51.9

----

===January 26, 2007===

I am trying to determine the exact relationship among exon three of ''Ciona intestinalis'' and the two alpha exons of ''Branchiostoma floridae''; in order to do so, I am aligning the exons with a very low gap opening penalty (~0) and a relatively high gap extension penalty (~12):

'''''Ciona'' x ''Branchiostoma'' (Alpha One):'''

Percent Similarity: 88.095   Percent Identity: 88.095<br>
Quality:    217<br>
Average quality based on 100 randomizations: 110.1 +/- 17.8

'''''Ciona'' x ''Branchiostoma'' (Alpha Two):'''

Percent Similarity: 78.049   Percent Identity: 78.049<br>
Quality:    477<br>
Average quality based on 100 randomizations: 207.6 +/- 34.2

'''''Ciona'' x ''Branchiostoma'' (2nd Region of Alpha One):'''

Percent Similarity: 79.412   Percent Identity: 79.412<br>
Quality:    147<br>
Average quality based on 100 randomizations: 125.6 +/- 18.9

'''''Ciona'' x ''Takifugu'' (Alpha One):'''

Percent Similarity: 81.513   Percent Identity: 81.513<br>
Quality:    400<br>
Average quality based on 100 randomizations: 190.8 +/- 33.9

'''''Ciona'' x ''Takifugu'' (Alpha Two):'''

Percent Similarity: 79.487   Percent Identity: 79.487<br>
Quality:    450<br>
Average quality based on 100 randomizations: 203.2 +/- 36.5

'''''Branchiostoma'' x ''Takifugu'' (Alpha One):'''

Percent Similarity: 83.721   Percent Identity: 83.721<br>
Quality:    177<br>
Average quality based on 100 randomizations: 119.4 +/- 18.2

'''''Branchiostoma'' x ''Takifugu'' (Alpha Two):'''

Percent Similarity: 79.130   Percent Identity: 79.130<br>
Quality:    454<br>
Average quality based on 100 randomizations: 188.0 +/- 30.0

'''''Branchiostoma'' (2nd Region of Alpha One) x ''Takifugu'' (Alpha One):'''

Percent Similarity: 80.435   Percent Identity: 80.435<br>
Quality:    133<br>
Average quality based on 100 randomizations: 113.5 +/- 22.3

----

===January 24, 2007===

Due to the appearance of a higher amount of DNA than expected in the gel electrophoresis of digestions of "pTol2 L/R" and "corrected pcmv hDα #3", I re-checked the OD<sub>260/280</sub> of "pTol2 L/R" and "corrected pcmv hDα #3" (diluted with 200 µL TE, subsequently diluted 5 µL of that with 995 µL of ddH<sub>2</sub>O for OD) midipreparations:

'''pTol2 L/R'''

260 nm Absorbance: 0.131 AU<br/>
280 nm Absorbance: 0.080 AU<br/>
A260/A280: 1.6424

[DNA] = (0.131 AU)(0.05 [µg/µL]/AU)(1000 µL/ 5µL) = 1.31 µg/µL

'''corrected pcmv hDα #3'''

260 nm Absorbance: 0.128 AU<br/>
280 nm Absorbance: 0.084 AU<br/>
A260/A280: 1.5249

[DNA] = (0.128 AU)(0.05 [µg/µL]/AU)(1000 µL/ 5µL) = 1.28 µg/µL

----

===January 23, 2007===

I performed a PCR screen of 12 colonies of pTol2 L/R kpy47+kpy48 (1X, 0.1X, 0.01X) using primers kpy47 and kpy49 for the first 6 colonies, kpy48 and kpy49 for the last 6 colonies.  Only colony 11 was positive.  Colony 11 will be cultured overnight for minipreparation tomorrow.

Minipreparation of 6 pTol2 L/R kpy47+kpy48 colonies were also performed, resulting in one positive when digested with EcoRI and PvuII.

----

===January 22, 2007===

I checked the OD<sub>260/280</sub> of "pTol2 L/R" (diluted with 200 µL TE, subsequently diluted 5 µL of that with 995 µL of ddH<sub>2</sub>O for OD) midipreparations from yesterday:

'''pTol2 L/R'''

260 nm Absorbance: 0.108 AU<br/>
280 nm Absorbance: 0.042 AU<br/>
A260/A280: 2.5534

[DNA] = (0.108 AU)(0.05 [µg/µL]/AU)(1000 µL/ 5µL) = 1.08 µg/µL

 kpy47: CGCGTGACTGGATCCGACTGTATAC
 
 Hetero-Dimers:
 
 kpy47: 5' CGCGTGACTGGATCCGACTGTATAC             3'
               |||||||||||||||||||||    
 kpy48: 3'     ACTGACCTAGGCTGACATATGGCGC         5'
 
 SEQUENCE: 5'- CGC GTG ACT GGA TCC GAC TGT ATA C -3'
 COMPLEMENT: 5'- GTA TAC AGT CGG ATC CAG TCA CGC G -3'
 LENGTH: 25
 GC CONTENT: 56.0%
 MELT TEMP: 60.6ºC
 MOLECULAR WEIGHT: 7658.0 g/mole
 
 kpy48: CGCGGTATACAGTCGGATCCAGTCA
 
 Hetero-Dimers:
 
 kpy47: 5' CGCGTGACTGGATCCGACTGTATAC             3'
               |||||||||||||||||||||    
 kpy48: 3'     ACTGACCTAGGCTGACATATGGCGC         5'
 
 SEQUENCE: 5'- CGC GGT ATA CAG TCG GAT CCA GTC A -3'
 COMPLEMENT: 5'- TGA CTG GAT CCG ACT GTA TAC CGC G -3'
 LENGTH: 25
 GC CONTENT: 56.0%
 MELT TEMP: 61.7ºC
 MOLECULAR WEIGHT: 7667.0 g/mole

2 µg of each fragment were annealed together in an Eppendorf by placing the tube into 100 mL of H<sub>2</sub>O at 90<sup>o</sup>C and allowing the water to cool to below 40<sup>o</sup>C:

'''50 µL Total Reaction'''

*1 µL kpy47 (2 µg)
*1 µL kpy48 (2 µg)
*5 µL 10X NEBuffer 2
*43 µL ddH<sub>2</sub>O

I then generated serial dilutions of 1X, 0.1X, and 0.01X of the resulting reaction and ligated this fragment with the pTol2 L/R, digested with MluI for 2 hours:

'''10 µL Ligation Reaction (30 min)'''

*2 µL pTol2 L/R (MluI)
*4 µL kpy47+kpy48 Insert
*1 µL 10X T4 DNA Ligase Buffer
*2.5 µL ddH<sub>2</sub>O
*0.5 µL T4 DNA Ligase

I sent the two constructs of "corrected pcmv hDα #3 and #4" for sequencing again with the kpy51 reverse sequencing primer (MGH Sequencing Core: #58892).  I also sent "pTol2 L/R" for sequencing with the kpy49 forward sequencing primer and the kpy50 reverse sequencing primer (MGH Sequencing Core: #58905).

Midipreparations of "pcmv fAβ", "corrected pcmv hDα #3", and "pTol2 L/R" were placed into 20 µL reactions containing new 10X NEBuffer 3 as well as into 20 µL enzymatic digestion reactions containing the new 10X NEBuffer 3 and PstI to be incubated for 1 hour.

----

===January 19, 2007===

Out of the 250 mL of "pTol2 L/R" culture started from last night, 10 mL was plucked out for minipreparation.  The remainder of the culture was used for maxipreparation.

I then digested the "pTol2 L/R" minipreps with PciI, MluI, PvuII (NEB3) for 3 hours, leaving a 2.0 kb PciI to MluI vector, and two unnecessary fragments of 0.2 and 0.4 kb.

At the same time, I digested the "pTol2 L/R" minipreps with MluI for 1 hour, added 0.5 μL of Taq Polymerase/2.0 μL (1 mM) dNTPS, incubated for 20 minutes, deactivate MluI for 20 minutes at 65 degrees in the PCR machine, added 0.5 μL PciI, and finally incubuated for 1 hour.  The results in a 0.6 kb MluI/Blunt to PciI fragment and an unnecessary 2.0 kb fragment.

Following gel purification, I ligated/transformed "pTol2 L/R" (PciI-MluI 2.0kb) 3 μL, "pTol2 L/R" (MluI/Blunt-PciI 0.6kb) 3 μL, and the vectors (pcmv fAβ, corrected pcmv hDα #3 and #4, pIRES-R2GR2-pAC) (MluI-BstZ17I/Blunt) 2 μL.

----

===January 18, 2007===

I checked the OD<sub>260/280</sub> of both versions of "corrected pcmv hDα", 3 and 4, (diluted with 200 µL TE, subsequently diluted 5 µL of that with 995 µL of ddH<sub>2</sub>O for OD) midipreparations from yesterday:

'''corrected pcmv hDα #3'''

260 nm Absorbance: 0.100 AU<br/>
280 nm Absorbance: 0.065 AU<br/>
A260/A280: 1.5358

[DNA] = 1.00 µg/µL

'''corrected pcmv hDα #4'''

260 nm Absorbance: 0.099 AU<br/>
280 nm Absorbance: 0.060 AU<br/>
A260/A280: 1.6329

[DNA] = 0.99 µg/µL

Both A260/A280 ratios were below 1.8, suggesting that the plasmid sample contains some protein contamination.

=====The Secondary Structure of MEF2 Alpha Domain pre-mRNA=====

[http://www.bioinfo.rpi.edu/applications/mfold/rna/form1.cgi mfold Server].

====="Tol2 pcmv PIRES-R2GR2-pAC" Construction=====

Digestion of "pcmv Faβ", "corrected pcmv hDα", and pIRES pAC with BstZ17I and MluI (NEB3).

Minipreparation of "pTol2 L/R".

Culture start for midipreparation of "pTol2 L/R".

=====pIRES Vector Information=====

From Clontech:

pIRES is a mammalian expression vector that allows you to express two genes of interest at high levels by cloning them into multiple cloning sites (MCS) A and B. These MCSs are located on either side of the internal ribosome entry site (IRES) from the encephalomyocarditis virus (ECMV), which allows translation of two consecutive open reading frames from the same messenger RNA (1–3). pIRES utilizes a partially disabled IRES sequence (1) which will have the affect of a reduced rate of translation initiation at the second, downstream cloned gene relative to that of the first.

[[Image:pIRES.png|center]]

Go to [http://www.clontech.com/products/detail.asp?product_id=10480&tabno=4 pIRES Vector Information] at [http://www.clontech.com/ Clontech].

=====''Polistotrema/Myxine'' Evolutionary PCR=====

In order to further elaborate on the evolution of the MEF2 gene, more information is required from genomic sequences of ancient fish and their immediate ancestors/descendants.  Unfortunately, sequences of the marine chordates in the class Myxini, known more popularly as hagfish, are incomplete.  We have decided to sequence a specimen ourselves following total RNA isolation, reverse transcription to create cDNA, and the subsequent PCR of the cDNAs.  In order to perform the sequencing, we need primers, in our case degenerate primers.

Following amino acid sequence alignments of the MADS Domain and the end of the MEF2 Domain of the MEF2 genes of ''[http://www.ncbi.nlm.nih.gov/UniGene/UGOrg.cgi?TAXID=7739 Branchiostoma floridae], [http://www.ncbi.nlm.nih.gov/UniGene/UGOrg.cgi?TAXID=7955 Danio rerio], [http://www.ncbi.nlm.nih.gov/UniGene/UGOrg.cgi?TAXID=69293 Gasterosteus aculeatus], [http://www.ncbi.nlm.nih.gov/UniGene/UGOrg.cgi?TAXID=8022 Oncorhynchus mykiss], [http://www.ncbi.nlm.nih.gov/UniGene/UGOrg.cgi?TAXID=8030 Salmo salar], [http://www.ncbi.nlm.nih.gov/UniGene/UGOrg.cgi?TAXID=31033 Takifugu rubripes],'' and'' Tetraodon nigroviridis'', there appears to be a highly conserved region of nucleotides about 50 nucleotides upstream of the 3' terminus of exon 2.

             1                                  34
      FuguA  AGAGTATAAT GAA CCA CAC GAG AGC AGA ACC AAC
     TetraA  AGAGTATAAC GAA CCG CAT GAG AGC AGA ACC AAC
     ZebraA  AGAATACAAT GAA CCC CAC GAG AGC AGA ACC AAC
      FuguB  CGAGTACAGC GAG CCG CAC GAG AGC CGC ACC AAC
     TetraB  CGAGTACAGC GAG CCG CAC GAG AGC CGC ACC AAC
      FuguC  CGAGTACAAC GAG CCC CAC GAG AGC AGG ACC AAC
     ZebraC  AGAGTACAAC GAG CCA CAC GAG AGC AGG ACC AAC
     TetraC  CGAATACAAC GAG CCC CAC GAG AGC AGG ACC AAC
     ZebraB  CGAATACAGT GAA CCT CAC GAG AGC CGC ACT AAC

In this alignment, I judge that the last 24 nucleotides will serve as a great starting point for my reverse degenerate sequencing primer.  With the exception of arginine, most of the amino acid residues contain relatively little degeneracy potential.

''kpy53'': Reverse Sequencing Primer for Myxine:

 kpy53: GTTRGTICKGCTCTCRTGIGGYTC (16 Fold Degeneracy, 2 Inosines)
 
 Self-Dimers:
 
 5' GTTRGTICKGCTCTCRTGIGGYTC              3'
            :   ||||||||   :         
 3'         CTYGGIGTRCTCTCGKCITGRTTG      5'
 
 5'             GTTRGTICKGCTCTCRTGIGGYTC  3'
                   ||||||               
 3' CTYGGIGTRCTCTCGKCITGRTTG              5'
 
 SEQUENCE: 5'- GTT RGT ICK GCT CTC RTG IGG YTC -3'
 COMPLEMENT: 5'- GAR CCI CAY GAG AGC MGI ACY AAC -3'
 LENGTH: 22
 GC CONTENT: 59.1%
 MELT RANGE: MIN: 55.1ºC, MEAN: 59.8ºC, MAX: 64.5ºC
 MOLECULAR WEIGHT: 7372.8 g/mole

Similarly for the MADS Domain,

             1                              30
      FuguA  ATG GGG CGG AAG AAG ATA CAG AT CACCAGG 
     TetraA  ATG GGG CGG AAG AAG ATA CAG AT CACCAGG 
     ZebraA  ATG GGA CGG AAG AAG ATA CAG AT CACGCGG 
      FuguC  ATG GGG AGG AAA AAG ATT CAG AT CACGCGG 
     TetraC  ATG GGG AGG AAA AAG ATT CAG AT CACGAGG 
     ZebraC  ATG GGG AGA AAA AAG ATT CAG AT CACACGG 
     ZebraB  ATG GGG AGG AAG AAA ATT CAG AT CTCTCGG 
      FuguB  ATG GGA AGA AAG AAA ATA CAG AT CTCTCGT 
     TetraB  ATG GGA AGA AAG AAA ATA CAA AT TTCTCGT

Unfortunately, this region contains an arginine and an isoleucine, and any region downstream of this area contains too many six fold degeneracy residues such as leucine, serine, or arginine; therefore, the first 30 nucleotides is most likely the best candidate.

''kpy52'': Forward Sequencing Primer for Myxine:

 kpy52: ATGGGRMGIAARAARATHCARAT (96 Fold Degeneracy, 1 Inosines)
 
 Self-Dimers:
 
 5'          ATGGGRMGIAARAARATHCARAT      3'
              :   ||||   :          
 3' TARACHTARAARAAIGMRGGGTA               5'
 
 5'  ATGGGRMGIAARAARATHCARAT              3'
      : |||  :    :  ::: :  
 3' TARACHTARAARAAIGMRGGGTA               5'
 
 SEQUENCE: 5'- ATG GGR MGI AAR AAR ATH CAR AT -3'
 COMPLEMENT: 5'- ATY TGD ATY TTY TTC CKY CCC AT -3'
 LENGTH: 23
 GC CONTENT: 35.6%
 MELT RANGE: MIN: 46.7ºC, MEAN: 51.9ºC, MAX: 58.2ºC
 MOLECULAR WEIGHT: 7164.7 g/mole

Total RNA isolation of the hagfish will begin immediately after its arrival.

----

===January 17, 2007===

I checked the OD<sub>260/280</sub> of the "pTsT3 Tol2" (diluted with 200 µL TE, subsequently diluted 5 µL of that with 995 µL of ddH<sub>2</sub>O for OD) midipreparations from last Friday again:

'''pTsT3 Tol2'''

260 nm Absorbance: 0.039 AU<br/>
280 nm Absorbance: 0.016 AU<br/>
A260/A280: 2.4646

[DNA] = 0.39 µg/µL

I also checked the OD<sub>260/280</sub> of the "pcmv Faβ R2G IRES pAC" (diluted with 200 µL TE, subsequently diluted 5 µL of that with 995 µL of ddH<sub>2</sub>O for OD) midipreparations from yesterday:

'''pcmv Faβ R2G IRES pAC'''

260 nm Absorbance: 0.077 AU<br/>
280 nm Absorbance: 0.038 AU<br/>
A260/A280: 2.0141

[DNA] = 0.77 µg/µL

Although the yield was much lower for the midipreparation kit with respect to the maxipreparations, both A260/A280 ratios were greater than 2.0, suggesting that the DNA solution is very pure with regard to protein contaminaton.

I went ahead and performed midipreparations on both the corrected versions of "pcmv Faβ" cultures in advance of new sequencing with the kpy51 primer.  Both QIAfilter were flowing normally; I redissolved the DNA pellet with 200 µL TE buffer after a couple of hours of air-drying following the last wash with 70% ethanol.  I will check the optical density of both versions of the plasmid tomorrow morning.

=====''Branchiostoma'' Alignment=====

Due to issues concerning the reliability of the relatively large alignment of the entire MEF2 genes of ''Branchiostoma floridae'' and ''Homo sapiens'', I decided to manually align the sequences.  The first 100 amino acid residues were highly conserved and therefore were trivially aligned using PileUp.  A second PileUp operation is performed with the Human genes only to generate a base alignment.  Issues regarding gap alignments caused by the larger ''Branchiostoma'' sequence were addressed by running a [http://www.ncbi.nlm.nih.gov/BLAST/Blast.cgi?CMD=Web&LAYOUT=TwoWindows&AUTO_FORMAT=Semiauto&ALIGNMENTS=50&ALIGNMENT_VIEW=Pairwise&CLIENT=web&DATABASE=nr&DESCRIPTIONS=100&ENTREZ_QUERY=%28none%29&EXPECT=10&FILTER=L&FORMAT_OBJECT=Alignment&FORMAT_TYPE=HTML&GENETIC_CODE=0&NCBI_GI=on&PAGE=Translations&PROGRAM=tblastn&SERVICE=plain&SET_DEFAULTS.x=23&SET_DEFAULTS.y=10&SHOW_OVERVIEW=on&UNGAPPED_ALIGNMENT=no&END_OF_HTTPGET=Yes&SHOW_LINKOUT=yes&GET_SEQUENCE=yes&SEARCH_NAME=tbn TBLASTN] Protein Query vs. Translated Database search with the entire MEF2 gene of ''Branchiostoma'' in the Expressed Sequence Tags (Other) Database, Whole Genome Shotgun Database, and High Throughput Genomic Sequences Database.  Database hits are recoded in; subsequently, PileUp is performed on the translated sequences of the Human genes and the focused section of the ''Branchiostoma'' sequence.  The new alignment is then used as a guide to edit to master alignment with the pre-aligned Human MEF2 genes.  Sequences that did not register a database hit on TBLASTN were partitioned into 50 amino acid sequences and serve as input into PileUp + Human MEF2 genes.  Further editing of the master alignment then proceeds with the new alignment.

[http://www.ncbi.nlm.nih.gov/BLAST/Blast.cgi?CMD=Web&LAYOUT=TwoWindows&AUTO_FORMAT=Semiauto&ALIGNMENTS=50&ALIGNMENT_VIEW=Pairwise&CLIENT=web&DATABASE=nr&DESCRIPTIONS=100&ENTREZ_QUERY=%28none%29&EXPECT=10&FILTER=L&FORMAT_OBJECT=Alignment&FORMAT_TYPE=HTML&GENETIC_CODE=0&NCBI_GI=on&PAGE=Translations&PROGRAM=tblastn&SERVICE=plain&SET_DEFAULTS.x=23&SET_DEFAULTS.y=10&SHOW_OVERVIEW=on&UNGAPPED_ALIGNMENT=no&END_OF_HTTPGET=Yes&SHOW_LINKOUT=yes&GET_SEQUENCE=yes&SEARCH_NAME=tbn TBLASTN] Alignment Hit Demonstrating Limits of MEF2 Exons In ''Branchiostoma'':

 >gi|30580756|gb|AC099301.6| Download subject sequence spanning the HSP 
 Rattus norvegicus clone CH230-20O17, WORKING DRAFT SEQUENCE 
 6 unordered pieces
 Length=263584
 
 Score = 63.5 bits (153),  Expect = 2e-07
 Identities = 50/119 (42%), Positives = 67/119 (56%), Gaps = 9/119 (7%)
 Frame = -1
 
 Query  449    GIPSGGPSTTPATLSQGQSLTVQTNQGPMQIKSEPVSPPQDR-HTPAVNNQGGLHANNTS  507
               G  S   + T   LSQ  +L++ + Q  + IKSEPVSPP+DR  TP+   Q   H    S
 Sbjct  58189  GFLSSYRACTSTHLSQSSNLSLPSTQS-LNIKSEPVSPPRDRTTTPSRYPQHTRHEAGRS   58013
 
 Query  508    PVNSLCssssssPFPADGEDRGD--GSFHSPGGSRMRKAMEDGDNPAAKRPRLNDNWGT  564
               PV+SL S SSS     DG DR D    FHSP G   R + ++ ++P+ KR RL++ W T
 Sbjct  58012  PVDSLSSCSSSY----DGSDREDHRNEFHSPIG-LTRPSPDERESPSVKRMRLSEGWAT  57851

----

===January 16, 2007===

I checked the OD<sub>260/280</sub> of the "pTsT3 Tol2" (diluted with 200 µL TE, subsequently diluted 5 µL of that with 995 µL of ddH<sub>2</sub>O for OD) midipreparations from last Friday:

'''pTsT3 Tol2'''

260 nm Absorbance: 0.016 AU<br/>
280 nm Absorbance: -0.004 AU<br/>
A260/A280: -3.5529

[DNA] = 0.16 µg/µL

Since this appears really low, I will rechecked again, tomorrow morning along with the "pcmv Faβ R2G IRES pAC" midipreparation.

I performed another alignment of the MADS+MEF2 Region, MADS+MEF2+Alpha2 Region, and Whole Protein of the Amphioxus (''Branchiostoma floridae'') MEF2 gene, Human MEF2A, MEF2B, MEF2C, and MEF2D.  The ''Danio rerio'' sequences for MEF2D that I have are not correct, and may have driven previous alignments incorrectly.

An alignment of the MADS and MEF2 domains of the proteins yield a phylogeny tree of which Human A and Human C are monophyletic and Human D and Human B are monophyletic with the Amphioxus as the root.  Subsequent alignments of the MADS/MEF2 Domains and Alpha 2 Exons of the proteins also yield the same phylogenetic relationships between the different forms of MEF2 for humans.  The alignment for the entire protein of MEF2 (Exons 1 to 9) again yields the same relationship.

A midipreparation of "pcmv Faβ R2G IRES pAC" was performed today according to Qiagen protocol.  The air-dried DNA was redissolved with 200 µL of TE buffer.  I performed filtration of the supernatant soon after I equilibriated the QIAfilter.  Subsequent flow through the QIAfilter was extremely slow.

Sequences of CPCM3 & CPCM4 were checked.  It appears that the KPY8 primer used for sequencing was too far downstream for conclusions to be drawn from the alpha region to exon 4 of the corrected minigene.  I designed a new reverse sequencing primer today in the DSRED2 sequence, about 120 nucleotides upstream of the methionine start codon as TG1040 is too far upstream of the region (in alpha 2) for sequencing into exon 4:

kpy51: Reverse Sequencing Primer for DSRED2:

 kpy51: CACCTTCAGCTTCACGGTGTTG
 
 Self-Dimers:
 
 5'    CACCTTCAGCTTCACGGTGTTG   3'
       |||| : :   : : ::::   
 3' GTTGTGGCACTTCGACTTCCAC      5'
  
 5'     CACCTTCAGCTTCACGGTGTTG  3'
          : :  ||||  : :      
 3' GTTGTGGCACTTCGACTTCCAC      5'
 
 SEQUENCE: 5'- CAC CTT CAG CTT CAC GGT GTT G -3'
 COMPLEMENT: 5'- CAA CAC CGT GAA GCT GAA GGT G -3'
 LENGTH: 22
 GC CONTENT: 54.5%
 MELT TEMP: 58.6ºC
 MOLECULAR WEIGHT: 6677.4 g/mole

====The Bootstrapping Process====

Bootstrapping is a method of estimating confidence levels of inferred relationships.

*1. Align the given sequences using PileUp, a multiple sequence alignment program that utilizes a progressive, pairwise alignment method:
 
 !!AA_MULTIPLE_ALIGNMENT 1.0
 PileUp of: *.pep
 
 Symbol comparison table: GenRunData:blosum62.cmp  CompCheck: 1102
 
                   GapWeight: 8
             GapLengthWeight: 2 
 
 pileup.msf  MSF: 529  Type: P  January 16, 2007 15:39  Check: 970 ..
 
 Name: FuguC            Len:   529  Check: 4363  Weight:  1.00
 Name: TetraC           Len:   529  Check: 3118  Weight:  1.00
 Name: ZebraC           Len:   529  Check: 3684  Weight:  1.00
 Name: HumanC           Len:   529  Check: 2247  Weight:  1.00
 Name: FuguA            Len:   529  Check: 6929  Weight:  1.00
 Name: TetraA           Len:   529  Check: 4354  Weight:  1.00
 Name: ZebraA           Len:   529  Check: 3876  Weight:  1.00
 Name: HumanA           Len:   529  Check: 2399  Weight:  1.00
 
 //
 
        1                                                   50
  FuguC  MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 TetraC  MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 ZebraC  MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 HumanC  MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
  FuguA  MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 TetraA  MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 ZebraA  MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 HumanA  MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS

*2. Re-align the PileUp output using [http://www.ebi.ac.uk/clustalw/ ClustalW], output PHYLIP format, to generate a Clustal Alignment .aln file.  This file, renamed as 'infile', will become the input for the SEQBOOT bootstrapping program.

     8    543
  FuguC     MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 TetraC     MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 ZebraC     MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 HumanC     MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
  FuguA     MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 TetraA     MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 ZebraA     MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS 
 HumanA     MGRKKIQITR IMDERNRQVT FTKRKFGLMK KAYELSVLCD CEIALIIFNS

*3. SEQBOOT resamples the input data set to create multiple bootstrapped or jackknifed data sets.  Bootstrapping involves sampling and replacing N characters randomly; these new data sets can then be analyzed to estimate statistically the sampling distribution of data.  SEQBOOT results are placed in a file named 'outfile'.  [http://evolution.genetics.washington.edu/phylip.html PHYLIP Package.] [http://artedi.ebc.uu.se/programs/seqboot.html SEQBOOT Web Server].

    8   543
 FuguC      MMMKKKQTRR RERRRRRQQQ VVTFTKRKKK GLLLKAYEEE LLVLLCDIAA IIIINSLYYK
 TetraC     MMMKKKQTRR RERRRRRQQQ VVTFTKRKKK GLLLKAYEEE LLVLLCDIAA IIIINSLYYK
 ZebraC     MMMKKKQTRR RERRRRRQQQ VVTFTKRKKK GLLLKAYEEE LLVLLCDIAA IIIINSLYYK
 HumanC     MMMKKKQTRR RERRRRRQQQ VVTFTKRKKK GLLLKAYEEE LLVLLCDIAA IIIINSLYYK
 FuguA      MMMKKKQTRR RERRRRRQQQ VVTFTKRKKK GLLLKAYEEE LLVLLCDIAA IIIINSLYYK
 TetraA     MMMKKKQTRR RERRRRRQQQ VVTFTKRKKK GLLLKAYEEE LLVLLCDIAA IIIINSLYYK
 ZebraA     MMMKKKQTRR RERRRRRQQQ VVTFTKRKKK GLLLKAYEEE LLVLLCDIAA IIIINSLYYK
 HumanA     MMMKKKQTRR RERRRRRQQQ VVTFTKRKKK GLLLKAYEEE LLVLLCDIAA IIIINSLYYK

*4. To find the phylogeny estimate for these new data sets, rename the 'outfile' to 'infile', and run PROTPARS (PROTein PARSimony) for protein sequences or similarly DNAPARS (DNA PARSimony) for nucleotide sequences, resetting its multiple data set option.  PROTPARS infers an unrooted phylogeny from these data sets using a method similar to counting the number of changes required to evolve from one protein to another in different phylogenies and counting the minimum number of substitutions required to recreate a given phylogeny.  Two files, the 'outfile' file and the 'outtree' file which contains all of the trees generated from the data sets. [http://bioweb.pasteur.fr/seqanal/interfaces/protpars.html PROTPARS Web Server.] [http://artedi.ebc.uu.se/programs/protpars.html PROTPARS Mirror.]

 Protein parsimony algorithm, version 3.66
 Data set # 1:
 One most parsimonious tree found:
 
              +--------HumanA    
           +--7  
           !  !  +-----ZebraA    
           !  +--6  
        +--4     !  +--TetraA    
        !  !     +--5  
        !  !        +--FuguA     
     +--3  !  
     !  !  +-----------HumanC    
  +--2  !  
  !  !  +--------------ZebraC    
  1  !  
  !  +-----------------TetraC    
  !  
  +--------------------FuguC     
 
  remember: this is an unrooted tree!
 requires a total of    823.000

*5. Finally, the 'outfile' file needs to be renamed to 'infile', the input file for CONSENSE, the program that outputs the majority rule consensus tree.  [http://artedi.ebc.uu.se/programs/consense.html CONSENSE Web Server.]

 Consensus tree program, version 3.66
 Species in order: 
 
  1. HumanA
  2. ZebraA
  3. TetraA
  4. FuguA
  5. HumanC
  6. ZebraC
  7. TetraC
  8. FuguC
 
 Sets included in the consensus tree
 
 Set (species in order)     How many times out of  100.00
 
 ....****                   100.00
 ..**....                   100.00
 .....***                   100.00
 ......**                   100.00
 .***....                   99.00
 
 Sets NOT included in consensus tree:
 
 Set (species in order)     How many times out of  100.00
 
 .*..****                    1.00
 
 Extended majority rule consensus tree
 
 CONSENSUS TREE:
 the numbers on the branches indicate the number
 of times the partition of the species into the two sets
 which are separated by that branch occurred
 among the trees, out of 100.00 trees
 
                              +------FuguC
                       +100.0-|
                +100.0-|      +------TetraC
                |      |
         +100.0-|      +-------------ZebraC
         |      |
  +------|      +--------------------HumanC
  |      |
  |      |             +-------------ZebraA
  |      +--------99.0-|
  |                    |      +------TetraA
  |                    +100.0-|
  |                           +------FuguA
  |
  +----------------------------------HumanA

*6. The 'outtree' file can be further manipulated (such as rooting) using [http://taxonomy.zoology.gla.ac.uk/rod/treeview.html TREEVIEW].

----

===January 15, 2007===

Martin Luther King, Jr. Day.

----

===January 12, 2007===

The "pcmv Faβ R2G IRES pAC" construct is being grown up in 2 mL of LB + Ampicillin in the incubator, to be added to a much larger 250 mL flask of LB + Ampicillin for overnight growth in the shaking incubator.  A 100 mL sample of this culture will be centrifuged for the midipreparation process on Monday, January 15.

The "pTsT3 Tol2" construct cultured in a 100 mL flask on Wednesday night was midiprepped today following Qiagen midipreparation protocol.  The DNA will be eluted with 200 μL of TE buffer to be quantified on Monday.

The "corrected pcmv hDα" constructs #3 and #4 cultures (submitted for sequencing on Wednesday morning, transformed for miniprepreparation overnight Thursday) are growing in the incubator to be miniprepped following confirmation of sequences.

----

===January 11, 2007===

The 3.0kb size of the Tol2 miniprepped plasmid was confirmed with a digestion of 6 μL of Tol2 miniprepped plasmid and 0.5 μL of HindIII, NEB4 in a 20 μL reaction.

The #3 and #4 minipreps of the "corrected pcmv hDα" constructs were both transformed again using 0.5 μL of each plasmid in MC1061 cells.

Meanwhile, a variety of MEF2A promoters were aligned using PILEUP.  The alignment was fed into CLUSTALW to create a Phylip phylogenetic tree (edited with TreeView).

----

===January 10, 2007===

Minipreparations of the "corrected pcmv hDα" transformations were made, of which, 6 μL sample of each plasmid were digested with EcoRI and BamHI, NEB4 in a 20 μL reaction.  Only the 3rd and 4th minipreparations contained the correct 2.0kb size band.  

The #3 and #4 "corrected pcmv hDα" constructs were then sent for sequencing with reverse primer KPY8 (Tube IDs: CPCMV3, CPCMV4, MGH Sequencing Core: #58529):

'''20 µL Final Volume'''

*8 µL "corrected pcmv hDα" Construct Minipreparation n
*4 µL KPY8 Reverse Primer (200 ng, 10 ng/µL)
*8 µL ddH<sub>2</sub>O

Lastly, 100 mL cultures to be used for midipreparation of the "pcmv Faβ R2G IRES pAC" and "pTsT3 Tol2" constructs were started.

----

===November 29, 2006===

A western blot of the R2GR2, R2GG, GR2G, and GR2R2 constructs was started.

----

===November 15, 2006===

====Western Blotting (11/11/06 Transfections) Part II====

*Rabbit GFP Antibody (Primary Antibody)
*Anti-Goat GFP Antibody (Secondary Antibody)
*Powdered Milk
*Tris-Buffered Saline
*Tween 20

After agitation with the primary antibody (rabbit GFP antibody) overnight (until 3:30 PM) at 4<sup>o</sup>C, the membrane was washed with approximately 20 to 25 mL of a solution containing 250 mL of Tris-Buffered Saline and 500 µL of Tween 20 and agitated more vigorously than the blocking agitation for 5 minutes.  I repeated this for two more times.

At the same time, I created a 10% solution consisting of 10 mL TBS/Tween 20 and 1 g of powdered milk; add 0.6 µL of the secondary anti-goat GFP antibody to the 10% milk solution to create a 1:15000 dilution.

Following the rinsing of the nitrocellulose membrane to remove any unbound primary rabbit GFP antibody, the membrane is then exposed to the secondary antibody by the addition of the entire 10% solution; the membrane and solution is shaken for 1 hour at room temperature.

Again, the membrane is washed three times with approximately 20 to 25 mL of the TBS and Tween solution to remove any unbound secondary antibody.

An additional wash consisting of just 20 to 25 mL of TBS is performed for 5 minutes.

Following the blotting of the membrane to remove excess solution, a total of 1 mL of chemiluminiscent detection solution is spread onto the membrane.  The membrane is transfered to the photoplate, where x-ray photopaper is exposed to the membrane for 5 minutes, 1 minute, etc. in the dark room and developed:

[[Image:KPY-Western-11-15-06-GFP.jpg|center|300px|frame|Western Blot]]

The membrane is then frozen at -4<sup>o</sup>C for future probing with the RFP antibody following another round of blocking of non-specific binding of protein between the membrane and the antibodies.

The western blot reveals that pcDNA R2GG-transfected cells produced the correct protein size (~60 kDa) of the RFP-GFP fusion.

====Polymerase Chain Reaction of cDNA (CDM-hDalpha GFP) Round II====

The polymerase chain reaction of the cDNA of CDM-hDalpha GFP-transfected cells' RNA is performed again with a very low annealing temperature of 50<sup>o</sup>C.  Three pairs of primers will be used: TG1037 to KPY8, covering from exon 2 to 3 to 4 to GFP; TG1037 to KPY27, covering exon 2 to exon 3alpha1; and KPY26 to KPY8, covering exon 3alpha2 to 4 to GFP.  Both the CDM-hDalpha GFP cDNA and RT control will be used as template:

1. Creation of master solution:

*30 µL MgSO<sub>4</sub>
*30 µL 10X HIFI Buffer
*30 µL dNTPs
*166 µL Nuclease-Free Water

2. Total solution is equal to 256 µL, divided into two Eppendorfs of 128 µL each, in which 5 µL of the +RT mix is added to one and 5 µL of the -RT mix is added to the other one for a total of 133 µL of solution each.

3. The two Eppendorfs are then divided into three Eppendorfs of 44 µL solution.

4. The three pairs of primers TG1037 (4 µL, 200 ng)/KPY8 (2 µL, 200 ng), TG1037 (4 µL, 200 ng)/KPY27 (2 µL, 200 ng), and KPY26 (2 µL, 200 ng)/KPY8 (2 µL, 200 ng)/ddH<sub>2</sub>O (2 µL) are added to their respective reaction tubes, for a total of 50 µL of solution in each.

5. 0.4 µL of Taq Polymerase is added to each of the reactions.

The PCR is performed with the first heating window of 95<sup>o</sup>C for 3 minutes.

The second window consists of a 45 second heating period at 95<sup>o</sup>C, a 45 second annealing period at 50<sup>o</sup>C, and a 60 second extension period at 72<sup>o</sup>C, for 35 cycles.

The gel electrophoresis (5 µL of each PCR reaction) revealed no amplification.

TG will perform the PCR again with actin primers as a positive control for the reverse transcription.

----

===November 14, 2006===

====Reverse Transcription-Polymerase Chain Reaction (CDM-hDalpha GFP)====

I checked the OD<sub>260/280</sub> of the RNA [[Kai_Yuet/Protocols:RNA Isolation | isolated ]] from the CDM-hDalpha GFP transfection (diluted with 200 µL TE, subsequently diluted 4 µL of that with 996 µL of ddH<sub>2</sub>O for OD) from yesterday:

'''CDM-hDalpha GFP'''

260 nm Absorbance: 0.222 AU<br/>
280 nm Absorbance: 0.132 AU<br/>
A260/A280: 1.6835

[RNA] = 2.22 µg/µL

The RNA that I will be using for reverse transcription is in an Eppendorf: 15 µL aliquot of isolated RNA from the -80<sup>o</sup>C freezer.

'''First Strand Synthesis of cDNA'''

* 1 µL RNA (2.22 µg)
* 1.5 µL Oligo dT (500 ng)
* 12.5 µL Nuclease-Free Water

Total reaction of 15 µL.

Incubate the tube to 75<sup>o</sup>C to 70<sup>o</sup>C for 5 minutes in heated water.

Immediately cool the tube on ice, and then spin briefly.

Add the following components to the tube in this specific order:

* 2.5 µL M-MLV 10X Reaction Buffer
* 5 µL 10 mM dNTPs
* 1 µL Recombinant RNasin Ribonuclease Inhibitor
* 1 µL M-MLV RT

Mix and incubate tube at 37<sup>o</sup>C for 60 minutes.

Incubate the tube at 90<sup>o</sup>C for 3-4 minutes to denature reverse transcriptase.

Quick spin, and ice until use.

Meanwhile, an identical tube of the same content with the exception of reverse transcriptase is incubated at the same temperature for only 30 minutes.  This control tube will be used to evaluate the presence of DNA.

'''Polymerase Chain Reaction'''

*10 µL MgCl2
*10 µg 10X Buffer
*10 µL dNTPs
*4 µL KPY8 Primer (400 ng)
*8 µL TG1037 Primer (400 ng)
*48 µL Nuclease Free H2O

Total reaction of 90 µL.

Split into two Eppendorfs of 45 µL.

*4 µL TEMPLATE (+/- RT)
*1 µL Taq Polymerase

Split each into 25 µL PCR tubes (x4 tubes).

The PCR is performed with the first heating window of 95<sup>o</sup>C for 3 minutes.

The second window consists of a 60 second heating period at 95<sup>o</sup>C, a 60 second annealing period at 58 and 62<sup>o</sup>C, and a two minute extension period at 72<sup>o</sup>C, for 35 cycles.

The gel electrophoresis (5 µL of each PCR reaction) revealed no amplification.

====Western Blotting (11/11/06 Transfections)====

Creation of stacking gel...

Creation of running buffer, 350 mL

Heat loading dye + cells up to boiling

Add 12 µL of the loading dye/cells mixture to each lane of the gel.

Run the gel with 80V until the loading dye/cells mixture runs to the end of the stacking gel, at which point increase the voltage to 130V.

Creation of transfer buffer (Tris, glycine, methanol) 1L...

Prepare membrane in methanol...

Prepare gel in transfer buffer...

Stack sponges, filter, gel (in right orientation),membrane, filter, sponges...

Fill apparatus with transfer buffer, run at 100V in cold room with spinner/block of ice for 1 hr.

Meanwhile, prepare blocking buffer with milk and TBS, TWEEN 20...

Aliquot 5 mL blocking buffer for antibody...

Soak membrane in blocking buffer and shake for 1 hour at room temp.

Shortly before draining of blocking buffer, create 1:5000 dilution with the 5 mL of blocking buffer and GFP antibody (1 uL in 5 mL of blocking buffer)

Drain blocking buffer, and replace with blocking/GFP antibody solution, shake overnight at 4<sup>o</sup>C.

----

===November 13, 2006===

====SDS-PAGE of 11/08/06 Transfections====

The 11/09/06 transfections contain cells that did not grow that well due to the use of new FBS which leads to changes in cell shape, growth rate, and adhesion.  TG retransfected the cells on Saturday night using higher density cells.  

TG says of the transfections on Sunday, November 12, 2006:

<blockquote><i>The R2G constructs appear to give a better matched red and green fl signal than the GR2 constructs, so we will probably use the R2G going forward (as long as the western gives good news re: stability of the fusion).</i></blockquote>

I scoped the cells today at 3PM and checked for fluorescence quality:

{| border="1" cellpadding="5" cellspacing="0" align="center"
|+ 6-Well Plate Transfections (Fluorescence)
! Construct !! Column One !! Column Two 
|-
! pcDNA Control (1), -- (2)
| -- || --
|-
! pcDNA R2GG (1), R2GR2 (2)
| G (Nuclear) || R+
|-
! pcDNA GR2R2 (1), GR2G (2)
| R (Nuclear), G- || G+
|-
|}

Both the pcDNA R2GR2 and GR2G constructs correctly expressed fluorescence; their fluorescence is spread thoroughly throughout the cell as one would expect for a normal transfection with just pcDNA EGFP.  The pcDNA R2GG and GR2R2 constructs, however, have compact fluorescence centered within the nucleus.  In addition, GR2R2 also contains a hint of green fluroscence in some of the transfected cells.

TG Replied:

<blockquote><i>This sounds like what we have seen before.  I think that the nuclear localization of the fusions relates to the very high content of basic residues, including some strings of Arg.  This could be re-engineered eventually to neutralize the fusions by making mutations, but we should get proof of principle first.  The GRR dual fluorescence should go away if there is a stop codon upstream of construct that is in frame with the GFP coding region - as we designed in the the minigene constructs.</i></blockquote>

I then aspirated the medium from the 4 fluorescent protein fusion construct wells in addition to the control well and replaced it with 1 mL of PBS.  Shortly after, I pipetted the solution back and forth to loosen the 293 cells, which was subsequently iced in an Eppendorf.  The tubes were then centrifuged at 1.5 x g for 6 minutes.  In the meantime, I prepared 1X loading dye (containing SDS, Tris, and DTT) from a 6X stock and heated it into cooler than boiling water.  The PBS was carefully aspirated, leaving behind the pellet of protein.  500 uL was added to each tube.  To reduce viscosity caused by the presence of DNA, a 25 gauge needle was used to aspirate in and out the solution in order to shear the DNA.  The solution is then stored at -80<sup>o</sup>C.

To prepare the stacking gel (12% acrylamide for 15 mL), 5 mL of ddH20, 6 mL of acrylamide (29:1 normal to bis)...

The clean glass templates are then set, at which point water is poured in to ensure there are no leaks.  The stacking gel is then poured ~7 mL so that 5 mm of clearance is between the surface of the gel and the combs used to create the wells.  300 uL of running buffer is used to ensure that the surface does not dry.  The gels can then be stored in template/apparatus at -4<sup>o</sup>C until the next day.

====RNA Isolation: RNA Phase Separation, Precipitation & Wash====

The TRIZOL solution is pulled from the -80<sup>o</sup>C freezer and thawed.  Followed [[Kai Yuet/Protocols:RNA Isolation | RNA Isolation ]] protocol.

----

===November 9, 2006===

I performed [[Kai_Yuet/Protocols:RNA Isolation | RNA Isolation]] of CDM-hDalpha GFP plate with 1 mL of TRIZOL (Immediately storing the sample following homogenization at -80<sup>o</sup>C).

I also checked the fluorescence of the 6-well plate transfection of pcDNA GR2R2 (Maxi), R2GG (Maxi), GR2G (Maxi), and R2GR2 (Maxi) transfected yesterday:

{| border="1" cellpadding="5" cellspacing="0" align="center"
|+ 6-Well Plate Transfections (Fluorescence)
! Construct !! Column One !! Column Two 
|-
! pcDNA Control (1), -- (2)
| -- || --
|-
! pcDNA GR2R2 (1), R2GG (2)
| G, R || G
|-
! pcDNA GR2G (1), R2GR2 (2)
| G+ || G, R
|-
|}

In all of the constructs designed to fluoresce the color red, I am observing green interspersed evenly with red fluorescence.  I ensured yesterday that there is a lack of communication between wells by using new pipettes for operating in each well.  Also, I am observing these red constructs using the same microscope designed only to observe green fluorescence, so the green fluorescence could be attributed to using this microscope versus one that is designed with red filters in order to truly distinguish between red and green fluorescence.

----

===November 8, 2006===

The 11/07/06 three fragment ligation of pIRES Vector (XbaI-PstI) + pAC (AgeI-XbaI) + IRES (NsiI-AgeI) was not successful.  TG will run a second PCR to amplify the IRES and pAC fragments again.  I [[Kai_Yuet/Protocols:Transformation | retransformed]] the "C" pAC construct using 1 µL of the "C" minipreparation 1 in MC1061 cells.  The ampicillin plate is labeled "TG 'C' pAC Construct: 11/08/06".

A 6-well plate of 293 cells was transfected today (3:30 PM) with pcDNA GR2R2, R2GG, GR2G, R2GR2, and pcDNA control:

{| border="1" cellpadding="5" cellspacing="0" align="center"
|+ 6-Well Plate Transfections
! Column One !! Column Two
|-
| pcDNA || --
|-
| GR2R2 || R2GG
|-
| GR2G || R2GR2
|-
|}


'''6-well Plate Transfection (Per Well)'''

*x µL Maxiprepped Plasmid (2 µg)
*100-x µL 1X DMEM (No Serum/Antibiotic)
*10 µL SuperFect Reagent [10 minute incubation at room temperature]
*600 µL 1X DMEM (10%)
*3 mL 1X DMEM (10%) (Addition of medium, 2 hours later)

No medium change today until tomorrow due to sensitivity of cell adherence to well following transfections.

----

===November 7, 2006===

I [[Kai_Yuet/Protocols:Plasmid Minipreparation | miniprepped]] the two cultures of TG's C construct and digested 6 µL of each with AgeI and XbaI:

'''20 µL Total Reaction'''

*6 µL Miniprepped Plasmid
*2 µL NEBuffer 4
*11 µL ddH<sub>2</sub>O
*0.5 µL AgeI
*0.5 µL XbaI

And found that both minipreparations contain the desired 0.6 kb fragment containing the pAC insert.

The minipreparations are in two Eppendorfs labeled "TG-C-pAC-11/07/06."

The TG "C" construct was then sent for sequencing with forward primer T7 (Tube ID: TGCT7, MGH Sequencing Core: #56539):

'''20 µL Final Volume'''

*8 µL TG "C" Construct Minipreparation 1
*4 µL T7 Primer (200 ng, 10 ng/µL)
*8 µL ddH<sub>2</sub>O

From the digestion, I isolated the 0.6 kb AgeI-XbaI band (digestion of 6 µL out of 30 µL of miniprepped TG "C" construct) and performed a three fragment ligation with TG's vector (XbaI-PstI) and the 0.6 kb NsiI-AgeI fragment containing the IRES:

'''10 µL Ligation Reaction'''

*2 µL Purified, Linearized Vector (XbaI-PstI)
*3.25 µL Purified, Linearized pAC Insert (AgeI-XbaI)
*3.25 µL Purified, Linearized IRES Insert (NsiI-AgeI)
*1 µL 10X T4 DNA Ligase Buffer
*0.5 µL T4 DNA Ligase

#Incubated ligation reaction and control at room temperature for 40 minutes.
#Added 2.5 µL out of 10 µL of ligation reaction/control to appropriate MC1061 cell-containing tubes.
#Followed the [[Kai_Yuet/Protocols:Transformation | Bacterial Transformation]] protocol from this point forward.

The ampicillin plates used for this transformation are labeled:

*"R/O: pAC+IRES+pIRES, 11/07/06"
*"CONTROL: pIRES, 11/07/06"

TG will start cultures from transformations tomorrow morning if appropriate.

----

===November 6, 2006===

I checked the OD<sub>260/280</sub> of the pcDNA GR2G and pcDNA R2GR2 (diluted with 200 µL TE, subsequently diluted 5 µL of that with 995 µL of ddH<sub>2</sub>O for OD) [[Kai_Yuet/Protocols:Plasmid Maxipreparation | maxipreparations]] from last Wednesday:

'''pcDNA GR2G'''

260 nm Absorbance: 0.274 AU<br/>
280 nm Absorbance: 0.155 AU<br/>
A260/A280: 1.7675

[DNA] = 2.74 µg/µL

'''pcDNA R2GR2'''

260 nm Absorbance: 0.242 AU<br/>
280 nm Absorbance: 0.138 AU<br/>
A260/A280: 1.7605

[DNA] = 2.42 µg/µL

I will transfect pcDNA GR2G, R2GR2, GR2R2, R2GG, and control (2 µg) into a 6-well plate with newly split 293 cells (TG) and verify with SDS-PAGE.

BR has created cultures of TG's construct A (Vector + pAC + IRES) (x6) and C (pAC + IRES) (x2).  I [[Kai_Yuet/Protocols:Plasmid Minipreparation | miniprepped]] the cultures this morning and checked for a 1.0 kb AgeI to AgeI fragment containing the internal ribosome entry site with a AgeI (NEBuffer 1) digestion (1 hr).

None of the digestions of TG's construct A (6 samples) revealed a 1.0 kb fragment.

Also, I transfected (1:25 PM) 293 cells in a 100 mm dish with 5.6 µg of Brad's CDM-hDalphaGFP construct:

*2 µL CDM-hDalpha GFP (5.6 µg)
*98 µL DMEM (No Serum/Antibiotics)
*33 µL SuperFect Reagent

Incubate for 8 minutes at room temperature.

*Added 3000 µL DMEM (10%)
*Immediate transfer to 100 mm dish.

Remove media and replace with fresh 10% DMEM (10 mL) after 2 hours.

I also checked the fluorescence of the 12-well plate transfection of pcDNA GR2R2 (Maxi), R2GG (Maxi), GR2G (Minis x 3), and R2GR2 (Minis x 3):

{| border="1" cellpadding="5" cellspacing="0" align="center"
|+ 12-Well Plate Transfections (Fluorescence)
! Construct !! Column One !! Column Two !! Column Three 
|-
! pcDNA GR2R2(1), R2GG(2)
| G+, R+ || G++ || --
|-
! pcDNA GR2G
| G++ || G++ || G++
|-
! pcDNA R2GR2
| R++ || R++ || R++
|-
| --
| -- || -- || --
|}


Lastly, I did not perform [[Kai_Yuet/Protocols:RNA Isolation | RNA isolaton]] on November 4, 2006; the cells in the 12-well plates were overgrown, and another [[Kai_Yuet/Protocols:Transfection | transfection]] will need to be performed following the splitting of more cells.

----

===November 3, 2006===

The primers kpy21 and kpy22 were designed by TG on October 22, 2006; the pair of primers are created to dimerize with one another, leaving a four-base overhang at both ends complementary to overhangs generated by NheI (5'-GCTAGC-3') and BamHI (5'-GGATCC-3').

 kpy21: CTAGCGTTTAAACTTAAGCTTACC'''ATG'''GGTACCGCTGCTG
 kpy22:     GCAAATTTGAATTCGAATGG'''TAC'''CCATGGCGACGACCTAG

1 µg of each fragment were annealed together in an Eppendorf by placing the tube into 100 mL of H₂O at 90<sup>o</sup>C and allowing the water to cool to below 40<sup>o</sup>C:

'''50 µL Total Reaction'''

*2 µL kpy21 (1 µg)
*2 µL kpy22 (1 µg)
*5 µL 10X NEBuffer 2
*41 µL ddH<sub>2</sub>O

I then generated serial dilutions of 1X, 0.1X, and 0.01X of the resulting reaction and ligated this fragment with the pcDNA-GR2R2 and pcDNA-R2GG vectors, digested with NheI and BamHI for 1.5 hours:

'''10 µL Ligation Reaction'''

*2 µL pcDNA-GR2R2 or pcDNA-R2GG
*4 µL kpy21+kpy22 Insert
*1 µL 10X T4 DNA Ligase Buffer
*2.5 µL ddH<sub>2</sub>O
*0.5 µL T4 DNA Ligase

The kpy21+kpy22 establishes the reading frame at +1 relative to the first fluorescent protein and out of frame with respect to the second fluorescent protein; therefore, our pcDNA-GR2R2 construct (containing the green-red fusion) will, when transfected, express green fluorescence, and our pcDNA-R2GG will express red.

The construct pcDNA-GR2R2 containing the kpy21+kpy22 insert will therefore be referred to as 'pcDNA-GR2G', and pcDNA-R2GG as 'pcDNA-R2GR2.'

TG then miniprepped pcDNA-GR2G and pcDNA-R2GR2 the day after the transformation.  I sent the two constructs for sequencing on October 31, 2006 (MGH Sequencing Core: #56291). 

 GRGT7  Length: 814  November 3, 2006 18:25  Type: N  Check: 3119  ..
 
      1  NNNNNNNNNN NNNNNNNNTT NNN'''CTTACC'''n tg'''GGTACCGC TGCTGGATCC'''
     51  GTGAGCAAGG GCGAGGAGCT GTTCACCGGG GTGGTGCCCA TCCTGGTCGA
    101  GCTGGACGGC GACGTAAACG GCCACAAGTT CAGCGTGTCC GGCGAGGGCG
    151  AGGGCGATGC CACCTACGGC AAGCTGACCC TGAAGTTCAT CTGCACCACC
    201  GGCAAGCTGC CCGTGCCCTG GCCCACCCTC GTGACCACCC TGACCTACGG
                                                                             
 RGRT7  Length: 832  November 3, 2006 18:30  Type: N  Check: 3002  ..        
                                                                             
      1  GNNNNNNTTN NNN'''TTAAGCT TACC'''ntg'''GGT ACCGCTGCTG GATCC'''GTGGC     
     51  CTCCTCCGAG AACGTCATCA CCGAGTTCAT GCGCTTCAAG GTGCGCATGG 
    101  AGGGCACCGT GAACGGCCAC GAGTTCGAGA TCGAGGGCGA GGGCGAGGGC
    151  CGCCCCTACG AGGGCCACAA CACCGTGAAG CTGAAGGTGA CCAAGGGCGG
    201  CCCCCTGCCC TTCGCCTGGG ACATCCTGTC CCCCCAGTTC CAGTACGGCT

Sequencing of the two constructs indicate that the kpy21+kpy22 was successfully ligated into the pcDNA-GR2G and pcDNA-R2GR2 vectors.

==ClustrMap==

<html>
<a href="http://www2.clustrmaps.com/counter/maps.php?url=http://openwetware.org/wiki/Kai_Yuet/" id="clustrMapsLink"><img src="http://www2.clustrmaps.com/counter/index2.php?url=http://openwetware.org/wiki/Kai_Yuet/" />
</a>
</html>