{{KeatingLabToolbar}}
= Amy-approved candidates =



= List of candidates (from group) =

* Barth P, Schonbrun J, Baker D., "Toward high-resolution prediction and design of transmembrane helical protein structures", Proc Natl Acad Sci U S A. 2007 Oct 2;104(40):15682-7 [http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=17905872&ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum PubMed].

The prediction and design at the atomic level of membrane protein structures and interactions is a critical but unsolved challenge. To address this problem, we have developed an all-atom physical model that describes intraprotein and protein-solvent interactions in the membrane environment. We evaluated the ability of the model to recapitulate the energetics and structural specificities of polytopic membrane proteins by using a battery of in silico prediction and design tests. First, in side-chain packing and design tests, the model successfully predicts the side-chain conformations at 73% of nonexposed positions and the native amino acid identities at 34% of positions in naturally occurring membrane proteins. Second, the model predicts significant energy gaps between native and nonnative structures of transmembrane helical interfaces and polytopic membrane proteins. Third, distortions in transmembrane helices are successfully recapitulated in docking experiments by using fragments of ideal helices judiciously defined around helical kinks. Finally, de novo structure prediction reaches near-atomic accuracy (<2.5 A) for several small membrane protein domains (<150 residues). The success of the model highlights the critical role of van der Waals and hydrogen-bonding interactions in the stability and structural specificity of membrane protein structures and sets the stage for the high-resolution prediction and design of complex membrane protein architectures. 

PMID: 17905872 [PubMed - in process]

<br>
* Joshi R, Passner JM, Rohs R, Jain R, Sosinsky A, Crickmore MA, Jacob V, Aggarwal AK, Honig B, Mann RS., "Functional specificity of a hox protein mediated by the recognition of minor groove structure", Cell. 2007 Nov 2;131(3):530-43.

The recognition of specific DNA-binding sites by transcription factors is a critical yet poorly understood step in the control of gene expression. Members of the Hox family of transcription factors bind DNA by making nearly identical major groove contacts via the recognition helices of their homeodomains. In vivo specificity, however, often depends on extended and unstructured regions that link Hox homeodomains to a DNA-bound cofactor, Extradenticle (Exd). Using a combination of structure determination, computational analysis, and in vitro and in vivo assays, we show that Hox proteins recognize specific Hox-Exd binding sites via residues located in these extended regions that insert into the minor groove but only when presented with the correct DNA sequence. Our results suggest that these residues, which are conserved in a paralog-specific manner, confer specificity by recognizing a sequence-dependent DNA structure instead of directly reading a specific DNA sequence.

PMID: 17981120 [PubMed - in process]<br>
[http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6WSN-4R1NGP6-K&_user=501045&_coverDate=11%2F02%2F2007&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000022659&_version=1&_urlVersion=0&_userid=501045&md5=d681c66c45f639397a98d150a9d34ec4 Full Text] (MIT-only access)

<br>
* The molecular architecture of the nuclear pore complex 
* Nature 450, 695-701 (29 November 2007) | doi:10.1038/nature06405; Received 20 April 2007; Accepted 22 October 2007 
Frank Alber1,4, Svetlana Dokudovskaya2,4,5, Liesbeth M. Veenhoff2,4,5, Wenzhu Zhang3, Julia Kipper2,5, Damien Devos1,5, Adisetyantari Suprapto2,5, Orit Karni-Schmidt2,5, Rosemary Williams2, Brian T. Chait3, Andrej Sali1 & Michael P. Rout2

Abstract

Nuclear pore complexes (NPCs) are proteinaceous assemblies of approximately 50 MDa that selectively transport cargoes across the nuclear envelope. To determine the molecular architecture of the yeast NPC, we collected a diverse set of biophysical and proteomic data, and developed a method for using these data to localize the NPC's 456 constituent proteins (see the accompanying paper). Our structure reveals that half of the NPC is made up of a core scaffold, which is structurally analogous to vesicle-coating complexes. This scaffold forms an interlaced network that coats the entire curved surface of the nuclear envelope membrane within which the NPC is embedded. The selective barrier for transport is formed by large numbers of proteins with disordered regions that line the inner face of the scaffold. The NPC consists of only a few structural modules that resemble each other in terms of the configuration of their homologous constituents, the most striking of these being a 16-fold repetition of 'columns'. These findings provide clues to the evolutionary origins of the NPC.

Orr comments: Paper uses several biophysical/proteomic along with computational methods to study macromolecular complex. Maybe Nora or Emiko would be interested.