''See [[Colony PCR]] for general information about this protocol and other variants''
==Materials==
*Sterile 0.6mL plastic tubes
*LB-agar plate with appropriate antibiotic
*[[Designing Primers | Oligonucleotide pair]] (usually VF2 and VR)
*[http://www.invitrogen.com/content/sfs/manuals/11306016.pdf PCR supermix]
*PCR machine
*Pipetman
*Sterile pipet tips

==Procedure==
===Picking colonies===
#Prepare one sterile 0.6mL tube with 20&mu;L ddH<sub>2</sub>O for each colony you intend to pick. 
#Prepare LB-agar plate with appropriate antibiotic to use as index plate.
#Pick single colony using a pipetman with sterile tip.  The pipettor should be set to 3&mu;L
#Inoculate tip with colony into tube.  Pipet up and down to ensure cells are transferred to tube.  Pipet 3&mu;L of cells suspended in water onto index plate.
#*''Alternatively, you can use a multi-channel pipettor to make an index plate once you are done picking colonies.''
#Repeat for as many colonies as you intend to pick.
#*''For routine assemblies, I pick 4-8 colonies per construct.''

===Reaction mixture===
1X Reaction
*9 &mu;L PCR supermix
*0.25 &mu;L 40&mu;M VF2
*0.25 &mu;L 40&mu;M VR
*0.5 &mu;L colony template

===PCR conditions===
#95&deg;C for 15 mins
#94&deg;C for 30 secs
#56&deg;C for 30 secs
#68&deg;C for 1 min per kb of expected product
#*''I typically round up for this step.  i.e. For a 3.6kb construct, I used a 4 min elongation time.  It seems to help to be a bit generous with the elongation time.''
#Repeat 2-4 39 times.
#68&deg;C for 20 mins
#4&deg;C forever

[[Knight:Agarose gel electrophoresis | Run a gel]] to determine amplification product length.

==Notes==
#Lately, I've been carrying out this procedure using a multichannel pipettor to speed steps up.  I use a P50 multichannel pipettor to add 20&mu;L water to the appropriate number of wells in a 96 well plate/  Picking colonies is essentially the same except that I inoculate the colony into a 96 well plate rather than a tube.  The PCR is carried out in 8 tube strips rather than individual tubes.  After adding 9.5&mu;L of the primer + PCR supermix master mix to each PCR tube, I transfer 1&mu;L of colony-water mixture to the reaction tube using a P10 multichannel pipettor.  (I transfer 1 &mu;L rather than 0.5&mu;L because it is easier to pipette.)  I then touch spin the PCR tube strips in a rack using a swinging bucket rotor in order to collect the contents of the tube to the bottom.  The PCR conditions are the same.  --[[Reshma Shetty | RS]]
# We use a [[Seth:Quick_Transformant_Screen|very similar protocol]] with different PCR conditions. Also we use half the DNA for our PCRs and add media to the remainder. Enough cells survive to innoculate overnight cultures for minipreps of which colonies you decide are needed. [[User:Seth|SK]] <small>''January 29th, 2007''</small>
#In a normal PCR reaction setting, a template of 2&mu;L from a 20&mu;L + one colony mix (nicely vortexed) works great for ''S.aureus'', provided we load at least 10&mu;L of the PCR reaction mix in the gel. [[User:Vijay|NV]]

[[Category:Protocol]]
[[Category:Escherichia coli]] 
[[Category:DNA]]