''Back to [[Site-directed mutagenesis]]''

Note that this is a work in progress and needs to be verified against my lab notebooks.  --[[User:Rshetty|Reshma]] 08:49, 24 Aug 2005 (EDT)

==Materials==
*Template plasmid purified from a dam<sup>+</sup> ''E. coli'' strain
*Mutatagenesis primers
*[http://www.stratagene.com/products/showProduct.aspx?pid=105 ''PfuTurbo'' DNA polymerase] (nonstrand-displacing) and associated reaction buffer
*dNTPs
*PCR Thermocycler
*[http://www.neb.com/nebecomm/products/productR0176.asp Dpn I]
*Competent cells

==Mutagenesis PCR mix==
*5 &mu;L of 10X reaction buffer (comes with PfuTurbo DNA polymerase)
*X &mu;L (5â€“50 ng) of dsDNA template
*X &mu;L (125 ng) of oligonucleotide primer #1
*X &mu;L (125 ng) of oligonucleotide primer #2
*1 &mu;L of dNTP mix (100mM total dNTP mix with 25 mM each individual dNTP)
*ddH<sub>2</sub>O to a final volume of 50 &mu;L
Then add
*1 &mu;L of PfuTurbo DNA polymerase (2.5 U/&mu;L)

==Procedure==
This procedure is primarily derived from the [http://www.stratagene.com/manuals/200518.pdf Stratagene QuikChange Site-Directed Mutagenesis manual] with some modifications based on past experience.

#Design mutagenesis primers.  
#*The primer should be designed so that the desired mutation occurs at the exact center of the primer.  The forward and reverse primers should be complementary to each other (i.e. anneal to the same location on opposite strands of the template).
#*Aim for ~15 or more bases of identical flanking sequence on each side of the mutation.  Stratagene recommends not using primers greater than 45 bp in order to avoid formation of secondary structure.
#*Stratagene strongly recommends using FPLC or PAGE purified primers.  See [[Designing primers#Oligo synthesis information | here]] for links to information on oligo purification.  5' phosphorylation is not necessary.
#*See the [http://www.stratagene.com/manuals/200518.pdf Stratagene manual] for more detailed information.  In particular, adhere to their formula for calculating the melting temperature of your primers and design your primers to have a melting temperature >=78&deg;C.
#*See [[Designing primers | designing primers]] for general advice on primer design.
#Purify template plasmid from a dam<sup>+</sup> ''E. coli'' strain via [[Miniprep | miniprep]].
#Set up mutagenesis PCR mix as described above.
#Run PCR
##95&deg;C for 2 mins
##95&deg;C for 30 secs
##55&deg;C for 1 min
##68&deg;C for 1 min/kb of plasmid length minimum
##Run PCR for 18 cycles for point mutations (even though the Stratagene manual recommends 12, I've found 18 to be preferable).
##68&deg;C for 20 mins
#*Stratagene recommends using a PCR machine with heated lid or overlaying the reaction with mineral oil.
#*Note that it is pretty common to not be able to visualize the PCR product on a gel but yet have the mutagenesis still work.
#Cool the reaction to <=37&deg;C
#Add 1&mu;L DpnI restriction enzyme to the PCR tube directly.  (Purification is not necessary at this stage).
#Incubate 2-3 hours at 37&deg;C (even though the Stratagene manual only recommends 1 hr).
#Purify PCR product.  
#*I typically do this step using a [http://www1.qiagen.com/literature/handbooks/PDF/DNACleanupAndConcentration/QQ_Spin/1021422_HBQQSpin_072002WW.pdf QIAgen PCR Purification kit] but any purification which removes the salts, dNTPs, oligos and proteins from the PCR should be fine.
#[[Knight:Electroporation | Transform]] purified DNA into highly competent cells.
#Screen the transformants for the desired mutation using [[Colony PCR  | colony PCR]], [[Restriction Digest | restriction digest]] or [[Sequencing DNA | sequencing]] as appropriate.  Typically 1/4 or 1/8 colonies are correct.

[[Category:Protocol]] [[Category:DNA]] [[Category:In vitro]]