''Back to [[Sequencing DNA]]''

==Purpose==

To confirm the physical DNA resulting from a BioBrick standard assembly step has the same sequence as that part in the [http://parts.mit.edu registry], the physical DNA must be sequenced.  This is done by the [http://web.mit.edu/biopolymers/www/dna_sequencing.html BioPolymers lab] in the Cancer center.

==Materials==

* VF2 and/or VR1 verification primers (10x dilution especially for sequencing reactions)
* Sterile DI water
* Prepped plasmid for sequencing
**You may want to read the [[Optimizing Sample for Sequencing|notes]] from the Biopolymers lab for optimizing your prepped DNA

{|border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;" <!-- This line here formats your table for you.  Change the code to change the formatting of your table.-->
| style="background:#f0f0f0;" | '''Template'''
| style="background:#f0f0f0;" | '''Quantity'''
|-
| colspan="2"| ''PCR products''
|- 
|&bull;  100-200 bp 
| 1-3 ng
|-
|&bull;  200-500 bp 
| 3-10 ng
|-
|&bull;  500-1000 bp 
| 5-20 ng
|-
|&bull;  1000-2000 bp 
| 10-40 ng
|-
|&bull;  >2000 bp 
| 40-100 ng
|-
|Single-stranded DNA 
| 50-100 ng
|-
|Double-stranded DNA 
| 200-500 ng
|-
|Large DNA BACs, PACs, YACs, cosmids, fosmids 
| 0.5-1.0 ug (use 5-10 pmoles of primer)
|-
|Bacterial genomic DNA 
| 2-3 ug (use 6-13 pmoles of primer)
|-  	 
|Primer (unless noted above) 
| 3.2 pmol
|}

===Procedure===

#Fill out the [http://dnalims.mit.edu/index.html online submission form]
#*Print one sheet for your records, and bring one sheet with your tubes to the biopolymers lab.
#Make a 12 &mu;L mix as specified in the submission form.
#*From the 40&mu;M stock solutions, you'll want 0.8&mu;L of a 1:10 dilution of the stock.
#The sequencing request form is numbered. Put each mix into a PCR tube (if you have >4 samples, the sequencing center requests the use of PCR strips) and write the sample number from the form on the top of the tube.
#Deliver all the tubes and the form to the Bio Polymers lab in the Cancer Center. Go to the Cancer center, take the elevator to the 4th floor, take a right, it will be your 2nd-ish door on the left.
#*The Biopolymers lab is room [http://whereis.mit.edu/map-jpg?mapterms=E17-415&mapsearch=go E17-415]. Inside, there is a brown fridge with a clear plastic thing attached to the front. Put your request from in the clear plastic thing and your large eppendorf tube inside the fridge. 
#When you arrive put all the small PCR tubes into a large falcon tube (there is a stash of used falcons next to the small fridge). Then label the side of the large falcon with your name, the date, and Knight Lab and the order number.
#Now you get to wait for your sequencing results to come back (usually 3-4 days).  You should be notified by email when they are available.  They are trying to reduce their turnaround time to 1 day.  

Note that the Biopolymers lab is generally only open during normal business hours, Mon-Fri, 9-5.

==Retrieve sequence data==

[http://dnalims.mit.edu/index.html Login to dnalims] to retrieve sequence data.  

==Verify sequence data==

One option is to use [[VectorNTI]] to align your expected sequences with the .abi data files generated by the sequencing center via the ContigExpress module.  

The registry permits [http://parts.mit.edu/r/parts/partsdb/blast.cgi blasting against the parts database] with the option of only blasting against basic parts.  This can be a quick way to determine whether your sequence is right or not.

There are also various programs that the sequencing center recommends listed [http://web.mit.edu/biopolymers/www/dna_sequencing.html here].

The dnalims online software also permits sequence analysis.  According to the sequencing center, it is quite good.

[[Category: Protocol]]
[[Category: In vitro]]
[[Category: DNA]]