{{Koch_Lab}}
<div style="padding: 10px; width: 720px; border: 5px solid #008;">
Our lab at UNM was started in August 2006, and full-time students will start in Summer of 2007.  So as of March 2007, we are mostly cooking up ideas, ordering equipment and supplies, and starting to build some instrumentation.  But we can generally describe our biophysics expertise that we intend to leverage and further develop as well as the general biological problems that we have learned a lot about and are planning to address with our collaborators.
==Some of the biological problems we are actively interested in==
Repair of [http://en.wikipedia.org/wiki/Dna_repair#Double-strand_breaks double-strand breaks] in DNA, [http://en.wikipedia.org/wiki/Chromatin chromatin] remodeling during [http://en.wikipedia.org/wiki/Transcription_%28genetics%29 gene transcription] (both of these are very important to genome stability and the development and progression of cancer), [http://en.wikipedia.org/wiki/RNA_polymerase_II RNA Polymerase II] elongation, site-specific protein-DNA interactions and binding kinetics, [http://en.wikipedia.org/wiki/Cytokinesis cytokinesis] and [http://en.wikipedia.org/wiki/Mitosis mitosis].  More details coming.

==Some of our biophysics techniques==
===Optical Tweezers===
[[Image:Unzipping DNA with Protein.png|right|thumb|Probing protein-DNA interactions by unzipping single DNA molecules with optical tweezers.]]
Optical tweezers are formed by shining laser light into a microscope objective.  The laser light can trap and move micron-sized particles (such as polystyrene microspheres) which are easily attached to biomolecules (such as DNA).  A carefully constructed and calibrated optical tweezers system can position particles with sub-nanometer precision and apply forces over 100 piconewtons (which is enough to unravel a single double-stranded DNA molecule and break most non-covalent bonds).  A very good description of optical tweezers can be found at the [http://www2.physics.umd.edu/~alaporta/ALPTech.html La Porta Lab website].  Optical tweezers now enable a wide array of fascinating experiments that provide critical information about biomolecular systems (See [http://www.stanford.edu/group/blocklab/ Steve Block Lab] for some famous examples with Kinesin and RNA Polymerase).  To the right is a diagram of using optical tweezers to unzip single DNA molecules when there are DNA-binding proteins present in the solution.  Our lab has expertise in this technique (see PMID 12124289) which we will use to answer key biological questions such as in chromatin remodeling during DNA damage repair and Pol II transcription.
<br style="clear:both;"/>

===Magnetic Tweezers===
[[Image:Mag Tweezers EWD.png|right|thumb|Schematic of magnetic tweezers apparatus under construction at CINT with P. Goodwin, J. Werner, D. Keller, G. Thayer, J. Martin, G. Bachand.]]
In contrast to optical tweezers, the magnetic "tweezers" that we use are not really tweezers per se.  They are really apparatuses that can apply calibrated, uni-directional, dynamic forces to many single-molecule tethers simultaneously.  For example, the picture to the right shows an apparatus designed to apply vertical forces to many single-molecule tethers at once.  Tether length information is provided by evanescent wave scattering (as the [http://www.physics.ucla.edu/zocchi_lab/ Zocchi lab] does), but there is no control over microsphere position...only applied force.  The idea is to be able to perform experiments more rapidly, but with less exquisite control.  Furthermore, magnetic experiments are much more amenable to varying the temperature and for samples containing a lot of debris that would interfere with optical traps.  An example of an experiment we would like to perform is to set up single-molecule tethers in an unzipping configuration and then detect whether or not a DNA-binding protein is present based on the unzipping force.  Because of the increased efficiency and robustness compared with optical tweezers, it would be much easier to probe how protein binding changes when the osmotic pressure, temperature, and other factors are varied. 
<br style="clear:both;"/>

===MEMS and Nanofabricated Devices===
[[Image:PN_186_sensor_45deg_08a_big.PNG|right|thumb|SEM image of a MEMS force sensor described in our [http://link.aip.org/link/?APL/89/173901 APL paper]]]
Our lab also has interest in developing micro- and nano-fabricated devices that can enable biophysical experiments that are impossible or extremely difficult with other methods.  In principle, microfabricated experiment platforms can be more versatile, much more robust, and more widely accessible than optical tweezers setups.  However, there is a very long way to go towards this goal.  In collaboration with Maarten de Boer, Alex Corwin, Gayle Thayer, and George Bachand at Sandia, we have recently taken initial steps towards using MEMS devices for real biophysics experiments (see picture to the right).  The device we used was fabricated with Sandia's SUMMiT process (via the [http://mems.sandia.gov/samples/index.html SAMPLES] program, and so is easily incorporated into other MEMS designs including actuation (comb drive or inch worm) and detection.  Also, our lab is now located at the Center for High Technology Materials ([http://www.chtm.unm.edu CHTM]) which houses one of the [http://www.nnin.org/nnin_newmexico.html NNIN nanofabrication facilities].  We are working on ideas for leveraging the unique capabilities at the CHTM (for example, nanochannels fabricated via optical interference lithography over square-centimeter areas) for biological experiments, partucularly to complement or combine with the biological studies being performed with optical and magnetic tweezers.
<br style="clear:both;"/>

===Tethered Particle Motion===
A prerequisite for most our optical and magnetic tweezers experiments is to tether microspheres to a coverglass via a single-molecule (usually DNA).  When viewed under a microscope, tethered microspheres (approximately 1 micron) undergo constrained Brownian motion.  An expert at tethering can tell by eye whether the "amount" of Brownian motion is correct for the length and nature of the single-molecule tethers.  It turns out that even without applying and measuring forces with optical or magnetic tweezers, useful biological experiments can be carried out by video microscopic analysis of the tethered particle motion (TPM).  A famous early example of this was published by the [http://www.bact.wisc.edu/landick/ Landick Lab] in 1991: PMID 1861724.  The hardware requirements for this method are only a microscope, video camera, and computer, making the method very appropriate to carry out in any biology lab with standard microscopy facilities.  However, there is still a barrier presented by data acquisition and data analysis software.  Our lab has expertise using LabVIEW for this application and we intend to both carry out our own TPM studies (starting with chromatin remodeling during double-strand break repair) as well as share our LabVIEW software and methods with other labs who can utilize the technique.

===Microfluidics and DNA Force Sensor Assay===
[[Image:Forceassay1.png|thumb||300px|right|'''Concept for Protein Binding Assay''' The purple polygons indicate DNA binding proteins.  If a DNA binding protein is present, the fluorophore will stay with the bottom duplex.]]There are two aspects to this project.  The first is to develop useful microfluidic techniques for DNA tethered particle motion, particle stamping, and our force sensor assay experiment.  The second is to develop a new force assay technique to analyze interactions between DNA and DNA-binding proteins.  

Microfluidics:<br>
We use a polymer known as PDMS ([http://en.wikipedia.org/wiki/Polydimethylsiloxane polydimethylsiloxane]) and [http://en.wikipedia.org/wiki/Soft_lithography soft lithography] to create devices for microfluidics and / or force sensor assays.  PDMS is very widely used and is a microscope friendly substance, in that it is a clear substance allowing the experimenter to use it as a sort of cover glass.  To create a device one simply makes a template (a negative of the cast you want) and pours a mixture of PDMS with curing agent onto the template.  Using a defined volume one can create devices of specified height, width, and depth.  We are practicing methods to achieve a depth equivalent to that of a glass slide (~1mm).  Current templates include a microfluidic channel template and a protein stamper design.  The channel template is a square (60mm x 60mm) with 9 channels equally spaced of equal dimensions (~1mm x 60mm x 100um).  The PDMS will harden so that the ridges of the template become channels on the mold.  When affixed to glass, one will be able to perform single molecule tethering experiments.  The other template contains many (about 30 or so) indented tiny circles.  In this case the PDMS will harden to have raised circles to create a stamp so that one can just press the PDMS into a protein solution and they will be attached to the raised PDMS surface (not in the channels created between the circles).

Force Assay:<br>
In a previous experiment Gaub et al. used the PDMS stamp to analyze interactions between different DNA geometries and between perfectly matching duplexes and duplexes with a single mismatched base pair (see "DNA: A Programmable Force Sensor" PMID 12869761,  and "A force-based protein biochip" PMID 12975526).  We intend to expand upon this work and create a new method for detecting the presence of DNA binding proteins based upon the resistance to DNA unzipping caused by bound proteins.  We will use a similar setup, and have designed DNA constructs from scratch using [http://frontend.bioinfo.rpi.edu/applications/mfold/cgi-bin/dna-form1.cgi mFold] (a web program that details the thermodynamics of specific DNA chains and gives all kinds of other goodies) to perform this experiment (see picture).  We will use the PDMS protein stamping technique mentioned above in conjunction with simple tethering techniques to create the experimental setup.  Using a fluorescent label we will be able to analyze the results via the "naked eye" using a comparison approach.  The setup will be as follows:  We will tether our DNA molecule complexes (with label affixed to one end) to the surface of a glass slide.  We will then place the PDMS stamp on top of the slide, so that the DNA can then bind to the PDMS side as well.  Next we pull the PDMS off the glass and analyze.  The label will either appear on the glass side or the PDMS side, depending on the strength of the tether.  For example, one specific setup may have a shear DNA geometry that is affixed to the PDMS side with an unzip DNA geometry on the glass side.  Once the sides have been pulled apart one would expect a higher ratio of fluorescent labels on the glass side because the unzip geometry isn't as stable as the shear.  In the protein bound case we may compare bound protein DNA with unbound protein DNA and would compare the ratio of labels on both the glass and PDMS sides.  If things work out well, a 50 / 50 split of the fluorescence will result when the DNA is incubated in the presence of DNA binding protein at a concentration equal to the <math>K_d</math>.

===Simulation, Analysis, Modeling, Automation===
Our lab has expertise (mostly in LabVIEW) with software applications for a variety of purposes, as is common with experimental biophysics labs.  Software for data acquisition and data analysis are critical for all of the above techniques.  Furthermore, for many of the biological systems we are interested in, it comes in very handy to create simple models for the kinetic processes and then to perform [http://en.wikipedia.org/wiki/Monte_Carlo_method Monte Carlo simulations] to help guide our thinking in both designing and interpreting experiments.  Currently, most of our software is only available in house, but with some work, we could make the software useful to other labs around the world (particularly those using LabVIEW).  The more effort we put into making user-friendly software and also explaining all of the code, the more likely it is to be useful.  Any inquiries from potential end users (see our [[Koch_Lab:Contact|contact page]]) would likely be very useful for spurring us on in these endeavors.  Some possibly useful software:
* Optical tweezers feedback control LabVIEW application.
* Dynamic Force Spectroscopy (DFS) analysis software (would require data file interpreter).  Also DFS simulation software.
* Top-level application for simulation of enzymatic processes on chromatin (such as Pol II transcription) coupled with simulted Chromatin Immunoprecipitation ([http://en.wikipedia.org/wiki/Chromatin_immunoprecipitation ChIP]) analysis.  Useful for evaluating alternative interpretations of sometimes-compliated ChIP experiments.
* Tethered Particle Motion (TPM) software.

==Some of the other labs that we talk with and may collaborate with==
[http://hsc.unm.edu/som/micro/Maryann.html Mary Ann Osley Lab], UNM, HSC Department of Molecular Genetics and Microbiology<br>
[http://dir.niehs.nih.gov/dirlmc/transcript.htm Karen Adelman Lab], NIH, NIEHS<br>
[http://mcdb.omrf.org/gorbskylab/ Gary Gorbsky Lab], Oklahoma Medical Research Foundation<br> 
[http://cint.lanl.gov/user_call/scientist_summaries.shtml George Bachand Lab], Sandia National Labs and CINT<br>
Alex Corwin, Maarten de Boer, and Gayle Thayer Labs, Sandia National Labs<br>
[http://cint.lanl.gov/user_call/scientist_summaries.shtml Peter Goodwin & Jim Werner Labs], Los Alamos National Lab and CINT<br>
Kim Rasmussen Lab, Los Alamos National Lab

</div>