==Syllabus==


===Class Format===

The Class will meet twice a week, one 2 hour classroom session, and one 3 hour lab session. 
Problem sets will be assigned weekly for the first eight weeks.
A midterm exam will cover the classroom material and the first several weeks of lab instruction.
Lab exercises will be assigned, and cover the lab techniques taught.  A small project to assemble and measure a BioBrick system follows the lab instruction.  Finally, students will design, build and analyze their own BioBrick system. 

===Grades===

The final grade will be as follows:
20% Problem Sets
30% Midterm Exam
20% Lab Evaluations
30% Final Project

===Schedule===
*Introduction - Synthetic Biology: History, current applications and future directions [[Media:SB-Lecture1.ppt| Powerpoint]] (w/content from Drew Endy)

Assignments: Endy Article and Comic Strip 
*The Biology (4 sessions)
** Cells, DNA, RNA and Protein
** DNA - information encoding, structure, sequencing and synthesis
** RNA - encoding, structure, function (RNA Enzymes, RNA Aptamers)
** Proteins - Crystallography, functions, scaffolds

Introductory packet about DNA, RNA, Protein.  Ron Weiss paper on Vibrio Fischerii quorum sensing (decompose into parts).  Artemisnin overview article.  
Hellinga Paper on protein sensors. 
*Foundational Technologies (4 sessions)
** BioBricks - Definition of a BioBrick, black-box encapsulation, PoPs and RiPs [[Media:SB-Lecture2.ppt| Powerpoint]] (w/content from Drew Endy)
** The Registry of Standard Biological Parts
** Analytical -omics tools - Proteomics, Genomics, Metabolomics
** Simulation tools - stochastic simulators, BioJADE, matlab

Create a biobrick out of a sequence (force them to re-optimize a coding region into e. coli and remove a biobrick incompatibility).
An introductory packet on PCR and cloning.  


*Basic Lab Techniques (8 sessions)
**Introductory Lab - PCR Primer Design
**Lab 2 - PCR, Restriction Digestion, Gel electrophoresis
**Lab 3 - Insert Preparation, Gel Extraction, Ligation and Transformation
*BioBrick Discovery and Creation Technique
**Lab 4 - PCRing New Parts [[Media:assignment-1.pdf| PDF]]
**Lab 5 - Cloning your new Part into A BioBrick Vector[[Media:lab6.pdf | PDF]]




** Cloning Basics - restriction enzymes, running analytical gels, QiaPreps
** Sterile Technique - autoclaving, flaming technique, 
** Competent Cells - Production & QC
** PCR - About PCR, Polymerases, Primer design 
** Screening - Antibotics, Blue/White screening
** Growth Curves
** Metabolite measurement 
** Standardized Measurements for BioBricks

*Trial Project (2 weeks)
** Assembly and Measurement of a BioBrick system

*Final Project

===Reading List===

=Intro=
*Bio FAB, G., Baker, D., Church, G., Collins, J., Endy, D., Jacobson, J., Keasling, J., Modrich, P., Smolke, C., and Weiss, R. (2006). Engineering life: building a fab for biology. Sci. Am. 294, 44-51.
*Aldhous, P (2006). Redesigning life. New. Sci. 43-47
=Fundamentals of SynthBio=
*Gardner, T.S., Cantor, C.R., and Collins, J.J. (2000). Construction of a genetic toggle switch in Escherichia coli. Nature 403, 339-342.
*Judd, E.M., Laub, M.T., and McAdams, H.H. (2000). Toggles and oscillators: new genetic circuit designs. Bioessays 22, 507-509.
*Elowitz, M.B., and Leibler, S. (2000). A synthetic oscillatory network of transcriptional regulators. Nature 403, 335-338.
*Ramachandra, M., Rahman, A., Zou, A., Vaillancourt, M., Howe, J.A., Antelman, D., Sugarman, B., Demers, G.W., Engler, H., Johnson, D., and Shabram, P. (2001). Re-engineering adenovirus regulatory pathways to enhance oncolytic specificity and efficacy. Nat. Biotechnol. 19, 1035-1041.
*Sprinzak, D., and Elowitz, M.B. (2005). Reconstruction of genetic circuits. Nature 438, 443-448.
*Levskaya, A., Chevalier, A.A., Tabor, J.J., Simpson, Z.B., Lavery, L.A., Levy, M., Davidson, E.A., Scouras, A., Ellington, A.D., Marcotte, E.M., and Voigt, C.A. (2005). Synthetic biology: engineering Escherichia coli to see light. Nature 438, 441-442.
*Basu, S., Gerchman, Y., Collins, C.H., Arnold, F.H., and Weiss, R. (2005). A synthetic multicellular system for programmed pattern formation. Nature 434, 1130-1134.
*Dueber, J.E., Yeh, B.J., Chak, K., and Lim, W.A. (2003). Reprogramming control of an allosteric signaling switch through modular recombination. Science 301, 1904-1908.
*Bayer, T.S., and Smolke, C.D. (2005). Programmable ligand-controlled riboregulators of eukaryotic gene expression. Nat. Biotechnol. 23, 337-343.
*Chan, L.Y., Kosuri, S., and Endy, D. (2005). Refactoring bacteriophage T7. Mol. Syst. Biol. 1, 2005.0018.
=Ethics: The 1918 Spanish Flu Debate=
*Tumpey, T.M., Basler, C.F., Aguilar, P.V., Zeng, H., Solorzano, A., Swayne, D.E., Cox, N.J., Katz, J.M., Taubenberger, J.K., Palese, P., and *Garcia-Sastre, A. (2005). Characterization of the reconstructed 1918 Spanish influenza pandemic virus. Science 310, 77-80.
*Kennedy, D. (2005). Science 310, 195.
*Sharp, P.A. (2005). 1918 flu and responsible science. Science 310, 17.
*Kaiser, J. (2005). Virology. Resurrected influenza virus yields secrets of deadly 1918 pandemic. Science 310, 28-29.
*Tumpey, T.M., Basler, C.F., Aguilar, P.V., Zeng, H., Solorzano, A., Swayne, D.E., Cox, N.J., Katz, J.M., Taubenberger, J.K., Palese, P., and Garcia-Sastre, A. (2005). Characterization of the reconstructed 1918 Spanish influenza pandemic virus. Science 310, 77-80.
*Simonson, S. (2006). Advising on publication. Science 311, 336-337..
*Jensenius, J.C. (2006). Vaccine against Spanish flu. Science 311, 1552; author reply 1552.
*Gostin, L.O. (2006). Medical countermeasures for pandemic influenza: ethics and the law. JAMA 295, 554-556.

<html>
<a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/"><img alt="Creative Commons License" style="border-width:0" src="http://i.creativecommons.org/l/by-sa/3.0/88x31.png"/></a><br/>This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/3.0/">Creative Commons Attribution-Share Alike 3.0  License</a>.
</html>