{{Template:Li Lab}}

<div style=" width: 1024px; padding: .1em 1.5em .9em">
{| cellspacing="0"
|- valign="top"
|width="70%" style="border: 1px solid #dddddd; color: #000; background-color: #dddddd"|
<div style="padding: left; text-align: left; float: left; ">

Our lab is focused on the design and application of statistical and computational algorithms to elucidate global transcription regulatory mechanism of key cancer transcription factors, by integrating the data from ChIP-chip/seq, gene expression profiling and sequence motif finding.

An elaborate system of transcription regulation is responsible for the morphological and behavioral complexity in higher eukaryotes. This regulatory system consists of diverse trans-acting protein factors and cis-acting regulatory DNA sequences. Recently, Chromatin ImmunoPrecipitation coupled with whole genome tiled microarray (ChIP-chip) and/or next-generation sequencing (Solexa, SOLiD and 454) has evolved as a powerful and unbiased technique to study genome-wide in vivo binding of the trans-acting protein factors. The application of this technology to multiple factors and/or in multiple conditions allows biologists to study how trans-factors differentially regulate transcription in a combinatorial manner. However, it also poses great challenges for the development of effective algorithms, the key link between massive raw data and biological hypotheses. 

We developed a series of  algorithms to reliably detect and annotate ChIP-enriched regions using Affymetrix whole-genome tiling arrays, including 1) Model-based Analysis of Tiling-arrays (MAT; PNAS 2006) and a hidden Markov model (Bioinformatics 2005) for ChIP-region detection, 2) extreme MApping of OligoNucleotide (xMAN, BMC Genomics 2007) for microarray probe mapping, 3) Cis-regulatory Element Annotation System (CEAS; NAR 2006) for ChIP-region annotation. Since the inception in early 2006, they have been adopted by hundreds of academic users and are now considered as the ChIP-chip data analysis standard in many labs. We are also working with the ENCODE spike-in consortium, which consists of more than 10 transcriptional regulation groups worldwide, to systematically analyze the performance variability introduced in ChIP-chip protocols, array platforms, and analysis methods.	
		
We are also in close collaboration with several labs on identifying global regulation targets of a series of key transcription factors, including estrogen receptor (Cell 2005; Nature Genetics 2006); androgen receptor (Molecular Cell 2007); FoxA1 (Cell 2008); STAT5 (J. Biological Chemistry 2006); TAL1 (Blood 2006); and p53 (Bioinformatics 2005). 



</div>

|width="30%" style="border: 1px solid #dddddd; color: #000; background-color: #dddddd"|
<div style="clear: right; text-align: left; float: left; padding: .1em 1.5em .9em; ">

==[http://lilab.openwetware.org/Positions.html Open Positions]==
Join Dr. Wei Li's lab at Baylor College of Medicine in Houston's Texas Medical Center, one of the most exciting places in the world for biomedical research! 

* Postdoctoral position in Bioinformatics/Computational Biology


* Scientific Programmer position 


* Multiple openings for graduate students




</div>
</div>