[[User:Smoore|Smoore]]
==Background==
Genetic libraries are collections of genes present in some recombinant DNA form so they can be propagated.  When people refer to “screening a library” they usually have some phenotype that they are able to select or screen for and evaluate a large number of library clones to look for a gene that alters the phenotype.  People interested in eukaryotic biology usually make cDNA libraries that are derived from pools of mRNA isolated from an organism of interest.  This allows them to isolate DNA fragments that encode proteins or RNA that are produced form the spliced form of the RNAs found in the cells.  It has been a long time since I have worked with cDNA libraries, so I won’t go into that here (perhaps someone in another group can add a section?).  

For example, suppose we have a strain of bacteria that can’t grow on lactose (like <i>Salmonella</i>) and we are interested in finding genes that are needed for lactose metabolism.  First, we prepare a plasmid that has been digested with two different restriction enzymes so that is can accept similarly-digested DNA fragments.  Second, we digest the genomic DNA of an organism that can metabolize lactose (like <i>E. coli</i>) and ligate the fragments into the plasmid.  Third, we transform the Salmonella with the recombinant plasmids we have made and look for <i>Salmonella</i> that can grow on lactose as a carbon source.  The plasmid that contains the genes responsible for lactose metabolism can then be isolated and sequenced to identify, hopefully, the <i>lac</i> operon of <i>E. coli</i>.

In the above example, a <i>selection</i> was used because only the cells with the ability to use lactose for food could grow.  We could have also <i>screened</i> for the ability to cleave lactose with &beta;-galactosidase by putting X-Gal in the plates and looking at thousands of white colonies for a blue colony

There are many variations on library creation.  An investigator may choose to randomize a small segment of a cloned gene and screen the variants for a mutant with a new phenotype.  A whole gene or plasmid can be mutated can be transformed for screening.  A screen can be set up for “multi-copy suppressors” that rely on having an excess of a gene to obtain a phenotype.  It’s all up you and your smart noodle to figure out the best way.

==Design Strategy==
I am presenting this strategy to make a library of genomic <i>E. coli</i> fragments in plasmids that replicate in <i>E. coli</I>.  In searching for genes or gene clusters, it’s important to keep in mind that any given pair of restriction enzymes will only cover a fraction of the DNA present in the chromosome.  Because the <i>E. coli</i> genome is sequenced (at least several K strains), you can make an estimate of the number of times your genomic DNA prep will be cut by a particular endonuclease.  When you are cutting with two enzymes to make library fragments, the number of clonable fragments from a complete digest will, at most, be twice the number of times the “least cutting” endonuclease cuts.  Because of this, it is wise to set up several libraries with different endonucleases.  Not only will this allow better coverage, but it will also allow the identification of a gene that may have one of the sites within it (that would never be isolated because it would always be cut in the library).

It is a good idea to use endonucleases that leave 4 bp overhangs so that the ligation efficiency is high.  There are four enzymes that leave the same CTAG overhang (Avr II, Nhe I, Spe I, and Xba I).  This is quite useful.  You can prepare a single vector preparation cut with, say, EcoR I and Avr II, and ligate four different genomic digests into it (EcoR I and each of the four enzymes that generate complementary overhangs).

When preparing the vector for your library, you want to minimize the background of transformants lacking an insert.  A trace amount of singly-cut vector can produce a lot of transformants after ligation.  I generally employ one of two strategies to get around this.  

*The easiest is to add a third restriction enzyme to your digest that cuts between the two sites of interest.  In doing so, vectors that were cut by only one of your library enzymes get secondarily cut to prevent self-ligation.  In general cloning of fragments, I find this to be far more effective than using a phosphatase (which reduces overall ligation/transformation efficiency).

*If your vector doesn’t have convenient sites for making the library, or if your vector is a low-copy vector, you can use PCR to amplify the replication origin and drug-resistance gene while appending convenient restriction sites on the ends (see [['Round-the-horn site-directed mutagenesis]]).  If you follow this route, keep in mind that most of your ligated plasmids will have large segments that are not host modified.  Therefore, transform the library into cells that have no restriction system.  This method greatly reduces “vector-only” transformants.

==Protocol==

===Genomic DNA Preparation===

*Use a strain that you think contains genes you’re interested in.
*Use a strain that does not contain obvious genes that will allow the genetic screen to be duped.  For example, I did a screen in cells that were expressing a toxic gene from the <i>araBAD</i> promoter.  The library clones that got selected as suppressors contained genes that shot down the <i>araBAD</i> promoter: suppressors, yes; relevant, no.

<b>--Harvest and Lysis--</b>

*Grow ~30 mLs of culture to late log phase (~2-3 X 10<sup>9</sup> c.f.u./mL).
*Harvest 20 mLs of cells and resuspend in <b>2 mLs Rinse Buffer</b> and re-harvest.  This step greatly improves lysis.  Divide the cells evenly into 4 aliquots in 1.5mL microfuge tubes.
*Resuspend each pellet in <b>100 &mu;L Lysis Buffer</b> by pipetting up-and-down.  Vortexing will foam the solution and make a mess.
*There should be visible clearing in about 30 secs to 1 min at room temperature.
*After clearing (there will be some turbidity, but nothing like the original suspension), add <b>467 &mu;L TE Buffer</b> and let stand 10 minutes. Here, you are allowing further lysis and RNA degradation.
*Add <b>30 &mu;L 10% SDS</b>, mix gently to prevent foaming.
*Add <b>3 &mu;L Proteinase K</b> stock solution (20 mg/mL).  You now have 600 &mu;L per tube (plus cell volume) and the proteinase K is ready to chew.  Sometimes, you’ll see a knot of white material at this step, it should go away as the proteinase does its thing.
*Incubate at 37 degC for about 1 hour.  The solution will get pretty clear.

<b>--DNA isolation--</b>

*Add <b>185 &mu;L of 3M NaCl</b> to each tube.  ~700 mM final.  Needed for CTAB precipitation step.
*Add <b>90 &mu;L of 10 % CTAB solution</b>, mix, let stand 5 minutes.  CTAB precipitates polysaccharides.  In high salt, DNA is soluble in CTAB.
*Add <b>~700 &mu;L of chloroform</b> and vortex to form an emulsion.
*Spin 5 min in microfuge.  You should see a nice, bright-white interface.  This is your CTAB precipitate.
*Carefully remove the supernatant to a new tube.
*Extract the solution with <b>~700 &mu;L phenol/chloroform mix</b>, emulsify and spin.
*Extract once more with chloroform to remove the phenol.
*Precipitate the DNA with <b>~700 &mu;L Isopropanol</b>.  After mixing, let stand on ice for a few minutes before centrifuging.
*Spin in a microfuge for 15 minutes to collect the DNA.
*Wash once with <b>~1 mL 75% ethanol</b>.
*Wash once with <b>~1 mL 95% ethanol</b>.
*Dry the pellets by setting the tubes on their side.
*Resuspend and pool the DNA in <b>100 &mu; TE</b>.  This is 200 X the original culture volume.  Assuming each gene was present in one copy per cell (an under-estimate), you’ll have about 6 X 10<sup>10</sup> copies of each gene in your tube.  Plenty.
*Determine the concentration of DNA using absorbance at 260 nm.  The DNA prep will most likely still have RNA fragemts in it, don't sweat it, but keep in mind that they also add to the absorbance reading.

===Digestion and Ligation===

I generally don't fuss too much here.  Determine the activities of the restriction enzymes you will be using (they're printed on the tube in units per mL).
*Digest the DNA with a 10- to 20-fold excess of restriction enzyme.  The kicker here is that companies always present activity in terms of "degrades XX &mu;g DNA per unit time"...what does this mean?  What if my plasmid has 1 site?  What if it has 100 sites?  So, use the activity as a guide.  Let the digest go a long time to ensure complete digestion (4-6 hours).
*Use a Qiagen "PCR Cleanup" kit to purify the fragments.  The kit not only removes protein and salts, but also gts rid of very large and very small DNA fragments that you don't want in your library.  I have still cloned ?10 kb fragments after purifying my digest this way so don't worry.
*Determine concentration of fragments by absorbance.
*If you want, you can run a gel of the material, but it's generally a waste of time.

I won't detail how to prepare a vector for receiving an insert here.

*Set up test ligations to determine the best insert-to-vector ratio.  I set up about 5 reactions in a volume of about 5 &mu;L.  Vary the insert amount and keep the vector constant.

*Transform each test ligation, you can probably plate the whole thing on one plate.  Count the colonies that form and use the insert-to-vector ratio that gave the most colonies to set up the large ligation reactions.

====<u>Recipes</u>====

<b>Rinse Buffer</b>

10 mM Tris-Cl, pH 8.0

100 mM NaCl


<b>Lysis Buffer</b>

B-Per (Pierce) supplemented with:

5 mM EDTA (from neutral pH stock)

0.1 mg/mL Lysozyme (from fresh 10 mg/mL stock in PBS or TBS)

1 &mu;g/mL RNase A

 
<b>TE Buffer</b> (Tris-EDTA)

10 mM Tris-Cl, pH 8.0

1 mM EDTA


*You can use other detergents instead of B-Per, that's what I use.  Bug-Buster is another commercial detergent that doesn't disrupt proteins.  There are protocols for using Tween, NP-40, and Triton as well.  You just need to get the outer membranr disrupted so the lysozyme has access.  I have had good luck with ~1-2% chloroform as well.


<b>10% CTAB solution</b>

10% CTAB (some crazy name that doesn't start with a "C")

700 mM NaCl

===Transformation and Plating===

*Using the test ligations as a guide, set up a ligation reaction that will generate enough clones to thoroughly screen the number of clonable fragments you expect from the genome.  Keep in mind that some genomic fragments may be under-represented in your final mixture and that some fragments may not ligate efficiently (like large fragments).

*Transform the whole big ligation reaction into a big aliquot of competent cells.  Some people are married to electroporating everything... if you want to electroporate 20 samples, go ahead.  I like to mix the whole ligation reaction into fresh TSS competent cells.

*Heal the cells for about an hour and harvest.  Healing too long will cause duplications of clones.

*Resuspend the cells at about 1/20th the volume of the healing culture.

*Add glycerol to ~15%, mix.

*Remove a small aliquot into a separate microfuge tube.

*Freeze both the large and small aliquots at -80 degC.

*After the small aliquot is frozen, thaw it on ice.

*Plate the cells to determine how many colony-forming units there are per set volume of frozen, transformed cells in your big aliquot.

*Plate the cells so that there are an appopriate number of colonies on each plate.  For a screen, 100-500 colonies per plate will keep them mostly separated.  For a selection, you can plate higher densities to reduce the number of plates.

==Final Thoughts==
In any genetic screen, you will find a colony that grows or has the phenotype if you look hard enough.  It is a good idea to know the frequency of "spontaneous" revertants in the cell type you are screening and to be aware of this number when you go to pick colonies from your library.  If your transformed library is generating the same number of positive clones as your background, it ain't working.

Always purify the library plasmids from your selection/screen and re-transform them into naive hosts to ensure that the phenotype is propegated with the clone.

Be aware that there are a lot of ways to overcome selective pressure that you may not have thought of.  If the cell finds a way to alter the copy number of a plasmid, or up-regulate a toxin exporter, they will look like good clones, but have nothing to do with what you were looking for.

[[Category:Protocol]]
[[Category:DNA]]
[[Category:In vitro]]