{{Template:Maloof_Lab}}
{| cellspacing="2px" cellpadding="0" border="0" style="padding: 0px; width: 680px; color: #000000; background-color: #ffffff;"
|-valign="top"
|width=700px style="padding: 15px; background-color: #ffffff; border: 2px solid #F8B603;" |<br/>
{| cellspacing="2px" cellpadding="0" border="0" style="padding: 5px; width: 680px; color: #000000;"
|
<h2>Jose M Jimenez-Gomez, PhD.</h2>
[mailto:jmjimenez@ucdavis.edu Contact]
|[[Image:Pepe_b&w.jpg|right|135px]]
|}

<br>
I am a Postdoctoral fellow in [[Maloof_Lab |Julin Maloof's lab]] in the [http://www-plb.ucdavis.edu/ Section of Plant Biology] at the [http://www.ucdavis.edu University of California Davis].<br>
<br>
In 2005, I completed my PhD. in JM Martinez-Zapater's lab at the [http://www.cnb.uam.es CNB] (National Center for Biotechnology) in Madrid, Spain, where I performed a quantitative genetic analysis of flowering time in tomato <cite>Jimenez-Gomez07</cite>.
<br>
<br>

<h3><font style="color:#F8B603;">QTL analysis of the shade avoidance response in Arabidopsis</font></h3>
----
<br>
Plants from different environments exhibit different degrees of responsiveness to the same light stimulus. For example, when plants accommodated to sunny environments detect foliar shade from neighboring vegetation they respond increasing petiole and stem elongation and reducing the time to reproduction, a phenomenon called the "shade avoidance response". On the other hand, plants surrounded by tall vegetation are familiarized with the shade and do not present this response. 
To identify the molecular mechanisms underlying this differences we are performing QTL analysis using a previously developed, well characterized Recombinant Inbred Line set descent from two different natural populations of <i>Arabidopsis thaliana</i>: Bayreuth, originary from the German low altitude fallow lands, and Shahdara, from the high mountains of Tadjikistan <cite>Loudet02</cite>.<br>
We grew replicated individual RILs in environments simulating shade and sun conditions and measured them for a number of traits characteristic of the shade avoidance response syndrome. For the QTL analysis we modeled this phenotipic data to calculate a shade avoidance response index and used an available genetic map that includes more than 500 Single Feature Polymorphism (SFP) markers <cite>West06</cite>.<br>
<br>
<br>
{|
|-
|align="center"|[[Image:QTL_analysis.jpg|center]]
<small>LOD score graph for several of the traits measured</small>
|-
|}
<br>
<br>
We are focusing now in a chromosomal region containing about 200 genes to fine map and identify the gene  responsible for the differential response to shade between the two natural populations. To do this we employ traditional genetic approaches as well as genomic and network analysis. We are developing a protocol to construct gene networks that will help us consider candidate genes based on coexpression with other genes across microarray experiments <cite>Riken</cite>, colocalization with expression QTLs <cite>West07</cite>, functional categorization <cite>GO_Classification</cite> and presence of polymorphisms <cite>Clark07</cite>.
<br>
<br>
{|
|-
|align="center"|[[Image:Network_fragment.jpg|center]]
<small>Fragment of a gene network</small>
|-
|}
<br>
<h3><font style="color:#F8B603;">Single Nuncleotide Polymorphism discovery between wild Tomato species</font></h3>
----
<br>
We use a bioinformatic approach to scrutinize the available tomato EST sequences and detect Single Nucleotide Polymorphisms. This will allow us to estimate the divergence between wild and cultivated tomato species, and will serve to have an idea of the effectiveness of the high throughput genomic methods that are and will be available soon for these species.  
<br>
<br>
<h3><font style="color:#F8B603;">Molecular evolution of PHYTOCHROME B</font></h3>
----
<br>
PHYTOCHROME B (PHYB) is the main plant photoreceptor involved in the shade avoidance response. This gene has been reported to be under selective pressure, suggesting that plants with different shade avoidance responses may have different functional alleles of PHYB. Under these presumptions we are sequencing and cloning PHYB genes from a number of species with diverse shade avoidance behaviors. We will soon test if the variation in light responses between these plants are due to particular amino-acid changes in this photoreceptor.
<br>
<br>
{|
|-
|align="center"|[[Image:PHYB_alignment.jpg|center]]
<small>amino-acid changes in a fragment of the PHYB gene in 8 speceis, red and black bars indicate non conserverd/ conserved amino-acid changes respectively</small>
|-
|}
<br>
<br>
<h3><font style="color:#F8B603;">Proteomics of light perception</font></h3>
----
<br>
When plants are exposed to light a number of changes occur that are controlled by complex signaling processes. Light perception includes interaction with flowering time pathways, the circadian clock and hormone pathways between others. Genetics and genomic analysis have so far allowed us to identify and understand part of how this signals occur at the gene expression level, but very little is known about the changes produced in the plant at protein level. The new advances in Proteomics make possible to identify small protein changes with high precision. In collaboration with the [http://proteomics.ucdavis.edu/ <font style="color:#000;">Proteomics Facility at the UC Davis Genome Center</font>] we are preparing a set of experiments that will allow us to determine the accuracy and power of the newest techniques in protein quantification and to better understand how the proteome is regulated by light.     

<br>
<br>
<h3><font style="color:#F8B603;">References</font></h3>
----
<biblio>
#Jimenez-Gomez07 pmid=17502904
#Loudet02 pmid=12582628
#West06 pmid=16702412
#West07 pmid=17179097
#Clark07 pmid=17641193
#Riken [http://prime.psc.riken.jp/ Platform for Riken Metabolomics]
#GO_Classification pmid=10802651
</biblio>
|}

__NOTOC__