{{Template:Maloof Lab}}
{| cellspacing="2px" cellpadding="0" border="0" style="padding: 0px; width: 700px; color: #000000; background-color: #ffffff;"
|-valign="top"
|width=750px style="padding: 5px; background-color: #ffffff; border: 2px solid #F8B603;" |

<h3><font style="color:#F8B603;">Background</font></h3>

Plants are dependent on light for their photosynthetic growth. In order to optimize their growth, plants have evolved a sophisticated set of photoreceptors and light responses that are used to interpret and respond to their light environment<sup>1</sup>.  At the seedling stage, light triggers the switch from heterotrophic to photoautotrophic growth at soil emergence. Later in life, light is used to determine season (and hence flowering<sup>2</sup>) and neighbor proximity; close neighbors can trigger a shade avoidance response<sup>3</sup>.
Interestingly, the correct response to a specific light cue depends on the environmental context.  Thus, plants native to different light environments have evolved different adaptive responses.  One clear example is that of shade avoidance.  Light that is transmitted through or reflected from neighboring plants is reduced in its red to far-red (R/FR) ratio, because chlorophyll absorbs red but not far-red light.  The phytochrome photoreceptors detect this change in light quality and trigger a suite of responses that increase light competiveness, including stem and petiole elongation and early reproduction<sup>3</sup> (Figure 1).

[[Image:Research_fig_1.jpg | center | Figure 1]]

Plants native to sunny environments are very sensitive to shade from their neighbors and show robust shade avoidance responses, allowing them to compete for sunlight.  In contrast, shade avoidance is much reduced in plants native to constitutively shady environments such as forest understory.  Such changes in shade avoidance response are adaptive and are due to heritable genetic differences<sup>4-6</sup>.  Although mutational studies have defined a number of genes involved in phytochrome signaling<sup>7,8</sup>, the genes and molecular changes responsible for adaptive changes in light response remain unknown.

Until very recently, most studies of population or species differences have been descriptive rather than mechanistic.  Recent developments in genomics are changing the level at which questions about adaptation and evolution can be asked.  We are taking a molecular-genetic approach to the study of plant light adaptation and are interested in determining which genes are responsible for adaptive changes in light response, the mechanisms by which changes in these genes affect light signaling, and the evolutionary forces that have acted on these genes.  Because of the fundamental importance of light perception to plant growth, the genomics tools and information available in the "model" plant ''Arabidopsis thaliana'', and the long history of light signaling research, this is an ideal system in which to study the molecular basis of adaptive responses. Our general approach is to study natural variation in ''Arabidopsis thaliana'' light signaling to find genes important for variation in light response. Once genes causing natural variation in shade avoidance have been defined, these genes (and changes in them) can be studied at mechanistic, evolutionary, and ecological levels. These studies can provide insight into the molecular basis of quantitative variation and adaptation. In addition, this work could be helpful for crop improvement since shade avoidance reduces crop yield under crowded conditions

<h3><font style="color:#F8B603;">Prior Work</font></h3>

To determine if ''A. thaliana'' is an appropriate organism for study of light adaptation, we characterized the extent of light response variation in 140 ''A. thaliana'' accessions using a simple response, seedling emergence<sup>9</sup>. A wide range of heritable differences in light response was found (Figure 2).

[[Image:Research_fig_2.jpg  | center | Figure 2]]

Interestingly, there was a significant inverse correlation between latitude and light sensitivity, suggesting adaptation to an environmental factor that varies over latitudinal clines (perhaps light itself). Some accessions had novel response patterns whereas others had patterns suggesting changes in known pathways. One accession, ''Lm-2'', was insensitive to far-red light, similar to phytochromeA (phyA)  mutants. We found that ''Lm-2'' fails to complement phyA  due to a single amino acid change and that this change causes production of a protein that is less sensitive to light. Biochemical characterization of ''Lm-2'' phytochrome demonstrated that it has reduced kinase activity and altered spectral properties.
We used Quantitative Trait Loci (QTL) mapping to identify loci involved in light-response variation for seedling emergence between two other strains, ''Ler'' and ''Cvi'' (Figure 3-4), and identified an average of four loci per light condition examined<sup>10</sup>.


[[Image:Research_fig_3.jpg | center | Figure 3]]
[[Image:Research_fig_4.jpg | center | Figure 4]]

Some loci map to regions with no known photomorphogenic mutants, suggesting that new genes involved in light response have been identified. In contrast, one QTL maps near PHYTOCHROMEB (PHYB),  known to be important for response to red light. We created a near-isogenic line (NIL) that confirmed that this region is important for light response, and preliminary results from transgenic plants suggest that PHYB  may indeed be the QTL. Strikingly, association testing suggests that the PHYB  region is an important determinant of light response across many ''A. thaliana''  accessions. Furthermore, sequence comparison between ''A. thaliana'' and ''Arabidopsis lyrata'' show that PHYB is evolving in a non-neutral fashion. Combined, these studies demonstrate that Arabidopsis  is an excellent organism for studying the molecular basis of natural variation in light response, and suggest that some changes in light response in Arabidopsis and its relatives is adaptive.

<h3><font style="color:#F8B603;">Current Projects</font></h3>

====QTL mapping of shade avoidance traits====


We are continuing to use QTL mapping to find genes important for variation in light response. Our preliminary studies focused on seedling emergence. We are now studying shade avoidance responses, because of the clear adaptive value of variation in shade avoidance responses<sup>4-6</sup> and possible benefit to agriculture. To separate phytochrome-mediated responses from those caused by reduced irradiance we are growing plants in three light conditions: high R/FR, low R/FR (simulating shade), and high R/FR but with photosynthetically active radiation (PAR) reduced to match that in the low R/FR environment. Preliminary results on one Recombinant Inbred Line (RIL) population demonstrates that there is substantial genetic variance in R/FR response (Figure 5; p < 0.0001).


[[Image:Research_fig_5.jpg | center | Figure 5]]

In each environment we are measuring a variety of shade avoidance characters, including hypocotyl, stem, and petiole elongation, leaf angle, leaf shape, and flowering time.   This work is part of an NSF funded collaboration with Cynthia Weinig  at the University of Minnesota who is examining these lines under low and high competitive conditions in the field to study group versus individual selection.

====Expression Profiling and eQTL mapping====

Recent reports demonstrate that EST or oligo microarrays can be used to detect significant natural variation in the transcriptome among strains of yeast, flies, fish, maize, mice, and humans<sup>11-14</sup>. In several of these studies, the expression levels were used as traits for linkage or QTL analysis. Notably, about one third of the transcripts with expression differences in yeast and mice are closely linked to loci or expression QTL (eQTL) predicted to control their expression differences<sup>11-14</sup>. For these genes it is likely that the differences in expression are caused by changes in the regulatory sequences or the transcript itself, meaning that a strong candidate gene for the eQTL has been identified.

To better understand the mechanisms underlying variation in Arabidopsis shade avoidance traits, we are coupling expression profiling with the classical QTL mapping experiments described above.   We hypothesize that eQTL that control expression differences in low and high R/FR and that co-localize with QTL for shade avoidance traits are likely to be causal for those traits. Thus we can learn about the molecular effects of particular QTL by examining the types of genes that it regulates. For example, a QTL affecting expression primarily of cell wall enzymes likely acts downstream in the shade avoidance pathway, whereas a QTL controlling expression of transcription factors may be more upstream. Comparing genes affected by eQTL with those controlled by known shade avoidance regulators will further help define the mechanisms of QTL action.

====QTL cloning====

QTL maps typically resolve the involved loci into regions that contain hundreds of genes. However a number of QTL have been recently cloned, demonstrating that it is possible to identify the causative genes. The full genome sequence, abundant genetic markers, and nearly saturating gene knock-out collection make Arabidopsis perhaps the premier multi-cellular organism for QTL cloning<sup>15</sup>. To better understand the molecular basis of quantitative genetic variation and to find the genes controlling variation in shade-avoidance, the full suite of Arabidopsis molecular genetic tools is being used to clone QTL identified in the lab.

====Functional molecular evolution====

Recently developed likelihood models of codon based nucleotide substitution greatly increase the ability to detect positive selection and allow prediction of particular codons that have been subject to positive selection<sup>16-18</sup>. Thus, evolutionary data can be used to predict functionally interesting amino acid residues. PHYB is an ideal candidate for the application of these methods. While selective pressure on R/FR sensitivity could affect any gene in the PHYB pathway, there is evidence that PHYB itself is under selection. First, phys are evolving more rapidly than average plant genes<sup>19</sup>; second, we found that PHYB co-localizes with a quantitative trait locus (QTL) affecting response to white and red light in Arabidopsis<sup>10</sup>; and last, our unpublished analysis of ''A. thaliana'' and ''A. lyrata'' PHYB sequences suggests positive selection using the McDonald-Krietman test<sup>20</sup>. Codon based substitution models will be used to analyze PHYB sequence across the brassicaceae. Site-directed mutagenesis of interesting residues will be used to test the functional consequence of amino acid substitution using transgenic Arabidopsis and in vitro tests. This work is part of a HFSP funded collaboration with Ulrich Genick and Christian Fankhauser to explore variation in Phy structure and function.

<h3><font style="color:#F8B603;">Bibliography</font></h3>

#Neff, M. M., Fankhauser, C. & Chory, J. Light: an indicator of time and place. Genes Dev 14 , 257-71 (2000).
#Yanovsky, M. J. & Kay, S. A. Living by the calendar: how plants know when to flower. Nat Rev Mol Cell Biol 4 , 265-75 (2003).
#Smith, H. Phytochromes and light signal perception by plants--an emerging synthesis. Nature 407 , 585-91 (2000).
#Dudley, S. A. & Schmitt, J. Testing the adaptive plasticity hypothesis: Density-dependent selection on manipulated stem length in Impatiens capensis. American Naturalist 147 , 445-465 (1996).
#Schmitt, J., Dudley, S. A. & Pigliucci, M. Manipulative approaches to testing adaptive plasticity: Phytochrome-mediated shade-avoidance responses in plants. American Naturalist 154 , S43-S54 (1999).
#Dorn, L. A., Pyle, E. H. & Schmitt, J. Plasticity to light cues and resources in Arabidopsis thaliana: Testing for adaptive value and costs. Evolution 54 , 1982-1994 (2000).
#Hudson, M. E. The genetics of phytochrome signalling in Arabidopsis. Semin Cell Dev Biol 11 , 475-83 (2000).
#Quail, P. H. Phytochrome photosensory signalling networks. Nat Rev Mol Cell Biol 3 , 85-93. (2002).
#Maloof, J. N. et al. Natural variation in light sensitivity of Arabidopsis. Nat Genet 29 , 441-6 (2001).
#Borevitz, J. O. et al. Quantitative trait loci controlling light and hormone response in two accessions of Arabidopsis thaliana. Genetics 160 , 683-96 (2002).
#Brem, R. B., Yvert, G., Clinton, R. & Kruglyak, L. Genetic dissection of transcriptional regulation in budding yeast. Science 296 , 752-5 (2002).
#Jin, W. et al. The contributions of sex, genotype and age to transcriptional variance in Drosophila melanogaster. Nat Genet 29 , 389-95 (2001).
#Oleksiak, M. F., Churchill, G. A. & Crawford, D. L. Variation in gene expression within and among natural populations. Nat Genet 32 , 261-6 (2002).
#Schadt, E. E. et al. Genetics of gene expression surveyed in maize, mouse and man. Nature 422 , 297-302 (2003).
#Borevitz, J. O. & Nordborg, M. The impact of genomics on the study of natural variation in Arabidopsis. Plant Physiol 132 , 718-25 (2003).
#Nielsen, R. & Yang, Z. Likelihood models for detecting positively selected amino acid sites and applications to the HIV-1 envelope gene. Genetics 148 , 929-36 (1998).
#Yang, Z., Nielsen, R., Goldman, N. & Pedersen, A. M. Codon-substitution models for heterogeneous selection pressure at amino acid sites. Genetics 155 , 431-49 (2000).
#Yang, Z. & Nielsen, R. Codon-substitution models for detecting   molecular adaptation at individual sites along specific lineages. Molecular   Biology and Evolution 19 , 908-17 (2002).
#Alba, R., Kelmenson, P. M., Cordonnier-Pratt, M.-M. & Pratt, L. H. The phytochrome gene family in tomato and the rapid differential evolution of this family in angiosperms. Molecular Biology and Evolution 17 , 362-373 (2000).
#McDonald, J. H. & Kreitman, M. Adaptive protein evolution at the Adh locus in Drosophila. Nature 351 , 652-4 (1991).
|}