'''University of Connecticut Department of Molecular and Cell Biology Course 214'''

[[MCB214:Overview]]

'''Course Lectures'''
Each semester the overall project for the course is different. Because the experiment is producing real data, its generally desirable to change the goals. Thus some of the material below may not be introduced each semester. However, regardless of specific experiment we generally incorporate many of the same techniques that we consider fundamental to work in molecular genetics.



==Micropipettors==
Introduction and review to proper micropipettor operation. Should cover basics of gripping the pipette, and typical precautions.

===Basic Parts===
*Plunger
*Body & Barrel
*Ejection button
*Digital display
*Adjustment dial
*Ejector

===The micropipettes===
All listed ranges are what we use in class. Some may be smaller than the manufacturer’s recommended range.
*P10
**0.5-2.0 &mu;L <small>Note: As the name denotes the P10 actually functions from 0.5-10.0 &mu;L. Since the P20 is accurate in the range above 2.0 to the same degree it is often faster to use the P20. Practically, P10s tend to be the most demanded pipette and should be reserved for the smallest volumes.</small>
**Accurate to 0.1 &mu;L <small>Note: This applies to current blue Fisher models in the lab as of 1/2007. Other models may be accurate to 0.01 &mu;L</small>

*P20
**2.0-20.0 &mu;L <small>Can dial safely to 0.5 &mu;L, but the P10 is considerably more accurate below 2.0.</small>
**Accurate to 0.1 &mu;L

*P200
**20-200 &mu;L
**Accurate to 1 &mu;L

*P1000
**200-1000 &mu;L <small>Note: P1000 pipettes can be safely used from 100-1000 &mu;L. Because the P200 is more accurate below 200 &mu;L there is no reason to dial the P1000 below that point.</small>
**Accurate to 10 &mu;Ls. Thus if you need to pipette 808 &mu;L you can add 800 &mu;L with the P1000 but must use the P20 to pipette 8 additional &mu;L.

===Do’s & Don’ts===
*Always use the pipette with a tip. The barrel should never come in contact with anything.
**For class we will always use filtered ARTs (aerosol resistant tip).
**Be sure to choose the right tip for the pipette. Note that although you can fit a p20 tip on the P200 pipette, it will not work properly. You will pull liquid up into the filter and will not be able to eject.
*The pipette plunger has two stops. Be sure not to press beyond the first stop when preparing to draw up liquid. It is generally a good idea to use the second stop when ejecting to clear any liquid from the tip.
*Press down the plunger before the tip breaks the liquids surface. Otherwise you’ll blow air bubbles.
*If there is liquid in the tip the pipette should never be tilted such that the liquid could flow back into the barrel.
**<small>Except in gravity-free environments during our field trip to the Sun.</small>
*Apply and release pressure on the plunger slowly
**Letting go quickly will make air bubbles and could splash liquid up onto the filter or barrel.
*Grip the pipette in a fist like position so that the top of the index finger rests under the guard. The thumb should be in position to comfortably press the plunger and ejection button.
*Remember: Micropipettes are least accurate at the bottom end of their range. To pipette 2.0 &mu;L it is more accurate to use the P10 than the P20.
*Pipettes often will show digits if you dial above or below their set ranges. ''You are breaking the pipette.''
*It is generally easier to pipette small volumes (ie below 4 or 5 &mu;L) into a large volume of liquid. Break the surface of the receiving volume with the pipette tip before ejecting to ensure no volume loss.
*Be aware when pipetting particularly viscous solutions. Many enzymes are packaged in sticky chemicals such as glycerol. It is good practice to only push the tip in as far as needed to get adequate volume. Viscous materials will coat the outside of the tip and result in a large waste of reagents.

[[MCB214:Lectures#Top|To top of page]]

==Gel Electrophoresis==

The course exclusively involves running nucleotide gels. For the initial overview of the topic we cover the basic concept, matrices and buffers, and loading dyes. 

===The Concept===
Nucleic acids have an overall negative charge due to their phosphate backbone. Thus in an electric field they are repelled from the negative terminal. By placing DNA or RNA in a gel matrix we can apply an electric field and seperate the molecules by length. Gels are used for a variety of purposes in molecular biology. They can be simple diagnostics to see if a reaction (eg PCR) worked, or can be used to seperate DNA for another experiment (eg Southern Blot).

*Larger DNA molecules will travel at a slower rate than a smaller molecule.
*The resolving power of a gel depends on the concentration of the matrix, the voltage applied, and the duration of electrophoresis.
**Higher concentration gels spread DNA out more, resulting in slower movement, but better resolution.
**Raising the voltage will cause the DNA to migrate more rapidly. Bands cannot spread out as much at high voltage.
**Since the speed of a given length nucleic acid is constant in the gel, the longer a gel is run the more bands can spread out.
**Thus the best gel image is obtained from an appropriately high concentration gel run at low voltage over a long period.

===The Gel Matrix===
A gel is a liquid suspended in a solid. For electrophoresis, gels are made by polymerizing a compound dissolved in a water based solution. The gel forms a dense mesh which electricity pushes DNA molecules through. There are two standard matrices used in molecular biology.

*Agarose
**Polysaccharide purified from agar, which is derived from red algae or kelp.
**0.6%-5.0% gels typically the limit
**Electrophorese at voltage up to 120V

*Polyacrylamide
**'''Warning''' polyacrylamide monomers are neurotoxic and should be handled very carefully. Polymerized acrylamide, while nontoxic, may still contain monomers and is potentially dangerous.
**Polymer of acrylic acid ions, this is the same material used to make soft contact lenses.
**Concentration ranges from 5.0%-20.0%
**Voltage limited by thickness of gel and apparatus used to run. In a typical rig 200-300V is normal, though in a circulating

===Electrophoresis Buffers===
In class we generally only run DNA gels. Because of this there are only two buffers we ever use. We cover the components of the buffers, and what each chemical does.

*TBE
**'''T'''ris
***pH buffer
**'''B'''oric Acid
**'''E'''DTA
***ethylenediaminetetraacetic acid
***Sequesters divalent and trivalent cations
***Mg(II) is a cofactor for many DNA modifying enzymes

*TAE
**'''T'''ris
**'''A'''cetic Acid
**'''E'''DTA

To run RNA for a northern the gel must be made with formaldehyde, and RNA needs to be denatured before running. TBE can be used when running DNA on PAGE, but other buffers are used if running proteins. SDS-PAGE is beyond the scope of this course.

===Loading Dyes===
Protonated DNA is soluble, and so adding it directly to the well of a gel will allow it to disperse throughout the buffer solution. Loading dyes contain a material to add density and viscosity to samples, causing them to settle in the well. They also contain a colored dye that will act as a visual indicator of the progress of the gel. <small>''Note: The color in a loading dye does not interact with DNA and is not an indicator of where a particular band has migrated.''</small>

Various substances are used to weight samples:
*Ficoll
**High mass polysaccharide
*Sucrose
**Table sugar
*Glycerol
**Also called glycerin
**Sugar alcohol

There are also several different dyes available, generally added as a disodium salt.
*Cresol Red
*Orange G
*Bromophenol Blue

===Procedure===
#Pouring the gel
##Mass out desired amount of electrophoresis grade agarose
##Add appropriate amount of 1x TBE
##Bring to a boil in microwave, swirling occassionally
##*Avoid burns from steam.
##When completely clear, remove from microwave to cool.
##Add 1 &mu;L/100 mL of 0.01% Ethidium Bromide to the cooling gel
##When cooled to about 65&deg; C, pour into cast with comb.
##*A 1% agarose gel typically takes 20 minutes to completely polymerize. Do not disturb.
##*Always pour gel on a level surface.
#Preparing samples for electrophoresis
##Choose loading dye and add to sample to final concentration of 1x.
##*Dye fronts all run differently. For very small products dyes may need to be selected that do not obscure bands.
##*When running gels as a diagnostic of a previous experiment, typically a small portion of the sample is needed for the gel. Avoid adding loading dye, and possibly hampering downstream experiments, to your entire sample if its not all entering the gel. Take an appropriate volume to a small PCR tube, add loading dye, and dilute with TBE buffer or diH2O.
#Load samples
#*Avoid pushing air bubbles into the buffer. Pressing the pipette to its second stop will force extra air through, possibly dispersing your sample into the buffer.
#*The pipette tip does not need to be completely inside the well. Because of sucrose, glycerol or ficoll in the loading dye, your sample will fall into the well if the tip breaks of the surface of the buffer over the well. This can allow you to add more volume of sample.
#Run gel at a reasonable voltage.
#*This will vary with gel concentration, product sizes, and the purpose of the gel. For quick diagnostics (''eg'' Did this PCR work?) high voltage on a 0.8-1.2% gel for 45-60 minutes will give a good idea of what products you obtained, and a rough idea of size.
#*For large pools of different size products such as genomic digests, run at very low voltage overnight or longer. We typically will run a gel for Southern Blots at 30V for 20-24 hours. Image the gel periodically with UV light.

===Imaging===
Our labs and the MCB214 lab course prestain agarose gels in ethidium bromide. We do this by adding a dilute solution to the cooling gel mixture prior to pouring. In other labs and courses students may have post-stained by incubating the gel in a very dilute solution and then washing.

Ethidium bromide intercalates into the backbone of dsDNA. Because of internal basepairing, RNA has sufficient secondary for visualization with ethidium bromide. The molecule fluoresces a bright orange under UV light. When bound to a nucleic acid this fluorescence is about 20 times brighter.

Because it binds the DNA backbone, ethidium bromide is very strong mutagen. It is a potential carcinogen and teratogen. Always wear gloves when handling containers with ethidium. If gloves become contaminated they should be disposed of, and be very aware of what other objects you touch with that hand. Latex gloves offer very little protection from EtBr. Nitrile gloves provide a very good barrier.

===Comments===
# Even with a loading dye, samples will eventually diffuse into the buffer and be lost. Pipette carefully, but don't tarry.
# Be sure to load samples at the correct terminal. DNA and RNA will electrophorese towards the positive terminal, and should be loaded at the negative terminal.
#* Red = Positive
#* Black = Negative
# Ethidium bromide is a very strong carcinogen. If prestaining gels with EtBr treat the used buffer and gel rig as though it were contaminated. When handling the gel be sure to wear gloves, and be '''very''' aware of what you touch.
# Ultraviolet light damages DNA both in your cells and in your gel. If you are being exposed to UV light, make sure to wear UV protective goggles and shield all exposed skin. Prolonged exposure of your gel to UV light will also degrade the DNA, and might affect later use of that DNA if purifying samples.
# Be careful when loading samples not to disturb the gel. The bottom of the well can be punctured easily. Loading a sample too quickly can cause it to fly out of the well. Pipette slowly and steadily.

[[MCB214:Lectures#Top|To top of page]]

==Ethanol Precipitation==
===Summary===
Using alcohols to precipitate nucleic acids is a fundamental technique in molecular biology. It will be used following any [[MCB214:Lectures#Genomic_DNA_Extraction|DNA]] or [[MCB214:Lectures#Trizol_RNA_Extraction|RNA]] extraction. Dideoxy sequencing and ''in vitro'' transcription must also be followed by precipitation. In general this technique can be used any time a nucleic acid sample needs to be concentrated or cleaned of excess salts.

===Materials===
*Alcohol
**95-100% Ethanol is best for precipitations
**Isopropanol can be used if necessary (see [[MCB214:Lectures#Comments_2|comments]] below)
*Cation source
**Sodium acetate, pH 5.2: This is the most common salt used for recovery of DNA or RNA.
**Lithium chloride: Ideal for purifying large RNAs, can help exclude tRNAs and 5S RNAs.
**Sodium chloride: Only necessary if precipitating DNA from solutions with SDS.
**Ammonium acetate: Used if precipitating DNA from digested agarose (agarased) gels or to remove excess nucleotides without use of a spin column.

===Procedure===
#Add 2 volumes 100% Ethanol (or 2.5 volumes 95% Ethanol) and 1/10 volume of 3M Sodium acetate, pH 5.2
#Invert sample to mix and precipitate at -20&deg;C for 10 minutes (see [[MCB214:Lectures#Comments_2|comments]])
#Centrifuge at maximum speed for 10 minutes
#Decant ethanol
#Wash with 150 &mu;L 70% ethanol
#Centrifuge 2 minutes max speed
#Remove ethanol by pipetting or aspiration
#Air dry
#Resuspend in water or Tricine-EDTA buffer.

===Concept===
The goal is to purify a nucleic acid by removing it from its current solution and resuspending in a pure solvent. By resuspending in a reduced volume, precipitation can also be used to increase the concentration of the target molecules.
Protonated DNA is soluble in water, so the key to precipitation is getting rid of the water. Ethanol and other alcohols are non-polar relative to water, and adequate strip DNA of the hydration shell. This leaves exposed charges from the phosphate backbone. The cation source (sodium in the above protocol) neutralizes this charge and helps desalt the nucleic acid.
Washing in 70% ethanol removes residual salts from the pellet, including the sodium acetate, while keeping the DNA from resuspending.

===Comments===
*The protocol given calls for 2 to 2.5 volumes of ethanol. In some cases (see [[MCB214:Lectures#Trizol_RNA_Extraction|Trizol]] this is impractical. 1 volume of isopropanol will precipiate nucleic acids without addition of a cation source. This is helpful when constrained by volume. However, isopropanol pellets tend to be clear or gel-like and loosen from the tube bottom easily. Use of ethanol is preferable whenever possible.

[[MCB214:Lectures#Top|To top of page]]

==Genomic DNA Extraction==
===Summary===
Extracting DNA from tissue or cells is a critical and standard technique in genetics. We use phenol and chloroform to extract genomic DNA. In practice this extraction will always be followed by ethanol or isopropanol precipitation to purify and concentrate the collected DNA. Since alcohol precipitation is used in various other experiments, and is a stand-alone procedure, its protocol is listed in a seperate section.

===Materials===
*Phenol
*Chloroform:Isoamyl Alcohol (24:1) ''Note: This mix is stable and can be stored at room temperature''
*ProK
*Reagent B
**TEN
***0.1 M NaCl
***10 mM Tris pH 8.0
***1 mM EDTA
**1% SDS
*Tissue or cell pellet

===Procedure===
#Place tissue sample in microfuge tube
#Cover in equal volume of extraction buffer
#*Reagent B + 2mg/ml ProK
#Place tubes in water bath at 50&deg;C overnight
#Add equal volume phenol:chloroform:isoamyl alcohol (PCI) in 25:24:1 mix
#Shake tubes vigorously for 10 seconds
#Spin at max speed for 5 minutes
#Remove aqueous phase to new tube
#Repeat steps 4-7 until interphase space is clear
#*This will vary with DNA source. Liver or other organ tissue will take 3-5 rounds. Cells or phage may only require 1 or 2.
#Add equal volume chloroform:isoamyl alcohol 24:1 mix
#Shake vigorously for 10 seconds
#Spin at max speed for 5 minutes
#Remove aqueous phase to new tube
#Proceed to [[MCB214:Lectures#Ethanol_Precipitation|Precipitation]]

===Concept and Application===
ProK or Proteinase K is a serine protease produced by ''Tritirachium album''. The enzyme catalyzes hydrolysis of most peptide bonds. Its used in the extraction to digest the majority of native proteins in the tissue or cell sample. This has the two-fold benefit of making the DNA accessible while destroying potentially damaging nucleases.

Phenol and chloroform are both organic solvents that are used in the extraction to dissolve and denature proteins. Phenol alone is insufficient to denature RNases, so it is crucial to have another organic solvent present for RNA extractions. In addition the specific gravity of phenol is close to pure water (1.07 compared to 1.00). High salt content in the aqueous phase can result in the organic phase being on top, making extractions difficult. The high specific gravity of chloroform ensures that the aqueous phase is always on top after centrifugation. Isoamyl alcohol helps keep proteins denatured, but also functions as an anti-foaming agent.

The final pass with chloroform and isoamyl alcohol is intended to remove any residual phenol before proceeding to precipitation.

PCI is used whenever proteins need to be disabled and removed from a solution.

===Safety===
*When doing an extraction always work in the hood while wearing gloves and safety glasses. Do not remove samples from the hood until the final aqueous phase has been transferred.
*Leave all tips and tubes that have been exposed to phenol in the hood for several days to dry. All PCI waste must be stored for proper disposal.
*Chloroform is a carcinogen and also damages the liver and kidneys. Vapors will irritate the skin, eyes, mucous membranes and respiratory tract. Because it is volatile it must always be used in a fume hood.
*Isoamyl alcohol should not be inhaled or ingested, and is harmful if absorbed through the skin. It causes serious damage to the eyes. Keep away from open flames.
*ProK is also an irritant and should not contact the skin.
*Phenol is extremely toxic and highly corrosive. Exposure can cause severe burns, and it is harmful by inhalation. If phenol comes in contact with any part of body wash with soap and water for at least 15 minutes. Do not use ethanol.

[[MCB214:Lectures#Top|To top of page]]

==Trizol RNA Extraction==
===Intro===
Extracting RNA is, in principle, the same as a DNA extraction. In this one-step extraction, RNA is separated from DNA by using acidic phenol. Genomic DNA can be extracted from cells or tissues by overnight digestion with ProK in Reagent B. However, because of stability and risk of enzymatic degradation, RNA must be recovered quickly. Low concentrations of ionic detergents such as SDS denature proteins too slowly to be sufficient in preventing RNase degradation of the target molecules. Trizol uses the acid of guanidine isothiocyanate, a chaotropic agent. Such chemicals rapidly disrupt non-covalent molecular interactions; eg. Van der Waals forces and hydrogen bonding. As a result, proteins secondary and tertiary structures are dismantled. Thus, RNA in tissue in Trizol is fairly stable.

===Protocol===
See [[Oneill_Lab:Trizol|Trizol: Protocol]]

===Precautions===
*See [[MCB214:Lectures#Safety|Safety]] 
*RNA is a far less stable molecule than DNA. Extra care must be taken to keep samples free of contamination.
**Always wear clean gloves when performing an RNA extraction.
**Wipe down your workspace, instruments and containers with RNase ZAP from Ambion
***RNase ZAP is a corrosive irritant and should never come in contact with your skin or be inhaled.
**Use DEPC treated water and RNA stocks of isopropanol and chloroform for extractions and precipitations.
*RNA does not last long on the bench. Resuspended material should be stored at -80&deg;C. Avoid unnecessary freeze/thaw cycles by aliquoting out large samples.

[[MCB214:Lectures#Top|To top of page]]
 
==Vectors==
==Restriction Digests==
==Gel Purification==
==Polymerase Chain Reaction==
==Southern Blot==
==Dideoxy sequencing==
==cDNA synthesis==
==Molecular cloning==
==Genomic libraries==
==Contact==
*[[User:Seth]]

[[Category:MCB214]]