DESIGN & PREPARATION OF PCR PRODUCTS AS INSERTS ~ 85 BP AND LARGER: PRIMER AMPLIFICATION OF DNA
*
== A. Primer design for PCR ==
*For typical Biofusion construction, you want a PCR product which has an XbaI site upstream of your part, and SpeI, NotI, and PstI sites downstream of your part. <br>Then, the forward primer should be of the form: <br>5' ''CCTT'''''TCTAGA''' (15-20 bp of the coding strand) 3' <br>and the reverse primer should be of the form: <br>5' ''AAGG'''''CTGCAGCGGCCGCTACTAGT''' (15-20 bp reverse complement) 3'<br> Here there are a four nucleotides (in italics) flanking the restriction sites (in bold); such spacers are required to allow the restriction enzymes to cut properly.
*Design considerations.  Make sure that:
** The T<sub>m</sub> of each primer is between 55-65 °C
** The T<sub>m</sub> of both primers are very similar, i.e., within ~2 °C
** Either primer will not form a stable internal hairpin structure, i.e., &Delta;G <-3 kcal/mol
** The forward and reverse primers do not combine to form a stable hairpin structure or dimer
** The genomic DNA to be amplified does not contain any EcoRI, PstI, SpeI, XbaI, or AarI sites 
** The insert for the forward primer does not begin with TC (or else a DAM I site (GATC) is formed, and XbaI cannot cut) 
** If possible the 3' end of each primer should end with a GC
*An easy way to design primers is to use [http://www.invitrogen.com/content.cfm?pageid=10071 Vector NTI].
*# Find the genomic DNA sequence that you want to amplify as your part at [http://www.yeastgenome.org yeastgenome.org] and save it into Vector NTI.
*# Highlight the sequence that you want as your part, and select Analyses -> Primer Design -> Amplify Selection.  
*# Under the Primer tab, set "Before" and "After" to 0 bp. Adjust the T<sub>m</sub>, primer length, GC content, et c. as noted above.  Also click More>> and insert the flanking sense and anti-sense sequences (given at top) in the boxes "Attach to 5' terminus of Sense primer" and "Attach to 5' terminus of Anti-sense primer". Lastly, click "Apply" then "OK."
*# Three possible sets of designed primers will appear in a folder on the left side of the screen, ranked by their score. Usually, the best possible score is 171.  The lowest score that I (Caroline) have successfully used is ~120. 
*# Double-check your primers by highlighting a given sequence, then right-clicking and selecting "Analyze".
*#* For more information, check out Vector NTI's [http://www.invitrogen.com/content.cfm?pageid=10141 user manual], Chapters 8 and 20.
*Order 25 nmol DNA oligo with standard desalting from [http://www.idtdna.com IDT].
*Note: if it is not possible to make a good set of primers with the flanking regions described above, try changing the first 4 bases - which are external to the restriction site - of each primer, e.g. <br>5' ''AAGG'''''TCTAGA''' (15-20 bp of the coding strand) 3' <br>
*Note: if you are still not able to get a good set of primers, try using a completely different set of flanking regions to improve the primers. For example, you can also use a PCR product that has the EcoRI, NotI, and XbaI sites upstream of your part, while the SpeI site is downstream of your part. In this case, the forward primer would be of the form:  5' ''CCTT'''''GAATTCGCGGCCGCATCTAGA''' (15-20 bp complement to coding strand)3' and the reverse primer should be of the form: <br> 5' ''AAGG'''''ACTAGT''' (15-20 bp complement to coding strand) 3'.

== B. PCR from yeast chromosomal or plasmid template ==

Both are known to work.
My two cents (Caroline): Using Vent (condition A) works for most (>90%) parts. However, there have been few parts for which I couldn't get pcr products using condition A. I have been able to pcr out these difficult parts using Pfx (condition B) -- Pfx has worked for *all* pcr reactions I've tried.
=== Condition A: Vent polymerase ===
#Resuspend each primer in Tris buffer pH 8.0 or distilled water to 100 µM.  
#Mix:
#*5 µL 10x ThermoPol buffer
#*0.4 µL 25 mM dNTPs
#*0.5 µL 100 µM forward primer
#*0.5 µL 100 µM reverse primer
#*≤1 µL plasmid DNA or 2 µL genomic DNA
#*1 µL Vent DNA polymerase
#*distilled water to 50 µL total volume
#PCR program:
#*Start: 95 °C for 2 min. (melt)
#*Cycle 95 °C for 0.5 min (melt)
#*Tm minus 5 °C for 0.5 min. (anneal)
#*74 °C for (# bp/1000) min. (extension) - no less than 0.5 min.
#*No. of Cycles: 30
#*End: keep at 4 °C forever

=== Condition B: Pfx Polymerase ===
#Resuspend each primer in Tris buffer pH 8.0 or distilled water to 100 µM.  
#Use Stratagene's Pfx kit. 
#Mix:
#* 3 µL primer mix (10µM of each primer)
#* 0.8 µL template DNA
#* 25 µL 10X PFx amplification buffer
#* 3 µL 10mM dNTPs
#* 2 µL 50mM MgSO<sub>4</sub>
#* 30 µL 10X PFx enhancer buffer
#* 34.2 µL water (to 100 µL)
#* 2 µL PFx DNA polymerase
# PCR Program
#*Start: 94 °C for 5 min. (melt)
#*Cycle 94 °C for 15 sec (melt) (cycle start)
#*55 °C for 0.5 min. (anneal)
#*68 °C for 3.5 min (extension) (cycle end)
#*68 °C for 7 min
#*No. of Cycles: 35
#*End: keep at 4 °C forever

== C. Gel analysis and purification of PCR products ==
* Run the PCR reaction on an appropriate percentage agarose gel and purify using Qiagen's PCR purification kit. Elute with 30 µL or another volume depending on the band's brightness.  

[[Category:Protocol]]
[[Category:DNA]]
[[Category:In vitro]]