[[Image:Zen garden Nijman.png|750px]]
<div style="padding: 10px; color: #ffffff; background-color: #000; width: 730px">
<center>

[[Nijman:Contact | <font face="trebuchet ms" style="color:#ffffff"> '''Contact''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Nijman:Positions | <font face="trebuchet ms" style="color:#ffffff"> '''Positions''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Nijman:Lab Members | <font face="trebuchet ms" style="color:#ffffff"> '''Lab Members''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Nijman:Publications | <font face="trebuchet ms" style="color:#ffffff"> '''Publications''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Nijman:Links | <font face="trebuchet ms" style="color:#ffffff"> '''Links''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
[[Nijman:Other | <font face="trebuchet ms" style="color:#ffffff"> '''Other''' </font>]] &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;

</center>
</div><br>

[[Image:Quote Asimov.png |450px|right]]

[[Category:Nijman]]

==Ce-M-M-==
[[Image:Logo CeMM 3.gif|left|400px|Logo CeMM 3.gif]] The Nijman lab is the latest addition to [http://www.cemm.oeaw.ac.at/ CeMM]. CeMM is a novel, international, independent and interdisciplinary research Center in Molecular Medicine of the Austrian Academy of Sciences. “From the clinic to the clinic”: Driven by medical needs, CeMM integrates basic research and clinical expertise to pursue innovative diagnostic and therapeutic approaches focused on cancer, inflammation and immune disorders. At the center of CeMM’s interest are patients and associated diseases. CeMM’s mission is to combine insight obtained from basic and clinical research and use it to implement the development of innovative therapeutic and diagnostic strategies – from the clinic to the clinic. Located at the [http://en.wikipedia.org/wiki/Vienna_General_Hospital Vienna General Hospital] (AKH), Austria’s largest medical research complex, CeMM functions as bidirectional channel between basic research and clinical applications. CeMM provides access to post-genomic technologies for the biomedical community and is a training and teaching center for a new generation of researchers in molecular medicine. CeMM’s innovative approaches are based on a novel, post-genomic, molecular understanding of biological and pathological systems.The goal of CeMM is to assist in preparing the predictive, preventive and personalized medicine of the future.

==Research scope==

[[Image:Cancer vs crab.jpg|200px|right]] In the post-genomic era, a major challenge is to understand the molecular networks that allow the cell perform it's physiological function. Within the Nijman lab the aim is to produce high quality science that contributes to the understanding of the cellular circuitry in health and disease with a particular focus on cancer. In order to achieve this, the lab employs state-of-the-art technology, [[genomics]] and bioinformatics.


''Figure (right). Breast cancer in a mastectomy specimen (top). The cancerous tumour (pale yellow) resembles the figure of a crab, giving the disease its name.
''

==Projects==
===Synthetic lethal interactions in mammalian cells===
[[Image:Quote1.png|400px|right]] During the last 5 years,'' in vitro'' human cell genetics has become within reach for many laboratories around the world. One of the challenges for the coming years is to use the available tools in new and creative ways to fully exploit the power of experimental genetics. In this context, a major focus of the lab is the high-throughput assessment of [[synthetic lethality|synthetic lethal/sick]] (SSL) interactions using chemical compounds and [http://en.wikipedia.org/wiki/Rnai RNAi]. In particular we are trying to identify genotype-specific cancer vulnerabilites or <nowiki>"</nowiki>[http://en.wikipedia.org/wiki/Achilles%27_heel Achilles' heels]<nowiki>"</nowiki>. However, the identification of SSL interactions can be useful for the study of any cell biological process and is thus not limited to the study of cell growth.

===Cell circuits and cancer===
====Ubiquitin dynamics====
Other projects in the lab are generally focussed on cancer-relevant pathways and signaling dynamics, such as [http://en.wikipedia.org/wiki/Ubiquitination ubiquitination]/deubiquitination. To identify deubiquitinating enzymes (DUBs) in pathways or processes of interest we employ [http://en.wikipedia.org/wiki/Rnai RNAi] libraries directed against all DUBs in the human genome. Using this approach we have previously identifid DUBs in [http://en.wikipedia.org/wiki/NF-kB NF-kappaB] signaling, DNA repair and [http://en.wikipedia.org/wiki/Hypoxia_%28medical%29 hypoxia] signaling. 

[[Image:Ubiquitin cartoon.png|250px|right]] <br>

''Figure (right). Ribbon representation of ubiquitin protein, highlighting the secondary structure. α-helices are coloured in blue and the β-sheet in green. The typical attachment point for a further ubiquitin molecule in polyubiquitin chain formation, lysine 48, is shown in pink.''

====Genetic screens====
The lab also employs genome wide gain-of-function and loss-of-function screens using cDNA and shRNA libraries, respectively. 
[[Image:TFE3 rescue.png|300px|left]] 




''Figure (left). Colony formation assay showing that overexpression of TFE3 can by-pass the temperature sensitive growth arrest induced by pRB. Figure taken from Nijman et. al. JBC (2006) [http://www.jbc.org/cgi/content/full/281/31/21582 Free access].'' <br>








For instance, we identified the transcription factor TFE3 as a regulator of cell-cycle arrest induced by the Retinoblastoma tumor suppressor gene ([http://en.wikipedia.org/wiki/Retinoblastoma_protein pRB]).

==Collaborations==
Collaborations are crucial for modern day science!

Listed here are some (science) friends and collaborators:

[http://www.med.nyu.edu/people/huangt03.html Tony Huang] (NYU)

[http://www.wi.mit.edu/research/fellows/brummelkamp.html Thijn Brummelkamp] (MIT, Whitehead Institute)

[http://www.biomedicalgenetics.nl/Members/Bernards/bernards.html Rene Bernards ] (Netherlands Cancer Institute)

[http://www.broad.mit.edu/about/bios/bio-golub.html Todd Golub] (Broad Institute, MIT/Harvard)

[http://www.broad.mit.edu/about/bios/bio-root.html Dave Root] (Broad Institute, MIT/Harvard)


http://openwetware.org/images/9/96/02_JoinOpenWetWare.png