==Background==
Sometimes when designing DNA parts (e.g. promoters, RBSs etc.) there is a need to include spacer DNA that has no function other than to separate two other DNA sequences.  I've often been unsure how to go about designing such a spacer sequence.    To increase the likelihood of designing an adequately non-functional piece of DNA, I thought it would be a good idea to have a checklist of all the types of sequences you should make sure are '''not''' in the spacer sequence.  This list is unlikely to be comprehensive so please add anything else you can think of.  In addition if anyone has any sequences that they know to be entirely non-functional they might post them here.

==Sequences to avoid==
''Don't forget to consider both the spacer DNA and the flanking DNA lest you create an undesirable sequence overlapping your DNA and the flanking DNA sequences.''
===Operator sequences===
Repressor/activator binding sites, &sigma;-factor binding sites and promoters should all be avoided if possible. 
===Start codons===
e.g ATG<br>
===RBSs===
There is a lot of variability in RBS sequences.  The ''E.coli'' consensus RBS sequence can be found [http://www-lmmb.ncifcrf.gov/%7Etoms/sequencelogo.html here]
The Shine-Dalgarno sequence is -<br>
TAAGGAGGT<br>
Even a part of this sequence can induce translation if there is a start codon nearby.  See Chen et al.<cite>Chen</cite> for more details.

===Restriction Sites===
Depending on your construction strategy, you may want to avoid certain restriction sites in your spacer sequence.  If you are using the BioBricks standard assembly method for cloning, you need to avoid [http://parts.mit.edu/r/parts/htdocs/Assembly/restriction_enzymes.cgi these] restriction sites - EcoRI, XbaI, SpeI and PstI.

===Methylation Sites===
Depending on your cellular chassis, you may need to avoid sequences that could be methylated.  For example, GATC will be methylated in some ''E. coli'' strains.

===Repetitive Sequences===
Either self-repetitive or similar sequence to other non-coding sequences can be problematic for a variety of reasons such as recombination and difficulty to PCR.

===GC ratio===
It seems that maintaining a GC ratio close to the native organism's would be desirable.

===Promoters===

===Structured sequences===
If the sequence will be transcribed into RNA, you will want to make sure there isn't any significant secondary structure that may interfere with translation of your RNA or its function, if applicable.

==Translational stop sequence==

<bbpart>BBa_B0042</bbpart>: This sequence provides a stop codon in all 6 frames.

       123123123123
       TTAGTTAGTTAG
       AATCAATCAATC
       321321321321
 R1    TTA
 F2     TAG
 R3        TTA
 F3         TAG
 R2            TTA
 F1             TAG

==Tools==

*[http://www.lifesci.ucsb.edu/~maduro/random.htm Random DNA sequence generator]
*[http://slam.bs.jhmi.edu/cgi-bin/gd/gdRandDNA.cgi Random DNA sequence generator and link to Gene Design software]

==References==
<biblio>
Chen pmid=7528374
</biblio>

[[Category:Needs attention]]