==Presentation Notes==

<b>BioSketch (Harvard)</b>
(NB: Diagram drawn on board of a BioBricks circuit)

*RBS gene terminator promoter = INVERTER (ie. 1 -> 0)

*promoterA | RBS lac-gene terminator promoterB | RBS lambda-gene terminator promoterA | RBS reporter(MCherry)-gene terminator 


*temp represses lac-gene
*lac represses promoterB
*lamda represses promoterA
*UV represses lambda

<b>Problem:</b> mutant component caused 23bp to disappear from promoter
<b>Problem:</b> low yield after repeated insert/vector separation; Solution: use triple ligation, with ccdB gene (which kills bacteria) in the final vector to select only out the triple-ligated plasmid

<b>Creating BioBricks</b>

*PCR with E/X- and S/P-overhang primers used to create BioBrick

*|--A--|--B--|--C--|  -> |--A--|--C--| -> amplification
*amplify by cloning into plasmid, PCR using primers for ends of A and C 


==Berkeley 2005: Cryptography through conjugation==

In 2005, Berkeley attempted to build on Farren's work on ribostructures by using their specificity as a "lock and key" structure. From my knowledge it is a bit like a bio-RSA system (though they didn't specify this so I'm not positive I'm intrep. it correctly...). In general, they use conjugation in F and RP4 plasmid containing bacteria to move a "locked item" from one cell to another, which can then be unlocked by a "key" - this allows the transcription of a protein which causes backwards conjugation, whereupon the same process occurs backwards. Reporters (YFP, GFP, RFP) are used to determine success.

*RSA
In the RSA encryption system, each person has their own key which others can use to encrypt messages. For example, if person A wants to send a message to person B, person B would take person A's "public key" to encrypt his/her message and send it to A who has the corresponding "private decrypting key". This way there is no chance for interception, unless you want to factor very very large primes - which is very very painful.

*Conjugation
Conjugation is a system where bacteria which have a "fertility" (F, RP4) can mate with other bacteria, sharing large amounts of DNA (up to 100k)! The "fertility" is caused by the plasmid, usually "F" or "RP4" in their case. This plasmid has a region which encodes for TraJ protein, which initiates DNA transfer, and also an OriT origin of transfer. Berkeley took advantage of this by creating two structures per cell: a plasmid which contained the TraJ gene and one which contained the OriT origin of transfer; TraJ would be an immobile plasmid containing the key. When encoded it causes OriT + lock to be transformed into the second cell, whereupon the second cell's key is used for unlocking, reinitiating conjugation (through c1 and spo0A).

*Farren's Riboregulors
In 2004, Farren created a very specific system of post-transcriptional control of mRNA products by hiding a ribosome binding site (RBS) behind a hairpin through cis-repression. The binding, however, is not ideal; when one adds an mRNA with better binding specificity, the RBS is revealed. He showed through experiments that it was a very specific interaction. Behind this is the idea of lock and key.

*Conclusions
Berkeley was unable to get all of the constructs made because of time constraints and an inordinate amount of cloning. But if they could have, this would have been an amazing project I think due to its uniqueness and its applications; you have the basis for a "cellular internet" where one cell can act as a server and another a host requesting genetic information. Though for this to be truly effective, the efficiency of Farren's Riboregulators would have to be improved - possible avenue of exploration? 

*What could we do here
So my idea for biocryptography was different drastically from their approach, as it took use of a "biocard" which would have information readable only so many times before degradation (which was mentioned with the telomere counter idea). Prehaps a chip with a scaffold DNA on it which degrades over X time; to use it as a key you put it in this reader which throws the correct primers into the mix and sees if the scaffold folds; if it does it allows entry and triggers something to destroy the resulting nanostructure. Berkeley's idea, however, has a huge amount of potential I think because it is a cellular RSA; it would be interesting to see if we could do a proof-of-concept of this through prehaps synthesis? We could also expand on Farren's riboregulators.

*Good references
[[http://openwetware.org/wiki/IGEM:UC_Berkeley/2006 Website with 2005 ppt]]<br>
Isaacs, Dwyer DJ, Ding C, Pervouchine DD, Cantor CR, Collins JJ. "Engineered riboregulators enable post-transcriptional control of gene expression."<br>
[[http://parts2.mit.edu/wiki/index.php/UC_Berkeley_2005 Old 2005 site]]


==UT Austin: EdgeDetector==

<b>Goal:</b> Shine light on plate of bacteria, bacteria will define edges of light areas

<b>How to Make Bact Light-Sensitive:</b>
Combine cyanobacteria's light-sensitive pathway and E.coli's histidine kinase pathway

<b>Accomplished:</b>
Light-sensitive bacteria

<b>Didn't Accomplish (?):</b>
Edge-detection

<b>Suggestions:</b>
* Quorum Sensing, Ron Weiss (PMID: 15858574) and the Bacterial Picture (PMID: 16306980)
* Bacterial Pong - one light-hit, one dark-ened, conjugation and both glow

==UT Austin: Light wire==

Print a biochemical circuit board using light to define the wires.

<b>Two Inputs:</b> point sources of HSL and PAI, which are exogenous chemicals which cause bacteria to amplify the signal when the bacteria are in the dark.

<b>Wires:</b> Bacteria which are propagating the signal when in the dark.

<b>Output:</b> When both chemicals are present (ie. when the two lines of bacteria propagating HSL or PAI production join), a reporter is expressed. 

<b>Truth Table for AND Gate</b>
HSL      PAI      Reporter
0         0        0
1         0        0
0         1        0
1         1        1

==Penn State: Bacterial Relay Race==
http://parts2.mit.edu/wiki/index.php/Image:Strategies.jpg

<b>Runners:</b> Bacteria
<b>Baton:</b> HSL, sent forward by moving bacteria

<b>Go Signal:</b> messenger molecule, switches on propagation, causes flagella to move (motB expressed), bacteria sending initial Go Signal destroyed after start 

==UCSF: BioThermometer==

Goal: create a lawn of bacteria that will glow when low or high temperature is inputted, a la the early 90s shirts that glow when you put your hand on them

<b>Achieved</b>: screened 40+ heat/cold-shock promoters, found 4 that worked, amplified one (hybB) and found that it worked very well, tried to invert the same one and found that the system lost temperature-dependence and was constitutively off