{{Template:Paulsson header}}

From the [http://sysbio.med.harvard.edu/faculty/paulsson/ HMS, Department of Systems Biology website:]<br>

===Research Summary===
Life in single cells is dictated by chance: reactions that involve small numbers of molecules generate spontaneous fluctuations that then enslave all dependent processes. Such ‘noise’ can randomize developmental pathways, disrupt cell cycle control or force metabolites away from their optimal levels. It can also be exploited when heterogeneity is advantageous, or even to obtain more reliable and deterministic control. The goal of the laboratory is to identify and understand the guiding principles behind these different phenomena. To this end we derive mathematical methods to interpret fluctuations, develop experimental methods to count molecules in single cells, and combine the two to study the simplest natural and engineered networks. Different applications may use different organisms, but E. coli is the first choice.

The individual network analyses form three more comprehensive projects: 1) A study of basic dynamic differences between different motifs in replication, gene expression and metabolism. 2) A comparison of different evolutionary solutions to the same regulatory problem, exemplified by homeostatic noise suppression. 3) A thorough quantitative characterization of the perhaps most tractable life form on the planet, bacterial plasmid R1. This includes evaluating the molecular mechanisms behind replication control and DNA segregation in terms of precision, cost and selfishness/altruism.

The mathematical projects also aim to build a more systematic stochastic theory for biochemical systems, approximating structural classes of random processes collectively and concretizing the interpretations rather than the assumptions.