{{Template:Paulsson header}}
'''What's hot?  What's not?'''

Paste in ".ezp1.harvard.edu" between "journal.com" and "/articleID" to access full text.<br>

Go [http://dx.doi.org.ezp1.harvard.edu/ here] to get the article using its DOI (Digital Object Identifier)<br>

Who's responsible for keeping track of which journal?  Find out [[Paulsson:Journal responsibilities | here]].

[[Paulsson:Journal watch archive | Archive of previous entries]]

==List of Journals==

===Biophysical Journal===


===Cell===

*A metabolic sensor governing cell size in bacteria.
Weart RB, Lee AH, Chien AC, Haeusser DP, Hill NS, Levin PA.

Nutrient availability is one of the strongest determinants of cell size. When grown in rich media, single-celled organisms such as yeast and bacteria can be up to twice the size of their slow-growing counterparts. The ability to modulate size in a nutrient-dependent manner requires cells to: (1) detect when they have reached the appropriate mass for a given growth rate and (2) transmit this information to the division apparatus. We report the identification of a metabolic sensor that couples nutritional availability to division in Bacillus subtilis. A key component of this sensor is an effector, UgtP, which localizes to the division site in a nutrient-dependent manner and inhibits assembly of the tubulin-like cell division protein FtsZ. This sensor serves to maintain a constant ratio of FtsZ rings to cell length regardless of growth rate and ensures that cells reach the appropriate mass and complete chromosome segregation prior to cytokinesis.
[http://sfx.hul.harvard.edu/sfx_local?id=pmid:17662947&sid=libx:hul.harvard]

*A molecular caliper mechanism for determining very long-chain Fatty Acid length.
Denic V, Weissman JS.

Very long-chain fatty acids (VLCFAs) are essential lipids whose functional diversity is enabled by variation in their chain length. The full VLCFA biosynthetic machinery and how this machinery generates structural diversity remain elusive. Proteoliposomes reconstituted here from purified membrane components-an elongase protein (Elop), a novel dehydratase, and two reductases-catalyzed repeated rounds of two-carbon addition that elongated shorter FAs into VLCFAs whose length was dictated by the specific Elop homolog present. Mutational analysis revealed that the Elop active site faces the cytosol, whereas VLCFA length is determined by a lysine near the luminal end of an Elop transmembrane helix. By stepping the lysine residue along one face of the helix toward the cytosol, we engineered novel synthases with correspondingly shorter VLCFA outputs. Thus the distance between the active site and the lysine residue determines chain length. Our results uncover a mutationally adjustable, caliper-like mechanism that generates the repertoire of cellular VLCFAs.
[http://sfx.hul.harvard.edu/sfx_local?id=pmid:17719544&sid=libx:hul.harvard]

===EMBO===


===Genetics===


===Journal of Bacteriology===


===Journal of Chemical Physcis===


===Journal of Molecular Biology===


===Journal of Physical Chemistry-A===


===Journal of Physical Chemistry-B===


===Journal of Physical Chemistry-C===


===Journal of Physical Chemistry-D===


===Journal of Physical Chemistry-E===


===Journal of Statistical Physics===


===Journal of Theoretical Biology===
*'''Noisy attractors and ergodic sets in models of gene regulatory networks.'''
Andre S. Ribeiroa and Stuart A. Kauffman

We investigate the hypothesis that cell types are attractors. This hypothesis was criticized with the fact that real gene networks are noisy systems and, thus, do not have attractors [Kadanoff, L., Coppersmith, S., Aldana, M., 2002. Boolean Dynamics with Random Couplings. left angle brackethttp://www.citebase.org/abstract?id=oai:arXiv.org:nlin/0204062right-pointing angle bracket]. Given the concept of “ergodic set” as a set of states from which the system, once entering, does not leave when subject to internal noise, first, using the Boolean network model, we show that if all nodes of states on attractors are subject to internal state change with a probability p due to noise, multiple ergodic sets are very unlikely. Thereafter, we show that if a fraction of those nodes are “locked” (not subject to state fluctuations caused by internal noise), multiple ergodic sets emerge. Finally, we present an example of a gene network, modelled with a realistic model of transcription and translation and gene–gene interaction, driven by a stochastic simulation algorithm with multiple time-delayed reactions, which has internal noise and that we also subject to external perturbations. We show that, in this case, two distinct ergodic sets exist and are stable within a wide range of parameters variations and, to some extent, to external perturbations. [http://www.sciencedirect.com.ezp1.harvard.edu/science?_ob=ArticleURL&_udi=B6WMD-4NKJ0PH-1&_user=10&_coverDate=08%252F21%252F2007&_rdoc=16&_fmt=full&_orig=browse&_srch=doc-info(%2523toc%25236932%25232007%2523997529995%2523663700%2523FLA%2523display%2523Volume)&_cdi=6932&_sort=d&_docanchor=&view=c&_ct=29&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=65f6bbbc7040f5fb09ffed38abe7ff92]

===Lab on a Chip===


===Molecular Microbiology===

===Molecular Systems Biology===
*'''Construction of consecutive deletions of the Escherichia coli chromosome.'''
Jun-ichi Kato & Masayuki Hashimoto.

The minimal set of genetic information necessary and sufficient to sustain a functioning cell contains not only trans-acting genes, but also cis-acting chromosomal regions that cannot be complemented by plasmids carrying these regions. In Escherichia coli (E. coli), only one chromosomal region, the origin of replication has been identified to be cis-acting. We constructed a series of mutants with long-range deletions, and the chromosomal regions containing trans-acting essential genes were deleted in the presence of plasmids complementing the deleted genes. The deleted regions cover all regions of the chromosome except for the origin and terminus of replication. The terminus affects cell growth, but is not essential. Our results indicate that the origin of DNA replication is the only vital, unique cis-acting DNA sequence in the E. coli chromosome necessary for survival.
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v3/n1/full/msb4100174.html]


*'''Single-cell zeroth-order protein degradation enhances the robustness of synthetic oscillator.'''
Wilson W Wong, Tony Y Tsai & James C Liao.

In Escherichia coli, protein degradation in synthetic circuits is commonly achieved by the ssrA-tagged degradation system. In this work, we show that the degradation kinetics for the green fluorescent protein fused with the native ssrA tag in each cell exhibits the zeroth-order limit of the Michaelis–Menten kinetics, rather than the commonly assumed first-order. When measured in a population, the wide distribution of protein levels in the cells distorts the true kinetics and results in a first-order protein degradation kinetics as a population average. Using the synthetic gene-metabolic oscillator constructed previously, we demonstrated theoretically that the zeroth-order kinetics significantly enlarges the parameter space for oscillation and thus enhances the robustness of the design under parametric uncertainty.
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v3/n1/full/msb4100172.html]


*'''A synthetic gene network for tuning protein degradation in Saccharomyces cerevisiae.'''
Chris Grilly, Jesse Stricker, Wyming Lee Pang, Matthew R Bennett & Jeff Hasty.

Protein decay rates are regulated by degradation machinery that clears unnecessary housekeeping proteins and maintains appropriate dynamic resolution for transcriptional regulators. Turnover rates are also crucial for fluorescence reporters that must strike a balance between sufficient fluorescence for signal detection and temporal resolution for tracking dynamic responses. Here, we use components of the Escherichia coli degradation machinery to construct a Saccharomyces cerevisiae strain that allows for tunable degradation of a tagged protein. Using a microfluidic platform tailored for single-cell fluorescence measurements, we monitor protein decay rates after repression using an ssrA-tagged fluorescent reporter. We observe a half-life ranging from 91 to 22 min, depending on the level of activation of the degradation genes. Computational modeling of the underlying set of enzymatic reactions leads to GFP decay curves that are in excellent agreement with the observations, implying that degradation is governed by Michaelis–Menten-type interactions. In addition to providing a reporter with tunable dynamic resolution, our findings set the stage for explorations of the effect of protein degradation on gene regulatory and signalling pathways.
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v3/n1/full/msb4100168.html]


*'''Nano-enabled synthetic biology.'''
Mitchel J Doktycz & Michael L Simpson.

Biological systems display a functional diversity, density and efficiency that make them a paradigm for synthetic systems. In natural systems, the cell is the elemental unit and efforts to emulate cells, their components, and organization have relied primarily on the use of bioorganic materials. Impressive advances have been made towards assembling simple genetic systems within cellular scale containers. These biological system assembly efforts are particularly instructive, as we gain command over the directed synthesis and assembly of synthetic nanoscale structures. Advances in nanoscale fabrication, assembly, and characterization are providing the tools and materials for characterizing and emulating the smallest scale features of biology. Further, they are revealing unique physical properties that emerge at the nanoscale. Realizing these properties in useful ways will require attention to the assembly of these nanoscale components. Attention to systems biology principles can lead to the practical development of nanoscale technologies with possible realization of synthetic systems with cell-like complexity. In turn, useful tools for interpreting biological complexity and for interfacing to biological processes will result.
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v3/n1/full/msb4100165.html]

===Nature===

Letter

Nature 448, 947-951 (23 August 2007) | doi:10.1038/nature06072; Received 12 March 2007; Accepted 6 July 2007

*'''The effects of molecular noise and size control on variability in the budding yeast cell cycle'''

Stefano Di Talia1,2, Jan M. Skotheim2, James M. Bean1,3, Eric D. Siggia2 & Frederick R. Cross1

   1. The Rockefeller University,
   2. Center for Studies in Physics and Biology, The Rockefeller University, New York, New York 10021, USA
   3. Present address: Human Oncology and Pathogenesis Program, Memorial Sloan-Kettering Cancer Center, New York, New York 10021, USA.

Correspondence to: Frederick R. Cross1 Correspondence and requests for materials should be addressed to F.R.C. (Email: fcross@mail.rockefeller.edu).
Top of page
Abstract

Molecular noise in gene expression can generate substantial variability in protein concentration1. However, its effect on the precision of a natural eukaryotic circuit such as the control of cell cycle remains unclear. We use single-cell imaging of fluorescently labelled budding yeast to measure times from division to budding (G1) and from budding to the next division. The variability in G1 decreases with the square root of the ploidy through a 1N/2N/4N ploidy series, consistent with simple stochastic models for molecular noise. Also, increasing the gene dosage of G1 cyclins decreases the variability in G1. A new single-cell reporter for cell protein content allows us to determine the contribution to temporal G1 variability of deterministic size control (that is, smaller cells extending G1). Cell size control contributes significantly to G1 variability in daughter cells but not in mother cells. However, even in daughters, size-independent noise is the largest quantitative contributor to G1 variability. Exit of the transcriptional repressor Whi5 from the nucleus partitions G1 into two temporally uncorrelated and functionally distinct steps. The first step, which depends on the G1 cyclin gene CLN3, corresponds to noisy size control that extends G1 in small daughters, but is of negligible duration in mothers. The second step, whose variability decreases with increasing CLN2 gene dosage, is similar in mothers and daughters. This analysis decomposes the regulatory dynamics of the Start transition into two independent modules, a size sensing module and a timing module, each of which is predominantly controlled by a different G1 cyclin.

Letter

Nature 447, 210-212 (10 May 2007) | doi:10.1038/nature05764; Received 12 January 2007; Accepted 20 March 2007

*'''Maintaining a behaviour polymorphism by frequency-dependent selection on a single gene'''

Mark J. Fitzpatrick1, Elah Feder2, Locke Rowe2 & Marla B. Sokolowski1

   1. Department of Biology, University of Toronto at Mississauga, Mississauga, Ontario L5L 1C6, Canada
   2. Department of Ecology and Evolutionary Biology, University of Toronto, Toronto, Ontario M5S 3G5, Canada

Correspondence to: Marla B. Sokolowski1 Correspondence and requests for materials should be addressed to M.B.S. (Email: msokolow@utm.utoronto.ca).
Top of page
Abstract

Accounting for the abundance of genetic variation in the face of natural selection remains a central problem of evolutionary biology1, 2. Genetic polymorphisms are constantly arising through mutation, and although most are promptly eliminated3, polymorphisms in functionally important traits are common. One mechanism that can maintain polymorphisms is negative frequency-dependent selection on alternative alleles, whereby the fitness of each decreases as its frequency increases4, 5. Examples of frequency-dependent selection are rare, especially when attempting to describe the genetic basis of the phenotype under selection. Here we show frequency-dependent selection in a well-known natural genetic polymorphism affecting fruitfly foraging behaviour. When raised in low nutrient conditions, both of the naturally occurring alleles of the foraging gene (fors and forR) have their highest fitness when rare—the hallmark of negative frequency-dependent selection. This effect disappears at higher resources levels, demonstrating the role of larval competition. We are able to confirm the involvement of the foraging gene by showing that a sitter-like mutant allele on a rover background has similar frequency-dependent fitness as the natural sitter allele. Our study represents a clear demonstration of frequency-dependent selection, and we are able to attribute this effect to a single, naturally polymorphic gene known to affect behaviour.

===Nature Biotechnology===
*'''Engineering synthetic signaling proteins with ultrasensitive input/output control.'''
John E Dueber, Ethan A Mirsky & Wendell A Lim.

Many signaling proteins are built from simple, modular components, yet display highly complex signal-processing behavior. Here we explore how modular domains can be used to build an ultrasensitive switch—a nonlinear input/output function that is central to many complex biological behaviors. By systematically altering the number and affinity of modular autoinhibitory interactions, we show that we can predictably convert a simple linear signaling protein into an ultrasensitive switch.
[http://www.nature.com.ezp1.harvard.edu/nbt/journal/v25/n6/full/nbt1308.html]

===Nature Genetics===
*'''Evolution of chromosome organization driven by selection for reduced gene expression noise.'''
Nizar N Batada & Laurence D Hurst.

The distribution of genes on eukaryotic chromosomes is nonrandom, but the reasons behind this are not well understood. The commonly observed clustering of essential genes is a case in point. Here we model and test a new hypothesis. Essential proteins are unusual in that random fluctuations in abundance (noise) can be highly deleterious. We hypothesize that persistently open chromatin domains are sinks for essential genes, as they enable reduced noise by avoidance of transcriptional bursting associated with chromatin remodeling. Simulation of the model captures clustering and correctly predicts that (i) essential gene clusters are associated with low nucleosome occupancy (ii) noise-sensitive nonessential genes cluster with essential genes (iii) nonessential genes of similar knockout fitness are physically linked (iv) genes in domains rich in essential genes have low noise (v) essential genes are rare subtelomerically and (vi) essential gene clusters are preferentially conserved. We conclude that different noise characteristics of different genomic domains favors nonrandom gene positioning. This has implications for gene therapy and understanding transgenic phenotypes.
[http://www.nature.com.ezp1.harvard.edu/ng/journal/v39/n8/full/ng2071.html]

===Nature Methods===

Nature Methods - 4, 629 - 632 (2007)
Published online: 1 July 2007; | doi:10.1038/nmeth1064
*'''Label-free continuous enzyme assays with macrocycle-fluorescent dye complexes'''

Andreas Hennig1, 2, Hüseyin Bakirci1, 2 & Werner M Nau1

1 School of Engineering and Science, Jacobs University Bremen, Campus Ring 1, D-28759 Bremen, Germany.

2 These authors contributed equally to this work.
Correspondence should be addressed to Werner M Nau w.nau@jacobs-university.de
We introduce a new economic, convenient and general assay principle based on the reversible interaction of water-soluble macrocycles and fluorescent dyes. We show that amino acid decarboxylase activity can be continuously monitored by measuring changes in fluorescence, which result from the competition of the enzymatic product and the dye for forming a complex with a cucurbituril or calixarene macrocycle. The new assay provides a complementary method to the use of antibodies, radioactive markers and labeled substrates.

Nature Methods - 4, 633 - 636 (2007)
Published online: 1 July 2007; | doi:10.1038/nmeth1065
*'''lambdaN-GFP: an RNA reporter system for live-cell imaging'''
Nathalie Daigle & Jan Ellenberg

Gene Expression Unit, EMBL, Meyerhofstr. 1, D-69117 Heidelberg, Germany.
Correspondence should be addressed to Jan Ellenberg jan.ellenberg@embl.de
We describe a GFP-based RNA reporter system (lambdaN-GFP) to visualize RNA molecules in live mammalian cells. It consists of GFP fused to an arginine-rich peptide derived from the phage lambda N protein, lambdaN22, which binds a unique minimal RNA motif and can be used to tag any RNA molecule. lambdaN-GFP uses a small and easy to engineer RNA tag, reducing the likelihood of perturbing the function of the tagged RNA molecule.

Nature Methods - 4, 637 - 639 (2007)
Published online: 8 July 2007; | doi:10.1038/nmeth1069
*'''Luciferase-YFP fusion tag with enhanced emission for single-cell luminescence imaging'''

Hideto Hoshino1, Yoshihiro Nakajima1 & Yoshihiro Ohmiya1, 2

1 Research Institute for Cell Engineering, National Institute of Advanced Industrial Science and Technology (AIST), Osaka 563-8577, Japan.

2 Department of Photobiology, Hokkaido University, Graduate School of Medicine, Sapporo 060-8638, Japan.
Correspondence should be addressed to Hideto Hoshino ah-hoshino@aist.go.jp
Taking advantage of the phenomenon of bioluminescence resonance energy transfer (BRET), we developed a bioluminescent probe composed of EYFP and Renilla reniformis luciferase (RLuc)—BRET-based autoilluminated fluorescent protein on EYFP (BAF-Y)—for near-real-time single-cell imaging. We show that BAF-Y exhibits enhanced RLuc luminescence intensity and appropriate subcellular distribution when it was fused to targeting-signal peptides or histone H2AX, thus allowing high spatial and temporal resolution microscopy of living cells.

Nature Methods - 4, 641 - 643 (2007)
Published online: 8 July 2007; | doi:10.1038/nmeth1070
*'''Red-shifted Renilla reniformis luciferase variants for imaging in living subjects'''

Andreas Markus Loening1, Anna M Wu2 & Sanjiv Sam Gambhir1, 2

1 Molecular Imaging Program at Stanford, Departments of Radiology and Bioengineering, Bio-X Program, Stanford University, Stanford, California 94305, USA.

2 The Crump Institute for Molecular Imaging, Department of Molecular and Medical Pharmacology, Geffen School of Medicine at UCLA, Los Angeles, California 90025, USA.
Correspondence should be addressed to Sanjiv Sam Gambhir sgambhir@stanford.edu
The use of R. reniformis luciferase (RLuc) as a reporter gene in small-animal imaging has been hampered by its 481 nm peaked emission spectrum, as blue wavelengths are strongly attenuated in biological tissues. To overcome this, we generated variants of RLuc with bathochromic (red) shifts of up to 66 nm (547 nm peak) that also had greater stability and higher light emission than native RLuc.

Nature Methods - 4, 619 - 628 (2007)
Published online: 30 July 2007; | doi:10.1038/nmeth1072
*'''Caged compounds: photorelease technology for control of cellular chemistry and physiology'''

Graham C R Ellis-Davies

Department of Pharmacology & Physiology, Drexel University College of Medicine, Philadelphia, Pennsylvania 19102, USA.
Correspondence should be addressed to Graham C R Ellis-Davies ged@drexel.edu
Caged compounds are light-sensitive probes that functionally encapsulate biomolecules in an inactive form. Irradiation liberates the trapped molecule, permitting targeted perturbation of a biological process. Uncaging technology and fluorescence microscopy are 'optically orthogonal': the former allows control, and the latter, observation of cellular function. Used in conjunction with other technologies (for example, patch clamp and/or genetics), the light beam becomes a uniquely powerful tool to stimulate a selected biological target in space or time. Here I describe important examples of widely used caged compounds, their design features and synthesis, as well as practical details of how to use them with living cells.

Nature Methods - 4, 555 - 557 (2007)
Published online: 17 June 2007; | doi:10.1038/nmeth1062
*'''Bright monomeric red fluorescent protein with an extended fluorescence lifetime'''

Ekaterina M Merzlyak1, Joachim Goedhart2, Dmitry Shcherbo3, Mariya E Bulina3, Aleksandr S Shcheglov3, Arkady F Fradkov3, Anna Gaintzeva1, Konstantin A Lukyanov3, Sergey Lukyanov3, Theodorus W J Gadella2 & Dmitriy M Chudakov3

1 Evrogen JSC, Miklukho-Maklaya 16/10, Moscow 117997, Russia.

2 Swammerdam Institute for Life Sciences, Section of Molecular Cytology, Centre for Advanced Microscopy, University of Amsterdam, Kruislaan 316, NL-1098 SM, Amsterdam, The Netherlands.

3 Shemiakin-Ovchinnikov Institute of Bioorganic Chemistry, Miklukho-Maklaya 16/10, 117997 Moscow, Russia.
Correspondence should be addressed to Dmitriy M Chudakov chudakovdm@mail.ru
Fluorescent proteins have become extremely popular tools for in vivo imaging and especially for the study of localization, motility and interaction of proteins in living cells. Here we report TagRFP, a monomeric red fluorescent protein, which is characterized by high brightness, complete chromophore maturation, prolonged fluorescence lifetime and high pH-stability. These properties make TagRFP an excellent tag for protein localization studies and fluorescence resonance energy transfer (FRET) applications.

Nature Methods - 4, 571 - 576 (2007)
Published online: 10 June 2007; | doi:10.1038/nmeth1058
*'''Recombinant RNA technology: the tRNA scaffold'''

Luc Ponchon & Frédéric Dardel

Cristallographie & RMN Biologiques, Université Paris Descartes, CNRS, 4 avenue de l'Observatoire, 75006, Paris, France.
Correspondence should be addressed to Frédéric Dardel frederic.dardel@univ-paris5.fr
RNA has emerged as a major player in most cellular processes. Understanding these processes at the molecular level requires homogeneous RNA samples for structural, biochemical and pharmacological studies. So far, this has been a bottleneck, as the only methods for producing such pure RNA have been in vitro syntheses. Here we describe a generic approach for expressing and purifying structured RNA in Escherichia coli, using tools that parallel those available for recombinant proteins. Our system is based on a camouflage strategy, the 'tRNA scaffold', in which the recombinant RNA is disguised as a natural RNA and thus hijacks the host machinery, escaping cellular RNases. This opens the way to large-scale structural and molecular investigations of RNA function.

Nature Methods - 4, 409 - 412 (2007)
Published online: 8 April 2007; | doi:10.1038/nmeth1040
*'''A genomic integration method to visualize localization of endogenous mRNAs in living yeast'''

Liora Haim, Gadi Zipor, Stella Aronov & Jeffrey E Gerst

Department of Molecular Genetics, Weizmann Institute of Science, Rehovot 76100, Israel.
Correspondence should be addressed to Jeffrey E Gerst jeffrey.gerst@weizmann.ac.il
mRNA localization may be an important determinant for protein localization. We describe a simple PCR-based genomic-tagging strategy (m-TAG) that uses homologous recombination to insert binding sites for the RNA-binding MS2 coat protein (MS2-CP) between the coding region and 3' untranslated region (UTR) of any yeast gene. Upon coexpression of MS2-CP fused with GFP, we demonstrate the localization of endogenous mRNAs (ASH1, SRO7, PEX3 and OXA1) in living yeast (Saccharomyces cerevisiae).

Nature Methods - 4, 413 - 419 (2007)
Published online: 1 April 2007; Corrected online: 04 April 2007 | doi:10.1038/nmeth1030
*'''Imaging dynamics of endogenous mitochondrial RNA in single living cells'''

Takeaki Ozawa1, 2, Yutaka Natori1, Moritoshi Sato1 & Yoshio Umezawa1

1 Department of Chemistry, School of Science, The University of Tokyo, Hongo Bunkyo-ku, Tokyo 113-0033, Japan, and Japan Science and Technology Corporation, Tokyo, Japan.

2 Department of Molecular Structure, Institute for Molecular Science, 38 Nishigonaka, Myodaiji, Okazaki, Aichi 444-8585, and PREST, Japan Science and Technology Agency, 4-1-8 Honcho Kawaguchi, Saitama, Japan.
Correspondence should be addressed to Yoshio Umezawa umezawa@chem.s.u-tokyo.ac.jp
We developed genetically encoded RNA probes for characterizing localization and dynamics of mitochondrial RNA (mtRNA) in single living cells. The probes consist of two RNA-binding domains of PUMILIO1, each connected with split fragments of a fluorescent protein capable of reconstituting upon binding to a target RNA. We designed the probes to specifically recognize a 16-base sequence of mtRNA encoding NADH dehydrogenase subunit 6 (ND6) and to be targeted into the mitochondrial matrix, which allowed real-time imaging of ND6 mtRNA localization in living cells. We showed that ND6 mtRNA is localized within mitochondria and concentrated particularly on mitochondrial DNA (mtDNA). Movement of the ND6 mtRNA is restricted but oxidative stress induces the mtRNA to disperse in the mitochondria and gradually decompose. These probes provide a means to study spatial and temporal mRNA dynamics in intracellular compartments in living mammalian cells.

===Plasmid===
*The incC korB region of RK2 repositions a mini-RK2 replicon in Escherichia coli.
Verheust C, Helinski DR.

Analysis by fluorescence microscopy has established that plasmid RK2 in Escherichia coli and other gram-negative bacteria is present as discrete clusters that are located inside the nucleoid at the mid- or quarter-cell positions. A mini-RK2 replicon containing an array of tetO repeats was visualized in E. coli cells that express a TetR-EYFP fusion protein. Unlike intact RK2, the RK2 mini-replicon (pCV1) was localized as a cluster at the cell poles outside of the nucleoid. Insertion of the O(B1)incC korB partitioning (par) region of RK2 into pCV1 resulted in a shift of the mini-replicon to within the nucleoid region at the mid- and quarter-cell positions. Despite the repositioning of the mini-RK2 replicon to the cellular positions where intact RK2 is normally located, the insertion of the intact O(B1) incC korB region did not significantly stabilize the mini-RK2 plasmid during cell growth. Deletions within the O(B1)incC or the korB region resulted in a failure of this par region to move pCV1 out of its polar position. The insertion of the par system of plasmid F into pCV1 resulted in a similar shift in the location of pCV1 to the nucleoid region. Unlike O(B1)incC korB, the insertion of the RK2 parABC resolvase system into pCV1 did not affect the polar positioning of pCV1. This effect of O(B1)incC korB on the location of pCV1 provides additional evidence for a partitioning role of this region of plasmid RK2. However, the failure of this region to significantly increase the stability of the mini-RK2 plasmid indicates that the localization of the plasmid to the mid- and quarter cell positions in E. coli is not in itself sufficient for the stable maintenance of plasmid RK2.
[http://sfx.hul.harvard.edu/sfx_local?id=pmid:17521722&sid=libx:hul.harvard]

===PLOS===


===PLOS Computational Biology===
*'''The Effect of Stochasticity on the Lac Operon: An Evolutionary Perspective'''
Milan van Hoek and Paulien Hogeweg

The role of stochasticity on gene expression is widely discussed. Both potential advantages and disadvantages have been revealed. In some systems, noise in gene expression has been quantified, in among others the lac operon of Escherichia coli. Whether stochastic gene expression in this system is detrimental or beneficial for the cells is, however, still unclear. We are interested in the effects of stochasticity from an evolutionary point of view. We study this question in the lac operon, taking a computational approach: using a detailed, quantitative, spatial model, we evolve through a mutation–selection process the shape of the promoter function and therewith the effective amount of stochasticity. We find that noise values for lactose, the natural inducer, are much lower than for artificial, nonmetabolizable inducers, because these artificial inducers experience a stronger positive feedback. In the evolved promoter functions, noise due to stochasticity in gene expression, when induced by lactose, only plays a very minor role in short-term physiological adaptation, because other sources of population heterogeneity dominate. Finally, promoter functions evolved in the stochastic model evolve to higher repressed transcription rates than those evolved in a deterministic version of the model. This causes these promoter functions to experience less stochasticity in gene expression. We show that a high repression rate and hence high stochasticity increases the delay in lactose uptake in a variable environment. We conclude that the lac operon evolved such that the impact of stochastic gene expression is minor in its natural environment, but happens to respond with much stronger stochasticity when confronted with artificial inducers. In this particular system, we have shown that stochasticity is detrimental. Moreover, we demonstrate that in silico evolution in a quantitative model, by mutating the parameters of interest, is a promising way to unravel the functional properties of biological systems. [http://compbiol.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pcbi.0030111]

===PNAS===
*'''Combinatorial promoter design for engineering noisy gene expression.'''
Kevin F. Murphy, Gábor Balázsi, and James J. Collins

Understanding the behavior of basic biomolecular components as parts of larger systems is one of the goals of the developing field of synthetic biology. A multidisciplinary approach, involving mathematical and computational modeling in parallel with experimentation, is often crucial for gaining such insights and improving the efficiency of artificial gene network design. Here we used such an approach and developed a combinatorial promoter design strategy to characterize how the position and multiplicity of tetO2 operator sites within the GAL1 promoter affect gene expression levels and gene expression noise in Saccharomyces cerevisiae. We observed stronger transcriptional repression and higher gene expression noise as a single operator site was moved closer to the TATA box, whereas for multiple operator-containing promoters, we found that the position and number of operator sites together determined the dose–response curve and gene expression noise. We developed a generic computational model that captured the experimentally observed differences for each of the promoters, and more detailed models to successively predict the behavior of multiple operator-containing promoters from single operator-containing promoters. Our results suggest that the independent binding of single repressors is not sufficient to explain the more complex behavior of the multiple operator-containing promoters. Taken together, our findings highlight the importance of joint experimental–computational efforts and some of the challenges of using a bottom-up approach based on well characterized, isolated biomolecular components for predicting the behavior of complex, synthetic gene networks, e.g., the whole can be different from the sum of its parts. [http://www.pnas.org.ezp1.harvard.edu/cgi/content/full/104/31/12726]

*'''Combinatorial transcriptional control of the lactose operon of Escherichia coli'''
Thomas Kuhlman, Zhongge Zhang, Milton H. Saier, Jr., and Terence Hwa

The goal of systems biology is to understand the behavior of the whole in terms of knowledge of the parts. This is hard to achieve in many cases due to the difficulty of characterizing the many constituents involved in a biological system and their complex web of interactions. The lac promoter of Escherichia coli offers the possibility of confronting "system-level" properties of transcriptional regulation with the known biochemistry of the molecular constituents and their mutual interactions. Such confrontations can reveal previously unknown constituents and interactions, as well as offer insight into how the components work together as a whole. Here we study the combinatorial control of the lac promoter by the regulators Lac repressor (LacR) and cAMP-receptor protein (CRP). A previous in vivo study [Setty Y, Mayo AE, Surette MG, Alon U (2003) Proc Natl Acad Sci USA 100:7702–7707] found gross disagreement between the observed promoter activities and the expected behavior based on the known molecular mechanisms. We repeated the study by identifying and removing several extraneous factors that significantly modulated the expression of the lac promoter. Through quantitative, systematic characterization of promoter activity for a number of key mutants and guided by the thermodynamic model of transcriptional regulation, we were able to account for the combinatorial control of the lac promoter quantitatively, in terms of a cooperative interaction between CRP and LacR-mediated DNA looping. Specifically, our analysis indicates that the sensitivity of the inducer response results from LacR-mediated DNA looping, which is significantly enhanced by CRP. [http://www.pnas.org.ezp1.harvard.edu/cgi/content/full/104/14/6043]

===PRLandE===


===Quarterly Reviews of Biophysics===


===Science===
*'''Noise in Gene Expression Determines Cell Fate in Bacillus subtilis.'''
Hédia Maamar, Arjun Raj and David Dubnau

Random cell-to-cell variations in gene expression within an isogenic population can lead to transitions between alternative states of gene expression. Little is known about how these variations (noise) in natural systems affect such transitions. In Bacillus subtilis, noise in ComK, the protein that regulates competence for DNA uptake, is thought to cause cells to transition to the competent state in which genes encoding DNA uptake proteins are expressed. We demonstrate that noise in comK expression selects cells for competence and that experimental reduction of this noise decreases the number of competent cells. We also show that transitions are limited temporally by a reduction in comK transcription. These results illustrate how such stochastic transitions are regulated in a natural system and suggest that noise characteristics are subject to evolutionary forces. [http://www.sciencemag.org.ezp1.harvard.edu/cgi/content/short/317/5837/526]

===Systems Biology===