{{Template:Paulsson header}}
'''What's hot?  What's not?'''

Paste in ".ezp1.harvard.edu" between "journal.com" and "/articleID" to access full text.<br>

Go [http://dx.doi.org.ezp1.harvard.edu/ here] to get the article using its DOI (Digital Object Identifier)<br>

Who's responsible for keeping track of which journal?  Find out [[Paulsson:Journal responsibilities | here]].

[[Paulsson:Journal watch archive | Archive of previous entries]]

==List of Journals==

===Biophysical Journal===


===Cell===
*Substrate Competition as a Source of Ultrasensitivity in the Inactivation of Wee1

Sun Young Kim1 and James E. Ferrell, Jr.

The mitotic regulators Wee1 and Cdk1 can inactivate each other through inhibitory phosphorylations. This double-negative feedback loop is part of a bistable trigger that makes the transition into mitosis abrupt and decisive. To generate a bistable response, some component of a double-negative feedback loop must exhibit an ultrasensitive response to its upstream regulator. Here, we experimentally demonstrate that Wee1 exhibits a highly ultrasensitive response to Cdk1. Several mechanisms can, in principle, give rise to ultrasensitivity, including zero-order effects, multisite phosphorylation, and competition mechanisms. We found that the ultrasensitivity in the inactivation of Wee1 arises mainly through two competition mechanisms: competition between two sets of phosphorylation sites in Wee1 and between Wee1 and other high-affinity Cdk1 targets. Based on these findings, we were able to reconstitute a highly ultrasensitive Wee1 response with purified components. Competition provides a simple way of generating the equivalent of a highly cooperative allosteric response.
[http://www.sciencedirect.com.ezp1.harvard.edu/science?_ob=MImg&_imagekey=B6WSN-4N9WKW2-F-2&_cdi=7051&_user=209690&_orig=browse&_coverDate=03%2F23%2F2007&_sk=998719993&view=c&wchp=dGLzVlz-zSkWA&md5=0251d42880645addc797c1f63cb355ae&ie=/sdarticle.pdf]

===EMBO===

===Genetics===


===Journal of Bacteriology===


===Journal of Chemical Physcis===


===Journal of Molecular Biology===


===Journal of Physical Chemistry-A===


===Journal of Physical Chemistry-B===


===Journal of Physical Chemistry-C===


===Journal of Physical Chemistry-D===


===Journal of Physical Chemistry-E===


===Journal of Statistical Physics===


===Journal of Theoretical Biology===


===Molecular Microbiology===
*The molecular basis of selective promoter activation by the σS subunit of RNA polymerase  (REVIEW)

Athanasios Typas, Gisela Becker and Regine Hengge

Different environmental stimuli cause bacteria to exchange the sigma subunit in the RNA polymerase (RNAP) and, thereby, tune their gene expression according to the newly emerging needs. Sigma factors are usually thought to recognize clearly distinguishable promoter DNA determinants, and thereby activate distinct gene sets, known as their regulons. In this review, we illustrate how the principle sigma factor in stationary phase and in stressful conditions in Escherichia coli, σS (RpoS), can specifically target its large regulon in vivo, although it is known to recognize the same core promoter elements in vitro as the housekeeping sigma factor, σ70 (RpoD). Variable combinations of cis-acting promoter features and trans-acting protein factors determine whether a promoter is recognized by RNAP containing σS or σ70, or by both holoenzymes. How these promoter features impose σS selectivity is further discussed. Moreover, additional pathways allow σS to compete more efficiently than σ70 for limiting amounts of core RNAP (E) and thereby enhance EσS formation and effectiveness. Finally, these topics are discussed in the context of sigma factor evolution and the benefits a cell gains from retaining competing and closely related sigma factors with overlapping sets of target genes.
[http://www.blackwell-synergy.com.ezp1.harvard.edu/doi/pdf/10.1111/j.1365-2958.2007.05601.x]

===Molecular Systems Biology===

===Nature===


===Nature Biotechnology===


===Nature Genetics===

===Plasmid===


===PLOS===


===PLoS Biology===


===PNAS===
*Combinatorial transcriptional control of the lactose operon of Escherichia coli

Thomas Kuhlman*, Zhongge Zhang, Milton H. Saier, Jr., and Terence Hwa*

The goal of systems biology is to understand the behavior of the whole in terms of knowledge of the parts. This is hard to achieve in many cases due to the difficulty of characterizing the many constituents involved in a biological system and their complex web of interactions. The lac promoter of Escherichia coli offers the possibility of confronting "system-level" properties of transcriptional regulation with the known biochemistry of the molecular constituents and their mutual interactions. Such confrontations can reveal previously unknown constituents and interactions, as well as offer insight into how the components work together as a whole. Here we study the combinatorial control of the lac promoter by the regulators Lac repressor (LacR) and cAMP-receptor protein (CRP). A previous in vivo study [Setty Y, Mayo AE, Surette MG, Alon U (2003) Proc Natl Acad Sci USA 100:7702–7707] found gross disagreement between the observed promoter activities and the expected behavior based on the known molecular mechanisms. We repeated the study by identifying and removing several extraneous factors that significantly modulated the expression of the lac promoter. Through quantitative, systematic characterization of promoter activity for a number of key mutants and guided by the thermodynamic model of transcriptional regulation, we were able to account for the combinatorial control of the lac promoter quantitatively, in terms of a cooperative interaction between CRP and LacR-mediated DNA looping. Specifically, our analysis indicates that the sensitivity of the inducer response results from LacR-mediated DNA looping, which is significantly enhanced by CRP.
[http://www.pnas.org.ezp1.harvard.edu/cgi/reprint/104/14/6043.pdf]


===PRLandE===
Stochastic simulations of genetic switch systems<br>
<br>
Adiel Loinger,1 Azi Lipshtat,2 Nathalie Q. Balaban,1 and Ofer Biham1<br>
<br>
Genetic switch systems with mutual repression of two transcription factors are studied using deterministic methods (rate equations) and stochastic methods (the master equation and Monte Carlo simulations). These systems exhibit bistability, namely two stable states such that spontaneous transitions between them are rare. Induced transitions may take place as a result of an external stimulus. We study several variants of the genetic switch and examine the effects of cooperative binding, exclusive binding, protein-protein interactions, and degradation of bound repressors. We identify the range of parameters in which bistability takes place, enabling the system to function as a switch. Numerous studies have concluded that cooperative binding is a necessary condition for the emergence of bistability in these systems. We show that a suitable combination of network structure and stochastic effects gives rise to bistability even without cooperative binding. The average time between spontaneous transitions is evaluated as a function of the biological parameters.
<br>
<br>
Stochastic suppression of gene expression oscillators under intercell coupling<br>
<br>
A. Koseska,1 A. Zaikin,1,2 J. García-Ojalvo,3 and J. Kurths1<br>
<br>
This paper examines the dynamics of an ensemble of hysteresis-based genetic relaxation oscillators, focusing on the influence of noise and cell-to-cell coupling on the appearance of new dynamical regimes. In particular, we show that control of the coupling strength and noise can effectively change the dynamics of the system leading to behaviors such as clustering, synchronous and asynchronous oscillations, and suppression. Moreover, under certain conditions an optimal amount of noise can lead to increased order in the system. The results obtained are correlated with relevant biological processes that occur in living organisms.
<br>

===Quaterly Reviews of Biophysics===

===Science===
Science 23 March 2007:
Vol. 315. no. 5819, pp. 1716 - 1719<br>
<br>
Tunability and Noise Dependence in Differentiation Dynamics<br>
Gürol M. Süel,1 Rajan P. Kulkarni,2 Jonathan Dworkin,3 Jordi Garcia-Ojalvo,4 Michael B. Elowitz2*<br>
<br>
The dynamic process of differentiation depends on the architecture, quantitative parameters, and noise of underlying genetic circuits. However, it remains unclear how these elements combine to control cellular behavior. We analyzed the probabilistic and transient differentiation of Bacillus subtilis cells into the state of competence. A few key parameters independently tuned the frequency of initiation and the duration of competence episodes and allowed the circuit to access different dynamic regimes, including oscillation. Altering circuit architecture showed that the duration of competence events can be made more precise. We used an experimental method to reduce global cellular noise and showed that noise levels are correlated with frequency of differentiation events. Together, the data reveal a noise-dependent circuit that is remarkably resilient and tunable in terms of its dynamic behavior.
<br>


===Systems Biology===