{{Template:Paulsson header}}
'''What's hot?  What's not?'''

Paste in ".ezp1.harvard.edu" between "journal.com" and "/articleID" to access full text.<br>

Go [http://dx.doi.org.ezp1.harvard.edu/ here] to get the article using its DOI (Digital Object Identifier)<br>

Who's responsible for keeping track of which journal?  Find out [[Paulsson:Journal responsibilities | here]].

[[Paulsson:Journal watch archive | Archive of previous entries]]

==List of Journals==

===Biophysical Journal===


===Cell===
*Tumor Morphology and Phenotypic Evolution Driven by Selective Pressure from the Microenvironment.  Alexander R.A. Anderson, Alissa M. Weaver, Peter T. Cummings, and Vito Quaranta.<br>
Emergence of invasive behavior in cancer is life-threatening, yet ill-defined due to its multifactorial nature. We present a multiscale mathematical model of cancer invasion, which considers cellular and microenvironmental factors simultaneously and interactively. Unexpectedly, the model simulations predict that harsh tumor microenvironment conditions (e.g., hypoxia, heterogenous extracellular matrix) exert a dramatic selective force on the tumor, which grows as an invasive mass with fingering margins, dominated by a few clones with aggressive traits. In contrast, mild microenvironment conditions (e.g., normoxia, homogeneous matrix) allow clones with similar aggressive traits to coexist with less aggressive phenotypes in a heterogeneous tumor mass with smooth, noninvasive margins. Thus, the genetic make-up of a cancer cell may realize its invasive potential through a clonal evolution process driven by definable microenvironmental selective forces. Our mathematical model provides a theoretical/experimental framework to quantitatively characterize this selective pressure for invasion and test ways to eliminate it.
[http://dx.doi.org.ezp1.harvard.edu/10.1016/j.cell.2006.09.042]

===EMBO===


===Genetics===
Genetics, Vol. 174, 1625-1634, November 2006, Copyright © 2006
doi:10.1534/genetics.106.061218<br>
Dynamical Analysis of the Regulatory Network Defining the Dorsal–Ventral Boundary of the Drosophila Wing Imaginal Disc <br>
Aitor González1, Claudine Chaouiya and Denis Thieffry2 
<br>
The larval development of the Drosophila melanogaster wings is organized by the protein Wingless, which is secreted by cells adjacent to the dorsal–ventral (DV) boundary. Two signaling processes acting between the second and early third instars and between the mid- and late third instar control the expression of Wingless in these boundary cells. Here, we integrate both signaling processes into a logical multivalued model encompassing four cells, i.e., a boundary and a flanking cell at each side of the boundary. Computer simulations of this model enable a qualitative reproduction of the main wild-type and mutant phenotypes described in the experimental literature. During the first signaling process, Notch becomes activated by the first signaling process in an Apterous-dependent manner. In silico perturbation experiments show that this early activation of Notch is unstable in the absence of Apterous. However, during the second signaling process, the Notch pattern becomes consolidated, and thus independent of Apterous, through activation of the paracrine positive feedback circuit of Wingless. Consequently, we propose that appropriate delays for Apterous inactivation and Wingless induction by Notch are crucial to maintain the wild-type expression at the dorsal–ventral boundary. Finally, another mutant simulation shows that cut expression might be shifted to late larval stages because of a potential interference with the early signaling process.



Genetics, Vol. 174, 511-518, September 2006, Copyright © 2006
doi:10.1534/genetics.106.058560<br>
Introns Regulate RNA and Protein Abundance in Yeast <br>
Kara Juneau*, ,1, Molly Miranda , Maureen E. Hillenmeyer , , Corey Nislow ,  and Ronald W. Davis*,  
<br>
The purpose of introns in the architecturally simple genome of Saccharomyces cerevisiae is not well understood. To assay the functional relevance of introns, a series of computational analyses and several detailed deletion studies were completed on the intronic genes of S. cerevisiae. Mining existing data from genomewide studies on yeast revealed that intron-containing genes produce more RNA and more protein and are more likely to be haplo-insufficient than nonintronic genes. These observations for all intronic genes held true for distinct subsets of genes including ribosomal, nonribosomal, duplicated, and nonduplicated. Corroborating the result of computational analyses, deletion of introns from three essential genes decreased cellular RNA levels and caused measurable growth defects. These data provide evidence that introns improve transcriptional and translational yield and are required for competitive growth of yeast.<br>

Note: the paper doesn’t have more info than the abstract.


===Journal of Bacteriology===


===Journal of Chemical Physcis===


===Journal of Molecular Biology===


===Journal of Physical Chemistry-A===


===Journal of Physical Chemistry-B===


===Journal of Physical Chemistry-C===


===Journal of Physical Chemistry-D===


===Journal of Physical Chemistry-E===


===Journal of Statistical Physics===


===Journal of Theoretical Biology===


===Molecular Microbiology===


===Molecular Systems Biology===


===Nature===


===Nature Biotechnology===


===Nature Genetics===


===Plasmid===




===PLoS Computational Biology===
*'''Optimal Noise Filtering in the Chemotactic Response of Escherichia coli'''<br>
Andrews BW, Yi TM, Iglesias PA<br><br>
Information-carrying signals in the real world are often obscured by noise. A challenge for any system is to filter the signal from the corrupting noise. This task is particularly acute for the signal transduction network that mediates bacterial chemotaxis, because the signals are subtle, the noise arising from stochastic fluctuations is substantial, and the system is effectively acting as a differentiator which amplifies noise. Here, we investigated the filtering properties of this biological system. Through simulation, we first show that the cutoff frequency has a dramatic effect on the chemotactic efficiency of the cell. Then, using a mathematical model to describe the signal, noise, and system, we formulated and solved an optimal filtering problem to determine the cutoff frequency that bests separates the low-frequency signal from the high-frequency noise. There was good agreement between the theory, simulations, and published experimental data. Finally, we propose that an elegant implementation of the optimal filter in combination with a differentiator can be achieved via an integral control system. This paper furnishes a simple quantitative framework for interpreting many of the key notions about bacterial chemotaxis, and, more generally, it highlights the constraints on biological systems imposed by noise.<br>
PLoS Computational Biology Vol. 2, No. 11, e154 [http://dx.doi.org.ezp1.harvard.edu/10.1371/journal.pcbi.0020154]

===PNAS===


===PRLandE===
Phys. Rev. Lett. 97, 168302 (2006)	<br>
Linking Stochastic Dynamics to Population Distribution: An Analytical Framework of Gene Expression<br>
Nir Friedman, Long Cai, and X. Sunney Xie<br>

We present an analytical framework describing the steady-state distribution of protein concentration in live cells, considering that protein production occurs in random bursts with an exponentially distributed number of molecules. We extend this framework for cases of transcription autoregulation and noise propagation in a simple genetic network. This model allows for the extraction of kinetic parameters of gene expression from steady-state distributions of protein concentration in a cell population, which are available from single cell data obtained by flow cytometry or fluorescence microscopy.


===Quarterly Reviews of Biophysics===


===Science===
Science 8 December 2006:
Vol. 314. no. 5805, pp. 1569 - 1572
DOI: 10.1126/science.1134829<br>
Research Articles
Group Competition, Reproductive Leveling, and the Evolution of Human Altruism <br>
Samuel Bowles <br>
Humans behave altruistically in natural settings and experiments. A possible explanation—that groups with more altruists survive when groups compete—has long been judged untenable on empirical grounds for most species. But there have been no empirical tests of this explanation for humans. My empirical estimates show that genetic differences between early human groups are likely to have been great enough so that lethal intergroup competition could account for the evolution of altruism. Crucial to this process were distinctive human practices such as sharing food beyond the immediate family, monogamy, and other forms of reproductive leveling. These culturally transmitted practices presuppose advanced cognitive and linguistic capacities, possibly accounting for the distinctive forms of altruism found in our species.
<br>
<br>
Science 8 December 2006:
Vol. 314. no. 5805, pp. 1560 - 1563
DOI: 10.1126/science.1133755<br>
Review
Five Rules for the Evolution of Cooperation <br>
Martin A. Nowak <br>
Cooperation is needed for evolution to construct new levels of organization. Genomes, cells, multicellular organisms, social insects, and human society are all based on cooperation. Cooperation means that selfish replicators forgo some of their reproductive potential to help one another. But natural selection implies competition and therefore opposes cooperation unless a specific mechanism is at work. Here I discuss five mechanisms for the evolution of cooperation: kin selection, direct reciprocity, indirect reciprocity, network reciprocity, and group selection. For each mechanism, a simple rule is derived that specifies whether natural selection can lead to cooperation.
<br>
<br>
Science 8 December 2006:
Vol. 314. no. 5805, pp. 1585 - 1588
DOI: 10.1126/science.1132493<br>
Reports<br>
Enzyme-Free Nucleic Acid Logic Circuits <br>
Georg Seelig,1 David Soloveichik,2 David Yu Zhang,2 Erik Winfree2,3* <br>
Biological organisms perform complex information processing and control tasks using sophisticated biochemical circuits, yet the engineering of such circuits remains ineffective compared with that of electronic circuits. To systematically create complex yet reliable circuits, electrical engineers use digital logic, wherein gates and subcircuits are composed modularly and signal restoration prevents signal degradation. We report the design and experimental implementation of DNA-based digital logic circuits. We demonstrate AND, OR, and NOT gates, signal restoration, amplification, feedback, and cascading. Gate design and circuit construction is modular. The gates use single-stranded nucleic acids as inputs and outputs, and the mechanism relies exclusively on sequence recognition and strand displacement. Biological nucleic acids such as microRNAs can serve as inputs, suggesting applications in biotechnology and bioengineering.
<br>
<br>
Science 8 December 2006:
Vol. 314. no. 5805, pp. 1595 - 1598
DOI: 10.1126/science.1133141<br>
Reports
A Complex Oscillating Network of Signaling Genes Underlies the Mouse Segmentation Clock <br>
Mary-Lee Dequéant,1,2,3 Earl Glynn,3 Karin Gaudenz,3 Matthias Wahl,1,3 Jie Chen,4 Arcady Mushegian,3 Olivier Pourquié1,2,3* <br>
The segmental pattern of the spine is established early in development, when the vertebral precursors, the somites, are rhythmically produced from the presomitic mesoderm. Microarray studies of the mouse presomitic mesoderm transcriptome reveal that the oscillator associated with this process, the segmentation clock, drives the periodic expression of a large network of cyclic genes involved in cell signaling. Mutually exclusive activation of the notch–fibroblast growth factor and Wnt pathways during each cycle suggests that coordinated regulation of these three pathways underlies the clock oscillator.
<br>
<br>
Science 8 December 2006:
Vol. 314. no. 5805, pp. 1601 - 1603
DOI: 10.1126/science.1134830<br>
Reports
Synthesis-Mediated Release of a Small RNA Inhibitor of RNA Polymerase <br>
Karen M. Wassarman1* and Ruth M. Saecker2 <br>
Noncoding small RNAs regulate gene expression in all organisms, in some cases through direct association with RNA polymerase (RNAP). Here we report that the mechanism of 6S RNA inhibition of transcription is through specific, stable interactions with the active site of Escherichia coli RNAP that exclude promoter DNA binding. In fact, the DNA-dependent RNAP uses bound 6S RNA as a template for RNA synthesis, producing 14-to 20-nucleotide RNA products (pRNA). These results demonstrate that 6S RNA is functionally engaged in the active site of RNAP. Synthesis of pRNA destabilizes 6S RNA–RNAP complexes leading to release of the pRNA-6S RNA hybrid. In vivo, 6S RNA–directed RNA synthesis occurs during outgrowth from the stationary phase and likely is responsible for liberating RNAP from 6S RNA in response to nutrient availability.
<br>
<br>
Science 8 December 2006:
Vol. 314. no. 5805, pp. 1607 - 1609
DOI: 10.1126/science.1134930<br>
Reports
A Positive Feedback Loop Promotes Transcription Surge That Jump-Starts Salmonella Virulence Circuit <br>
Dongwoo Shin,* Eun-Jin Lee, Henry Huang, Eduardo A. Groisman  <br>
The PhoP/PhoQ two-component system is a master regulator of Salmonella pathogenicity. Here we report that induction of the PhoP/PhoQ system results in an initial surge of PhoP phosphorylation; the occupancy of target promoters by the PhoP protein; and the transcription of PhoP-activated genes, which then subsides to reach new steady-state levels. This surge in PhoP activity is due to PhoP positively activating its own transcription, because a strain constitutively expressing the PhoP protein attained steady-state levels of activation asymptotically, without the surge. The strain constitutively expressing the PhoP protein was attenuated for virulence in mice, demonstrating that the surge conferred by PhoP's positive feedback loop is necessary to jump-start Salmonella's virulence program.
<br>
<br>
Science 1 December 2006:
Vol. 314. no. 5804, pp. 1447 - 1450
DOI: 10.1126/science.1130088<br>
Reports
WNT and DKK Determine Hair Follicle Spacing Through a Reaction-Diffusion Mechanism <br>
Stefanie Sick,1 Stefan Reinker,2* Jens Timmer,2 Thomas Schlake1  <br>
Mathematical reaction-diffusion models have been suggested to describe formation of animal pigmentation patterns and distribution of epidermal appendages. However, the crucial signals and in vivo mechanisms are still elusive. Here we identify WNT and its inhibitor DKK as primary determinants of murine hair follicle spacing, using a combined experimental and computational modeling approach. Transgenic DKK overexpression reduces overall appendage density. Moderate suppression of endogenous WNT signaling forces follicles to form clusters during an otherwise normal morphogenetic program. These results confirm predictions of a WNT/DKK-specific mathematical model and provide in vivo corroboration of the reaction-diffusion mechanism for epidermal appendage formation.
<br>
<br>
Science 1 December 2006:
Vol. 314. no. 5804, pp. 1464 - 1467
DOI: 10.1126/science.1131370<br>
Reports
Microfluidic Digital PCR Enables Multigene Analysis of Individual Environmental Bacteria<br> 
Elizabeth A. Ottesen,1 Jong Wook Hong,2 Stephen R. Quake,3 Jared R. Leadbetter4* <br>
Gene inventory and metagenomic techniques have allowed rapid exploration of bacterial diversity and the potential physiologies present within microbial communities. However, it remains nontrivial to discover the identities of environmental bacteria carrying two or more genes of interest. We have used microfluidic digital polymerase chain reaction (PCR) to amplify and analyze multiple, different genes obtained from single bacterial cells harvested from nature. A gene encoding a key enzyme involved in the mutualistic symbiosis occurring between termites and their gut microbiota was used as an experimental hook to discover the previously unknown ribosomal RNA–based species identity of several symbionts. The ability to systematically identify bacteria carrying a particular gene and to link any two or more genes of interest to single species residing in complex ecosystems opens up new opportunities for research on the environment.
<br>
<br>
Science 17 November 2006
Vol. 314. no. 5802, pp. 1139 - 1143
DOI: 10.1126/science.1131398<br>
Reports
Abortive Initiation and Productive Initiation by RNA Polymerase Involve DNA Scrunching <br>
Andrey Revyakin,1,2* Chenyu Liu,1,2,3 Richard H. Ebright,1  Terence R. Strick2,3  <br>
Using single-molecule DNA nanomanipulation, we show that abortive initiation involves DNA "scrunching"—in which RNA polymerase (RNAP) remains stationary and unwinds and pulls downstream DNA into itself—and that scrunching requires RNA synthesis and depends on RNA length. We show further that promoter escape involves scrunching, and that scrunching occurs in most or all instances of promoter escape. Our results support the existence of an obligatory stressed intermediate, with approximately one turn of additional DNA unwinding, in escape and are consistent with the proposal that stress in this intermediate provides the driving force to break RNAP-promoter and RNAP-initiation-factor interactions in escape.
<br>
<br>

===Systems Biology===