{{Template:Paulsson header}}
'''What's hot?  What's not?'''

Paste in ".ezp1.harvard.edu" between "journal.com" and "/articleID" to access full text.<br>

Go [http://dx.doi.org.ezp1.harvard.edu/ here] to get the article using its DOI (Digital Object Identifier)<br>

Who's responsible for keeping track of which journal?  Find out [[Paulsson:Journal responsibilities | here]].

[[Paulsson:Journal watch archive | Archive of previous entries]]

==List of Journals==

===Biophysical Journal===


===Cell===


===EMBO===
*DNA segregation by the bacterial actin AlfA during Bacillus subtilis growth and development. 
Eric Becker1, Nick C Herrera1, Felizza Q Gunderson, Alan I Derman, Amber L Dance, Jennifer Sims, Rachel A Larsen and Joe Pogliano<br>
We here identify a protein (AlfA; actin like filament) that defines a new family of actins that are only distantly related to MreB and ParM. AlfA is required for segregation of Bacillus subtilis plasmid pBET131 (a mini pLS32-derivative) during growth and sporulation. A 3-kb DNA fragment encoding alfA and a downstream gene (alfB) is necessary and sufficient for plasmid stability. AlfA-GFP assembles dynamic cytoskeletal filaments that rapidly turn over (t1/2<approx45 s) in fluorescence recovery after photobleaching experiments. A point mutation (alfA D168A) that completely inhibits AlfA subunit exchange in vivo is strongly defective for plasmid segregation, demonstrating that dynamic polymerization of AlfA is necessary for function. During sporulation, plasmid segregation occurs before septation and independently of the DNA translocase SpoIIIE and the chromosomal Par proteins Soj and Spo0J. The absence of the RacA chromosome anchoring protein reduces the efficiency of plasmid segregation (by about two-fold), suggesting that it might contribute to anchoring the plasmid at the pole during sporulation. Our results suggest that the dynamic polymerization of AlfA mediates plasmid separation during both growth and sporulation.
[http://www.nature.com.ezp1.harvard.edu/emboj/journal/v25/n24/full/7601443a.html]

===Genetics===


===Journal of Bacteriology===


===Journal of Chemical Physcis===


===Journal of Molecular Biology===


===Journal of Physical Chemistry-A===


===Journal of Physical Chemistry-B===


===Journal of Physical Chemistry-C===


===Journal of Physical Chemistry-D===


===Journal of Physical Chemistry-E===


===Journal of Statistical Physics===


===Journal of Theoretical Biology===


===Molecular Microbiology===


===Molecular Systems Biology===


===Nature===


===Nature Biotechnology===


===Nature Genetics===


===Plasmid===


===PLOS===
===PLoS Biology===
*Polarised Asymmetric Inheritance of Accumulated Protein Damage in Higher Eukaryotes.<br> Rujano MA, Bosveld F, Salomons FA, Dijk F, van Waarde MA, et al.<br>
Disease-associated misfolded proteins or proteins damaged due to cellular stress are generally disposed via the cellular protein quality-control system. However, under saturating conditions, misfolded proteins will aggregate. In higher eukaryotes, these aggregates can be transported to accumulate in aggresomes at the microtubule organizing center. The fate of cells that contain aggresomes is currently unknown. Here we report that cells that have formed aggresomes can undergo normal mitosis. As a result, the aggregated proteins are asymmetrically distributed to one of the daughter cells, leaving the other daughter free of accumulated protein damage. Using both epithelial crypts of the small intestine of patients with a protein folding disease and Drosophila melanogaster neural precursor cells as models, we found that the inheritance of protein aggregates during mitosis occurs with a fixed polarity indicative of a mechanism to preserve the long-lived progeny.
[http://dx.doi.org/10.1371/journal.pbio.0040417]

*An HIV Feedback Resistor: Auto-Regulatory Circuit Deactivator and Noise Buffer.<br> Weinberger LS, Shenk T<br>
Animal viruses (e.g., lentiviruses and herpesviruses) use transcriptional positive feedback (i.e., transactivation) to regulate their gene expression. But positive-feedback circuits are inherently unstable when turned off, which presents a particular dilemma for latent viruses that lack transcriptional repressor motifs. Here we show that a dissipative feedback resistor, composed of enzymatic interconversion of the transactivator, converts transactivation circuits into excitable systems that generate transient pulses of expression, which decay to zero. We use HIV-1 as a model system and analyze single-cell expression kinetics to explore whether the HIV-1 transactivator of transcription (Tat) uses a resistor to shut off transactivation. The Tat feedback circuit was found to lack bi-stability and Tat self-cooperativity but exhibited a pulse of activity upon transactivation, all in agreement with the feedback resistor model. Guided by a mathematical model, biochemical and genetic perturbation of the suspected Tat feedback resistor altered the circuit's stability and reduced susceptibility to molecular noise, in agreement with model predictions. We propose that the feedback resistor is a necessary, but possibly not sufficient, condition for turning off noisy transactivation circuits lacking a repressor motif (e.g., HIV-1 Tat). Feedback resistors may be a paradigm for examining other auto-regulatory circuits and may inform upon how viral latency is established, maintained, and broken. 
[http://dx.doi.org/10.1371/journal.pbio.0050009]

===PNAS===
*Cooperation and conflict in microbial biofilms<br>Xavier JB and Foster KR<br>
Biofilms, in which cells attach to surfaces and secrete slime (polymeric substances), are central to microbial life. Biofilms are often thought to require high levels of cooperation because extracellular polymeric substances are a shared resource produced by one cell that can be used by others. Here we examine this hypothesis by using a detailed individual-based simulation of a biofilm to investigate the outcome of evolutionary competitions between strains that differ in their level of polymer production. Our model includes a biochemical description of the carbon fluxes for growth and polymer production, and it explicitly calculates diffusionâ€“reaction effects and the resulting solute gradients in the biofilm. An emergent property of these simple but realistic mechanistic assumptions is a strong evolutionary advantage to extracellular polymer production. Polymer secretion is altruistic to cells above a focal cell: it pushes later generations in their lineage up and out into better oxygen conditions, but it harms others; polymer production suffocates neighboring nonpolymer producers. This property, analogous to vertical growth in plants, suggests that polymer secretion provides a strong competitive advantage to cell lineages within mixed-genotype biofilms: global cooperation is not required. Our model fundamentally changes how biofilms are expected to respond to changing social conditions; the presence of multiple strains in a biofilm should promote rather than inhibit polymer secretion. [http://dx.doi.org.ezp1.harvard.edu/10.1073/pnas.0607651104]

===PRLandE===


===Quarterly Reviews of Biophysics===


===Science===


===Systems Biology===