{{Template:Paulsson header}}
[[Paulsson:Journal watch archive | back to Archive]]

==List of Journals==

===Biophysical Journal===

===Cell===


===EMBO===


===Genetics===


===Journal of Bacteriology===


===Journal of Chemical Physcis===
*COAST: Controllable approximative stochastic reaction algorithm.  Wagner H, Moller M, Prank K.
We present an approximative algorithm for stochastic simulations of chemical reaction systems, called COAST, based on three different modeling levels: for small numbers of particles an exact stochastic model; for intermediate numbers an approximative, but computationally more efficient stochastic model based on discrete Gaussian distributions; and for large numbers the deterministic reaction kinetics. In every simulation time step, the subdivision of the reaction channels into the three different modeling levels is done automatically, where all approximations applied can be controlled by a single error parameter for which an appropriate value can easily be found. Test simulations show that the results of COAST simulations agree well with the outcomes of exact algorithms; however, the asymptotic run times of COAST are asymptotically proportional to smaller powers of the particle numbers than exact algorithms.
[http://scitation.aip.org.ezp1.harvard.edu/journals/doc/JCPSA6-ft/vol_125/iss_17/174104_1.html]

===Journal of Molecular Biology===
*SOS Repair and DNA Supercoiling Influence the Genetic Stability of DNA Triplet Repeats in Escherichia coli.  Majchrzak M, Bowater RP, Staczek P, Parniewski P.<br>
Molecular mechanisms responsible for the genetic instability of DNA trinucleotide sequences (TRS) account for at least 20 human hereditary disorders. Many aspects of DNA metabolism influence the frequency of length changes in such repeats. Herein, we demonstrate that expression of Escherichia coli SOS repair proteins dramatically decreases the genetic stability of long (CTG/CAG)(n) tracts contained in plasmids. Furthermore, the growth characteristics of the bacteria are affected by the (CTG/CAG)(n) tract, with the effect dependent on the length of the TRS. In an E. coli host strain with constitutive expression of the SOS regulon, the frequency of deletions to the repeat is substantially higher than that in a strain with no SOS response. Analyses of the topology of reporter plasmids isolated from the SOS+ and SOS- strains revealed higher levels of negative supercoiling in strains with the constitutively expressed SOS network. Hence, we used strains with mutations in topoisomerases to examine the effect of DNA topology upon the TRS instability. Higher levels of negative DNA supercoiling correlated with increased deletions in long (CTG/CAG)(n), (CGG/CCG)(n) and (GAA/TTC)(n.) These observations suggest a link between the induction of bacterial SOS repair, changes in DNA topology and the mechanisms leading to genetic instability of repetitive DNA sequences.
[http://www.sciencedirect.com.ezp1.harvard.edu/science?_ob=ArticleURL&_udi=B6WK7-4KV3W0T-5&_coverDate=12%2F08%2F2006&_alid=499957763&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=6899&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=375d16e39f2b34b282ea5501f8e95794]



===Journal of Physical Chemistry-A===

===Journal of Physical Chemistry-B===

===Journal of Physical Chemistry-C===

===Journal of Physical Chemistry-D===

===Journal of Physical Chemistry-E===

===Journal of Statistical Physics===

===Journal of Theoretical Biology===

===Molecular Microbiology===


===Molecular Systems Biology===
*Imaging single molecules in living cells for systems biology.  Sako Y.<br>
In this work, I present the application of single-molecule imaging to systems biology and discuss the relevant technical issues within this context. Imaging single molecules has made it possible to visualize individual molecules at work in living cells. This continuously improving technique allows the measurement of non-invasively quantitative parameters of intracellular reactions, such as the number of molecules, reaction rate constants and diffusion coefficients with spatial distributions and temporal fluctuations. This detailed information about unitary intracellular reactions is essential for constructing quantitative models of reaction networks that provide a systems-level understanding of the mechanisms by which various cellular behaviors are emerging. 
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v2/n1/full/msb4100100.html]

*A novel computational model of the circadian clock in Arabidopsis that incorporates PRR7 and PRR9. Zeilinger MN, Farre EM, Taylor SR, Kay SA, Doyle FJ 3rd<br>
In plants, as in animals, the core mechanism to retain rhythmic gene expression relies on the interaction of multiple feedback loops. In recent years, molecular genetic techniques have revealed a complex network of clock components in Arabidopsis. To gain insight into the dynamics of these interactions, new components need to be integrated into the mathematical model of the plant clock. Our approach accelerates the iterative process of model identification, to incorporate new components, and to systematically test different proposed structural hypotheses. Recent studies indicate that the pseudo-response regulators PRR7 and PRR9 play a key role in the core clock of Arabidopsis. We incorporate PRR7 and PRR9 into an existing model involving the transcription factors TIMING OF CAB (TOC1), LATE ELONGATED HYPOCOTYL (LHY) and CIRCADIAN CLOCK ASSOCIATED (CCA1). We propose candidate models based on experimental hypotheses and identify the computational models with the application of an optimization routine. Validation is accomplished through systematic analysis of various mutant phenotypes. We introduce and apply sensitivity analysis as a novel tool for analyzing and distinguishing the characteristics of proposed architectures, which also allows for further validation of the hypothesized structures.
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v2/n1/full/msb4100101.html]

===Nature===


===Nature Biotechnology===
*A sequence-oriented comparison of gene expression measurements across different hybridization-based technologies. Kuo, et. al.<br>
Over the last decade, gene expression microarrays have had a profound impact on biomedical research. The diversity of platforms and analytical methods available to researchers have made the comparison of data from multiple platforms challenging. In this study, we describe a framework for comparisons across platforms and laboratories. We have attempted to include nearly all the available commercial and 'in-house' platforms. Using probe sequences matched at the exon level improved consistency of measurements across the different microarray platforms compared to annotation-based matches. Generally, consistency was good for highly expressed genes, and variable for genes with lower expression values as confirmed by quantitative real-time (QRT)-PCR. Concordance of measurements was higher between laboratories on the same platform than across platforms. We demonstrate that, after stringent preprocessing, commercial arrays were more consistent than in-house arrays, and by most measures, one-dye platforms were more consistent than two-dye platforms.
[http://www.nature.com.ezp1.harvard.edu/nbt/journal/v24/n7/full/nbt1217.html]

===Nature Genetics===
*Comparative genome sequencing of Escherichia coli allows observation of bacterial evolution on a laboratory timescale.  Herring et al...Palsson BO.<br>
We applied whole-genome resequencing of Escherichia coli to monitor the acquisition and fixation of mutations that conveyed a selective growth advantage during adaptation to a glycerol-based growth medium. We identified 13 different de novo mutations in five different E. coli strains and monitored their fixation over a 44-d period of adaptation. We obtained proof that the observed spontaneous mutations were responsible for improved fitness by creating single, double and triple site-directed mutants that had growth rates matching those of the evolved strains. The success of this new genome-scale approach indicates that real-time evolution studies will now be practical in a wide variety of contexts.
[http://www.nature.com.ezp1.harvard.edu/ng/journal/v38/n12/full/ng1906.html]


===Plasmid===
*Interaction of initiator proteins with the origin of replication of an IncL/M plasmid.  Borrell L, Yang J, Pittard AJ, Praszkier J. <br>
The origin of replication of the IncL/M plasmid pMU604 was analyzed to identify sequences important for binding of initiator proteins and origin activity. A thrice repeated sequence motif 5'-NANCYGCAA-3' was identified as the binding site (RepA box) of the initiator protein, RepA. All three copies of the RepA box were required for in vivo activity and binding of RepA to these boxes appeared to be cooperative. A DnaA R box (box 1), located immediately upstream of the RepA boxes, was not required for recruitment of DnaA during initiation of replication by RepA of pMU604 unless a DnaA R box located at the distal end of the origin (box 3) had been inactivated. However, DnaA R box 1 was important for recruitment of DnaA to the origin of replication of pMU604 when the initiator RepA was that from a distantly related plasmid, pMU720. A mutation which scrambled DnaA R boxes 1 and 3 and one which scrambled DnaA R boxes 1, 3 and 4 had much more deleterious effects on initiation by RepA of pMU720 than on initiation by RepA of pMU604. Neither Rep protein could initiate replication from the origin of pMU604 in the absence of DnaA, suggesting that the difference between them might lie in the mechanism of recruitment of DnaA to this origin. DnaA protein enhanced the binding and origin unwinding activities of RepA of pMU604, but appeared unable to bind to a linear DNA fragment bearing the origin of replication of pMU604 in the absence of other proteins.
[http://www.sciencedirect.com.ezp1.harvard.edu/science?_ob=ArticleURL&_udi=B6WPF-4K5ST53-1&_coverDate=09%2F30%2F2006&_alid=499960173&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=6989&_sort=d&view=c&_acct=C000014438&_version=1&_urlVersion=0&_userid=209690&md5=9a341cf1af9bc82719d8d0357ee7f76a]

*Inhibition of protein and RNA synthesis in Escherichia coli results in declustering of plasmid RK2.  Yao S, Toukdarian A, Helinski DR<br>
Multi-copy plasmids in Escherichia coli are not randomly distributed throughout the cell but are present as clusters of plasmid molecules that are localized at preferred cellular locations. A plasmid RK2 derivative (pZZ15) that can be tagged with a green fluorescent protein-LacI fusion protein normally exists as clusters that are localized at the mid- and quarter-cell positions. In this study the effect of the protein synthesis inhibitor, chloramphenicol, and the RNA synthesis inhibitor, rifampicin, on RK2 clustering and localization was examined. The addition of either inhibitor to exponentially growing E. coli cells carrying pZZ15 results in a displacement of the position and a declustering of this multi-copy plasmid indicating that continued protein synthesis and RNA synthesis are required for clustering and localization of this plasmid. It is likely that it is not just the process of transcription or translation that is important for clustering but rather some host or plasmid encoded factor(s) that is required.
[http://dx.doi.org.ezp1.harvard.edu/10.1016/j.plasmid.2006.04.003 to article]


===PLOS===
*Stochastic mRNA synthesis in Mammalian cells.  Raj A, Peskin CS, Tranchina D, Vargas DY, Tyagi S.<br>
Individual cells in genetically homogeneous populations have been found to express different numbers of molecules of specific proteins. We investigated the origins of these variations in mammalian cells by counting individual molecules of mRNA produced from a reporter gene that was stably integrated into the cell's genome. We found that there are massive variations in the number of mRNA molecules present in each cell. These variations occur because mRNAs are synthesized in short but intense bursts of transcription beginning when the gene transitions from an inactive to an active state and ending when they transition back to the inactive state. We show that these transitions are intrinsically random and not due to global, extrinsic factors such as the levels of transcriptional activators. Moreover, the gene activation causes burst-like expression of all genes within a wider genomic locus. We further found that bursts are also exhibited in the synthesis of natural genes. The bursts of mRNA expression can be buffered at the protein level by slow protein degradation rates. A stochastic model of gene activation and inactivation was developed to explain the statistical properties of the bursts. The model showed that increasing the level of transcription factors increases the average size of the bursts rather than their frequency. These results demonstrate that gene expression in mammalian cells is subject to large, intrinsically random fluctuations and raise questions about how cells are able to function in the face of such noise.
[http://biology.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pbio.0040309]


===PNAS===
*Quantification of protein half-lives in the budding yeast proteome.  Belle A, Tanay A, Bitincka L, Shamir R, O'Shea EK.<br>
A complete description of protein metabolism requires knowledge of the rates of protein production and destruction within cells. Using an epitope-tagged strain collection, we measured the half-life of >3,750 proteins in the yeast proteome after inhibition of translation. By integrating our data with previous measurements of protein and mRNA abundance and translation rate, we provide evidence that many proteins partition into one of two regimes for protein metabolism: one optimized for efficient production or a second optimized for regulatory efficiency. Incorporation of protein half-life information into a simple quantitative model for protein production improves our ability to predict steady-state protein abundance values. Analysis of a simple dynamic protein production model reveals a remarkable correlation between transcriptional regulation and protein half-life within some groups of coregulated genes, suggesting that cells coordinate these two processes to achieve uniform effects on protein abundances. Our experimental data and theoretical analysis underscore the importance of an integrative approach to the complex interplay between protein degradation, transcriptional regulation, and other determinants of protein metabolism.
[http://www.pnas.org/cgi/content/full/103/35/13004]


===PRLandE===

===Quarterly Reviews of Biophysics===


===Science===


===Systems Biology===
Is there an optimal ribosome concentration for maximal protein production?  Snoep JL, Westerhoff HV, Rohwer JM, Hofmeyr JH.<br>
A core model is presented for protein production in Escherichia coli to address the question whether there is an optimal ribosomal concentration for non-ribosome protein production. Analysing the steady-state solution of the model over a range of mRNA concentrations, indicates that such an optimum ribosomal content exists, and that the optimum shifts to higher ribosomal contents at higher specific growth rates.<br>