{{Template:Paulsson header}}
'''What's hot?  What's not?'''

Paste in ".ezp1.harvard.edu" between "journal.com" and "/articleID" to access full text.<br>

Go [http://dx.doi.org.ezp1.harvard.edu/ here] to get the article using its DOI (Digital Object Identifier)<br>

Who's responsible for keeping track of which journal?  Find out [[Paulsson:Journal responsibilities | here]].

[[Paulsson:Journal watch archive | Archive of previous entries]]

==List of Journals==

===Biophysical Journal===


===Cell===
*Excitatory Local Circuits and Their Implications for Olfactory Processing in the Fly Antennal Lobe
Yuhua Shang1, Adam Claridge-Chang1, Lucas Sjulson1, Marc Pypaert1 and Gero Miesenböck<br>
Conflicting views exist of how circuits of the antennal lobe, the insect equivalent of the olfactory bulb, translate input from olfactory receptor neurons (ORNs) into projection-neuron (PN) output. Synaptic connections between ORNs and PNs are one-to-one, yet PNs are more broadly tuned to odors than ORNs. The basis for this difference in receptive range remains unknown. Analyzing a Drosophila mutant lacking ORN input to one glomerulus, we show that some of the apparent complexity in the antennal lobe's output arises from lateral, interglomerular excitation of PNs. We describe a previously unidentified population of cholinergic local neurons (LNs) with multiglomerular processes. These excitatory LNs respond broadly to odors but exhibit little glomerular specificity in their synaptic output, suggesting that PNs are driven by a combination of glomerulus-specific ORN afferents and diffuse LN excitation. Lateral excitation may boost PN signals and enhance their transmission to third-order neurons in a mechanism akin to stochastic resonance.
[http://www.sciencedirect.com.ezp1.harvard.edu/science?_ob=ArticleURL&_udi=B6WSN-4N0Y27X-R&_user=209690&_coverDate=02%2F09%2F2007&_rdoc=21&_fmt=summary&_orig=browse&_srch=doc-info(%23toc%237051%232007%23998719996%23643816%23FLA%23display%23Volume)&_cdi=7051&_sort=d&_docanchor=&_ct=24&_acct=C000014438&_version=1&_urlVersion=0&_userid=209690&md5=820ef90d8b289e80c8cb343a987183cd]
<br>


*The Highly Cooperative Folding of Small Naturally Occurring Proteins Is Likely the Result of Natural Selection
Alexander L. Watters1, 6, Pritilekha Deka2, Colin Corrent3, David Callender3, Gabriele Varani2, 3, Tobin Sosnick4 and David Baker<br>
To illuminate the evolutionary pressure acting on the folding free energy landscapes of naturally occurring proteins, we have systematically characterized the folding free energy landscape of Top7, a computationally designed protein lacking an evolutionary history. Stopped-flow kinetics, circular dichroism, and NMR experiments reveal that there are at least three distinct phases in the folding of Top7, that a nonnative conformation is stable at equilibrium, and that multiple fragments of Top7 are stable in isolation. These results indicate that the folding of Top7 is significantly less cooperative than the folding of similarly sized naturally occurring proteins, suggesting that the cooperative folding and smooth free energy landscapes observed for small naturally occurring proteins are not general properties of polypeptide chains that fold to unique stable structures but are instead a product of natural selection.
[http://www.cell.com.ezp1.harvard.edu/content/article/fulltext?uid=PIIS0092867407001171]

===EMBO===

===Genetics===


===Journal of Bacteriology===


===Journal of Chemical Physcis===


===Journal of Molecular Biology===


===Journal of Physical Chemistry-A===


===Journal of Physical Chemistry-B===


===Journal of Physical Chemistry-C===


===Journal of Physical Chemistry-D===


===Journal of Physical Chemistry-E===


===Journal of Statistical Physics===


===Journal of Theoretical Biology===


===Molecular Microbiology===


===Molecular Systems Biology===
*'''Noise in timing and precision of gene activities in a genetic cascade'''
''Amnon Amir, Oren Kobiler, Assaf Rokney, Amos B Oppenheim and Joel Stavans''<br>
Biological developmental pathways require proper timing of gene expression. We investigated timing variations of defined steps along the lytic cascade of bacteriophage . Gene expression was followed in individual lysogenic cells, after induction with a pulse of UV irradiation. At low UV doses, some cells undergo partial induction and eventually divide, whereas others follow the lytic pathway. The timing of events in cells committed to lysis is independent of the level of activation of the SOS response, suggesting that the lambda network proceeds autonomously after induction. An increased loss of temporal coherence of specific events from prophage induction to lysis is observed, even though the coefficient of variation of timing fluctuations decreases. The observed temporal variations are not due to cell factors uniformly dilating the timing of execution of the cascade. This behavior is reproduced by a simple model composed of independent stages, which for a given mean duration predicts higher temporal precision, when a cascade consists of a large number of steps. Evidence for the independence of regulatory modules in the network is presented.
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v3/n1/pdf/msb4100113.pdf]
<br>


*'''Random timing in signaling cascades'''
''Juan M Pedraza and Johan Paulsson''<br>
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v3/n1/pdf/msb4100121.pdf]


*'''Comparative proteomic and transcriptomic profiling the fission yeast Schizosaccharomyces pombe'''
''Michael W Schmidt, Andres Houseman, Alexander R Ivanov and Dieter A Wolf''<br>
The fission yeast Schizosaccharomyces pombe is a widely used model organism to study basic mechanisms of eukaryotic biology, but unlike other model organisms, its proteome remains largely uncharacterized. Using a shotgun proteomics approach based on multidimensional prefractionation and tandem mass spectrometry, we have detected 30% of the theoretical fission yeast proteome. Applying statistical modelling to normalize spectral counts to the number of predicted tryptic peptides, we have performed label-free quantification of 1465 proteins. The fission yeast protein data showed considerable correlations with mRNA levels and with the abundance of orthologous proteins in budding yeast. Functional pathway analysis indicated that the mRNA–protein correlation is strong for proteins involved in signalling and metabolic processes, but increasingly discordant for components of protein complexes, which clustered in groups with similar mRNA–protein ratios. Self-organizing map clustering of large-scale protein and mRNA data from fission and budding yeast revealed coordinate but not always concordant expression of components of functional pathways and protein complexes. This finding reaffirms at the protein level the considerable divergence in gene expression patterns of the two model organisms that was noticed in previous transcriptomic studies.
[http://www.nature.com.ezp1.harvard.edu/msb/journal/v3/n1/pdf/msb4100117.pdf]
<br>


===Nature===


===Nature Biotechnology===


===Nature Genetics===


===Plasmid===


===PLOS===


===PLoS Biology===


===PNAS===
*'''Regulation of yeast oscillatory dynamics'''
''Douglas B. Murray, Manfred Beckmann and Hiroaki Kitano''<br>

When yeast cells are grown continuously at high cell density, a respiratory oscillation percolates throughout the population. Many essential cellular functions have been shown to be separated temporally during each cycle; however, the regulatory mechanisms involved in oscillatory dynamics remain to be elucidated. Through GC-MS analysis we found that the majority of metabolites show oscillatory dynamics, with 70% of the identified metabolite concentrations peaking in conjunction with NAD(P)H. Through statistical analyses of microarray data, we identified that biosynthetic events have a defined order, and this program is initiated when respiration rates are increasing. We then combined metabolic, transcriptional data and statistical analyses of transcription factor activity, identified the top oscillatory parameters, and filtered a large-scale yeast interaction network according to these parameters. The analyses and controlled experimental perturbation provided evidence that a transcriptional complex formed part of the timing circuit for biosynthetic, reductive, and cell cycle programs in the cell. This circuitry does not act in isolation because both have strong translational, proteomic, and metabolic regulatory mechanisms. Our data lead us to conclude that the regulation of the respiratory oscillation revolves around coupled subgraphs containing large numbers of proteins and metabolites, with a potential to oscillate, and no definable hierarchy, i.e., heterarchical control. [http://www.pnas.org/cgi/reprint/104/7/2241]

*'''A synthetic time-delay circuit in mammalian cells and mice'''
''Wilfried Weber, Jörg Stelling{ddagger}, Markus Rimann, Bettina Keller, Marie Daoud-El Baba, Cornelia C. Weber, Dominique Aubel, and Martin Fussenegger'' <br>

Time-delay circuitries in which a transcription factor processes independent input parameters can modulate NF-{kappa}B activation, manage quorum-sensing cross-talk, and control the circadian clock. We have constructed a synthetic mammalian gene network that processes four different input signals to control either immediate or time-delayed transcription of specific target genes. BirA-mediated ligation of biotin to a biotinylation signal-containing VP16 transactivation domain triggers heterodimerization of chimeric VP16 to a streptavidin-linked tetracycline repressor (TetR). At increasing biotin concentrations up to 20 nM, TetR-specific promoters are gradually activated (off to on, input signal 1), are maximally induced at concentrations between 20 nM and 10 µM, and are adjustably shut off at biotin levels exceeding 10 µM (on to off, input signal 2). These specific expression characteristics with a discrete biotin concentration window emulate a biotin-triggered bandpass filter. Removal of biotin from the culture environment (input signal 3) results in time-delayed transgene expression until the intracellular biotinylated VP16 pool is degraded. Because the TetR component of the chimeric transactivator retains its tetracycline responsiveness, addition of this antibiotic (input signal 4) overrides biotin control and immediately shuts off target gene expression. Biotin-responsive immediate, bandpass filter, and time-delay transcription characteristics were predicted by a computational model and have been validated in standard cultivation settings or biopharmaceutical manufacturing scenarios using trangenic CHO-K1 cell derivatives and have been confirmed in mice. Synthetic gene circuitries provide insight into structure–function correlations of native signaling networks and foster advances in gene therapy and biopharmaceutical manufacturing. [http://www.pnas.org/cgi/content/full/104/8/2643]


===PRLandE===


===Quarterly Reviews of Biophysics===


===Science===


===Systems Biology===