'''Quantitative reverse transcriptase PCR''' (QRT-PCR) is a [[PCR techniques|PCR technique]] used to determine the amount of cDNA in a sample. It is the most commonly used form of [[quantitative PCR]] (qPCR). This technique is also called ''real-time reverse transcriptase PCR''.

== Reference mRNAs ==

A mRNA used as reference or standard of a Q-PCR (and other experiments) should have the following properties:
* expressed in all cells
* constant copy number in all cells
* medium copy number for more accuracy (or similar copy number to gene of interest)

Common reference mRNAs linked to known mouse primer pairs:
* &beta;-actin (common cytoskeletal enzyme) [http://medgen.ugent.be/rtprimerdb/search_results.php?id=&application=-1&organism=2&gene=actin&string_type=substring&detection=-1&primer=&locuslink=&snp=&last_name=&pubmed=&order=0&search=Search+Primers&first_result=0], [http://www.realtimeprimers.org/SYBR%20Green/Mouse%20SYBR%20Green.html]
* glyceraldehyde-3-phosphate dehydrogenase GAPDH (common metabolic enzyme) [http://medgen.ugent.be/rtprimerdb/search_results.php?primer=&application=-1&detection=-1&locuslink=&snp=&last_name=&order=0&search=Search+Primers&first_result=0&pubmed=&string_type=substring&gene=gapdh&organism=2&Search=Search&id=], [http://www.realtimeprimers.org/SYBR%20Green/Mouse%20SYBR%20Green.html] 
* ribosomal proteins (e.g. RPLP0) and RNAs (28S or 18S)
* cyclophilin mRNA
* MHC I (major histocompatibility complex I)

Search primer repositories like [http://medgen.ugent.be/rtprimerdb/ RTPrimerDB] (see also below) and check the literature before doing it from scratch. <br>
Check out the Eccles Lab collection of human and mouse [[Eccles:QPCR_reference_genes| qPCR reference genes]] on OWW.

=== Additional considerations in choosing reference genes ===
==== Stability ====
*'''[[User:Ajeffs|Ajeffs]] 06:55, 21 April 2007 (EDT):''' In addition to the given requirements of good (well, acceptable) specificity and efficiency of the reference gene primers, the next most important aspect of reference gene selection is stability. I don't care if the CT value of my reference genes (yes, genes, not gene) is close to the target genes/s or not - as long as the efficiency of all the primers is similar, and they are all working within their respective limits of detection i.e. linear range, then the stability of the reference genes between samples, treatments, etc. is the most crucial aspect of generating believable qPCR results.

==== Selection ====
*'''[[User:Ajeffs|Ajeffs]] 06:55, 21 April 2007 (EDT):''' Screen a handful of ref genes, select the most stable using genorm, bestkeeper etc, use at least 2 reference genes for subsequent reactions and normalisation. Inlcude your genorm M values when publishing qPCR data. 

==== Use of 18S ====
*'''[[User:Ajeffs|Ajeffs]] 06:55, 21 April 2007 (EDT):''' 18S is generally a terrible choice for a reference gene thanks to the combination of (i) high abundance (creating a 1:100 dilution of template to run in parallel with neat template just for 18S is a complete drag); and (ii) having different degradation characteristics to mRNAs (it appears to be more resistant to degradation). However, if you can show that you have screened 5-10 reference genes, and 18S is still the best for your specific situation then so be it (but do try 28S if you or you PI is hung-up on 18S).

== Primer selection ==
=== Primer repositories ===
*[http://medgen.ugent.be/rtprimerdb/ RTPrimerDB] is an excellent database storing known primer and probe sequences for popular techniques (SYBR Green I, Taqman, Hybridisation Probes, Molecular Beacon). This can help you save the time of designing and testing your own primers. It is also intended to facilitate standardisation among different laboratories. The database is hosted by the Center for Medical Genetics, Gent, Belgium. Please submit you tested primer pairs.

*[http://www.realtimeprimers.org/ RealTimePrimers.org] is another such database but it contains only few primers.

*[http://pga.mgh.harvard.edu/primerbank/ PrimerBank]. From the website "''PrimerBank is a public resource for PCR primers. These primers are designed for gene expression detection or quantification (real-time PCR). PrimerBank contains over 306,800 primers covering most known human and mouse genes. ...The primer design algorithm has been extensively tested by real-time PCR experiments for PCR specificity and efficiency. We have tested 26,855 primer pairs that correspond to 27,681 mouse genes by Real Time PCR followed by agarose gel electrophoresis and sequencing of the PCR products. The design success rate is 82.6% (22,187 successful primer pairs) based on agarose gel electrophoresis''". Don't neglect to check the efficiency and specificity of the oligos yourself though. Link to [http://nar.oxfordjournals.org/cgi/content/abstract/31/24/e154 paper].

*[http://primerdepot.nci.nih.gov/ qPrimerDepot]. From the website "''This database provides qRT PCR primers for 99.96% human RefSeq sequences. For 99% of intron-bearing genes, the PCR product will cross an exon-exon border which overlaps one of the largest introns. All primers have annealing temperatures approximately 60C''". Link to [http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1635330 paper].

=== Primer design ===
An excellent and fast way to select primers is with the free online-tool [http://fokker.wi.mit.edu/primer3/input.htm Primer3], currently in v0.3. [http://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi Primer3Plus], a variation of Primer3 has qPCR settings. Or just apply the following or similar settings to Primer3:
*pair towards 3' end (often more specific, some cDNAs don't contain)
*pair separated by an exon-exon boundary (reduces genomic background) e.g. last exon & penultimate
*amplified region must be no biger than 200 bp; usally 60-150 bp
*GC content: 50-60%
*min length: 18, max length 24 (best: 20 nt)
*melting temperature: min 60, max 63, best 60
*max Tm difference: 10 (shouldn't be more than 1 in final pair)
*max 3' self complementary: 1
*max poly-x: 3

Verify by blasting the primers sequences. Target gene should come out with the lowest E value. No other gene should be close. Also check whether possible isoforms will be detected by the candidate primer pair. See also: [[Designing primers]]

== Notes ==
*The most commonly used specialist reverse transcriptase enzyme for cDNA production is AMV reverse transcriptase.  It has RNase H activity (so that RNA molecules are only transcribed once) and has a high temperature stability (to reduce RNA secondary structure and nonspecific primer annealing) <cite>MeasuringGeneExpression</cite>.
*Since RNA can degrade with repeated freeze-thaw steps, experimental variability is often seen during successive reverse transcription reactions of the same RNA sample <cite>MeasuringGeneExpression</cite>.
*Reverse transcriptase enzymes are notorious for their thermal instability.  Repeated removals from the freezer can degrade the efficiency of the enzyme <cite>MeasuringGeneExpression</cite>.
*Producing total cDNA from total RNA can be advantageous because 
*#cDNA is more stable than RNA so making total cDNA allows you to make multiple sequence-specific RNA measurements <cite>MeasuringGeneExpression</cite>.
*#This approach could reduce experimental variability stemming from RNA degradation <cite>MeasuringGeneExpression</cite>.
*To make total cDNA
*#Use a polyT primer (most but not all eukaryotic mRNA) or random decamers (prokaryotic mRNA) <cite>MeasuringGeneExpression</cite>.
*#Random decamers give longer cDNAs on average than random hexamer primers <cite>MeasuringGeneExpression</cite>.
*#Use longer reverse transcription reaction times <cite>MeasuringGeneExpression</cite>.
*#Ensure that the concentration of deoxynucleotides doesn't run out <cite>MeasuringGeneExpression</cite>.

== Specific protocols ==
*[[Endy:Real-time RT-PCR]] (in progress)
*[[qRT-PCR/Two tubes]] (in progress)
*[[qRT-PCR/Single tube]]

== See also ==
* [[Q-PCR]] and [[Real-time PCR]]
* [[PCR techniques]]

== External links ==
* An excellent, detailed [http://pathmicro.med.sc.edu/pcr/realtime-home.htm Q-PCR tutorial] by Margaret and Richard Hunt, University of South Carolina
* The venerable [http://tech.groups.yahoo.com/group/qpcrlistserver/ qpcrlistserv]. Anyone doing qPCR should be subscribed to this list.

== References ==
<biblio>
#MeasuringGeneExpression isbn=0415374723
</biblio>

[[Category:Protocol]] [[Category:In vitro]] [[Category:RNA]]