==Overview==
Single-gene knockouts using λ red system, adapted from [http://www.pnas.org/cgi/content/full/97/12/6640 Datsenko and Wanner] paper.  The goal of this protocol was to create an endA (endonuclease I) knockout, but obviously it can be adapted to any gene.  The knocked-out gene is replaced with an antibiotic resistance gene, usually for kanamycin or chloramphenicol.  In this example, the target strain was already kanamycin resistant, so the chloramphenicol resistance gene was used.

==Materials==
''Not a complete list — see protocol for details, or update this list'' --[[User:ShawnDouglas|smd]] 20:20, 5 March 2007 (EST)
*plasmids
**pKD46
**pKD3 (chloramphenicol)
**pKD4 (kanamycin)
**pCP20 (optional)
*reagents
**L-arabinose
*equipment
**incubators (30{{c}} and 37{{c}})
**electroporator

==Procedure==
*'''General outline'''
**Grow up pKD46, pKD3, and pCP20 in host strains
**Perform minipreps to extract plasmids
**Transform pKD46 into target strain, plate out on LB-amp plates
**PCR amplify linear fragment from pKD3 or pKD4 using oligos ''A'' and ''B'' (see below for design)
**Make target strain (now maintaining pKD46) electrocompetent by growing at 30{{c}} with L-arabinose
**Electroprate linear DNA into electrocompetent cells
**Grow at 37{{c}} on chloramphenicol plates
**PCR verify the deletion with oligos ''C'' and ''D'' (see below for design)
*'''Detailed procedure'''
*'''Day 0: Start overnight culture'''
**Start overnight culture of strain containing gene to knock out.
*'''Day 1: Preparation and transformation of competent cells'''
**Make new glycerol stock of overnight strain (grown from single colony)
**Add 300 {{ul}} overnight culture to 30 mL LB medium (1:100 dilution)
**Check culture density every 30 minutes starting at +1 hour; grow to {{od|600}} of 0.3 to 0.4
**{{od|600}} measurements of K91
***+2:00 hrs: 0.06
***+2:45 hrs: 0.3008
**Spin at 2500rcf for 10 minutes at 4{{c}} in two 50 mL centrifuge tubes (JA-20 rotor)
**Decant supernatant, discard
**Resuspend each pellet in 5 mL ice cold transformation buffer; swirl or pipette gently to mix
***'''Transformation Buffer'''
***10 mM Pipes
***15 mM {{cacl2}}
***250 mM KCl
***Titrate to pH 6.7 before adding {{mncl2}}
***55 mM {{mncl2}}
***Filter sterilize
**Incubate on ice for 10 min
**Spin at 2500rcf for 10 minutes at 4{{c}}
**Decant supernatant, discard
**Resuspend each pellet in 1.25 mL ice cold transformation buffer
**Combine resuspended pellets in single tube
**Remove 400 {{ul}} for immediate transformation
**Add DMSO to a final concentration of 7% (160 {{ul}}). Drip the DMSO slowly into the cell suspension, with constant swirling by hand.
**Incubate on ice for 10 min
**Aliquot 400 {{ul}} each into five 1.5 mL tubes
**Store in -80{{c}} freezer.
**Transform strain with pKD46 and grow on LB-amp plate at 30{{c}}
***Prepare four tubes with 0, 1 ng, 10 ng, and 100 ng pKD46 plasmid DNA
***Add 100 {{ul}} of competent cell mix to each tube
***Incubate on ice 30 min
***Heat shock 30 seconds at 42{{c}}
***Incubate on ice 2 min
***Spread all 100 {{ul}} on LB-amp-kan plate
***Incubate at 30{{c}} overnight
**U45endA—pKD3[Cat]—D45endA fragment is chloramphenicol cassette with FRT sequences, flanked by 45 bp upstream and downstream of endA.
**PCR U45endA— pKD3[Cat]—D45endA from pKD3 and verify on gel
***PCR program: 95{{c}} 7min &rarr; 35*[94{{c}} 15s &rarr; 50{{c}} 30s &rarr; 72{{c}} 90s]
::{| {{table}}
| align="center" style="background:#f0f0f0;"|'''Contents'''
| align="center" style="background:#f0f0f0;"|'''Concentration'''
| align="center" style="background:#f0f0f0;"|'''Volume'''
|-
| pKD3 template||45 ng/{{ul}}||0.5 {{ul}}
|-
| forward primer||10 {{um}}||5 {{ul}}
|-
| reverse primer||10 {{um}}||5 {{ul}}
|-
| 10x KOD buffer||-||10 {{ul}}
|-
| dNTP||2 mM||10 {{ul}}
|-
| {{mgso4}}||25 mM||4 {{ul}}
|-
| KOD polymerase||||2 {{ul}}
|-
| d{{h2o}}||-||64.5 {{ul}}
|}
**Make ten 10 {{ug}}/ml chloramphenicol plates and ten 25 {{ug}}/ml chloramphenicol plates

*'''Day 2'''
**Make 1 M stock of L-arabinose
***MW of L-arabinose is 150.13
***Add 1501.3 mg of L-arabinose to 8.5 g d{{h2o}} to make 1 M stock
**Retrieve plates from incubator
**Check results from the transformation
***1 ng pKD46 transformation yielded about 10 colonies
***10 ng pKD46 transformation yielded about 100 colonies
***100 ng pKD46 transformation yielded several hundred colonies
**Pick some colonies and grow at 30{{c}} in 2 mL LB + 50 {{ug}}/mL Amp
***add 50 {{ul}} 10 mg/mL Ampicillin stock
**Include enough samples for two conditions: +/- L-arabinose induction
**When {{od|600}} of cells(+pKD46) reaches 0.1, add L-arabinose to concentration of 10 mM to induce pKD46 &lambda;-red expression
***add 20 {{ul}} 1 M L-arabinose to 2 mL culture
**Continue to grow at 30{{c}} to {{od|600}} = 0.4
**Aliquot 1 mL each into two 1.5 mL centrifuge tubes
**Chill cells in ice-water bath 10 minutes
**Centrifuge 10 min at 4000rcf 4{{c}}
**Pipette off supernatant and resuspend pellets in 1 mL ice-cold d{{h2o}}
**Centrifuge 10 min at 4000rcf 4{{c}}
**Resuspend pellet in 50 {{ul}} d{{h2o}}
**For electroporation step, include 2 conditions: +/- PCR fragment
**Chill electroporation cuvettes for 5 minutes on ice
**Add 5 pg to 0.5 {{ug}} PCR amplified DNA to cells
**Set electroporation apparatus to 2.5 kV, 25 &mu;F.  Set the pulse controller to 200 ohms
**Place the cuvette into the sample chamber
**Apply the pulse by pushing the button
**Remove the cuvette.  Immediately add 1 mL LB medium and transfer to a sterile culture tube
**Incubate 60-120 min with moderate shaking at 37{{c}}
**Plate aliquots of the transformation culture on LB plates supplemented with chloramphenicol (10 {{ug}}/mL, 25 {{ug}}/mL)

==Notes==
'''Designing necessary primers'''
*First look at the sequence of the plasmid containing the resistance marker you wish to swap in for your target gene.
**[http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&val=15554329 NCBI sequence viewer: pKD3]
***priming site 1: {{blue|GTGTAGGCTGGAGCTGCTTC}}
***priming site 2: {{purple|GGACCATGGCTAATTCCCAT}}
***priming site 2 reverse complement: {{green|ATGGGAATTAGCCATGGTCC}}


*pKD3 Cat sequence (1033 bases)
<span style="font-family:Courier,monospace;">
{{blue|GTGTAGGCTGGAGCTGCTTC}}GAAGTTCCTATACTTTCTAGAGAATAGGAACTTCGGAATAGGAACTTCTTTAAATGGCGCGCCTTACGCCCCGCCC
TGCCACTCATCGCAGTACTGTTGTATTCATTAAGCATCTGCCGACATGGAAGCCATCACAAACGGCATGATGAACCTGAATCGCCAGCGGCATCAGCACCTTGTC
GCCTTGCGTATAATATTTGCCCATGGTGAAAACGGGGGCGAAGAAGTTGTCCATATTGGCCACGTTTAAATCAAAACTGGTGAAACTCACCCAGGGATTGGCTGA
GACGAAAAACATATTCTCAATAAACCCTTTAGGGAAATAGGCCAGGTTTTCACCGTAACACGCCACATCTTGCGAATATATGTGTAGAAACTGCCGGAAATCGTC
GTGGTATTCACTCCAGAGCGATGAAAACGTTTCAGTTTGCTCATGGAAAACGGTGTAACAAGGGTGAACACTATCCCATATCACCAGCTCACCGTCTTTCATTGC
CATACGTAATTCCGGATGAGCATTCATCAGGCGGGCAAGAATGTGAATAAAGGCCGGATAAAACTTGTGCTTATTTTTCTTTACGGTCTTTAAAAAGGCCGTAAT
ATCCAGCTGAACGGTCTGGTTATAGGTACATTGAGCAACTGACTGAAATGCCTCAAAATGTTCTTTACGATGCCATTGGGATATATCAACGGTGGTATATCCAGT
GATTTTTTTCTCCATTTTAGCTTCCTTAGCTCCTGAAAATCTCGACAACTCAAAAAATACGCCCGGTAGTGATCTTATTTCATTATGGTGAAAGTTGGAACCTCT
TACGTGCCGATCAACGTCTCATTTTCGCCAAAAGTTGGCCCAGGGCTTCCCGGTATCAACAGGGACACCAGGATTTATTTATTCTGCGAAGTGATCTTCCGTCAC
AGGTAGGCGCGCCGAAGTTCCTATACTTTCTAGAGAATAGGAACTTCGGAATAGGAACTAAGGAGGATATTCATAT{{purple|GGACCATGGCTAATTCCCAT}}

*Next, find the sequence (and context sequence) of the gene you wish to remove
**In this example, I found the sequence and context for endA (808 bases total)
***MG1655_m56_ABE-0009661 +50bp upstream +50bp downstream
<span style="font-family:Courier,monospace;">
{{orange|
CCAAAACAGCTTTCGCTACGTTGCTGGCTCGTTTTAACACGGAGTAAGTG}}ATGTACCGTTATTTGTCTATTGCTGCGGTGGTACTGAGCGCAGCATTTTCCGGC
CCGGCGTTGGCCGAAGGTATCAATAGTTTTTCTCAGGCGAAAGCCGCGGCGGTAAAAGTCCACGCTGACGCGCCCGGTACGTTTTATTGCGGATGTAAAATTAA
CTGGCAGGGCAAAAAAGGCGTTGTTGATCTGCAATCGTGCGGCTATCAGGTGCGCAAAAATGAAAACCGCGCCAGCCGCGTAGAGTGGGAACATGTCGTTCCCG
CCTGGCAGTTCGGTCACCAGCGCCAGTGCTGGCAGGACGGTGGACGTAAAAACTGCGCTAAAGATCCGGTCTATCGCAAGATGGAAAGCGATATGCATAACCTG
CAGCCGTCAGTCGGTGAGGTGAATGGCGATCGCGGCAACTTTATGTACAGCCAGTGGAATGGCGGTGAAGGCCAGTACGGTCAATGCGCCATGAAGGTCGATTT
CAAAGAAAAAGCTGCCGAACCACCAGCGCGTGCACGCGGTGCCATTGCGCGCACCTACTTCTATATGCGCGACCAATACAACCTGACACTCTCTCGCCAGCAAA
CGCAGCTGTTCAACGCATGGAACAAGATGTATCCGGTTACCGACTGGGAGTGCGAGCGCGATGAACGCATCGCGAAGGTGCAGGGCAATCATAACCCGTATGTG
CAACGCGCTTGCCAGGCGCGAAAGAGCTAA{{red|CCTACACTAGCGGGATTCTTTTTGTTAACCCCTACCCCACGCGTACAACC}}
</span>

*Construct primers that have internal overlap with the resistance marker (pKD3) and external overlap with the target knockout gene (endA).
**Forward primer: ''A''
***{{orange|CCAAAACAGCTTTCGCTACGTTGCTGGCTCGTTTTAACACGGAGTAAGTG}}{{blue|GTGTAGGCTGGAGCTGCTTC}}
**Reverse primer: ''B''
***{{red|GGTTGTACGCGTGGGGTAGGGGTTAACAAAAAGAATCCCGCTAGTGTAGG}}{{green|ATGGGAATTAGCCATGGTCC}}

*Construct primers that only flank the target gene (endA) for PCR verification
**Forward primer: ''C''
***{{orange|CCAAAACAGCTTTCGCTACGTTGCT}} (25 bases)
**Reverse primer: ''D''
***{{red|GGTTGTACGCGTGGGGTAGGGGTTA}} (25 bases)

*Figure out the sequence and size of what you should expect if everything works.  In this case, it's Cat inserted into endA flanking region''' (1132 bases total)
<span style="font-family:Courier,monospace;">
{{orange|CCAAAACAGCTTTCGCTACGTTGCTGGCTCGTTTTAACACGGAGTAAGTG}}{{blue|GTGTAGGCTGGAGCTGCTTC}}GAAGTTCCTATACTTTCTAGAGAATAGGAACTTC
GGAATAGGAACTTCATTTAAATGGCGCGCCTTACGCCCCGCCCTGCCACTCATCGCAGTACTGTTGTATTCATTAAGCATCTGCCGCATGGAAGCCATCACAAA
CGGCATGATGAACCTGAATCGCCAGCGGCATCAGCACCTTGTCGCCTTGCGTATAATATTTGCCCATGGTGAAAACGGGGGCGAAGAAGTTGTCCATATTGGCC
ACGTTTAAATCAAAACTGGTGAAACTCACCCAGGGATTGGCTGAGACGAAAAACATATTCTCAATAAACCCTTTAGGGAAATAGGCCAGGTTTTCACCGTAACA
CGCCACATCTTGCGAATATATGTGTAGAAACTGCCGGAAATCGTCGTGGTATTCACTCCAGAGCGATGAAAACGTTTCAGTTTGCTCATGGAAAACGGTGTAAC
AAGGGTGAACACTATCCCATATCACCAGCTCACCGTCTTTCATTGCCATACGTAATTCCGGATGAGCATTCATCAGGCGGGCAAGAATGTGAATAAAGGCCGGA
TAAAACTTGTGCTTATTTTTCTTTACGGTCTTTAAAAAGGCCGTAATATCCAGCTGAACGGTCTGGTTATAGGTACATTGAGCAACTGACTGAAATGCCTCAAA
ATGTTCTTTACGATGCCATTGGGATATATCAACGGTGGTATATCCAGTGATTTTTTTCTCCATTTTAGCTTCCTTAGCTCCTGAAAATCTCGACAACTCAAAAA
ATACGCCCGGTAGTGATCTTATTTCATTATGGTGAAAGTTGGAACCTCTTACGTGCCGATCAACGTCTCATTTTCGCCAAAAGTTGGCCCAGGGCTTCCCGGTA
TCAACAGGGACACCAGGATTTATTTATTCTGCGAAGTGATCTTCCGTCACAGGTAGGCGCGCCGAAGTTCCTATACTTTCTAGAGAATAGGAACTTCGGAATAG
GAACTAAGGAGGATATTCATAT{{purple|GGACCATGGCTAATTCCCAT}}{{red|CCTACACTAGCGGGATTCTTTTTGTTAACCCCTACCCCACGCGTACAACC}}
</span>

==References==
'''Literature'''
<biblio>
#Datsenko-PNAS-2000 pmid=10829079
</biblio>

'''λ red Links'''
*[http://www.genome.wisc.edu/functional/tnmutagenesis.htm Schematic Overview]
*[http://falkow.stanford.edu/whatwedo/general/wanner.pdf Protocol from Charlie Kim]

'''endA Links'''
*[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene&cmd=Retrieve&dopt=full_report&list_uids=949092 Entrez Gene: endA]
*[https://asap.ahabs.wisc.edu/asap/feature_info.php?FeatureID=ABE-0009661 ASAP annotation for K12 endA]

==Contact==
*[[User:ShawnDouglas|Shawn Douglas]] — posted original protocol, performed once in August 2006.

[[Category:Protocol]] [[Category:Escherichia coli]]