==Introduction==
When receiving oligonucleotide primers from a manufacturer such as Invitrogen, the oligos arrive dry and must be resuspended in buffer.  The proper choice of buffer will depend on the intended application of the primers, some common ones are:
#Sequencing/PCR
#[[Annealing Primers|Annealing]]

==Method==
Invitrogen recommends the following [http://www.invitrogen.com/content.cfm?pageid=9974 reconstitution procedure] - 

*Centrifuge the tube for a few seconds to get all the DNA to the bottom of the tube.
*Resuspend in [[TE buffer]], pH 8.0 at a concentration greater than 10&mu;M.
*Allow to sit for 2mins, then vortex for 15s.

However, if you are using primers for [[PCR]]/Sequencing you may want to resuspend in water or 100nM Tris, since the [[EDTA]] in [[TE buffer]] chelates Mg<sup>2+</sup> ions inhibiting PCR.  You could also keep track of the amount of EDTA in the mix to make sure there is still enough Mg<sup>2+</sup> for your reaction to proceed successfully. Each [[EDTA]] molecule chelates one Mg<sup>2+</sup> ion.
*You typically want a small amount of [[EDTA]] around to chelate metals other than Mg. Heavier metals, like Fe, are more likely to wreck havoc on your favorite biological macromolecules and are chelated more strongly by [[EDTA]] than Mg. That said, I typically resuspend all of my oligos in [[TE buffer]] (mine is 10 mM Tris-HCl, pH 7.5, 1 mM [[EDTA]]) at a concentration of 100 &mu;M. I use 0.5 &mu;L of this in a 100 &mu;L PCR reaction. This leaves me with 5 &mu;M [[EDTA]] final, which is insignificant compared to the mM concentrations of Mg<sup>2+</sup> used in the reaction. [http://www.chm.bris.ac.uk/motm/edta/edtah.htm Here's] a great page I found about [[EDTA]], including [http://www.cem.msu.edu/~cem333/EDTATable.html formation constant] (K<sub>f</sub>) values for metal-EDTA complexes.--[[User:Kathmc|Kathleen]]

[[Category:Protocol]]
[[Category:DNA]]
[[Category:In vitro]]