==Frequently asked questions and objections==

'''Disclaimer: These are personal opinions developed as a result of my own thinking on this subject and interactions with others (which I have tried to note as applicable).  They are highly likely to evolve over time.  If you have a comment/question on something here, send me an email.'''

===But life isn't digital!===
The most common question I receive when talking about my work with others is "Biology isn't digital, so why concentrate on digital logic"?  There's an answer on the [[SB:FAQ | Synthetic Biology FAQ]] but I'll give my own two cents here.  In my mind, there are two valid responses to this question.

====Maybe not, but we are better at thinking digitally.====
This is the answer I usually give.  After thinking about this question a fair amount and talking with others in the MIT Synthetic Biology Working Group (especially Tom Knight), I came to the conclusion that the first "digital" devices in electrical engineering probably weren't very digital either.  In fact, even today's devices aren't 100% digital.  Calling a device "digital" is really an abstraction we place upon a physical object that behaves according to certain specifications.  By carefully determining those behavior requirements and carefully engineering the device, the digital abstraction holds up sufficiently well for the device to work as desired.  A lot of work has already been done in electrical engineering in terms of engineering digital circuits from analog electrical components.  We should be able to leverage this expertise to design digital devices from analog, biological components.

====Sure it is.====
Information in biology is encoded by DNA which consists of strings of 4 kinds of nucleotides.  Therefore life is fundamentally digital.  I first heard this argument in a talk given by [http://www.systemsbiology.org/Scientists_and_Research/Faculty_Groups/Hood_Group Leroy Hood] at MIT in 2003 but I am sure that others use it as well.  If biology at its core is digital, then it is no longer so unreasonable to design digital devices from biological parts.

===BioBricks assembly is too cumbersome.===
Agreed.  I would really like to be able to email an arbitrarily long DNA sequence to a synthesizer sitting in my lab and have it given me a tube with that DNA in it for very little cost.  It would make my Ph.D. go much faster.  Unfortunately, we aren't quite there yet (though some might justifiably disagree).  So as a stopgap measure, BioBricks assembly allows us to construct systems from parts in a *relatively* cheap, efficient and reproducible manner.

However, in my mind there is a clear difference between the concept behind BioBricks itself (and by BioBricks, I am referring to the parts in the [http://parts.mit.edu MIT Registry of Standard Biological Parts]) and the method used to assemble BioBricks together.  There are various ways that one could imagine to [http://www.csail.mit.edu/research/abstracts/abstracts04/html/328/328.html improve assembly] or even eliminate it all together by using DNA synthesis instead.  Regardless of the technique used to fabricate a system, it is still useful to maintain a library of reusable, well-characterized biological parts from which systems can be made.  This is how engineers avoid "reinventing the wheel" so to speak.  In my mind, it is this concept (not the particular assembly technique) that is the key idea behind BioBricks.  Thus, I think BioBricks and the Registry of Standard Biological Parts will still be useful even if/when long DNA synthesis is easily available.

===What's the difference between parts, devices and systems?===

See the discussion on an [[Synthetic Biology:Abstraction hierarchy | abstraction hierarchy]] which initially arose from a discussion with [[Jason Kelly]] and [[Ilya Sytchev]].

===What's the deal with PoPS?===
Content moved to [[PoPS]].

==Random musings==

'''Here's some thoughts that I decided to post.  Feedback is welcome.'''

===Diatribe on ''Escherichia coli'' strains===
One thing that surprises me is how difficult it is to find an ''Escherichia coli'' strain that meets a certain set of specifications.  For instance, I want a strain with the lactose permease knocked out and the arabinose permease under the control of a constitutive promoter so that I can get linear induction with both lactose and arabinose.  I don't think one exists (if it does please email me!).  I also can't find a strain that is lacI<sup>q</sup> and has the lactose permease deleted (again email me if you have this strain!).  I find this situation mildly frustrating.

I also find the nomenclature of ''Escherichia coli'' genotype information to be unnecessarily  confusing but I am willing to let it slide as a historical artifact.  (See the attempt to [[E. coli genotypes#Nomenclature & Abbreviations | decipher the code]].)

However, in my mind what is truly astonishing is the dearth of information available on existing strains and the fact that some of this information is wrong!  Case in point:  I was interested in using a strain of ''Escherichia coli'' with the lacI<sup>q</sup> mutation.  I found various strains that are supposed to have this mutation: [[E. coli genotypes#D1210 | D1210]], [[E. coli genotypes#JM109 | JM109]], [[E. coli genotypes#BW26434, CGSC Strain # 7658 | BW26434]].  Then, since I was getting some anomalous experimental results, Tom suggested that I sequence verify the fact that my strains were lacI<sup>q</sup>.  So I did and lo and behold, none of my sequences had the lacI<sup>q</sup> mutation on the genome.  Now based on my anomalous experimental results (which are no longer so anomalous) and reading of some papers, I think that [[E. coli genotypes#D1210 | D1210]] really is lacI<sup>q</sup> but that it just has lacI<sup>q</sup> on the F plasmid rather than on the genome.  But [[E. coli genotypes#JM109 | JM109]] and [[E. coli genotypes#BW26434, CGSC Strain # 7658 | BW26434]] ... or at least the versions that I sequenced ... are not lacI<sup>q</sup> as documented.  I don't understand how people use these strains without having correct genotype information.  I also don't understand that with all the sequencing centers there are and how many people work on or with ''Escherichia coli'', why all the common lab strains at least don't get sequenced.    Some claim it is a combination of the lack of resources and the fact that this isn't an interesting thing to do.  Quite possibly this is true, but nevertheless, I find this situation unbelievable.  Anyway, it was these experiences that led me to populate the [[Standard E. coli Strain for BioBricks | standard strain page]].

===Inducible promoters are obnoxious===

I am so annoyed at the lack of inducible promoters that provide linear, single cell control of gene expression.  I would vent on this more but Kathleen already presented an objective discussion of this topic  at [[Titratable control of pBAD and lac promoters in individual E. coli cells]].