=Expression Engineering, Day 5, 4/20/07=
==cDNA Synthesis==
;Purpose: To create DNA copies of the mRNA within our yeast cells.  Later they will be fluorescently modified to visualize the levels of each mRNA signal.

'''Protocol'''
#Move 0.684 &mu;g of each sample into eppendorf tubes.
#*FY2068: 6 &mu;L + 4 &mu;L RNAase-free water + 1 &mu;L Cy3 (Green) polyT primer
#*NY389: 10 &mu;L + 1 &mu;L Cy5 (Red) polyT primer
#Heat the annealing reactions to 80째C for 10 minutes then place the tubes on ice for 2 minutes. 
#Microfuge the tubes briefly to spin any condensation or droplets down to the bottom of the tube then add:
#*1 ul of "Superase," an RNase inhibitor (Vial 4)
#*8 ul of cDNA synthesis cocktail. (Received from teaching staff)
#Pipet the contents of your tubes up and down to gently mix, then incubate the cDNA synthesis reactions at 42째 for 1.5 hours.
#Microfuge the tubes briefly then add 3.5 ul of 0.5M NaOH/0.5M EDTA to each tube and pipet up and down to mix. Heat to 65째C for 10 minutes. This step will denature your RNA/cDNA hybrids and degrade the RNA. 
#Add 5 ul of 1M Tris, pH7 to neutralize the contents of each tube.

==Spot Test Results==
{|border=1
!Sample !! Removed !! Notes
|-
|YPD || 4/15 6AM || pH = 5.5
|-
|SC-Lys || 4/15 6AM || FY2068 didn't grow
|-
|Low PH || 4/15 6AM || pH = 5.5
|-
|YP Gal || 4/16 8PM || Didn't grow as well as YPD
|-
|YPD + Rap || 4/16 8PM|| Didn't grow very well
|}

==Sample Microarray Analysis==
;Purpose: To lean how to read and interpret microarray data.

===Small Data Set===
#MFA2 is at Row 5, Column 27
#SLG1's systematic name isis YOR008C
#SNF2's function: chromatin modeling, general RNA polymerase II transcription 
#FUN11 is RBG1 in the yeast genome database
#PSD1 is reported to be localized in the mitochondrian, which agrees with the database.
#MFA2 -> Red:280 Green:114
#Highest Red Value: RPS20, Lowest: SDC25
##These are also the highest/lowest values in Green
##38% of the Red genes have higher values than in Green
#Mean SCD25: 443, Median:1059
#*This tells us that there is one value way off
#Average/Medians
#*Red -> Mean: 1084, Median:465
#*Green -> Mean: 3496, Median: 635
#*This shows us that the green signal may be stronger than the red signal

===Large Data Set===

'''Normalization'''
#Mean Green: 709, Mean Red: 1340, k = 1.89
#Median Green: 697, Median Red: 1330, k = 1.90

These values for k are very close.

'''Background correction'''
#Mean Green BG: 67.9, Mean Red BG: 83.6, k = 1.23
#*Median Green BG: 67.8, Median Red BG: 83.4, k = 1.23
#This difference is smaller than the difference in the data.
#My spot was considerably higher in both backgrounds.  Maybe it's close to the center of the grid and gets light from lots of neighbors.

'''Intensity Ratios'''
#Green is 2.22 times stronger than red on average.
#The average of Log2(Green/Red) is 0.32, indicating that overall, which is the same information as the ratio average.

==Microarray Hybridization==

#Begin by mixing the cDNA pools you have synthesized into one eppendorf tube. The total volume should be 57 ul. Add 43 ul of H2O and 100 ul of 2X Hybridization Buffer. Pipet up and down several times to mix the contents. 
#Heat your hybridization solutions to 80째 for 10 minutes then cool them to room temperature. During this time you will be shown how to assemble the hybridization chambers. 
#Load your samples into the hybridization chambers as follows:
#*open the gasket and place the "Agilent" sticker facing up in the rectangular side of the holder. 
#*Pipet 200 ul of the hyb solution into place 1 or place 2 (see figure)[[Image:Macintosh HD-Users-nkuldell-Desktop-agilentarray.png]]
#*Place the microarray <b><font color = red> "agilent" sticker down</font color> </b> onto the hybridization solution that's in the gasket. 
#*Slide on the top portion of the hybridization chamber and the brace. 
#*Tighten and tape them with your colored tape.

==Summary==
Today we prepared microarrays.  First, we created cDNA from the mRNA in our samples (recognized by the polyA tail).  Then we set the cDNA to hybridize to the microarrays overnight.  We also looked at the results of our spot tests, which didn't seem all that interesting to me, most everything grew, except for FY2068 on -lys.

We collaborated with the purple group today.  We both had the same negative PCR results and are using the same samples (to get higher concentrations of RNA, since we were using the same strains).  Because we were doing the same experiment, we decided to do a dye-swap, to add a control to our data.