*Brainstorming:
**I have for sometime thought of producing synthetic spider silk proteins that would be modified to have RGD (Arg-Gly-Asp) residues to facilitate mammalian cell attachement. Traditionally, biomaterials scientists incorporate these residues using chemical reactions prior to cell seeding. If we were to encode these modified silk proteins in a DNA sequence optimized forprotein ,achinery in E coli, we would be well on our way to exacting the kind of control we need to modify these materials for various medical applications.

*So, thus far, I have made one such monomer with adequate provision for RGD repeats. We could use the E coli quorum sense to harness the E coli machinery for production of such proteins. From the quorum-sensing group's results, it appears that the point of optimum signalling and reception may very well coincide with the point n time when the cell's protein machinery is at a max -- so why not use that to our advantage and produce proteins of interest when and if the cells are at close vicinity to each other and are primed for protein-o-genesis.<br>


==ODs and dilution factors==
[[Image:graph.jpg|thumb|left|250px]]
*Did this experiment to verify that OD and cell dilution factors were linearly related.<br><br><br><br>

==protein denaturation electrophoresis gels==
[[Image:2z_last_denaturation_electrophoresis_27.jpg|thumb|left|400px]]<br>

Description: 6/27 denaturation electrophoresis gel:The rightmost lane is the ladder<br/>
lane 1 Ladder <br>
lane 2 OmpA1 + strep before induction with IPTG<br>
lane 3 OmpA1 + his6 before induction<br>
lane 4 OmpA1 + strep after induction<br> 
lane 5 OmpA1 + his6 after induction<br>
lane 6 OmpA1 + his6 after induction<br>
lane 7 OmpA1 + strep after induction<br>
lane 8 OmpA1 + his6 before induction<br>
lane 9 OmpA1 + strep before induction<br><br><br>
''Note that there were 2 copies of each sample: samples in lanes 2 to 5 were denatured for a 2 minute period as per protocol while samples in lanes 6 to 9 were denatured for a 10 minute period, Sammy S.''<br><br><br>


2. Then went on to investigate the appropriate times to induce expression of the OmpA-fusion protein constructs on the E.coli surface using IPTG<br>
'''Week 3 gel'''<br/>
[[Image: 2007-07-06protGel2.jpg|thumb|left|400px]]<br>
'''Gel 2: induced immediately'''<br>
Lane 1: Ladder<br>
Lane 2: Strep No IPTG<br>
Lane 3: Strep denatured after 30 min<br>
Lane 4: Strep denatured after 60 min<br>
Lane 5: Strep denatured after 90 min<br>
Lane 6: Strep denatured after 120 min <br>
Lane 7: His   No IPTG<br>
Lane 8: His denatured after 30 min <br>
Lane 9: His denatured after 60 min <br>
Lane 10: His denatured after 90 min <br>
This result seems to indicate that immediate induction and denaturation (stoppage) at 30 mins gives maximum expression of proteins. But we suspected this to be an artifact so we redid the assay again with a higher concentration for the samples<br><br><br><br>

[[Image: 2007-07-06protGel1.jpg|thumb|left|400px]]<br>
'''Gel 1'''<br>
Lane 1: Ladder<br>
Lane 2: His 0 min induction<br>
Lane 3: His 30 min induction<br>
Lane 4: His 60 min induction<br>
Lane 5: His 90 min induction<br>
Lane 6: His 120 min induction<br>
Lane 7: Strep 0 min induction<br>
Lane 8: Strep 30 min induction<br>
Lane 9: Strep 60 min induction<br>
Lane 10: Strep 90 min induction<br>
Lane 11: Strep 120 min induction<br>
Lane 12: His 0 min induction 120 min denature<br><br><br><br>

'''Gel 2 samples run again'''

[[Image:1_protGel_Jul7.jpg|thumb|left|400px]]<br>

Lane 1: Ladder<br>
Lane 2: Strep induced at 0 min<br>
Lane 3: Strep denatured after 30 min<br>
Lane 4: Strep denatured after 60 min<br>
Lane 5: Strep denatured after 90 min<br>
Lane 6: Strep denatured after 120 min <br>
Lane 7: His induced at 0 min<br>
Lane 8: His denatured after 30 min <br>
Lane 9: His denatured after 60 min <br>
Lane 10: His denatured after 90 min <br>
Lane 11: His denatured after 120 min <br>
Lane 12: His denatured after 120 min <br><br><br><br><br>

<br><br>
<br><br><br>

==LIGATING RANDOMERS ONTO THE PART OF OMPA'S LOOP 1==
Started off with PCRs to pcr-out OmpA from 2 templates:
+K-12 genomic DNA
+OmpA in Topo plasmid (Kan/Amp resistant)
For the first few days, this procedure gave negative results. Then we changed the protocol a little bit:
#Denatured for 2 mins as opposed to initial 10 mins
#Used fresh dilutions of primers (final conc. was 10 uM)
#Extension temp lowered to 68 degC as opposed to initial 72 degC
*These changes gave us good results as seen in image below:
[[Image: pcr_results.jpg|thumb|left|400px]]<br>
The rightmost lane is the ladder<br/>
lane 1 Ladder <br>
lane 2 BB OmpA from K-12 genome<br>
lane 3 BB OmpA from Topo plasmid<br>
lane 4 BB OmpA from K-12 genome<br> 
lane 5 BB OmpA from Topo plasmid<br>
NB that all the pcr products here have BB ends<br><br><br><br><br><br><br><br><br><br><br><br><br>

*The next step will be to use overlap pcr to create a pcr product that will have the randomer inserted at the beginning of OmpA. So we've ordered for a forward primer with the random (15 mer) library on it. This will be an ingredient in the 2nd step where we use it together with a reverse primer that takes care of the very end of OmpA.
*The first step will therefore utilize the forward primer matching the beginning of OmpA and the reverse primer going only until just before the beginning of OmpA. 
*The two pcr products will be ligated with, say, SpeI (product 1) and XbaI (product 2) and then ligated to produce a sequence much like:
OmpABgeginning-XbaISpeIFusionSite-Randomer(15 nt)-Rest of OmpA with appropriate BB sites

*The first procwedure I utilized involved digesting and ligating the inserts together to form a completet insert -- OmpA modified to include a strep or his tag at the extracellular portion of ompA
* The second strategy depended ona 3 part ligation involving the cip treated vector(already digested with e/p, and the two parts of modified ompA -- portion 1(e/s) and 2(x/p). 

* While all these attempts gave us transformant colonies, and the colony pcr results (see below) were promising, the sequencing results imply something else all together. Pls. note that we did not sequence all the colonies.

*did a cpcr targetting the part 5 IU had inserted into psb1ak3
[[Image: cuts_pcr_part5_focus.jpg|thumb|left|400px]]<br>
and the lanes are:<br>
lane 1: ladder<br>
lane 2: 5jc8<br>
lane 3: 5j8s/p<br>
lane 4: 5j3<br>
Lane 5: 5j3s/p<br>
Lane 6: 85j<br>
Lane 7: 65j<br>
Lane 8: 55j<br>
Lane 9: 75j<br><br><br><br><br><br><br>
j refers to the plasmid that contained the j04500 and part 5
<br><br><br><br><br>

*This week (of 8/10), I decided to enhance my efforts in using the OmpA gene with an NheI site at the psn where we expect to have a extracellular addition to native OmpA. There are 2 main strategies involved:
**(i)whence we will use a double digestion N/P and then fuse in X/P-ended inserts for 2 randomers, strep2 and 6xHis
**(ii)I got good results for this -- they are indicative of a successful transformation in that the unligated, yet digested plasmids had no colonies whatsoever. see below:<br>
[[Image: OmpA_neg.jpg|thumb|left|400px]]
Negative control<br><br><br><br><br>
[[Image: OmpA_pos.jpg|thumb|left|400px]]<br><br><br><br><br>
<br><br><br><br><br><br><br><br><br>Note the white colonies in the experiment proper.<br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
Bead assay for OmpAHis (6xhis added to the c terminus)
[[Image: plates.jpg|thumb|left|400px]]<br>