Submitted by [[User:Smoore|Sean Moore]]
==My Experience Purifying RNA from E. coli==

Regarding RNA extraction, there is a horrible tendency of people to use kits for RNA extraction with bacteria.  With Qiagen (anion exchange) or other silica-binding technologies, the smaller RNAs are not recovered. In addition, cell lysis is completely variable. This led me to find methods of extraction that recovered the RNAs in the same proportions they are found in the cell.  

I first noticed that the ratio of 16S and 23S to 5S (and tmRNA) was WAY too low in RNA prep'd by resuspending E. coli in tri-reagent (called Trizol from here on) as the "lysis" step.  Even though Trizol has phenol and guanidinium thiocyanate, and the samples were heated to 65 degrees for 30 minutes, the large RNAs just didn't get out (a concern for anyone doing quantitative mRNA Northerns as well).

So, I did several things.  This lead to my current protocols.

For some experiments I was doing, I had chased <sup>14</sup>C-uracil into a growing culture for several generations to label stable RNAs.  This was a good reagent to compare the losses of radioactivity with different procedures.  <sup>32</sup>P-PO4 is no good for this type of experiment, it labels all sorts of stuff.  For all of my experiments, I use an RNase I- (<i>rna-</i>) strain to prevent this voracious periplasmic RNase from ruining my stable RNAs during lysis.  If you plan on working with RNA in E. coli, I recommend knocking at least this gene out.  It has no known phenotype.

Keeping in mind that with long chases, radiolabeled substrate pools in the most stable material, so I was primarily looking at ribosomal RNA extraction and recovery.  As I suspected, straight into Trizol only recovered about 20-30% of the counts in the soluble supernatant.  As mentioned above, this was primarily loss of large rRNAs.

I tried resuspending the cells in trizol, and using zirconium beads in a disruptor (not that crappy one on a vortexer, but a special machine that obliterates the sample).  This busted the carcasses and liberated most of the counts.  It told me that the cell wall carcass was probably the problem.

I then prep'd the cells by completely lysing in B-PER (a commercial lysis detergent) under native conditions with egg lysozyme and magnesium present (to keep the ribosomes from falling apart during lysis), then I added Trizol.  This does a good job at liberating practically all of the radioactivity from the insoluble debris; however, I found that precipitations from Trizol (the next step), even with a precipitation carrier, only recovered about half of the RNA!

I no longer use Trizol.

A comparison of the RNAs from the above procedures showed that the recovered amounts of the small tmRNA and 5S rRNA per cell was similar in each case, and that the large rRNAs were reduced in the Trizol-only samples.

==My Current RNA isolation protocol for E. coli==

===For unstable RNAs like mRNAs===
Immediately stabilize the RNA, I like to add one volume of culture to one volume of cold methanol in tubes sitting on ice.  Then I add beads and mechanically disrupt the cells.  I have also used "RNA later" and similar stabilization reagents.  Then I proceed to the organic extraction described down below.

===For stable RNAs===
I dilute the cells into an ice-cold salty washing buffer (seems to really help lysis) 10 mM Tris-Cl pH 8.0, 150 mM NaCl. Then harvest in the cold.  Resuspend the cells in a lysis buffer such as the one that follows.  The Mg++ is there to keep ribonucleoprotein assemblies together and to let the DNase work during lysis.

Native Lysis:
B-Per (probably optional, but I have it on hand) supplement with 1 mM Mg-OAc, 0.5 mM CaCl, ~0.1 mM EDTA, 0.01-0.1 mg/mL lysozyme, and RNase-free DNase (I use Roche DNase I diluted 1/1000 in the B-Per after the Mg and Ca to 10 units per mL final).   DNase I requires calcium in addition to Mg++ and works better in low salt conditions (frequently overlooked facts, Ambion's web site has some good info).

See: [[Sauer:Lysing E. coli with Lysozymes]]


So, You have a mL or so of cells in a microfuge tube, harvest them in the cold and place on ice, and then aspirate the medium (aspiration works better than pipette and is faster). Go just above the pellet, but not TO the pellet.  The liquid will rise up the wall the of the tube and get sucked away leaving a practically dry pellet.

Pipette a fixed defined volume of the lysis solution to each sample (generally 50-100 &mu;L) and vortex into solution at room temp.  Place the tube in a rack at room temp and proceed to the next pellet.  After resuspension, the cells will clear in less than a minute when placed at room temp.

So now you have intact ribosomes in a low-salt solution.  Add about 200-400 uL of salty extraction buffer.  This solution chelates all of the Mg++, disrupts ribosomes, increases ionic strength, and provides a carrier for precipitation.  I use 50 mM bis-Tris pH 6.5, containing 400 mM NaCl, 5 mM EDTA and 1 &mu;L/400 &mu;L of linear polyacrylamide (LPA)at 10 mg/mL.

===Organic Extraction and Precipitation===
Add about 300-500 &mu;L of acidic phenol/chloroform (I buy Ambion's, I still have the same bottle I bought years ago).  If you're adding a control RNA for quantification, now is when you do it.  Vortex the samples, let them stand about 5 min on the bench.   Spin a few minutes and remove the supernatant to a new tube.  If I need cleaner RNA, I repeat the extraction, if I'm just running denaturing gels, I precipitate.

Add slightly-greater-than-equal volume of isopropanol to each sup., vortex.  I see different opinion about the temp this should be done.  I put my tubes on ice for 30 minutes or overnight at -20.  Then I spin for 30-45 minutes.  I think it was Ambionâ€™s site where they showed "spin time" as the most critical factor.  Either your precipitation conditions cause large aggregates that pellet fast, or small aggregates than pellet slow, a long spin here covers both.

Aspirate the isopropanol (don't go near the pellet, it'll jump into your tip and go bye-bye).  Wash with ~700 &mu;L of 75% Ethanol to de-salt, re-spin, aspirate, then wash with 95% ethanol to dehydrate, re-spin, aspirate, and place the tubes on their side on the bench to dry.  Before resuspending the pellets MAKE SURE they are dry.  It seems a small amount of ethanol will cause distortion in the gels, probably from partial precipitation.  Be careful, when they are dry, static can cause the RNA pellet to fly out, just keep your eyes on the prize.

After drying, resuspend either in non-denaturing solution (10 mM bis-Tris, pH 6.5, 0.1 mM EDTA) or denaturing solution (95%  stabilized formamide, 10 mM bis-Tris pH 6.5, 0.1 mM EDTA, some bromophenol blue to taste).  About 50 &mu;L for each mL of culture harvested.  Expect 50-100 &mu;g of total RNA per 10^9 cells.  I run about 1 ug total or 0.5 &mu;g ribosomal RNA per well when staining with Sybr-Green II.

For determining the concentration of your RNA, see this article:  [[Sauer:Nucleic acid quantitation]]

===Images of E. coli RNAs===

[[Image:SDM_RNA1a.tif]]   [[Image:SDM_RNA2a.tif]]   [[Image:SDM_RNA3a.tif]]

====E. coli RNAs resolved in a denaturing gel and subsequently stained with SYBR Green II dye (Invitrogen)====

<b>Left:</b>  Four samples of RNA resolved so that the smallest and largest RNAs are visible.
<b>Middle:</b>  RNA resolved such that larger RNAs are separated.
<b>Right:</b>  RNAs extracted from sucrose gradient fractions.  The gradient was made with low Mg<sup>++</sup> so the 70S ribosomes dissociated into 30S and 50S subunits.



==Notes on RNA stability==

I have seen so many people freak out about having an "RNase-Free" environment.  Washing benches, buying special tips and tubes, using special RNase removal reagents, etc.  This is usually completely unnecessary.  RNases are not gas-phase molecules (ask any mass spectroscopist what it takes to get proteins into the gas phase), once you have purified the RNA, any RNase that gets into your samples is from poor lab technique.  Wear gloves.  Remember that when you handle a pipettor with bare hands, you load it with RNase, this will transfer to your gloves and get on other things. Keyboards and mice are RNase farms too, be mindful of your hands.  The RNA you prep should only see the inside of clean tubes or tips.  If no humans handled them, then they are most likely fine.  Dust settling from the air will provide plenty of RNase if you want it, so keep tubes and tips covered as a general rule, but aside from that, don't panic.  Also, gel running equipment: think about it, under denaturing conditions, no magic RNase is going to eat you RNA while it is being electrophoresed.  If there is active RNase there, it won't co-migrate with your RNA anyway.

Commercial RNase inhibitors generally only work well against eukaryotic RNases.  Chelating divalent cations doesn't work either because many bacterial RNases (like RNase I) use a histidine residue and not metal to activate the water nucleophile.      

Keep your RNA away from high pH.  I see a lot of procedures wherein the RNA was heated or stored in TBE.  Alkaline pH are you crazy?  Use lower pH solutions, I like 10 mM bis-Tris at 6.5.  RNA in this can be diluted into a primer extension reaction without affecting the results.  The acidic phenol/chloroform used in the above protocol does two things.  First, acid stabilizes RNA and denatures protein (aside from the phenol) and, under acidic conditions, DNA fragments will partition to the organic phase.  So you do two things, you protect your RNA and reduce the DNA contamination (this alone is not enough to rid your sample of DNA though).

Keep you RNA away from metal, especially when heated. Use the cleanest water you can find (we have a millipore system, but you can buy "HPLC-grade" water that's quite clean).  Not because of protein RNases, but because of the metals crappy water contains.  If you have to heat an RNA sample for primer extension annealing reaction, alter the protocol so there is EDTA present and the solution is buffered.  Again, I add my primer and RNA to water (they both contain buffer and EDTA), heat that sample and cool it, THEN add the buffer with Mg++, NTPs, and enzyme.  Some commercial protocols have you heat your RNA with metal present.  For short extensions, a little degradation won't be noticed, but for mapping something long like a promoter start site, you donâ€™t' want damaged RNA.

Also, regarding EDTA.  You'll notice in people's protocols, people will have a small amount of EDTA (like 0.1 mM) in solution that contains Mg++.  You may say, what the heck?  Doesn't EDTA chelate Mg++? Yes it does, and it does so quite well (K<sub>d</sub> = ~1.5 nM).  However, it chelates many other metals with much higher affinity (e.g., lead (Pb<sup>2+</sup>, K<sub>d</sub> = 10<sup>-18</sup> M) and iron (Fe<sup>2+</sup>, K<sub>d</sub> = 10<sup>-15</sup> M, Fe<sup>3+</sup>, K<sub>d</sub> = 10<sup>-25</sup> M)).  These are bound so tightly that there is practically none of these metals free when EDTA is present in excess.  So what the EDTA does is removes most of the RNA-damaging metals and whatever's left gets bound with Mg++.

Many people resuspend RNA in water.  This is fine if the samples are to be kept in a -80 freezer.  As a general rule, I always buffer solutions.  Resuspending directly in formamide is a great way to protect your RNA.  Formamide is very dense and makes an excellent loading dye.  It also denatures better than urea so after you heat your samples prior to electrophoresis, there is no panic to get them loaded.  I buy "stabilized" formamide from a company called Molecular Research Center (MRC), add some bis-Tris to it, some EDTA, and some bromophenol blue.  RNA in this denaturant is stable at 4 degrees, but I generally keep mine at -20. An additional benefit is that you can re-precipitate your RNA from the formamide if you feel you need it for other applications.

[[Category:Protocol]]
[[Category:In vitro]]
[[Category:RNA]]