{{Sauer lab sidebar}}

[[Protocols|Return to Protocols]]

Adapted from: Cazenave, C., Uhlenbeck, O.C. ''Proc. Natl. Acad. Sci. USA'' '''1994''', ''91'', 6972–6976.

T7 RNAP=T7 RNA polymerase

U=units

==Protocol==

===Template DNA===
PCR product or linearized plasmid (run-off transcription)

If you use a PCR product, make sure there are at least 5 base pairs upstream of the T7 RNAP promoter. The polymerase needs something to bind to. It is a good idea to have a generic T7 promoter primer that you can use to PCR any template that has the promoter. The one I use has the sequence 5´-GAA AT'''T AAT ACG ACT CAC TAT A'''-3´ (promoter sequence in bold). This primer is also useful for sequencing plasmids that have the T7 RNAP promoter.

Note that if you are designing a template for transcription, T7 RNAP has certain base requirements for transcription initiation. If possible, the first two nucleotides after the promoter should be GG (to be transcribed; CC in the template strand), as these are preferred by T7 RNAP. The polymerase also works reasonably well with AG or GA as the starting nucleotides. If you need the 5´ end of your RNA to be something other than a purine, there are some post-transcriptional modifications that can be employed, typically involving ribozyme- or deoxyribozyme-mediated cleavage of your RNA. Maybe someone will put up a protocol for some of those strategies soon.

I generally recommend using 5–10 pmol of DNA template in a 100 &mu;L transcription reaction. Does this mean you need to determine the concentration of your DNA? Not really, a reasonable estimate is good enough. For a 5000 base pair plasmid, 5 pmol is approximately 16 &mu;g of DNA. For a PCR reaction, estimate the total number of pmols in your PCR by assuming that the reaction went to completion and half of your primers were used up (ex. a reaction with 50 pmol of each primer should yield approximately 25 pmol of extended product).

===Transcription buffer and other components=== 
'''1X buffer:'''

50 mM Tris-HCl, pH 7.5

15 mM MgCl<sub>2</sub>

5 mM dithiothreitol (DTT)

2 mM spermidine

Make 10X stock and store at -20&deg;C.

'''10X NTPs'''

20 mM each of ATP, CTP, GTP, and UTP

Store at -20&deg;C.

'''Inorganic pyrophosphatase'''

Inorganic pyrophosphatase (PPase) catalyzes the hydrolysis of inorganic pyrophosphate to form orthophosphate: 
P<sub>2</sub>O<sub>7</sub><sup>-4</sup> + H<sub>2</sub>O -> 2HPO<sub>4</sub><sup>-2</sup>. Inorganic pyrophosphate is released when a nucleoside triphosphate is incorporated/polymerized into the growing chain. This helps to prevent against any inhibitory effect of having pyrophosphate around (i.e. prevents the "reverse" reaction.) This is an optional component of the transcription reaction. If you leave it out, often you will see something precipitate (white) in your transcription reaction. This is the pyrophosphate.

Make a 0.1 U/&mu;L stock solution in H<sub>2</sub>O and store at -20&deg;C. 

===T7 RNA polymerase===
Clones of T7 RNA polymerase with an N-terminal His-6 tag are available. (see [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=9116496&query_hl=1 He B, Rong M, Lyakhov D, Gartenstein H, Diaz G, Castagna R, McAllister WT, Durbin RK. ''Protein Expr Purif.'' '''1997''', ''9'', 142–151.])

It is highly recommended that you obtain this clone and purify your own polymerase. The prep is easy, you should obtain a large amount of polymerase with high activity from a single prep, and you will save a lot of money by not buying the polymerase.

===Transcription reaction===

For a 100 &mu;L reaction ("preparative scale"):

10 &mu;L 10X transcription buffer

10 &mu;L 10X NTPs

?? &mu;L  DNA template (5–10 pmol) *see above for better description

5 &mu;L inorganic pyrophosphatase (0.1 U/&mu;L): 0.005 U/&mu;L final concentration

?? &mu;L T7 RNAP (25 U/&mu;L final concentration)

OK, so this does sound like a ridiculous amount of enzyme to use. You can get away with a lot less if you are still a slave to corporate America and are purchasing your polymerase. However, if you have produced your own enzyme (see note above), this is not a big deal and you will obtain a truckload of RNA.

Incubate reaction at 37 &deg;C for 2 hr. (You can get away with less time here, you'll just get less RNA).

Add 10U of RNase-free DNase I and incubate at 37 &deg;C, 30 min.

Add 5 &mu;L of 500 mM [[EDTA]] to stop the reaction.

===Clean-up/process the RNA===

*[[phenol/chloroform extraction]] followed by [[nucleic acid precipitation]]

*[[denaturing acrylamide gel purification of nucleic acids]]

*Qiagen RNeasy mini kit

===Determine the concentration of your RNA===

* [[Quantification of nucleic acids]]

===Storing RNA===

*Always store RNA at a neutral pH with some amount of EDTA. I recommend TE buffer (10 mM Tris-HCl, pH 7.5, 1 mM EDTA). Thinking "I'll just put it in water" is a bad idea for RNA (and DNA and proteins and...). Do you really know what's in that water?

Store RNA at -20&deg;C.