==Marine microbes==
===Going against the grain: Chemotaxis and infection in ''Vibrio cholerae''===
[http://www.nature.com/nrmicro/journal/v3/n8/full/nrmicro1205_fs.html]
*one polar flagellum powered by sodium motive force, 100,000 rpm, 60 cell-body lengths/s (60 um/s)
*also twitching motility (extension, adherence, retraction of type IV pili), gliding (slime secretion)
*120,000 deaths from cholera/year, '''how do you determine minimum infectious dose? ~10 cells'''
*humans only known vertebrate host, patients can shed 10 trillion/day, shed cells highly motile
*''V. cholerae'' not particularly resistant to low pH (stomach), requires high infectious dose, acid-adapted V. have competitive advantage
*genes for virulence factors are tightly regulated so not expressed inappropriately
*associates with abiotic and chitinous surfaces in biofilms, motility required for this process
*''E. coli'' and ''B. subtilis'' have only one gene for each che, may be exception
*"Many bacteria possess multiple paralogues of each component, often organized into independent systems." V. has 3 operons, only #2 req'd
*chemotaxis inhibits ability to colonize (infant mice), wt outcompeted by che- '''(how do you measure competition?)'''
*''V. fischeri'' requires motility to colonize host squid Euprymna scolopes, chemotaxis may be needed for symbiosis
*''V. parahaemolyticus'' flagellum serves as mechanosensor of increased viscosity, ''V. cholerae'' might sense in biofilm formation
*Motility has to be switched back on prior to exiting the host, transient phenotype of motile, non-chemotactic shed bacteria
*'''Do they modulate chemotaxis to optimize particular stages of infection?'''

===Light stimulates growth of proteorhodopsin-containing marine Flavobacteria===
[http://www.nature.com/nature/journal/v445/n7124/full/nature05381.html]

Alphaproteobacteria and Gammaproteobacteria, together with members of the Bacteroidetes phylum, are the most abundant groups of heterotrophic bacteria in the sea.

From supplemental info:
Annotation and phylogenetic analysis. Open reading frames were predicted and 
autoannotated using GenDB (Meyer et al. 2003). In addition, all relevant genes discussed here 
were manually annotated. For the phylogenetic analysis of PR, amino acid sequences were 
aligned using Clustal W, and a tree was constructed based on a Kimura’s distance matrix and 
the Neighbour-Joining method using the PHYLIP package (Version 3.2) (Felsenstein 1989). 

For bacterial counts, samples were fixed with 0.2 μm-pore-size filtered formaldehyde (4%, 
final concentration), stained with SYBR Gold (1:100 dilution, Molecular Probes), filtered 
onto black 0.2 μm-pore-size polycarbonate filters (Poretics, Osmonics Inc.) and counted by 
epifluorescence microscopy within 48 hours. 

RNA extraction and purification. Cells were harvested from seawater cultures by pipetting 
10 ml samples into 15-ml tubes on ice. Samples were centrifuged and the pellets were stored 
with 0.5 ml RNAlater® (Ambion Inc., Austin, TX) at –80 oC. For RNA extraction, samples 
were thawed on ice. The RNAlater was discarded and the pellet was washed in PBS 1X. A 
total of 500 μl of lysis/binding solution provided by the RNAqueous®-4PCR kit (Ambion, 
Inc.) were added to the cells. The samples were transferred to 2 ml screw-cap microcentrifuge 
tubes containing 1.2 g of 100-μm-diameter zirconia-silica beads (BioSpec Products, Inc.). 
Samples were mechanically disrupted in a Mini-beadbeater-8TM cell disrupter (BioSpec 
Products Inc., Bartlesville, OK). After disruption, samples were incubated on ice for 5 min 
and the beads were allowed to settle out of the lysis mixture. The lysate was clarified by 
centrifugation and the aqueous phase was transferred to a new tube. An equal volume of 64% 
ethanol was added to the lysate and samples were purified according to the RNAqueous®- 
4PCR Kit. The isolated total RNA was treated with DNase I - RNase-free (Ambion Inc.). 
RNA preparations were checked for DNA contamination with PCR using primers 358F and 
907RM for the 16S rRNA gene. '''Total RNA was quantified by spectrophotometry at 260 nm.'''

==Genomes==
===honeybee genome===

Honey Bee Genome Sequencing Consortium. Insights into social insects from the genome of the honey bee Apis mellifera. Nature 443, 931–949 (2006)

AT-rich, ~240 Mb genome

has improved the annotation of ESTs, which the authors have already used to identify candidate genes that are involved in caste differentiation — their data show that changes in metabolism are important during this process

haplodiploid, polyandrous creature

==Evolution==
===Tempo and Mode in Evolution: Genetics and Paleontology 50 Years After Simpson (1995)===
National Academy of Sciences

In Tempo and Mode, Simpson coined terms for three decidedly different rate distributions in evolution, inferred from morphological comparisons of Phanerozoic and living taxa: tachytelic, for "fast"-evolving lineages; horotelic, the standard rate distribution, typical of most Phanerozoic animals; and bradytelic, for "slow" morphological evolution (Simpson, 1944). Included among the bradytelic lineages are so-called living fossils (such as linguloid brachiopods, horseshoe crabs, coelacanth fish, crocodilians, opossums), "groups that survive today and show relatively little change since the very remote time when they first appeared in the fossil record" (Simpson, 1944, p. 125). Simpson's bradytely closely approximates Ruedemann's earlier developed concept of "arrested evolution'' (Ruedemann, 1918, 1922a, 1922b), both based on comparison of modern taxa with fossil forms that are virtually indistinguishable in morphology but are 100 Ma or more older.

Hypobradytely. Recently, a fourth term—hypobradytely—has been added to this list of rate distributions (Schopf, 1987) "to refer to the exceptionally low rate of evolutionary change exhibited by cyanobacterial taxa, morphospecies that show little or no evident _morphological_ change over many hundreds of millions of years and commonly over more than one or even two thousand million years" (Schopf, 1992e, p. 596).

This strategy of using uniquely paleontological data about tempo to infer mode, and thus to develop theory directly from the domain of macroevolution, pervades Simpson's book and underlies all his examples. To cite just two cases:  

# Designation of the three modes. Simpson's last, and best-known, chapter (1944, pp. 197–217) uses data of tempo to propose a fundamental division of evolutionary processes into three modes, each with different meaning: speciation for a low-level process of iterating diversity, with no significant input to trends or other larger-scale patterns; phyletic evolution for the ordinary style of directional change, leading to evolutionary trends and accounting for some 90% of paleontological data; and quantum evolution for rapid and rare, but efficacious, "all-or-nothing" transitions from one adaptive zone to another through an inadaptive phase (a process analogized with Wright's model of genetic drift).
# The theory of horotely, tachytely, and bradytely. This fascinating and brilliant, if ultimately flawed, theory has been widely misunderstood by people who do not grasp Simpson's central strategy of using tempo to infer mode. Many critics have stated that Simpson only invented some arcane, Greek-based jargon to divide the ordinary continuum of evolutionary rates into slow (brady), ordinary (horo), and fast (tachy). Not at all. Simpson was trying to identify separate peaks (modes in the statistical sense) in the distribution of tempos in order to specify distinct modes (in the ordinary sense) of evolution. Thus, horotely is not the central tendency of a single distribution of rates (with tachytely as the right tail, and bradytely as the left tail, as in the conventional misinterpretation); horotely is the entire distribution of ordinary rates, while tachytely and bradytely are, in Simpson's hypothesis, smaller distributions with distinct central tendencies at much larger and much smaller values than the central tendency of the horotelic distribution.