==Specific Protocols==
*[[MIT:Sequencing BioBrick DNA]] -- protocol for DNA sequencing of BioBrick parts
*[[Knight:Sequencing DNA]] 
*[[CRI DNA sequencing|CRI DNA sequencing]]

==Optimal Primer Design for Sequencing in E. coli==

Agencourt has designed new primers for their CopyControl vector series which use 3' ends which are low frequency in the E. coli genome.  The two 7-mers are GTCTAGG and CTAGGAA which occur 3 and 7 times in the genome, respectively, and have near 50% GC content.  The 7-mer GCCTAGG does not occur in the genome, but was rejected due to self-complementarity.   They chose an 8-mer which was unique, and the two primers they finally chose for their sequencing vector were GTA CAA CGA CAC CTA GAC and CAG GAA ACA GCC TAG GAA (Note that their final choice is inconsistent with their reported plan).   The low frequence of the CTAG 4-mer appears to be the key insight, and this might be useful in designing novel primers and primer binding sites.  See Epicentre Forum Vol 13(1) p 17,  "An improved CopyControl fosmid vector maximizes end-sequencing results.
 
==Sequencing Mammalian Genomic DNA==
This approach goes for sequencing exons or DNA fragments from most any organism.  

*Template source: Genomic DNA can be isolated from tissues using a variety of protocols.  I prefer the Qiagen gDNA Tissue kit but man work.  10 - 50 nanograms of gDNA is recommended for consistent amplification using a nested PCR protocol.  PCR from less DNA template is possible (even a single cell in theory) but is inconsistent.  
*Primer design: For a nested PCR approach, you'll need 4 or 6 total primers.  Nested PCR primers designed around a single exon (or ~400 bp fragment) are most likely to yield the desired product.  Outside primers should be designed according to regular guidelines.  For many human and mouse genes, tested primer sets may be published.  The primers should be non-overlapping.  For high throughput protocols, many groups will engineer M13F & M13R sequencing into the internal sequencing primers.  This allows the sequencing lab to then use the same tested primers to sequence every sample.   
*Basic protocol
#Extract DNA.
#Complete external primer PCR reaction.
#Use 1ul of the first reaction as a template for second reaction with internal primers.
#Clean up the PCR reaction with your favorite PCR cleanup kit (Qiagen, Wizard) or use ExoSap-it enzyme to degrade oligos (less handling in high throughput).  
#Elute in WATER and submit samples to sequencing according to the eColi protocol.

*Controls and other comments: 
#Always use a no-template control, especially in multi-sample experiments.
#A mutation is confirmed after detection in two ''independent'' reactions.

==Notes==

At the MIT biopolymers facility and probably with most sequencing centers, when doing a run-off sequencing reaction, an extra (sometimes pretty high amplitude) 'A' peak is seen at the end before the template ends. The workers at the biopolymers facility seemed surprised when they were told about this, but they were able to find out that the sequenase enzyme used in the sequencing reaction is a genetically modified form of Taq. Therefore, it is most probable that the extra A is the template-independent A that Taq tends to add to the 3'-end of DNA.

'''Q: The MIT sequencing center recommends an amount of template DNA for a sequencing reaction.  If I have a mixed sample of DNA (for example, two plasmids purified from a culture) and want some of it sequenced (for example, one of the plasmids), should I submit the recommended amount of total DNA or the recommended amount of the DNA of interest?'''<br>

A: It's best to submit a pure sample...but if you cannot, I would make sure that you submit the required amount of the template that the primer that you are using will anneal to.

Any other DNA will not get labeled with the fluorescent dyes that we use to call a sequence....and will hopefully be inert...it should be inert unless your primer also anneals to the contaminant DNA.

The dynamic range of the DNAsequencer is between 10 to 1000 nanograms...so if you are in that ballpark it should work.

[[Category:Protocol]]
[[Category:DNA]]
[[Category:In vitro]]