==Overview==
This allows you to screen a large number of inserts the day after completing a transformation, rather than waiting for overnight cultures and performing many unnecessary minipreps. A small amount of DNA is isolated directly from colonies on your plate. PCR amplification using primers
complementary to your plasmid lets you determine the presence and approximate size of inserts. The protocol works best for ligation reactions with known insert size(s).

==Materials==
*LB (or other suitable media)
*Selectable marker of choice
*Sterile deionized water

*[[PCR]] Reagents
**dNTPs
**Thermostable polymerase
**Buffer
***50 mM MgCl<sub>2</sub>
***500 mM KCl
***200 mM Tris-Hcl (ph 8.4)
**Primers
**Red Juice
***Cresol Red
***Sucrose
***MilliQ Water
***''<small>Note: Red Juice is a loading dye for agarose gels. Our lab adds it to PCRs as it saves a step between performing the reaction and running a sample on a gel. There is no negative impact to including this particular loading dye, and anecdotal evidence that it actually improves PCR results. Other loading dyes may have undesirable effects or completely impede the reaction.</small>''

*Agarose
*Ethidium Bromide
*DNA Marker

==Procedure==
#After performing your ligation reactions and transformations, plate the transformed cells on agar plates with an appropriate selectable marker. We typically use plasmids with ampicillin resistance.
#Grow plates overnight at 37&deg;C
#The next day prepare a 96-well plate by adding 20 &mu;L of MilliQ water to as many wells as you have colonies for.
#Pick a colony with a micropipet tip and resuspend thoroughly in the water by pipetting. Cells will lyse over the next several minutes. Begin setting up your PCR immediately. Alternatively, bacteria could be resuspended in LB without risk of lysis.  Include a negative control.  
#Prepare a master mix for PCR.
#Use 10&mu;L of your lysed cells for template.
#As soon as you have taken your template DNA, add 50&mu;L of LB + ampicillin (or whatever selection you are using) to the remaining cells. Cover the plate with parafilm and place at 4&deg;C.
#Place samples in a thermal cycler and set program
#*When using M13, T7, or Sp6 primers we use an annealling temperature of 55&deg;C.
#*Adjust extension time depending on your insert size. Be sure to keep in mind the amplified portion of your vector when determining size.
#Pour a 1% agarose gel with ethidium bromide.
#Load PCR samples and an appropriate ladder (ie 2 log, 1kb, 100 bp)
#Electrophorese gel 
#Visualize your gel and determine which clones have the desired insert.
#In a 14 ml culture tube, innoculate 2ml LB + ampicillin with 5&mu;L of the cells from your 96-well plate.
#Grow overnight at 37&deg;C and continue to [[miniprep]].

==Notes==
*This protocol is meant to be a time and resource saving method, but is not appropriate for every ligation and 
transformation. Experiments that yield a large range of fragment sizes or very large fragments, such as BAC or genomic 
digests (shotgun type cloning), may exceed the limitations of PCR or your particular ''taq''. Very large inserts may necessitate such lengthy extension times that the PCR is no longer a time saving tool. Still, with a pool of fragments we often find this technique useful as it lets us select one or two clones of each size, or choose the clone with our expected length product.

*Because you are priming your PCR off the plasmid, there will be a small fragment in every lane of your gel, even if a clone had no insert. This also means that when looking on your gel you want to factor in that additional DNA when approximating fragment length. If a lane is completely empty there may have been a problem with your PCR. A template negative PCR reaction is always a recommended control.

*Btarlow: note
I do this with a couple modifications.  
#I resuspend my colony directly in 10ul LB broth.  This way, I don't worry about lysis.  Sterile LB won't affect the PCR, but it's always good to include a negative control.  
#Then I use 1uL of the culture as a template for 50uL PCR reaction.  
#When avaiable, I like to use primers to the vector that span the multiple cloning site (MCS) so that I always get PCR product for every template.  Many plasmids already have sequences designed for sequencing primers built in (ie T7, T4, M13F, M13R) and these work well.  In a recent experiment, I ligated a 1800bp fragment into the backbone.  If I had empty vector, I saw a 250bp band.  If the plasmid contained the insert, I got a 2050bp band.

==Contact==
*[[User:Seth]]

[[Category:Protocol]] [[Category:In vitro]] [[Category:Escherichia coli]] [[Category:DNA]]