This is the text-only version of Stephanieâ€™s Harvard iGEM 2007 wiki. For the full version including images, please refer to [[shlo/notebook| my full notebook]]



<b> Helpful Results/Links</b>

[[shlo/FACSresultsiGEM07| FACS results]]<br>
[[Shlo/notebook/growthcurves| Growth Curves]] <br>
[[Shlo/notebook/platereaderexcelsheets| Plate Reader Data]]<br>
[[IGEM:Harvard/2007/Laboratory_Notebooks/Quorum_Sensing/Plate_Reader_Protocol|  Plate Reader Protocol]]<br>
[[shlo/fluormicroiGEM07| Fluorescent Microscope Results]]<br>
[[shlo/flmicroprotstickiness | Fluorescence Microscopy Protocol for Stickiness]]<br>
[[shlo/directmagneticprotocol| Direct Magnetic Bead Protocol]]<br>

<br>[http://openwetware.org/images/5/5d/SLo0702presentation.ppt Stephanie's Week 3 Powerpoint]
<br>[http://openwetware.org/images/f/f6/Sloweek5igem.ppt Stephanie's Week 5 Powerpoint]
<br>[http://openwetware.org/images/3/3a/Shloweek6igem2007.ppt Stephanie's Week 6 Powerpoint]
<br>[http://openwetware.org/images/6/63/080607slo.ppt Stephanie's Week 8 Powerpoint]
<br>[http://openwetware.org/images/e/ed/Shlo0813igem.ppt Stephanie's Week 9 Powerpoint]
<br>[[IGEM:Harvard/2007| Harvard iGEM]]
<br>
[[shlo/aboutme| About Me]]

=Week 1=

<br>
Doing some reading on Fec now. Really brief notes and some of the more interesting readings follow.<br>
Fec genes encode proteins essential for ferric citrate transport in e.coliK12. <br>
FecA is an outer membrane protein<br>
=N-proximal 79-residue extension: deletion of this extension abolishes induction by ferric citrate but retains feric citrate transport: Kim et al, Transcription induction of the ferric citrate transport genes via the N-terminus of the FecA outer membrane protein, the Ton system, and the electrochemical potential of the cytoplasmic membrane [http://www.blackwell-synergy.com.ezp1.harvard.edu/doi/pdf/10.1046/j.1365-2958.1997.2401593.x?cookieSet=1 Kim article] 
<br>
[http://www.ncbi.nlm.nih.gov.ezp1.harvard.edu/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=16718597&ordinalpos=10&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum Gene regulation by transmembrane signaling] Some really nice info on structure of Fec and interactions with ferric citrate
<br>
[http://biocyc.org/ECOLI/NEW-IMAGE?type=ENZYME&object=EG10292-MONOMER Biocyc]<br>
<biblio>
#reading1 pmid=11137298<br>
#reading2 pmid=12000971
</biblio>

<br>
NanoDrop results (6/20) performed by Ellenor and Stephanie:<br>
(1.5 uL used out of a 30uL elution with nuclease free water)<br>

S: 10.9 ng/uL<br>
S2: 15.4 ng/uL<br>
B: 59.1 ng/uL<br>
B1: 25.1 ng/uL<br>

== Brainstorming ==
Brainstorming for the two-component systems (really for my own use for now - not expected to be coherent)<br>
[http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=10653699&ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVAbstractPlus Structural comparison of the PhoB and OmpR DNA binding/transactivation domains and the arrangement of PhoB molecules on the phosphate box]<br>
-NMR used to determine 3DE structure of PhoB DNA-binding/transactivation domain. Very similar to OmpR DNA-binding/transactivation domain, except for conformation of the long turn region of PhoB (interaction site for sigma subunit, rather than interaction with alpha subunit for OmpR)
<br><br>
[http://jb.asm.org.ezp1.harvard.edu/cgi/content/full/185/1/317?view=long&pmid=12486069 Interdomain linkers of homologous response regulators determine their mechanism of action]<br>
Focuses on OmpR and PhoB and, as the title suggests, supports that phosphorylation of sites (particularly N-terminus of both proteins) improves affinity to bind DNA. Isolated C terminus of OmpR is insufficient to productively interact with RNA polymerase. <br>
I've been told by some of the lab members that OmpR is an inner-membrane protein and therefore cannot be used for our assays. It seems that we'll have to find another protein ...<br>
<br>
[http://www.ncbi.nlm.nih.gov.ezp1.harvard.edu/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=11325944&ordinalpos=6&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum The phosphoryl transfer domain of UhpB interacts with the response regulator UhpA]
<br> UhpB = histidine kinase protein that controls production of sugar phosphate transporter UhpT<br>
UhpA = response regulator; when phosphate is transferred from histidine to aspartate, ability of kinase to bind to target DNA sequences and to alter gene transcription is altered.
<br> Major result: indication that phosphoryl, transfer-dimerization of UhpB participates in specific binding of UhpA, in control of autokinase activity, and dephosphorylation of P-UhpA<br>
... So I found this paper on a search on PubMed for e coli outer membrane protein signaling dimerization. However, I've found that UhpA resides in the cytoplasm, whereas UhpB is an inner-membrane protein. Boo.<br>
More thoughts: we could potentially try to target some of these inner-membrane proteins to the outer-membrane. I don't know if this is really feasible - while we can attach the appropriate signal sequence, I'm not sure the environment would allow for correct conformation and activity of said proteins.


= Week 2 =
== 6/27 ==

Yesterday night I left some vectors & plasmids to ligate in the 37C incubator. Kevin took the vectors for nucleotide removal, and I speed Vac'd the plasmids. Originally I was going to make a gel and run the plasmids on multiple lanes for gel extraction, but given the large Nanodrop values (242 - 371 ng/uL) and the large amount of Midiprep used (50uL), Bill suggested that I PCR purify them instead, which gives 95% recovery rather than 80% recovery. A gel will still need to be run to confirm that the samples were indeed ligated.<br>
The plasmids are Lpp+OmpA1+pet29 and Lpp+OmpA2+pet29.<br>
*Reconstitution into 30uL of H2O per plasmid (two plasmids total - OmpA1, OmpA2), 
*split into two samples each (so 15uL total volume)
*PCR purified (PCR purification kit from Qiagen)
*Eluted with 50uL nuclease-free H2O per tube 
*prepared a gel using TBE
*prepared 5 lanes<br>
#Lane 1: 10uL ladder (1kB +)
#Lane 2: 1uL OmpA1 plasmid (diluted to 20uL total using H2O)
#Lane 3: 3uL OmpA1 plasmid (diluted to 20)
#Lane 4: 1uL OmpA2 plasmid (diluted to 20)
#Lane 5: 3uL OmpA2 plasmid (diluted to 20)
<br> The gel was run at 130V for 45 minutes.<br>
4 tubes total should remain: 2 tubes of each type of plasmid. 2 tubes should have 49uL left (labeled "1"), 2 tubes should have 47uL left (labeled "2"). After gel ran for 45 minutes, bands appeared about 3/4 way down the gel. Ethidium bromide (2uL) and 100mL TBE buffer used, put on shaker for 45 minutes. Visualized under Trans-UV.
<br>
<br>

<br>
<br>
Nanodrop, 6/27:<br>
10mer: 122.3 ng/uL<br>
15mer: 156.3 ng/uL<br>
20mer: 167.4 ng/uL<br>
plasmidOmpA1: 39.6 ng/uL <br>
plasmidOmpA2: 42.8 ng/uL <br>
<br>
We Speed Vac'd the plasmids and resuspended both in 33uL of nuclease-free H2O. To calculate volumes of sample needed , we approximated that the vector was 5800 in size and the insert was 50. Therefore, we'd need 1200ng of vector and 103ng of insert. <br>
<b>Protocol for Ligation of Plasmid given Nanodrop results and volumes available</b><br>
OmpA1 or A2: 10uL<br>
10mer, 15mer, or 20mer: 1uL<br>
buffer: 1.5uL<br>
ligase: 1uL<br>
dH2O: 1.5uL<br>
(total of 15uL)<br>
Leave in PCR machine (left in the one at George's workstation) overnight for 15C. The extra 10,15,and20mer tubes were put in the "Mike Strong iGEM plasmid" box in the freezer in the small laboratory. We used shorthand for the small PCR tubes: 110 means OmpA1+10mer, 115 means OmpA1+15mer, 210 means OmpA2+10mer, etc. 

== 6/28 ==
George, Perry, and I transformed some of the BioBricks into 20uL of Top Ten cells.<br>
We want to change the pL promoter because it does not allow for constitutive expression. Some of the Biobrick parts we're using may facilitate this.<br>
Also: grew cells in liquid media, digested BB, looked at sequences and discovered that the BB isn't what it should be - should be about 800bp, with ribosome binding region and terminator region, but was only about 300bp, with terminator and something following it.<br><br>

== 6/29 ==
... oh lord. i have so much more to add. sorry for the delay guys ... mad busy day.<br><br>
George, Perry, and I arrived at 8:15am to start to prepare the bacteria for the FACS appointment at 3:30. We took the optical densities of the cells we grew in liquid medium overnight and got 1.785, 1.793, 1.780. <br><br>
9am: We prepared 3 tubes to 2mL each, with dilutions of 1:40, 1:80, 1:100. 2uL of Amp was added in each, and the following:<br>
1:40 - 50uL cells, 1498uL LB<br>
1:80 - 25uL cells, 1748uL LB<br>
1:100- 20uL cells, 1792uL LB<br>
<br>9:15am: Made and set a 1% agarose gel for the digest we prepared yesterday.<br>
Also helped Perry with some of his colony-PCRs. Gel has 3 lanes- 1: ladder, 2: yesterday's digest, 3: Perry's sample. Will be visualized using SyberGold <br><br>
10:30am: OD of 4-hr samples. Realized we had not diluted them enough. Diluted 1:100 for the spectrometer (so we did not use up too many samples), had reading of 0.03 for 1:40, 0.018 for 1:80, 0.013 for 1:100. Decided to re-dilute 1:100 for samples originally marked 1:40 and 1:100, and 1:30 for sample originally 1:80. Gives us an absolute OD of 0.03 for the 1:40, 0.01 for the 1:100, and 0.05 for the 1:80. (made 2mL total for each sample, adding in cells, LB, and Amp)<br>1:40- 20uL cells, 2mL LB, 2uL Amp, 1:80 - 65uL cells, 2mL LB, 2uL Amp, 1:100 - 20uL cells, 2mL LB, 2uL Amp.<br>
Also: prepared 2hr samples<br>
<br><br>
11am: George prepares HSL solution using 200proof ethanol. 
<br><br>
11:30am: Prepare the 1-hr incubations<br><br>
12 noon: OD of 4-hour induction, right before protein added. 1:100 dilution done for spectrometer. (10uL cells, 990LB). The numbers below are the raw numbers from the machine - so our samples actually have OD 100x this.<br>
1:40 - 0.002<br>
1:80 - 0.002<br>
1:100 - 0.001<br>
Since we didn't have as much volume of cells as we had anticipated, instead of doing 2mL cell samples as planned, we used 1mL of cells with 10.5 uL HSL (diluted by George - see his personal notebook page). 2 samples were taken from the 1:40 first dilution (8:30am) - used for the 10 and 100nM HSL runs - and 1 from the 1:80, with 1nM HSL. <br>
<br>
1:30pm: Induced the two-hour samples. First, OD taken of 2-hr and 1-hr (1hr had been growing for about 1.5 hours at this point). <br>
Again, all ODs diluted 1:100 first. <br>
Two hours: 1:40 - 0.078, 1:80 - 0.035, 1:100 - 0.005. One hour: 1:40 - 0.016, 1:80 - 0.008, 1:100 - 0.007. Also, we had left the 4-hour cells on the bench (from before) - ODs of 0.002 (1:80) and 0.004 (1:100). 
<br>
<br>
2 samples from the 2-hour 1:100 (0.005 OD) were used to make the 10, 100nM HSL.<br>
Since the ODs were already so high for the 1-hr, we also decided to take our original overnight liquid cultures and dilute them for the 1-hr. We used the same formula as above and diluted to 1:80 and 1:100. <br><br>
2:30pm: 1-hr rediluted samples gave 0.003OD (1:80) and 0.002 (1:100). We used the 1:80 for the 1nM HSL and the 1:100 for the 10, 100nM HSL. <br>
A negative control was also created from 1mL of the 1:80 solution. 
<br><br>
3:15pm: All samples spun down for 2 minutes at 10,000rpm, supernatant removed, reconstituted with 0.5mL PBS (1x)<br>
<br>
Colony counts from 06/28: <br><br>
<u>275uL</u><br>
OmpA1 + 10mer: 167<br>
OmpA1 + 15mer: 266<br>
OmpA1 + 20mer: 224<br>
OmpA2 + 10mer: 60<br>
OmpA2 + 15mer: 9<br>
OmpA2 + 20mer: 28<br>
<u>10uL</u><br>
OmpA1 + 10mer: 15<br>
OmpA1 + 15mer: 8<br>
OmpA1 + 20mer: 0<br>
OmpA2 + 10mer: 1<br>
OmpA2 + 15mer: 0<br>
OmpA2 + 20mer: 0<br>

== 6/30 ==
Helped miniprep some Biobricks, but 3 of the labels smeared and I was unable to discern them. PCR'd them, Will run an E-gel.
<br>
E-gel: <br>
Lane 1: 1kb+ ladder<br>
Lane 2: Sample 1<br>
Lane 3: Sample 2<br>
Lane 4: Sample 3<br>
Rest of lanes loaded with water as instructions dictate.

== 7/01 ==
Did another miniprep of some Biobricks that Perry had grown in liquid culture. The tubes are in the freezer at my workstation. Perry helped me visualize the Egel from yesterday (the machine is somewhat broken ... I had left the gel in the fridge overnight. Though some of the gel diffused, the bands are still distinguishable). The samples were known to be either R0051, B0015, or T9002. Band 3 ran the slowest, indicating the highest molecular weight, and therefore T9002. B0015 is 129 bp whereas R0011 is 55bp, so sample 2 (which ran the furthest) was likely to be R0011, and sample 1 was B0015.  <br><br>

== Readings and Brainstorming ==
Idea: increasing ligation efficiency<br>
According to a paper I just read, (Lund et al) "temperature cycle ligations" may provide a 4-8 fold increase on cloning efficiency, since the cycling balances high enzyme activity and DNA annealing. The temperature cycle should run for 12-16 hours, cycling between 30 second bouts at 10C and 30C. <br>
<br><b>Readings toward QS and lux</b>
<br>
[http://departments.kings.edu/biology/lux/bacterial.html Bacterial Transformation Experiment] James Slock has a detailed protocol for transforming the lux genes into e.coli. Actually, the transformation protocol is the same as what we've been doing, but there is some specific info on lux stuff. Not really essential, I guess.<br>
[http://parts.mit.edu/registry/index.php/Featured_Parts:Cell-Cell-Signaling MIT Parts Registry: Overview of LuxR system]<br>
[ww.ai.mit.edu/projects/ cellular-robotics/rweiss-dna6.ps Paper] Really nice, but needs to be converted to pdf - my Mac did this automatically ...<br>
[http://parts.mit.edu/registry/index.php/Part:BBa_F2620:Response_time MIT Parts, BBa_F2620]<br>
[http://www.pnas.org/cgi/content/abstract/101/17/6355 spatiotemporal control reading] Look at the supplementary information as well.
<br>
[http://www.ncbi.nlm.nih.gov.ezp1.harvard.edu/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=16487323&ordinalpos=1&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum Paper] ...some other genes that may be turned on by HSL are mentioned ...
<br>
[http://www.ncbi.nlm.nih.gov.ezp1.harvard.edu/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=12658522&ordinalpos=6&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum somewhat relevant ... but not completely] Seems that the e. coli were grown with OHL... and a different strain was used
<br>
[http://www.biocompare.com/technicalarticle/216/High-efficiency-Derivatives-Of-The-BL21-Series-For-Protein-Expression-from-Stratagene.html BL21] More info on the cells we're using<br><br>
[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=539822 look in methods for  treatment of HSL powder] ... mammalian cells, but for method only
<br> [[shlo/notebook/luxfluorescenceduration]]
----

= Week 3 =
== 07/02 ==
Nanodropped some of the samples from yesterday and Saturday before Perry digested them and we sent them out for sequencing.

{| {{table}}
| align="center" style="background:#f0f0f0;"|'''Sample'''
| align="center" style="background:#f0f0f0;"|'''Nanodrop result (ng/uL)'''

|-
| S03623, Sample 1||104.5
|-
| S03623, Sample 2||111.2
|-
| S03608, Sample 1||126.7
|-
| S03608, Sample 2||110.3
|-
| J37034, Sample 1||196.5
|-
| J37034, Sample 2||97.6
|-
| J23039||65.8
|-
| R0011||61.4
|-
| R0140||86.1
|-
| T9002||185.1
|-
| B0015||71.6
|-
| R0051||100.2
|-
|}
<br>
We wanted to construct a growth curve for our cells, so we took the refrigerated BL21+I13263 cells (from Friday) and made 800uL of different dilutions for the plate reader. Ampicillin and LB were added to each sample, and samples were made for 1:10 (wells A1-3), 1:20 (B1-3), 1:30 (C1-3), 1:40 (D1-3), 1:50 (E1-3), 1:60 (F1-3), 1:70 (G1-3), 1:80 (H1-3), 1:90 (A4-6), 1:100 (B4-6), and of the BL21 cells (1:10, C4). 
<br>
Also, diluted the OHHL to 0.05M. There is 938uL total.<br><br>
Grew liquid cultures of I5311, OmpA1+his, I13522, T9002, and I13263. These will be used when we induce tomorrow and use the Plate Reader to detect fluorescence. 
<br>

See Perry's notebook for more info on this.


== 07/03 ==
[[shlo/notebook/growthcurves0703]] ... Growth curves for BL-21 with the lux insertions. Most of them look pretty good; should serve as a general reference for all groups, until (if?) you make your own growth curves.<br><br>

Helped Perry with some ligations/plating, and started a 5-hour fluorescent/induction experiment. First, I loaded the machine with cells (dilution - 1:10 ... maybe will dilute in greater amounts next time, eg 1:100?) and watched OD. Around 0.3OD I took them out, transferred to a clear/black 96-well plate (next time, will start with one of these), and induced cells with OHHL. <br>
<br>
Wells A1-3: I5311 (constitutive YFP, kan)<br>
Wells A4-6: I13522 (constitutive GFP, amp)<br>
Wells B1-3: BL-21 with PET29 OmpA1+His (negative control, kan)<br>
Wells B4-6: BL-21 with I13263 (negative control, inducible for YFP, amp)<br>
Wells C1-3: BL-21 with I13263 (10nM OHHL added)<br>
Wells C4-6: Top10 with T9002 (10nM OHHL added)<br>
Wells D1-3: BL-21 with I13263 (100nM OHHL added)<br>
Wells D4-6: Top10 with T9002 (100nM OHHL added)<br>
<br>
Also, digested some E0240 and plated S03608 and S03623, along with B0015. We'll use this to re-create the J37034 that hasn't seemed right (though our own constructs will have different promoters ... which is good in the long run). All of these samples (plates and microcentrifuge tube) are in the incubator, and someone will have to move the plates to the fridge and the tube to the freezer tomorrow.

== 07/04 ==
Took plates out of incubator, put into fridge<br>
F2620, top 10: lawns of colonies<br>
JL0159: lawns<br>
S03623-B0015: a few<br>
JL0159-PDZ1+2: minimal growth<br>
JL0159-PDZ2: some growth<br>
S03608-B0015 in Top 10... are these air bubbles or colonies?<Br>
JL0159-PDZ1: Minimal growth<br>
<br>
I put the E0240 in the freezer. Be careful; the non-digested microcentrifuge tube is there as well!! The digested one is clearly labeled "digested".
<br>
[[shlo/notebook/0704fluorescence]]

== 07/05 ==
 
Prepared a colony PCR using some of the plates that I moved yesterday.<br>The PCR looks really bad/has hardly any bands, which is really confusing. We might have to redo this all ...<br>
<br>

== 07/06 ==
Helped Perry prepare a colony PCR of F2620 and a few other samples<br>
Planning to do some transformations at the end of the day, including some of the parts we found: J06702, F2622, F2621, I15030 (we should have this somewhere), !6042, I13273, J37015 (?)<br><br> 
Found Imperial's page from last year, with testing of the T9002 part we're using. Interestingly, as they varied AHL concentration, fluorescence did not seem to change too significantly. [http://openwetware.org/wiki/IGEM:IMPERIAL/2006/Results/T9002 Imperial T9002 Results]

== Additional Readings and Thoughts ==
Trying to read to trigger brainstorming ideas for fluorescence assay as well as gauge what to expect. <br><br>Harris poses a good question: why do we have luxR/luxI constructs in different bacteria? Will probably be a lot less sensitive than if we had both in one ... though perhaps the applications of having a sender/receiver are more diverse? Time to discuss..<br><br>
Again, the Andersen et al paper (gfp-based N-acyl homoserine lactone sensor systems for detection ...) is extremely helpful in this regard: <br>
*Their construct (e. coli MT102) was much more sensitive to OHHL relative to other proteins in the acyl-homoserine lactone family. 
*1 nM OHHL concentrations did not cause fluorescence to be any higher than background. Significant fluorescence began at about 3nM OHHL. 100nM OHHL was only slightly higher in fluorescence compared to 10nM OHHL; as I've noted before, "maximal induction" was coined to be at 10nM OHHL. 
*GFP: excitation of 475nm and emission at 515nm
*Interestingly, the fluorescence peaked around 250 minutes, and then started to decrease (... this could do with their different GFP constructs, though).
<br>
<br>
Ripp et al: Linking bacteriophage infection to quorum sensing signalling and bioluminescent bioreporter monitoring for direct detection of bacterial agents<br>
*We've been talking about trying to measure amounts of OHHL produced, so we can detect whether our lux I construct is working. n this paper, OHHL concentrations were analytically determined using liquid chromatograph-mass spec. At least it's feasible, though I don't think our lab could/would do this.
*Induced E. coli OHHLux bioreporter using synthetic OHHL. 
**"significant" signals were considered to be those 2 standard deviations above background. 
**Saturation-type behavior was observed at OHHL concentrations exceeding 50uL ... which is much higher than anything we've used
**Found a linear-type response at OHHL concentrations from 20nM to 2uM
<br><br>
Burmolle M et al: Presence of N-acyl homoserine lactones in soil detected by a whole-cell biosensor and flow cytometry<br>
*To figure out dose-response, OHL and samples shook at 37C for 20 hours (including antibiotic and LB, 5mL total). Afterward, 1mL was washed once in PBS (for each sample) and resuspended in 3mL PBS for fluorescence readings
<br><br>

A bunch of papers also seem to reference TLC (thin layer chromo) as a possibility to 
detect/characterize OHHL levels.

= Week 4 =
==7/9==
<br>
Perry, George, and I did a number of colony PCRs on various samples. Results below. The ones that worked are marked.<br>
<br>
*Lane 1: Ladder
*2: J37015, 1
*3: J37015, 2
*4: J06702, 1 (as expected)
*5: J06702, 2
*6: I6042, 1 (as expected)
*7: I6042, 2 (as expected)
*8: I13273, 1
*9: I13273, 2 (as expected)
*10: F2622, 1 (as expected)
*11: F2622, 2
*12: H2O<br><br>
<br>
*Lane 1: Ladder
*2: F26-I07
*3: S08-E40
*4: S23-I07 (as expected)
*5: S23-E0240 (as expected)
*6: F26-E40
*7: S08-I07 (as expected)
*8: F2621, 1 (as expected)
*9: F2621, 2 (as expected)
*10: I15030, 1
*11: I15030, 2
*12: H2O
<br>
Made liquid cultures of a number of samples for plate reader tomorrow.<br>
Also, left one portion of T9002 incubating with about 1000 nM OHHL to serve as comparison/control of fluorescence, to J37015.
==7/10==
===Plate Reader===
Prepared plate reader for fluorescence with OHHL. <br>
A1-3: T9002 (-), 10nM OHHL<br>
A4-6: T9002 (+), 10nM<br>
B1-3: T9002 (-), 100nM<br>
B4-6: T9002 (+), 100nM<br>
C1-3: I13273, 10nM<br>
C4-6: I13263, 10nM<br>
D1-3: I13273, 100nM<br>
D4-6: I13263, 100nM<br>
E1: B0015, negative control<br>
E2: T9002(-), no induction<br>
E3: T9002(+), no induction<br>
E4: I13273, no induction<br>
E5: I13263, no induction<br>
E6: I13522, no induction (constitutive GFP => positive control)<br>
Note: T9002 (+) indicates that the T9002 sample was incubated with 1000nM OHHL.<br>
The samples were all put in 1:20 (sample:total volume), total volume = 200uL, inclubes LB + Amp. <br>
Induction expected to occur around 1hr20min after dilution, at 0.3OD, around 12:15.<br>
===Readings: SdiA and AHL===
Interestingly, on our plate reader today, T9002(+) samples grew consistently faster than the T9002 (-) and other samples. The I63 and I73 samples grew at the same rate despite being in different types of cells (Top 10 vs BL21). <br>
Some papers seem to suggest that SdiA acts similarly to a LuxR and is involved in cell division processes (though papers also mention that exact mechanisms remain unknown). One paper has found that addition of AHL to e coli should not affect their overall growth, though a number of genes are upregulated and downregulated, including OmpC (upregulated). [[http://www.blackwell-synergy.com/doi/abs/10.1111/j.1574-6968.2006.00103.x]]
<br><br>
Additional Readings:<br>
#[[http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=8552633 Control of cell division in Escherichia coli: regulation of transcription of ftsQA involves both rpoS and SdiA-mediated autoinduction.]]
#[[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=1169494#r1 Effect of sdiA on Biosensors of N-Acylhomoserine Lactones]]

===Colony PCRs===
Prepared 3 different colony PCRs & E-gels. <br>
Extension for 1min15sec<br><br>
<br>
Lanes 2-6: F2620<br>
Lanes 7-12: B0015
<br>Extension for 2min15sec<br>
<br>

#Ladder
#I15030 1
#I15030 2
#F26-E 1
#F26-E 2
#F26-E 3
#F26-I13507 1
#F26-I13507 2
#F26-I13507 3
#S08-E0240 1
#S08-E0240 2
#S08-E0240 3<br>
<br>Extension for 3min15sec<br><br>
#Ladder
#J37015 1
#J37015 2
#J37015 3
#J37015 4
#J37015 5
#J39-T9002 1
#J-T 2
#J-T 3
#J-T 4
#J-T 5
#J-T 6

==7/11==
Note: We had set up some fluorescent readings on the plate reader to run for a few hours yesterday, but somehow the PC restarted and failed to save/restore our information. Other than a few observations that we had made during the process - such as T9002 (+) growing much faster than T9002 (-), all our data was lost ... very, very unfortunate and aggrevating. 
===Plans===
*Figure out J-T: start with low concentration, see if fluorescence spikes (which might indicate quorum-like activity) rather than slow growth of fluorescence. 
*Re-Colony PCR some of the constructs 
**I5311 (... actually, we've never PCR'd this)
**F2620-I13507
**S08-E40
**F2620-E40
**I15030
*New growth curves?
===Setup for FACS===
ODs and fluorescence taken from the plate reader to figure out dilution and efficacy of some of the overnight dilutions/combinations. <br>
<br><br>
Induction, 100nM final concentration OHHL

{| {{table}}
| align="center" style="background:#f0f0f0;"|'''Sample'''
| align="center" style="background:#f0f0f0;"|'''OD'''
| align="center" style="background:#f0f0f0;"|'''Fluorescence (excitation485, em38)'''
| align="center" style="background:#f0f0f0;"|'''OD after one hour of growth, at induction'''
|-
| T9002 (+)||1.486||675.85||not used
|-
| T9002 (-)||1.212||212.61||0.379
|-
| I13273||1.283||208.69||0.430
|-
| F2620-E0240||1.487||191.98||0.527
|-
| J23039+T9002||1.174||4698.8||0.317
|-
| B0015||1.470||174||0.496
|-
| I13522||1.571||7978.7||0.397
|-
| I5311||1.643||2258||0.482
|-
| S08-I07 w/T9002||1.313||2538||0.427
|-
| S08-I07 w/ I73||1.248||1701||0.395
|-
|}

<br>
Interestingly, the T9002 (+) had relatively little fluorescence compared to the constitutive samples. Perhaps the fluorescence  had started to decrease over time, or perhaps the T9002 was not activated that strongly ... this is something we can measure over time with the plate reader ... if it does not turn off again and lose our data.
<br><br>
8mL LB + Amp (except for I5311, with Kan) added to enough uL of cells to make a "cell mass" of 371.5.<br>

===Results from FACS===
All inductions were at 2 hours with 100nM OHHL.
<br> We've also learned that we should do about a 1:50 dilution of the overnight cultures. 0.03 OD is probably optimal. 
*T9002 (non-induced) had some leaky fluorescence.
*T9002 (induced) showed strong fluorescence as expected.
*I73 (non-induced) had a little less leaky fluorescence than T9002, but still had some leakiness.
*I73 (induced) showed strong fluorescence as expected. S08-I07 + T9002 had little fluorescence in the GFP but had lots of fluorescence for RFP. 
*S08-I07 + I73 had strong fluorescence for both YFP and RFP. It should be noted, though, that the count of YFP was around 500 at peak, while some of the induced were around 1250 at peak. Perhaps this has to do with inconsistent cell numbers?
*S08 I07-T9002 had very little GFP (low count, both negative and positive) but strong RFP. 
*J39-T9002 showed strong GFP fluorescence.
*15311 showed strong YFP fluorescence as expected, with peak counts around 1600. 
*I13522 showed strong GFP, as expected, with peak around 1750.
*B0015 served as the negative control
*F2620/E40 (induced) showed no fluorescence, nor did the F2620/E40 (non-induced). Probably time to let go of the F2620.

===Setup for Overnight Plate Reader===
A1-3: T9002 (noninduced)<br>
A4-6: T9002 (+100nM OHHL)<br>
B1-3: I13273 (noninduced)<br>
B4-6: I13273 (+100nM OHHL)<br>
C1-3: F2620-E0240 (noninduced)<br>
C4-6: F2620-E0240 (+100nM OHHL)<br>
D1-3: J39-T9002<br>
D4-6: B0015 <br>
E1-3: I13522<br>
E4-6: I5211<br>
F1-3: S08-I07 w/T9002<br>
F4-6: S08-I07 1/ I73<br>
G1-6: Swabs of J-T (noninduced)<br>
1:100 initial dilution<br>
[[shlo/notebook/flu0711]]
<br>The decrease in fluorescence that is seen on many of the samples is probably due to the wells drying out over time.



==7/12==
I miniprepped some liquid cultures that were grown last night:
*R0040
*R0011
*P0140(-1)
*P0340(-1)
*R0051<br>
<br>
George and I also Colony PCR'd the biobricks that I listed in my plans for yesterday.

<br>
#Ladder
#I15030 1 (NO)
#I15030 2 (NO)
#I5311 1 (a little too big)
#I5311 2 (a little too big)
#I5311 3 (no band)
#S08-E40 1 (as expected)
#S08-E40 2 (as expected)
#F20-I07 1 (as expected)
#F20-I07 2 (as expected)
#F20-E40 1 (as expected)
#F20-E40 2 (as expected)
<br>
We will grow overnight cultures of:
*JT
*S08-E
*S23-E
*S08-I07
*S23-I07
*F26-E<br>
Looks like the plate reader can do more than one wavelength, so having the RFP constructs is going to be really awesome.

===Overnight Plate Reader===
Hopefully, this will allow us to compare the shape of the J+T (full construct) curve versus the T9002+OHHL curve. We hope the T9002+OHHL will prove to be a somewhat linear growth in fluorescence while J+T proves to be sigmoidal (showing quorum activity). <br>
Initial ODs were all about 0.05 or 0.06.
<br>
<br>
All of the following were swabs (touch of a pipet tip to the liquid culture, such that the initial number of cells is very low). Unfortunately, J-T was not available in liquid culture, so a direct touch from the plate had to be done.<br>
A1-6: J-T<br>
B1-3: I13522<br>
B4-6: I13263<br>
C1-3: B0015<br>
C4-6: I13263 w/ 10nM OHHL<br>
D1-3: I5311<br>
D4-6: I13263 w/ 100nM OHHL<br>
E1-3: I13273 <br>
E4-6: T9002<br>
F1-3: I13273 w/ 10nM OHHL<br>
F4-6: T9002 w/ 10nM OHHL<br>
G1-3: I13273 w/ 100nM OHHL<br>
G4-6: T9002 w/ 100nM OHHL<br>
H1-3: J-T 1/ 10nM OHHL
<br>
<b>Some Results:</b> [[ShLo/notebook/0712overnightplate]]
==7/13==
Created a 96-well plate for the plate reader (overnight, will pick up tomorrow). read is on fluorescence for both RFP and GFP. I was more careful to do dilutions of cells rather than swabs ... so hopefully growth and number of cells will be more consistent. Final dilution for most of the samples ended up being .5uM of the original sample, though I varied this in some of the wells. Some things that are being tested:<br>
*Fluorescence of J-T with OHHL versus fluorescence of just JT versus fluorescence of T02 with OHHL. 
*Different relative concentrations of lux I (S08-I07 and S23-I07) constructs versus lux R (FE ... this part hasn't worked in the past so I can only hope). I probably should have used T02 but I didn't grow this overnight, and had to use a liquid culture that was about a week old (from refrigerator)...<br>
Unless otherwise specified, the dilutions below are 1:100, add 1uL to final 200uL well for a 5x10^-5 final dilution. All are in LB+Amp.<br>

{| {{table}}
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''1to3'''
| align="center" style="background:#f0f0f0;"|'''4to6'''
| align="center" style="background:#f0f0f0;"|'''7to9'''
| align="center" style="background:#f0f0f0;"|'''10to12'''
|-
| A||J-T||J-T w/OHHL||FE w/OHHL||FE alone
|-
| B||S08-I||S08-I w/FE||S08-I w/T02||T02 alone
|-
| C||S23-I||S23-I w/FE||S23-I w/T02||S23 alone
|-
| D||T02 w/OHHL||I13522, 0.5 uL of original||I13522 w/ OHHL||I13522
|-
| E||2xS08-I w/ FE||4xS08-I w/ FE||S08-I w/ 2xFE||S08-I w/ 4xFE
|-
| F||2x S23-I w/FE||4x S23-I w/FE||S23-I w/ 2x FE||S23-I w/ 4xFE
|-
| G||10xS08-I w/FE||100xS08-I w/FE||S23-I w/ 10x FE||S08-I w/ 100x FE
|-
| H||T02 w/ 10nM OHHL ||FE w/ 10nM OHHL||JT w/ 10nM OHHL||I13522, 1uL from original
|}

==7/14==
Picked up plate from yesterday. RFP didn't seem to be detected at all, but I double checked my em/ex wavelengths online and they look right. Maybe I'll change them up a little and see if it helps.<br>
Set up an OD plate to accompany results. Will probably come in tomorrow or late tonight to start new plate reader, etc.<br>
===Plate Reader===
{| {{table}}
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''1to3'''
| align="center" style="background:#f0f0f0;"|'''4to6'''
|-
| A||J-T||J-T w/100nM OHHL
|-
| B||S08-I||13522
|-
| C||S23-I||T02
|-
| D||J-T||JT w/ 100 nM OHHL
|-
| E||S08-I07||13522
|-
| F||S23-I07||T02
|-
| G||FE 1:100||FE 1uL
|-
| 
|}
A-C rows: Original liquid overnight, 1:100 dilution, add 1uL of dilution to 200uL well.<br>
D-F rows: 1uL of original overnight added to 200uL well for final 1:200 dilution.<br>

Grew up some liquid cultures of BL-21 to start testing those in similar ways:<br>
*I13522
*J-T
*S08-I07
*S23-I07
*T02
*I73
*S08-B
*S23-B<br>
All of these constructs were colony-PCR'd by George and Perry, though the gel did not appear on either of their sites so I'm posting it here. (I found it in the quorum folder on the usual imaging computer) They deserve full credit, however, and should feel free to take this off and replace with a link to their page instead.
<br>
*Well 2: J-T
*Well 3: S23-I07
*Well 4: I11
*Well 5: I22
*Well 6: S08-I07
*Well 7: S08-B1
*Well 8: S08-B2
*Well 9: S23-B1
*Well 10: S23-B2
*Well 11: T02
*Well 12: I73 
<br> It's unclear how long the BL-21s will take to grow, so I stuck them in the shaking incubator near the plate reader at 25C (since I suspect they'll take a shorter time to saturate), and will pick them up either late tonight, or more likely, early tomorrow morning to start another plate reader. <br>
<br> Started a colony PCR on the P-R (tetR-creating) constructs. They are in the PCR machine closest to the door. The plate (with streaks) is in the incubator in the other room, near the UV machine.
<br> br>
Will redo the promoters from scratch, adding a new one for more options, with phosphotase treatment before clonewell this time. Also, will make enough miniprep to send some for sequencing and use the rest for digest/phosphotase etc.
<br>Grew up in liquid culture:<br>
*R0051
*R0011
*R0052 (kan)<br>
<br>To do on Sunday (tomorrow): Miniprep (at least); start another plate reader ... maybe use BL-21 constructs this time?

==7/15==
Miniprepped the promoters: R0011, R0051, R0052<br>
Elution in 50uL
<br>
Though ODs were pretty low for R0051 and R0011 (42.9 and 45.6, respectively), I still took all samples and removed 10uL for sequencing.<br>
9:50am, put into incubator: Digested 38uL of each sample with SP. They are in the plate incubator in the other room. Should be ready by noon today. Remember to phosphotase-treat before clonewell this time ...
<br>
Also, set up another plate reader with the BL-21, am testing for 'quorum like activity' versus induction in J-T and T constructs, and also interaction between S08-I07 and S23-I07 with the T. Added a positive RFP control this time, though I took fluorescent readings of the saturated cultures and I'm a little concerned because of the difference (already!) between the S-I and positive control in terms of RFP fluorescence
<br>OD and Fluorescence readings of RFP Saturated Liquid Cultures:
*J04450 (constitutive): 1.695, 4987.1
*S23I07: 1.807, 752.85
*S08I07: 1.690, 512.57
===Overnight Plate Reader Setup===
All samples used 1uL of original (saturated) liquid culture into the 200uL well.
*A1-3: J-T alone
*A4-6: J-T with 100nM OHHL
*B1-3: T02 alone
*B4-6: T02 with 100nM OHHL
*C1-3: T02 with S23I07
*C4-6: T02 with S08I07
*D1-3: J04450
*D4-6: I13522

=Week 5=
==7/16==

Came in early to colony-PCR some of the promoter/tet constructs. Most of the plates had appropriate growth, with the exception of R0052+P0140, which had hardly any (3?) colonies.
<br>
#Ladder
#R52-P01-1(worked)
#R52-P01-2
#R51-P01-1(worked)
#R51-P01-2(worked)
#R51-P03-1(faint)
#R51-P03-2
#R11-P03-1
#R11-P03-2(worked)
#R52-P03-1(worked)
#R52-P03-2
#R11-P01-1
===Plate Reader===
{| {{table}}
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''1to3'''
| align="center" style="background:#f0f0f0;"|'''4to6'''
| align="center" style="background:#f0f0f0;"|'''7to9'''
| align="center" style="background:#f0f0f0;"|'''10to12'''
|-
| A||J04450||I13522||I13263||S23-I / 12 has just LB
|-
| B||J-T||J-T + 100nM OHHL||J-T + 10nM OHHL||S08-I07
|-
| C||T02 noninduced||T02+100nM OHHL||T02 + 10nM OHHL||S23-E
|-
| D||0.5uL T02 + 0.5uL S23-I07||0.25uL T02 + 0.75uL S23-I07||0.75uL T02+ 0.25 uL S23-I07||S08-E
|-
| E||0.5uL T02 + 0.5uL S08-I07||0.25uL T02 + 0.75uL S08-I07||0.75uL T02+ 0.25 uL S08-I07||T02
|-
| 
|}
The "0.25uL" samples were actually diluted down, such that the addition of cells was equal to 0.25uL of the original liquid overnight culture, but the volume was within our pipet's capabilities.
<br>Read at the following wavelengths: <br>
584/607 (RFP)<br>
485/538 (GFP)<br>
Two wavelength sets, one at upper limit and one at lower limit of plate reader capability, to try to detect cell number through scattering.
===More Random Readings===
Goal: Figure out quorum "slope", assay for washing cells if necessary. This is more for me than for anyone; I'll have notes once I actually read them. Ha.<br>
[http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=28106 Quorum-sensing acts at initiation of DNA replication in E.coli]
<Br>
[http://www.jbc.org/cgi/reprint/M311194200v1.pdf Looks amazing]
<br>[http://www.jbc.org/cgi/reprint/279/14/13645.pdf]

==7/17==

Learned how to use the microscope (guess this is pretty important, since now I'm developing new assays with this), will do an overnight plate (per usual - I'm getting pretty efficient at these, at least), will grow up some liquid cultures. Also, troubleshooting/randomly trying a new diluting/quorum(?) assay with the JTs. <br><br>

Plate Reader Results from last night:
*

Reading notes:<br>
Recombinant GFP E coli: [http://pubs.acs.org.ezp1.harvard.edu/cgi-bin/article.cgi/langd5/2006/22/i22/html/la060661v.html]<br>
*"Time-dependent changes in bacterial growth were monitored by measuring OD595"
*Macromolecules present in LB media such as sugar and carbohydrates absorb at A400-600
<br><Br>
<u> Problems with Plate Reader Assay</u>
*LB dries out before mid-log phase is achieved
**Perhaps dilute such that initial cell number is higher: use 1:50 or 1:100 rather than 1:200?
**Use 300uL in each well instead of 200
*Control (13522) shows increasing fluorescence (divided by) OD over time
**May be function of cells not yet reaching log - protein expression differs over growth curve?

<br>
<u> Next FACS Appt Ideas</u>
*Bring samples at different points of growth (J-T). Take OD before going, measure fluorescence. Determine whether cells at beginning of growth are already fluorescing (residual fluorescence? self-induction?)
**Cells would have to be collected throughout the day, stored at room/non-permissive temp. Also, shake: so that cells do not settle and therefore cause false quorum reading?
**How long do cells last in PBS?
<br>
<u> Misc</u>
*Continue to use microscope to look at individual cells, determine when fluorescence turns off and whether self-induction actually occrs
*To determine quorum-activity:
**I13263 ... assuming luxpL is downregulated at a concentration of dimerized luxR/OHHL that is above quorum concentration, fluorescence should shoot up at quorum but then be downregulated, such that the liquid culture does not fluoresce. 
**The "slope" of fluorescence at supposed quorum can be compared to induced ... not sure if this will imply quorum occurs or not ...

*Set up an OD plate reading for tonight, with higher concentrations of cells to begin with, the plate cover on, and 300uL of media. 
*Grew up 9 different liquid cultures (mainly BL-21, maybe except I13263)
<br>
<Br>
Some exciting microscope things that I've done ... to come soon. Yay images!
<br>
http://openwetware.org/index.php?title=Shlo/fluormicroiGEM07
<br>
<u> To Do tomorrow:</u><br>
*Fluorescence assays with 13263, lower dilutions
*More controlled scope pics with regular time points, this time with endpoint fl/OD readings
*Make more lysine-plate covers
*Clean up this wiki. Word.

==7/18==

Overnight cultures: Interestingly, some did not grow: S08-B in BL-21, J04450.
<br>
Prepared some poly-L-lysine slide covers; need to put this protocol (=easy) and the microscope protocol on the wiki. <br>
Also, diluting and looking at J-T fluorescence under scope. Hopefully it turns off at some point; growth is easier to monitor today since endpoint readings will be available in the afternoon on the plate reader.
<br><br>
{| {{table}}
| align="center" style="background:#f0f0f0;"|'''18-Jul'''
| align="center" style="background:#f0f0f0;"|'''Fluorescence of Saturated Overnight Liquids'''
| align="center" style="background:#f0f0f0;"|''''''
|-
| ||485/538||584/607
|-
| I13263||316.2||155.8
|-
| I13273||232.01||160.1
|-
| J-t||3990.6||174.59
|-
| T02||143.03||144.42
|-
| J04450||78.3||179.86
|-
| S08-B||76.2||184.7
|-
| S23-B||99.7||174.4
|-
| S08-I07||104.36||2126.1
|-
| S23-I07||97.849||2111.1
|-
| LB||67.67||141.26
|-
| 
|}
<br>
Notes: 
*I63 has leakier expression than I73
*J04450 must not have grown up; not a concern since it won't be needed in an assay in the near future
<Br>
I made 5 different dilutions on a 96-well plate and took OD/Fluorescence readings intermittently (ever 15-30 minutes). In attempts to keep the ODs roughly constant, I made dilutions every once in a while ... this needs to be much more normalized (and systematic) though. Will be redone ... tomorrow? Late tonight?
<br>
The TFs suggested that I make larger liquid cultures, spin down cells and reconstitute in PBS to view under the fluorescent microscope. All of these were grown for 55 minutes.


{| {{table}}
| align="center" style="background:#f0f0f0;"|'''Original Dil.'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''Original'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''After 55 minutes'''
| align="center" style="background:#f0f0f0;"|''''''
|-
| ||||OD||Fluor.||OD||Fluor
|-
| 1in10||||0.245||79||0.39||575.33
|-
| 1in50||||0.128||40.6||0.218||183.24
|-
| 1in100||||0.118||38.2||0.217||124.14
|-
| 1in1000||||0.101||34.1||0.221||67.1
|-
| 1in5000||||0.099||36.4||0.11||80.036
|-
| Original JT||||1.277||513||1.217||4423.3
|-
| 
|}

Images found here: [[Shlo/fluormicroiGEM07 |Fluorescent Images]]

<Br> Also, set up a 96-well (full) plate overnight to test I13263 and a few other things ... will continue doing quorum microscopy tomorrow. Growing up some liquid cultures tonight in preparation for this.
I also plan to keep OD of a J-T culture constant (hopefully pretty low) and hopefully fluorescence will decrease over time. With the growth curves I created of the BL-21s last night, this shouldn't be excessively hard, though it will probably be time-consuming ...
<br><Br>
<u>Goals</u>
*Determine quorum-like activity of JT through time/fluorescent/keeping OD constant-assay. 
*Determine self-induction through huge dilutions, spin downs, and looks at individual cells
*Continue to organize Wiki
*Fastest way to count percentage of total cells fluorescent?
*Comparison of FACS, plate reader, and microscopy
*Characterizing promoter lux pL downregulation through overnight fluorescent assay
*Write protocol for fluorescent scope, preparing slides, be more clear about plate reader parameters, figure out normalization by LB cuvette
*Organize Excel sheets, post relevant graphs and label all data (this will probably be done in NJ next week) ... have never taken the time to fully analyze or cross-inspect, so perhaps this break from benchwork and manipulation of data will reveal something

==7/19==
<u>Thoughts from last night's plate reader</u>
*I63 (1:200 original dilution) never saturates in fluorescence, even when induced T02 does. However, 1:100 dilution seems to ...?
*Concerning 7/17 ODs, S23/S08-I07 do not seem to have different growth curves. 
*Curves from last night (S23/S08-I07 with T02 or I63) indicate that S23-I07 is a better sender (causing greater fluorescence in recievers; or greater absolute fluorescence), yet S08-I07 shows greater RFP (in an absolute manner) compared to S23-I07. This means perhaps that S08-I07 takes up more of the culture as a percentage of cells versus the receiver, but perhaps a lower sender:receiver ratio is more efficient [since lower S23-I07 was more effective ...?]

<Br>
Started another plate reader / OD / Fluorescence on JT to try to determine quorum-like activity; hopefully, will be more systematic this time.
<br>Called the SpectraMax Company on why RFUs are different ... troubleshooting will take a few hours, perhaps.
<br>Am forgetting about quorum-detection/microscope/benchwork for at least a couple of hours while I analyze the pile of data that I was planning to look at in NJ next week.
<br><br>
===Notes from Reading===
*Papers often track proteins expressed for quorum ... increase 1000-fold
*In light organs: When <i> Vibrio fischeri</i> cells reach a density of 10^10 or 10^11 cells per mL, the autoinducers can reach a critical concentration (5 to 10nM) required for lux operon stimulation
*In sea water, since the autoinducers diffuse out of cells and density of bacteria is less than 10^2 cells per ml, no observable luminescence
*At sufficiently high AHL concentration, AHL interacts with LuxR to form a positive transcriptional complex, which then activates transcription approximately a thousand fold
*AHL of V. fischeri was shown to stimulate PQ2-lacZ expression through plasmid-encoded sdiA slightly in e.coli
**Appears that e.coli cells use SdiA to detect AHLs produced by the other bacteria
**SdiA may allow multidrug resistant
*http://pubs.acs.org.ezp1.harvard.edu/cgi-bin/article.cgi/acbcct/2007/2/i05/html/cb700036x.html
**High cell densities (greater than 100nM in vivo) required for quorum by LuxR/OHHL
*[http://jb.asm.org.ezp1.harvard.edu/cgi/reprint/176/2/269?ijkey=e8bcd34a2e0de21dfcc973597b3a514b74770f5f&keytype2=tf_ipsecsha Quorum Sensing in Bacteria: the LuxR-LuxI Family of Cell Density-Responsive Transcriptional Regulators]
*Really nice paper on what we'd might expect: [http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&pubmedid=5473898 Paper]
*[http://www.sciencedirect.com.ezp1.harvard.edu/science?_ob=ArticleURL&_udi=B6WK7-457D0X9-9&_user=209690&_coverDate=06%2F08%2F2001&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000014438&_version=1&_urlVersion=0&_userid=209690&md5=ca5d2b9abc9431e23dbac53af1972341 This paper is awesome like whoa]

===Analysis===

I did a bunch of this today. Most results indicate that S23-I07 is a better sender than S08-I07. Their growth curves (7/17 overnight) are equivalent, yet RFP appears higher in S08I07 than in S23I07. Yet S23I07 has consistently yielded stronger absolute GFP in I63, I73, and T02. This may mean that a) it is competing less vigorously, so more receiver cells are present as a percentage and this may be optimal for maximizing absolute GFP, or b) somehow, its luxI is synthesizing OHHL more strongly, despite its lower RFP compared to S08I07. <br>
From 7/16 overnight:<Br>

===Really ...===

If you are reading my wiki, [http://www.sciencedirect.com.ezp1.harvard.edu/science?_ob=ArticleURL&_udi=B6WK7-457D0X9-9&_user=209690&_coverDate=06%2F08%2F2001&_rdoc=1&_fmt=&_orig=search&_sort=d&view=c&_acct=C000014438&_version=1&_urlVersion=0&_userid=209690&md5=ca5d2b9abc9431e23dbac53af1972341 read this!!!]

==7/20==

{| {{table}}
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''T9'''
| align="center" style="background:#f0f0f0;"|'''I07'''
| align="center" style="background:#f0f0f0;"|'''J04450-JT'''
| align="center" style="background:#f0f0f0;"|'''S08I07'''
| align="center" style="background:#f0f0f0;"|'''S23I07'''
| align="center" style="background:#f0f0f0;"|'''JT'''
| align="center" style="background:#f0f0f0;"|'''LB'''
| align="center" style="background:#f0f0f0;"|'''J04450'''
| align="center" style="background:#f0f0f0;"|'''I13522'''
|-
| Fluorescence||||||||||||||||||||
|-
| 485/538||||855||118||11438||193||159.3||3327||68.78||90.335||5943
|-
| 584/607||||196||187||176.81||4802||337||215.3||242.38||3360||180.58
|-
| 
|-
| OD||||1.737||1.46||1.913||1.907||1.032||1.823||0.368||1.492||1.729
|-
| 
|}

For some reason, the J04450-JTs have much stronger (about 3.5x) GFP readings than just the JT, whereas the J04450 RFP reading is lower than just the JT and much, much lower (about 20 fold) than the J04450 alone. This is because the J-JT was not induced with IPTG. However, if we are going to try to use RFP as a marker for growth, than this probably isn't a good way to go about it ... we need a constitutive RFP. There's not much available from the BB website: I13521 has a tetR promoter, which might not be good if we plan to use tet system to regulate JT, and I13520 is pBad/araC, which we aren't sure is appropriate for our system. S03473 is also a possibility (cI/lam) but is not available on our plates. What to do?<br>
Consistent with our kinetic assays, the S08I07 shows much stronger RFP readings (over 10x) versus S23I07; also, interestingly, the OD of S08I07 is almost double that of the S23I07. This supports the hypothesis that S23I07 is less competitive in a S23I07+reciever mixed culture, but the fact that it induces more overal GFP fluorescence indicates that a smaller amount of sender relative to reciever may be optimal. This is also supported by the fact that S23I07 alone has less GFP than the S08I07, so the difference in GFP fluorescence is probably from the reciever (as it should be).
<br>Anyways, most of what I just wrote is apparently completely wrong.

===Readings===

Luminescence Control in the Marine Bacterium Vibrio fischeri: An Analysis of the Dynamics of lux Regulation (James et al 2000)
*Build-up of autoinducer subject to many factors in addition to pop. density: growth rate, permeability of cell membrane, shape and degree of enclosure of culture
*Other levels of control: catabolite repression effects the expression of lux genes cloned in e. coli through cAMP receptor protein (CRP), Dunlap and Greenberg, 1985
*Luminescence in the absence of extracellular OHHL
**Three steady-states, two are non-zero
***One is non-zero and unstable
****Therefore, small differences in certain parameters (cellular concentration of OHHL [A], cellular concentration of luxR [R], and cellular concentration of LuxR/OHHL complex [C]) determine whether system ends up in non-induced (zero) steady state [S0], or the luminescent state S2. 
****This explains the switch-like behavior of the lux system, where the cell suddenly "turns on" luminescence. 
****If S2 is
<br>
Obviously I got sidetracked as I as writing this. Because ...

==="New" Assay===

The assay's really just a more regulated, controlled plate reading of what I did before, plus I'm pretty much manually using the machine so I can read both OD and fluorescence. Harris taught me how to have a blank on my plate (how I wish I knew this before), which probably will make things 1000x better. Will do a similar overnight assay, except I won't be here to monitor OD ... It's really hard to make this Excel when I use the machine manually on "endpoints", actually, since I can't make one big excel so I have to export each mini-experiment (endpoint) on its own and I'll have to compile it later. Also, my time points are spaced so close together, there's really no time to manipulate data between runs, just to copy-paste it.
<br><br>
Original concentrations of S08I07 and T02 were as close as I could possibly make the dilutions: 0.636 and 0.642.<br>
Different combinations (ratios) of constructs were used, but always adding up to the same volume (and presumably, approximately about the same number of initial cells). ODs and fluorescence (for both GFP and RFP) were taken as endpoints every ten minutes, translated onto an Excel. Will be posting results ASAP. "Blanks" were two wells of LB.
====Results====
To be updated, of course.<br>

As expected ... since these aren't normalized per sender cell. So, "per cell", there should be more RFP as sender:reciever ratio increases since there are more senders than recievers.
<br>


===Overnight Plate Reader===
Started ~ 9pm

==7/21==

Miniprepped and Nanodropped JT (84.1 ng/uL), R0051-P0140-1 (85.6 ng/uL), R0051-P0340-1 (48.8 ng/uL), R0011-P0340-2 (57.4 ng/uL). Eluted in 50uL but wanted to use as small a volume as possible for the digest, so with help from Perry, vacufuged and then resuspended.<Br>

Also, digested, and put in plate incubator at 10:30:
* 21 ÂµL DNA
* 0.5 ÂµL BSA
* 1.5 uL Buffer 2
* 1 ÂµL EcoRI & 1 ÂµL SpeI for R-P; 1 ÂµL EcoRI & 1 ÂµL XbaI for JT
<br>(not sure this is correct, actually)
===Plate Reader===
Am running another plate reader this morning - raw excel info will be available via "Plate Reader Data" - but will not be doing one overnight.
<br>
=Week 6=
==7/23==
Group meeting this morning, see the top part of wiki for Powerpoint link.<br>
This morning, diluted a liquid culture of T02, put in shaking incubator. Allowed growth for 2.5 hour, took OD (mean = 0.479), GFP low as expected (about 176 RFU). Induced 2 mL of T02 in 1000 nM (= 1uM) OHHL, put in incubator at 12:45.<br>
2hr 15 min later - measured OD and GFP fluorescence, OD was about 0.96 and RFU was 3400 (approaching maximum for T02). Washed 3 times in PBS. Reconstituted in LB-Amp.

==7/28==
Wanted to start an assay with cultures that had been grown for over 12 hours, but interestingly, T02 induced did not show that much fluorescence, nor did J04450; ODs were pretty low for many cultures. Decided to wait until Sunday 7/29 to work on things.

==7/29== 
Started a JT/T02 assay to test for self-induction. Preliminary data already available via my Plate Reader Data page (link on top).
===Determining self induction via JT: Basic protocol===
*Overnight cultures of JT, T02 induced, T02 noninduced, I13522 (const GFP), B0015 (negative control)
*Wash/resuspend in LB: JT and T02 induced
*Measure ODs and fluorescence from overnight to determine dilution quantities
*Dilute to about 0.3, take fluorescence (should have a somewhat linear relationship to original dilution)
*Dilute again to 0.1, take fluorescence (again, should be linear)
*Start kinetic OD run, read every 5 minutes (so we can monitor pretty closely)
*When OD reaches 0.3, take fluorescence
*Dilute again to 0.1 (should be a 1:3 dilution ... so remove 133.3 of sln in well and replace with fresh LB) [read fluorescence just to check for linearity again .. not essential]
*Let grow until 0.3, repeat readings
*Continue until fluorescence at 0.3 is substantially lower than original fluorescence, indicating lack of self-induction
*Switch to fluorescent kinetic reads, ensure JT returns to original fluorescence

===Observations===
*From 0.1 to 0.3, it takes approximately 45 minutes for T02 but we anticipate that it will take about 1.5 hr for the rest of the samples (we didn't stop it at 45 to dilute down the T02, so we will just have to dilute them more later)
*It seems that some wells grow significantly faster than others, despite initial ODs that are relatively consistent ... in particular, T02 induced grows very quickly
*T02 induced did not act as if it were induced ... initial fluorescence from overnight was about twice that of T02 (noninduced), and after washing, it seemed fine - but once I rediluted and started kinetic, fluorescence was virtually same as noninduced
**This could do with sensitivity (or lack of) in plate reader for GFP
=Week 7=
==7/30==
As per tradition, find my presented data (... not much this week) and lab meeting notes on Meetings subpage of Harvard iGEM.


Did a lot of brainstorming today w/ other members + Harris. Data analysis (George's last plate reader) indicates RFP takes at least 5 hrs to fold & detect ... a huge lag, not good to measure cell number in our eventual bead assay. Maybe use another version (mOrange and mCherry? Some logistical problems - ie lacI regulator, not in BL21 - need to figure out), test to make sure this fluorescence corresponds to growth of GFP and cells. Or membrane stain (remains in liq culture) or lacZ/betagal. Harris' search (spectral scan) for autofluorescence outside the GFP range was null.


Need to find membrane cover for plate to avoid cross-contamination? Or, if I have many wells for myself, spread out samples. Should try this tomorrow. Also, GFP: 485/518 from now on, though I'll do 485/538 if I have a spare wavelength for comparison.


Will redo plate assay tomorrow w/ JT dilutions and perhaps a few other random samples (supernatant removed from S23I07 for induction of T02?), grew up new liquid cultures. Hopefully they'll be ready for tomorrow, but last time George grew me samples, they took a surprisingly long time to grow ...

==7/31==
Re-ran the JT experiment, with 6 samples of each and three different dilutions (... very time-consuming and definitely not something I want to redo). Results turned out really nicely, actually. 

<br>
*Indicates that final ODs of 0.3 and 0.5 (diluted down to 0.1 and 0.3, respectively, in sextiplicate (?) ... 6 samples each) are below quorum; fluorescence returns to near-level of T02
**JT samples were done in 12-plicate
*Complements my results from past Sunday
*0.7 is likely above quorum, but my dilution was probably below quorum (/quorum is not a "defined" concentration; this is the grey area)
*Need to add error bars to this graph



[[shlo/notebook/73107results| More Results Here]]

==8/1==
Will do suggested growth assay. Dilute to .05 (not measurable, but estimated), 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0(11 dilutions), T02 noninduced (6 wells), T02 induced at innoculation w/ SI supernatant (6-12 wells; vary amt added?), B0015 (3 wells?), I13522 (3 wells), LB (2-3 wells).
For further results, refer to presentation [http://openwetware.org/images/6/63/080607slo.ppt here]<br>

===OD Curve===
Blue line is from George's last growth; is not for JT, rather, is just for J, but shows "expected" growth for all exponential samples ... does not really correlate with mine, except for perhaps 0.05. Will need to repeat this again (why aren't my JT growing right?)
<Br>
To do tomorrow:
*Endpoint read on fluorescence (left this plate in shaker, since Shaunak and Ellenor are using plate reader)
*Calibration curve w/ photospectometer and plate reader
*Bill's suggested "assay", similar to redoing this except OD/fluorescence in parallel, with photospectrometer/plate reader
*Continue to look into this data

===Fluorescence over OD===
Comparison of initial, 200 minutes (near minimum for all samples), and after overnight. For further results, refer to presentation [http://openwetware.org/images/6/63/080607slo.ppt here]<br>


===T02 induced with SI supernatant after 390 minutes===

===T02 induced with SI Supernatant after Overnight===


===Overall thoughts===
While I feel the results are heading in the right direction, the non-exponential-ness of the growth curves is worrisome enough to make me redo this assay. Also, though endpoint readings seem to be fine after a certain amount of time, I'm still unsure enough to want to try a kinetic fluorescence read.

==8/2==


Similar assay to yesterday's, except fluorescence readings are left kinetic on the plate reader, and parallel dilution readings are done on the photospectrometer.
===Preliminary Results===
*T02 shows quick increase in fluorescence (in all six wells ...); perhaps it was contaminated with I22 or something? I'm not sure how/why this would hapen. 
*JT only shows "exponential" growth in more-diluted samples (1:100, 1:200).
*Fluorescence per OD varies quite a bit in JT samples, even at beginning - I'm led to believe that our ODs aren't as measured ...?
===Thoughts===

So it turns out the covering-cuvette with parafilm is a bad idea, may have affected growth; I won't use these results. At all.

==8/3==


Redid the plate reader assay from 8/1, with endpt OD and fluorescence. Readings every half hour, starting at 10:15. 


Set up a plate to test optimal induction mixes using 20uL T02 and varying amounts of (different kinds of) supernatant, from spun-down overnights.

For further results, refer to presentation [http://openwetware.org/images/6/63/080607slo.ppt here]<br>

===JT Experiment: Preliminary Results===

===Sender/Receiver Experiment: Preliminary Results===
For further results, refer to presentation [http://openwetware.org/images/6/63/080607slo.ppt here]<br>

=Week 8=
For further results from week 7, refer to presentation [http://openwetware.org/images/6/63/080607slo.ppt here]<br>
==8/6==


Starting to bridge gap between magnetic assays / AIDA targeting and quorum sensing.


Made new streak plates and cultures with help from MACS-constructs (from Mike's plates); AIDA-his-S23I07 and AIDA-strep-S23I07. We don't have AIDA-receiver (T02) yet apparently. 


<u> Combined Magnetic Assay & Agar Plate </u>

*20uL Magnetic his beads (in fridge near plate incubator)
*1000 uL saturated AIDA-sender constructs (Amp/Kan)
*5 washes 
**Used MACS buffer 
**Let beads re-aggregate for 3-5 minutes per wash
*200 uL T02 plated on Amp
*Reconstitute beads in 10uL LB-Amp, plate
**Let sit for at least 30 minutes before moving to incubator to allow sender cells to settle


Expected: AIDA-Senders (nonwashed) should both serve as positive controls; AIDA-strep should show no fluorescence/induction since they are nonspecific to beads.

{| {{table}}
| align="center" style="background:#f0f0f0;"|'''OD'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
|-
| ||From Culture||Supernatant from first wash||Supernatant from 5th wash
|-
| His||1.738||0.365||0.296
|-
| Strep||1.784||1.372||0.132
|-
| 
|-
| 
|-
| Fluorescence (RFP)||||||
|-
| ||From Culture||Supernatant from first wash||Supernatant from 5th wash
|-
| His||4526.3||1372.2||819.02
|-
| Strep||4565.1||2955.9||315.16
|-
| 
|}

==8/7==

Am electroporating and co-transforming AIDA-his and AIDA-strep into liquids of T02. Let recover for 1 hr, 1:100 dilution, plate 50 into selective Amp/Kan plate. 

Time constants: 
*Control: 5.2
*Strep: 5.1
*His: 4.5

Yesterday's results = decent ...?




==8/8==

Nothing grew up last night, so took rest of liquid culture (left at room temp) - about 1mL - spun down for 3 min at top speed in microcentrifuge, reconstituted in 100uL LB, plated on Amp-Kan warmed plates, put in incubator ~ 11:15. Hopefully it was an issue of number of cells and not an electroporation that didn't work. I did notice that the pellet of cells was pretty small.


Did a lot of reading on imidazole and I think it's toxic to cells, so to reduce background color (to allow for better detection of red senders on beads), I'll try Agarose-Nickel beads. Started growing up a small amount of tagged senders and T02 so hopefully I can do the assay within a few assays. Essentially a redo of my last Dynabead assay, except washes will have to be done with microcentrifugation (optimal number of beads? not sure; will probably try 20uL for now, maybe move downward later as necessary). I'd like to put some in the plate reader for quantitative stuff from the beginning, as well as streaks later on. I have plates in the incubator warming for the assay as well.


Strep direct beads came in; tonight, grow up strep and his cultures for both dynabead assays ...?


Realized mid-assay that Agarose-Nickel beads would be pulled down by microcentrifuge ... as would the cells. Trying to optimize what speed/length to speed such that beads, rather than cells themselves, pellet. Testing supernatant in plate reader.

==8/9==

Electroporation plates had lawns of colonies. Yesterday's were probably just diluted down too much. 


Yesterday's plates (agarose beads) turned out pretty well; unfortunately, there was a high level of fluorescence, probably because spinning down the agarose beads also caused cells to accumulate. Still, a noticeably higher green halo around the his-agarose beads compared to the strep-agarose. Images!

===Strep sender with his bead on T02 plate===

===His sender with his bead on T02 plate===
===Control: strep sender with no beads on T02 plate===




Since I had leftover beads, I took the same amount from each microcentrifuge tube and grew them up in about 3mL last night along with 1uL of RAE. Here are my results (not sure what the bearing of this is, but it appears that the His- had more sender (RFP) and therefore more induction (GFP)). This is true also when I normalize for OD.

{| {{table}}
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''his bead + RAE'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''strep bead + RAE'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
|-
| ||E7||E8||E9||E10||E11||E12
|-
| GFP||3143.045||3309.642||3161.45||1591.069||1830.288||1837.943
|-
| 
|-
| RFP||1989.934||2118.741||1982.111||702.742||722.174||710.781
|-
| 
|-
| OD||1.8996||1.9272||1.8937||1.5945||1.5996||1.5878
|-
| 
|-
| GFP per OD||1654.582544||1717.33188||1669.456619||997.8482283||1144.216054||1157.540622
|-
| 
|-
| RFP per OD||1047.554222||1099.388232||1046.686909||440.7287551||451.4716179||447.6514674
|-
| 
|-
| ||Average GFP per OD||||||||||
|-
| ||1680.457014||||||1099.868302
|-
| 
|-
| ||Average RFP per OD||||||
|-
| ||1064.543121||||||446.6172801
|-
| 
|}



Nick suggested that I try an assay in which I measure background for magnetic beads. Rough protocol, after collaboration with Mike: 
*Mixed culture of I22 nontagged (Const GFP) and sender-RFP with tag
*Grow up to about 0.3 OD each; don't want to flood magnetic beads with RFP-tags in case the nontagged does show nonspecific binding; use about 25uL beads
*Mix together, redilute, take ODs ("best" OD not yet characterized)
*Plate beads, hope for no green colonies
*Control: mix red/green cells without beads and see proportion; since I diluted to same ODs before experiment, plates should show equal amounts of red/green. 
<br>Also did this for T02 strep/his with strep beads (his did not grow ... need to make streak plates tomorrow)
<br>AS a side note, kept supernatant from all washes and measured these for RFP/GFP. Strep beads do not appear too optimistic, but I will watch plates and post numerical data tomorrow as necessary. 

Grew up liquid cultures for tomorrow - senders (his/strep/nontagged), receivers (his/strep/nontagged), I22 (control)

==8/10==

Results from yesterday: Way too many cells. Apparently, I need to start with a lower OD than .4 or .5 (had used about 1/4 to plate). Still, a few optimistic results, perhaps ...?

===T02-strep with S23I (nontagged): all results===

===S23I-his with I22 (nontagged): all results===
===S23I-strep with I22 (nontagged): all results===



It turns out the AIDA- and Omp- constructs are IPTG inducible (1000x), which I didn't do and hadn't known. It seems there's a substantial amount of leakiness and expression still because some of my plates had decent results ...


Made an imidazole 50mM + 1x PBS stock to do washes in; should get rid of nonspecific binding but allow other specific cells to stay attached to beads. 


Also, in today's assay, will rotissiere in refrigerator to agitate/allow for more specific binding while preventing growth. Mike suggested I try testing Omp constructs as well. I was able to image the Talon beads under the 100x microscope in the cell culture room so I'll also save some beads after my assay to visualize.

===Cool microscopy images: all with his-sender attached to his bead===

==8/11==


Results from yesterday's assay:

===Summary in Words===
*T02 strep w/ S23 nontagged
**Control (200uL plated from mixed liquid culture)
***About 50% red (S23I) and 50% "white" (green?) T02 colonies. Whereas red fills the space, creating a "background", the T02 white colonies tend to clump. This is consistent with Mike's results. 
**TS with strep bead (10uL)
***5 colonies, all white. This is pretty good, in that there are no red colonies, but there were probably too few initial beads. 
**TS with strep bead (50uL)
***About 20 colonies - again, clumps (around beads, or by each other?), but at least no red!
**TS with strep bead (100uL)
***Only about 20 colonies again - does this indicate that they weren't from beads? At least no red, again. 
**TS with his bead (10uL)
***6 white colonies - almost as expected - we didn't want any colonies
**TS with his bead (75uL)
***A good number of colonies - maybe around 100. They all look red-ish. I wonder if this indicates that the senders (red), even nontagged, have affinity for the His beads. 
*SI-his and I22 
**Control (200uL plated from mixed liquid culture)
***Almost all green, though some red colonies are detectable at the edge where the green stops. Is this because it's hard to see the red with the naked eye (green shields red?), because the red takes longer to express, or simply because I22 has taken over (competitive advantage ...?). I may be able to rule some of this out by either swabbing and growing up liquid culture (seeing if red eventually expresses in that, which would indicate presence of sender as well) or by letting the plate grow more and hoping red is eventually expressed in greater quantities. We'd expect the eventual ratio of cells to be 1:1. 
**with his bead, 10uL
***As expected, quite a robust selection of red colonies - almost all red! Some of the outer colonies aren't that red yet, which makes me think they just haven't expressed the RFP, but they're definitely not the green we're seeing in the I22 control. There are probably about 1000 colonies on this plate. 
**with his bead, 50uL
***Way too many colonies to count, but I only see red, with maybe a few white colonies at the edge, but it's hard to tell whether it's simply because the RFP isn't strongly expressed yet. 
**with his bead, 150uL
***Way too many colonies to count, but I see all red and no green ...!
**with strep bead, 10uL
***4 colonies, all white. This is good and as expected - we didn't want to see any colonies.
*SI-strep and I22
**Control (200uL plated from mixed liquid culture)
***Compared to SI-his control, there are more detectable (small) red colonies intermixed with the green in the plate. The green still seems more robust.
**SIS with HB (10uL)
***There are about 1000 red colonies. This is totally unexpected - the his bead should not select for the strep-tagged sender
**SIS with SB (10uL)
***About 30 colonies - all are starting to show red, I think - it's hard to tell ..
**SIS with SB (150uL)
***A really good number of colonies and a really good result: 37 green, 293 red.. This indicates really good selection!

===Images===
====Promising Strep results that indicate good selection====
Strep sender with strep beads: control (left) versus experimental (right)
====Unspecific binding?====
Strep-receiver, mixed with nontagged constitutive GFP, with his bead still yields red colonies, but surprisingly, no green. (the background on the plate is due to the brown opacity of the beads). This indicates that the strep-aida construct on the receiver may have some affinity for the his bead. <br>
====Controls: Growth rate issues?====
Controls: equal ODs of both cultures were added. In sender experiments, tagged T02-strep (white/green) was added to untagged sender (red). In reciever experiments, tagged senders (strep/his, Red) were added to untagged constitutive green. The plates below indicate that the untagged cells tend to "take over" the plates ... which is probably bad, in general, but for our experiments, this makes our "enrichment" results even more outstanding (ie strep receiver experiment above: in control, almost all green, but able to select for red in assay).<br>


====Sender-His plus His Beads indicates good selection====
Compare to control (above, far right), which is mainly green; this is almost all red (though colonies are too plentiful to count); compared to control, indicates really good selection. 

=Week 9=

==8/13==
Group Meeting: my [[Media: Shlo0813igem.ppt | powerpoint]]

===Morning microscopy===

Wanted to visualize stickiness of AIDA tags (without senders), but so far, no luck - I only see clumps of cells with beads. I guess this is positive ...? Will re-try with liquid culture, no beads. 

===Today's Assay===

Used T02 his and T02 strep (OD = 0.30) along with strep and his senders (OD's also 0.3) and beads to try to induce quorum sensing and see binding ratios, etc. 
<br> 4 Samples: 
*His-receiver + His-sender + His bead
**Expect equal mix of red and green colonies upon plating
*Strep-receiver + Strep-sender + Strep bead
**Expect equal mix as above
*His-receiver + Strep-sender + His bead
**Expect white (green, if induced?) colonies only
*Strep-receiver + His-sender + Strep bead
**Same as above

==8/14==

Yesterday's results: (I still need to do colony counts on all of these)

===Controls===
====T02-his with his and strep senders====

<br> About 250 colonies each. The T02-his with his sender definitely has at least 50% red colonies, but it's hard to tell whether the rest are white T02 or just haven't expressed red yet. I will swab and look at under fluorescent microscope (plate reader?) to determine this. The T02-his with strep-sender has mostly red though there are some small colonies toward the center that may be white (or, again, just haven't expressed red yet). 
====T02-strep with his and strep senders====
<br>There are about 250 colonies on each plate. The red on these plates is lighter than the His-T02 controls, so I'm led to believe that there may be some T02 that is also grouping with the red colonies (since AIDA-tags may be sticky to one another). I'll do some microscopy to see if I can detect this. Also, there are a few very large white colonies, which is a weird phenotype also detectable on the plate from the electroporation.
<br>I took a swab from one of the large white colonies on the control on right (strep-T02 and his-sender) and I got about 50% RFP, 50% white cells. This would be promising, except now I realize that I have no clue whether these are receiver cells etc.

===Experimental mix===
Expect mix of white and red.
====His-receiver plus His-sender plus His-beads====
<br>Colony counts to come. There are hints of red in these white colonies, which makes me believe there may have been good selection. Under fluorescent microscope, one colony from 75uL plate only showed rfp (and some cells seem to have problems dividing - image to come). One colony from 25uL plate also seems to have this issue. (rfp, no gfp)

====Strep-receiver plus strep sender plus strep beads====

<br>The 10 ul has 3 white colonies. The 100 ul 15 white-ish (maybe some pink in the center?) colonies.
===Experimental receiver selection===
Expect all white.
====Strep-receiver plus his-sender plus strep beads====
<Br> Literally no colonies on either plate.

====His-receiver plus strep-sender plus his beads====

<br>Lots of colonies. I should have selected for white colonies but most of these look pretty red, which is worrisome. I picked one colony from the 10ul and got all red under the fluorescent microscope. Gah.

===More Assays===
I grew up some liquid cultures of T02-tagged last night. None of the T02 his seems to have grown (maybe one?). I have a grown T02 strep and a T02 strep induced; will use the plate reader to hopefully get some GFP. If I can induce GFP, then a) I can potentially plate these as "correct" receivers, and b) I can redo the bead assay and seeing GFP would be really nice on the fluorescent microscope (and a nice contrast to the RFP senders)

I had incubated strep sender, strep receiver, and strep beads in the rotissiere in the fridge overnight, imaged them. Only saw RFP, again, no GFP, and to my knowledge, no white cells either.

Also, am colony-PCRing the T02 with tags to see if they're right. I'll probably tag/electroporate some RAE receivers and/or re-try the T02 electroporation.
<br>None of these worked. Gross.

==8/15==

Started re-growing up some liquid cultures for electroporation later today. Intend to try a lot of different combinations: into T02 and RAE cells, and also the receiver into the AIDA-his and AIDA-strep. (8 electroporations total? maybe not.)

Redoing JT assay (with Top10 cells, which should work) so I can have good results to add to the iGEM Jamboree presentation. I'm doing the kinetic run today (reads every 20 min), but I might do the redilution assay if this doesn't work.

Yesterday I had taken I22 and AIDA-senders, incubated same amt of cells in the refrigerated rotissiere to see whether cell clumping could be detected on a microscope-level. If the clumping were specific (AIDA-to-AIDA homogenous only), I'd expect to see RFP clumped but GFP splattered; otherwise if clumping and AIDA-stickiness were nonspecific, I might see green and red clumps. On the microscope, for both AIDA-his and AIDA-strep, I noticed that the red cells tended to be clumped, whereas the green were more evenly distributed and tended to be alone. I'm not sure what the merit of observations of the individual cells (and such small clumps) has, but it's an interesting and consistent observation nonetheless. ... Also, images show that there are significantly more red cells than green - not sure why this is - but could cause cells to *appear* like they're clumping when it's a simple issue of cell density. 

11:20am: started incubation of flooded gfp (2:1 ratio) to tagged senders. Hopefully, with more green cells, I'll be able to conclude that clumping is due to AIDA-specific-clumping (which is still bad in the big picture, unless beads cause much higher enrichment) rather than simple differences in cell density.
3:30pm: Visualized, still had far more red than green cells total, though red cells again seemed to be clumping more readily. Will need to start an overnight incubation with far, far more gfp. Maybe 200 ul I22 and 20 sender. <br>

Also, sometime (hopefully by the end of this week) I need to write up my protocol for the direct magnetic bead and imidazole-washing assay.


Electroporation: <br>
Receiver put into cells already tagged:
*strep-T02: 4.8 ms
*his-T02: 5.3 ms
*his-RAE: 5.1 ms
*strep-RAE: 5.1 ms
<br>
Tags put into receiver cells:
*his-RAE: 4.3 ms
*his-T02: 4.40 ms
*strep-RAE: 5.4 ms
*strep-T02: 5.2 ms



==8/16==

Came in early this morning to start a colony PCR (put into machine at 9am), check plates, etc. 

From electroporation:<br>
{| {{table}}
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''Tags into reciever cells'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''Reciever cells into tags'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
|-
| ||His||||Strep||||His||||Strep||
|-
| ||T02||RAE||T02||RAE||T02||RAE||T02||RAE
|-
| 50ul||9 colonies||Tend to be near edges, somewhat clumpish, about 200 but not very isolated||Very robust and clumpy colonies (I guess, a phenotype we\'d expect with AIDA), evenly distributed over plate, maybe 100ish||Colonies look pretty big - good; somewhat evenly distributed, about 25 colonies||A good amount, though more on one half of the plate - maybe 150ish colonies||About 50, on edge||Very asymmetrical (on edges) colonies, smaller than many of the others, about 100||Very small colonies, makes me think that electroporation was unsuccessful; lots of small colonies but none isolated and not evenly distributed
|-
| 500ul||Somewhat evenly distributed on one side, about 20 colonies||Lots of small colonies all over plate, but not very isolated and not very clumpy-looking||Lots of colonies - a few are isolated - about 400 colonies||Again, phenotype we\'d expect from AIDA (clumpy), about 200 colonies, evenly distributed and somewhat isolated - yay||A few colonies on side, maybe 50 total||Hardly any colonies, except on side||Very small colonies on edges, looks unsuccessful â€¦?||Hardly any colonies, except on side
|-
| 
|}

===JT Results from Yesterday===
1:200 gave best results, grew up to normal OD after overnight, seems to be a positive result that we can present. 
===Microscopy Images of AIDAhis/strep RFP and nontagged GFP to test for stickiness===
About equal number of red and green cells, yet red clumps in both his- and strep- tagged constructs. This is good to know and seems like a pretty good characterization result. <br>

<br>This is actually the strep-tagged RFP with non-tagged GFP. Clumps of red visible.

<br>His-tagged RFP with non-tagged GFP. Again, clumps of red visible.

==8/17==
Came in this morning, started growing liquid cultures of T02 and RAE for attempts at electroporation.

PCR'd up random libraries + AIDA, will run gel to check for bands, then digest, pcr purify, ligation by end of today, hopefully transformation by end of today and/or by Sunday. Will start magnetic bead assays (ask Alain for beads/proteins etc) either Sunday or Monday.


PCR: protocol is same as Omp-library used earlier. Product is something like 1.5 kb, so extension time needs to = at least 2 min.
*Template: AIDA1 in Pet296+ (1 ul)
*Primer 1: MNS27-Strong (10uM) (1.5 ul)
*Primer 2: MNSiGEM2 (10uM) (1.5 ul)
*H2O (1 ul)
*Platinum PCR Mix (45 ul)

After Harris and I realized the extension time on our initial 3 samples (which were eventually PCR-purified and put in my freezer) was too low, we remade the mixture, except I used 2.5 ul of each primer instead, at Mike's recommendation to use more primer. 


Lunchtime meeting about presentation. My notes available on Week 9 meeting page.


After PCR, ran gel: bands as expected, wells 2 & 3 indicate low PCR product (faint bands), probably due to our initial extension time mistake. The bands for 4 & 5, our new PCR products (I have 3 tubes, just used 2 for this) were pretty faint. I used 0.5 ul in 19.5 ul water for the E-gel. 
<br> 

After gel, digested using 35 ul insert, 2.5 NotI (50% efficiency), 2 Nhe1 (100% efficiency in our buffer), 4.5 ul 10X NEBuffer 2, 1.5 ul H2O. Left in plate incubator for about 2 hr 45 min, then PCR-purified. These 3 tubes (1.5 ml tubes, with 35 ul in each) are in the plate incubator. 


Also, digested vector (20 ul) - labeled something like AIDA1 in PET296+, though the tube is really small - also for nearly 3 hrs, then put in black thermal controller near my bench for CIP-treatment. Put it in around 8:15pm; programmed it for 2 hr at 37C before heat inactivation at 65C and then a hold at 37C. 


Needs to be done after these are done: Clonewell (or gel extraction) of digested vector, then ligation (VERY preferably overnight, since Harris is skeptical of the quick ligase), and transformation tomorrow (chemically competent? or electro-competent ... I grew up cells for this. Yes, I remember now). 


Goal: Sunday: new calmodulin / EGF assays. Have been discussing available reagents with Alain, ordered some things in a hurry before the end of the day today. Should have enough to do some on Sunday? Also, if George's electroporations are successful, I should try the his/strep targeting with both senders and receivers - exciting.

==8/19==

Induced RAE-his, RAE-strep, and T02-strep with IPTG after diluting down, put in shaking incubator for magnetic assay later today. 
*Update: after three hours of growth, RAE-his did not seem to grow at al, RAE-strep grew extremely slowly (reached OD of 0.144 after blanking ...), and T02 strep showed expected growth (0.682). Added more cells to RAE-samples, used T02 for assay. 
*Also, RAE-strep seemed acceptable after a couple of hours of more growth; started that w/ the direct bead assay as well. 



Assay: 
*T02-strep OD = 0.433
*S23-I Nontagged: 0.414
*S23-I Strep: 0.386


Add 25ul beads, 750 of each type of culture (2 per experimental sample; eg one type of sender and T02-strep). For controls, appropriate plate with resistance and 10ul of each type of diluted sample.


Microscopy images to come...? Clumping seems evident, which is good - receivers that I viewed likely have tags.


==8/20==
===Yesterday's Results===
{| {{table}}
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''SI-strep'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|'''SI-nontagged'''
| align="center" style="background:#f0f0f0;"|''''''
|-
| ||RAE-strep||T02-strep||RAE-strep||T02-strep
|-
| Control||8 White, 356 Red||Lawns of white||Lawns of red||Lawns of colonies; mainly red except for a few green colonies on the outskirts
|-
| 10||5 Red/white colonies||12 White, 22 Red||2 red/white colonies||41 White, 39 Red
|-
| 75||3 Red/white colinies||132 White, 7 Red||4 red/white colonies||198 White, 237 red
|-
| 
|}
<br>
Red/white refers to a pink-tinted center with a white bulk around it. Maybe RFP isn't being expressed, or it's some red/white colony surrounding a bead?
====Controls====
====Experimental Plates====

===Today's Assay===

Tested receivers with his and strep, along with his and strep senders. Hoped to see signaling with combinations of tagged sender/receivers that were complementary, and tested for background with mixed-tagged cultures. 

====Microscopy====
Exciting - Sender-His with RAE-His combined with His-bead showed microscopy image with targeted cells and only 50% red (indicating that the rest may be receivers that have not been activated yet). I kept the slide and hope to image again later, hopefully seeing green this time.
<br>
Below: left: phase; right: RFP<br>

==8/21==

===George's RACE results===
Came in @6am to run Egel. Results below. <br>

<br>
*1: ladder
*2: RS-2
*3: RS-5
*4: RH-1
*5: RH-2
*6: RH-3
*7: RH-5

<br>As i was loading the samples, I noticed that lane 7 did not seem to have 10ul of PCR reaction mixture. I'm guessing the mastermix wasn't made with excess, and as things go, Lane 7 got less than anticipated.
<br>
Ugh. Although lane 2 appears to have sample, I re-imaged the gel and it looks like it was just a smudge on the imaging device. I don't know why the samples didn't seem to run (there didn't seem to be anything in them, except for the randomness that almost came out of the one lane) - I made sure to make total well volume = 20ul. My one concern was that most of the PCR tubes didn't seem to have a full 10ul of sample. 
What to do? I think I'm going to induce these with IPTG and wait for morning discussion on how to proceed; at least if i induce, I can still test if we decide not to completely rely on the colony PCR data.

===Yesterday's Plate Results===

====Colony Counts / Interpretation ====
{| {{table}}
| align="center" style="background:#f0f0f0;"|'''Testing for background'''
| align="center" style="background:#f0f0f0;"|''''''
| align="center" style="background:#f0f0f0;"|''''''
|-
| ||T02-strep with SI-his and strep-bead||RAE-his with SI-strep and his-bead
|-
| control||28 white, 572 red||Thousands of red colonies and a lot less small white/green colonies dispersed throughout. Estimates: 80 White, 1000 (â€¦ to be improved, estimate) Red
|-
| 10||1 white, 16 red||Lawns of colonies, but about equal amounts of red and white
|-
| 75||18 white, 199 red||Ditto above
|-
| 
|-
| 
|-
| Testing for binding||||
|-
| ||T02-strep with SI-strep and strep-bead||RAE-his with SI-his and his-bead
|-
| control||Thousands of red, a very few green (~30 white throughout)||88 white, 141 red
|-
| 10||2 white, 130 red||Lawns, about 75% white and 25% red
|-
| 75||16 white, 856 red||Lawns, about 80% white and 20% red
|-
| 
|}

===Today's Assay===
*Trying rotissiere at room temp to provoke expression of GFP. 
**Want to be able to visualize, on fluorescent microscope, localization and GFP. I have not seen any GFP at all yet, which is sketchy given the usual leakiness we associate with T02. 
**Wanted a higher number of cells to beads, so reaction mixture was as follows (done for TS and RH): 
***1000 receiver (tagged for bead)
***10 sender (nontagged) OR 200 sender (also tagged for bead)
***10 bead

<br>At 1.5 hr: Visualization of HSHRHB sample. Definite clumping of cells to beads, rfp clumps visible, rest of cells (presumably receivers) nonfluorescent :-(. Will not wash, continue to rotissiere at room temp. 
Am making new page for all today's images; there are quite a few. 
<br>At 2 hr: Still no fluorescence in HSHRHB sample. Induced 200ul with 125 nM OHHL.
<br>At 2.5 hr: Looked at strep-reciever, strep-sender, plus strep bead, a few green (yay?) though hard to see clumps (I think strep beads are much less sticky); also, strep-reciever with nontagged sender and strep beads; still not seeing much (images to come? nothing substantial)
<br>At 4ish hr: HSHRHB sample still had no fluorescence. Ditto the OHHL induced.
Since I'm afraid these results are due to a concentration of cells lower than quorum, I microcentrifuged the HSHRHB sample, pulled out supernatant into separate tube. Had about 20ul of beads/cells, put 10ul in original supernatant, reconstituted the rest in 50ul LB.
<br>Again, SSSRSB shows clumping, but no GFP. RFP only takes half clump though ... rest should be receivers.
<br>At 5hr, (1hr after concentration of HSHRHB sample), I'm seeing at least some green in HSHRHB (even though nothing concentrated yet). Yay?! This is more optimistic than the HSHRHB sample I again visualized and saw *no* green. Will need to visualize again tonight.

===Other Progress / Plans / Onward===
*Redoing controls for my plate experiments from the past week so I can count / quantify enrichment. 
*Will play with concentrations of receiver vs sender ...? Need more receiver and less sender. 
*Need to start working with random library / calmodulin / EGF etc.
*Write-up LB plate pouring protocol 
*Made a new 50uM stock of OHHL in LB, will test tagged RAE/T02 for induction tonight
*Group presentations need to be done by tomorrow. Ugh?
*Visualized one RACE sample under microscope, and it clumps, somewhat to our surprise.

===Images from Today===

[[shlo/notebook/082107images | Images from today]]
(to be updated after group meetings etc?)

===Preliminary Draft: My Slides for Final Presentation ===

http://openwetware.org/images/5/52/Shlofinalpptrough0821.ppt


==8/22==

Colony counts on SIS + TS control (redone): 
<br>
2320 Red<br>
288 White


More images here: http://openwetware.org/wiki/Shlo/notebook/082107images


[[shlo/notebook/822meetingnotes]]


==8/23==

Plate-drop experiment with tagged RAE and tagged T02 completely failed. The "receivers" were all red (entire plate was pink). I looked for GFP under fluorescent microscope (swab from halo) as well as stereoscope but saw no positive results.


Also, imaged Rae-his plus SI-his plus Ni-NTA agarose bead that had been sitting. Cells were still alive, but almost all red (some seemingly nonfluorescent). Again, an indication that the receivers are not what we expect them to be.


I don't expect the entire plate of receivers to be contaminated, though, since I've run plate selection assays on them and they were white, and the T02-strep even showed characteristic leakiness.


Edited the powerpoint (see Final Presentation part of Harvard wiki), wrote up abstract (http://parts.mit.edu/igem07/index.php/Harvard#The_Harvard_University_2007_iGEM_Team). 


Grew up liquids of tagged senders for frozen stock tomorrow. 


Chemical transformation of AIDA + 15mer into BL21 DE3 cells.


==8/24==

Set up a plate to try to induce tagged receivers - variables: w or w/o IPTG, w or w/o OHHL, w or w/o his-sender swab. It's in plate reader; will check back later for fluorescence values and OD values. Unfortunately, with swabs, it's not too well controlled, but it's more to ensure that *some* GFP expression occurs from these receivers. 


I also grew up and induced (12:45) tagged senders and tagged receivers & will likely repeat the plate experiment (tonight? early tomorrow?).

=Onward=

[[Shlo/notebook/fall07 | Fall iGEM work]]