==notes about existing M13K07 genome (11 genes, circular ssDNA, Kan-R, 8669 bp)==
#Independent genes:
#*g3, g5, g6
#Overlapping gene groups:
#*g10, g2
#*g7, g9, g8
#*g11, g1, g4
#'''Overlap details:'''
#*g10 contained within g2 (share same STOP)
#*g7 STOP overlaps g9 START by 2 bp 
#*g9 STOP overlaps g8 START by 2 bp
#*g11 contained within g1 (share same STOP)
#*g4 STARTs 23 bp upstream of the STOP that is shared by g1/g11
#Other:
#*g3 has a weaker START codon (GTG)

==gene/protein function==
{| border="1"
! gene/protein   
! function
|-
| 1 
| assembly
|-
| 2 
| replication of DNA + strand
|-
| 3 
| phage tail protein (5 copies)
|-
| 4 
| assembly
|-
| 5 
| binds ssDNA
|-
| 6 
| phage tail protein (5 copies)
|-
| 7 
| phage head protein (5 copies)
|-
| 8 
| phage coat protein (2700 copies) 
|-
| 9 
| phage head protein (5 copies)
|-
| 10 
| DNA replication
|-
| 11 
| assembly
|}

==analysis of overlapping gene groups==
#g10, g2 = '''both''' internal dna replication proteins
#g7, g9, g8 = '''all''' surface proteins 
#g11, g1, g4 = '''all''' internal dna assembly proteins
#*'''It is not surprising to me that the overlapping genes in these groups share similar functions'''
#*'''Ideally I would like to separate all of these genes. I know that some guy at MIT recently removed all overlaps from a bacterial genome and that while the colonies grew smaller for an unexplainable reason, basic functionality was preserved. If we started here, we would have much more freedom to re-engineer individual parts of the phage without affecting the rest of the machinery.'''

==re-engineering ideas==
#So this seems to be a pretty good phage, but I see the following limitation (from phage's perspective):
#*It can only infect F containing bacteria because it needs to contact the TolA protein in the F pilus to begin transfer of its genetic material into the cell. This reminds me of how HIV cannot infect cells without first attaching to chemokine receptors. Could WE think to (has somebody already/ what is stopping them from doing so) re-engineer a smarter HIV virus that can bypass this initial interaction and infect cells that don't have chemokine receptors?? -- and following this (in a less dangerous manner), could we alter '''p3''' such that the phage can infect F minus cells? surely ALL existing phage do not need to bind to TolA to start infection, so why should M13 be thus limited.
#Limitation (from engineer's perspective):
#*'''(From 2/9 Introduction: Amazingly, new M13 phage particles are secreted within 10 minutes from a newly infected host and can arise at a rate of 1000/cell within the first hour of infection. Also amazing is how the bacterial host can continue to grow and divide, allowing this process to continue indefinitely)'''. This is nice if you are a phage, but for me it would be much cooler if I could control the rate of the phage's replication/infection and then measure/graph this change. If we replaced the natural promoter of key replication protein '''p2''' '''(from 2/9 Introduction: Without p2, no replication of the phage genome can occur)''' with a specific inducible promoter, we could then induce/control the phage for some sort of interesting application in the future. Maybe our chosen inducible promoter turns ON when it senses some environmental condition.
#Other:
#*'''p8''' could be bound to nano-scale inorganic materials to be expressed in large quantities on surface: big challenge = try to control and pattern the display of 2 inorganic materials at same time!'''
#**could we use mutant trna anticodons here? and predict/control the ratio at which they are used in protein translation versus wild-type trna? I know that the attachments are short, but how different are the 2 amino acid sequences necessary for binding GaN and InN?
#*'''p1, p4, p11''' are all involved in creating the transmembrane pore that the phage secretes itself from cell through. It seems to me that we know too little about this actual process and that there are way too many interactions involved for us to mess with this group of assembly proteins (and they don't really interest me, so yeah)
#*'''p3 and p9''' could continue to be used/tested as a hook for other material interactions, etc.

==questions:==
#Would you expect the phage to tolerate transcriptional terminators that are: 
#*100X weaker? '''No.''' Will get dramatically more read-through and therefore '''MIS-REGULATION''' of downstream genes/'''MIS-EXPRESSION''' of downstream protein products.

==M13 evolutionary relatives, and how differ?==
#Lineage: Viruses ; ssDNA viruses ; Inoviridae ; Inovirus
#'''Closest relatives = phage f1 and phage fd'''
#*f1 does not need/have gene 11

==Test for myc-tag insertion==

{| border="1"
! Diagnostic digest 1
| '''plasmid with insert'''
| '''plasmid no insert'''
|-
| Enzyme(s) used 
| BamHI and HindIII
| BamHI and HindIII
|-
| Buffer used
| #2
| #2
|-
| Temperature
| 37 C incubate, 65 C heat inactivate     
| 37 C incubate, 65 C heat inactivate     
|-
| Predicted fragments		
| 8705
| 5182, 3487
|-
| 
|
|
|-
! Diagnostic digest 2	 	
| 
|
|-
| Enzyme(s) used		
| EcoRV, NruI
| EcoRV, NruI
|-
| Buffer used		
| #3
| #3
|-
| Temperature		
| 37 C incubate, 65 C heat inactivate
| 37 C incubate, 65 C heat inactivate
|-
| Predicted fragments		
| 5647, 3022 (+ or - 20)
| 8669
|}

==Ligation/Transformation Results==

{| border="1"
! Reaction:
! # Colonies Expected:
! Blue Team Results:
|-
| '''neg. control:''' cells, no dna
| none
| 0
|-
| '''pos. control:''' 5 ng plasmid
| lots
| 124
|-
| bkb - lig + killcut
| none
| 0
|-
| bkb + lig + killcut
| none
| 0
|-
| bkb + lig + insert + killcut
| colonies
| 2-3
|}


#Transformation Efficiency = 124 cols/5ng  = '''2.5 x 10^4 colonies/ug plasmid dna'''
#Interpretation of Results: Our negative control plate shows that our cells were not contaminated and that our plates did have a selectable resistance. Our positive control plate shows that our cells were competent, but that our transformation was not very efficient. The lack of any colonies on both killcut plates shows that our BamHI killcut cocktail was viable. The yield on our experimental plates (2-3 colonies per plate) was pretty small. They have some chance of being correct, but more colonies would be desirable. Should troubleshoot and refine this ligation/transformation protocol. Our gel also suggests that our colonies are not correct assemblies of insert + backbone.