==SAGA??==
#"SAGA" is an acronym for "Spt-Ada-Gcn5-acetyltransferase". It has 19 identified subunits.
#The SAGA complex is a chromatin modifying complex, specifically a histone modifier, that is involved in all of s. cerevisae's DNA-templated processes --esp. in the transcription of SAGA-dependent genes. Our interest and expectation, therefore, is that the deletion of one of SAGA's 13 non-essential subunits will have an interesting and assayable effect on cell growth and gene expression.

==Sus1?? (reference: Rodriguez-Navarro et al., 2004)==
#Sus1 stands for "'''S'''ynthetic lethal '''u'''pstream of y'''S'''a'''1'''". It is located on Chromosome 2 and contains an intron.
#Sus1 functions in both transcription and in mRNA export from the nucleus...
#*Transcription/interactions with the SAGA complex:
#**'''Sus1 TAP-tagging:''' essentially all of the reported SAGA components were identified in Sus1-TAP eluate (pulldown)
#**'''DNA macroarray: '''showed that Sus1 was required for transcription regulation. ~9% of yeast genes were altered by loss of Sus1. PH084 was most significantly decreased transcript in Sus1 mutant cells (--result is similar to other SAGA subunits).
#**'''Chromatin immunoprecipitation:''' Sus1 associated with the promoter of a SAGA-dependent gene (GAL1) upon transcription activation. Did not associate with middle or end of GAL1 or the promoter of a non SAGA-dependent gene (PMA1) (--result is similar to other SAGA subunits).
#*mRNA export from the nucleus/interactions with the Sac3-Thp1 complex:
#**'''Sus1-GFP localization:''' Sus1 found to have an intranuclear location with a concentration at nuclear periphery. Sus1-GFP was no longer detected at nuclear periphery in Sac3 mutant cells, suggesting that Sus1 '''via Sac3''' can associate with nuclear pore complexes (NPC) at nuclear periphery
#**'''Synthetic lethality tests of sus1Δ with yra1-ΔRRM, sub2-85, and mex67-5: '''Found that Sus1 interacts '''genetically''' with these essential components of the conserved mRNA export machinery
#**'''Sus1 TAP-tagging:''' Thp1 and Sac3 identified in pulldown. Reciprocally, Thp1 was TAP tagged: Sac 3 and Sus1'''-myc''' identified in pulldown. 
#***Note: Thp1 and Sac3 are both required for mRNA export and interact physically and genetically with the general mRNA export receptor Mex67. Sac 3 is also functionally linked to the cell cycle and cell cycle genes.
#**'''In situ poly (A)+ RNA hybridization with Sus1 mutant cells:''' showed a significant defect in mRNA export
#Interesting Question: do these two complexes become physically linked (via Sus1) to funcion in transcription-coupled mRNA export??
#*Sus1 confirmed to exist in both complexes (SAGA and Sac3-Thp1)
#*Gel filtration chromatography and subsequent Western Analysis showed that not only Sus1, but also Thp1 (from Thp1-Sac3 complex) fractionates with the rest of SAGA. Potentially suggestive of a 'supercomplex'
#Interesting, and thus far unexplained finding about Sus1:
#*Cdc31 was also identified in Sus1-TAP pulldown. Cdc31 binds calcium and is involved in the cell division cycle. It is, specifically, a component of the spindle pole body (SPB) half-bridge, required for SPB duplication in mitosis and meiosis II; homolog of mammalian centrin. Cdc31Δ mutants are inviable.

==Sus1?? (reference: Kohler et al., 2006) ==
#Cdc31 confirmed to be part of the Sac3-Thp1-Sus1 complex!! boo :)

==Lab Notes==

#Day 1: Start-up expression engineering --in M&M? '''yes'''
#*Our project goal is to "knock out" the Sus1 gene of S. cerevisae's SAGA complex by "knocking in" a URA3 marker gene in its place. To do this, we have today designed forward and reverse PCR primers to bind to and amplify the URA3 gene. We have also given the primers 39 bp tails with sequence homology to the 39 bp flanking the Sus1 gene in S. cerevisiae, so that when transformed into S. cerevisiae, the PCR product can homologously recombine with the genome to insert URA3 and delete SUS1. 
#Day 2: Yeast transformation --in M&M? '''yes'''
#*Today we transformed our Day 1 PCR product, conveniently, into a URA minus strain of yeast and plated the mixture onto SC-ura plates to prevent the growth of any colonies that did NOT homologously recombine with URA3. If our transformation controls indicate no contamination, any viable colonies on the SC-ura plates are likely to be desired Sus1 "knock outs".
#Day 3: Colony PCR --in M&M? '''yes'''
#*Today we found 10 viable colonies on our SC-ura plates. We performed Colony PCR on 3 of these colonies using a forward primer that anneals upstream of Sus1 in the S. cerevisiae genome and a reverse primer that anneals to a region within the URA3 gene. We expect to observe a ~.8kb band when we run the PCR product out on a gel if URA3 correctly integrated and deleted Sus1. Note: only positive result will be meaningful.
#Day 4: Screen for phenotypes, isolate RNA --in M&M? ''' yes'''
#*Today we first spotted serial dilutions of FY2068 parent yeast cells and 3 samples (A, B, C) of Sus1 mutant cells onto 10 different plates. We expect to observe differences in growth between parental and Sus1 mutant cells on different media types and in different growth conditions (ex: temperature). Also, it is possible that we will microscopically observe altered cell morphology in the Sus1 mutant cells. Secondly, we isolated RNA from Sus1 mutant colony B (confirmed via Colony PCR to have URA3 in place of Sus1) to use next time in cDNA synthesis and microarray.
#Day 5: cDNA synthesis and microarray --in M&M? '''yes'''

==(Sus1)_URA3 Deletion Primer Design==
{| border="1"  
! primer name  
! primer number
! sequence
|-   
| sus1_URA3_KO_fwd 
| NO177
| 5' GCGACAAAATCAGAAGTAACAATTCTGGCCTTCACTCCAatgtcgaaagctacatataa
|-
| sus1_URA3_KO_rev 
| NO178
| 5' TGTAATAATATTGGGAATTAAGGTGCATTTTCGTATCCTttagttttgctggccgcatc
|}

==(Sus1)_URA3 PCR Cycle==
#*95° 4 minutes
#*95° 1 minute
#*40° 1 minute
#*72° 3 minute
#*repeat steps 2-4 5 times
#*95° 1 minute
#*45° 1 minute
#*72° 3 minute
#*repeat steps 6-8 5 times
#*95° 1 minute
#*50° 1 minute
#*72° 3 minute
#*repeat steps 10-12 30 times
#*72° 10 minutes
#*4° forever (or until one of the teaching faculty removes the reactions and stores them in the freezer)

==Yeast Transformation Results==

{| border="1"  
! (+) control: pRS416 DNA template
! (-) control: No DNA template
! URA3 PCR product
|-   
| Lots
| 3
| 10
|}

==Colony PCR Primer Design==
{| border="1"  
! primer name  
! primer number
! sequence
! anneals
|-   
| URA3_220to191_rev 
| NO179
| 5' CTG TGC CCT CCA TGG AAA AAT CAG TCA AGA 
| 191-220 (btm strand) after ATG 
|-
| sus1_minus520_fwd 
| NO193
| 5' AATGGTTAAG ATACCAATGC CGTCTACACC
| 520bp upstream of ATG
|}

==Colony PCR Cycle==	
#*95&deg; 4 minutes
#*95&deg; 1 minute
#*52&deg; 1 minute
#*72&deg; 2 minute
#*repeat steps 2-4 35 times 
#*72&deg; 10 minutes
#*4&deg; forever

==Colony PCR gel results==
[[Image:Gel mod3day4 transform check 1 less.jpg|thumb|center]]

==10 Phenotype screen plates (spotting):==
#YPDextrose 23C
#YPDextrose 30C
#YPDextrose 37C
#YPGalactose 23C
#YPGalactose 30C
#YPGalactose 37C
#SC-Lysine 30C
#SC-Tryptophan 30C
#YPD + Formamide 30C
#YPD + Rapamycin 30C

==RNA isolation (concentration)==

{| border="1"
! RNA Sample
! A260
! Conc of dilute RNA
! Conc of undiluted RNA
|-
| FY2068
| .023
| .92 ug/mL
| 92 ug/mL
|-
| Sus1Δ		
| .0325
| 1.3 ug/mL
| 130 ug/mL
|}

#Note: 40 μg/ml of RNA will give a reading of 1 A260
#Note: Will need to add 10 uL FY2068 RNA and 7 uL Sus1Δ RNA next time (= .92 ug)