{{Smolke_Top}}

<h1><center>Research</center></h1>

__TOC__

==Engineering switch platforms as temporal and spatial cellular biosensors==
''Researchers: Andrew Babiskin, Travis Bayer, Chase Beisel, Stephanie Culler, Katie Galloway, Kevin Hoff, Maung Nyan Win''

We are exploring the design strategies for constructing molecular switches that act in vivo as both biosensors and ligand-controlled regulators of gene expression in bacteria, yeast, and mammalian cell culture. Much of our effort is focused on the design of nucleic acid-based molecular sensors, although the design of some protein-based sensors is being explored as well. In the area of trans-acting molecular switches, we are exploring the design of sensors that act through diverse gene regulation mechanisms such as the RNA interference (RNAi) pathway, ribozyme-based cleavage, and the antisense pathway. In the area of cis-acting molecular switches, we are exploring the design of sensors that act through regulatory mechanisms such as alternative splicing, RNase III cleavage, ribozyme-based cleavage, and internal ribosome entry site (IRES) activity.

In order to effectively monitor information flow through cellular networks, projects are examining different sensor platforms that can provide temporal and spatial information regarding fluctuations in biomolecule levels. These platforms couple molecular recognition of a ligand-binding event to a conformational change in the sensor molecule. This regulated conformational change is linked to an appropriate readout signal, which enables these molecules to act as cellular biosensors. For example, the output from a molecular binding event may be coupled to the regulation of a targeted gene expression event. Therefore, these platforms enable allosteric regulation of the activity of a general gene expression platform, toward a target gene through the binding of a small molecule, protein, or transcript input. Our work on ncRNA design demonstrates the modularity, design predictability, and specificity inherent in these molecules for cellular control. By modifying the input and output modules on these platforms we can achieve user-specified probing and programming of cellular events. By modifying the regulatory modules on these platforms we can construct sensors with different temporal and spatial resolution properties. For instance, projects are exploring other sensor platforms built around the integration of nucleic acid-based switches with protein-based sensors to develop rapidly responsive biosensors that act through fluorescence resonance energy transfer (FRET) signals. Protein-based switches are also being explored that function through nuclear receptor-based transcriptional mechanisms. Molecular design techniques are incorporating a combination of rational design strategies and library-based screens.

[[#Research|Back to Top]]

==Designing cellular information processing capabilities==
''Researchers: Katie Galloway''

We are constructing circuits of interacting molecular switches to engineer complex cellular information processing capabilities. Projects in this area explore the design strategies and parameters necessary for programming higher-level cellular logic. Specifically, efforts are currently focused on the design of different logic gates and filtering circuits composed of molecular switches that regulate the expression of a target gene in response to different combinations of small molecule and protein inputs. Current projects are focused on the construction of AND, OR, and NOR gates, which will later be combined to build more complex information processing capabilities. In addition, filtering circuits are being constructed to process distinct patterns of cellular information. Due to the exhibited modular and programmable nature of these switches, the molecular sensor components comprising these signal integration circuits will be applied to project areas described below in metabolic engineering and cellular programming.

[[#Research|Back to Top]]

==Designing ‘intelligent’ therapeutic molecules==
''Researchers: Chase Beisel, Yvonne Chen, Leo d’Espaux''

We are designing molecular switches to act as targeted or ‘intelligent’ therapeutic molecules. Projects in this area focus on the construction of ligand-regulated RNA-based regulators of gene expression that function in mammalian cells through diverse regulatory mechanisms such as the RNAi pathway or ribozyme-based cleavage. Research areas have been initiated in the design of molecular switches for advancing targeted breast cancer treatments and immunotherapy strategies. In the area of developing next-generation cancer therapies and detection strategies, RNA switches will be constructed to take different hormone and growth factor biomarkers identified for different breast cancers as input signals. In response to the presence of particular set of biomarker indicative of breast cancer, these molecules will regulate the expression of target output genes such as genes involved in regulating cellular behavior (apoptosis, cancer phenotype) or genes associated with a monitorable signal (detection/diagnosis strategies). In the latter area, RNA switches will be constructed to take target small molecules or biomarkers for different tumor cells as input signals. These molecular switches will be engineered into T cells and respond to the presence of these localized inputs by activating the T cell to kill the nearby tumor cells, thereby developing more effective and safe immunotherapy treatments. Cellular engineering projects are currently being conducted in model cell lines and will later effectively be transferred into animal models for these diseases. Both of these projects have translational clinical collaborators at the City of Hope (Professors Carlotta Glackin - breast cancer therapies; Mike Jensen - T cell engineering).

[[#Research|Back to Top]]

==Metabolic engineering of Saccharomyces cerevisiae for alkaloid production==
''Researchers: Kristy Hawkins, Maung Nyan Win''

We are engineering synthetic circuits in yeast for the production of different value-added compounds. Current research efforts are focused on the development of Saccharomyces cerevisiae as a microbial host for the total biosynthesis of diverse alkaloid compounds. Synthetic metabolic pathways are being assembled for the production of two different classes of alkaloids, the purine alkaloids and the benzylisoquinoline alkaloids (BIAs). The purine alkaloid pathway, resulting in the synthesis of caffeine and similar analogs, is being engineered in yeast largely as a model pathway through which to explore general design principles and strategies for integrating molecular switches and assembling signal processing schemes with synthetic metabolic pathways. Strategies will be developed for applying these engineered molecular switches for establishing rapid and generalizable pathway optimization screens and selections. Furthermore, control theory will be used to explore the design parameters for constructing dynamically regulated networks with switch-based control loops as a way to optimize pathway flux. The purine alkaloid pathway enables a more immediate demonstration of these strategies and design principles as aptamers to these metabolites are readily available.

The BIA pathway, resulting in the synthesis of codeine, morphine, and sanguinarine, is being explored for generating a microbial host that can (i) readily synthesize an array of BIA molecules with diverse pharmacological activities and (ii) be used to set up rapid functional genomics screens to effectively identify enzymes that can act on these molecules from EST libraries of native plant hosts. The BIA pathway is particularly appropriate to this type of metabolic engineering effort as they are a complex class of molecules that are not effectively synthesized through traditional chemical means. In addition, there is no source for many of the intermediate metabolites of pharmaceutical interest, as they do not accumulate in the native hosts and genetic engineering efforts remain challenging in plants. As the pathway including the early steps resulting in the synthesis of the BIA backbone (norcoclaurine) has not been entirely elucidated from plant hosts, a synthetic network composed of genes from bacteria, humans, and various plants is being assembled and optimized for BIA production in yeast. It is anticipated that the molecular tools developed in the purine alkaloid pathway will be readily transferable to the BIA pathway. This research effort has a plant biologist collaborator from the University of Calgary (Peter Facchini).

[[#Research|Back to Top]]

==Integration of molecular switches into synthetic and endogenous cellular networks==
''Researchers: Travis Bayer, Katie Galloway, Kristy Hawkins''

We are exploring the integration of different molecular switch platforms into synthetic and endogenous cellular networks. These research efforts focus on transferring from researcher-controlled model inputs (for which sensor domains have previously been developed and characterized) to target metabolite, enzyme, or transcript inputs. Outputs that regulate reporter gene expression will enable these molecular switches to serve as cellular biosensors, providing data regarding information flow or flux through networks of interest. Outputs that regulate target enzyme levels will be connected to two different types of inputs. Inputs that are directly associated with the pathway of interest, such as precursors or toxic intermediates in a synthetic pathway, will enable dynamic control schemes. Inputs not associated with a pathway output will be used to generate synthetic connections between naturally unconnected pathways. In the area of synthetic networks, efforts are focused on integrating switches as both real-time cellular biosensors and dynamic regulators of gene expression with the purine alkaloid and BIA pathways. In the area of endogenous networks, efforts are focused on integrating switches into signal transduction pathways, pathways associated with cell cycle and apoptosis, and pathways associated with yeast mating type and differentiated cellular state.

[[#Research|Back to Top]]

==Nucleic acid switches as molecular recognition components of nanosensor devices==
''Researchers: Arwen Brown''

We are constructing nucleic acid-based switches that will function as the molecular recognition components of different nanosensor devices. These types of switches may provide an alternative to antibody-based molecular sensors in that they are more stable, easier to generate to diverse analytes of interest, and more amenable to engineering strategies for integration into different detection platforms. Research efforts have developed an amplifiable, solution-based detection system, where the conformational change in the molecular sensor upon analyte binding is coupled to an amplification event. The output from this sensing event may be integrated into existing microarray detection technologies for high-throughput, multiplexed identification and quantification of small molecules and proteins on a single platform. Future research efforts will explore the integration of this amplifiable molecular detection system with microfluidic devices. In addition, research efforts are exploring the integration of these molecules with nanoelectromechanical systems (NEMS) and nanowires. In the former platform, the device will convert the mechanical input signal (realized as a change in conformational state upon binding analyte) into an electrical signal, whereas in the latter platform, the device will convert the charge input signal (realized as binding of the switch molecule to the functionalized surface in the ‘ON’ state) into an electrical signal. Research efforts are focused on establishing the design principles to optimize the mechanical or charge input signal upon switching of the sensor from a bound to unbound state.

[[#Research|Back to Top]]

==Rapid generation and characterization of switches responsive to novel molecular inputs==
''Researchers: Arwen Brown, Maung Nyan Win''

We are developing methods to rapidly generate and characterize new molecular switches in a high-throughput manner. The sensor domains of the molecular switches are aptamers or nucleic acid molecules able to bind ligands with high specificity and affinity. In vitro and in vivo methods are being developed to rapidly select for new sensor domains within the switch platform. In vitro methods focus on the development of novel capillary electrophoresis (CE)-based methods for selecting small molecule- and protein-responsive switches. CE-based methods are extremely effective at partitioning bound from unbound species, enabling selection of novel sensor domains in fewer cycles and with greater control over binding properties, such as equilibrium binding and kinetic rate constants, than commonly employed affinity-based techniques. In addition, by selecting sensor domains within the switch platform (and therefore selecting indirectly for activity through conformational changes) small molecule sensor domains will be generated with this technique. In vivo methods are focused on the development of bacterial and yeast library screening systems. In these systems functional sensor domains are generated by assessing the activity of a switch indirectly through its ability to regulate the expression of a fluorescent protein in the presence and absence of the desired ligand molecule. This is a generalizable screening system, as many of the engineered switch platforms are transferable between bacteria, yeast, and mammalian cells and the sensor domains within these platforms are modular and swappable between platforms. Furthermore, small molecule and protein-responsive switches, as well as switches responsive to post-translational modification states and transcript levels, will be rapidly generated through the in vivo screening strategies. Finally, high-throughput surface plasmon resonance-based methods have been developed to precisely and robustly characterize the equilibrium and kinetic binding properties of these sensor domains either as aptamers or directly as switches.

[[#Research|Back to Top]]