{| border="0" cellpadding="10" cellspacing="10" align="left" width=800px 
{| border="0" colspan="2" rowspan="3" align="left" valign="top" height="200px"
| align="center" colspan="2" | [[Image:Model2000.gif|400px]]
|-
| align="center" | [[Image:Isotope effects.jpg|200px]]
| align="center" | [[Image:GCN4 bHxlink.jpg|150px]]
|}
{|
!border="0" colspan="2" cellpadding="10" align="left" valign="top" | <font color=#000000 size=2> 


The over-all goal of my research program is to understand how complex biological molecules adopt their functional, 3-dimensional conformations: The Protein and RNA Folding Problem. These synergistic studies, both experimental and computational, are based on the premise that rigorous and innovative studies of basic processes have broad implications in many areas of biological research. The folding process has considerable biological significance while posing a tremendous intellectual challenge. Beyond aiding in the prediction of structure from sequence, the relevance of folding studies is underscored by the involvement of misfolded conformers in a variety of human diseases and the role of “natively-unfolded” proteins in regulation, recognition, and binding. From the RNA standpoint, recent discoveries of the diverse roles structural RNAs play in gene expression, such as riboswitches, illustrate the importance of understanding their dynamics and folding as well. Other classes of non-coding RNAs are likely to be found with new regulatory roles. 

In the past 7 years, we have learned to direct folding and dynamics of these biopolymers. We have developed [[Sosnick:Psi-analysis| &Psi;-analysis ]], a widely applicable method that identifies structures along folding routes. The method utilizes designed metal binding sites in conjunction with a new theory to analyze the accompanying non-linear Br&oslash;nsted behavior. It has provided one of the most accurate descriptions of a protein folding pathway yet obtained. In RNA folding, we generated atomic level models of folding intermediates and revealed the principles of tertiary RNA stability using homologous RNAs from mesophilic and thermophilic organisms. Our studies have led to the concept of thermodynamic and kinetic hotspots. These fundamental studies have helped us examine how non-coding RNAs may fold in the cell. We found that species-specific pausing during RNA transcription improves the folding behavior, thereby suggesting that RNA sequences have co-evolved with the pausing properties of their cognate RNA polymerases. 

In addition to continuing the experimental protein and RNA folding studies, new research areas in the Sosnick lab involve the rational design of proteins and RNAs to control their structure and function, and the expansion of the computational component. We have developed a method to “fuse” two proteins together in order to create a single light-triggered allosteric repressor protein. We are also predicting protein structures by simulating folding pathways, taking advantage of our experimental results and an approach that dynamically integrates local and global information. In addition, we are refining crystal structures and homology models using our new move-set and statistical potentials. With Wah Chiu (Baylor), we recently have pushed the molecular weight limit for cryoEM studies down by 5-fold. This work opens up a whole new avenue of structural studies on non-coding RNAs. Our long-term goal is to create residue-level structural models using the cryoEM images, sequence information, and all-atom simulations, building on our modeling studies of RNA folding intermediates. 

Graduate students in my lab enter through the new  [http://biophysics.uchicago.edu/ Biophysical Sciences Graduate Program], in addition to the Programs in [http://bmb.bsd.uchicago.edu/Students/01_Graduate_Programs/ Biochemistry & Molecular Biology], [http://chemistry.uchicago.edu/ Chemistry], and [http://physics.uchicago.edu/ Physics].
</font>
|-
| style="height:10px" |
|-
!border="0" colspan="2" cellpadding="10" align="left" valign="top" |[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=PureSearch&db=PubMed&details_term=Sosnick%20TR%5BAuthor%5D  <font color=#000000 size=2> Do a PubMed search for "Sosnick TR." </font>]
|}
|}



{{SosnickLab}}