Questions 1 and 2 are theoretical but they should help prepare you to interpret the results you will collect next time. 

<b>1.</b> You have purchased some supercompetent bacteria that are provided at a transformation efficiency of 10<sup>9</sup> colony forming units/ug of DNA. You transform the cells with 1 ng of plasmid DNA and plate 1/1000th of the cells. How many colonies do you expect? 

(1 ng DNA)(1e9 colonies/ug)(1 ug/1000 ng)(1/1000) = 1000 colonies.  

Next you transform another aliquot of cells, also at 10<sup>9</sup> colony forming units/ug of DNA, with 2 &mu;l of plasmid DNA. You spread 1/100th of the cells and find 50 colonies growing on the plate after 24 hours at 37&deg;C. What is the concentration of plasmid? 

(50 colonies)(1 ug DNA/1e9 colonies)(100/2 ul) = 2.5e-6 ug/ul.

<b>2.</b> To illustrate your understanding and the importance of the controls you performed today, please write a one-sentence interpretation for each of the following transformation outcomes. 

*'''Outcomes:'''
**Outcome 1: no colonies on any plate. Lack on function of the Kanamycin resistance gene.<br>
**Outcome 2: thousands of colonies on all the plates. Kanamycin not selecting properly, so that cells without the plasmid are not being killed off<br>
**Outcome 3: approximately the same number of colonies on the backbone+ligase+kill cut as the backbone+insert+ligase+kill cut. The kill cut did not function as the backbone+ligase+kill colonies stayed alive which means they were transformed by an intact plasmid.

<b>3.</b> Next time you will isolate DNA from four transformants and begin to characterize the plasmids in these bacteria. To prepare for this experiment, you should draw a plasmid map of the M13K07 genome. Start by printing out the M13K07 plasmid map from NEB by using their [http://tools.neb.com/NEBcutter2/index.php NEB Cutter tool], selecting M13K07 from the "Viral and phage" drop down menu on the right, changing the default minimum ORF to 25 amino acids (do you remember which of the M13 proteins are very small?), and finally telling the program that you are entering circular DNA. Modify the map by hand to indicate which restriction site you are changing, which enzymes you are adding, and how many basepairs of DNA this modification needs. Next, use the plasmid map to help you plan at least two restriction digests that will confirm the presence of the oligonucleotide insert. Recall that the lab does not have every enzyme available so you should double check your idea against the [[20.109(F07): Available enzymes | list of available enzymes]]. It will help to read the introduction for the next lab before you complete this part of the assignment. Be sure to predict the size of the fragments you expect when the plasmid does and doesnâ€™t have the oligonucleotide insert. Also include reaction conditions such as buffer and temperature. Use the [http://www.neb.com/nebecomm/default.asp NEB website] for details on various enzymes and reaction conditions.
<center>

{| border="1"
! Diagnostic digest 1
| plasmid with insert
| plasmid no insert
|-
| Enzyme(s) used 
| BamHI
| BamHI
|-
| Buffer used
| 1xNEB Buffer 3
| 1xNEB Buffer 3
|-
| Temperature
| 37 degees C
| 37 degees C
|-
| Predicted fragments		
| Plasmid, supercoiled
| 8,600 bp fragment
|-
| 
|
|
|-
! Diagnostic digest 2	 	
|
|
|-
| Enzyme(s) used		
| EcoRI
| EcoRI
|-
| Buffer used		
| 1xNEB Buffer EcoRI
| 1xNEB Buffer EcoRI
|-
| Temperature
| 37 degees C
| 37 degees C
|-
| Predicted fragments		
| 8,600 bp fragment
| Plasmid, supercoiled
|}
</center>
<br>
<b>4.</b> Based on the results of your plaque assay, what is the titer of each stock solution of phage? Please show your work. If the plaques appeared different, please consider how the phage genomes differ (M13K07 is a "helper phage" while E4 is identical the the M13 genome except four glutamic acids are presented on the N-terminus of the p8 protein) and suggest how these differences might account for the differences in plaque morphology. <br>

K07: (80 pfu)(10e6)(200/10) = 1.6e9 pfu in the original stock

E4: (783 pfu)(10e6)(200/10) = 1.6e10 pfu in the original stock

M13K07 is the more artificial genome while E4 is practically wild type phage. This makes E4 more efficient because it was guided by many, many years of evolution, while M1307 has added parts which detract from its efficiency as a phage in infecting bacteria and reproducing. This explains why the E4 appeared to infect Ecoli more succesfully than K07 on the plates, leading to larger infection areas for E4 phage for a particular colony of phage.