==Aim==

This project intends to build a standard strain for use with [http://parts.mit.edu BioBricks] in which most systems can be run.

We will be making a series of mutations from wildtype K12 ''Escherichia coli'' that need to be fully determined and prioritized.

==Motivation==

Even some of the simple devices I work with appear to behave differently in different strains.  Therefore, the strain in which your systems operates is yet another variable that can affect system performance.  Ideally, we should be able to design our systems so that they are independent of the strain in which they operate but currently we are not yet there.  Thus, this project seeks to build a strain in which we can characterize and test most devices in the hopes that standardization of background will solve some of these observed problems.

==Approach==

The key feature of this project is that it is likely of use to several people but yet a lot of work for any one person to tackle.  Add to these features the fact that most people can scrape along by using existing strains and you have a project that everyone would like to see done but that no one wants to commit to.  I hope to use this wiki page to achieve a detailed project plan.  Then perhaps several people can work on this project in the background and update this page with project status in order to keep it going forward.  It is unclear at this point whether this approach is at all feasible.

==Plan of action==
''Note that this is a work in progress''

#Begin with MG1655 - a fully sequenced ''Escherichia coli'' strain.
#Identify desired list of deletions, insertions and mutations and their order of priority.
#* no lambda lysogen (we use lambda cI)
#* no F plasmid (we want to be able to use F ori (cosmid)  plasmids)
#**I believe these first two steps are already completed in MG1665 (see note in the discussion)
#* no lacY
#* no hsdR restriction system (we don't want our DNA cut up randomly on transformation)
#* Delete the methylation specific restriction systems encoded by the mcrA, mcrBC and mrr loci.
#* appropriate mutation of arabinose operon/transport (see Keasling's paper)
#* deletion of lac operon
#* delete endA (as a cloning strain)
#Construct kanamycin resistance gene with constitutive promoter in BioBricks form.
#*8/16/2006: This cassette is being synthesized as a part of the [[Synthetic Biology:Vectors/Parts/Synthesis order|standard BioBricks vector synthesis order]].
#Construct pheS(mut) gene with constitutive promoter in BioBricks form.
#*pheS(mut) can be obtained from plasmid pKSS (Genbank accession U01668).
#*pheS(mut) encodes "a phenylalanyl-tRNA synthetase &alpha; subunit with relaxed substrate specificity"  due to A294G mutation (Kast 1994).  Cells with this gene will not grow on plates with p-chlorophenylalanine (D,L-''p''-Cl-Phe).
#*This selection marker is independent of host strain since high copy number pheS(mut) gene is dominant lethal over chromosomal wildtype pheS (Kast 1994).
#Assemble kanamycin resistance part with pheS(mut) part.
#Select the highest priority genome change to do.
#Isolate left and right flanking regions around the point of insertion/deletion as insertion sites.  These need only be 36-50 bp with 20 bp of priming sequence complementary to the KanR.pheS(mut) cassette (Datsenko 2000) or 30-50 bp of homology with greater than 40 bp being optimal (Yu 2000).
#Construct a linear DNA fragment with left.kanR.pheS(mut).right where kanR and pheS(mut) genes have constitutive promoters.
#*This construction can be done via PCR.  The PCR product should be subsequently treated with DpnI to eliminate plasmid template DNA.  Alternatively, linear template DNA may be prepared via digestion and gel purification to minimize the likelihood of plasmid transformation into the target strain.
#The cells into which the linear DNA is transformed must be induced to express the &lambda; recombination genes ''exo'', ''bet'' and ''gam''.  This can be done by transforming the cells with pKD46 which has these 3 genes under the control of an arabinose-derived promoter.  This plasmid may be cured from the cells by growing the cells at 43&deg;C.  However, Court's lab reports that plasmid based expression of Exo and Beta is not as efficient as chromosomal based expression possibly because it is more difficult to regulate expression of these genes at higher copy (Yu 2000) but Wanner's lab used this plasmid successfully (Datsenko 2000).
#Transform the &lambda; Red cells with the linear DNA and select with kanamycin, giving cells with chromosomal insertion of the cassette (and deletion of any intervening region).
#*Gam inhibits RecBCD nuclease from degrading linear DNA (Yu 2000).  Exo and Beta are responsible for the recombination activity for the linear DNA.
#*Recombination efficiency is reduced 10 fold in ''recA<sup>-</sup>'' cells.
#*If the kanR.pheS(mut) cassette was amplified from a plasmid with another antibiotic resistance gene, then transformed cells can be checked for the absence of resistance to the other antibiotic via replica plating.
#*300ng or more of linear 1kbp DNA was found to be optimal for recombination which is equivalent to about 300 molecules per cell (Yu 2000).
#Construct a linear DNA fragment with left.insert.right  for any (or null) insert.
#Transform result of kanamycin insertion with this fragment, select on p-chlorophenylalanine plates, selecting for removal of the cassette.
#*This step is referred to as positive selection (Kast 1994).
#*Some background growth may occur on LB agar plates.  This can be eliminated by using either minimal media (with a concentration of 1.1mM racemic ''p''-Cl-Phe) or omitting tryptone from the LB such that the media becomes YEG-Cl agar (0.5% yeast extract, 1% NaCl, 0.4% glucose, 1.5% agar, 10 mM ''p''-Cl-Phe, 200 &mu;g Ap per mL) (Kast 1994).
#Cure the &lambda; Red plasmid by growing the cells at 43&deg;C.  Check for ampicillin sensitivity to ensure that the plasmid has been cured from the strain.
#PCR across the gap and sequence the PCR product to verify the edit.  You can use the same primers for the PCR and the sequencing steps.

==Current status==
''Note: the status of these steps needs to be verified.''
*Tom has in the freezer multiple copies of first-generation MG1655 duplicates from the ATCC stock.
*Tom also has ATCC stock (currently ungrown) of E. coli with plasmid pKSS, carrying the mutant pheS gene described in the paper Kast94 (see below).
*Tom has sigma p-chlorophenylalanine in the freezer.
*Bram took the cassette that the Sauer Lab uses for selection during chromosomal recombination.  Tried to biobrick the markers.  Only got KanR in the end.  However, there were problems b/c kanR has a PstI cut site within it.  Ended up using an Eco/Spe insertion to get it in.  The sequence is in the attached file.
*To insert PheS into the KanR plasmid using BioBricks restriction enzymes, PheS needed to be PCR'd out of pKSS using a forward primer with the  BioBricks [http://rosalind.csail.mit.edu/r/parts/htdocs/Assembly/rbs_cds.cgi prefix] on the 5' end.
*These primers were ordered by Bram and Caitlin.
*As the location of the PheS promoter was unclear, the left primer (PheS_Extended) was designed to bind quite far upstream of the ORF.
*The PCR product could then be cut using either Xba1 or Xba1 and Spe1.  The former would lead to a non-directional ligation of PheS into the KanR plasmid.
*Initial attempts to PCR PheS from pKSS using the existing primers failed both in Bram and Barry's hands.  Two different annealing temperatures were used but no other experimental conditions were changed.
*Sri gave Tom the main Wanner plasmid already (pKD46).
*All parts, primers and documentation from Bram's work are currently in Barry's possession.

==Notes==

===Email correspondence===
Below I have included some email correspondence concerning the status of this project so that all the information is in one place.  I'll be organizing this information further as I try to more definitely outline what all steps need to be done.  Feel free to contribute as well. These emails are in chronological order from earliest to latest.

'''Tom:'''

I have in the freezer multiple copies of first-generation MG1655 duplicates from the ATCC stock.

I also have ATCC stock (currently ungrown) of E. coli with plasmid pKSS, carrying the mutant pheS gene described in the paper Kast94 (see below).

I also have sigma p-chlorophenylalanine in the freezer.

There is a set of other relevant papers in this directory:
[link not included cause it contains copyrighted papers]

The literature confusingly (to me) uses the words "positive selection" to mean what I think of as negative selection -- namely, selection of mutants that *do not* express some protein.  Apparently the idea is that when you clone something (the positive?) into them, then the gene is no longer functional.  In any case, the paper Young-Jun02 has a review of some "positive selection" markers.

The sequence for pKSS is in Genbank U01668.

'''Tom:'''

The Reyrat98 paper basically describes the approach I think we want to make to chromosome editing, except by doing it with the Wanner approach, we eliminate the plasmid ori integration.
Johnston99 uses a plasmid integration approach with N. gonorrhoeae, using a mutant version of rpsL which conveys  dominant sensitivity to streptomycin, in the same way that pheS(mut) conveys dominant sensitivity to p-chlorophenylalanine.

So, in summary the approach looks like this:

#Isolate left and right flanking regions around the point of insertion/deletion as insertion sites.  These need only be 35-50 bp if I recall correctly.
#Construct a linear DNA fragment (pcr?) with left.kanR.pheS(mut).right where kanR and pheS(mut) genes have constitutive promoters.
#Transform and select with kanamycin, giving cells with chromosomal insertion of the cassette (and deletion of any intervening region).
#Construct a linear DNA fragment with left.insert.right  for any (or null) insert.
#Transform result of (3) with this fragment, select on p-chlorophenylalanine plates, selecting for removal of the cassette.
#PCR across the gap and sequence verify edit.

Saving some of the strains from (3) in the freezer would allow us to easily insert constructs into the genome at that spot in one step.

Strain selection:

I dug out the Yanisch-Perron/Messing paper on the JM1xx strains.  They really a hodge-podge -- a result of mating experiments with unclear and confusing parentage.  But the paper is worth a look to see what features they were aiming for.

I also scanned the Doug Hanahan paper describing the DHxx strains as Hanahan83.

These are the things I think we clearly want:

* no lambda lysogen (we use lambda cI)
* no F plasmid (we want to be able to use F ori (cosmid)  plasmids)
* no lacY
* no hsdR restriction system (we don't want our DNA cut up randomly on transformation)
* appropriate mutation of ARA operon/transport 

I'm not sure I know how to cure the F plasmid, but I'm sure it's in the literature somewhere.

There is a note in Yanish-Perron85 that recA+ strains cause plasmids to form multimers.
This is interesting.  I'm not sure we know how to detect these, or if it just looks like a larger copy count.

'''Tom:'''

I'm ready to move on to trying to chromosome engineer MG1655 with pKSS.  Can you bring me up to speed on where you are with recombination, the strains for recombination?  Can you remind me of the edits we initially proposed before Christmas for MG1655?
I think I heard someone say that we had biobricked some antibiotic cassettes ?  I'm looking for Kan or Cm.

'''Sri's response:'''

There wasn't much success with the work that was attempted. But here is what was done mostly by Bram (a new IAP UROP) working with Caitlin and myself. As Bram is no longer with us, and Caitlin is away, let me know if you need any of these raw ingredients, including primers/plasmids/strains.  

Selection cassettes:

We took the cassette that the Sauer Lab uses for selection during chromosomal recombination.  Tried to biobrick the markers.  Only got KanR in the end.  However, there were problems b/c kanR has a PstI cut site within it.  Ended up using an Eco/Spe insertion to get it in.  The sequence is in the attached file.

pKSS:

designed primers for amplification of pKSS using published sequence in genbank (LOCUS:U01668   gi:405986)

two sets of forward primers, w/ and w/o possible promoter driving expression

PheS forward

TTC CGA ATT CGC GGC  CGC TTC TAG AGA TGT CAC ATC TCG CAG AAC TG

PheS forward w/ possible promoter

TTC CGA ATT CGC GGC CGC TTC TAG AGC ACG ACA GGT TTC CCG AC

PheS reverse

TTC CAC TAG TAT TAT TAT TTA AAC TGT TTG AGG AAA CG

PCR didn't work in Bram's hands.  

Recombineering

Haven't done any recently, but we have all the requisite materials.  I think I gave you the main Wanner plasmid already (pKD46).  
I think lambda red genes on this plasmid are controlled by arabinose induction if my memory serves me correctly.  the plasmid is also temperature sensitive.  
I would look up the recombineering papers by Don Court (PMID: 12429697) and the paper by Datsenko and Wanner (PMID: 10829079)

Edits to MG1655

*First is perhaps a deletion of lac operon
*delete endA (as a cloning strain)
*make some of the edits needed for arabinose linearity (see Keasling's paper)
*can't remember others off the top of my head.

'''Reshma's response:'''

I don't have the antibiotics biobricked but I have primers to amplify Kan and Cm with flanking XhoI sites.  It includes a constitutive promoter.  (Though I don't know anything about this promoter except that it works in pSB1AK3-1 ... a high copy plasmid).  It never had a PstI cut site in it (it used to have an Xho I site in it, but that has been mutated out).

===Comments from others===
'''Kathleen's 2 cents'''

Chris Farrell in the Sauer lab would be a great person to talk to about all of this. He has knocked out numerous genes with the Wanner method, is currently using the PheS selection system, and has removed lacY to allow for dose-responsive induction of lac promoters.

In general, I think our lab uses a modified Wanner method for knock-outs. We typically have 300 base pair recombination regions on either side of the antibiotic resistance marker (generated by PCR). Also, we typically clone these pieces upstream and downstream of the antibiotic resistance marker and then cut the entire region out of the plasmid for transformation into ''E. coli''. Although it takes more steps, this seems to work better than trying to transform a PCR product. May have something to do with methylation, etc. Hope this helps, and be sure to contact Chris for details.

==References==

This list of references was compiled by Tom.  I have electronic copies of all of these if you need them.

#K. A. Datsenko and B. L. Wanner. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. ''Proc Natl Acad Sci USA'', 97(12):6640–5, 2000.
#H. M. Ellis, D. Yu, T. DiTizio, and D. L. Court. High efficiency mutagenesis, repair, and engineering of chromosomal DNA using single-stranded oligonucleotides. ''Proc Natl Acad Sci USA'', 98(12):6742–6, 2001.
#J. Gal, S. Szekeres, R. Schnell, S. Pongor, A. Simoncsits, and M. Kalman. A positive selection cloning system based on the gltS gene of Escherichia coli. ''Anal Biochem'', 266(2):235–8, 1999.
#I. Golovliov, V. Baranov, Z. Krocova, H. Kovarova, and A. Sjostedt. An attenuated strain of the facultative intracellular bacterium francisella tularensis can escape the phagosome of monocytic cells. ''Infect Immun'', 71(10):5940–50, 2003.
#A. Haldimann and B. L. Wanner. Conditional-replication, integration, excision, and retrieval plasmid-host systems for gene structure-function studies of bacteria. ''J Bacteriol'', 183(21):6384–93, 2001.
#D. Hanahan. Studies on transformation of ''Escherichia coli'' with plasmids. ''J Mol Biol'', 166(4):557–80, 1983.
#P. Higgins, D. Court, C. McGill, and K. Scheirer. Recombineering clinic, 2001.
#M. Ibba, P. Kast, and H. Hennecke. Substrate specificity is determined by amino acid binding pocket size in ''Escherichia coli'' phenylalanyl-tRNA synthetase. ''Biochemistry'', 33(23):7107–12, 1994.
#D. M. Johnston and J. G. Cannon. Construction of mutant strains of Neisseria gonorrhoeae lacking new antibiotic resistance markers using a two gene cassette with positive and negative selection. ''Gene'', 236(1):179–84, 1999.
#P. Kast and H. Hennecke. Amino acid substrate specificity of ''Escherichia coli'' phenylalanyl-tRNA synthetase altered by distinct mutations. ''J Mol Biol'', 222(1):99–124, 1991.
#P. Kast. pKSS–a second-generation general purpose cloning vector for efficient positive selection of recombinant clones. ''Gene'', 138(1-2):109–14, 1994.
#A. Khlebnikov, K. A. Datsenko, T. Skaug, B. L. Wanner, and J. D. Keasling. Homogeneous expression of the P promoter in ''Escherichia coli'' by constitutive expression of the low-affinity high-capacity AraE transporter. ''Microbiology'', 147(Pt 12):3241–7, 2001.
#A. Martinez, S. J. Kolvek, C. L. T. Yip, J. Hopke, K. A. Brown, I. A. MacNeil, and M. S. Osburne. Genetically modified bacterial strains and novel bacterial artificial chromosome shuttle vectors for constructing environmental libraries and detecting heterologous natural products in multiple expression hosts. ''Appl Environ Microbiol'', 70(4):2452–63, 2004.
#C. Matsumoto-Mashimo, A.-M. Guerout, and D. Mazel. A new family of conditional replicating plasmids and their cognate ''Escherichia coli'' host strains. ''Res Microbiol'', 155(6):455–61, 2004.
#N. Philippe, J.-P. Alcaraz, E. Coursange, J. Geiselmann, and D. Schneider. Improvement of pCVD442, a suicide plasmid for gene allele exchange in bacteria. ''Plasmid'', 51(3):246–55, 2004.
#J. M. Reyrat, V. Pelicic, B. Gicquel, and R. Rappuoli. Counterselectable markers: untapped tools for bacterial genetics and pathogenesis. ''Infect Immun'', 66(9):4011–7, 1998.
#D. Schlieper, B. von Wilcken-Bergmann, M. Schmidt, H. Sobek, and B. Muller-Hill. A positive selection vector for cloning of long polymerase chain reaction fragments based on a lethal mutant of the crp gene of ''Escherichia coli''. ''Anal Biochem'', 257(2):203–9, 1998.
#N. Sharma, R. Furter, P. Kast, and D. A. Tirrell. Efficient introduction of aryl bromide functionality into proteins ''in vivo''. ''FEBS Lett'', 467(1):37–40, 2000.
#M. Xia, R. D. Lunsford, D. McDevitt, and S. Iordanescu. Rapid method for the identification of essential genes in ''Staphylococcus aureus''. ''Plasmid'', 42(2):144–9, 1999.
#C. Yanisch-Perron, J. Vieira, and J. Messing. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. ''Gene'', 33(1):103–19, 1985.
#S. Yazynin, H. Lange, T. Mokros, S. Deyev, and H. Lemke. A new phagemid vector for positive selection of recombinants based on a conditionally lethal barnase gene. ''FEBS Lett'', 452(3):351–4, 1999.
#C. Young-Jun, W. Tsung-Tsan, and B. H. Lee. Positive selection vectors. ''Critical Reviews in Biotechnology'', 22(3):225–244, 2002.
#D. Yu, H. M. Ellis, E. C. Lee, N. A. Jenkins, N. G. Copeland, and D. L. Court. An efficient recombination system for chromosome engineering in ''Escherichia coli''. ''Proc Natl Acad Sci USA'', 97(11):5978–83, 2000.
#D. Yu, J. A. Sawitzke, H. Ellis, and D. L. Court. Recombineering with overlapping single-stranded DNA oligonucleotides: testing a recombination intermediate. ''Proc Natl Acad Sci USA'', 100(12):7207–12, 2003.
#United States Application 20030138937:  Bacteria with reduced genome [http://www.freepatentsonline.com/20030138937.html html]
[[Category:Project]]
[[Category:Escherichia coli]]