==Protocols - Conjugation==
<!--INSERT THE TEMPLATE HEADER-->
{{Streptomyces}}
{|cellspacing="5" cellpadding="10" style="background:#000; width: 800px;"
|-valign="top"
|style="background:#ffffff"|
<!--THESE CATEGORY LINES ADD THE PROTOCOL TO THE OWW PROTOCOLS PAGE-->
[[Category:Protocol]]
[[Category:Streptomyces]]
<!--CONTENT START-->

==Intergeneric Conjugation and Overlay==

'''Description'''
<br/>
Transfer of plasmid/cosmid DNA from a host strain, e.g. E.coli ET12567 [pUZ8002], to the recipient strain, e.g. Streptomyces coelicolor.
{| cellspacing="0" cellpadding="0" 
|-
| style="width: 150px;"| Approx. Duration: || style="width: 175px;"| || style="width: 75px;"| ||
|-
| || Conjugation preparation || ~30  minutes
|-
| || Preparing the host cells  || ~25 minutes
|-
| || Preparing the recipient cells || ~15 minutes
|- 
| || Plating the conjugant mix || ~20 minutes
|-
| || Whole Conjugation || ~1  hour 30 minutes
|-
| || colspan="2" | 
----
|- 
| || Overlay preparation || ~5  minutes
|-
| || Overlaying || ~10 minutes
|}


'''Uses'''
<br/>
Transfer a construct (cloned plasmid/cosmid) into Streptomyces to knock-out, knock-in, complement or over-express a gene.


'''Prerequisites'''
<br/>
You need to know if the Streptomyces species you are using has a methyl-specific restriction mechanism. If this is the case, intergeneric conjugation into Streptomyces will require the use of the E.coli strain ET12567 [pUZ8002]; otherwise DH5α [pUZ8002] may be used. You should also know the antibiotic resistance markers that the E.coli strain, construct and Streptomyces strain confer.


'''Safety'''
<br/>
General laboratory & molecular microbiology safety rules apply.


'''Requirements'''
<br/>
LB/LB-NaCl, SFM+MgCl<sub>2</sub>, 2xYT, sterile distilled water (sdH<sub>2</sub>O), 15mL falcon tube(s), micro-centrifuge tube(s) (MCT), heating block @ 50°C, incubator @ 30°C, falcon centrifuge, micro-centrifuge, vortexer, Antibiotic(s). 


'''Preparation'''
<br/>
Pour 4 plates per conjugation (plus parallels if necessary) of SFM+MgCl<sub>2</sub> (10mM final) and dry for ~1 hour. Place the LB/LB-NaCl on ice. Set a heating block to 50°C. Label all tubes and plates to be used.


'''Method'''
<br/>
All parts are to be performed under sterile conditions, i.e. a Bunsen burner/laminar flow hood. Spins should be done at the maximum capacity of the centrifuge/tubes. Before plating the conjugation, each MCT should be thoroughly mixed by vortexing. 
<br/>
Making the E.coli conjugationally competent: 
# After sub-culturing decant the 10mL culture into a 15mL falcon tube. 
# Using the appropriate centrifuge in the cold room, spin the cells at 5000g for 3 minutes. 
# Decant the supernatant and re-suspend the pellet in 1mL ice cold LB/LB-NaCl, then top-up to 10mL.
# Repeat steps 2-3-2 in this order then progress to step 5.
# Re-suspend the pellet in 1mL LB/LB-NaCl.
# Place the cell suspension on ice.
<br/>
Synchronising the germination of Streptomyces spores: 
# Thaw the spore prep, on ice.
# In a laminar flow hood, add 500µL 2xYT to a MCT, to this transfer 5µL of spores.
# Use a heating block to incubate the spore mix @ 50°C for 10 minutes.
<br/>
Initiating conjugation:
# Add 500µL of the relevant E.coli cells to the pre-germinated spores and mix by inversion.
# Centrifuge the cells for 2 minutes and decant the supernatant. 
# Re-suspend the pellet in the remaining fluid so that the final volume is 50µL
<br/>
Setting up serial dilutions:
# Pipette 270µL sdH<sub>2</sub>O into each of 4 MCTs.
# Label the first – 10-1, second 10-2, etc…
# Transfer 30µL of the conjugation mix to the 10-1 MCT.
# Mix well by vortexing.
# Immediately transfer 30µL from 10-1 MCT to the 10-2 MCT.
# Repeat steps 4 & 5, continuing the serial dilution for the 10-3 and 10-4 MCTs.
<br/>
Plating:
# Pipette 100µL of each serial dilution onto an SFM+MgCl<sub>2</sub> plate.
# Use a glass spreader in a circular motion to evenly spread the conjugation mix over the plate. 
# Allow the liquid to soak into the plate before transferring to an incubator @ 30°C.
# Plate parallels if necessary – see notes.
<br/>
*Overlay:
**After an overnight incubation, the conjugation plates need to be overlaid with antibiotics, to select for successful exconjugants. Typically, the antibiotics used are Nalidixic Acid and whatever antibiotic the conjugated construct confers resistance to; normally Apramycin. 
** Create a 500µL solution per plate using sdH<sub>2</sub>O and the usual concentrations of antibiotics.
<br/>
<br/>
To a suitable sized falcon, add:
# The volumes of antibiotics required to overlay all the plates +1.
# sdH<sub>2</sub>O. Volume to add = (No of plates +1 * 500µL) – Volume of antibiotics.
# Mix solution well.
# Pipette 500µL onto a plate. Using a sterile ‘L’ shaped spreader, very lightly, barely touching the plate; spread the liquid over the whole surface.
# Replace the plates in the incubator @ 30°C for 3-4 more days.


'''Notes'''
<br/>
Conjugation of a plasmid or cosmid into Streptomyces coelicolor is restricted by two main factors. Firstly, intergeneric conjugation can only be achieved if the vector contains the Streptomyces origin of replication – oriT. Secondly, the construct must be transformed into the E.coli strain ET12567 [pUZ8002]. This is due to methyl-specific restriction mechanisms that some Streptomyces use to protect themselves against the introduction of heterologous DNA. ET12567 [pUZ8002] is methylation deficient and therefore a convenient donor to use. If the Streptomyces species does not have methyl-sensing mechanisms then DH5α [pUZ8002] may be used. The plasmid pUZ8002 is required in both cases because of the tra gene, which encodes a transfer protein Tra. This enables RP4 derived in trans mobilisation of tra defective vectors in intergeneric conjugations.

<br/>
The number of parallels performed depends on the type of conjugation:
{| cellspacing="0" cellpadding="0" rules="rows" frame=hsides
|-
| Integrative || Conjugation with an integrative plasmid; those with a phage ΦC31 integrase gene, i.e. pSET152 is relatively efficient. Integration occurs between the attP sites in the Streptomyces genome and the plasmid/construct. A single set of plates should suffice for the four dilutions.
|-
| Homologous Recombination || When attempting PCR targeted mutagenesis with a library cosmid; the integration or removal of the resistance cassette relies on homologous recombination. This is quite an inefficient process, especially when looking for complete recombination (double crossovers). It is recommended to plate at least one parallel set of plates (2x4 dilution sets = 8 plates in total).
|-
| Selective || Conjugation with a plasmid that does not have integrase or homologous recombination activity, i.e. pCJW93 requires continuous selection with thiostrepton to keep the plasmid/construct. This is also an efficient conjugation that should only require a single set of plates – one for each of the dilutions. 
|}

<br/>
A consideration with the E.coli strain ET12567 is its growth rate. Its doubling rate is slightly longer than DH5α. Thus the subculture needs a longer incubation (up to 4 hours) and the plates a few extra hours.

<br/>
The antibiotics in the overlay select for the correct exconjugants. It is done after the incubation overnight to allow for the construct replication / integration into the genome of Streptomyces. If a construct confers Apramycin resistance then all successful exconjugants will be Apramycin resistant. When used at the suggested concentrations Nalidixic Acid bacteriostatically inhibits the growth E.coli cells. It is thought to do this by disrupting DNA repair mechanisms. This prevents E.coli from over growing the Streptomyces, allowing it to develop normally. Also, mechanical selection of a Streptomyces colony will prove easier in later steps.

<!--CONTENT END-->
|}<br style="clear:both" />