What will we be doing in SynBERC year 1?  (August 2006 -> August 2007)

==Undergraduates==
==Graduate Students==
===[[Barry Canton]]===
I am engineering bacterial cells to act as [[SynBERC:Chassis|chassis]] for engineered biological systems.  Today we use bacterial cells to power engineered biological systems in an ad hoc fashion with an unknown probability of success.  One technology that might improve the process of engineering biological systems is to decouple the function of the system from that of the cellular chassis.  Specifically, I am working to enable a protein synthesis system that is dedicated to the engineered biological system and to elucidate the interaction between the system and the chassis.  You can read more about my project [[Engineering the Host/System Interface|here]].

===[[Austin Che]]===

I am engineering trans-splicing ribozymes derived from the Tetrahymena group I intron. Splicing ribozymes enable the design of synthetic biological circuits by providing real-time control over what is expressed using post-transcription and pre-translation logic. The ability to splice into an existing biological system provides a minimally invasive hook for measurement and debugging purposes. In addition, splicing can be used to patch or modify the operation of an existing system. For building novel synthetic systems, splicing ribozymes can be used as a macro expansion library, expanding short tags to longer sequences. Another application is the implementation of logic using a modular, reusable, and scalable family of splicing ribozyme logic gates. 

===[[Jason Kelly]]===
I am developing a system for construction of biological systems via screening of subcomponent ([[SynBERC:Parts|parts]] or [[SynBERC:Devices|devices]]) libraries and rational re-assembly. I have begun development of tools to enable this approach, including a [[Endy:Screening plasmid|FACS-based screening system]] to rapidly [[Endy:Screening plasmid/Inverter characterization|measure the input/output function of a genetic circuit]]. Additionally, I have designed a microfluidic system that enables more sophisticated screening and selection functions. Specifically, a microfluidic chemostat integrated with a cell sorter (i.e., a [[Sortostat|sort-o-stat]]). This microscope-based system will enable us to evaluate whether or not more complicated screens and selections will be of practical use in service of evolving engineered biological systems.

===[[Reshma Shetty]]===
My work seeks to develop a novel device implementation of transcription based logic.  Ideally, the resulting devices should be well-characterized, composable and scalable.  I hope to begin to address these goals by engineering devices based on modular, synthetic transcription factors.  These synthetic transcription factors are made of two domains: DNA binding (to confer specificity to the devices) and dimerization (to confer nonlinear or digital behavior to the devices).  I use zinc fingers as the DNA binding domain and leucine zippers as the dimerization domain.  Since both protein domain families are well-studied, highly specific and large, this design is both modular and scalable.  Finally, I am also interested in defining the key performance characteristics of these devices as well as approaches for measuring those characteristics experimentally.

[Editorial note: Given that I have been working on this project for a few years now, it is unlikely that I will complete it in year 1 of the SynBERC grant.]

===[[Samantha Sutton]]===

I am developing a framework for engineering Post-translational Devices (PTDs), which are devices whose input/output characteristics are modulated by protein modification. I have built two instances of translocating PTDs, called Phospholocators, and am characterizing them to ensure the easy reusual for future engineers. I am using the knowledge gained in building the Phospholocators to develop an abstraction hierarchy, a  common signal carrier, and device boundaries for PTDs.

==Research Assistants==

===[[Heather Keller]]===

I am currently working to clone the 60 bacteriophage T7 Ribosome Binding Sites (RBSs) upstream of two fluorescent reporters, either GFP or mCherry.  The fluorsecent output of these constructs, as measured on a plate reader, will help to establish the relative strength of these RBS sequences in support of our efforts to rebuild the T7 genome, while also generating a library of well characterized RBSs for use in Biobricks assembly.  I also hope to examine issues related to the predictable composition of biological parts by comparing the output of the two flurophores for a given RBS, as well as constructing a second version of each RBS that contains that first 5 amino acids of the native T7 protein.

==Investigators==