'''This page is intended as a how-to guide for constructing novel BioBrick parts for submission to the [http://parts.mit.edu Registry of Standard Biological Parts].''' 

'''For those already familiar with making BioBrick parts, see the [[Synthetic Biology:BioBricks/Part fabrication/quick reference | quick reference guide]].'''

'''For help in assembling two preexisting BioBricks parts together, see one of the following pages.'''
#[[BioBricks construction tutorial]]
#[[Synthetic Biology:BioBricks/3A assembly | 3A assembly]]
#[[Silver: BB Strategy | Silver lab strategy]]

==Introduction==

This page serves as documentation for how to construct a BioBrick part.  For a more detailed explanation of the development of the BioBricks scheme, see Tom Knight's technical report [http://hdl.handle.net/1721.1/21168 Idempotent Vector Design for Standard Assembly of BioBricks].

The exact approach used when fabricating a BioBrick part depends on the fabrication method (PCR or direct synthesis) as well as the type of part being constructed (standard part or protein coding sequence).

==Constructing a BioBrick part via PCR==

A BioBrick can be constructed via PCR if there already exists template DNA from which the BioBricks can be amplified or if the part is short enough that it can be created by [[Annealing and primer extension | primer annealing and extension]].  When constructing a BioBrick part via PCR, specific sequences corresponding to the BioBrick ends must be included on the 5' end of each of the two PCR primers.  The exact sequence of that primer tail depends on whether the desired BioBrick parts is a protein coding sequence or some other kind of BioBrick part.

===Standard part fabrication===

Use this approach for promoters, ribosome binding sites, terminators and most other BioBricks parts.

====Prefix====

 5' GTTTCTT C GAATTC GCGGCCGC  T  TCTAGA  G   [part] 3'
 3' CAAAGAA G CTTAAG CGCCGGCG  A  AGATCT  C   [part] 5'
    (1)    (2)(3)    (4)      (5) (6)    (7)  (8)

#Extra bases designed to both
##permit cutting of the PCR product with EcoRI by providing extra "spacer" bases.  See [[Restriction digest#Notes | notes on cutting near the ends of linear DNA fragments]].
##promote addition of an A base on the opposite strand by Taq polymerase for high efficiency TA cloning if desired.  See [[Knight:TOPO TA cloning#Notes | notes on TOPO TA cloning]].
#Random extra spacer base
#EcoRI recognition site
#NotI recognition site
#Extra base to prevent inadvertent creation of EcoBI or EcoKI methylation sites which could inhibit efficient digestion by the BioBricks enzymes.
#XbaI recognition site
#Extra G base to prevent inadvertent creation of either
##a GATC site (which can undergo methylation in some strains thereby inhibiting digestion by the BioBricks enzymes.)
##an ATG start codon
#Approximately 20 bp of sequence that matches the 5' end of the part you wish to construct.

====Suffix====

 5' [part] T ACTAGT  A  GCGGCCG CTGCAG G AAGAAAC   3'
 3' [part] A TGATCA  T  CGCCGGC GACGTC C TTCTTTG   5'
    (1)      (2)    (3) (4)     (5)   (6)  (7)

*The above sequence assumes that your part is on the forward strand running in the 5' to 3' direction.  To construct a PCR primer, you will need to use the bottom strand in the reverse direction.

#Approximately 20 bp of sequence that matches the 3' end of the part you wish to construct.
#SpeI recognition site
#Extra base to prevent inadvertent creation of EcoBI or EcoKI methylation sites which could inhibit efficient digestion by the BioBricks enzymes.
#NotI recognition site
#PstI recognition site
#Random extra spacer base
#Extra bases designed to both
##permit cutting of the PCR product with PstI by providing extra "spacer" bases.  See [[Restriction digest#Notes | notes on cutting near the ends of linear DNA fragments]].
##promote addition of an A base on the opposite strand by Taq polymerase for high efficiency TA cloning if desired.  See [[Knight:TOPO TA cloning#Notes | notes on TOPO TA cloning]].

====Quick reference====
Once you are ready to design your primers for making a BioBrick, you can copy and paste the following sequences into your primers.

 Copy and paste the following 30 bp sequence onto the 5' end of your upstream primer:
 5' ---> 3'
 GTT TCT TCG AAT TCG CGG CCG CTT CTA GAG

 Copy and paste the following 29 bp sequence onto the 5' end of your downstream primer:
 5' ---> 3'
 GTT TCT TCC TGC AGC GGC CGC TAC TAG TA

===Protein coding sequence fabrication===

Construction of protein coding sequences in BioBricks form requires slightly specialized BioBricks prefixes and suffixes for two reasons.
#The prefix is slightly altered to ensure appropriate spacing between the ribsome binding site and the start codon.
#BioBricks coding sequences standardly end with two sequential TAA stop codons.  

====Prefix====

 5' GTTTCTT C GAATTC GCGGCCGC  T  TCTAG [ATG Remaining CDS] 3'
 3' CAAAGAA G CTTAAG CGCCGGCG  A  AGATC [TAC Remaining CDS] 5'
    (1)    (2)(3)    (4)      (5) (6)    (7) (8)

#Extra bases designed to both
##permit cutting of the PCR product with EcoRI by providing extra "spacer" bases.  See [[Restriction digest#Notes | notes on cutting near the ends of linear DNA fragments]].
##promote addition of an A base on the opposite strand by Taq polymerase for high efficiency TA cloning if desired.  See [[Knight:TOPO TA cloning#Notes | notes on TOPO TA cloning]].
#Random extra spacer base
#EcoRI recognition site
#NotI recognition site
#Extra base to prevent inadvertent creation of EcoBI or EcoKI methylation sites which could inhibit efficient digestion by the BioBricks enzymes.
#XbaI recognition site
#An ATG start codon
#Approximately 20 bp of sequence that matches the 5' end of the coding sequence you wish to construct (excluding the start codon).

====Suffix====

 5' [part] TAATAA T ACTAGT  A  GCGGCCG CTGCAG G AAGAAAC  3'
 3' [part] ATTATT A TGATCA  T  CGCCGGC GACGTC C TTCTTTG  5'
    (1)    (2)      (3)    (4) (5)     (6)   (7)  (8)

*The above sequence assumes that your part is on the forward strand running in the 5' to 3' direction.  To construct a PCR primer, you will need to use the bottom strand in the reverse direction.

#Approximately 20 bp of sequence that matches the 3' end of the CDS you wish to construct (excluding the stop codon).
#Two sequential stop codons.  TAA is the default stop codon used in all BioBricks coding sequences.
#SpeI recognition site
#Extra base to prevent inadvertent creation of EcoBI or EcoKI methylation sites which could inhibit efficient digestion by the BioBricks enzymes.
#NotI recognition site
#PstI recognition site
#Random extra spacer base
#Extra bases designed to both
##permit cutting of the PCR product with PstI by providing extra "spacer" bases.  See [[Restriction digest#Notes | notes on cutting near the ends of linear DNA fragments]].
##promote addition of an A base on the opposite strand by Taq polymerase for high efficiency TA cloning if desired.  See [[Knight:TOPO TA cloning#Notes | notes on TOPO TA cloning]].

====Quick reference====
Once you are ready to design your primers for making a BioBrick, you can copy and paste the following sequences into your primers.

 Copy and paste the following 31 bp sequence onto the 5' end of your upstream primer for your coding sequence:
 includes the ATG start codon!
 5' ---> 3'
 GTT TCT TCG AAT TCG CGG CCG CTT CTA G [ATG start]

 Copy and paste the following 35 bp sequence onto the 5' end of your downstream primer for your coding sequence:
 includes the TAATAA double stop codon!
 5' ---> 3'
 GTT TCT TCC TGC AGC GGC CGC TAC TAG TA [TTA TTA double stop codon]

==Constructing a BioBrick part via direct synthesis==

If you are constructing a BioBrick part via direct synthesis ... especially a coding sequence (in which the sequence is flexible), you may want to consider eliminating the following restriction sites that might be useful in the future to others.  This list of sites is prioritized.

===These sites MUST be absent===
*[[EcoRI]], [[SpeI]], [[XbaI]], [[PstI]], [[NotI]]

The exception to this rule is for the BioBricks prefixes, BioBricks suffixes and the multiple cloning site of BioBricks vectors.

===Strongly prefer that these sites be absent===
These restriction enzymes are those that generate compatible cohesive ends to the BioBrick sites and therefore can be useful for various projects.

*[http://www.neb.com/nebecomm/products/productR0566.asp ApoI] (probably impossible because it occurs frequently), [http://www.neb.com/nebecomm/products/productR0589.asp Mfe I]
*[http://www.neb.com/nebecomm/products/productR0174.asp AvrII], [http://www.neb.com/nebecomm/products/productR0131.asp NheI]
*[http://www.neb.com/nebecomm/products/productR0127.asp NsiI], [http://www.neb.com/nebecomm/products/productR0642.asp SbfI]

===Would prefer that these sites be absent===

====Offset cutters====
These are offset cutters that can generate arbitrary overhangs.  In addition, some are used in the BioBricks++ assembly scheme.

*[http://www.fermentas.com/catalog/re/aari.htm AarI], [http://www.neb.com/nebecomm/products/productR0641.asp AcuI] (medium priority), [http://www.neb.com/nebecomm/products/productR0539.asp BbsI], [http://www.neb.com/nebecomm/products/productR0601.asp BbvCI], [http://www.neb.com/nebecomm/products/productR0596.asp BciVI] (would be nice, medium priority), [http://www.neb.com/nebecomm/products/productR0701.asp BfuAI] (high priority), [http://www.neb.com/nebecomm/products/productR0600.asp BmrI] (high priority),   [http://www.neb.com/nebecomm/products/productR0580.asp BsmBI], [http://www.neb.com/nebecomm/products/productR0535.asp BsaI], [http://www.neb.com/nebecomm/products/productR0559.asp BsgI] (medium priority), [http://www.neb.com/nebecomm/products/productR0134.asp BsmI] (includes nicking enzyme, high priority), [http://www.neb.com/nebecomm/products/productR0502.asp BspMI], [http://www.neb.com/nebecomm/products/productR0574.asp BsrDI] (includes nicking enzyme, high priority), [http://www.neb.com/nebecomm/products/productR0703.asp BtgZI], [http://www.neb.com/nebecomm/products/productR0528.asp EarI], [http://www.neb.com/nebecomm/products/productR0646.asp EcoP15I] (high priority),  [http://www.neb.com/nebecomm/products/productR0109.asp FokI] (best effort), [http://www.neb.com/nebecomm/products/productR0607.asp Nt.BstNBI], [http://www.neb.com/nebecomm/products/productR0569.asp SapI] (high priority, should already be eliminated from EarI), [http://www.neb.com/nebecomm/products/productR0582.asp TspRI] (probably difficult, best effort)

====Nicking enzymes====
*[http://www.neb.com/nebecomm/products/productR0607.asp Nt.BstNBI], BbvCI, [http://www.neb.com/nebecomm/products/productR0627.asp Nt.AlwI] (best effort to at least remove sites near each other)

====Homing endonucleases====
These enzymes have very long recognition sites and are unlikely to be in your part.

*I-CeuI, I-SceI, PI-PspI, PI-SceI, I-PpoI

====Others====
*[http://www.neb.com/nebecomm/products/productR0552.asp AgeI], AscI, FseI, [http://www.neb.com/nebecomm/products/productR0501.asp RsrII], [http://www.neb.com/nebecomm/products/productR0603.asp SgrAI], [http://www.neb.com/nebecomm/products/productR0194.asp XmnI], [http://www.neb.com/nebecomm/products/productR0533.asp XcmI]

===Would be convenient if these sites were removed===
These are common, efficient cutters that people might want to use.
*HindIII, BamHI, XhoI, NcoI, SacI, NdeI

Here are other low priority cut sites to remove
* KasI, MssI, NgoMIV, PacI, PmeI, SalI, SfiI, SgfI, SmiI, SrfI, SwaI, XmaI, ZraI

===Sites to include===
*Having GATC sites in your part can sometimes be useful because it allows plasmid purified DNA to be cut (via DpnI) whereas PCR product DNA is not.  Such a trick is useful for some site directed mutagenesis protocols.

===Miscellaneous===
*BaeI is an enzyme whose is exact cut position is unknown.  It is explicitly included in some pSB plasmid replications origins so that the plasmid can optionally be destroyed via another enzyme.  You may want to eliminate this site from your part if you intend to use this feature of the BioBricks plasmids.

==References==
#[http://hdl.handle.net/1721.1/21168 Idempotent Vector Design for Standard Assembly of BioBricks] by [[Tom Knight]]
#http://parts.mit.edu/r/parts/htdocs/Assembly/rbs_cds.cgi

==Contacts==
The BioBricks idempotent assembly scheme was designed by [[Tom Knight]].

This page was developed by [[Reshma Shetty]] for instructional purposes.