These are some design notes on the new version 5 vectors.

==Overall design==
[[Image:SBvectordesign.png|thumb|600px|left|Layout sketch of new SB vectors. Updated 8/21/2006 by [[Reshma Shetty]].  Click to view larger version.]]
<br style="clear:both" />

==Features==

#Positive selection marker (''ccdB'') internal to the BioBricks insertion site to facilitate cloning.
#A high copy origin internal to the BioBricks insertion site to enable easy purification of the vector.
#Strong terminators flanking the BioBricks insertion site (both orientations).
#Standard BioBricks verification primer binding sites (VF2 and VR) positioned to yield good sequencing results of insert.
#Enabled for [[Topoisomerase I mediated TA cloning]]
#Translational stops in all 6 frames flanking the BioBricks insertion site
#Modular scaffold design so that new vectors can be constructed in the future.  (Replications origin and resistance marker should be assembled together in BioBricks form and cloned via the NheI sites into the vector scaffold).

==Vector components==
Genbank files of the sequences below are available for [[Media:PSB**5vectors.zip|download]].  Each of these vectors is available via the [http://parts.mit.edu Registry of Standard Biological Parts].

This is the list of DNA fragments that need to be synthesized (with the exception of the '''Insert''' which comes as part of the scaffold).

===Vector scaffold===
This sequence is the vector scaffold.  It has BBa_P1016 (ccdB, a toxic gene ... note that original design called for BBa_P1011) and BBa_I50020 (pUC19 origin) in the multiple cloning site.  It also has BBa_P1002 (ampicillin resistance cassette in the reverse orientation) flanked by two NheI sites.  Digestion of this plasmid with NheI will permit any BioBrick part that has been cut with XbaI and Spe I to be inserted into the vector.

*<bbpart>BBa_I51001</bbpart>   Vector scaffold Length: 2438 Base Pairs 

*'''[[User:Rshetty|Reshma]] 14:34, 29 January 2007 (EST)''': Due to synthesis issues, rather than synthesizing BBa_P1011, <bbpart>BBa_P1016</bbpart> was synthesized.  Thus, the vector scaffold is documented at <bbpart>BBa_I51001</bbpart> (<bbpart>BBa_I51000</bbpart> contains the original design except with BioBricks scars included).

===Antibiotic resistance marker===
We want four different resistance markers.  For these parts, in addition to getting them inserted into the vector, we'd also want them as separate pieces in BioBricks format.  There is flexibility in the coding sequence of these parts.  One of them, BBa_P1002 is present in the vector scaffold but we would also like it as a separate piece in BioBricks format.  The other three (BBa_P1000, BBa_P1001, BBa_P1003) are modules that need to be assembled with each replication origin and then inserted at the two NheI restriction sites in the vector scaffold (to replace BBa_P1002).  Note the orientation in which these modules (once combined with a replication origin) are inserted into the intact vector.  Although the insertion step itself is not directional, we want clones in our preferred orientation.

*BBa_P1000   CmR Length: 835 Base Pairs including BioBrick ends
**renamed to <bbpart>BBa_P1004</bbpart>
*BBa_P1001   TetR Length: 1325 Base Pairs including BioBrick ends
**renamed to <bbpart>BBa_P1005</bbpart>
*<bbpart>BBa_P1002</bbpart>   AmpR Length: 987 Base Pairs including BioBrick ends
*<bbpart>BBa_P1003</bbpart>   KanR Length: 1039 Base Pairs including BioBrick ends

===Replication origin===
We want one replication origin.  In addition to getting it inserted into the vector, we also want it as separate piece in BioBricks format.  It is a module that need to be assembled with each antibiotic resistance marker and then inserted at the two NheI restriction sites in the vector scaffold (to replace BBa_P1002).  Note the orientation in which this module (once combined with an antibiotic resistance marker) is inserted into the intact vector.  Although the insertion step itself is not directional, we want clones our preferred orientation.

*<bbpart>BBa_I50040</bbpart>   pSC101 origin of replication Length: 2269 Base Pairs including BioBrick ends

Other origins can be generated in BioBricks format in the future via PCR and/or direct synthesis.

===Insert (or MCS module)===
Note that BBa_P1011 and BBa_I52000 encode ccdB, a toxic gene to ''E. coli'', and therefore will only propagate in special strains which have a mutation that confers resistance to ccdB (i.e. DB3.1).  BBa_I52000 is the insert present in the vector scaffold.  We will not ask for the BBa_I50020 and BBa_P1011 as separate pieces in BioBricks format but they could later be generated by PCR.

*<bbpart>BBa_P1016</bbpart>   ccdB cassette Length: 416 Base Pairs excluding BioBrick ends
**'''[[User:Rshetty|Reshma]] 14:34, 29 January 2007 (EST)''': Due to synthesis issues, rather than making <bbpart>BBa_P1011</bbpart>, <bbpart>BBa_P1016</bbpart> was made.
*<bbpart>BBa_I50020</bbpart>   high copy origin of replication from pUC19 Length: 861 Base Pairs including BioBrick ends
**'''[[User:Rshetty|Reshma]] 14:34, 29 January 2007 (EST)''': contains several ambiguous bases as documented in the Registry
*<bbpart>BBa_I52001</bbpart>   MCS cassette, ccdB with high copy origin Length: 1303 Base Pairs including BioBrick ends (without a scar)
**'''[[User:Rshetty|Reshma]] 14:51, 29 January 2007 (EST)''': Originally designed as <bbpart>BBa_I52000</bbpart>

===Intact vector===
In total, there are 3 antibiotic resistance choices * 1 replication origins = 3 total intact vector combinations.  The antibiotic resistance markers and replication origins have been assembled as a unit and inserted into the vector scaffold at the NheI sites.  Note that this cloning step is not directional although we do want the replication origin and antibiotic resistance marker to be inserted into the vector in a particular orientation.  Therefore, this cloning step will require some screening to verify orientation.

*pSB4C5-I52001
*pSB4K5-I52001
*pSB4T5-I52001

*'''[[User:Rshetty|Reshma]] 14:34, 29 January 2007 (EST)''': Codon Devices elected not to complete the project.  So these three vectors have yet to be synthesized.  However, I successfully constructed vectors on my own.  They have been made and sequence verified.

==Individual BioBrick vector parts==
{| border="1"
|-
| '''Part number'''
| '''Description'''
| '''Size'''
| '''Want synthesized'''
| '''Want synthesized as an individual part in BioBricks format'''
|-- 
| colspan="6" | '''RESISTANCE MARKERS'''
|--
| BBa_P1000 (now BBa_P1004)
| CmR
| 812bp
| Yes
| Yes, BBa_P1004
|--
| BBa_P1001 (now BBa_P1005)
| TetR
| 1326bp
| Yes
| Yes, BBa_P1005
|--
| BBa_P1002
| AmpR
| 986bp
| Yes
| Yes
|--
| BBa_P1003
| KanR
| 1010bp
| Yes
| Yes
|--
| colspan="6" |
|-- 
| colspan="6" | '''REPLICATION ORIGINS'''
|--
| BBa_I50040
| low copy origin (pSC101)
| 2269bp
| Yes
| Yes, BBa_I50040
|--
| colspan="6" |
|-- 
| colspan="6" | '''SCAFFOLD'''
|--
| BBa_I51000
| Vector scaffold
| 2439bp
| Yes, BBa_I51001
| No
|--
| colspan="6" |
|-- 
| colspan="6" | '''COMPLETE VECTORS'''
|--
| pSB4C5-I52000
| Vector
| 4500bp
| Yes
| No
|--
| pSB4K5-I52000
| Vector
| 4698bp
| Yes
| No
|--
| pSB4T5-I52000
| Vector
| 5014bp
| Yes
| No
|}

==BioBricks format==
The antibiotic resistance markers and the replication origins will be supplied separately in BioBricks format.

These parts will be provided in a standard vector (like pUC19).

==Vector assembly scheme==
[[Image:SBvectorassemblyscheme.png|thumb|600px|left|Sample assembly scheme for vectors.  The same scheme can be used for all combinations of replication origins and antibiotic resistance markers.  Click to view larger version.]]
<br style="clear:both" />

==Restriction enzyme site removal==
*Tried to remove the suggested restriction sites from the vector components whenever possible.  Some sites which occurred in non-coding regions (like nicking enzymes and ApoI sites) were not removed.

Here are the restriction enzyme sites that I tried to remove.

===BioBrick enzymes sites===
[[EcoRI]], [[SpeI]], [[XbaI]], [[PstI]], [[NotI]]

===Enzymes sites that generate compatible cohesive ends to BioBrick sites===
*[http://www.neb.com/nebecomm/EnzymeFinderSearchByEnd.asp?txtSequence=AATT&selDirection=2&radSearchType=0 '''EcoRI''']:  [http://www.neb.com/nebecomm/products/productR0566.asp ApoI], [http://www.neb.com/nebecomm/products/productR0589.asp Mfe I]
*[http://www.neb.com/nebecomm/EnzymeFinderSearchByEnd.asp?txtSequence=CTAG&selDirection=2&radSearchType=0 '''SpeI/XbaI''']:  [http://www.neb.com/nebecomm/products/productR0174.asp AvrII], [http://www.neb.com/nebecomm/products/productR0131.asp NheI]
*[http://www.neb.com/nebecomm/EnzymeFinderSearchByEnd.asp?txtSequence=TGCA&selDirection=1&radSearchType=0 '''PstI''']:  [http://www.neb.com/nebecomm/products/productR0127.asp NsiI], [http://www.neb.com/nebecomm/products/productR0642.asp SbfI]

===Offset cutters===
[http://www.fermentas.com/catalog/re/aari.htm AarI], [http://www.neb.com/nebecomm/products/productR0580.asp BsmBI], [http://www.neb.com/nebecomm/products/productR0535.asp BsaI], [http://www.neb.com/nebecomm/products/productR0539.asp BbsI], [http://www.neb.com/nebecomm/products/productR0502.asp BspMI], [http://www.neb.com/nebecomm/products/productR0703.asp BtgZI],
[http://www.neb.com/nebecomm/products/productR0528.asp EarI]

===Nicking enzymes===
[http://www.neb.com/nebecomm/products/productR0607.asp Nt.BstNBI], BbvCI

===Other common enzymes===
'''Arbitrary list, feel free to add more.'''

HindIII (wasn't able to remove from Bba_P1001), BamHI, XhoI, NcoI (wasn't able to remove from I50000, pSB5** series plamids), SacI, NdeI,

===Discussion===
Would be simplest to destroy all 6 bp palindromic sequences. This destroys a lot of the common 'normal' restriction sites.
*Is there a tool that does this?  [http://slam.bs.jhmi.edu/gd/ GeneDesign] removes sites and optimizes the resulting codons for a particular species.  But it requires that you select which enzymes you want to remove from their list.
* Don't know off the top of my head but you should ask Sri or Leon if there's a tool. They did that for their plasmid for the T7.1 rebuild so they could use more restriction enzymes. GeneDesign appears to actually list the enzyme sites that are in the sequence so assuming it has a good database of enzymes, the question is whether just removing everything it knows about is good enough. I could pretty easily write a program to find all 6 bp palidromes but fixing it with silent mutations would take more work.

===GATC===

Another idea I had was whether we want to remove all GATC (DpnI) sites from the plasmid. There are potential benefits and drawbacks to this. One potential downside is that some mutation protocols assume that you can chew up the plasmid by adding DpnI. However if our plasmid had no GATC, we could perhaps use this to our advantage in some way. For example, adding DpnI to cut up only the genomic DNA and not our plasmid (of course, this would only likely work with the base plasmids).

*'''[[User:Rshetty|RS]] 11:55, 15 May 2006 (EDT)''': Tom actually requested that I specifically include GATC sites in the plasmid to ensure that digestion by DpnI works well.  For cloning purposes, I think that treatment of the destination plasmid digestion with antarctic phosphatase is generally sufficient for reducing the likelihood of cloning genomic DNA.  So I would favor that approach over removing DpnI sites (given that the presence of DpnI sites is useful for site-directed mutagenesis).

*'''[[User:Rshetty|Reshma]] 23:14, 19 July 2006 (EDT)''': Note that I have left the GATC sites and actually included a few extra ones per Drew's instructions.

==Unique restriction sites in the backbone==
===Sites for barcode insertion===
Drew wanted a unique restriction site in the backbone via which a barcode could be inserted at a later date.  Austin argues that we shouldn't arbitrarily include 6bp cutters in the backbone because they could be used in parts.

Barcodes can be inserted easily into future vectors by simply assembling them with an origin and resistance marker in BioBricks format and inserting them into the NheI sites in the vector scaffold.  Currently, no barcodes have been generated.

===Sites to cut vector backbone===
Josh initially requested unique cutters in the backbone so that in the event that the BioBrick part is the same size as the vector backbone, you could cut the backbone to enable size separation of the two.  

Austin suggested using BaeI ... a so-called useless enzyme because there is some uncertainty in exactly where it cuts.  BaeI already cuts in the middle of <bbpart>BBa_I50000</bbpart> but does not cut <bbpart>BBa_I50040</bbpart>.  

I introduced a BaeI cut site immediately downstream of BBa_I50040.  This enzyme will serve as the backbone cutter.

==Additional features==
#How much would it cost to include 4 unique bases adjacent to VF2 and VR primer binding sites (present in the scaffold)?  These 4 unique bases should uniquely identify the vector.  What would this add to the cost?
#*Inclusion of these bases seems to add about $6000 to the cost of the project therefore we are not including barcodes at this time.