<wikionly>
Go to [http://syntheticbiology.org/ http://syntheticbiology.org/]
</wikionly>
<div style="padding: 10px; width: 730px; color: #ffffff; background-color: #181860">
<font face="trebuchet ms" style="color:#ffffff">
'''<big>Synthetic Biology</big> is '''<br>
'''A) the design and construction of new biological parts, devices, and systems, and'''<br>
'''B) the re-design of existing, natural biological systems for useful purposes.'''
</font>
</div>
<div>[[Image:Cells-to-gears-big.png|750px|Synthetic Biology Logo]]</div>
{{Synthetic biology top}}

<div style="padding: 10px; width: 730px; color: #000000; background-color: #ccccff">

This page contains a collection of links to online tools that are related to Synthetic Biology.  <wikionly>'''If you are editing this page, please try to keep the tools in alphabetical order, by tool name.'''</wikionly>

===DNA Tools===
*[http://alpha.dmi.unict.it/~ctnyu/bbocus.html BBOCUS] by Ferro and Purrello lab
**BBOCUS (BackTranslation Based On Codon Usage Strategy) is a re-implementation of the algorithm in Graziano Pesole's BACKTR. It's based on cluster analysis (Complete Linkage algorithm), that requires a similarity matrix D containing distance between each pair of sequences of mRNA. After cluster analysis, backtranslation is performed on sequences of homogeneous pool. Unlike in Graziano Pesole's method, where the choose of homogeneous pool was made by a biologist, the choose of cluster in BBOCUS is fully automatic. A Codon Usage Table (CUT) is created through sequences of homogeneuos pool. Generally, to backtranslate an amino acid in a protein the codon chosen it is the one has maximum frequency.

*[http://paris.chem.yale.edu/extinct.html Biopolymer calculator] by Schepartz lab
**Calculate extinction coefficients, Tm's, and base composition for your DNA or RNA; calculate amino acid composition and extinction coefficient for your protein

*[http://synbio.mit.edu/tools/clipboard.cgi Clipboard] by [[Austin Che]]
**Web tool for getting complement, reverse complement, translation and restriction enzyme analysis of a DNA sequence

*[http://helixweb.nih.gov/dnaworks/ DNAWorks] by Hoover and Lubkowski
**A web tool for optimizing melting temperature during gene synthesis.

*[http://synbio.mit.edu/tools/converter.cgi File format converter] by [[Austin Che]]
**Web tool for converting between sequence file formats.

*[http://slam.bs.jhmi.edu/gd/ GeneDesign] by Boeke lab
**Collection of online tools for codon optimization and shuffling, restriction site editing, and so on.

*[http://www.dna20.com/commerce/customer/emailsupp.jsp GeneDesigner] by DNA2.0
**Combine genetic building blocks by drag-and-drop, codon optimize, restriction site editing, sequence oligo design etc. See BMC Bioinformatics 2006 Jun 6;7(1):285 for more detail.
**Free to download and works on Mac or PC.  User agreement is somewhat restrictive (i.e., does not allow you to sell genes designed on the program without permission).

*[http://www.genocad.org/ GenoCAD]
** An experimental site that relies on formal grammars to design and verify the synthetic DNA sequences. 
** The theoretical foundations of GenoCAD were published in [http://bioinformatics.oxfordjournals.org/cgi/content/abstract/btm446 Bioinformatics].

*[http://tools.neb.com/NEBcutter2/index.php NEB Cutter] by New England Biolabs, Inc.
**Tool for finding restriction sites, et cetera.

*[http://www.evolvingcode.net/codon/sgd/index.php Synthetic Gene Designer] by [[Gang Wu]]
**A web platform that allows codon optimization to various extent.
**Compatible with non-standard genetic codes
**Integrate CUTG
**Restriction site avoidance
**Oligo design

*[http://www.invitrogen.com/content.cfm?pageid=10373 Vector NTI] by Informax, Inc.
**Free-to-academics tool for sequence analysis and data management.

===RNA Tools===
*[http://www.ambion.com/techlib/append/na_mw_tables.html Appendix] by Ambion, Inc.
**Website with many useful nucleic acid parameters.

*[http://www.bioinfo.rpi.edu/applications/mfold/ mFold] by Michael Zuker.
**This is the main page with links to sites for predicting RNA and DNA folds, calculating Tm's and free energies.

===Protein Tools===
*[http://ncbi.nih.gov/Structure/CN3D/cn3d.shtml Cn3D] by NCBI
**A helper application for your web browser that allows you to view 3-dimensional structures from NCBI's  Entrez retrieval service.  It doesn't read PDB files but can be more straightforward to use than DeepView.

*[http://www.expasy.org/spdbv/ DeepView] by GlaxoSmithKline & Swiss Institute of Bioinformatics
**Awesome program for viewing and studying protein structure.

*[http://us.expasy.org/ ExPASy Proteomics server] by the Swiss Institute of Bioinformatics 
**Collection of links to many pages to calculate parameters of your favorite proteins

*[http://salilab.org/modeller/ Modeller] by Sali Lab
**For homology or comparative modeling of protein three-dimensional structures.

*[http://www.scripps.edu/mb/barbas/zfdesign/zfdesignhome.php Zinc Finger Tools] by Barbas Lab
**Design Zinc Finger DNA binding proteins

===General Tools===
*[http://genolist.pasteur.fr/Colibri/ Colibri] by Institut Pasteur
**''E. coli'' genome site; get sequences, see the position of your gene in the chromosome, see the function of your gene, and other fun stuff. You can also search for protein sequences/motifs within the ''E. coli'' genome.

*[http://parts.mit.edu/ Registry of Standard Biological Parts] by MIT
**Open repository of BioBricks; the place for all your standard biological parts.
</div>
{{Synthetic biology bottom}}