{{Template:Rebuilding T7}}
==Why evolve T7.1?==
The refactoring process of the T7 phage to produce the [[T7.1|T7.1 phage]] was a success in the fact that the un-overlapping of the alpha and beta sections of the T7 genome still produced a viable phage.  Though it is viable, it is less fit than the wild-type T7 phage with respect to growth rate.  This observation gives us an opportunity to use short term evolution as a debugging tool to let nature help us discover what was disrupted during the refactoring process that causes reduced fitness.


==T7.1 Evolution Protocol==
NOTE: The below protocol has been adapted from various papers from the references listed below


===Adaption protocol:===

Adapted from ''Springman et al., 2005'':

#1 mL (experimentally confirmed on [[BjornsMethods/ExperimentLog| Dec 16, 2005]]) of Klett 80 (~4e8 cells/mL) E. coli frozen aliquot (-80C freezer) is thawed, and put into 10 mL LB solution in a 250 mL flask
#The culture is placed in an orbital water bath at 37C, 200 rpm for 60 min where cells are grown to a concentration of 7.5e7 cells/mL (want between 5e7-2e8 cells/mL).
#1e4 - 1e7 phage is added to the flask and grown for 20-60 min until density of free phage was approx. equal to cell density
#*A large variation in the number of phage initially inpu.  Makes sense for the first transfer, but what about subsequent transfers.  
#*Also, how do we know when to stop?  So we usually try to stop before final lysis, but how much before?
#*In the Springman paper, cultures of the continuous transfer protocol were occasionally allowed to lyse.  They claim it is to ensure high levels of co-infection which promotes recombination.  
#A portion of the infected culture was quickly transfered to the next flask containing fresh cells
#*Once again, how much?
#Lysates from each passage are [[BjornsMethods/PhageStorage|stored]] at 4&deg;C
#The above steps will be repeated at least ~4-5 times a day.  
#At the start of each day, the stored phage stock from last previous day's passage will be used as starting phage for current day's passages


'''Note:''' We intend to change this protocol to allow for complete lysis in order to increase repeatability and ease of experimentation.

===Duration of adaptation:===

Continue serial transfers long enough to ensure no major fitness changes occur.  

''Springman 2005''  propagated 57 h, 65 h, 50.5 h (3 different strains of phage);  Since generation time shortens as the phage gains fitness it is difficult to determine exact number of generations, however they state all phage were grown in excess of 100 phage generations.


===Media===

Past experiments have used [[LB]].  We could use a more restrictive media, such as M9 with sugar source (IJM recommendation)

===Bacterial Host:===
Ideally we wanted to find a bacterial host in which the T7+ (WT) had minimal nucleotide evolution, yet still provided selection pressures for the T7.1.  


''Springman 2005'' notes:

"Wild-type T7 has not been extensively grown in the laboratory since its isolation in 1945 (Studier, 1969, I. J. M., unpublished). Its usual hosts are E. coli B derivatives, rather than the K-12 strain used here, and the usual temperature of growth is 30 -C, rather than 37 -C. It was therefore expected that the T7+ was not well adapted to our culture conditions. "  <-- They used IJ1133


''Heineman 2005'' notes :
"The T7+ had already been passaged in this lab condition on IJ1133 for some time and was passaged for 20.5 more hours to create a control for the expected fitness increase in the abscence of any genomic defect"


Host chosen:

BL-21

'''Reason''':  It is supposedly the ''usual'' host but we still performed adaptation procedures for T7+, as in ''Heineman 2005'', to confirm it has reached its final steady state of adaptation



How to make a [[BjornsMethods/SmallOvernight |small bacterial overnight culture]]

How to make a [[BjornsMethods/LargeOvernight |large bacterial liquid culture]]


===Phage Strains:===

2 strains of phage were used in these experiments:

(1) T7+ (wild-type)

(2) T7.1 alpha-beta


===Determination of number of phage:===

[[BjornsMethods/PhageTitering |phage titering]] was used to determine the approximate number of phage in a solution


===Measuring Phage Lysis Curves===

Taken from ''Chan 2005 - Supp. Materials'':

1mL containing 2E8 cells of BL21 was infected at a MOI of 5 and 200uL of the resulting mixture was loaded per well into a 96 well ViewPlate (Packard) at 30°C.  Mineral oil was layered into each well and the OD was monitored at 30°C with agitation by a Wallac Victor2 plate reader (Perkin-Elmer). The half-lysis time was taken at the time when the absorbance of a culture equals the average of its absorbance at (i) the time of infection and (ii) the end of lysis. 


===Fitness Test:===

Fitness was measured using a doublings/hour metric as calculated:

'''Fitness = [log2(Nt/N0)]t'''

N0 is the number of phage after 1 hour, to ensure asynchronous infection and exponential growth phase; 

Nt is the number of phage at time i (i>1 hr), corrected for dilutions over multiple transfers,

t is time measured in hours. 


===References===

* Bull JJ, Badgett MR, Rokyta D, Molineux IJ, (2003) Experimental evolution yields hundreds of mutations in a functional viral genome.  ''J Mol Evol'' 57:241-248
* Bull JJ, Badget MR, Wichman HA, Huelsenbeck JP, Hillis DM, Gulati A, Ho C, Molineux IJ, (1997) Exceptional convergent evolution in a virus.  ''Genetics'' 147: 1497-1507
* Chan L, Kosuri S, Endy D, (2005) Refactoring bacteriophage T7.  ''Molecular Systems Biology'' 10.1038/msb4100025
* Cunningham CW, Jeng K, Husti J, Badgett M, Molineux IJ, Hillis DM, Bull JJ, (1997) Parallel molecular evolution of deletions and nonsense mutations in bacteriophage T7.  ''Mol. Biol. Evol.'' 14(1): 113-116
* Garcia LR, Molineux IJ, (1996) Transcription-Independent DNA translocation of bacteriophage T7 DNA into E. coli.  ''J of Bacteriology'' p. 6921-6929
* Heineman R, Molineux IJ, Bull JJ, (2005) Evolutionary robustness of an optimal phenotype: Re-evolution of lysis in a bacteriophage deleted for its lysin gene.  ''J Mol Evol'' 61:181-191
* Springman R, Badgett MR, Molineux IJ, Bull JJ, (2005) Gene order constrains adaptation in bacteriophage T7. ''Virology'' 341 141-152
* Rokyta D, Badgett MR, Molineux IJ, Bull JJ, (2002) Experimental genomic evolution: extensive compensation for loss of DNA ligase activity in a virus.  ''Mol. Biol. Evol.'' 19(3): 230-238


===Outstanding questions===

# '''What bacterial strain should we use?  '''
#* BL21 seems like a good choice, but why did the reference papers often use IJ1133 for evolution?
#* How about doing BR3 in parallel with BL21?
# '''How do you know when to stop the infected culture?'''
#* Papers say stop when the phage density is about equal to bacteria density... but how do you test that?
# '''How much of the infected culture gets transfered to next the next culture for initial infection?'''
# '''Is the phage [[BjornsMethods/PhageStorage|storage]] protocol sufficient?'''