{{Rebuilding T7}}__NOTOC__
This page contains information on what genes will be included in the specification of [[T7.2]].  Most of this information was blatently taken/copied/modified from two references.  First, from Ian Molineux's upcoming chapter in Calender's new Bacteriophages book <cite>calendar2005</cite>.  Second, from the original sequence paper from Studier and Dunn<cite>studier1983</cite>.
==Genes included in [http://parts.mit.edu/registry/index.php/Part:BBa_Z0501 current T7.2 design]==

===gene 0.3===
Gene ''0.3'' is expressed at very high levels and protects T7 from various type I restriction systems by mimicking the shape of B DNA. Gp0.3 has been shown to bind the EcoKI restriction enzyme in a 2:1 complex and to displace the enzyme from its target DNA.

Second site suppressors of 0.7-,2- infections map to gene 0.3 (Molineux, PC).  Unknown why this is the case.  

The calculated RBS strength is 7.28 <cite>stormo1994</cite>.

[http://openwetware.org/images/3/3b/0.3_Z0261_ss.pdf RBS-CDS predicted secondary structure], dG = -5.12.

===gene 0.7===
Gene ''0.7'' codes for a protein that inactivates host-catalyzed transcription, it is also a serine-threonine protein kinase.  The two activities are separable.  Gp0.7 phosphorylates many host proteins, including ribosome components, ribonucleases III and E, and the β and β´ subunits of E. coli RNAP.  The BR3 and Y49 strains contain the rpoC-E1158K mutation, which renders gp0.7 essential for phage growth.  The mutation affects the region of β´ to which the T7 protein gp2 binds.  Gene ''0.7'' is non-essential in wild-type hosts and is actually detrimental when T7 is grown at 30˚C in rich media.  Early T7 RNA and protein synthesis is prolonged after infection by gp0.7 mutants.  However, phage growth at elevated temperatures or in minimal media, or in cells harboring the Col Ib plasmid renders gene ''0.7'' essential.  The processes underlying the conditional necessity for gp0.7 function have not been determined.

The calculated RBS strength is 4.94 <cite>stormo1994</cite>.


The E. coli C promoter -35 region is located on the 3' end of the coding sequence.  Removing the downstream region will probably be sufficient to delete functioning of the promoter.

[http://openwetware.org/images/d/d8/0.7_Z0262_ss.pdf RBS-CDS secondary structure], deltaG = -3.2

===gene 1===
T7 RNA Polymerase
The calculated RBS strength is 5.59 <cite>stormo1994</cite>.

Suppressor mutation in 19 allows no gene 1 late in infection.  It's thought that gene 1 is hanging out at the CJ terminator at the pause site... recruiting 18 and 19 after packaging.  (Studier FW, PC)

sequence changes:

For wild-type CDS and new RBS, [http://openwetware.org/images/7/77/1_Z0262_ss.pdf RBS-CDS secondary structure] has dG = -9.90. Optimized via codon-shuffling to have dG = -0.01 kCal/mol. 

===gene 1.1===
conserved; non-essential

The calculated RBS strength is 6.48 <cite>stormo1994</cite>.

For wild-type CDS, [http://openwetware.org/images/7/7f/1.1_Z0262_ss.pdf RBS-CDS secondary structure] has dG = -4.60.

===gene 1.2===
Gene 1.2 inhibits E. coli dGTPase and converts it into a rGTP-binding protein, which is important for T7 only when dgt is over-expressed (optA1 mutation).  T7 gene 1.2 also causes phage exclusion by the F plasmid, but T3 gene 1.2 overcomes the exclusion system.

Need to think more about this for consistency of which genes will be considered definitely in and which should be optional

The calculated RBS strength is 7.84 <cite>stormo1994</cite>.

[http://openwetware.org/images/3/3c/1.2_Z0261_ss.pdf RBS-CDS secondary structure], dG = -3.60

===gene 1.3===
The ATP-dependent DNA ligase is only essential in E. coli lig mutants, but its presence enhances phage growth in wild-type hosts.

The calculated RBS strength is 7.61 <cite>stormo1994</cite>.

[http://openwetware.org/images/5/5e/1.3_Z0261_ss.pdf RBS-CDS secondary structure], dG = -6.27

===gene 1.7===
full-length gene not conserved; beneficial for growth

The calculated RBS strength is 6.18 <cite>stormo1994</cite>.

[http://openwetware.org/images/d/de/1.7_Z0262_ss.pdf RBS-CDS secondary structure], dG = -5.30

===gene 2===
Gp2 binds to and is an inhibitor of E. coli RNAP but in K-12 or B strains the defects due to a lack of gene ''2'' activity are reduced DNA replication and premature breakdown of replicating DNA, specifically at the left genome end where the major E. coli promoters are found.

The calculated RBS strength is 7.94 <cite>stormo1994</cite>.

For wild-type CDS and new RBS, [http://openwetware.org/images/0/0a/2_Z0261_ss.pdf RBS-CDS secondary structure]has dG = -10.0. Codon-shuffled to have dG = -3.10 kCal/mol.

===gene 2.5===
Gene ''2.5'' encodes a single-stranded binding protein and is essential for T7 DNA replication and recombination.

Very important to get the level correct.  Too much or too little is bad.  (Studier FW, PC)

The calculated RBS strength is 6.85 <cite>stormo1994</cite>.

For wild-type CDS and new RBS, [http://openwetware.org/images/5/5d/2.5_Z0262_ss.pdf RBS-CDS junction has] dG = -9.5 kCal/mol. Codon-shuffled to have dG = -1.53 kCal/mol.

===gene 3===
Encodes for a endonuclease that prefers ssDNA.  It functions along with gene ''6'' to degrade the host chromosome during infection and provide nucleotides for subsequent phage DNA replication.  Gene ''3'' is also a Holliday junction resolvase.

The calculated RBS strength is 5.89 <cite>stormo1994</cite>.

[http://openwetware.org/images/a/a8/3_Z0262_ss.pdf RBS-CDS ss], dG = -6.30

===gene 3.5===
Gene ''3.5'' codes for lysozyme, which is actually not a lysozyme but an N-acetylmuramyl-L-alanine amidase.  The most important functions of lysozyme are also not in cell lysis but lie in controlling initiation and termination by T7 RNAP, which in turn affect DNA replication and packaging.

The calculated RBS strength is 8.32 <cite>stormo1994</cite>.

[http://openwetware.org/images/5/56/3.5_Z0262_ss.pdf RBS-CDS ss], dG = -5.30

===gene 4A/4B===
The full-length gene ''4'' protein is a 5´->3´ helicase of the E. coli DnaB family and a DNA primase.  An in-frame internal start site within gene 4 results in a protein, gp4B, that lacks the N-terminal 63 amino acids of the 63 kDa gp4A, and consequently lacks primase activity.  The primase recognition sites on ssDNA are 5´-G/TGGTC, and 5´-GTGTC.

The calculated RBS strength for 4A is 6.76 and 4B is 6.8 <cite>stormo1994</cite>.

For wild-type CDS and new RBS, [http://openwetware.org/images/d/de/4A_Z0262_ss.pdf RBS-CDS junction] has dG = -14.50. Codon-shuffled to have dG = -4.6 kCal/mol. 

===gene 4.3===
conserved; non-essential

The calculated RBS strength is 7.31 <cite>stormo1994</cite>.

[http://openwetware.org/images/d/d1/4.3_Z0261_ss.pdf RBS-CDS ss], dG = -8.00

===gene 4.5===
conserved; non-essential

The calculated RBS strength is 8.33 <cite>stormo1994</cite>.

[http://openwetware.org/images/e/e2/4.5_Z0261_ss.pdf RBS-CDS ss], dG = -7.20

===gene 5===
T7 DNA Polymerase consists of two proteins, the gene ''5'' protein and the processivity factor ''E. coli'' thioredoxin. Thioredoxin forms a 1:1 complex with the T7 protein and increases its processivity by at least 1000-fold, the complex also increases the activity of the associated 5´->3´ exonuclease on dsDNA but has little effect on ssDNA exonucleolytic activity.

The calculated RBS strength is 8.97 <cite>stormo1994</cite>.

For wild-type CDS and new RBS, [http://openwetware.org/images/1/1f/5_Z0261_ss.pdf RBS-CDS junction] has dG = -9.70. Codon-shuffled to have dG = -4.3 kCal/mol.

===gene 5.5-5.7===
conserved; non-essential; binds E. coli HNS; λrex exclusion; 
non-conserved –1 frameshift leads to T7 5.5-5.7 fusion.  The frameshift may not occur in other T7-like phages

The calculated RBS strength for 5.5 is 8.47 and 5.7 is 6.18 <cite>stormo1994</cite>.

[http://openwetware.org/images/5/51/5.5_Z0261_ss.pdf RBS-CDS ss], dG = -4.50

===gene 6===
Gene ''6'' encodes for a 5´->3´ exonuclease.  In combination with gene ''3'', it degrades the host chromosome during infection and provides nucleotides for subsequent phage DNA replication.  Gene ''6'' also possessses RNase H activity that removes RNA primers and also prometes concatemer formation during DNA replication.

The calculated RBS strength is 7.34 <cite>stormo1994</cite>.

[http://openwetware.org/images/1/1a/6_Z0261_ss.pdf RBS-CDS ss], dG = -4.61


===gene 6.5===
conserved; non-essential

Although Studier said that one of 6.5 and 6.5 is essential. problably 6.7.  (STudier FW, PC)

The calculated RBS strength is 6.78 <cite>stormo1994</cite>.

[http://openwetware.org/images/6/69/6.5_Z0262_ss.pdf RBS-CDS ss], dG = -1.50

===gene 6.7===
Head protein.  ~18 copies per virion.  Gp6.7 is not absolutely essential, morphological normal T7 particles are produced in its absence but have reduced infectivity.  <cite>molineux2005</cite>

The calculated RBS strength is 7.64 <cite>stormo1994</cite>.

[http://openwetware.org/images/e/eb/6.7_Z0261_ss.pdf RBS-CDS ss], dG = -7.80

===gene 7.3===
Gene ''7'' was originally defined by mutations that mapped between genes ''6'' and ''8'' but was subsequently shown to be two genes, ''7'' and ''7.3''.  Gp7.3 is a tail protein, present in ~30 copies per particle, that is essential for infectivity though not particle formation <cite>molineux2005</cite>
.

The calculated RBS strength is 6.88 <cite>stormo1994</cite>.

[http://openwetware.org/images/6/63/7.3_Z0262_ss.pdf RBS-CDS ss], dG = -6.04

===gene 8===
Head-tail connector.  12 copies per virion.<cite>molineux2005</cite>

The calculated RBS strength is 4.99 <cite>stormo1994</cite>.

[http://openwetware.org/images/8/8e/8_Z0262_ss.pdf RBS-CDS ss], dG = -4.6

===gene 9===
scaffolding protein
The calculated RBS strength is 6.65 <cite>stormo1994</cite>.

For wild-type CDS and new RBS, [http://openwetware.org/images/2/21/9_Z0262_ss.pdf RBS-CDS junction] has dG = -10.00. Codon-shuffled to have dG = -3.0 kCal/mol. 

===gene 10A/10B===
major/minor capsid protein.  The 10B portion is non-essential, 10A-only phages seem to grow normally.  ~430 copies of 10A and 16 copies of 10B in capsid (415 expected from symmetry) <cite>molineux2005</cite>.


JJ Dunn removed various pieces of terminator.. if you remove hairpin, the gene 10 RNA is obliterated.  Also, got rid of gene 10 pre-shine dalgarno and could not get translation off gene 10.  (Dunn JJ, PC)


The calculated RBS strength is 7.37 <cite>stormo1994</cite>.

[http://openwetware.org/images/e/ef/10A_Z0261_ss.pdf 10A RBS-CDS ss], dG = -7.1

===gene 11===
structural tail protein of phage capsid; 12 copies per virion <cite>molineux2005</cite>.

The calculated RBS strength is 7.8 <cite>stormo1994</cite>.

Wild-type CDS and new RBS has [http://openwetware.org/images/a/af/11_Z0261_ss.pdf RBS-CDS junction] with dG = -9.90. Codon-shuffled to have dG = -3.8 kCal/mol.

===gene 12===
a structural tail protein of the phage capsid; 6 copies per virion <cite>molineux2005</cite>.

The calculated RBS strength is 6.69 <cite>stormo1994</cite>.

Wild-type CDS and new RBS has [http://openwetware.org/images/5/57/12_Z0262_ss.pdf RBS-CDS junction] with dG = -8.80. Codon-shuffled to have dG = -1.7 kCal/mol. 

===gene 13===
Essential gene that is NOT part of the virion.  In part required to incorporate gene ''6.7'' (migrates similarly in many SDS gels) but also must have an additional role as virions made from 13 null mutants are more defective than virions lacking only gp6.7.

The calculated RBS strength is 6.21 <cite>stormo1994</cite>.

[http://openwetware.org/images/6/61/13_Z0262_ss.pdf RBS-CDS ss], dG = -4.50

===gene 14===
internal core protein of phage capsid.  ~10 copies per capsid <cite>molineux2005</cite>.

14 expression is dependent upon 13  (FW Studier, PC).

The calculated RBS strength is 6.86 <cite>stormo1994</cite>.

Wild-type CDS and new RBS has [http://openwetware.org/images/5/55/14_Z0262_ss.pdf RBS-CDS junction] with dG = -10.10. Codon-shuffled to have dG = -5.4 kCal/mol.

===gene 15===
internal core protein of phage capsid; ~8 copies per virion <cite>molineux2005</cite>.

The calculated RBS strength is 6.37 <cite>stormo1994</cite>.

[http://openwetware.org/images/a/a8/15_Z0262_ss.pdf RBS-CDS ss], dG = -7.90

===gene 16===
internal core protein of phage capsid; ~4 copies per virion <cite>molineux2005</cite>.

The calculated RBS strength is 6.55 <cite>stormo1994</cite>.

[http://openwetware.org/images/3/33/16_Z0262_ss.pdf RBS-CDS ss], dG = -4.70

===gene 17===
T7 tail fiber protein of phage capsid (adsorption); 14 +/- 3 (eighteen expected by symmettry) copies per virion; 3 copies per tail fiber <cite>molineux2005</cite>.

The calculated RBS strength is 5.87 <cite>stormo1994</cite>.

[http://openwetware.org/images/c/c9/17_Z0262_ss.pdf RBS-CDs ss], dG = -5.40

===gene 17.5===
lysis associated gene.

The calculated RBS strength is 5.92 <cite>stormo1994</cite>.

[http://openwetware.org/images/b/b3/17.5_Z0262_ss.pdf RBS-CDS ss], dG = -4.10

===gene 18===
small terminase subunit.

The calculated RBS strength is 7.26 <cite>stormo1994</cite>.

[http://openwetware.org/images/b/b6/18_Z0261_ss.pdf RBS-CDs ss], dG = -7.10


===gene 18.5===
lysis associated protein.  Homologs of 18.5/18.7 are found in all phages (in T1 the two proteins are fused)

The calculated RBS strength is 6.9 <cite>stormo1994</cite>.

[http://openwetware.org/images/c/c4/18.5_Z0261_ss.pdf RBS-CDS ss], dG = -7.24

===gene 19===
large terminase subunit.  binds capsid pro-head.

The calculated RBS strength is 4.89 <cite>stormo1994</cite>.

Wild-type CDS and new RBS has [http://openwetware.org/images/5/5e/19_Z0262_ss.pdf RBS-CDS junction] with dG = -10.40. Codon-shuffled to have dG = -2.7 kCal/mol. 


===gene 19.5===
non-essential; conserved

The calculated RBS strength is 7.01 <cite>stormo1994</cite>.

[http://openwetware.org/images/6/62/19.5_Z0261_ss.pdf RBS-CDS ss], dG = -7.93

==Possibly included==


===gene 5.7===
conserved; non-essential

The calculated RBS strength is 6.18 <cite>stormo1994</cite>.

===gene 7===
Gene ''7'' was originally defined by mutations tha tmapped between genes ''6'' and ''8'' but was subsequently shown to be two genes, ''7'' and ''7.3''.  Gp7 is a non-essential head protein.  There are an extremely small number of proteins per virion.  Not totally conserved in the T7 group.

The calculated RBS strength is 6.61 <cite>stormo1994</cite>.

===gene 18.7===
lysis associated protein

The calculated RBS strength is 7.64 <cite>stormo1994</cite>.

===gene 19.2===
not totally conserved; overlap with gene ''19''

The calculated RBS strength is 5.33 <cite>stormo1994</cite>.

===gene 19.3===
not totally conserved; overlap with gene ''19''

==Not included==
===gene 0.4===
not conserved; non-essential

The calculated RBS strength is 5.64 <cite>stormo1994</cite>.

===gene 0.5===
not conserved; non-essential

The calculated RBS strength is 5.28 <cite>stormo1994</cite>.

===gene 0.6A/B===
not conserved; non-essential

The calculated RBS strength is 5.62 <cite>stormo1994</cite>.

===gene 1.4===
not conserved; non-essential

The calculated RBS strength is 6.85 <cite>stormo1994</cite>.

===gene 1.5===
not conserved; non-essential

The calculated RBS strength is 7 <cite>stormo1994</cite>.

===gene 1.6===
not conserved; non-essential

The calculated RBS strength is 7.13 <cite>stormo1994</cite>.

===gene 1.8===
poorly conserved; non-essential

The calculated RBS strength is 7.56 <cite>stormo1994</cite>.

===gene 2.8===
not conserved; non-essential; possible homing endonuclease.

The calculated RBS strength is 4.3 <cite>stormo1994</cite>.

===gene 3.8===
not conserved; non-essential; homing endonuclease

The calculated RBS strength is 6.79 <cite>stormo1994</cite>.

===gene 4.1===
not conserved; overlaps with gene ''4''

The calculated RBS strength is 6.35 <cite>stormo1994</cite>.

===gene 4.2===
not conserved; overlaps with gene ''4''

The calculated RBS strength is 7.91 <cite>stormo1994</cite>.

===gene 4.7===
not conserved; non-essential

The calculated RBS strength is 7.34 <cite>stormo1994</cite>.

===gene 5.3===
not conserved; non-essential; homing endonuclease

Very very toxic to E. coli.  The only gene S&D couldn't clone. (FW Studier, PC)

The calculated RBS strength is 6.61 <cite>stormo1994</cite>.

===gene 5.9===
inhibits RecBCD nuclease; non-essential; not conserved

The calculated RBS strength is 6.69 <cite>stormo1994</cite>.

===gene 6.3===
poorly conserved; non-essential

The calculated RBS strength is 4.8 <cite>stormo1994</cite>.

===gene 7.7===
not conserved; homing endonuclease

The calculated RBS strength is 5.51 <cite>stormo1994</cite>.

==References==
<biblio>
#molineux2005 pmid=16054667
#stormo1994 pmid=8165145
#studier1983 pmid=6864790
#calendar2005 isbn=0195148509
</biblio>