{{Rebuilding_T7}}
==T7.2 Alpha Design==
===Design===
[[Image:T7-2 alpha design.png|thumb|An initial draft design of T7.2]]
Based on the gene, RBS, RNaseIII, and promoter classifications described below, I have constructed a draft design shown in the thumbnail to the right.  This is the component level specificaiton.  Further refinement will be needed at the sequence level (DpnI optimization, restriction sites, codon shuffling, and how much extra DNA we will have to add).

===Comments===
'''[[User:Skosuri|Sri Kosuri]] 19:17, 28 March 2006 (EST)''':  I'm wondering what to do when there are there are elements removed that cause two promoter elements to be right next to each other.  For example, in this initial design, ø1.5 and ø1.6 (standardized to ø2.5) are right next to each other because we are planning to eliminate ''1.5''.

'''[[User:Skosuri|Sri Kosuri]] 19:35, 28 March 2006 (EST)''': By the definition of what coding domains we are keeping and removing (below), I think we should probably get rid of ''0.3''.  Is there any other assignable function for this gene other than host restriction evasion?

==Functional Genetic Element Design Considerations==
===Coding Domains===
A number of the CDS on the natural T7 genome that do not contribute to phage development in the set of  conditions that we wish to assay them, and thus confound our systems-level analysis. We would like to remove a number of genes which we have reason to believe encode no observable function.

The following genes are unconserved across other T7 likes and encode no known function during T7 development:
''0.4'', ''0.5'', ''0.6A/B'', ''1.4'', ''1.5'', ''1.6'', ''1.8'', ''2.8'', ''3.8'', ''4.1'', ''4.2'', ''4.7'', ''5.3'', ''5.9'', ''6.3'', ''7'', ''7.7''.
We will begin by removing these genes first.

There is a larger subset of genes that we are interested in removing, but will continue to keep these in the genome for a variety of reasons (evolutionary conservation, contribution to development unclear).  We will it make easier to delete them in the future.  This subset include genes ''0.3'', ''1.1'', ''4.3'', ''4.5'', ''5.5-5.7'', ''5.7'', ''6.5'', ''18.5'', ''18.7'', ''19.2'', ''19.3'', ''19.5''.

For details on individual genes, check out the [[T7.2/gene specification|gene specification]] page.

===Ribosome Binding Sites===
We will standardize the use of Ribosome binding sites (RBS).  We will assign one of these parts to each coding domain based on both knowledge of the amounts of total protein production necessary, as well as computational analysis of the existing ribosome binding sites strength.  We will standardize to ribosome binding sites that were characterized by initial experiments by Heather Keller to experimentally charactize all the natural RBS strengths: <br>

'''<bbpart>Z0261</bbpart> -- strong RBS '''-- based on gene 5 RBS<br>
'''<bbpart>Z0262</bbpart> -- medium RBS '''-- based on [http://parts2.mit.edu/r/parts/partsdb/view.cgi?part_id=147 BBa_B0030]<br>

For details on individual genes, check out the [[T7.2/gene specification|gene specification]] page.

===Host Promoters===
We will encode only the the 3 strong host promoters, A1, A2, and A3.  We will actively take steps to remove the other host promoters identified in the [[T7.1/Reannotation|annotation]].  Finally, we will keep the ''boxA'' anti-termination site intact.  It is worth considering if we should remove all but one of the T7 promoters, such as A1.

'''A1,A2,A3,BoxA region -- <bbpart>Z0294</bbpart>'''<br>

====Removed Host Promoters====
'''B''': Located mostly in gene 0.5, which is not encoded<br>
'''C''': Only -35 region is included within coding domain of gene 0.7; no further mutation necessary<br>
'''E''': Strong -10 region mutated, -35 region not included<br>
'''S1''': -35 and -10 regions edited to remove promoter<br>
'''S2''': -35 region edited, -10 region already nonoptimal<br>
'''S3''': -35 and -10 regions edited to remove promoter<br>
'''S4''': -35 and -10 regions edited to remove promoter<br>
'''S5''': Located within gene 5.3 which is not included<br>
'''S6''': -35 and -10 regions edited to remove promoter<br>
'''F1''': Only -35 region is included within coding domain of gene 6.5; no further mutation necessary<br>
'''F2''': -35 region mutated, -10 region already nonoptimal<br>

===Phage Promoters===
We want to standardize the phage promoters that we encode onto the genome.  In our models, we split the promoters into three class; weak binding and processivity, weak processivity, or strong promoters.  The natural T7 promoters on the T7 genome are listed below, along the classification they fall into.  The promoters in bold are the canonical promoter in each class that have the most kinetic studies associated with them.  Thus we will use the three promoters in bold as the prototype sequences in each class that we will standardize the other promoters to. <br>
øOL -- weak processivity (-11,1) -- standardized to <bbpart>Z0253</bbpart><br>
ø1.1A -- weak binding & processivity (-17,2,4) -- standardized to <bbpart>Z0252</bbpart><br>
ø1.1B -- weak processivity (3,4,5) -- standardized to <bbpart>Z0253</bbpart><br>
ø1.3 -- weak binding & processivity (-5,5) -- standardized to <bbpart>Z0252</bbpart><br>
ø1.5 -- weak processivity (-2,3,4,5) -- standardized to <bbpart>Z0253</bbpart><br>
ø1.6 -- weak processivity (-2,3,4,5,6) -- standardized to <bbpart>Z0253</bbpart><br>
'''ø2.5 -- weak processivity (-1,1,4,5) -- defined as <bbpart>Z0253</bbpart>'''<br> 
'''ø3.8 -- weak binding & processivity (-13,-12,-11,-2) -- defined as <bbpart>Z0252</bbpart>'''<br>
ø4c -- weak binding & processivity (-17,-13,2) -- standardized to <bbpart>Z0252</bbpart><br>
ø4.3 -- weak processivity (-2,3,4,5,6) -- standardized to <bbpart>Z0253</bbpart><br>
ø4.7 -- weak binding & processivity (-17,-16,-13,3,4,5,6) -- standardized to <bbpart>Z0252</bbpart><br>
ø6.5 -- strong -- standardized to <bbpart>Z0251</bbpart><br>
ø9 -- strong -- standardized to <bbpart>Z0251</bbpart><br>
'''ø10 -- strong -- defined as <bbpart>Z0251</bbpart>'''<br>
ø13 -- strong -- standardized to <bbpart>Z0251</bbpart><br>
ø17 -- strong -- standardized to <bbpart>Z0251</bbpart><br>
øOR -- strong -- standardized to <bbpart>Z0251</bbpart><br>

===Terminators===
We will keep our defiinitions for the terminators as they were in [[T7.1]].  TE and Tø will remain intact, while effort will be made to not alter the CJ terminator.

'''TE -- <bbpart>Z0271</bbpart>'''<br>
'''Tø -- <bbpart>Z0272</bbpart>'''<br>

===RNA===
====RNaseIII sites====
We will use the R0.3 as a standard RNaseIII site for all encoded RnaseIII sites in T7.2 (except R1.1, which lies in the primary origin of replication).

'''R0.3 -- <bbpart>Z0273</bbpart>'''<br>
===Miscellaenous Elements===
====Terminal Repeats====
'''Left End -- <bbpart>Z0292</bbpart>'''<br>
'''Right End -- <bbpart>Z0293</bbpart>'''<br>
====Primary Origin of Replication====
The primary origin of replication is located at the phi1.1A/R1.1/phi1.1B locus.  In order to not disturb this critical section of the T7 genome, we will not standardize the promoters or RNAseIII site located in this section.  

'''Primary Origin of Replication -- <bbpart>Z0291</bbpart>'''<br>

===Other Changes===
====Reporters====
====DpnI optimization====
*[http://openwetware.org/wiki/Image:DpnIproposal.doc Word file proposal for DpnI remapping]
*[http://openwetware.org/wiki/Image:DpnI.pdf PDF map of resulting restriction pattern]
*[http://openwetware.org/wiki/Image:DpnI.M1.xls Excel file containing sites on DNA to be changed]
*[http://openwetware.org/wiki/Image:DpnI.analysis.xls Excel file containing graphical depiction of restriction fragments]

====Restriction enzyme considerations====
==Assemblies==
===Component Makeups===
====In Words====
Left End -- Phi2.5 -- A1,A2,A3,BoxA -- R0.3 -- 0.3 -- R0.3 -- 0.7 -- R0.3 -- 1 -- ORI -- 1.1 -- 1.2 -- phi 3.8 -- R0.3 -- 1.3 -- TE -- phi2.5 -- 1.7 -- 2 -- phi2.5 -- 2.5 -- 3 -- 3.5 -- R0.3 -- phi3.8 -- 4A/B -- phi2.5 -- 4.3 -- 4.5 -- R0.3 -- phi3.8 -- 5 -- 5.5 -- 5.7 -- 6 -- R0.3 -- phi10 -- 6.5 -- 6.7 - 7.3 -- 8 -- phi10 -- 9 -- phi10 -- 10 -- Tphi -- 11 -- 12 -- R0.3 -- phi10 -- 13 -- 14 -- 15 -- 16 -- phi10 -- 17 -- 17.5 -- 18 -- R0.3 -- 18.5/18.7 -- 19/19.2/19.3 -- phi10 -- 19.5 -- Right End

====In BioBrick Numbers====
Z0292.Z0253.Z0294.Z0273.Z0261.Z0158.Z0273.Z0262.Z0106.Z0273.Z0262.Z0107.Z0291.Z0262.
Z0108.Z0261.Z0115.Z0252.Z0273.Z0261.Z0109.Z0271.Z0253.Z0262.Z0151.Z0261.Z0116.Z0253.
Z0262.Z0152.Z0262.Z0118.Z0262.Z0119.Z0273.Z0252.Z0262.Z0120.Z0253.Z0261.Z0138.Z0261.
Z0139.Z0273.Z0252.Z0261.Z0121.Z0261.Z0160.Z0261.Z0154.Z0273.Z0251.Z0262.Z0159.Z0261.
Z0123.Z0262.Z0124.Z0262.Z0125.Z0251.Z0262.Z0126.Z0251.Z0261.Z0127.Z0272.Z0261.Z0128.
Z0262.Z0129.Z0273.Z0251.Z0262.Z0155.Z0262.Z0156.Z0262.Z0132.Z0262.Z0133.Z0251.Z0262.
Z0134.Z0262.Z0157.Z0261.Z0136.Z0273.Z0261.Z0144.Z0262.Z0137.Z0251.Z0261.Z0148.Z0293

====Initial Design====
'''Initial Compilation: <bbpart>Z0501</bbpart>'''