Questions 1 and 2 are theoretical but they should help prepare you to interpret the results you will collect next time. 

<b>1.</b> You have purchased some supercompetent bacteria that are provided at a transformation efficiency of 10<sup>9</sup> colony forming units/ug of DNA. You transform the cells with 1 ng of plasmid DNA and plate 1/1000th of the cells. How many colonies do you expect? 

(1 ng DNA)*(1 ug/1000 ng)*(10<sup>9</sup> colonies/ug)*(1/1000) = about 1000 colonies.  

Next you transform another aliquot of cells, also at 10<sup>9</sup> colony forming units/ug of DNA, with 2 &mu;l of plasmid DNA. You spread 1/100th of the cells and find 50 colonies growing on the plate after 24 hours at 37&deg;C. What is the concentration of plasmid? 

(50 colonies)*(1 ug DNA/10<sup>9</sup> colonies)*100/2ul = 2.5 x 10<sup>-6</sup> ug/ul.

<b>2.</b> To illustrate your understanding and the importance of the controls you performed today, please write a one-sentence interpretation for each of the following transformation outcomes. 

*'''Outcomes:'''
**Outcome 1: no colonies on any plate. Maybe something is wrong with the Kanamycin resistance gene.
**Outcome 2: thousands of colonies on all the plates. Maybe we forgot to add the Kanamycin, or maybe the plates were contaminated with Kanamycin resistant bacteria.  
**Outcome 3: approximately the same number of colonies on the backbone+ligase+kill cut as the backbone+insert+ligase+kill cut. Maybe the kill cut did not work because the restriction enzyme was denatured.
**There may be more than one valid interpretation for some of the data (only one answer for each is required for the assignment).

<b>3.</b> Next time you will isolate DNA from four transformants and begin to characterize the plasmids in these bacteria. To prepare for this experiment, you should draw a plasmid map of the M13K07 genome. Start by printing out the M13K07 plasmid map from NEB by using their [http://tools.neb.com/NEBcutter2/index.php NEB Cutter tool], selecting M13K07 from the "Viral and phage" drop down menu on the right, changing the default minimum ORF to 25 amino acids (do you remember which of the M13 proteins are very small?), and finally telling the program that you are entering circular DNA. Modify the map by hand to indicate which restriction site you are changing, which enzymes you are adding, and how many basepairs of DNA this modification needs. Next, use the plasmid map to help you plan at least two restriction digests that will confirm the presence of the oligonucleotide insert. Recall that the lab does not have every enzyme available so you should double check your idea against the [[20.109(F07): Available enzymes | list of available enzymes]]. It will help to read the introduction for the next lab before you complete this part of the assignment. Be sure to predict the size of the fragments you expect when the plasmid does and doesnâ€™t have the oligonucleotide insert. Also include reaction conditions such as buffer and temperature. Use the [http://www.neb.com/nebecomm/default.asp NEB website] for details on various enzymes and reaction conditions.
<center>

{| border="1" 
| <b>Diagnostic digest 1</b>
| plasmid with insert
| plasmid no insert
|-
| Enzyme(s) used 
| EcoRI and HindIII
| 	 EcoRI and HindIII
|-
| Buffer used
| NEB buffer 2
|	 NEB buffer 2
|-
| Temperature
| 37 degrees C
|37 degrees C
|-
| Predicted fragments		
| about 5200 bp and 3500 bp
|8700 bp
|-
| 
|
|
|-
! Diagnostic digest 2	 	
| 
|
|-
| Enzyme(s) used		
| HpaI and XhoI
|HpaI and XhoI
|-
| Buffer used		
| NEB buffer 4
| 	 NEB buffer 4
|-
| Temperature		
| 37 degrees C
|37 degrees C
|-
| Predicted fragments		
| about 5700 bp and 3000 bp
| 8700 bp
|}
</center>
<br>
<b>4.</b> Based on the results of your plaque assay, what is the titer of each stock solution of phage? Please show your work. If the plaques appeared different, please consider how the phage genomes differ (M13K07 is a "helper phage" while E4 is identical the the M13 genome except four glutamic acids are presented on the N-terminus of the p8 protein) and suggest how these differences might account for the differences in plaque morphology. <br>

K07: (80 pfu)*(10<sup>6</sup>)*(200/10) = 1.6 x 10<sup>9</sup> pfu in stock.

E4: (783 pfu)*(10<sup>6</sup>)*(200/10) = 1.6 x 10<sup>10</sup> pfu in stock.

The M13K07 genome is more different from wild type than the E4 genome. Subsequently, M13K07 does not have as efficient a life cycle as the E4 virus because E4 retains most of the fine tuning it received from nature while M13K07 lost much of its fine tuning when it was genetically engineered. This explains why the E4 plaques were larger than their M13K07 counterparts. Also, the glutamic acids would make the E4 more water soluble, and this might help the bacteriaphages travel more quickly from bacteria to bacteria. 

<br>
<b>5.</b> Read the article by Chan, Kosuri and Endy. "Refactoring bacteriophage T7" ''Nature/EMBO Molecular Systems Biology'' 13 September 2005 [[doi:10.1038/msb4100025]] and [http://www.nature.com/msb/journal/v1/n1/full/msb4100028.html News & Views]. Come prepared to discuss this paper during lab next time. To guide your reading and test your understanding, try to answer the following questions (<b>note: these questions are just to guide your reading and the answers do not have to be turned in</b>):
*'''from the Introduction:'''
**What is "refactoring" and what makes is T7 an attractive candidate for this approach?
**What experimental techniques give us "component level" understanding, i.e. allow us to attribute particular functions to particular sequences in the genome? How completely can "component level" understanding provide "system level" understanding?  
**How predictive have computational and quantitative models for T7 behavior proven to be? What's important about predicting behavior?  
*'''from the Results:'''
**What design principles were the authors pursuing? How well do these map to our class effort at M13 re-design? 
**Was the entire T7 genome refactored? 
**What techniques were used to verify the refactoring? What techniques were used to evaluate it? 
*'''from the Discussion:'''
**How do the authors' findings extend knowledge of T7 biology? 
**Does T7.1 resolve disagreements between model-based behavior predictions and those that are observed though experimental approaches? 
**Could nature have produced the T7 phage that now exists in the Endy lab in Building 68?
**What's next for this phage?