===Ab hac una re===
17 Nov 2007

SLA sites in MF genome.  Calcutt99a shows motif for an unknown protein site in lipoproteins, concensus sequence TDXGXXDDKSFNQSAWE.

Found a match in AAT76025, possible match in AAT75382.

SLA-2 motif Rosati99, IGVDxDQ


 
3 June 2007

N15 cloning vectors, plasmids:
* AF064539
* EF583812  pJAZZ-OC.  Copies 24802 (telomere site) - 35005  [MCS, resistance] 23696 - 24802 (telomere site)
* DQ391279

Sloning patent: EP1411122A1

22 Nov 2006

* pRML1-L plasmid constructed.
* 401349-402237 fragment from MF
* pRML2-R plasmid constructed.
* 400990-400038 fragment from MF
* potential mutation in MF sequence at TATTTTCTTCTc/tTTTTCAT  c sequenced in the plasmid; t present in genome sequence.
* coordinates of MF genome region is 400012-400990 (RH); gap 400991-401348; 401349-402253 (LH)


17 Oct 2006

Errors in the primers:

* gcgagtgcggccgc TGATTGAGGAGGAAATTTTGAA, LH-chr2
** LH-Chr primer was wrong.  Misprimes at another chromosome location, but no easy change.
* tagactgcggccgc TCCAATTCCTTGTCCAGCTC,  MID-chr2
** MID-chr primer had strong primer-dimer due to gaga extension.
* gagagaattc TTTTTGGTTCTGTTCCACCAT, LH-tel2
** LH-tel had mispriming potential at alternative site in chromosome.  


10 Oct 2006

Finished sequencing pRML1 / pRML2.  Tiime to go on to contstruction of the YAC.  Primers for a 1000 bp sequence near the terminator.

* Primers for the gene 399881 - 40119

 OLIGO            start  len      tm     gc%   any    3' seq 
 LEFT PRIMER        132   20   59.41   40.00  8.00  0.00 TGAATGCCGGGATTAGTTTT
 RIGHT PRIMER      1110   20   60.92   50.00  5.00  0.00 TCCAGGACTTGGTGGTTTTG  (terminator is at the 5' end of this)
 SEQUENCE SIZE: 1398
 INCLUDED REGION SIZE: 1398
 PRODUCT SIZE: 979, PAIR ANY COMPL: 4.00, PAIR 3' COMPL: 0.00

EcoRI site goes next to the terminator,  NotI site goes away from terminator

* gagagagaattc TCCAGGACTTGGTGGTTTTG, RH-tel
* gagagcggccgc TGAATGCCGGGATTAGTTTT, RH-chr

EcoRI does not cut this sequence



* Primers for the gene 401278-404xxx (Chitinase)
 OLIGO            start  len      tm     gc%   any    3' seq 
 LEFT PRIMER         72   20   59.00   40.00  4.00  2.00 TTTTTGGTTCTGTTCCACCA  (terminator is at the 5' end of this)
 RIGHT PRIMER       976   22   58.67   31.82  6.00  3.00 TGATTGAGGAGGAAATTTTGAA
 SEQUENCE SIZE: 1123
 INCLUDED REGION SIZE: 1123
 PRODUCT SIZE: 905, PAIR ANY COMPL: 5.00, PAIR 3' COMPL: 1.00

* gagagaattc TTTTTGGTTCTGTTCCACCA, LH-tel
* gagagcggccgc  TGAATGCCGGGATTAGTTTT, LH-chr

EcoRI does not cut this sequence.


Sequence at 200Kbp.  EcoRI site at the low base position.

 OLIGO            start  len      tm     gc%   any    3' seq 
 LEFT PRIMER         16   20   59.94   45.00  3.00  1.00 TTGGAACAGGTGGATGACAA
 RIGHT PRIMER      1042   20   60.20   50.00  4.00  2.00 TCCAATTCCTTGTCCAGCTC
 SEQUENCE SIZE: 2001
 PRODUCT SIZE: 1027, PAIR ANY COMPL: 5.00, PAIR 3' COMPL: 3.00

* gagagaattc TTGGAACAGGTGGATGACAA, MID-tel
* gagagcggccgc TCCAATTCCTTGTCCAGCTC,  MID-chr

No EcoRI sites in this sequence


----
9 September 2006

primers for further sequencing of pRML1/2

primers for aacC1 coding region, formation of the promoter sequence

finish ligation tests

page for Lim

[[Knight:Part construction plans]]


----
3 September 2006

yebU also is a close relative of AAT75553.  Andersen06.  E-25 match.


----
2 September 2006

Established role of protein AAT75660, previously unknown function, now clearly yggJ or rsmE (Basturea06, PMID: 16431987).  Methyltransferase modifying the 16S base at location U1498.  E value match 2E-16 with the E. coli gene.

No evidence for tusBDC complex in MF.

ksgA (rsmA), another 16S ribosomal RNA modifying protein is AAT75361.  VanBuul85

rsmB is AAT75553 (5-methyl cytosine transferase for tRNA and rRNA)  Tscherne99

rsmC appears to be missing.  


----
1 September 2006

design and order fluorescent primers for debugging construction plasmid PCR, cutting and ligation.  Pacific blue and Alexa Fluor 488 appear to work well with UV excitation.  Prefix-R and Suffix-F ordered.

Miniprep of pRML1, pRML2, pYAC4.  Design and order sequencing primers and half of the needed construction primers for YAC left and right arms.  Remainder awaiting sequencing results.


----
29 August 2006

----

Order medium components (Spencer 93)

Sequencing primers for pRML1, pRML2
* pBluescript II SK+
* pCGS966
* URA3
* TRP
* ARS1
* ARSH4
* CEN4

Construct parts from Left, Right telomere, Left, Right chromosome ends

Construct parts from 1000 bp sequences at MF terminator

----

17 August 2006

----

Wood's Hole ISAT
TCDB transporter database.  Useful in characterizing and searching for transporter protein properties and motifs.

----

Demo of "cool stuff" minty fresh E. coli and banana smelling E. coli.   Discussions with Carl Landweber and Bob Colwell especially imporant.  Possible teaming on energy production with SRI / Sandia.

----

12 August 2006

----

Sci Foo camp.  Jonathan Eisen discussions.  Dyson / Danny / Brand

----

10 August 2006

----

Peter Karp.  Discussons on SRI EcoCyc/Biocyc uses for Mesoplasma florum design and editing.

----

9 August 2006

----

Rocha paper on survey of recombination systems in sequenced bacteria.  Florum has RecO, RecR, but not RecF in the RecFOR system.  RecA present.  RecD present.  RuvAB present, ruvC not, but replaced with RecU.  yqgF gene part of the recomination system.   PriA missing.  RecBC missing, RecD form is different from the form found in organisms with RecBC also.

----

Received yeast strains and plasmids from Weiss lab.  Confirmed that they cannot PCR through the centromere CDS II region.

----

Codon usage of the P1014 cassette.  CTC codon (Lys) is rare and used 4+ times.  CGG is not present in the CDS.  Three GATC sequences.

----

7 August 2006

----

SMC proteins maintain chromosome condensation.  Co-factors scpA and scpB.  Me. florum proteins are AAT7589, AAT75908, AAT75907 respectively.  B. subtilis homologs: P51834, P35154, P35155.  Volkov03.  Notarnicola91 reports SMC protein in M. hyorhinis (P115).

----
Compromise codon usage:  multiply codon probabilities for corresponding codons from each organism, then take the square root (geometric mean).  Renormalize within each amino acid to establish new relative probabilities.
----