[Editorial note: The information on this page pertains to bacteria and primarily ''Escherichia coli''.  If you're interested in adding information on transposable elements in other organisms, please move the contents of this page to [[Transposable elements/Bacteria]].]

==Overview==
'''Transposable elements''' or '''transpososns''' are mobile segments of DNA that relocate in the genome occasionally.  '''Transposition''' refers to the movement of these segements.  Transposons must encode a '''tranposase''': an enzyme which makes two single-stranded breaks in the target DNA which is a necessary step for transposition.  Certain positions in ''Escherichia coli'' seem to be more or less favorable targets for transposon insertion (hot or cold spots).  

==Nomenclature==
Transposons are named by the abbreviation Tn followed by a number (like Tn''1'', Tn''2'', Tn''3'' and so on).  Genes on transposons are referred to by their genotypic name (like Tn1(''ampR'')).  The transposons that were first  identified did not carry any host genes and were called '''insertion sequences''' or '''IS elements''' (labelled IS''1'', IS''2'' and so on).  When transposons occur within genes, they are designated by the name of the gene, the allele number, two colons and the transposon name (like ''lacZ87''::Tn''3'').   

==Classification==

There are 4 classes of transposons ...

#Insertion sequences or IS elements
#*don't contain any host genes
#*termini have inverted repeat sequences
#*can frequently contain transcription-stop signals or chain termination mutations in all reading frames
#*contain two coding sequences
#Composite transposons or composite type-I transposons
#*an antibiotic resistance gene flanked by two highly similar copies of an IS element (either direct or inverted repeats)
#*sometimes the terminal IS elements can transpose alone
#*transposons in plasmids can act as two transposons
#The Tn3 transposon family
#*~5000bp
#*contains &beta;-lactamase (for ampicillin resistance), a transposase and a resolvase (for transposition)
#*contain short inverted repeats but are not flanked by IS elements
#The transposable phages
#*Mu and D108 phages use tranposition in their life cycles (for DNA replication)

===Insertion elements===

{|{{table}}
|-
!align="center" style="background:#f0f0f0;"|Element
!align="center" style="background:#f0f0f0;"|Number and copies and location
!align="center" style="background:#f0f0f0;"|Length
!align="center" style="background:#f0f0f0;"|Genbank Accession
|-
|IS''1''
|5-8 in chromosome
|768
|
|-
|IS''2''
|5 in chromosome; 1 in F
|1327
|
|-
|IS''3''
|5 in chromosome; 2 in F
|1258
|
|-
|IS''4''
|1 or 2 in chromosome
|1426
|
|-
|IS''5''
|Unknown
|1195
|[http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&val=149079 J01734]
|-
|Tn''1000'' (&gamma;&delta;)
|1 or more in chromosome; 1 in F
|5980
|
|-
|}

===Composite type I tranposons===

{|{{table}}
|-
!align="center" style="background:#f0f0f0;"|Element
!align="center" style="background:#f0f0f0;"|Genes carried
!align="center" style="background:#f0f0f0;"|Length in bp
!align="center" style="background:#f0f0f0;"|Terminal IS element (Length)
!align="center" style="background:#f0f0f0;"|Relative direction of terminal IS elements
!align="center" style="background:#f0f0f0;"|Genbank Accession
|-
|Tn''5''
|kan
|5818
|IS''50''(1533)
|Inverted
|
|-
|Tn''9''
|cam
|2638
|IS''1''(768)
|Direct
|
|-
|Tn''10''
|tet
|9300
|IS''10''(1329)
|Inverted
|[http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?db=nucleotide&val=37987870 AJ601386]
|-
|Tn''204''
|cam,fus
|2457
|IS''1''(768)
|Direct
|
|-
|Tn''903''
|kan
|3094
|IS''903''(1057)
|Inverted
|
|-
|Tn''1681''
|ent
|2088
|IS''1''(768)
|Inverted
|
|}

==Why do I care?==
Transposable elements can hop into your plasmids with out warning.  Note that this can happen not just during cloning but also when transforming intact plasmid into cells.  [[Tom Knight]], [[Jason Kelly]] and [[Reshma Shetty]] have each observed this phenomenon.  

==References==
<biblio>
#MicrobialGenetics isbn=0-86720-248-3
</biblio>

[[Category:Escherichia coli]]