* [http://genereg.molgen.mpg.de/ Vingron group] Max Planck, Berlin, Germany:
** Stefan Roepke: reg database, matrix comparison 
** Szymon Kielbasa: Matrix comparison, motif discovery

* [http://rulai.cshl.edu/ Michael Q. Zhang group]
** lot's of good software: matrix comparison (matcompare), (DME) motif discovery, chip-chip analysis, regulatory database, etc. etc.

* [http://rana.lbl.gov/ Eisen Lab], Berkeley, CA: 
** [http://rana.lbl.gov/~dan/index.html Daniel Pollard] [http://rana.lbl.gov/~dan/publications.html (pubs)] is working on evolution of binding sites. He did a benchmark on alignment programs for noncoding seqs and an algorithm that searches matrices in multiple alignments.
** These people are part of the [http://bdtnp.lbl.gov/ berkeley drosophila trascnription network project]

*[http://www.sladeklab.ucr.edu/aboutpi.html  Sladek lab], Riverside, CA;
** HNF4  research.

* [http://web.wi.mit.edu/fraenkel/people.html Fraenkel lab]
** Yeast (science 2003), motif discovery, TAMO, very good software

* [http://biowiki.org/HolmesLab Ian Holmes Lab]
** more like conservation and alignments, not too much about binding sites

* [http://www.bx.psu.edu/miller_lab/ Miller Lab]
** Whole Genome alignments and search of alignments for conserved sites (blastz, pipmaker, etc)

* [http://www.cbse.ucsc.edu/staff/hausslerlab.shtml David Haussler Group, UCSC]:
** [http://www.google.com/url?sa=t&ct=res&cd=1&url=http%3A%2F%2Fwww.cse.ucsc.edu%2F~jill%2F&ei=uEBXRIroKaGuogKW6az_Bw&sig2=yK0JZtxhCHOLjXodKFUdgg Gill Bejerano]: Ultraconserved Region

* [http://www.ucmb.ulb.ac.be/~jvanheld/ Jacques van Helden]: Motif Discovery with (spaced) words

* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=PureSearch&db=pubmed&details_term=%22Kadonaga%20J%22%5BAll%20Fields%5D Jim Kadonaga] coauthor of about 100 articles about transcription factors and interactions

* Students:
** [http://www.cs.washington.edu/homes/jcarlson Jonathan Carlson] adds another motif discovery algorithm and is confident that it outperforms others.