There are some websites where you can see Precalculated alignments and Matrix hits. Similar to ready-made food, I don't like them and they are not applicable for my model organism.

* [[http://mordor.cgb.ki.se/cgi-bin/CONSITE/consite/ Consite]] Any 2 sequences and hits on Transfac
* famous [[http://genome.ucsc.edu UCSC]] Genome Browser has a TFBS-track for the human genome, where all hits against Transfac Pro 8.3 are documented and can be downloaded as GFF (hint: What can you do with many hits?)
* [http://corg.molgen.mpg.de/ corg] shows alignments of upstream regions of many model species and can display conserved binding sites. Unfortunately, I have only seen pairwise alignments. Look at UCSC for multiple alignments with an easy to use interface.