* Alignment Algorithms:
** [http://www.drive5.com/muscle Muscle] quotedon bio-bb board to be better than clustal and t-coffee, according to Casey Bergman's analysis and posters at SMBE06 to be the best multiple aligner out there. Used by [http://www.metazome.net/ metazome]
** [http://www.biophys.kyoto-u.ac.jp/~katoh/programs/align/mafft/ mafft] according to genome technology 9/06 sometimes better than clustalw
** [http://genome.lbl.gov/vista/index.shtml AVID+VISTA] is the standard tool to produce multiple alignments (AVID) and plots (VISTA) where you can see which bases are conserved. The global aligner AVID can be replaced by LAGAN (pairwise), MLAGAN (multiple) or SLAGAN (similar to TBA, tolerates inversions/translocations, output are MANY alignments).
** [[http://www.biostat.wisc.edu/~cdewey/mercator/ Mercator]] actually a pipeline combining BLAT and MAVID to obtain whole-genome exon-anchored global alignments by using best-matches to form "runs" (~ortholog genes) that can then be aligned with their flanking sequences with mavid
** [[ftp://ftp.ebi.ac.uk/pub/software/wise2 PromotorWise]] designed for cis-regulatory sequences, but only for two sequences (NB: there is a second dir on the same server with the same name but older versions, make sure to get the right one. This is the correct link.)
** Orca (used by consite)
** [http://dialign.gobics.de/chaos-dialign-submission Dialign2] Two sequences
** [http://msa.cgb.ki.se/cgi-bin/msa.cgi Kalign] aimed rather at proteins, the newest in the crowd, claims to be better than TCoffee and 50 times faster.
** [http://www.cbs.dtu.dk/services/MatrixPlot/introduction/index.html MatrixPlot] Not really an alignment, more a dotplotter that can add information content to its plots. Interesting visualization, though.
** [http://probcons.stanford.edu/ Probcons] only proteins (!) is the best but slowest aligner according to genome technology sept 2006 page 17 

* Visualizing alignment hits:
** [http://www.jalview.org/ jalview] good old multiple alignment viewer, better for proteins than dna but with clustal and mafft included
** [http://alggen.lsi.upc.es/recerca/align/mgcat/intro-mgcat.html M-GCAT] for many and LONG multiple alignments
** [http://www.sanger.ac.uk/Software/ACT/ ACT] for synteny plots, based on BLAST-results
** Apollo also has a synteny mode, which can be configured to read something like GFF-files

* Aligning non-coding fragments:
** Phylogenetic distances < mammals: You can use Vista with it's alignment algorithms AVID(global), LAGAN(global) or SLAGAN(seeded-global)
** High Distance, e.g. human-fish: Use pipmaker, as it is based on local alignments. Global Aligners will noch find synteny  if you 10 MB of sequence on human and only 2 MB on zebrafish.

Also see the page on [[Finding interspecies-orthologs]] as some of the ortholog predictions already include the alignments.