*[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16839656&itool=pubmed_DocSum a 3' utr region contacts a 5' utr region and influences transcription]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17217514&query_hl=18&itool=pubmed_docsum motif discovery, microarray, muscle, human]
* whole-genome scanning again: [http://nar.oxfordjournals.org/cgi/content/full/33/11/3479?ijkey=f1709b51ef59f4334df8c7564b8a9f1b20781ed5#SEC2 retina specific factors and their sites]
* [http://www.sciencemag.org/cgi/content/short/315/5809/233 microfluidics to the rescue:] using a microfluidics platform, binding sites were determined and the genome scanned for them.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17046742&query_hl=11&itool=pubmed_docsum  anther whole-genome promoter-scanning search:] pan-neuronal genes in general
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17187676&query_hl=11&itool=pubmed_docsum yet another whole-genome promoter-scanning example:] c.elegans, ciliary/ciliopathy neurons
* [http://www.sciencedirect.com.gate1.inist.fr/science?_ob=ArticleURL&_udi=B6WG1-4MMP2CV-3&_coverDate=12%2F22%2F2006&_alid=519360218&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=6809&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=37a652e861753450b20c663fe87a1f1e#secx13 SMARTs are also being systematically analyzed]: Interesting is the short list of smart-predictors that are benchmarked against the data set. I have exported the data from the paper into UCSC but haven't seen a lot of conservation in these SMART-regions.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16966362&itool=pubmed_DocSum trying to learn transcription factor concentrations from chip and network data]
* [http://www.genetics.org/cgi/content/full/174/2/735 hedgehog signalling regulates at least 220 different genes (supression of HH signalling combined with microarray analysis)], the gli motifs is overrepresented upstream of these genes
* [http://www.newscientist.com/article.ns?id=mg19325853.200&feedId=online-news_rss20 Motif discovery with a twist:]Searching for the most UNDERreprensented motifs gives "nullomers", 15bp-long motifs that are completely avoided by nature and cannot be found one single time in genbank. Cool idea.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16816430&itool=pubmed_DocSum Yet another putative link between repeats and regulatory sequences:] regulatory elements that are repeated are prone for recombination and quickly create new combinations of existing regulatory elements
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17084362&itool=pubmed_DocSum ever wondered why only certain alleles is expressed?] Zac1 seems to be a factor that restricts expression to alleles
* [http://hedgehog.sfsu.edu/ complex - the sonic hedgehog signaling database:] Reviews, papers, diagrams, gene lists, diseases, etc.
* [http://www.genome.org/cgi/content/full/17/1/117 =2&itool=pubmed_docsum  now even in plants: motif discovery and alignment combined, phylogenetic footprinting (van der peer)]

* [http://genomebiology.com/content/pdf/gb-2006-7-11-r102.pdf a "boosted" classifier (motif discovery) is trained to find decision trees that describe the relation between motif substitution patterns and the outcome of the chip-chip experiment] 
* [http://www.biomedcentral.com/content/pdf/1471-2105-7-478.pdf the yeast regulatory network  contains certain types of motifs (feed-forward, sinle link/convergence + mutual regulation) and not other types (divergence, cascade, divergence)], is organized into "organizers", "Regulatory connections inside organizers are dense, while inter-organizer connections are sparse.", interactions as "waves" (davidson calls that batteries since 20 years) that are triggered by few signals 
* [http://www.biomedcentral.com/1471-213X/6/50 regulation of OTP in zebrafish: HH, FGF, nodal]
* [http://www.molecular-cancer.com/content/pdf/1476-4598-5-60.pdf nice review about chromatin structure and inf jumps into neuronal genes, can act on methylation patterns]
* Coevolution of binding sites and transcription factors: [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16143608&itool=pubmed_DocSum new hybrid-based experiments to prove it]
* Review: [http://www.nature.com.gate1.inist.fr/hdy/journal/v97/n3/full/6800869a.html review about computational versus biological methods] in uncovering binding sites in flies. Conclusion: back to the bench :-)
* Silencing useless binding sites
** [http://www.nature.com.gate1.inist.fr/nsmb/journal/v10/n11/full/nsb1103-879.html Spt6 and FACT silence superflous binding sites while PolII is elongating the DNA]
* Protein Microarrays
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17018151&itool=pubmed_DocSum algorithm to design protein microarrays]
* Success Stories (scanning & discovery)
** [http://www.biomedcentral.com/1471-2164/3/25 a real-life example] how to apply promoter prediction and binding site scanning (NNPP and alibaba)
** [[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15314660&itool=pubmed_DocSum Application of motif discovery]]
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16289377&itool=pubmed_DocSum 
Application of motif search but simple ERE-steroird inverted repeat model]
** [http://www.biomedcentral.com/content/pdf/1471-2164-7-21.pdf Motif Discvoery] finds a 19bp motif, software used was ClustalX (!!), model: bacteria
**[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16412085&itool=pubmed_DocSum 
** [http://dmd.aspetjournals.org.gate1.inist.fr/cgi/content/full/34/1/138 matinspector] to discover and prove one binding site. Important: This binding site is NOT conserved at all among most mammals!
** [http://molehr.oxfordjournals.org/cgi/content/abstract/12/1/31?ct Vasarhelyi] seach for overrepresented binding sites in micrarray data
** [http://eutils.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16644048&itool=pubmed_DocSum SNPs and TFBS binding sites] in two mouse genes: SNPs change expression
** [http://nar.oxfordjournals.org/cgi/content/full/33/1/409 Applikation of MODEL to upstream sequences of microarray chosen genes]


* Whole genome searches for sites:
** [http://www.genome.org/cgi/content/abstract/16/10/1208?ct starting from only a single site], by looking at the conservation pattern, you can build a completely network by conservation and PWM matches.
* Coding theory and regulation
** [http://www.biomedcentral.com/1471-2164/7/239 the number of transcription factors] are limited by genome size and information content of binding domain

* Relation between cis-regulatory sequences and trans-acting factors
** Ohler Cornell 2000, pmid 10679430: Vnd/nkx, ind/gsh, and msh/msx: conserved regulators of dorsoventral neural patterning? ... while recent loss-oncRNA-search benchmark]: Wu-Blast is better (faster, better options) than NCBI Blast, more specific algorithms are better than generic aligners (surprise, surprise)
* [http://www.genome.org/cgi/content/abstract/17/1/16?etoc indels in the human lineage] follow the trend set by HERs (see UCSC papers, Siepel etc) and cluster around trancription/regulation genes.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16922677&itool=pubmed_DocSum histones and gene regulation]: NFAT, YY, CCAAT, GATA3 recruit HDAC4 to the IL5-promoter which in turn regulates expression
* [http://bioinformatics.oxfordjournals.org/cgi/content/abstract/22/23/2870?ct background models for dna motifs: adding gc-context seems to improve them]
* [http://mbe.oxfordjournals.org.gate1.inist.fr/cgi/content/abstract/msl205v1 long-range synteny amphioxus-vertebrates:] Why doesn't anyone try to add sea-urchin + ciona + lancelet?? ([http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17182894&itool=pubmed_DocSum pubmed])
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17189297&itool=pubmed_DocSum in yeast, promotors are REALLY much better characterized than in eurcaryotes... (schneider)]
* [http://www.nature.com.gate1.inist.fr/ng/journal/v38/n10/full/ng1006-1102.html Once again, the even-skipped enhancer:] Mathematical modelling of expression using TF concentrations and binding site data (Halfon)
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16697139&itool=pubmed_DocSum conserved sequences have certain properties that allow to distinguish between exons, CNS and Utrs:] Exons/CNS have longer regions. In Exons, mutations show the 3n+2 pattern. Long CNS can perhaps be explained by overlapping TF sites.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17082449&itool=pubmed_DocSum and finally in science:] CNS evolution is faster close to neuronal genes (Prabhakar, Rubin)
* [http://www.nature.com.gate1.inist.fr/ng/journal/v38/n6s/full/ng1794.html a great review on methods in miRNA prediction/validation] from nature genetics special issue on miRNA: if just we had this in binding site predictions...
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17152067&itool=pubmed_DocSum a point mutation in distal SHH enhancers is responsible for a genetic disease]: another good reason why it is a good idea to test all these conserved sequences...
* [http://www.sciencemag.org.gate1.inist.fr/cgi/content/short/314/5805/1601 pRNA is competing for the polymerase and this regulates gene expression]. I wonder if there is something similar with enhancers competing for TFs...
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&db=pubmed&term=8673282%20 transposable elements are linked to gene expression],old review that I didn't know before, strange examples, strange conclusions, no single example where the transposabl direct binding ones identified by chipchip

* Co-occurence of binding sites
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16844711&itool=pubmed_DocSum combination in yeast] of matrix matches at preferred distances to find protein interactions
** [http://nar.oxfordjournals.org/cgi/reprint/34/17/4925 same for human factors] using tissue-based sorting

* PSSM and extensions:
** [http://nar.oxfordjournals.org/cgi/content/abstract/gkl585v1?ct graph based approach for motif recognition]
* Transcription factor binding domains:
** [http://nar.oxfordjournals.org/cgi/content/abstract/gkl683v1?ct dna elasticity rather than direct base-touching is used to recognize binding sites]
* Ultraconserved and Co:
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16705037&query_hl=3&itool=pubmed_docsum The strange story of Evf-2] which is at the same time enhancer AND ncRNA binding to a homf-function studies in flies and mice indicate that these three genes may have a conserved role in regional specification, there is no obvious conservation of the particular cell fates deriving from corresponding domains. The three-column expression pattern may thus represent a developmental mechanism that is more resistant to evolutionary changes than genetic events upstream or downstream of it.
** [http://www.nature.com.gate1.inist.fr/nature/journal/v443/n7110/full/443401a.html nature paper] describing how regulation of yeast has changed yet phenotype is almost identical between S cerevisia and C albins

* ultraconserved elements:
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&dopt=Abstract&db=PubMed&list_uids=16998490 ultras] avoid segmental duplications

* RNA and regulation:
** [http://nar.oxfordjournals.org/cgi/content/abstract/34/15/4206?etoc RNA as cofactors] in the transcriptional complex for RNA Poly II

* the "billboard" model:
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&lluence on cancer]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17082449&itool=pubmed_DocSum accelerated regions between human/monkey are non-coding and located around neural proteins (rubin) ]
* [http://nar.oxfordjournals.org/cgi/content/abstract/34/19/5638?ct yet another try to predict binding sites from TF protein structure]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17053094&query_hl=13&itool=pubmed_docsum review: locating TF binding sites (jones)]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=15475268&query_hl=9&itool=pubmed_docsum conserved fugu regions correspond to most conserved (>350bp, 77%) human-mouse regions (ovcharenko)]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16179648&query_hl=4&itool=pubmed_docsum human-zebrafish comparison and enhancer test (ovcharenko?)]
* [http://www.pnas.org/cgi/content/full/102/14/4936 the sea urchin transcriptional regulatory network (davidson) ]
* [http://www.genome.org/cgi/content/full/15/1/184 mulan tools from dcode.org (ovcharenko) ]
* [http://www.nature.com.gate1.inist.fr/nature/journal/v443/n7111/full/443521a.html The advantages of transposons, review in nature]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16970547&itool=pubmed_DocSum properties of binding sites in transfac analyzed:] GC-content and palindromic structure, depends on species and promoter/enhancer localization
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17050569&query_hl=3&itool=pubmed_Summary CREAD in action:] fax2/wnt target genes analysed and new motifs found, new factor found to be expressed in the lung and verified with rtPCR
* [http://nar.oxfordjournals.org/cgi/content/abstract/gkl734v1?ct a second type of binding sites for CRP exist that exchange a T for a C in the core site]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17055246&itool=pubmed_DocSum functional evolution and variation of non-coding dna]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16973595&itool=pubmed_DocSum The predicted NfKappab-binding sites in the HCMV gene are not functional] Surprise, surprise!
* [http://nar.oxfordjournals.org/cgi/content/abstract/gkl773v1?ct genes with intermediate complex expression have the longest introns (genome design theory)]
* [http://bioinformatics.oxfordjournals.org/cgi/content/abstract/22/21/2585?ct mirnas in arabidopsis have been copied around together with their promoters]
* [http://www.genome.org/cgi/reprint/gr.4140006v1 survey how to located binding sites in genomes]
* [http://www.genome.org/cgi/reprint/gr.4537706v1 enhancer prediction with 83% accuracy on the vista browser] see also biology paper: * [http://www.ncbi.nlm.nih.eodomain
* Contribution of transposable elements to expression: [http://www.nature.com.gate1.inist.fr/nature/journal/v443/n7111/full/443521a.html TEs can influence expression , as silencers, LINE-1 preferentiallye element really is proven to be the minimal enhancer]
* a [http://nar.oxfordjournals.org/cgi/content/abstract/34/21/e146?etoc third party] opinion about chip-chip data can confirm most findinings but warns that conclusions from some studies might not be generalizable to all transcription factors
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17132051&itool=pubmed_DocSum the correct phylogentic tree for drosophila depends on the gene you're looking at]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=14707170&dopt=Citation birectional promoters are ubiquitous on the human genome] and 90% are functional
* the protein interaction community have a "[http://www.nature.com/nbt/consult/index.html minimum information document]" that is needed for publications. Why don't we have this for protein-DNA interactions?
* an update for [http://www.sciencemag.org.gate1.inist.fr/cgi/content/abstract/295/5558/1306 an absolutely fascinating technology to capture enhancer interactions: chromatin conformation capture (3C) ] called [http://www.nature.com.gate1.inist.fr/ng/journal/v38/n11/full/ng1896.html 4C]
* [http://www.nature.com/emboj/journal/v24/n2/full/7600531a.html nature article showing that dna structure can be altered and this can be used to block a given enhancer]
* [http://www.genome.org/cgi/content/full/15/12/1675?ck=nck fish genomics review:] the 2R hypothesis, differences between assemblies, a phylogenetic tree, etc.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&db=pubmed&term=7698641 retrotransposons are transcribed and have their own promotors/enhancers]
* How NOT to use binding site databases: 
[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16843737&query_hl=27&itool=pubmed_docsum quote from abstract:] "Finally, we conducted DNAse I-footprinting assays in nuclear extracts for the 184bp region and detected two protected sequences. Data bank search indicates that these sites contain consensus binding sites for transcription factors." Wonder if they ever pasted a random sequence into transfac match... 
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17106362&itool=pubmed_DocSum transcription factors are related by OMIM], example pitx2, pax6 and foxc
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=12011440&query_hl=1&itool=pubmed_docsum  In ascidians, eletroporation of random 1kb fragments] has an enhancer-hit-rate of around 8% (levine)
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16602820&itool=pubmed_DocSum Great news for electronic freaks: all AND/OR combinations are possible for the e.coli lac operon, SNPs can change binding sites and change the corresponding logic]
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17090307&query_hlgov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17038566&query_hl=4&itool=pubmed_docsum application of RP on thyroid enhancer set]

* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?itool=pubmed_DocSum&db=pubmed&cmd=Display&dopt=pubmed_pubmed&from_uid=16752383 gene CEL is located within irx2]
* many genome sizes...
** [http://www3.interscience.wiley.com/cgi-bin/abstract/33963/ABSTRACT?CRETRY=1&SRETRY=0 old paper]

* Combinations (Disc,Scan,Expression,chip2chip, deleteion):
** [http://eutils.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16644789&itool=pubmed_DocSum TFBSScan] combines alignments, expression, Chip-Chip to find new motifs in yeast.
** [http://www.genome.org/cgi/content/full/16/5/627 combining binding site data with protein interactions predicts phylegenetic result of deletions]
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16923194&itool=pubmed_DocSum deletion of transcription factor leu3] does identify targets but notist_uids=16847329&itool=pubmed_DocSum NFkappaB example] showing how the distal enhancers make the promoter accessible via chromatin modifications and thus starts transcription
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17002502&itool=pubmed_DocSum promoter that can be dissected into 6 states]

* Randomization techniques when scanning for motifs:
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16819796&itool=pubmed_DocSum Scanning human/mouse/rat, rating conservation and rating motif matches]

* Matrices are bad:
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16961928&itool=pubmed_DocSum by re-building ERE matrices, < 15% error rate in prediction was obtained]
* alignment
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16733547&itool=pubmed_DocSum tf-maps] is an alignment without sequence similarity

* Bio papers about chosen transcription factors:
** Regulation of the transcription factor FOXM1c by Cyclin E/CDK2 PMID 16504183

* Mathmatics, statistics, background models:
** [http://www.biomedcentral.com/content/pdf/1471-2105-7-279.pdf Various Markov Models] to improve the PWM scheme.

* combinatorial detection
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16733548&itool=pubmed_DocSum CodeFinder] tries to find overrepresented combinations

* cis-reg evoluation is important, few bps enough to create dots on wings
** [http://www.nature.com.gate1.inist.fr/nature/journal/v440/n7087/full/nature04597.html yellow dots in drosophila]

* motif disc reviews
** [http://eutils.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16683017&itool=pubmed_DocSum motif discovery review]
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16722558&itool=pubmed_DocSum tompa review proposing new formula]

Applikation of motif discovery in plants]
** [http://www.jbc.org/cgi/content/abstract/281/7/4132?ct Gibbs Sampler applied] to find a ccaat-box
** [http://www.plantmethods.com/content/pdf/1746-4811-2-8.pdf Clustering of genes ] according to a) co-expression and b) co-occurence of hexamers and transfac motifs, programmed in R
* Developmental regulation
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16344563 ncRNA] has 2700 ncRNAs that might be involved in development
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&dopt=Abstract&list_uids=16365385 transposon deserts] are associated with developmental regulators

* Motif Scanning
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16464824&itool=pubmed_DocSum Yeast Dyad scanning] distances conserved, direction as well. Tf active in Multiple conditions -> variable distances

* Tools:
** [http://www.biomedcentral.com/content/pdf/1471-2105-7-188.pdf Bio++] C++ library specialized in phygenetics can at least do some sequence reading/writing
* Biology
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16465403&itool=pubmed_DocSum Strange article by the infamous Katoh/Katoh team] about spray being indentified with double TCF/LEF binding sites conserved]
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16365378&itool=pubmed_DocSum How to validate binding sites to be REALY REALLY sure that they are valid?] 
** [http://www.sciencedirect.com.gate1.inist.fr/science?_ob=ArticleURL&_udi=B6TD1-4JRKWSW-4&_coverDate=05%2F31%2F2006&_alid=418401753&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=5185&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=89452db0bc9ef6ae44604e8b87a9892a How to validate binding sites?]

* Structure of the DNA
** [http://www.sciencemag.org.gate1.inist.fr/cgi/content/full/311/5762/844 modification of histone] to unwind chromatin structure
** [http://nar.oxfordjournals.org/cgi/content/full/34/4/e34/FIG4 DHS] search overlayed with UCSC conservation
** [http://eutils.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16568947&itool=pubmed_DocSum review nucleosome structure]

* Chip-Chip or similar, large sequence biology data:
** [http://www.genome.org/cgi/content/abstract/16/5/595?etoc E2F1 Chip2Chip] suggests that 20% of all promoters are bound by E2F1, usual proximal, sites do not correspond to matrix (!)
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=PureSearch&db=pubmed&details_term=Meng%5BAll%20Fields%5D%20AND%20Brodsky%5BAll%20Fields%5D%20AND%20Wolfe%5BAll%20Fields%5D Yeast one hybrid assay] to determine matrices for given transcription factor, WITHOUT selex and without Chip-Chip

* Link between SNPs and binding sites:
** [http://hmg.oxfordjournals.org/cgi/content/abstract/15/suppl_1/R9?ct problems faced at the moment]
** [http://www.sciencemag.org/cgi/content/abstract/312/5778/1392?etoc Rice Example] published in Science: SNP in 5' causes phylogenetic difference
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16728641&itool=pubmed_DocSum regulatory SNP causes disease] SCIENCE 05/06
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16978413&itool=pubmed_DocSum genome-wide survey] of linkage between genes that are differently expressed and their regulatory SNPs

* Networks and binding sites:
** [http://www.genome.org/cgi/content/abstract/16/5/627?etoc yeast data] suggests that sequences (knockout-data) can predict phenotypes based on network (manual) and binding site (predicted and chip-chip)
* Pitx stuff:
** [http://eutils.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=16638982&itool=pubmed_DocSum regulates muscle and eye development]

* Applications of conservation to elucidate regulation
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=retrieve&db=pubmed&list_uids=16613900&dopt=Abstract MCEP2 dissection of regulatory region]

* Relation between phenotype and cis-regulatory variation
** [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=abstract&list_uids=16472943&itool=pubmed_DocSum Carp acclimated to 10/30 deg.] is analyzed in respect to cis-reg variation

* Redundancy in Regulation:
** Demonstration that you can delete functional binding sites [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=Abstract&list_uids=16818466&itool=pubmed_DocSum without changing expression ] (note: at a certain tissue/development stage/pathway status!!)
** Another example that binding sites [http://www.pnas.org.gate1.inist.fr/cgi/content/full/103/27/10305 cluster] together
** [http://www.pnas.org.gate1.inist.fr/cgi/content/full/103/32/12027 Drosophila colocalization hotspots bind many different factors]: "we predict that many more factors will show strong colocalization and that hotspots may recruit up to a few hundred different proteins. This observation contrasts with results in yeast where overlap between transcription factors was relatively rare and hotspots were not apparent (7, 8)."