These tools simplify the search for all homologous genes across as many species as possible.
Apart from Unigene, all of these are the result of massiv protein-to-genome or genome-to-genome BLAST runs:

* [http://www.bcgsc.ca/services/thor THOR] is designed for low-coverage genomes. It will try to infer the order of contigs by aligning them on the human genome. Works of course only species close to human.
* [http://inparanoid.cgb.ki.se/ Inparanoid] is a database of orthologous proteins between 26 model organisms and a downloadable PERL script that should run on any machine (beware of the BLAST-version issue, read the readme-file)
* [http://cvsweb.sanger.ac.uk/cgi-bin/cvsweb.cgi/ensembl-compara/docs/ Compara] The part of ensembl that is called Compara could be used for this. Search a gene in Ensembl and it will display all orthologs from all sequenced model species. You can use biomart.org to download the data in compara. As of this writing (01/2008) there is still no paper describing the pipeline of compara and compara still doesn't seem to include synteny. In addition, when using compara, as everything related to Ensembl, make sure that your clearly keep track of the version of Ensembl that was used. Compara data might suddenly change from one version to another and since Ensembl is updated it's complete database every few months (unlike UCSC, which only updates the programs) you might end up a with "version issues" in your database.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=homologene Homologene ] 17 main model organisms
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=unigene Unigene ] not directly usable but can give some hints
* [http://genome.ucsc.edu/cgi-bin/pbGateway UCSC Proteome Browser ] The famous Browser imports data from Unigene and also points to similiarities, but this seems to be deduced from gene names instead of comprehensive searches
* [http://www.tigr.org/tdb/tgi/ego/index.shtml EGO] from tigr is yet another cross-blast-clustering database
* the [http://www.metazome.net/ Metazome] consists of more than 26 genomes which you can search for clusters of proteins, align and display the domains. What makes metazome interesting is its "neighbors"-view: Type in a gene-name and it will immediately display all 5 flanking genes. With this you can easily discover genes located in syntenic regions.
* [http://synteny.cnr.berkeley.edu/CoGe/ Comparative Plant Genomics at Berkeley] lets you view two genomes and their conserved elements around homologous genes using short non-coding blast hits
* [http://www.sbc.su.se/~andale/multiparanoid/html/index.html Multiparanoid is like Inparanoid, but outputs clusters. Yeah!]
* [http://orthomcl.cbil.upenn.edu/ OrthoMCL] uses [http://micans.org/mcl MCL] to build gene clusters and identify orthologs. The basic difference between OrthoMCL and Inparanoid is that Inparanoid gives you only pairwise results and OrthoMCL creates clusters across ALL genomes.
* [ http://www.ebi.ac.uk/research/cgg/tribe/ TribeMCL is similar to OrthoMCL but older, seems to be used in compara?]
* A couple of other options include [http://www.treefam.org treefam] though it is not really meant for this task and includes only selected organisms, TribeMCL (which is well integrated into Bioperl). However, in a recent comparison [http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&Cmd=ShowDetailView&TermToSearch=17440619&ordinalpos=3&itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum] of various orthology-detection techniques, Inparanoid and OrthoMCL scored best. 

For closer model species, you can project features from one genome to the other, based on whole genome (nucleotide-based) alignments:
* [http://bio.math.berkeley.edu/genemapper/ Genmapper] seems to be one of the better programs (Projector and Genewise are two other options)
* Net and chain tracks on UCSC can be used to map from one species to the other, and the [http://genome.ucsc.edu/cgi-bin/hgConvert?hgsid=68458980&db=hg17&position=chr7:127471196-127495720 UCSC liftOver-tool] can be used to do it automatically (download as part of jim kent's source tree)
* Do not forget Ensembl's [http://www.ensembl.org/Homo_sapiens/multicontigview?bottom=%7Cinfo%3Aon&s1=tn&w=2824576&c=4%3A113108453.5%3A1&h=newsinfrug00000155006%7Censg00000164093&w1=2529628&c1=20%3A1264814.5%3A-1 multicontigview]. There is nothing similar in UCSC. You can display several genomes one atop the other and elements that can be aligned (only blat?) are connected by lines. Try a developmental gene with highly conserved flanking regions and follow the compaction of genomes down to fugu. Impressive visualization.
* If you need a less cluttered visualization and more sensitivity: try [http://gata.sourceforge.net/ Gata] from the Eisen lab. It will run blast on your regions and then visualize the results with a synteny-plot. If your regions is longer, try syn-apollo from my toolbox