* [http://genomequebec.mcgill.ca/PReMod/ PreMod:] searches in whole genome alignments of hs/mm/rn for conserved matches of transfac-matrices in non-coding regions and predicts 100.000 enhancers on the human genomen, searchable, with nice displays.
* [http://www.stanford.edu/group/wonglab/software.html CisModScan: ] searches for clusters of binding sites
* [http://giscompute.gis.a-star.edu.sg/~vega/BEARR1.0/ BEARR] lets you download sequences and searches for sites
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=16630346&itool=pubmed_DocSum XCisClique] searches for cluster of sites called Bicliques in Arabidopsis: Genes have to be co-expressed + share a combinatin of motifs. It works only on Arabidopsis.
* [http://gala.cse.psu.edu GALA] lets you download multiz-alignments of human-mouse-rat, already scanned for conserved Transfac matrices at 0.85. There's a lot of filtering options available.
* [http://lgsun.grc.nia.nih.gov/cisview/ cisView] has the results of a whole-genome scan for TFBSs with patterns from the literature, Jaspar, Xie et al. They add a whole lot of analyzation, overrepresented distances, flanking genes, GOs of flanking genes. Regions can be narrowed down to conserved distal regions or proximal upstream promoters.
* [http://hscl.cimr.cam.ac.uk/CoMoDis_human35.html CoMoDis] searches mammalian alignments for matches to a given binding site model around a set of flanking sequence for a list of genes.

==== Bacteria/Yeast ====
* [http://www.swissregulon.unibas.ch/ Swissregulon]: Alignments and predicted regulative features on GBrowse