These programs expect an alignment or sequences and try to find the best-conserved fragments within them:

* Originally, the classical [http://research.nhgri.nih.gov/MCS/details.shtml VISTA] was used for this, with a cutoff of 70% identity over 100 bp.
* UCSC's most conserved is based on coding regions. Covers 5% of the genome, no configurable threshold.
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15965027&query_hl=2 Genomic Evolutionary Rate Profiling (GERP)] : is only available in perl and does not seem to be threshold-based, it's available on UCSC Encode though
* [http://research.nhgri.nih.gov/MCS/details.shtml MCS] wants an alignment, a tree, genomic coordiantes for UCSC (bravo!), repeats and exons and a P-Value cutoff value. Nice, but only a webversion.
* [http://pga.lbl.gov/gumby/ gumby] expects somewhat less parameters and is downloadable in source (seems to be part of [http://genome.lbl.gov/vista/rankVISTA.shtml RankVista]

Note: When in doubt, prefer multiple alignments ([http://www.sciencedirect.com.gate1.inist.fr/science?_ob=ArticleURL&_udi=B6TCY-4JBGHS7-3&_coverDate=04%2F30%2F2006&_alid=420883063&_rdoc=1&_fmt=&_orig=search&_qd=1&_cdi=5183&_sort=d&view=c&_acct=C000050221&_version=1&_urlVersion=0&_userid=10&md5=3be40332a858e9a0ac4203a5ce410da2#SECX4 paper])