* Converters:
** [http://idconverter.bioinfo.cipf.es/ gene id converter]
* Bibliography, News
** [http://barf.jcowboy.org/ BARF:] Get RSS-feeds for most biology/bioinfo journals here and put them into your...
** [http://reader.google.com Google Reader] to see everyday where new articles appeared in your subjects of interest, then look up bibtex with
** [http://www.jabref.org Jabref] Free Citation Manager for Latex
** [http://www.highwire.com highwire] Slower, but can show who's citing whom
** [http://scholar.google.com Google Scholar] Well, you know it, searches journals in fulltext
** [http://www.connotea.com Connotea] is Nature's bibliography and generic link exchange site. You can register, order your bookmarks and bibliography data, tag them and share lists with all other participants.
** [http://www.citeulike.org Citulike] is (as far as I know, TODO) almost the same thing. You can search both connotea and Citeulike for your keywords and possibly find similar articles in your domain.
** [http://www-tsujii.is.s.u-tokyo.ac.jp/medie/search.cgi MEDIE] looks pretty useful, a copy of pubmed with different operators, you specify e.g. certain verbs or sequences of words to search

*Tools for Pubmed
** [http://www.hubmed.org hubmed] Faster, better than pubmed, can also export to bibtex and put the bibtex into...
** [http://www.glycosciences.de/tools/PubFinder/ Pubfinder] [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&list_uids=15980583&query_hl=1&itool=pubmed_docsum (article)]: Give it a list of articles, it will extract the discriminative words and return similar articles
** [http://pubmatrix.grc.nia.nih.gov/ Pubmatrix] can compare a list of terms against a list of terms, searches all combinations and displays the results in a kind of "cross-matrix" view
** [http://www.chilibot.net/ Chilibot] can do multiple pubmed queries for you and displays nice graphics which word is occuring in which abstract.
** [http://www.gopubmed.org/ Gopubmed] launches pubmed-searches and sorts the results by Gene ontology.
** [http://www.ihop-net.org/UniPub/ iHOP] shows a gene-centric pubmed view. Type in any gene and iHOP will display all sentences that contain this gene together with other genes or MeSH terms.
** [http://clustermed.info/ Clustermed of Vivisimo] is a search engine like Google. Unlike google, it can display it's results clustered into groups which can make sense for pubmed.
** [http://www.ogic.ca/projects/xplormed/ XPloreMed] scan all abstract of a pubmed search and displays the most recurrent words in them.

* Useless Pubmed Tools:
** [http://metnet.vrac.iastate.edu/browser/ Pubmed Friend] a Java interface for pubmed...well...
** [http://pmi.nlm.nih.gov/slide/index.php Pubmed Sliders] This is the same as pubmed, the only difference is that you can shift sliders to narrow down on your search results. Well...

*Blogs:
** [http://www.nodalpoint.org Nodelpoint] Bioinformatics Blog, I guess the most famous one, in any case, this was the first one that I discovered
** [http://www.ghastlyfop.com/blog/index.html Flags and Lollypops] Bioinformatics Blog 
** [http://blog.moseshohman.com/ Moses Hohman on Inforbiomatica] writes a lot about programming languages and bioinformatics and the best approach to develop software
** [http://fjossinet.u-strasbg.fr/ Propeller Twist] a lot about RNA, general stuff.
** [http://scienceblogs.com/goodmath/] Good math, bad math is a funny read, but nor really bioinfo-related
** [http://naturesnumbers.blogspot.com/ NaturesNumbers] is a much newer blog, less well-known
** [http://www.dalkescientific.com/writings/diary/ Andrew Dalke] an important developer for biopython, is also blogging from time to time

* Programming, Statistics
** [http://www.r-project.org/ R] The statistics package. I merely use it for simple plotting.

* Sequence Tools
** [http://www.mbio.ncsu.edu/BioEdit/page2.html Bioedit] The good old free sequence editor (Windows only). Outdated but still usable, especially for biologists. (Update Oct 05: Seems that the author decided to stop distributing it!!! You can still download it if you use google wisely...)
** [http://www.fruitfly.org/annot/apollo/ Apollo] and its siblings ACT/Artemis are not for everyday work, they are just great graphical interfaces to display annotations in any format (GFF, among others).
** [http://galaxy.cse.psu.edu/ Galaxy] looks very promising. Though far from straightforward, it allows every one without programming skills to run analyses on a genomic scale. I guess "extract of all coding regions and display a graph of their lenghts" is something that should be easy with galaxy... (todo)

* Phylogenetics
** [http://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi The NCBI Taxonomy Browser] is handy when dealing with biologists as you can quickly decrypt the names of all those weird species AND you filter on all those animals which are already sequenced (!!). More data on all genomes can be found in the [http://www.genomesonline.org/ Gold Database].
** Daniel Pollard made a nice [http://rana.lbl.gov/~dan/trees.html tree of the evoluationary distance of all sequenced flies]
** [http://www.cisreg.ca/gsb/ The Gene Set Builder] let's you create lists of genes, share with other people and then transfer them automatically to IDs of other species. Very handy if you want to download the same set of promoters from another species.

* Workbench-like tools: (list from Jeff Bizarro)
** [http://taverna.sourceforge.net/ Taverna]
** [http://www.ngbw.org Swami]
** [http://www.openflow.it/EN/index_html OpenFlow]
** [http://tikiwiki.org/tiki-index.php?page=GalaxiaWorkflow GalaxiaWorkflow]
** [http://wildfire.bii.a-star.edu.sg/ Wildfire]
** [http://home.cc.umanitoba.ca/~psgendb/ BIRCH]
** [http://emboss.org/ EMBOSS]
** [http://www.wemboss.org/ wEMBOSS Portal]
** [http://www.alphaworks.ibm.com/tech/wsbaw WSBAW]
** [http://www.scicraft.org/ SciCraft]
** [http://scitegic.com/products_services/pipeline_pilot.htm Scitegic]
** [http://bioinformatics.ubc.ca/pegasys/ Pegasys]
** [http://www.angis.org.au/ ANGIS]
** [http://www.w2h.dkfz-heidelberg.de/ W2H]

* Whole Genome Data:
** [http://www.genomesize.com/ GenomeSizes] : Does what its name implies
** [http://metazome.org Metazome] : Data of all coding regions clustered into ortholog groups, with genomic neighbours and phylogenetic relationships

* Tutorials:
** [http://www.worldscinet.com/jbcb/03/0302/S0219720005001077.html HMMs]
** Pharyngula: [http://scienceblogs.com/pharyngula/2006/07/upstream_plasticity_and_downst.php Upstream factor turnover] (see discussion papers!) and the [http://scienceblogs.com/pharyngula/2006/07/bicoid_nanos_and_bricolage.php bicoid gradient]