* Prediction of enhancers:
** [http://www.bx.psu.edu/projects/esperr/ esperr]: a newer version of the "regulatory potential" program
** "Regulatory Pontential" track on UCSC: searches multiple alignments for transfac matches

* Prediction of promotors:
Since there are now CAGE-data available from fantom2 and dbtss (included in UCSC?) I never really got into any detail for this topic. here are at least some keywords if you want to try this:
**[http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=pubmed&cmd=Retrieve&dopt=AbstractPlus&list_uids=17000749&query_hl=5&itool=pubmed_docsum promoter explorer], a quite new addition compared to the others
** [http://www.fruitfly.org/seq_tools/promoter.html NNPP]: BGDP's prom prediction
** [[http://www.softberry.com/berry.phtml?topic=promhg&group=programs&subgroup=promoter Promh]] if you have two orthologous regions (oct 05: doesn't seem to work anymore)
** EponineTSS
** [http://www.biomedcentral.com/1471-2105/6/262 HexDiff]
** [http://rulai.cshl.org/tools/FirstEF/ First EF]: not really promotor prediction, rather First Exon Prediction, available on UCSC as a Track.
** TSSG: rather old promoter predictor...

* Note: Have a look at a very recent [http://genomebiology.com/2006/7/S1/S3 benchmark of promoter predictors] done for Encode