* Good old [http://www.epd.isb-sib.ch/ EPD], of course
* On the [ftp://hgdownload.cse.ucsc.edu/goldenPath/hg17/bigZips/ UCSC FTP-Server] you can find upstream1000, upstream2000 and upstream5000-files, that are simply the given number of bps upstream of mRNA sequences in the genome browser.
* [[http://www.ensembl.org/Multi/martview Ensmart]] is the part of ensembl that corresponds to the UCSC table viewer. It allows you to retrieve upstream regions if you have a list of Gene-IDs. There is a [http://insects.eugenes.org/BioMart/martview special biomart] for the 12 sequenced flies now.
* [[http://genome-www6.stanford.edu/cgi-bin/Traser/traser Traser]] Retrieve upstream or transcript sequences via LocuslinkID, for human/mouse/rat
* [[http://doop.abc.hu/ Doop]] The database of orthologous promoters lists clusters of promotors from orthologous genes, sequences and common motifs within them. The best resource at the moment.
* [http://biowulf.bu.edu/zlab/PromoSer/ Promoser] Human, mouse, rat. Promoser tries to extend existing mRNA sequences from Genbank using ESTs. Whereas this might not be as interesting as before since the advent of [http://dbtss_old.hgc.jp/hg17/ DBTSS] (that got incorporated into EPD,Biobase and Genomatix), Promoser has still nice clickable images that give you an impression of the distribution of promoters.
* [[http://rulai.cshl.edu/CSHLmpd2/ CSHLMPD]] collects promotors from human, mouse and rat: using 
** all mRNAs from Genebank 
** all DBTSS and EPD entries
** FirstEF predictions
** multispecies alignments of the promotors. This should be similar to Genomatix.
See this [[http://genomebiology.com/2005/6/8/R72 article]] for details
* [http://giscompute.gis.a-star.edu.sg/~vega/BEARR1.0/ BEARR] extract upstream sequences, can use DBTSS to delineate the TSS, can also directly scan the results
* [http://rulai.cshl.edu/TRED TRED] takes promotors from CSHLMPD and maps '''binding sites''' on them
* [http://dbtgr.hgc.jp/ DBTGR ] collects promotor AND '''validated binding sites''' (!) from tunicates, a rare but cute model species. Update: It seems that they don't have more than a handful (around 20) binding sites with known transcription factor. Most are not very well dissected regulatory elements.

* Sometimes you need to [[assemble part of a genome from traces]] before you find your promotor of interest 

=== Commercial products ===

* [http://www.genomatix.de Genomatix] is selling a database of promotors of the main mammalian model species that is a combination of DBTSS-hits, PromotorInspector, EST- and mRNA-hits. However, without access to the data, you cannot verify any claims about it. Genomatix people claim that they are the best, of course. It is certainly one of the most usable sites for promoter analysis and may be well worth the 500 $ per year, given it's nice and very usable graphical interface. You might check their "promotors" in your favorite genome browser though... (try comparing it in UCSC against AceView, Known Genes and FirstEF predictions on your gene of interest.)
* [http://www.biobase.de/pages/products/transpro.html Transpro] by the Transfac-people is a very similar thing, without any predictions. I wonder who's buying it, given that there is virtually no interface at all.