Whole Genome Alignments are handy: They can save you a lot of time. 

* The most famous resource here is [http://genome.ucsc.edu/cgi-bin/hgTrackUi?hgsid=70063309&c=chr4&g=multiz17way The Conservation Track ] of the UCSC browser based on chaining/netting and re-aligning regions with Blastz which was done by the [http://www.bx.psu.edu/miller_lab/ Miller Lab]. You can download bed-regions with the table browser and then convert to fasta with my maf2fa and maf2fa-files tools. At the moment, they combine 28 genomes into one single alignment. If you wonder how this works, the best reference (very humble as I am) is [http://genomewiki.ucsc.edu/index.php/Whole_genome_alignment_howto my howto].
* The other main resource is Lawrence Livermore's Comparative Genomics tools, well known as [http://genome.lbl.gov/vista/index.shtml VISTA]. Apart from beautiful percent-identity-pots, you can [download http://genome.lbl.gov/vista/downloads.shtml] the alignments in fasta format. They are different from UCSC, I've only seen pairwise alignments to download here until now though you can display them together and possibly weave together several files on your own. But you stick with UCSC as they really support their alignments.
* Plants: [http://synteny.cnr.berkeley.edu/CoGe/ Comparative Plant Genomics at Berkeley] lets you view two genomes and their conserved elements using pairwise short non-coding blast hits
* '''Note:''' There is some important difference between UCSC and the VISTA servers: Vista uses MLAGAN, which is global alignment program whereas UCSC is using only BLASTZ, which is local aligner, optimized for speed and perhaps somewhat less sensitive than BLAST. However, if you working on conserved regions, you probably won't to focus only on the best ones anyways. UCSC has definitely the best format, best tool support and fastest servers out there, so unless you have very good reason to do otherwise, I'd suggest you stick with them.