* Conserved Matches, precalculated: There are some websites where you can see Precalculated alignments and Matrix hits. I don't like them too much and they are not applicable for my model organism.
** [http://corg.molgen.mpg.de/ corg] shows alignments of upstream regions of many model species and can display conserved binding sites. Unfortunately, I have only seen pairwise alignments. Look at UCSC for multiple alignments with an easy to use interface.
** [[http://mordor.cgb.ki.se/cgi-bin/CONSITE/consite/ Consite]] Any 2 sequences and hits on Transfac
** famous [[http://genome.ucsc.edu UCSC]] Genome Browser has a TFBS-track for the human genome, where all hits against Transfac Pro 8.3 are documented and can be downloaded as GFF (hint: What can you do with many hits?)

* Conserved Matches, configurable:
** [http://synor.dcode.org/ synor] searches a pair of human/chicken/mouse/frog for a combination of conserved binding sites
** [http://www.cisreg.ca/cgi-bin/oPOSSUM/opossum oPOSSUM] Looks better than the other conserved genomic searchers as you can configure the parameters. Only human/mouse? (Thanks to Malcolm Cook!)
** [http://hscl.cimr.cam.ac.uk/TFBScluster_genome_34.html TFBSCluster] Searches human-mouse for matches to a subset of matrices from Transfac4 or your IUPAC-strings.