This page contains various information relating to vectors used in OpenWetWare labs.

==General information==

===Online vector databases===
You can often find vector information at [http://www.ncbi.nlm.nih.gov/ NCBI,] either directly or in their list of vectors screened for contamination of new sequence at [http://www.ncbi.nlm.nih.gov/VecScreen/replist.html Vecscreen]. 

[http://seq.yeastgenome.org/vectordb/ VectorDB] contains information about many common vectors, including yeast vectors.

[http://www.embl-hamburg.de/services/protein/vectors_strains/vectors.html The EMBL Hamburg outstation] maintains a large database of vectors.

[http://image.llnl.gov/image/html/vectors.shtml I.M.A.G.E. Consortium Vectors] contains information about plasmids commonly used in EST collections such as those sold by [http://www.openbiosystems.com OpenBioSystems] and [http://www.invitrogen.com Invitrogen].

For eukaryotic vectors (Fish, Xenopus) see [http://beckmancenter.ahc.umn.edu/cgi-bin/plasmidlookup.pl Minnesota].

The [http://www-rcf.usc.edu/~forsburg/vectors.html Forsburg Lab] maintains a list of Fisson Yeast vectors.

[http://www.promega.com/vectors/ Promega] maintains a list of their vectors.

[http://www.neb.com/nebecomm/products/category24.asp?#29 NEB] maintains a list of common vectors.

[http://www.epibio.com/sequences.asp Epicentre] also maintains its own list.

[http://www.lucigen.com/catalog/index.php?cPath=14_15 Lucigen] provides transcription-free vectors for cloning AT-rich and other difficult DNAs.

[http://www.addgene.org/vectors Addgene's Vector DB] contains most of the information from Stanford's VectorDB, plus more vector information they have curated from commercial websites and added through our plasmid curation efforts.  (However, it seems to be rather sparse when it comes to ''Escherichia coli'' vectors.)  Note that [http://www.addgene.org Addgene] is a a non-profit plasmid repository where scientists can archive and share their plasmids.  They encourage and invite labs to deposit plasmids at Addgene. They help you with data submission and all tech transfer issues.  Plasmids can be requested from Addgene for a fee to cover expenses.

[http://bioinfoman.com/bm/pcs.php Bioinfoman] also has a 5000+ long list of vector sequences.

[http://plasmid.med.harvard.edu/PLASMID/ PlasmID] is an online database and DNA repository maintained for  plasmids, including [http://plasmid.med.harvard.edu/PLASMID/GetAllVectors.do empty vectors] useful for a variety of techniques, and open reading frame (ORF) clones relevant to human, yeast, and several bacterial pathogens. All clones can be requested individually, and ORF clones can also be viewed and requested as grouped [http://plasmid.med.harvard.edu/PLASMID/GetCollectionList.do collections]. PlasmID is maintained by the [http://dnaseq.med.harvard.edu DF/HCC DNA Resource Core] at the Harvard Institute of Proteomics, Harvard Medical School.

===Annotation===
[http://wishart.biology.ualberta.ca/PlasMapper/ PlasMapper]: "automatically generates and annotates plasmid maps using only the plasmid DNA sequence as input. Plasmid sequences up to 20,000 bp may be annotated and displayed. Plasmid figures may be rendered in PNG, JPG, SVG or SVGZ format."  It can also output GenBank format.  Reference: Xiaoli Dong, Paul Stothard, Ian J. Forsythe, and David S. Wishart "PlasMapper: a web server for drawing and auto-annotating plasmid maps" Nucleic Acids Res. 2004 Jul 1;32(Web Server issue):W660-4.
*One drawback to this tool is that although it finds ORFs, it doesn't necessarily identify them. --[[Reshma Shetty | RS]]
*A very nice aspect of PlasMapper is that it identifies features of a sequence and puts them into GenBank format - very handy to prepare sequences for submission to GenBank. --[[User:Rgodiska]]

Also check out [http://www.biology.utah.edu/jorgensen/wayned/ape/ APe], A Plasmid Editor.
*Has some built-in features for annotation (a few genes, many restriction enzymes).  Allows you to make custom "feature libraries" for annotating sequences
*Can edit features found, unlike Plasmapper --[[User:Melissali|mel]] 18:50, 12 June 2007 (EDT)

==''Escherichia coli''==

See [[Escherichia coli/Vectors]].

==Yeast==

[[Yeast_vectors|Nomenclature and types]]

[[Yeast artificial chromosomes]]

==Plant==
[http://www.cambia.org/daisy/cambia/materials/vectors.html CAMBIA Plant Transformation Vectors]

==Other==
See [[Gateway cloning system]].

[[Category:DNA]]