'''Please add new resources useful to the community here.  Keep this page organized by following the format and organization already here.'''

<b>
*See also [[Vectors]] for information and links related to cloning vectors.
*See also [[Searching the literature]] for information and links on searching the biological literature.
*See also [[Synthetic Biology]] for information and links on Synthetic Biology.
</b>

== Videos ==
* [http://sciencehack.com/videos/category/7 ScienceHack]: Search engine for science videos with a review system for videos.

==Information resources==

'''References texts and information resources.'''

===Bacterial physiology===
*[http://redpoll.pharmacy.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi ''Escherichia coli'' statistics] by Institute for Biomolecular Design
**Several tables describing statistical data on ''E. coli'' compiled from several sources.

*[[Physiology of the Bacterial Cell]] by Neidhardt, Ingraham and Schaechter
**Reference text on bacterial cell physiology.

*[http://www.ecosal.org/ecosal/index.jsp EcoSal] by ASM Press
**An incomplete web version of the publication ''Escherichia coli and Salmonella: Cellular and Molecular Biology''.  [Subscription required]

*[http://ecogene.org/index.php EcoGene]
**Database of all E. coli genes and sequences

*[http://ecocyc.org Ecocyc]
** Comprehensive biochemical pathway and gene function site for ''E. coli''

===Lab techniques===
*[http://www.mrw.interscience.wiley.com/cp/cpmb/cpmb_contents_fs.html Current Protocols in Molecular Biology]
**Handbook of protocols.

*[[Molecular Cloning]] by Sambrook and Russell
**Handbook of protocols.

*[[A Short Course in Bacterial Genetics]] by Miller
**Handbook of protocols.

*[http://arep.med.harvard.edu/polony/polony_protocols Polony Protocols] by Church and Mitra Lab
**Polonies are colonies of PCR amplicons derived from a single molecule of nucleic acid.  

*[http://www.cf.ac.uk/biosi/staff/ehrmann/tools/geneexpression.htm Gene expression in ''E. coli''] by Ehrmann lab
**Tips and information on gene expression in ''E. coli''

*[http://protocol-online.org Protocol-online] by Dr. Long-Cheng Li
**A database of research protocols in a variety of life science fields.  It has a popular discussion forum.

===Protein===
*[http://www.hamptonresearch.com/support/ Technical suppport on protein crystallization] by Hampton Research
**Focused on protein crystallization but contains a lot of generally useful information about various reagents with respect to proteins.

===RNA===
*[http://www.ambion.com/techlib/append/na_mw_tables.html Appendix] by Ambion, Inc.
**Website with many useful nucleic acid parameters.

===Research===
*[http://www.nap.edu/readingroom/books/obas/ On Being a Scientist] by National Academy of Science
**NAS report on responsible conduct in research.

==Online tools==

'''Tools available to use online'''.

===General===
*[http://genolist.pasteur.fr/Colibri/ Colibri] by Institut Pasteur
**''E. coli'' genome site; get sequences, see the position of your gene in the chromosome, see the function of your gene, and other fun stuff. You can also search for protein sequences/motifs within the ''E. coli'' genome.

*[http://www.ncbi.nlm.nih.gov/ NCBI]
**Various tools and databases for molecular biology.

*[http://parts.mit.edu/ Registry of Standard Biological Parts] by MIT
**Open repository of BioBricks; the place for all your standard biological parts.

*[http://paris.chem.yale.edu/extinct.html Biopolymer calculator] by Schepartz lab
**Calculate extinction coefficients, Tm's, and base composition for your DNA or RNA; calculate amino acid composition and extinction coefficient for your protein
**Currently under [http://www.schepartzlab.yale.edu/construction.html construction], with promises that it will be back online soon.

*[http://www.dnatwopointo.com/commerce/misc/sms2.jsp Bioinformatics toolbox] from DNA 2.0
**Contains many Javascript tools to do common tasks.

*[http://www.sciencegateway.org/ Science Gateway]
**Collection of science-related links, including links to journals, catalogs, and tools. See their [http://www.sciencegateway.org/about.htm about page].

*[[BioNumbers]]
**The database of useful biological numbers.

===DNA===
*[http://synbio.mit.edu/tools/clipboard.cgi Clipboard] by [[Austin Che]]
**Web tool for getting complement, reverse complement, translation and restriction enzyme analysis of a DNA sequence

*[http://synbio.mit.edu/tools/converter.cgi File format converter] by [[Austin Che]]
**Web tool for converting between sequence file formats.

*[http://slam.bs.jhmi.edu/gd/ Gene Design] by Boeke lab
**Collection of online tools for codon optimization and shuffling, restriction site editing, and so on.

*[http://tools.neb.com/NEBcutter2/index.php NEB Cutter] by New England Biolabs, Inc.
**Tool for finding restriction sites, et cetera.

*[http://frodo.wi.mit.edu/cgi-bin/primer3/primer3_www.cgi Primer3]
** Tool that lets you pick & evaluate primers from a DNA sequence

*[http://molbiol.edu.ru/eng/scripts/01_11.html Search of rare codons in nucleotide sequence] by "Practical Molecular Biology"
*[http://nihserver.mbi.ucla.edu/RACC/ Rare Codon Calculator (RaCC)] by NIH MBI Laboratory for Structural Genomics and Proteomics
** Finds rare codons in a coding sequence.

*[http://www.bioinfo.rpi.edu/applications/hybrid/download.php UNAFold]
** Replacement for mFold for predicting nucleic acid folding. Downloadable and some applications are available online also.

*[http://genomics.ucdavis.edu/benham/sidd/index.php SIDD]
** Stress induced DNA duplex destabilization. Finds destabilized sites in superhelical DNA.

===RNA===
* [http://www.fruitfly.org/seq_tools/promoter.html Promoter prediction]: both prokaryote and eukaryotic promoter prediction

*[http://www.bioinfo.rpi.edu/applications/mfold/ mFold] by Michael Zuker.
**This is the main page with links to sites for predicting RNA and DNA folds, calculating Tm's and free energies.
** (This has been replaced with the [http://www.bioinfo.rpi.edu/applications/hybrid/ DINAMelt server] running UNAfold)
*[http://www.tbi.univie.ac.at/~ivo/RNA/ Vienna RNA] software package: RNA secondary structure prediction and design
* [http://www.bioinfo.rpi.edu/applications/sfold/index.pl Sfold]: Statistical Folding and Rational Design of Nucleic Acids. Predicts accessible RNA sites

===Protein===
*[http://www.embl-heidelberg.de/Services/serrano/agadir/agadir-start.html AGADIR] by Serrano lab
**An algorithm to predict the helical content of peptides.

*[http://web.mit.edu/biology/keating/AVID/ AVID] by Keating lab
**An integrative framework for discovering functional relationships among proteins.

*[http://dunbrack.fccc.edu/bbdep/ Backbone-dependent rotamer library] by Dunbrack lab
**Libraries of sidechain rotamer from protein structures

*[http://ncbi.nih.gov/Structure/CN3D/cn3d.shtml Cn3D] by NCBI
**A helper application for your web browser that allows you to view 3-dimensional structures from NCBI's  Entrez retrieval service.  It doesn't read PDB files but can be more straightforward to use than DeepView.

*[http://sosnick.uchicago.edu/contact_order.html Contact Order Calulator] by Baker lab
**Calculator to determine a protein's contact order 

*[http://us.expasy.org/ ExPASy Proteomics server] by the Swiss Institute of Bioinformatics 
**Collection of links to many pages to calculate parameters of your favorite proteins
*#[http://sosnick.uchicago.edu/pi_tool.html Compute pI/Mw]
*#*A tool which allows the computation of the theoretical pI (isoelectric point) and Mw (molecular weight) for a list of Swiss-Prot and/or TrEMBL entries or for user entered sequences.
*#[http://us.expasy.org/tools/protparam.html ProtParam]
*#*A tool which allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user entered sequence. 
*#[http://sosnick.uchicago.edu/translate_dna.html Translate] 
*#*Translate is a tool which allows the translation of a nucleotide (DNA/RNA) sequence to a protein sequence. 

*[http://www.scsb.utmb.edu/cgi-bin/get_a_form.tcl GETAREA 1.1] by Sealy Center for Structural Biology
**Solvent accessible surface areas, atomic solvation energies, and their gradients for macromolecules 

*[http://paircoil2.csail.mit.edu/ PAIRCOIL2] by Keating and Berger labs
**Tool to predict the parallel coiled coil fold from sequence using pairwise residue probabilities.

*[http://prospector.ucsf.edu/ ProteinProspector] by UCSF Mass Spectrometry Facility
**Proteomics tools for mining sequence databases in conjunction with Mass Spectrometry experiments.

*[http://www.scripps.edu/mb/barbas/zfdesign/zfdesignhome.php Zinc Finger Tools] by Barbas lab
**Design Zinc Finger DNA binding proteins

===Statistical tests===
*[http://www.physics.csbsju.edu/stats/ Statistics to use] from Saint John's University.

===Calculators===
*[http://sosnick.uchicago.edu/gdmcl.html <nowiki>[GdmCl]</nowiki> and <nowiki>[Urea]</nowiki> from refractive index] by Sosnick lab
**GdmCl and urea concentration calculator from index of refraction.

===Office applications===
*[http://www.goffice.com/ gOffice] by Silveroffice, Inc.
**Suite of office applications offered over the web

*[http://online.thinkfree.com/ ThinkFree Office Online] by ThinkFree
**Beta suite of office applicatins offered over the web

====Writing====
*[http://www.writely.com/ Writely] by Upstartle, LLC.
**A beta word processor run over the web

*[http://www.writeboard.com/ WriteBoard] by 37signals
**Permits writing of shareable, web-based text documents

====Spreadsheets====
*[http://numsum.com/ NumSum] by TrimPath
**Permits sharable, web-based spreadsheets

*[http://www.irows.com/xo/Welcome.do iRows] by Itai Raz and Yoah Bar-David
**Permits web-based spreadsheets

==Software resources==

'''Software available to download.'''

===General===
*[http://www.invitrogen.com/content.cfm?pageid=10373 Vector NTI] by Informax, Inc.
**Free-to-academics tool for sequence analysis and data management.

===DNA===
*[http://perlprimer.sourceforge.net/download.html PerlPrimer]: Open source PCR primer design. Written in Perl/Tk.
*[http://www.sanger.ac.uk/Software/Artemis/ Artemis] by the Sanger Center
**A free DNA sequence viewer and annotation tool (Java based).

*[http://www.dna20.com/commerce/customer/emailsupp.jsp GeneDesigner] by DNA2.0
**Combine genetic building blocks by drag-and-drop, codon optimize, restriction site editing, sequence oligo design etc. See BMC Bioinformatics 2006 Jun 6;7(1):285 for more detail.
**Free to download and works on Mac or PC.  User agreement is somewhat restrictive.

===RNA===
* [http://rna.ucsc.edu/rnacenter/xrna/xrna.html XRNA]: Java tools for creating RNA secondary structure diagrams

===Protein===
*[http://kinemage.biochem.duke.edu/software/softdocs/README.dang.html Dang] by Richardsons' lab
**A command-line tool that generates a table of several useful geometric measurements for each residue or base from a PDB file.

*[http://www.expasy.org/spdbv/ DeepView] by GlaxoSmithKline & Swiss Institute of Bioinformatics
**Awesome program for viewing and studying protein structure.

*[http://salilab.org/modeller/ Modeller] by Sali lab
**Program for homology or comparative modeling of protein three-dimensional structures by satisfaction of spatial restraints.

*[http://pymol.sourceforge.net/ PyMOL]
**Molecular graphics system with an embedded Python interpreter designed for real-time visualization and rapid generation of high-quality molecular graphics images and animations.  The latest version does not run on OSX 10.3.  (from [[User:Kathmc | Kathleen]]).

*[http://www.ks.uiuc.edu/Research/vmd/ VMD] by Theoretical and Computational Biophysics Group at UIUC
**Molecular visualization program for displaying, animating, and analyzing large biomolecular systems using 3-D graphics and built-in scripting.  Generates pretty high resolution pictures of protein structures.

===Geospatial modelling===
*[http://docs.codehaus.org/display/GEOS/Home GeoServer] by [http://www.openplans.org/projects/projects/topp-the-organization/project-home The Open Planning Project]
**Although aimed at the geographic information systems community, the GeoTools toolkit from the [http://www.openplans.org/projects/topp-the-organization/geoserver GeoServer Project] may be useful for geospatial problems in biology.

===Gene Network Modeling===
*[http://ingeneue.org Ingeneue] by George von Dassow, Eli Meir, Edwin Munro, and Garret Odell at the [http://www.celldynamics.org Center for Cell Dynamics]
**An open-source java program for modeling gene regulatory networks.  Users can rapidly build networks by specifying their topology, initial conditions, connectivity, and known parameters.  Ingeneue can then search/explore paramter space for desired behavior, simulate the effects of noise and mutation, and generate statistics/time graphs of the system.

===Toolbars===
*[http://www.bitesizebio.com/toolbars/molecular-biologists-toolbar.htm The Molecular Biologist's Toolbar] by [http://www.bitesizebio.com Bitesize Bio]