===Probabilistic methods (SCFG/CM and HMMs)===

* [http://infernal.janelia.org Infernal] is an implementation of covariance models (profile stochastic context-free grammars) for RNA sequence/structure analysis. The [http://www.sanger.ac.uk/Software/Rfam/index.shtml Rfam] database is maintained using Infernal.
* [http://tagc.univ-mrs.fr/erpin/ ERPIN] (Easy RNA Profile IdentificatioN) is an RNA motif search program developed by Daniel Gautheret and Andr√© Lambert
* [http://protein3d.ncifcrf.gov/shuyun/homostrscan.html HomoStRscan] A program for discovering homologous RNAs in complete genomes by taking a single RNA sequence with its secondary structure.
* [http://phmmts.dna.bio.keio.ac.jp/ PHMMTS] An implementation of "pair hidden Markov models on tree structures (PHMMTSs)".
* [http://bio.cs.washington.edu/supplements/zasha-ravenna/ RaveNnA] A software package for faster covariance models whilst provably sacrificing no accuracy for rigorous filtering. Also includes heuristic filters.
* [http://www.cse.ucsc.edu/~mpbrown/rnacad/index.html RNACAD] A stochastic context-free grammar(SCFG) RNA modeling package that accounts for both primary and secondary structure information. SCFGs are robust and allow arbitrary deviations from a pattern.
* [http://exon.umdnj.edu/software/RSmatch/ RSmatch] RNA Secondary Structure Matcher: provides four functions: (1) regular database search, (2) multiple structure alignment, (3) iterative database search, and (4) pairwise sequence alignment.


===RNA descriptor methods===
* [http://bibiserv.techfak.uni-bielefeld.de/HyPa/ HyPa] A system for the declarative description and efficient search of hybrid patterns in large genomic data sets.
* [http://carlit.toulouse.inra.fr/MilPaT/MilPat.pl MilPat] Motifs and Inter-moLecular motifs searching tool using Constraints Solving Problems formAlism and solving Technique.
* [http://bibiserv.techfak.uni-bielefeld.de/guugle/ GUUGle] A utility for fast exact matching under RNA base pairing rules. GUUGle efficiently locates potential helical regions under RNA base pairing rules, which include Watson-Crick as well as G-U pairs. It accepts a target and a query set of sequences, and determines all exact matches under RNA rules between target and query sequences that exceed a specified length.
* [http://www-unix.mcs.anl.gov/compbio/PatScan/HTML/patscan.html PatScan] is a pattern matcher which searches protein or nucleotide (DNA, RNA, tRNA etc.) sequence archives for instances of a pattern which you input.
* [http://selab.janelia.org/software/#rnabob RNABOB] is an implementation of D. Gautheret's RNAMOT, but with a different underlying algorithm using a nondeterministic finite state machine with node rewriting rules.
* [http://www.scripps.edu/case/casegr-sh-3.5.html RNAMotif] Construct descriptors based upon consensus models of structural RNAs, these can be used to scan data-bases for matches.
* [http://bioinfo.csie.ncu.edu.tw/~rnamst/index.php RNAMST] an efficient and flexible RNA Motif Search Tool for RNA structural homologs. RNAMST web server accepts four different kinds of input formats to facilitate user to describe a RNA structure easily.

===Family specific searches===
* [http://bioinfo.thep.lu.se/ ARAGORN] A program to detect tRNA genes and tmRNA genes in nucleotide sequences.
* [http://artedi.ebc.uu.se/Bjorn/Tmrna/index.html BRUCE] A program for the detection of transfer-messenger RNA genes in nucleotide sequences
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve