==Existing tutorials==
===general introduction===
* [http://www.ensembl.org/info/index.html Ensembl Help System startup page]

* [http://www.ensembl.org/info/helpdesk/tutorials/worked_example.pdf Ensembl worked example] (PDF) 28 steps of viewing contigs, finding information about a gene, its homologues, performing a blast seaerch etc.

* [http://www.ensembl.org/common/Workshops_Online Flash examples]

* [http://bioinformatics.ca/workshop_pages/genomics/lectures2004/3_1.pdf Workshop in Canada] A bit dated (2004?) in a part describing Ensmart but otherwise good tutorial.

===For programers===
* [http://www.ebi.ac.uk/2can/pdf/ensembl_tutorial.pdf by Michele Clamp (PDF) @EBI] perl API, accessing servers using mysql. 
* [http://www.ensembl.org/info/software/core/core_tutorial.html Perl API] web based

=Random bits=
==Getting promoter sequences using BioMart for a list of IDs==
* go to [http://www.biomart.org/ BioMart] 
* click on '''Central Server''' link
* from a pull down menu 'CHOOSE DATABASE' select ENSEMBL 45 GENES
* from a new pull down menu 'CHOOSE DATASET' select organism of interest, i.e. Rattus norvegicus genes
* on a left side a frame appears. Click on '''Filters'''
* in the central panel click on '''+''' next to '''Gene'''
* check a box next to '''ID list limit'''
* from pull down menu right to '''ID list limit''' select type of your IDs, i.e. Unigene ID(s)
* either cut and paste your IDs (try four UniGene entries below) in a form or select text file with your IDs, one ID per line.
<pre>
Rn.141972
Rn.34912
Rn.6497
Rn.88235
</pre> 

* click on the Left Panel '''Atributes'''
* click on the Central Panel on the top: Radio button (round)'''Sequences''' 
* in the Central Panel click on '''+''' next to '''SEQUENCES'''
* depending on what you want to analyse click on   Radio button (round) next to '''Flank (Transcript)''' (makes sense if there are multiple startng exons) or '''Flank (Gene)''' (one flank per gene)
* check box next to '''Upstream flank''', put desired flank size, i.e. 1000
* you will get:
<pre>
>1|ENSRNOG00000021093|protein_coding
GTCACTAGGCTCTTAGACCACGGATGGGCGGTACCTGTATCAGGAGGCGGAGCAGCTGCT
<snip>
>10|ENSRNOG00000012620|protein_coding
ATGATCTCTAACTACACCCTAGCTTCTGTGACCACGAAGATGGAGCCATTCAGTCTCCAA
<snip>
>7|ENSRNOG00000017108|protein_coding
GTTTAATAGCCGCCTCTAAGATTTCTCATGGGTATGGTAACACAGGCCTGAAACTCCATT
<snip>
</pre>

* as you noted one of the entries in input IDs does not have proper entry recognised by ENSEMBL. 


== Credits ==

{{credits}}

* [[User:Darked|Darek Kedra]] wrote this tutorial
<!-- other contributors, put yourself here -->


{{stub}}