'''Sequence motifs''' - for more general background see Wikipedias [http://en.wikipedia.org/wiki/Sequence_motif Sequence_motif]. 


It is important to use few complementary programs as well as allow predictions of several motifs per program. There is threshold when it comes to number of sequences used, above which there is no improvement in sensitivity.


==Motif finding programs==
There are obvious trade offs between speed and accuracy/sensitivity when running these programs. Few rules based on review by Hu et al.:
* avoid using longer sequencess than necessary. It does increase noise signal and increases running time. 
* for programs like MEME there is a plateu of motifs found after about 10 input sequences, so one can randomly select 10 sequences out of a larger set and  
then check for occurence of detected motifs in the remaining ones. 
* scores between different programs are incompatible

==Tested set==
===AlignACE=== 
Gibbs sampling algorithm (stochastic), requires multiple runs to get all top hits.

* Web server:  [http://atlas.med.harvard.edu/cgi-bin/alignace.pl ]  Gibbs sampling algorithm

* command line C++, requires compilation

* to run it:
<pre>
./AlignACE -i GAL.seq > test.ace
</pre>

* Accessory programs for motif comparisons and motif finding
CompareACE [http://atlas.med.harvard.edu/cgi-bin/compareace.pl] Compares set of found motifs to a database of TFs (in yeast on the web)

<!--
Usage 1a: CompareACE ace_file1 mot1 ace_file2 mot2
 This compares motif mot1 of ace_file1 with motif number mot2 of ace_file2.
Usage 1b: CompareACE ace_file1 ace_file2
 This performs all pairwise comparisons between the motifs in the specified files (assumes all filenames have some non-digit characters).
Usage 2: CompareACE -all file ace_col mot_col (-c [cutoff])
 This usage performs all pairwise comparisons between all motifs in the specified file and returns those scoring better than the given cutoff (default all).
 The file should be tab-delimited with the name of the AlignACE file in column ace_col and the motif number in column mot_col, specifying one motif per line.
Usage 3: CompareACE -form file1 file2
 These files should contain a set of aligned sites, optionally with asterisks underneath for selection of active columns, and any comment lines (starting with #).
-->

===MEME===
Web: [http://meme.sdsc.edu/meme/intro.html]
* Command line
<pre>
meme -nmotifs numberof_motifs_2_find -model oops  -protein -sequence your_fasta_file -outfile your_fasta_file.meme
</pre>

Where 'model' could be oops/zoops/anr.

oops: One Occurrence Per Sequence

zoops: Zero or One Occurrence Per Sequence

anr: Any Number of Repetitions


===Weeder===
<pre>
./weederlauncher.out your_promoters_file.fasta  MM large A M S t15 
</pre>
Where MM stands for Mus musculus, HS Homo Sapiens atc. See ./FreqFiles/ directory for more.

The output files:
your_promoters_file.mix 
your_promoters_file.wee (output as text)
your_promoters_file.html (output summary in HTML)


===Bioprospector=== 
* Web server: [http://ai.stanford.edu/~xsliu/BioProspector/]
accepts only FastA files with single sequence line
<pre>
>sequence1 name
GGTGACGAC sequence1 as ONE LINE
>sequence2 name
GTAGCCTCATG sequence2 as ONE LINE
</pre>
* fixed widith of motifs (default 10, range: 4-50)
* binaries for Linux, Sun and Cygwin.
* creata background file:
<pre>
./genomebg.linux -i your_background -o your_background.genomeBG
</pre>
* run BioProspector
<pre>
./BioProspector.linux -n 200 -d 1 -r 30 -i target_fasta_seqs -f your_background.genomeBG -o outputbiop1
</pre>

===Improbizer===
[http://www.cse.ucsc.edu/~kent/improbizer/improbizer.html  Web] takes into account position of the motifs
Command line utility called '''ameme'''

<pre>
 ./ameme good=your_test_set.fasta bad=set_of_random_promoter_sequences.fasta numMotifs=10 mo tifOutpu=ameme.out outputLogo background=m2
</pre>


===NestedMica=== 
(java + C program, requires compilation ) [http://www.sanger.ac.uk/Software/analysis/nmica/]

* Calculate background:
<pre>
makemosaicbg -seqs your_input_sequences.fasta -mosaicClasses 1 -mosaicOrder 1 -out your_sequences_background.sbg
</pre>

* Calculate motifs:
<pre>
motiffinder -seqs your_input_sequences.fasta -backgroundModel your_sequences_background.sbg -numMotifs 2
</pre>

* View the motifs:
<pre>
motifviewer motifs.xms
</pre>

Output in xms (an XML variant) format.

* MotifExplorer (Java motif viewer compatible with NestedMica): [http://www.sanger.ac.uk/Software/analysis/nmica/mxt.shtml]
as for Jan 22nd 2007 works on MacOS. Problems on Windows and Ubuntu Linux.

===SOMBRERO=== 
Self Organising Maps [http://bioinf.nuigalway.ie/sombrero/index.html]
* create background model (takec  >15min on 35Mbytes fasta file/2.26GHz Pentium) :
<pre>
perl ./BackExtract.pl -seq your_background_sequences.fasta
</pre>

* Create SOM with motif lenght 8, 10, 12, 14 &16 nucleotides: 
<pre>
./SOMBRERO -t your_target_sequences.fasta -b out.back -lm 8 16 -out target_sombrero.out
</pre>

* View the SOM (requires installation of Tkperl)
<pre>
perl ./SOMBREROView.pl  target_sombrero.ou
</pre>

===MDScan===

* uses restricted fasta format!
<pre>
>sequence1 name
ATGGTGACGAC sequence1 as ONE LINE
>sequence2 name
GTAGCCTCATG sequence2 as ONE LINE
</pre>

* requires sequence background file
<pre>
./genomebg -i inputSequenceFile -o outputDistributionFile
</pre>

* running it:
<pre>
./MDscan -i inseq -w 15 -f yeast_all.bg -t 10 -c 80 -r 10 -n 5 -g 1 
</pre>

find motif of width 15 from sequence file ''inseq'', ''use yeast_all.bg'' as the background distribution. Find candidate motifs from top 10 sequences, and refine 5 iterations from the top 0 sequences. Report the final best 5 motifs to stdout, and do not print out progress messages on the way.

===YMF (not working)===
* C++ program, requires compilation
* running it:
<pre>
./stats stats.config 800 6 ../ymftables/yeast -sort ../examples/abf/abf1 ../examples/abf/cha1
</pre>

segfaults. 

===DIPS===
Obtainable from author. Requires compilation
* example ( ~50 Kbp takes 70mins)
<pre>
dmotif -positive positive.fna -negative negative.fna -len 9 -bkg fly_background.fna -niter 5 -nmotif 1 > dmotifoutput 2> dmotiflog
</pre>
Change '-nmotif 1' to a '-nmotif 10' if you want to get top ten motifs.

==Novel algorithms==

* The AMADEUS Motif Discovery Tool (whole platform)
http://www.cs.tau.ac.il/~rshamir/amadeus/Amadeus.htm

* GAME (java, genetic algorithm) [http://mail.med.upenn.edu/~zhiwei/GAME/]
paper http://bioinformatics.oxfordjournals.org/cgi/content/abstract/22/13/1577

* MotifCut (maximum density subgraphs) [http://motifcut.stanford.edu/]
Paper: http://bioinformatics.oxfordjournals.org/cgi/reprint/22/14/e150
motif lenght: fixed, between 6,31

* GibbsST [http://gbr.cir.tohoku.ac.jp/] Not working yet
Paper(PDF): [http://scholar.google.com/url?sa=U&q=http://www.biomedcentral.com/content/pdf/1471-2105-7-486.pdf]

* THEME (ChIP-chip only??)
http://bioinformatics.oxfordjournals.org/cgi/content/abstract/22/4/423

==Ortholog sequences==
* PhyME  [http://edsc.rockefeller.edu/cgi-bin/phyme/download.pl]
Paper: [http://www.biomedcentral.com/1471-2105/5/170]

* PhyloGibbs [http://www.imsc.res.in/~rsidd/phylogibbs/]
Paper: [http://compbiol.plosjournals.org/perlserv?request=get-document&doi=10.1371/journal.pcbi.0010067]
* PhyloCon [http://ural.wustl.edu/~twang/PhyloCon/]
Paper: [http://bioinformatics.oxfordjournals.org/cgi/content/abstract/19/18/2369]

==Multiple algorithms / metaservers==
* Credo [http://mips.gsf.de/proj/regulomips/credo.htm]
Visualisation of AlignACE, DIALIGN, FootPrinter, MEME and MotifSampler results.
Paper: [http://bioinformatics.oxfordjournals.org/cgi/content/full/21/23/4304]

* BEST 
Download: [http://www.cs.uga.edu/~che/BEST/] (Qt on linux)
BEST includes four commonly used motif-finding programs: AlignACE, BioProspector, CONSENSUS and MEME, as well as the optimization program BioOptimizer. 
Paper: [http://bioinformatics.oxfordjournals.org/cgi/content/abstract/21/12/2909]

* Multifinder http://the_brain.bwh.harvard.edu/MultiFinderSuppl/ (download)
* RgS-Miner (web, as of 2007.05.22: uses gene list but not sequences yet) http://rgsminer.csie.ncu.edu.tw/

==Combining/comparing motifs==
* MatCompare 
** set of user provided PWMs: [http://rulai.cshl.edu/cgi-bin/MatCompare/home.cgi?process=compareMatricesForm]
** single PWM against TRANSFAC or JASPAR [http://rulai.cshl.edu/cgi-bin/MatCompare/home.cgi?process=compare2knownForm]
** simple correlation vlue between each apir of motifs

* CompareACE [http://atlas.med.harvard.edu/download/extra.html] supporting program for AlignAce

* '''STAMP''' [http://biodev.hgen.pitt.edu/stamp/]
Handles 12 various output formats from a wide range of motif finding programs. Compares these motifs to known TFBS from JASPAR and other, also user-defined databases of motifs.
Converts the output to intermediate format accessible on the server under "X motifs loaded".

==APIs==
* Cistematic http://cistematic.caltech.edu/
Python package with interfaces to i.e. MEME, AlignACE, Co-Bind, and FootPrinter
Paper [http://www.genome.org/cgi/content/full/16/10/1208]

* TAMO http://fraenkel.mit.edu/TAMO/
Python/C++ package with Interfaces to MDscan, AlignACE, and MEME.
Paper: [http://bioinformatics.oxfordjournals.org/cgi/content/full/21/14/3164]

==Databases==
* [http://www.oreganno.org/oregano/Index.jsp ORegAnno] database as a dynamic collection of literature-curated regulatory regions, transcription factor binding sites and regulatory mutations (polymorphisms and haplotypes).
Paper: [http://bioinformatics.oxfordjournals.org/cgi/content/abstract/22/5/637]


==To Read ==
* Tompa review of 14 programs Nat Biotech. 2005 g[http://www.nature.com/nbt/journal/v23/n1/full/nbt1053.html]
* Maximilian Haeussler's [http://www.stud.uni-potsdam.de/~haussler/master/index.php  master thesis] and his [[Tregwiki]] @Openwetware:
* Comparison of several programs: Hu NAR 2005 [http://nar.oxfordjournals.org/cgi/content/full/33/15/4899#TBL1]
* [http://bioinformatics.caltech.edu/cis-analysis.txt Erich Schwarz's list from 2002]
* [http://biobase.ist.unomaha.edu/mediawiki/index.php/Main_Page Motif Tool Assessment Platform (MTAP)] wiki from Omaha

See also: http://www.connotea.org/user/darked89/tag/motif

<!--
* montgomery_stephen PhD thesis 2 read
https://dspace.library.ubc.ca/dspace/bitstream/2429/58/1/ubc_2006_fall_montgomery_stephen.pdf
Sockeye Chipotle
-->

==External courses/tutorials==
* EMBL 2001 [http://www1.imim.es/courses/embl2001/promoters/]
* Harward (AlignAce) [http://arep.med.harvard.edu/labgc/amcguire/IBSS/]

==stuff to incorporate ==

*  [http://biodev.hgen.pitt.edu/footer_php/Footerv2_0.php FOOTER] and [http://nar.oxfordjournals.org/cgi/content/abstract/33/suppl_2/W442 FOOTER paper]

* [http://melina2.hgc.jp/public/index.html Melina II ] Metaserver (uses four out of five programs: CONSENSUS, MEME, Gibbs Sampler, MDScan and Weeder) by Kenta Nakai @Tokyo University. [http://nar.oxfordjournals.org/cgi/content/abstract/35/suppl_2/W227 Peper]

== Credits ==

{{credits}}

* [[User:Darked|Darek Kedra]] wrote this tutorial
<!-- other contributors, put yourself here -->



{{stub}}