[[Wikiomics:Bioinfo tutorial (exercises)]]

==Introduction==
The goal of this document is to provide a set of illustrative cookbook recipes for wet lab biologists. As there is more than one way of performing a given task, it does point to different tools, but it does not even try to become a yet another directory of links. 

You are strongly encouraged to post your comments/ideas on the discussion page of this tutorial.

==Before you start==

* use text files/plain e-mail whenever possible
* give meaningful names to your files (i.e. humP53_XP12345.fas)
* do not use spaces or Unix special characters in file names (!?"'%&^~*$|/\{}[]()<>:)
* create separate folders/directories for each project with meaningful names
* do not waste paper for printing out your blast search results or sequences
* if primary data needs to be archived burn it on CD using (if possible) non vendor-specific data format

==Operating systems/General software==
For installing standalone bioinformatics programs the order of preference is:
Linux/Unix > MacOS > Windows

Once you move from worksation to bioinformatics application server (almost always Linux/Unix) then as long as you have a Xwindows connection (preferably ssh tunneled) operating system of client workstation plays a minor (if any) role.
 
<!-- While many tasks can be completed using web servers for a number of others one needs to have  -->

== DNA sequence processing ==

===Sequence formats===

There are multiple sequence formats commonly used. These can be divided into text-only and binary formats.

====Text-only====
=====Unaligned sequences=====
* '''Raw sequence''' ([[Wikiomics:Bioinfo_tutorial_%28exercises%29#text_only.2Fraw_sequence| example]]) The text file contains nothing but sequence. The information that's inside has to be encoded in the name of the file. Some web servers still require it.

* '''FASTA''' ([http://en.wikipedia.org/wiki/FASTA_format description], [[Wikiomics:Bioinfo_tutorial_%28exercises%29#FASTA | example]]) The most commonly used format (by programs/servers). It can contain either single or multiple sequences, each having a name line, also known as the header, starting with ">" in the first column. Keep in mind that various programs/servers introduce extra restrictions how the FastA header should look like. The "safe" standard:
** Alphanumeric characters only [a-z, A-Z, 0-9], avoid spaces!
** Keep the names short [i.e. 30 characters are significant for clustalw, only 8 for older versions of t_coffee]
** In multiple FastA format keep the names unique

Whenever you plan to analyze your sequence using web servers or command line programs, keep it as FASTA. It is easy to cut the raw sequence out of it, and if your file name/sequence names contain accession numbers to find the GenBank entry.

* '''GenBank''' format of choice used by GenBank to display extra information about the sequence. It is fairly complex, but easily readable by humans. Keep your sequences as GenBank when you need extra annotations provided by this format.

* '''GCG''' GCG package own format. If you need to use GCG, you will have to convert your sequences into it

* '''XML''' eXtended Markeup Language. Standard language slowly taking over the Net. XML files are computer-readable (= information is fairly easy to extract from them) and possible to read by humans. Format of choice when complex sequence information need to be described and computer accessible.

* '''EXP''' format used by [[Wikiomics:Staden]] sequencing package. Contains both sequence and annotations understandable by [[Wikiomics:gap4]].

* '''GFF''' format for (mostly) genomic sequence annotations used by Ensembl [http://www.sanger.ac.uk/Software/formats/GFF/GFF_Spec.shtml]

=====Aligned sequences=====     
* '''MSF''' Multiple Sequence Format. Used as output for multiple sequence alignments (pileup from GCG) and as input for alignment editors and shadowing programs.

* '''ALN''' standard for multiple sequence alignments. Output format of clustalw and t_coffee.

* '''PHYLIP''' used as an input in such programs like PHYLIP package, phyml

* '''NEXUS''' advanced format allowing more annotation, used by MrBayes

====Binary====

*'''ABI''' binary output format of ABI sequencers. Bulky (= unnecessary large), but so far the only one readable on Mac-OS. Can be easily converted into SCF.

*'''SCF''' Sequence Common Format. Standard output for ALF sequencers. More compact than ABI, and commonly used in sequencing projects. Readable on PC and Unix, binary format of the Staden package.

*'''ZTR''' successor of SCF with internal data compression. Currently the default binary format for Staden.

===Sequence conversion tools ===

* [http://bimas.dcrt.nih.gov/molbio/readseq/ readseq]
OK for simple format, has problems with aligned sequences
* [http://www.bioperl.org/wiki/SQUID sreformat]  from [ftp://selab.janelia.org/pub/software/squid/squid.tar.gz SQUID]
A function in Sean Eddy's SQUID package for converting between unaligned and aligned sequence formats.
* [http://www.bioperl.org/wiki/Bptutorial.pl#Transforming_sequence_files_.28SeqIO.29 BioPerl::SeqIO] 
can be used for sequence conversion, also for many other bioinformatic related functions.
* for convertions between aligned formats (ALN, MSF, NEXUS, PHYLIP) use: 
# [http://pbil.univ-lyon1.fr/software/seaview.html SeaView] GUI program 
# [http://insilico.ehu.es/tophylip/ ToPhylip] (WWW) ALN and MSF -> Phylip

===Processing ABI files and sequence assembly===

Before you even start sequencing, plan the way the sequences will be processed. The most important is to establish a proper naming scheme compatible with various programs you (or others) may use downstream for sequence processing. The basic rules for file names are:
* use alphanumeric characters [a-z, A-Z, 0-9]
* use underscore "_" instead of spaces
* full stop shall separate information about sequencing template and primer, like: 
<pre>
prefix_my-template.my-primer_reaction-number
</pre>

Even small sized projects become cumbersome if file names are not unique (you can not assembly files having the same name!).
Once these requirements are satisfied, file name alone will give you enough information about:
* whose sample is it?
* to which sequencing project does it belong?
* what was the sequencing template?
* which primer was used?
* was this sample sequenced before?

===Programs for DNA sequence assembly===

====Fasta files (no traces)====
* phrap
<pre>
phrap your_multiple_fasta_file.fa 
</pre>

Results in: your_multiple_fasta_file.fa.contigs  

Used for i.e assembly of ESTs.

====ABI/SCF files (traces)====

'''recommended:'''
*  [http://staden.sourceforge.net/ Staden package] (Unix, Windows, OSX)
see also another Wikiomics page [[Wikiomics:Staden]]

**pregap4 (sequence pre-processing) 
Takes a directory or a list of files for preprocessing (creating EXP files, clipping bad quality sequence, marking sequencing vector, repeats etc. 

Creating sequence out of ABI chromatogram file is done by a basecalling program. One can go along with ABI-provided basecaller or preferably use: 
* '''phred'''  [http://bozeman.genome.washington.edu/phrap.docs/phred.html] giving more accurate calls for less accurate part of the sequence (like at the end of the run, say 600bp and more) . Phred also gives a probability/quality values for each of the bases allowing more accurate assembly.

<pre>
pregap4 %
</pre>

**gap4 (sequence assembly, editing, verification and much more)
<pre>
gap4 &
</pre>
or (to open a specific database at startup):
<pre>
gap4 database.version.aux &
</pre>

Other packages:
* Consed [http://www.genome.washington.edu/UWGC/analysistools/consed.cfm]
* Seqman from Lasergene (PC & Mac) [http://www.dnastar.com/products/seqmanpro.php]
* Sequencher  [http://www.genecodes.com/]

<!-- other stuff
http://informagen.com/SA/
http://www.mbio.ncsu.edu/BioEdit/bioedit.html
-->

===Vector masking===
<!-- $$ needs checking $$ -->
* EMVEC (web and blast2 based) [http://www.ebi.ac.uk/blastall/vectors.html]
Good for a single sequence.

* vectorstrip (Emboss)

<pre>
vectorstrip -sequence  my_fasta_file -vectors my_vectors_fasta_file -mismatch 30
   -linkera TAGGATACAG -linkerb TTTTTCAGGACAGACA 
</pre>

;linkera: 5' sequence
;linkerb: 3' sequence

* cross_match (from phrap package)

<pre>
cross_match my_fasta_file my_vectors_fasta_file -minmatch 10 -minscore 20 -screen > screen.out
</pre>

You will get a file: my_fasta_file.screen --- "devectorised" my_fasta_file

For masking out, let's say, one gene/overlapping clone, etc., type: 
<pre>
cross_match fasta_file_to_be_masked masking_sequence -minmatch 10 -minscore 20 -screen > screen.out
</pre>

You will get a file: fasta_file_to_be_masked.screen without the sequences matching masking_sequence.

===Masking repetitive sequences using RepeatMasker (WWW/Unix)===

* Online: [http://www.repeatmasker.org/cgi-bin/WEBRepeatMasker]

* command line    

You have to have a FastA file (it can be multiple FastA). Type:

<pre>
repmask your_sequence_in_fasta_format
</pre>
You will get a file: your_sequence_in_fasta_format.masked --- name tells all

species options (choose only one):
<pre>
-m(us) masks rodent specific and mammalian wide repeats
-rod(ent) same as -mus
-mam(mal) masks repeats found in non-primate, non-rodent mammals
-ar(abidopsis) masks repeats found in Arabidopsis
-dr(osophila) masks repeats found in Drosophilas
-el(egans) masks repeats found in C. elegans
</pre>

===Primer design===

General info about primer design/PCR:
* http://www.mcb.uct.ac.za/pcroptim.htm
<!-- gone... http://www.hgmp.mrc.ac.uk/MANUAL/faq/faq-primers.html -->

online:
* '''Primer3''' [http://www-genome.wi.mit.edu/cgi-bin/primer/primer3_www.cgi]
* '''Oligo''' (PC & Mac ) - $$$, first choice. [http://www.oligo.net/]
* '''Primer Select''' from Lasergene (PC & Mac)
<!--
Primo?? http://www.changbioscience.com/primo/ 
-->

===DNA -> protein translation===
* transeq (Emboss) [http://www.ebi.ac.uk/emboss/transeq/ WWW server]
* Protein Translate (ExPasy): [http://www.expasy.ch./tools/dna.html WWW server]
* ORF Finder [http://www.ncbi.nlm.nih.gov/gorf/gorf.html WWW server]
<!-- * EditSeq (Lasergene, Mac, PC) -->

==Cloning in silico==
===ESTs based===
Pre-clustered ESTs:
* STACKdb (limited access, tissue specific splice forms) [http://ww2.sanbi.ac.za/Dbases.html]
* Unigene (no consensus sequence) [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=unigene]
* TIGR [http://www.tigr.org/tdb/tgi/index.shtml]

===Genome based===
* based on homology
* de novo

see also separate page: [[Wikiomics:Cloning_in_silico]]

===Genome annotation using ESTs assembly===

* PASA http://www.tigr.org/tdb/e2k1/ath1/pasa_annot_updates/pasa_annot_updates.shtml

==Synthetic genes==
Many genes due to an extreme codon bias can not be expressed in model organisms, such as E.coli. To fix this problem one can back-translate protein sequence into nucleotides, using codons preferably used by E.coli. Moreover one can construct sucgh gene from a set of overlapping oligos, introducing restriction sites or other features at will. Programs performing such tasks:

* '''JCAT''' http://www.jcat.de/
* '''GeneDesign''' http://slam.bs.jhmi.edu/gd 
* [http://www.evolvingcode.net/codon/sgd/index.php SGD] Syntetic Gene Designer 
* '''Synthetic Gene DataBase''' [http://www.evolvingcode.net/codon/sgdb/index.php]
* stand alone program UpGene (SmallTalk) http://www.vectorcore.pitt.edu/upgene/upgene.html

* Codon preference in ''Pichia pastoris''
http://www.microbialcellfactories.com/content/4/1/33


* Ref
<pre>
Applied Microbiology and Biotechnology
Issue:  Volume 67, Number 3
Date:  May 2005
Pages: 289 - 298
Strategies for efficient production of heterologous proteins in Escherichia coli
S. Jana1 and J. K. Deb
</pre>

==Codon usage==
There is a bias in codon usage resulting in variation in translation speed/efficiency. The bias itself varies between organisms (i.e. it is strong in yeast, then in E.coli, much weaker in B.subtilis).

* codon usage database [http://www.kazusa.or.jp/codon/]
* graphical codon usage analysis http://gcua.schoedl.de/

One can calculate a number of codon bias indexes using programs:
* CodonW [http://bioweb.pasteur.fr/seqanal/interfaces/codonw.html web server], [http://sourceforge.net/projects/codonw/ Sourceforge project]

==RNA==

The [http://www.tbi.univie.ac.at/~ivo/RNA/ Vienna RNA package] has solutions for many classical RNA problems. It well worth taking some time to familiarize oneself with the algorithms in this package before checking out the alternative methods.

===Secondary structure prediction=== 

For [[Wikiomics:RNA secondary structure prediction]] from a single sequence the most commonly used tools are [http://www.bioinfo.rpi.edu/applications/mfold/ MFold] and [http://www.tbi.univie.ac.at/~ivo/RNA/ RNAfold].
For more single sequence and other comparative methods a large range of [[Wikiomics:RNA_secondary_structure_prediction | methods are available]]. 

For the best accuracy (and evolutionary information) [[Wikiomics:RNA_secondary_structure_prediction#Comparative_structure_prediction |comparative]] approaches, in particular the alignment folding methods, have proven rather successful. Typically a researcher will align sequences using tools such as ClustalW, MAFFT and ProAlign and then fold the alignment using tools such as RNAalifold or Pfold.

===RNA homology search===
For the [[Wikiomics:RNA homology search]] problem (including secondary structure) a number of algorithms exist. E.g. [http://www.genetics.wustl.edu/eddy/infernal/ Infernal] and [http://bio.cs.washington.edu/supplements/zasha-ravenna/ RaveNnA]. A number of tools specializing in a specific RNA family are also available eg. tRNA, miRNA etc.

===RNA gene-finding===
Tools for [[Wikiomics:RNA gene-finding]] usually use as input sequence alignmments which are then scanned for potential conserved secondary structure signals. Useful examples of these are [http://www.genetics.wustl.edu/eddy/software/#qrna QRNA], [http://www.tbi.univie.ac.at/~wash/RNAz/ RNAz] and [http://www.cbse.ucsc.edu/~jsp/EvoFold/ EvoFold].

===RNA databases===
A number of [[Wikiomics:RNA databases]] are available.
* [http://microrna.sanger.ac.uk/ microRNA database] (sequences, targets, sumbmission for publications)
* [http://www.sanger.ac.uk/Software/Rfam/ Rfam] a large collection of multiple sequence alignments and covariance models covering many common non-coding RNA families.
* [http://www.binf.ku.dk/~pgardner/bralibase/ BRaliBase] Benchmarks of RNA analysis tools on trusted [http://en.wikipedia.org/wiki/NcRNA ncRNA] datasets.

==Protein Alignment==
See also article in wikipedia: [http://en.wikipedia.org/wiki/Sequence_alignment_software Sequence alignment software]

===Two sequences===
There are two kinds of alignment: local and global. 

;Local : finds the best region of similarity between two sequences
* '''water''' (Emboss) uses the Smith-Waterman algorithm (modified for speed) [http://www.ebi.ac.uk/emboss/align/] 
* '''lalign''' Huang and Miller algorithm [http://www.ch.embnet.org/software/LALIGN_form.html]
* '''blast2''' [http://www.ncbi.nlm.nih.gov/blast/bl2seq/wblast2.cgi]
* '''bestfit''' (GCG) Smith-Waterman algorithm


;Global : covers the whole length of both sequences
* '''needle''' (Emboss) Needleman-Wunsch global alignment [http://www.ebi.ac.uk/emboss/align/]
* '''gap''' (GCG) Needleman-Wunsch global alignment

<!-- Estimating accuracy of pairwise sequence alignment: -->

===Multiple sequences===
For a recent review see [http://compbiol.plosjournals.org/perlserv/?request=get-document&doi=10.1371%2Fjournal.pcbi.0030123 Notredame 2007 @PLOS]

There is a trade off between accuracy and speed. For speed select MUSCLE. Accuracy of algorithms differs, with the best results on average being achieved using PRALINE and t-coffee/M-coffee. For poteins with multiple/repeated domains you may try graph based algorithms: '''poa''' or '''aba'''. 

* '''t_coffee''' (good at producing alignments of distant (~30% similarity) sequences. [http://www.ch.embnet.org/software/TCoffee.html]
<pre>
t_coffee your_fasta_file.fa
</pre>
T-Coffee has a limit of 50  sequences. The new version is called M-Coffee, it is a meta-aligner using output of several other algorithms (ClustalW, Poa, Muscle, ProbCons, MAFFT, Dialign-T, PCMA). To run it (bash):
<pre>
export POA_DIR=/absolute/path/to/Poa/installation/directory
export DIALIGNT_DIR=/absolute/path/to/DIALIGN-T/conf
t_coffee your_fasta_file.fa -special_mode mcoffee
</pre>

* '''PRALINE''' (slow but takes into account blast results) [http://ibivu.cs.vu.nl/programs/pralinewww/]
Uses PSI-BLAST for finding most conserved residues in each of the sequences. Very slow but could be more accurate as compared with other methods. 

* '''clustalw''' (standard algorithm most commonly used and "good enough" in most cases) [http://www.ebi.ac.uk/clustalw/]
<pre>
clustalw your_fasta_file.fa
</pre>
output as aln format
<!--* '''emma''' (Emboss) (clustalw front end) -->

* '''dialign''' (good for creating alignments with long gaps as with non-orthologues sharing domains only) [http://bibiserv.techfak.uni-bielefeld.de/cgi-bin/dialign_submit]
Improved, command line version is DIALIGN-T
[http://dialign-t.gobics.de/submission?type=protein]
<pre>
dialign-t ./path/to/dialign-t/conf/ your_fasta_file.fa your_fasta_file.dialign-t
</pre>
output is in fasta format

* '''MUSCLE''' http://www.ebi.ac.uk/muscle/
<pre>
muscle -clw -in your_fasta_file.fa -out your_fasta_file.aln -clw
</pre>

* '''probcons''' http://probcons.stanford.edu/
<pre>
probcons -clustalw -ir 100 -pre 20 -a your_sequence_in_fasta_format.fa > probcons.aln
</pre>
<!--Caveat: there is a problem with version 1.1 with clustalw: One has to add:
'CLUSTAL FORMAT for ' before PROBCONS version 1.1 multiple sequence alignment
-->
* '''poa''' http://www.bioinformatics.ucla.edu/poa/ 
Graph based algorithm replacing columns with directed acyclic graph. Permits domain recombinations of domains/exons: useful for the alignment of multidomain proteins and ESTs.
<pre>
 poa -read_fasta your_fasta_file.fa  -clustal output_file.aln /path/to/similarity/matrix/i.e/blosum80.mat 
</pre>

* '''aba''' http://aba.nbcr.net/aba.html
Method using A-Bruijn graph. Good for alignment of proteins with:
** domains that are not present in all proteins
** domains that are present in different orders in different proteins
** domains that are present in multiple copies in some proteins
DNA: ABA is useful in the alignment of genomic sequences that contain duplications and inversions.
Based on: http://www.genome.org/cgi/content/abstract/14/11/2336

* '''MEME''' (good for finding as-yet-unknown domains in a set of proteins) [http://meme.sdsc.edu/meme/meme.html]

<pre>
meme -nmotifs numberof_motifs_2_find -model oops  -protein -sequence your_fasta_file.fa.fa -outfile your_fasta_file.fa.fa.meme
</pre>
Where 'model' could be oops/zoops/anr.

oops:    One Occurrence Per Sequence

zoops:   Zero or One Occurrence Per Sequence

anr: 	Any Number of Repetitions


<!--* '''pileup''' (GCG) 

* '''satchmo''' new (2003) algorithm (constructs a tree and a set of multiple sequence alignments, one for each internal node of the tree). [http://phylogenomics.berkeley.edu/cgi-bin/satchmo/input_satchmo.py]-->

MEME 3.5.3 syntax:
<pre>
meme your_input_fasta_file.fas         -nmotifs 20 -mod anr  -dna     > your_meme_output.htm &
</pre>

===Alignment accuracy assessment===
* [http://msa.cgb.ki.se/cgi-bin/msa.cgi MUMSA] (WWW) 
all alignments have to in aligned fasta format  and contain the same sequences in the same order.

Paper:
http://nar.oxfordjournals.org/cgi/content/full/33/22/7120

===Alignment editors & beautifiers===
* '''Jalview''' (Java) [http://www.jalview.org/]
good for sequence conversion (ALN, MSF, Fasta,Pfam, PIR, BLC)
* '''clustalx''' (PC, Unix) [http://bips.u-strasbg.fr/fr/Documentation/ClustalX/]
ALN only
* '''seaview''' (Linux) [http://pbil.univ-lyon1.fr/software/seaview.html]
good for sequence conversion  (ALN, MSF, Fasta, Mase, Phylip, Nexus)
* '''genedoc''' (PC only) [http://www.psc.edu/biomed/genedoc/]

* '''boxshade''' (no editing, pretty prints only) [http://www.ch.embnet.org/software/BOX_form.html] 
ALN, MSF

* '''TeXshade'''  [http://www.ctan.org/tex-archive/help/Catalogue/entries/texshade.html]
LaTeX package. accepts ALN and MSF formats.

===Sequence logos===
'''MSA or matrix'''
* [http://biodev.hgen.pitt.edu/cgi-bin/enologos/enologos.cgi enoLOGOS] (first choice)

'''MSA'''
*  [http://weblogo.berkeley.edu/logo.cgi WebLogo] (input as Fasta, ALN formats)

'''HMM'''
* LogoMat-M http://www.sanger.ac.uk/cgi-bin/software/analysis/logomat-m.cgi
(single HMM profile)

*  LogoMat-P  http://www.sanger.ac.uk/cgi-bin/software/analysis/logomat-p.cgi
(pair-wise HMM profiles)


==DNA alignment==

===coding DNA alignment based on existing protein alignment===
This is a method of constructing more accurate DNA alignment for downstream use in phylogenic analyses taking into account coding properties of given DNA sequences.

* [http://www.personal.uni-jena.de/~b6biol2/ProgramsMain.html transAlign] Perl script, requires Clustalw [http://www.biomedcentral.com/1471-2105/6/156 article]
* [http://www.cbs.dtu.dk/services/RevTrans/ RevTrans]
* [http://bioweb.pasteur.fr/seqanal/interfaces/protal2dna.html protal2dna]

===Genomic DNA  against genomic DNA ===
* [http://pipmaker.bx.psu.edu/cgi-bin/pipmaker?basic PipMaker] aligns two DNA sequences
* [http://pipmaker.bx.psu.edu/cgi-bin/multipipmaker MultiPipMaker (web server)] [http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=168985 article] Aligns 2-30 DNA sequences

* [http://www.biomedcentral.com/1471-2105/6/9 GATA paper]
GATA: a graphic alignment tool for comparative sequence analysis

* [http://zeus.cs.vu.nl/programs/aubergenewww/ AuberGene] stand alone Python scripts. Requires blastz for pairs comparison. 


===Non-coding DNA sequences===

* '''Sigma''' [http://www.imsc.res.in/~rsidd/sigma/]
Designed specially  for non-coding DNA sequence
Paper: [http://www.biomedcentral.com/1471-2105/7/143/abstract]

==Similarity plots (dot-plots)==

* '''dotter''' (Unix, PC) [http://www.cgr.ki.se/cgr/groups/sonnhammer/Dotter.html] 
Create dot-plots (DNA-DNA, protein-protein and DNA-protein) between one or more sequences. Ideal for quick estimation of sequence similarity, gene organization (genomic DNA vs cDNA), domain finding for small/medium size sequences. One can also zoom an area of interest.
Memory and CPU is the limit but one can compare 200kbp genomic DNA vs 6kbp cDNA in ca 2min on 2GHz Pentium 4.  

<pre>
dotter first_fasta_DNA.seq second_fasta_DNA.seq
dotter first_fasta_AA.seq second_fasta_AA.seq
dotter first_fasta_DNA.seq second_fasta_AA.seq
</pre>
Note that the order of DNA vs protein comparison matters. Try to keep sequence names within your multiple_fasta file short.

* '''dottup''' (Emboss, Unix/Web) [http://bioweb.pasteur.fr/docs/EMBOSS/dottup.html info] [http://bioweb.pasteur.fr/seqanal/interfaces/dottup.html web form]

<!-- * '''Megaalign''' (Lasergene, PC/Mac) [http://www.dnastar.com/web/r10.php] -->

==Database searches (sequence similarity)==
There is a tradeoff between finding all possible matches (like with Smith-Waterman) and speed. For most applications '''Blast''' (and '''Psi-Blast''') is a first choice. For massive searches, where speed is essential (i.e. mapping all cDNAs to a mammalian genome) use '''blat'''

===Regular expressions===
Searches for short motifs, like [MC]-A-x-E(2)-{IE} (M or C followed by A followed by any followed by EE followed by anything but I or E)

* fuzznuc (from Emboss)
search nucleotide searches
<pre>
fuzznuc -sequence your_sequence_for_searching.fasta -pattern 'GATTACCA' -mismatch number_of_mismatches_allowed -outfile output_file.txt
</pre>
* fuzzpro(from Emboss)
search protein sequences
<pre>
fuzzpro -sequence your_sequence_for_searching.fasta -pattern 'MEGGADE' -mismatch number_of_mismatches_allowed -outfile output_file.txt
</pre>

* dreg (from Emboss)- 
Regular expression search of a nucleotide sequence
<pre>
dreg -sequence your_sequence_for_searching.fasta -pattern '[CG](5)TG{A}N(1,5)C' -mismatch number_of_mismatches_allowed -outfile output_file.txt
</pre>
* preg (from Emboss)
Regular expression search of a protein sequence 
<pre>
preg -sequence your_sequence_for_searching.fasta -pattern '[MARE](2)YDY{AGWK}' -mismatch number_of_mismatches_allowed -outfile output_file.txt
</pre>

For more help about options/switches in Emboss programs type:
<pre>
program_name -help -verbose
</pre> 
* '''ScanProsite''' [http://us.expasy.org/tools/scanprosite/]
[CG](5)TG{A}N(1,5)C 

corresponds to:

[C or G] (five times), T, G, {anything except A},1-5 of any, C 

More about pattern syntax: [http://us.expasy.org/tools/scanprosite/scanprosite-doc.html#patsyntax]

===First tier algorithms===
* '''Blast''' [http://www.ncbi.nlm.nih.gov/BLAST/]
Command line:
check first if $BLASTDB and $BLASTMAT are set (it should display these names sans $ signs & pathways):
<pre>
env | grep BLAST
</pre>

If you get a blank and these are not set in your '~/.ncbirc' file (tcsh):
<pre>
setenv $BLASTDB /path/to/directory/with/blast/databases/
setenv $BLASTMAT /path/to/your/director/blast/data
</pre>
or follow instructions for creating '~/.ncbirc' from here: [http://genome.nhgri.nih.gov/blastall/blast_install/]

to create a database out of your fasta file:
<pre>
formatdb -i your_multiple_protein_file.fa -p T -V
formatdb -i your_multiple_DNAseq_file.fa  -p F -V
</pre>
For more options type 'formatdb -h'

**nucleotide search
<pre>
blastall -p blastn -d ecoli_nn -i my_nn_fasta_file.fa -o my_nn_fasta_file_vs_ecolidb.blastn_out
</pre>
**protein search
<pre>
blastall -p blastp -d ecoli_aa -i my_nn_fasta_file.fa -o my_nn_fasta_file_vs_ecolidb.blastp_out
</pre>

* Blast2 [http://www.ncbi.nlm.nih.gov/blast/bl2seq/wblast2.cgi]
finding similarity between two sequences/alignment + dotplot on the web f

comparing protein sequence with DNA:
<pre>
bl2seq -p tblastn -F N -i your_peptide_sequence.fasta -j your_nucleotide_sequence.fasta
</pre>
* '''Blat''' 
** description: http://www.genome.org/cgi/content/full/12/4/656

** web server: http://genome.ucsc.edu/cgi-bin/hgBlat?command=start
** command line
<pre>
blat /path/to/database your_query_in_fasta.fa [-ooc=11.ooc] output.psl
</pre>
-ooc=11.ooc option speeds up search by using a set of over-occuring 11-mers
to create your database out of DNA sequence file:
<pre>
faToTwoBit input_fasta.fa [in2.fa in3.fa ...] output.2bit
</pre>
use output.2bit as a database in your blat searches.

===Slower but more sensitive===
* '''FastA''' [http://www.ebi.ac.uk/fasta33/]
* '''Smith-Waterman/MPsrch''' [http://www.ebi.ac.uk/MPsrch/]
** only Protien queries. Emboss '''water''' [http://emboss.sourceforge.net/apps/cvs/water.html] can be used to search local nucleotide databases .
* '''Scanps (SW variant)''' [http://www2.ebi.ac.uk/scanps/]


===HMM and profile searches===
* PSI-BLAST 

* HMM searches <!--(separate doc in preparation)-->

** hmmer [http://hmmer.wustl.edu/]
<pre>
hmmbuild your_ALIGNED_sequences.hmm your_ALIGNED_sequences.fasta
hmmcalibrate your_ALIGNED_sequences.hmm
hmmsearch your_ALIGNED_sequences.hmm protein_database.fasta
</pre>

** MEME MAST [http://meme.sdsc.edu/meme/intro.html]
Command line:
<pre>
mast your_meme_motifs.meme -d your_database.multiple_fasta
</pre>


<!--===Genome 2 Genome comparisons===
* Wobble-Aware Bulk Aligner (WABA) [http://www.soe.ucsc.edu/%7Ekent/xenoAli/index.html]-->

==Data mining==
finding basic info about known gene

* EntrezGene [http://www.ncbi.nih.gov/entrez/query.fcgi?db=gene]
* Omim [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=OMIM]
* GeneCards [http://bioinfo.weizmann.ac.il/cards/index.html] 
* HomoloGene [http://www.ncbi.nlm.nih.gov/HomoloGene/]

==Phylogeny==
$$$ very preliminary at the moment $$$

# select a set of sequences
# create alignment
# remove bad regions of the alignment/gaps (tree constructing programs deal badly with indels)
# for more than 12 sequences tha space of all possible tree is to large to search one-by-one -> heuristic methods are used

===Methods===

* maximum parsimony MP
sensitive to the order of sequences.

* distance methods
** Neighbour joining NJ
** UPGMA (Unweighted Pair Group Method with Arithmetic mean) assumes rate of mutation constant along branches of a tree. Less acurate for sequence data than NJ.

* maximum likehood (computationaly intense, but more accurate)


To check validy of a tree:
* bootstrap
After creating a tree from MSA bootstraping program selects number of columns from MSA and calculates a tree. Thsi process is repeated 100-1000 times and resulting trees are compared with previously created one. Branches present in more than 70% of bootstrap samples are believed to have a good support. 
These estimates are conservative 
<!-- * jackknife ??parsimony jackknife -->

* multiple samples from the posterior distribution (MrBayes)
Probably overestimate confidence values [http://mbe.oxfordjournals.org/cgi/content/full/21/1/188]

===Programs===
* PHYLYP (phylogeny standard package) see a guide to it:  [http://koti.mbnet.fi/tuimala/oppaat/phylip2.pdf]
Individual programs can be scripted. On unix (bash): 
** create a new directory 
** put your_aligned_proteins.phylip there
** start any text editor, modify content below to suit your options, save as i.e. run_proml_with_my_options.sh
<pre>
#!/usr/bin/sh
proml << EOF
your_aligned_proteins.phylip
1 
S
y
EOF
</pre> 

** run it
<pre>
source run_proml_with_my_options.sh
</pre>
** after completein the run it reports:
<pre>
Output written to file "outfile"
Tree also written onto file "outtree"
</pre>
* PHYML http://atgc.lirmm.fr/phyml/ (ML)
* MrBayes http://mrbayes.csit.fsu.edu/ (Bayes / Markov chain Monte Carlo.)
  requires NEXUS format. To tun it on command line: 'mb'
* RAxML-VI http://www.ics.forth.gr/~stamatak/index-Dateien/Page443.htm (ML for large phylogenetic trees)
* TREE-PUZZLE http://www.tree-puzzle.de/ (ML)
* [http://mesquiteproject.org/mesquite/mesquite.html Mesquite] collection of modules for performing a wide range of analyses from various branches of evolutionary biology (e.g., phylogenetics, molecular evolution, population genetics, geometric morphometrics)

===Molecular clock===
* BEAST http://evolve.zoo.ox.ac.uk/beast/ (Bayes) 

===Tree display===
* Phylodendron http://iubio.bio.indiana.edu/treeapp/treeprint-form.html
* njplot http://pbil.univ-lyon1.fr/software/njplot.html
* forester http://www.genetics.wustl.edu/eddy/forester/

===References===
* extensive list of programs: http://evolution.genetics.washington.edu/phylip/software.html
* tutorial articles:
** http://eebweb.arizona.edu/courses/ecol335/Spring%202006%20active/Papers/Phylogeny%20faint%20heart.pdf
** http://scholar.google.com/url?sa=U&q=http://www-edlab.cs.umass.edu/cs691k/blanchard/HolderLewis2003.pdf

==Genomic structure==

===One gene in one species === 
Almost 20 species have as for today (May 2006) separate entries in Ensembl. Possible searches include: blast, GeneBank accession number, gene symbol, chromosomal band, marker etc. To elucidate genomic organization (exons/introns) of a gene starting from a known gene name/accession number.

* '''Ensembl''' (known transcript)
# go to [http://www.ensembl.org/ Ensembl]
# click on a link to an organism of interest say Human (Homo sapiens)
# on the top  of the page, "Search for:" (select "Gene" from pull down menu)
# put the gene symbol (i.e. p53), Search
# select from the list of hits Ensembl gene (click on the red link)
# you are in "Ensembl Gene Report"
# right to "Sequence Markup" click on " View genomic sequence for this gene with exons highlighted"
# the sequence (including SNPs) is displayed

* '''Spidey''' (novel transcript)
# get cDNA sequence in FASTA format
# check it for repetitive sequences using RepeatMasker
# get genomic sequence containing the gene with 1-10kb flanks
# check the gene structure using dotter this is to detect cDNA/genomic DNA miss-assembly/gaps etc.
# go to http://www.ncbi.nlm.nih.gov/IEB/Research/Ostell/Spidey/spideyweb.cgi
# fill in the form selecting the proper splice site matrix (so far: vertebrates, Drosophila, C. elegans, or plants)
# you get a table with genomic coordinates/mRNA coordinates, exon length, % of identity, number ofmismatches, gaps, precence or abcence of splice donor and acceptor sites

Very good at finding exons in the same species (i.e human cDNA/human genomic sequence), more problematic with cross-species comparisons (i.e. human cDNA, bovine genomic sequence -> most of the exons OK, with few bad predictions). 

* '''exalin''' http://blast.wustl.edu/exalin/
comand line only. Slow but good at finding small exons. So far has matrices for human (default), mouse, rice and C.elegans.
<pre>
exalin your_genomic_sequence.fa  your_cDNA_sequence.fa
</pre>
* '''wise2''' [http://www.ebi.ac.uk/Wise2/]
Best for: comparing a protein sequence or hmm profile to a genomic/cDNA sequence from other organism
Restrictions: fasta headers longer than 15 characters are truncated

One needs to set up env variable for wise, i.e bash:
<pre>
export WISECONFIGDIR=/usr/biosoft/packages/wise/wise2.2.3-rc7//wisecfg/
</pre>

*** peptide vs genomic DNA:
<pre>
genewise your_peptide_sequence.single_fasta  your_genomic_sequence.single_fasta
</pre>
***HMM profile vs genomic DNA:
<pre>
genewise -hmmer your_hmm_profile.HMM your_genomic_sequence.single_fasta
</pre>
*** peptide vs cDNA/EST
<pre>
estwise your_peptide_sequence.single_fasta your_cDNA_sequence.single_fasta 
</pre>
*** HMM profile vs cDNA/EST
<pre>
estwise -hmmer  your_hmm_profile.HMM your_cDNA_sequence.single_fasta
</pre>

*'''[http://deepc2.psi.iastate.edu/cgi-bin/gs.cgi GeneSeqer] WWW server + stand alone program. Multiple species models.

*''' Pairagon ''' [http://mblab.wustl.edu/software/pairagon/]
standalone cDNA-to-genome alignment program

* [http://www.fml.mpg.de/raetsch/projects/palma PALMA] 2007 program stand alone claimed to outperform other algorithms. Parameter files for human, C.elegans, arabidopsis and drosophila (Nov 2007)

===One gene in multiple species===
* SVC 
Structured Visualization of Evolutionary Conserved Sequences. Provides colorful graphs depicting intron exon orientation with sizes.

# go to [http://svc.molgen.mpg.de/ SVC]
# put gene symbol (i.e. SYNGR3) or better Ensembl ID (i.e. ENSG00000127561) into 'Input ID'
# check species of interest in 'Search for orthologs in:'
# Click 'Submit' next to 'Input ID'

==Bacterial genomics==

* [http://theseed.uchicago.edu/FIG/index.cgi SEED] precomputed similarities, annotations using colocalization in operons. Very efective for annotation new genes. 

* [http://www.ebi.ac.uk/integr8/EBI-Integr8-HomePage.do Integr8] Ensembl-like interface to bacterial genomes

* [http://www.webact.org/WebACT/home WebACT] WebACT genome-2-genome comparisons

* [http://webclu.bio.wzw.tum.de/prompt/index.php PROMPT] Protein Mapping and Comparison Tool

==Promoter prediction==
Predicting promoters based on a single sequence is rather unaccurate. Transcription factor (TF) recognition sequences are fairly short, therefore any sequence 1kbp in lenght will contain hundreds of such motifs, only a handful of which will be a true positives. There are two major methods used for finding relevant sites
* phylogenetic fingerprinting (comparison of promoter regions between orthologues)
* comparison of promoter regions of genes with similar expression profiles in the same species

We can look for either known TFs recognition sequences (quality of which depends on database used) or perform motif finding de novo, looking simply for any streach of DNA shared between species/co-expressed genes. 


* before you start 
make sure that you got a real 5' end exon of the gene (a lot of longer genes in the databases may be missing the true 5'end). You verify quality of your gene of interest by:
** comparing with orthologues ( Ensembl or Homologene) If close orhologues are longer on the protein level, look for the presumptive missing exon(s). 
** looking at the genomic context (other genes flanking yours) 
** blasting cDNAs 5' end against ESTs database looking for any ESTs extending your gene 
** blasting masked genomic sequence, from 5' end exon to the start/end of the flanking gene
Interpretation of this one may be tricky, on the simplest level look for spliced matches with ORFs in the same orientation as your gene without clear stop codons.     

* find position of repeats
After checking the 5' end of the gene, always look for repetitive sequences using RepeatMasker. It does not mean that say Alu/LINE1 can not be a part of the promoter. Just that you may concentrate on parts which are a bit more specific for the given gene. 

===phylogenetic fingerprinting===
comparisons of promoter regions of orthologous genes in other species
#obtain these using Ensembl [http://www.ensembl.org/]
# use tools described below to find out shared TF binding sites/comon motifs

Caveats:
* comparing closely related genomic sequences (human and chimp) is not stringent enough
* comparing wildely different sequences (human and zebrafish) is too stringent for all but small set of genes (i.e. homeobox?)

===comparing promoters of coexpressed genes in the same species===
# use GEO (Gene Expression Omnibus) [http://www.ncbi.nih.gov/geo/] to find gene(s) with simmilar expression
# get promoter sequences using Ensembl [http://www.ensembl.org/]

'''Tools''':

===Shared TF binding sites===
Quality depends on the underlying database of TF binding sites

* [http://mordor.cgb.ki.se/cgi-bin/CONSITE/consite Consite] produces a set of transcription factors binding sites shared between two sequences
* [http://zpicture.dcode.org/ zpicture plus rVISTA] 
* Conreal (multiple alignment algorithms) [http://conreal.niob.knaw.nl/]
* Confac [http://morenolab.whitehead.emory.edu/cgi-bin/confac.pl?id=1]
* [http://mulan.dcode.org/ Mulan] multiple sequences
* Footprinter [http://wingless.cs.washington.edu/htbin-post/unrestricted/FootPrinterWeb/FootPrinterInput2.pl]
<!-- PromH [http://www.softberry.com/ £problems with access 060517 -->

===Shared DNA motifs only===
For more in depth coverage see: [[Wikiomics:Sequence_motifs]]

This method will work even for unknown TF binding sites, but it may skip some more degenerate motifs. 

* [http://meme.sdsc.edu/meme/meme.html MEME] capable of discovering longer motifs, very sensitive
* [http://www.cse.ucsc.edu/~kent/improbizer/improbizer.html  Improbizer] takes into account position of the motifs
* [http://atlas.med.harvard.edu/cgi-bin/alignace.pl AlignACE]  Gibbs sampling algorithm
* [http://ai.stanford.edu/~xsliu/MDscan/ MDscan ]

It is important to use few complementary programs as well as allow predictions of several motifs per program. There is threshold when it comes to number of sequences used, above which there is no improvement in sensitivity.

Comparison of several programs:
http://nar.oxfordjournals.org/cgi/content/full/33/15/4899#TBL1

<!-- * [http://www.bioinfo.rpi.edu/applications/bayesian/gibbs/gibbs.9.pl?data_type=DNA Gibbs sampler] #black  hole == no results -->

Others:

'''Promoter Databases'''
* [http://www.epd.isb-sib.ch/ Eukaryotic Promoter Database]
* [http://bioinformatics.med.ohio-state.edu/MPromDb/ Mammalian Promoter  Database MPromDb]
* [http://doop.abc.hu/ DoOP] eukaryotic promoter sequences
* [http://dbtss.hgc.jp/ DBTSS]  Database of Transcriptional Start Sites

'''Transcription Factiors Databases'''
* [http://jaspar.genereg.net JASPAR] (open transcription factors database)
* [http://www.cbil.upenn.edu/cgi-bin/tess/tess?RQ=SEA-FR-Query TESS](finding transcription factor binding sites within your sequence)

Be aware that simply listing all recognizable TF binding sites in a given stretch of DNA is almost of no value. Only 0.1% of the putative sites listed that way will be functional (Applied bioinformatics for the identification of regulatory elements WW Wasserman, A Sandelin - Nature Reviews Genetics, 2004)

===Reviews & tutorials===

* [http://sullivan.bu.edu/zlab/gene.shtml gene regulation tools site] very comprehensive

* Other tutorial (you need an access to Nature Immunology): [http://www.nature.com/cgi-taf/DynaPage.taf?file=/ni/journal/v5/n8/full/ni0804-768.html]
* PLOS Apr 2006 MacIsaac and Fraenkel article [http://compbiol.plosjournals.org/perlserv/?request=get-document&doi=10.1371/journal.pcbi.0020036]

* For an extensive table of known programs used for finding motifs see [http://www.stud.uni-potsdam.de/~haussler/master/list.php]

===Transcription start finding===
There are various clases of such programs starting with completa ab initio promoter finding using genomic sequence only, through combining it with ab initio gene fining, finally taking into account ESTs data and/or gene annotations. Most of the programs were evaluated on human or mammalian sequences and may not be suitable for other groups of organisms. Masking repeats using RepeatMasker improves performance of McPromoter(strongly) and FirstEF (mildly), whereas has no effect on Eponine and DragonGSF.

====Programs====
* N-SCAN (standalone only derivative of TwinSCAN) [http://mblab.wustl.edu/software/twinscan/] 
* Eponine (mammalian) [http://servlet.sanger.ac.uk:8080/eponine/]
* FirstEF (human) [http://rulai.cshl.edu/tools/FirstEF/]
* DragonGSF [http://sdmc.lit.org.sg/promoter/dragonGSF1_0/genestart.htm]

* McPromoter (fly, with older version for human) [http://tools.genome.duke.edu/generegulation/McPromoter/]

* PromH (promoters identification using orthologous genomic sequences) [http://www.softberry.com/berry.phtml?topic=promhw&group=programs&subgroup=promoter]

* ARTS (superior but not as Jan 2007 publicly available) [http://www2.fml.tuebingen.mpg.de/raetsch/projects/arts]

====Databases====
* [http://www.epd.isb-sib.ch/ EPD ] eukaryotic promoter database
* [http://doop.abc.hu/ DoOP ] eukaryotic promoter sequences

* [http://enhancer.lbl.gov/ VISTA Enhancer Browser] Human/vertebrates enhancers DB

====Reviews====
* Bajic et al. Performance assessment of promoter predictions on ENCODE regions in the EGASP experiment
Genome Biology August 2006 [http://genomebiology.com/2006/7/S1/S3]
* Bajic VB, Tan SL, Suzuki Y, Sugano S: Promoter prediction analysis on the whole human genome. Nat Biotechnol 2004, [http://www.nature.com/nbt/journal/v22/n11/full/nbt1032.html]

==SNPs (Single Nucleotide Polymorphism)==
* NCBI SNP database http://www.ncbi.nlm.nih.gov/SNP/ (strongly recommended)
* HGVbase http://hgvbase.cgb.ki.se/
* JSNP http://snp.ims.u-tokyo.ac.jp/ (some access problems)
* CSHL http://snp.cshl.org/

==Gene expression==

===Unigene===
# go to [http://www.ncbi.nlm.nih.gov/UniGene Unigene] (ESTs derived from the gene) 
# search Unigene with gene symbol (i.e SYNGR1) or Unigene cluster (i.e. SYNGR1)
# look at Expression and distribution of ESTs [http://www.ncbi.nlm.nih.gov/UniGene/clust.cgi?ORG=Hs&CID=216226&MAXEST=250]
# click on [http://www.ncbi.nlm.nih.gov/UniGene/ESTProfileViewer.cgi?uglist=Hs.216226 Expression profile ] to see electronic Northern

===Symatlas===
Wide array of tissues and genes.
* go to [http://symatlas.gnf.org Symatlas]
* put your gene symbol, i.e. SYNGR1
* click on a small icon left to human or mouse genes

===in situ hybridisation images===
* EMAGE
http://genex.hgu.mrc.ac.uk/Emage/database/emageIntro.html

* Gensat
Atlas of in situ hybridization of mouse brain
* http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gensat

* Genepaint
Atlas of in situ hybridization in mouse embryos. One can order hybridization of a gene of interest.
* http://www.genepaint.org/Frameset.html

===Digital Differential Display===
http://www.ncbi.nlm.nih.gov/UniGene/info_ddd.shtml

==Searching info about gene homologues in other databases==
* Mouse [http://www.informatics.jax.org/]
* C.elegans & co. [http://www.wormbase.org/]
* D. melanogaster [http://www.flybase.org/]
* S. cerevisiae SGD@stanford [http://genome-www.stanford.edu/Saccharomyces/]
* Proteome (multiple organisms) [http://www.proteome.com/databases/] ($$$) (databases of S. cerevisiae, S.pombe and WormPB [C.elegans+]

==Finding protein domains==
* [http://pfam.wustl.edu/ '''Pfam''']
* [http://smart.embl-heidelberg.de/ '''SMART''' ]
* [http://www.ncbi.nlm.nih.gov/Structure/lexington/lexington.cgi?cmd=rps '''DART''' ]
* [http://www.ebi.ac.uk/InterProScan/ '''InterPro''' ]

==Finding protein motifs==

===Phosphorylation===
* [http://www.hprd.org/PhosphoMotif_finder PhosphoMotif Finder]
* [http://scansite.mit.edu/motifscan_seq.phtml ScanSite]
* [http://www.cbs.dtu.dk/services/NetPhos/ NetPhos]

===GPI-Anchors===
* http://mendel.imp.ac.at/gpi/gpi_server.html

===Disulfide bond predictions===
* [http://disulfind.dsi.unifi.it/ DISULFIND] Newest

* [http://clavius.bc.edu/~clotelab/DiANNA/ DiANNA]
* [http://bioinfo.csie.ntu.edu.tw:5433/Disulfide/ PreCys]

==Sequence/structure retrieval==
* [http://www.ncbi.nlm.nih.gov/Entrez/ '''Entrez''' ]
* [http://www.ncbi.nlm.nih.gov/entrez/batchentrez.cgi?db=Nucleotide '''Batch Entrez''' ] (retrieving multiple DNA/protein entries at once):
* [http://srs.ebi.ac.uk/ '''SRS''' ] (complex queries )

==Protein localization and structure prediction==
* collection of links: [http://molbiol-tools.ca/Protein_secondary_structure.htm]
===localization===
* [http://psort.nibb.ac.jp/ Psort]
* [http://www.cbs.dtu.dk/services/TargetP/ TargetP]

* Evaluating eukaryotic secreted protein prediction [http://www.biomedcentral.com/1471-2105/6/256]

In evaluation (localization):
* MultiLoc [http://www-bs.informatik.uni-tuebingen.de/Services/MultiLoc/]
* LOCSVMPSI (localization, results by email) [http://bioinformatics.ustc.edu.cn/LOCSVMPSI/LOCSVMPSI.php]
* Localizome server [http://www.localizome.org Localizome.org]
* BaCelLo (max five fasta sequences) [http://gpcr.biocomp.unibo.it/bacello/pred.htm]
* ProteinProwler (plant/nonplant and four localizations only) [http://pprowler.imb.uq.edu.au/]

===transmembrane===
* top servers were evaluated by Cuthbertson et al.( Protein Engineering Design and Selection 2005) [http://peds.oxfordjournals.org/cgi/content/full/18/6/295  full text]:
** [http://split.pmfst.hr/split/4/ SPLIT4] (first choice)
** [http://www.cbs.dtu.dk/services/TMHMM-2.0/ TMHMM-2.0]
** [http://www.enzim.hu/hmmtop/html/submit.html HMMTOP 2]
** [http://bioweb.pasteur.fr/seqanal/interfaces/toppred.html TopPred] 

* Other servers:
** [http://bioinfo.si.hirosaki-u.ac.jp/~ConPred2/ ConPred2 ] Consensus TM prediction (metaserver)
** [http://mendel.imp.ac.at/sat/DAS/DAS.html DAS-TM]
** [http://phobius.cgb.ki.se/ Phobius] (includes signal peptide predictions)
** [http://www.pdc.kth.se/~hakanv/prodiv-tmhmm/ PRODIV-TMHMM] ref: [http://www.proteinscience.org/cgi/content/full/13/7/1908] (results send by email, prediction of TM topology)
<!-- older stuff 
** [http://gpcr.biocomp.unibo.it/biodec/ ENSEMBLE] (login: anynymous passwd: guest)
** [http://www.ch.embnet.org/software/TMPRED_form.html TMPRED]
** [http://minnou.cchmc.org/ MINNOU]
-->

see also links here: [http://www.biodirectory.com/directory/Protein_structure/Transmembrane_domains_89.html] and the Signal Peptide section

* Older reviews
** Chen State-of-the-art in membrane protein prediction. (Applied Bioinformatics 2002) [http://cubic.bioc.columbia.edu/papers/2002_rev_htm/paper.html]

===signal peptides===
* [http://www.cbs.dtu.dk/services/SignalP/ SignalP]
* [http://phobius.cgb.ki.se/index.html Phobius] (predicts signal peptide, TM and orientation)
* [http://phobius.cgb.ki.se/poly.html PolyPhobius] Homology supported signal peptide predictions
Accepts both single fasts seqence as well as aligned sequences in multiple fasta format (only the top sequence appears in the output).

* Phobius author PhD thesis: [http://scholar.google.com/url?sa=U&q=http://diss.kib.ki.se/2006/91-7140-719-7/thesis.pdf]

Other Programs:
* [http://gpcr.biocomp.unibo.it/cgi/predictors/spep/pred_spepcgi.cgi SPEPlip] Eukaryots/Prokaryots
* [http://www.cbs.dtu.dk/services/TatP/ TatP] Twin-arginine signal peptide cleavage sites in bacteria  Paper: [http://www.biomedcentral.com/1471-2105/6/167]
* [http://www.cbs.dtu.dk/services/LipoP/ LipoP] lipoprotein signal peptides in Gram-negative bacteria.  Paper: [http://protsci.highwire.org/cgi/content/abstract/12/8/1652]
<!--

ChloroP
http://www.cbs.dtu.dk/services/ChloroP/
MitoProt
http://www.mips.biochem.mpg.de/cgi-bin/proj/medgen/mito¢lter
predictNLS
http://maple.bioc.columbia.edu/predictNLS/
Predotar
http://www.inra.fr/Internet/Produits/Predotar/
PSORT
http://psort.nibb.ac.jp/

TargetP
http://www.cbs.dtu.dk/services/Targe
-->

=== protein structure (no-3D)===
* '''Expasy''' [http://www.expasy.ch/tools/]
* '''Coils''' (coiled coil prediction) [http://www.ch.embnet.org/software/COILS_form.html]
* '''Protscale''' (various prediction based on AA sequence) [http://us.expasy.org/cgi-bin/protscale.pl]
* '''Jpred''' (multiple secondary structure predictions run in parallel) [http://www.compbio.dundee.ac.uk/~www-jpred/]
<!--* '''DomainFinder''' (finds dynamical domains in proteins, local) http://dirac.cnrs-orleans.fr/DomainFinder/-->

==Protein structure databases/comparisons==
* [http://www.rcsb.org/pdb/ '''PDB''']
* [http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=Structure '''NCBI''']
* [http://www.biochem.ucl.ac.uk/bsm/cath_new/index.html '''CATH''']
* [http://www2.ebi.ac.uk/dali/ '''DALI''']
* [http://scop.berkeley.edu/ '''SCOP''']

==Metabolic networks ==
$$ needs more work$$
* [http://www.reactome.org/ Reactome]
* [http://www.genome.ad.jp/kegg/kegg2.html KEGG]
* [http://biocyc.org/download.shtml BioCyc]
* [http://nemo-cyclone.sourceforge.net Cyclone] - provides an open source Java API for easier access to BioCyc.
* [http://regulondb.ccg.unam.mx/ RegulonDB] E.coli K12 DB (operons/genes/regulatory elements)

==Protein-protein interactions==
Large scale methods such as yeast-two-hybrids (Y2H), immuno-co-precipiatation (IP) produce a large number of false positives. Only about 30-50% of interactions seem to be relevant. While newer improvements of these procedures tend to be more accurate, existing databases struggle to separate good quality data. 

* [http://www.ebi.ac.uk/intact/site/ IntAct@EBI] literature and user submission data.
* [http://www.bind.ca/Action?pg=0 BIND] (registration required)
* [http://dip.doe-mbi.ucla.edu/ DIP]
* [http://string.embl.de/ STRING]
* [http://www.bindingdb.org/bind/index.jsp BindingDB ]
* [http://www.thebiogrid.org BioGRID]
* [http://mips.gsf.de/proj/ppi/ Mammalian Protein-Protein Interaction Database @MIPS]
* [http://interweaver.i2r.a-star.edu.sg/exp/ Interweaver] protein sequence searches for orthologues and their interactions

'''Tools'''
* [http://bioinfow.dep.usal.es/apid/index.htm APID] Agile Protein Interaction DataAnalyzer 

'''Papers'''
* http://www.nature.com/nbt/journal/v22/n1/full/nbt924.html

==Where to find more information==

* [http://bioinformatics.ubc.ca/resources/links_directory/ Bioinformatics Links Directory @UBiC]
* Well maintained protein analysis links from U. of Nottingham [http://bioresearch.ac.uk/browse/mesh/D020539.html]
* [http://medicalcomputing.net/tools_dna1.php Approaches to Web Development for Bioinformatics]
* [http://medicalcomputing.net/resources/bioinformatics_resources.php Bioinformatics Resources] - Medical Computing .Net




Also there are relevant pages on Wikiomics web site:
* [[Wikiomics:Bioinformatics courses(external)]]

* [[Wikiomics:SRS_tutorials]]
* [[Wikiomics:Ensembl_tutorials]]
* [[Wikiomics:Staden]]
* [[Wikiomics:Cloning_in_silico]]


* [[Wikiomics:ChIP-chip]]
* [[Wikiomics:Protein mass spectrometry]]
* [[Wikiomics:Sequence motifs]]


* [[Wikiomics:NMR_Toolbox]]
* [[Wikiomics:Crystallography_toolbox]]
* [[Wikiomics:Docking_toolbox]]

== Credits ==

{{credits}}

* [[User:Darked|Darek Kedra]] wrote this tutorial
<!-- other contributors, put yourself here -->