===Sequence databases===
* [http://www.bioinfo.org.cn/noncode/ NONCODE] a database of all kinds of noncoding RNAs (except tRNAs and rRNAs). 
* [http://biobases.ibch.poznan.pl/ncRNA/ ncRNAdb] The noncoding RNA (ncRNA) database is intended to provide information on the sequences and functions of transcripts which do not code for proteins, but perform regulatory roles in the cell. The sequences included in the database have been at least partially characterized in terms of their function or expression.
* [http://www-snorna.biotoul.fr snoRNAbase] a comprehensive database of human H/ACA and C/D box snoRNAs.
* [http://lowelab.ucsc.edu/snoRNAdb/ snoRNAdb] Methylation Guide snoRNA Database: Saccharomyces cerevisiae, Archaeal and Arabidopsis thaliana.
* [http://people.biochem.umass.edu/sfournier/fournierlab/snornadb/main.php snoRNAdb] Small Nucleolar RNAs (snoRNAs) from the Yeast Saccharomyces cerevisiae: A comprehensive database of S. cerevisiae H/ACA and C/D box snoRNAs. VERSION 2.0.

===Alignment and consensus secondary structure databases===
* [http://www.sanger.ac.uk/Software/Rfam/ Rfam] Rfam is a large collection of multiple sequence alignments and covariance models covering many common non-coding RNA families.
* [http://rdp.cme.msu.edu/index.jsp Ribosomal Database Project-II] The Ribosomal Database Project (RDP) provides ribosome related data and services to the scientific community, including online data analysis and aligned and annotated 16S rRNA sequences.
* [http://www.psb.ugent.be/rRNA/ European rRNA database] Compiles all complete or nearly complete SSU (small subunit) and LSU (large subunit) ribosomal RNA sequences. Sequences are provided in aligned format. The alignment takes into account the secondary structure information derived by comparative sequence analysis of thousands of sequences. Additional information such as literature references, taxonomy, secondary structure modles and nucleotide variability maps, is also available.
* [http://www.rna.ccbb.utexas.edu/ CRW Site] Alignments, structure models and phylogenetic analyses of 5S, 16S and 23S rRNA, Group I and II introns and tRNA.
* [http://psyche.uthct.edu/dbs/SRPDB/SRPDB.html SRPDB] SRPDB (Signal Recognition Particle Database) provides aligned, annotated and phylogenetically ordered sequences related to structure and function of SRP.
* [http://www.indiana.edu/~tmrna/ The tmRNA Website] Research on the mechanism, structure and biology of tmRNA is served by the tmRNA website, a collection of sequences, alignments, secondary structures and other information.
* [http://psyche.uthct.edu/dbs/tmRDB/tmRDB.html tmRDB] tmRDB (tmRNA Database) provides aligned, annotated and phylogenetically ordered sequences related to structure and function of tmRNA.
* [http://jwbrown.mbio.ncsu.edu/RNaseP/home.html The RNase P Database] A compilation of RNase P sequences, sequence alignments, secondary structures, three-dimensional models, and accessory information.
* [http://rna.tbi.univie.ac.at/cgi-bin/virusdb.cgi Viral RNA Structure Database] Viral structures from TBI, Vienna.

===Tertiary structure databases===
* [http://ndbserver.rutgers.edu/ NDB] The nucleic acid database: a repository of three-dimensional structural information about nucleic acids.
* [http://scor.lbl.gov/index.html SCOR] a database designed to provide a comprehensive perspective and understanding of RNA motif structure, function, tertiary interactions and their relationships.

[[Category:RNA]]
[[Category:Databases]]
[[Category:Sequence]]
[[Category:Structure]]


{{Paul P. Gardner}}