'''Staden''' - program package for DNA sequence assembly. It handles all steps from sequence pre-processing, through assembly, editing and finishing. Suitable for both small and very large sequence projects. Along [[Wikiomics:Consed]] used in many leading DNA sequencing centers. Excellent contig editor in gap4.

Executables for Linux, Windows, MacOS X and Solaris.

Current version: 1.6.0 with 1.7.0 in the works.

==Program page==
http://staden.sourceforge.net/

==Sylabus==

===Start up===
* connecting to server using ssh and Exceed/Hummingbird
* setting staden environmental variables
before starting to work with Staden on Unix one has to set up environment

**tcsh
<pre>
setenv STADENROOT /path/to/your/staden/directory
source $STADENROOT/staden.login

/usr/biosoft/packages/staden/staden/current/


</pre>
**bash (Note '.' [dot] in the second line)
<pre>
export STADENROOT=/path/to/your/staden/directory
. $STADENROOT/staden.profile
</pre>
* basics of moving between directories, listing files etc. on Unix
<pre>
whoami
pwd
ls
ls *.exp
ls -l
ls -ltr
cd 
cd directory_name
mkdir new_directory
</pre>

* viewing trace files with trev
<pre>
trev file_name.scf &
trev file_name.exp &
trev project1_file*.exp
</pre>

===Sequence preprocessing===
* data quality
* base calling
* vector(s) masking
* annotation of repeats
* starting pregap4
* setting up pregap4 run
* checking the results

===gap4===

* starting it
* menu options
* opening existing database/starting a new one
* contig editor
**trace display
**translation
**diff
**creating annotations
**searching/annotation searchin/offset searching
**breaking miss-assembled contigs
* Find internal joins function
* finding restriction sites
* saving copies of database
* saving consensus as Fasta/exp files

== Credits ==

{{credits}}

* [[User:Darked|Darek Kedra]] wrote this tutorial
<!-- other contributors, put yourself here -->

{{stub}}