After determining a list of genes involved in a given biological process the next step is to map these genes to known pathways/Gene Ontology terms and determine i.e. which pathways are overrepresented in a given set of genes. 

'''Recent review''' (Jan 2008 !): 
Nam, Dougu, and Seon-Young Kim. “Gene-set approach for expression pattern analysis.” Brief Bioinform (17, 2008): bbn001. [http://bib.oxfordjournals.org/cgi/content/full/bbn001v1 HTML]
See table 1 for complete list of tools.

=Recommended= 
* [http://biit.cs.ut.ee/gprofiler/ g:Profiler] a web-based toolset for functional profiling of gene lists from large-scale experiments. Easy to use web server

* [http://kobas.cbi.pku.edu.cn. KOBAS] server used for i.e. elucidating pathways in addicion
Li, Chuan-Yun, Xizeng Mao, and Liping Wei. “Genes and (Common) Pathways Underlying Drug Addiction.” PLoS Computational Biology 4, no. 1 (1, 2008) [http://compbiol.plosjournals.org/perlserv/?request=get-document&doi=10.1371%2Fjournal.pcbi.0040002 HTML]

* GSEA http://www.broad.mit.edu/gsea/software/software_index.html
objections (Damian D, Gorfine M. Statistical concerns about the GSEA procedure): http://www.nature.com/ng/journal/v36/n7/full/ng0704-663a.html
and reply: http://www.nature.com/ng/journal/v36/n7/full/ng0704-663b.html

=Other tools to check=
* [http://gepat.sourceforge.net/ GEPAT] Genome Expression Pathway Analysis Tool. Performs standard microarray analyzes plus "Ensembl database and provides information about gene names, chromosomal location, GO categories and enzymatic activity for each probe on the chip.". Complex installation of java jars/MySQL etc.  
* [http://www.bioinformatics.ubc.ca/ermineJ/ ErmineJ] Java stand-alone program "designed to be used by biologists with little or no informatics background" + command line for expert
* [http://www.biomedcentral.com/1471-2105/6/144#B12 PAGE]  Parametric Analysis of Gene Set Enrichment
* [http://cbio.mskcc.org/software/cpath/ CPath] database and software suite for storing, visualizing, and analyzing biological pathways [http://cbio.mskcc.org/cpath/ demo page]
* EASE (old?) http://www.pubmedcentral.gov/articlerender.fcgi?tool=pubmed&pubmedid=14519205

* nonparametric multivariate analysis Nettleton et al. [http://bioinformatics.oxfordjournals.org/cgi/content/full/24/2/192 HTML]. R code availebla from author.

=Pathway/graph visualisation=
* [Cytoscape http://www.cytoscape.org/ Cytoscape] leader in the field
* [http://ondex.sourceforge.net/ ONDEX] [http://bioinformatics.oxfordjournals.org/cgi/content/full/22/11/1383 HTML] "enables data from diverse biological data sets to be linked, integrated and visualised through graph analysis techniques"

=Pathway Databases=

* [http://www.genome.ad.jp/kegg/kegg2.html KEGG] first choice for scope
* [http://www.reactome.org/ Reactome] human + model organisms pathways. Expert annotations from literature. 
* [http://pid.nci.nih.gov/ PID] Pathway Interaction Database @NIH 

* [http://biocyc.org/download.shtml BioCyc]
* [http://nemo-cyclone.sourceforge.net Cyclone] - provides an open source Java API for easier access to BioCyc.
* [http://regulondb.ccg.unam.mx/ RegulonDB] E.coli K12 DB (operons/genes/regulatory elements)

=Pathway specific languages=

* [http://www.biopax.org/ BioPAX] Biological Pathway Exchange Language

=Stuff 2 check=

*GenMapp, Pathway Processor  GeneXpress see:
 Cavalieri D, De Filippo C. Bioinformatic methods for integrating whole-genome expression results into cellular networks. Drug Discov Today. 2005;10:727–734. doi: 10.1016/S1359-6446(05)03433-1
* KaPPA-View
* VANTED
* [http://omicspace.riken.jp/osml/GSCope3] [http://www.plantmethods.com/content/2/1/5 HTML] OSML Editor

==Pathway analysis==
* [http://www.patika.org/ PATIKA]  and [http://www.cs.bilkent.edu.tr/~patikaweb/ PATIKAweb]

=Biography=
#Aittokallio, Tero, and Benno Schwikowski. “Graph-based methods for analysing networks in cell biology.” Brief Bioinform 7, no. 3 (September 1, 2006): 243-255.
#Li, Chuan-Yun, Xizeng Mao, and Liping Wei. “Genes and (Common) Pathways Underlying Drug Addiction.” PLoS Computational Biology 4, no. 1 (1, 2008): e2 EP -.
#Nam, Dougu, and Seon-Young Kim. “Gene-set approach for expression pattern analysis.” Brief Bioinform (17, 2008): bbn001.
#Resources for integrative systems biology: from data through databases to networks and dynamic system models -- Ng et al. 7 (4): 318 -- Briefings in Bioinformatics.” http://bib.oxfordjournals.org/cgi/content/full/7/4/318.
#Stromback, Lena, Vaida Jakoniene, He Tan, and Patrick Lambrix. “Representing, storing and accessing molecular interaction data: a review of models and tools.” Brief Bioinform 7, no. 4 (December 1, 2006): 331-338.
#“Tools for visually exploring biological networks -- Suderman and Hallett 23 (20): 2651 -- Bioinformatics.” http://bioinformatics.oxfordjournals.org/cgi/content/full/23/20/2651.



{{stub}}