===Conserved structure search methods===
* [http://www.tbi.univie.ac.at/papers/SUPPLEMENTS/Alifoldz/ Alifoldz] Assessing a multiple sequence alignment for the existence of an unusual stable and conserved RNA secondary structure.
* [http://www.tigem.it/Research/Personal%20Web%20Page_files/dibernardo/Website/Group_Frame_blue.htm ddbRNA] detection of conserved secondary structures in multiple alignments
* [http://www.cbse.ucsc.edu/~jsp/EvoFold/ EvoFold] A comparative method for identifying functional RNA structures in multiple-sequence alignments. It is based on a probabilistic model-construction called a phylo-SCFG and exploits the characteristic differences of the substitution process in stem-pairing and unpaired regions to make its predictions. Each prediction consists of a specific secondary structure and a folding potential score.
* [http://theory.csail.mit.edu/MSARi/ MSARi] searches orthologous nucleotide sequences for statistically signficant variations conserving a candidate secondary structure. It is described in "MSARi: Multiple sequence alignments for statistical detection of RNA secondary structure", by Coventry, Kleitman and Berger.
* [http://selab.wustl.edu/cgi-bin/selab.pl?mode=software#qrna QRNA] This is the code from Elena Rivas that accompanies a submitted manuscript "Noncoding RNA gene detection using camparative sequence analysis". QRNA uses comparative genome sequence analysis to detect conserved RNA secondary structures, including both ncRNA genes and cis-regulatory RNA structures.
* [ftp://www.pesolelab.it/pub/ RNAProfile] An algorithm for finding conserved secondary structure motifs in unaligned RNA sequences. Input a set of unaligned RNA sequences expected to share a common motif, RNAProfile outputs the regions that are most conserved throughout the sequences, according to a similarity measure that takes into account both the sequence of the regions and the secondary structure they can form according to base-pairing and thermodynamic rules.
* [http://www.tbi.univie.ac.at/~wash/RNAz/ RNAz] is program for predicting structurally conserved and thermodynamic stable RNA secondary structures in multiple sequence alignments. It can be used in genome wide screens to detect functional RNA structures, as found in noncoding RNAs and cis-acting regulatory elements of mRNAs.

===Miscellaneous approaches===
* [http://selab.wustl.edu/publications/Klein02/Klein02-hmmgcc.tar.gz HMMGCC] Noncoding RNA genes identified in AT-rich hyperthermophiles
* [http://www.genetics.wustl.edu/eddy/software/#ncrnascan  NCRNASCAN] This is the code from Elena Rivas that goes with the paper "Secondary structure alone is generally not statistically significant for the detection of noncoding RNAs" by Elena Rivas and Sean Eddy. As the title indicates, the genefinder doesn't work, because real RNAs don't generally have any more secondary structure content than random sequence, contrary to what we expected.
* [http://bioinformatics.psb.ugent.be/supplementary_data/erbon/nov2003/ RANDfold] Assess the stability of non-coding RNA secondary structures such as micro RNAs, transfer RNAs and ribosomal RNAs compared to random sequences.
* [http://rnaanalyzer.bioapps.biozentrum.uni-wuerzburg.de/ RNAAnalyzer] you can search your RNA/DNA for several known RNA structures and retrieve some general information of your sequence.
* [http://rnagene.lbl.gov/ RNAGenie] A computational approach to identify genes for functional RNAs in genomic sequences
* [http://www.tbi.univie.ac.at/~ivo/RNA/ RNALfold] computes locally stable RNA secondary structure with a maximal base pair span. For a sequence of length n and a base pair span of L the algorithm uses only O(n+L*L) memory and O(n*L*L) CPU time. Thus it is practical to  "scan"  very  large  genomes  for short RNA structures.


[[Category:RNA]]
[[Category:Sequence]]
[[Category:Structure]]


{{Paul P. Gardner}}