Given a protein structure, which are the potentially interesting sites? Approaches which are based only on sequence patterns or backbone architecture are often insufficient to find similarities between sites of similar biochemical function.

The set of methods which are shown here use the 3D arrangement of the atoms of proteins to find putative functional sites, such as ligand binding sites or catalytic sites.

== Search by comparison against annotated sites ==

Comparing 3D structures locally at the atomic level is not a simple problem, and there is no standard method in this field. However, many of these recent techniques are available from web servers, which makes them relatively easy to use.

An advantage of comparing a query protein structure against 3D sites of known biological activity is that both sites can be compared and the similarity can be further investigated either visually or using other tools.

=== Methods and tools ===

[http://pdbfun.uniroma2.it/ PdbFun] <cite>pdbfun</cite> is a web server for the identification of local structural similarities between annotated residues in proteins, gives fast access to the whole PDB organized as a database of annotated residues, helps selecting any residue subset by combining the available features, compares query and target selections with a fast and sequence-independent 3D comparison algorithm representing each amino acid by one point located at its centroid.

[http://wwwmgs.bionet.nsc.ru/mgs/gnw/pdbsitescan/ PDBSiteScan] <cite>pdbsitescan</cite> will scan a protein structure against its PDBSite <cite>pdbsite</cite> database. Each amino acid is represented by its 3 backbone atoms (N, C-alpha, C).

[http://www.russell.embl.de/pints/ PINTS] <cite>pints_method pints_assessment</cite> defines types of atoms for certain atoms of the lateral chains of amino acids. 2 atoms of the same type such as an oxygen of a carboxyl group (in Asp or Glu) can be considered as equivalent. The search is based on interatomic distances and the scoring is based on [[Wikiomics:RMSD]] values.

[http://www.biochem.ucl.ac.uk/bsm/PROCAT/PROCAT.html PROCAT] <cite>procat</cite> and now [http://www.ebi.ac.uk/thornton-srv/databases/CSA/ Catalytic Site Atlas] <cite>csa1 csa2 csa3</cite> use the TESS <cite>tess</cite> and Jess <cite>jess</cite> methods for searching a database of 3D templates of catalytic sites.

[http://pvsoar.bioengr.uic.edu/ pvSOAR] <cite>pvsoar_method pvsoar_server</cite> uses centroids of amino acids forming pockets and the pseudosequence they form: if a pocket is made of amino acids Ala45, Tyr12, Ser124 and His32 then the corresponding sequence would be Tyr-Ala-His-Ser. The default comparison procedure uses an alignment between the sequences associated with 2 pockets. This constraint can be removed if only 2 pockets are being compared. 

[http://bioinfo3d.cs.tau.ac.il/SiteEngine/ SiteEngine] <cite>siteengine</cite> uses surface exposed functional groups that describe the physico-chemical properties of amino acids. It is possible to compare a protein structure against a given site on the web server. The program is also available for download.

[http://portray.bmc.uu.se/cgi-bin/spasm/scripts/spasm.pl SPASM/RIGOR] <cite>spasm_rigor</cite> was the first webserver to propose sequence- and fold-independent search in 3D structures of proteins. It represents each residue by it's C-alpha or the centroid of the lateral chain.

[[Image:Micro-poster-sumo-ismb05.png|right|frame|Poster showing the main concepts of SuMo. [http://martin.jambon.free.fr/poster-ismb05.png Enlarge] ]]
[http://sumo-pbil.ibcp.fr/ SuMo] <cite>sumo2003 sumo2005 sumo_method</cite> uses chemical groups with their own geometry and symmetry plus a complementary local shape comparison technique. It does not require a low [[Wikiomics:RMSD]] between 2 sites to consider them as similar although local pairwise matching is required. Given a protein structure, it will scan the PDB for similar ligand binding sites and return a list of sites, sorted by decreasing size. Clicking on each individual result gives a parallel view of the matched sites.

== Prediction of functional sites from geometrical or physico-chemical properties ==

These tools do not try to match 3D sites between a query and sites of biological importance. Based on the geometry or the chemistry of the protein sites, they are associated with a given function.

* [http://www.weizmann.ac.il/SARIG/ SARIG] <cite>sarig</cite> predicts functional sites using residue interaction graphs (contact maps)
* [http://feature.stanford.edu/webfeature/ WebFEATURE] <cite>feature webfeature</cite> scans a protein structure for local environments of a given type. An RNA version exists too, naFEATURE <cite>nafeature</cite>.
* THEMATICS <cite>thematics2005a thematics2005b thematics2005c</cite> catalytic sites are predicted from deviations in theoretical titration curves of proteins

== Prediction using phylogenetic information ==

Combined with projections onto 3D structures, the degree of conservation of aligned residues within a family of proteins can indicate amino acids which are functionally important.

* [http://www-cryst.bioc.cam.ac.uk/~jiye/evoltrace/evoltrace.html Evolutionary Trace (Cambridge)] <cite>et1996 et2000</cite>
* [http://mammoth.bcm.tmc.edu/traceview Evolutionary Trace Viewer] and [http://mammoth.bcm.tmc.edu/report_maker Evolutionary Trace report_maker] (Baylor) <cite>et1996 et2004</cite>
* [http://consurf.tau.ac.il/ ConSurf] <cite>consurf</cite>

== See also ==

* [[Wikiomics:Protein function prediction]]

== References ==

<biblio>
#sumo2005 pmid=16141250 // application note about the SuMo web server
#sumo2003 pmid=12833538 // describes the basic method, which has been considerably refined since. Read <cite>sumo_method</cite> for a good understanding of the current method and the concepts on which it relies.
#sumo_method Jambon M. ''A bioinformatic system for searching functional similarities in 3D structures of proteins''. [http://martin.jambon.free.fr/phd.html PhD thesis], 2003.
#pints_method pmid=9642096
#pints_assessment pmid=12595245 // read <cite>pints_method</cite> first
#pdbsitescan pmid=15215447
#pdbsite pmid=15608173
#siteengine pmid=15147845
#pvsoar_method pmid=12948498
#pvsoar_server pmid=15215448
#procat pmid=8762132
#tess pmid=9385633 // successor of PROCAT <cite>procat</cite>
#jess pmid=12967960 // successor of TESS <cite>tess</cite>
#csa1 pmid=14681376
#csa2 pmid=12421562
#csa3 pmid=15755451
#spasm_rigor pmid=9917419
#pdbfun pmid=15980442
#sarig pmid=15544817
#et1996 pmid=8609628
#et2000 pmid=11239083
#et2004 pmid=15037084
#consurf pmid=12499312
#thematics2005a pmid=15961465
#thematics2005b pmid=15751116
#thematics2005c pmid=15739204
#feature pmid=9697207
#webfeature pmid=12824318
#nafeature pmid=12888505
#polacco pmid=16410325 // uses the same technique as SPASM <cite>spasm_rigor</cite>
#schmitt2002 pmid=12381328 // one of the most advanced technique with SuMo <cite>sumo2003 sumo2005 sumo_method</cite>, but not available online (?). (more details needed)
</biblio>

== Credits ==
{{credits}}
* [[User:Martin Jambon | Martin Jambon]]: list of tools and papers, initially imported from a [http://martin.jambon.free.fr/search-protein-3D-sites.html traditional web page] and later extended

[[Category:Protein]]
[[Category:Structure]]