== Definition ==

RMSD means ''Root Mean Square Deviation''. It is often used in 3D geometry of molecules to compare two conformations of a given set of points, typically atoms. In other words, given a list of paired points, it gives a measure of the distance between these points.

Normally a rigid superposition which minimizes the RMSD is performed, and this minimum is returned. Given two sets of <math>n</math> points <math>\mathbf{v}</math> and
<math>\mathbf{w}</math>, the RMSD is defined as follows:

{|
| <math>\mathrm{RMSD}(\mathbf{v}, \mathbf{w})</math>
| <math>= \sqrt{\frac{1}{n}\sum_{i=1}^{n} \|v_i - w_i\|^2}</math>
|-
|
| <math>= \sqrt{\frac{1}{n}\sum_{i=1}^{n} 
      ({v_i}_x - {w_i}_x)^2 + ({v_i}_y - {w_i}_y)^2 + ({v_i}_z - {w_i}_z)^2}</math>
|}

An RMSD value is expressed in length units. The most commonly used unit in structural biology is the Ångström (Å) which is equal to 10<sup>–10</sup>m.

== Usage ==

RMSD values are commonly used to measure the structural similarity between proteins structures. If the proteins are different, an alignment between amino acids is required. It can be a sequence alignment or a structural alignment, both of which can be performed in a variety of ways.

Often, the atoms for which the RMSD is given are restricted to the alignable C<sub><math>\alpha</math></sub> of a pair of proteins. This can be assumed to be the default. However, it is possible to compute the RMSD for all atoms of identical molecules. For a protein, the RMSD of all atoms is usually larger than the RMSD of C<sub><math>\alpha</math></sub> since the main chain is more constrained than the lateral chains of the amino acids.

== Tools ==

The following tool can be used to perform a [[Wikipedia:sequence alignment|sequence alignment]] of protein structures and then compute the corresponding RMSD. This is not [[Wikipedia:protein structural alignment | structural alignment]], so it should be used with protein structures of very similar sequences:
* [http://wishart.biology.ualberta.ca/SuperPose/ SuperPose] <cite>superpose</cite> provides the RMSD computed on alpha carbons, backbone atoms or heavy atoms, but doesn't give the choice of which residues should be superposed.

There are also a good number of structural alignment tools, which should choose a good alignment even if the sequence conservation is very low. For aligning 2 given structures, the following online tools are available:
* [http://www.ebi.ac.uk/DaliLite/ DaliLite] <cite>dalilite</cite> uses the classic Dali heuristics.
* [http://cl.sdsc.edu/ce/ce_align.html CE] <cite>ce</cite> is another good structure alignment method (requires Java).

Note that Dali, CE and several other algorithms of structural alignment can be used to search a database for structural homologs.

== References ==

<biblio>
#superpose pmid=15215457
#dalilite pmid=10980157
#ce pmid=9796821
</biblio>

== Credits ==
{{credits}}
* this page was initiated (definition, usage, a few links) by [[User:Martin Jambon|Martin Jambon]]

[[Category:Structure]]
[[Category:Definitions]]
[[Category:Protein]]