<!DOCTYPE doctype PUBLIC "-//w3c//dtd html 4.0 transitional//en">
<html>
<head>
        
  <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
        
  <meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
        
  <meta name="GENERATOR" content="Mozilla/4.77 [en] (X11; U; Linux 2.4.3-20mdk i686) [Netscape]">
  <title>10-Gene_Discovery.html</title>
</head>
  <body>
   
<p>  </p>
    
<center>  
<h1> CLCG Tutorial #10</h1>
 </center>
    
<center>  
<h2> Gene&nbsp; Discovery</h2>
 </center>
   <br>
 <b>Introduction</b>  
<p>The goal of all the Gene Discovery projects is to identify as many genes
 as possible (or practical) as efficiently and cost effectively as possible.
 All of our Gene Discovery efforts focus on the creation of cDNA libraries.
 A cDNA library is created by isolating the mRNA molecules from a tissue
or cell sample,&nbsp; and creating a DNA copy of those mRNA molecules. This 
is done because the DNA copy can then be manipulated in an easier fashion 
(mRNA is single-stranded, DNA is double-stranded).&nbsp; <br>
 The cDNA copy is then cloned into a plasmid vector and transformed into
bacteria. This allows the original single-copy of the cDNA to be amplified
greatly. After sufficient growth within the bacteria, the cDNAs are arrayed
into 96-well (or 384-well) plates for sequencing. </p>
 
<p><br>
 The processes of normalization and subtraction are used in creating new
libraries with imporoved discovery rates. Specifically, cDNA library normalization 
reduces the prevalence of highly prevalent clones with the library sss a self-hybridization.
The subtraction process differs in that it reduces the prevalence of a specific
set of clones. </p>
 
<p>We use the process of clustering to assess the novelty of sequences obtained 
from a library. This infomration is used in determining when a library should 
no longer be sequenced. Clustering is also used to define a non-redundant 
set. Such a set is useful in expression studies (e.g., microarray) and in 
efficient sequencing of 5' ESTs. </p>
 
<p> </p>
 
<hr width="100%" size="3" noshade=""> &nbsp; 
<p><b>cDNA libraries</b> </p>
 
<p>The creation of a cDNA library consists of four primary steps: (1) isolation 
of RNA, (2) creation of the cDNAs, (3) cloning, and (4) electroporation into 
bacteria. </p>
 
<p>The first step is isolation and purification of RNA. This is done either 
from a tissue sample, or from a cell culture. The first step is in removing 
random detritus from the mix. The garbage in this instance includes cellular 
components, and proteins that may or may not be associated with the RNA molecules. 
Next, the DNA in the mixture is removed using DNase, which degrades any DNA 
remaining while leaving RNA alone. Finally, those RNA molecules with a polyA 
tail (from polyadenylation - see the Molecular Biology tutorial) are isolated. 
This is referred to as polyA+ selection. Only molecules with a polyA tail 
are desired for two reasons. First, these should represent a polyadenylation 
tail at the 3' end of a messenger RNA. Knowing we are at the 3' end is beneficial 
during later analysis when we are estimating a non-redundant set of transcripts. 
The second reason that polyA select is performed is to remove the various 
rRNA sequences from the mix. These sequences are in very high copy number 
in every cell, and as such can be thought of as a contaminant when attempting 
to discover novel genes.</p>
 
<p>To create cDNA molecules from mRNA transcripts, we have to break the Central 
Dogma. Rather than going from DNA to RNA we are going in the reverse direction. 
To do this, the enzyme reverse transcriptase is used. This enzyme takes a 
single-stranded RNA molecule, and creates a double-stranded RNA-DNA heteroduplex 
where the DNA is the reverse complement sequence to that in the RNA. (A heteroduplex 
is a double-helix formed where one strand is RNA and the other is DNA.) This 
reaction requires a primer upon which the remaining nucleotides will be added. 
A polyT primer is used, that anneals (hybridizes) to the polyA regions of 
the mRNA molecules. We're half-way there. Next, and adaptor sequence (with 
a restriction site) is ligated on to both ends of the heteroduplex. Now, the
RNA is degraded with RNase leaving behind the DNA copies of the messenger 
RNAs. Because the adaptor sequence was DNA is is not degraded, and it is used
as the primer to fill in the second strand of the cDNA. At this point, we
know have a double-stranded cDNA clone with cloning sites at both ends.</p>
 
<p>The third step in the creation of a cDNA library is to clone the cDNA molecules
in a vector. But this begs the question - What is a vector? A vector is a
circular piece of DNA used to stably propagate cDNA clones within a host
(typically bacteria). Without a vector to stay in, the cDNA clone would quickly
be degraded within the host. The important features of a vector are the polylinker,
the selectable markers, and an origin of replication. The polylinker provides
a multitude of restriction sites in this the cDNA can be cloned. The origin
of replication is important in propagating the cDNA from generation to generation
in the host. When bacteria divide, the create an extra copy of their genome
(a large circle of DNA - about 3-5 million bases) and one copy goes to each
of the daughter cells. So two copies of the vector + cDNA are needed to be
distributed to the daughter cells as well. Finally, one or more selectable
markers are used to guarantee that the host retains the vector. This is done
by growing the host under conditions such that if they do not have a copy
of the vector (with a cDNA clone) within them they can not grow. &nbsp;[Contrast
pT3T7 and pYX]</p>
 
<p>The final step in creation of a cDNA library is the electroporation of 
the vector into the hosts. Electroporation is the process in which cells
(in our case bacteria) are induced to "take up" DNA into themselves with
an electric current. </p>
 
<p><br>
 <b>Summary of EST/cDNA features</b> </p>

<p>
There are several common features in EST sequences. These are presented
in Figure 1 below. The most fundamental feature is the restriction site, often
called the cloning site. This site is in red in the example below. 
In this example, the restriction site is the NotI site (GCGGCCGC). This is
the recognition site for the NotI restriction enzyme. The sequence
upstream of the restriction site is part of the vector sequecne (in purple).
These two features should be present in <i>every</i> EST sequence.

<p>
The remaining features are present in most 3' ESTs. These ESTs are derived 
from polyT primed cDNA libraries. These features are the polyadenylation tail,
polyadenylation signal, and tissue tag. The polyadenylation is in blue, and
is the reverse complement of the polyadenylation tail from the original
mRNA transcript. The polyadenylation signal is in yellow, and should appear
11 - 30 bp downstream from the polyadenylation tail in 3' ESTs. The final
3' specific feature is the tissue tag. These are (mostly) a UI-specific
feature. It is used to identify the tissue of origin from a pooled-tissue
library.



<center>
<table cellspacing=0 cellpadding=0 border=0>
<tr>
  <td> <img src="pics/features-3prime.png">
<tr>
  <td>
<center>
 <b>Figure 1.</b> Figure of common 3' EST features.
</center>
</table>
</center>

</p>
 
<blockquote> &nbsp; <br>
  </blockquote>
 <b>EST sequencing</b> (add pic of 96 and 384 well plates)    
  <blockquote>1. 3' vs. 5' -- typically cloned directionally -&gt; can read
 the cDNA from either direction <br>
 2. How sequencing works <br>
 3. Feature identification <br>
 ESTprep      
    <ul>
  <li> requires FASTA sequence and quality files</li>
   <li> requires restrcition site + vector, 3' polyA tail, signal, lib tag</li>
   <li> trims based on quality</li>
   <li> requires min # bases</li>
       
    </ul>
  RepeatMasker      
    <ul>
  <li> req. library of known repeats&nbsp; <br>
      </li>
       
    </ul>
 &nbsp;</blockquote>
    <b>  Normalization of cDNA libraries</b><br>
 <br>
The process of cDNA library normalization is used to reduce the prevalence
of the most highly represented clones. The goal is to bring the relative
prevalence of the highest and lewest represented clones from 1:10,000 to
1:100 (or better). This is done in a "blind" manner for all clones within
the library; a complete description of this method can be found in Bonaldo
et al. 1996. In brief, the library is hybridized to itself for a moderate
period of time. Because of hybridization kinetics, the most prevalent sequences
will find a complementary strand becoming double-stranded. The double-stranded
portion is removed, and the single-stranded portions are isolated. These
single-stranded cDNAs are then completed (made into double-stranded cDNAs),
and electroporated into the host.<b><br>
    <br>
Subtraction of cDNA libraries</b>     
    <p>The process of cDNA library subtraction seeks to completely remove
a targeted set of clones from a cDNA library. This differs from a normalized
library, in that this method is a <i>directed</i> attempt to remove <i>specific</i>
 clones. Realistically, it greatly reduces the prevalence of the targeted
clones, but may not remove them from the library entirely. This method is
also described in Bonaldo et al. 1996. This method is very similar to that
for creating a normalized cDNA library, but in this instance a specific set
of cDNA clones are used as the driver (those clones to be removed) rather
than the entire library. In addition, the hybridization is carried out for
a much longer time, in an attempt to remove the rare clones that may be represented
in the driver. Again, the bound portion (i.e., the double-stranded segments)
are what is removed. The remainder is electroporated in the host.<br>
    <b><br>
Clustering</b></p>
    <p>Generically, clustering is the process of grouping elements together
based upon some criteria. With respect to ESTs, clustering is done based
on seqeunce similarity. The goal of clustering is to either (1) assess the
novelty of a set of ESTs, or (2) generate a non-redundant set of clones.
The assessment of novelty places an important role as vital feedback for
the Soares lab used to determine when to stop sequencing from a library.</p>
    <p>The program in use (and constant development) in the CLCG is UIcluster.
The current version is v3.0.5. UIcluster can be used to group sequences based
upon sequence similarity.<br>
 </p>
     
    <blockquote><br>
 3. Parameters available to uicluster (plus standard procedure) <br>
 4. How does the UI clustering differ from other clustering systems?</blockquote>
          
      <p><big><br>
      <big><b>Sequence Processing</b></big> </big><br>
 <br>
 <b>High-level view of the processing</b> </p>
       
      <p>The individual cDNA clones are sequenced generating ESTs (expressed
 sequence tags; a sequence derived from a cDNA clone) at the Oakdale sequencing
 facility, and transferred from there to a local server in the Casavant lab. 
Once local in the lab, they are placed appropriately within our file system 
hierarchy and the sequence processing begins. The first step in the processing 
is to remove low-quality sequences from the processing pipeline. The second 
step is to identify features within the EST sequence. Next the EST sequences 
are clustered together, bringing similar sequences together into units called 
clusters. This is done to derive an estimate for the number of genes that 
have been discovered. Clustering is necessary because of the high percentage 
of redundancy in the mRNA molecules expressed within cells. A final step in
the processing is to generate pages summarizing the results for each gel individually,
and for all ESTs within a given project,&nbsp; and also to load all of the
pertinent sequence and annotation into the project database.        </p>
       
      <p align="center"><img src="seqPipe.jpeg" alt="Image of Sequence Processing Pipeline" width="549" height="509">
      <br>
 </p>
      <p><br>
 </p>
       
      <p><b>Processing details</b> </p>
       
      <p><u>File system hierarchy</u> - </p>
       
      <blockquote>The EST projects are all located on the RAID at /mnt/r5-projects/....../sequence/&lt;direction&gt;.
 The direction depends upon whether the ESTs within the gel were derived
from the 3' or 5' end of the cDNA. Within this directory, gel directories
are located with a directory structure of the form YYYY-MM/DD/MACHINE/GEL_NAME.</blockquote>
            
        <blockquote>Within each gel directory, the following directories should
be present: scf/, qual/, and seq/. These directories hold the chromatograph
 (a.k.a. trace) files, and FASTA formatted quality and sequence files respectively.</blockquote>
  <u>Sample-sheet generator</u> -            
          <blockquote>Prior to sequencing a given plate of cDNA clones, the 
ABI sequencing hardware requires a file containing the information of which 
cDNA clone will be run in which lane or capillary. This is accomplished with 
the sample sheet generator, which is run from brainest (madb). This program 
allows sample sheets to be generated for ABI 377's and 3700's for each project.</blockquote>
                
            <p><br>
 &nbsp; <u>setup-abi</u> - </p>
             
            <blockquote>The first step of processing performed within our 
lab is the setup-abi program, which moves the raw sequencing data transferred 
from Oakdale into the appropriate location based upon project, date, and machine
name. This program also takes the original SCF files derived from the ABI
sequencers, placing it in the ABIscf directory within the gel directory,
 and calling phred on those SCFs, generating the SCF, sequence and quality
 files to be used in the remainder of the processing pipeline.</blockquote>
                  
              <p><u>ESTprep</u> - </p>
               
              <blockquote>The next step in the EST processing pipeline is 
the ESTprep program, written by T. Casavant and currently maintained by N. 
Trivedi, both in the CLCG. This program uses the sequence and quality files 
as inputs, and performs a series of analyses on the data. Each analysis is 
configurable within the prep.params file, which is global to the gel being 
processed within the full pipeline (i.e., when runprep is used).        
         
                <p>First, the overall quality of the sequence is assessed, 
to determine if it is of sufficient quality to continue. Next, a search is 
made within the sequence for the specified restriction site. This search is
limited in our pipeline to the first 150 bases, as the probability of observing
a site similar to an eight-base restriction site (allowing errors) is relatively
high (1:1000 ?). If the ESTs are 3' reads, then an attempt is made to identify
the poly-adenylation tail (which in 3' ESTs is represented by a string of
T's). If the poly-adenylation tail is found, then a search is made for both
library tag and polyadenylation signal. The library tag is a synthetic sequence
inserted between the restriction site and the poly-adenylation  tail during
cDNA library synthesis. Depending upon the set of library tags  used within
a pooled library, it may or may not be appropriate to allow errors within
the library tag sequence. In identifying the poly-adenylation signal, no
errors are allowed as these are often&nbsp; alternative polyadenylation  signals.
(Nishank - HOW is the canonical and alternative PAS search regulated?)  
               </p>
                 
                <p>If at any time, it is determined that the sequence should 
not continue in the sequence processing pipeline (e.g., insufficient quality, 
no restriction site found) the resulting .prpsmry file is annotated with a
'<tt>STATUS= STOP</tt>'. </p>
                 
                <p></p>
                <p>Below are two output files from ESTprep. The convention
is that these files are given the same name as the sequence file, with an
extension of ".prpsmry". In the first file, we see the results for the sequence
UI-R-CV0-bro-h-12-0-UI.s1. For this sequence, the restriction site (<tt>RESTRSITEFOUND</tt>
) was successfully located, as was upstream vector (<tt>VECTORFOUND</tt>),
polyadenylation tail (<tt>POLYATAILFOUND</tt>), canonical polyadenylation
signal (<tt>POLYASIGNALFOUND</tt>), and a library tag (<tt>LIBTAGFOUND</tt>
). Information on the length of vector (<tt>VECTORLENGTH</tt>), polyadenylation
tail length (<tt>POLYATAILLENGTH</tt>), polyadenylation signal sequence (<tt>
POLYASIGNAL</tt>), and the library tag sequence and tissue designation (<tt>
LIBTAG</tt>) are also included in the file, if the appropriate feature has
been identified. Additional information on the sequence quality (<tt>GOODQUALITY,
QUAL_FILTERS</tt>), and final status (<tt>STATUS</tt>) of the sequence are
also included. If the sequence is to continue in our sequence processing
pipeline, the status field must be "GO" and trim locations (<tt>TRIMLOC</tt>
) will be identified. These are the left and right locations within the original
sequence that will be excised for further processing. Current standards are
to leave 18 bases of the polyA tail, if present, or else the restriction
site sequence (NotI for 3' sequences). The right trimming position is determined
by sequence quality.<br>
                </p>
                <p><tt>CLONENAME: seq/UI-R-CV0-bro-h-12-0-UI.s1.seq<br>
RESTRSITEFOUND: TRUE<br>
VECTORFOUND: TRUE<br>
VECTORLENGTH: 33<br>
POLYATAILFOUND: TRUE<br>
POLYATAILLENGTH: 25<br>
LIBTAGFOUND: TRUE<br>
LIBTAG: CAGCC rat-eye<br>
POLYASIGNALFOUND: TRUE<br>
POLYASIGNAL: TTTATT<br>
TRIMLOC: 71 451<br>
GOODQUALITY: 25.36<br>
QUAL_FILTERS: 25.36 39.84<br>
STATUS: GO</tt></p>
                <p>The second sequence (below) contains similar results as
above, with one difference. This sequence contains an alternative (non-canonical)
polyadenylation signal (<tt>ALTPOLYASIGNALFOUND, ALTPOLYASIGNAL</tt>).<br>
                </p>
                <p><tt>CLONENAME: seq/UI-R-CV0-bro-g-01-0-UI.s1.seq<br>
RESTRSITEFOUND: TRUE<br>
VECTORFOUND: TRUE<br>
VECTORLENGTH: 33<br>
POLYATAILFOUND: TRUE<br>
POLYATAILLENGTH: 20<br>
LIBTAGFOUND: TRUE<br>
LIBTAG: CAGCC rat-eye<br>
ACTUALLIBTAG: TAGCC<br>
POLYASIGNALFOUND: FALSE<br>
ALTPOLYASIGNALFOUND: TRUE<br>
ALTPOLYASIGNAL: TTTATG<br>
TRIMLOC: 76 452<br>
GOODQUALITY: 31.20<br>
QUAL_FILTERS: 31.20 18.67<br>
STATUS: GO</tt><br>
                <br>
                </p>
                 </blockquote>
                    
                <p><br>
                 <u>generic-process (trimming + repeat masking)</u> - </p>
                 
                <blockquote>This program is run on a gel of sequneces at a
time. The first step is the trimming of the sequence to remove the leading 
region prior to the restriction site or polyadenylation tail and at the right-hand 
side to remove the low-quality portion of the sequence. The trimmed sequence
 (and quality) file is placed into .seq.trim (and .qual.trim). This program
 also performs several calls to RepeatMasker (below) and will trim the sequence
 and quality files again if additional vector sequence is found at the end
 of good EST sequence.</blockquote>
                      
                  <p><br>
                   <u>RepeatMasker</u> - </p>
                   
                  <blockquote>The RepeatMasker program was written by Arian 
Smit (Univeristy of Washingtion), and utilizes the cross-match program (from 
the phrap package - D. Gordon, P. Green, University of Washington). We use 
RepeatMasker for two separate purposes. First, it is used to detect contaminating 
sequences - bacterial, mitochondrial, and vector. For both bacterial and mitochondrial
contamination, the analysis of results is straight-forward, as the expected
contamination would be an all-or-nothing event. However, vector contamination
is more difficult, as the contamination can be complete (e.g., empty insert),
or very small - due to read-through of a short insert. The second use of
RepeatMasker is to mask the known repetitive elements from the species under
study.                      
                    <p>(HAVE an exceprt from a few .out files here with annotations 
of what each feature tag means.)</p>
                     </blockquote>
                        
                    <p><br>
                     <u>Clustering</u> - </p>
                     
                    <blockquote>The process of clustering is carried out using
the uicluster program, developed locally by K. Pedretti and T. Casavant. We
use a sequence similarity requirement of 38 out of 40 bases identical.  
                     
                      <p>(HAVE an excerpt from a .clus file with annotation, 
plus show some of the basic options to uicluster.)</p>
                       </blockquote>
                          
                      <p><br>
                       <u>Blasting</u> - </p>
                       
                      <p><u>Summary Scripts</u> - BrianO and RyanH -- write 
me </p>
                       
                      <p><u>Database Loading</u> - RyanH -- write me <br>
 &nbsp; <br>
 &nbsp; </p>
                       
                      <p>IV. Contacts: Nat, BrianO, Todd, Nishank (ESTprep)
                       <br>
 &nbsp; </p>
                       
                      <p> </p>
                       
                      <hr width="100%" size="3" noshade=""><br>
Updated: March 3, 2002 by TS<br>
Updated: March 2, 2002 by TS<br>
 Updated: January 29, 2002 by TS                        
                      </body>
                      </html>
