<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>

<HEAD>
	<META NAME="GENERATOR" Content="Symantec Visual Page 1.0.1">
	<META HTTP-EQUIV="Content-Type" CONTENT="text/html;CHARSET=iso-8859-1">
	<TITLE>Bacterial Plasmids</TITLE>
</HEAD>

<BODY BGCOLOR="#FFFFFF">

<P>
<P ALIGN="CENTER"><FONT SIZE="5" COLOR="#990033" face="Arial, Helvetica, sans-serif"><B>Bacterial
Plasmids</B></FONT></P>
<P>
<CENTER>
<P>
<HR>
</P>
</CENTER>
<P><A NAME="plasmids"></A><FONT SIZE="4" COLOR="#AA0000" face="Arial, Helvetica, sans-serif"><B>What
is a plasmid?</B></FONT><FONT face="Arial, Helvetica, sans-serif"><B><BR>
<BR>
</B>
<TABLE BORDER="0" WIDTH="100%">
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000"><I>&#160;</I></FONT></TD>
		<TD WIDTH="80%">
			<P><FONT face="Arial, Helvetica, sans-serif">A </FONT><FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif">plasmid</FONT><FONT
			face="Arial, Helvetica, sans-serif"> is an extra-chromosomal element, often a circular
			DNA. The plasmids we will use in this class typically have three important elements:</P>
			<UL>
				<LI><FONT face="Arial, Helvetica, sans-serif">An origin of replication
				<LI><FONT face="Arial, Helvetica, sans-serif">A selectable marker gene (e.g. resistance
				to ampicillin)
				<LI><FONT face="Arial, Helvetica, sans-serif">A cloning site (a place to insert foreign
				DNAs)
			</UL>
			<P>
		</TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><A NAME="plasmidElmnt"></A><FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif"><I><B>Origins
			of replication:</B></I></FONT></TD>
		<TD WIDTH="80%">Since a plasmid is (by definition) an extrachromosomal element, it cannot make use
			of any <FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif">origin of DNA replication</FONT><FONT
			face="Arial, Helvetica, sans-serif"> in a chromosome. That is, DNA synthesis within
			(i.e. copying of) a plasmid depends on its having an origin of DNA synthesis of its
			own. Obviously, if a plasmid couldn't be copied, it would be rapidly diluted out
			in a population of dividing cells because it couldn't be passed on to daughter cells.</TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif"><B> <A NAME="selection"></A><I>Selectable
			markers</I></B></FONT></TD>
		<TD WIDTH="80%">A selectable marker is not actually a required element of a plasmid, but it makes
			it possible for us to maintain stocks of cells that contain the plasmid uniformly.
			Sometimes, carrying a plasmid puts a cell at a <I>selective disadvantage</I> compared
			to its plasmid-free neighbors, so the cells with plasmids grow more slowly. <A NAME="cured"></A>Cells
			that happen to &quot;kick out&quot; their plasmid during division may be &quot;rewarded&quot;
			by having a higher rate of growth, and so these plasmid-free (sometimes referred
			to as &quot;<FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif">cured</FONT><FONT
			face="Arial, Helvetica, sans-serif">&quot;) cells may take over a population. If
			a plasmid contains a gene that the cell needs to survive (for example, a gene encoding
			an enzyme that destroys an antibiotic), then cells that happen to kick out a plasmid
			are &quot;punished&quot; (by subsequent death) rather than &quot;rewarded&quot; (as
			in the previous scenario). That selective pressure helps to maintain a plasmid in
			a population.</TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif"><I><B>Cloning sites</B></I></FONT></TD>
		<TD WIDTH="80%">A cloning site is not required at all, but it sure is nice to have! What I mean by
			&quot;cloning site&quot; is a place where the DNA can be digested by specific restriction
			enzymes - a point of entry or analysis for genetic engineering work. This is a matter
			we will be discussing in great detail at a later point. For now, think of the following
			example: Suppose you are really thirsty and you buy a can of soda. Does it occur
			to you that one end of the can (the &quot;top&quot;) is designed so that you can
			open it easily? If you bought a can of soda with two bottom ends and no top, you
			would have a hard time drinking it! It's the same way with plasmids. You can have
			a plasmid with lots of terrific features, but you might lack an easy way of &quot;getting
			it open&quot; with restriction enzymes.</TD>
	</TR>
</TABLE>
<FONT face="Arial, Helvetica, sans-serif"></P>

<P>
<TABLE BORDER="0">
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif"><I><B>Coiling in a plasmid</B></I></FONT></TD>
		<TD WIDTH="80%">
			<P><FONT face="Arial, Helvetica, sans-serif">You probably remember that double-stranded
			DNA has the form of a &quot;double helix&quot; which looks a bit like a telephone
			handset cord (except that the telephone cord is a single helix). You may also recall
			that the double helix is right-handed (for an expose on the difference, take a look
			at the </FONT><A HREF="http://www.lecb.ncifcrf.gov/~toms/Leftyear1989.html"><FONT
			face="Arial, Helvetica, sans-serif">Left Handed DNA Hall of Fame Site</FONT></A><FONT
			face="Arial, Helvetica, sans-serif">.)</P>
			<P><FONT face="Arial, Helvetica, sans-serif">You've probably also noticed how knotted
			up a telephone cord can get, if your roommate twists the handset around a few times
			before hanging up. Those knots are a higher order structure that lead to &quot;coiled
			coils.&quot;</P>
			<P><A NAME="supercoil"></A><FONT face="Arial, Helvetica, sans-serif">DNA has the
			same problem, though your roommate isn't to blame this time! Aside from the double-helical
			structure that we all know and love, DNA can take on a higher order coiling that
			twists one double helix around another. We call this &quot;superhelical coiling&quot;
			or simply &quot;</FONT><FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif">supercoiling</FONT><FONT
			face="Arial, Helvetica, sans-serif">.&quot; In a linear molecule these twists can
			unravel by themselves, provided the ends are not prevented from rotating. In a circular
			molecule with no free ends, the superhelical twists are &quot;locked in&quot; and
			the molecule cannot relax. This coiling is not the same as the right-handed double
			helix coil with which you are all familiar. The supercoiled molecule is a coiled
			coil.<BR>
			<BR>
			You can click on the image below to see an electron micrograph of a supercoiled circular
			DNA.</P>
			<CENTER>
			<P>
			<TABLE BORDER="1" WIDTH="50%">
				<TR>
					<TD WIDTH="100%">
						<P ALIGN="CENTER"><A HREF="http://phage.bocklabs.wisc.edu/0014b.jpg" target="_blank">Supercoiled
						DNA (Bock lab)</A>
					</TD>
				</TR>
				<TR>
					<TD WIDTH="100%">
						<CENTER>
						<P><A HREF="http://phage.bocklabs.wisc.edu/0014b.jpg"><IMG SRC="images/sc.gif" WIDTH="160"
						HEIGHT="160" ALIGN="BOTTOM" BORDER="2"></A>
</CENTER>
					</TD>
				</TR>
			</TABLE>

</CENTER>
		</TD>
	</TR>
</TABLE>
<FONT face="Arial, Helvetica, sans-serif"></P>

<P>
<TABLE BORDER="0">
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000"><I><B>Relaxation</B></I></FONT></TD>
		<TD WIDTH="80%">
			<P><A NAME="relax"></A><FONT face="Arial, Helvetica, sans-serif">What's needed to
			get supercoiled circular DNA to relax? A few weeks of pampering at a spa perhaps?
			No! If one of the two strands is broken so that it has free 5' and 3' ends, the supercoils
			can relax even though the overall structure of the molecule remains a circle. The
			free ends of the broken strand rotate around the phosphate backbone of the intact
			strand (the one that wasn't broken). This loss of superhelical stress puts the plasmid
			into a &quot;</FONT><FONT COLOR="#DD0000" face="Arial, Helvetica, sans-serif">relaxed
			DNA</FONT><FONT face="Arial, Helvetica, sans-serif">&quot; form.<BR>
			<BR>
			Another electron micrograph: This one is of relaxed DNA</P>
			<CENTER>
			<P>
			<TABLE BORDER="1" WIDTH="50%">
				<TR>
					<TD WIDTH="100%">
						<P ALIGN="CENTER"><A HREF="http://phage.bocklabs.wisc.edu/0001a.jpg" target="_blank">Relaxed
						DNA (Bock lab)</A>
					</TD>
				</TR>
				<TR>
					<TD WIDTH="100%">
						<CENTER>
						<P><A HREF="http://phage.bocklabs.wisc.edu/0001a.jpg"><IMG SRC="images/rel.gif" WIDTH="160"
						HEIGHT="160" ALIGN="BOTTOM" BORDER="2"></A>
</CENTER>
					</TD>
				</TR>
			</TABLE>

</CENTER>
		</TD>
	</TR>
</TABLE>
<FONT face="Arial, Helvetica, sans-serif"></P>

<P><FONT SIZE="4" COLOR="#AA0000" face="Arial, Helvetica, sans-serif"><B>What is
a vector?</B></FONT><FONT face="Arial, Helvetica, sans-serif"><BR>
<BR>

<TABLE BORDER="0" WIDTH="100%">
	<TR>
		<TD WIDTH="20%" VALIGN="TOP">&#160;</TD>
		<TD WIDTH="80%">
			<P><A NAME="vector"></A><FONT face="Arial, Helvetica, sans-serif">Plasmids are sometimes
			called &quot;</FONT><FONT COLOR="#AA0000" face="Arial, Helvetica, sans-serif">vectors</FONT><FONT
			face="Arial, Helvetica, sans-serif">&quot;, because they can take DNA from one organism
			to the next. Not all vectors are plasmids, however. We commonly use engineered viruses,
			for example <A NAME="lambda"></A>bacteriophage lambda, which can carry large pieces
			of foreign DNA.</P>
			<CENTER>
			<P>
			<TABLE BORDER="1">
				<TR>
					<TD>
						<P ALIGN="CENTER"><A HREF="http://phage.bocklabs.wisc.edu/0022a.jpg" target="_blank">Bacteriophage
						lambda (Bock lab)</A>
					</TD>
				</TR>
				<TR>
					<TD>
						<CENTER>
						<P><A HREF="http://phage.bocklabs.wisc.edu/0022a.jpg"><IMG SRC="images/lambda.gif"
						WIDTH="144" HEIGHT="173" ALIGN="BOTTOM" BORDER="2"></A>
</CENTER>
					</TD>
				</TR>
			</TABLE>

</CENTER>
		</TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP">&#160;</TD>
		<TD WIDTH="80%">Why do we use the word &quot;vector,&quot; which we've been trying to forget ever
			since we took Physics 100? The word has a connotation of taking something from one
			place to another. A mosquito is said to be a &quot;vector&quot; for malarial parasites,
			and a velocity &quot;vector&quot; in physics indicates a direction in which an object
			is travelling. In molecular biology, a &quot;vector&quot; is a piece of DNA that
			may be introduced into a cell, usually after we've played around with it a bit in
			a test tube.</TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000"><I><B>Orientation</B></I></FONT></TD>
		<TD WIDTH="80%">One important concept is that depending on the cloning strategy employed, a gene
			could be inserted into the plasmid in either of two orientations:<BR>
			<BR>
			
			<TABLE BORDER="1" WIDTH="381">
				<TR>
					<TD WIDTH="177">
						<P ALIGN="CENTER">Left to right orientation
					</TD>
					<TD WIDTH="185">
						<P ALIGN="CENTER">Right to left orientation
					</TD>
				</TR>
				<TR>
					<TD COLSPAN="2">
						<CENTER>
						<P><IMG SRC="images/orientate.gif" WIDTH="348" HEIGHT="177" ALIGN="BOTTOM" BORDER="0">
</CENTER>
					</TD>
				</TR>
			</TABLE>
<BR>
			<BR>
			Perhaps we don't care which orientation we obtain as our final product, but we should
			note that there is a fundamental difference between the two. The arrow in the diagram
			shows the direction of transcription/translation of the &quot;red gene&quot; coding
			sequence, and the two orientations differ with respect to the outside markers Amp
			and ori.</TD>
	</TR>
</TABLE>
<FONT face="Arial, Helvetica, sans-serif"></P>

<P><FONT face="Arial, Helvetica, sans-serif"><BR>
<BR>
<A NAME="Refinement"></A></FONT><FONT SIZE="4" COLOR="#AA0000" face="Arial, Helvetica, sans-serif"><B>How
do we clone a plasmid?</B></FONT></P>

<P>
<TABLE BORDER="0" WIDTH="100%">
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000"><I><B>How do we isolate a plasmid we want?</B></I></FONT></TD>
		<TD WIDTH="80%">
			<UL>
				<LI><FONT COLOR="#000000">We introduce the reclosed (</FONT><FONT COLOR="#AA0000">ligated</FONT><FONT
				COLOR="#000000">) products into E. coli, a process called &quot;</FONT><FONT COLOR="#AA0000">transformation</FONT><FONT
				COLOR="#000000">&quot;, and </FONT><FONT COLOR="#AA0000">select</FONT><FONT COLOR="#000000">
				for bacteria resistant to a drug (such as amipicillin, for example).</FONT>
				<LI><FONT COLOR="#000000">We </FONT><FONT COLOR="#AA0000">screen</FONT><FONT COLOR="#000000">
				the individual bacterial colonies to find one that contains a plasmid of the correct
				structure.</FONT>
			</UL>
			<P>
		</TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000"><I><B>Transformation is natural.</B></I></FONT></TD>
		<TD WIDTH="80%"><A NAME="Transformation"></A><FONT COLOR="#000000">Bacteria naturally take up DNA
			from their environment, and we call that process transformation.</FONT></TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><FONT COLOR="#AA0000"><I><B>Efficiency of transformation in the lab.</B></I></FONT></TD>
		<TD WIDTH="80%">
			<P><A NAME="transform_ways"></A><FONT COLOR="#000000">When we are transforming DNA
			in the laboratory (i.e. for experimental purposes), we have several ways of making
			the uptake of DNA by E. coli cells more efficient.</FONT></P>
			<UL>
				<LI><FONT COLOR="#000000">One method is to starve the cells in ice-cold calcium chloride
				solution, add a sample of ligated DNA, and &quot;heat-shock&quot; the cells at 42
				degrees C for a short period of time - about 45 seconds. The fluidity of the membrane
				increases to the point where DNA is taken up by the cell.</FONT>
				<LI><FONT COLOR="#000000">A second method is to deliver an electric shock to the
				cell, releasing a charged capacitor with a field strength in the sample of approximately
				1200 volts per millimeter. The DNA is swept into the cells as the membranes are temporarily
				breached. This process is called </FONT><FONT COLOR="#AA0000">electroporation</FONT><FONT
				COLOR="#000000">.</FONT>
			</UL>
		</TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><A NAME="Selection"></A><FONT COLOR="#AA0000"><I><B>Selection</B></I></FONT></TD>
		<TD WIDTH="80%"><FONT COLOR="#000000">After transformation, we challenge the bacteria with an antibiotic
			(such as ampicillin). If the E. coli have taken up and expressed an ampicillin resistance
			gene on a plasmid, they will live - otherwise they will die. This process is called
			</FONT><FONT COLOR="#AA0000">selection</FONT><FONT COLOR="#000000">, because we are
			selecting which bacteria may survive.<BR>
			<BR>
			Transformation is a rare event, so most bacteria in an experiment are killed by the
			antibiotic. If a bacterium takes up a piece of DNA that cannot be maintained in a
			cell (e.g. if it lacks an </FONT><A HREF="../L1/FrameMain1.html#plasmidElmnt" target="Frame323050">origin
			of DNA replication</A><FONT COLOR="#000000">) that cell also will not survive. It's
			a tough world!</FONT></TD>
	</TR>
	<TR>
		<TD WIDTH="20%" VALIGN="TOP"><A NAME="Screening"></A><FONT COLOR="#AA0000"><I><B>Screening</B></I></FONT></TD>
		<TD WIDTH="80%"><FONT COLOR="#000000">At this stage we have a bacteriological plate (agar medium
			containing ampicillin) with bacterial colonies on it. Each colony contains a different
			plasmid type, because each was grown up from a single transformed cell. What we do
			now is to isolate DNA from each colony (or a small growth of cells propagated from
			the colony), and analyze the structure of the plasmid with restriction enzymes or
			by DNA sequencing. We can use gel electrophoresis to identify the sizes of restriction
			fragments that are released from the plasmid and to check the purity of the preparation.<BR>
			<BR>
			If you are unfamiliar with the principles of gel electrophoresis, </FONT><A HREF="gels.htm">you
			may be helped by this explanation (in which fish and DNA are one and the same)</A></TD>
	</TR>
</TABLE>



</BODY>

</HTML>