<HTML>  <HEAD>  <META NAME="GENERATOR" CONTENT="Adobe PageMill 3.0 Mac">  <LINK REV="made" HREF="mailto:mulligan@morgan.ucs.mun.ca">  <TITLE>Bacterial Plasmids</TITLE></HEAD><BODY BGCOLOR="#ffffbd"><BLOCKQUOTE>  <H2><CENTER>&nbsp;</CENTER></H2></BLOCKQUOTE><H2><CENTER><TABLE WIDTH="90%" BORDER="1" CELLSPACING="0" CELLPADDING="15">  <TR>    <TD WIDTH="100%" BGCOLOR="#002600">      <H1><CENTER><FONT COLOR="#ffffff" FACE="Arial">Bacterial Plasmids</FONT></CENTER></H1>    </TD>  </TR></TABLE></CENTER></H2><BLOCKQUOTE>  <H2><CENTER>&nbsp;</CENTER></H2>  <P>Bacterial cells may contain a number of plasmids which have  different and varied functions such as funtions for colicin resistance,  symbiosis, nitrogen fixation, and tumour induction in plants.  Among the first to be discovered -- soon after the discovery  of the F plasmid -- were the resistance transfer factors, which  have been extensively modified and adapted over the years since  to become the modern cloning plasmids.</P>  <P>&nbsp;</P></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">ResistanceTransfer Factors</FONT></B></P><BLOCKQUOTE>  <P>In the late 1950's, Japanese hospitals were faced with outbreaks  of <I><FONT COLOR="#00194c">Shigella</FONT></I> dysentery which  were resistant to the usual antibiotics. The following table  shows the data for the occurrence of antibiotic resistant strains  between 1953 and 1960:</P>  <P>&nbsp;</P>  <P><CENTER><TABLE WIDTH="450" BORDER="0" CELLSPACING="2" CELLPADDING="1">    <TR>      <TD WIDTH="11%" BGCOLOR="#0099ff">        <P><CENTER><B><FONT FACE="Arial">&nbsp;Year</FONT></B></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#0099ff">        <P><CENTER><B><FONT FACE="Arial">&nbsp;# Strains tested</FONT></B></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#0099ff">        <P><CENTER>&nbsp;<B><FONT FACE="Arial">Sm</FONT></B></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#0099ff">        <P><CENTER>&nbsp;<B><FONT FACE="Arial">Tc</FONT></B></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#0099ff">        <P><CENTER>&nbsp;<B><FONT FACE="Arial">Cm</FONT></B></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#0099ff">        <P><CENTER><B><FONT FACE="Arial">&nbsp;Sm + Cm</FONT></B></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#0099ff">        <P><CENTER>&nbsp;<B><FONT FACE="Arial">Sm + Tc</FONT></B></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#0099ff">        <P><CENTER>&nbsp;<B><FONT FACE="Arial">Cm + Tc</FONT></B></CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#0099ff">        <P><CENTER>&nbsp;<B><FONT FACE="Arial">Sm, Cm &amp; Tc</FONT></B></CENTER></TD>    </TR>    <TR>      <TD WIDTH="11%" BGCOLOR="#33ccff">        <P><CENTER><FONT FACE="Arial">&nbsp;1953</FONT></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#99ffff">        <P><CENTER>&nbsp;4900</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;5</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;2</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>    </TR>    <TR>      <TD WIDTH="11%" BGCOLOR="#33ccff">        <P><CENTER>&nbsp;<FONT FACE="Arial">1954</FONT></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#99ffff">        <P><CENTER>&nbsp;4876</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;11</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>    </TR>    <TR>      <TD WIDTH="11%" BGCOLOR="#33ccff">        <P><CENTER>&nbsp;<FONT FACE="Arial">1955</FONT></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#99ffff">        <P><CENTER>&nbsp;5327</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;4</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;1</CENTER></TD>    </TR>    <TR>      <TD WIDTH="11%" BGCOLOR="#33ccff">        <P><CENTER>&nbsp;<FONT FACE="Arial">1956</FONT></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#99ffff">        <P><CENTER>&nbsp;4399</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;8</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;4</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;1</CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>    </TR>    <TR>      <TD WIDTH="11%" BGCOLOR="#33ccff">        <P><CENTER>&nbsp;<FONT FACE="Arial">1957</FONT></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#99ffff">        <P><CENTER>&nbsp;4873</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;13</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;46</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;2</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;2</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;37</CENTER></TD>    </TR>    <TR>      <TD WIDTH="11%" BGCOLOR="#33ccff">        <P><CENTER>&nbsp;<FONT FACE="Arial">1958</FONT></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#99ffff">        <P><CENTER>&nbsp;6563</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;18</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;20</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;7</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;2</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;193</CENTER></TD>    </TR>    <TR>      <TD WIDTH="11%" BGCOLOR="#33ccff">        <P><CENTER>&nbsp;<FONT FACE="Arial">1959</FONT></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#99ffff">        <P><CENTER>&nbsp;4071</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;16</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;32</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;71</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;74</CENTER></TD>    </TR>    <TR>      <TD WIDTH="11%" BGCOLOR="#33ccff">        <P><CENTER>&nbsp;<FONT FACE="Arial">1960</FONT></CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#99ffff">        <P><CENTER>&nbsp;3396</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;29</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;36</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;0</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;61</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;9</CENTER></TD>      <TD WIDTH="11%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;7</CENTER></TD>      <TD WIDTH="12%" BGCOLOR="#ffffff">        <P><CENTER>&nbsp;308</CENTER></TD>    </TR>  </TABLE></CENTER></P>  <P>&nbsp;</P>  <P>These resistant strains were odd in a number of way:</P>  <P>&nbsp;</P>  <UL>    <LI>Multiple drug resistances appeared rather than single resistances<BR>    <LI>Chloramphenicol resistance was never found (by itself)<BR>  </UL>  <P>A clue to the mystery of these strains was found when a multiple-drug  resistant strain of <I><FONT COLOR="#00194c">E.&nbsp;coli</FONT></I>  was identified. It turned out that the <I><FONT COLOR="#00194c">Shigella</FONT></I>  strains had obtained their multiple-drug resistant phenotype  from <I><FONT COLOR="#00194c">E.&nbsp;coli</FONT></I>.</P>  <P>We now know that resistance in these and similar cases is  due to the inheritance of large conjugative plasmids (<FONT COLOR="#ff0000">R  plasmids</FONT>) that carry genes for more than one antibiotic  resistance. R plasmids code for their own replication, they have  genes for their own conjugal transfer, and they have mobile genetic  elements.</P>  <P>[<A HREF="../figures/Kornberg18-1.jpg">K18.1</A>]</P>  <P>(<FONT COLOR="#333333">One might note, in passing, that --  even as early as the 1960's -- the use of antibiotics was considered  commonplace and routine. Is it any wonder that we are now facing  the consequences of 40 years of antibiotic abuse?</FONT>)</P>  <P>&nbsp;</P></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">Other NaturalPlasmids</FONT></B></P><BLOCKQUOTE>  <P>Some other naturally-occurring plasmids are:</P>  <P>[<A HREF="../figures/Snyder-Champness/T4-1.jpg">T4.1</A>]  <FONT COLOR="#990000" FACE="Arial Rounded MT Bold">T4-1</FONT></P>  <P>&nbsp;</P>  <UL>    <LI>The colicin plasmids of <I><FONT COLOR="#00194c">E.&nbsp;coli</FONT></I>.    These are small plasmids which encode the genes to synthesize    toxins -- colicins -- that will kill other <I><FONT COLOR="#00194c">E.&nbsp;coli</FONT></I>    cells. The host is not killed because the colicin plasmid also    codes for an immunity function which protects the host cell.<BR><BR>    <LI>The symbiotic plasmids of <I><FONT COLOR="#00194c">Rhizobium    spp</FONT></I>. are responsible for the formation of root nodules    in susceptible plants.<BR><BR>    <LI>The nitrogen-fixation operon genes of <I><FONT COLOR="#00194c">Klebsiella    pneumoniae</FONT></I> are carried on a large plasmid.<BR><BR>    <LI><I><FONT COLOR="#00194c">Agrobacterium tumefaciens</FONT></I>    will infect wounds in dicotyledonous plants and cause the growth    of a mass of undifferentiated cells. This loss of control of    cell growth and division is the result of the transfer of a section    of DNA from a plasmid in <I><FONT COLOR="#00194c">Agrobacterium</FONT></I>    into the plant cells. The plasmid is known as a Ti (tumour inducing)    plasmid. The DNA that is transferred may code for the synthesis    of a number of plant growth hormones. When these genes are expressed    in a plant cell they promote unregulated cell division.<BR><BR>  </UL></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">Structure ofPlasmids</FONT></B></P><BLOCKQUOTE>  <P>Plasmids are typically circular dsDNA molecules which range  in size from 2 Kb to 100 Kb. These plasmids will be supercoiled  in the cell.</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-1.jpg">4.1</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-1</FONT></P>  <P>Plasmids can also exist as linear dsDNA molecules.</P>  <P>The spirochaete <I><FONT COLOR="#00194c">Borrelia burgdorfferi</FONT></I>  has at least 17 plasmids. 11 of these have recently been sequenced;  2 are circular and 9 are linear molecules. These linear molecules  are covalently closed and have terminal sequences that function  as telomeres.</P>  <P>[<A HREF="../figures/Snyder-Champness/B4-1.jpg">B4.1</A>]  <FONT COLOR="#990000" FACE="Arial Rounded MT Bold">B4-2</FONT></P>  <BLOCKQUOTE>    <P>&nbsp;</P></BLOCKQUOTE></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">Plasmid Replication</FONT></B></P><BLOCKQUOTE>  <P>Plasmids must be able to replicate independently of the host  chromosome. They have their own <FONT COLOR="#351a00">origin  of replication</FONT> (<I><FONT COLOR="#00194c">ori</FONT></I>).  There are two general modes of replication: <FONT COLOR="#0000ff"   FACE="Arial">THETA Replication</FONT> -- which is found in most  of the common gram negative plamids such as ColE1, RK2, and F;  and, <FONT COLOR="#0000ff" FACE="Arial">ROLLING&nbsp;CIRCLE Replication</FONT><FONT   FACE="Arial"> </FONT>-- which is found in some gram positive  bacterial plasmids.</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-2.jpg">4.2</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-4 ***</FONT></P>  <P>Replication is central to the control of a number of important  plasmid properties: <FONT COLOR="#351a00">HOST RANGE</FONT>;  <FONT COLOR="#351a00">COPY NUMBER</FONT>; <FONT COLOR="#351a00">INCOMPATIBILITY</FONT>;  and <FONT COLOR="#351a00">MOBILITY</FONT>.</P>  <P><TABLE WIDTH="99%" HEIGHT="167" BORDER="0" CELLSPACING="2"  CELLPADDING="5">    <TR>      <TD WIDTH="100%" HEIGHT="134" BGCOLOR="#cccccc">        <P><A HREF="../figures/N15_plasmid_rep_5019f04.gif"><IMG SRC="../figures/N15_plasmid_rep_5019f04.gif"        WIDTH="106" HEIGHT="83" ALIGN="RIGHT" BORDER="0" NATURALSIZEFLAG="2"></A>Little        is known about the mechanism of replication of linear covalently        closed DNA such as the linear plasmids found in <I>Borrleia burgdorfferi</I>.        Some recent evidence suggests that the model shown at right may        be reasonable for the bacteriophage N15 prophage.</P>        <P>Replication requires a &quot;protelomerase&quot; which has        been found to be required for maintenance of a linear plasmid        because it acts as a resolvase of replicative intermediates.        Repression of protelomerase results in the accumulation of circular        head-to-head dimer molecules.      </TD>    </TR>    <TR>      <TD HEIGHT="26"><FONT SIZE="-1" FACE="Arial">Nikolai V. Ravin,        Taisia S. Strakhova, Victor V. Kuprianov (2001) The Protelomerase        of the Phage-plasmid N15 is Responsible for its Maintenance in        Linear Form. <A HREF="http://www.idealibrary.com/links/doi/10.1006/jmbi.2001.5019/full">J.        Mol. Biol. 312(5): 899-906.</A></FONT></TD>     </TR>  </TABLE></P>  <BLOCKQUOTE>    <P>&nbsp;</P></BLOCKQUOTE></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">Plasmid HostRange</FONT></B></P><BLOCKQUOTE>  <P>Some plasmids are able to replicate in a limited number of  bacterial species; they have a <FONT COLOR="#351a00">NARROW </FONT>host  range. Examples are ColE1, pBR322, pUC18 plasmids which are limited  to <I><FONT COLOR="#00194c">E.&nbsp;coli</FONT></I> and some  closely related species.</P>  <P>Other plasmids are able to replicate in a wide range of bacterial  species; they have a <FONT COLOR="#351a00">BROAD</FONT> host  range.</P>  <P>[<A HREF="../figures/Kornberg18-1.jpg">K18.1</A>]</P>  <P>The ability of a plasmid to replicate in many different species  of bacteria means that many of the sequence signals for gene  expression, control of gene expression, and for replication,  must all work in many different species.</P>  <BLOCKQUOTE>    <P>&nbsp;</P></BLOCKQUOTE></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">Control ofPlasmid Copy Number</FONT></B></P><BLOCKQUOTE>  <P>The number of copies of a plasmid can vary from 1 (the F plasmid)  to over a hundred (pUC18). This number is a property of the plasmid  itself and depends on the mechanism by which it regulates its  own replication.</P>  <P>[<A HREF="../figures/Snyder-Champness/T4-2.jpg">T4.2</A>]  <FONT COLOR="#990000" FACE="Arial Rounded MT Bold">T4-2</FONT></P>  <P>We discussed four different mechanisms of control of copy  number in the course. This description should complement that  in your textbook.</P>  <P>&nbsp;</P>  <P><B><U><FONT COLOR="#351a00" FACE="Arial">Replication of ColE1  based plasmids</FONT></U></B></P>  <P>ColE1 is a small circular plasmid which codes for a 57 kDal  protein toxin (colicin E1) which can kill other <B><I><FONT COLOR="#000066">E.&nbsp;coli</FONT></I></B>  cells by depolarizing the bacterial membrane.</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-3.jpg">4.3</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-5</FONT></P>  <P>The ColE1 plasmid carries the following genes:</P>  <P><CENTER><TABLE WIDTH="300" HEIGHT="270" BORDER="0" CELLSPACING="2"  CELLPADDING="2">    <TR>      <TD WIDTH="22%" BGCOLOR="#996633"><FONT COLOR="#cc0000">&nbsp;</FONT><B><FONT         COLOR="#cc0000" FACE="Arial">gene</FONT></B></TD>       <TD WIDTH="78%" BGCOLOR="#996633"><FONT COLOR="#cc0000" FACE="Arial">&nbsp;<B>function</B></FONT></TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966"><B><I><FONT COLOR="#00194c">&nbsp;cea</FONT></I></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;the colicin toxin</TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966"><B><I><FONT COLOR="#00194c">&nbsp;imm</FONT></I></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;immunity protein</TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966"><B><I><FONT COLOR="#00194c">&nbsp;kil</FONT></I></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;lysis protein</TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966">&nbsp;<B><I><FONT COLOR="#00194c">inc</FONT></I></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;RNA I - incompatibility determinant</TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966"><B><FONT COLOR="#00194c">&nbsp;RNA II</FONT></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;primer for replication</TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966">&nbsp;<B><I><FONT COLOR="#00194c">rop</FONT></I></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;protein that regulates priming and        copy number</TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966"><B><I><FONT COLOR="#00194c">&nbsp;mob</FONT></I></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;proteins for mobilization during        conjugation</TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966"><B><I><FONT COLOR="#00194c">&nbsp;cer</FONT></I></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;maintains plasmid as monomers</TD>     </TR>    <TR>      <TD BGCOLOR="#cc9966">&nbsp;<B><I><FONT COLOR="#00194c">exc</FONT></I></B></TD>       <TD BGCOLOR="#ffcc99">&nbsp;exclusion protein</TD>     </TR>  </TABLE></CENTER></P>  <P>&nbsp;</P>  <P>It also carries and origin of replication (<B><I><FONT COLOR="#00194c">oriV</FONT></I></B>)  and a region (<B><I><FONT COLOR="#00194c">bom</FONT></I></B>)  at which it can be mobilized for transfer to other bacteria.</P>  <P>Replication of ColE1 proceeds unidirectionally from <B><I><FONT   COLOR="#00194c">oriV</FONT></I></B> and depends on RNA II which  is transcribed from a promoter 555 bp away from <B><I><FONT COLOR="#00194c">oriV</FONT></I></B>.  When this transcript reaches <B><I><FONT COLOR="#00194c">oriV</FONT></I></B>,  one of two possible fates is possible:</P>  <P>[<A HREF="../figures/Kornberg18-2.jpg">K18.2</A>]</P>  <P>&nbsp;</P>  <UL>    <LI>The RNA-DNA hybrid is cleaved by RNaseH to generate a free    3'-OH end which serves as a primer for DNA polymerase I to initiate    replication. DNA polymerase III takes over after a short distance.<BR><BR>    <LI>The RNA-DNA hybrid is NOT cleaved by RNaseH and transcription    continues nonproductively.  </UL>  <P>&nbsp;</P>  <P>The choice between these fates depends on the formation of  a specific secondary structure in the nascent RNA II transcript.  If the structure forms correctly then RNaseH processing occurs  correctly; if not, it does not.</P>  <P>Clearly, anything that interferes with the formation of the  correct secondary structure will interfere with plasmid replication.</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-4.jpg">4.4</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-6</FONT></P>  <P>There is a specific RNA whose function is to do just that!  RNA I is a 108 nt molecule which is transcribed -- in the opposite  direction to RNA II -- from a promoter located between the RNA  II promoter and the origin of replication:</P>  <P>&nbsp;</P>  <P><CENTER><IMG SRC="../figures/ColE1.GIF" WIDTH="224" HEIGHT="119"  ALIGN="BOTTOM" NATURALSIZEFLAG="3"></CENTER></P>  <P>&nbsp;</P>  <P>Since RNA I is perfectly complementary to RNA II, it is able  to form a stable RNA-RNA hybrid with it. If it does, then RNA  II will not be able to function as a primer.</P>  <P>This is an example of <B>antisense RNA</B> regulation.</P>  <P>Replication of ColE1 also depends on the <FONT COLOR="#004300">Rop</FONT>  protein.</P>  <BLOCKQUOTE>    <P><FONT COLOR="#004300">Rop</FONT> is a small protein (63 aas)    which increases the rate of binding of RNA I to RNA II. If <FONT     COLOR="#004300">Rop</FONT> is absent, plasmid replication will    be more frequent.</P>    <P>The relative copy numbers of pBR322 and pUC18 illustrate the    <A HREF="#anchor246135">role of Rop</A> perfectly. There are    approximately 15 copies of pBR322 per cell; however, there are    50-100 copies of pUC18, which is derived from pBR322. The relevant    difference between the two plasmids is the fact that pBR322 contains    the <I><FONT COLOR="#00194c">rop</FONT></I> gene but pUC18 does    not.</P></BLOCKQUOTE>  <P>Turnover of the RNAI molecule also plays a role in the control  of replication of these plasmids.</P>  <BLOCKQUOTE>    <P>RNAI is a rare example of a prokaryotic RNA that is polyadenylylated.    It contains a short, 25-50 nt, long polyA tail. This tail acts    as a starting site for exonucleolytic degradation of RNAI by    polynucleotide phosphorylase, a degradation that is facilitated    if RNAI has been cleaved by RNAseE.</P>    <P>[<A HREF="../figures/RNAI_coleEI.gif">RNAI structure</A>]</P></BLOCKQUOTE>  <P><TABLE WIDTH="99%" BORDER="0" CELLSPACING="2" CELLPADDING="5">    <TR>      <TD WIDTH="100%" BGCOLOR="#e6e6e6">        <P><A HREF="../figures/RNA_decay_chao.jpg" TARGET="_blank"><IMG         SRC="../figures/RNA_decay_chao.jpg" WIDTH="141" HEIGHT="130"        ALIGN="RIGHT" BORDER="0" NATURALSIZEFLAG="1"></A>RNase E is an        essential ribonuclease with a key role in the degradation and/or        processing of RNA. In E.&nbsp;coli, RNase&nbsp;E is present as        a membrane bound multicomponent complex called a &quot;degradosome&quot;.        The other major components are polynucleotide phosphorylase (PNPase),        the RhlB RNA helicase, and enolase while the DnaK chaperonin        protein, GroEL, and polynucleotide phosphate kinase (PPK) are        minor components. The ratios of the major components may vary        during different stages of cell growth though the RhlB helicase        remains at close to a 1:1 ratio.</P>        <P>The C-terminal part of RNaseE appears to be responsible for        binding these other components. However, these other components        are not essential for RNaseE function since a truncated RNase        E protein lacking the C-terminal half is sufficient for cell        viability and for RNA degradation and processing <I>in vivo</I>        in <I>E. coli</I>.</P>        <P>The degradosome has also been shown to contain RNA molecules        such as RNAI and<BR>        fragmented rRNAs which are known to be substrates for RNase E        . RNaseE can cleave RNAI at three or four internal sites. More        surprising was an observation that degradosomes could also contain        the10Sa/ssrA RNA/tmRNA.</P>        <P>The ssrA RNA is transcribed as a 457-nt precursor which is        processed at both 5' and 3' ends to generate the mature tmRNA.        Processing at the 5' end requires RNase P; processing at the        3' end requires RNase E, which cuts the precursor at three sites        immediately adjacent to the CCA-3'end.      </TD>    </TR>    <TR>      <TD WIDTH="100%" BGCOLOR="#cccccc">        <P><B><FONT SIZE="-1" FACE="Arial">&nbsp;Image from <A HREF="http://www.sinica.edu.tw/imb/researcher/Sue-Lin-Chao/Sue-Lin-Chao.html">Sue        Lin-Chao's web site</A>.</FONT></B></P>        <P><B><FONT SIZE="-1" FACE="Arial">See also:</FONT></B></P>        <UL>          <LI><B><FONT SIZE="-1" FACE="Arial">Gunn-Guang Liou, Wann-Neng          Jane, Stanley N. Cohen, Na-Sheng Lin, and Sue Lin-Chao (2001)          <A HREF="http://www.pnas.org/cgi/content/full/98/1/63">RNA degradosomes          exist in vivo in Escherichia coli as<BR>          multicomponent complexes associated with the cytoplasmic membrane<BR>          via the N-terminal region of ribonuclease E.</A> <I>Proc. Natl.          Acad. Sci. USA</I>, 98(1): 63-68.</FONT></B>          <LI><B><FONT SIZE="-1" FACE="Arial">Sue Lin-Chao, Chia-Li Wei,          and Yi-Tzu Lin (1999) <A HREF="http://www.pnas.org/cgi/content/full/96/22/12406">RNase          E is required for the maturation of ssrA RNA and normal ssrA          RNA peptide-tagging activity.</A> <I>Proc. Natl. Acad. Sci. USA</I>,          96(22): 12406-12411.</FONT></B>          <LI><B><FONT SIZE="-1" FACE="Arial">Vladimir R. Kaberdin , Yen-Huei          Chao and Sue Lin-Chao (1996) <A HREF="http://www.jbc.org/cgi/content/full/271/22/13103">RNase          E Cleaves at Multiple Sites in Bubble Regions of RNA I Stem Loops          Yielding Products That Dissociate Differentially from the Enzyme.</A>          <I>J.Biol.Chem.</I> 271(22): 13103-13109.</FONT></B>        </UL>      </TD>    </TR>  </TABLE></P>  <P><IMG SRC="../figures/new.gif" WIDTH="23" HEIGHT="14" ALIGN="RIGHT"  BORDER="0" NATURALSIZEFLAG="3">The role of polyadenylation in  regulating copy number control of ColEI plasmids was discovered  when a new mutation affecing <FONT COLOR="#ff0000">p</FONT>lasmid  <FONT COLOR="#ff0000">c</FONT>opy <FONT COLOR="#ff0000">n</FONT>umber  (<I><FONT COLOR="#ff0000">pcnB</FONT></I>) was discovered and  shown to be the the same gene as that coding for a bacterial  polyA polymerase. More recently, Sarkar, Cao and Jain (Mol Genet  Genomics (2002) <A HREF="http://www.springerlink.com/app/home/content.asp?wasp=lp3efhmqxrdtv877tmdq&referrer=contribution&format=13&page=1&pagecount=8">268:  62-69</A> [<A HREF="http://www.springerlink.com/app/home/content.asp?wasp=lp3efhmqxrdtv877tmdq&referrer=contribution&format=2&page=1&pagecount=8">PDF</A>])  carrried out a study to identify additional polyA polymerases  and, instead, discovered two unexpected regulatory molecules.  (<I>pcnB</I> mutants display residual polyadenylation activity)</P>  <P>They started with a <FONT FACE="Symbol">D</FONT><I>pcnB</I>  strain of <I>E.&nbsp;coli</I> carrying a ColEI based plasmid  which contained the <I>lac</I> operon. Expression of <FONT FACE="Symbol">b</FONT>-galactosidase  is then an approximate measure of plasmid copy number. Colonies  containing this plasmid were less blue than wild-type strains  of <I>E.&nbsp;coli</I> carrying the same plasmid.</P>  <P>A genomic library of <I>E.&nbsp;coli</I> DNA was constructed  in a p15 based plasmid. This plasmid has a replicon which is  compatible with that of ColEI plasmids. The plasmid library was  transformed into the <FONT FACE="Symbol">D</FONT><I>pcnB</I>  strain of <I>E.&nbsp;coli</I> carrying the <I>lac</I> operon  plasmid.</P>  <P>A small number of colonies appeared to have a darker blue  colour that the others. This indicated that the p15 plasmid was  expressing some function which was interfering with the copy  number control of the ColEI plasmid. The isolated and measure  the <FONT FACE="Symbol">b</FONT>-galactosidase acticvity in 14  colonies and found it to be 1.5 to 3.6 times higher than in the  'control' colonies.</P>  <P>[<A HREF="../figures/Sarkar_Cao_Jain/table.jpg">table</A>]</P>  <P>When total plasmid DNA was isolated from these 14 strains,  they found that the copy number of the ColEI based plasmid was  increased in 13 of the 14 cases. Restriction digestion of the  p15 plasmids indicated that there were 6 different plasmids isolated.  Sequence analysis revealed that the six plasmids fell into two  groups - 4 plasmids belonged to one group; the other group consisted  of two plasmids.</P>  <P>[<A HREF="../figures/Sarkar_Cao_Jain/plasmid_digest.jpg">plasmid_digest</A>]</P>  <P>The smaller group both contained an insert with the <I>rssB</I>  gene and the larger group was found to contain an insert with  the <I>eutE</I> gene.</P>  <P>RssB is a response regulator protein whose function appears  to be to present the sigma factor RpoS to the protease ClpXP  for degradation. It is not clear why RssB should affect plasmid  copy number.</P>  <P>EutE is similar to a <I>Salmonella</I> protein which functions  in ethanolamine metabolism.</P>  <P>In order to confirm the role of these two proteins in regulating  plasmid copy number, each was cloned into an expression vector,  pET19. This vector permits T7 RNA polymerase driven expression  of cloned reading frames.</P>  <P>When RssB was over-expressed in this way, plasmid copy number  increased ~4 fold.</P>  <P>[<A HREF="../figures/Sarkar_Cao_Jain/expression1.jpg">expression1</A>]</P>  <P>When EutE was over-expressed, there was no change in plasmid  copy number. However, when the <I>eutE</I> gene was cloned in  the reverse orientation (which was expected to have no effect),  plasmid copy number increased.</P>  <P>[<A HREF="../figures/Sarkar_Cao_Jain/expression2.jpg">expression2</A>]</P>  <P>Therefore, it was not <I>eutE</I> that affected plasmid copy  number, but something encoded on the other strand and expressed  in the opposite orientation. Analysis of this strand, revealed  a 16 nt sequence that was complementary to RNAI in 15 of the  16 positions. It is possible that this noncoding RNA regulates  plasmid copy number by sequestering RNAI and preventing it from  interacting with RNAII. However, there were a number of unexplained  observations so this remains just a hypothetical mechanism.</P>  <P>[<A HREF="../figures/Sarkar_Cao_Jain/model.jpg">model</A>]</P>  <P>&nbsp;</P>  <P><B>Genetic experiments examining the mechanism of replication  of ColEI plasmids, copy number and incompatibility, are described  in your textbook.</B></P>  <P>[<A HREF="../figures/Snyder-Champness/F16-2.jpg">16.2</A>]  <FONT COLOR="#990000" FACE="Arial Rounded MT Bold">14-9 </FONT>[<A  HREF="../figures/Snyder-Champness/F16-3.jpg">16.3</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">14-10 </FONT>[<A  HREF="../figures/Snyder-Champness/F4-10.jpg">4.10</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-14</FONT></P>  <P>&nbsp;</P>  <P><B><U><FONT COLOR="#351a00" FACE="Arial">Replication of R1  plasmids</FONT></U></B></P>  <P>R1 is a large (100 kb) broad host-range plasmid which replicates  unidirectionally from a single origin of replication (<B><I><FONT   COLOR="#00194c">oriR</FONT></I></B>). Plasmid replication --  and hence, copy number -- is also controlled by an antisense  RNA mechanism, though the mechanism is different from that of  ColE1.</P>  <P>R1 requires a special plasmid-encoded protein, <B><FONT COLOR="#004300">RepA</FONT></B>,  to initiate replication. <B><FONT COLOR="#004300">RepA</FONT></B>  binding is required to permit <B><FONT COLOR="#004300">DnaA</FONT></B>  to bind to th origin which in turn allows <B><FONT COLOR="#004300">DnaB</FONT></B>  to bind and start the assembly of a replisome. <B><FONT COLOR="#004300">RepA</FONT></B>  is very unusual because it acts only in <B><I><FONT COLOR="#ce0000">cis</FONT></I></B>.  It is thought that <B><FONT COLOR="#004300">RepA</FONT></B> must  bind to <B><I><FONT COLOR="#00194c">oriR</FONT></I></B> as it  is being synthesized and folded. A DNA bending sequence is located  between <B><I><FONT COLOR="#00194c">repA</FONT></I></B> and <B><I><FONT   COLOR="#00194c">oriR</FONT></I></B> which may facilitate this  direct interaction.</P>  <P>[<A HREF="../figures/Kornberg18-6.jpg">K18.6</A>]</P>  <P>Since <B><FONT COLOR="#004300">RepA</FONT></B> is essential  for replication, control of <B><FONT COLOR="#004300">RepA</FONT></B>  expression will regulate replication and plasmid copy number.  This expression is controlled in two ways: by the <B><I><FONT   COLOR="#00194c">copA</FONT></I></B> antisense RNA; and, by the  <B><FONT COLOR="#004300">CopB</FONT></B> protein.</P>  <P>The organization of these elements is shown below:</P>  <P><CENTER><IMG SRC="../figures/repA.GIF" WIDTH="307" HEIGHT="158"  ALIGN="BOTTOM" NATURALSIZEFLAG="3"></CENTER></P>  <P><CENTER>&nbsp;</CENTER></P>  <P>[<A HREF="../figures/Snyder-Champness/F4-5.jpg">4.5</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-7 ***</FONT></P>  <P>Expression of <B><FONT COLOR="#004300">RepA</FONT></B> occurs  primarily as a result transcription from the <B><I><FONT COLOR="#00194c">repA</FONT></I></B>  promoter. However when the R1 plasmid enters a cell for the first  time, <B><FONT COLOR="#004300">RepA</FONT></B> can also be expressed  due to transcription from the <B><I><FONT COLOR="#00194c">copB</FONT></I></B>  promoter.</P>  <P>Transcription from the <B><I><FONT COLOR="#00194c">repA</FONT></I></B>  promoter is negatively regulated by the <B><FONT COLOR="#004300">CopB</FONT></B>  protein, which is a small -- 10&nbsp;kD -- protein.</P>  <P>Expression from the <B><I><FONT COLOR="#00194c">repA</FONT></I></B>  promoter is regulated by the <B><I><FONT COLOR="#00194c">copA</FONT></I></B>  RNA. This small -- 91 nt -- antisense RNA is complementary to  the <B><I><FONT COLOR="#00194c">repA</FONT></I></B> transcript  and it will hybridize with it. When it does so, it sequesters  the Shine-Dalgarno ribosome binding site sequence so that the  16S rRNA in the ribosome is unable to be positioned correctly  and translation of the mRNA does not take place.</P>  <P>This mechanism of antisense RNA control -- i.e. prevention  of translation -- is found in a number of systems.</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-8.jpg">4.8</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-12 </FONT>[<A  HREF="../figures/Snyder-Champness/F4-9.jpg">4.9</A>] <FONT COLOR="#990000"   FACE="Arial Rounded MT Bold">4-13</FONT></P>  <P><B><FONT COLOR="#999999">[</FONT><A HREF="http://www.emboj.org/cgi/content/full/18/14/4076">partitioning</A><FONT   COLOR="#999999">]</FONT></B></P>  <BLOCKQUOTE>    <P>&nbsp;</P></BLOCKQUOTE>  <P><B><U><FONT COLOR="#351a00" FACE="Arial">Replication of pSC101</FONT></U></B></P>  <P>pSC101 is the best-studied example of a plasmid whose replication  is controlled by iterated RepA binding sites - <B><FONT FACE="Arial">ITERONS</FONT></B>.</P>  <P>RepA is required for replication of these plasmids. It binds  to several iterons in the vicinity of the origin of replication:</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-6.jpg">4.6</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-8 </FONT>[<A  HREF="../figures/Kornberg18-5.jpg">K18.5</A>]</P>  <P>2 sites are inverted repeat sequences that are located near  the <I>repA</I> promoter. RepA binding to these two sites will  inhibit its own expression.</P>  <P>3 other sites are 18 bp tandem repeats. These sites may serve  to sequester RepA so that it is not available for replication  possibly by linking or &quot;handcuffing&quot; plasmids containing  these sequences together so that they cannot replicate.</P>  <P>Some experiments that support this handcuffing model are described  in your textbook.</P>  <P>[<A HREF="../figures/Snyder-Champness/F16-4.jpg">16.4</A>]  <FONT COLOR="#990000" FACE="Arial Rounded MT Bold">4-10 </FONT>[<A  HREF="../figures/Snyder-Champness/F4-7.jpg">4.7</A>] <FONT COLOR="#990000"   FACE="Arial Rounded MT Bold">4-9</FONT></P>  <P>&nbsp;</P>  <BLOCKQUOTE>    <P>&nbsp;</P></BLOCKQUOTE>  <P><B><U><FONT COLOR="#351a00" FACE="Arial">Replication of colIb-P9  plasmids</FONT></U></B></P>  <P>Replication of ColIb-P9 plasmids is also regulated via an  antisense RNA mechanism - but it works in a different way to  that found in ColEI and R1 plasmids.</P>  <P>ColIb-P9 is a large (93 Kb) low copy-number plasmid which  replicates autonomously and whose replication required the <B>RepZ</B>  protein. <B>RepZ</B> is expressed from its own promoter. An antisense  RNA is expressed from the <B><I><FONT COLOR="#000066">inc</FONT></I></B>  promoter.</P>  <P><CENTER><A HREF="../figures/repZ-a.GIF"><IMG SRC="../figures/repZ-a.GIF"  WIDTH="284" HEIGHT="60" ALIGN="BOTTOM" BORDER="0" NATURALSIZEFLAG="3"></A></CENTER></P>  <P>&nbsp;</P>  <P>At first glance, it might appear that this is a classical  antisense RNA control system. However, the Shine-Dalgarno sequence  of the <B><I><FONT COLOR="#000066">repZ</FONT></I></B> mRNA is  not sequestered by the <B><I><FONT COLOR="#000066">inc</FONT></I></B>  RNA. Nevertheless, down mutations in the <B><I><FONT COLOR="#000066">inc</FONT></I></B>  promoter cause a drop in the amount of <B><I><FONT COLOR="#000066">inc</FONT></I></B>  mRNA, an increase in the amount of <B>RepZ</B> protein, but no  change in the amount of <B><I><FONT COLOR="#000066">repZ</FONT></I></B>  mRNA.</P>  <P>Thus, expression of <B>RepZ</B> is controlled at the level  of translation even though it is not by means of blocking the  Shine-Dalgarno ribosome binding site.</P>  <P>Translation termination of <B><I><FONT COLOR="#006633">repY</FONT></I></B>,  a small gene immediately upstream of <B><I><FONT COLOR="#000066">repZ</FONT></I></B>  is critical for this control. <BR>  </P>  <UL>    <LI>If the stop codon for <B><I><FONT COLOR="#006633">repY</FONT></I></B>    is removed (by changing it to a sense codon, for example), or<BR><BR>    <LI>If an amber mutation is created within <B><I><FONT COLOR="#006633">repY</FONT></I></B>    so that it terminates upstream of its normal site  </UL>  <P>then translation of <B><I><FONT COLOR="#000066">repZ</FONT></I></B>  is abolished.</P>  <P>It turns out that the stop codon of <B><I><FONT COLOR="#006633">repY</FONT></I></B>  is just after the start codon of <B><I><FONT COLOR="#000066">repZ</FONT></I></B>.  A ribosome that stalls at this stop codon can easily jump to  the start of <B><I><FONT COLOR="#000066">repZ</FONT></I></B>.  So, as long as <B><I><FONT COLOR="#006633">repY</FONT></I></B>  translation stops in the correct place, <B><I><FONT COLOR="#000066">repZ</FONT></I></B>  translation can start properly.</P>  <P><CENTER><A HREF="../figures/repZ-b.GIF"><IMG SRC="../figures/repZ-b.GIF"  WIDTH="284" HEIGHT="63" ALIGN="BOTTOM" BORDER="0" NATURALSIZEFLAG="3"></A></CENTER></P>  <P>&nbsp;</P>  <P>Translation of <B><I><FONT COLOR="#006633">repY</FONT></I></B>  also permits a second event to occur. The <B><I><FONT COLOR="#000066">repZ</FONT></I></B>  mRNA can form a stem-loop structure. A 4 nt sequence within the  loop of this structure is complementary to a sequence immediately  upstream of the <B><I><FONT COLOR="#000066">repZ</FONT></I></B>  Shine-Dalgarno sequence. These two sequences can anneal to one  another forming a structure called a <B><FONT COLOR="#663300"   FACE="Arial">pseudoknot</FONT></B>. Formation of the pseudoknot  and translation of <B><I><FONT COLOR="#006633">repY</FONT></I></B>  depends on one another because they prevents competing reactions  from taking place. Plasmids which carry mutations in the 4 nt  sequence that destabilize pseudoknot formation are replication-defective.  These mutations can be complemented by compensating mutations  in the 4 nt complementary sequence upstream of the <B><I><FONT   COLOR="#000066">repZ</FONT></I></B> Shine-Dalgarno site.</P>  <P><CENTER><A HREF="../figures/repZ-c.GIF"><IMG SRC="../figures/repZ-c.GIF"  WIDTH="425" HEIGHT="336" ALIGN="BOTTOM" BORDER="0" NATURALSIZEFLAG="3"></A></CENTER></P>  <P><CENTER><TABLE WIDTH="450" BORDER="0" CELLSPACING="2" CELLPADDING="2">    <TR>      <TD WIDTH="100%" BGCOLOR="#bfbfbf">&nbsp;<FONT SIZE="-1" FACE="Arial">Figure        adapted from Fig. 1 of <A HREF="http://www.jbc.org/cgi/content/full/273/19/11815">Asana        &amp; Mizobuchi (1998) J. Biol. Chem. 273: 11815-11825</A>.</FONT></TD>     </TR>  </TABLE></CENTER></P>  <P>&nbsp;</P>  <P>The stem-loop formed by the <B><I><FONT COLOR="#000066">repZ</FONT></I></B>  mRNA can also anneal with the complementary stem-loop formed  by the <B><I><FONT COLOR="#000066">inc</FONT></I></B> RNA in  a manner similar to that between RNAI and RNAII of ColEI plasmids.  If this happens, then the Shine-Dalgarno sequence of <B><I><FONT   COLOR="#006633">repY</FONT></I></B> is sequestered and it cannot  be translated.</P>  <P>In addition, the <B><I><FONT COLOR="#006633">repY</FONT></I></B>  portion of the <B><I><FONT COLOR="#000066">repZ</FONT></I></B>  mRNA can form secondary structures that sequester the <B>RepZ</B>  Shine-Dalgarno sequence. Translation of <B><I><FONT COLOR="#006633">repY</FONT></I></B>  prevents these from forming.</P>  <BLOCKQUOTE>    <P>&nbsp;</P>    <P>&nbsp;</P></BLOCKQUOTE></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">IncompatibilityGroups</FONT></B></P><BLOCKQUOTE>  <P>In the above mechanisms for the control of plasmid copy number,  <B>trans-acting</B> molecules were responsible for the control  of plasmid replication. For example, in the case of ColE1 plasmids,  RNA I acts <B><I><FONT COLOR="#351a00">in trans</FONT></I></B>.  In the case of R1 plasmids, the <B><I><FONT COLOR="#00194c">copA</FONT></I></B>  RNA and <B><FONT COLOR="#004300">CopB</FONT></B> both act <B><I><FONT   COLOR="#351a00">in trans</FONT></I></B>.</P>  <P>If a cell contained two different plasmids, each of which  had the same mechanisms of control, then because control is exercised  through <B><I><FONT COLOR="#351a00">trans</FONT></I></B>-acting  molecules, each plasmid would be able to control the replication  of the other. The inevitable consequence of this is that one  of the two plasmids would eventually be lost from the cell simply  as a result of random partitioning of plasmids into daughter  cells during cell division.</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-10.jpg">4.10</A>]  <FONT COLOR="#990000" FACE="Arial Rounded MT Bold">4-14</FONT></P>  <P>Thus the two plasmids would appear to be incompatible.</P>  <P>Plasmids which have different mechanisms of control would  replicate independently of one another and each would be partitioned  between daughter cells. Thus, both plasmids would be maintained.</P>  <P>This is a basis for classifying plasmids. We can organize  them into <B><FONT COLOR="#ff0000" FACE="Arial">INCOMPATIBILITY  GROUPS</FONT></B>. Two plasmids in the same incompatibilty group  cannot coexist in the same cell. The following table lists some  incompatibility groups and plasmids.</P>  <P><CENTER><TABLE WIDTH="400" HEIGHT="233" BORDER="0" CELLSPACING="2"  CELLPADDING="2">    <TR>      <TH WIDTH="25%" HEIGHT="17" BGCOLOR="#006633"><FONT FACE="Arial">&nbsp;Incompatibilty        group</FONT></TH>       <TH WIDTH="50%" BGCOLOR="#006633">&nbsp;<FONT FACE="Arial">Plasmids</FONT></TH>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;FI</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;<B><FONT COLOR="#ce0000">F</FONT></B>,        R386</TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;FII</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;<B><FONT COLOR="#ce0000">R1</FONT></B></TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;FIII</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;Col B-K99, Col B-K166</TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;FIV</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;R124</TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;I</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;R62, R64, R483 (at least        5 subgroups)</TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;J</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;R391</TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;N</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;R46</TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;O</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;R724</TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;P</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;<B><FONT COLOR="#ce0000">RP4</FONT></B>,        <B><FONT COLOR="#ce0000">RK2</FONT></B></TD>     </TR>    <TR>      <TD BGCOLOR="#339966">        <P><CENTER>&nbsp;Q</CENTER></TD>      <TD BGCOLOR="#99ffcc">&nbsp;<B><FONT COLOR="#ce0000">RSF1010</FONT></B></TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;T</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;R401</TD>     </TR>    <TR>      <TD WIDTH="50%" HEIGHT="17" ALIGN="CENTER" BGCOLOR="#339966">&nbsp;W</TD>       <TD WIDTH="50%" BGCOLOR="#99ffcc">&nbsp;R388, S-a</TD>     </TR>  </TABLE></CENTER></P>  <P>&nbsp;</P>  <P>&nbsp;</P></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">Plasmid Transfer</FONT></B></P><BLOCKQUOTE>  <P>Some plasmids can transfer themselves between bacterial cells  through bacterial conjugation. These plasmids encode all of the  protein functions that are necessary and required to carry out  mobilization and transfer from a host cell to a recipient cell.</P>  <P>Many plasmids are unable to do this. They do not code for  the necessary proteins themselves. However, if they coexist in  a cell with a self-mobilizable plasmid, then they may be able  to be mobilized if they carry the correct sequences or sites  that are recognized by the proteins that are synthesized from  the other plasmid.</P>  <P>[<A HREF="../figures/Snyder-Champness/F5-4.jpg">5.4</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">5-8 ***</FONT></P>  <P>The Col E1 plasmid is a good example of a mobilizable plasmid.  It contains a <B><I><FONT COLOR="#00194c">bom</FONT></I></B>  (basis of mobility) region with a <B><I><FONT COLOR="#00194c">nic</FONT></I></B>  site, which is nicked by proteins coded in the <B><I><FONT COLOR="#00194c">mob</FONT></I></B>  (mobility) region. Upon bacterial mating, the nicked strand forms  a complex with a <B><I><FONT COLOR="#00194c">mob</FONT></I></B>  protein which leads it through a mating channel into a recipient  cell.</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-3.jpg">4.3</A>] <FONT   COLOR="#990000" FACE="Arial Rounded MT Bold">4-5</FONT></P>  <P>&nbsp;</P>  <P><B><U><FONT COLOR="#004300" FACE="Arial">TRIPARENTAL MATING</FONT></U></B></P>  <P>Molecular biologists make use of plasmid mobilization in a  technique called triparental mating.</P>  <P>[<A HREF="../figures/Snyder-Champness/F5-10.jpg">5.10</A>]  <FONT COLOR="#990000" FACE="Arial Rounded MT Bold">5-9 ***</FONT></P>  <P>Because it is difficult to transform may species of bacteria,  conjugation often offers the only practical method of introducing  new DNA -- particularly DNA which has been engineered with some  genetic purpose or test in mind.</P>  <P>However, mating strains are also not commonly used for cloning  DNA. The presence of other plasmids makes it difficult to purify  plasmid DNA routinely.</P>  <P>Triparental mating allows plasmids to be contructed and propagated  in one strain (<B><FONT COLOR="#351a00">A</FONT></B>), and then  mated with a broad-host range plasmid containing strain (<B><FONT   COLOR="#351a00">B</FONT></B>). Once the broad-host range plasmid  has conjugated into strain <B><FONT COLOR="#351a00">A</FONT></B>,  it can now mobilize the engineered plasmid into another strain  (<B><FONT COLOR="#351a00">C</FONT></B>).</P>  <P>&nbsp;</P>  <P>&nbsp;</P></BLOCKQUOTE><P><A NAME="anchor246135"></A><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">Modern Cloning Plasmids</FONT></B></P><BLOCKQUOTE>  <P>Modern cloning plasmids are derived from various combinations  of naturally occurring plasmids or other genetic elements.</P>  <P><B><U><FONT COLOR="#ff0000" FACE="Arial">pBR322</FONT></U></B></P>  <P><A HREF="../figures/NEB_pBR322.jpg">pBR322</A> was the first  popular modern cloning vector. It was constructed by Ray Rodriguez  and Herbert Boyer from three basic parts:</P>  <P><CENTER><IMG SRC="../figures/pBR322.GIF" WIDTH="417" HEIGHT="386"  ALIGN="BOTTOM" NATURALSIZEFLAG="3" BORDER="0"></CENTER></P>  <BLOCKQUOTE>    <P>&nbsp;</P></BLOCKQUOTE>  <UL>    <LI>the tetracycline resistance gene segment was derived from    the plasmid pSC101 (constructed by <B>Stanley Cohen</B>) which    was, in turn, derived from the broad host range conjugative plasmid,    R6.<BR><BR>    <LI>an ampicillin resistance gene segment was obtained from the    transposon, Tn3.<BR><BR>    <LI>the origin of replication segment was obtained from pMB9    (constructed by <B>Mary Betlach</B>), which was, in turn, derived    from the colicin plasmid, ColEI.  </UL>  <BLOCKQUOTE>    <P>&nbsp;</P></BLOCKQUOTE>  <P><B><U><FONT COLOR="#ff0000" FACE="Arial">pUC plasmids</FONT></U></B></P>  <P>The <A HREF="../figures/NEB_pUC19.jpg">pUC</A> series of plasmids  were all constructed by <B>Jeffrey Vieira</B> and <B>Joachim  Messing</B> and all are derived from pBR322. The tetracycline  resistance gene region of pBR322 has been replaced by a gene  segment coding for part of the <FONT FACE="Symbol">b</FONT>-galactosidase  enzyme into which a series of unique restriction sites have been  designed (the multiple cloning site, MCS).</P>  <P>In making this construction, part of the segment containing  the origin of replication was deleted. As a consequence, replication  of the pUC plasmids is not properly controlled so that we have  a higher copy number of the plasmids in the cell.</P>  <P>&nbsp;</P>  <P>&nbsp;</P></BLOCKQUOTE><P><B><FONT COLOR="#330099" SIZE="+1" FACE="Arial">Shuttle Vectors</FONT></B></P><BLOCKQUOTE>  <P>Plasmids with narrow host ranges will only replicate in certain  bacteria. In particular, most of the modern cloning vectors are  derived from the Col E1 plasmid which has a narrow host range.  Thus if you want to do molecular genetics in strains outside  this range, you have to either (<B>a</B>) construct a new set  of vectors suitable for general cloning in your chosen strain,  or (<B>b</B>) modify one of the common vectors so that it will  also replicate in your chosen organism.</P>  <P>The second option is the easier option since all it requires  is the addition of an origin of replication which will function  in your chosen organism. Such vectors, with two different origins  of replication suitable for two different bacteria, are called  shuttle vectors.</P>  <P>[<A HREF="../figures/Snyder-Champness/F4-18.jpg">4.18</A>]  <FONT COLOR="#990000" FACE="Arial Rounded MT Bold">4-20</FONT></P>  <P>&nbsp;</P>  <P>&nbsp;</P></BLOCKQUOTE><P><CENTER><IMG SRC="../figures/grey_ruler.gif" WIDTH="90%" HEIGHT="5"ALIGN="BOTTOM" BORDER="0" NATURALSIZEFLAG="0"></CENTER></P><BLOCKQUOTE>  <P><CENTER><B><FONT COLOR="#333333" SIZE="-1" FACE="Arial">Format  and Original Material &copy; Martin E. Mulligan, 1997-2004</FONT><FONT   FACE="Arial"> <BR><BR>  </FONT></B></CENTER></BLOCKQUOTE></BODY></HTML>