<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
  <meta name="description" content="" />
  <meta name="keywords" content="" />
  <link rel="stylesheet" type="text/css" href="http://dendrome.ucdavis.edu/style.css" media="screen" title="screen" />
	<!--[if IE 6]>
		<link rel="stylesheet"href="http://dendrome.ucdavis.edu/stylesheet-ie6.css" type="text/css" />
	<![endif]-->
  <link rel="stylesheet" type="text/css" href="http://dendrome.ucdavis.edu/print.css" media="print" />
  <title>Dendrome Project</title>
</head>
<body>

<div id="toptabs">
	<table cellspacing=0 cellpadding=0 border=0 align="left" valign="top"><form class="searchBox" method="get" action="/search.php">
		<tr>
			<td width="400"></td>
			<td valign="top"><span class="searchbar">
			<input class="searchBoxInput" style='background:url(/images/GGrad14.png)' type="text" name="query" size="20" maxlength="255" value=""/>&nbsp;<input type="submit" name="sa" value="Search" class="goSearchButton" /></span></td></form>
		</tr>
	</table>
	<div class="toptabcont">
	<a class="activetoptab" href="/">Home</a>
    <span class="hide"> | </span><a class="toptab" href="http://dendrome.ucdavis.edu/sitemap.php">Site Map</a>
    <span class="hide"> | </span><a class="toptab" href="http://dendrome.ucdavis.edu/awstats/awstats.pl">Site Stats</a>
    <span class="hide"> | </span><a class="toptab" href="http://dendrome.ucdavis.edu/about/contact/">Contact Us</a>
    <span class="hide"> | </span><a class="toptab" href="http://dendrome.ucdavis.edu/help/forum/">Discussion Forum</a>

    </div>
</div>


<div id="container">
<div id="logo">
  <h1>
    <a href="/"><img src="http://dendrome.ucdavis.edu/images/logo.jpg" /></a>
  </h1>
</div>
<div id="navitabs">
  <h2 class="hide">Site menu:</h2>
    <a class="navitab" href="http://dendrome.ucdavis.edu/index.php">Welcome</a>
  <span class="hide"> | </span>
      <a class="navitab" href="http://dendrome.ucdavis.edu/NealeLab/index.html">Research</a>
  <span class="hide"> | </span>
      <a class="navitab" href="http://dendrome.ucdavis.edu/treegenes/">TreeGenes</a>
  <span class="hide"> | </span>
      <a class="navitab" href="http://dendrome.ucdavis.edu/resources/" href="http://dendrome.ucdavis.edu/resources/">Resources</a>
  <span class="hide"> | </span>
      <a class="navitab" href="http://dendrome.ucdavis.edu/news_events/">Events</a>
  <span class="hide"> | </span>
    <a class="navitab" href="http://dendrome.ucdavis.edu/news_events/?section=jobs">Employment</a>
  <span class="hide"> | </span>
    <a class="navitab" href="http://dendrome.ucdavis.edu/links_directory/">Links</a>
  <span class="hide"> | </span>
    <a class="navitab" href="http://dendrome.ucdavis.edu/help/">Help</a>
</div>

<div id="desc">
  <h2>Welcome to the Dendrome Project!</h2>
  </p>

</div>
	<div id="leftNavBar">
       
<h3><img src="http://dendrome.ucdavis.edu/images/05.jpg" width="30" height="30" alt="Icon 05" />&nbsp;Resources</h3><p>

     <p>
            <a href="http://dendrome.ucdavis.edu/resources/scripts/" class="leftNavlink">Scripts</a>
	    <a href="http://dendrome.ucdavis.edu//resources/blast.php" class="leftNavlink">BLAST</a>
			<a href="http://dendrome.ucdavis.edu/resources/under_dev/" class="leftNavlink">FASTA</a>
			<a href="http://dendrome.ucdavis.edu//resources/display.php" class="leftNavlink">Tools</a>
            <a href="http://dendrome.ucdavis.edu/resources/databases/" class="leftNavlink">Databases</a>
			<a href="http://dendrome.ucdavis.edu/resources/public_est/" class="leftNavlink">Public ESTs</a>
            <a href="http://dendrome.ucdavis.edu/treegenes/" class="leftNavlink">TreeGenes</a>
	    <a href="http://www.plantontology.org/" class="leftNavlink">Plant Ontology</a>
        </p>


    <h3><img src="http://dendrome.ucdavis.edu/images/03.jpg" width="30" height="30" alt="Icon 03" />&nbsp;Updates</h3>

	<p>
        <a class="sidelink" href="http://dendrome.ucdavis.edu/TreeGenes/">New EST analysis and submission pipeline available for use!</a><span class="hide"> | </span>
        <a class="sidelink" href="http://dendrome.ucdavis.edu/NealeLab/index.html">Plant Gene Ontology database ported into mysql</a>
        <span class="hide"> | </span><p>


     <h3><img src="http://dendrome.ucdavis.edu/images/02.jpg" width="30" height="30" alt="Icon 02" />&nbsp;Links</h3><p>
        <a class="sidelink" href="http://www.pinegenome.org">Conifer Genome Network</a><span class="hide"> | </span>
        <a class="sidelink" href="http://www.pinegenome.org/cgp">Conifer Genome Project</a><span class="hide"> | </span>
        <a class="sidelink" href="http://dendrome.ucdavis.edu/TreeGenes/">TreeGenes Database</a><span class="hide"> | </span>
	<a class="sidelink" href="http://dendrome.ucdavis.edu/wiki/">Dendrome Wiki</a><span class="hide"> | </span>
        <a class="sidelink" href="http://dendrome.ucdavis.edu/NealeLab/index.html">Neale Lab</a>
        <span class="hide"> | </span>
        
	</div>

  <link rel='stylesheet' type='text/css' href='style.css' media='screen' title='screen'/>
<div id='main'>
<br><br><h4>Tool Listing</h4><TABLE ID='margins' cellpadding=0 cellspacing=0><TR><TH style='width:100px;'>Tool</TH><TH>Version</TH><TH>Description</TH><TH>Documents</TH></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=1'>Local BLAST</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>TreeGenes has installed and maintains a local version of the WWW-BLAST server called Local BLAST with databases which are updated weekly. Having a local server offers improved performance and security and allows us to create our own BLAST DBs.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=2'>NCBI C Toolkit</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>The toolkit is used internally at NCBI to process and analyze data from a variety of sources to build and maintain the unified databases and also serves as the components for the end-user applications NCBI distributes.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=3'>R</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.1.0&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>R is a language and environment for statistical computing and graphics.</TD>
	    <TD style='vertical-align:top;'>Admin<BR>Data_Import_Export<BR>Writing_Extensions<BR>Introduction<BR>Internals<BR>Language_Definition<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=4'>bioperl</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.2.3&nbsp;(2006)</TD>
	    <TD style='vertical-align:top;'>Bioperl is a collection of perl libraries that ease common bioinformatic needs and integrate with common file formats  </TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=5'>bl2seq</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.2.12&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>	bl2seq is part of NCBI&#39;s BLAST package and allows alignment between 2 sequences  </TD>
	    <TD style='vertical-align:top;'>ManPage<BR>BLAST Readme<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=6'>blastall</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.2.12&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>	blastall is part of NCBI&#39;s BLAST package and uses all BLAST implementations: blastp, blastn, blastx, tblastx, and tblastn</TD>
	    <TD style='vertical-align:top;'>BLAST Readme<BR>ManPage<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=7'>blastclust</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.2.12&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>blastclust clusters both protein and DNA sequences based on resulting BLAST alignments  </TD>
	    <TD style='vertical-align:top;'>BLAST Readme<BR>Helpfile<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=8'>blastpgp</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.2.12&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>	blastpgp is part of NCBI&#39;s package and is intended for gapped blastp searches  </TD>
	    <TD style='vertical-align:top;'>BLAST Readme<BR>Helpfile<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=9'>boxshade</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;3.2.1&nbsp;</TD>
	    <TD style='vertical-align:top;'>BOXSHADE provides displya options for multiple alignments from other tools  </TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=10'>cap3</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;(1999)</TD>
	    <TD style='vertical-align:top;'>CAP 3 is a contig assembly program that can take in FASTA formatted sequences and quality scores  </TD>
	    <TD style='vertical-align:top;'>Readme<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=11'>clustalw</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.81&nbsp;(2000)</TD>
	    <TD style='vertical-align:top;'>ClustalW is a multiple alignment utility for DNA and protein sequences </TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=12'>clustalx</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.83&nbsp;</TD>
	    <TD style='vertical-align:top;'>windows interface for clustalW</TD>
	    <TD style='vertical-align:top;'>User Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=13'>consed</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;15.0&nbsp;(2001)</TD>
	    <TD style='vertical-align:top;'>Consed is a graphical interface for editing phrap output.  </TD>
	    <TD style='vertical-align:top;'>Readme<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=14'>emboss</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.10.0&nbsp;(2006)</TD>
	    <TD style='vertical-align:top;'>EMBOSS is a free Open Source software analysis package specially developed for the needs of the molecular biology user community. EMBOSS integrates a range of currently available packages and tools for sequence analysis into a seamless whole.</TD>
	    <TD style='vertical-align:top;'>About<BR>Sequence Formats<BR>Command Line<BR>Stabdards<BR>User Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=15'>ensembl</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>Ensembl is a joint project between EMBL - EBI and the Sanger Institute to develop a software system which produces and maintains automatic annotation on eukaryotic genomes.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=16'>epcr</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.3.7&nbsp;(2007)</TD>
	    <TD style='vertical-align:top;'>Electronic PCR (e-PCR) is a program for finding STSs in DNA sequences</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=17'>fbat</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.73&nbsp;(2006)</TD>
	    <TD style='vertical-align:top;'>FBAT implements a broad class of Family Based Association Tests, adjusted for population admixture.</TD>
	    <TD style='vertical-align:top;'>Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=18'>formatdb</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.2.12&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>Formatdb must be used in order to format protein or nucleotide sourcedatabases before these databases can be searched by blastall, blastpgpor MegaBLAST.</TD>
	    <TD style='vertical-align:top;'>BLAST Readme<BR>Readme<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=19'>fpc</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;8.5.3&nbsp;(2006)</TD>
	    <TD style='vertical-align:top;'>FPC (fingerprinted contigs) is an interactive program for building contigs from fingerprinted clones, where the fingerprint for a clone is a set of restriction fragments.</TD>
	    <TD style='vertical-align:top;'>Tutorial<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=20'>gap4</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.6.0&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>Gap4 is a Genome Assembly Program. The program contains all the tools that would be expected from an assembly program plus many unique features and a very easily used interface. It is distributed as part of the Staden package.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=21'>genscan</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>GENSCAN is a general-purpose gene identification program whichanalyzes genomic DNA sequences from a variety of organisms including human, other vertebrates, invertebrates and plants.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=22'>gold</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.1.0&nbsp;(2001)</TD>
	    <TD style='vertical-align:top;'>GOLD (graphical overview of linkage disequilibrium) is a tool for the graphical summary of linkage disequilibrium in genetic data. The summary is well suited to the analysis of dense genetic maps, where contingency tables are cumbersome to interpret.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=23'>haplo</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;3.3.6&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>Haplo uses the EM algorithm to estimate the frequencies of multi-site haplotypes </TD>
	    <TD style='vertical-align:top;'>Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=24'>hmmer</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.3.2&nbsp;(2003)</TD>
	    <TD style='vertical-align:top;'>find additional homologues of sequence family</TD>
	    <TD style='vertical-align:top;'>Readme<BR>UserGuide(PDF)<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=25'>hmmgene</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>HMMgene is a program for prediction of genes in anonymous DNA.</TD>
	    <TD style='vertical-align:top;'>Usage<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=26'>interproscan</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;4.3.1&nbsp;(2007)</TD>
	    <TD style='vertical-align:top;'>protein signature recognition</TD>
	    <TD style='vertical-align:top;'>Readme<BR>User Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=27'>joinmap</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;4.0&nbsp;(2001)</TD>
	    <TD style='vertical-align:top;'>JoinMap provides high quality tools that allow detailed study of the experimental data and the generation of publication-ready map charts.</TD>
	    <TD style='vertical-align:top;'>Manual (PDF)<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=28'>lagan</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.0&nbsp;(2003)</TD>
	    <TD style='vertical-align:top;'>The Lagan Tookit is a set of alignment programs for comparative genomics.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=29'>mapQTL</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;5.0&nbsp;</TD>
	    <TD style='vertical-align:top;'>MapQTL is computer software for the calculation of positions of quantitative trait loci (QTLs) on genetic maps.</TD>
	    <TD style='vertical-align:top;'>Manual (PDF)<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=30'>maskeraid</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>OBSOLETE : MaskerAid is a drop-in accelerator that increases the speed of RepeatMasker about 30-fold while maintaining sensitivity.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=31'>megablast</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>nucleotide sequence alignment search</TD>
	    <TD style='vertical-align:top;'>BLAST Readme<BR>Helpfile<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=32'>multiclustal</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.1&nbsp;(2000)</TD>
	    <TD style='vertical-align:top;'>MULTICLUSTAL is a Perl script designed to automate the process of alignment parameter choice for Clustal W with the goal of generating high quality multiple sequence alignments.</TD>
	    <TD style='vertical-align:top;'>Readme<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=33'>njplot</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;(2007)</TD>
	    <TD style='vertical-align:top;'>Phylogenetic tree drawing tool</TD>
	    <TD style='vertical-align:top;'>About<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=34'>phrap</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;0.990329&nbsp;</TD>
	    <TD style='vertical-align:top;'>Phrap (&quot;phragment assembly program&quot;, or &quot;phil&#39;s revised assembly program&quot;) is aprogram for assembling shotgun DNA sequence data.</TD>
	    <TD style='vertical-align:top;'>About<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=35'>phred</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;0.020425.c&nbsp;</TD>
	    <TD style='vertical-align:top;'>Phred reads DNA sequencer trace data, calls bases, assigns quality values to the bases, and writes the base calls and quality values to output files.</TD>
	    <TD style='vertical-align:top;'>Documentation<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=36'>phylip</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;3.66&nbsp;</TD>
	    <TD style='vertical-align:top;'>A package of tools: PHYLIP (the PHYLogeny Inference Package) is a package of programs for inferring phylogenies (evolutionary trees).</TD>
	    <TD style='vertical-align:top;'>User Manual<BR>Primer (PDF)<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=37'>phyml</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;(2003)</TD>
	    <TD style='vertical-align:top;'>Phyml is a program that constructs phylogenetic trees from sequence alignments using the maximum likelihood method.</TD>
	    <TD style='vertical-align:top;'>Web Manual<BR>Command Line Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=38'>plato</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.11&nbsp;</TD>
	    <TD style='vertical-align:top;'>detect recombination and selection</TD>
	    <TD style='vertical-align:top;'>Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=62'>polybayes</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;3.0&nbsp;(2001)</TD>
	    <TD style='vertical-align:top;'>PolyBayes is a computer program for the automated analysis of single-nucleotide polymorphism (SNP) discovery in redundant DNA sequences.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=39'>polyphred</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;5.04&nbsp;(2006)</TD>
	    <TD style='vertical-align:top;'>PolyPhred identifies substitution SNPs as potential heterozygotes by comparing traces in a sequence assembly.</TD>
	    <TD style='vertical-align:top;'>Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=61'>probconsRNA</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.12&nbsp;</TD>
	    <TD style='vertical-align:top;'></TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=40'>puzzleboot</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.03&nbsp;</TD>
	    <TD style='vertical-align:top;'>Puzzleboot is a UNIX shell script facilitating bootstrap analysis using TREE-PUZZLE and PHYLIP. It enhances TREE-PUZZLE by allowing one to analyse multiple datasets, and an be used for both protein and DNA distance bootstrap analysis.</TD>
	    <TD style='vertical-align:top;'>Readme<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=41'>qtdt</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.5.1&nbsp;(2006)</TD>
	    <TD style='vertical-align:top;'>Test linkage disequilibrium</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=42'>readseq</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.1.22&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>Reads and writes nucleic/protein sequences in various formats. Data files may have multiple sequences.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=43'>repeatmasker</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;3.1.8&nbsp;(2007)</TD>
	    <TD style='vertical-align:top;'>Screens DNA sequences for interspersed repeats and low complexity DNA sequences.</TD>
	    <TD style='vertical-align:top;'>Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=44'>rnabob</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.1&nbsp;</TD>
	    <TD style='vertical-align:top;'>The program &quot;rnabob&quot; searches for RNA motifs in sequence databases.Rather than searching simply for primary sequence matches, rnaboballows one to search for secondary and tertiary structural motifs aswell.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=45'>rnamot</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.1&nbsp;</TD>
	    <TD style='vertical-align:top;'>The RNAMOT program finds primary or secondary structure motifs in nucleic acid sequences.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=46'>rpsblast</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>RPS-BLAST (Reverse PSI-BLAST) searches a query sequence against a database of profiles, or score matrices, prepared by copymat, producing BLAST-like output.</TD>
	    <TD style='vertical-align:top;'>BLAST Readme<BR>Helpfile<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=47'>seaview</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;(2005)</TD>
	    <TD style='vertical-align:top;'>A graphical multiple sequence alignment editor.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=48'>sequin</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;7.35&nbsp;</TD>
	    <TD style='vertical-align:top;'>Sequin is a program designed to aid in the submission of sequences to the GenBank, EMBL, and DDBJ sequence databases.</TD>
	    <TD style='vertical-align:top;'>Help<BR>QuickGuide<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=49'>sim4</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>Sim4 is a similarity-based tool for aligning an expressed DNA sequence (EST, cDNA, mRNA) with a genomic sequence for the gene.</TD>
	    <TD style='vertical-align:top;'>Helpfile<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=50'>sputnik</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;(2003)</TD>
	    <TD style='vertical-align:top;'>Sputnik searches DNA sequence files in Fasta format for microsatellite repeats.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=51'>staden</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.6&nbsp;</TD>
	    <TD style='vertical-align:top;'>The Staden Package is a suite of tools for: sequence assembly, analysis, and mutation detection.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=52'>structure</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.2&nbsp;(2007)</TD>
	    <TD style='vertical-align:top;'>Structure implements a model-based clustering method for inferring population structure using genotype data consisting of unlinked markers. The methods used are introduced in a paper by Pritchard, Stephens and Donnelly (2000)</TD>
	    <TD style='vertical-align:top;'>Manual (PDF)<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=53'>tree-puzzle</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;5.2&nbsp;(2004)</TD>
	    <TD style='vertical-align:top;'>Tree-puzzle is a program that constructs phylogenetic trees from sequence alignments using the maximum likelihood method.</TD>
	    <TD style='vertical-align:top;'>Manual (PDF)<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=54'>twinscan</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;4.0&nbsp;(2006)</TD>
	    <TD style='vertical-align:top;'>predicting gene structure</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=55'>vista</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>VISTA is an application that allows the visualization of long sequence alignments with annotation information.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=56'>weighbor</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;1.2&nbsp;</TD>
	    <TD style='vertical-align:top;'>Weighbor is a tool for building phylogenetic trees from distance matrices. It employs a weighted version of the neighbour-joining method in which longer distances in the matrix are given less weight.</TD>
	    <TD style='vertical-align:top;'>Helpfile<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=57'>wise2</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.1.16&nbsp;(2002)</TD>
	    <TD style='vertical-align:top;'>Wise2 is a package focused on comparisons of DNA sequence and protein sequence.</TD>
	    <TD style='vertical-align:top;'>Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=58'>wublast</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>v.&nbsp;2.0&nbsp;(2006)</TD>
	    <TD style='vertical-align:top;'>Washington University BLAST - performs sensitive, selective and rapid similarity searches of protein and nucleotide sequence databases.</TD>
	    <TD style='vertical-align:top;'>Manual<BR></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=59'>xgrail</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;</TD>
	    <TD style='vertical-align:top;'>GRAIL is a suite of tools designed to provide analysis and putative annotationof DNA sequences both interactively and through the use of automated computation.</TD>
	    <TD style='vertical-align:top;'></TD></TR><TR>
	    <TD class='right' style='font-size:1.0em;vertical-align:top;text-align:left;font-weight:bold;'><A HREF='display.php?prog=60'>zt</A></TD>
	    <TD style='font-size:1.0em;vertical-align:top;text-align:center'>?&nbsp;(2001)</TD>
	    <TD style='vertical-align:top;'>ZT is a tool which performs simple and partial Mantel tests. The partial Mantel test investigates the correlation between two matrices while controlling the effect of a third one, and thus tries to remove spurious correlations.</TD>
	    <TD style='vertical-align:top;'></TD></TR></TABLE></p></div><div id="sidebar">
    <h3>
        <img src="http://dendrome.ucdavis.edu/images/01.jpg" width="30" height="30" alt="Icon 01" />&nbsp;Events
    </h3>

     <p>
        <a class="sidelink" href="http://www.agbt.org/flash/about.html">Advances in Genome Biology and Technology<br /><br/>
        <font style="color: #3C746B;">Marco Island, Florida<br/>February 6-9, 2008</font></a><span class="hide"> | </span>
    </p>
        <p>
        <a class="sidelink" href="http://www.geocities.com/sylva.monde/first.htm?20075">SYLVA-WORLD for Development and the protection of forests and the Environment<br /><br/>
        <font style="color: #3C746B;">Casablanca, Morocco<br/>March 20-21, 2008</font></a><span class="hide"> | </span>
    </p>
        <p>
        <a class="sidelink" href="http://iufro.kfrigene.net/"> The Breeding and Genetic Resources of Five-Needle Pines<br /><br/>
        <font style="color: #3C746B;">Yangyang, Korea<br/>September 22-26, 2008</font></a><span class="hide"> | </span>
    </p>
    
<p><img class="photo" src="http://dendrome.ucdavis.edu/images/redwood.jpg" height="100" width="130" alt="" /></p>
    <h3><img src="http://dendrome.ucdavis.edu/images/04.jpg" width="30" height="30" alt="Icon 03" />&nbsp;People</h3>
	
	<p>
	        <a class="sidelink" href="#">David Neale<br>Principal Investigator</a>
	        <a class="sidelink" href="#">Jennifer Lee<br>Bioinformaticist</a>
	        <a class="sidelink" href="#">Jill Wegrzyn<br>Bioinformaticist</a>
	        <a class="sidelink" href="#">Brandon Tearse<br>Developer</a>
	</p>




</div>


    <div id="footer">
      <a href="http://www.plantsciences.ucdavis.edu/plantsciences/">
          <img src="http://dendrome.ucdavis.edu/images/ps_logo.jpg" width="133" height="60" border="0" alt="University of California Davis" />
      </a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      <a href="http://www.fs.fed.us/">
          <img src="http://dendrome.ucdavis.edu/images/ifg.gif" width="59" height="60" border="0" alt="US Forestry Service" />
      </a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
      <a href="http://www.fs.fed.us/">
          <img src="http://dendrome.ucdavis.edu/images/fs.gif" width="49" height="55" border="0" alt="US Forestry Service" />
      </a>
    </div>
</body>
</html>
