<html>

<head>
 <meta name="author" content="Books">

<meta name="keywords" content="national center for biotechnology information, ncbi, national library of medicine, nlm, national institutes of health, nih, pubmed, bookshelf, free access, book, online, textbook, genomics, proteomics, microarray, chromosome painting, chromatin, vesicle, golgi, cell signaling">

<meta name="description" content="Molecular Cell Biology concentrates on the macromolecules and reactions studied by biochemists, the processes described by cell biologists, and the gene control pathways identified by molecular biologists and geneticists. Lodish, Harvey; Berk, Arnold; Zipursky, S. Lawrence; Matsudaira, Paul; Baltimore, David; Darnell, James E. New York: W. H. Freeman & Co.. c1999.">

 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
 <!-- Google indexing --> <meta name="robots" content="noindex,nofollow,NOARCHIVE"> 
 <title>DNA Cloning with Plasmid Vectors</title>
 <link rel="stylesheet" href="/books/bookres.fcgi/mcb/books.css">
 <!-- link type="text/css" rel="stylesheet" href="/corehtml/ncbi_test.css" -->
 <LINK REL="SHORTCUT ICON" HREF="favicon.ico">
</head>
<body bgcolor="#FFFFFF" topmargin=0 leftmargin=0 marginwidth=0 marginheight=0>
 <a name="top"></a>

<script type="text/javascript" language="JavaScript1.2">
var PopUpMenu2_GlobalConfig = [  ["UseThisGlobalConfig","yes"] ];
</script>
<!-- script language="JavaScript1.2" src="http://www.ncbi.nlm.nih.gov/coreweb/javascript/popupmenu2/popupmenu2_5loader.js"></script -->
<script language="JavaScript1.2" src="http://www.ncbi.nlm.nih.gov/coreweb/javascript/popupmenu2/popupmenu2_7loader.js"></script>

 <!-- logo -->
<table width="100%" border="0" cellpadding="0" cellspacing="0">
 <tr>
  <td width="100%"  align="left" valign="center" bgcolor="#638cb5">
  <img src="/books/bookres.fcgi/mcb/logo-mcb.gif" USEMAP="#map" ALIGN="left" BORDER="0"><MAP NAME="map">
  <AREA SHAPE="rect" coords="0,0 599,99" HREF="http://www.whfreeman.com/college/book.asp?disc=BIO&id_product=1124001436&@id_course=1058000157&disc_name=Biology"></MAP>
  <!-- <span style="font-size:large;color:white">Molecular Cell Biology</span> -->
</td> 
 </tr> 
</table>
   



<table width="100%" border="0" cellpadding="0" cellspacing="0">
<tr>
<td bgcolor="#9cbd9c">&nbsp;&nbsp;&nbsp;
<a class="toolbar-link" href="/books/bv.fcgi?call=bv.View..ShowTOC&rid=mcb.TOC">Short Contents</a> | 
<a class="toolbar-link" href="/books/bv.fcgi?call=bv.View..ShowTOC&rid=mcb.TOC&depth=10">Full Contents</a></td>
<td bgcolor="#9cbd9c">
<a class="toolbar-link" href="/entrez/query.fcgi?db=Books">Other books</a> @ 
<a class="toolbar-link" href="http://www.ncbi.nlm.nih.gov">NCBI</a></td>
</tr>
</tr>
</table>

<br>
        <table width="100%">
         <tr>       
         <td width="2%" valign="top">&nbsp;</td>              
          <td width="73%" valign="top"><a name="mcb.section.1582"></a><a class="toc-sidebar-passive-0" href="javascript:BVShow('mcb')">Molecular Cell Biology</a> <img src="/books/bookres.fcgi/system_default/clickpar.gif" border="0"> <span class="toc-sidebar-passive-1"> 7. </span><a class="toc-sidebar-passive-1" href="javascript:BVShow('mcb.chapter.1556')">Recombinant DNA and Genomics</a><br><br><span class="section-header">7.1. </span> <span class="section-header">DNA Cloning with Plasmid Vectors</span><br><p>The essence of cell chemistry is to isolate a particular cellular component and then analyze its chemical structure and activity. In the case of DNA, this is feasible for relatively short molecules such as the genomes of small viruses. But genomes of even the simplest cells are much too large to directly analyze in detail at the molecular level. The problem is compounded for complex organisms. The human genome, for example, contains about 6 × 10<sup>9</sup> base pairs (bp) in the 23 pairs of chromosomes. Cleavage of human DNA with restriction enzymes that produce about one cut for every 3000 base pairs yields some 2 million fragments, far too many to separate from each other directly. This obstacle to obtaining pure DNA samples from large genomes has been overcome by recombinant DNA technology. With these methods virtually any gene can be purified, its sequence determined, and the functional regions of the sequence explored by altering it in planned ways and reintroducing the DNA into cells and into whole organisms.</p><p>The essence of recombinant DNA technology is the prep-aration of large numbers of identical DNA molecules. A DNA fragment of interest is linked through standard 3′ → 5′ phosphodiester bonds to a <a class="text-ref" href="/books/bv.fcgi?rid=mcb.glossdiv.7894#7859">vector</a> DNA molecule, which can replicate when introduced into a host cell. When a single recombinant DNA molecule, composed of a vector plus an inserted DNA fragment, is introduced into a host cell, the inserted DNA is reproduced along with the vector, producing large numbers of recombinant DNA molecules that include the fragment of DNA originally linked to the vector. Two types of vectors are most commonly used: <i>E. coli</i> <a class="text-ref" href="/books/bv.fcgi?rid=mcb.glossdiv.7888#7728">plasmid</a> vectors and bacteriophage λ vectors. Plasmid vectors replicate along with their host cells, while λ vectors replicate as lytic viruses, killing the host cell and packaging the DNA into virions (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.chapter.1350">Chapter 6</a>). In this section, the general procedure for cloning DNA fragments in <i>E. coli</i> plasmids is described.</p><a name="1583"></a><span class="subsect1-header">Plasmids Are Extrachromosomal Self-Replicating DNA Molecules</span><br><p>Plasmids are circular, double-stranded DNA (dsDNA) molecules that are separate from a cell’s chromosomal DNA. These extrachromosomal DNAs, which occur naturally in bacteria, yeast, and some higher eukaryotic cells, exist in a parasitic or symbiotic relationship with their host cell. Plasmids range in size from a few thousand base pairs to more than 100 kilobases (kb). Like the host-cell chromosomal DNA, plasmid DNA is duplicated before every cell division. During cell division, at least one copy of the plasmid DNA is segregated to each daughter cell, assuring continued propagation of the plasmid through successive generations of the host cell.</p><p><img src="/books/bookres.fcgi/mcb/med.gif" align="left" valign="top" hspace="3">Many naturally occurring plasmids contain genes that provide some benefit to the host cell, fulfilling the plasmid’s portion of the symbiotic relationship. For example, some bacterial plasmids encode enzymes that inactivate antibiotics. Such drug-resistance plasmids have become a major problem in the treatment of a number of common bacterial pathogens. As antibiotic use became widespread, plasmids containing several drug-resistance genes evolved, making their host cells resistant to a variety of different antibiotics simultaneously. Many of these plasmids also contain “transfer genes” encoding proteins that can form a macromolecular tube, or<i> pilus,</i> through which a copy of the plasmid can be transferred to other host cells of the same or related bacterial species. Such transfer can result in the rapid spread of drug-resistance plasmids, expanding the number of antibiotic-resistant bacteria in an environment such as a hospital. Coping with the spread of drug-resistance plasmids is an important challenge for modern medicine.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1584"></a><span class="subsect1-header"><b><i>E. Coli</i></b> Plasmids Can Be Engineered for Use as Cloning Vectors</span><br><p>The plasmids most commonly used in recombinant DNA technology replicate in <i> E. coli.</i>Generally, these plasmids have been engineered to optimize their use as vectors in DNA cloning. For instance, to simplify working with plasmids, their length is reduced; many plasmid vectors are only <img border="0" src="/books/bookres.fcgi/system_default/ap.gif" alt="[approximate]">3kb in length, which is much shorter than in naturally occurring <i> E. coli</i> plasmids. (The circumference of plasmids usually is referred to as their “length,” even though plasmids are almost always circular DNA molecules.) Most plasmid vectors contain little more than the essential nucleotide sequences required for their use in DNA cloning: a replication origin, a drug-resistance gene, and a region in which exogenous DNA fragments can be inserted (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1585">Figure 7-1</a>).</p><a name="1586"></a><span class="subsect2-header">Plasmid DNA Replication</span><br><p>The <a class="text-ref" href="/books/bv.fcgi?rid=mcb.glossdiv.7890#7773">replication origin</a> (ORI) is a specific DNA sequence of 50 <img border="0" src="/books/bookres.fcgi/system_default/ndash.gif" alt=" --"> 100 base pairs that must be present in a plasmid for it to replicate. Host-cell enzymes bind to ORI, initiating replication of the circular plasmid. Once DNA replication is initiated at ORI, it continues around the circular plasmid regardless of its nucleotide sequence (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1587">Figure 7-2</a>). Thus any DNA sequence inserted into such a plasmid is replicated along with the rest of the plasmid DNA; this property is the basis of molecular DNA cloning.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1588"></a><span class="subsect2-header">Selection of Transformed Cells</span><br><p>In 1944, O. T. Avery, C. M. Macleod, and M. McCarty first demonstrated gene transfer with isolated DNA obtained from <i>Streptococcus pneumoniae.</i> This process involved the genetic alteration of a bacterial cell by the uptake of DNA isolated from a genetically different bacterium and its recombination with the host-cell genome. Their experiments provided the first evidence that DNA is the genetic material. Later studies showed that such genetic alteration of a recipient cell can result from the uptake of exogenous extrachromosomal DNA (e.g., plasmids) that does not integrate into the host-cell chromosome. The term <a class="text-ref" href="/books/bv.fcgi?rid=mcb.glossdiv.7892#7842">transformation</a> is used to denote the genetic alteration of a cell caused by the uptake and expression of foreign DNA regardless of the mechanism involved. (Note that <i>transformation</i> has a second meaning defined in <a class="text-ref" href="/books/bv.fcgi?rid=mcb.chapter.1350">Chapter 6</a>, namely, the process by which normal cells with a finite life span in culture are converted into continuously growing cells similar to cancer cells.)</p><p>The phenomenon of transformation permits plasmid vectors to be introduced into and expressed by <i>E. coli</i> cells. In order to be useful in DNA cloning, however, a plasmid vector must contain a <i>selectable gene,</i> most commonly a drug-resistance gene encoding an enzyme that inactivates a specific antibiotic. As we’ve seen, the ampicillin-resistance gene (<i>amp</i><sup>r</sup>) encodes β-lactamase, which inactivates the antibiotic ampicillin. After plasmid vectors are incubated with <i>E. coli,</i> those cells that take up the plasmid can be easily selected from the larger number of cells that do not by growing them in an ampicillin-containing medium. The ability to select transformed cells is critical to DNA cloning by plasmid vector technology because the transformation of <i>E. coli</i> with isolated plasmid DNA is inefficient.</p><p>Normal <i>E. coli</i> cells cannot take up plasmid DNA from the medium. Exposure of cells to high concentrations of certain divalent cations, however, makes a small fraction of cells permeable to foreign DNA by a mechanism that is not understood. In a typical procedure, <i>E. coli</i> cells are treated with CaCl<sub>2</sub> and mixed with plasmid vectors; commonly, only 1 cell in about 10,000 or more cells becomes competent to take up the foreign DNA. Each competent cell incorporates a <i>single</i> plasmid DNA molecule, which carries an antibiotic-resistance gene. When the treated cells are plated on a petri dish of nutrient agar containing the antibiotic, only the rare transformed cells containing the antibiotic-resistance gene on the plasmid vector will survive. All the plasmids in such a colony of selected transformed cells are descended from the single plasmid taken up by the cell that established the colony.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1589"></a><span class="subsect1-header">Plasmid Cloning Permits Isolation of DNA Fragments from Complex Mixtures</span><br><p>A DNA fragment of a few base pairs up to ≈20 kb can be inserted into a plasmid vector. When such a recombinant plasmid transforms an <i>E. coli</i> cell, all the antibiotic-resistant progeny cells that arise from the initial transformed cell will contain plasmids with the same inserted sequence of DNA (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1590">Figure 7-3</a>). The inserted DNA is replicated along with the rest of the plasmid DNA and segregates to daughter cells as the colony grows. In this way, the initial fragment of DNA is replicated in the colony of cells into a large number of identical copies. Since all the cells in a colony arise from a single transformed parental cell, they constitute a clone of cells. The initial fragment of DNA inserted into the parental plasmid is referred to as <i>cloned DNA,</i> since it can be isolated from the clone of cells.</p><p>DNA cloning allows fragments of DNA with a particular nucleotide sequence to be isolated from a complex mixture of fragments with many different sequences. As a simple example, assume you have a solution containing four different types of DNA fragments, each with a unique sequence (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1592">Figure 7-4</a>). Each fragment type is individually inserted into a plasmid vector. The resulting mixture of recombinant plasmids is incubated with <i>E. coli</i> cells under conditions that facilitate transformation; the cells then are cultured on antibiotic selective plates. Since each colony that develops arose from a single cell that took up a single plasmid, all the cells in a colony harbor the identical type of plasmid characterized by the DNA fragment inserted into it. As a result, copies of the DNA fragments in the initial mixture are isolated from one another in the separate bacterial colonies. DNA cloning thus is a powerful, yet simple method for purifying a particular DNA fragment from a complex mixture of fragments and producing large numbers of the fragment of interest.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1594"></a><span class="subsect1-header">Restriction Enzymes Cut DNA Molecules at Specific Sequences</span><br><p>To clone specific DNA fragments in a plasmid vector, as just described, or in other vectors discussed in later sections, the fragments must be produced and then inserted into the vector DNA. As noted in the introduction, restriction enzymes and DNA ligases are utilized to produce such recombinant DNA molecules.</p><p>Restriction enzymes are bacterial enzymes that recognize specific 4- to 8-bp sequences, called <i>restriction sites,</i> and then cleave both DNA strands at this site. Since these enzymes cleave DNA within the molecule, they are also called <i>restriction endonucleases</i> to distinguish them from exonucleases<i>,</i> which digest nucleic acids from an end. Many restriction sites, like the <i>Eco</i>RI site shown in <a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1595">Figure 7-5a</a>, are short inverted repeat sequences; that is, the restriction-site sequence is the same on each DNA strand when read in the 5′ → 3′ direction. Because the DNA isolated from an individual organism has a specific sequence, restriction enzymes cut the DNA into a reproducible set of fragments called <a class="text-ref" href="/books/bv.fcgi?rid=mcb.glossdiv.7890#7778">restriction fragments</a> (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1596">Figure 7-6</a>).</p><p>The word <i>restriction</i> in the name of these enzymes refers to their function in the bacteria from which they are isolated: a restriction endonuclease destroys (restricts) incoming foreign DNA (e.g., bacteriophage DNA or DNA taken up during transformation) by cleaving it at all the restriction sites in the DNA. Another enzyme, called a <i>modification enzyme,</i> protects a bacterium’s own DNA from cleavage by modifying it at or near each potential cleavage site. The modification enzyme adds a methyl group to one or two bases, usually within the restriction site. When a methyl group is present there, the restriction endonuclease is prevented from cutting the DNA (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1595">Figure 7-5b</a>). Together with the restriction endonuclease, the methylating enzyme forms a restriction-modification system that protects the host DNA while it destroys foreign DNA. Restriction enzymes have been purified from several hundred different species of bacteria, allowing DNA molecules to be cut at a large number of different sequences corresponding to the recognition sites of these enzymes (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.table.1597">Table 7-1</a>).</p><a name="1598"></a><span class="subsect1-header">Restriction Fragments with Complementary “Sticky Ends” Are Ligated Easily</span><br><p>As illustrated in <a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1595">Figure 7-5a</a>, <i>Eco</i>RI makes staggered cuts in the two DNA strands. Many other restriction enzymes make similar cuts, generating fragments that have a single-stranded “tail” at both ends. The tails on the fragments generated at a given restriction site are complementary to those on all other fragments generated by the same restriction enzyme. At room temperature, these single-stranded regions, often called “sticky ends,” can transiently base-pair with those on other DNA fragments generated with the same restriction enzyme, regardless of the source of the DNA. This base pairing of sticky ends permits DNA from widely differing species to be ligated, forming chimeric molecules.</p><p>During in vivo DNA replication, DNA ligase catalyzes formation of 3′ → 5′ phosphodiester bonds between the short fragments of the discontinuously synthesized DNA strand at a replication fork (see <a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.847">Figure 4-16</a>). In recombinant DNA technology, purified DNA ligase is used to covalently join the ends of restriction fragments in vitro. This enzyme can catalyze the formation of a 3′ → 5′ phosphodiester bond between the 3′-hydroxyl end of one restriction-fragment strand and the 5′-phosphate end of another restriction-fragment strand during the time that the sticky ends are transiently base-paired (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1599">Figure 7-7</a>). When DNA ligase and ATP are added to a solution containing restriction fragments with sticky ends, the restriction fragments are covalently ligated together through the standard 3′ → 5′ phosphodiester bonds of DNA.</p><p>Some restriction enzymes, such as <i>Alu</i>I and <i>Sma</i>I, cleave both DNA strands at the same point within the recognition site (see <a class="text-ref" href="/books/bv.fcgi?rid=mcb.table.1597">Table 7-1</a>). These restriction enzymes generate DNA restriction fragments with “blunt” (flush) ends in which all the nucleotides at the fragment ends are base-paired to nucleotides in the complementary strand. In addition to ligating complementary sticky ends, the DNA ligase from bacteriophage T4 can ligate any two blunt DNA ends. However, blunt-end ligation requires a higher DNA concentration than ligation of sticky ends.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1600"></a><span class="subsect1-header">Polylinkers Facilitate Insertion of Restriction Fragments into Plasmid Vectors</span><br><p>Restriction enzymes to create fragments with sticky ends and DNA ligase to covalently link them allow foreign DNA to be inserted into plasmid vectors in vitro in a straightforward procedure. <i>E. coli</i> plasmid vectors can be constructed with a <i>polylinker,</i> a synthetic multiple-cloning-site sequence that contains one copy of several different restriction sites (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1601">Figure 7-8a</a>). When such a vector is treated with a restriction enzyme that recognizes a recognition sequence in the polylinker, it is cut at that sequence, generating sticky ends. In the presence of DNA ligase, DNA fragments produced with the same restriction enzyme will be inserted into the plasmid (<a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1601">Figure 7-8b</a>). The ratio of DNA fragments to be inserted to cut vectors and other reaction conditions are chosen to maximize the insertion of one restriction fragment per plasmid vector. The recombinant plasmids produced in in vitro ligation reactions then can be used to transform antibiotic-sensitive <i>E. coli</i> cells as shown in <a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1592">Figure 7-4</a>. All the cells in each antibiotic-resistant clone that remains after selection contain plasmids with the same inserted DNA fragment, but different clones carry different fragments.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1602"></a><span class="subsect1-header">Small DNA Molecules Can Be Chemically Synthesized</span><br><p>Advances in synthetic chemistry now permit the chemical synthesis of single-stranded DNA (ssDNA) molecules of any sequence up to about 100 nucleotides in length. Synthetic DNA has a number of applications in recombinant DNA technology. Complementary ssDNAs can be synthesized and hybridized to each other to form a dsDNA with sticky ends. Such completely synthetic dsDNAs can be cloned into plasmid vectors just as DNA restriction fragments prepared from living organisms are. For example, the 57-bp polylinker sequence shown in <a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1601">Figure 7-8</a> was chemically synthesized and then inserted into plasmid vectors to facilitate the cloning of fragments generated by different restriction enzymes. This example illustrates the use of synthetic DNAs to add convenient restriction sites where they otherwise do not occur. As described later in the chapter, synthetic DNAs are used in sequencing DNA and as probes to identify clones of interest. Synthetic DNAs also can be substituted for natural DNA sequences in cloned DNA to study the effects of specific mutations; this topic is examined in <a class="text-ref" href="/books/bv.fcgi?rid=mcb.chapter.1850">Chapter 8</a>.</p><p>The technique for chemical synthesis of DNA oligonucleotides is outlined in <a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1603">Figure 7-9</a>. Note that chains grow in the 3′ → 5′ direction, opposite to the direction of DNA chain growth catalyzed by DNA polymerases. Once the chemistry for producing synthetic DNA was standardized, automated instruments were developed that allow researchers to program the synthesis of oligonucleotides of specific sequences up to about 100 nucleotides long.<a href="#top"><img border="0" name="top link" alt="top link" src="/books/bookres.fcgi/system_default/top.gif" width="40" height="12"></a></p><a name="1604"></a><span class="subsect1-header">SUMMARY</span><br><br><ul>
<li>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*">  In DNA cloning, recombinant DNA molecules are formed in vitro by inserting DNA fragments of interest into vector DNA molecules. The recombinant DNA molecules are then introduced into host cells, where they replicate, producing large numbers of recombinant DNA molecules that include the fragment of DNA originally linked to the vector.</li>
<li>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*">  The most commonly used cloning vectors are <i>E. coli</i> plasmids, small circular DNA molecules that include three functional regions: (1) an origin of replication, (2) a drug-resistance gene, and (3) a region where DNA can be inserted without interfering with plasmid replication or expression of the drug-resistance gene.</li>
<li>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*">  Two enzymes are used to produce recombinant plasmids. Restriction enzymes cut DNA at specific 4- to 8-bp sequences, often leaving self-complementary single-stranded tails (sticky ends). These enzymes are used to cut long DNA molecules into multiple restriction fragments and to cut a plasmid vector at a single site. If a restriction fragment and cut plasmid vector with complementary ends are mixed under the proper conditions, DNA ligase will form phosphodiester bonds between the restriction fragment and vector DNA (see <a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1599">Figure 7-7</a>).</li>
<li>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*">  When recombinant plasmids are incubated with <i>E. coli</i> cells first treated with a high concentration of divalent cations, a very small fraction of the cells take up a single recombinant plasmid. These transformed cells, which carry the plasmid drug-resistance gene, can be selected by plating on nutrient agar containing the antibiotic (see <a class="text-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1590">Figure 7-3</a>). All the cells in each colony that grows on this medium contain identical plasmids descended from the single plasmid that entered the founder cell of the colony. Isolated colonies thus represent clones of the different restriction fragments originally inserted into the plasmid vector.</li>
<li>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*">  Polylinkers are synthetic oligonucleotides composed of one copy of several different restriction sites. Plasmid vectors that contain a polylinker will be cut only once by multiple restriction enzymes, each acting at its own site. Inclusion of a polylinker in a plasmid vector thus permits cloning of restriction fragments generated by cleavage of DNA with multiple different restriction enzymes.</li>
<li>
<img border="0" src="/books/bookres.fcgi/system_default/bull.gif" alt="*">  Single-stranded DNA containing up to 100 nucleotides of any desired sequence can be chemically synthesized using automated instruments. Synthetic dsDNAs are produced by synthesizing complementary ssDNAs and then hybridizing them.</li>
</ul>
</td>
          <td valign="top"><table width="100%" class="sidebar-table">
<tr><th class="sidebar-header">Navigation</th></tr>
<tr><td><div class="toc-sidebar-text"><table cellspacing="2" cellpadding="2">
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-0" href="javascript:BVShow('mcb')">About this book</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-1">7. Recombinant DNA and Genomics</a></td>
</tr>
<tr>
<td width="8">
<img src="/books/bookres.fcgi/system_default/click.gif" border="0"> </td>
<td><a class="toc-sidebar-active-2">7.1. DNA Cloning with Plasmid Vectors</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">7.2. Constructing DNA Libraries with λ Phage and Other Cloning Vectors</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">7.3. Identifying, Analyzing, and Sequencing Cloned DNA</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">7.4. Bioinformatics</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">7.5. Analyzing Specific Nucleic Acids in Complex Mixtures</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">7.6. Producing High Levels of Proteins from Cloned cDNAs</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">7.7. Polymerase Chain Reaction: An Alternative to Cloning</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">7.8. DNA Microarrays: Analyzing Genome-Wide Expression</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">PERSPECTIVES for the Future</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">PERSPECTIVES in the Literature</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">Testing Yourself on the Concepts</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2">MCAT/GRE-Style Questions</a></td>
</tr>
<tr>
<td width="8"></td>
<td><a class="toc-sidebar-passive-2" href="javascript:BVShow('mcb.biblist.1799')">References</a></td>
</tr>
</table></div></td></tr>
</table>
<table width="100%" class="sidebar-table">
<tr><th class="sidebar-header">Figures</th></tr>
<tr><td><span class="sidebar-text"><table cellspacing="2" cellpadding="2">
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1585">Figure 7-1. <link_caption>Diagram of a simple cloning...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1587">Figure 7-2. <link_caption>Plasmid DNA replication.</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1590">Figure 7-3. <link_caption>General procedure for cloning a...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1592">Figure 7-4. <link_caption>Isolation of DNA fragments from...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1595">Figure 7-5. <link_caption>Restriction-recognition sites are short DNA...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1596">Figure 7-6. <link_caption>Fragments produced by cleavage of...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1599">Figure 7-7. <link_caption>Ligation of restriction fragments with...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1601">Figure 7-8. <link_caption>Plasmid vectors containing a polylinker,...</link_caption></a></td>
</tr>
<tr>
<td><img src="/books/bookres.fcgi/system_default/figure.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.figgrp.1603">Figure 7-9. <link_caption>Chemical synthesis of oligonucleotides by...</link_caption></a></td>
</tr>
</table></span></td></tr>
</table><table width="100%" class="sidebar-table">
<tr><th class="sidebar-header">Tables</th></tr>
<tr><td><span class="sidebar-text"><table cellspacing="2" cellpadding="2"><tr>
<td><img src="/books/bookres.fcgi/system_default/table.gif" border="0"></td>
<td><a class="sidebar-ref" href="/books/bv.fcgi?rid=mcb.table.1597">Table 7-1. <link_caption>Selected Restriction Endonucleases and Their...</link_caption></a></td>
</tr></table></span></td></tr>
</table>
<br><script type="text/javascript" src="/books/bookres.fcgi/system_default/search.js"></script>

<form name="frmGo" method="get" action="/entrez/query.fcgi">
 <input type="hidden" name="db" value="Books">
 <input type="hidden" name="cmd" value="Search">
 <input type="hidden" name="term" value=".">
 <input type="hidden" name="doptcmdl" value="Books">
</form>

<form name="frmSearch" onSubmit="return Process('mcb')">
<table class="sidebar-table" width="100%">
<tr><th class="sidebar-header">Search</th></tr>
<tr><td>
	<input type="text" name="__term" size="12" 
	      onKeyPress="KeyPress('mcb',event)">
	&nbsp;
	<a href="javascript:Process('mcb');"><img src="/books/bookres.fcgi/system_default/searchbtn.gif" border="0"></a>
	<br><small>
  <input type="radio" name="__sbook" checked> This book&nbsp; &nbsp;
  <input type="radio" name="__sbook"> All books<br>
  <input type="radio" name="__sbook"> PubMed</small>
</td></tr></table>
</form>
</td>
        </tr>
        </table>
        
                <br><small><center>&copy; 2000 by <a href="http://www.whfreeman.com/">W. H. Freeman and Company.</a></center></small><br>
      

 <br>

<!-- redirect empty query -->
<script type="text/javascript">
  var s = document.location.href;
  if(document.forms.length>0 && document.forms[0].name=='redirect_form')
  {
      var rs = document.forms[0].redirect.value;
      if(rs.indexOf('google')>-1 && (rs.indexOf('medline')>-1 || rs.indexOf('pubmed')>-1))
      {
//          document.write('Redirect to PubMed site: '+s.substr(0, s.indexOf('books/bv.fcgi'))+'entrez/');
          document.location.href = s.substr(0, s.indexOf('books/bv.fcgi'))+'entrez/';

       }
       else
      {
//         document.write('Redirect to Bookshelf site: '+s.substr(0, s.indexOf('bv.fcgi')));
         document.location.href = s.substr(0, s.indexOf('bv.fcgi'));
       }
  }  
</script>


</body>
</html>
