<HTML>
<HEAD>
   <TITLE>CloningVectors</TITLE>
</HEAD>
<BODY BGCOLOR="#FFFFFF">
<P ALIGN=center><IMG SRC="WSS.gif" WIDTH=528 HEIGHT=90 ALIGN=bottom>

<H2 ALIGN=center><FONT SIZE="-1">Session 12</FONT></H2>

<P>

<HR SIZE=4 WIDTH="25%">

<FONT SIZE="-1">&#91;Some of this material is updated from my book,
"Introduction to Genetic Engineering", Butterworth/Heinemann,
1991&#93; </FONT>

<H1 ALIGN=center>Genetic Engineering Vectors</H1>

<P>The second step in molecular cloning is to join the passenger DNA
to the DNA of a suitable cloning vehicle. These vehicles (or
<B>vectors</B>) have the property that they replicate themselves and
any attached passenger DNA so that the passenger is amplified and can
be eventually isolated. A number of different vectors have been
developed for genetic engineering. Each has special distinguishing
properties.

<H2>Plasmids</H2>

<P>Plasmids are relatively small, double-stranded, closed-circular
DNA molecules that exist apart from the chromosomes of their hosts.
They are a kind of molecular parasite and are present in a wide
variety of bacterial and fungal species. Naturally occurring plasmids
carry one or more genes. For example, some plasmids carry genes which
confer resistance to certain antibiotics. Some plasmids may bear
genes that code for the restriction and modification enzymes that
were discussed previously. Some may carry genes that direct the
synthesis of enzymes that aid in the production of bacterial poisons
or antibiotics.

<P>However, from the viewpoint of the genetic engineer, the most
important property of plasmids is that they bear a special region of
DNA called an <B>origin of replication</B>, or more simply an
<B>origin</B>. This region allows the plasmid to multiply within and
semi-independently of its host. It confers on the plasmid the
property that if a foreign DNA segment is inserted into it, the
plasmid doesn't much care. It will replicate its own DNA as well as
any passenger DNA that may be attached to it, producing many copies
of the recombinant molecule.

<P>In a typical cloning experiment, the circular plasmid DNA is cut
once by treating it with a restriction endonuclease. This converts
the circular molecule into a linear one. Then, a foreign DNA fragment
adds on to the ends of the vector with the help of the enzyme DNA
ligase. The ligase creates a circular molecule containing both the
plasmid and its passenger. The next step is transformation -- the
introduction of the recombinant DNA molecule into a single bacterium.
Once inside, the plasmid produces many copies of itself as the
bacteria themselves grow and reproduce.

<H3>Plasmid replication</H3>

<P>Many of the plasmids that are commonly utilized in recombinant
technology are derivatives of pMB1, a plasmid that was originally
isolated from bacteria that had infected a human patient. pMB1 is an
example of a <B>relaxed</B> replicating plasmid. Relaxed in this case
refers not to a state of mind, but to the regulation of the number of
copies of the plasmid in the cell. Plasmids under relaxed control are
commonly maintained in bacteria in large numbers -- sometimes
hundreds of copies. By contrast, there are normally only one or a few
copies of stringently controlled plasmids in a cell.

<H3>Desirable properties of plasmids</H3>

<P>Plasmids have proven extremely useful as cloning vectors,
particularly when they are modified specifically for that purpose. A
bacterial plasmid that is designed for cloning should have some or
all of the following properties:

<UL>
   <LI><B>It should be small.</B>
   
   <P>The aim of most cloning experiments is to isolate the passenger
   DNA. The vector only serves to carry and amplify the passenger. A
   small plasmid has the advantage of contributing only a minimal
   amount of extraneous DNA to the plasmid/passenger construct,
   thereby making it easier to prepare large amounts of the passenger
   DNA. In addition, it is easier to get smaller pieces of DNA into
   bacteria than larger ones -- everything else being equal -- and
   the smaller the plasmid is, the less it contributes to total size.
   Another point is that small plasmids replicate faster and require
   less energy for replication than large ones. Finally, small
   plasmids are easier to purify than large ones because they are
   less fragile.</LI>
   
   <LI><B>Its DNA sequence should be known.</B>
   
   <P>Knowledge of the DNA sequence of a plasmid allows the genetic
   engineer to manipulate it with the full complement of recombinant
   DNA techniques.</LI>
   
   <LI><B>It should grow to high copy number in the host cell.</B>
   
   <P>In other words, relaxed replication plasmids are most often
   preferred to those that are under stringent control. In most
   cases, having many copies makes it easier to purify the plasmid
   away from the chromosomal DNA, and of course increases its yield.
   However, there are circumstances when just one or a few copies of
   a plasmid are desirable. Can you think of one?</LI>
   
   <LI><B>It should contain a selectable marker that allows cells
   containing the plasmid to be isolated.</B>
   
   <P>Bacterial cells that harbor plasmids don't necessarily look or
   act different than those that do not. It is only when the plasmid
   carries a gene that lends a special trait to the bacteria that the
   molecular engineer can tell if a bacterium contains a plasmid.
   Antibiotic resistance is one such trait or marker, and the genes
   for ampicillin resistance and tetracycline resistance are common
   genes carried by frequently used plasmids. Some plasmids are also
   readily lost from cells, and unless there is a selectable gene
   contributed by the plasmid and the bacteria are kept under
   selective conditions, the molecular engineer may end up with a
   plasmid-less population of cells at the end of an experiment.</LI>
   
   <LI><B>It should also contain a second selectable gene that is
   inactivated by insertion of the passenger.</B>
   
   <P>Imagine the following scenario. A scientist is trying to insert
   a passenger molecule into a plasmid. The plasmid has only a single
   selectable marker. After ligation, some of the molecules contain
   the passenger. Others do not. These may be plasmids that have
   recircularized in the presence of DNA ligase. This mixture of
   plasmids is then used to transform bacteria. Both recombinant
   plasmids and recircularized plasmids contain the same marker and
   cannot readily be told apart genetically. How are colonies
   carrying a plasmid with a passenger distinguished from those with
   a plasmid with no passenger? One way is to grow many individual
   colonies, isolate plasmid DNA from each, run the DNA out on an
   agarose gel, and identify the recombinant plasmid molecules
   because they are bigger by virtue of the passenger that they
   carry. While this procedure works and is very commonly carried
   out, it is laborious, time consuming, and expensive. <IMG SRC="plasmid1.gif" WIDTH=196 HEIGHT=184 ALIGN=right>
   
   <P>There is a better way. Suppose that a plasmid carries two
   different antibiotic resistance genes, A and B. What would be the
   consequences of cloning the passenger into the middle of gene B?
   In most cases, the foreign DNA will disrupt gene B and not allow
   it to work. On the other hand, if the plasmid simply
   recircularizes, gene B will be unaffected. With this arrangement
   it is possible to tell which cells harbor plasmids that carry
   passengers. Those that contain any plasmid at all will have gene A
   activity. And those that have a plasmid bearing a passenger, will
   lack activity from gene B.</LI>
   
   <LI><B>There should be a large number of unique restriction sites
   lying within one of the two selectable markers described
   above.</B>
   
   <P>Why unique? As described above, most cloning is done by
   inserting a fragment of DNA that is cut with a specific
   restriction endonuclease into a vector that is cut with the same
   enzyme (remember, this ensures that the two have compatible ends).
   If the vector contains more than one such site, it will be cut
   into multiple pieces by the restriction enzyme, complicating
   matters unnecessarily. The presence of many unique sites allows
   for maximum flexibility and ease in cloning.</LI>
</UL>

<H3>Plasmid purification</H3>

<P>Plasmids may be purified from bacteria by taking advantage of the
difference between the small, circular plasmid molecules and the
large, broken (hence linear) pieces of chromosomal DNA. (Chromosomal
DNA from <I>E. coli</I> is also circular, but during isolation its
relatively large size (six million nucleotide pairs) and consequent
fragility cause it to break easily.)

<P>The most common method for purifying plasmid DNA involves three
steps.

<UL>
   <LI>First, the bacteria are broken open and the is DNA
   isolated.</LI>
   
   <LI>Then the DNA is denatured.</LI>
   
   <LI>Finally, the DNA is renatured and centrifuged.</LI>
</UL>

<P>Upon denaturation, the two circular single-stranded chains of the
plasmid DNA remain entwined and don't separate fully. When conditions
are set up so that renaturation can occur, each strand rapidly finds
its complement. The chromosomal DNA, on the other hand, breaks
readily and therefore consists of noncircular pieces. Under these
circumstances, the two strands easily denature and separate. Upon
renaturation, they have difficulty finding complete copies of their
complements. Often partial renaturation occurs between different
sizes of single-stranded fragments and large insoluble aggregates
form. These can be simply separated from the small circular plasmid
DNA by high-speed centrifugation.

<H3>Some popular plasmids</H3>

<P>The first really useful plasmid for genetic engineering, pBR322,
was pieced together by Francisco Bolivar, and others in Herbert
Boyer's laboratory in the 1970s (The "B" stands for Bolivar and the
"R" for Rodriguez, another scientist in Boyer's laboratory). What
makes pBR322 useful is that it contains an ampicillin resistance gene
and a tetracycline resistance gene. In addition it has a relaxed
origin of replication (indicated by the green arrow shown at the
bottom of the illustration) and accumulates to high numbers in <I>E.
coli</I>. Its entire 4363 base-pair sequence has been determined, and
21 common enzymes are available that recognize only a single site
within it. (However, only 11 are in either of the two antibiotic
resistance genes.).<IMG SRC="pBR322GCK.gif" WIDTH=419 HEIGHT=340 ALIGN=right>

<P>More recently, a series of small plasmids (about 2.7 kilobase
pairs) have been developed in Joachim Messing'e laboratory that have
several properties that have made them very popular with genetic
engineers. These pUC (pronounced PUCK) plasmids, exemplified by pUC18
pictured at the right, carry an ampicillin resistance gene and an
origin of replication, both from pBR322. They also bear a multiple
cloning site -- a sequence of DNA that carries many restriction sites
(13, in the case of pUC18). The multiple cloning site of the pUC
plasmids is special because it also codes for a small peptide. This
peptide will correct a specific mutation in the chromosomal gene that
codes for the enzyme beta-galactosidase.

<P>When plasmids containing this sequence are cloned into a specific
<I>E. coli</I> strain that lacks beta-galactosidase activity, they
make the peptide and thereby begin to express active enzyme. If,
however, one of the restriction endonuclease sites in the multiple
cloning site is opened and a foreign gene inserted, the peptide is no
longer produced (because the protein coding region is disturbed), and
no beta-galactosidase activity appears.<IMG SRC="pUC18.gif" WIDTH=279 HEIGHT=252 ALIGN=bottom>

<P>Cells that harbor an active beta-galactosidase enzyme can be made
to turn blue in the presence of certain substrates. Those colonies
that have a passenger inserted at the multiple cloning site of the
pUC plasmids will lack the enzyme and will be white, while those that
have simply recircularized (those that don't contain a passenger)
will stain blue. In this way, pUC plasmids containing a foreign
insert of DNA can be distinguished from plasmids without a passenger.

<P>

<HR>

<H2>Lambdoid phage</H2>

<P>Lambda is a temperate bacteriophage with a genome size of about
48.5 kilobase pairs. <IMG SRC="lambda1.gif" WIDTH=180 HEIGHT=131 ALIGN=right>Its
entire DNA sequence is known. In phage particles, the lambda genome
exists as a linear, double-stranded molecule with single-stranded,
complementary ends. These ends can hybridize with each other (and do
so when the DNA is within an infected cell) and are thus termed
cohesive. They are similar to, but longer than, the sticky ends that
were encountered when we discussed restriction enzymes.

<H3>Lysis and lysogeny</H3>

<P>Bacteriophage lambda can enjoy one of two life styles -- that's
what's meant by the word "temperate". It can enter the <B>lytic</B>
<B>cycle</B>, replicate many times, produce more phage, and destroy
its bacterial host. Alternatively, it can take up <B>lysogenic
growth</B>, meaning that it integrates its DNA into the bacterial
chromosome.

<P>The lysogenic state is highly stable, but not permanent. Although
the viral DNA may be passed through hundreds of generations of
bacteria while integrated into the chromosome, a number of influences
can cause "induction" and a return to the lytic cycle. We're going to
ignore the tendency of lambda phage to engage in the lysogenic state.

<P>Lambda is assayed by plating phage particles on a "lawn" of
bacterial colonies spread on plates so that they form a continuous
group of cells. If a lambda phage infects one of the cell, it will
multiply quickly and lyse that cell, releasing lambda phage. These in
turn will infect adjacent cells and also cause lysis. After a while,
a round hole -- technically, a <B>plaque</B> -- is seen in the lawn,
representing the breakage of many bacterial cells and also marking a
single original event: the entrance of a single virus into a single
cell.

<P>Shown below is the genetic map of phage lambda (not all the genes
are included) and a scale indicating the extent of various regions,
expressed as a percentage of the total genome. Notice that the genes
that form the proteins of the head are gathered together toward the
extreme left end of the DNA, and the tail-forming genes follow.
Similarly, the genes for DNA synthesis (O and P) and those that cause
bacterial lysis (S, R, and Rz) are clustered near the right end of
the lambda chromosome. In the middle of the chromosome -- between the
J and int genes -- is a large region whose function is unclear but
apparently unessential. Also pictured at the ends of the chromosome
are the two single-stranded cohesive ends.

<P><IMG SRC="lambdaMap.gif" WIDTH=514 HEIGHT=90 ALIGN=bottom>

<H3>Three reasons why bacteriophage lambda is a good cloning
vehicle</H3>

<OL>
   <LI>It can accept very large pieces of foreign DNA. About 20kb of
   DNA can be deleted from its central region (between the J and N
   genes) and elsewhere, and replaced with an equal quantity of
   foreign DNA without affecting the ability of the phage to grow
   lytically. As already discussed in the section on plasmids, a good
   cloning vehicle should accept foreign DNA and amplify it without
   regard to the organism from which it comes. Phage lambda fills
   this bill nicely.</LI>
   
   <LI>It has been extensively reworked over the years. Genetic
   engineers have constructed numerous derivatives of lambda that
   contain only one or two sites for a variety of restriction
   enzymes.</LI>
   
   <LI>Finally -- its main advantage -- bacteriophage lambda is one
   of the few organisms that can be reconstituted in a test tube. By
   simply mixing phage DNA with a mixture of phage proteins, an
   infective viral particle with the DNA inside the phage head can be
   produced. This process is called in vitro packaging and it
   normally occurs when long tandem arrays of phage chromosomes (they
   are produced in this form when they replicate in the bacterial
   cell) are cleaved into monomers by certain phage proteins. The DNA
   is cut at a site on the chromosome called cos, and this operation
   produces the cohesive ends that were mentioned above. Ordinarily
   the DNA is cut into 48.5 kb fragments that are incorporated into
   the head of newly formed phage particles.</LI>
</OL>

<P>However -- and this is very important -- there is a strict size
requirement for the piece of DNA that goes into the head. That is, if
the distance between successive cos sites is either too long or too
short (longer than about 105% or shorter than about 78% of a
wild-type phage's DNA), the resultant phage will have markedly
decreased viability. For example, if a piece of phage DNA of 36 kb
(about 75% of 48.5 kb) is packaged, it will fail to yield significant
numbers of active phage. The same thing will happen if the distance
between two successive cos sites is 53.4kb, approximately 110%
greater than the normal size of a lambda chromosome.

<P>The molecular engineer can take advantage of this situation to
distinguish among recombinant molecules that carry foreign DNA and
those that do not. The essential idea is to begin with a vector that
carries a piece of useless DNA (a so-called stuffer fragment) that
can be excised with the aid of one or more restriction endonucleases.
The stuffer fragment is specially designed so that when it is
removed, the spacing between the cos sites will be too short for
successful packaging. If a passenger is not substituted in place of
the stuffer, no infective phage particles will be produced.

<P>Phage that have a stuffer fragment are called substitution vectors
because they are designed to have a piece removed and substituted
with something else.

<P>Large numbers of different lambda strains have been created that
allow efficient cloning of a variety of foreign DNA's. The various
strains are designed to have differing amounts of DNA removed, and
they contain a variety of restriction enzyme sites for cloning. In
addition, some lambda strains have a stuffer fragment that carries
the beta-galactosidase gene. When it is removed or when foreign DNA
is cloned within the gene, beta-galactosidase activity may be
abolished. The accompanying loss of activity may be used to select
recombinant clones.

<H3>Why lambda?</H3>

<P>Why use bacteriophage lambda as a vector rather than plasmids?
Large pieces of DNA (up to about 20 kilobase pairs) can be easily
cloned in bacteriophage lambda substitution vectors. Plasmid vectors
are less useful for cloning big passengers.

<P>But why clone large pieces of DNA in the first place? One obvious
reason is that some genes are very big and it is advantageous to have
them all in one piece. Interestingly, often it is not the
protein-coding portion of these large genes that contributes to their
size. Rather, the regulatory regions of several genes are dispersed
tens of thousands of base pairs away from the structural gene.

<P>Another reason for cloning in lambda is the efficiency it offers
in DNA transformation. In general, plasmids are much less efficiently
moved into bacterial cells than bacteriophage, which, after all, were
designed by Nature for their ability to project their chromosome into
their hosts.

<H2>Summary</H2>

<UL>
   <LI>The second step in molecular cloning is to join the passenger
   DNA with a cloning vector.</LI>
   
   <LI>Plasmids are closed-circular, extrachromosomal DNA's that are
   frequently used for cloning relatively small pieces of DNA.</LI>
   
   <LI>Bacteriophage lambda DNA carries a dispensable region that may
   be substituted by a foreign passenger.</LI>
</UL>
</BODY>
</HTML>
