<html>

<head>
<title>Sequencer Pipeline</title>
</head>

<body bgcolor="#FFFFCC">
<!--<link rel=stylesheet href=try.css>-->
<form name="N1" method="post">
  <p align="right">
  <select onchange="window.open(N1.M1.options[N1.M1.selectedIndex].value)" size="1" name="M1">
  <option value="http://gnomix.ansci.umn.edu/software1.html" selected>
  BioInformatics_Resources_LINKs.......</option>
  <option value="http://www.mysql.com/doc/en/index.html">MySQL</option>
  <option value="http://www.ncbi.nlm.nih.gov">NCBI</option>
  <option value="http://www.ebi.ac.uk">EBI</option>
  <option value="http://www.ensembl.org">ENSEMBL</option>
  <option value="http://www.tigr.org">Tigr</option>
  <option value="http://www.intlgenome.org">LargeGenomeDataBase</option>
  <option value="http://www.cbs.dtu.dk/services">CBS</option>
  <option value="http://workbench.sdsc.edu">Biology_WorkBench</option>
  <option value="http://freeprogrammingresources.com/">FreeProgrammingResources</option>
  <option value="http://www.msi.umn.edu/cgl">
  Computational_Genetics_Laboratory_At_UofM</option>
  <option value="http://www.ccgb.umn.edu">CCGB_at_UofM</option>
  <option value="http://www.angis.org.au/Documents/RecipeBook/top.html">
  The_ANGIS_On-Line Recipe_Book</option>
  <option value="http://biosci.cbs.umn.edu/cbri">
  Center_for_Biodegradation_Research_and_Informatics</option>
  <option value="http://www.molbiol.ox.ac.uk/help/bioinformatics_software_page.html">
  Oxford bioinformatics software</option>
  <option value="http://bio.perl.org">bioperl</option>
  <option value="http://bioinformatics.org/software/index.php3">Molecular_Linux
  </option>
  <option value="http://videocast.nih.gov/PastEvents.asp?c=7">
  NIH_archives_of_Genome_Analysis_talks</option>
  <option value="http://mail.ahc.umn.edu/mailman/listinfo/bioinf-general">
  Biolinks_at_UofM</option>
  <option value="http://www.binf.umn.edu/whatsbinf.html">What_is_bioinformatics
  </option>
  <option value="http://www.isb-sib.ch">Swiss_Institute_of_Bioinformatics
  </option>
  <option value="http://conferences.oreillynet.com/bio2003">
  O'Relly_Bioinformatics_Conference</option>
  <option value="http://safari.oreilly.com">Safari_Bookshelf</option>
  <option value="http://www.biomedcentral.com/bmcbioinformatics">Bio_IT_World
  </option>
  <option value="http://online.kitp.ucsb.edu/online/infobio01">
  ITP_Workshop_on_Stat._Physics_and_Biol</option>
  <option value="http://www.nibib.nih.gov/training/coursetable.html">
  US_Graduate_Programs_in_Bioinformatics</option>
  <option value="http://us.expasy.org/tools">ExPASy_Proteomics_tools</option>
  <option value="http://restools.sdsc.edu">CMS_Molecular_Boilogy_Resources
  </option>
  <option value="http://motif.stanford.edu">eMotif</option>
  <option value="http://www.apbionet.org">ApBNet</option>
  <option value="http://www.cernet.edu.cn">CERNet</option>
  <option value="http://pir.georgetown.edu">PIR</option>
  <option value="http://www.rcsb.org/pdb">PDB</option>
  <option value="http://123genomics.homestead.com/files/home.html">123Genomics
  </option>
  <option value="http://www.ndsu.edu/virtual-genomics/seminars.htm">
  VirtualSeminars</option>
  <option value="http://biocore.ncsa.uiuc.edu">BioCore</option>
  <option value="http://www.sciencejobs.com/">Sciencejobs</option>
  <option value="http://bio.oreilly.com">O'ReillyBioinformaticsWebsite</option>
  </select> </p>
</form>
<p align="center">
<img height="135" src="evolution1.gif" width="349" border="0"></p>
<div align="center">
  <center>
  <table height="1" cellSpacing="0" cellPadding="0" width="756" border="0">
    <tr>
      <td width="23" bgColor="#ffffcc" height="39" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="81" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="16">
      <font color="#0000ff"><b><font color="#800000">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/array/access.cgi">
      <font color="#800000" face="Franklin Gothic Book">ArrayBrowser</font></a></font></b></font>
      </td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="23">
      <p align="justify"><font size="2" face="Franklin Gothic Book">ArrayBrowser is developed to support livestock animal gene expression studies and 
	eventually integrate gene expression data with physical and genetic data of different 
	species. We are working towards the understanding of genes and pathways involved in 
	the islet biology, and quality control of porcine islet cells prior to 
	xeno-transplantation monitored by gene expression profiling with porcine specific 
	genome chips, which contains more than 13K 70mer oligos.  We are also developing 
	bovine whole genome oligo chip to understand the genes/pathways involved in milk 
	production. Those genome chips developed for livestock animals will allow more 
	experiments designed to address the function of genes in process of medical and 
	agricultural importance.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="39" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="81" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="16">
      <font color="#0000ff"><b><font color="#800000">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/SMD/login.pl">
      <font color="#800000" face="Franklin Gothic Book">Longhorn Array Database</font></a></b>
      </td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="23">
      <p align="justify"><font size="2" face="Franklin Gothic Book">A local copy of
       the fully open 
       source version of the Stanford Microarray Database (SMD), one of the largest 
       microarray databases.</font></td>
    </tr>
 <tr>
      <td width="23" bgColor="#ffffcc" height="39" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="81" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="16">
      <font color="#0000ff"><b><font color="#800000">
      <a style="TEXT-DECORATION: none" href="http://genome.coafes.umn.edu/">
      <font color="#800000" face="Franklin Gothic Book">Stanford MicroArray Database</font></a></b>
      </td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="23">
      <p align="justify"><font size="2" face="Franklin Gothic Book">A local copy of
       Full implementation of Stanford Microarray database using Oracle database and Linux OS  
       .</font></td>
    </tr>
    
<tr>
      <td width="23" bgColor="#ffffcc" height="39" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="81" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="16">
      <font color="#0000ff"><b><font color="#800000">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/abi3100.html">
      <font color="#800000" face="Franklin Gothic Book">Sequencer Pipeline </font></a></font></b></font>
      </td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="23">
      <p align="justify"><font size="2" face="Franklin Gothic Book">These are a series of tools for coordinating the
      generation and database storage of DNA sequence data.&nbsp;This includes the
      automated generation of run sheets for the ABI3100 and the submission or
      editing of sequence data to our Comparative and Functional Genomics
      database.</font></td>
    </tr>
<tr>
      <td width="23" bgColor="#ffffcc" height="35" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="60" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="14">
      <font color="#800000" face="Franklin Gothic Book">
      <a href="http://primer.ansci.umn.edu/cgi-bin/EST/basket/estSubmission.cgi" style="text-decoration: none; font-weight: 700">
      <i><font color="#800000">GenBank</font></i><font color="#800000"> 
      submission Pipeline</font></a></font></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="21">
      <p align="justify"><font size="2" face="Franklin Gothic Book">This is a tool to assist in the submission of data from 
      high volume DNA sequencing projects to public databases, including dbEST 
      and dbSTS. Header files are prepared for individual or batch submission of 
      sequences. Submission is coordinated with submission of trace files (TRACEdb) 
      and user interactive masking, trimming and quality control. </font> </td>
    </tr>
<tr>
      <td width="23" bgColor="#ffffcc" height="35" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="60" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="14">
      <font color="#800000" face="Franklin Gothic Book">
      <a href="http://134.84.29.73/cgi-bin/Getiria/sequence-annotation2.0/sequence_annotation.cgi" style="text-decoration: none; font-weight: 700">
      <i><font color="#800000">Sequence Annotation Pipeline.</font></i><font color="#800000"></font></a></font></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="21">
      <p align="justify"><font size="2" face="Franklin Gothic Book">This is a tool to assist in the annotation of sequence  data.</font> </td>
    </tr>
  <tr>
      <td width="23" bgColor="#ffffcc" height="48" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="61" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="20"><b>
      <font color="#0000ff">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/chromatAnalyzerIndex.html">
      <font color="#800000" face="Franklin Gothic Book">Chromat Analyzer</font></a></font></b></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="28">
      <p align="justify"><font size="2" face="Franklin Gothic Book">This tool permits secure viewing of chromatograms and 
      FASTA sequences from remote locations.&nbsp; You can view data already resident 
      on our server, or upload data from remote locations. Please use Internet 
      Explorer or Netscape 6.2 web browser for this tool.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="1" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="70" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="13">
      <font color="#000080"><b>
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/workset/index.cgi?job=primer3">
      <font color="#800000" face="Franklin Gothic Book">Primer3</font></a></b></font></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="2">
      <p align="justify"><font size="2" face="Franklin Gothic Book">Primer3 selects primers for polymerase chain reaction (PCR), 
      hybridization and DNA sequencing using sequence provided by the user.&nbsp; If 
      desired, sequence quality can be considered in the selection of 
      primers.&nbsp;Target sequence can be provided individually or in batch as 
      either Chromatograms OR <a href="http://primer.ansci.umn.edu/FASTA.html">
      FASTA</a> files.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="10" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="59" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="12">
      <font color="#000080"><b>
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/workset/index.cgi?job=overgo">
      <font color="#800000" face="Franklin Gothic Book">Overgo4.0</font></a></b></font></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="1">
      <p align="justify"><font size="2" face="Franklin Gothic Book">Overgo 4.0 selects Overgo oligos for the synthesis of 
      high specific activity hybridization probes. Target sequence can be 
      provided individually or in batch as either Chromatograms OR
      <a href="http://primer.ansci.umn.edu/FASTA.html">FASTA</a> files.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="48" rowSpan="2">
      <p align="center">
      <font face="Franklin Gothic Book">
      <img height="119" src="bioinf3.gif" width="23" align="right" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="8">
      <font face="Franklin Gothic Book"><b>
      <font color="#000080">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/workset/index.cgi?job=concate">
      <font color="#800000">Sequence Concatenation Tool</font></a></font></b><font size="2">
      </font></font></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="40">
      <p align="justify"><font size="2" face="Franklin Gothic Book">This program allows for the concatenation of two 
      sequences, and their associated quality scores, directly from 
      chromatograms.&nbsp; The user has the option to reverse complement either 
      sequence before concatenation.&nbsp; Virtual unsequenced stuffer fragments of 
      selectable length can be inserted between the concatenated sequences.&nbsp; 
      This software is particularly valuable for creating sequence to be used in 
      the design of PCR primers that span unsequenced gaps. If you wish to 
      concatenate a large collection of sequences (batch) please contact system 
      administrator.</font></td>
    </tr>
    <tr>
      <td vAlign="top" align="left" width="23" bgColor="#ffffcc" height="51" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="57" src="bioinformatics2.gif" width="32" border="0"></font></td>
      <td vAlign="top" align="left" width="717" bgColor="#ffffcc" height="21">
      <b><font color="#800000">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/workset/index.cgi">
      <font color="#800000" face="Franklin Gothic Book">SeqCart</font></a></font></b></td>
    </tr>
    <tr>
      <td vAlign="top" align="left" width="717" bgColor="#ffffcc" height="30">
      <p align="justify"><font size="2" face="Franklin Gothic Book">This allows user to freely group sequences into 
      different groups according to various analysis tasks. These groups can be 
      used in later calculations such as primer design, sequence submission, 
      Blast search, and data mining. </font> </td>
    </tr>
    <tr>
      <td vAlign="top" align="left" width="23" bgColor="#ffffcc" height="93" rowspan="2">
      <font face="Franklin Gothic Book">
      <img height="103" src="bioinf3.gif" width="23" align="right" border="0"></font></td>
      <td vAlign="top" align="left" width="717" bgColor="#ffffcc" height="2"><b>
      <font color="#800000">
      <a href="http://primer.ansci.umn.edu/cgi-bin/Mapping/geneMap.cgi" style="text-decoration: none">
      <font color="#800000" face="Franklin Gothic Book">OrthoMap</font></a></font></b></td>
    </tr>
    <tr>
      <td vAlign="top" align="left" width="717" bgColor="#ffffcc" height="70">
      <p align="justify"><font size="2" face="Franklin Gothic Book">This tool is used to identify sequence orthologs and 
      their position in reference genomes. FASTA files can be uploaded or trace 
      files can be selected from our server via SeqCart. Based on parameters set 
      by the user, sequences are masked and compared to the sequence of selected 
      vertebrate genomes currated at ENSEMBL. Sequences are hierarchically 
      compared using WU-BLASTX to identify orthologous proteins, WU-BLASTN to 
      identify orthologous cDNAs, and finally WU-BLASTN to identify orthologous 
      intergenic DNA. Eventually, this tool will permit PCR primer design that 
      makes use of information about orthologous gene-structure.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="1" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="64" src="bioinf4.gif" width="22" align="right" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="1"><b>
      <font color="#800000">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/workset/index.cgi?job=blast">
      <font color="#800000" face="Franklin Gothic Book">BLAST</font></a></font></b></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="1">
      <p align="justify"><font size="2" face="Franklin Gothic Book">DNA Sequences can be selected from our server for 
      comparision to public data repositories based on the algorithm developed 
      by Altschul et al. (1990). Masks for submission and reporting defined by 
      the user.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="16" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="84" src="bioinf4.gif" width="22" align="right" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="1"><b>
      <font color="#800000">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/blat1.html">
      <font color="#800000" face="Franklin Gothic Book">BLAT</font></a></font></b></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="29">
      <font size="2" face="Franklin Gothic Book">This tool was developed by 
      Jim Kent and can be used to quickly find regions of the human or mouse 
      genomes with high similarity to your sequence (95% and greater similarity 
      over 40 bases or more). DNA BLAT works by keeping an index of the entire 
      human OR mouse genomes in memory.&nbsp; Coming soon, local BLAT of individual 
      and batches of sequences selectable from our server or loaded from a 
      remote location. </font> </td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="1" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="77" src="bioinf4.gif" width="24" align="right" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="1"><b>
      <font color="#000080">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/PolyView/PolyView.cgi">
      <font color="#800000" face="Franklin Gothic Book">PolyView</font></a></font></b></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="1">
      <p align="justify"><font size="2" face="Franklin Gothic Book">This tool was developed for annotating and summarizing 
      the results from comparative re-sequencing projects (for SNP discovery) 
      based on the PHRED/POLYPHRED/CONSED suite.&nbsp; Sequences can be submitted for 
      analysis from our server or from remote locations.&nbsp; In order to annotate 
      or edit contigs requires the implementation of X-windows.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="58" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="72" src="bioinf4.gif" width="23" align="right" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="29"><b>
      <font color="#0000ff">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/cgi-bin/seqMapper/seqMapper.cgi">
      <font color="#800000" face="Franklin Gothic Book">ChromatAnnotator</font></a></font></b></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="29">
      <p align="justify"><font size="2" face="Franklin Gothic Book">This is a tool based on the PHRED/POLYPHRED/CONSED 
      suite and allows the user to assemble and annotate sequence contigs.&nbsp; 
      Sequences on our server or loaded from remote locations can be analyzed.&nbsp; 
      Annotation can be directly parsed to our database for later query.&nbsp; In 
      order to annotate or edit contigs requires the implementation of 
      X-windows.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="58" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="69" src="bioinf4.gif" width="25" align="right" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="29">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/pairwise_sequence_alignment.html">
      <b><font color="#800000" face="Franklin Gothic Book">Pairwise SEQALIGN</font></b></a></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="29">
      <p class="MsoNormal" align="justify">
      <font size="2" face="Franklin Gothic Book">This tool allows the pairwise 
      comparison of chromatograms to reference chromatograms or GenBank entries. 
      Individual or batches of sequences can be selected from our server or 
      loaded from a remote location.&nbsp; If you want to conduct a pairwise 
      comparison of a large collection of sequences (batch) contact the system 
      administrator.</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="28" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="44" src="bioinformatics2.gif" width="30" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="14"><b>
      <font color="#000080">
      <a style="TEXT-DECORATION: none" href="http://primer.ansci.umn.edu/RHmaps/index.html">
      <font color="#800000" face="Franklin Gothic Book">Rhmaps</font></a></font></b></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="14">
      <font face="Franklin Gothic Book" size="2">The INRA-University of 
      Minnesota porcine Radiation Hybrid (IMpRH) map</font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="87" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="88" src="bioinf3.gif" width="25" align="right" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="29">
      <font color="#800000"><b>
      <a style="TEXT-DECORATION: none" href="http://animalgene.umn.edu">
      <font color="#800000" face="Franklin Gothic Book">Locusmap and Pedigraph</font></a></b></font></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="58">
      <p style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px" align="justify">
      <font face="Franklin Gothic Book">
      <font size="3">&nbsp;</font><font size="2">(These tools are provided by Dr. Yang,&nbsp; Da, Department of 
      Animal Science, U of MN.)</font></font></p>
      <p style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px" align="justify">
      <font face="Franklin Gothic Book"><i><font size="2">Locusmap</font></i><font size="2"> 
      is a computer package designed for rapid linkage analysis and map 
      construction using a variety of inheritance modes. <i>Pedigraph</i> 
      permits pedigree visualization for studying population structure and 
      inheritance patterns.</font></font></td>
    </tr>
    <tr>
      <td width="23" bgColor="#ffffcc" height="51" rowSpan="2">
      <font face="Franklin Gothic Book">
      <img height="62" src="bioinf4.gif" width="25" align="right" border="0"></font></td>
      <td width="717" bgColor="#ffffcc" height="22"><b>
      <a href="http://primer.ansci.umn.edu/ermodel/index.html" style="text-decoration: none">
      <font color="#800000" face="Franklin Gothic Book">Comparative and Functional Genomics Database Schema</font></a></b></td>
    </tr>
    <tr>
      <td width="717" bgColor="#ffffcc" height="29">
      <p align="justify"><font face="Franklin Gothic Book"><a href="http://primer.ansci.umn.edu/ermodel/index.html"><font size="2">A schema </font></a><font size="2"> from the USDA Meat Animal Research Center has 
      been developed for dealing with genetic and DNA sequence data in a species 
      independent manner.&nbsp; We are collaborating with the USDA to modernize and 
      distribute this schema. </font></font> </td>
    </tr>
    <tr>
    <td><img height="62" src="bioinf4.gif" width="25" align="right" border="0"></font></td>
    <td valign="top">
      <a href="http://gnomix.ansci.umn.edu/microarray/qiagen01.html" style="text-decoration: 
none"><b><font color="#800000" face="Franklin Gothic Book">QiagenArray</font></b></a>
    </td>
    </tr>
  </table>
  </center>
</div>
<p align="center"><font size="2">Please Contact
<a href="mailto:junej003@umn.edu">Bhupinder S Juneja</a> 
with questions or suggestions</font></p>

</body>

</html>
