<HTML>
<!--This file created 8/30/05 2:21 PM by Claris Home Page version 3.0-->
<HEAD>
   <TITLE>Tutorial in Bioinformatics</TITLE>
   <META NAME=GENERATOR CONTENT="Claris Home Page 3.0">
   <X-CLARIS-WINDOW TOP=78 BOTTOM=842 LEFT=21 RIGHT=700>
   <X-CLARIS-REMOTESAVE SERVER="www.usm.maine.edu" USER=rhodes DIR="/home/users/rhodes/public.www/Goodies/" FILE="">
   <X-CLARIS-TAGVIEW MODE=minimal>
</HEAD>
<BODY BGCOLOR="#FFFFFF">
<H1><font color="#FF0000">Bioinformatics and Homology Modeling: <BR>
A Student-Tested Tutorial for Beginners</font></H1>

<H2>Exploring Human Visual Pigments</H2>
<ol>
  <li><a href="#intro">Introduction</a></li>
  <li><a href="#bioinformatics">Bioinformatics</a></li>
  <li><a href="#encore">Homology Modeling</a></li>
  <li><a href="#test">Test Your Bioinformatics Skills</a> </li>
</ol>
<p><FONT COLOR="#009900"><B>Last revision: 2007/01/08, to </B></FONT><a href="#revised"><B>here</B></a><FONT COLOR="#009900"><B>.</B></FONT></p>
<p>Gale Rhodes<BR>
  Chemistry Department
  <BR>
University of Southern Maine<FONT COLOR="#0033FF"><br>
</FONT><A HREF="../OffHrs.html" TARGET="_blank">Contact
Information</A></p>
<p>If you would like to use this tutorial in your teaching,<BR>
please ask permission. <A HREF="../Permit.html" target="_blank">Here's
why</A>.</p>
<P>Do you know about the <a href="http://www.grameenfoundation.org/" target="_blank">Grameen
Foundation</a>?</P>
<P>&nbsp;</P>

<CENTER><IMG SRC="images/MyHand.gif" WIDTH=492 HEIGHT=234 X-CLARIS-USEIMAGEWIDTH X-CLARIS-USEIMAGEHEIGHT ALIGN=bottom>
</CENTER>

<HR>

</P>

<h2><a name="intro"></a>Introduction</h2>
<P>This tutorial allows you to explore opsins -- the proteins that
catch light for our eyes -- and the genes that code for opsins. But
the real subject of this exercise is <B>bioinformatics</B> -- the use
of computers to search for, explore, and use information about genes,
nucleic acids, and proteins. While learning about the human opsins,
you will use some of today's most powerful bioinformatics tools. You
can follow up this tutorial with a study of opsins from other
organisms, or by exploring any class of biomolecules that interest
you.</P>

<P>I assume that you are conversant with biochemistry and molecular
biology. If you see unfamiliar terms pertaining to the genes, mRNAs,
and proteins used as examples here, break out your biochemistry text,
head for the index, and review, review, review.</P>

<P>For more information about each database or tool, go to its home
page and read, read, read.</P>

<P>If you are a student in my biochemistry course (CHY 463 at the
University of Southern Maine), you will find that this tutorial
follows closely my classroom demonstration of bioinformatics tools
applied to finding desired information in databases.</P>
<h3>History</h3>
<P>This web page was originally composed of somewhat sketchy procedures
  that I pasted in here as I devised them by playing* with bioinformatics
  tools on the web. Each year my biochemistry students have run through the tutorial,
  which has led to many improvements. Long about March each, I get it ready for them by trying it out to see what has changed in this fast-changing world. Take a look at the &quot;last revision&quot; date above, to see the last time I checked
  it. </P>
<P>*My play with bioinformatics tools was initiated by the book <A HREF="http://www.amazon.com/Bioinformatics-Dummies-Math-Science/dp/0470089857/sr=11-1/qid=1167952448/ref=sr_11_1/103-7926611-2337432"><B>Bioinformatics
  for Dummies</B></A><B> </B>(really!), by Claverie and Notredame,
  Wiley Publishing, Inc., 2003 (a new edition appeared in December of 2006). Not considering myself a dummie in most
  arenas, I had never looked very hard at Wiley's "Dummies" books. I'm
  so glad I looked at this one. The authors are on the frontiers of the
  field, and they have produced a serious, high quality book. If you
  want to work through lots of clear tutorials in all areas of
  bioinformatics, buy it. It's the best $30 I have spent on a book in
  quite a few years. Just click the title to learn more about the
  latest edition.</P>
<P></P>
<P>

<HR>

</P>

<H1><a name="bioinformatics"></a>Cast of Characters</H1>

<H2>I. The Databases (and their acronyms!)</H2>

<UL>
   <LI><B>Genbank</B>, operated by <B>NCBI</B> (National Center for
   Biotechnology Information)<BR>
   Contains all publicly available <B>sequences of DNA</B>, with
   annotations<BR>
   Same DNA sequence content as <B>EMBL</B> (European Molecular
   Biology Laboratory) and <B>DDBJ</B> (DNA Data Bank of Japan)</LI>
   
   <LI><B>UniProt Knowledgebase (Swiss-Prot and TrEMBL)</B>, operated by <B>SIB</B> (Swiss
   Institute of Bioinformatics) and <B>EBI</B> (European
   Bioinformatics Institute)<BR>
   Contains most of the publicly available <B>sequences of
   proteins</B>. Sequences in Swiss-Prot are annotated manually, and provide or link you to just about all published information about the sequence. Sequences in TrEMBL collected automatically from sequence databases, and are on their way to Swiss-Prot, but are not yet fully annotated. </LI>
   
   <LI><B>Protein Data Bank</B><BR>
   Contains all publicly availalble <B>experimentally determined
   structural models of proteins</B> and nucleic acids (determined by
   x-ray crystallography and NMR)</LI>
   
   <LI><B>Swiss-Model Repository</B><BR>
   Contains many <B>theoretical structural models of proteins</B>
   (determined by automated homology modeling)</LI>
   
   <LI><B>Online Mendelian Inheritance in Man<BR>
   </B>A catalog of human genes and genetic disorders, linked to gene
   entries in GenBank<B><BR>
   </B></LI>
</UL>

<H2>II. The Tools</H2>

<UL>
   <LI><B>NCBI Map Viewer</B><BR>
   For finding genes and gene products (RNAs and proteins) that
   interest you</LI>
   
   <LI><B>BLAST</B><BR>
   For finding genes or proteins with sequences similar to yours</LI>
   
   <LI><B>ClustalW</B><BR>
   For comparing your sequence with others, and lots of sequences
   with each other</LI>
   
   <LI><B>Phylip</B><BR>
   For making phylogenetic trees, which show how sequences are
   related to each other</LI>
   
   <LI><B>Treeprint</B><BR>
   For printing phylogenetic trees</LI>
   
   <LI><B>PSIPRED</B><BR>
   For predicting the location of helices, pleated sheets, and
   transmembrane elements of proteins of unknown structure</LI>
   
   <LI><B>Swiss-Model</B><BR>
   For automated building theoretical structural models of your
   sequence based on known structures (homology modeling)</LI>
   
   <LI><B>Deep View</B> (also knows as <B>Swiss-PdbViewer</B>)<BR>
   For seeing and exploring macromolecular models in three
   dimensions, and for manual and semiautomated homology
   modeling</LI>
   
   <LI><B>PubMed</B><BR>
   For searching ALL the literature of the life sciences</LI>
   
   <LI><B>ExPASy</B> (Expert Protein Analysis System)<BR>
   Not so much a tool as a tool box -- a very complete set of protein
   analysis tools<BR>
   </LI>
</UL>

<H1>Here We Go</H1>

<P>Our subject is human opsins, those proteins, found in the cells of
your retina, that catch light and begin the process of vision. We
will proceed by asking questions about opsins and opsin genes, and
then using bioinformatics to answer them.</P>

<P>When I provide a web address, I'll also make it a link -- just
click it to go to the site in a new browser window. Then make it a
bookmark so you can find it again. This tutorial will still be open
in the window behind the new one.</P>

<P><FONT COLOR="#FF0000">WARNING: Bioinformatics tools evolve
rapidly, faster than I can make changes to this tutorial. So if a
page does not look exactly like I say it should, or if its title is
different, look around and try to do what the tutorial says. You
should find the same links, but names may be slightly different. If
the differences are so great that you can't proceed, send me email
(see contact link at top of this page), and I'll adapt the instructions to the
changes as soon as I can.</FONT></P>

<H2>Where are the opsin genes in the human genome?</H2>

<P>Point your browser to <A HREF="http://www.ncbi.nlm.nih.gov/mapview/" TARGET="_blank"><B>http://www.ncbi.nlm.nih.gov/mapview/</B></A>.</P>

<P>Note that you can look at a genome by
clicking on the NAME of the species, not the word<strong> BLAST </strong>beside
it.
The species name takes you to a viewer for the genome of that organism.
BLAST takes you to a tool for searching that genome  (later).</P>

<P>Find <B>Homo sapiens (human)</B>, and click on the OLDEST &quot;build&quot; or
  version of the genome.</P>
<blockquote>
  <p>NOTE: Why should you use the oldest build in this tutorial?
    Because sometimes not all links are
    hooked up to the tools for the newest build. </p>
</blockquote>
<P>You see a diagram of the human chromosomes, and a search box at
the top. Enter "opsin" in the box next to <B>Search for</B>.</P>

<P>Click <B>Find</B>.</P>

<P>You see the diagram again, with red marks at your "hits", the
locations of genes whose entries contain "opsin" as a whole or
partial word. Below the diagram is a list of the indicated genes. Among them are the rhodopsin gene (RHO), and three cone pigments, short-, medium-,
and long-wavelength sensitive opsins (for blue, green, and red light detection).
Four hits look like visual pigments, which probably does not surprise you. To
the left of each entry is the chromosome number, allowing you to tell which red
mark corresponds to each entry. Note that several hits are on the X
chromosome, one of the sex-determining chromosomes. You can pursue
multiple hits on the same chromosome with the <B>all matches</B> link
for that chromosome.</P>
<blockquote>
  <p>NOTE: In the human genome lists, you will often see duplicates marked &quot;reference&quot;
    or &quot;Celera&quot;, referring to the results from two major efforts to sequence the
    human genome. At first, these two efforts were separate, but eventually they
    came together. When you have a choice,
    choose &quot;reference,&quot; so you will be following the same path I followed
    in setting up the tutorial. </p>
</blockquote>
<P>Click <B>all matches</B> next to X. Be patient: the next page may take as much as one or two minutes to load. </P>

<P>You see a very complicated display (don't sweat -- we're going to
use only a part of this now). On the left is a diagram of the X
chromosome, with red marks at the positions of the gene(s) you've
followed to this page -- in our case, the two opsins, medium- and
long-wave, which are located near the bottom tip of the X chromosome.
To the right are various representations of the X chromosome, with
listings of annotated areas. The two opsin genes are highlighted in
pink. If you pass your cursor over this page without clicking, you
will find that some symbols provide brief information, mostly about
regions that are not yet characterized well enough to have a full
entry.</P>

<P>As you can see, there is a tremendous amount of information on
this page, with links to much more. If you want full information
about the meanings of abbreviations and symbols on this page, as well
as the kinds of information linked to the page, you can use <B>Map
Viewer Help</B> at the top of the page. You will find abundant
information about the Map Viewer, explanations of all symbols and
links, and even tutorials about how to ask and answer all kinds of
questions about the genome.</P>

<P>For now, note the information provided for the first of the two
highlighted opsin genes, <B>OPN1LW</B> (this is called the gene
symbol). You see that this is the long-wavelength-sensitive (red)
opsin, and that it's a gene involved in color blindness (a sex-linked
trait -- no surprise).</P>

<H2>What do scientists know about the opsins?</H2>

<P>Click <B>OPN1LW</B>.</P>

<P>You have entered <B>Entrez Gene</B>, which is a sort of highway interchange
  with routing to all sorts of information about this gene. Scan down the page.
  Some of the information is very plain and understandable, while some is very
cryptic. One of the most accessible links is to <B>OMIM</B> (for
Online Mendeliam Inheritance in Man), a catalog of human genes and
genetic disorders. Despite the name, the database includes genes of
women, too.</P>

<P>Look down the page and find <B>Phenotypes</B>, and notice the
links marked <B>MIM</B>. These are links to <B>OMIM</B> entries.
Click one of them.</P>

<P>Each <B>OMIM</B> entry tells you about this gene and types of
colorblindness, genetic disorders associated with mutations in this
gene. Read as much as your interest dictates. Follow links to other
information. For more information about <B>OMIM</B> itself, click the
<B>OMIM</B> logo at the top of the page. Once you've satisfied your
appetite, return to the <B>Entrez Gene</B> page (use the <B>Back</B>
button of your browser or your browser's history list -- if you're
lost, click <A HREF="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?db=gene&cmd=retrieve&dopt=default&list_uids=5956" TARGET="_blank">HERE</A>).</P>

<P>Next to the <B>Display</B> button, pull down the menu and select
<B>PubMed Links</B>.</P>

<P>You have entered <B>PubMed</B>, a free database of scientific
literature, to a list of articles directly associated with this gene
locus. By clicking on the authors of each article, you can see
abstracts of the article. If you are on a university campus where
there is online access to specific journals, you might also see links
to full articles. <B>PubMed</B> is your entry point to a wide variety
of scientfic literature in the life sciences. On the left side of any
<B>PubMed</B> page, you will find links to a description of the
database, help, and tutorials on searching. Use the Find tool of your browser
to find the name <em>Nathans</em> on this page. Read the abstract of the article
by Nathans and co-workers before returning to <B>Entrez
Gene</B>.</P>

<P>NB to GR: Add some guided searches in PubMed? (Pa Kettle said, &quot;I'll get around to that... one of these days.) </P>
<H2>What is the nucleotide sequence of this gene?</H2>

<P>Remember that we are looking at the gene for the red-sensitive
opsin in human vision, and it is located near the bottom tip of the X
chromosome. Scroll down (way down!) to <B>NCBI Reference Sequences</B>
(<B>RefSeq</B>). In the first section, <strong>mRNA and Protein(s)</strong>, all of the following are available:</P>

<UL>
   <LI>the protein sequence  (sequence of this gene's protein product, the
   red opsin), here listed as <strong>NP_064445.1</strong> (P for protein);</LI>
   
   <LI>the mRNA Sequence (sequence of nucleotide bases in the
   messenger RNA), here listed as <b><strong>NM_020061.3</strong> </b>(M for mRNA);</LI>
   <LI>the source sequence, listed as <strong>Z68193</strong> (sequence of the entire genome fragment
     that contains this sequence, from <b>GenBank</b>).</LI>
</UL>

<P>Note that the two links to mRNA sequence and protein sequence are given as NM_020061.3&rarr;NP_064445.1, the arrow implying that the sequence of the NM entry is translated (by protein synthesis) to give the sequence of the NP entry. </P>
<P>Click the entry number for the <B></B>mRNA sequence: NM_020061.3</P>
<P>This is a typical <B>GenBank</B> <U>nucleotide</U> file, and a lot
of it is hard to read, but a few things are clear. First note, under
references, citations to the publication of this sequence in the
scientific literature. To see an abstract of the article in which
this gene was described, click the <B>PubMed</B> link (a number) below the first
reference and read it. Or instead, find the word <em>Nathans </em>on the page,
and and click the <strong>PubMed</strong> link
below the related article.  As
you see, you've been here before. There are many ways to move from one database
to another, which is both a blessing and a curse. You have to keep your eyes
open for useful links, and <B>when
you find a path that you think you might use again, make a note of it
and bookmark the web pages</B>. It is frustrating to know there's an
easier way to do something, and not remember how you did it.</P>

<P>NB to GR: point back to this <A HREF="http://www.ncbi.nlm.nih.gov/entrez/viewer.fcgi?val=9910525&db=Nucleotide&dopt=GenBank" TARGET="_blank">abstract</A>
when you get the phylogenetic tree.</P>

<P>Scroll to the bottom of this long page. The last thing, labeled <strong>ORIGIN</strong>, is
  the sequence of this messenger RNA. You are seeing the actual list of As,
Ts, Gs, and Cs that make up the message for synthesis of this opsin.
But wait! You know that RNA contains no T. In most nucleotide
databases, U from RNA is represented as T, to make for easy
comparison of DNA and RNA sequences. This sequence information is
<U>not</U> in the form that is most useful for searching in
databases, say, searching for related genes. Let's display this entry
in a form more useful for searching.</P>

<P>At the top of the page, beside the <B>Display</B> button, pull
down the menu that says <B>GenBank</B> (the default
 display format for each entry), and select <B>FASTA</B> (note that several other
display options are available). Now you see one descriptive or "comment" line that begins with "&gt;",
followed by the nucleotide sequence. This little file is just what
you need to search nucleotide databases for similar sequences. Let's
keep it for future use.</P>

<P>Click and drag on the web page to select everything from the
"&gt;" through the last nucleotide. Be careful not to select anything
else. From your browser's <B>Edit</B> menu, select <B>Copy</B> to
make a copy of this information on your clipboard, for pasting
elsewhere. Now start your favorite word processor, make a new
document, and paste. The FASTA comment and sequence should appear.
Select all of the text and change the font to Courier or Monaco --
these "typewriter" fonts make it easy to align letters into columns,
because all letter are the same width. Save this file, choosing
<U>text</U> or <U>plain</U> <U>text</U> as the file type. Call it
<B>mrnared.txt</B>. Save it to a convenient location for the files
you'll be making later. Click your browser's <B>Back</B> button until
you return to the <B>Entrez Gene</B> page for this gene. </P>

<H2>What is the amino-acid sequence of this gene?</H2>

<P>Under <strong>NCBI Reference Sequences (RefSeq), </strong>click the entry number
  NP_064445.1 for the protein sequence. </P>

<P>Things look a lot like before, but this is a <U>protein</U> entry
(the classical view is that gene products are proteins, but not all
of them are), containing the amino-acid sequence in one-letter
abbreviations. Just as with the mRNA entry, turn this into a FASTA
display, and copy it into a new word-processor document. Save it in
<U>text</U> format as <B>protred.txt</B>. Return to <B>Entrez
Gene</B>.</P>

<H2>What does the neighborhood of this gene look like?</H2>

<P>Click the entry number beside <B>Source Sequence</B>.</P>

<P>This entry shows the
sequence of the specific DNA clone that contains the opsin gene,
along with information about how this clone was produced. This entry
thus shows the gene in the slightly larger context of the cloned
fragment in which this gene was found. This sequence would allow you
to see flanking regions around the gene, and perhaps to design PCR
primers for making useful quantities of the nucleotide sequence so
you could express this gene in a cloning vector. From this page, you
could also find neighboring sequences if you wanted to look farther
afield. As before, display this entry in FASTA format. You will get several entries,
  each a different clone that was found to contain this region of the genome.
  Save the first FASTA entry (from the &quot;&gt;&quot; to the end of the nucleotide sequence)
  as a word processor <U>text</U> document
  entitled <B>GBred.txt</B>.
  (Why GB? Because the last time I looked [still true 2006/03/21], these entries
  were called GenBank entries. But things change fast in this business.)</P>

<H2>What proteins in humans are similar to the red opsin?</H2>

<P>Now return to the<B> </B><A HREF="http://www.ncbi.nlm.nih.gov/mapview/" TARGET="_blank"><B>NCBI
Map Viewer</B></A>. We're going to search the human genome for
sequences similar to that of the red opsin.</P>

<P>Click <B>Blast</B> next to <B>Homo sapiens (human)</B>, OLDEST build.</P>

<P>This is the NCBI's BLAST search tool. BLAST is a widely used
program for finding sequences similar to a "query" sequence that
you're interest in. Pick these options from the various menus:</P>

<UL>
   <LI>Database: Build Protein for OLDEST build (look at bottom of the Database
     menu). This means that you will search the protein sequences in this build
     of the database.)</LI>
   
   <LI>Program: BLASTP (Use the version of BLAST that compares
   <U>protein</U> sequences, unlike BLASTN, which compares
   <U>nucleotide</U> sequences.)</LI>
   
   <LI>Other Parameters, Expect: 10 (The higher the number, the less
   stringent the matching, and the more hits you'll get)</LI>
</UL>

<P>Next, copy the FASTA data from your file <B>protred.txt</B> to
your clipboard, and paste it into the BLAST search box, above which
it says, "Enter an accession..." Check to be sure that the first
character in the box is the "&gt;" at the beginning of the FASTA
data. Then click<B> Begin Search</B>.</P>

<P>The next page is for formatting your search results. Just click
that enthusiastic <B>Format!</B> button. When your results are ready,
the <B>results of BLAST</B> page appears. Look down the page to the
graphical display, a box containing lots of colored lines. Each line
represents a hit from your blast search. If you pass your mouse
cursor over a red line, the narrow box just above the box gives a
brief description of the hit. You'll find that the first hit is your
red opsin. That's encouraging, because the best match should be to
the query sequence itself, and you got this sequence from that gene
entry. The second hit is the green opsin -- remember that the PubMed
entry reported that the red and green pigments are the most similar.
The third and fourth hits are the blue opsin and the rod-cell pigment
rhodopsin. Other hits have lower numbers of matching residues, and
are color coded according to a score of matches. If you click on any
of the colored lines, you'll skip down to more information about that
hit, and you can see how much similarity each one has to the red
opsin, your original query sequence. As you go down the list, each
succeeding sequence has less in common with red opsin. Each sequence
is shown in comparison with red opsin in what is called a <B>pairwise
sequence alignment</B>. Later, you'll make <B>multiple sequence
alignments </B>from which you can discern relationships among
genes.</P>

<P>See what you can figure out about what the scores mean.
<U>Identities</U> are residues that are identical in the hit and the
query (red opsin), when the two are optimally aligned.
<U>Positives</U> are residues that are very similar to each other
(see residue number 1 in the blue opsin -- it's threonine in red
opsin, and the very similar serine in the blue). <U>Gaps</U> are
sometimes introduced into a hit to improve its alignment with the
query. The more identities and positives, and the fewer gaps, the
higher the score. Note that blue opsin and rhodopsin are only about
45% identical to the red opsin. Other proteins, which are apparently
not visual pigments, have even lower scores. Now let's take a look at
where all these hits are in the human genome.</P>

<H2>Where are all the genes for these other proteins?</H2>

<P>Click the <B>Genome View</B> button near just below the
introductory information at the top of this result page. If this button does
  not appear, go back and make sure you are searching the database for the OLDEST
  build of the human genome.</P>

<P>You have come full circle. You are back that the human chromosome
diagram, and all the hits of your search, in the colors that signify
their BLAST scores, are located for you on the diagram. Notice that
there are about 100 proteins (discovered so far, that is) that have
40% or more positives in alignment with red opsin. The opsins are
members of the very large family of <U>G</U>
<U>protein</U>-<U>coupled</U> <U>receptors</U>, key players in signal
transduction.</P>
<H2>How are the opsin genes related to each other?</H2>

<P>Answering this question requires making a <B>multiple sequence
alignment</B> and then using it to make a <B>phylogenetic tree</B>.
For these tasks, we move to another database where it's a little
easier to gather a bunch of sequences into a single FASTA file.</P>

<P>Point your browser to <A HREF="http://us.expasy.org" TARGET="_blank"><B>http://us.expasy.org</B></A>.</P>

<P>You see the home page of <B>ExPASy</B>, the Expert Protein
Analysis System. As I said earlier, ExPASy is a complete protein tool
box. With ExPASy, you can do almost any imaginable analysis or
comparison of protein sequences and structures. In my <font size="1">humble</font> opinion, Swiss sequence database tools are among the easiest ones to use. </P>

<P>Click <B>UniProt Knowledgebase (SwissProt and TrEMBL) </B> under <B>Databases</B>.</P>

<P>Read the introduction to these databases. They are high quality
protein sequence databases with abundant annotation, minimal
redundancy, and many connections to other databases.</P>

<P>Click <B>Advanced search in the UniProt Knowledgebase</B>.</P>

<P>With advance searching, you can limit your search to specific
genes and organisms, and you can search on descriptive information in
the entries.</P>

<P>Set up a search for human opsins, as follows:</P>

<UL>
   <LI>Search <B>UniProtKB/Swiss-Prot</B> only.</LI>
   
   <LI>Enter Description: opsin</LI>
   
   <LI>Organism: Choose "Human" from the pull-down menu</LI>
   
   <LI>Check "Append and prefix * to query terms.&quot; The * is a "wild
   card". You are searching for all entries that contain "opsin" as a
   whole or partial word.</LI>
</UL>

<P>Click <B>Submit</B>.</P>

<P>The page <B>UniProtKB/Swiss-Prot description</B> is your search result
page.</P>

<P>Look over the results. On 2007/01/05, this search gave 15 hits, including
  the rod pigment rhodopsin (<B>OPSD</B>), along with the three cone
pigments (<B>OPSB</B>, <B>OPSG</B>, <B>POSR</B>). There is also a
"visual pigment-like receptor peropsin", <B>OPSX</B>. Sound
mysterious. Let's find out more about it, and in the process, see a
typical Swiss-Prot entry.</P>

<P>Next to   <B>OPSX_Human</B>, click on the number (014718) in the column headed
  AC (accession code?). </P>

<P>You see the <B>UniProtKB/Swiss-Prot </B> View of entry O14718. Peruse
this entry and try to find out just what this rhodopsin-like protein
is thought to do. Under <B>Comments</B>, you'll learn that it's found
in the retina (the RPE or retinal pigment epithelium), and that it
may detect light, or perhaps monitors levels of retinoids, the
general class of compounds that are the actual light absorbers in
opsins. Also under <B>Comments &bull;Similarity</B>, you see, as
mentioned earlier, that this protein is a member of the large family
of G protein-coupled receptors (GPCRs). If you click <strong>G-protein coupled receptor
1 family. Opsin subfamily</strong>, you find a list of all purported members of this
subfamily in <B>SwissProt</B>.
Clicking &quot;View Classification&quot; produces a list of all GPCRs in <strong>SwissProt</strong>,
with a summary at the bottom indicating that the human genome alone contains
772 of them (as of 2007/01/05)!</P>
<P>Now back up
  to the <B>UniProtKB/Swiss-Prot </B>entry page for 014718, <strong>OPSX_HUMAN</strong>.</P>
<P>Under <B>References</B> click the journal citation, "Proc. Natl.
Acad. Sci. U.S.A. 94:9893-9898(1997). From the resulting page, you
can read a full article in the <B>Journal of the National Academy of
Sciences</B> (PNAS) about this protein. Like many journals, PNAS puts
full articles online just 6 to 12 months after publication.</P>

<P>Looking further down the page, you find cross-references to the
protein or its gene in other databases, predicted structural features
of the protein, and last, the sequence. Note also, at the bottom of
the page, links to a number of <B>ExPASy</B> tools listed for further
analysis of this sequence. Try some of them. For example, I just
learned in about ten seconds from <B>Compute pI/MW </B>that the
isoelectric pH (or pI) of this protein is 8.78. And I learned in no
time at all from <B>ScanProSite</B> that the sequence contains
signatures indicating that the protein is probably a G
protein-coupled receptor (no surprise, but comforting) and that it
has a retinal binding site. <A NAME=prosite></A><B>ProSite</B> is a
tool for finding signatures of function in new sequences.When you
finish playing with these powerful tools, return to your
<B>SwissProt</B> search results by use of the <B>back</B> button of
your browser. If you're lost, go back to <B>ExPASy</B> and do the
search again.</P>

<P>Now let's compare the sequences with each other. We'll use the
program <B>ClustalW</B> to make a multiple sequence alignment.</P>

<P>Scroll down the result page and check the boxes at the left of
these entries</P>

<UL>
   <LI><B>OPSB</B> (blue-sensitive opsin)</LI>
   
   <LI><B>OPSD</B> (rhodopsin)</LI>
   
   <LI><B>OPSG</B> (green-sensitive opsin)</LI>
   
   <LI><B>OPSR</B> (red-sensitive opsin)</LI>
   
   <LI><B>OPSX</B> (visual pigment-like receptor opsin)</LI>
</UL>

<P>At the top of the page, at <B>Send selected sequences to</B>,
select <B>Clustal W (multiple alignment)</B> from the menu, and click
<B>Submit</B>.</P>

<P><B>ClustalW</B> has been implemented at many web sites. This one,
located at <A HREF="http://www.ch.embnet.org/software/ClustalW.html" TARGET="_blank"><B>EMBnet.org</B></A>,
automatically receives the FASTA files from the selected entries,
allows you to make some settings of the alignment criteria, and then
does the alignment. We will just accept the default alignment
settings. First, scroll in the <B>Input Sequences</B> box and verify
that it contains five FASTA files, one right after the other. To make
them easier to identify in subsequent outputs, edit the name of each
FASTA comment line (begins with "&gt;") as follows:</P>

<UL>
   <LI>Change "sp|P03999|OPSB_HUMAN Blue-sensitive opsin (Blue cone
   photoreceptor pigment) - Homo sapiens (Human)." to "Blue".</LI>
   
   <LI>Change "sp|P08100|OPSD_HUMAN Rhodopsin (Opsin 2) - Homo
   sapiens (Human)." to "Rhodopsin".</LI>
   
   <LI>Change "sp|P04001|OPSG_HUMAN Green-sensitive opsin (Green cone
   photoreceptor pigment) - Homo sapiens (Human)." to "Green".</LI>
   
   <LI>Change "sp|P04000|OPSR_HUMAN Red-sensitive opsin (Red cone
   photoreceptor pigment) - Homo sapiens (Human)." to "Red".</LI>
   
   <LI>Change "sp|O14718|OPSX_HUMAN Visual pigment-like receptor
   peropsin - Homo sapiens (Human)." to "Peropsin".</LI>
</UL>

<P>In all cases, be sure to leave the "&gt;" in the first line of
each FASTA entry. To save some work in case something goes wrong,
select the edited contents of the <B>Input Sequences</B> box, copy
it, and paste it onto an empty word-processor page, and save the file
in text format. Name it <B>Opsins.txt</B>.</P>
<P>Do not enter your email address. </P>
<P>Click <B>Run ClustalW</B>.</P>

<P>The resulting page is called <B>ClustalW query receipt</B>. Once the task is complete, links to results files will appear. </P>
<P>Click <B>clustalw (aln)</B>, the link to your multiple-sequence alignment file. </P>
<P>You see the typical <B>ClustalW</B> alignment file, showing our
five protein sequences aligned to maximize identical and similar
residues. Below each line of five sequences are symbols to show the
extent of similarity among the sequences. An asterisk (*) means that
the same residue is always (that is, for all of <U>these</U>
sequences) found at that location; for example, the first asterisk
marks a location where only N (asparagine) is found. Colon (:) means
that all residues at this location are very similar; for example, the
first colon is where only F (phenylaline), I (isoleucine), and L
(leucine) -- residues with large, nonpolar sidechains -- occur.
Period (.) means somewhat similar residues; for example, at the first
period, serine, threonine, and glutamine occur -- all polar, but
varied in size. If there is no mark then the residues at that
location display no predominant common properties.</P>

<P>Once more, as a safety measure, copy this alignment to your
clipboard, and paste it onto an empty word-processor page. Then save
the file in text format. Name it <B>OpsMSA.txt</B>. Remember that it
is still on your clipboard, for pasting at our next stop. This
multiple sequence alignment is one type of input you can use to make
a phylogenetic tree.</P>
<H2>What does the family tree of human opsins look like?</H2>

<P>Point your browser to <A HREF="http://bioweb.pasteur.fr/seqanal/phylogeny/phylip-uk.html" TARGET="_blank"><B>http://bioweb.pasteur.fr/seqanal/phylogeny/phylip-uk.html</B></A></P>

<P>This is one home of the program Phylip, One of the most rigorous
tools for constructing phylogentic trees from aligned sequences.</P>

<P>Under Proteins, next to protdist, click "advanced form."</P>

<P>You are about to run <B>protdist</B>, a program that computes the
"distance" of sequences from each other. These so-called <B>distance
matrices</B> will be used by <B>Phylip</B> to construct your
tree.</P>

<P>Enter your email into the top box.</P>

<P>In the alignment file box, paste your mutiple sequence alignment
from ClustalW.</P>

<P>Click "Bootstrap Options" to move down this page, and then make these settings:</P>

<UL>
   <LI>Check the box for "Perform a bootstrap before analysis"</LI>
   
   <LI>Enter any odd number for a seed</LI>
   
   <LI>Enter 100 replicates</LI>
</UL>

<P>Scroll back up to  the top of the page, and click "<B>Run</B> <B>protdist</B>".</P>

<P><B>protdist</B> constructs distance matrices by a process called
"bootstrapping". Bootstrapping is a bias-reducing procedure in which
<B>protdist</B> builds an alignment of pseudosequences by picking
residue positions at random and stringing the residues at those
positions together until the sequence is the same length as the
original ClustalW alignment. From this pseudosequence alignment,
<B>protdist</B> determines the relative number of sequence difference
among the five proteins, as determined from a random sampling of
their sequences. The result of the process is a called distance
matrix, and you will see it soon. This process is repeated, 100 times
in our case, to make 100 distance matrices. The tree we will
ultimately produce represents a consensus of the 100 matrices.</P>

<P>There may be a delay of a few minutes before the result page
appears. If the server is busy, you may be informed that results are
being sent by email. If so, check you email in two or three minutes.
You will receive five messages. The one whose subject starts with &quot;access&quot; contains a
link to your result page. Click the URL, or paste it into your
browser and press &lt;return&gt; to open the page.</P>
<P>On the <B>Phylip: protdist</B> page that results, click
<B>outfile</B> to see the output from <B>protdist</B>. The file
contains 100 matrices containing numbers that represent the relative
number of differences among the five sequences. Each matrix has the
sequence names in the first column, and you should imagine that these
sequence names are also the headings for the remaining columns. The
number at the intersection of the row Blue and the column with the
imaginary heading Peropsin gives the relative magnitude of the
sequence differences between the blue opsin and peropsin. The
matrices have zeros on the diagonal because each pseudosequence is
identical to itself.</P>

<P>Click the <B>Back</B> button of your browser to return to the
<B>Phylip: protdist</B> page.</P>

<P>On the first pull-down menu of the <B>Phylip: protdist</B> page,
pick "<B>neighbor</B>." Read the menu carefully: don't pick
"<B>w</B>eighbor".</P>

<P>Click "Run the selected program on outfile" to run Phylip with the
output file of matrices you just examined. You are running a
procedure called "neighbor joining" to construct an evolutionary
tree.</P>

<P>On the <B>Phylip: neighbor</B> page that appears next, beside
"Distance method?" Make sure "Neighbor-joining" is selected.</P>

<P>Click "Bootstrap options" to move down the page, and then make these settings:</P>

<UL>
   <LI>Check "Analyze multiple data sets (M)"</LI>
   
   <LI>Enter 100 data sets (using all of the replicates from
   <B>protdist</B>)</LI>
   
   <LI>Enter an odd number for a seed</LI>
   
   <LI>Check "Compute a consensus tree"</LI>
</UL>

<P>&nbsp;Scroll down to "Other options".</P>

<P>This entry area gives you the option of designating an
<U>outgroup</U> for the root of your tree. An <U>outgroup</U> is the
sequence you think is most distant from the others, possibly the
common ancestor of all. We don't know that in this case, so leave the
default of 1.</P>

<P>&nbsp;At the top of the page, click "Run neighbor".</P>

<P>&nbsp;The resulting files are</P>

<P><B>outfile.consense</B> -- your tree, in a text file, and
<B>outtree.consense</B> -- your tree in a format used by
tree-printing programs.</P>

<P>Click on <B>outfile.consense</B> to see the tree.</P>

<P>Scroll down to the bottom of this file to see the consensus tree.
This tree is "unrooted", meaning that we do not know the ancestor of
all these sequences. We learn from this tree which sequences are most
alike and which are most different. We also learn how often the
connections of this tree were made the same way in the 100 trees made
from those 100 difference matrices. The numbers on the branches
indicate the number of times that partition of the species into the
two sets separated by that branch occurred among the 100 trees. For
example, the separation of Red and Green from the other three,
indicating that Red and Green are more similar to each other than to
the other three, occurred in all 100 trees. The separation of Blue
and Peropsin from the other three occurred in only 62 of the 100
trees. In the other 38 trees, Rhodopsin and Peropsin were separated
from the other three. (Can you extract this information from this
file?) In the tree branching shown, the majority rules, and the
results of  38 of the trees are discarded.</P>
<blockquote>
  <p>Note: Your results may be slightly different from mine. Because of the random choices made in constructing the tree, the percentages
    in the paragraph above my vary. I have gotten as high as 82% consensus on
    the separation of Blue and Peropsin from the other three. </p>
</blockquote>
<P>You can save this file by selecting all and pasting it into a
word-processor document. Call it <B>outfile_consense.txt</B>.</P>

<P>Return to the <B>Phylip: neighbor</B> page and click on
<B>outtree.consense</B>. This is your tree in Newick format, which is
widely used by tree-printing programs like <B>Phylodendron</B>. Let's
use this program to give us a tree in attractive graphics, rather
than text. Select:All and copy this little file to your clipboard. </P>

<P>Point your browser to <a href="http://iubio.bio.indiana.edu/treeapp/treeprint-form.html" target="_blank"><strong>http://iubio.bio.indiana.edu/treeapp/treeprint-form.html</strong></a>.</P>

<P>Paste the contents of your <B>outtree.consense</B> file into the
  Tree Data box. Select <B>Phenogram</B> from among the Tree Styles.
  From the menu at <B>Extra Options</B>, <B>Output</B>, select <strong>GIF image </strong> (not Gif Image map) format
  for your output file. Give your tree a title, such as "Human
  Visual Opsins and Opsin-Like Proteins". Set Tree Growth to <strong>horizontal</strong>. Finally, click <B>Submit</B>.</P>
<P>Your GIF-format tree appears in your browser window. To keep it,
chose <B>Save As ...</B> from the <B>File</B> menu. Call the file
OpsinTree.gif. My tree looks like this:</P>

<CENTER>
  <img src="images/OpsinTree.gif" width=590 height=590 x-claris-useimagewidth x-claris-useimageheight align=bottom>
</CENTER>

<p>Look again  at this <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=pubmed&dopt=Abstract&query_hl=1&list_uids=2937147" target="_blank">abstract</a> in the light of your tree. </p>
<H2>What is the structure of an opsin?</H2>
<P>By now, I'm particularly curious about peropsin, but it's not
likely that the structure of a recently discovered protein of unknown
function has been determined. But it <U>is</U> likely that all opsins
are similar in structure, so let's see is we can find an opsin in
<U>the</U> database for macromolecular structures, the <B>Protein
Data Bank</B> (<B>PDB</B>). It will give us an idea of what kind of
thing an opsin is.</P>

<P>In fact, the PDB does not contain molecular structures at all. Is
is better to say that it contains <U>models</U> of macromolecules.
These models are interpretations of data from one of the two main
methods of macromolecular structure determination: <A HREF="../SPVTut/text/STut09aTN.html" TARGET="_blank"><B>x-ray
crystallography</B></A> and <A HREF="../SPVTut/text/STut09bTN.html" TARGET="_blank"><B>NMR
spectroscopy</B></A>. When researchers determine the structure of a
macromolecule, they deposit a file containing the three-dimensional
coordinates of all the atoms in the model. This <B>coordinate
file</B> -- along with an online molecular graphics tool (like the PDB's Jmol Viewer) or
a computer graphics program like <A HREF="../SPVTut/index.html" TARGET="_blank"><B>DeepView</B></A> -- are all that you need to see and study the molecule
on your computer. Next we will retrieve a model from the PDB and view
it with an online graphics tool. We'll also visit the home of a
topnotch computer graphics program that you can download FREE and use
on your home computer.</P>

<P>Point your browser to <A HREF="http://www.rcsb.org/pdb/" TARGET="_blank"><B>http://www.rcsb.org/pdb/</B></A>.</P>
<P>The PDB home page contains a simple search box at the top. You can search for models using simple keywords or PDB
ID codes. An PDB code has four characters, like <B>1CYO</B>. How
would you ever know a model by its code? When a new structure is
published, the authors usually give the PDB code in the last
reference of the bibiography. With that code, you can go straight to
the model you want to see. But more often, your question, like ours,
is more general. For such cases, PDB also provides forms for more
sophisticated searches. For now, let's just see if any opsin models
are availalble. Type "opsin" into the search box, make sure the <strong>PDB ID or keyword</strong> is selected, and click <strong>Search</strong>.  </P>

<P>On 2007/01/05, this search returned only one model, which is quite puzzling, because a search for &quot;rhodopsin&quot; returns 48 models. So this tells me that the quicky (quirky?) search tool at the PDB still needs some work. But this shortcoming is a gift for us right now. We've bagged a live opsin of known structure ; the PDB contains only models derived experimentally&mdash;either by x-ray crystallography or NMR spectroscopy. Lets take a look at it.</P>
<P>Click the PDB file code <strong>1GZM</strong>  above the tiny image of the model.</P>
<P>You have come to the <strong>Structure Summary </strong> page for this model, which is its home page at the PDB. This page is connected to just about everything you could possible do with this model. At the PDB, your first goal is always to get to the Structure Summary page for the model you are seeking. </P>
<blockquote>
  <p>NOTE: <strong>Structure Summary</strong> does not exactly jump out at you on this page. It's the tab selected over the main part of the entry. Those tabs should be more prominent&mdash;they are what distinguishes each of the important pages for a model. If you want to know where you are in the PDB, look at two sets of tabs at the top of the page. The other tabs open LOTS of information about this model, but we will stick with structure.</p>
</blockquote>
<p>In the left column of all PDB pages, you find a set of nested menus. Click <strong>Display Molecule</strong> to open the PDB display options. If you already own or use one of the listed viewers, like the free program <strong>DeepView</strong>, you are in business. Click your viewer to download the model and view it in a familiar environment. But let's assume that you are new to all this, and use a handy viewer that works in your browser.</p>
<p>Click <strong>Jmol Viewer</strong>. Assuming that your computer has up-to-date Java software, your browser will load the viewer, and it will load the file <strong>1GZM</strong>. Your should see  models of two rhodopsin molecules&mdash;with backbones shown as ribbon-like cartoons, one green, one blue&mdash;and several ball-and-stick models of smaller molecules. Is rhodopsin a dimer? No, but in the crystals of rhodopsin from which this model was derived contained two rhodopsin molecules per<strong> asymmetric unit</strong> (the smallest portion from which the entire crystal is constructed. PDB files usually show the full contents of the asymmetric unit. If more than one molecule is present, they are referred to as <strong>chains</strong> in the model. </p>
<blockquote>
  <p><strong>NOTE ON VIEWERS</strong>: The viewer on display is the widely used <strong><a href="http://jmol.sourceforge.net/" target="_blank">Jmol</a></strong>, which you will find in use as a molecular viewer at many web sites. If you take time to get to know this viewer fairly well, you will get more out of the many sites that use it. Like most of the other viewers listed at PDB, Jmol is quite limited in its capacity for analysis of protein structure.<br>
  <strong>In my </strong><font size="1">humble</font><strong> opinion, the most powerful  protein-analysis tool listed at PDB is <a href="http://www.usm.maine.edu/spdbv/" target="_blank">DeepView</a>.</strong>  DeepView may be the only protein-structure viewing and analysis tool you will ever need. You will learn about it in the <a href="#encore">Homology Modeling</a> section, below.</p>
</blockquote>
<p>Here are some other things you can do to get to know models in a Jmol frame (to get back to the original rendition,  reload the page): </p>
<UL>
  <LI>Click/drag (left button if you have more than one) on the image to rotate the structure. You should be
    able to tell that is has a lot of alpha helix.</LI>
  <LI>Hold down <strong>option</strong> (for Macintosh; <strong>alt</strong> for Windows) and click/drag to zoom in (drag towards you) or out (drag away) or the rotate the model in the plane of the screen (drag left or right).</LI>
  <LI>Hold down <strong>ctrl</strong> (or right-click) the image: up pops a set of menus, and if you browse around on them, you'll see that there is much more to Jmol. Let's try just a couple of things to give you some general ideas. </LI>
  <LI>Using the pop-up menus,<strong> Select:Protein:All</strong><br>
  Nothing appears to happen. You have selected part of the model (the protein part, but not the small molecules). Now let's change it.</LI>
  <LI><strong>Color:Cartoon:By Scheme:Secondary Structure</strong><br>
  The cartoons become red (well, bright pink) for alpha helix, and yellow for beta sheet. I'll bet you had not noticed the beta sheet in the models before. Look one of the chains  over carefully to get a feeling for its structure. How many helices are present? How many strands of bet a sheet? Are the strands parallel or antiparallel? </LI>
  <LI>Do you know how to view stereo pairs? (If not click <a href="../0Help/StereoView.html" target="_blank"><strong>HERE</strong></a> to learn how.) Then <strong>Render:Stereographic:(choose your favorite mode)</strong>. Due to a bug in Jmol, pick the opposite mode from what you really want. Cross-eyed viewing gives you wall-eyed, and vice versa. Now you can see the models as solid object with convincing depth. If you are ever going to do anything serious with protein structure, you'll need to find a way to view them in 3D.</LI>
  <LI>Work in stereo or not, as you prefer. Clear the display: <strong>Select:None</strong>; then <strong>Select:Display Selected Only</strong>. The display goes blank; nothing is selected and you are displaying only the selection (very logical!).</LI>
  <LI><strong>Select:Protein:All</strong> (means select both backbond and sidechains). Then <strong>Render:Scheme:CPK Spacefilling</strong>. The protein portion is now show as a spacefilling model. In this rendition, you get a good idea of the overall shape of the protein. Unfortunately, the Jmol menu does not allow you to color the two chains separately or get rid of one of them. </LI>
  <LI> <strong>Render:Scheme:Wireframe</strong>. Now you see all of the protein parts of this model in wireframe. This is not as impressive as some other schemes, but is actually the most useful when you start exploring models in detail, because the wires do not hide each other like ball and sticks or spacefilling models. </LI>
</UL>
<p>To learn more about Jmol, consult the help links at PDB below the display. You can also find extensive help for all viewers listed there. </p>
<h3>Finding Opsin Homologs in the PDB</h3>
<p>Now, let's try to find other models in the PDB that are homologous to the human opsins. You will ask the PDB, &quot;List all models whose sequences can be aligned with that of human red opsin, in order of sequence similarity.&quot; In PDB terminology, the red opsin sequence is the <strong>query</strong>, and similar models found are called <strong>subjects</strong>. </p>
<p>First, open your query file <strong>protred.txt</strong> (FASTA sequence of red human opsin), and copy the sequence portion only to the clipboard; omit the comment line that begins with &gt;.</p>
<p>At the top right of any PDB page, click <strong>Search</strong>. From the list of search types, click <strong>Sequence</strong>. On the resulting page, click the button next to<strong> use Sequence</strong>, and paste your red opsin sequence into the box just below. Not that the search tool is your new friend <strong>Blast</strong>, and that a <strong>E cut-off value</strong> of 10 is given as a default. This will limit the results to subjects of quite high similarity. Higher E-value cutoffs will be less restrictive, and give more hits or subjects. Click the <strong>search</strong> button. The search tool is now looking for PDB models whose sequences are similar to the human red opsin sequence. </p>
<p>On 2007/01/07, I got 14 subjects, or 14 PDB models whose sequences are homologous to the search sequence.  Each is listed with an <strong>E-value</strong> (not the same as the E cut-off value on the previous page), which is the probability that the sequence similarity between query and subject is a coincidence. The first result or subject is PDB model <strong>1F88</strong>, a model of bovine rhodopsin. The E-value is 2.4 x 10<sup> -76 </sup>. In other words, while the probability that a coin flip and your call will agree just by chance is 0.5, the probability that the similarity between human red opsin and bovine rhodopsin is just a chance occurence is </p>
<p align="center">0.00000000000000000000000000000000000000000000000000000000000000000000000000024,</p>
<p align="left">which means, to any sane biologist, that these two molecules descended from a common ancestor. There is no chance that, in the history of the universe, two proteins could arrive at sequences this similar by chance. This also means that the structure of the bovine rhodopsin is a sure bet to be very similar to that of the human red opsin, whose structure is unknown (if if were known, this search would have found it). </p>
<p align="left">Now look down the list of the models you found. They are all models of the same substance: bovine rhodopsin. The only molecule of known structure that is highly homologous to the human red opsin is this one. As you know by now, these molecules are in the family of G-protein coupled receptors, membrane proteins whose structures are proving to be quite elusive. </p>
<p align="left">Use the results page  to answer these questions about the comparison between human red opsin and the bovine rhodopsin in PDB 1F88:</p>
<ol>
  <li>How many corresponding residues, and what percent of the residues, do the two proteins have in common (exact matches)?</li>
  <li>How many and what percent of corresponding residues are similar in chemical properties?</li>
  <li>How many gaps did the alignment program introduce, and how many residues in each gap, to get best alignment between human red opsin and 1F88?</li>
  <li>Find the longest string of exact matches between the two proteins. How many matches does it contain, and what are the beginning and ending residue numbers?</li>
</ol>
<p align="left">Now you know how to search the PDB for models whose sequences are similar to a target or query sequence. Structural biologists use such searches when they have a new protein sequence and want to know its structure. If the structure is known, this search would find it. If not, any hits with high sequence similarity can tell researchers the overall fold of the newly discovered protein. </p>
<H2>Where Do You Go From Here?</H2>

<P>Well, there you have a basic introduction to bioinformatics. With
the tools you've tried out, you can explore the vast stores of
genetic and structural information available on the Internet. Every
page we've visited has many more links to other tools. You can figure
out a lot just by visiting them and playing around, and there's
usually built-in help and tutorials. I hope this tutorial spurs you
to learn more about how to use bioinformatics in your explorations.
If you want to learn more about using databases in <U>structural</U>
investigations of proteins, see the <B>Encore!</B> next.</P>

<H2><A HREF="#test">Test Yourself</A></H2>

<P>Click <A HREF="#test"><B>HERE</B></A> for a series of questions to
answer using the tools of this tutorial. These questions make an
appropriate assignment for assessing your (or your students') ability
to use the tools of bioinformatics.</P>
<HR>

</P>

<H2><a name="encore"></a>Encore! Exploring Protein Structure by Homology Modeling</H2>

<P>How would you like to create a model of that mysterious <B>peropsin</B> we found among sequences
homologous to the human opsins, and to find out whether it contains a
binding site for retinal? This is a reasonable goal, if you are
willing to learn how to use a more powerful tool for protein viewing
and analysis. </P>

<P><B>Jmol</B> is a great structure viewer for beginners, but
among free (my favorite price) viewers, there are much more sophisticated, powerful, and versatile, viewers than Jmol. My all-time favorite is
<B><a href="http://www.usm.maine.edu/spdbv/" target="_blank">DeepView</a></B> (also known as <B>Swiss-PdbViewer</B>). The
encore section of this tutorial requires <B>DeepView</B>. We will use
it to look more closely at bovine rhodopsin, including exploring its
bound retinal molecule. We will also probe more deeply into that
mysterious <B>peropsin</B> we noticed among the sequences similar to
the human opsins. In fact, we will make a model of peropsin, by threading its sequence onto the three
dimensional structure of bovine rhodopsin. (You can't do <U>that</U>
with any other free protein viewer!!)</P>
<P>This process is called <B>homology modeling</B>, and it can
provide sort of an educated guess about the structure of a protein
(the <B>target)</B> from its sequence, as long as one of more
structures of homologous proteins (<B>templates</B>) are available in
the <B>Protein Data Bank</B>. How good a guess? In short, the more
similar are the sequence and function of your target and your
template(s), the better the model. So in our case, if we can find a
template of similar sequence and function (a homologous
retinal-binding opsin, that is), we should be able to get a decent
model of peropsin, and see whether it's really feasible to think that
it binds retinal. We might even be able to tell whether it binds
retinal covalently or noncovalently, depending on whether an
appropriate covalent-binding residue is suitably located in the
homology model. At the very best, however, you cannot learn fine
details of side-chain conformations, as you can from determining
protein structure by X-ray crystallography or NMR. But homology
models can be useful for preliminary exploration, and might also
point to useful target residues for chemical analyses of structure,
such as site-directed mutagenesis.</P>
<blockquote>
  <p>NOTE: Creating a model is loosely called <strong>structure determination</strong>, but in fact, we never really determine the structures of molecules. What we call structure determination is really creating molecular models that fit data or fit what we know about a substance. A model based on data from x-ray crystallography or spectroscopy is called an <strong>experimental model</strong>. A model not based on experimental data is a <strong>theoretical model</strong>, and a theoretical model based on homology to one or more experimental models is a <strong>homology model</strong>. </p>
</blockquote>
<P>If you already have and know how to use <B>DeepView</B>, just
continue with this tutorial. If not, take time to learn
<B>DeepView</B> from this <A HREF="http://www.usm.maine.edu/~rhodes/SPVTut/index.html" TARGET="_blank"><B>Beginner's
Tutorial</B></A>, and then return here.</P>

<H2>Exploring Rhodopsin and Peropsin</H2>

<P>I'm assuming that you know how to use <B>DeepView</B>, also known
as <B>Swiss-PdbViewer</B>, and how to download models (called
coordinate files) from the <B>Protein Data Bank</B>. If not, please
take sections 1-11 of the <A HREF="http://www.usm.maine.edu/~rhodes/SPVTut/index.html"><B>DeepView
Tutorial</B></A> and call me in the morning -- er, I mean -- come
back to this section. The conventions used in the rest of this
tutorial are the same as those in the DeepView Tutorial.</P>
<h3>Quick Homology Models (but you don't learn anything).</h3>
<P>Many genomics databases include homology models generated by automated servers, or quick and easy ways to submit a sequence or database entry for modeling. You get a model quickly, but you don't learn much about the process. Let's get a peropsin model by this route, and then we will do it by a more hands-on method that helps you to see a lttle bit about what the automated servers are doing. Then later on, if an automated server gives you a poor model (one that conflicts with things that other research has told you, or one with obvious errors, like charged residues in its interior), you will know how to intervene in the modeling in hopes of getting a better model (one that agrees with reliable evidence about the structure). </P>
<P>Go to <A HREF="http://us.expasy.org/" target="_blank"><B>http://us.expasy.org/</B></A>,
select <B>UniProt Knowledgebase (Swiss-Prot and TrEMBL)</B>, and enter <B>OPSX</B>, the
protein name for peropsin, in the <B>Search Swiss-Prot and TrEMBL
for</B> box. Click <B>Go</B>. On the result page, click <B>OPSX
Human</B>. You are now at <B>UniProtKB/Swiss-Prot </B>entry <B>O14718</B>. Look over the wealth of information that makes up the annotations for this entry. Through this page, you are connected to the information about this gene is databases all over the world of structural biology.</P>
<P>After looking over this page, and perhaps visiting some other sites to see what thay have on OPSX, scroll to the bottom and click on <strong>Submit a homology modeling request to SWISS-MODEL</strong>.  </P>
<P>At <strong>Swiss-Model</strong>, you see that the server has already started filling out the form for you, by displaying the accession number <B>O14718</B> to the top. Fill in your email address and your name. Click <strong>Submit to SWISS-MODEL</strong>. You just asked <strong>Swiss-Model</strong> to make a homology model for you. The server will search (using pBlast) for templates with sufficient sequence similarity; gather the templates and thread your model onto them, search databases of protein loops to build parts that do not match well with the templates; to build loops similar to those in databases, if possible; to build remaining loops that are mostly guesswork, but with reasonable conformations; and finally, to optimize the model by minimizing its conformational energy.</P>
<blockquote>
  <p>NOTE: You can also paste any FASTA sequence into a form for a homology modeling request by going to <a href="http://swissmodel.expasy.org/" target="_blank"><strong>http://swissmodel.expasy.org/</strong></a> and clicking on <strong>First Approach Mode</strong> under <strong>Modeling Requests</strong>. </p>
</blockquote>
<P><strong>Swiss-Model</strong> will send your model, along with the template models, to you by email. The file will be all ready for viewing in <strong>DeepView</strong>, and it will include all templates used in the modeling. </P>
<P>&nbsp;</P>
<h3>Homology Modeling Step by Step </h3>
<P>Now let's make a peropsin homology model by the do-it-yourself (well, partically, at least) method. This will give you a much clearer picture of what's going on when an automated server makes a model for you.</P>
<P>First, we'll need a FASTA file for human peropsin. Go to <A HREF="http://us.expasy.org/" target="_blank"><B>http://us.expasy.org/</B></A>,
  select <B>UniProt Knowledgebase (Swiss-Prot and TrEMBL)</B>, and enter <B>OPSX</B>, the
  protein name for peropsin, in the <B>Search Swiss-Prot and TrEMBL
    for</B> box. Click <B>Go</B>. On the result page, click <B>OPSX
      Human</B>. Near the bottom of the resulting <B>UniProtKB/Swiss-Prot </B>entry,
  click on <B>O14718 in FASTA format</B>. Save the resulting text file
  as <B>peropsin.txt</B>. (I selected and copied the file from the
  browser display, pasted it into a new word processor file, and saved
  it in <B>text</B> format).</P>
<blockquote>
  <p>NOTE: You can also paste any FASTA sequence into a form for homology modeling by going to <a href="http://swissmodel.expasy.org/" target="_blank"><strong>http://swissmodel.expasy.org/</strong></a> and clicking on <strong>First Approach Mode</strong> under Modeling Requests. </p>
</blockquote>
<P>Start <B>DeepView</B>. Then <B>Cancel</B> the initial dialog box,
which is expecting you to load a PDB file. Your peropsin file is in
FASTA format, so you have to load it by a different procedure.</P>

<P><B>SwissModel: Load Raw Sequence to Model...</B><BR>
A reminder about <B>DeepView Tutorial</B> format: the instruction above this line
tells you to select the command <B>Load Raw Sequence to Model..</B>
from the <B>SwissModel</B> menu.</P>

<P>The resulting dialog box looks just like one for loading a PDB
file, but now DeepView is looking for a sequence file in FASTA
format. Navigate to your <B>peropsin.txt</B> file, select it, and
click <B>Open</B>.</P>

<P>DeepView displays the sequence of peropsin as an alpha helix. This
is a compact way to get the 337 residues onto the screen. First,
let's see what <B>Prosite</B> (<A HREF="#prosite">above</A>) has to
say about the nature of this protein. <B>DeepView</B> contains an
internal link to <B>ProSite</B> through the command <B>Edit: Search
for ProSite Patterns</B>. Because <B>ProSite</B> works on sequence
only, we don't need to know anything about structure to see what
signatures of protein function <B>ProSite</B> can find.</P>

<P><B>Edit: Search for ProSite Patterns</B></P>

<UL>
   <LI>If <B>DeepView</B> returns an error message, your
   <B>DeepView</B> installation may not have included the
   <B>ProSite</B> data. Go <A HREF="http://www.usm.maine.edu/spdbv/text/getpc.htm" TARGET="_blank"><B>HERE</B></A>
   and follow instructions for this installation. You'll probably
   have to restart <B>DeepView</B>, but not your computer, after
   installing.</LI>
</UL>

<P>This command elicits a small window listing signatures or patterns
found in peropsin. Note the last two entries, indicating that
<B>ProSite</B> recognizes patterns indicating that peropsin is a G
protein-coupled receptor (GPCR) with a retinal-binding site. Click the black
descriptions of sites to highlight the residues that <B>ProSite</B>
recognized. Click the  red <B>ProSite</B> entry numbers to
download a full description of the <B>ProSite</B> documentation for
recognizing a specific type of protein. The entry for GPCRs contains a full list of
sequences from this family in <B>SwissProt/TrEMBL</B>. It's a huge list. The entry for retinal binding sites contains a list of sequences that appear to contain this site,
followed by a description of the types of proteins in which this
pattern is found. All entries end with the specific patterns that <B>ProSite
</B>looks for to a particular type of protein. Study these lines to find out what <strong>ProSite</strong> is looking for when it scans a sequence. </P>

<P>Now you know that there is likely to be a retinal-binding site in
peropsin. Let's see if we can find it.</P>

<P><B>SwissModel: Find Appropriate ExPDB Templates</B></P>

<UL>
   <LI>If <B>DeepView</B> returns an error message, you need to
   <B>Preferences: Swiss-Model</B>, and make the following settings:
   
   <UL>
      <LI><B>Modeling Server</B>:
      http://swissmodel.expasy.org/cgi-bin/sm-submit-request.cgi</LI>
      
      <LI><B>Template Server</B>:
      http://swissmodel.expasy.org/cgi-bin/blastexpdb.cgi</LI>
      
      <LI>Also specify your name, an email address where you can
      receive large files, and your preferred browser.</LI>
      
      <LI>Then repeat the command <B>SwissModel: Find Appropriate
      ExPDB Templates</B></LI>
   </UL>
   </LI>
</UL>

<P><B>DeepView</B> starts up your preferred browser and completes a
form containing the peropsin sequence in FASTA format. Click
<B>Submit</B> to conduct a search for proteins of <U>known</U>
<U>three</U>-<U>dimensional</U> <U>structures</U> that are homologous
to peropsin. The <B>Swiss-Model Template Server</B> returns a list of
model that should server as suitable templates for making a model of
peropsin. These models are in a special structure database called
<B>ExPDB</B>, for <B>ex</B>cerpts of <B>PDB</B> models. An
<B>ExPDB</B> entry is usually one domain from a multidomain protein,
or one chain from a <B>PDB</B> model that contains more than one
chain.</P>

<P>On 2007/01/08, I got sixteen possible templates, all of which were
various models of bovine rod rhodopsin. As of this date, rhodopsin
from the rod cells of good ol' <I>Bos taurus</I> (whence "Come,
Bossy!") is the only visual pigment of known structure. The first
recommended template, <B>1f88A</B>, is chain A from <B>PDB</B> entry
1F88, a model determined by X-ray crystallography. The <B>BLAST</B>
score for <B>1f88A</B> indicates that the odds that the sequencs of
of peropsin and model <B>1f88A</B> are similar by chance is
9x10<SUP>-37</SUP>, which implies strong reason for similarity.
Biologists would say that the only reasonable explanation for such
similarity is evolution: peropsin and bovine rhodopsin evolved from a
common ancestor.</P>

<P>Because all the potential templates are the same protein, we will
work with just one template, <B>1f88A</B>. Click its file name in the
<B>download ExPDB</B> column of the Template Selection table.
Depending on how your browser and <B>DeepView</B> are set up, the
file might open automatically in <B>DeepView</B>, or you may have to
specify <B>DeepView</B> as the helper application, or you may have to
save the file to your desktop. If you've been using <B>DeepView</B>
before, you've probably worked out the best way to handle <B>PDB</B>
downloads. Use your favorite method, and then open the file in
<B>DeepView</B> (by <B>File: Open PDB File</B>, if you saved the file
to your desktop).</P>

<P>The model should appear near your large alpha helix of peropsin
residues.</P>

<P><B>Wind: Sequences Alignment<BR>
DeepView</B> displays the sequences of both models, right-justified
-- that is, aligned at their N-terminal residues. Now for some real
magic.</P>

<P><B>Fit: Magic Fit<BR>
</B>In the blink of an eye, it appears that your peropsin helix is
gone. But its sequence has been aligned with that of <B>1f88A</B>,
and each its residues has been superimposed upon the residue with
which it aligns sequentially. In short, the peropsin chain has been
<B>threaded</B> onto the 3-dimensional model of bovine rhodopsin.</P>

<P><B>&lt;control-tab-tab-tab...&gt;<BR>
</B>Holding down the<B> control</B> key and pressing <B>tab</B> tells
<B>DeepView</B> to "blink" between the structures. You are seeing,
alternately, <B>1f88A</B> and the target peropsin homology model.
Your target might still be colored gray, with a <B>ProSite
</B>pattern highlighted in cyan; if so, display the target, and
<B>Color: CPK</B> to give it "normal" color. As you blink back and
forth between the models, you might see some strange things about the
target. For example, look for a very long peptide bond at one end
(between residues 315 and 316). Such features are obviously not
chemically realistic; they just represent the best <B>DeepView</B>
could do at aligning the sequences. Such a problem suggests that, in
this region, the two proteins are not <U>structurally</U> very
similar. But the overall alignment seems to work well.</P>

<P>In the <B>Sequences Alignment </B>window, notice that the
sequences are not longer left-justified; they are aligned by
homology. Straight vertical lines connect residues that are identical
in the two models, two dots connect quite similar residues (like
valine and leucine, both bulky nonpolar), one dots connects less
similar pairs (serine and glutamine, both polar), and dissimilar
pairs are unconnected (glycine and isoleucine). To see the full
alignment conveniently, click the little document icon at the left
end of the <B>Sequences Alignment</B> window. You can save this
diagram as a plain text file for printing (<B>File: Save: Sequence
Alignment</B>).</P>

<P><B>Wind: Layer Infos<BR>
</B>This window gives information about the display property of all
models currently loaded. With what we are doing, it's a handy way to
be more quantitative about similarities. According to the <B>Sel</B>
column (far right), 299 residues are currently selected. This is the
number of residues in the aligned regions of the two models. Blink to
display <B>1f88A</B>. <B>Select: All</B>. The <B>Sel</B> column tells
you that <B>1f88A</B> contains 346 residues. Blink to display
peropsin. How many residues does it contain?</P>

<P>With the peropsin model remaining on display, <B>Select: aa Identical to
ref. Structure</B>. <B>DeepView </B>tells you that 88 residues of
peropsin align with identical amino-acid residues of <B>1f88A</B>.
This is 88/346 or only 25% sequence identity. <B>Select: aa Similar
to ref. Structure</B>. The percentage of aligned residues that are
chemically similar (double dots in the <B>Sequences Alignment
</B>window) is 182/346 or 53%. If two proteins show more than about
35% sequence similarity under best alignment, then they are almost
certain to be of similar structure.</P>

<P><B>Select: aa Making Clashes<BR>
</B>A large number of residues in the homology model are trying to
occupy the same space. This is obviously unrealistic. Make the model
more feasible by using <B>Tools: Fix Selected Sidechains: Quick and
Dirty</B>. Again, <B>Select: aa Making Clashes</B> to see if
<B>DeepView</B> has improved these problems. Press <B>&lt;return&gt;
</B>to show only the selected residues. Pink dotted lines reveal the
clashes. Some of them look pretty serious. </P>

<P>You can solve all of the problems of this primitive model by
sending it off to the <B>Swiss-Model</B> server for optimization.</P>

<P><B>SwissModel: Submit Modeling Request<BR>
</B>Your browser appears again, this time with a new form. The form tells you that a project file has been created, and gives you its location (on a Macintosh, the location is /Applications/SPDBV_3.9b1.01_univ/temp/, and the file name is SwissModelRequest-xxx.<strong>spd</strong>. On the form, under <B>Your Swiss-Model
project file can be found in:</B>, click <B>Browse</B>,
and navigate to the file listed on the form. This is the project file that your browser will
send to <B>Swiss-Model</B> for optimization. (The other file in the same location, SwissModelRequest-xxx.<strong>htm</strong>  is just the web page form you are viewing. Don't select this file by accident.) Complete
the form by checking your email information, select
<B>Swiss-PdbViewer mode</B> for the format of your final model, and
<U>uncheck</U> the option of getting a <B>WhatCheck report</B> of the
final model. Then click <B>Send Request</B>.</P>
<blockquote>
  <p>NOTE: After clicking the <strong>Browse</strong> button on the form, you could select and send ANY DeepView project file that you created by <strong>Fit:Magic Fit</strong>ting a FASTA sequence on to one or more templates. So your procedure to this point does not have to follow the tutorial exactly. But the project file must contain one target sequence, loaded as the first model, and one or more templates loaded afterward. If you are working on several modeling projects, you might want to move a copy of SwissModelRequest-xxx.<strong>spd</strong> to a working folder for this model, and use the same folder for the results files that you will receive. </p>
</blockquote>
<P><B>ALTERNATIVE REQUEST METHOD</B>: If you have problems sending
the project file by way of your browser, you can submit it directly
to <B>Swiss-Model</B>. Go to <A HREF="http://swissmodel.expasy.org/"><B>http://swissmodel.expasy.org/</B></A>.
Under <B>Modeling Requests</B>, click <B>Project (optimise) mode</B>.
You will see a form similar to the one the <B>DeepView</B> creates.
Fill in your email address, name, and a project title. Click
<B>Choose File</B>, navigate to your project file,
and choose it. Select <B>Swiss-PdbViewer mode</B> for the format of
your final model, and <U>uncheck</U> the option of getting a
<B>WhatCheck report</B> of the final model. Then click <B>Send
Request</B>.</P>

<P>By either submission method, your browser should return a message
indicating successful uploading of your project file (385787 bytes
for this project), and provide further information. You will receive
your optimized model by email. It may take several hours. Once you
receive several email files from <B>Swiss-Model</B>, you are ready to
resume this tutorial.</P>
<P><FONT COLOR="#009900"><B><a name="revised"></a>NOTE TO USERS: 2007/01/13 -- revised to here, then found that this project was failing at Swiss-Model. The folks there are trying to track down the problem. Tutorial beyond this point is revised to fit the <u>expected</u> results from  the hands-on method, and the <u>actual</u> results from the automated method, but it might need additional changes. </B></FONT></P>
<P>On 2007/01/11, my automated modeling request elicited four email messages
from <B>Swiss-Model</B>, with  one that contains this subject line:
<B>SwissModel-Model-AAAa080xq</B>. The AAA number is a
<B>Swiss-Model</B> project number (yours will be different, of
course), and this is the email that contains your homology model as
an attachment (mine was named <B>AAAa080xq.pdb</B>). Save your model
file to a convenient location, and then start <B>DeepView</B> and
open the file.</P>
<blockquote>
  <p>NOTE: The other emails contain information about how your modeling project was carried out, and some news for <strong>Swiss-Model</strong> users. If a modeling project fails, one of these emails will tell you (perhaps cryptically) just what happened. The mail whose subject line includes the word TraceLog gives a list of all operations in making your model. </p>
  <p>ANOTHER NOTE: For an automated request, there may be more than one template. In the remaining instructions, I will assume more than one template, in order to include instructions for looking at only the ones you want. </p>
</blockquote>
<h3>Studying and Evaluating Your Model </h3>
<P>Blink (hold down <strong>ctrl</strong> and press <strong>tab</strong> repeatedly) to see  the <strong>target</strong> (peropsin) and the templates in sequence. In the <strong>cyc</strong> column of the <strong>Layers Info </strong>window, for all models except the target and the first template, click each checkmark once to turn it to +, than again to turn it to -. This prevent the other templates from appearing during blinking. Now blinking will simply alternate between showing the target and the first template.</P>
<P>By default, templates are displayed as backbone only. Turn on all of the first templatse side chains by shift-clicking anywhere in the <B>side</B> column of the <B>Control Panel</B>. With <B>peropsin</B> on display, shift-click any checkmark in the <B>show</B> column to
  turn off display of all residues, leaving a ribbon model. By
  default, the ribbon is colored to show the quality of the model. Most
  of the ribbon is blue or green, and some short segments are red. Blue
  indicates residues that fit very well with the template, green means not bad, while red
  indicates residues that did not match up well with template residues.
  (The menu command option for showing this color scheme is <B>Color:
    B-Factor</B>. although the term <B>B-Factor</B> does not apply
  here.)</P>
<P>It is typical in a modeling project like this that scaffold
residues, such as those in the seven helices, model well, but surface
loops, which define the specific function of the protein, constitute
the most significant differences between target and template, and do
not model as well. Ironically, you learn mostly what you already know
about your target (in this case, that it's a seven-helix bundle), and
you learn least about the most interesting parts, the parts that
differ most from your template, and that give your target protein a
different function from that of your template.</P>

<P>Let's see whether optimization really improved our hand-made model
noticeably. With the <B>peropsin</B> layer displayed, and the
<B>Layer Infos </B>window open, <B>Select: aa Making Clashes</B>. If you find any, fix them as outlined above. Look for other funny stuff, such as long peptide bonds.
If such things are all gone, and the model is at least structurally realistic. To
learn more about judging the quality of models -- homology models and
others -- visit these two resources:</P>

<OL>
   <LI><A HREF="http://www.expasy.org/swissmod/course/course-index.htm" TARGET="_blank"><B>Principles
   of Protein Structure, Comparative Protein Modelling and
   Visualisation</B></A>, by Nicolas Guex (creator of
   <B>DeepView</B>) and Manual C. Peitsch</LI>
   
   <LI><A HREF="http://www.usm.maine.edu/~rhodes/ModQual/index.html" TARGET="_blank"><B>Judging
   the Quality of Macromolecular Models</B></A>, by Gale Rhodes</LI>
</OL>

<P>Now let's see whether it appears that <B>peropsin</B> contains a
pocket for retinal binding. Blink to the <B>1f88A</B> layer (in my project, the first template). In the
<B>Control Panel</B>, scroll down to the bottom and click
<B>RET977</B>, to select the retinal molecule in the bovine rhodopsin
model. The line <B>RET977</B> should turn red when you click it. Press
<B>&lt;return&gt;</B> to remove all but retinal from the display.</P>

<P><B>Select: Neighbors of Selected aa..: Select groups that are
within <U>4.0</U> A of the picked atom</B>. Click <B>OK</B> and press
<B>&lt;return</B>&gt;. Click the <B>labl</B> heading in the
<B>Control Panel</B> to label all displayed residues, and click the <strong>side</strong> heading to add side chaines . Next to RET977
at the bottom of the Control Panel, add a checkmark to the column headed by four
dots and a small <B>v</B> (van der Waals surface column). You should
now have a dramatic display of the retinal-binding pocket of bovine
rhodopsin. Note that LYS296 forms a covalent <U>imide</U> link, with
retinal, the result of addition of the LYS296 amino group to the
aldehyde group of retinal. Now the exquisite fit of the retinal among
other residues, with most hydrophobic side chains snuggling up to the
hydrophobic retinal molecule.</P>

<P>Now let's see whether our model of peropsin allows such a pocket,
and provides an appropriately placed lysine residue for covalent
bonding. <B>Select: Extend to other layers</B>. This selects the
residues in model <B>peropsin</B> that are aligned with the displayed
residues of <B>af88A</B>. Blink to the <B>peropsin layer</B>,
press<B>&lt;return&gt;</B> to display the selected residues,
shift-click any checkmark in the <B>ribn</B> column to remove the
ribbon display, and click the <B>labl</B> heading to label all
displayed residues. Now blink between the layers, looking for
similarities and differences. You should see that LYS284 of
<B>peropsin</B> is perfectly placed to link to retinal, that no
pocket residues of <B>peropsin</B> intrude upon the retinal, and that
many of the pocket residues are identical to those in bovine
rhodopsin. It appears that <B>peropsin</B> could accomodate a retinal
molecule, and could attach it by an imide link as well. Finally, <strong>Color</strong>:<strong>Layer</strong> to give the the target and template different colors, and then make both layers visible (in <strong>Layer</strong> <strong>Infos</strong> window, click to put check marks in the <strong>vis</strong> colum for both. With the models superimposed, you can readily see that the target model has a nicely formed retinal pocket. </P>

<P>So. <B>Does</B> peropsin carry retinal in the human eye? I don't know.
This homology model certainly suggests that retinal binding is
feasible. Proving that peropsin is a retinal carrier in the eye
requires more than just building models. It requires purifying
peropsin from retinal tissue, followed by chemical analysis to detect
retinal. Finding out if the binding is just like what we are seeing
in the model would require determining the structure of the
peropsin-retinal complex by X-ray crystallography or NMR. Or a
researcher could use the model we've made to select residues to
change (by site-directed mutagenesis) and see if the changes affect
binding. <B>Modeling only begins the quest to determine what peropsin
actually does.</B> To fully understand peropsin will require a
conversation between theory (which includes model building) and
experiment (chemical analysis, spectroscopy, structure determination,
monitoring peropsin gene expression). This powerful dialog is the
engine that propels science, and our growing understanding of
nature.</P>

<P>PS<BR>
You have just seen an example of how well things can go in homology
modeling. For a look at what can go wrong, and how to recognize and
avoid some of the pitfalls, click <A HREF="../SPVTut/text/HGHomMod.html"><B>HERE</B></A>.)</P>

<HR>

</P>

<H2><FONT COLOR="#FF0000"><B><A NAME=test></A></B></FONT><FONT COLOR="#009900"><B>Test
Your Bioinformatics Skills</B></FONT></H2>

<P><FONT COLOR="#FF0000"><B>Exam #3 for USM Biochemistry
(CHY463/563), spring semester, 2004.<BR>
See class schedule for instructions.</B></FONT></P>

<P><FONT COLOR="#009900"><B>The subject of this assignment is
</B></FONT><FONT COLOR="#FF0000"><B>homeodomain
genes</B></FONT><FONT COLOR="#009900"><B>, which are involved in
numerous aspects of development, including establishment of body plan
and differentiation of various types of stem cells into specialized
cells.</B></FONT></P>

<P><FONT COLOR="#009900"><B>If your instructor assigns this test,
follow these instructions:</B></FONT></P>

<UL>
   <LI><FONT COLOR="#009900"><B>Copy and paste the Questions sections
   into email.</B></FONT></LI>
   
   <LI><FONT COLOR="#009900"><B>Fill in answers in the spaces
   provided.</B></FONT></LI>
   
   <LI><FONT COLOR="#009900"><B>If the answer is a file, save your
   file with the </B></FONT><FONT COLOR="#FF0099"><B>name
   provided</B></FONT><FONT COLOR="#009900"><B>. Submit all requested
   files in a format acceptable to your instructor.<BR>
   For USM Biochemistry students, the following are
   acceptable:</B></FONT>
   
   <UL>
      <LI><FONT COLOR="#009900"><B>email attachment</B></FONT></LI>
      
      <LI><FONT COLOR="#009900"><B>CD (full size
      only)</B></FONT></LI>
      
      <LI><FONT COLOR="#009900"><B>floppy disk</B></FONT></LI>
      
      <LI><FONT COLOR="#009900"><B>USB memory device</B></FONT></LI>
   </UL>
   </LI>
</UL>

<H2><FONT COLOR="#009900"><B>Questions</B></FONT></H2>

<H3><FONT COLOR="#009900"><B>Part 1. Sequence Work</B></FONT></H3>

<P><FONT COLOR="#009900"><B>Requires completion of all of this
tutorial EXCEPT the Encore! structural work.</B></FONT></P>

<OL>
   <LI><B>Start at the </B><A HREF="http://www.ncbi.nlm.nih.gov/mapview/" TARGET="_blank"><B>NCBI
   Map Viewer</B></A><B>. How many genes in the human genome contain
   the term "homeo" in their name? To be sure you find them all,
   search for "*homeo*". The asterisks are <U>wild</U> <U>cards</U>.,
   which means that you are searching for "homeo" preceded or
   followed by any other characters.<BR>
   Number found: ______ .</B></LI>
   
   <LI><B>Which chromosome contains the largest number of these
   genes? How many?<BR>
   Chromosome # ______ ; Number of "homeo" genes on this chromosome:
   ______ .</B></LI>
   
   <LI><B>Among the genes found in question 1, find one that has a
   role in insulin action.<BR>
   Name of the gene: _____________________________. Four-character
   ID: ______ .</B></LI>
   
   <LI><B>What chromosome contains this gene?<BR>
   Chromosome # ______ .</B></LI>
   
   <LI><B>According to OMIM, what is the role of the protein encoded
   by this gene?<BR>
   Role (limit to 25 words):
   ________________________________________________ .</B></LI>
   
   <LI><B>Obtain the <U>protein</U> sequence of this gene, in FASTA
   format.<BR>
   File name: </B><FONT COLOR="#FF0099"><B>HmPrt.txt</B></FONT><B>
   (You will use this file in Part 2.)</B></LI>
   
   <LI><B>Go to </B><A HREF="http://us.expasy.org/" TARGET="_blank"><B>ExPASy</B></A><B>.
   How many annotated human genes in SwissProt and trEMBL contain the
   term "homeo"? Note that "*" is automatically used as prefix and
   suffix unless you specify otherwise.<BR>
   Number found: ______ .</B></LI>
   
   <LI><B>Make a phylogenetic tree of the first 24 of these genes
   plus the insulin-related gene found in question 3, a total of 25
   sequences. Use the insulin-related gene as the "outgroup". (NOTE:
   ClustalW at EMBnet accepts no more than 30 sequences for
   alignment.)<BR>
   Hint: After conducting the search, you will need to use, in
   sequence, ClustalW, Phylip, and Phylodendron to complete this
   task.<BR>
   Files:<BR>
   ClustalW input file name:
   </B><FONT COLOR="#FF0099"><B>HmCWIn.txt</B></FONT><B><BR>
   ClustalW output file name for Phylip &#91;clustalw (aln)
   file&#93;:
   </B><FONT COLOR="#FF0099"><B>HmCWOut.txt</B></FONT><B><BR>
   Phylip output file name &#91;text tree (outfile.consense)
   file&#93;: </B><FONT COLOR="#FF0099"><B>HmCons.txt<BR>
   </B></FONT><B>Phylip input for Phylodendron &#91;Newick
   (outtree.consense) file&#93;:
   </B><FONT COLOR="#FF0099"><B>HmOutTree.txt<BR>
   </B></FONT><B>Phylodendron output file hame:
   </B><FONT COLOR="#FF0099"><B>HmTree.gif</B></FONT></LI>
   
   <LI><B>According to your tree, what two SwissProt entries in this
   group are the most similar?<BR>
   SwissProt entry numbers ______ and ______ .</B></LI>
   
   <LI><B>What SwissProt entry is most similar to the insulin-related
   gene?<BR>
   SwissProt entry number ______ .</B></LI>
   
   <LI><B>What can you find out about the function of this similar
   gene?</B></LI>
   
   <LI><B>Go to the </B><A HREF="http://www.rcsb.org/pdb/" TARGET="_blank"><B>Protein
   Data Bank</B></A><B>. Search for models of <U>human</U>
   homeodomain proteins. </B>
   
   <OL>
      <LI><B>How many models do you find?<BR>
      Number: ______ .</B></LI>
      
      <LI><B>What method of structure determination produced the
      first of these models?<BR>
      PDB ID code: ______ . Method: _______________________
      .</B></LI>
      
      <LI><B>View the first model on the list with QuickPDB or your
      favorite molecular viewer.<BR>
      What are the main secondary structural elements (helix, sheet,
      coil) in this protein?<BR>
      Secondary structural elements: ___________ . </B></LI>
      
      <LI><B>Give beginning and ending residue numbers of three
      secondary structural elements.<BR>
      1. start: ______ end: ______ .<BR>
      2. start: ______ end: ______ .<BR>
      3. start: ______ end: ______ .</B></LI>
   </OL>
   </LI>
   
   <LI><B>Find a model of a human homeodomain/DNA complex.</B>
   
   <OL>
      <LI><B>How many models do you find?<BR>
      Number: ______ .</B></LI>
      
      <LI><B>What method of structure determination produced the
      first of these models?<BR>
      PDB ID code: ______ . Method: _______________________
      .</B></LI>
      
      <LI><B>View the first model on the list with QuickPDB or your
      favorite molecular viewer.<BR>
      What secondary structural element(s) (helix, sheet, coil)
      interact with DNA?<BR>
      Secondary structural elements: ___________ . </B></LI>
      
      <LI><B>Give beginning and ending residues of main secondary
      structural element in contact with DNA.<BR>
      Residue start: ______ end: ______ .<BR>
      </B></LI>
   </OL>
   </LI>
</OL>

<H3><FONT COLOR="#009900"><B>Part 2. Structural Work</B></FONT></H3>

<P><FONT COLOR="#009900"><B>Requires completion of the Encore!
structure section of this tutorial.</B></FONT></P>

<P><B>Search the </B><A HREF="http://www.rcsb.org/pdb/" TARGET="_blank"><B>Protein
Data Bank</B></A><B> for <U>human</U> homeodomain/DNA complexes. View
the first model you find with DeepView (Swiss-PdbViewer).</B></P>

<OL>
   <LI><B>PDB File ID: _________ .</B></LI>
   
   <LI><B>What patterns does ProSite recognize in this protein?<BR>
   List ProSite patterns:<BR>
   ________________________<BR>
   ________________________<BR>
   etc<BR>
   </B></LI>
   
   <LI><B>What secondary-structural element (helix, sheet, coil)
   contains most of the residues that ProSite recognizes as a
   homeodomain?<BR>
   Element: ________________ ; Residue numbers, start: ______
   end:______ .</B></LI>
   
   <LI><B>What secondary-structural element(s) of the homeodomain
   protein interact(s) directly with DNA?</B></LI>
   
   <LI><B>List three residues involved with hydrogen bonds to
   DNA.<BR>
   List interactions like this: "LYS89-T6" means a hydrogen bond
   between lysine-89 of the protein and thymine-6 of the DNA.<BR>
   Interactions:<BR>
   _______________________<BR>
   _______________________<BR>
   _______________________<BR>
   </B></LI>
   
   <LI><B>Comment on the quality of this model, especially in areas
   of protein-DNA interaction.<BR>
   Quality criteria (limit to 10 words):
   ___________________________________________ .<BR>
   Comments: (limit to 25 words):
   _______________________________________________ .</B></LI>
   
   <LI><B>Use DeepView and the file
   </B><FONT COLOR="#FF0099"><B>HmPrt.txt</B></FONT><B>, which you
   saved in Part 1, to make a homology model of the insulin-related
   human homeodomain protein (the target), using the best available
   template. Submit your project file returned from Swiss-Model.<BR>
   Project file name: _________________.pdb</B></LI>
   
   <LI><B>How many residues does the target protein contain?<BR>
   Number of residues in file HmPrt.txt: ______ .</B></LI>
   
   <LI><B>How many residues of the target are modeled by the best
   template?<BR>
   Number of residues in the homology model: ______ .</B></LI>
   
   <LI><B>Why are the remaining residues missing?<BR>
   Reason(limit to 25 words):
   ___________________________________________________ .</B></LI>
   
   <LI><B>How many residues of the template are <U>identical</U> to
   corresponding residues of the target? How many are
   <U>similar</U>?<BR>
   Number identical: ______ .<BR>
   Number similar: ______ .</B></LI>
   
   <LI><B>How many residues of the target are modeled well by the
   template?<BR>
   Number modeled well: ______ .<BR>
   Criteria (limit to 25 words):
   __________________________________________________ .</B></LI>
</OL>

<P><FONT SIZE="+1">

<HR>

</FONT></P>

<P>Back to <A HREF="Goodies.html#lis">Goodies List</A></P>

<P>Back to <A HREF="../Biochem/index.html">Biochemistry
Resources</A></P>

<P><A HREF="../GRHomePage.html">HOME</A></P>

<P><FONT SIZE="+1">

<HR>

</FONT></P>

<CENTER>&nbsp;You are visitor number <IMG SRC="http://www.usm.maine.edu/cgi/wwwcount?df=BI030405.dat&amp;dd=C&amp;pad=N&amp;ft=2" X-CLARIS-USEIMAGEWIDTH X-CLARIS-USEIMAGEHEIGHT ALIGN=bottom></CENTER>

<P>&nbsp;</P>
</BODY>
</HTML>
