<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
				<meta name="keywords" content="Polymerase chain reaction,Polymerase chain reaction,1992,1993,2005,Agarose gel electrophoresis,Allele,Amplicon,Annealing (biology),Antibody,Arithmetic" />
		<link rel="shortcut icon" href="/favicon.ico" />
		<link rel="apple-touch-icon" href="http://en.wikipedia.org/apple-touch-icon.png" />
		<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (English)" />
		<link rel="copyright" href="http://www.gnu.org/copyleft/fdl.html" />
<link rel="alternate" type="application/rss+xml" title="Wikipedia RSS Feed" href="http://en.wikipedia.org/w/index.php?title=Special:Recentchanges&amp;feed=rss" />
<link rel="alternate" type="application/atom+xml" title="Wikipedia Atom Feed" href="http://en.wikipedia.org/w/index.php?title=Special:Recentchanges&amp;feed=atom" />
		<title>Polymerase chain reaction - Wikipedia, the free encyclopedia</title>
		<style type="text/css" media="screen, projection">/*<![CDATA[*/
			@import "/skins-1.5/common/shared.css?107";
			@import "/skins-1.5/monobook/main.css?107";
		/*]]>*/</style>
		<link rel="stylesheet" type="text/css" media="print" href="/skins-1.5/common/commonPrint.css?107" />
		<!--[if lt IE 5.5000]><style type="text/css">@import "/skins-1.5/monobook/IE50Fixes.css?107";</style><![endif]-->
		<!--[if IE 5.5000]><style type="text/css">@import "/skins-1.5/monobook/IE55Fixes.css?107";</style><![endif]-->
		<!--[if IE 6]><style type="text/css">@import "/skins-1.5/monobook/IE60Fixes.css?107";</style><![endif]-->
		<!--[if IE 7]><style type="text/css">@import "/skins-1.5/monobook/IE70Fixes.css?107";</style><![endif]-->
		<!--[if lt IE 7]><script type="text/javascript" src="/skins-1.5/common/IEFixes.js?107"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
		
		<script type= "text/javascript">/*<![CDATA[*/
var skin = "monobook";
var stylepath = "/skins-1.5";
var wgArticlePath = "/wiki/$1";
var wgScriptPath = "/w";
var wgScript = "/w/index.php";
var wgServer = "http://en.wikipedia.org";
var wgCanonicalNamespace = "";
var wgCanonicalSpecialPageName = false;
var wgNamespaceNumber = 0;
var wgPageName = "Polymerase_chain_reaction";
var wgTitle = "Polymerase chain reaction";
var wgAction = "view";
var wgRestrictionEdit = [];
var wgRestrictionMove = [];
var wgArticleId = "23647";
var wgIsArticle = true;
var wgUserName = "Kanzure";
var wgUserGroups = ["*", "user", "autoconfirmed", "emailconfirmed"];
var wgUserLanguage = "en";
var wgContentLanguage = "en";
var wgBreakFrames = false;
var wgCurRevisionId = "185291002";
var wgVersion = "1.12alpha";
var wgEnableAPI = true;
var wgEnableWriteAPI = false;
var wgAjaxWatch = {"watchMsg": "Watch", "unwatchMsg": "Unwatch", "watchingMsg": "Watching...", "unwatchingMsg": "Unwatching..."};
/*]]>*/</script>
                
		<script type="text/javascript" src="/skins-1.5/common/wikibits.js?107"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="/skins-1.5/common/ajax.js?107"></script>
		<script type="text/javascript" src="/skins-1.5/common/ajaxwatch.js?107"></script>
		<script type="text/javascript" src="/w/index.php?title=-&amp;action=raw&amp;smaxage=0&amp;gen=js&amp;useskin=monobook"><!-- site js --></script>
		<style type="text/css">/*<![CDATA[*/
@import "/w/index.php?title=MediaWiki:Common.css&usemsgcache=yes&action=raw&ctype=text/css&smaxage=2678400";
@import "/w/index.php?title=MediaWiki:Monobook.css&usemsgcache=yes&action=raw&ctype=text/css&smaxage=2678400";
@import "/w/index.php?title=-&action=raw&gen=css&maxage=2678400&smaxage=0&ts=20080125141459";
@import "/w/index.php?title=User:Kanzure/monobook.css&action=raw&ctype=text/css";
/*]]>*/</style>
		<script type="text/javascript" src="/w/index.php?title=User:Kanzure/monobook.js&amp;action=raw&amp;ctype=text/javascript&amp;dontcountme=s"></script>
	</head>
<body class="mediawiki ns-0 ltr page-Polymerase_chain_reaction">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 class="firstHeading">Polymerase chain reaction</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From Wikipedia, the free encyclopedia</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<div class="dablink">"PCR" redirects here. For other uses, see <a href="/wiki/PCR_%28disambiguation%29" title="PCR (disambiguation)">PCR (disambiguation)</a>.</div>
<div class="thumb tright">
<div class="thumbinner" style="width:302px;"><a href="/wiki/Image:PCR_tubes.png" class="image" title="A strip of eight PCR tubes, each containing a 100μl reaction."><img alt="A strip of eight PCR tubes, each containing a 100μl reaction." src="http://upload.wikimedia.org/wikipedia/commons/thumb/8/81/PCR_tubes.png/300px-PCR_tubes.png" width="300" height="246" border="0" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="/wiki/Image:PCR_tubes.png" class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
A strip of eight PCR tubes, each containing a 100μl reaction.</div>
</div>
</div>
<p>The <b>polymerase chain reaction</b> (<b>PCR</b>) is a technique widely used in <a href="/wiki/Molecular_biology" title="Molecular biology">molecular biology</a>. It derives its name from one of its key components, a <a href="/wiki/DNA_polymerase" title="DNA polymerase">DNA polymerase</a> used to amplify (i.e., replicate) a piece of <a href="/wiki/DNA" title="DNA">DNA</a> by <i><a href="/wiki/In_vitro" title="In vitro">in vitro</a></i> <a href="/wiki/Enzyme" title="Enzyme">enzymatic</a> <a href="/wiki/DNA_replication" title="DNA replication">replication</a>. As PCR progresses, the DNA thus generated is itself used as template for replication. This sets in motion a <a href="/wiki/Chain_reaction" title="Chain reaction">chain reaction</a> in which the DNA template is <a href="/wiki/Exponential_growth" title="Exponential growth">exponentially</a> amplified. With PCR it is possible to amplify a single or few copies of a piece of DNA across several orders of magnitude, generating millions or more copies of the DNA piece. PCR can be performed without restrictions on the form of DNA, and it can be extensively modified to perform a wide array of <a href="/wiki/Genetic_engineering" title="Genetic engineering">genetic manipulations</a>.</p>
<p>Almost all PCR applications employ a heat-stable DNA polymerase, such as <a href="/wiki/Taq_polymerase" title="Taq polymerase">Taq polymerase</a>, an enzyme derived from the bacterium <i><a href="/wiki/Thermus_aquaticus" title="Thermus aquaticus">Thermus aquaticus</a></i>. This DNA polymerase <a href="/wiki/Enzyme" title="Enzyme">enzymatically</a> assembles a new DNA strand from DNA building blocks, the <a href="/wiki/Nucleotide" title="Nucleotide">nucleotides</a>, using single-stranded DNA as template and <a href="/wiki/Oligonucleotide" title="Oligonucleotide">DNA oligonucleotides</a> (also called <a href="/wiki/Primer_%28molecular_biology%29" title="Primer (molecular biology)">DNA primers</a>) required for initiation of DNA synthesis. The vast majority of PCR methods use <a href="/wiki/Thermocycler" title="Thermocycler">thermal cycling</a>, i.e., alternately heating and cooling the PCR sample to a defined series of temperature steps. These different temperature steps are necessary to bring about physical separation of the strands in a DNA double helix (<a href="/wiki/DNA_melting" title="DNA melting">DNA melting</a>), and permit DNA synthesis by the DNA polymerase to selectively amplify the target DNA. The power and selectivity of PCR are primarily due to selecting <a href="/wiki/Primer_%28molecular_biology%29" title="Primer (molecular biology)">primers</a> that are highly <a href="/wiki/Complementary_DNA" title="Complementary DNA">complementary</a> to the DNA region targeted for amplification, and to the thermal cycling conditions used.</p>
<p>Developed in 1983 by <a href="/wiki/Kary_Mullis" title="Kary Mullis">Kary Mullis</a>,<sup id="_ref-Bartlett_.26_Stirling_0" class="reference"><a href="#_note-Bartlett_.26_Stirling" title="">[1]</a></sup> PCR is now a common and often indispensable technique used in medical and biological research labs for a variety of applications. These include <a href="/wiki/DNA_cloning" title="DNA cloning">DNA cloning</a> for <a href="/wiki/DNA_sequencing" title="DNA sequencing">sequencing</a>, DNA-based <a href="/wiki/Phylogeny" title="Phylogeny">phylogeny</a>, or functional analysis of <a href="/wiki/Gene" title="Gene">genes</a>; the diagnosis of <a href="/wiki/Hereditary_disease" title="Hereditary disease">hereditary diseases</a>; the identification of <a href="/wiki/Genetic_fingerprint" title="Genetic fingerprint">genetic fingerprints</a> (used in forensics and <a href="/wiki/Parental_testing" title="Parental testing">paternity testing</a>); and the detection and diagnosis of <a href="/wiki/Infectious_disease" title="Infectious disease">infectious diseases</a>. Mullis won the <a href="/wiki/Nobel_Prize" title="Nobel Prize">Nobel Prize</a> for his work on PCR.<sup id="_ref-Karry_Mullis_Nobel_Lecture_0" class="reference"><a href="#_note-Karry_Mullis_Nobel_Lecture" title="">[2]</a></sup></p>
<table id="toc" class="toc" summary="Contents">
<tr>
<td>
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1"><a href="#PCR_principle_and_procedure"><span class="tocnumber">1</span> <span class="toctext">PCR principle and procedure</span></a>
<ul>
<li class="toclevel-2"><a href="#Procedure"><span class="tocnumber">1.1</span> <span class="toctext">Procedure</span></a></li>
<li class="toclevel-2"><a href="#PCR_optimization"><span class="tocnumber">1.2</span> <span class="toctext">PCR optimization</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a href="#Application_of_PCR"><span class="tocnumber">2</span> <span class="toctext">Application of PCR</span></a>
<ul>
<li class="toclevel-2"><a href="#Isolation_of_genomic_DNA"><span class="tocnumber">2.1</span> <span class="toctext">Isolation of genomic DNA</span></a></li>
<li class="toclevel-2"><a href="#Amplification_and_quantitation_of_DNA"><span class="tocnumber">2.2</span> <span class="toctext">Amplification and quantitation of DNA</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a href="#Variations_on_the_basic_PCR_technique"><span class="tocnumber">3</span> <span class="toctext">Variations on the basic PCR technique</span></a></li>
<li class="toclevel-1"><a href="#History"><span class="tocnumber">4</span> <span class="toctext">History</span></a>
<ul>
<li class="toclevel-2"><a href="#Patent_wars"><span class="tocnumber">4.1</span> <span class="toctext">Patent wars</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a href="#References"><span class="tocnumber">5</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1"><a href="#External_links"><span class="tocnumber">6</span> <span class="toctext">External links</span></a></li>
</ul>
</td>
</tr>
</table>
<script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script>
<p><a name="PCR_principle_and_procedure" id="PCR_principle_and_procedure"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=1" title="Edit section: PCR principle and procedure">edit</a>]</span> <span class="mw-headline">PCR principle and procedure</span></h2>
<div class="thumb tright">
<div class="thumbinner" style="width:202px;"><a href="/wiki/Image:Pcr_machine.jpg" class="image" title="Figure 1a: An old thermal cycler for PCR"><img alt="Figure 1a: An old thermal cycler for PCR" src="http://upload.wikimedia.org/wikipedia/commons/thumb/d/da/Pcr_machine.jpg/200px-Pcr_machine.jpg" width="200" height="194" border="0" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="/wiki/Image:Pcr_machine.jpg" class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
<b>Figure 1a</b>: An old thermal cycler for PCR</div>
</div>
</div>
<div class="thumb tright">
<div class="thumbinner" style="width:182px;"><a href="/wiki/Image:Primitive_PCR_machine_for_scrap.JPG" class="image" title="Figure 1b: A very old three-temperature thermal cycler for PCR"><img alt="Figure 1b: A very old three-temperature thermal cycler for PCR" src="http://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/Primitive_PCR_machine_for_scrap.JPG/180px-Primitive_PCR_machine_for_scrap.JPG" width="180" height="203" border="0" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="/wiki/Image:Primitive_PCR_machine_for_scrap.JPG" class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
<b>Figure 1b</b>: A very old three-temperature thermal cycler for PCR</div>
</div>
</div>
<p>PCR is used to amplify specific regions of a DNA strand (the DNA target). This can be a single gene, a part of a gene, or a non-coding sequence. Most PCR methods typically amplify DNA fragments of up to 10 <a href="/wiki/Kilo_base_pair" title="Kilo base pair">kilo base pairs</a> (kb), although some techniques allow for amplification of fragments up to 40 kb in size.<sup id="_ref-0" class="reference"><a href="#_note-0" title="">[3]</a></sup></p>
<p>A basic PCR set up requires several components and reagents.<sup id="_ref-molecular_cloning_0" class="reference"><a href="#_note-molecular_cloning" title="">[4]</a></sup> These components include:</p>
<ul>
<li><i>DNA template</i> that contains the DNA region (target) to be amplified.</li>
<li>One or more <i><a href="/wiki/Primer_%28molecular_biology%29" title="Primer (molecular biology)">primers</a>,</i> which are <a href="/wiki/Complementarity_%28molecular_biology%29" title="Complementarity (molecular biology)">complementary</a> to the DNA regions at the <a href="/wiki/Directionality_%28molecular_biology%29" title="Directionality (molecular biology)">5'</a> (five prime) and <a href="/wiki/Directionality_%28molecular_biology%29" title="Directionality (molecular biology)">3'</a> (three prime) ends of the DNA region.</li>
<li>a <a href="/wiki/DNA_polymerase" title="DNA polymerase">DNA polymerase</a> such as <i><a href="/wiki/Taq_polymerase" title="Taq polymerase">Taq polymerase</a></i> or another DNA polymerase with a temperature optimum at around 70°C.</li>
<li><i><a href="/wiki/Nucleoside" title="Nucleoside">Deoxynucleoside triphosphates</a></i> (dNTPs; also very commonly and erroneously called <a href="/wiki/Nucleotide" title="Nucleotide">deoxynucleotide triphosphates</a>), the building blocks from which the DNA polymerases synthesizes a new DNA strand.</li>
<li><i><a href="/wiki/Buffer_solution" title="Buffer solution">Buffer solution</a>,</i> providing a suitable chemical environment for optimum activity and stability of the DNA polymerase.</li>
<li><i><a href="/wiki/Divalent" title="Divalent">Divalent</a> <a href="/wiki/Cations" title="Cations">cations</a>,</i> <a href="/wiki/Magnesium" title="Magnesium">magnesium</a> or <a href="/wiki/Manganese" title="Manganese">manganese</a> ions; generally Mg<sup>2+</sup> is used, but Mn<sup>2+</sup> can be utilized for PCR-mediated DNA mutagenesis, as higher Mn<sup>2+</sup> concentration increases the error rate during DNA synthesis<sup id="_ref-1" class="reference"><a href="#_note-1" title="">[5]</a></sup></li>
<li><i>Monovalent cation</i> <a href="/wiki/Potassium" title="Potassium">potassium</a> ions.</li>
</ul>
<p>The PCR is commonly carried out in a reaction volume of 15-100 μl in small reaction tubes (0.2-0.5 ml volumes) in a <a href="/wiki/Thermal_cycler" title="Thermal cycler">thermal cycler</a>. The thermal cycler allows heating and cooling of the reaction tubes to control the temperature required at each reaction step (see below). Many modern thermal cyclers make use of the <a href="/wiki/Thermoelectric_cooling" title="Thermoelectric cooling">Peltier effect</a> which permits both heating and cooling of the block holding the PCR tubes simply by reversing the electric current. Thin-walled reaction tubes permit favorable thermal conductivity to allow for rapid thermal equilibration. Most thermal cyclers have heated lids to prevent condensation at the top of the reaction tube. Older thermocyclers lacking a heated lid require a layer of oil on top of the reaction mixture or a ball of wax inside the tube.</p>
<p><a name="Procedure" id="Procedure"></a></p>
<h3><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=2" title="Edit section: Procedure">edit</a>]</span> <span class="mw-headline">Procedure</span></h3>
<div class="thumb tright">
<div class="thumbinner" style="width:302px;"><a href="/wiki/Image:PCR.svg" class="image" title="Figure 2: Schematic drawing of the PCR cycle. (1) Denaturing at 94-96°C. (2) Annealing at ~65°C (3) Elongation at 72°C. Four cycles are shown here."><img alt="Figure 2: Schematic drawing of the PCR cycle. (1) Denaturing at 94-96°C. (2) Annealing at ~65°C (3) Elongation at 72°C. Four cycles are shown here." src="http://upload.wikimedia.org/wikipedia/commons/thumb/8/87/PCR.svg/300px-PCR.svg.png" width="300" height="675" border="0" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="/wiki/Image:PCR.svg" class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
<b>Figure 2</b>: Schematic drawing of the PCR cycle. <b>(1) Denaturing at 94-96°C. (2) Annealing at ~65°C (3) Elongation at 72°C</b>. Four cycles are shown here.</div>
</div>
</div>
<p>The PCR usually consists of a series of 20 to 35 repeated temperature changes called cycles; each cycle typically consists of 2-3 discrete temperature steps. Most commonly PCR is carried out with cycles that have three temperature steps (Fig. 2). The cycling is often preceded by a single temperature step (called <i>hold</i>) at a high temperature (&gt;90°C), and followed by one hold at the end for final product extension or brief storage. The temperatures used and the length of time they are applied in each cycle depend on a variety of parameters. These include the enzyme used for DNA synthesis, the concentration of divalent ions and dNTPs in the reaction, and the melting temperature (<a href="/wiki/DNA_melting" title="DNA melting">Tm</a>) of the primers.<sup id="_ref-2" class="reference"><a href="#_note-2" title="">[6]</a></sup></p>
<ul>
<li><i>Initialization step</i>: This step consists of heating the reaction to a temperature of 94-96°C (or 98°C if extremely thermostable polymerases are used), which is held for 1-9 minutes. It is only required for DNA polymerases that require heat activation by <a href="/wiki/Polymerase_chain_reaction#Practical_modifications_to_the_PCR_technique" title="Polymerase chain reaction">hot-start PCR</a>.<sup id="_ref-antibody_hot_start_0" class="reference"><a href="#_note-antibody_hot_start" title="">[7]</a></sup></li>
</ul>
<ul>
<li><i><a href="/wiki/Denaturation_%28biochemistry%29#Nucleic_acid_denaturation" title="Denaturation (biochemistry)">Denaturation step</a></i>: This step is the first regular cycling event and consists of heating the reaction to 94-98°C for 20-30 seconds. It causes melting of DNA template and primers by disrupting the hydrogen bonds between complementary bases of the DNA strands, yielding single strands of DNA.</li>
</ul>
<ul>
<li><i><a href="/wiki/Annealing_%28biology%29" title="Annealing (biology)">Annealing step</a></i>: The reaction temperature is lowered to 50-65°C for 20-40 seconds allowing annealing of the primers to the single-stranded DNA template. Typically the annealing temperature is about 3-5 degrees Celsius below the Tm of the primers used. Stable DNA-DNA hydrogen bonds are only formed when the primer sequence very closely matches the template sequence. The polymerase binds to the primer-template hybrid and begins DNA synthesis.</li>
</ul>
<ul>
<li><i>Extension/elongation step</i>: The temperature at this step depends on the DNA polymerase used; <a href="/wiki/Taq_polymerase" title="Taq polymerase">Taq polymerase</a> has its optimum <a href="/wiki/Enzyme" title="Enzyme">activity</a> temperature at 75-80°C,<sup id="_ref-Chien_et_al_0" class="reference"><a href="#_note-Chien_et_al" title="">[8]</a></sup><sup id="_ref-Lawyer_et_al_0" class="reference"><a href="#_note-Lawyer_et_al" title="">[9]</a></sup> and commonly a temperature of 72°C is used with this enzyme. At this step the DNA polymerase synthesizes a new DNA strand complementary to the DNA template strand by adding dNTP's that are complementary to the template in 5' to 3' direction, condensing the 5'-<a href="/wiki/Phosphate_group" title="Phosphate group">phosphate group</a> of the dNTPs with the 3'-<a href="/wiki/Hydroxyl_group" title="Hydroxyl group">hydroxyl group</a> at the end of the nascent (extending) DNA strand. The extension time depends both on the DNA polymerase used and on the length of the DNA fragment to be amplified. As a rule-of-thumb, at its optimum temperature, the DNA polymerase will polymerize a thousand bases in one minute.</li>
</ul>
<ul>
<li><i>Final elongation</i>: This single step is occasionally performed at a temperature of 70-74°C for 5-15 minutes after the last PCR cycle to ensure that any remaining single-stranded DNA is fully extended.</li>
</ul>
<ul>
<li><i>Final hold</i>: This step at 4-15°C for an indefinite time may be employed for short-term storage of the reaction.</li>
</ul>
<div class="thumb tright">
<div class="thumbinner" style="width:302px;"><a href="/wiki/Image:Roland_Gel.JPG" class="image" title="Figure 3: Ethidium bromide-stained PCR products after gel electrophoresis. Two sets of primers were used to amplify a target sequence from three different tissue samples. No amplification is present in sample #1; DNA bands in sample #2 and #3 indicate successful amplification of the target sequence. The gel also shows a positive control, and a DNA ladder containing DNA fragments of defined length for sizing the bands in the experimental PCRs."><img alt="Figure 3: Ethidium bromide-stained PCR products after gel electrophoresis. Two sets of primers were used to amplify a target sequence from three different tissue samples. No amplification is present in sample #1; DNA bands in sample #2 and #3 indicate successful amplification of the target sequence. The gel also shows a positive control, and a DNA ladder containing DNA fragments of defined length for sizing the bands in the experimental PCRs." src="http://upload.wikimedia.org/wikipedia/en/thumb/d/d0/Roland_Gel.JPG/300px-Roland_Gel.JPG" width="300" height="283" border="0" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="/wiki/Image:Roland_Gel.JPG" class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
<b>Figure 3</b>: Ethidium bromide-stained PCR products after <a href="/wiki/Gel_electrophoresis" title="Gel electrophoresis">gel electrophoresis</a>. Two sets of primers were used to amplify a target sequence from three different tissue samples. No amplification is present in sample #1; DNA bands in sample #2 and #3 indicate successful amplification of the target sequence. The gel also shows a positive control, and a DNA ladder containing DNA fragments of defined length for sizing the bands in the experimental PCRs.</div>
</div>
</div>
<p>To check whether the PCR generated the anticipated DNA fragment (also sometimes referred to as the amplimer or <a href="/wiki/Amplicon" title="Amplicon">amplicon</a>), <a href="/wiki/Agarose_gel_electrophoresis" title="Agarose gel electrophoresis">agarose gel electrophoresis</a> is employed for size separation of the PCR products. The size(s) of PCR products is determined by comparison with a <i>DNA ladder</i> (a molecular weight marker), which contains DNA fragments of known size, run on the gel alongside the PCR products (see Fig. 3).</p>
<p><a name="PCR_optimization" id="PCR_optimization"></a></p>
<h3><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=3" title="Edit section: PCR optimization">edit</a>]</span> <span class="mw-headline">PCR optimization</span></h3>
<dl>
<dd>
<div class="noprint relarticle mainarticle"><i>Main article: <a href="/wiki/PCR_optimization" title="PCR optimization">PCR optimization</a></i></div>
</dd>
</dl>
<p>In practice, PCR can fail for various reasons, in part due to its sensitivity to contamination causing amplification of spurious DNA products. Because of this, a number of techniques and procedures have been developed for optimizing PCR conditions.<sup id="_ref-3" class="reference"><a href="#_note-3" title="">[10]</a></sup><sup id="_ref-4" class="reference"><a href="#_note-4" title="">[11]</a></sup> Contamination with extraneous DNA is addressed with lab protocols and procedures that separate pre-PCR reactions from potential DNA contaminants.<sup id="_ref-molecular_cloning_1" class="reference"><a href="#_note-molecular_cloning" title="">[4]</a></sup> This usually involves spatial separation of PCR-setup areas from areas for analysis or purification of PCR products, and thoroughly cleaning the work surface between reaction setups. Primer-design techniques are important in improving PCR product yield and in avoiding the formation of spurious products, and the usage of alternate buffer components or polymerase enzymes can help with amplification of long or otherwise problematic regions of DNA.</p>
<p><a name="Application_of_PCR" id="Application_of_PCR"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=4" title="Edit section: Application of PCR">edit</a>]</span> <span class="mw-headline">Application of PCR</span></h2>
<p><a name="Isolation_of_genomic_DNA" id="Isolation_of_genomic_DNA"></a></p>
<h3><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=5" title="Edit section: Isolation of genomic DNA">edit</a>]</span> <span class="mw-headline">Isolation of genomic DNA</span></h3>
<p>PCR allows isolation of DNA fragments from genomic DNA by selective amplification of a specific region of DNA. This use of PCR augments many methods, such as generating <a href="/wiki/Hybridization_probe" title="Hybridization probe">hybridization probes</a> for <a href="/wiki/Southern_blot" title="Southern blot">Southern</a> or <a href="/wiki/Northern_blot" title="Northern blot">northern</a> hybridization and <a href="/wiki/DNA_cloning" title="DNA cloning">DNA cloning</a>, which require larger amounts of DNA, representing a specific DNA region. PCR supplies these techniques with high amounts of pure DNA, enabling analysis of DNA samples even from very small amounts of starting material.</p>
<p>Other applications of PCR include <a href="/wiki/DNA_sequencing" title="DNA sequencing">DNA sequencing</a> to determine unknown PCR-amplified sequences in which one of the amplification primers may be used in Sanger sequencing, isolation of a DNA sequence to expedite <a href="/wiki/Recombinant_DNA" title="Recombinant DNA">recombinant DNA</a> technologies involving the insertion of a DNA sequence into a <a href="/wiki/Plasmid" title="Plasmid">plasmid</a> or the genetic material of another organism. PCR may also be used for <a href="/wiki/Genetic_fingerprinting" title="Genetic fingerprinting">genetic fingerprinting</a>; a forensic technique used to identify a person or organism by comparing experimental DNAs through different PCR-based methods.</p>
<p>Some PCR 'fingerprints' methods have high discriminative power and can be used to identify genetic relationships between individuals, such as parent-child or between siblings, and are used in paternity testing (Fig. 4). This technique may also be used to determine evolutionary relationships among organisms.</p>
<div class="thumb tright">
<div class="thumbinner" style="width:202px;"><a href="/wiki/Image:Pcr_fingerprint.png" class="image" title="Figure 4: Electrophoresis of PCR-amplified DNA fragments. (1) Father. (2) Child. (3) Mother. The child has inherited some, but not all of the fingerprint of each of its parents, giving it a new, unique fingerprint."><img alt="Figure 4: Electrophoresis of PCR-amplified DNA fragments. (1) Father. (2) Child. (3) Mother. The child has inherited some, but not all of the fingerprint of each of its parents, giving it a new, unique fingerprint." src="http://upload.wikimedia.org/wikipedia/commons/thumb/0/02/Pcr_fingerprint.png/200px-Pcr_fingerprint.png" width="200" height="301" border="0" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="/wiki/Image:Pcr_fingerprint.png" class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
<b>Figure 4</b>: Electrophoresis of PCR-amplified DNA fragments. (1) Father. (2) Child. (3) Mother. The child has inherited some, but not all of the fingerprint of each of its parents, giving it a new, unique fingerprint.</div>
</div>
</div>
<p><a name="Amplification_and_quantitation_of_DNA" id="Amplification_and_quantitation_of_DNA"></a></p>
<h3><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=6" title="Edit section: Amplification and quantitation of DNA">edit</a>]</span> <span class="mw-headline">Amplification and quantitation of DNA</span></h3>
<p>Because PCR amplifies the regions of DNA that it targets, PCR can be used to analyze extremely small amounts of sample. This is often critical for <a href="/wiki/Forensic_analysis" title="Forensic analysis">forensic analysis</a>, when only a trace amount of DNA is available as evidence. PCR may also be used in the analysis of ancient DNA that is thousands of years old. These PCR-based techniques have been successfully used on animals, such as a forty-thousand-year-old <a href="/wiki/Mammoth" title="Mammoth">mammoth</a>, and also on human DNA, in applications ranging from the analysis of Egyptian <a href="/wiki/Mummy" title="Mummy">mummies</a> to the identification of a <a href="/wiki/Russia" title="Russia">Russian</a> <a href="/wiki/Tsar" title="Tsar">Tsar</a>.<sup id="_ref-5" class="reference"><a href="#_note-5" title="">[12]</a></sup></p>
<p><a href="/wiki/Virus" title="Virus">Viral</a> DNA can likewise be detected by PCR. The primers used need to be specific to the targeted sequences in the DNA of a virus, and the PCR can be used for diagnostic analyses or DNA sequencing of the viral genome. The high sensitivity of PCR permits virus detection soon after infection and even before the onset of disease. Such early detection may give physicians a significant lead in treatment. The amount of virus ("viral load") in a patient can also be quantified by PCR-based DNA quantitation techniques (see below).</p>
<p>Quantitative PCR methods allow the estimation of the amount of a given sequence present in a sample – a technique often applied to quantitatively determine levels of <a href="/wiki/Gene_expression" title="Gene expression">gene expression</a>. <a href="/wiki/Real-time_polymerase_chain_reaction" title="Real-time polymerase chain reaction">Real-time PCR</a> is an established tool for DNA quantification that measures the accumulation of DNA product after each round of PCR amplification.</p>
<p><a name="Variations_on_the_basic_PCR_technique" id="Variations_on_the_basic_PCR_technique"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=7" title="Edit section: Variations on the basic PCR technique">edit</a>]</span> <span class="mw-headline">Variations on the basic PCR technique</span></h2>
<ul>
<li><b>Allele-specific PCR</b>: This diagnostic or cloning technique is used to identify or utilize <a href="/wiki/Single-nucleotide_polymorphism" title="Single-nucleotide polymorphism">single-nucleotide polymorphisms</a> (SNPs) (single base differences in DNA). It requires prior knowledge of a DNA sequence, including differences between <a href="/wiki/Allele" title="Allele">alleles</a>, and uses primers whose 3' ends encompass the SNP. PCR amplification under stringent conditions is much less efficient in the presence of a mismatch between template and primer, so successful amplification with an SNP-specific primer signals presence of the specific SNP in a sequence.<sup id="_ref-Saiki_et_al_0" class="reference"><a href="#_note-Saiki_et_al" title="">[13]</a></sup> See <a href="/wiki/SNP_genotyping" title="SNP genotyping">SNP genotyping</a> for more information.</li>
</ul>
<ul>
<li><b>Assembly PCR</b>: Assembly PCR is the artificial synthesis of long DNA sequences by performing PCR on a pool of long oligonucleotides with short overlapping segments. The oligonucleotides alternate between sense and antisense directions, and the overlapping segments determine the order of the PCR fragments thereby selectively producing the final long DNA product.<sup id="_ref-Stemmer_et_al_0" class="reference"><a href="#_note-Stemmer_et_al" title="">[14]</a></sup></li>
</ul>
<ul>
<li><b>Asymmetric PCR</b>: Asymmetric PCR is used to preferentially amplify one strand of the original <a href="/wiki/DNA" title="DNA">DNA</a> more than the other. It finds use in some types of <a href="/wiki/Sequencing" title="Sequencing">sequencing</a> and hybridization probing where having only one of the two complementary stands is required. PCR is carried out as usual, but with a great excess of the primers for the chosen strand. Due to the slow (<a href="/wiki/Arithmetic" title="Arithmetic">arithmetic</a>) amplification later in the reaction after the limiting primer has been used up, extra cycles of PCR are required.<sup id="_ref-Innis_et_al_0" class="reference"><a href="#_note-Innis_et_al" title="">[15]</a></sup> A recent modification on this process, known as <b>L</b>inear-<b>A</b>fter-<b>T</b>he-<b>E</b>xponential-PCR (LATE-PCR), uses a limiting primer with a higher melting temperature (<a href="/wiki/DNA_melting" title="DNA melting">Melting temperature|Tm</a>) than the excess primer to maintain reaction efficiency as the limiting primer concentration decreases mid-reaction.<sup id="_ref-Pierce_and_Wangh_0" class="reference"><a href="#_note-Pierce_and_Wangh" title="">[16]</a></sup></li>
</ul>
<ul>
<li><b>Colony PCR</b>: Bacterial colonies (<a href="/wiki/E.coli" title="E.coli">E.coli</a>) can be rapidly screened by PCR for correct DNA <a href="/wiki/Plasmid" title="Plasmid">vector</a> constructs. Selected bacterial colonies are picked with a sterile toothpick and dabbed into the PCR master mix or sterile water. The PCR is started with an extended time at 95˚C when standard polymerase is used or with a shortened denaturation step at 100˚C and special chimeric DNA polymerase.<sup id="_ref-Hybrid_0" class="reference"><a href="#_note-Hybrid" title="">[17]</a></sup></li>
</ul>
<ul>
<li><b><a href="/wiki/Helicase-dependent_amplification" title="Helicase-dependent amplification">Helicase-dependent amplification</a></b>: This technique is similar to traditional PCR, but uses a constant temperature rather than cycling through denaturation and annealing/extension cycles. <a href="/wiki/DNA_Helicase" title="DNA Helicase">DNA Helicase</a>, an enzyme that unwinds DNA, is used in place of thermal denaturation.<sup id="_ref-6" class="reference"><a href="#_note-6" title="">[18]</a></sup></li>
</ul>
<ul>
<li><b>Hot-start PCR</b>: This is a technique that reduces non-specific amplification during the initial set up stages of the PCR. The technique may be performed manually by heating the reaction components to the melting temperature (e.g., 95˚C) before adding the polymerase.<sup id="_ref-general_hot_start_0" class="reference"><a href="#_note-general_hot_start" title="">[19]</a></sup> Specialized enzyme systems have been developed that inhibit the polymerase's activity at ambient temperature, either by the binding of an <a href="/wiki/Antibody" title="Antibody">antibody</a><sup id="_ref-antibody_hot_start_1" class="reference"><a href="#_note-antibody_hot_start" title="">[7]</a></sup> or by the presence of covalently bound inhibitors that only dissociate after a high-temperature activation step. Hot-start/cold-finish PCR is achieved with new hybrid polymerases that are inactive at ambient temperature and are instantly activated at elongation temperature.</li>
</ul>
<ul>
<li><b>Intersequence-specific (ISSR) PCR</b>: a PCR method for DNA fingerprinting that amplifies regions between some simple sequence repeats to produce a unique fingerprint of amplified fragment lengths.<sup id="_ref-7" class="reference"><a href="#_note-7" title="">[20]</a></sup></li>
</ul>
<ul>
<li><b><a href="/wiki/Inverse_polymerase_chain_reaction" title="Inverse polymerase chain reaction">Inverse PCR</a></b>: a method used to allow PCR when only one internal sequence is known. This is especially useful in identifying flanking sequences to various <a href="/wiki/Genomic" title="Genomic">genomic</a> inserts. This involves a series of <a href="/wiki/Restriction_digest" title="Restriction digest">DNA digestions</a> and <a href="/wiki/Self_ligation" title="Self ligation">self ligation</a>, resulting in known sequences at either end of the unknown sequence.<sup id="_ref-Ochman_et_al_0" class="reference"><a href="#_note-Ochman_et_al" title="">[21]</a></sup></li>
</ul>
<ul>
<li><b>Ligation-mediated PCR</b>: This method uses small DNA linkers ligated to the DNA of interest and multiple primers annealing to the DNA linkers; it has been used for DNA sequencing, genome walking, and <a href="/w/index.php?title=DNA_footprinting&amp;action=edit" class="new" title="DNA footprinting">DNA footprinting</a>. <sup id="_ref-Mueller_and_Wold_0" class="reference"><a href="#_note-Mueller_and_Wold" title="">[22]</a></sup></li>
</ul>
<ul>
<li><b>Methylation-specific PCR</b> (MSP): The MSP method was developed by Stephen Baylin and Jim Herman at the Johns Hopkins School of Medicine,<sup id="_ref-8" class="reference"><a href="#_note-8" title="">[23]</a></sup> and is used to detect methylation of CpG islands in genomic DNA. DNA is first treated with sodium bisulfite, which converts unmethylated cytosine bases to uracil, which is recognized by PCR primers as thymine. Two PCR reactions are then carried out on the modified DNA, using primer sets identical except at any CpG islands within the primer sequences. At these points, one primer set recognizes DNA with cytosines to amplify methylated DNA, and one set recognizes DNA with uracil or thymine to amplify unmethylated DNA. MSP using qPCR can also be performed to obtain quantitative rather than qualitative information about methylation.</li>
</ul>
<ul>
<li><b><a href="/wiki/Multiplex_Ligation-dependent_Probe_Amplification" title="Multiplex Ligation-dependent Probe Amplification">Multiplex Ligation-dependent Probe Amplification</a></b> (<b>MLPA</b>): permits multiple targets to be amplified with only a single primer pair, thus avoiding the resolution limitations of multiplex PCR (see below).</li>
</ul>
<ul>
<li><b>Multiplex-PCR</b>: The use of multiple, unique primer sets within a single PCR reaction to produce <a href="/wiki/Amplicon" title="Amplicon">amplicons</a> of varying sizes specific to different DNA sequences. By targeting multiple genes at once, additional information may be gained from a single test run that otherwise would require several times the reagents and more time to perform. Annealing temperatures for each of the primer sets must be optimized to work correctly within a single reaction, and amplicon sizes, i.e., their base pair length, should be different enough to form distinct bands when visualized by <a href="/wiki/Gel_electrophoresis" title="Gel electrophoresis">gel electrophoresis</a>.</li>
</ul>
<ul>
<li><b><a href="/wiki/Nested_PCR" title="Nested PCR">Nested PCR</a></b>: increases the specificity of DNA amplification, by reducing background due to non-specific amplification of DNA. Two sets of primers are being used in two successive PCR reactions. In the first reaction, one pair of primers is used to generate DNA products, which besides the intended target, may still consist of non-specifically amplified DNA fragments. The product(s) are then used in a second PCR reaction with a set of primers whose binding sites are completely or partially different from and located 3' of each of the primers used in the first reaction. Nested PCR is often more successful in specifically amplifying long DNA fragments than conventional PCR, but it requires more detailed knowledge of the target sequences.</li>
</ul>
<ul>
<li><b><a href="/wiki/Overlap-extension_PCR" title="Overlap-extension PCR">Overlap-extension PCR</a></b>: is a <a href="/wiki/Genetic_engineering" title="Genetic engineering">genetic engineering</a> technique allowing the construction of a DNA sequence with an alteration inserted beyond the limit of the longest practical primer length.</li>
</ul>
<ul>
<li><b><a href="/wiki/Q-PCR" title="Q-PCR">Quantitative PCR</a> (Q-PCR)</b>: is used to measure the quantity of a PCR product (preferably real-time). It is the method of choice to quantitatively measure starting amounts of DNA, cDNA or RNA. Q-PCR is commonly used to determine whether a DNA sequence is present in a sample and the number of its copies in the sample. The method with currently the highest level of accuracy is <b>Quantitative real-time PCR</b>. It is often confusingly known as <a href="/wiki/Real-time_PCR" title="Real-time PCR">RT-PCR</a> (<b>R</b>eal <b>T</b>ime PCR) or RQ-PCR. QRT-PCR or RTQ-PCR are more appropriate contractions. RT-PCR commonly refers to <a href="/wiki/RT-PCR" title="RT-PCR">reverse transcription PCR</a> (see below), which is often used in conjunction with Q-PCR. QRT-PCR methods use fluorescent dyes, such as Sybr Green, or <a href="/wiki/Fluorophore" title="Fluorophore">fluorophore</a>-containing DNA probes, such as <a href="/wiki/TaqMan" title="TaqMan">TaqMan</a>, to measure the amount of amplified product in real time.</li>
</ul>
<ul>
<li><b><a href="/wiki/RT-PCR" title="RT-PCR">RT-PCR</a></b>: (<b>R</b>everse <b>T</b>ranscription PCR) is a method used to amplify, isolate or identify a known sequence from a <a href="/wiki/Cell_%28biology%29" title="Cell (biology)">cellular</a> or <a href="/wiki/Biological_tissue" title="Biological tissue">tissue</a> <a href="/wiki/RNA" title="RNA">RNA</a>. The PCR is preceded by a reaction using <a href="/wiki/Reverse_transcriptase" title="Reverse transcriptase">reverse transcriptase</a> to convert <a href="/wiki/RNA" title="RNA">RNA</a> to <a href="/wiki/CDNA" title="CDNA">cDNA</a>. RT-PCR is widely used in <a href="/wiki/Expression_profiling" title="Expression profiling">expression profiling</a>, to determine the expression of a gene or to identify the sequence of an RNA transcript, including transcription start and termination sites and, if the genomic DNA sequence of a gene is known, to map the location of <a href="/wiki/Exons" title="Exons">exons</a> and <a href="/wiki/Introns" title="Introns">introns</a> in the gene. The 5' end of a gene (corresponding to the transcription start site) is typically identified by an RT-PCR method, named <a href="/wiki/RACE_%28biology%29" title="RACE (biology)">RACE-PCR</a>, short for <i>Rapid Amplification of cDNA Ends</i>.</li>
</ul>
<ul>
<li><b>TAIL-PCR</b>: <b>Thermal asymmetric interlaced</b> PCR is used to isolate unknown sequence flanking a known sequence. Within the known sequence TAIL-PCR uses a nested pair of primers with differing annealing temperatures; a degenerate primer is used to amplify in the other direction from the unknown sequence.<sup id="_ref-9" class="reference"><a href="#_note-9" title="">[24]</a></sup></li>
</ul>
<ul>
<li><b><a href="/wiki/Touchdown_PCR" title="Touchdown PCR">Touchdown PCR</a></b>: a variant of PCR that aims to reduce nonspecific background by gradually lowering the annealing temperature as PCR cycling progresses. The annealing temperature at the initial cycles is usually a few degrees (3-5˚C) above the T<sub>m</sub> of the primers used, while at the later cycles, it is a few degrees (3-5˚C) below the primer T<sub>m</sub>. The higher temperatures give greater specificity for primer binding, and the lower temperatures permit more efficient amplification from the specific products formed during the initial cycles.<sup id="_ref-10" class="reference"><a href="#_note-10" title="">[25]</a></sup></li>
</ul>
<ul>
<li><b>PAN-AC</b>: This method uses isothermal conditions for amplification, and may be used in living cells.<sup id="_ref-11" class="reference"><a href="#_note-11" title="">[26]</a></sup><sup id="_ref-12" class="reference"><a href="#_note-12" title="">[27]</a></sup></li>
</ul>
<p><a name="History" id="History"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=8" title="Edit section: History">edit</a>]</span> <span class="mw-headline">History</span></h2>
<p>A 1971 paper in the <a href="/wiki/Journal_of_Molecular_Biology" title="Journal of Molecular Biology">Journal of Molecular Biology</a> by Kleppe and co-workers first described a method using an enzymatic assay to replicate a short DNA template with primers <i>in vitro</i>.<sup id="_ref-13" class="reference"><a href="#_note-13" title="">[28]</a></sup> However, this early manifestation of the basic PCR principle did not receive much attention, and the invention of the polymerase chain reaction in 1983 is generally credited to <a href="/wiki/Kary_Mullis" title="Kary Mullis">Kary Mullis</a>.<sup id="_ref-Mullis_0" class="reference"><a href="#_note-Mullis" title="">[29]</a></sup><sup id="_ref-14" class="reference"><a href="#_note-14" title="">[30]</a></sup></p>
<p>At the core of the PCR method is the use of a suitable <a href="/wiki/DNA_polymerase" title="DNA polymerase">DNA polymerase</a> able to withstand the high temperatures of &gt;90°C (&gt;195°F) required for separation of the two DNA strands in the <a href="/wiki/DNA_double_helix" title="DNA double helix">DNA double helix</a> after each <a href="/wiki/DNA_replication" title="DNA replication">replication</a> cycle. The DNA polymerases initially employed for <a href="/wiki/In_vitro" title="In vitro">in vitro</a> experiments presaging PCR were unable to withstand these high temperatures. So the early procedures for DNA replication were very inefficient, time consuming, and required large amounts of DNA polymerase and continual handling throughout the process.</p>
<p>A 1976 discovery of <a href="/wiki/Taq_polymerase" title="Taq polymerase">Taq polymerase</a> a DNA polymerase purified from the <a href="/wiki/Thermophile" title="Thermophile">thermophilic bacterium</a>, <i><a href="/wiki/Thermus_aquaticus" title="Thermus aquaticus">Thermus aquaticus</a></i>, which naturally occurs in hot (50 to 80 °C (120 to 175 °F)) environments<sup id="_ref-Chien_et_al_1" class="reference"><a href="#_note-Chien_et_al" title="">[8]</a></sup> paved the way for dramatic improvements of the PCR method. The DNA polymerase isolated from <i>T. aquaticus</i> is stable at high temperatures remaining active even after DNA denaturation,<sup id="_ref-Lawyer_et_al_1" class="reference"><a href="#_note-Lawyer_et_al" title="">[9]</a></sup> thus obviating the need to add new DNA polymerase after each cycle. This allowed an automated thermocycler-based process for DNA amplification.</p>
<p>At the time he developed PCR in 1983, Mullis was working in <a href="/wiki/Emeryville" title="Emeryville">Emeryville</a>, California for <a href="/wiki/Cetus_Corporation" title="Cetus Corporation">Cetus Corporation</a>, one of the first <a href="/wiki/Biotechnology" title="Biotechnology">biotechnology</a> companies. There, he was responsible for synthesizing short chains of DNA. Mullis has written that he conceived of PCR while cruising along the <a href="/wiki/Pacific_Coast_Highway_%28United_States%29" title="Pacific Coast Highway (United States)">Pacific Coast Highway</a> one night in his car.<sup id="_ref-Mullis_1" class="reference"><a href="#_note-Mullis" title="">[29]</a></sup> He was playing in his mind with a new way of analyzing changes (mutations) in DNA when he realized that he had instead invented a method of amplifying any DNA region through repeated cycles of duplication driven by DNA polymerase. Mullis has credited the <a href="/wiki/Psychedelic" title="Psychedelic">psychedelic</a> drug <a href="/wiki/LSD" title="LSD">LSD</a> for his invention of the technique.<sup id="_ref-Mullisp18_0" class="reference"><a href="#_note-Mullisp18" title="">[31]</a></sup></p>
<p>In <i><a href="/wiki/Scientific_American" title="Scientific American">Scientific American</a></i>, Mullis summarized the procedure: "Beginning with a single molecule of the genetic material DNA, the PCR can generate 100 billion similar molecules in an afternoon. The reaction is easy to execute. It requires no more than a test tube, a few simple reagents, and a source of heat."<sup id="_ref-15" class="reference"><a href="#_note-15" title="">[32]</a></sup> He was awarded the <a href="/wiki/Nobel_Prize_in_Chemistry" title="Nobel Prize in Chemistry">Nobel Prize in Chemistry</a> in <a href="/wiki/1993" title="1993">1993</a> for his invention,<sup id="_ref-Karry_Mullis_Nobel_Lecture_1" class="reference"><a href="#_note-Karry_Mullis_Nobel_Lecture" title="">[2]</a></sup> seven years after he and his colleagues at Cetus first put his proposal to practice. However, some controversies have remained about the intellectual and practical contributions of other scientists to Mullis' work, and whether he had been the sole inventor of the PCR principle. (see main article: <a href="/wiki/Kary_Mullis" title="Kary Mullis">Kary Mullis</a>)</p>
<p><a name="Patent_wars" id="Patent_wars"></a></p>
<h3><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=9" title="Edit section: Patent wars">edit</a>]</span> <span class="mw-headline">Patent wars</span></h3>
<p>The PCR technique was patented by <a href="/wiki/Cetus_Corporation" title="Cetus Corporation">Cetus Corporation</a>, where Mullis worked when he invented the technique in 1983. The <i>Taq</i> polymerase enzyme was also covered by patents. There have been several high-profile lawsuits related to the technique, including an unsuccessful lawsuit brought by <a href="/wiki/DuPont" title="DuPont">DuPont</a>. The pharmaceutical company <a href="/wiki/Hoffmann-La_Roche" title="Hoffmann-La Roche">Hoffmann-La Roche</a> purchased the rights to the patents in <a href="/wiki/1992" title="1992">1992</a> and currently holds those that are still protected.</p>
<p>A related patent battle over the Taq polymerase enzyme is still ongoing in several jurisdictions around the world between Roche and <a href="/wiki/Promega" title="Promega">Promega</a>. The legal arguments have extended beyond the life of the original PCR and Taq polymerase patents, which expired on <a href="/wiki/March_28" title="March 28">March 28</a>, <a href="/wiki/2005" title="2005">2005</a>.<sup id="_ref-16" class="reference"><a href="#_note-16" title="">[33]</a></sup></p>
<p><a name="References" id="References"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=10" title="Edit section: References">edit</a>]</span> <span class="mw-headline">References</span></h2>
<div class="references-small" style="-moz-column-count:2; -webkit-column-count:2; column-count:2;">
<ol class="references">
<li id="_note-Bartlett_.26_Stirling"><b><a href="#_ref-Bartlett_.26_Stirling_0" title="">^</a></b> <a href="http://biomed.humanapress.com/index.php?option=com_opbookdetails&amp;task=chapterdetails&amp;chapter_code=1-59259-384-4:3&amp;category=biomedprotocols" class="external text" title="http://biomed.humanapress.com/index.php?option=com_opbookdetails&amp;task=chapterdetails&amp;chapter_code=1-59259-384-4:3&amp;category=biomedprotocols" rel="nofollow">Bartlett &amp; Stirling (2003)—A Short History of the Polymerase Chain Reaction. In: Methods Mol Biol. 226:3-6</a></li>
<li id="_note-Karry_Mullis_Nobel_Lecture">^ <a href="#_ref-Karry_Mullis_Nobel_Lecture_0" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-Karry_Mullis_Nobel_Lecture_1" title=""><sup><i><b>b</b></i></sup></a> <a href="http://nobelprize.org/nobel_prizes/chemistry/laureates/1993/mullis-lecture.html" class="external text" title="http://nobelprize.org/nobel_prizes/chemistry/laureates/1993/mullis-lecture.html" rel="nofollow">Karry Mullis Nobel Lecture, December 8, 1993</a></li>
<li id="_note-0"><b><a href="#_ref-0" title="">^</a></b> <cite style="font-style:normal">Cheng S, Fockler C, Barnes WM, Higuchi R (1994). "Effective amplification of long targets from cloned inserts and human genomic DNA". <i>Proc Natl Acad Sci.</i> <b>91</b>: 5695-5699. <a href="http://www.ncbi.nlm.nih.gov/pubmed/8202550" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/8202550">PMID 8202550</a>.</cite></li>
<li id="_note-molecular_cloning">^ <a href="#_ref-molecular_cloning_0" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-molecular_cloning_1" title=""><sup><i><b>b</b></i></sup></a> <cite class="book" style="font-style:normal">Joseph Sambrook and David W. Russel (2001). <i>Molecular Cloning: A Laboratory Manual</i>, 3rd ed., Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press. <a href="/w/index.php?title=Special:Booksources&amp;isbn=0879695765" class="internal">ISBN 0-87969-576-5</a>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Molecular+Cloning%3A+A+Laboratory+Manual&amp;rft.au=Joseph+Sambrook+and+David+W.+Russel&amp;rft.edition=3rd+ed.&amp;rft.pub=Cold+Spring+Harbor+Laboratory+Press&amp;rft.place=Cold+Spring+Harbor%2C+N.Y.">&#160;</span> Chapter 8: In vitro Amplification of DNA by the Polymerase Chain Reaction</li>
<li id="_note-1"><b><a href="#_ref-1" title="">^</a></b> <cite style="font-style:normal">Pavlov AR, Pavlova NV, Kozyavkin SA, Slesarev AI (2004). "Recent developments in the optimization of thermostable DNA polymerases for efficient applications". <i>Trends Biotechnol.</i> <b>22</b>: 253-260. <a href="http://www.ncbi.nlm.nih.gov/pubmed/15109812" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/15109812">PMID 15109812</a>.</cite></li>
<li id="_note-2"><b><a href="#_ref-2" title="">^</a></b> <cite style="font-style:normal">Rychlik W, Spencer WJ, Rhoads RE (1990). "<a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=2243783" class="external text" title="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=2243783" rel="nofollow">Optimization of the annealing temperature for DNA amplification <i>in vitro</i></a>". <i>Nucl Acids Res</i> <b>18</b>: 6409-6412.</cite></li>
<li id="_note-antibody_hot_start">^ <a href="#_ref-antibody_hot_start_0" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-antibody_hot_start_1" title=""><sup><i><b>b</b></i></sup></a> <cite style="font-style:normal">D.J. Sharkey, E.R. Scalice, K.G. Christy Jr., S.M. Atwood, and J.L. Daiss (1994). "Antibodies as Thermolabile Switches: High Temperature Triggering for the Polymerase Chain Reaction". <i>Bio/Technology</i> <b>12</b>: 506-509.</cite></li>
<li id="_note-Chien_et_al">^ <a href="#_ref-Chien_et_al_0" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-Chien_et_al_1" title=""><sup><i><b>b</b></i></sup></a> <cite style="font-style:normal">Chien A, Edgar DB, Trela JM (1976). "Deoxyribonucleic acid polymerase from the extreme thermophile Thermus aquaticus". <i>J. Bacteriol</i> <b>174</b>: 1550-1557. <a href="http://www.ncbi.nlm.nih.gov/pubmed/8432" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/8432">PMID 8432</a>.</cite></li>
<li id="_note-Lawyer_et_al">^ <a href="#_ref-Lawyer_et_al_0" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-Lawyer_et_al_1" title=""><sup><i><b>b</b></i></sup></a> <cite style="font-style:normal">Lawyer FC, Stoffel S, Saiki RK, Chang SY, Landre PA, Abramson RD, Gelfand DH (1993). "High-level expression, purification, and enzymatic characterization of full-length Thermus aquaticus DNA polymerase and a truncated form deficient in 5' to 3' exonuclease activity". <i>PCR Methods Appl.</i> <b>2</b>: 275-287. <a href="http://www.ncbi.nlm.nih.gov/pubmed/8324500" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/8324500">PMID 8324500</a>.</cite></li>
<li id="_note-3"><b><a href="#_ref-3" title="">^</a></b> PCR from problematic templates. Focus 22:1 p.10 (2000).</li>
<li id="_note-4"><b><a href="#_ref-4" title="">^</a></b> Helpful tips for PCR. Focus 22:1 p.12 (2000).</li>
<li id="_note-5"><b><a href="#_ref-5" title="">^</a></b> <a href="http://photoscience.la.asu.edu/photosyn/courses/BIO_343/lecture/DNAtech.html" class="external text" title="http://photoscience.la.asu.edu/photosyn/courses/BIO_343/lecture/DNAtech.html" rel="nofollow">Chemical Synthesis, Sequencing, and Amplification of DNA (class notes on MBB/BIO 343)</a>. Arizona State University. Retrieved on <a href="/w/index.php?title=29_October_2007&amp;action=edit" class="new" title="29 October 2007">29 October 2007</a>.</li>
<li id="_note-Saiki_et_al"><b><a href="#_ref-Saiki_et_al_0" title="">^</a></b> <cite style="font-style:normal">Saiki RK, Bugawan TL, Horn GT, Mullis KB, Erlich HA (1986). "Analysis of enzymatically amplified beta-globin and HLA-DQ alpha DNA with allele-specific oligonucleotide probes.". <i>Nature</i> <b>324</b>: 163-166. <a href="http://www.ncbi.nlm.nih.gov/pubmed/3785382" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/3785382">PMID 3785382</a>.</cite></li>
<li id="_note-Stemmer_et_al"><b><a href="#_ref-Stemmer_et_al_0" title="">^</a></b> <cite style="font-style:normal">Stemmer WP, Crameri A, Ha KD, Brennan TM, Heyneker HL (1995). "Single-step assembly of a gene and entire plasmid from large numbers of oligodeoxyribonucleotides". <i>Gene</i> <b>164</b>: 49-53. <a href="http://www.ncbi.nlm.nih.gov/pubmed/7590320" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/7590320">PMID 7590320</a>.</cite></li>
<li id="_note-Innis_et_al"><b><a href="#_ref-Innis_et_al_0" title="">^</a></b> <cite style="font-style:normal">Innis MA, Myambo KB, Gelfand DH, Brow MA. (1988). "DNA sequencing with Thermus aquaticus DNA polymerase and direct sequencing of polymerase chain reaction-amplified DNA". <i>Proc Natl Acad Sci USA</i> <b>85</b>: 9436-4940. <a href="http://www.ncbi.nlm.nih.gov/pubmed/3200828" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/3200828">PMID 3200828</a>.</cite></li>
<li id="_note-Pierce_and_Wangh"><b><a href="#_ref-Pierce_and_Wangh_0" title="">^</a></b> <cite style="font-style:normal">Pierce KE and Wangh LJ (2007). "Linear-after-the-exponential polymerase chain reaction and allied technologies Real-time detection strategies for rapid, reliable diagnosis from single cells". <i>Methods Mol Med.</i> <b>132</b>: 65-85. <a href="http://www.ncbi.nlm.nih.gov/pubmed/17876077" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/17876077">PMID 17876077</a>.</cite></li>
<li id="_note-Hybrid"><b><a href="#_ref-Hybrid_0" title="">^</a></b> <cite class="book" style="font-style:normal">Pavlov AR, Pavlova NV, Kozyavkin SA, Slesarev AI (2006). "Thermostable DNA Polymerases for a Wide Spectrum of Applications: Comparison of a Robust Hybrid TopoTaq to other enzymes", in Kieleczawa J: <i>DNA Sequencing II: Optimizing Preparation and Cleanup</i>. Jones and Bartlett, pp. 241-257. <a href="/w/index.php?title=Special:Booksources&amp;isbn=0763733830" class="internal">ISBN 0-7637338-3-0</a>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=DNA+Sequencing+II%3A+Optimizing+Preparation+and+Cleanup&amp;rft.atitle=Thermostable+DNA+Polymerases+for+a+Wide+Spectrum+of+Applications%3A+Comparison+of+a+Robust+Hybrid+TopoTaq+to+other+enzymes&amp;rft.au=Pavlov+AR%2C+Pavlova+NV%2C+Kozyavkin+SA%2C+Slesarev+AI&amp;rft.pub=Jones+and+Bartlett&amp;rft.pages=pp.+241-257">&#160;</span></li>
<li id="_note-6"><b><a href="#_ref-6" title="">^</a></b> <cite style="font-style:normal">Myriam Vincent, Yan Xu and Huimin Kong (2004). "Helicase-dependent isothermal DNA amplification". <i>EMBO reports</i> <b>5</b> (8): 795–800.</cite></li>
<li id="_note-general_hot_start"><b><a href="#_ref-general_hot_start_0" title="">^</a></b> <cite style="font-style:normal">Q. Chou, M. Russell, D.E. Birch, J. Raymond and W. Bloch (1992). "Prevention of pre-PCR mis-priming and primer dimerization improves low-copy-number amplifications". <i>Nucleic Acids Research</i> <b>20</b>: 1717-1723.</cite></li>
<li id="_note-7"><b><a href="#_ref-7" title="">^</a></b> <cite style="font-style:normal">E. Zietkiewicz, A. Rafalski, and D. Labuda (1994). "Genome fingerprinting by simple sequence repeat (SSR)-anchored polymerase chain reaction amplification". <i>Genomics</i> <b>20</b> (2): 176-83.</cite></li>
<li id="_note-Ochman_et_al"><b><a href="#_ref-Ochman_et_al_0" title="">^</a></b> <cite style="font-style:normal">Ochman H, Gerber AS, Hartl DL (1988). "Genetic applications of an inverse polymerase chain reaction". <i>Genetics</i> <b>120</b>: 621-623. <a href="http://www.ncbi.nlm.nih.gov/pubmed/2852134" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/2852134">PMID 2852134</a>.</cite></li>
<li id="_note-Mueller_and_Wold"><b><a href="#_ref-Mueller_and_Wold_0" title="">^</a></b> <cite style="font-style:normal">Mueller PR, Wold B (1988). "<i>In vivo</i> footprinting of a muscle specific enhancer by ligation mediated PCR". <i>Science</i> <b>246</b>: 780-786. <a href="http://www.ncbi.nlm.nih.gov/pubmed/2814500" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/2814500">PMID 2814500</a>.</cite></li>
<li id="_note-8"><b><a href="#_ref-8" title="">^</a></b> <cite style="font-style:normal">Herman JG, Graff JR, Myöhänen S, Nelkin BD, Baylin SB (1996). "Methylation-specific PCR: a novel PCR assay for methylation status of CpG islands". <i>Proc Natl Acad Sci U S A</i> <b>93</b> (13): 9821-9826. <a href="http://www.ncbi.nlm.nih.gov/pubmed/8790415" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/8790415">PMID 8790415</a>.</cite></li>
<li id="_note-9"><b><a href="#_ref-9" title="">^</a></b> <cite style="font-style:normal">Y.G. Liu and R. F. Whittier (1995). "Thermal asymmetric interlaced PCR: automatable amplification and sequencing of insert end fragments from P1 and YAC clones for chromosome walking". <i>Genomics</i> <b>25</b> (3): 674-81.</cite></li>
<li id="_note-10"><b><a href="#_ref-10" title="">^</a></b> <cite style="font-style:normal">Don RH, Cox PT, Wainwright BJ, Baker K, Mattick JS (1991). "<a href="http://www.pubmedcentral.nih.gov/pagerender.fcgi?artid=328507&amp;pageindex=1" class="external text" title="http://www.pubmedcentral.nih.gov/pagerender.fcgi?artid=328507&amp;pageindex=1" rel="nofollow">'Touchdown' PCR to circumvent spurious priming during gene amplification.</a>". <i>Nucl Acids Res</i> <b>19</b>: 4008.</cite></li>
<li id="_note-11"><b><a href="#_ref-11" title="">^</a></b> <cite style="font-style:normal">David, F.Turlotte, E., (1998). "An Isothermal Amplification Method". <i>C.R.Acad. Sci Paris, Life Science</i> <b>321</b> (1): 909-914.</cite></li>
<li id="_note-12"><b><a href="#_ref-12" title="">^</a></b> Fabrice David (September-October 2002). <a href="http://www.lab-rech-associatives.com/pdf/Utiliser%20la%20Topologie%20de%20l'ADN.pdf" class="external text" title="http://www.lab-rech-associatives.com/pdf/Utiliser%20la%20Topologie%20de%20l'ADN.pdf" rel="nofollow">Utiliser les propriétés topologiques de l’ADN: une nouvelle arme contre les agents pathogènes</a>. Fusion.(in French)</li>
<li id="_note-13"><b><a href="#_ref-13" title="">^</a></b> <cite style="font-style:normal">Kleppe K, Ohtsuka E, Kleppe R, Molineux I, Khorana HG (1971). "<a href="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WK7-4DKM9WR-36&amp;_user=103681&amp;_coverDate=03%2F14%2F1971&amp;_rdoc=10&amp;_fmt=summary&amp;_orig=browse&amp;_srch=doc-info(%23toc%236899%231971%23999439997%23524037%23FLA%23display%23Volume)&amp;_cdi=6899&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_ct=18&amp;_acct=C000007920&amp;_version=1&amp;_urlVersion=0&amp;_userid=103681&amp;md5=5e1e29e72e51ebd62d1b60d1bd5f9058" class="external text" title="http://www.sciencedirect.com/science?_ob=ArticleURL&amp;_udi=B6WK7-4DKM9WR-36&amp;_user=103681&amp;_coverDate=03%2F14%2F1971&amp;_rdoc=10&amp;_fmt=summary&amp;_orig=browse&amp;_srch=doc-info(%23toc%236899%231971%23999439997%23524037%23FLA%23display%23Volume)&amp;_cdi=6899&amp;_sort=d&amp;_docanchor=&amp;view=c&amp;_ct=18&amp;_acct=C000007920&amp;_version=1&amp;_urlVersion=0&amp;_userid=103681&amp;md5=5e1e29e72e51ebd62d1b60d1bd5f9058" rel="nofollow">Studies on polynucleotides. XCVI. Repair replications of short synthetic DNA's as catalyzed by DNA polymerases</a>". <i>J. Mol. Biol.</i> <b>56</b>: 341-361.</cite></li>
<li id="_note-Mullis">^ <a href="#_ref-Mullis_0" title=""><sup><i><b>a</b></i></sup></a> <a href="#_ref-Mullis_1" title=""><sup><i><b>b</b></i></sup></a> <cite class="book" style="font-style:normal" id="Reference-Mullis-1998"><a href="/wiki/Kary_Mullis" title="Kary Mullis">Mullis, Kary</a> (1998). <i>Dancing Naked in the Mind Field</i>. New York: Pantheon Books. <a href="/w/index.php?title=Special:Booksources&amp;isbn=0679442553" class="internal">ISBN 0-679-44255-3</a>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Dancing+Naked+in+the+Mind+Field&amp;rft.aulast=Mullis&amp;rft.aufirst=Kary&amp;rft.pub=Pantheon+Books&amp;rft.place=New+York&amp;rft.isbn=0-679-44255-3">&#160;</span></li>
<li id="_note-14"><b><a href="#_ref-14" title="">^</a></b> <cite class="book" style="font-style:normal" id="Reference-Rabinow-1996"><a href="/wiki/Paul_Rabinow" title="Paul Rabinow">Rabinow, Paul</a> (1996). <i>Making PCR: A Story of Biotechnology</i>. Chicago: University of Chicago Press. <a href="/w/index.php?title=Special:Booksources&amp;isbn=0226701468" class="internal">ISBN 0-226-70146-8</a>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Making+PCR%3A+A+Story+of+Biotechnology&amp;rft.aulast=Rabinow&amp;rft.aufirst=Paul&amp;rft.pub=University+of+Chicago+Press&amp;rft.place=Chicago&amp;rft.isbn=0-226-70146-8">&#160;</span></li>
<li id="_note-Mullisp18"><b><a href="#_ref-Mullisp18_0" title="">^</a></b> <cite class="book" style="font-style:normal" id="Reference-Mullis-1998">Mullis, Kary (1998). <i>Dancing Naked in the Mind Field</i>. New York: Pantheon Books, 18. <a href="/w/index.php?title=Special:Booksources&amp;isbn=0679442553" class="internal">ISBN 0-679-44255-3</a>.</cite><span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook&amp;rft.genre=book&amp;rft.btitle=Dancing+Naked+in+the+Mind+Field&amp;rft.aulast=Mullis&amp;rft.aufirst=Kary&amp;rft.au=Karry+Mullis&amp;rft.pub=Pantheon+Books&amp;rft.place=New+York&amp;rft.pages=18&amp;rft.isbn=0-679-44255-3">&#160;</span></li>
<li id="_note-15"><b><a href="#_ref-15" title="">^</a></b> <cite style="font-style:normal">Mullis, Kary (1990). "The unusual origin of the polymerase chain reaction". <i>Scientific American</i> <b>262</b> (4): 56-61, 64-5.</cite></li>
<li id="_note-16"><b><a href="#_ref-16" title="">^</a></b> <a href="http://www.genengnews.com/articles/chitem.aspx?aid=1656&amp;chid=0" class="external text" title="http://www.genengnews.com/articles/chitem.aspx?aid=1656&amp;chid=0" rel="nofollow">Advice on How to Survive the Taq Wars ¶2</a>: <a href="/w/index.php?title=GEN_Genetic_Engineering_News&amp;action=edit" class="new" title="GEN Genetic Engineering News">GEN Genetic Engineering News</a> Biobusiness Channel: Article. May 1 2006 (Vol. 26, No. 9).</li>
</ol>
</div>
<p><a name="External_links" id="External_links"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit&amp;section=11" title="Edit section: External links">edit</a>]</span> <span class="mw-headline">External links</span></h2>
<div class="infobox sisterproject">
<div class="floatleft"><span><a href="/wiki/Image:Commons-logo.svg" class="image" title="Commons-logo.svg"><img alt="" src="http://upload.wikimedia.org/wikipedia/en/thumb/4/4a/Commons-logo.svg/50px-Commons-logo.svg.png" width="50" height="67" border="0" /></a></span></div>
<div style="margin-left: 60px;">Wikimedia Commons has media related to:
<div style="margin-left: 10px;"><i><b><a href="http://commons.wikimedia.org/wiki/Polymerase_chain_reaction" class="extiw" title="commons:Polymerase_chain_reaction">Polymerase chain reaction</a></b></i></div>
</div>
</div>
<ul>
<li><a href="http://www.sciam.com/article.cfm?articleID=00035C6C-229B-1C74-9B81809EC588EF21" class="external text" title="http://www.sciam.com/article.cfm?articleID=00035C6C-229B-1C74-9B81809EC588EF21" rel="nofollow">PCR at Home</a> - Amateur Scientist article in the July 2000 issue of <a href="/wiki/Scientific_American" title="Scientific American">Scientific American</a> on performing PCR reactions with low-cost household materials.</li>
<li><a href="http://patft.uspto.gov/netacgi/nph-Parser?Sect2=PTO1&amp;Sect2=HITOFF&amp;p=1&amp;u=%2Fnetahtml%2FPTO%2Fsearch-bool.html&amp;r=1&amp;f=G&amp;l=50&amp;d=PALL&amp;RefSrch=yes&amp;Query=PN%2F4683202" class="external text" title="http://patft.uspto.gov/netacgi/nph-Parser?Sect2=PTO1&amp;Sect2=HITOFF&amp;p=1&amp;u=%2Fnetahtml%2FPTO%2Fsearch-bool.html&amp;r=1&amp;f=G&amp;l=50&amp;d=PALL&amp;RefSrch=yes&amp;Query=PN%2F4683202" rel="nofollow">US Patent for PCR</a></li>
<li><a href="http://www.sumanasinc.com/webcontent/animations/content/pcr.html" class="external text" title="http://www.sumanasinc.com/webcontent/animations/content/pcr.html" rel="nofollow">Narrated animation and step-through animation of PCR</a> - From the educational multimedia company Sumanas. Adobe Flash required.</li>
<li><a href="http://www.dnalc.org/ddnalc/resources/pcr.html" class="external text" title="http://www.dnalc.org/ddnalc/resources/pcr.html" rel="nofollow">Step-through animation of PCR</a> - From <a href="/wiki/Cold_Spring_Harbor_Laboratory" title="Cold Spring Harbor Laboratory">Cold Spring Harbor</a>'s Dolan DNA Learning Center. Adobe Flash required.</li>
<li><a href="http://insilico.ehu.es/edu/PCR_en" class="external text" title="http://insilico.ehu.es/edu/PCR_en" rel="nofollow">PCR exercise</a> - A PCR problem will be generated and students will be requested to design an experiment to solve and simulate it.</li>
<li><a href="http://bio-rad.cnpg.com/lsca/videos/ScientistsForBetterPCR/" class="external text" title="http://bio-rad.cnpg.com/lsca/videos/ScientistsForBetterPCR/" rel="nofollow">PCR music video</a></li>
</ul>
<p><span id="interwiki-de-fa"></span></p>

<!-- 
Pre-expand include size: 179511/2048000 bytes
Post-expand include size: 51510/2048000 bytes
Template argument size: 39261/2048000 bytes
#ifexist count: 0/500
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:23647-0!1!0!default!!en!2 and timestamp 20080123122557 -->
<div class="printfooter">
Retrieved from "<a href="http://en.wikipedia.org/wiki/Polymerase_chain_reaction">http://en.wikipedia.org/wiki/Polymerase_chain_reaction</a>"</div>
			<div id="catlinks"><p class='catlinks'><a href="/wiki/Special:Categories" title="Special:Categories">Categories</a>: <span dir='ltr'><a href="/wiki/Category:Molecular_biology" title="Category:Molecular biology">Molecular biology</a></span> | <span dir='ltr'><a href="/wiki/Category:Polymerase_chain_reaction" title="Category:Polymerase chain reaction">Polymerase chain reaction</a></span> | <span dir='ltr'><a href="/wiki/Category:Laboratory_techniques" title="Category:Laboratory techniques">Laboratory techniques</a></span> | <span dir='ltr'><a href="/wiki/Category:DNA_profiling_techniques" title="Category:DNA profiling techniques">DNA profiling techniques</a></span> | <span dir='ltr'><a href="/wiki/Category:Amplifiers" title="Category:Amplifiers">Amplifiers</a></span> | <span dir='ltr'><a href="/wiki/Category:Hoffmann-La_Roche" title="Category:Hoffmann-La Roche">Hoffmann-La Roche</a></span></p></div>			<!-- end content -->
			<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
					 <li id="ca-nstab-main" class="selected"><a href="/wiki/Polymerase_chain_reaction" title="View the content page [c]" accesskey="c">Article</a></li>
					 <li id="ca-talk"><a href="/wiki/Talk:Polymerase_chain_reaction" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
					 <li id="ca-edit"><a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=edit" title="You can edit this page. Please use the preview button before saving. [e]" accesskey="e">Edit this page</a></li>
					 <li id="ca-history"><a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=history" title="Past versions of this page [h]" accesskey="h">History</a></li>
					 <li id="ca-move"><a href="/wiki/Special:Movepage/Polymerase_chain_reaction" title="Rename this page [m]" accesskey="m">Move</a></li>
					 <li id="ca-watch"><a href="/w/index.php?title=Polymerase_chain_reaction&amp;action=watch" title="Add this page to your watchlist [w]" accesskey="w">Watch</a></li>
				</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-userpage"><a href="/wiki/User:Kanzure" title="My user page [.]" accesskey=".">Kanzure</a></li>
				<li id="pt-mytalk"><a href="/wiki/User_talk:Kanzure" title="My talk page [n]" accesskey="n">My talk</a></li>
				<li id="pt-preferences"><a href="/wiki/Special:Preferences" title="My preferences">My preferences</a></li>
				<li id="pt-watchlist"><a href="/wiki/Special:Watchlist" title="The list of pages that you are monitoring for changes [l]" accesskey="l">My watchlist</a></li>
				<li id="pt-mycontris"><a href="/wiki/Special:Contributions/Kanzure" title="List of my contributions [y]" accesskey="y">My contributions</a></li>
				<li id="pt-logout"><a href="/w/index.php?title=Special:Userlogout&amp;returnto=Polymerase_chain_reaction" title="Log out">Log out</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(/images/wiki-en.png);" href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
		<div class='portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-Contents"><a href="/wiki/Portal:Contents" title="Guides to browsing Wikipedia">Contents</a></li>
				<li id="n-featuredcontent"><a href="/wiki/Portal:Featured_content">Featured content</a></li>
				<li id="n-currentevents"><a href="/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li>
				<li id="n-randompage"><a href="/wiki/Special:Random" title="Load a random article [x]" accesskey="x">Random article</a></li>
			</ul>
		</div>
	</div>
		<div class='portlet' id='p-interaction'>
		<h5>Interaction</h5>
		<div class='pBody'>
			<ul>
				<li id="n-About-Wikipedia"><a href="/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li>
				<li id="n-portal"><a href="/wiki/Wikipedia:Community_Portal" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="/wiki/Special:Recentchanges" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-contact"><a href="/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact Wikipedia</a></li>
				<li id="n-sitesupport"><a href="http://wikimediafoundation.org/wiki/Donate" title="Support us">Donate to Wikipedia</a></li>
				<li id="n-help"><a href="/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li>
			</ul>
		</div>
	</div>
		<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="/wiki/Special:Search" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search Wikipedia [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if one exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the wiki for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="/wiki/Special:Whatlinkshere/Polymerase_chain_reaction" title="List of all English Wikipedia pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="/wiki/Special:Recentchangeslinked/Polymerase_chain_reaction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-upload"><a href="/wiki/Wikipedia:Upload" title="Upload images or media files [u]" accesskey="u">Upload file</a></li>
<li id="t-specialpages"><a href="/wiki/Special:Specialpages" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="/w/index.php?title=Polymerase_chain_reaction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="/w/index.php?title=Polymerase_chain_reaction&amp;oldid=185291002" title="Permanent link to this version of the page">Permanent link</a></li><li id="t-cite"><a href="/w/index.php?title=Special:Cite&amp;page=Polymerase_chain_reaction&amp;id=185291002">Cite this page</a></li>			</ul>
		</div>
	</div>
	<div id="p-lang" class="portlet">
		<h5>Languages</h5>
		<div class="pBody">
			<ul>
				<li class="interwiki-ar"><a href="http://ar.wikipedia.org/wiki/%D8%AA%D9%81%D8%A7%D8%B9%D9%84_%D8%A7%D9%84%D8%A8%D9%84%D9%85%D8%B1%D8%A9_%D8%A7%D9%84%D9%85%D8%AA%D8%B3%D9%84%D8%B3%D9%84">العربية</a></li>
				<li class="interwiki-az"><a href="http://az.wikipedia.org/wiki/Polimeraz_z%C9%99ncir_reaksiyas%C4%B1">Azərbaycan</a></li>
				<li class="interwiki-bg"><a href="http://bg.wikipedia.org/wiki/%D0%9F%D0%BE%D0%BB%D0%B8%D0%BC%D0%B5%D1%80%D0%B0%D0%B7%D0%BD%D0%B0_%D0%B2%D0%B5%D1%80%D0%B8%D0%B6%D0%BD%D0%B0_%D1%80%D0%B5%D0%B0%D0%BA%D1%86%D0%B8%D1%8F">Български</a></li>
				<li class="interwiki-ca"><a href="http://ca.wikipedia.org/wiki/Reacci%C3%B3_en_cadena_per_la_polimerasa">Català</a></li>
				<li class="interwiki-cs"><a href="http://cs.wikipedia.org/wiki/Polymer%C3%A1zov%C3%A1_%C5%99et%C4%9Bzov%C3%A1_reakce">Česky</a></li>
				<li class="interwiki-da"><a href="http://da.wikipedia.org/wiki/PCR">Dansk</a></li>
				<li class="interwiki-de"><a href="http://de.wikipedia.org/wiki/Polymerase-Kettenreaktion">Deutsch</a></li>
				<li class="interwiki-el"><a href="http://el.wikipedia.org/wiki/%CE%91%CE%BB%CF%85%CF%83%CE%B9%CE%B4%CF%89%CF%84%CE%AE_%CE%B1%CE%BD%CF%84%CE%AF%CE%B4%CF%81%CE%B1%CF%83%CE%B7_%CF%80%CE%BF%CE%BB%CF%85%CE%BC%CE%B5%CF%81%CE%AC%CF%83%CE%B7%CF%82">Ελληνικά</a></li>
				<li class="interwiki-es"><a href="http://es.wikipedia.org/wiki/Reacci%C3%B3n_en_cadena_de_la_polimerasa">Español</a></li>
				<li class="interwiki-eo"><a href="http://eo.wikipedia.org/wiki/P%C4%88R">Esperanto</a></li>
				<li class="interwiki-fr"><a href="http://fr.wikipedia.org/wiki/R%C3%A9action_en_cha%C3%AEne_par_polym%C3%A9rase">Français</a></li>
				<li class="interwiki-ko"><a href="http://ko.wikipedia.org/wiki/%EC%A4%91%ED%95%A9_%ED%9A%A8%EC%86%8C_%EC%97%B0%EC%87%84_%EB%B0%98%EC%9D%91">한국어</a></li>
				<li class="interwiki-hr"><a href="http://hr.wikipedia.org/wiki/PCR">Hrvatski</a></li>
				<li class="interwiki-id"><a href="http://id.wikipedia.org/wiki/Reaksi_berantai_polimerase">Bahasa Indonesia</a></li>
				<li class="interwiki-it"><a href="http://it.wikipedia.org/wiki/Reazione_a_catena_della_polimerasi">Italiano</a></li>
				<li class="interwiki-he"><a href="http://he.wikipedia.org/wiki/PCR">עברית</a></li>
				<li class="interwiki-lt"><a href="http://lt.wikipedia.org/wiki/PGR">Lietuvių</a></li>
				<li class="interwiki-hu"><a href="http://hu.wikipedia.org/wiki/PCR">Magyar</a></li>
				<li class="interwiki-mn"><a href="http://mn.wikipedia.org/wiki/%D0%9F%D0%BE%D0%BB%D0%B8%D0%BC%D0%B5%D1%80%D0%B0%D0%B7%D1%8B%D0%BD_%D0%B3%D0%B8%D0%BD%D0%B6%D0%B8%D0%BD_%D1%83%D1%80%D0%B2%D0%B0%D0%BB">Монгол</a></li>
				<li class="interwiki-nl"><a href="http://nl.wikipedia.org/wiki/Polymerase-kettingreactie">Nederlands</a></li>
				<li class="interwiki-ja"><a href="http://ja.wikipedia.org/wiki/%E3%83%9D%E3%83%AA%E3%83%A1%E3%83%A9%E3%83%BC%E3%82%BC%E9%80%A3%E9%8E%96%E5%8F%8D%E5%BF%9C">日本語</a></li>
				<li class="interwiki-no"><a href="http://no.wikipedia.org/wiki/PCR">‪Norsk (bokmål)‬</a></li>
				<li class="interwiki-pl"><a href="http://pl.wikipedia.org/wiki/Reakcja_%C5%82a%C5%84cuchowa_polimerazy">Polski</a></li>
				<li class="interwiki-pt"><a href="http://pt.wikipedia.org/wiki/Rea%C3%A7%C3%A3o_em_cadeia_da_polimerase">Português</a></li>
				<li class="interwiki-ru"><a href="http://ru.wikipedia.org/wiki/%D0%9F%D0%BE%D0%BB%D0%B8%D0%BC%D0%B5%D1%80%D0%B0%D0%B7%D0%BD%D0%B0%D1%8F_%D1%86%D0%B5%D0%BF%D0%BD%D0%B0%D1%8F_%D1%80%D0%B5%D0%B0%D0%BA%D1%86%D0%B8%D1%8F">Русский</a></li>
				<li class="interwiki-sl"><a href="http://sl.wikipedia.org/wiki/Veri%C5%BEna_reakcija_s_polimerazo">Slovenščina</a></li>
				<li class="interwiki-sr"><a href="http://sr.wikipedia.org/wiki/%D0%9F%D0%BE%D0%BB%D0%B8%D0%BC%D0%B5%D1%80%D0%B0%D0%B7%D0%BD%D0%B0_%D0%BB%D0%B0%D0%BD%D1%87%D0%B0%D0%BD%D0%B0_%D1%80%D0%B5%D0%B0%D0%BA%D1%86%D0%B8%D1%98%D0%B0">Српски / Srpski</a></li>
				<li class="interwiki-fi"><a href="http://fi.wikipedia.org/wiki/PCR">Suomi</a></li>
				<li class="interwiki-sv"><a href="http://sv.wikipedia.org/wiki/PCR">Svenska</a></li>
				<li class="interwiki-vi"><a href="http://vi.wikipedia.org/wiki/PCR">Tiếng Việt</a></li>
				<li class="interwiki-tr"><a href="http://tr.wikipedia.org/wiki/Polimeraz_zincir_tepkimesi">Türkçe</a></li>
				<li class="interwiki-uk"><a href="http://uk.wikipedia.org/wiki/%D0%9F%D0%BE%D0%BB%D1%96%D0%BC%D0%B5%D1%80%D0%B0%D0%B7%D0%BD%D0%B0_%D0%BB%D0%B0%D0%BD%D1%86%D1%8E%D0%B3%D0%BE%D0%B2%D0%B0_%D1%80%D0%B5%D0%B0%D0%BA%D1%86%D1%96%D1%8F">Українська</a></li>
				<li class="interwiki-ur"><a href="http://ur.wikipedia.org/wiki/%D9%BE%DB%8C_%D8%B3%DB%8C_%D8%A2%D8%B1">اردو</a></li>
				<li class="interwiki-zh"><a href="http://zh.wikipedia.org/wiki/%E8%81%9A%E5%90%88%E9%85%B6%E9%93%BE%E9%94%81%E5%8F%8D%E5%BA%94">中文</a></li>
			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="/skins-1.5/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="http://wikimediafoundation.org/"><img src="/images/wikimedia-button.png" border="0" alt="Wikimedia Foundation"/></a></div>
			<ul id="f-list">
				<li id="lastmod"> This page was last modified 21:47, 18 January 2008.</li>
				<li id="copyright">All text is available under the terms of the <a class='internal' href="http://en.wikipedia.org/wiki/Wikipedia:Text_of_the_GNU_Free_Documentation_License" title="Wikipedia:Text of the GNU Free Documentation License">GNU Free Documentation License</a>. (See <b><a class='internal' href="http://en.wikipedia.org/wiki/Wikipedia:Copyrights" title="Wikipedia:Copyrights">Copyrights</a></b> for details.) <br /> Wikipedia&reg; is a registered trademark of the <a href="http://www.wikimediafoundation.org">Wikimedia Foundation, Inc</a>., a U.S. registered <a class='internal' href="http://en.wikipedia.org/wiki/501%28c%29#501.28c.29.283.29" title="501(c)(3)">501(c)(3)</a> <a href="http://wikimediafoundation.org/wiki/Deductibility_of_donations">tax-deductible</a> <a class='internal' href="http://en.wikipedia.org/wiki/Non-profit_organization" title="Non-profit organization">nonprofit</a> <a href="http://en.wikipedia.org/wiki/Charitable_organization" title="Charitable organization">charity</a>.<br /></li>
				<li id="privacy"><a href="http://wikimediafoundation.org/wiki/Privacy_policy" title="wikimedia:Privacy policy">Privacy policy</a></li>
				<li id="about"><a href="/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
				<li id="disclaimer"><a href="/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
		
	
		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
</div>
<!-- Served by srv156 in 0.045 secs. --></body></html>
