<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
	<head>
		<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
				<meta name="keywords" content="Proteomics,Molecular and Cellular Biology,Portals,Articles with unsourced statements since August 2007,2-D electrophoresis,Acute myocardial infarction,Affinity chromatography,Alternative splicing,Alzheimer’s disease,Biochemistry,Bioinformatics" />
		<link rel="shortcut icon" href="/favicon.ico" />
		<link rel="apple-touch-icon" href="http://en.wikipedia.org/apple-touch-icon.png" />
		<link rel="search" type="application/opensearchdescription+xml" href="/w/opensearch_desc.php" title="Wikipedia (English)" />
		<link rel="copyright" href="http://www.gnu.org/copyleft/fdl.html" />
<link rel="alternate" type="application/rss+xml" title="Wikipedia RSS Feed" href="http://en.wikipedia.org/w/index.php?title=Special:Recentchanges&amp;feed=rss" />
<link rel="alternate" type="application/atom+xml" title="Wikipedia Atom Feed" href="http://en.wikipedia.org/w/index.php?title=Special:Recentchanges&amp;feed=atom" />
		<title>Proteomics - Wikipedia, the free encyclopedia</title>
		<style type="text/css" media="screen, projection">/*<![CDATA[*/
			@import "/skins-1.5/common/shared.css?107";
			@import "/skins-1.5/monobook/main.css?107";
		/*]]>*/</style>
		<link rel="stylesheet" type="text/css" media="print" href="/skins-1.5/common/commonPrint.css?107" />
		<!--[if lt IE 5.5000]><style type="text/css">@import "/skins-1.5/monobook/IE50Fixes.css?107";</style><![endif]-->
		<!--[if IE 5.5000]><style type="text/css">@import "/skins-1.5/monobook/IE55Fixes.css?107";</style><![endif]-->
		<!--[if IE 6]><style type="text/css">@import "/skins-1.5/monobook/IE60Fixes.css?107";</style><![endif]-->
		<!--[if IE 7]><style type="text/css">@import "/skins-1.5/monobook/IE70Fixes.css?107";</style><![endif]-->
		<!--[if lt IE 7]><script type="text/javascript" src="/skins-1.5/common/IEFixes.js?107"></script>
		<meta http-equiv="imagetoolbar" content="no" /><![endif]-->
		
		<script type= "text/javascript">/*<![CDATA[*/
var skin = "monobook";
var stylepath = "/skins-1.5";
var wgArticlePath = "/wiki/$1";
var wgScriptPath = "/w";
var wgScript = "/w/index.php";
var wgServer = "http://en.wikipedia.org";
var wgCanonicalNamespace = "";
var wgCanonicalSpecialPageName = false;
var wgNamespaceNumber = 0;
var wgPageName = "Proteomics";
var wgTitle = "Proteomics";
var wgAction = "view";
var wgRestrictionEdit = [];
var wgRestrictionMove = [];
var wgArticleId = "55172";
var wgIsArticle = true;
var wgUserName = "Kanzure";
var wgUserGroups = ["*", "user", "autoconfirmed", "emailconfirmed"];
var wgUserLanguage = "en";
var wgContentLanguage = "en";
var wgBreakFrames = false;
var wgCurRevisionId = "186074133";
var wgVersion = "1.12alpha";
var wgEnableAPI = true;
var wgEnableWriteAPI = false;
var wgAjaxWatch = {"watchMsg": "Watch", "unwatchMsg": "Unwatch", "watchingMsg": "Watching...", "unwatchingMsg": "Unwatching..."};
/*]]>*/</script>
                
		<script type="text/javascript" src="/skins-1.5/common/wikibits.js?107"><!-- wikibits js --></script>
		<!-- Head Scripts -->
		<script type="text/javascript" src="/skins-1.5/common/ajax.js?107"></script>
		<script type="text/javascript" src="/skins-1.5/common/ajaxwatch.js?107"></script>
		<script type="text/javascript" src="/w/index.php?title=-&amp;action=raw&amp;smaxage=0&amp;gen=js&amp;useskin=monobook"><!-- site js --></script>
		<style type="text/css">/*<![CDATA[*/
@import "/w/index.php?title=MediaWiki:Common.css&usemsgcache=yes&action=raw&ctype=text/css&smaxage=2678400";
@import "/w/index.php?title=MediaWiki:Monobook.css&usemsgcache=yes&action=raw&ctype=text/css&smaxage=2678400";
@import "/w/index.php?title=-&action=raw&gen=css&maxage=2678400&smaxage=0&ts=20080125141459";
@import "/w/index.php?title=User:Kanzure/monobook.css&action=raw&ctype=text/css";
/*]]>*/</style>
		<script type="text/javascript" src="/w/index.php?title=User:Kanzure/monobook.js&amp;action=raw&amp;ctype=text/javascript&amp;dontcountme=s"></script>
	</head>
<body class="mediawiki ns-0 ltr page-Proteomics">
	<div id="globalWrapper">
		<div id="column-content">
	<div id="content">
		<a name="top" id="top"></a>
				<h1 class="firstHeading">Proteomics</h1>
		<div id="bodyContent">
			<h3 id="siteSub">From Wikipedia, the free encyclopedia</h3>
			<div id="contentSub"></div>
									<div id="jump-to-nav">Jump to: <a href="#column-one">navigation</a>, <a href="#searchInput">search</a></div>			<!-- start content -->
			<div class="dablink">For the journal Proteomics, see <a href="/wiki/Proteomics_%28journal%29" title="Proteomics (journal)">Proteomics (journal)</a>.</div>
<table style="" class="metadata plainlinks ambox ambox-content">
<tr>
<td class="ambox-image">
<div style="width:52px;"><a href="/wiki/Image:Emblem-important.svg" class="image" title="Emblem-important.svg"><img alt="" src="http://upload.wikimedia.org/wikipedia/commons/thumb/4/4c/Emblem-important.svg/40px-Emblem-important.svg.png" width="40" height="40" border="0" /></a></div>
</td>
<td class="ambox-text"><b>This article or section is in need of attention from an expert on the subject.</b><br />
<p><a href="/wiki/Wikipedia:WikiProject_Molecular_and_Cellular_Biology" title="Wikipedia:WikiProject Molecular and Cellular Biology">WikiProject Molecular and Cellular Biology</a> or the <a href="/wiki/Portal:Molecular_and_Cellular_Biology" title="Portal:Molecular and Cellular Biology">Molecular and Cellular Biology Portal</a> may be able to help recruit one.<br />
If a more appropriate <a href="/wiki/Wikipedia:WikiProject" title="Wikipedia:WikiProject">WikiProject</a> or <a href="/wiki/Portal:Portals" title="Portal:Portals">portal</a> exists, please adjust this template accordingly.<br /></p>
</td>
</tr>
</table>
<div class="thumb tright">
<div class="thumbinner" style="width:302px;"><a href="/wiki/Image:Protein_pattern_analyzer.jpg" class="image" title="Robotic preparation of MALDI mass spectrometry samples on a sample carrier."><img alt="Robotic preparation of MALDI mass spectrometry samples on a sample carrier." src="http://upload.wikimedia.org/wikipedia/en/thumb/5/5b/Protein_pattern_analyzer.jpg/300px-Protein_pattern_analyzer.jpg" width="300" height="225" border="0" class="thumbimage" /></a>
<div class="thumbcaption">
<div class="magnify" style="float:right"><a href="/wiki/Image:Protein_pattern_analyzer.jpg" class="internal" title="Enlarge"><img src="/skins-1.5/common/images/magnify-clip.png" width="15" height="11" alt="" /></a></div>
Robotic preparation of MALDI mass spectrometry samples on a sample carrier.</div>
</div>
</div>
<p><b>Proteomics</b> is the large-scale study of <a href="/wiki/Protein" title="Protein">proteins</a>, particularly their <a href="/wiki/Protein_structure" title="Protein structure">structures</a> and <a href="/wiki/Functional_genomics" title="Functional genomics">functions</a>.<sup id="_ref-pmid9740045_0" class="reference"><a href="#_note-pmid9740045" title="">[1]</a></sup><sup id="_ref-pmid10189717_0" class="reference"><a href="#_note-pmid10189717" title="">[2]</a></sup> Proteins are vital parts of living organisms, as they are the main components of the physiological <a href="/wiki/Metabolic_pathways" title="Metabolic pathways">metabolic pathways</a> of <a href="/wiki/Biological_cell" title="Biological cell">cells</a>. The term "proteomics" was coined to make an analogy with <a href="/wiki/Genomics" title="Genomics">genomics</a>, the study of the <a href="/wiki/Gene" title="Gene">genes</a>. The word "proteome" is a <a href="/wiki/Portmanteau" title="Portmanteau">portmanteau</a> of "<i>prote</i>in" and "gen<i>ome</i>". The <a href="/wiki/Proteome" title="Proteome">proteome</a> of an organism is the set of proteins produced by it during its life, and its genome is its set of genes.</p>
<p>Proteomics is often considered the next step in the study of biological systems, after genomics. It is much more complicated than genomics, mostly because while an organism's <a href="/wiki/Genome" title="Genome">genome</a> is rather constant, a <a href="/wiki/Proteome" title="Proteome">proteome</a> differs from cell to cell and constantly changes through its <a href="/wiki/Biochemistry" title="Biochemistry">biochemical</a> interactions with the genome and the environment. One organism has radically different <a href="/wiki/Protein_expression" title="Protein expression">protein expression</a> in different parts of its body, different stages of its life cycle and different environmental conditions. Another major difficulty is the complexity of proteins relative to <a href="/wiki/Nucleic_acid" title="Nucleic acid">nucleic acids</a>. E.g., in human there are about 25 000 identified genes but an estimated &gt;500 000 proteins that are derived from these genes. This increased complexity derives from mechanisms such as alternative splicing, protein modification (<a href="/wiki/Glycosylation" title="Glycosylation">glycosylation</a>,<a href="/wiki/Phosphorylation" title="Phosphorylation">phosphorylation</a>) and protein degradation.</p>
<p>Scientists are very interested in proteomics because it gives a much better understanding of an organism than genomics. First, the level of transcription of a gene gives only a rough estimate of its level of expression into a protein. An <a href="/wiki/MRNA" title="MRNA">mRNA</a> produced in abundance may be degraded rapidly or translated inefficiently, resulting in a small amount of protein. Second, many proteins experience <a href="/wiki/Post-translational_modification" title="Post-translational modification">post-translational modifications</a> that profoundly affect their activities; for example some proteins are not active until they become phosphorylated. Methods such as <a href="/wiki/Phosphoproteomics" title="Phosphoproteomics">phosphoproteomics</a> and <a href="/wiki/Glycoproteomics" title="Glycoproteomics">glycoproteomics</a> are used to study post-translational modifications. Third, many transcripts give rise to more than one protein, through <a href="/wiki/Alternative_splicing" title="Alternative splicing">alternative splicing</a> or alternative post-translational modifications. Finally, many proteins form complexes with other proteins or RNA molecules, and only function in the presence of these other molecules.</p>
<p>Since proteins play a central role in the life of an organism, proteomics is instrumental in discovery of <a href="/wiki/Biomarker" title="Biomarker">biomarkers</a>, such as markers that indicate a particular disease.</p>
<p>With the completion of a rough draft of the <a href="/wiki/Human" title="Human">human</a> genome, many researchers are looking at how genes and proteins interact to form other proteins. A surprising finding of the <a href="/wiki/Human_Genome_Project" title="Human Genome Project">Human Genome Project</a> is that there are far fewer protein-coding genes in the human genome than proteins in the human proteome (20,000 to 25,000 genes vs. &gt; 500,000 proteins). The human body may even contain more than 2 million proteins, each having different functions. The protein diversity is thought to be due to alternative splicing and post-translational modification of proteins. The discrepancy implies that protein diversity cannot be fully characterized by gene expression analysis, thus proteomics is useful for characterizing cells and tissues.</p>
<p>To catalog all human proteins, their functions and interactions is a great challenge for scientists. An international collaboration with these goals is co-ordinated by the <a href="/wiki/Human_Proteome_Organization" title="Human Proteome Organization">Human Proteome Organization</a> (<a href="http://www.hupo.org" class="external text" title="http://www.hupo.org" rel="nofollow">HUPO</a>).</p>
<table id="toc" class="toc" summary="Contents">
<tr>
<td>
<div id="toctitle">
<h2>Contents</h2>
</div>
<ul>
<li class="toclevel-1"><a href="#Studying_proteomics"><span class="tocnumber">1</span> <span class="toctext">Studying proteomics</span></a></li>
<li class="toclevel-1"><a href="#Biomarkers"><span class="tocnumber">2</span> <span class="toctext">Biomarkers</span></a></li>
<li class="toclevel-1"><a href="#Branches"><span class="tocnumber">3</span> <span class="toctext">Branches</span></a></li>
<li class="toclevel-1"><a href="#Technologies"><span class="tocnumber">4</span> <span class="toctext">Technologies</span></a></li>
<li class="toclevel-1"><a href="#See_also"><span class="tocnumber">5</span> <span class="toctext">See also</span></a>
<ul>
<li class="toclevel-2"><a href="#Protein_databases"><span class="tocnumber">5.1</span> <span class="toctext">Protein databases</span></a></li>
</ul>
</li>
<li class="toclevel-1"><a href="#References"><span class="tocnumber">6</span> <span class="toctext">References</span></a></li>
<li class="toclevel-1"><a href="#Bibliography"><span class="tocnumber">7</span> <span class="toctext">Bibliography</span></a></li>
<li class="toclevel-1"><a href="#External_links"><span class="tocnumber">8</span> <span class="toctext">External links</span></a></li>
</ul>
</td>
</tr>
</table>
<script type="text/javascript">
//<![CDATA[
 if (window.showTocToggle) { var tocShowText = "show"; var tocHideText = "hide"; showTocToggle(); } 
//]]>
</script>
<p><a name="Studying_proteomics" id="Studying_proteomics"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=1" title="Edit section: Studying proteomics">edit</a>]</span> <span class="mw-headline">Studying proteomics</span></h2>
<p>Most proteins function in collaboration with other proteins, and one goal of proteomics is to identify which proteins interact. This often gives important clues about the functions of newly discovered proteins. Several methods are available to probe protein-protein interactions. The traditional method is yeast <a href="/wiki/Two-hybrid_screening" title="Two-hybrid screening">two-hybrid analysis</a>. New methods include <a href="/wiki/Protein_microarray" title="Protein microarray">protein microarrays</a>, <a href="/wiki/Immunoaffinity_chromatography" title="Immunoaffinity chromatography">immunoaffinity chromatography</a> followed by <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a>, and combinations of experimental methods such as phage display and computational methods.</p>
<p>Current research in proteomics requires first that proteins be resolved, sometimes on a massive scale. Protein separation can be performed using <a href="/wiki/Two-dimensional_gel_electrophoresis" title="Two-dimensional gel electrophoresis">two-dimensional gel electrophoresis</a>, which usually separates proteins first by <a href="/wiki/Isoelectric_point" title="Isoelectric point">isoelectric point</a> and then by molecular weight. Protein spots in a gel can be visualized using a variety of chemical stains or fluorescent markers. Proteins can often be quantified by the intensity of their stain. Once proteins are separated and quantified, they are identified. Individual spots are cut out of the gel and cleaved into peptides with proteolytic <a href="/wiki/Enzyme" title="Enzyme">enzymes</a>. These peptides can then be identified by mass spectrometry, specifically matrix-assisted laser desorption-ionization time-of-flight (<a href="/wiki/MALDI-TOF" title="MALDI-TOF">MALDI-TOF</a>) mass spectrometry. In this procedure, a peptide is placed on a matrix, which causes the peptide to form crystals<sup class="noprint Template-Fact"><span title="This claim needs references to reliable sources&#160;since August 2007" style="white-space: nowrap;">[<i><a href="/wiki/Wikipedia:Citation_needed" title="Wikipedia:Citation needed">citation needed</a></i>]</span></sup>. Then the peptide on the matrix is ionized with a <a href="/wiki/Laser" title="Laser">laser</a> beam and an increase in voltage at the matrix is used to shoot the ions toward a detector in which the time it takes an ion to reach the detector depends on its mass. The higher the mass, the longer the time of flight of the ion. In a MALDI-TOF mass spectrometer, the ions can also be deflected with an electrostatic reflector that also focuses the ion beam. Thus, the masses of the ions reaching the second detector can be determined with high precision and these masses can reveal the exact chemical compositions of the peptides, and therefore their identities.</p>
<p>Protein mixtures can also be analyzed without prior separation. These procedures begin with proteolytic digestion of the proteins in a complex mixture. The resulting peptides are often injected onto a <a href="/wiki/High_pressure_liquid_chromatography" title="High pressure liquid chromatography">high pressure liquid chromatography</a> column (HPLC) that separates peptides based on hydrophobicity. HPLC can be coupled directly to a time-of-flight mass spectrometer using <a href="/wiki/Electrospray_ionization" title="Electrospray ionization">electrospray ionization</a>. Peptides eluting from the column can be identified by tandem mass spectrometry (MS/MS). The first stage of tandem MS/MS isolates individual peptide ions, and the second breaks the peptides into fragments and uses the fragmentation pattern to determine their amino acid sequences. Labeling with isotope tags can be used to quantitatively compare proteins concentration among two or more protein samples.</p>
<p>One of the most promising developments to come from the study of human genes and proteins has been the identification of potential new drugs for the treatment of disease. This relies on genome and proteome information to identify proteins associated with a disease, which computer software can then use as targets for new drugs. For example, if a certain protein is implicated in a disease, its 3D structure provides the information to design drugs to interfere with the action of the protein. A molecule that fits the active site of an enzyme, but cannot be released by the enzyme, will inactivate the enzyme. This is the basis of new drug-discovery tools, which aim to find new drugs to inactivate proteins involved in disease. As genetic differences among individuals are found, researchers expect to use these techniques to develop personalized drugs that are more effective for the individual.</p>
<p>A computer technique which attempts to fit millions of small molecules to the three-dimensional structure of a protein is called "virtual ligand screening". The computer rates the quality of the fit to various sites in the protein, with the goal of either enhancing or disabling the function of the protein, depending on its function in the cell. A good example of this is the identification of new drugs to target and inactivate the HIV-1 protease. The HIV-1 protease is an enzyme that cleaves a very large HIV protein into smaller, functional proteins. The virus cannot survive without this enzyme; therefore, it is one of the most effective protein targets for killing HIV.</p>
<p>There are many distributed computing programs, such as the <a href="/wiki/World_community_grid" title="World community grid">world community grid</a>, which allows people around the world to help scientists by computing calculations. The software adds to the use of super computers by using the unused processing power of millions of home computers. The world community grid works on HIV, cancer, and protein folding. All three projects centre around protein modelling and protein modification models. Using the data gained from distributed computing models of proteins, scientists can develop more specific and effective therapies. In addition, most enzymes act as part of complexes and networks, which also affect the way an enzyme acts in a cell. Understanding these complex networks will assist in developing drugs that affect the function of these complexes.</p>
<p><a name="Biomarkers" id="Biomarkers"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=2" title="Edit section: Biomarkers">edit</a>]</span> <span class="mw-headline">Biomarkers</span></h2>
<p>Understanding the proteome, the structure and function of each protein and the complexities of protein-protein interactions will be critical for developing the most effective diagnostic techniques and disease treatments in the future.</p>
<p>An interesting use of proteomics is using specific protein biomarkers to diagnose disease. A number of techniques allow to test for proteins produced during a particular disease, which helps to diagnose the disease quickly. Techniques include western blot, <a href="/wiki/Immunohistochemical_staining" title="Immunohistochemical staining">immunohistochemical staining</a>, <a href="/wiki/Enzyme_linked_immunosorbent_assay" title="Enzyme linked immunosorbent assay">enzyme linked immunosorbent assay</a> (ELISA) or <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a>. The following are some of the diseases that have characteristic biomarkers that physicians can use for diagnosis:</p>
<ul>
<li>In <a href="/wiki/Alzheimer%E2%80%99s_disease" title="Alzheimer’s disease">Alzheimer’s disease</a>, elevations in beta secretase creates amyloid/beta-protein, which causes plaque to build up in the patient's brain, which causes dementia. Targeting this enzyme decreases the amyloid/beta-protein and so slows the progression of the disease. A procedure to test for the increase in amyloid/beta-protein is immunohistochemical staining, in which antibodies bind to specific antigens or biological tissue of amyloid/beta-protein.</li>
<li>Heart disease is commonly assessed using several key protein based biomarkers. Standard protein biomarkers for CVD include interleukin-6, interleukin-8, serum amyloid A protein, fibrinogen, and troponins. cTnI cardiac troponin I increases in concentration within 3 to 12 hours of initial cardiac injury and can be found elevated days after an <a href="/wiki/Acute_myocardial_infarction" title="Acute myocardial infarction">acute myocardial infarction</a>. A number of commercial antibody based assays as well as other methods are used in hospitals as primary tests for acute MI.</li>
<li>Proteomic analysis of kidney cells and cancerous kidney cells is producing promising leads for biomarkers for <a href="/wiki/Renal_cell_carcinoma" title="Renal cell carcinoma">renal cell carcinoma</a> and developing assays to test for this disease. In kidney-related diseases, urine is a potential source for such biomarkers. Recently, it has been shown that the identification of urinary polypeptides as biomarkers of kidney-related diseases allows to diagnose the severity of the disease several months before the appearance of the pathology.<a href="http://www.nature.com/nm/journal/v12/n4/abs/nm1384.html;jsessionid=3F9707D2B671CA69E12EC68E65919D60" class="external text" title="http://www.nature.com/nm/journal/v12/n4/abs/nm1384.html;jsessionid=3F9707D2B671CA69E12EC68E65919D60" rel="nofollow">Article</a></li>
</ul>
<p><a name="Branches" id="Branches"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=3" title="Edit section: Branches">edit</a>]</span> <span class="mw-headline">Branches</span></h2>
<ol>
<li><i>Protein separation</i>. Proteomic technologies rely on the ability to separate a complex mixture so that individual proteins are more easily processed with other techniques.</li>
<li><i>Protein identification</i>. Well-known methods include low-throughput sequencing through <a href="/wiki/Edman_degradation" title="Edman degradation">Edman degradation</a>. Higher-throughput proteomic techniques are based on <a href="/wiki/Mass_spectrometry" title="Mass spectrometry">mass spectrometry</a>, commonly <a href="/wiki/Peptide_mass_fingerprinting" title="Peptide mass fingerprinting">peptide mass fingerprinting</a> on MALDI-TOF instruments, or <a href="/wiki/De_novo_repeat_detection" title="De novo repeat detection">De novo repeat detection</a> MS/MS on instruments capable of more than one round of mass spectrometry. MS/MS data can be analyzed by simple database searches as is the case for PMFs and additionally they can be analyzed by de novo sequencing and homology searching. This particular approach allows to even identify similar (homolog) proteins, e.g. across species in case a protein was derived from an organism with unsequenced genome. Antibody-based assays can also be used, but are unique to one sequence motif.</li>
<li>In <a href="/wiki/Quantitative_proteomics" title="Quantitative proteomics">quantitative proteomics</a> different methods are used to obtain quantitative information on a proteome-wide scale. Rather than just lists of proteins, quantitative proteomics provides functional information and reveals temporal changes in the proteome.</li>
<li><i>Protein sequence analysis</i> is a branch of bioinformatics that deals with searching databases for possible protein or peptide matches by algorithms such as <a href="/wiki/Mascot_%28software%29" title="Mascot (software)">Mascot</a>, <a href="/wiki/PEAKS%28software%29" title="PEAKS(software)">PEAKS(software)</a>, OMSSA, <a href="/wiki/SEQUEST" title="SEQUEST">SEQUEST</a> and X!Tandem, PWB Protein Identification Cluster Software Solution, functional assignment of domains, prediction of function from sequence, and evolutionary relationships of proteins.</li>
<li><i>Structural proteomics</i> concerns the high-throughput determination of protein structures in three-dimensional space. Common methods are <a href="/wiki/X-ray_crystallography" title="X-ray crystallography">x-ray crystallography</a> and <a href="/wiki/Protein_NMR" title="Protein NMR">NMR spectroscopy</a>.</li>
<li><i>Interaction proteomics</i> concerns the investigation of protein interactions on the atomic, molecular and cellular levels. see related article on <a href="/wiki/Protein-protein_interaction_prediction" title="Protein-protein interaction prediction">Protein-protein interaction prediction</a>.</li>
<li><i>Protein modification</i> studies the modified forms of proteins. Almost all proteins are modified from their pure translated amino-acid sequence, by so-called post-translational modification. Specialized methods have been developed to study phosporylation (<a href="/wiki/Phosphoproteomics" title="Phosphoproteomics">phosphoproteomics</a>) and glycosylation (<a href="/wiki/Glycoproteomics" title="Glycoproteomics">glycoproteomics</a>).</li>
<li><i>Cellular proteomics</i> is a new branch of proteomics aiming to map the location of proteins and protein-protein interactions in whole cells during key cell events. Centers around the use of techniques such as X-ray Tomography and optical fluorescence microscopy.</li>
<li><i>Experimental bioinformatics</i> is a branch of bioinformatics, as it is applied in proteomics, coined by Mathias Mann. It involves the mutual design of experimental and bioinformatics methods to create (extract) new types of information from proteomics experiments.</li>
</ol>
<p><a name="Technologies" id="Technologies"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=4" title="Edit section: Technologies">edit</a>]</span> <span class="mw-headline">Technologies</span></h2>
<p>Proteomics uses various technologies:</p>
<ul>
<li>One- and two-dimensional <a href="/wiki/Gel_electrophoresis" title="Gel electrophoresis">gel electrophoresis</a> is used to identify the relative mass of a protein and its <a href="/wiki/Isoelectric_point" title="Isoelectric point">isoelectric point</a>.</li>
<li><a href="/wiki/X-ray_crystallography" title="X-ray crystallography">X-ray crystallography</a> and <a href="/wiki/Nuclear_magnetic_resonance" title="Nuclear magnetic resonance">nuclear magnetic resonance</a> are used to characterize the three-dimensional structure of <a href="/wiki/Peptides" title="Peptides">peptides</a> and <a href="/wiki/Proteins" title="Proteins">proteins</a>. However, low-resolution techniques such as <a href="/wiki/Circular_dichroism" title="Circular dichroism">circular dichroism</a>, <a href="/wiki/Fourier_transform_spectroscopy" title="Fourier transform spectroscopy">Fourier transform infrared spectroscopy</a> and <a href="/wiki/Small_angle_X-ray_scattering_%28SAXS%29" title="Small angle X-ray scattering (SAXS)">Small angle X-ray scattering (SAXS)</a> can be used to study the secondary structure of proteins.</li>
<li><a href="/wiki/Tandem_mass_spectrometry" title="Tandem mass spectrometry">Tandem mass spectrometry</a> combined with <a href="/wiki/Reverse_phase_chromatography" title="Reverse phase chromatography">reverse phase chromatography</a> or <a href="/wiki/2-D_electrophoresis" title="2-D electrophoresis">2-D electrophoresis</a> is used to identify proteins using database search tools such as <a href="/wiki/Mascot_%28software%29" title="Mascot (software)">Mascot</a>, Phenyx, <a href="/wiki/PEAKS%28software%29" title="PEAKS(software)">PEAKS(software)</a>, OMSSA, X!Tandem and <a href="/wiki/SEQUEST" title="SEQUEST">SEQUEST</a> or de novo algorithms and quantify all the levels of proteins found in cells.</li>
<li>Scaffold, a software tool useful in visualization of tandem mass spectrometry results.</li>
<li><a href="/wiki/Tandem_mass_spectrometry" title="Tandem mass spectrometry">Tandem mass spectrometry</a> combined with tagging technologies such as <a href="/wiki/Tandem_Mass_Tags" title="Tandem Mass Tags">TMT</a>, ICPL or <a href="/w/index.php?title=ITRAQ&amp;action=edit" class="new" title="ITRAQ">iTRAQ</a> is used for quantification of proteins and peptides.</li>
<li><a href="/wiki/Mass_spectrometry" title="Mass spectrometry">Mass spectrometry</a> (no-tandem), often <a href="/wiki/MALDI-TOF" title="MALDI-TOF">MALDI-TOF</a>, is used to identify proteins by <a href="/wiki/Peptide_mass_fingerprinting" title="Peptide mass fingerprinting">peptide mass fingerprinting</a>. This technology is also used in so-called "MALDI-TOF MS protein profiling" where samples (i.e. serum) are prepared by either protein chips (<a href="/wiki/SELDI-TOF" title="SELDI-TOF">SELDI-TOF</a> MS), magnetic beads (The Bruker Daltonics protein profiling platform) or with other methods of sample treatment, such as liquid chromatography, size-exclusion and immunoaffinity. Protein peaks of interest must be identified by tandem mass spectrometry. Protein profiling with MALDI-TOF MS could be of high use in clinical diagnostics, but so far there has been little success with advancing MALDI-TOF MS protein profiling into clinical validation due to high analytical variation.</li>
<li><a href="/wiki/ICP-MS" title="ICP-MS">ICP-MS</a> combined with MeCAT - Metal Coded Tagging - technology is used for ultrasensitive quantification of proteins and peptides down to low attomol range</li>
<li><a href="/wiki/Affinity_chromatography" title="Affinity chromatography">Affinity chromatography</a>, <a href="/wiki/Yeast_two_hybrid" title="Yeast two hybrid">yeast two hybrid</a> techniques, <a href="/wiki/Fluorescence_resonance_energy_transfer" title="Fluorescence resonance energy transfer">fluorescence resonance energy transfer</a> (FRET), and Surface Plasmon Resonance (SPR) are used to identify protein-protein and protein-DNA binding reactions.</li>
<li><a href="/wiki/X-ray_Tomography" title="X-ray Tomography">X-ray Tomography</a> used to determine the location of labeled proteins or protein complexes in an intact cell. Frequently correlated with images of cells from light based microscopes.</li>
<li><a href="/wiki/Image_analysis" title="Image analysis">Software based image analysis</a> is utilized to automate the quantification and detection of spots within and among gels samples. While this technology is widely utilized, the intelligence has not been perfected yet. For example, the leading software tools in this area tend to agree on the analysis of well-defined, well-separated protein spots, but they deliver different results and tendencies with less-defined less-separated spots - thus necessitating manual verification of results.</li>
</ul>
<p><a name="See_also" id="See_also"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=5" title="Edit section: See also">edit</a>]</span> <span class="mw-headline">See also</span></h2>
<ul>
<li><a href="/wiki/Proteomic_chemistry" title="Proteomic chemistry">proteomic chemistry</a></li>
<li><a href="/wiki/Bioinformatics" title="Bioinformatics">bioinformatics</a></li>
<li><a href="/wiki/Cytomics" title="Cytomics">cytomics</a></li>
<li><a href="/wiki/Genomics" title="Genomics">genomics</a></li>
<li><a href="/wiki/List_of_omics_topics_in_biology" title="List of omics topics in biology">List of omics topics in biology</a></li>
<li><a href="/wiki/Metabolomics" title="Metabolomics">metabolomics</a></li>
<li><a href="/wiki/Lipidomics" title="Lipidomics">lipidomics</a></li>
<li><a href="/wiki/Shotgun_proteomics" title="Shotgun proteomics">Shotgun proteomics</a></li>
<li><a href="/wiki/Top-down_proteomics" title="Top-down proteomics">Top-down proteomics</a></li>
<li><a href="/wiki/Bottom-up_proteomics" title="Bottom-up proteomics">Bottom-up proteomics</a></li>
<li><a href="/wiki/Systems_biology" title="Systems biology">systems biology</a></li>
<li><a href="/wiki/Transcriptomics" title="Transcriptomics">transcriptomics</a></li>
<li><a href="/wiki/Phosphoproteomics" title="Phosphoproteomics">phosphoproteomics</a></li>
<li><a href="/wiki/PEGylation" title="PEGylation">PEGylation</a></li>
</ul>
<p><a name="Protein_databases" id="Protein_databases"></a></p>
<h3><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=6" title="Edit section: Protein databases">edit</a>]</span> <span class="mw-headline">Protein databases</span></h3>
<ul>
<li><a href="/wiki/UniProt" title="UniProt">UniProt</a></li>
<li><a href="/wiki/Protein_Information_Resource" title="Protein Information Resource">PIR</a></li>
<li><a href="/wiki/Swiss-Prot" title="Swiss-Prot">Swiss-Prot</a></li>
<li><a href="/wiki/Protein_Data_Bank" title="Protein Data Bank">PDB</a></li>
<li><a href="/wiki/NCBI" title="NCBI">NCBI</a></li>
<li><a href="/wiki/Human_Protein_Reference_Database" title="Human Protein Reference Database">Human Protein Reference Database</a></li>
</ul>
<p><a name="References" id="References"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=7" title="Edit section: References">edit</a>]</span> <span class="mw-headline">References</span></h2>
<div class="references-small">
<ol class="references">
<li id="_note-pmid9740045"><b><a href="#_ref-pmid9740045_0" title="">^</a></b> <cite style="font-style:normal">Anderson NL, Anderson NG (1998). "Proteome and proteomics: new technologies, new concepts, and new words". <i>Electrophoresis</i> <b>19</b> (11): 1853–61. <a href="/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a href="http://dx.doi.org/10.1002/elps.1150191103" class="external text" title="http://dx.doi.org/10.1002/elps.1150191103" rel="nofollow">10.1002/elps.1150191103</a>. <a href="http://www.ncbi.nlm.nih.gov/pubmed/9740045" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/9740045">PMID 9740045</a>.</cite></li>
<li id="_note-pmid10189717"><b><a href="#_ref-pmid10189717_0" title="">^</a></b> <cite style="font-style:normal">Blackstock WP, Weir MP (1999). "Proteomics: quantitative and physical mapping of cellular proteins". <i>Trends Biotechnol.</i> <b>17</b> (3): 121–7. <a href="http://www.ncbi.nlm.nih.gov/pubmed/10189717" class="external" title="http://www.ncbi.nlm.nih.gov/pubmed/10189717">PMID 10189717</a>.</cite></li>
</ol>
</div>
<p><a name="Bibliography" id="Bibliography"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=8" title="Edit section: Bibliography">edit</a>]</span> <span class="mw-headline">Bibliography</span></h2>
<div class="references-small" style="margin-left:1.5em;">
<ul>
<li>Belhajjame, K. et al. <a href="http://www.allhands.org.uk/2005/proceedings/papers/525.pdf" class="external text" title="http://www.allhands.org.uk/2005/proceedings/papers/525.pdf" rel="nofollow">Proteome Data Integration: Characteristics and Challenges</a>. Proceedings of the UK e-Science All Hands Meeting, <a href="/w/index.php?title=Special:Booksources&amp;isbn=1904425534" class="internal">ISBN 1-904425-53-4</a>, September 2005, Nottingham, UK.</li>
<li>Twyman, R. M. 2004. Principles of proteomics. BIOS Scientific Publishers, New York. <a href="/w/index.php?title=Special:Booksources&amp;isbn=1859962734" class="internal">ISBN 1-85996-273-4</a>.(covers almost all branches of proteomics)</li>
<li>Westermeier, R. and T. Naven. 2002. Proteomics in practice: a laboratory manual of proteome analysis. Wiley-VCH, Weinheim. <a href="/w/index.php?title=Special:Booksources&amp;isbn=3527303545" class="internal">ISBN 3-527-30354-5</a>.(focused on 2D-gels, good on detail)</li>
<li>Liebler, D. C. 2002. Introduction to proteomics: tools for the new biology. Humana Press, Totowa, NJ. <a href="/w/index.php?title=Special:Booksources&amp;isbn=0585418799" class="internal">ISBN 0-585-41879-9</a> (electronic, on Netlibrary?), <a href="/w/index.php?title=Special:Booksources&amp;isbn=0896039919" class="internal">ISBN 0-89603-991-9</a> hardback, <a href="/w/index.php?title=Special:Booksources&amp;isbn=0896039927" class="internal">ISBN 0-89603-992-7</a> paperback.</li>
<li><a href="/w/index.php?title=Wilkins_MR&amp;action=edit" class="new" title="Wilkins MR">Wilkins MR</a>, <a href="/w/index.php?title=Williams_KL&amp;action=edit" class="new" title="Williams KL">Williams KL</a>, <a href="/w/index.php?title=Appel_RD&amp;action=edit" class="new" title="Appel RD">Appel RD</a>, <a href="/w/index.php?title=Hochstrasser_DF&amp;action=edit" class="new" title="Hochstrasser DF">Hochstrasser DF</a>. <i>Proteome research: new frontiers in functional genomics</i>. Berlin Heidelberg, Springer Verlag; 1997, <a href="/w/index.php?title=Special:Booksources&amp;isbn=3540627537" class="internal">ISBN 3-540-62753-7</a>.</li>
<li><cite style="font-style:normal">Arora, Pankaj S., et al. (2005). "Comparative evaluation of two two-dimensional gel electrophoresis image analysis software applications using synovial fluids from patients with joint disease". <i>Journal of Orthopaedic Science</i> <b>10</b> (2): 160-166.</cite> <a href="http://www.springerlink.com/openurl.asp?genre=article&amp;id=doi:10.1007/s00776-004-0878-0" class="external autonumber" title="http://www.springerlink.com/openurl.asp?genre=article&amp;id=doi:10.1007/s00776-004-0878-0" rel="nofollow">[1]</a></li>
<li>Rediscovering Biology Online Textbook. Unit 2 Proteins and Proteomics. 1997-2006.</li>
<li>Weaver. R.F. Molecular Biology. Third Edition. The McGraw-Hill Companies Inc. 2005. pgs 840-849.</li>
<li>Campbell and Reece. Biology. Sixth Edition. Pearson Education Inc. 2002. pg 392-393.</li>
<li>Hye A, Lynham S, Thambisetty M, et al. " Proteome-based plasma biomarkers for Alzheimer's disease." Brain 129: 3042-3050, (2006).</li>
<li>Perroud B, Lee J, Valkova N, et al. "Pathway Analysis of Kidney Cancer Using Proteomics and *Metabolic Profiling." Biomed Central: 65-82, (24 November 2006).</li>
<li>Macaulay IC, Carr P, Gusnanto A, et al. "Platelet Genomics and Proteomics in Human Health and Disease." The Journal of Clinical Investigation 115: 3370-3377, (December 2005).</li>
<li>Rogers MA, Clarke P, Noble J, et al. "Proteomic Profiling of Urinary Proteins in Renal Cancer by Surface Enhanced Laser Desorption Ionization, and Neural-Network Analysis: Identification of Key Issues Affecting Clinical Potential Utility." Cancer Research 63: 6971-6983, (15 October 2003).</li>
<li>Vasan RS. “Biomarkers of cardiovascular disease: molecular basis and practical considerations” Circulation. 2006;113:2335-2362.</li>
<li>“Myocardial Infaction”. <a href="http://medlib.med.utah.edu/WebPath/TUTORIAL/MYOCARD/MYOCARD.html" class="external free" title="http://medlib.med.utah.edu/WebPath/TUTORIAL/MYOCARD/MYOCARD.html" rel="nofollow">http://medlib.med.utah.edu/WebPath/TUTORIAL/MYOCARD/MYOCARD.html</a> (Retrieved 29 Nov 2006)</li>
<li>World Community Grid. <a href="http://www.worldcommunitygrid.org" class="external free" title="http://www.worldcommunitygrid.org" rel="nofollow">http://www.worldcommunitygrid.org</a> (Retrieved 29 Nov 2006)</li>
<li>Introduction to Antibodies - Enzyme-Linked Immunosorbent Assay (ELISA). <a href="http://www.chemicon.com/resource/ANT101/a2C.asp" class="external free" title="http://www.chemicon.com/resource/ANT101/a2C.asp" rel="nofollow">http://www.chemicon.com/resource/ANT101/a2C.asp</a>. (Retrieved 29 Nov 2006)</li>
<li>Decramer S et al "Predicting the clinical outcome of congenital unilateral ureteropelvic junction obstruction in newborn by urinary proteome analysis" Nature Medicine 2006; 12:398-400 <a href="http://www.nature.com/nm/journal/v12/n4/abs/nm1384.html;jsessionid=3F9707D2B671CA69E12EC68E65919D60" class="external text" title="http://www.nature.com/nm/journal/v12/n4/abs/nm1384.html;jsessionid=3F9707D2B671CA69E12EC68E65919D60" rel="nofollow">Article</a></li>
</ul>
</div>
<p><a name="External_links" id="External_links"></a></p>
<h2><span class="editsection">[<a href="/w/index.php?title=Proteomics&amp;action=edit&amp;section=9" title="Edit section: External links">edit</a>]</span> <span class="mw-headline">External links</span></h2>
<div class="infobox sisterproject">
<div style="float: left;">
<div class="floatnone"><span><a href="/wiki/Image:Wikibooks-logo-en.svg" class="image" title="Wikibooks"><img alt="Wikibooks" src="http://upload.wikimedia.org/wikipedia/commons/thumb/7/7c/Wikibooks-logo-en.svg/50px-Wikibooks-logo-en.svg.png" width="50" height="57" border="0" /></a></span></div>
</div>
<div style="margin-left: 60px;"><a href="/wiki/Wikibooks" title="Wikibooks">Wikibooks</a> has more on the topic of
<div style="margin-left:10px;"><i><b><a href="http://en.wikibooks.org/wiki/Special:Search/Proteomics" class="extiw" title="wikibooks:Special:Search/Proteomics">Proteomics</a></b></i></div>
</div>
</div>
<div class="infobox sisterproject">
<div style="float: left;">
<div class="floatnone"><span><a href="/wiki/Image:Wikiversity-logo-Snorky.svg" class="image" title="Wikiversity"><img alt="Wikiversity" src="http://upload.wikimedia.org/wikipedia/commons/thumb/3/34/Wikiversity-logo-Snorky.svg/50px-Wikiversity-logo-Snorky.svg.png" width="50" height="45" border="0" /></a></span></div>
</div>
<div style="margin-left: 60px;">At <a href="/wiki/Wikiversity" title="Wikiversity">Wikiversity</a> you can learn more and teach others about <b>Proteomics</b> at:
<div style="margin-left: 10px;"><a href="http://en.wikiversity.org/wiki/Topic:Proteomics" class="extiw" title="v:Topic:Proteomics">The Department of Proteomics</a></div>
</div>
</div>
<table style="margin:0 auto;" align="center" class="toccolours">
<tr>
<td align="center" style="background:#ccccff"><b><a href="/wiki/Genomics" title="Genomics">Genomics</a> topics</b></td>
</tr>
<tr>
<td align="center" style="font-size:90%;"><a href="/wiki/Genome_project" title="Genome project">Genome project</a> | <a href="/wiki/Paleopolyploidy" title="Paleopolyploidy">Paleopolyploidy</a> | <a href="/wiki/Glycomics" title="Glycomics">Glycomics</a> | <a href="/wiki/Human_Genome_Project" title="Human Genome Project">Human Genome Project</a> | <strong class="selflink">Proteomics</strong></td>
</tr>
<tr>
<td align="center" style="font-size:90%;"><a href="/wiki/Chemogenomics" title="Chemogenomics">Chemogenomics</a> | <a href="/wiki/Structural_genomics" title="Structural genomics">Structural genomics</a> | <a href="/wiki/Pharmacogenetics" title="Pharmacogenetics">Pharmacogenetics</a> | <a href="/wiki/Pharmacogenomics" title="Pharmacogenomics">Pharmacogenomics</a> | <a href="/wiki/Toxicogenomics" title="Toxicogenomics">Toxicogenomics</a> | <a href="/wiki/Computational_genomics" title="Computational genomics">Computational genomics</a></td>
</tr>
<tr>
<td align="center" style="font-size:90%;"><a href="/wiki/Bioinformatics" title="Bioinformatics">Bioinformatics</a> | <a href="/wiki/Cheminformatics" title="Cheminformatics">Cheminformatics</a> | <a href="/wiki/Systems_biology" title="Systems biology">Systems biology</a></td>
</tr>
</table>

<!-- 
Pre-expand include size: 26448/2048000 bytes
Post-expand include size: 10261/2048000 bytes
Template argument size: 5106/2048000 bytes
#ifexist count: 2/500
-->

<!-- Saved in parser cache with key enwiki:pcache:idhash:55172-0!1!0!default!!en!2 and timestamp 20080122083519 -->
<div class="printfooter">
Retrieved from "<a href="http://en.wikipedia.org/wiki/Proteomics">http://en.wikipedia.org/wiki/Proteomics</a>"</div>
			<div id="catlinks"><p class='catlinks'><a href="/wiki/Special:Categories" title="Special:Categories">Categories</a>: <span dir='ltr'><a href="/wiki/Category:Molecular_and_Cellular_Biology_articles_needing_expert_attention" title="Category:Molecular and Cellular Biology articles needing expert attention">Molecular and Cellular Biology articles needing expert attention</a></span> | <span dir='ltr'><a href="/wiki/Category:Articles_needing_expert_attention" title="Category:Articles needing expert attention">Articles needing expert attention</a></span> | <span dir='ltr'><a href="/wiki/Category:Pages_needing_expert_attention" title="Category:Pages needing expert attention">Pages needing expert attention</a></span> | <span dir='ltr'><a href="/wiki/Category:All_articles_with_unsourced_statements" title="Category:All articles with unsourced statements">All articles with unsourced statements</a></span> | <span dir='ltr'><a href="/wiki/Category:Articles_with_unsourced_statements_since_August_2007" title="Category:Articles with unsourced statements since August 2007">Articles with unsourced statements since August 2007</a></span> | <span dir='ltr'><a href="/wiki/Category:Proteomics" title="Category:Proteomics">Proteomics</a></span> | <span dir='ltr'><a href="/wiki/Category:Genomics" title="Category:Genomics">Genomics</a></span> | <span dir='ltr'><a href="/wiki/Category:Bioinformatics" title="Category:Bioinformatics">Bioinformatics</a></span> | <span dir='ltr'><a href="/wiki/Category:Proteins" title="Category:Proteins">Proteins</a></span> | <span dir='ltr'><a href="/wiki/Category:Mass_spectrometry" title="Category:Mass spectrometry">Mass spectrometry</a></span></p></div>			<!-- end content -->
			<div class="visualClear"></div>
		</div>
	</div>
		</div>
		<div id="column-one">
	<div id="p-cactions" class="portlet">
		<h5>Views</h5>
		<div class="pBody">
			<ul>
					 <li id="ca-nstab-main" class="selected"><a href="/wiki/Proteomics" title="View the content page [c]" accesskey="c">Article</a></li>
					 <li id="ca-talk"><a href="/wiki/Talk:Proteomics" title="Discussion about the content page [t]" accesskey="t">Discussion</a></li>
					 <li id="ca-edit"><a href="/w/index.php?title=Proteomics&amp;action=edit" title="You can edit this page. Please use the preview button before saving. [e]" accesskey="e">Edit this page</a></li>
					 <li id="ca-history"><a href="/w/index.php?title=Proteomics&amp;action=history" title="Past versions of this page [h]" accesskey="h">History</a></li>
					 <li id="ca-move"><a href="/wiki/Special:Movepage/Proteomics" title="Rename this page [m]" accesskey="m">Move</a></li>
					 <li id="ca-watch"><a href="/w/index.php?title=Proteomics&amp;action=watch" title="Add this page to your watchlist [w]" accesskey="w">Watch</a></li>
				</ul>
		</div>
	</div>
	<div class="portlet" id="p-personal">
		<h5>Personal tools</h5>
		<div class="pBody">
			<ul>
				<li id="pt-userpage"><a href="/wiki/User:Kanzure" title="My user page [.]" accesskey=".">Kanzure</a></li>
				<li id="pt-mytalk"><a href="/wiki/User_talk:Kanzure" title="My talk page [n]" accesskey="n">My talk</a></li>
				<li id="pt-preferences"><a href="/wiki/Special:Preferences" title="My preferences">My preferences</a></li>
				<li id="pt-watchlist"><a href="/wiki/Special:Watchlist" title="The list of pages that you are monitoring for changes [l]" accesskey="l">My watchlist</a></li>
				<li id="pt-mycontris"><a href="/wiki/Special:Contributions/Kanzure" title="List of my contributions [y]" accesskey="y">My contributions</a></li>
				<li id="pt-logout"><a href="/w/index.php?title=Special:Userlogout&amp;returnto=Proteomics" title="Log out">Log out</a></li>
			</ul>
		</div>
	</div>
	<div class="portlet" id="p-logo">
		<a style="background-image: url(/images/wiki-en.png);" href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z"></a>
	</div>
	<script type="text/javascript"> if (window.isMSIE55) fixalpha(); </script>
		<div class='portlet' id='p-navigation'>
		<h5>Navigation</h5>
		<div class='pBody'>
			<ul>
				<li id="n-mainpage"><a href="/wiki/Main_Page" title="Visit the main page [z]" accesskey="z">Main Page</a></li>
				<li id="n-Contents"><a href="/wiki/Portal:Contents" title="Guides to browsing Wikipedia">Contents</a></li>
				<li id="n-featuredcontent"><a href="/wiki/Portal:Featured_content">Featured content</a></li>
				<li id="n-currentevents"><a href="/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li>
				<li id="n-randompage"><a href="/wiki/Special:Random" title="Load a random article [x]" accesskey="x">Random article</a></li>
			</ul>
		</div>
	</div>
		<div class='portlet' id='p-interaction'>
		<h5>Interaction</h5>
		<div class='pBody'>
			<ul>
				<li id="n-About-Wikipedia"><a href="/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li>
				<li id="n-portal"><a href="/wiki/Wikipedia:Community_Portal" title="About the project, what you can do, where to find things">Community portal</a></li>
				<li id="n-recentchanges"><a href="/wiki/Special:Recentchanges" title="The list of recent changes in the wiki [r]" accesskey="r">Recent changes</a></li>
				<li id="n-contact"><a href="/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact Wikipedia</a></li>
				<li id="n-sitesupport"><a href="http://wikimediafoundation.org/wiki/Donate" title="Support us">Donate to Wikipedia</a></li>
				<li id="n-help"><a href="/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li>
			</ul>
		</div>
	</div>
		<div id="p-search" class="portlet">
		<h5><label for="searchInput">Search</label></h5>
		<div id="searchBody" class="pBody">
			<form action="/wiki/Special:Search" id="searchform"><div>
				<input id="searchInput" name="search" type="text" title="Search Wikipedia [f]" accesskey="f" value="" />
				<input type='submit' name="go" class="searchButton" id="searchGoButton"	value="Go" title="Go to a page with this exact name if one exists" />&nbsp;
				<input type='submit' name="fulltext" class="searchButton" id="mw-searchButton" value="Search" title="Search the wiki for this text" />
			</div></form>
		</div>
	</div>
	<div class="portlet" id="p-tb">
		<h5>Toolbox</h5>
		<div class="pBody">
			<ul>
				<li id="t-whatlinkshere"><a href="/wiki/Special:Whatlinkshere/Proteomics" title="List of all English Wikipedia pages that link here [j]" accesskey="j">What links here</a></li>
				<li id="t-recentchangeslinked"><a href="/wiki/Special:Recentchangeslinked/Proteomics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</a></li>
<li id="t-upload"><a href="/wiki/Wikipedia:Upload" title="Upload images or media files [u]" accesskey="u">Upload file</a></li>
<li id="t-specialpages"><a href="/wiki/Special:Specialpages" title="List of all special pages [q]" accesskey="q">Special pages</a></li>
				<li id="t-print"><a href="/w/index.php?title=Proteomics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</a></li>				<li id="t-permalink"><a href="/w/index.php?title=Proteomics&amp;oldid=186074133" title="Permanent link to this version of the page">Permanent link</a></li><li id="t-cite"><a href="/w/index.php?title=Special:Cite&amp;page=Proteomics&amp;id=186074133">Cite this page</a></li>			</ul>
		</div>
	</div>
	<div id="p-lang" class="portlet">
		<h5>Languages</h5>
		<div class="pBody">
			<ul>
				<li class="interwiki-bn"><a href="http://bn.wikipedia.org/wiki/%E0%A6%AA%E0%A7%8D%E0%A6%B0%E0%A7%8B%E0%A6%9F%E0%A6%BF%E0%A6%AF%E0%A6%BC%E0%A7%8B%E0%A6%AE%E0%A6%BF%E0%A6%95%E0%A7%8D%E2%80%8C%E0%A6%B8">বাংলা</a></li>
				<li class="interwiki-ca"><a href="http://ca.wikipedia.org/wiki/Prote%C3%B2mica">Català</a></li>
				<li class="interwiki-de"><a href="http://de.wikipedia.org/wiki/Proteomik">Deutsch</a></li>
				<li class="interwiki-es"><a href="http://es.wikipedia.org/wiki/Prote%C3%B3mica">Español</a></li>
				<li class="interwiki-fa"><a href="http://fa.wikipedia.org/wiki/%D9%BE%D8%B1%D9%88%D8%AA%D8%A6%D9%88%D9%85%DB%8C%DA%A9">فارسی</a></li>
				<li class="interwiki-fr"><a href="http://fr.wikipedia.org/wiki/Prot%C3%A9omique">Français</a></li>
				<li class="interwiki-ko"><a href="http://ko.wikipedia.org/wiki/%EB%8B%A8%EB%B0%B1%EC%B2%B4%ED%95%99">한국어</a></li>
				<li class="interwiki-id"><a href="http://id.wikipedia.org/wiki/Proteomika">Bahasa Indonesia</a></li>
				<li class="interwiki-lt"><a href="http://lt.wikipedia.org/wiki/Proteomika">Lietuvių</a></li>
				<li class="interwiki-ja"><a href="http://ja.wikipedia.org/wiki/%E3%83%97%E3%83%AD%E3%83%86%E3%82%AA%E3%83%BC%E3%83%A0%E8%A7%A3%E6%9E%90">日本語</a></li>
				<li class="interwiki-no"><a href="http://no.wikipedia.org/wiki/Proteomikk">‪Norsk (bokmål)‬</a></li>
				<li class="interwiki-pl"><a href="http://pl.wikipedia.org/wiki/Proteomika">Polski</a></li>
				<li class="interwiki-pt"><a href="http://pt.wikipedia.org/wiki/Prote%C3%B4mica">Português</a></li>
				<li class="interwiki-ru"><a href="http://ru.wikipedia.org/wiki/%D0%9F%D1%80%D0%BE%D1%82%D0%B5%D0%BE%D0%BC%D0%B8%D0%BA%D0%B0">Русский</a></li>
				<li class="interwiki-sv"><a href="http://sv.wikipedia.org/wiki/Proteomik">Svenska</a></li>
				<li class="interwiki-zh"><a href="http://zh.wikipedia.org/wiki/%E8%9B%8B%E7%99%BD%E8%B4%A8%E7%BB%84%E5%AD%A6">中文</a></li>
			</ul>
		</div>
	</div>
		</div><!-- end of the left (by default at least) column -->
			<div class="visualClear"></div>
			<div id="footer">
				<div id="f-poweredbyico"><a href="http://www.mediawiki.org/"><img src="/skins-1.5/common/images/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" /></a></div>
				<div id="f-copyrightico"><a href="http://wikimediafoundation.org/"><img src="/images/wikimedia-button.png" border="0" alt="Wikimedia Foundation"/></a></div>
			<ul id="f-list">
				<li id="lastmod"> This page was last modified 08:32, 22 January 2008.</li>
				<li id="copyright">All text is available under the terms of the <a class='internal' href="http://en.wikipedia.org/wiki/Wikipedia:Text_of_the_GNU_Free_Documentation_License" title="Wikipedia:Text of the GNU Free Documentation License">GNU Free Documentation License</a>. (See <b><a class='internal' href="http://en.wikipedia.org/wiki/Wikipedia:Copyrights" title="Wikipedia:Copyrights">Copyrights</a></b> for details.) <br /> Wikipedia&reg; is a registered trademark of the <a href="http://www.wikimediafoundation.org">Wikimedia Foundation, Inc</a>., a U.S. registered <a class='internal' href="http://en.wikipedia.org/wiki/501%28c%29#501.28c.29.283.29" title="501(c)(3)">501(c)(3)</a> <a href="http://wikimediafoundation.org/wiki/Deductibility_of_donations">tax-deductible</a> <a class='internal' href="http://en.wikipedia.org/wiki/Non-profit_organization" title="Non-profit organization">nonprofit</a> <a href="http://en.wikipedia.org/wiki/Charitable_organization" title="Charitable organization">charity</a>.<br /></li>
				<li id="privacy"><a href="http://wikimediafoundation.org/wiki/Privacy_policy" title="wikimedia:Privacy policy">Privacy policy</a></li>
				<li id="about"><a href="/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
				<li id="disclaimer"><a href="/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
			</ul>
		</div>
		
	
		<script type="text/javascript">if (window.runOnloadHook) runOnloadHook();</script>
</div>
<!-- Served by srv173 in 0.043 secs. --></body></html>
